-- MySQL dump 10.13  Distrib 8.0.25, for macos11.3 (x86_64)
--
-- Host: 127.0.0.1    Database: gutmgene
-- ------------------------------------------------------
-- Server version	8.0.25

/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!50503 SET NAMES utf8mb4 */;
/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
/*!40103 SET TIME_ZONE='+00:00' */;
/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;

create database if not exists gutmgene;
use gutmgene;
--
-- Table structure for table `details`
--

DROP TABLE IF EXISTS `details`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `details` (
  `ncbi_gene_id` text COLLATE utf8mb4_bin,
  `kegg_id` text COLLATE utf8mb4_bin,
  `kegg_symbol` longtext COLLATE utf8mb4_bin,
  `kegg_name` longtext COLLATE utf8mb4_bin,
  `kegg_orthology` longtext COLLATE utf8mb4_bin,
  `kegg_organism` longtext COLLATE utf8mb4_bin,
  `kegg_pathway` longtext COLLATE utf8mb4_bin,
  `kegg_network` longtext COLLATE utf8mb4_bin,
  `kegg_network_element` longtext COLLATE utf8mb4_bin,
  `kegg_brite` longtext COLLATE utf8mb4_bin,
  `kegg_motif` longtext COLLATE utf8mb4_bin
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `details`
--

LOCK TABLES `details` WRITE;
/*!40000 ALTER TABLE `details` DISABLE KEYS */;
INSERT INTO `details` VALUES ('6372','hsa:6372','[CXCL6, CKA-3, GCP-2, GCP2, SCYB6]','[(RefSeq) C-X-C motif chemokine ligand 6]','[C-X-C motif chemokine 5/6]','[Homo sapiens (human)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, Chemokine signaling pathway, IL-17 signaling pathway, TNF signaling pathway, Pertussis, Rheumatoid arthritis]','[nt06150  Cytokine-cytokine receptor interaction (viruses)]','[N01233  CXCL1/5/6/8 to CXCR1 signaling, N01235  CXCL6 to KSHV vGPCR signaling, N01238  CXCL1/2/3/5/6/7/8 to CXCR2 signaling]','04668 TNF signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04657 IL-17 signaling pathway; 04062 Chemokine signaling pathway; 05133 Pertussis; 05323 Rheumatoid arthritis; 04052 Cytokines and growth factors [BR:hsa04052]; Chemokines; ','[Pfam: IL8]'),('6364','hsa:6364','[CCL20, CKb4, Exodus, LARC, MIP-3-alpha, MIP-3a, MIP3A, SCYA20, ST38]','[(RefSeq) C-C motif chemokine ligand 20]','[C-C motif chemokine 20]','[Homo sapiens (human)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, Chemokine signaling pathway, IL-17 signaling pathway, TNF signaling pathway, Rheumatoid arthritis]','[nt06150  Cytokine-cytokine receptor interaction (viruses)]','[N01213  CCL20 to CCR6 signaling]','04668 TNF signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04657 IL-17 signaling pathway; 04062 Chemokine signaling pathway; 05323 Rheumatoid arthritis; 04052 Cytokines and growth factors [BR:hsa04052]; Chemokines; ','[Pfam: IL8]'),('11199','hsa:11199','[ANXA10, ANX14]','[(RefSeq) annexin A10]','[annexin A10]','[Homo sapiens (human)]',NULL,'[]','[]','04147 Exosome [BR:hsa04147]; Exosomal proteins of ovarian cancer cells; ','[Pfam: Annexin DUF758]'),('50506','hsa:50506','[DUOX2, LNOX2, NOXEF2, P138-TOX, TDH6, THOX2]','[(RefSeq) dual oxidase 2]','[dual oxidase [EC:1.6.3.1 1.11.1.-]]','[Homo sapiens (human)]','[Thyroid hormone synthesis]','[nt06322  TRH-TSH-TH signaling]','[N00793  TSH-DUOX2-TG signaling pathway, N00794  Mutation-inactivated DUOX2 to TSH-DUOX2-TG signaling pathway, N00795  DUOX2-generated H2O2 production, N00796  Mutation-inactivated DUOXA2 to DUOX2-generated H2O2 production]','04918 Thyroid hormone synthesis; 04131 Membrane trafficking [BR:hsa04131]; 1.6.3.1  NAD(P)H oxidase (H2O2-forming); 1.11.1.-; NADPH oxidases; ','[Pfam: An_peroxidase NAD_binding_6 Ferric_reduct FAD_binding_8 EF-hand_1 EF-hand_7 EF-hand_6 EF-hand_8 EF-hand_5 NAD_binding_1 Rv0078B]'),('1956','hsa:1956','[EGFR, ERBB, ERBB1, ERRP, HER1, NISBD2, PIG61, mENA]','[(RefSeq) epidermal growth factor receptor]','[epidermal growth factor receptor [EC:2.7.10.1]]','[Homo sapiens (human)]','[EGFR tyrosine kinase inhibitor resistance, Endocrine resistance, MAPK signaling pathway, ErbB signaling pathway, Ras signaling pathway, Rap1 signaling pathway, Calcium signaling pathway, HIF-1 signaling pathway, FoxO signaling pathway, Phospholipase D signaling pathway, Endocytosis, PI3K-Akt signaling pathway, Focal adhesion, Adherens junction, Gap junction, JAK-STAT signaling pathway, Regulation of actin cytoskeleton, GnRH signaling pathway, Estrogen signaling pathway, Oxytocin signaling pathway, Relaxin signaling pathway, Parathyroid hormone synthesis, secretion and action, Cushing syndrome, Epithelial cell signaling in Helicobacter pylori infection, Shigellosis, Hepatitis C, Human cytomegalovirus infection, Human papillomavirus infection, Coronavirus disease - COVID-19, Pathways in cancer, Proteoglycans in cancer, MicroRNAs in cancer, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species, Colorectal cancer, Pancreatic cancer, Endometrial cancer, Glioma, Prostate cancer, Melanoma, Bladder cancer, Non-small cell lung cancer, Breast cancer, Hepatocellular carcinoma, Gastric cancer, Central carbon metabolism in cancer, Choline metabolism in cancer, PD-L1 expression and PD-1 checkpoint pathway in cancer]','[c(\"nt06110  MAPK signaling (viruses and bacteria)\", \"nt06114  PI3K signaling (viruses)\", \"nt06120  Calcium signaling (viruses and bacteria)\", \"nt06135  Cytoskeletal regulation (viruses and bacteria)\", \"nt06160  Human T-cell leukemia virus 1 (HTLV-1)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06163  Hepatitis C virus (HCV)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06165  Epstein-Barr virus (EBV)\", \"nt06166  Human papillomavirus (HPV)\", \"nt06167  Human cytomegalovirus (HCMV)\", \"nt06170  Influenza A virus (IAV)\", \n\"nt06210  ERK signaling\", \"nt06213  Other RAS signaling\", \"nt06214  PI3K signaling\", \"nt06219  JAK-STAT signaling\", \"nt06220  Calcium signaling\", \"nt06260  Colorectal cancer\", \"nt06261  Gastric cancer\", \"nt06262  Pancreatic cancer\", \"nt06263  Hepatocellular carcinoma\", \"nt06265  Bladder cancer\", \"nt06266  Non-small cell lung cancer\", \"nt06268  Melanoma\", \"nt06270  Breast cancer\", \"nt06271  Endometrial cancer\", \"nt06273  Glioma\", \"nt06274  Thyroid cancer\", \"nt06276  Chronic myeloid leukemia\", \"nt06310  CRH-ACTH-cortisol signaling\", \n\"nt06360  Cushing syndrome\")]','[N00001  EGF-EGFR-RAS-ERK signaling pathway, N00006  Amplified EGFR to RAS-ERK signaling pathway, N00014  Mutation-activated EGFR to RAS-ERK signaling pathway, N00021  EGF-ERBB2-RAS-ERK signaling pathway, N00022  ERBB2-overexpression to RAS-ERK signaling pathway, N00023  EGF-EGFR-PLCG-ERK signaling pathway, N00024  Mutation-activated EGFR to PLCG-ERK signaling pathway, N00026  EGF-EGFR-PLCG-CAMK signaling pathway, N00027  Amplified EGFR to PLCG-CAMK signaling pathway, N00030  EGF-EGFR-RAS-PI3K signaling pathway, N00033  EGF-EGFR-PI3K signaling pathway, N00034  ERBB2-overexpression to PI3K signaling pathway, N00035  Amplified EGFR to PI3K signaling pathway, N00036  Mutation-activated EGFR to PI3K signaling pathway, N00041  EGFR-overexpression to RAS-ERK signaling pathway, N00042  EGFR-overexpression to PI3K signaling pathway, N00094  EGF-Jak-STAT signaling pathway, N00095  ERBB2-overexpression to EGF-Jak-STAT signaling pathway, N00096  EGF-EGFR-RAS-RASSF1 signaling pathway, N00103  EGF-EGFR-RAS-RalGDS signaling pathway, N00147  EGF-EGFR-PLCG-calcineurin signaling pathway, N00227  TGFA-EGFR-PLCG-PKC signaling pathway, N00228  TGFA-overexpression to PLCG-PKC signaling pathway, N00229  TGFA-EGFR-RAS-ERK signaling pathway, N00230  TGFA-overexpression to RAS-ERK signaling pathway, N00231  TGFA-EGFR-PI3K signaling pathway, N00232  TGFA-overexpression to PI3K signaling pathway, N00252  Amplified ERBB2 to RAS-ERK signaling pathway, N00253  Amplified ERBB2 to PI3K signaling pathway, N00276  EGF-overexpression to RAS-ERK signaling pathway, N00277  EREG-EGFR-RAS-ERK signaling pathway, N00278  EREG-overexpression to RAS-ERK signaling pathway, N00279  AREG-EGFR-RAS-ERK signaling pathway, N00280  AREG-overexpression to RAS-ERK signaling pathway, N00281  EGF-overexpression to PI3K signaling pathway, N00282  EREG-EGFR-PI3K signaling pathway, N00283  EREG-overexpression to PI3K signaling pathway, N00284  AREG-EGFR-PI3K signaling pathway, N00285  AREG-overexpression to PI3K signaling pathway, N00318  EGFR-ERK-ACTH signaling pathway, N00319  Mutation-activated USP8 to EGFR-ERK-ACTH signaling pathway, N00366  HPV E5 to EGFR-PI3K signaling pathway, N00367  HPV E5 to EGFR-RAS-ERK signaling pathway, N00390  EGF-EGFR-PI3K-NFKB signaling pathway, N00391  HCMV gB to EGFR-PI3K-NFKB signaling pathway, N00392  HCMV gB to EGFR-RAS-ERK signaling pathway, N00513  Mutation-activated EGFR to RAS-ERK signaling pathway, N00514  Mutation-activated EGFR to PI3K signaling pathway, N00542  EGF-EGFR-RAS-JNK signaling pathway, N01078  EGF-EGFR-Actin signaling pathway, N01354  BPA to RAS-ERK signaling pathway, N01355  Arsenic to PI3K signaling pathway, N10005  First/second-generation tyrosine kinase inhibitor to EGFR mutation, N10006  Third-generation tyrosine kinase inhibitor to EGFR mutation]','04010 MAPK signaling pathway; 04012 ErbB signaling pathway; 04014 Ras signaling pathway; 04015 Rap1 signaling pathway; 04630 JAK-STAT signaling pathway; 04066 HIF-1 signaling pathway; 04068 FoxO signaling pathway; 04020 Calcium signaling pathway; 04072 Phospholipase D signaling pathway; 04151 PI3K-Akt signaling pathway; 04144 Endocytosis; 04510 Focal adhesion; 04520 Adherens junction; 04540 Gap junction; 04810 Regulation of actin cytoskeleton; 04912 GnRH signaling pathway; 04915 Estrogen signaling pathway; 04921 Oxytocin signaling pathway; 04926 Relaxin signaling pathway; 04928 Parathyroid hormone synthesis, secretion and action; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05205 Proteoglycans in cancer; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 05230 Central carbon metabolism in cancer; 05231 Choline metabolism in cancer; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05210 Colorectal cancer; 05212 Pancreatic cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05214 Glioma; 05218 Melanoma; 05219 Bladder cancer; 05215 Prostate cancer; 05213 Endometrial cancer; 05224 Breast cancer; 05223 Non-small cell lung cancer; 05160 Hepatitis C; 05171 Coronavirus disease - COVID-19; 05163 Human cytomegalovirus infection; 05165 Human papillomavirus infection; 05120 Epithelial cell signaling in Helicobacter pylori infection; 05131 Shigellosis; 04934 Cushing syndrome; 01521 EGFR tyrosine kinase inhibitor resistance; 01522 Endocrine resistance; 01001 Protein kinases [BR:hsa01001]; 04147 Exosome [BR:hsa04147]; 2.7.10.1  receptor protein-tyrosine kinase; EGFR family [OT]; Exosomal proteins of colorectal cancer cells; Exosomal proteins of pancreatic cancer cells; Exosomal proteins of bladder cancer cells; ','[Pfam: PK_Tyr_Ser-Thr Recep_L_domain GF_recep_IV Pkinase Furin-like Kinase-like APH GAPT ABC1 Kdo]'),('2865','hsa:2865','[FFAR3, FFA3R, GPR41, GPR42]','[(RefSeq) free fatty acid receptor 3]','[free fatty acid receptor 3]','[Homo sapiens (human)]',NULL,'[]','[]','04030 G protein-coupled receptors [BR:hsa04030]; Free fatty acid; ','[Pfam: 7tm_1 7TM_GPCR_Srx 7TM_GPCR_Srsx DUF6332]'),('2867','hsa:2867','[FFAR2, FFA2R, GPR43]','[(RefSeq) free fatty acid receptor 2]','[free fatty acid receptor 2]','[Homo sapiens (human)]','[cAMP signaling pathway]','[]','[]','04024 cAMP signaling pathway; 04030 G protein-coupled receptors [BR:hsa04030]; Free fatty acid; ','[Pfam: 7tm_1 7TM_GPCR_Srv TAS2R 7TM_GPCR_Srsx 7TM_GPCR_Srx]'),('6347','hsa:6347','[CCL2, GDCF-2, HC11, HSMCR30, MCAF, MCP-1, MCP1, SCYA2, SMC-CF]','[(RefSeq) C-C motif chemokine ligand 2]','[C-C motif chemokine 2]','[Homo sapiens (human)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, Chemokine signaling pathway, NOD-like receptor signaling pathway, IL-17 signaling pathway, TNF signaling pathway, AGE-RAGE signaling pathway in diabetic complications, Yersinia infection, Chagas disease, Malaria, Human cytomegalovirus infection, Influenza A, Herpes simplex virus 1 infection, Coronavirus disease - COVID-19, Rheumatoid arthritis, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[c(\"nt06123  TNF signaling (viruses and bacteria)\", \"nt06124  Chemokine signaling (viruses)\", \"nt06150  Cytokine-cytokine receptor interaction (viruses)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06167  Human cytomegalovirus (HCMV)\", \"nt06168  Herpes simplex virus 1 (HSV-1)\", \"nt06171  Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)\", \"nt06224  CXCR signaling\")]','[N00153  CCR/CXCR-GNB/G-PI3K-RAC signaling pathway, N00399  CCR2-GNB/G-PI3K-NFKB signaling pathway, N00406  HCMV US28 to GNA12/13-Rho signaling pathway, N00560  HSV gD to HVEM-NFKB signaling pathway, N01173  CCL2/7/8/13/27 to CCR2 signaling, N01174  VAV vCCI to CCR signaling, N01175  CCL2 to HCMV US28 signaling, N01176  CCL2 to HHV6 U12 signaling, N01177  CCL2 to HHV6 U51 signaling, N01307  SARS-CoV-2 S to AngII-AT1R-NOX2 signaling pathway]','04668 TNF signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04621 NOD-like receptor signaling pathway; 04657 IL-17 signaling pathway; 04062 Chemokine signaling pathway; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05168 Herpes simplex virus 1 infection; 05163 Human cytomegalovirus infection; 05135 Yersinia infection; 05144 Malaria; 05142 Chagas disease; 05323 Rheumatoid arthritis; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 04933 AGE-RAGE signaling pathway in diabetic complications; 04052 Cytokines and growth factors [BR:hsa04052]; 00536 Glycosaminoglycan binding proteins [BR:hsa00536]; Chemokines; Chemokines; ','[Pfam: IL8]'),('3586','hsa:3586','[IL10, CSIF, GVHDS, IL-10, IL10A, TGIF]','[(RefSeq) interleukin 10]','[interleukin 10]','[Homo sapiens (human)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, FoxO signaling pathway, C-type lectin receptor signaling pathway, JAK-STAT signaling pathway, T cell receptor signaling pathway, Intestinal immune network for IgA production, Pertussis, Yersinia infection, Leishmaniasis, Chagas disease, African trypanosomiasis, Malaria, Toxoplasmosis, Amoebiasis, Staphylococcus aureus infection, Tuberculosis, Asthma, Autoimmune thyroid disease, Inflammatory bowel disease, Systemic lupus erythematosus, Allograft rejection]','[c(\"nt06119  JAK-STAT signaling (viruses)\", \"nt06150  Cytokine-cytokine receptor interaction (viruses)\", \"nt06167  Human cytomegalovirus (HCMV)\")]','[N00415  IL10-Jak-STAT signaling pathway]','04630 JAK-STAT signaling pathway; 04068 FoxO signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04625 C-type lectin receptor signaling pathway; 04660 T cell receptor signaling pathway; 04672 Intestinal immune network for IgA production; 05135 Yersinia infection; 05133 Pertussis; 05150 Staphylococcus aureus infection; 05152 Tuberculosis; 05146 Amoebiasis; 05144 Malaria; 05145 Toxoplasmosis; 05140 Leishmaniasis; 05142 Chagas disease; 05143 African trypanosomiasis; 05310 Asthma; 05322 Systemic lupus erythematosus; 05320 Autoimmune thyroid disease; 05321 Inflammatory bowel disease; 05330 Allograft rejection; 04052 Cytokines and growth factors [BR:hsa04052]; 00536 Glycosaminoglycan binding proteins [BR:hsa00536]; Interleukins; Cytokines; ','[Pfam: IL10 IL22 CLZ]'),('3593','hsa:3593','[IL12B, CLMF, CLMF2, IL-12B, IMD28, IMD29, NKSF, NKSF2]','[(RefSeq) interleukin 12B]','[interleukin 12B]','[Homo sapiens (human)]','[Cytokine-cytokine receptor interaction, Toll-like receptor signaling pathway, RIG-I-like receptor signaling pathway, C-type lectin receptor signaling pathway, JAK-STAT signaling pathway, Th1 and Th2 cell differentiation, Alcoholic liver disease, Type I diabetes mellitus, Pertussis, Legionellosis, Leishmaniasis, Chagas disease, African trypanosomiasis, Toxoplasmosis, Amoebiasis, Tuberculosis, Measles, Influenza A, Herpes simplex virus 1 infection, Coronavirus disease - COVID-19, Pathways in cancer, Proteoglycans in cancer, Inflammatory bowel disease, Allograft rejection, Lipid and atherosclerosis]','[c(\"nt06119  JAK-STAT signaling (viruses)\", \"nt06121  TLR signaling (viruses and bacteria)\", \"nt06160  Human T-cell leukemia virus 1 (HTLV-1)\", \"nt06161  Human immunodeficiency virus type 1 (HIV-1)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06165  Epstein-Barr virus (EBV)\", \"nt06168  Herpes simplex virus 1 (HSV-1)\", \"nt06169  Measles virus (MV)\", \"nt06171  Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)\", \"nt06219  JAK-STAT signaling\", \n\"nt06263  Hepatocellular carcinoma\", \"nt06266  Non-small cell lung cancer\", \"nt06276  Chronic myeloid leukemia\")]','[N00053  Cytokine-Jak-STAT signaling pathway, N00435  TLR2/4-NFKB signaling pathway, N01307  SARS-CoV-2 S to AngII-AT1R-NOX2 signaling pathway]','04630 JAK-STAT signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04620 Toll-like receptor signaling pathway; 04622 RIG-I-like receptor signaling pathway; 04625 C-type lectin receptor signaling pathway; 04658 Th1 and Th2 cell differentiation; 05200 Pathways in cancer; 05205 Proteoglycans in cancer; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05133 Pertussis; 05134 Legionellosis; 05152 Tuberculosis; 05146 Amoebiasis; 05145 Toxoplasmosis; 05140 Leishmaniasis; 05142 Chagas disease; 05143 African trypanosomiasis; 05321 Inflammatory bowel disease; 05330 Allograft rejection; 05417 Lipid and atherosclerosis; 04940 Type I diabetes mellitus; 04936 Alcoholic liver disease; 04052 Cytokines and growth factors [BR:hsa04052]; 00536 Glycosaminoglycan binding proteins [BR:hsa00536]; Interleukins; Cytokines; ','[Pfam: IL12p40_C Ig_3 Ig_5 Ig_2 fn3 Ig_4 I-set]'),('3592','hsa:3592','[IL12A, CLMF, IL-12A, NFSK, NKSF1, P35]','[(RefSeq) interleukin 12A]','[interleukin 12A]','[Homo sapiens (human)]','[Cytokine-cytokine receptor interaction, Toll-like receptor signaling pathway, RIG-I-like receptor signaling pathway, C-type lectin receptor signaling pathway, JAK-STAT signaling pathway, Th1 and Th2 cell differentiation, Alcoholic liver disease, Type I diabetes mellitus, Pertussis, Legionellosis, Leishmaniasis, Chagas disease, African trypanosomiasis, Malaria, Toxoplasmosis, Amoebiasis, Tuberculosis, Measles, Influenza A, Herpes simplex virus 1 infection, Coronavirus disease - COVID-19, Pathways in cancer, Inflammatory bowel disease, Allograft rejection, Lipid and atherosclerosis]','[c(\"nt06119  JAK-STAT signaling (viruses)\", \"nt06121  TLR signaling (viruses and bacteria)\", \"nt06160  Human T-cell leukemia virus 1 (HTLV-1)\", \"nt06161  Human immunodeficiency virus type 1 (HIV-1)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06165  Epstein-Barr virus (EBV)\", \"nt06168  Herpes simplex virus 1 (HSV-1)\", \"nt06169  Measles virus (MV)\", \"nt06171  Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)\", \"nt06219  JAK-STAT signaling\", \n\"nt06263  Hepatocellular carcinoma\", \"nt06266  Non-small cell lung cancer\", \"nt06276  Chronic myeloid leukemia\")]','[N00053  Cytokine-Jak-STAT signaling pathway, N00435  TLR2/4-NFKB signaling pathway, N01307  SARS-CoV-2 S to AngII-AT1R-NOX2 signaling pathway]','04630 JAK-STAT signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04620 Toll-like receptor signaling pathway; 04622 RIG-I-like receptor signaling pathway; 04625 C-type lectin receptor signaling pathway; 04658 Th1 and Th2 cell differentiation; 05200 Pathways in cancer; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05133 Pertussis; 05134 Legionellosis; 05152 Tuberculosis; 05146 Amoebiasis; 05144 Malaria; 05145 Toxoplasmosis; 05140 Leishmaniasis; 05142 Chagas disease; 05143 African trypanosomiasis; 05321 Inflammatory bowel disease; 05330 Allograft rejection; 05417 Lipid and atherosclerosis; 04940 Type I diabetes mellitus; 04936 Alcoholic liver disease; 04052 Cytokines and growth factors [BR:hsa04052]; 00536 Glycosaminoglycan binding proteins [BR:hsa00536]; Interleukins; Cytokines; ','[Pfam: IL12]'),('1235','hsa:1235','[CCR6, BN-1, C-C_CKR-6, CC-CKR-6, CCR-6, CD196, CKR-L3, CKRL3, CMKBR6, DCR2, DRY6, GPR29, GPRCY4, STRL22]','[(RefSeq) C-C motif chemokine receptor 6]','[C-C chemokine receptor type 6]','[Homo sapiens (human)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, Chemokine signaling pathway]','[nt06150  Cytokine-cytokine receptor interaction (viruses)]','[N01213  CCL20 to CCR6 signaling, N01214  HHV6A vCCL4a to CCR6 signaling]','04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04062 Chemokine signaling pathway; 03037 Cilium and associated proteins [BR:hsa03037]; 04030 G protein-coupled receptors [BR:hsa04030]; 04050 Cytokine receptors [BR:hsa04050]; 04090 CD molecules [BR:hsa04090]; Sperm flagellum proteins; CC Chemokine; CC Chemokine; Proteins; ','[Pfam: 7tm_1]'),('10663','hsa:10663','[CXCR6, BONZO, CD186, CDw186, STRL33, TYMSTR]','[(RefSeq) C-X-C motif chemokine receptor 6]','[C-X-C chemokine receptor type 6]','[Homo sapiens (human)]','[Cytokine-cytokine receptor interaction, Chemokine signaling pathway]','[]','[]','04060 Cytokine-cytokine receptor interaction; 04062 Chemokine signaling pathway; 04030 G protein-coupled receptors [BR:hsa04030]; 04050 Cytokine receptors [BR:hsa04050]; 04090 CD molecules [BR:hsa04090]; CXC Chemokine; CXC Chemokine; Proteins; ','[Pfam: 7tm_1 7TM_GPCR_Srsx Peroxin-13_N]'),('2833','hsa:2833','[CXCR3, CD182, CD183, CKR-L2, CMKAR3, GPR9, IP10-R, Mig-R, MigR]','[(RefSeq) C-X-C motif chemokine receptor 3]','[C-X-C chemokine receptor type 3]','[Homo sapiens (human)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, Chemokine signaling pathway]','[nt06150  Cytokine-cytokine receptor interaction (viruses)]','[N01241  CXCL4/9/10/11/13 to CXCR3/5 signaling]','04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04062 Chemokine signaling pathway; 04030 G protein-coupled receptors [BR:hsa04030]; 04050 Cytokine receptors [BR:hsa04050]; 04090 CD molecules [BR:hsa04090]; CXC Chemokine; CXC Chemokine; Proteins; ','[Pfam: 7tm_1 Ctr 7TM_GPCR_Srsx]'),('9414','hsa:9414','[TJP2, C9DUPq21.11, DFNA51, DUP9q21.11, FHCA1, PFIC4, X104, ZO2]','[(RefSeq) tight junction protein 2]','[tight junction protein 2]','[Homo sapiens (human)]','[Tight junction, Vibrio cholerae infection]','[]','[]','04530 Tight junction; 05110 Vibrio cholerae infection; ','[Pfam: PDZ PDZ_6 PDZ_2 Guanylate_kin Peptidase_M50 SH3_2 SH3_1]'),('5590','hsa:5590','[PRKCZ, PKC-ZETA, PKC2]','[(RefSeq) protein kinase C zeta]','[atypical protein kinase C zeta type [EC:2.7.11.13]]','[Homo sapiens (human)]','[Rap1 signaling pathway, Chemokine signaling pathway, Sphingolipid signaling pathway, Endocytosis, Axon guidance, Hippo signaling pathway, Tight junction, Platelet activation, Insulin signaling pathway, Relaxin signaling pathway, Type II diabetes mellitus, Insulin resistance, AGE-RAGE signaling pathway in diabetic complications, Human papillomavirus infection, Diabetic cardiomyopathy, Fluid shear stress and atherosclerosis]','[]','[]','04015 Rap1 signaling pathway; 04390 Hippo signaling pathway; 04071 Sphingolipid signaling pathway; 04144 Endocytosis; 04530 Tight junction; 04611 Platelet activation; 04062 Chemokine signaling pathway; 04910 Insulin signaling pathway; 04926 Relaxin signaling pathway; 04360 Axon guidance; 05165 Human papillomavirus infection; 05418 Fluid shear stress and atherosclerosis; 05415 Diabetic cardiomyopathy; 04930 Type II diabetes mellitus; 04931 Insulin resistance; 04933 AGE-RAGE signaling pathway in diabetic complications; 01001 Protein kinases [BR:hsa01001]; 2.7.11.13  protein kinase C; PKC family [OT]; ','[Pfam: Pkinase PK_Tyr_Ser-Thr PB1 C1_1 Pkinase_C Kinase-like ABC1 C1_2 APH Kdo]'),('3553','hsa:3553','[IL1B, IL-1, IL1-BETA, IL1F2, IL1beta]','[(RefSeq) interleukin 1 beta]','[interleukin 1 beta]','[Homo sapiens (human)]','[Antifolate resistance, MAPK signaling pathway, Cytokine-cytokine receptor interaction, NF-kappa B signaling pathway, Necroptosis, Osteoclast differentiation, Toll-like receptor signaling pathway, NOD-like receptor signaling pathway, Cytosolic DNA-sensing pathway, C-type lectin receptor signaling pathway, Hematopoietic cell lineage, IL-17 signaling pathway, Th17 cell differentiation, TNF signaling pathway, Inflammatory mediator regulation of TRP channels, Non-alcoholic fatty liver disease, AGE-RAGE signaling pathway in diabetic complications, Alcoholic liver disease, Type I diabetes mellitus, Alzheimer disease, Prion disease, Pathways of neurodegeneration - multiple diseases, Pathogenic Escherichia coli infection, Shigellosis, Salmonella infection, Pertussis, Legionellosis, Yersinia infection, Leishmaniasis, Chagas disease, African trypanosomiasis, Malaria, Amoebiasis, Tuberculosis, Measles, Human cytomegalovirus infection, Influenza A, Herpes simplex virus 1 infection, Coronavirus disease - COVID-19, Inflammatory bowel disease, Rheumatoid arthritis, Graft-versus-host disease, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[c(\"nt06110  MAPK signaling (viruses and bacteria)\", \"nt06139  NLR signaling (viruses and bacteria)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06170  Influenza A virus (IAV)\", \"nt06171  Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)\", \"nt06211  Other MAPK signaling\")]','[N00186  IL1-IL1R-p38 signaling pathway, N00188  IL1-IL1R-JNK signaling pathway, N00742  NLRP3 Inflammasome signaling pathway, N00867  NLRC4 Inflammasome signaling pathway, N00868  Pyrin Inflammasome signaling pathway, N00934  Non-canonical inflammasome signaling pathway, N00948  Shigella IpaH7.8 to NLRP3 Inflammasome signaling pathway, N00949  Shigella IpaB to NLRC4 Inflammasome signaling pathway, N01126  Salmonella SipB to Inflammasome signaling pathway, N01127  Salmonella SopE to Inflammasome signaling pathway, N01307  SARS-CoV-2 S to AngII-AT1R-NOX2 signaling pathway]','04010 MAPK signaling pathway; 04064 NF-kappa B signaling pathway; 04668 TNF signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04217 Necroptosis; 04640 Hematopoietic cell lineage; 04620 Toll-like receptor signaling pathway; 04621 NOD-like receptor signaling pathway; 04623 Cytosolic DNA-sensing pathway; 04625 C-type lectin receptor signaling pathway; 04659 Th17 cell differentiation; 04657 IL-17 signaling pathway; 04750 Inflammatory mediator regulation of TRP channels; 04380 Osteoclast differentiation; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05163 Human cytomegalovirus infection; 05130 Pathogenic Escherichia coli infection; 05132 Salmonella infection; 05131 Shigellosis; 05135 Yersinia infection; 05133 Pertussis; 05134 Legionellosis; 05152 Tuberculosis; 05146 Amoebiasis; 05144 Malaria; 05140 Leishmaniasis; 05142 Chagas disease; 05143 African trypanosomiasis; 05323 Rheumatoid arthritis; 05321 Inflammatory bowel disease; 05332 Graft-versus-host disease; 05010 Alzheimer disease; 05020 Prion disease; 05022 Pathways of neurodegeneration - multiple diseases; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 04940 Type I diabetes mellitus; 04936 Alcoholic liver disease; 04932 Non-alcoholic fatty liver disease; 04933 AGE-RAGE signaling pathway in diabetic complications; 01523 Antifolate resistance; 04052 Cytokines and growth factors [BR:hsa04052]; 00536 Glycosaminoglycan binding proteins [BR:hsa00536]; Interleukins; Cytokines; ','[Pfam: IL1 IL1_propep]'),('7124','hsa:7124','[TNF, DIF, TNF-alpha, TNFA, TNFSF2, TNLG1F]','[(RefSeq) tumor necrosis factor]','[tumor necrosis factor superfamily, member 2]','[Homo sapiens (human)]','[Antifolate resistance, MAPK signaling pathway, Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, NF-kappa B signaling pathway, Sphingolipid signaling pathway, mTOR signaling pathway, Apoptosis, Necroptosis, TGF-beta signaling pathway, Osteoclast differentiation, Antigen processing and presentation, Toll-like receptor signaling pathway, NOD-like receptor signaling pathway, RIG-I-like receptor signaling pathway, C-type lectin receptor signaling pathway, Hematopoietic cell lineage, Natural killer cell mediated cytotoxicity, IL-17 signaling pathway, T cell receptor signaling pathway, Fc epsilon RI signaling pathway, TNF signaling pathway, Adipocytokine signaling pathway, Type II diabetes mellitus, Insulin resistance, Non-alcoholic fatty liver disease, AGE-RAGE signaling pathway in diabetic complications, Alcoholic liver disease, Type I diabetes mellitus, Alzheimer disease, Amyotrophic lateral sclerosis, Prion disease, Pathways of neurodegeneration - multiple diseases, Pathogenic Escherichia coli infection, Shigellosis, Salmonella infection, Pertussis, Legionellosis, Yersinia infection, Leishmaniasis, Chagas disease, African trypanosomiasis, Malaria, Toxoplasmosis, Amoebiasis, Tuberculosis, Hepatitis C, Hepatitis B, Human cytomegalovirus infection, Influenza A, Human papillomavirus infection, Human T-cell leukemia virus 1 infection, Herpes simplex virus 1 infection, Epstein-Barr virus infection, Human immunodeficiency virus 1 infection, Coronavirus disease - COVID-19, Proteoglycans in cancer, Asthma, Inflammatory bowel disease, Systemic lupus erythematosus, Rheumatoid arthritis, Allograft rejection, Graft-versus-host disease, Hypertrophic cardiomyopathy, Dilated cardiomyopathy, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[c(\"nt06121  TLR signaling (viruses and bacteria)\", \"nt06123  TNF signaling (viruses and bacteria)\", \"nt06131  Apoptosis (viruses and bacteria)\", \"nt06150  Cytokine-cytokine receptor interaction (viruses)\", \"nt06160  Human T-cell leukemia virus 1 (HTLV-1)\", \"nt06161  Human immunodeficiency virus type 1 (HIV-1)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06163  Hepatitis C virus (HCV)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06165  Epstein-Barr virus (EBV)\", \"nt06166  Human papillomavirus (HPV)\", \n\"nt06167  Human cytomegalovirus (HCMV)\", \"nt06168  Herpes simplex virus 1 (HSV-1)\", \"nt06169  Measles virus (MV)\", \"nt06171  Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)\", \"nt06223  TNF signaling\", \"nt06231  Apoptosis\", \"nt06263  Hepatocellular carcinoma\", \"nt06414  Apoptosis\", \"nt06416  TNF signaling\", \"nt06417  AGE-RAGE signaling\", \"nt06460  Alzheimer disease\", \"nt06461  Huntington disease\")]','[N00145  Extrinsic apoptotic pathway, N00151  TNF-NFKB signaling pathway, N00374  TNF-IRF1 signaling pathway, N00435  TLR2/4-NFKB signaling pathway, N00436  HIV Tat to TLR2/4-NFKB signaling pathway, N00444  TNF-p38 signaling pathway, N00446  TNF-JNK signaling pathway, N00994  AGE-RAGE signaling pathway, N00996  Mutation-caused aberrant Abeta to AGE-RAGE signaling pathway, N01275  TNF to TNFR signaling, N01276  VAV CrmB to TNFR signaling, N01307  SARS-CoV-2 S to AngII-AT1R-NOX2 signaling pathway]','04010 MAPK signaling pathway; 04350 TGF-beta signaling pathway; 04064 NF-kappa B signaling pathway; 04668 TNF signaling pathway; 04071 Sphingolipid signaling pathway; 04150 mTOR signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04210 Apoptosis; 04217 Necroptosis; 04640 Hematopoietic cell lineage; 04620 Toll-like receptor signaling pathway; 04621 NOD-like receptor signaling pathway; 04622 RIG-I-like receptor signaling pathway; 04625 C-type lectin receptor signaling pathway; 04650 Natural killer cell mediated cytotoxicity; 04612 Antigen processing and presentation; 04660 T cell receptor signaling pathway; 04657 IL-17 signaling pathway; 04664 Fc epsilon RI signaling pathway; 04920 Adipocytokine signaling pathway; 04380 Osteoclast differentiation; 05205 Proteoglycans in cancer; 05166 Human T-cell leukemia virus 1 infection; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05160 Hepatitis C; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05168 Herpes simplex virus 1 infection; 05163 Human cytomegalovirus infection; 05169 Epstein-Barr virus infection; 05165 Human papillomavirus infection; 05130 Pathogenic Escherichia coli infection; 05132 Salmonella infection; 05131 Shigellosis; 05135 Yersinia infection; 05133 Pertussis; 05134 Legionellosis; 05152 Tuberculosis; 05146 Amoebiasis; 05144 Malaria; 05145 Toxoplasmosis; 05140 Leishmaniasis; 05142 Chagas disease; 05143 African trypanosomiasis; 05310 Asthma; 05322 Systemic lupus erythematosus; 05323 Rheumatoid arthritis; 05321 Inflammatory bowel disease; 05330 Allograft rejection; 05332 Graft-versus-host disease; 05010 Alzheimer disease; 05014 Amyotrophic lateral sclerosis; 05020 Prion disease; 05022 Pathways of neurodegeneration - multiple diseases; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 05410 Hypertrophic cardiomyopathy; 05414 Dilated cardiomyopathy; 04930 Type II diabetes mellitus; 04940 Type I diabetes mellitus; 04936 Alcoholic liver disease; 04932 Non-alcoholic fatty liver disease; 04931 Insulin resistance; 04933 AGE-RAGE signaling pathway in diabetic complications; 01523 Antifolate resistance; 04052 Cytokines and growth factors [BR:hsa04052]; 00536 Glycosaminoglycan binding proteins [BR:hsa00536]; Tumor necrosis fators; Cytokines; ','[Pfam: TNF]'),('5595','hsa:5595','[MAPK3, ERK-1, ERK1, ERT2, HS44KDAP, HUMKER1A, P44ERK1, P44MAPK, PRKM3, p44-ERK1, p44-MAPK]','[(RefSeq) mitogen-activated protein kinase 3]','[mitogen-activated protein kinase 1/3 [EC:2.7.11.24]]','[Homo sapiens (human)]','[EGFR tyrosine kinase inhibitor resistance, Endocrine resistance, Platinum drug resistance, MAPK signaling pathway, ErbB signaling pathway, Ras signaling pathway, Rap1 signaling pathway, cGMP-PKG signaling pathway, cAMP signaling pathway, Chemokine signaling pathway, HIF-1 signaling pathway, FoxO signaling pathway, Sphingolipid signaling pathway, Phospholipase D signaling pathway, Oocyte meiosis, Autophagy - animal, mTOR signaling pathway, PI3K-Akt signaling pathway, Apoptosis, Cellular senescence, Adrenergic signaling in cardiomyocytes, Vascular smooth muscle contraction, TGF-beta signaling pathway, Axon guidance, VEGF signaling pathway, Apelin signaling pathway, Osteoclast differentiation, Focal adhesion, Adherens junction, Gap junction, Signaling pathways regulating pluripotency of stem cells, Platelet activation, Neutrophil extracellular trap formation, Toll-like receptor signaling pathway, NOD-like receptor signaling pathway, C-type lectin receptor signaling pathway, Natural killer cell mediated cytotoxicity, IL-17 signaling pathway, Th1 and Th2 cell differentiation, Th17 cell differentiation, T cell receptor signaling pathway, B cell receptor signaling pathway, Fc epsilon RI signaling pathway, Fc gamma R-mediated phagocytosis, TNF signaling pathway, Circadian entrainment, Long-term potentiation, Neurotrophin signaling pathway, Retrograde endocannabinoid signaling, Glutamatergic synapse, Cholinergic synapse, Serotonergic synapse, Long-term depression, Regulation of actin cytoskeleton, Insulin signaling pathway, GnRH signaling pathway, Progesterone-mediated oocyte maturation, Estrogen signaling pathway, Melanogenesis, Prolactin signaling pathway, Thyroid hormone signaling pathway, Oxytocin signaling pathway, Relaxin signaling pathway, Parathyroid hormone synthesis, secretion and action, GnRH secretion, Type II diabetes mellitus, AGE-RAGE signaling pathway in diabetic complications, Cushing syndrome, Growth hormone synthesis, secretion and action, Aldosterone-regulated sodium reabsorption, Alzheimer disease, Prion disease, Pathways of neurodegeneration - multiple diseases, Alcoholism, Pathogenic Escherichia coli infection, Shigellosis, Salmonella infection, Pertussis, Yersinia infection, Leishmaniasis, Chagas disease, Toxoplasmosis, Tuberculosis, Hepatitis C, Hepatitis B, Human cytomegalovirus infection, Influenza A, Human papillomavirus infection, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Human immunodeficiency virus 1 infection, Coronavirus disease - COVID-19, Pathways in cancer, Viral carcinogenesis, Proteoglycans in cancer, MicroRNAs in cancer, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species, Colorectal cancer, Renal cell carcinoma, Pancreatic cancer, Endometrial cancer, Glioma, Prostate cancer, Thyroid cancer, Melanoma, Bladder cancer, Chronic myeloid leukemia, Acute myeloid leukemia, Non-small cell lung cancer, Breast cancer, Hepatocellular carcinoma, Gastric cancer, Central carbon metabolism in cancer, Choline metabolism in cancer, PD-L1 expression and PD-1 checkpoint pathway in cancer, Lipid and atherosclerosis]','[c(\"nt06110  MAPK signaling (viruses and bacteria)\", \"nt06121  TLR signaling (viruses and bacteria)\", \"nt06124  Chemokine signaling (viruses)\", \"nt06150  Cytokine-cytokine receptor interaction (viruses)\", \"nt06161  Human immunodeficiency virus type 1 (HIV-1)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06163  Hepatitis C virus (HCV)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06166  Human papillomavirus (HPV)\", \"nt06167  Human cytomegalovirus (HCMV)\", \"nt06170  Influenza A virus (IAV)\", \n\"nt06210  ERK signaling\", \"nt06220  Calcium signaling\", \"nt06224  CXCR signaling\", \"nt06260  Colorectal cancer\", \"nt06261  Gastric cancer\", \"nt06262  Pancreatic cancer\", \"nt06263  Hepatocellular carcinoma\", \"nt06264  Renal cell carcinoma\", \"nt06265  Bladder cancer\", \"nt06266  Non-small cell lung cancer\", \"nt06268  Melanoma\", \"nt06270  Breast cancer\", \"nt06271  Endometrial cancer\", \"nt06273  Glioma\", \"nt06274  Thyroid cancer\", \"nt06275  Acute myeloid leukemia\", \"nt06276  Chronic myeloid leukemia\", \n\"nt06310  CRH-ACTH-cortisol signaling\", \"nt06323  KISS1-GnRH-LH/FSH-E2 signaling\", \"nt06324  GHRH-GH-IGF signaling\", \"nt06360  Cushing syndrome\", \"nt06361  Hypogonadotropic hypogonadism\", \"nt06415  PI3K signaling\", \"nt06417  AGE-RAGE signaling\", \"nt06460  Alzheimer disease\", \"nt06465  Prion disease\")]','[N00001  EGF-EGFR-RAS-ERK signaling pathway, N00002  BCR-ABL fusion kinase to RAS-ERK signaling pathway, N00003  Mutation-activated KIT to RAS-ERK signaling pathway, N00004  Duplication or mutation-activated FLT3 to RAS-ERK signaling pathway, N00005  Mutation-activated MET to RAS-ERK signaling pathway, N00006  Amplified EGFR to RAS-ERK signaling pathway, N00007  EML4-ALK fusion kinase to RAS-ERK signaling pathway, N00008  RET fusion kinase to RAS-ERK signaling pathway, N00009  TRK fusion kinase to RAS-ERK signaling pathway, N00011  Mutation-activated FGFR3 to RAS-ERK signaling pathway, N00012  Mutation-activated KRAS/NRAS to ERK signaling pathway, N00013  Mutation-activated BRAF to ERK signaling pathway, N00014  Mutation-activated EGFR to RAS-ERK signaling pathway, N00015  PDGF-PDGFR-RAS-ERK signaling pathway, N00016  PDGF-overexpression to RAS-ERK signaling pathway, N00018  Amplified PDGFR to RAS-ERK signaling pathway, N00019  FGF-FGFR-RAS-ERK signaling pathway, N00020  Amplified FGFR to RAS-ERK signaling pathway, N00021  EGF-ERBB2-RAS-ERK signaling pathway, N00022  ERBB2-overexpression to RAS-ERK signaling pathway, N00023  EGF-EGFR-PLCG-ERK signaling pathway, N00024  Mutation-activated EGFR to PLCG-ERK signaling pathway, N00025  EML4-ALK fusion kinase to PLCG-ERK signaling pathway, N00041  EGFR-overexpression to RAS-ERK signaling pathway, N00077  HRAS-overexpression to ERK signaling pathway, N00078  Mutation-activated HRAS to ERK signaling pathway, N00152  CXCR-GNB/G-ERK signaling pathway, N00157  KSHV vGPCR to GNB/G-ERK signaling pathway, N00160  KSHV K1 to RAS-ERK signaling pathway, N00215  KITLG-KIT-RAS-ERK signaling pathway, N00216  HGF-MET-RAS-ERK signaling pathway, N00217  FLT3LG-FLT3-RAS-ERK signaling pathway, N00229  TGFA-EGFR-RAS-ERK signaling pathway, N00230  TGFA-overexpression to RAS-ERK signaling pathway, N00233  IGF-IGF1R-RAS-ERK signaling pathway, N00235  IGF2-overexpression to RAS-ERK signaling pathway, N00237  IGF1R-overexpression to RAS-ERK signaling pathway, N00246  HGF-overexpression to RAS-ERK signaling pathway, N00248  MET-overexpression to RAS-ERK signaling pathway, N00252  Amplified ERBB2 to RAS-ERK signaling pathway, N00259  Amplified MET to RAS-ERK signaling pathway, N00276  EGF-overexpression to RAS-ERK signaling pathway, N00277  EREG-EGFR-RAS-ERK signaling pathway, N00278  EREG-overexpression to RAS-ERK signaling pathway, N00279  AREG-EGFR-RAS-ERK signaling pathway, N00280  AREG-overexpression to RAS-ERK signaling pathway, N00318  EGFR-ERK-ACTH signaling pathway, N00319  Mutation-activated USP8 to EGFR-ERK-ACTH signaling pathway, N00367  HPV E5 to EGFR-RAS-ERK signaling pathway, N00369  HPV E5 to PDGFR-RAS-ERK signaling pathway, N00386  HCMV gB to PDGFR-RAS-ERK signaling pathway, N00392  HCMV gB to EGFR-RAS-ERK signaling pathway, N00438  TLR2/4-MAPK signaling pathway, N00439  HIV Nef to TLR2/4-MAPK signaling pathway, N00513  Mutation-activated EGFR to RAS-ERK signaling pathway, N00518  HCV Core to ERK signaling pathway, N00519  HCV Core to ERK signaling pathway, N00538  Ca2+-PYK2-RAS-ERK signaling pathway, N00539  HBV HBx to Ca2+-PYK2-RAS-ERK signaling pathway, N00540  HBV HBx to RAS-ERK signaling pathway, N00541  HBV HBx to RAS-ERK signaling pathway, N00545  HBV HBx to ERK signaling pathway, N00546  CXCL12-CXCR4-PKC-ERK signaling pathaway, N00547  HBV LHBs to PKC-ERK signaling pathway, N00744  IAV HA to ERK signaling pathway, N00873  GnRH-GnRHR-PLCB-PKC signaling pathway, N00875  Mutation-inactivated GnRHR to GnRH-GnRHR-PLCB-PKC signaling pathway, N00876  Mutation-inactivated FGF8 to RAS-ERK signaling pathway, N00877  Mutation-inactivated FGF17 to RAS-ERK signaling pathway, N00878  Mutation-inactivated FGFR1 to RAS-ERK signaling pathway, N00879  PROK-PRKR-Gi-ERK signaling pathway, N00880  Mutation-inactivated PROK2 to PROK-PRKR-Gi-ERK signaling pathway, N00881  Mutation-inactivated PROKR2 to PROK-PRKR-Gi-ERK signaling pathway, N00913  Mutation-inactivated IGF1 to RAS-ERK signaling pathway, N00914  Mutation-inactivated IGF1R to RAS-ERK signaling pathway, N00994  AGE-RAGE signaling pathway, N00996  Mutation-caused aberrant Abeta to AGE-RAGE signaling pathway, N01062  Mutation-activated MET to RAS-ERK signaling pathway, N01064  Mutation-activated RET to RAS-ERK signaling pathway, N01119  RAC/CDC42-PAK-ERK signaling pathway, N01121  Salmonella SpvC to ERK signaling pathway, N01204  PRNP-PI3K-NOX2 signaling pathway, N01283  Shigella OspF to TLR2/4-MAPK signaling pathway, N01343  ACH-CHRN-RAS-ERK signaling pathway, N01344  NNK/NNN to RAS-ERK signaling pathway, N01351  E2-ER-RAS-ERK signaling pathway, N01352  BPA to RAS-ERK signaling pathway, N01353  E2 to RAS-ERK signaling pathway, N01354  BPA to RAS-ERK signaling pathway, N01360  P4-PR-RAS-ERK signaling pathway, N01361  P4/MPA to PR-RAS-ERK signaling pathway, N01408  Metals to RAS-ERK signaling pathway]','04010 MAPK signaling pathway; 04012 ErbB signaling pathway; 04014 Ras signaling pathway; 04015 Rap1 signaling pathway; 04350 TGF-beta signaling pathway; 04370 VEGF signaling pathway; 04371 Apelin signaling pathway; 04668 TNF signaling pathway; 04066 HIF-1 signaling pathway; 04068 FoxO signaling pathway; 04072 Phospholipase D signaling pathway; 04071 Sphingolipid signaling pathway; 04024 cAMP signaling pathway; 04022 cGMP-PKG signaling pathway; 04151 PI3K-Akt signaling pathway; 04150 mTOR signaling pathway; 04140 Autophagy - animal; 04114 Oocyte meiosis; 04210 Apoptosis; 04218 Cellular senescence; 04510 Focal adhesion; 04520 Adherens junction; 04540 Gap junction; 04550 Signaling pathways regulating pluripotency of stem cells; 04810 Regulation of actin cytoskeleton; 04611 Platelet activation; 04613 Neutrophil extracellular trap formation; 04620 Toll-like receptor signaling pathway; 04621 NOD-like receptor signaling pathway; 04625 C-type lectin receptor signaling pathway; 04650 Natural killer cell mediated cytotoxicity; 04660 T cell receptor signaling pathway; 04658 Th1 and Th2 cell differentiation; 04659 Th17 cell differentiation; 04657 IL-17 signaling pathway; 04662 B cell receptor signaling pathway; 04664 Fc epsilon RI signaling pathway; 04666 Fc gamma R-mediated phagocytosis; 04062 Chemokine signaling pathway; 04910 Insulin signaling pathway; 04929 GnRH secretion; 04912 GnRH signaling pathway; 04915 Estrogen signaling pathway; 04914 Progesterone-mediated oocyte maturation; 04917 Prolactin signaling pathway; 04921 Oxytocin signaling pathway; 04926 Relaxin signaling pathway; 04935 Growth hormone synthesis, secretion and action; 04919 Thyroid hormone signaling pathway; 04928 Parathyroid hormone synthesis, secretion and action; 04916 Melanogenesis; 04261 Adrenergic signaling in cardiomyocytes; 04270 Vascular smooth muscle contraction; 04960 Aldosterone-regulated sodium reabsorption; 04724 Glutamatergic synapse; 04725 Cholinergic synapse; 04726 Serotonergic synapse; 04720 Long-term potentiation; 04730 Long-term depression; 04723 Retrograde endocannabinoid signaling; 04722 Neurotrophin signaling pathway; 04360 Axon guidance; 04380 Osteoclast differentiation; 04713 Circadian entrainment; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05205 Proteoglycans in cancer; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 05203 Viral carcinogenesis; 05230 Central carbon metabolism in cancer; 05231 Choline metabolism in cancer; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05210 Colorectal cancer; 05212 Pancreatic cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05214 Glioma; 05216 Thyroid cancer; 05221 Acute myeloid leukemia; 05220 Chronic myeloid leukemia; 05218 Melanoma; 05211 Renal cell carcinoma; 05219 Bladder cancer; 05215 Prostate cancer; 05213 Endometrial cancer; 05224 Breast cancer; 05223 Non-small cell lung cancer; 05166 Human T-cell leukemia virus 1 infection; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05160 Hepatitis C; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05165 Human papillomavirus infection; 05130 Pathogenic Escherichia coli infection; 05132 Salmonella infection; 05131 Shigellosis; 05135 Yersinia infection; 05133 Pertussis; 05152 Tuberculosis; 05145 Toxoplasmosis; 05140 Leishmaniasis; 05142 Chagas disease; 05010 Alzheimer disease; 05020 Prion disease; 05022 Pathways of neurodegeneration - multiple diseases; 05034 Alcoholism; 05417 Lipid and atherosclerosis; 04930 Type II diabetes mellitus; 04933 AGE-RAGE signaling pathway in diabetic complications; 04934 Cushing syndrome; 01521 EGFR tyrosine kinase inhibitor resistance; 01524 Platinum drug resistance; 01522 Endocrine resistance; 01001 Protein kinases [BR:hsa01001]; 03036 Chromosome and associated proteins [BR:hsa03036]; 04147 Exosome [BR:hsa04147]; 2.7.11.24  mitogen-activated protein kinase; MAPK family [OT]; Protein kinases; Exosomal proteins of haemopoietic cells  (B-cell, T-cell, DC-cell, reticulocyte, and mast cell); ','[Pfam: Pkinase PK_Tyr_Ser-Thr ABC1 APH Haspin_kinase FTA2 Choline_kinase Kdo Pkinase_fungal]'),('5594','hsa:5594','[MAPK1, ERK, ERK-2, ERK2, ERT1, MAPK2, NS13, P42MAPK, PRKM1, PRKM2, p38, p40, p41, p41mapk, p42-MAPK]','[(RefSeq) mitogen-activated protein kinase 1]','[mitogen-activated protein kinase 1/3 [EC:2.7.11.24]]','[Homo sapiens (human)]','[EGFR tyrosine kinase inhibitor resistance, Endocrine resistance, Platinum drug resistance, MAPK signaling pathway, ErbB signaling pathway, Ras signaling pathway, Rap1 signaling pathway, cGMP-PKG signaling pathway, cAMP signaling pathway, Chemokine signaling pathway, HIF-1 signaling pathway, FoxO signaling pathway, Sphingolipid signaling pathway, Phospholipase D signaling pathway, Oocyte meiosis, Autophagy - animal, mTOR signaling pathway, PI3K-Akt signaling pathway, Apoptosis, Cellular senescence, Adrenergic signaling in cardiomyocytes, Vascular smooth muscle contraction, TGF-beta signaling pathway, Axon guidance, VEGF signaling pathway, Apelin signaling pathway, Osteoclast differentiation, Focal adhesion, Adherens junction, Gap junction, Signaling pathways regulating pluripotency of stem cells, Platelet activation, Neutrophil extracellular trap formation, Toll-like receptor signaling pathway, NOD-like receptor signaling pathway, C-type lectin receptor signaling pathway, Natural killer cell mediated cytotoxicity, IL-17 signaling pathway, Th1 and Th2 cell differentiation, Th17 cell differentiation, T cell receptor signaling pathway, B cell receptor signaling pathway, Fc epsilon RI signaling pathway, Fc gamma R-mediated phagocytosis, TNF signaling pathway, Circadian entrainment, Long-term potentiation, Neurotrophin signaling pathway, Retrograde endocannabinoid signaling, Glutamatergic synapse, Cholinergic synapse, Serotonergic synapse, Long-term depression, Regulation of actin cytoskeleton, Insulin signaling pathway, GnRH signaling pathway, Progesterone-mediated oocyte maturation, Estrogen signaling pathway, Melanogenesis, Prolactin signaling pathway, Thyroid hormone signaling pathway, Oxytocin signaling pathway, Relaxin signaling pathway, Parathyroid hormone synthesis, secretion and action, GnRH secretion, Type II diabetes mellitus, AGE-RAGE signaling pathway in diabetic complications, Cushing syndrome, Growth hormone synthesis, secretion and action, Aldosterone-regulated sodium reabsorption, Alzheimer disease, Prion disease, Pathways of neurodegeneration - multiple diseases, Alcoholism, Pathogenic Escherichia coli infection, Shigellosis, Salmonella infection, Pertussis, Yersinia infection, Leishmaniasis, Chagas disease, Toxoplasmosis, Tuberculosis, Hepatitis C, Hepatitis B, Human cytomegalovirus infection, Influenza A, Human papillomavirus infection, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Human immunodeficiency virus 1 infection, Coronavirus disease - COVID-19, Pathways in cancer, Viral carcinogenesis, Proteoglycans in cancer, MicroRNAs in cancer, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species, Colorectal cancer, Renal cell carcinoma, Pancreatic cancer, Endometrial cancer, Glioma, Prostate cancer, Thyroid cancer, Melanoma, Bladder cancer, Chronic myeloid leukemia, Acute myeloid leukemia, Non-small cell lung cancer, Breast cancer, Hepatocellular carcinoma, Gastric cancer, Central carbon metabolism in cancer, Choline metabolism in cancer, PD-L1 expression and PD-1 checkpoint pathway in cancer, Lipid and atherosclerosis]','[c(\"nt06110  MAPK signaling (viruses and bacteria)\", \"nt06121  TLR signaling (viruses and bacteria)\", \"nt06124  Chemokine signaling (viruses)\", \"nt06150  Cytokine-cytokine receptor interaction (viruses)\", \"nt06161  Human immunodeficiency virus type 1 (HIV-1)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06163  Hepatitis C virus (HCV)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06166  Human papillomavirus (HPV)\", \"nt06167  Human cytomegalovirus (HCMV)\", \"nt06170  Influenza A virus (IAV)\", \n\"nt06210  ERK signaling\", \"nt06220  Calcium signaling\", \"nt06224  CXCR signaling\", \"nt06260  Colorectal cancer\", \"nt06261  Gastric cancer\", \"nt06262  Pancreatic cancer\", \"nt06263  Hepatocellular carcinoma\", \"nt06264  Renal cell carcinoma\", \"nt06265  Bladder cancer\", \"nt06266  Non-small cell lung cancer\", \"nt06268  Melanoma\", \"nt06270  Breast cancer\", \"nt06271  Endometrial cancer\", \"nt06273  Glioma\", \"nt06274  Thyroid cancer\", \"nt06275  Acute myeloid leukemia\", \"nt06276  Chronic myeloid leukemia\", \n\"nt06310  CRH-ACTH-cortisol signaling\", \"nt06323  KISS1-GnRH-LH/FSH-E2 signaling\", \"nt06324  GHRH-GH-IGF signaling\", \"nt06360  Cushing syndrome\", \"nt06361  Hypogonadotropic hypogonadism\", \"nt06415  PI3K signaling\", \"nt06417  AGE-RAGE signaling\", \"nt06460  Alzheimer disease\", \"nt06465  Prion disease\")]','[N00001  EGF-EGFR-RAS-ERK signaling pathway, N00002  BCR-ABL fusion kinase to RAS-ERK signaling pathway, N00003  Mutation-activated KIT to RAS-ERK signaling pathway, N00004  Duplication or mutation-activated FLT3 to RAS-ERK signaling pathway, N00005  Mutation-activated MET to RAS-ERK signaling pathway, N00006  Amplified EGFR to RAS-ERK signaling pathway, N00007  EML4-ALK fusion kinase to RAS-ERK signaling pathway, N00008  RET fusion kinase to RAS-ERK signaling pathway, N00009  TRK fusion kinase to RAS-ERK signaling pathway, N00011  Mutation-activated FGFR3 to RAS-ERK signaling pathway, N00012  Mutation-activated KRAS/NRAS to ERK signaling pathway, N00013  Mutation-activated BRAF to ERK signaling pathway, N00014  Mutation-activated EGFR to RAS-ERK signaling pathway, N00015  PDGF-PDGFR-RAS-ERK signaling pathway, N00016  PDGF-overexpression to RAS-ERK signaling pathway, N00018  Amplified PDGFR to RAS-ERK signaling pathway, N00019  FGF-FGFR-RAS-ERK signaling pathway, N00020  Amplified FGFR to RAS-ERK signaling pathway, N00021  EGF-ERBB2-RAS-ERK signaling pathway, N00022  ERBB2-overexpression to RAS-ERK signaling pathway, N00023  EGF-EGFR-PLCG-ERK signaling pathway, N00024  Mutation-activated EGFR to PLCG-ERK signaling pathway, N00025  EML4-ALK fusion kinase to PLCG-ERK signaling pathway, N00041  EGFR-overexpression to RAS-ERK signaling pathway, N00077  HRAS-overexpression to ERK signaling pathway, N00078  Mutation-activated HRAS to ERK signaling pathway, N00152  CXCR-GNB/G-ERK signaling pathway, N00157  KSHV vGPCR to GNB/G-ERK signaling pathway, N00160  KSHV K1 to RAS-ERK signaling pathway, N00215  KITLG-KIT-RAS-ERK signaling pathway, N00216  HGF-MET-RAS-ERK signaling pathway, N00217  FLT3LG-FLT3-RAS-ERK signaling pathway, N00229  TGFA-EGFR-RAS-ERK signaling pathway, N00230  TGFA-overexpression to RAS-ERK signaling pathway, N00233  IGF-IGF1R-RAS-ERK signaling pathway, N00235  IGF2-overexpression to RAS-ERK signaling pathway, N00237  IGF1R-overexpression to RAS-ERK signaling pathway, N00246  HGF-overexpression to RAS-ERK signaling pathway, N00248  MET-overexpression to RAS-ERK signaling pathway, N00252  Amplified ERBB2 to RAS-ERK signaling pathway, N00259  Amplified MET to RAS-ERK signaling pathway, N00276  EGF-overexpression to RAS-ERK signaling pathway, N00277  EREG-EGFR-RAS-ERK signaling pathway, N00278  EREG-overexpression to RAS-ERK signaling pathway, N00279  AREG-EGFR-RAS-ERK signaling pathway, N00280  AREG-overexpression to RAS-ERK signaling pathway, N00318  EGFR-ERK-ACTH signaling pathway, N00319  Mutation-activated USP8 to EGFR-ERK-ACTH signaling pathway, N00367  HPV E5 to EGFR-RAS-ERK signaling pathway, N00369  HPV E5 to PDGFR-RAS-ERK signaling pathway, N00386  HCMV gB to PDGFR-RAS-ERK signaling pathway, N00392  HCMV gB to EGFR-RAS-ERK signaling pathway, N00438  TLR2/4-MAPK signaling pathway, N00439  HIV Nef to TLR2/4-MAPK signaling pathway, N00513  Mutation-activated EGFR to RAS-ERK signaling pathway, N00518  HCV Core to ERK signaling pathway, N00519  HCV Core to ERK signaling pathway, N00538  Ca2+-PYK2-RAS-ERK signaling pathway, N00539  HBV HBx to Ca2+-PYK2-RAS-ERK signaling pathway, N00540  HBV HBx to RAS-ERK signaling pathway, N00541  HBV HBx to RAS-ERK signaling pathway, N00545  HBV HBx to ERK signaling pathway, N00546  CXCL12-CXCR4-PKC-ERK signaling pathaway, N00547  HBV LHBs to PKC-ERK signaling pathway, N00744  IAV HA to ERK signaling pathway, N00873  GnRH-GnRHR-PLCB-PKC signaling pathway, N00875  Mutation-inactivated GnRHR to GnRH-GnRHR-PLCB-PKC signaling pathway, N00876  Mutation-inactivated FGF8 to RAS-ERK signaling pathway, N00877  Mutation-inactivated FGF17 to RAS-ERK signaling pathway, N00878  Mutation-inactivated FGFR1 to RAS-ERK signaling pathway, N00879  PROK-PRKR-Gi-ERK signaling pathway, N00880  Mutation-inactivated PROK2 to PROK-PRKR-Gi-ERK signaling pathway, N00881  Mutation-inactivated PROKR2 to PROK-PRKR-Gi-ERK signaling pathway, N00913  Mutation-inactivated IGF1 to RAS-ERK signaling pathway, N00914  Mutation-inactivated IGF1R to RAS-ERK signaling pathway, N00994  AGE-RAGE signaling pathway, N00996  Mutation-caused aberrant Abeta to AGE-RAGE signaling pathway, N01062  Mutation-activated MET to RAS-ERK signaling pathway, N01064  Mutation-activated RET to RAS-ERK signaling pathway, N01119  RAC/CDC42-PAK-ERK signaling pathway, N01121  Salmonella SpvC to ERK signaling pathway, N01204  PRNP-PI3K-NOX2 signaling pathway, N01283  Shigella OspF to TLR2/4-MAPK signaling pathway, N01343  ACH-CHRN-RAS-ERK signaling pathway, N01344  NNK/NNN to RAS-ERK signaling pathway, N01351  E2-ER-RAS-ERK signaling pathway, N01352  BPA to RAS-ERK signaling pathway, N01353  E2 to RAS-ERK signaling pathway, N01354  BPA to RAS-ERK signaling pathway, N01360  P4-PR-RAS-ERK signaling pathway, N01361  P4/MPA to PR-RAS-ERK signaling pathway, N01408  Metals to RAS-ERK signaling pathway]','04010 MAPK signaling pathway; 04012 ErbB signaling pathway; 04014 Ras signaling pathway; 04015 Rap1 signaling pathway; 04350 TGF-beta signaling pathway; 04370 VEGF signaling pathway; 04371 Apelin signaling pathway; 04668 TNF signaling pathway; 04066 HIF-1 signaling pathway; 04068 FoxO signaling pathway; 04072 Phospholipase D signaling pathway; 04071 Sphingolipid signaling pathway; 04024 cAMP signaling pathway; 04022 cGMP-PKG signaling pathway; 04151 PI3K-Akt signaling pathway; 04150 mTOR signaling pathway; 04140 Autophagy - animal; 04114 Oocyte meiosis; 04210 Apoptosis; 04218 Cellular senescence; 04510 Focal adhesion; 04520 Adherens junction; 04540 Gap junction; 04550 Signaling pathways regulating pluripotency of stem cells; 04810 Regulation of actin cytoskeleton; 04611 Platelet activation; 04613 Neutrophil extracellular trap formation; 04620 Toll-like receptor signaling pathway; 04621 NOD-like receptor signaling pathway; 04625 C-type lectin receptor signaling pathway; 04650 Natural killer cell mediated cytotoxicity; 04660 T cell receptor signaling pathway; 04658 Th1 and Th2 cell differentiation; 04659 Th17 cell differentiation; 04657 IL-17 signaling pathway; 04662 B cell receptor signaling pathway; 04664 Fc epsilon RI signaling pathway; 04666 Fc gamma R-mediated phagocytosis; 04062 Chemokine signaling pathway; 04910 Insulin signaling pathway; 04929 GnRH secretion; 04912 GnRH signaling pathway; 04915 Estrogen signaling pathway; 04914 Progesterone-mediated oocyte maturation; 04917 Prolactin signaling pathway; 04921 Oxytocin signaling pathway; 04926 Relaxin signaling pathway; 04935 Growth hormone synthesis, secretion and action; 04919 Thyroid hormone signaling pathway; 04928 Parathyroid hormone synthesis, secretion and action; 04916 Melanogenesis; 04261 Adrenergic signaling in cardiomyocytes; 04270 Vascular smooth muscle contraction; 04960 Aldosterone-regulated sodium reabsorption; 04724 Glutamatergic synapse; 04725 Cholinergic synapse; 04726 Serotonergic synapse; 04720 Long-term potentiation; 04730 Long-term depression; 04723 Retrograde endocannabinoid signaling; 04722 Neurotrophin signaling pathway; 04360 Axon guidance; 04380 Osteoclast differentiation; 04713 Circadian entrainment; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05205 Proteoglycans in cancer; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 05203 Viral carcinogenesis; 05230 Central carbon metabolism in cancer; 05231 Choline metabolism in cancer; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05210 Colorectal cancer; 05212 Pancreatic cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05214 Glioma; 05216 Thyroid cancer; 05221 Acute myeloid leukemia; 05220 Chronic myeloid leukemia; 05218 Melanoma; 05211 Renal cell carcinoma; 05219 Bladder cancer; 05215 Prostate cancer; 05213 Endometrial cancer; 05224 Breast cancer; 05223 Non-small cell lung cancer; 05166 Human T-cell leukemia virus 1 infection; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05160 Hepatitis C; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05165 Human papillomavirus infection; 05130 Pathogenic Escherichia coli infection; 05132 Salmonella infection; 05131 Shigellosis; 05135 Yersinia infection; 05133 Pertussis; 05152 Tuberculosis; 05145 Toxoplasmosis; 05140 Leishmaniasis; 05142 Chagas disease; 05010 Alzheimer disease; 05020 Prion disease; 05022 Pathways of neurodegeneration - multiple diseases; 05034 Alcoholism; 05417 Lipid and atherosclerosis; 04930 Type II diabetes mellitus; 04933 AGE-RAGE signaling pathway in diabetic complications; 04934 Cushing syndrome; 01521 EGFR tyrosine kinase inhibitor resistance; 01524 Platinum drug resistance; 01522 Endocrine resistance; 01001 Protein kinases [BR:hsa01001]; 03036 Chromosome and associated proteins [BR:hsa03036]; 04147 Exosome [BR:hsa04147]; 2.7.11.24  mitogen-activated protein kinase; MAPK family [OT]; Protein kinases; Exosomal proteins of haemopoietic cells  (B-cell, T-cell, DC-cell, reticulocyte, and mast cell); ','[Pfam: Pkinase PK_Tyr_Ser-Thr ABC1 APH Haspin_kinase FTA2 Choline_kinase Kdo Pkinase_fungal Kinase-like]'),('3576','hsa:3576','[CXCL8, GCP-1, GCP1, IL8, LECT, LUCT, LYNAP, MDNCF, MONAP, NAF, NAP-1, NAP1, SCYB8]','[(RefSeq) C-X-C motif chemokine ligand 8]','[interleukin 8]','[Homo sapiens (human)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, Chemokine signaling pathway, NF-kappa B signaling pathway, Phospholipase D signaling pathway, Cellular senescence, Toll-like receptor signaling pathway, NOD-like receptor signaling pathway, RIG-I-like receptor signaling pathway, IL-17 signaling pathway, Non-alcoholic fatty liver disease, AGE-RAGE signaling pathway in diabetic complications, Alcoholic liver disease, Epithelial cell signaling in Helicobacter pylori infection, Pathogenic Escherichia coli infection, Shigellosis, Salmonella infection, Pertussis, Legionellosis, Yersinia infection, Chagas disease, Malaria, Amoebiasis, Hepatitis B, Human cytomegalovirus infection, Influenza A, Kaposi sarcoma-associated herpesvirus infection, Coronavirus disease - COVID-19, Pathways in cancer, Transcriptional misregulation in cancer, Bladder cancer, Rheumatoid arthritis, Lipid and atherosclerosis]','[c(\"nt06124  Chemokine signaling (viruses)\", \"nt06140  Transcription (viruses)\", \"nt06150  Cytokine-cytokine receptor interaction (viruses)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06167  Human cytomegalovirus (HCMV)\", \"nt06171  Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)\", \"nt06224  CXCR signaling\", \"nt06240  Transcription\", \"nt06263  Hepatocellular carcinoma\")]','[N00140  FUS-DDIT3 fusion to NFKB-mediated transcription, N00152  CXCR-GNB/G-ERK signaling pathway, N00153  CCR/CXCR-GNB/G-PI3K-RAC signaling pathway, N00154  CXCR-GNB/G-PI3K-AKT signaling pathway, N00178  KSHV vGPCR to GNB/G-PI3K-JNK signaling pathway, N00400  HCMV US28 to GNB/G-PI3K-NFKB signaling pathway, N00402  HCMV US28 to GNAQ-PLCB/G-calcineurin signaling pathway, N00544  HBV HBx to CREB-mediated transcription, N01233  CXCL1/5/6/8 to CXCR1 signaling, N01234  CXCL1/5/8 to KSHV vGPCR signaling, N01238  CXCL1/2/3/5/6/7/8 to CXCR2 signaling, N01307  SARS-CoV-2 S to AngII-AT1R-NOX2 signaling pathway]','04064 NF-kappa B signaling pathway; 04072 Phospholipase D signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04218 Cellular senescence; 04620 Toll-like receptor signaling pathway; 04621 NOD-like receptor signaling pathway; 04622 RIG-I-like receptor signaling pathway; 04657 IL-17 signaling pathway; 04062 Chemokine signaling pathway; 05200 Pathways in cancer; 05202 Transcriptional misregulation in cancer; 05219 Bladder cancer; 05161 Hepatitis B; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05120 Epithelial cell signaling in Helicobacter pylori infection; 05130 Pathogenic Escherichia coli infection; 05132 Salmonella infection; 05131 Shigellosis; 05135 Yersinia infection; 05133 Pertussis; 05134 Legionellosis; 05146 Amoebiasis; 05144 Malaria; 05142 Chagas disease; 05323 Rheumatoid arthritis; 05417 Lipid and atherosclerosis; 04936 Alcoholic liver disease; 04932 Non-alcoholic fatty liver disease; 04933 AGE-RAGE signaling pathway in diabetic complications; 04052 Cytokines and growth factors [BR:hsa04052]; 00536 Glycosaminoglycan binding proteins [BR:hsa00536]; Chemokines; Chemokines; ','[Pfam: IL8]'),('2353','hsa:2353','[FOS, AP-1, C-FOS, p55]','[(RefSeq) Fos proto-oncogene, AP-1 transcription factor subunit]','[proto-oncogene protein c-fos]','[Homo sapiens (human)]','[Endocrine resistance, MAPK signaling pathway, cAMP signaling pathway, Apoptosis, Osteoclast differentiation, Toll-like receptor signaling pathway, IL-17 signaling pathway, Th1 and Th2 cell differentiation, Th17 cell differentiation, T cell receptor signaling pathway, B cell receptor signaling pathway, TNF signaling pathway, Circadian entrainment, Cholinergic synapse, Dopaminergic synapse, Estrogen signaling pathway, Prolactin signaling pathway, Oxytocin signaling pathway, Relaxin signaling pathway, Parathyroid hormone synthesis, secretion and action, Non-alcoholic fatty liver disease, Growth hormone synthesis, secretion and action, Amphetamine addiction, Pathogenic Escherichia coli infection, Salmonella infection, Pertussis, Yersinia infection, Leishmaniasis, Chagas disease, Hepatitis B, Measles, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Human immunodeficiency virus 1 infection, Coronavirus disease - COVID-19, Pathways in cancer, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species, Colorectal cancer, Breast cancer, Choline metabolism in cancer, PD-L1 expression and PD-1 checkpoint pathway in cancer, Rheumatoid arthritis, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[c(\"nt06110  MAPK signaling (viruses and bacteria)\", \"nt06121  TLR signaling (viruses and bacteria)\", \"nt06123  TNF signaling (viruses and bacteria)\", \"nt06124  Chemokine signaling (viruses)\", \"nt06140  Transcription (viruses)\", \"nt06150  Cytokine-cytokine receptor interaction (viruses)\", \"nt06160  Human T-cell leukemia virus 1 (HTLV-1)\", \"nt06161  Human immunodeficiency virus type 1 (HIV-1)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06210  ERK signaling\", \n\"nt06211  Other MAPK signaling\", \"nt06214  PI3K signaling\", \"nt06223  TNF signaling\", \"nt06224  CXCR signaling\", \"nt06263  Hepatocellular carcinoma\", \"nt06416  TNF signaling\", \"nt06461  Huntington disease\")]','[N00157  KSHV vGPCR to GNB/G-ERK signaling pathway, N00188  IL1-IL1R-JNK signaling pathway, N00189  KSHV K15 to JNK signaling pathway, N00438  TLR2/4-MAPK signaling pathway, N00446  TNF-JNK signaling pathway, N00447  HIV Vpr/Tat to TNF-JNK signaling pathway, N00509  HTLV-1 Tax to SRF-mediated transcription, N00545  HBV HBx to ERK signaling pathway, N00547  HBV LHBs to PKC-ERK signaling pathway, N01344  NNK/NNN to RAS-ERK signaling pathway, N01350  NNK/NNN to PI3K signaling pathway, N01353  E2 to RAS-ERK signaling pathway, N01407  Metals to JNK signaling pathway, N01408  Metals to RAS-ERK signaling pathway]','04010 MAPK signaling pathway; 04668 TNF signaling pathway; 04024 cAMP signaling pathway; 04210 Apoptosis; 04620 Toll-like receptor signaling pathway; 04660 T cell receptor signaling pathway; 04658 Th1 and Th2 cell differentiation; 04659 Th17 cell differentiation; 04657 IL-17 signaling pathway; 04662 B cell receptor signaling pathway; 04915 Estrogen signaling pathway; 04917 Prolactin signaling pathway; 04921 Oxytocin signaling pathway; 04926 Relaxin signaling pathway; 04935 Growth hormone synthesis, secretion and action; 04928 Parathyroid hormone synthesis, secretion and action; 04725 Cholinergic synapse; 04728 Dopaminergic synapse; 04380 Osteoclast differentiation; 04713 Circadian entrainment; 05200 Pathways in cancer; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 05231 Choline metabolism in cancer; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05210 Colorectal cancer; 05224 Breast cancer; 05166 Human T-cell leukemia virus 1 infection; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05171 Coronavirus disease - COVID-19; 05162 Measles; 05167 Kaposi sarcoma-associated herpesvirus infection; 05130 Pathogenic Escherichia coli infection; 05132 Salmonella infection; 05135 Yersinia infection; 05133 Pertussis; 05140 Leishmaniasis; 05142 Chagas disease; 05323 Rheumatoid arthritis; 05031 Amphetamine addiction; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 04932 Non-alcoholic fatty liver disease; 01522 Endocrine resistance; 03000 Transcription factors [BR:hsa03000]; 04131 Membrane trafficking [BR:hsa04131]; AP-1(-like) components, Fos; Selective cargos; ','[Pfam: bZIP_1 bZIP_2 bZIP_Maf Macoilin DUF3450 HALZ]'),('3725','hsa:3725','[JUN, AP-1, AP1, c-Jun, cJUN, p39]','[(RefSeq) Jun proto-oncogene, AP-1 transcription factor subunit]','[transcription factor AP-1]','[Homo sapiens (human)]','[Endocrine resistance, MAPK signaling pathway, ErbB signaling pathway, cAMP signaling pathway, Mitophagy - animal, Apoptosis, Wnt signaling pathway, Osteoclast differentiation, Focal adhesion, Tight junction, Toll-like receptor signaling pathway, NOD-like receptor signaling pathway, C-type lectin receptor signaling pathway, IL-17 signaling pathway, Th1 and Th2 cell differentiation, Th17 cell differentiation, T cell receptor signaling pathway, B cell receptor signaling pathway, TNF signaling pathway, Neurotrophin signaling pathway, GnRH signaling pathway, Estrogen signaling pathway, Oxytocin signaling pathway, Relaxin signaling pathway, Non-alcoholic fatty liver disease, AGE-RAGE signaling pathway in diabetic complications, Cocaine addiction, Amphetamine addiction, Epithelial cell signaling in Helicobacter pylori infection, Pathogenic Escherichia coli infection, Shigellosis, Salmonella infection, Pertussis, Yersinia infection, Leishmaniasis, Chagas disease, Hepatitis B, Measles, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Epstein-Barr virus infection, Human immunodeficiency virus 1 infection, Coronavirus disease - COVID-19, Pathways in cancer, Viral carcinogenesis, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species, Colorectal cancer, Renal cell carcinoma, Breast cancer, Choline metabolism in cancer, PD-L1 expression and PD-1 checkpoint pathway in cancer, Inflammatory bowel disease, Rheumatoid arthritis, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[c(\"nt06110  MAPK signaling (viruses and bacteria)\", \"nt06121  TLR signaling (viruses and bacteria)\", \"nt06123  TNF signaling (viruses and bacteria)\", \"nt06124  Chemokine signaling (viruses)\", \"nt06150  Cytokine-cytokine receptor interaction (viruses)\", \"nt06160  Human T-cell leukemia virus 1 (HTLV-1)\", \"nt06161  Human immunodeficiency virus type 1 (HIV-1)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06210  ERK signaling\", \"nt06211  Other MAPK signaling\", \n\"nt06214  PI3K signaling\", \"nt06223  TNF signaling\", \"nt06224  CXCR signaling\", \"nt06263  Hepatocellular carcinoma\", \"nt06416  TNF signaling\", \"nt06461  Huntington disease\")]','[N00157  KSHV vGPCR to GNB/G-ERK signaling pathway, N00188  IL1-IL1R-JNK signaling pathway, N00189  KSHV K15 to JNK signaling pathway, N00438  TLR2/4-MAPK signaling pathway, N00446  TNF-JNK signaling pathway, N00447  HIV Vpr/Tat to TNF-JNK signaling pathway, N00503  HTLV-1 Tax to TNF-JNK signaling pathway, N00543  HBV HBx to JNK signaling pathway, N00545  HBV HBx to ERK signaling pathway, N00547  HBV LHBs to PKC-ERK signaling pathway, N01344  NNK/NNN to RAS-ERK signaling pathway, N01350  NNK/NNN to PI3K signaling pathway, N01353  E2 to RAS-ERK signaling pathway, N01407  Metals to JNK signaling pathway, N01408  Metals to RAS-ERK signaling pathway]','04010 MAPK signaling pathway; 04012 ErbB signaling pathway; 04310 Wnt signaling pathway; 04668 TNF signaling pathway; 04024 cAMP signaling pathway; 04137 Mitophagy - animal; 04210 Apoptosis; 04510 Focal adhesion; 04530 Tight junction; 04620 Toll-like receptor signaling pathway; 04621 NOD-like receptor signaling pathway; 04625 C-type lectin receptor signaling pathway; 04660 T cell receptor signaling pathway; 04658 Th1 and Th2 cell differentiation; 04659 Th17 cell differentiation; 04657 IL-17 signaling pathway; 04662 B cell receptor signaling pathway; 04912 GnRH signaling pathway; 04915 Estrogen signaling pathway; 04921 Oxytocin signaling pathway; 04926 Relaxin signaling pathway; 04722 Neurotrophin signaling pathway; 04380 Osteoclast differentiation; 05200 Pathways in cancer; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 05203 Viral carcinogenesis; 05231 Choline metabolism in cancer; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05210 Colorectal cancer; 05211 Renal cell carcinoma; 05224 Breast cancer; 05166 Human T-cell leukemia virus 1 infection; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05171 Coronavirus disease - COVID-19; 05162 Measles; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05120 Epithelial cell signaling in Helicobacter pylori infection; 05130 Pathogenic Escherichia coli infection; 05132 Salmonella infection; 05131 Shigellosis; 05135 Yersinia infection; 05133 Pertussis; 05140 Leishmaniasis; 05142 Chagas disease; 05323 Rheumatoid arthritis; 05321 Inflammatory bowel disease; 05030 Cocaine addiction; 05031 Amphetamine addiction; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 04932 Non-alcoholic fatty liver disease; 04933 AGE-RAGE signaling pathway in diabetic complications; 01522 Endocrine resistance; 03000 Transcription factors [BR:hsa03000]; AP-1(-like) components, Jun; ','[Pfam: Jun bZIP_1 bZIP_2 bZIP_Maf ZapB PspA_IM30 Fib_alpha FlaC_arch TFIIA NBP1 DUF3450]'),('4583','hsa:4583','[MUC2, MLP, MUC-2, SMUC]','[(RefSeq) mucin 2, oligomeric mucus/gel-forming]','[mucin-2]','[Homo sapiens (human)]','[Amoebiasis, Gastric cancer]','[c(\"nt06240  Transcription\", \"nt06261  Gastric cancer\")]','[N00250  CDX2-overexpression to transcriptional activation]','05226 Gastric cancer; 05146 Amoebiasis; 04131 Membrane trafficking [BR:hsa04131]; Secretory mucins; ','[Pfam: VWD C8 Mucin2_WxxW TIL VWC Pacifastin_I]'),('3065','hsa:3065','[HDAC1, GON-10, HD1, KDAC1, RPD3, RPD3L1]','[(RefSeq) histone deacetylase 1]','[histone deacetylase 1/2 [EC:3.5.1.98]]','[Homo sapiens (human)]','[Cell cycle, Longevity regulating pathway - multiple species, Notch signaling pathway, Neutrophil extracellular trap formation, Thyroid hormone signaling pathway, Huntington disease, Amphetamine addiction, Alcoholism, Human papillomavirus infection, Epstein-Barr virus infection, Pathways in cancer, Transcriptional misregulation in cancer, Viral carcinogenesis, MicroRNAs in cancer, Chronic myeloid leukemia]','[c(\"nt06123  TNF signaling (viruses and bacteria)\", \"nt06166  Human papillomavirus (HPV)\", \"nt06240  Transcription\", \"nt06440  Transcription\", \"nt06461  Huntington disease\")]','[N00118  TEL-AML1 fusion to transcriptional repression, N00375  HPV E7 to TNF-IRF1 signaling pathway, N00980  Mutation-caused aberrant Htt to REST-mediated transcriptional repression]','04330 Notch signaling pathway; 04110 Cell cycle; 04613 Neutrophil extracellular trap formation; 04919 Thyroid hormone signaling pathway; 04213 Longevity regulating pathway - multiple species; 05200 Pathways in cancer; 05202 Transcriptional misregulation in cancer; 05206 MicroRNAs in cancer; 05203 Viral carcinogenesis; 05220 Chronic myeloid leukemia; 05169 Epstein-Barr virus infection; 05165 Human papillomavirus infection; 05016 Huntington disease; 05031 Amphetamine addiction; 05034 Alcoholism; 03036 Chromosome and associated proteins [BR:hsa03036]; 3.5.1.98  histone deacetylase; Class I HDACs; Sin3A-HDAC complex; BRAF-HDAC complex; Other heterochromatin formation proteins; NuRD complex; ','[Pfam: Hist_deacetyl]'),('3066','hsa:3066','[HDAC2, HD2, KDAC2, RPD3, YAF1]','[(RefSeq) histone deacetylase 2]','[histone deacetylase 1/2 [EC:3.5.1.98]]','[Homo sapiens (human)]','[Cell cycle, Longevity regulating pathway - multiple species, Notch signaling pathway, Neutrophil extracellular trap formation, Thyroid hormone signaling pathway, Huntington disease, Amphetamine addiction, Alcoholism, Human papillomavirus infection, Epstein-Barr virus infection, Pathways in cancer, Transcriptional misregulation in cancer, Viral carcinogenesis, MicroRNAs in cancer, Chronic myeloid leukemia]','[c(\"nt06123  TNF signaling (viruses and bacteria)\", \"nt06166  Human papillomavirus (HPV)\", \"nt06240  Transcription\", \"nt06440  Transcription\", \"nt06461  Huntington disease\")]','[N00118  TEL-AML1 fusion to transcriptional repression, N00375  HPV E7 to TNF-IRF1 signaling pathway, N00980  Mutation-caused aberrant Htt to REST-mediated transcriptional repression]','04330 Notch signaling pathway; 04110 Cell cycle; 04613 Neutrophil extracellular trap formation; 04919 Thyroid hormone signaling pathway; 04213 Longevity regulating pathway - multiple species; 05200 Pathways in cancer; 05202 Transcriptional misregulation in cancer; 05206 MicroRNAs in cancer; 05203 Viral carcinogenesis; 05220 Chronic myeloid leukemia; 05169 Epstein-Barr virus infection; 05165 Human papillomavirus infection; 05016 Huntington disease; 05031 Amphetamine addiction; 05034 Alcoholism; 03036 Chromosome and associated proteins [BR:hsa03036]; 3.5.1.98  histone deacetylase; Class I HDACs; Sin3A-HDAC complex; BRAF-HDAC complex; Other heterochromatin formation proteins; NuRD complex; ','[Pfam: Hist_deacetyl]'),('8841','hsa:8841','[HDAC3, HD3, KDAC3, RPD3, RPD3-2]','[(RefSeq) histone deacetylase 3]','[histone deacetylase 3 [EC:3.5.1.98]]','[Homo sapiens (human)]','[Neutrophil extracellular trap formation, Thyroid hormone signaling pathway, Alcoholism, Viral carcinogenesis]','[]','[]','04613 Neutrophil extracellular trap formation; 04919 Thyroid hormone signaling pathway; 05203 Viral carcinogenesis; 05034 Alcoholism; 03036 Chromosome and associated proteins [BR:hsa03036]; 3.5.1.98  histone deacetylase; Class I HDACs; ','[Pfam: Hist_deacetyl]'),('9759','hsa:9759','[HDAC4, AHO3, BDMR, HA6116, HD4, HDAC-4, HDAC-A, HDACA, NEDCHF, NEDCHID]','[(RefSeq) histone deacetylase 4]','[histone deacetylase 4/5 [EC:3.5.1.98]]','[Homo sapiens (human)]','[Apelin signaling pathway, Neutrophil extracellular trap formation, Alcoholism, Viral carcinogenesis, MicroRNAs in cancer]','[]','[]','04371 Apelin signaling pathway; 04613 Neutrophil extracellular trap formation; 05206 MicroRNAs in cancer; 05203 Viral carcinogenesis; 05034 Alcoholism; 03036 Chromosome and associated proteins [BR:hsa03036]; 3.5.1.98  histone deacetylase; Class II HDACs; ','[Pfam: Hist_deacetyl HDAC4_Gln]'),('10014','hsa:10014','[HDAC5, HD5, NY-CO-9]','[(RefSeq) histone deacetylase 5]','[histone deacetylase 4/5 [EC:3.5.1.98]]','[Homo sapiens (human)]','[Apelin signaling pathway, Neutrophil extracellular trap formation, Alcoholism, Viral carcinogenesis, MicroRNAs in cancer]','[]','[]','04371 Apelin signaling pathway; 04613 Neutrophil extracellular trap formation; 05206 MicroRNAs in cancer; 05203 Viral carcinogenesis; 05034 Alcoholism; 03036 Chromosome and associated proteins [BR:hsa03036]; 3.5.1.98  histone deacetylase; Class II HDACs; ','[Pfam: Hist_deacetyl HDAC4_Gln]'),('51564','hsa:51564','[HDAC7, HD7, HD7A, HDAC7A]','[(RefSeq) histone deacetylase 7]','[histone deacetylase 7 [EC:3.5.1.98]]','[Homo sapiens (human)]','[Neutrophil extracellular trap formation, Alcoholism, Viral carcinogenesis]','[]','[]','04613 Neutrophil extracellular trap formation; 05203 Viral carcinogenesis; 05034 Alcoholism; 03036 Chromosome and associated proteins [BR:hsa03036]; 3.5.1.98  histone deacetylase; Class II HDACs; ','[Pfam: Hist_deacetyl]'),('55869','hsa:55869','[HDAC8, CDA07, CDLS5, HD8, HDACL1, KDAC8, MRXS6, RPD3, WTS]','[(RefSeq) histone deacetylase 8]','[histone deacetylase 8 [EC:3.5.1.98]]','[Homo sapiens (human)]','[Neutrophil extracellular trap formation, Alcoholism, Viral carcinogenesis]','[]','[]','04613 Neutrophil extracellular trap formation; 05203 Viral carcinogenesis; 05034 Alcoholism; 03036 Chromosome and associated proteins [BR:hsa03036]; 3.5.1.98  histone deacetylase; Class I HDACs; ','[Pfam: Hist_deacetyl BFN_dom]'),('9734','hsa:9734','[HDAC9, HD7, HD7b, HD9, HDAC, HDAC7, HDAC7B, HDAC9B, HDAC9FL, HDRP, MITR]','[(RefSeq) histone deacetylase 9]','[histone deacetylase 9 [EC:3.5.1.98]]','[Homo sapiens (human)]','[Neutrophil extracellular trap formation, Alcoholism, Viral carcinogenesis]','[]','[]','04613 Neutrophil extracellular trap formation; 05203 Viral carcinogenesis; 05034 Alcoholism; 03036 Chromosome and associated proteins [BR:hsa03036]; 3.5.1.98  histone deacetylase; Class II HDACs; ','[Pfam: Hist_deacetyl HDAC4_Gln]'),('79885','hsa:79885','[HDAC11, HD11]','[(RefSeq) histone deacetylase 11]','[histone deacetylase 11 [EC:3.5.1.98]]','[Homo sapiens (human)]','[Neutrophil extracellular trap formation, Alcoholism, Viral carcinogenesis]','[]','[]','04613 Neutrophil extracellular trap formation; 05203 Viral carcinogenesis; 05034 Alcoholism; 03036 Chromosome and associated proteins [BR:hsa03036]; 3.5.1.98  histone deacetylase; Class IV HDACs; ','[Pfam: Hist_deacetyl]'),('7100','hsa:7100','[TLR5, MELIOS, SLE1, SLEB1, TIL3]','[(RefSeq) toll like receptor 5]','[toll-like receptor 5]','[Homo sapiens (human)]','[Toll-like receptor signaling pathway, Pathogenic Escherichia coli infection, Shigellosis, Salmonella infection, Legionellosis, Inflammatory bowel disease]','[]','[]','04620 Toll-like receptor signaling pathway; 05130 Pathogenic Escherichia coli infection; 05132 Salmonella infection; 05131 Shigellosis; 05134 Legionellosis; 05321 Inflammatory bowel disease; 04054 Pattern recognition receptors [BR:hsa04054]; Toll-like receptors; ','[Pfam: LRR_8 TIR LRR_4 LRR_5 TIR_2 LRR_1 LRR_9 CoV_E]'),('7097','hsa:7097','[TLR2, CD282, TIL4]','[(RefSeq) toll like receptor 2]','[toll-like receptor 2]','[Homo sapiens (human)]','[Phagosome, PI3K-Akt signaling pathway, Neutrophil extracellular trap formation, Toll-like receptor signaling pathway, Salmonella infection, Legionellosis, Leishmaniasis, Chagas disease, Malaria, Toxoplasmosis, Amoebiasis, Tuberculosis, Hepatitis B, Measles, Herpes simplex virus 1 infection, Epstein-Barr virus infection, Human immunodeficiency virus 1 infection, Coronavirus disease - COVID-19, Proteoglycans in cancer, PD-L1 expression and PD-1 checkpoint pathway in cancer, Inflammatory bowel disease, Rheumatoid arthritis, Lipid and atherosclerosis]','[c(\"nt06121  TLR signaling (viruses and bacteria)\", \"nt06161  Human immunodeficiency virus type 1 (HIV-1)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06165  Epstein-Barr virus (EBV)\", \"nt06168  Herpes simplex virus 1 (HSV-1)\", \"nt06169  Measles virus (MV)\", \"nt06263  Hepatocellular carcinoma\")]','[N00435  TLR2/4-NFKB signaling pathway, N00438  TLR2/4-MAPK signaling pathway, N00556  HBV HBe to TLR2/4-NFKB signaling pathway]','04151 PI3K-Akt signaling pathway; 04145 Phagosome; 04613 Neutrophil extracellular trap formation; 04620 Toll-like receptor signaling pathway; 05205 Proteoglycans in cancer; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05171 Coronavirus disease - COVID-19; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05169 Epstein-Barr virus infection; 05132 Salmonella infection; 05134 Legionellosis; 05152 Tuberculosis; 05146 Amoebiasis; 05144 Malaria; 05145 Toxoplasmosis; 05140 Leishmaniasis; 05142 Chagas disease; 05323 Rheumatoid arthritis; 05321 Inflammatory bowel disease; 05417 Lipid and atherosclerosis; 04131 Membrane trafficking [BR:hsa04131]; 04147 Exosome [BR:hsa04147]; 04054 Pattern recognition receptors [BR:hsa04054]; 04090 CD molecules [BR:hsa04090]; 00536 Glycosaminoglycan binding proteins [BR:hsa00536]; Toll-like receptors; Exosomal proteins of breast milk; Toll-like receptors; Proteins; Cell surface receptors; ','[Pfam: LRR_8 TIR LRR_4 TIR_2 LRR_9 LRR_5 LRR_1 LRRCT]'),('7099','hsa:7099','[TLR4, ARMD10, CD284, TLR-4, TOLL]','[(RefSeq) toll like receptor 4]','[toll-like receptor 4]','[Homo sapiens (human)]','[NF-kappa B signaling pathway, HIF-1 signaling pathway, Phagosome, PI3K-Akt signaling pathway, Necroptosis, Neutrophil extracellular trap formation, Toll-like receptor signaling pathway, NOD-like receptor signaling pathway, Alcoholic liver disease, Pathogenic Escherichia coli infection, Shigellosis, Salmonella infection, Pertussis, Legionellosis, Yersinia infection, Leishmaniasis, Chagas disease, Malaria, Toxoplasmosis, Amoebiasis, Tuberculosis, Hepatitis B, Measles, Influenza A, Human immunodeficiency virus 1 infection, Coronavirus disease - COVID-19, Proteoglycans in cancer, PD-L1 expression and PD-1 checkpoint pathway in cancer, Inflammatory bowel disease, Rheumatoid arthritis, Lipid and atherosclerosis]','[c(\"nt06121  TLR signaling (viruses and bacteria)\", \"nt06161  Human immunodeficiency virus type 1 (HIV-1)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06165  Epstein-Barr virus (EBV)\", \"nt06168  Herpes simplex virus 1 (HSV-1)\", \"nt06169  Measles virus (MV)\", \"nt06263  Hepatocellular carcinoma\")]','[N00435  TLR2/4-NFKB signaling pathway, N00438  TLR2/4-MAPK signaling pathway, N00553  TLR4-IRF3/7 signaling pathway]','04064 NF-kappa B signaling pathway; 04066 HIF-1 signaling pathway; 04151 PI3K-Akt signaling pathway; 04145 Phagosome; 04217 Necroptosis; 04613 Neutrophil extracellular trap formation; 04620 Toll-like receptor signaling pathway; 04621 NOD-like receptor signaling pathway; 05205 Proteoglycans in cancer; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05162 Measles; 05130 Pathogenic Escherichia coli infection; 05132 Salmonella infection; 05131 Shigellosis; 05135 Yersinia infection; 05133 Pertussis; 05134 Legionellosis; 05152 Tuberculosis; 05146 Amoebiasis; 05144 Malaria; 05145 Toxoplasmosis; 05140 Leishmaniasis; 05142 Chagas disease; 05323 Rheumatoid arthritis; 05321 Inflammatory bowel disease; 05417 Lipid and atherosclerosis; 04936 Alcoholic liver disease; 04131 Membrane trafficking [BR:hsa04131]; 04054 Pattern recognition receptors [BR:hsa04054]; 04090 CD molecules [BR:hsa04090]; 00536 Glycosaminoglycan binding proteins [BR:hsa00536]; Toll-like receptors; Toll-like receptors; Proteins; Cell surface receptors; ','[Pfam: LRR_8 TIR LRR_4 TIR_2 LRR_9 LRR_1 LRR_6 DUF1863]'),('7082','hsa:7082','[TJP1, ZO-1]','[(RefSeq) tight junction protein 1]','[tight junction protein 1]','[Homo sapiens (human)]','[Adherens junction, Tight junction, Gap junction, Vibrio cholerae infection, Epithelial cell signaling in Helicobacter pylori infection, Pathogenic Escherichia coli infection]','[nt06135  Cytoskeletal regulation (viruses and bacteria)]','[N01287  Tight junction-Actin signaling pathway, N01288  Escherichia EspF to Tight junction-Actin signaling pathway]','04520 Adherens junction; 04530 Tight junction; 04540 Gap junction; 05110 Vibrio cholerae infection; 05120 Epithelial cell signaling in Helicobacter pylori infection; 05130 Pathogenic Escherichia coli infection; 04812 Cytoskeleton proteins [BR:hsa04812]; Others; ','[Pfam: PDZ ZU5 PDZ_6 PDZ_2 Peptidase_M50 Guanylate_kin SH3_2 DUF6288 SH3_1 RPOL_N]'),('9076','hsa:9076','[CLDN1, CLD1, ILVASC, SEMP1]','[(RefSeq) claudin 1]','[claudin]','[Homo sapiens (human)]','[Cell adhesion molecules, Tight junction, Leukocyte transendothelial migration, Pathogenic Escherichia coli infection, Hepatitis C]','[nt06135  Cytoskeletal regulation (viruses and bacteria)]','[N01287  Tight junction-Actin signaling pathway]','04514 Cell adhesion molecules; 04530 Tight junction; 04670 Leukocyte transendothelial migration; 05160 Hepatitis C; 05130 Pathogenic Escherichia coli infection; 03037 Cilium and associated proteins [BR:hsa03037]; 04147 Exosome [BR:hsa04147]; Stereociliary proteins; Exosomal proteins of ovarian cancer cells; Exosomal proteins of colorectal cancer cells; ','[Pfam: PMP22_Claudin Claudin_2 Amastin Cytochrom_B_C SUR7 DUF2613 YccF DUF4282 CD20]'),('100506658','hsa:100506658','[OCLN, BLCPMG, PPP1R115, PTORCH1]','[(RefSeq) occludin]','[occludin]','[Homo sapiens (human)]','[Cell adhesion molecules, Tight junction, Leukocyte transendothelial migration, Pathogenic Escherichia coli infection, Hepatitis C]','[nt06135  Cytoskeletal regulation (viruses and bacteria)]','[N01287  Tight junction-Actin signaling pathway]','04514 Cell adhesion molecules; 04530 Tight junction; 04670 Leukocyte transendothelial migration; 05160 Hepatitis C; 05130 Pathogenic Escherichia coli infection; 01009 Protein phosphatases and associated proteins [BR:hsa01009]; PP1-interacting proteins (PIPs); ','[Pfam: MARVEL Occludin_ELL Laminin_II HBM FapA FadA DUF4164 APG6_N TMF_TATA_bd KxDL AAA_23 UPF0184 Swi5]'),('57530','hsa:57530','[CGN]','[(RefSeq) cingulin]','[cingulin]','[Homo sapiens (human)]','[Tight junction]','[]','[]','04530 Tight junction; ','[Pfam: Myosin_tail_1 Nuf2_DHR10-like ADIP DUF1664]'),('1673','hsa:1673','[DEFB4A, BD-2, DEFB-2, DEFB102, DEFB2, DEFB4, HBD-2, SAP1]','[(RefSeq) defensin beta 4A]','[beta-defensin 4]','[Homo sapiens (human)]','[NOD-like receptor signaling pathway, IL-17 signaling pathway, Staphylococcus aureus infection]','[]','[]','04621 NOD-like receptor signaling pathway; 04657 IL-17 signaling pathway; 05150 Staphylococcus aureus infection; 02000 Transporters [BR:hsa02000]; Pores ion channels [TC:1]; ','[Pfam: Defensin_beta]'),('9356','hsa:9356','[SLC22A6, HOAT1, OAT1, PAHT, ROAT1]','[(RefSeq) solute carrier family 22 member 6]','[MFS transporter, OCT family, solute carrier family 22 (organic anion transporter), member 6]','[Homo sapiens (human)]',NULL,'[]','[]','02000 Transporters [BR:hsa02000]; SLC22: Organic cation/anion/zwitterion transporter; Organic cation transporter (OCT) family [TC:2.A.1.19]; ','[Pfam: Sugar_tr MFS_1 VIT1]'),('929','hsa:929','[CD14]','[(RefSeq) CD14 molecule]','[monocyte differentiation antigen CD14]','[Homo sapiens (human)]','[MAPK signaling pathway, NF-kappa B signaling pathway, Phagosome, Toll-like receptor signaling pathway, Hematopoietic cell lineage, Alcoholic liver disease, Shigellosis, Salmonella infection, Pertussis, Legionellosis, Amoebiasis, Tuberculosis, Transcriptional misregulation in cancer, Acute myeloid leukemia, Lipid and atherosclerosis]','[c(\"nt06240  Transcription\", \"nt06275  Acute myeloid leukemia\")]','[N00112  AML1-ETO fusion to PU.1-mediated transcription]','04010 MAPK signaling pathway; 04064 NF-kappa B signaling pathway; 04145 Phagosome; 04640 Hematopoietic cell lineage; 04620 Toll-like receptor signaling pathway; 05202 Transcriptional misregulation in cancer; 05221 Acute myeloid leukemia; 05132 Salmonella infection; 05131 Shigellosis; 05133 Pertussis; 05134 Legionellosis; 05152 Tuberculosis; 05146 Amoebiasis; 05417 Lipid and atherosclerosis; 04936 Alcoholic liver disease; 04147 Exosome [BR:hsa04147]; 04090 CD molecules [BR:hsa04090]; 00537 Glycosylphosphatidylinositol (GPI)-anchored proteins [BR:hsa00537]; Exosomal proteins of microglial cells; Proteins; Receptors; ','[Pfam: LRR_4 LRR_8 LRR_6]'),('9332','hsa:9332','[CD163, M130, MM130, SCARI1]','[(RefSeq) CD163 molecule]','[CD163 antigen]','[Homo sapiens (human)]',NULL,'[]','[]','04090 CD molecules [BR:hsa04090]; Proteins; ','[Pfam: SRCR SRCR_2]'),('51129','hsa:51129','[ANGPTL4, ARP4, FIAF, HARP, HFARP, NL2, PGAR, TGQTL, UNQ171, pp1158]','[(RefSeq) angiopoietin like 4]','[angiopoietin-like 4]','[Homo sapiens (human)]','[PPAR signaling pathway, Cholesterol metabolism]','[]','[]','03320 PPAR signaling pathway; 04979 Cholesterol metabolism; 04147 Exosome [BR:hsa04147]; Exosomal proteins of breast milk; ','[Pfam: Fibrinogen_C SlyX HeLo Exonuc_VII_L ATG14 DivIC Helo_like_N]'),('1364','hsa:1364','[CLDN4, CPE-R, CPER, CPETR, CPETR1, WBSCR8, hCPE-R]','[(RefSeq) claudin 4]','[claudin]','[Homo sapiens (human)]','[Cell adhesion molecules, Tight junction, Leukocyte transendothelial migration, Pathogenic Escherichia coli infection, Hepatitis C]','[nt06135  Cytoskeletal regulation (viruses and bacteria)]','[N01287  Tight junction-Actin signaling pathway]','04514 Cell adhesion molecules; 04530 Tight junction; 04670 Leukocyte transendothelial migration; 05160 Hepatitis C; 05130 Pathogenic Escherichia coli infection; 03037 Cilium and associated proteins [BR:hsa03037]; 04147 Exosome [BR:hsa04147]; Stereociliary proteins; Exosomal proteins of ovarian cancer cells; Exosomal proteins of colorectal cancer cells; ','[Pfam: PMP22_Claudin Claudin_2 TMEM37 TMEM_230_134 DUF4131 Amastin]'),('50848','hsa:50848','[F11R, CD321, JAM, JAM1, JAMA, JCAM, KAT, PAM-1]','[(RefSeq) F11 receptor]','[junctional adhesion molecule 1]','[Homo sapiens (human)]','[Cell adhesion molecules, Tight junction, Leukocyte transendothelial migration, Epithelial cell signaling in Helicobacter pylori infection]','[]','[]','04514 Cell adhesion molecules; 04530 Tight junction; 04670 Leukocyte transendothelial migration; 05120 Epithelial cell signaling in Helicobacter pylori infection; 04090 CD molecules [BR:hsa04090]; Proteins; ','[Pfam: Ig_3 ig I-set Ig_2 V-set C1-set C2-set_2 DUF5626 Ig_C17orf99 Adeno_E3_CR1 V-set_CD47]'),('7040','hsa:7040','[TGFB1, CED, DPD1, IBDIMDE, LAP, TGF-beta1, TGFB, TGFbeta]','[(RefSeq) transforming growth factor beta 1]','[transforming growth factor beta-1]','[Homo sapiens (human)]','[MAPK signaling pathway, Cytokine-cytokine receptor interaction, FoxO signaling pathway, Cell cycle, Cellular senescence, TGF-beta signaling pathway, Osteoclast differentiation, Hippo signaling pathway, Th17 cell differentiation, Intestinal immune network for IgA production, Relaxin signaling pathway, Non-alcoholic fatty liver disease, AGE-RAGE signaling pathway in diabetic complications, Leishmaniasis, Chagas disease, Malaria, Toxoplasmosis, Amoebiasis, Tuberculosis, Hepatitis B, Human T-cell leukemia virus 1 infection, Pathways in cancer, Proteoglycans in cancer, Colorectal cancer, Renal cell carcinoma, Pancreatic cancer, Chronic myeloid leukemia, Hepatocellular carcinoma, Gastric cancer, Inflammatory bowel disease, Rheumatoid arthritis, Hypertrophic cardiomyopathy, Dilated cardiomyopathy, Diabetic cardiomyopathy]','[c(\"nt06118  TGFB signaling (viruses)\", \"nt06160  Human T-cell leukemia virus 1 (HTLV-1)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06218  TGFB signaling\", \"nt06225  HIF-1 signaling\", \"nt06260  Colorectal cancer\", \"nt06261  Gastric cancer\", \"nt06263  Hepatocellular carcinoma\", \"nt06264  Renal cell carcinoma\", \"nt06276  Chronic myeloid leukemia\")]','[N00063  TGF-beta signaling pathway, N00079  HIF-1 signaling pathway, N00080  Loss of VHL to HIF-1 signaling pathway, N00081  Mutation-inactivated VHL to HIF-1 signaling pathway, N00531  HBV HBx to TGF-beta signaling pathway]','04010 MAPK signaling pathway; 04350 TGF-beta signaling pathway; 04390 Hippo signaling pathway; 04068 FoxO signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04110 Cell cycle; 04218 Cellular senescence; 04659 Th17 cell differentiation; 04672 Intestinal immune network for IgA production; 04926 Relaxin signaling pathway; 04380 Osteoclast differentiation; 05200 Pathways in cancer; 05205 Proteoglycans in cancer; 05210 Colorectal cancer; 05212 Pancreatic cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05220 Chronic myeloid leukemia; 05211 Renal cell carcinoma; 05166 Human T-cell leukemia virus 1 infection; 05161 Hepatitis B; 05152 Tuberculosis; 05146 Amoebiasis; 05144 Malaria; 05145 Toxoplasmosis; 05140 Leishmaniasis; 05142 Chagas disease; 05323 Rheumatoid arthritis; 05321 Inflammatory bowel disease; 05410 Hypertrophic cardiomyopathy; 05414 Dilated cardiomyopathy; 05415 Diabetic cardiomyopathy; 04932 Non-alcoholic fatty liver disease; 04933 AGE-RAGE signaling pathway in diabetic complications; 03110 Chaperones and folding catalysts [BR:hsa03110]; 04052 Cytokines and growth factors [BR:hsa04052]; 00536 Glycosaminoglycan binding proteins [BR:hsa00536]; Others; Transforming growth factors (RSTK binding); Growth factors/receptors; ','[Pfam: TGFb_propeptide TGF_beta]'),('50943','hsa:50943','[FOXP3, AIID, DIETER, IPEX, JM2, PIDX, XPID]','[(RefSeq) forkhead box P3]','[forkhead box P3]','[Homo sapiens (human)]','[Th17 cell differentiation, Inflammatory bowel disease]','[]','[]','04659 Th17 cell differentiation; 05321 Inflammatory bowel disease; 03000 Transcription factors [BR:hsa03000]; Fork head/winged helix other regulators; ','[Pfam: FOXP-CC Forkhead]'),('196','hsa:196','[AHR, RP85, bHLHe76]','[(RefSeq) aryl hydrocarbon receptor]','[aryl hydrocarbon receptor]','[Homo sapiens (human)]','[Th17 cell differentiation, Cushing syndrome, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species]','[c(\"nt06227  Nuclear receptor signaling\", \"nt06360  Cushing syndrome\")]','[N00315  Mutation-inactivated AIP to AhR-mediated transcription, N00317  AhR signaling pathway, N01365  TCDD to Ahr signaling pathway, N01366  BaP to Ahr signaling pathway, N01367  PCB to Ahr signaling pathway, N01368  HCB to Ahr signaling pathway]','04659 Th17 cell differentiation; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 04934 Cushing syndrome; 03000 Transcription factors [BR:hsa03000]; Factors with PAS domain; ','[Pfam: PAS_3 PAS PAS_11 HLH PAS_4 PAS_9]'),('1543','hsa:1543','[CYP1A1, AHH, AHRR, CP11, CYP1, CYPIA1, P1-450, P450-C, P450DX]','[(RefSeq) cytochrome P450 family 1 subfamily A member 1]','[cytochrome P450 family 1 subfamily A1 [EC:1.14.14.1]]','[Homo sapiens (human)]','[Steroid hormone biosynthesis, Tryptophan metabolism, Retinol metabolism, Metabolism of xenobiotics by cytochrome P450, Metabolic pathways, Ovarian steroidogenesis, Chemical carcinogenesis - DNA adducts, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species, Lipid and atherosclerosis]','[c(\"nt06227  Nuclear receptor signaling\", \"nt06250  DNA adduct formation\", \"nt06251  CYP-mediated ROS formation\", \"nt06360  Cushing syndrome\")]','[N00317  AhR signaling pathway, N01365  TCDD to Ahr signaling pathway, N01366  BaP to Ahr signaling pathway, N01367  PCB to Ahr signaling pathway, N01368  HCB to Ahr signaling pathway, N01374  BaP to DNA adducts, N01375  DMBA to DNA adducts, N01401  Benzo[a]pyrenre to CYP-mediated metabolism, N01404  17beta-estradiol to CYP-mediated metabolism]','00140 Steroid hormone biosynthesis; 00380 Tryptophan metabolism; 00830 Retinol metabolism; 00980 Metabolism of xenobiotics by cytochrome P450; 04913 Ovarian steroidogenesis; 05204 Chemical carcinogenesis - DNA adducts; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 05417 Lipid and atherosclerosis; 00199 Cytochrome P450 [BR:hsa00199]; 1.14.14.1  unspecific monooxygenase; CYP1 family; ','[Pfam: p450]'),('15550','mmu:15550','[Htr1a, Gpcr18]','[(RefSeq) 5-hydroxytryptamine (serotonin) receptor 1A]','[5-hydroxytryptamine receptor 1]','[Mus musculus (house mouse)]','[cAMP signaling pathway, Neuroactive ligand-receptor interaction, Serotonergic synapse, Taste transduction]','[]','[]','04024 cAMP signaling pathway; 04080 Neuroactive ligand-receptor interaction; 04726 Serotonergic synapse; 04742 Taste transduction; 04030 G protein-coupled receptors [BR:mmu04030]; Serotonin; ','[Pfam: 7tm_1 7TM_GPCR_Srsx 7tm_4 7TM_GPCR_Srx DUF2981]'),('8549','hsa:8549','[LGR5, FEX, GPR49, GPR67, GRP49, HG38]','[(RefSeq) leucine rich repeat containing G protein-coupled receptor 5]','[leucine-rich repeat-containing G protein-coupled receptor 5]','[Homo sapiens (human)]','[Wnt signaling pathway]','[]','[]','04310 Wnt signaling pathway; 04030 G protein-coupled receptors [BR:hsa04030]; R-spondin; ','[Pfam: LRR_8 LRR_4 LRR_5 LRR_9 7tm_1 LRRNT]'),('248','hsa:248','[ALPI, IAP]','[(RefSeq) alkaline phosphatase, intestinal]','[alkaline phosphatase [EC:3.1.3.1]]','[Homo sapiens (human)]','[Thiamine metabolism, Folate biosynthesis, Metabolic pathways, Biosynthesis of cofactors]','[]','[]','00730 Thiamine metabolism; 00790 Folate biosynthesis; 04147 Exosome [BR:hsa04147]; 00537 Glycosylphosphatidylinositol (GPI)-anchored proteins [BR:hsa00537]; 3.1.3.1  alkaline phosphatase; Exosomal proteins of other body fluids (saliva and urine); Enzymes; ','[Pfam: Alk_phosphatase Metalloenzyme]'),('4069','hsa:4069','[LYZ, LYZF1, LZM]','[(RefSeq) lysozyme]','[lysozyme C [EC:3.2.1.17]]','[Homo sapiens (human)]','[Salivary secretion]','[]','[]','04970 Salivary secretion; 04147 Exosome [BR:hsa04147]; 3.2.1.17  lysozyme; Exosomal proteins of other body fluids (saliva and urine); ','[Pfam: Lys SLT SLT_3 Destabilase]'),('1113','hsa:1113','[CHGA, CGA, PHE5, PHES]','[(RefSeq) chromogranin A]','[chromogranin-A]','[Homo sapiens (human)]',NULL,'[]','[]','04131 Membrane trafficking [BR:hsa04131]; Others; ','[Pfam: Granin PP1_bind]'),('9201','hsa:9201','[DCLK1, CL1, CLICK1, DCAMKL1, DCDC3A, DCLK]','[(RefSeq) doublecortin like kinase 1]','[doublecortin-like kinase 1/2 [EC:2.7.11.1]]','[Homo sapiens (human)]',NULL,'[]','[]','01001 Protein kinases [BR:hsa01001]; 2.7.11.1  non-specific serine/threonine protein kinase; DCAMKL family; ','[Pfam: Pkinase DCX PK_Tyr_Ser-Thr Kdo Pkinase_fungal RIO1]'),('130120','hsa:130120','[REG3G, LPPM429, PAP_IB, PAP-1B, PAP1B, PAPIB, REG_III, REG-III, UNQ429]','[(RefSeq) regenerating family member 3 gamma]','[regenerating islet-derived protein 3]','[Homo sapiens (human)]',NULL,'[]','[]','99995 Signaling proteins; ','[Pfam: Lectin_C]'),('1672','hsa:1672','[DEFB1, BD1, DEFB-1, DEFB101, HBD1]','[(RefSeq) defensin beta 1]','[beta-defensin 1]','[Homo sapiens (human)]','[ABC transporters, Staphylococcus aureus infection]','[]','[]','02010 ABC transporters; 05150 Staphylococcus aureus infection; 02000 Transporters [BR:hsa02000]; 03037 Cilium and associated proteins [BR:hsa03037]; Pores ion channels [TC:1]; Sperm flagellum proteins; ','[Pfam: Defensin_beta]'),('1365','hsa:1365','[CLDN3, C7orf1, CPE-R2, CPETR2, HRVP1, RVP1]','[(RefSeq) claudin 3]','[claudin]','[Homo sapiens (human)]','[Cell adhesion molecules, Tight junction, Leukocyte transendothelial migration, Pathogenic Escherichia coli infection, Hepatitis C]','[nt06135  Cytoskeletal regulation (viruses and bacteria)]','[N01287  Tight junction-Actin signaling pathway]','04514 Cell adhesion molecules; 04530 Tight junction; 04670 Leukocyte transendothelial migration; 05160 Hepatitis C; 05130 Pathogenic Escherichia coli infection; 03037 Cilium and associated proteins [BR:hsa03037]; 04147 Exosome [BR:hsa04147]; Stereociliary proteins; Exosomal proteins of ovarian cancer cells; Exosomal proteins of colorectal cancer cells; ','[Pfam: PMP22_Claudin Claudin_2 L_HMGIC_fpl TMEM37 Phage_holin_3_6 DUF6239]'),('51738','hsa:51738','[GHRL, MTLRP]','[(RefSeq) ghrelin and obestatin prepropeptide]','[ghrelin]','[Homo sapiens (human)]','[cAMP signaling pathway, Neuroactive ligand-receptor interaction, Growth hormone synthesis, secretion and action]','[]','[]','04024 cAMP signaling pathway; 04080 Neuroactive ligand-receptor interaction; 04935 Growth hormone synthesis, secretion and action; ','[Pfam: Motilin_assoc Motilin_ghrelin]'),('3556','hsa:3556','[IL1RAP, C3orf13, IL-1RAcP, IL1R3]','[(RefSeq) interleukin 1 receptor accessory protein]','[interleukin 1 receptor accessory protein]','[Homo sapiens (human)]','[MAPK signaling pathway, Cytokine-cytokine receptor interaction, Th17 cell differentiation, Inflammatory mediator regulation of TRP channels]','[c(\"nt06110  MAPK signaling (viruses and bacteria)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06211  Other MAPK signaling\")]','[N00186  IL1-IL1R-p38 signaling pathway, N00188  IL1-IL1R-JNK signaling pathway]','04010 MAPK signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04659 Th17 cell differentiation; 04750 Inflammatory mediator regulation of TRP channels; 04050 Cytokine receptors [BR:hsa04050]; IL1 receptor family; ','[Pfam: TIR Ig_3 Ig_2 Ig_6 TIR_2 ig I-set]'),('3579','hsa:3579','[CXCR2, CD182, CDw128b, CMKAR2, IL8R2, IL8RA, IL8RB, WHIMS2]','[(RefSeq) C-X-C motif chemokine receptor 2]','[C-X-C chemokine receptor type 2]','[Homo sapiens (human)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, Chemokine signaling pathway, Phospholipase D signaling pathway, Endocytosis, Epithelial cell signaling in Helicobacter pylori infection, Human cytomegalovirus infection]','[c(\"nt06124  Chemokine signaling (viruses)\", \"nt06150  Cytokine-cytokine receptor interaction (viruses)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06167  Human cytomegalovirus (HCMV)\", \"nt06224  CXCR signaling\")]','[N00152  CXCR-GNB/G-ERK signaling pathway, N00153  CCR/CXCR-GNB/G-PI3K-RAC signaling pathway, N00154  CXCR-GNB/G-PI3K-AKT signaling pathway, N00427  HCMV vCXCL to CXCR-GNB/G-PI3K-AKT signaling pathway, N01238  CXCL1/2/3/5/6/7/8 to CXCR2 signaling, N01240  MOCV1 MC148 to CXCR2 signaling]','04072 Phospholipase D signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04144 Endocytosis; 04062 Chemokine signaling pathway; 05163 Human cytomegalovirus infection; 05120 Epithelial cell signaling in Helicobacter pylori infection; 04030 G protein-coupled receptors [BR:hsa04030]; 04050 Cytokine receptors [BR:hsa04050]; 04090 CD molecules [BR:hsa04090]; CXC Chemokine; CXC Chemokine; Proteins; ','[Pfam: 7tm_1 TMEM70]'),('338442','hsa:338442','[HCAR2, GPR109A, HCA2, HM74a, HM74b, NIACR1, PUMAG, Puma-g]','[(RefSeq) hydroxycarboxylic acid receptor 2]','[hydroxycarboxylic acid receptor 2/3]','[Homo sapiens (human)]','[cAMP signaling pathway]','[]','[]','04024 cAMP signaling pathway; 04030 G protein-coupled receptors [BR:hsa04030]; Hydroxy-carboxylic acid; ','[Pfam: 7tm_1]'),('8843','hsa:8843','[HCAR3, GPR109B, HCA3, HM74, PUMAG, Puma-g]','[(RefSeq) hydroxycarboxylic acid receptor 3]','[hydroxycarboxylic acid receptor 2/3]','[Homo sapiens (human)]','[cAMP signaling pathway]','[]','[]','04024 cAMP signaling pathway; 04030 G protein-coupled receptors [BR:hsa04030]; Hydroxy-carboxylic acid; ','[Pfam: 7tm_1]'),('4254','hsa:4254','[KITLG, DCUA, DFNA69, FPH2, FPHH, KL-1, Kitl, MGF, SCF, SF, SHEP7, SLF]','[(RefSeq) KIT ligand]','[KIT ligand]','[Homo sapiens (human)]','[MAPK signaling pathway, Ras signaling pathway, Rap1 signaling pathway, Phospholipase D signaling pathway, PI3K-Akt signaling pathway, Hematopoietic cell lineage, Melanogenesis, Pathways in cancer]','[c(\"nt06210  ERK signaling\", \"nt06214  PI3K signaling\", \"nt06275  Acute myeloid leukemia\")]','[N00045  KITLG-KIT-PI3K signaling pathway, N00215  KITLG-KIT-RAS-ERK signaling pathway]','04010 MAPK signaling pathway; 04014 Ras signaling pathway; 04015 Rap1 signaling pathway; 04072 Phospholipase D signaling pathway; 04151 PI3K-Akt signaling pathway; 04640 Hematopoietic cell lineage; 04916 Melanogenesis; 05200 Pathways in cancer; 04052 Cytokines and growth factors [BR:hsa04052]; Growth factors (RTK binding); ','[Pfam: SCF Psg1]'),('160728','hsa:160728','[SLC5A8, AIT, SMCT, SMCT1]','[(RefSeq) solute carrier family 5 member 8]','[solute carrier family 5 (sodium-coupled monocarboxylate transporter), member 8/12]','[Homo sapiens (human)]',NULL,'[]','[]','02000 Transporters [BR:hsa02000]; SLC5: Sodium glucose cotransporter; ','[Pfam: SSF]'),('1432','hsa:1432','[MAPK14, CSBP, CSBP1, CSBP2, CSPB1, EXIP, Mxi2, PRKM14, PRKM15, RK, SAPK2A, p38, p38ALPHA]','[(RefSeq) mitogen-activated protein kinase 14]','[p38 MAP kinase [EC:2.7.11.24]]','[Homo sapiens (human)]','[Endocrine resistance, MAPK signaling pathway, Rap1 signaling pathway, FoxO signaling pathway, Sphingolipid signaling pathway, Oocyte meiosis, Cellular senescence, Adrenergic signaling in cardiomyocytes, VEGF signaling pathway, Osteoclast differentiation, Signaling pathways regulating pluripotency of stem cells, Platelet activation, Neutrophil extracellular trap formation, Toll-like receptor signaling pathway, NOD-like receptor signaling pathway, RIG-I-like receptor signaling pathway, C-type lectin receptor signaling pathway, IL-17 signaling pathway, Th1 and Th2 cell differentiation, Th17 cell differentiation, T cell receptor signaling pathway, Fc epsilon RI signaling pathway, TNF signaling pathway, Leukocyte transendothelial migration, Thermogenesis, Neurotrophin signaling pathway, Retrograde endocannabinoid signaling, Dopaminergic synapse, Inflammatory mediator regulation of TRP channels, GnRH signaling pathway, Progesterone-mediated oocyte maturation, Prolactin signaling pathway, Relaxin signaling pathway, Non-alcoholic fatty liver disease, AGE-RAGE signaling pathway in diabetic complications, Growth hormone synthesis, secretion and action, Alcoholic liver disease, Amyotrophic lateral sclerosis, Prion disease, Pathways of neurodegeneration - multiple diseases, Epithelial cell signaling in Helicobacter pylori infection, Pathogenic Escherichia coli infection, Shigellosis, Salmonella infection, Pertussis, Yersinia infection, Leishmaniasis, Chagas disease, Toxoplasmosis, Tuberculosis, Hepatitis B, Human cytomegalovirus infection, Kaposi sarcoma-associated herpesvirus infection, Epstein-Barr virus infection, Human immunodeficiency virus 1 infection, Coronavirus disease - COVID-19, Proteoglycans in cancer, Chemical carcinogenesis - reactive oxygen species, PD-L1 expression and PD-1 checkpoint pathway in cancer, Diabetic cardiomyopathy, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[c(\"nt06110  MAPK signaling (viruses and bacteria)\", \"nt06121  TLR signaling (viruses and bacteria)\", \"nt06123  TNF signaling (viruses and bacteria)\", \"nt06124  Chemokine signaling (viruses)\", \"nt06161  Human immunodeficiency virus type 1 (HIV-1)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06167  Human cytomegalovirus (HCMV)\", \"nt06211  Other MAPK signaling\", \"nt06263  Hepatocellular carcinoma\")]','[N00186  IL1-IL1R-p38 signaling pathway, N00187  KSHV Kaposin B to p38 signaling pathway, N00409  HCMV UL33 to GNB/G-Rho signaling pathway, N00438  TLR2/4-MAPK signaling pathway, N00444  TNF-p38 signaling pathway, N00445  HIV Tat/Nef to TNF-p38 signaling pathway, N00931  Escherichia NleD to TNF-p38 signaling pathway, N01283  Shigella OspF to TLR2/4-MAPK signaling pathway]','04010 MAPK signaling pathway; 04015 Rap1 signaling pathway; 04370 VEGF signaling pathway; 04668 TNF signaling pathway; 04068 FoxO signaling pathway; 04071 Sphingolipid signaling pathway; 04114 Oocyte meiosis; 04218 Cellular senescence; 04550 Signaling pathways regulating pluripotency of stem cells; 04611 Platelet activation; 04613 Neutrophil extracellular trap formation; 04620 Toll-like receptor signaling pathway; 04621 NOD-like receptor signaling pathway; 04622 RIG-I-like receptor signaling pathway; 04625 C-type lectin receptor signaling pathway; 04660 T cell receptor signaling pathway; 04658 Th1 and Th2 cell differentiation; 04659 Th17 cell differentiation; 04657 IL-17 signaling pathway; 04664 Fc epsilon RI signaling pathway; 04670 Leukocyte transendothelial migration; 04912 GnRH signaling pathway; 04914 Progesterone-mediated oocyte maturation; 04917 Prolactin signaling pathway; 04926 Relaxin signaling pathway; 04935 Growth hormone synthesis, secretion and action; 04261 Adrenergic signaling in cardiomyocytes; 04728 Dopaminergic synapse; 04723 Retrograde endocannabinoid signaling; 04722 Neurotrophin signaling pathway; 04750 Inflammatory mediator regulation of TRP channels; 04380 Osteoclast differentiation; 04714 Thermogenesis; 05205 Proteoglycans in cancer; 05208 Chemical carcinogenesis - reactive oxygen species; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05171 Coronavirus disease - COVID-19; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05120 Epithelial cell signaling in Helicobacter pylori infection; 05130 Pathogenic Escherichia coli infection; 05132 Salmonella infection; 05131 Shigellosis; 05135 Yersinia infection; 05133 Pertussis; 05152 Tuberculosis; 05145 Toxoplasmosis; 05140 Leishmaniasis; 05142 Chagas disease; 05014 Amyotrophic lateral sclerosis; 05020 Prion disease; 05022 Pathways of neurodegeneration - multiple diseases; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 05415 Diabetic cardiomyopathy; 04936 Alcoholic liver disease; 04932 Non-alcoholic fatty liver disease; 04933 AGE-RAGE signaling pathway in diabetic complications; 01522 Endocrine resistance; 01001 Protein kinases [BR:hsa01001]; 2.7.11.24  mitogen-activated protein kinase; MAPK family [OT]; ','[Pfam: Pkinase PK_Tyr_Ser-Thr ABC1 Kdo APH NAD_binding_8 Kinase-like]'),('9641','hsa:9641','[IKBKE, IKK-E, IKK-i, IKKE, IKKI]','[(RefSeq) inhibitor of nuclear factor kappa B kinase subunit epsilon]','[inhibitor of nuclear factor kappa-B kinase subunit epsilon [EC:2.7.11.10]]','[Homo sapiens (human)]','[Toll-like receptor signaling pathway, NOD-like receptor signaling pathway, RIG-I-like receptor signaling pathway, Cytosolic DNA-sensing pathway, C-type lectin receptor signaling pathway, IL-17 signaling pathway, Alcoholic liver disease, Hepatitis C, Hepatitis B, Measles, Influenza A, Human papillomavirus infection, Kaposi sarcoma-associated herpesvirus infection, Herpes simplex virus 1 infection, Epstein-Barr virus infection, Coronavirus disease - COVID-19, Chemical carcinogenesis - receptor activation, Lipid and atherosclerosis]','[c(\"nt06121  TLR signaling (viruses and bacteria)\", \"nt06133  RLR signaling (viruses)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06163  Hepatitis C virus (HCV)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06165  Epstein-Barr virus (EBV)\", \"nt06166  Human papillomavirus (HPV)\", \"nt06168  Herpes simplex virus 1 (HSV-1)\", \"nt06169  Measles virus (MV)\", \"nt06170  Influenza A virus (IAV)\", \"nt06171  Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)\", \"nt06219  JAK-STAT signaling\", \n\"nt06221  TLR signaling\", \"nt06263  Hepatocellular carcinoma\")]','[N00148  TLR3-IRF7 signaling pathway, N00149  TLR3-IRF3 signaling pathway, N00469  RIG-I-IRF7/3 signaling pathway, N00552  HBV pol to TLR3-IRF3 signaling pathway, N00553  TLR4-IRF3/7 signaling pathway, N00567  HSV ICP34.5 to TBK1 signaling pathway, N00734  IAV PB1-F2/PB2 to RIG-I-like receptor signaling pathway, N01308  MDA5-IRF7/3 signaling pathway, N01341  NNK/NNN to Jak-STAT signaling pathway]','04620 Toll-like receptor signaling pathway; 04621 NOD-like receptor signaling pathway; 04622 RIG-I-like receptor signaling pathway; 04623 Cytosolic DNA-sensing pathway; 04625 C-type lectin receptor signaling pathway; 04657 IL-17 signaling pathway; 05207 Chemical carcinogenesis - receptor activation; 05161 Hepatitis B; 05160 Hepatitis C; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05165 Human papillomavirus infection; 05417 Lipid and atherosclerosis; 04936 Alcoholic liver disease; 01001 Protein kinases [BR:hsa01001]; 2.7.11.10  IkappaB kinase; IKK family [OT]; ','[Pfam: TBK1_CCD1 Pkinase TBK1_ULD PK_Tyr_Ser-Thr ABC1 Pkinase_fungal Kinase-like APH zf-RING_9 RIO1 DUF5798]'),('406938','hsa:406938','[MIR146A, MIRN146, MIRN146A, miR-146a, miRNA146A]','[(RefSeq) microRNA 146a]','[microRNA 146a]','[Homo sapiens (human)]','[MicroRNAs in cancer]','[]','[]','05206 MicroRNAs in cancer; 04147 Exosome [BR:hsa04147]; 03100 Non-coding RNAs [BR:hsa03100]; Exosomal RNA of breast milk; Exosomal RNA of renal carcinoma cells; Exosomal RNA of lung cancer cells; Micro RNA; ','[]'),('406947','hsa:406947','[MIR155, MIRN155, miRNA155, mir-155]','[(RefSeq) microRNA 155]','[microRNA 155]','[Homo sapiens (human)]','[MicroRNAs in cancer]','[]','[]','05206 MicroRNAs in cancer; 04147 Exosome [BR:hsa04147]; 03100 Non-coding RNAs [BR:hsa03100]; Exosomal RNA of breast milk; Exosomal RNA of lung cancer cells; Micro RNA; ','[]'),('3569','hsa:3569','[IL6, BSF-2, BSF2, CDF, HGF, HSF, IFN-beta-2, IFNB2, IL-6]','[(RefSeq) interleukin 6]','[interleukin 6]','[Homo sapiens (human)]','[EGFR tyrosine kinase inhibitor resistance, Antifolate resistance, Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, HIF-1 signaling pathway, FoxO signaling pathway, PI3K-Akt signaling pathway, Cellular senescence, Toll-like receptor signaling pathway, NOD-like receptor signaling pathway, Cytosolic DNA-sensing pathway, C-type lectin receptor signaling pathway, JAK-STAT signaling pathway, Hematopoietic cell lineage, IL-17 signaling pathway, Th17 cell differentiation, TNF signaling pathway, Intestinal immune network for IgA production, Insulin resistance, Non-alcoholic fatty liver disease, AGE-RAGE signaling pathway in diabetic complications, Alcoholic liver disease, Alzheimer disease, Prion disease, Pathways of neurodegeneration - multiple diseases, Pathogenic Escherichia coli infection, Salmonella infection, Pertussis, Legionellosis, Yersinia infection, Chagas disease, African trypanosomiasis, Malaria, Amoebiasis, Tuberculosis, Hepatitis B, Measles, Human cytomegalovirus infection, Influenza A, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Herpes simplex virus 1 infection, Epstein-Barr virus infection, Coronavirus disease - COVID-19, Pathways in cancer, Transcriptional misregulation in cancer, Inflammatory bowel disease, Rheumatoid arthritis, Graft-versus-host disease, Hypertrophic cardiomyopathy, Lipid and atherosclerosis]','[c(\"nt06119  JAK-STAT signaling (viruses)\", \"nt06121  TLR signaling (viruses and bacteria)\", \"nt06124  Chemokine signaling (viruses)\", \"nt06150  Cytokine-cytokine receptor interaction (viruses)\", \"nt06160  Human T-cell leukemia virus 1 (HTLV-1)\", \"nt06161  Human immunodeficiency virus type 1 (HIV-1)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06165  Epstein-Barr virus (EBV)\", \"nt06167  Human cytomegalovirus (HCMV)\", \"nt06168  Herpes simplex virus 1 (HSV-1)\", \n\"nt06169  Measles virus (MV)\", \"nt06171  Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)\", \"nt06219  JAK-STAT signaling\", \"nt06224  CXCR signaling\", \"nt06227  Nuclear receptor signaling\", \"nt06240  Transcription\", \"nt06263  Hepatocellular carcinoma\", \"nt06266  Non-small cell lung cancer\", \"nt06276  Chronic myeloid leukemia\", \"nt06360  Cushing syndrome\", \"nt06417  AGE-RAGE signaling\", \"nt06460  Alzheimer disease\")]','[N00053  Cytokine-Jak-STAT signaling pathway, N00139  FUS-DDIT3 fusion to CEBPB-mediated transcription, N00178  KSHV vGPCR to GNB/G-PI3K-JNK signaling pathway, N00317  AhR signaling pathway, N00400  HCMV US28 to GNB/G-PI3K-NFKB signaling pathway, N00435  TLR2/4-NFKB signaling pathway, N00994  AGE-RAGE signaling pathway, N00996  Mutation-caused aberrant Abeta to AGE-RAGE signaling pathway, N01266  IL6 to IL6R signaling, N01306  AngII-AT1R-NOX2 signaling pathway, N01307  SARS-CoV-2 S to AngII-AT1R-NOX2 signaling pathway]','04630 JAK-STAT signaling pathway; 04668 TNF signaling pathway; 04066 HIF-1 signaling pathway; 04068 FoxO signaling pathway; 04151 PI3K-Akt signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04218 Cellular senescence; 04640 Hematopoietic cell lineage; 04620 Toll-like receptor signaling pathway; 04621 NOD-like receptor signaling pathway; 04623 Cytosolic DNA-sensing pathway; 04625 C-type lectin receptor signaling pathway; 04659 Th17 cell differentiation; 04657 IL-17 signaling pathway; 04672 Intestinal immune network for IgA production; 05200 Pathways in cancer; 05202 Transcriptional misregulation in cancer; 05166 Human T-cell leukemia virus 1 infection; 05161 Hepatitis B; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05130 Pathogenic Escherichia coli infection; 05132 Salmonella infection; 05135 Yersinia infection; 05133 Pertussis; 05134 Legionellosis; 05152 Tuberculosis; 05146 Amoebiasis; 05144 Malaria; 05142 Chagas disease; 05143 African trypanosomiasis; 05323 Rheumatoid arthritis; 05321 Inflammatory bowel disease; 05332 Graft-versus-host disease; 05010 Alzheimer disease; 05020 Prion disease; 05022 Pathways of neurodegeneration - multiple diseases; 05417 Lipid and atherosclerosis; 05410 Hypertrophic cardiomyopathy; 04936 Alcoholic liver disease; 04932 Non-alcoholic fatty liver disease; 04931 Insulin resistance; 04933 AGE-RAGE signaling pathway in diabetic complications; 01521 EGFR tyrosine kinase inhibitor resistance; 01523 Antifolate resistance; 04052 Cytokines and growth factors [BR:hsa04052]; Interleukins; ','[Pfam: IL6 IL11 IL23 Phage_int_SAM_1 GCSF]'),('2932','hsa:2932','[GSK3B]','[(RefSeq) glycogen synthase kinase 3 beta]','[glycogen synthase kinase 3 beta [EC:2.7.11.26]]','[Homo sapiens (human)]','[EGFR tyrosine kinase inhibitor resistance, ErbB signaling pathway, Chemokine signaling pathway, Cell cycle, mTOR signaling pathway, PI3K-Akt signaling pathway, Wnt signaling pathway, Hedgehog signaling pathway, Axon guidance, Hippo signaling pathway, Focal adhesion, Signaling pathways regulating pluripotency of stem cells, IL-17 signaling pathway, T cell receptor signaling pathway, B cell receptor signaling pathway, Neurotrophin signaling pathway, Dopaminergic synapse, Insulin signaling pathway, Melanogenesis, Prolactin signaling pathway, Thyroid hormone signaling pathway, Insulin resistance, Non-alcoholic fatty liver disease, Cushing syndrome, Growth hormone synthesis, secretion and action, Alcoholic liver disease, Alzheimer disease, Prion disease, Pathways of neurodegeneration - multiple diseases, Shigellosis, Yersinia infection, Hepatitis C, Measles, Human cytomegalovirus infection, Human papillomavirus infection, Kaposi sarcoma-associated herpesvirus infection, Pathways in cancer, Colorectal cancer, Endometrial cancer, Prostate cancer, Basal cell carcinoma, Breast cancer, Hepatocellular carcinoma, Gastric cancer, Diabetic cardiomyopathy, Lipid and atherosclerosis]','[c(\"nt06114  PI3K signaling (viruses)\", \"nt06115  WNT signaling (viruses and bacteria)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06169  Measles virus (MV)\", \"nt06215  WNT signaling\", \"nt06260  Colorectal cancer\", \"nt06261  Gastric cancer\", \"nt06263  Hepatocellular carcinoma\", \"nt06270  Breast cancer\", \"nt06271  Endometrial cancer\", \"nt06274  Thyroid cancer\", \"nt06419  Microtubule-based transport\", \"nt06460  Alzheimer disease\", \"nt06465  Prion disease\")]','[N00056  Wnt signaling pathway, N00057  Mutation-inactivated APC to Wnt signaling pathway, N00058  Mutation-activated CTNNB1 to Wnt signaling pathway, N00059  FZD7-overexpression to Wnt signaling pathway, N00060  LRP6-overexpression to Wnt signaling pathway, N00175  KSHV LANA to Wnt signaling pathway, N00242  Mutation-inactivated AXIN to Wnt signaling pathway, N00683  CD80/CD86-CD28-PI3K signaling pathway, N01017  Mutation-caused aberrant PSEN1 to anterograde axonal transport, N01202  Oligomeric conformation PrPc to anterograde axonal transport, N01403  Zn to anterograde axonal transport]','04012 ErbB signaling pathway; 04310 Wnt signaling pathway; 04340 Hedgehog signaling pathway; 04390 Hippo signaling pathway; 04151 PI3K-Akt signaling pathway; 04150 mTOR signaling pathway; 04110 Cell cycle; 04510 Focal adhesion; 04550 Signaling pathways regulating pluripotency of stem cells; 04660 T cell receptor signaling pathway; 04657 IL-17 signaling pathway; 04662 B cell receptor signaling pathway; 04062 Chemokine signaling pathway; 04910 Insulin signaling pathway; 04917 Prolactin signaling pathway; 04935 Growth hormone synthesis, secretion and action; 04919 Thyroid hormone signaling pathway; 04916 Melanogenesis; 04728 Dopaminergic synapse; 04722 Neurotrophin signaling pathway; 04360 Axon guidance; 05200 Pathways in cancer; 05210 Colorectal cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05217 Basal cell carcinoma; 05215 Prostate cancer; 05213 Endometrial cancer; 05224 Breast cancer; 05160 Hepatitis C; 05162 Measles; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05165 Human papillomavirus infection; 05131 Shigellosis; 05135 Yersinia infection; 05010 Alzheimer disease; 05020 Prion disease; 05022 Pathways of neurodegeneration - multiple diseases; 05417 Lipid and atherosclerosis; 05415 Diabetic cardiomyopathy; 04936 Alcoholic liver disease; 04932 Non-alcoholic fatty liver disease; 04931 Insulin resistance; 04934 Cushing syndrome; 01521 EGFR tyrosine kinase inhibitor resistance; 01001 Protein kinases [BR:hsa01001]; 03036 Chromosome and associated proteins [BR:hsa03036]; 2.7.11.26  tau-protein kinase; GSK family [OT]; Kinases and associated factors; ','[Pfam: Pkinase PK_Tyr_Ser-Thr ABC1 Kinase-like APH Kdo]'),('6934','hsa:6934','[TCF7L2, TCF-4, TCF4]','[(RefSeq) transcription factor 7 like 2]','[transcription factor 7-like 2]','[Homo sapiens (human)]','[Wnt signaling pathway, Hippo signaling pathway, Adherens junction, Melanogenesis, Cushing syndrome, Alcoholic liver disease, Salmonella infection, Human papillomavirus infection, Kaposi sarcoma-associated herpesvirus infection, Pathways in cancer, Colorectal cancer, Endometrial cancer, Prostate cancer, Thyroid cancer, Basal cell carcinoma, Acute myeloid leukemia, Breast cancer, Hepatocellular carcinoma, Gastric cancer, Arrhythmogenic right ventricular cardiomyopathy]','[c(\"nt06115  WNT signaling (viruses and bacteria)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06215  WNT signaling\", \"nt06260  Colorectal cancer\", \"nt06261  Gastric cancer\", \"nt06263  Hepatocellular carcinoma\", \"nt06270  Breast cancer\", \"nt06271  Endometrial cancer\", \"nt06274  Thyroid cancer\")]','[N00056  Wnt signaling pathway, N00057  Mutation-inactivated APC to Wnt signaling pathway, N00058  Mutation-activated CTNNB1 to Wnt signaling pathway, N00059  FZD7-overexpression to Wnt signaling pathway, N00060  LRP6-overexpression to Wnt signaling pathway, N00061  CDH1-reduced expression to beta-catenin signaling pathway, N00175  KSHV LANA to Wnt signaling pathway, N00242  Mutation-inactivated AXIN to Wnt signaling pathway, N00257  Loss of CDH1 to beta-catenin signaling pathway, N00258  Mutation-inactivated CDH1 to beta-catenin signaling pathway, N01124  Salmonella AvrA to beta-catenin signaling pathway]','04310 Wnt signaling pathway; 04390 Hippo signaling pathway; 04520 Adherens junction; 04916 Melanogenesis; 05200 Pathways in cancer; 05210 Colorectal cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05216 Thyroid cancer; 05221 Acute myeloid leukemia; 05217 Basal cell carcinoma; 05215 Prostate cancer; 05213 Endometrial cancer; 05224 Breast cancer; 05167 Kaposi sarcoma-associated herpesvirus infection; 05165 Human papillomavirus infection; 05132 Salmonella infection; 05412 Arrhythmogenic right ventricular cardiomyopathy; 04936 Alcoholic liver disease; 04934 Cushing syndrome; 03000 Transcription factors [BR:hsa03000]; HMG TCF-1; ','[Pfam: CTNNB1_binding HMG_box HMG_box_2]'),('595','hsa:595','[CCND1, BCL1, D11S287E, PRAD1, U21B31]','[(RefSeq) cyclin D1]','[G1/S-specific cyclin-D1]','[Homo sapiens (human)]','[Endocrine resistance, FoxO signaling pathway, Cell cycle, p53 signaling pathway, PI3K-Akt signaling pathway, AMPK signaling pathway, Cellular senescence, Wnt signaling pathway, Hedgehog signaling pathway, Apelin signaling pathway, Hippo signaling pathway, Focal adhesion, Tight junction, JAK-STAT signaling pathway, Prolactin signaling pathway, Thyroid hormone signaling pathway, Oxytocin signaling pathway, AGE-RAGE signaling pathway in diabetic complications, Cushing syndrome, Alcoholic liver disease, Hepatitis C, Measles, Human cytomegalovirus infection, Human papillomavirus infection, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Epstein-Barr virus infection, Pathways in cancer, Viral carcinogenesis, Proteoglycans in cancer, MicroRNAs in cancer, Chemical carcinogenesis - receptor activation, Colorectal cancer, Pancreatic cancer, Endometrial cancer, Glioma, Prostate cancer, Thyroid cancer, Melanoma, Bladder cancer, Chronic myeloid leukemia, Acute myeloid leukemia, Small cell lung cancer, Non-small cell lung cancer, Breast cancer, Hepatocellular carcinoma, Gastric cancer, Viral myocarditis]','[c(\"nt06110  MAPK signaling (viruses and bacteria)\", \"nt06115  WNT signaling (viruses and bacteria)\", \"nt06130  Cell cycle (viruses)\", \"nt06140  Transcription (viruses)\", \"nt06160  Human T-cell leukemia virus 1 (HTLV-1)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06163  Hepatitis C virus (HCV)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06165  Epstein-Barr virus (EBV)\", \"nt06166  Human papillomavirus (HPV)\", \"nt06167  Human cytomegalovirus (HCMV)\", \"nt06170  Influenza A virus (IAV)\", \n\"nt06210  ERK signaling\", \"nt06215  WNT signaling\", \"nt06219  JAK-STAT signaling\", \"nt06220  Calcium signaling\", \"nt06227  Nuclear receptor signaling\", \"nt06230  Cell cycle\", \"nt06260  Colorectal cancer\", \"nt06261  Gastric cancer\", \"nt06262  Pancreatic cancer\", \"nt06263  Hepatocellular carcinoma\", \"nt06264  Renal cell carcinoma\", \"nt06265  Bladder cancer\", \"nt06266  Non-small cell lung cancer\", \"nt06267  Small cell lung cancer\", \"nt06268  Melanoma\", \"nt06270  Breast cancer\", \"nt06271  Endometrial cancer\", \n\"nt06273  Glioma\", \"nt06274  Thyroid cancer\", \"nt06275  Acute myeloid leukemia\", \"nt06276  Chronic myeloid leukemia\")]','[N00001  EGF-EGFR-RAS-ERK signaling pathway, N00005  Mutation-activated MET to RAS-ERK signaling pathway, N00007  EML4-ALK fusion kinase to RAS-ERK signaling pathway, N00012  Mutation-activated KRAS/NRAS to ERK signaling pathway, N00014  Mutation-activated EGFR to RAS-ERK signaling pathway, N00023  EGF-EGFR-PLCG-ERK signaling pathway, N00024  Mutation-activated EGFR to PLCG-ERK signaling pathway, N00025  EML4-ALK fusion kinase to PLCG-ERK signaling pathway, N00056  Wnt signaling pathway, N00057  Mutation-inactivated APC to Wnt signaling pathway, N00058  Mutation-activated CTNNB1 to Wnt signaling pathway, N00059  FZD7-overexpression to Wnt signaling pathway, N00060  LRP6-overexpression to Wnt signaling pathway, N00061  CDH1-reduced expression to beta-catenin signaling pathway, N00066  MDM2-p21-Cell cycle G1/S, N00067  Deleted p14(ARF) to p21-cell cycle G1/S, N00068  Amplified MDM2 to p21-cell cycle G1/S, N00069  p16-Cell cycle G1/S, N00070  Mutation-inactivated p16(INK4a) to p16-cell cycle G1/S, N00071  Deleted p16(INK4a) to p16-cell cycle G1/S, N00072  Amplified CDK4 to cell cycle G1/S, N00073  Mutation-activated CDK4 to cell cycle G1/S, N00076  Mutation-inactivated p14(ARF) to p21-cell cycle G1/S, N00088  Amplified MYC to p15-cell cycle G1/S, N00089  Amplified MYC to cell cycle G1/S, N00090  p15-Cell cycle G1/S, N00175  KSHV LANA to Wnt signaling pathway, N00242  Mutation-inactivated AXIN to Wnt signaling pathway, N00257  Loss of CDH1 to beta-catenin signaling pathway, N00258  Mutation-inactivated CDH1 to beta-catenin signaling pathway, N00275  Amplified CCND1 to cell cycle G1/S, N00287  ESR1-positive to nuclear-initiated estrogen signaling pathway, N00347  p300-p21-Cell cycle G1/S, N00483  EBV EBNA3C to cell cycle G1/S, N00510  HTLV-1 Tax to CREB-mediated transcription, N00513  Mutation-activated EGFR to RAS-ERK signaling pathway, N00529  HCV core to RXRA/PPARA-mediated transcription, N00530  HCV core to RXRA/LXRA-mediated transcription, N01342  Nicotine to Jak-STAT signaling pathway, N01360  P4-PR-RAS-ERK signaling pathway, N01361  P4/MPA to PR-RAS-ERK signaling pathway, N01362  Nuclear-initiated progesterone signaling pathway]','04310 Wnt signaling pathway; 04340 Hedgehog signaling pathway; 04390 Hippo signaling pathway; 04371 Apelin signaling pathway; 04630 JAK-STAT signaling pathway; 04068 FoxO signaling pathway; 04151 PI3K-Akt signaling pathway; 04152 AMPK signaling pathway; 04110 Cell cycle; 04115 p53 signaling pathway; 04218 Cellular senescence; 04510 Focal adhesion; 04530 Tight junction; 04917 Prolactin signaling pathway; 04921 Oxytocin signaling pathway; 04919 Thyroid hormone signaling pathway; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05205 Proteoglycans in cancer; 05207 Chemical carcinogenesis - receptor activation; 05203 Viral carcinogenesis; 05210 Colorectal cancer; 05212 Pancreatic cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05214 Glioma; 05216 Thyroid cancer; 05221 Acute myeloid leukemia; 05220 Chronic myeloid leukemia; 05218 Melanoma; 05219 Bladder cancer; 05215 Prostate cancer; 05213 Endometrial cancer; 05224 Breast cancer; 05222 Small cell lung cancer; 05223 Non-small cell lung cancer; 05166 Human T-cell leukemia virus 1 infection; 05160 Hepatitis C; 05162 Measles; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05165 Human papillomavirus infection; 05416 Viral myocarditis; 04936 Alcoholic liver disease; 04933 AGE-RAGE signaling pathway in diabetic complications; 04934 Cushing syndrome; 01522 Endocrine resistance; ','[Pfam: Cyclin_N Cyclin_C]'),('4609','hsa:4609','[MYC, MRTL, MYCC, bHLHe39, c-Myc]','[(RefSeq) MYC proto-oncogene, bHLH transcription factor]','[Myc proto-oncogene protein]','[Homo sapiens (human)]','[MAPK signaling pathway, ErbB signaling pathway, Cell cycle, PI3K-Akt signaling pathway, Cellular senescence, Wnt signaling pathway, TGF-beta signaling pathway, Hippo signaling pathway, Signaling pathways regulating pluripotency of stem cells, JAK-STAT signaling pathway, Thyroid hormone signaling pathway, Salmonella infection, Hepatitis C, Hepatitis B, Human cytomegalovirus infection, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Epstein-Barr virus infection, Pathways in cancer, Transcriptional misregulation in cancer, Proteoglycans in cancer, MicroRNAs in cancer, Chemical carcinogenesis - receptor activation, Colorectal cancer, Endometrial cancer, Thyroid cancer, Bladder cancer, Chronic myeloid leukemia, Acute myeloid leukemia, Small cell lung cancer, Breast cancer, Hepatocellular carcinoma, Gastric cancer, Central carbon metabolism in cancer]','[c(\"nt06115  WNT signaling (viruses and bacteria)\", \"nt06130  Cell cycle (viruses)\", \"nt06140  Transcription (viruses)\", \"nt06160  Human T-cell leukemia virus 1 (HTLV-1)\", \"nt06163  Hepatitis C virus (HCV)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06165  Epstein-Barr virus (EBV)\", \"nt06210  ERK signaling\", \"nt06215  WNT signaling\", \"nt06227  Nuclear receptor signaling\", \"nt06230  Cell cycle\", \"nt06240  Transcription\", \"nt06260  Colorectal cancer\", \"nt06261  Gastric cancer\", \"nt06263  Hepatocellular carcinoma\", \n\"nt06265  Bladder cancer\", \"nt06267  Small cell lung cancer\", \"nt06270  Breast cancer\", \"nt06271  Endometrial cancer\", \"nt06274  Thyroid cancer\", \"nt06275  Acute myeloid leukemia\")]','[N00011  Mutation-activated FGFR3 to RAS-ERK signaling pathway, N00056  Wnt signaling pathway, N00057  Mutation-inactivated APC to Wnt signaling pathway, N00058  Mutation-activated CTNNB1 to Wnt signaling pathway, N00059  FZD7-overexpression to Wnt signaling pathway, N00060  LRP6-overexpression to Wnt signaling pathway, N00061  CDH1-reduced expression to beta-catenin signaling pathway, N00077  HRAS-overexpression to ERK signaling pathway, N00078  Mutation-activated HRAS to ERK signaling pathway, N00088  Amplified MYC to p15-cell cycle G1/S, N00089  Amplified MYC to cell cycle G1/S, N00090  p15-Cell cycle G1/S, N00092  Amplified MYC to p27-cell cycle G1/S, N00110  PLZF-RARA fusion to transcriptional activtion, N00133  EWSR1-FLI1 fusion to transcriptional activation, N00225  EBV EBNA2 to RBP-Jk-mediated transcription, N00242  Mutation-inactivated AXIN to Wnt signaling pathway, N00257  Loss of CDH1 to beta-catenin signaling pathway, N00258  Mutation-inactivated CDH1 to beta-catenin signaling pathway, N00287  ESR1-positive to nuclear-initiated estrogen signaling pathway, N00489  HTLV-1 p30II to c-myc-mediated transcription, N00512  HTLV-1 Tax to c-myc-mediated transcription, N00529  HCV core to RXRA/PPARA-mediated transcription, N00530  HCV core to RXRA/LXRA-mediated transcription, N01124  Salmonella AvrA to beta-catenin signaling pathway, N01344  NNK/NNN to RAS-ERK signaling pathway]','04010 MAPK signaling pathway; 04012 ErbB signaling pathway; 04310 Wnt signaling pathway; 04350 TGF-beta signaling pathway; 04390 Hippo signaling pathway; 04630 JAK-STAT signaling pathway; 04151 PI3K-Akt signaling pathway; 04110 Cell cycle; 04218 Cellular senescence; 04550 Signaling pathways regulating pluripotency of stem cells; 04919 Thyroid hormone signaling pathway; 05200 Pathways in cancer; 05202 Transcriptional misregulation in cancer; 05206 MicroRNAs in cancer; 05205 Proteoglycans in cancer; 05207 Chemical carcinogenesis - receptor activation; 05230 Central carbon metabolism in cancer; 05210 Colorectal cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05216 Thyroid cancer; 05221 Acute myeloid leukemia; 05220 Chronic myeloid leukemia; 05219 Bladder cancer; 05213 Endometrial cancer; 05224 Breast cancer; 05222 Small cell lung cancer; 05166 Human T-cell leukemia virus 1 infection; 05161 Hepatitis B; 05160 Hepatitis C; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05132 Salmonella infection; 03000 Transcription factors [BR:hsa03000]; 03019 Messenger RNA biogenesis [BR:hsa03019]; Cell cycle controlling factors, Myc; Factors that have mRNA localization signals; ','[Pfam: Myc_N Myc-LZ HLH TRH]'),('2475','hsa:2475','[MTOR, FRAP, FRAP1, FRAP2, RAFT1, RAPT1, SKS]','[(RefSeq) mechanistic target of rapamycin kinase]','[serine/threonine-protein kinase mTOR [EC:2.7.11.1]]','[Homo sapiens (human)]','[EGFR tyrosine kinase inhibitor resistance, Endocrine resistance, ErbB signaling pathway, HIF-1 signaling pathway, Phospholipase D signaling pathway, Autophagy - other, Autophagy - animal, mTOR signaling pathway, PI3K-Akt signaling pathway, AMPK signaling pathway, Longevity regulating pathway, Longevity regulating pathway - multiple species, Cellular senescence, Apelin signaling pathway, Neutrophil extracellular trap formation, JAK-STAT signaling pathway, Th17 cell differentiation, Thermogenesis, Insulin signaling pathway, Thyroid hormone signaling pathway, Adipocytokine signaling pathway, Type II diabetes mellitus, Insulin resistance, Growth hormone synthesis, secretion and action, Alzheimer disease, Amyotrophic lateral sclerosis, Huntington disease, Spinocerebellar ataxia, Pathways of neurodegeneration - multiple diseases, Shigellosis, Human cytomegalovirus infection, Human papillomavirus infection, Kaposi sarcoma-associated herpesvirus infection, Herpes simplex virus 1 infection, Human immunodeficiency virus 1 infection, Pathways in cancer, Proteoglycans in cancer, MicroRNAs in cancer, Chemical carcinogenesis - receptor activation, Colorectal cancer, Pancreatic cancer, Glioma, Prostate cancer, Acute myeloid leukemia, Breast cancer, Hepatocellular carcinoma, Gastric cancer, Central carbon metabolism in cancer, Choline metabolism in cancer, PD-L1 expression and PD-1 checkpoint pathway in cancer, Diabetic cardiomyopathy]','[c(\"nt06114  PI3K signaling (viruses)\", \"nt06124  Chemokine signaling (viruses)\", \"nt06150  Cytokine-cytokine receptor interaction (viruses)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06166  Human papillomavirus (HPV)\", \"nt06167  Human cytomegalovirus (HCMV)\", \"nt06168  Herpes simplex virus 1 (HSV-1)\", \"nt06214  PI3K signaling\", \"nt06224  CXCR signaling\", \"nt06260  Colorectal cancer\", \"nt06261  Gastric cancer\", \"nt06262  Pancreatic cancer\", \"nt06263  Hepatocellular carcinoma\", \"nt06266  Non-small cell lung cancer\", \n\"nt06270  Breast cancer\", \"nt06273  Glioma\", \"nt06275  Acute myeloid leukemia\", \"nt06415  PI3K signaling\", \"nt06462  Spinocerebellar ataxia\")]','[N00034  ERBB2-overexpression to PI3K signaling pathway, N00035  Amplified EGFR to PI3K signaling pathway, N00037  FGF-FGFR-PI3K signaling pathway, N00038  Amplified FGFR to PI3K signaling pathway, N00039  PDGF-PDGFR-PI3K signaling pathway, N00040  Amplified PDGFR to PI3K signaling pathway, N00042  EGFR-overexpression to PI3K signaling pathway, N00045  KITLG-KIT-PI3K signaling pathway, N00049  Mutation-activated PI3K to PI3K signaling pathway, N00050  Amplified PI3K to PI3K signaling pathway, N00154  CXCR-GNB/G-PI3K-AKT signaling pathway, N00158  KSHV vGPCR to GNB/G-PI3K-AKT signaling pathway, N00159  KSHV K1 to PI3K signaling pathway, N00218  FLT3LG-FLT3-RAS-PI3K signaling pathway, N00231  TGFA-EGFR-PI3K signaling pathway, N00234  IGF2-IGF1R-PI3K signaling pathway, N00236  IGF2-overexpression to PI3K signaling pathway, N00238  IGF1R-overexpression to PI3K signaling pathway, N00253  Amplified ERBB2 to PI3K signaling pathway, N00260  Amplified MET to PI3K signaling pathway, N00282  EREG-EGFR-PI3K signaling pathway, N00284  AREG-EGFR-PI3K signaling pathway, N00357  HPV E6 to MTOR signaling pathway, N00385  HCMV gB to PDGFR-PI3K signaling pathway, N00387  HCMV IE1-72/IE2-86 to PI3K signaling pathway, N00388  HCMV UL38 to MTOR signaling pathway, N00582  IGF-IGF1R-PI3K signaling pathway, N00583  HSV VP11/12 to PI3K signaling pathway, N00584  HSV US3 to MTOR signaling pathway, N00963  RELN-VLDLR-PI3K signaling pathway, N01063  Mutation-activated MET to PI3K signaling pathway, N01065  Mutation-activated RET to PI3K signaling pathway, N01339  NNK/NNN to PI3K signaling pathway, N01355  Arsenic to PI3K signaling pathway]','04012 ErbB signaling pathway; 04371 Apelin signaling pathway; 04630 JAK-STAT signaling pathway; 04066 HIF-1 signaling pathway; 04072 Phospholipase D signaling pathway; 04151 PI3K-Akt signaling pathway; 04152 AMPK signaling pathway; 04150 mTOR signaling pathway; 04140 Autophagy - animal; 04136 Autophagy - other; 04218 Cellular senescence; 04613 Neutrophil extracellular trap formation; 04659 Th17 cell differentiation; 04910 Insulin signaling pathway; 04920 Adipocytokine signaling pathway; 04935 Growth hormone synthesis, secretion and action; 04919 Thyroid hormone signaling pathway; 04211 Longevity regulating pathway; 04213 Longevity regulating pathway - multiple species; 04714 Thermogenesis; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05205 Proteoglycans in cancer; 05207 Chemical carcinogenesis - receptor activation; 05230 Central carbon metabolism in cancer; 05231 Choline metabolism in cancer; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05210 Colorectal cancer; 05212 Pancreatic cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05214 Glioma; 05221 Acute myeloid leukemia; 05215 Prostate cancer; 05224 Breast cancer; 05170 Human immunodeficiency virus 1 infection; 05168 Herpes simplex virus 1 infection; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05165 Human papillomavirus infection; 05131 Shigellosis; 05010 Alzheimer disease; 05014 Amyotrophic lateral sclerosis; 05016 Huntington disease; 05017 Spinocerebellar ataxia; 05022 Pathways of neurodegeneration - multiple diseases; 05415 Diabetic cardiomyopathy; 04930 Type II diabetes mellitus; 04931 Insulin resistance; 01521 EGFR tyrosine kinase inhibitor resistance; 01522 Endocrine resistance; 01001 Protein kinases [BR:hsa01001]; 04131 Membrane trafficking [BR:hsa04131]; 2.7.11.1  non-specific serine/threonine protein kinase; PIKK family [OT]; mTORC1 complex; mTORC2 complex; ','[Pfam: FAT PI3_PI4_kinase DUF3385 FRB_dom FATC HEAT_2 IFRD UME V-ATPase_H_N TPR_19 EPSP_synthase HEAT_EZ Nipped-B_C]'),('3627','hsa:3627','[CXCL10, C7, IFI10, INP10, IP-10, SCYB10, crg-2, gIP-10, mob-1]','[(RefSeq) C-X-C motif chemokine ligand 10]','[C-X-C motif chemokine 10]','[Homo sapiens (human)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, Chemokine signaling pathway, Toll-like receptor signaling pathway, RIG-I-like receptor signaling pathway, Cytosolic DNA-sensing pathway, IL-17 signaling pathway, TNF signaling pathway, Hepatitis C, Influenza A, Epstein-Barr virus infection, Coronavirus disease - COVID-19]','[nt06150  Cytokine-cytokine receptor interaction (viruses)]','[N01241  CXCL4/9/10/11/13 to CXCR3/5 signaling, N01243  CXCL10 to KSHV vGPCR signaling, N01244  HSV1 SgG1 to CXCR signaling, N01245  HSV2 SgG2 to CXCR signaling]','04668 TNF signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04620 Toll-like receptor signaling pathway; 04622 RIG-I-like receptor signaling pathway; 04623 Cytosolic DNA-sensing pathway; 04657 IL-17 signaling pathway; 04062 Chemokine signaling pathway; 05160 Hepatitis C; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05169 Epstein-Barr virus infection; 04052 Cytokines and growth factors [BR:hsa04052]; 00536 Glycosaminoglycan binding proteins [BR:hsa00536]; Chemokines; Chemokines; ','[Pfam: IL8]'),('10015','hsa:10015','[PDCD6IP, AIP1, ALIX, DRIP4, HP95]','[(RefSeq) programmed cell death 6 interacting protein]','[programmed cell death 6-interacting protein]','[Homo sapiens (human)]','[Viral life cycle - HIV-1, Endocytosis]','[]','[]','03250 Viral life cycle - HIV-1; 04144 Endocytosis; 04131 Membrane trafficking [BR:hsa04131]; 04147 Exosome [BR:hsa04147]; ESCRT associated proteins; Proteins found in most exosomes; ','[Pfam: BRO1 ALIX_LYPXL_bnd DUF5082 AAT Ly49]'),('79087','hsa:79087','[ALG12, CDG1G, ECM39, PP14673, hALG12]','[(RefSeq) ALG12 alpha-1,6-mannosyltransferase]','[alpha-1,6-mannosyltransferase [EC:2.4.1.260]]','[Homo sapiens (human)]','[N-Glycan biosynthesis, Various types of N-glycan biosynthesis, Metabolic pathways]','[nt06015  N-Glycan biosynthesis]','[N00680  N-Glycan precursor biosynthesis, ALG3 to ALG9]','00510 N-Glycan biosynthesis; 00513 Various types of N-glycan biosynthesis; 01003 Glycosyltransferases [BR:hsa01003]; 2.4.1.260  dolichyl-P-Man:Man7GlcNAc2-PP-dolichol alpha-1,6-mannosyltransferase; Dol-linked oligosaccharide; ','[Pfam: Glyco_transf_22 CoV_NSP4_N]'),('56922','hsa:56922','[MCCC1, MCC-B, MCCA, MCCCalpha]','[(RefSeq) methylcrotonyl-CoA carboxylase subunit 1]','[3-methylcrotonyl-CoA carboxylase alpha subunit [EC:6.4.1.4]]','[Homo sapiens (human)]','[Valine, leucine and isoleucine degradation, Metabolic pathways]','[nt06024  Valine, leucine and isoleucine degradation]','[N00851  Leucine degradation]','00280 Valine, leucine and isoleucine degradation; 6.4.1.4  methylcrotonoyl-CoA carboxylase; ','[Pfam: CPSase_L_D2 Biotin_carb_N Biotin_carb_C Biotin_lipoyl Dala_Dala_lig_C ATP-grasp Biotin_lipoyl_2 ATP-grasp_3 RimK ATPgrasp_ST ATP-grasp_4 GARS_A RnfC_N HlyD_D23 NQRA MCCA_BT DUF2118 2OG-FeII_Oxy_2]'),('1644','hsa:1644','[DDC, AADC]','[(RefSeq) dopa decarboxylase]','[aromatic-L-amino-acid/L-tryptophan decarboxylase [EC:4.1.1.28 4.1.1.105]]','[Homo sapiens (human)]','[Tyrosine metabolism, Phenylalanine metabolism, Tryptophan metabolism, Metabolic pathways, Serotonergic synapse, Dopaminergic synapse, Cocaine addiction, Amphetamine addiction, Alcoholism]','[]','[]','00350 Tyrosine metabolism; 00360 Phenylalanine metabolism; 00380 Tryptophan metabolism; 04728 Dopaminergic synapse; 04726 Serotonergic synapse; 05030 Cocaine addiction; 05031 Amphetamine addiction; 05034 Alcoholism; 04147 Exosome [BR:hsa04147]; 4.1.1.28  aromatic-L-amino-acid decarboxylase; 4.1.1.105  L-tryptophan decarboxylase; Exosomal proteins of other body fluids (saliva and urine); ','[Pfam: Pyridoxal_deC Beta_elim_lyase Aminotran_1_2]'),('1728','hsa:1728','[NQO1, DHQU, DIA4, DTD, NMOR1, NMORI, QR1]','[(RefSeq) NAD(P)H quinone dehydrogenase 1]','[NAD(P)H dehydrogenase (quinone) [EC:1.6.5.2]]','[Homo sapiens (human)]','[Ubiquinone and other terpenoid-quinone biosynthesis, Metabolic pathways, Biosynthesis of cofactors, Pathways in cancer, Chemical carcinogenesis - reactive oxygen species, Hepatocellular carcinoma, Fluid shear stress and atherosclerosis]','[c(\"nt06226  KEAP1-NRF2 signaling\", \"nt06263  Hepatocellular carcinoma\", \"nt06463  Parkinson disease\")]','[N00243  KEAP1-NRF2 signaling pathway, N00244  Mutation-inactivated KEAP1 to KEAP1-NRF2 signaling pathway, N00245  Mutation-activated NRF2 to KEAP1-NRF2 signaling pathway, N01413  Metals to KEAP1-NRF2 signalig pathway]','00130 Ubiquinone and other terpenoid-quinone biosynthesis; 05200 Pathways in cancer; 05208 Chemical carcinogenesis - reactive oxygen species; 05225 Hepatocellular carcinoma; 05418 Fluid shear stress and atherosclerosis; 1.6.5.2  NAD(P)H dehydrogenase (quinone); ','[Pfam: Flavodoxin_2 FMN_red]'),('27232','hsa:27232','[GNMT, HEL-S-182mP]','[(RefSeq) glycine N-methyltransferase]','[glycine N-methyltransferase [EC:2.1.1.20]]','[Homo sapiens (human)]','[Glycine, serine and threonine metabolism, Metabolic pathways]','[]','[]','00260 Glycine, serine and threonine metabolism; 2.1.1.20  glycine N-methyltransferase; ','[Pfam: Methyltransf_25 Methyltransf_31 Methyltransf_12 Methyltransf_11 Methyltransf_23 TehB Ubie_methyltran MTS PCMT TPMT CMAS Methyltransf_15]'),('1544','hsa:1544','[CYP1A2, CP12, CYPIA2, P3-450, P450(PA)]','[(RefSeq) cytochrome P450 family 1 subfamily A member 2]','[cytochrome P450 family 1 subfamily A2 [EC:1.14.14.1]]','[Homo sapiens (human)]','[Steroid hormone biosynthesis, Caffeine metabolism, Tryptophan metabolism, Linoleic acid metabolism, Retinol metabolism, Metabolism of xenobiotics by cytochrome P450, Drug metabolism - cytochrome P450, Metabolic pathways, Chemical carcinogenesis - DNA adducts, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species]','[c(\"nt06250  DNA adduct formation\", \"nt06251  CYP-mediated ROS formation\")]','[N01369  4-ABP to DNA adducts, N01370  PhIP to DNA adducts, N01371  PhIP to DNA adducts, N01372  IQ to DNA adducts, N01373  MeIQx to DNA adducts, N01378  AFB1 to DNA adducts, N01396  4-Aminobiphenyl to CYP-mediated metabolism, N01397  4-Aminobiphenyl to CYP-mediated metabolism, N01398  Pentachlorophenol to CYP-mediated metabolism]','00140 Steroid hormone biosynthesis; 00591 Linoleic acid metabolism; 00380 Tryptophan metabolism; 00830 Retinol metabolism; 00232 Caffeine metabolism; 00980 Metabolism of xenobiotics by cytochrome P450; 00982 Drug metabolism - cytochrome P450; 05204 Chemical carcinogenesis - DNA adducts; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 00199 Cytochrome P450 [BR:hsa00199]; 1.14.14.1  unspecific monooxygenase; CYP1 family; ','[Pfam: p450]'),('1577','hsa:1577','[CYP3A5, CP35, CYPIIIA5, P450PCN3, PCN3]','[(RefSeq) cytochrome P450 family 3 subfamily A member 5]','[cytochrome P450 family 3 subfamily A5 [EC:1.14.14.1]]','[Homo sapiens (human)]','[Steroid hormone biosynthesis, Retinol metabolism, Metabolism of xenobiotics by cytochrome P450, Drug metabolism - cytochrome P450, Metabolic pathways, Chemical carcinogenesis - DNA adducts]','[]','[]','00140 Steroid hormone biosynthesis; 00830 Retinol metabolism; 00980 Metabolism of xenobiotics by cytochrome P450; 00982 Drug metabolism - cytochrome P450; 05204 Chemical carcinogenesis - DNA adducts; 00199 Cytochrome P450 [BR:hsa00199]; 1.14.14.1  unspecific monooxygenase; CYP3 family; ','[Pfam: p450 Coa1]'),('2729','hsa:2729','[GCLC, GCL, GCS, GLCL, GLCLC]','[(RefSeq) glutamate-cysteine ligase catalytic subunit]','[glutamate--cysteine ligase catalytic subunit [EC:6.3.2.2]]','[Homo sapiens (human)]','[Cysteine and methionine metabolism, Glutathione metabolism, Metabolic pathways, Biosynthesis of cofactors, Ferroptosis]','[nt06026  Glutathione biosynthesis]','[N00899  Glutathione biosynthesis]','00270 Cysteine and methionine metabolism; 00480 Glutathione metabolism; 04216 Ferroptosis; 6.3.2.2  glutamate---cysteine ligase; ','[Pfam: GCS GCS2]'),('1565','hsa:1565','[CYP2D6, CPD6, CYP2D, CYP2D7AP, CYP2D7BP, CYP2D7P2, CYP2D8P2, CYP2DL1, CYPIID6, P450-DB1, P450C2D, P450DB1]','[(RefSeq) cytochrome P450 family 2 subfamily D member 6]','[cytochrome P450 family 2 subfamily D6 [EC:1.14.14.1]]','[Homo sapiens (human)]','[Metabolism of xenobiotics by cytochrome P450, Drug metabolism - cytochrome P450, Endocrine resistance, Serotonergic synapse]','[]','[]','00980 Metabolism of xenobiotics by cytochrome P450; 00982 Drug metabolism - cytochrome P450; 04726 Serotonergic synapse; 01522 Endocrine resistance; 00199 Cytochrome P450 [BR:hsa00199]; 1.14.14.1  unspecific monooxygenase; CYP2 family; ','[Pfam: p450]'),('1576','hsa:1576','[CYP3A4, CP33, CP34, CYP3A, CYP3A3, CYPIIIA3, CYPIIIA4, HLP, NF-25, P450C3, P450PCN1, VDDR3]','[(RefSeq) cytochrome P450 family 3 subfamily A member 4]','[cytochrome P450 family 3 subfamily A4 [EC:1.14.13.32 1.14.14.55 1.14.14.56 1.14.14.57 1.14.14.73 1.14.14.-]]','[Homo sapiens (human)]','[Steroid hormone biosynthesis, Linoleic acid metabolism, Retinol metabolism, Metabolism of xenobiotics by cytochrome P450, Drug metabolism - cytochrome P450, Drug metabolism - other enzymes, Metabolic pathways, Bile secretion, Chemical carcinogenesis - DNA adducts, Chemical carcinogenesis - receptor activation]','[nt06250  DNA adduct formation]','[N01378  AFB1 to DNA adducts]','00140 Steroid hormone biosynthesis; 00591 Linoleic acid metabolism; 00830 Retinol metabolism; 00980 Metabolism of xenobiotics by cytochrome P450; 00982 Drug metabolism - cytochrome P450; 00983 Drug metabolism - other enzymes; 04976 Bile secretion; 05204 Chemical carcinogenesis - DNA adducts; 05207 Chemical carcinogenesis - receptor activation; 00199 Cytochrome P450 [BR:hsa00199]; 1.14.13.32  albendazole monooxygenase; 1.14.14.55  quinine 3-monooxygenase; 1.14.14.56  1,8-cineole 2-exo-monooxygenase; 1.14.14.57  taurochenodeoxycholate 6alpha-hydroxylase; 1.14.14.73  albendazole monooxygenase (sulfoxide-forming); 1.14.14.-; CYP3 family; ','[Pfam: p450 Coa1]'),('4790','hsa:4790','[NFKB1, CVID12, EBP-1, KBF1, NF-kB, NF-kB1, NF-kappa-B1, NF-kappaB, NF-kappabeta, NFKB-p105, NFKB-p50, NFkappaB]','[(RefSeq) nuclear factor kappa B subunit 1]','[nuclear factor NF-kappa-B p105 subunit]','[Homo sapiens (human)]','[Antifolate resistance, MAPK signaling pathway, Ras signaling pathway, cAMP signaling pathway, Chemokine signaling pathway, NF-kappa B signaling pathway, HIF-1 signaling pathway, Sphingolipid signaling pathway, PI3K-Akt signaling pathway, Apoptosis, Longevity regulating pathway, Cellular senescence, Osteoclast differentiation, Neutrophil extracellular trap formation, Toll-like receptor signaling pathway, NOD-like receptor signaling pathway, RIG-I-like receptor signaling pathway, Cytosolic DNA-sensing pathway, C-type lectin receptor signaling pathway, IL-17 signaling pathway, Th1 and Th2 cell differentiation, Th17 cell differentiation, T cell receptor signaling pathway, B cell receptor signaling pathway, TNF signaling pathway, Neurotrophin signaling pathway, Prolactin signaling pathway, Adipocytokine signaling pathway, Relaxin signaling pathway, Insulin resistance, Non-alcoholic fatty liver disease, AGE-RAGE signaling pathway in diabetic complications, Alcoholic liver disease, Alzheimer disease, Pathways of neurodegeneration - multiple diseases, Cocaine addiction, Epithelial cell signaling in Helicobacter pylori infection, Pathogenic Escherichia coli infection, Shigellosis, Salmonella infection, Pertussis, Legionellosis, Yersinia infection, Leishmaniasis, Chagas disease, Toxoplasmosis, Amoebiasis, Tuberculosis, Hepatitis C, Hepatitis B, Measles, Human cytomegalovirus infection, Influenza A, Human papillomavirus infection, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Herpes simplex virus 1 infection, Epstein-Barr virus infection, Human immunodeficiency virus 1 infection, Coronavirus disease - COVID-19, Pathways in cancer, Transcriptional misregulation in cancer, Viral carcinogenesis, MicroRNAs in cancer, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species, Pancreatic cancer, Prostate cancer, Chronic myeloid leukemia, Acute myeloid leukemia, Small cell lung cancer, PD-L1 expression and PD-1 checkpoint pathway in cancer, Inflammatory bowel disease, Diabetic cardiomyopathy, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[c(\"nt06114  PI3K signaling (viruses)\", \"nt06121  TLR signaling (viruses and bacteria)\", \"nt06123  TNF signaling (viruses and bacteria)\", \"nt06124  Chemokine signaling (viruses)\", \"nt06133  RLR signaling (viruses)\", \"nt06139  NLR signaling (viruses and bacteria)\", \"nt06150  Cytokine-cytokine receptor interaction (viruses)\", \"nt06160  Human T-cell leukemia virus 1 (HTLV-1)\", \"nt06161  Human immunodeficiency virus type 1 (HIV-1)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06163  Hepatitis C virus (HCV)\", \n\"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06165  Epstein-Barr virus (EBV)\", \"nt06167  Human cytomegalovirus (HCMV)\", \"nt06168  Herpes simplex virus 1 (HSV-1)\", \"nt06169  Measles virus (MV)\", \"nt06170  Influenza A virus (IAV)\", \"nt06171  Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)\", \"nt06214  PI3K signaling\", \"nt06223  TNF signaling\", \"nt06224  CXCR signaling\", \"nt06240  Transcription\", \"nt06263  Hepatocellular carcinoma\", \"nt06417  AGE-RAGE signaling\", \"nt06460  Alzheimer disease\"\n)]','[N00140  FUS-DDIT3 fusion to NFKB-mediated transcription, N00151  TNF-NFKB signaling pathway, N00171  KSHV vFLIP to NFKB signaling pathway, N00173  KSHV K15 to TNF-NFKB signaling pathway, N00174  KSHV vFLIP to TNF-NFKB signaling pathway, N00178  KSHV vGPCR to GNB/G-PI3K-JNK signaling pathway, N00179  KSHV K1 to PI3K-NFKB signaling pathway, N00182  IGF-IGFR-PI3K-NFKB signaling pathway, N00265  EBV LMP1 to NFKB signaling pathway, N00269  HCV core to TNF-NFKB signaling pathway, N00389  HCMV IE1-72 to transcription, N00390  EGF-EGFR-PI3K-NFKB signaling pathway, N00391  HCMV gB to EGFR-PI3K-NFKB signaling pathway, N00398  HCMV IE2-86 to TNF-NFKB signaling pathway, N00399  CCR2-GNB/G-PI3K-NFKB signaling pathway, N00400  HCMV US28 to GNB/G-PI3K-NFKB signaling pathway, N00435  TLR2/4-NFKB signaling pathway, N00436  HIV Tat to TLR2/4-NFKB signaling pathway, N00437  HIV Vpu to TLR2/4-NFKB signaling pathway, N00441  HIV gp120 to TNF-NFKB signaling pathway, N00442  HIV Nef to TNF-NFKB signaling pathway, N00443  HIV Vpr/Nef/Tat to TNF-NFKB signaling pathway, N00501  HTLV-1 Tax to EGFR-PI3K-NFKB signaling pathway, N00504  HTLV-1 Tax to NFKB signaling pathway, N00550  HBV HBeAg to TLR2/4-NFKB signaling pathway, N00559  LIGHT-HVEM-NFKB signaling pathway, N00560  HSV gD to HVEM-NFKB signaling pathway, N00563  TLR3-NFKB signaling pathway, N00688  RIG-I-NFKB signaling pathway, N00689  MV V/P/C to RIG-I-NFKB signaling pathway, N00730  IAV NS1 to RIG-I-like receptor signaling pathway, N00932  Escherichia NleH1 to TNF-NFKB signaling pathway, N00940  NOD-NFKB signaling pathway, N00994  AGE-RAGE signaling pathway, N00996  Mutation-caused aberrant Abeta to AGE-RAGE signaling pathway, N01112  Salmonella SopE/E2 to NOD-NFKB signaling pathway, N01115  Salmonella SseL to TNF-NFKB signaling pathway, N01118  Salmonella SpvD to TNF-NFKB signaling pathway, N01306  AngII-AT1R-NOX2 signaling pathway, N01307  SARS-CoV-2 S to AngII-AT1R-NOX2 signaling pathway, N01339  NNK/NNN to PI3K signaling pathway, N01410  Metals to NFKB signaling pathway, N01411  Metals to NFKB signaling pathway]','04010 MAPK signaling pathway; 04014 Ras signaling pathway; 04064 NF-kappa B signaling pathway; 04668 TNF signaling pathway; 04066 HIF-1 signaling pathway; 04071 Sphingolipid signaling pathway; 04024 cAMP signaling pathway; 04151 PI3K-Akt signaling pathway; 04210 Apoptosis; 04218 Cellular senescence; 04613 Neutrophil extracellular trap formation; 04620 Toll-like receptor signaling pathway; 04621 NOD-like receptor signaling pathway; 04622 RIG-I-like receptor signaling pathway; 04623 Cytosolic DNA-sensing pathway; 04625 C-type lectin receptor signaling pathway; 04660 T cell receptor signaling pathway; 04658 Th1 and Th2 cell differentiation; 04659 Th17 cell differentiation; 04657 IL-17 signaling pathway; 04662 B cell receptor signaling pathway; 04062 Chemokine signaling pathway; 04920 Adipocytokine signaling pathway; 04917 Prolactin signaling pathway; 04926 Relaxin signaling pathway; 04722 Neurotrophin signaling pathway; 04380 Osteoclast differentiation; 04211 Longevity regulating pathway; 05200 Pathways in cancer; 05202 Transcriptional misregulation in cancer; 05206 MicroRNAs in cancer; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 05203 Viral carcinogenesis; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05212 Pancreatic cancer; 05221 Acute myeloid leukemia; 05220 Chronic myeloid leukemia; 05215 Prostate cancer; 05222 Small cell lung cancer; 05166 Human T-cell leukemia virus 1 infection; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05160 Hepatitis C; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05165 Human papillomavirus infection; 05120 Epithelial cell signaling in Helicobacter pylori infection; 05130 Pathogenic Escherichia coli infection; 05132 Salmonella infection; 05131 Shigellosis; 05135 Yersinia infection; 05133 Pertussis; 05134 Legionellosis; 05152 Tuberculosis; 05146 Amoebiasis; 05145 Toxoplasmosis; 05140 Leishmaniasis; 05142 Chagas disease; 05321 Inflammatory bowel disease; 05010 Alzheimer disease; 05022 Pathways of neurodegeneration - multiple diseases; 05030 Cocaine addiction; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 05415 Diabetic cardiomyopathy; 04936 Alcoholic liver disease; 04932 Non-alcoholic fatty liver disease; 04931 Insulin resistance; 04933 AGE-RAGE signaling pathway in diabetic complications; 01523 Antifolate resistance; 03000 Transcription factors [BR:hsa03000]; RHR (Rel homology region) Rel/Ankyrin; ','[Pfam: RHD_DNA_bind RHD_dimer Ank_2 Ank_4 Ank Ank_3 Ank_5 Death]'),('3552','hsa:3552','[IL1A, IL-1_alpha, IL-1A, IL1, IL1-ALPHA, IL1F1]','[(RefSeq) interleukin 1 alpha]','[interleukin 1 alpha]','[Homo sapiens (human)]','[MAPK signaling pathway, Cytokine-cytokine receptor interaction, Necroptosis, Cellular senescence, Osteoclast differentiation, Hematopoietic cell lineage, Non-alcoholic fatty liver disease, AGE-RAGE signaling pathway in diabetic complications, Type I diabetes mellitus, Alzheimer disease, Prion disease, Pathways of neurodegeneration - multiple diseases, Pertussis, Leishmaniasis, Tuberculosis, Measles, Influenza A, Inflammatory bowel disease, Rheumatoid arthritis, Graft-versus-host disease, Fluid shear stress and atherosclerosis]','[c(\"nt06110  MAPK signaling (viruses and bacteria)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06211  Other MAPK signaling\")]','[N00186  IL1-IL1R-p38 signaling pathway, N00188  IL1-IL1R-JNK signaling pathway]','04010 MAPK signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04217 Necroptosis; 04218 Cellular senescence; 04640 Hematopoietic cell lineage; 04380 Osteoclast differentiation; 05164 Influenza A; 05162 Measles; 05133 Pertussis; 05152 Tuberculosis; 05140 Leishmaniasis; 05323 Rheumatoid arthritis; 05321 Inflammatory bowel disease; 05332 Graft-versus-host disease; 05010 Alzheimer disease; 05020 Prion disease; 05022 Pathways of neurodegeneration - multiple diseases; 05418 Fluid shear stress and atherosclerosis; 04940 Type I diabetes mellitus; 04932 Non-alcoholic fatty liver disease; 04933 AGE-RAGE signaling pathway in diabetic complications; 04052 Cytokines and growth factors [BR:hsa04052]; Interleukins; ','[Pfam: IL1_propep IL1]'),('3558','hsa:3558','[IL2, IL-2, TCGF, lymphokine]','[(RefSeq) interleukin 2]','[interleukin 2]','[Homo sapiens (human)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, PI3K-Akt signaling pathway, C-type lectin receptor signaling pathway, JAK-STAT signaling pathway, Th1 and Th2 cell differentiation, Th17 cell differentiation, T cell receptor signaling pathway, Intestinal immune network for IgA production, Type I diabetes mellitus, Yersinia infection, Chagas disease, Measles, Human T-cell leukemia virus 1 infection, Coronavirus disease - COVID-19, Pathways in cancer, Autoimmune thyroid disease, Inflammatory bowel disease, Allograft rejection, Graft-versus-host disease]','[c(\"nt06119  JAK-STAT signaling (viruses)\", \"nt06120  Calcium signaling (viruses and bacteria)\", \"nt06150  Cytokine-cytokine receptor interaction (viruses)\", \"nt06160  Human T-cell leukemia virus 1 (HTLV-1)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06165  Epstein-Barr virus (EBV)\", \"nt06219  JAK-STAT signaling\", \"nt06220  Calcium signaling\", \"nt06263  Hepatocellular carcinoma\", \"nt06266  Non-small cell lung cancer\", \"nt06276  Chronic myeloid leukemia\"\n)]','[N00053  Cytokine-Jak-STAT signaling pathway, N00490  HTLV-1 p12 to calcineurin signaling pathway, N00491  HTLV-1 p12 to Jak-STAT signaling pathway, N01268  IL2 to IL2R signaling]','04630 JAK-STAT signaling pathway; 04151 PI3K-Akt signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04625 C-type lectin receptor signaling pathway; 04660 T cell receptor signaling pathway; 04658 Th1 and Th2 cell differentiation; 04659 Th17 cell differentiation; 04672 Intestinal immune network for IgA production; 05200 Pathways in cancer; 05166 Human T-cell leukemia virus 1 infection; 05171 Coronavirus disease - COVID-19; 05162 Measles; 05135 Yersinia infection; 05142 Chagas disease; 05320 Autoimmune thyroid disease; 05321 Inflammatory bowel disease; 05330 Allograft rejection; 05332 Graft-versus-host disease; 04940 Type I diabetes mellitus; 04052 Cytokines and growth factors [BR:hsa04052]; 00536 Glycosaminoglycan binding proteins [BR:hsa00536]; Interleukins; Cytokines; ','[Pfam: IL2 Plasmid_RAQPRD]'),('3567','hsa:3567','[IL5, EDF, IL-5, TRF]','[(RefSeq) interleukin 5]','[interleukin 5]','[Homo sapiens (human)]','[Cytokine-cytokine receptor interaction, JAK-STAT signaling pathway, Hematopoietic cell lineage, IL-17 signaling pathway, Th1 and Th2 cell differentiation, T cell receptor signaling pathway, Fc epsilon RI signaling pathway, Intestinal immune network for IgA production, Pathways in cancer, Asthma, Autoimmune thyroid disease, Inflammatory bowel disease, Allograft rejection]','[c(\"nt06119  JAK-STAT signaling (viruses)\", \"nt06160  Human T-cell leukemia virus 1 (HTLV-1)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06165  Epstein-Barr virus (EBV)\", \"nt06219  JAK-STAT signaling\", \"nt06263  Hepatocellular carcinoma\", \"nt06266  Non-small cell lung cancer\", \"nt06276  Chronic myeloid leukemia\")]','[N00053  Cytokine-Jak-STAT signaling pathway]','04630 JAK-STAT signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04640 Hematopoietic cell lineage; 04660 T cell receptor signaling pathway; 04658 Th1 and Th2 cell differentiation; 04657 IL-17 signaling pathway; 04664 Fc epsilon RI signaling pathway; 04672 Intestinal immune network for IgA production; 05200 Pathways in cancer; 05310 Asthma; 05320 Autoimmune thyroid disease; 05321 Inflammatory bowel disease; 05330 Allograft rejection; 04052 Cytokines and growth factors [BR:hsa04052]; 00536 Glycosaminoglycan binding proteins [BR:hsa00536]; Interleukins; Cytokines; ','[Pfam: IL5 DUF6034]'),('50616','hsa:50616','[IL22, IL-21, IL-22, IL-D110, IL-TIF, ILTIF, TIFIL-23, TIFa, zcyto18]','[(RefSeq) interleukin 22]','[interleukin 22]','[Homo sapiens (human)]','[Cytokine-cytokine receptor interaction, JAK-STAT signaling pathway, Th17 cell differentiation, Inflammatory bowel disease]','[c(\"nt06227  Nuclear receptor signaling\", \"nt06360  Cushing syndrome\")]','[N00317  AhR signaling pathway]','04630 JAK-STAT signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04659 Th17 cell differentiation; 05321 Inflammatory bowel disease; 04052 Cytokines and growth factors [BR:hsa04052]; Interleukins; ','[Pfam: IL22 IL10 IL28A]'),('3458','hsa:3458','[IFNG, IFG, IFI, IMD69]','[(RefSeq) interferon gamma]','[interferon gamma]','[Homo sapiens (human)]','[Proteasome, Cytokine-cytokine receptor interaction, HIF-1 signaling pathway, Necroptosis, TGF-beta signaling pathway, Osteoclast differentiation, Antigen processing and presentation, JAK-STAT signaling pathway, Natural killer cell mediated cytotoxicity, IL-17 signaling pathway, Th1 and Th2 cell differentiation, Th17 cell differentiation, T cell receptor signaling pathway, Type I diabetes mellitus, Leishmaniasis, Chagas disease, African trypanosomiasis, Malaria, Toxoplasmosis, Amoebiasis, Tuberculosis, Hepatitis C, Influenza A, Herpes simplex virus 1 infection, Pathways in cancer, PD-L1 expression and PD-1 checkpoint pathway in cancer, Inflammatory bowel disease, Systemic lupus erythematosus, Rheumatoid arthritis, Allograft rejection, Graft-versus-host disease, Fluid shear stress and atherosclerosis]','[c(\"nt06119  JAK-STAT signaling (viruses)\", \"nt06160  Human T-cell leukemia virus 1 (HTLV-1)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06165  Epstein-Barr virus (EBV)\", \"nt06219  JAK-STAT signaling\", \"nt06263  Hepatocellular carcinoma\", \"nt06266  Non-small cell lung cancer\", \"nt06276  Chronic myeloid leukemia\")]','[N00053  Cytokine-Jak-STAT signaling pathway]','03050 Proteasome; 04350 TGF-beta signaling pathway; 04630 JAK-STAT signaling pathway; 04066 HIF-1 signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04217 Necroptosis; 04650 Natural killer cell mediated cytotoxicity; 04612 Antigen processing and presentation; 04660 T cell receptor signaling pathway; 04658 Th1 and Th2 cell differentiation; 04659 Th17 cell differentiation; 04657 IL-17 signaling pathway; 04380 Osteoclast differentiation; 05200 Pathways in cancer; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05160 Hepatitis C; 05164 Influenza A; 05168 Herpes simplex virus 1 infection; 05152 Tuberculosis; 05146 Amoebiasis; 05144 Malaria; 05145 Toxoplasmosis; 05140 Leishmaniasis; 05142 Chagas disease; 05143 African trypanosomiasis; 05322 Systemic lupus erythematosus; 05323 Rheumatoid arthritis; 05321 Inflammatory bowel disease; 05330 Allograft rejection; 05332 Graft-versus-host disease; 05418 Fluid shear stress and atherosclerosis; 04940 Type I diabetes mellitus; 03051 Proteasome [BR:hsa03051]; 04052 Cytokines and growth factors [BR:hsa04052]; 00536 Glycosaminoglycan binding proteins [BR:hsa00536]; Other assembling factors; Interferons; Cytokines; ','[Pfam: IFN-gamma API5 Cna_B Tup_N]'),('3605','hsa:3605','[IL17A, CTLA-8, CTLA8, IL-17, IL-17A, IL17, ILA17]','[(RefSeq) interleukin 17A]','[interleukin 17A]','[Homo sapiens (human)]','[Cytokine-cytokine receptor interaction, IL-17 signaling pathway, Th17 cell differentiation, Alcoholic liver disease, Inflammatory bowel disease, Rheumatoid arthritis]','[]','[]','04060 Cytokine-cytokine receptor interaction; 04659 Th17 cell differentiation; 04657 IL-17 signaling pathway; 05323 Rheumatoid arthritis; 05321 Inflammatory bowel disease; 04936 Alcoholic liver disease; 04052 Cytokines and growth factors [BR:hsa04052]; Interleukins; ','[Pfam: IL17]'),('6095','hsa:6095','[RORA, IDDECA, NR1F1, ROR1, ROR2, ROR3, RZR-ALPHA, RZRA]','[(RefSeq) RAR related orphan receptor A]','[RAR-related orphan receptor alpha]','[Homo sapiens (human)]','[Th17 cell differentiation, Circadian rhythm, Spinocerebellar ataxia, Inflammatory bowel disease]','[c(\"nt06440  Transcription\", \"nt06462  Spinocerebellar ataxia\")]','[N00965  RORA-mediated transcription, N00966  Mutation-caused aberrant ATXN1 to RORA-mediated transcription]','04659 Th17 cell differentiation; 04710 Circadian rhythm; 05321 Inflammatory bowel disease; 05017 Spinocerebellar ataxia; 03000 Transcription factors [BR:hsa03000]; 03310 Nuclear receptors [BR:hsa03310]; Cys4 thyroid hormone receptor-like; 1F. RAR-related orphan receptor; ','[Pfam: zf-C4 Hormone_recep]'),('2919','hsa:2919','[CXCL1, FSP, GRO1, GROa, MGSA, MGSA-a, NAP-3, SCYB1]','[(RefSeq) C-X-C motif chemokine ligand 1]','[C-X-C motif chemokine 1/2/3]','[Homo sapiens (human)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, Chemokine signaling pathway, NF-kappa B signaling pathway, NOD-like receptor signaling pathway, IL-17 signaling pathway, TNF signaling pathway, Alcoholic liver disease, Epithelial cell signaling in Helicobacter pylori infection, Legionellosis, Amoebiasis, Kaposi sarcoma-associated herpesvirus infection, Rheumatoid arthritis, Lipid and atherosclerosis]','[nt06150  Cytokine-cytokine receptor interaction (viruses)]','[N01233  CXCL1/5/6/8 to CXCR1 signaling, N01234  CXCL1/5/8 to KSHV vGPCR signaling, N01238  CXCL1/2/3/5/6/7/8 to CXCR2 signaling]','04064 NF-kappa B signaling pathway; 04668 TNF signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04621 NOD-like receptor signaling pathway; 04657 IL-17 signaling pathway; 04062 Chemokine signaling pathway; 05167 Kaposi sarcoma-associated herpesvirus infection; 05120 Epithelial cell signaling in Helicobacter pylori infection; 05134 Legionellosis; 05146 Amoebiasis; 05323 Rheumatoid arthritis; 05417 Lipid and atherosclerosis; 04936 Alcoholic liver disease; 04052 Cytokines and growth factors [BR:hsa04052]; Chemokines; ','[Pfam: IL8]'),('6374','hsa:6374','[CXCL5, ENA-78, SCYB5]','[(RefSeq) C-X-C motif chemokine ligand 5]','[C-X-C motif chemokine 5/6]','[Homo sapiens (human)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, Chemokine signaling pathway, IL-17 signaling pathway, TNF signaling pathway, Pertussis, Rheumatoid arthritis]','[nt06150  Cytokine-cytokine receptor interaction (viruses)]','[N01233  CXCL1/5/6/8 to CXCR1 signaling, N01234  CXCL1/5/8 to KSHV vGPCR signaling, N01238  CXCL1/2/3/5/6/7/8 to CXCR2 signaling]','04668 TNF signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04657 IL-17 signaling pathway; 04062 Chemokine signaling pathway; 05133 Pertussis; 05323 Rheumatoid arthritis; 04052 Cytokines and growth factors [BR:hsa04052]; Chemokines; ','[Pfam: IL8]'),('5468','hsa:5468','[PPARG, CIMT1, GLM1, NR1C3, PPARG1, PPARG2, PPARG5, PPARgamma]','[(RefSeq) peroxisome proliferator activated receptor gamma]','[peroxisome proliferator-activated receptor gamma]','[Homo sapiens (human)]','[PPAR signaling pathway, AMPK signaling pathway, Longevity regulating pathway, Osteoclast differentiation, Thermogenesis, Non-alcoholic fatty liver disease, Huntington disease, Pathways in cancer, Transcriptional misregulation in cancer, Thyroid cancer, Lipid and atherosclerosis]','[c(\"nt06240  Transcription\", \"nt06274  Thyroid cancer\")]','[N00126  PAX8-PPARG fusion to PPARG-mediated transcription, N00138  EWSR1-NR4A3]','04152 AMPK signaling pathway; 03320 PPAR signaling pathway; 04380 Osteoclast differentiation; 04211 Longevity regulating pathway; 04714 Thermogenesis; 05200 Pathways in cancer; 05202 Transcriptional misregulation in cancer; 05216 Thyroid cancer; 05016 Huntington disease; 05417 Lipid and atherosclerosis; 04932 Non-alcoholic fatty liver disease; 03000 Transcription factors [BR:hsa03000]; 03310 Nuclear receptors [BR:hsa03310]; Cys4 thyroid hormone receptor-like; 1C. Peroxisome proliferator-activated receptor (PPAR); ','[Pfam: PPARgamma_N zf-C4 Hormone_recep Tmemb_55A]'),('123','hsa:123','[PLIN2, ADFP, ADRP]','[(RefSeq) perilipin 2]','[perilipin-2]','[Homo sapiens (human)]','[PPAR signaling pathway]','[]','[]','03320 PPAR signaling pathway; 04147 Exosome [BR:hsa04147]; Exosomal proteins of breast milk; ','[Pfam: Perilipin SH3BP5]'),('9971','hsa:9971','[NR1H4, BAR, FXR, HRR-1, HRR1, PFIC5, RIP14]','[(RefSeq) nuclear receptor subfamily 1 group H member 4]','[farnesoid X receptor]','[Homo sapiens (human)]','[Bile secretion]','[]','[]','04976 Bile secretion; 03000 Transcription factors [BR:hsa03000]; 03310 Nuclear receptors [BR:hsa03310]; Cys4 thyroid hormone receptor-like; 1H. Liver X receptor like receptor; ','[Pfam: zf-C4 Hormone_recep]'),('2740','hsa:2740','[GLP1R, GLP-1, GLP-1-R, GLP-1R]','[(RefSeq) glucagon like peptide 1 receptor]','[glucagon-like peptide 1 receptor]','[Homo sapiens (human)]','[cAMP signaling pathway, Neuroactive ligand-receptor interaction, Insulin secretion]','[]','[]','04024 cAMP signaling pathway; 04080 Neuroactive ligand-receptor interaction; 04911 Insulin secretion; 04030 G protein-coupled receptors [BR:hsa04030]; Glucagon; ','[Pfam: 7tm_2 HRM]'),('10052','hsa:10052','[GJC1, CX45, GJA7]','[(RefSeq) gap junction protein gamma 1]','[gap junction gamma-1 protein]','[Homo sapiens (human)]',NULL,'[]','[]','02000 Transporters [BR:hsa02000]; Pores ion channels [TC:1]; ','[Pfam: Connexin TLP-20]'),('171389','hsa:171389','[NLRP6, AVR, CLR11.4, NALP6, NAVR, NAVR/AVR, PAN3, PYPAF5]','[(RefSeq) NLR family pyrin domain containing 6]','[NACHT, LRR and PYD domains-containing protein 6]','[Homo sapiens (human)]','[NOD-like receptor signaling pathway]','[]','[]','04621 NOD-like receptor signaling pathway; 04054 Pattern recognition receptors [BR:hsa04054]; NOD-like receptors; ','[Pfam: NACHT PYRIN NLRC4_HD2 NOD2_WH AAA_16]'),('3952','hsa:3952','[LEP, LEPD, OB, OBS]','[(RefSeq) leptin]','[leptin]','[Homo sapiens (human)]','[Cytokine-cytokine receptor interaction, Neuroactive ligand-receptor interaction, AMPK signaling pathway, JAK-STAT signaling pathway, Adipocytokine signaling pathway, Non-alcoholic fatty liver disease]','[]','[]','04630 JAK-STAT signaling pathway; 04152 AMPK signaling pathway; 04080 Neuroactive ligand-receptor interaction; 04060 Cytokine-cytokine receptor interaction; 04920 Adipocytokine signaling pathway; 04932 Non-alcoholic fatty liver disease; 04052 Cytokines and growth factors [BR:hsa04052]; CSF and other factors; ','[Pfam: Leptin]'),('207','hsa:207','[AKT1, AKT, PKB, PKB-ALPHA, PRKBA, RAC, RAC-ALPHA]','[(RefSeq) AKT serine/threonine kinase 1]','[RAC serine/threonine-protein kinase [EC:2.7.11.1]]','[Homo sapiens (human)]','[EGFR tyrosine kinase inhibitor resistance, Endocrine resistance, Platinum drug resistance, MAPK signaling pathway, ErbB signaling pathway, Ras signaling pathway, Rap1 signaling pathway, cGMP-PKG signaling pathway, cAMP signaling pathway, Chemokine signaling pathway, HIF-1 signaling pathway, FoxO signaling pathway, Sphingolipid signaling pathway, Phospholipase D signaling pathway, Autophagy - animal, mTOR signaling pathway, PI3K-Akt signaling pathway, AMPK signaling pathway, Apoptosis, Longevity regulating pathway, Longevity regulating pathway - multiple species, Cellular senescence, Adrenergic signaling in cardiomyocytes, VEGF signaling pathway, Apelin signaling pathway, Osteoclast differentiation, Focal adhesion, Signaling pathways regulating pluripotency of stem cells, Platelet activation, Neutrophil extracellular trap formation, Toll-like receptor signaling pathway, C-type lectin receptor signaling pathway, JAK-STAT signaling pathway, T cell receptor signaling pathway, B cell receptor signaling pathway, Fc epsilon RI signaling pathway, Fc gamma R-mediated phagocytosis, TNF signaling pathway, Neurotrophin signaling pathway, Cholinergic synapse, Dopaminergic synapse, Insulin signaling pathway, Progesterone-mediated oocyte maturation, Estrogen signaling pathway, Prolactin signaling pathway, Thyroid hormone signaling pathway, Adipocytokine signaling pathway, Glucagon signaling pathway, Regulation of lipolysis in adipocytes, Relaxin signaling pathway, GnRH secretion, Insulin resistance, Non-alcoholic fatty liver disease, AGE-RAGE signaling pathway in diabetic complications, Growth hormone synthesis, secretion and action, Alcoholic liver disease, Carbohydrate digestion and absorption, Alzheimer disease, Spinocerebellar ataxia, Shigellosis, Salmonella infection, Yersinia infection, Chagas disease, Toxoplasmosis, Tuberculosis, Hepatitis C, Hepatitis B, Measles, Human cytomegalovirus infection, Influenza A, Human papillomavirus infection, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Herpes simplex virus 1 infection, Epstein-Barr virus infection, Human immunodeficiency virus 1 infection, Pathways in cancer, Proteoglycans in cancer, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species, Colorectal cancer, Renal cell carcinoma, Pancreatic cancer, Endometrial cancer, Glioma, Prostate cancer, Melanoma, Chronic myeloid leukemia, Acute myeloid leukemia, Small cell lung cancer, Non-small cell lung cancer, Breast cancer, Hepatocellular carcinoma, Gastric cancer, Central carbon metabolism in cancer, Choline metabolism in cancer, PD-L1 expression and PD-1 checkpoint pathway in cancer, Diabetic cardiomyopathy, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[c(\"nt06114  PI3K signaling (viruses)\", \"nt06124  Chemokine signaling (viruses)\", \"nt06135  Cytoskeletal regulation (viruses and bacteria)\", \"nt06150  Cytokine-cytokine receptor interaction (viruses)\", \"nt06160  Human T-cell leukemia virus 1 (HTLV-1)\", \"nt06161  Human immunodeficiency virus type 1 (HIV-1)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06163  Hepatitis C virus (HCV)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06165  Epstein-Barr virus (EBV)\", \"nt06166  Human papillomavirus (HPV)\", \n\"nt06167  Human cytomegalovirus (HCMV)\", \"nt06168  Herpes simplex virus 1 (HSV-1)\", \"nt06169  Measles virus (MV)\", \"nt06170  Influenza A virus (IAV)\", \"nt06214  PI3K signaling\", \"nt06224  CXCR signaling\", \"nt06260  Colorectal cancer\", \"nt06261  Gastric cancer\", \"nt06262  Pancreatic cancer\", \"nt06263  Hepatocellular carcinoma\", \"nt06264  Renal cell carcinoma\", \"nt06266  Non-small cell lung cancer\", \"nt06267  Small cell lung cancer\", \"nt06268  Melanoma\", \"nt06270  Breast cancer\", \"nt06271  Endometrial cancer\", \n\"nt06272  Prostate cancer\", \"nt06273  Glioma\", \"nt06275  Acute myeloid leukemia\", \"nt06276  Chronic myeloid leukemia\", \"nt06415  PI3K signaling\", \"nt06462  Spinocerebellar ataxia\", \"nt06464  Amyotrophic lateral sclerosis\")]','[N00030  EGF-EGFR-RAS-PI3K signaling pathway, N00031  Duplication or mutation-activated FLT3 to RAS-PI3K signaling pathway, N00032  Mutation-activated KRAS/NRAS to PI3K signaling pathway, N00033  EGF-EGFR-PI3K signaling pathway, N00034  ERBB2-overexpression to PI3K signaling pathway, N00035  Amplified EGFR to PI3K signaling pathway, N00036  Mutation-activated EGFR to PI3K signaling pathway, N00037  FGF-FGFR-PI3K signaling pathway, N00038  Amplified FGFR to PI3K signaling pathway, N00039  PDGF-PDGFR-PI3K signaling pathway, N00040  Amplified PDGFR to PI3K signaling pathway, N00042  EGFR-overexpression to PI3K signaling pathway, N00043  HGF-MET-PI3K signaling pathway, N00044  Mutation-activated MET to PI3K signaling pathway, N00045  KITLG-KIT-PI3K signaling pathway, N00046  Mutation-activated KIT to PI3K signaling pathway, N00047  EML4-ALK fusion kinase to PI3K signaling pathway, N00048  BCR-ABL fusion kinase to PI3K signaling pathway, N00049  Mutation-activated PI3K to PI3K signaling pathway, N00050  Amplified PI3K to PI3K signaling pathway, N00051  Deleted PTEN to PI3K signaling pathway, N00052  Mutation-inactivated PTEN to PI3K signaling pathway, N00082  Loss of NKX3-1 to PI3K signaling pathway, N00154  CXCR-GNB/G-PI3K-AKT signaling pathway, N00158  KSHV vGPCR to GNB/G-PI3K-AKT signaling pathway, N00159  KSHV K1 to PI3K signaling pathway, N00179  KSHV K1 to PI3K-NFKB signaling pathway, N00182  IGF-IGFR-PI3K-NFKB signaling pathway, N00218  FLT3LG-FLT3-RAS-PI3K signaling pathway, N00220  PTEN-PIP3-AKT signaling pathway, N00231  TGFA-EGFR-PI3K signaling pathway, N00232  TGFA-overexpression to PI3K signaling pathway, N00234  IGF2-IGF1R-PI3K signaling pathway, N00236  IGF2-overexpression to PI3K signaling pathway, N00238  IGF1R-overexpression to PI3K signaling pathway, N00247  HGF-overexpression to PI3K signaling pathway, N00249  MET-overexpression to PI3K signaling pathway, N00253  Amplified ERBB2 to PI3K signaling pathway, N00260  Amplified MET to PI3K signaling pathway, N00266  EBV LMP2A to PI3K signaling pathway, N00267  HBV HBx to PI3K signaling pathway, N00272  HCV NS5A to PI3K signaling pathway, N00281  EGF-overexpression to PI3K signaling pathway, N00282  EREG-EGFR-PI3K signaling pathway, N00283  EREG-overexpression to PI3K signaling pathway, N00284  AREG-EGFR-PI3K signaling pathway, N00285  AREG-overexpression to PI3K signaling pathway, N00342  MAGI-PTEN signaling pathway, N00355  PP2A-AKT signaling pathway, N00366  HPV E5 to EGFR-PI3K signaling pathway, N00368  HPV E5 to PDGFR-PI3K signaling pathway, N00385  HCMV gB to PDGFR-PI3K signaling pathway, N00387  HCMV IE1-72/IE2-86 to PI3K signaling pathway, N00390  EGF-EGFR-PI3K-NFKB signaling pathway, N00391  HCMV gB to EGFR-PI3K-NFKB signaling pathway, N00399  CCR2-GNB/G-PI3K-NFKB signaling pathway, N00400  HCMV US28 to GNB/G-PI3K-NFKB signaling pathway, N00430  CXCR4-GNAI-PI3K-BAD signaling pathway, N00431  HIV gp120 to CXCR4-GNAI-PI3K-BAD signaling pathway, N00485  EBV LMP1 to PI3K signaling pathway, N00501  HTLV-1 Tax to EGFR-PI3K-NFKB signaling pathway, N00514  Mutation-activated EGFR to PI3K signaling pathway, N00582  IGF-IGF1R-PI3K signaling pathway, N00583  HSV VP11/12 to PI3K signaling pathway, N00683  CD80/CD86-CD28-PI3K signaling pathway, N00684  MV F/H to CD28-PI3K signaling pathway, N00736  IAV NS1 to PI3K signaling pathway, N00963  RELN-VLDLR-PI3K signaling pathway, N00964  DAB1-overexpression to RELN-VLDLR-PI3K signaling pathway, N01063  Mutation-activated MET to PI3K signaling pathway, N01065  Mutation-activated RET to PI3K signaling pathway, N01163  NRG-ERBB4-PI3K signaling pathway, N01338  ACH-CHRN-PI3K signaling pathway, N01339  NNK/NNN to PI3K signaling pathway, N01347  EP/NE-ADRB-PI3K signaling pathway, N01348  Nicotine/NNK to PI3K signaling pathway, N01349  ACH-CHRN-PI3K signaling pathway, N01350  NNK/NNN to PI3K signaling pathway, N01355  Arsenic to PI3K signaling pathway, N01356  Membrane-initiated progesterone signaling pathway, N01357  P4/MPA to membrane-initiated progesterone signaling pathway, N01358  P4-PR-PI3K signaling pathway, N01359  P4/MPA to PR-PI3K signaling pathway, N01409  Metals to PI3K signaling pathway]','04010 MAPK signaling pathway; 04012 ErbB signaling pathway; 04014 Ras signaling pathway; 04015 Rap1 signaling pathway; 04370 VEGF signaling pathway; 04371 Apelin signaling pathway; 04630 JAK-STAT signaling pathway; 04668 TNF signaling pathway; 04066 HIF-1 signaling pathway; 04068 FoxO signaling pathway; 04072 Phospholipase D signaling pathway; 04071 Sphingolipid signaling pathway; 04024 cAMP signaling pathway; 04022 cGMP-PKG signaling pathway; 04151 PI3K-Akt signaling pathway; 04152 AMPK signaling pathway; 04150 mTOR signaling pathway; 04140 Autophagy - animal; 04210 Apoptosis; 04218 Cellular senescence; 04510 Focal adhesion; 04550 Signaling pathways regulating pluripotency of stem cells; 04611 Platelet activation; 04613 Neutrophil extracellular trap formation; 04620 Toll-like receptor signaling pathway; 04625 C-type lectin receptor signaling pathway; 04660 T cell receptor signaling pathway; 04662 B cell receptor signaling pathway; 04664 Fc epsilon RI signaling pathway; 04666 Fc gamma R-mediated phagocytosis; 04062 Chemokine signaling pathway; 04910 Insulin signaling pathway; 04922 Glucagon signaling pathway; 04923 Regulation of lipolysis in adipocytes; 04920 Adipocytokine signaling pathway; 04929 GnRH secretion; 04915 Estrogen signaling pathway; 04914 Progesterone-mediated oocyte maturation; 04917 Prolactin signaling pathway; 04926 Relaxin signaling pathway; 04935 Growth hormone synthesis, secretion and action; 04919 Thyroid hormone signaling pathway; 04261 Adrenergic signaling in cardiomyocytes; 04973 Carbohydrate digestion and absorption; 04725 Cholinergic synapse; 04728 Dopaminergic synapse; 04722 Neurotrophin signaling pathway; 04380 Osteoclast differentiation; 04211 Longevity regulating pathway; 04213 Longevity regulating pathway - multiple species; 05200 Pathways in cancer; 05205 Proteoglycans in cancer; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 05230 Central carbon metabolism in cancer; 05231 Choline metabolism in cancer; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05210 Colorectal cancer; 05212 Pancreatic cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05214 Glioma; 05221 Acute myeloid leukemia; 05220 Chronic myeloid leukemia; 05218 Melanoma; 05211 Renal cell carcinoma; 05215 Prostate cancer; 05213 Endometrial cancer; 05224 Breast cancer; 05222 Small cell lung cancer; 05223 Non-small cell lung cancer; 05166 Human T-cell leukemia virus 1 infection; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05160 Hepatitis C; 05164 Influenza A; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05165 Human papillomavirus infection; 05132 Salmonella infection; 05131 Shigellosis; 05135 Yersinia infection; 05152 Tuberculosis; 05145 Toxoplasmosis; 05142 Chagas disease; 05010 Alzheimer disease; 05017 Spinocerebellar ataxia; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 05415 Diabetic cardiomyopathy; 04936 Alcoholic liver disease; 04932 Non-alcoholic fatty liver disease; 04931 Insulin resistance; 04933 AGE-RAGE signaling pathway in diabetic complications; 01521 EGFR tyrosine kinase inhibitor resistance; 01524 Platinum drug resistance; 01522 Endocrine resistance; 01001 Protein kinases [BR:hsa01001]; 2.7.11.1  non-specific serine/threonine protein kinase; AKT/PKB family; ','[Pfam: Pkinase PK_Tyr_Ser-Thr PH Pkinase_C Kinase-like PH_11 PH_20 Seadorna_VP7 Pkinase_fungal ABC1 APH]'),('6647','hsa:6647','[SOD1, ALS, ALS1, HEL-S-44, IPOA, SOD, STAHP, hSod1, homodimer]','[(RefSeq) superoxide dismutase 1]','[superoxide dismutase, Cu-Zn family [EC:1.15.1.1]]','[Homo sapiens (human)]','[Peroxisome, Longevity regulating pathway - multiple species, Parkinson disease, Amyotrophic lateral sclerosis, Huntington disease, Prion disease, Pathways of neurodegeneration - multiple diseases, Chemical carcinogenesis - reactive oxygen species]','[c(\"nt06253  Antioxidant system\", \"nt06412  Unfolded protein response (UPR) signaling\", \"nt06414  Apoptosis\", \"nt06419  Microtubule-based transport\", \"nt06420  Ubiquitin-proteasome system\", \"nt06464  Amyotrophic lateral sclerosis\")]','[N01135  Mutation-caused aberrant SOD1 to intrinsic apoptotic pathway, N01144  Mutation-caused aberrant SOD1 to 26S proteasome-mediated protein degradation, N01147  Mutation-caused aberrant SOD1 to ATF6-mediated transcription, N01148  Mutation-caused aberrant SOD1 to IRE1a-XBP1 signaling pathway, N01149  Mutation-caused aberrant SOD1 to PERK-ATF4 signaling pathway, N01160  Mutation-caused aberrant SOD1 to retrograde axonal transport, N01388  SOD/Cat-mediated ROS neutralization, N01389  Lead to SOD/Cat-mediated ROS neutralization, N01390  p,p\'-DDT to SOD/Cat-mediated ROS neutralization]','04146 Peroxisome; 04213 Longevity regulating pathway - multiple species; 05208 Chemical carcinogenesis - reactive oxygen species; 05012 Parkinson disease; 05014 Amyotrophic lateral sclerosis; 05016 Huntington disease; 05020 Prion disease; 05022 Pathways of neurodegeneration - multiple diseases; 1.15.1.1  superoxide dismutase; ','[Pfam: Sod_Cu]'),('847','hsa:847','[CAT]','[(RefSeq) catalase]','[catalase [EC:1.11.1.6]]','[Homo sapiens (human)]','[Tryptophan metabolism, Glyoxylate and dicarboxylate metabolism, Metabolic pathways, Carbon metabolism, FoxO signaling pathway, Peroxisome, Longevity regulating pathway, Longevity regulating pathway - multiple species, Amyotrophic lateral sclerosis, Pathways of neurodegeneration - multiple diseases, Chemical carcinogenesis - reactive oxygen species]','[nt06253  Antioxidant system]','[N01388  SOD/Cat-mediated ROS neutralization, N01391  Lead to SOD/Cat-mediated ROS neutralization]','00630 Glyoxylate and dicarboxylate metabolism; 00380 Tryptophan metabolism; 04068 FoxO signaling pathway; 04146 Peroxisome; 04211 Longevity regulating pathway; 04213 Longevity regulating pathway - multiple species; 05208 Chemical carcinogenesis - reactive oxygen species; 05014 Amyotrophic lateral sclerosis; 05022 Pathways of neurodegeneration - multiple diseases; 1.11.1.6  catalase; ','[Pfam: Catalase Catalase-rel]'),('2878','hsa:2878','[GPX3, GPx-P, GSHPx-3, GSHPx-P]','[(RefSeq) glutathione peroxidase 3]','[glutathione peroxidase [EC:1.11.1.9]]','[Homo sapiens (human)]','[Glutathione metabolism, Arachidonic acid metabolism, Metabolic pathways, Thyroid hormone synthesis, Amyotrophic lateral sclerosis, Huntington disease, Pathways of neurodegeneration - multiple diseases]','[nt06026  Glutathione biosynthesis]','[N00899  Glutathione biosynthesis]','00590 Arachidonic acid metabolism; 00480 Glutathione metabolism; 04918 Thyroid hormone synthesis; 05014 Amyotrophic lateral sclerosis; 05016 Huntington disease; 05022 Pathways of neurodegeneration - multiple diseases; 1.11.1.9  glutathione peroxidase; ','[Pfam: GSHPx]'),('596','hsa:596','[BCL2, Bcl-2, PPP1R50]','[(RefSeq) BCL2 apoptosis regulator]','[apoptosis regulator Bcl-2]','[Homo sapiens (human)]','[EGFR tyrosine kinase inhibitor resistance, Endocrine resistance, Platinum drug resistance, NF-kappa B signaling pathway, HIF-1 signaling pathway, Sphingolipid signaling pathway, p53 signaling pathway, Autophagy - animal, Protein processing in endoplasmic reticulum, PI3K-Akt signaling pathway, Apoptosis, Apoptosis - multiple species, Necroptosis, Adrenergic signaling in cardiomyocytes, Hedgehog signaling pathway, Focal adhesion, NOD-like receptor signaling pathway, JAK-STAT signaling pathway, Neurotrophin signaling pathway, Cholinergic synapse, Estrogen signaling pathway, Parathyroid hormone synthesis, secretion and action, AGE-RAGE signaling pathway in diabetic complications, Amyotrophic lateral sclerosis, Pathways of neurodegeneration - multiple diseases, Shigellosis, Salmonella infection, Toxoplasmosis, Tuberculosis, Hepatitis B, Measles, Herpes simplex virus 1 infection, Epstein-Barr virus infection, Human immunodeficiency virus 1 infection, Pathways in cancer, MicroRNAs in cancer, Chemical carcinogenesis - receptor activation, Colorectal cancer, Prostate cancer, Small cell lung cancer, Gastric cancer, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[c(\"nt06131  Apoptosis (viruses and bacteria)\", \"nt06161  Human immunodeficiency virus type 1 (HIV-1)\", \"nt06165  Epstein-Barr virus (EBV)\", \"nt06166  Human papillomavirus (HPV)\", \"nt06168  Herpes simplex virus 1 (HSV-1)\", \"nt06170  Influenza A virus (IAV)\", \"nt06210  ERK signaling\", \"nt06227  Nuclear receptor signaling\", \"nt06231  Apoptosis\", \"nt06260  Colorectal cancer\", \"nt06267  Small cell lung cancer\", \"nt06270  Breast cancer\", \"nt06323  KISS1-GnRH-LH/FSH-E2 signaling\", \"nt06414  Apoptosis\", \n\"nt06463  Parkinson disease\", \"nt06464  Amyotrophic lateral sclerosis\")]','[N00098  Intrinsic apoptotic pathway, N00100  BCL2-overexpression to intrinsic apoptotic pathway, N00286  Nuclear-initiated estrogen signaling pathway, N00451  HIV Tat to intrinsic apoptotic pathway, N00453  HIV Vpr to intrinsic apoptotic pathway, N00475  EBV BHRF1 to intrinsic apoptotic pathway, N00478  EBV BARF1 to intrinsic apoptotic pathway, N01135  Mutation-caused aberrant SOD1 to intrinsic apoptotic pathway, N01344  NNK/NNN to RAS-ERK signaling pathway]','04141 Protein processing in endoplasmic reticulum; 04340 Hedgehog signaling pathway; 04630 JAK-STAT signaling pathway; 04064 NF-kappa B signaling pathway; 04066 HIF-1 signaling pathway; 04071 Sphingolipid signaling pathway; 04151 PI3K-Akt signaling pathway; 04140 Autophagy - animal; 04210 Apoptosis; 04215 Apoptosis - multiple species; 04217 Necroptosis; 04115 p53 signaling pathway; 04510 Focal adhesion; 04621 NOD-like receptor signaling pathway; 04915 Estrogen signaling pathway; 04928 Parathyroid hormone synthesis, secretion and action; 04261 Adrenergic signaling in cardiomyocytes; 04725 Cholinergic synapse; 04722 Neurotrophin signaling pathway; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05207 Chemical carcinogenesis - receptor activation; 05210 Colorectal cancer; 05226 Gastric cancer; 05215 Prostate cancer; 05222 Small cell lung cancer; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05169 Epstein-Barr virus infection; 05132 Salmonella infection; 05131 Shigellosis; 05152 Tuberculosis; 05145 Toxoplasmosis; 05014 Amyotrophic lateral sclerosis; 05022 Pathways of neurodegeneration - multiple diseases; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 04933 AGE-RAGE signaling pathway in diabetic complications; 01521 EGFR tyrosine kinase inhibitor resistance; 01524 Platinum drug resistance; 01522 Endocrine resistance; 01009 Protein phosphatases and associated proteins [BR:hsa01009]; PP1-interacting proteins (PIPs); ','[Pfam: Bcl-2 BH4 Bcl-2_3]'),('581','hsa:581','[BAX, BCL2L4]','[(RefSeq) BCL2 associated X, apoptosis regulator]','[apoptosis regulator BAX]','[Homo sapiens (human)]','[EGFR tyrosine kinase inhibitor resistance, Endocrine resistance, Platinum drug resistance, Sphingolipid signaling pathway, p53 signaling pathway, Protein processing in endoplasmic reticulum, Apoptosis, Longevity regulating pathway, Apoptosis - multiple species, Necroptosis, Neurotrophin signaling pathway, Non-alcoholic fatty liver disease, AGE-RAGE signaling pathway in diabetic complications, Parkinson disease, Amyotrophic lateral sclerosis, Huntington disease, Prion disease, Pathways of neurodegeneration - multiple diseases, Pathogenic Escherichia coli infection, Shigellosis, Salmonella infection, Tuberculosis, Hepatitis C, Hepatitis B, Measles, Human cytomegalovirus infection, Influenza A, Human papillomavirus infection, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Herpes simplex virus 1 infection, Epstein-Barr virus infection, Human immunodeficiency virus 1 infection, Pathways in cancer, Transcriptional misregulation in cancer, Viral carcinogenesis, Colorectal cancer, Pancreatic cancer, Endometrial cancer, Glioma, Thyroid cancer, Basal cell carcinoma, Melanoma, Chronic myeloid leukemia, Small cell lung cancer, Non-small cell lung cancer, Breast cancer, Hepatocellular carcinoma, Gastric cancer, Lipid and atherosclerosis]','[c(\"nt06131  Apoptosis (viruses and bacteria)\", \"nt06140  Transcription (viruses)\", \"nt06160  Human T-cell leukemia virus 1 (HTLV-1)\", \"nt06161  Human immunodeficiency virus type 1 (HIV-1)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06163  Hepatitis C virus (HCV)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06165  Epstein-Barr virus (EBV)\", \"nt06166  Human papillomavirus (HPV)\", \"nt06167  Human cytomegalovirus (HCMV)\", \"nt06168  Herpes simplex virus 1 (HSV-1)\", \"nt06170  Influenza A virus (IAV)\", \n\"nt06231  Apoptosis\", \"nt06240  Transcription\", \"nt06260  Colorectal cancer\", \"nt06261  Gastric cancer\", \"nt06262  Pancreatic cancer\", \"nt06263  Hepatocellular carcinoma\", \"nt06265  Bladder cancer\", \"nt06266  Non-small cell lung cancer\", \"nt06267  Small cell lung cancer\", \"nt06268  Melanoma\", \"nt06269  Basal cell carcinoma\", \"nt06270  Breast cancer\", \"nt06271  Endometrial cancer\", \"nt06273  Glioma\", \"nt06274  Thyroid cancer\", \"nt06276  Chronic myeloid leukemia\", \"nt06414  Apoptosis\", \"nt06440  Transcription\", \n\"nt06460  Alzheimer disease\", \"nt06461  Huntington disease\", \"nt06463  Parkinson disease\", \"nt06464  Amyotrophic lateral sclerosis\")]','[N00098  Intrinsic apoptotic pathway, N00099  Mutation-inactivated BAX to apoptotic pathway, N00100  BCL2-overexpression to intrinsic apoptotic pathway, N00115  Mutation-inactivated TP53 to transcription, N00146  Crosstalk between extrinsic and intrinsic apoptotic pathways, N00164  KSHV vBCL2 to crosstalk between extrinsic and intrinsic apoptotic pathways, N00223  EBV EBNA1 to p53-mediated transcription, N00263  EBV EBNA3C to p53-mediated transcription, N00426  HCMV UL37x1 to crosstalk between extrinsic and intrinsic apoptotic pathways, N00449  HIV Tat/Nef to crosstalk between extrinsic and intrinsic apoptotic pathways, N00450  HIV Tat to intrinsic apoptotic pathway, N00453  HIV Vpr to intrinsic apoptotic pathway, N00454  HIV Vpr to intrinsic apoptotic pathway, N00475  EBV BHRF1 to intrinsic apoptotic pathway, N00478  EBV BARF1 to intrinsic apoptotic pathway, N00481  EBV BZLF1 to p53-mediated transcription, N00511  HTLV-1 Tax to E47-mediated transcription, N00526  HCV NS3 to crosstalk between extrinsic and intrinsic apoptotic pathways, N00527  HCV Core to crosstalk between extrinsic and intrinsic apoptotic pathways, N00534  HBV HBx to Crosstalk between extrinsic and intrinsic apoptotic pathways, N00535  HBV HBx to p53-mediated transcription, N00938  Escherichia NleH to intrinsic apoptotic pathway, N00950  Shigella FimA to crosstalk between extrinsic and intrinsic apoptotic pathways, N00982  Mutation-caused aberrant Htt to p53-mediated transcription, N01005  Mutation-caused aberrant Abeta to crosstalk between extrinsic and intrinsic apoptotic pathways, N01049  Mutation-inactivated PRKN to intrinsic apoptotic pathway, N01050  Mutation-inactivated PINK1 to intrinsic apoptotic pathway, N01051  Mutation-inactivated DJ1 to intrinsic apoptotic pathway, N01135  Mutation-caused aberrant SOD1 to intrinsic apoptotic pathway]','04141 Protein processing in endoplasmic reticulum; 04071 Sphingolipid signaling pathway; 04210 Apoptosis; 04215 Apoptosis - multiple species; 04217 Necroptosis; 04115 p53 signaling pathway; 04722 Neurotrophin signaling pathway; 04211 Longevity regulating pathway; 05200 Pathways in cancer; 05202 Transcriptional misregulation in cancer; 05203 Viral carcinogenesis; 05210 Colorectal cancer; 05212 Pancreatic cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05214 Glioma; 05216 Thyroid cancer; 05220 Chronic myeloid leukemia; 05217 Basal cell carcinoma; 05218 Melanoma; 05213 Endometrial cancer; 05224 Breast cancer; 05222 Small cell lung cancer; 05223 Non-small cell lung cancer; 05166 Human T-cell leukemia virus 1 infection; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05160 Hepatitis C; 05164 Influenza A; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05165 Human papillomavirus infection; 05130 Pathogenic Escherichia coli infection; 05132 Salmonella infection; 05131 Shigellosis; 05152 Tuberculosis; 05012 Parkinson disease; 05014 Amyotrophic lateral sclerosis; 05016 Huntington disease; 05020 Prion disease; 05022 Pathways of neurodegeneration - multiple diseases; 05417 Lipid and atherosclerosis; 04932 Non-alcoholic fatty liver disease; 04933 AGE-RAGE signaling pathway in diabetic complications; 01521 EGFR tyrosine kinase inhibitor resistance; 01524 Platinum drug resistance; 01522 Endocrine resistance; 03029 Mitochondrial biogenesis [BR:hsa03029]; 02000 Transporters [BR:hsa02000]; Fission and Fusion factors; Pores ion channels [TC:1]; ','[Pfam: Bcl-2]'),('836','hsa:836','[CASP3, CPP32, CPP32B, SCA-1]','[(RefSeq) caspase 3]','[caspase 3 [EC:3.4.22.56]]','[Homo sapiens (human)]','[Platinum drug resistance, MAPK signaling pathway, p53 signaling pathway, Apoptosis, Apoptosis - multiple species, Natural killer cell mediated cytotoxicity, IL-17 signaling pathway, TNF signaling pathway, Serotonergic synapse, Non-alcoholic fatty liver disease, AGE-RAGE signaling pathway in diabetic complications, Alcoholic liver disease, Alzheimer disease, Parkinson disease, Amyotrophic lateral sclerosis, Huntington disease, Prion disease, Pathways of neurodegeneration - multiple diseases, Epithelial cell signaling in Helicobacter pylori infection, Pathogenic Escherichia coli infection, Salmonella infection, Pertussis, Legionellosis, Toxoplasmosis, Amoebiasis, Tuberculosis, Hepatitis C, Hepatitis B, Measles, Human cytomegalovirus infection, Influenza A, Human papillomavirus infection, Kaposi sarcoma-associated herpesvirus infection, Herpes simplex virus 1 infection, Epstein-Barr virus infection, Human immunodeficiency virus 1 infection, Pathways in cancer, Viral carcinogenesis, Proteoglycans in cancer, MicroRNAs in cancer, Colorectal cancer, Small cell lung cancer, Viral myocarditis, Lipid and atherosclerosis]','[c(\"nt06131  Apoptosis (viruses and bacteria)\", \"nt06161  Human immunodeficiency virus type 1 (HIV-1)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06163  Hepatitis C virus (HCV)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06165  Epstein-Barr virus (EBV)\", \"nt06166  Human papillomavirus (HPV)\", \"nt06167  Human cytomegalovirus (HCMV)\", \"nt06168  Herpes simplex virus 1 (HSV-1)\", \"nt06170  Influenza A virus (IAV)\", \"nt06231  Apoptosis\", \"nt06260  Colorectal cancer\", \"nt06263  Hepatocellular carcinoma\", \n\"nt06267  Small cell lung cancer\", \"nt06410  Calcium signaling\", \"nt06414  Apoptosis\", \"nt06460  Alzheimer disease\", \"nt06461  Huntington disease\", \"nt06462  Spinocerebellar ataxia\", \"nt06463  Parkinson disease\", \"nt06464  Amyotrophic lateral sclerosis\", \"nt06465  Prion disease\")]','[N00098  Intrinsic apoptotic pathway, N00100  BCL2-overexpression to intrinsic apoptotic pathway, N00101  DCC-apoptotic pathway, N00102  Loss of DCC to DCC-apoptotic pathway, N00145  Extrinsic apoptotic pathway, N00146  Crosstalk between extrinsic and intrinsic apoptotic pathways, N00165  KSHV vIAP to crosstalk between extrinsic and intrinsic apoptotic pathways, N00449  HIV Tat/Nef to crosstalk between extrinsic and intrinsic apoptotic pathways, N00450  HIV Tat to intrinsic apoptotic pathway, N00453  HIV Vpr to intrinsic apoptotic pathway, N00475  EBV BHRF1 to intrinsic apoptotic pathway, N00478  EBV BARF1 to intrinsic apoptotic pathway, N00526  HCV NS3 to crosstalk between extrinsic and intrinsic apoptotic pathways, N00527  HCV Core to crosstalk between extrinsic and intrinsic apoptotic pathways, N00534  HBV HBx to Crosstalk between extrinsic and intrinsic apoptotic pathways, N00580  HSV ICP0 to crosstalk between extrinsic and intrinsic apoptotic pathways, N00745  IAV PB1-F2 to intrinsic apoptotic pathway, N00939  Escherichia EspF to intrinsic apoptotic pathway, N00967  VGCC-Ca2+ -apoptotic pathway, N00983  Mutation-caused aberrant Htt to extrinsic apoptotic pathway, N00984  mGluR5-Ca2+ -apoptotic pathway, N00985  Mutation-caused aberrant Htt to mGluR5-Ca2+ -apoptotic pathway, N00987  Mutation-caused aberrant Htt to transport of calcium, N01002  Mutation-caused aberrant Abeta to mGluR5-Ca2+ -apoptotic pathway, N01005  Mutation-caused aberrant Abeta to crosstalk between extrinsic and intrinsic apoptotic pathways, N01007  Mutation-caused aberrant PSEN to mGluR5-Ca2+ -apoptotic pathway, N01031  Mutation-caused aberrant SNCA to VGCC-Ca2+ -apoptotic pathway, N01047  Mutation-activated LRRK2 to intrinsic apoptotic pathway, N01048  Mutation-inactivated PINK1 to intrinsic apoptotic pathway, N01049  Mutation-inactivated PRKN to intrinsic apoptotic pathway, N01050  Mutation-inactivated PINK1 to intrinsic apoptotic pathway, N01135  Mutation-caused aberrant SOD1 to intrinsic apoptotic pathway, N01199  Scrapie conformation PrPSc to mGluR5-Ca2+ -apoptotic pathway, N01200  Scrapie conformation PrPSc to transport of calcium]','04010 MAPK signaling pathway; 04668 TNF signaling pathway; 04210 Apoptosis; 04215 Apoptosis - multiple species; 04115 p53 signaling pathway; 04650 Natural killer cell mediated cytotoxicity; 04657 IL-17 signaling pathway; 04726 Serotonergic synapse; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05205 Proteoglycans in cancer; 05203 Viral carcinogenesis; 05210 Colorectal cancer; 05222 Small cell lung cancer; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05160 Hepatitis C; 05164 Influenza A; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05165 Human papillomavirus infection; 05120 Epithelial cell signaling in Helicobacter pylori infection; 05130 Pathogenic Escherichia coli infection; 05132 Salmonella infection; 05133 Pertussis; 05134 Legionellosis; 05152 Tuberculosis; 05146 Amoebiasis; 05145 Toxoplasmosis; 05010 Alzheimer disease; 05012 Parkinson disease; 05014 Amyotrophic lateral sclerosis; 05016 Huntington disease; 05020 Prion disease; 05022 Pathways of neurodegeneration - multiple diseases; 05417 Lipid and atherosclerosis; 05416 Viral myocarditis; 04936 Alcoholic liver disease; 04932 Non-alcoholic fatty liver disease; 04933 AGE-RAGE signaling pathway in diabetic complications; 01524 Platinum drug resistance; 01002 Peptidases and inhibitors [BR:hsa01002]; 3.4.22.56  caspase-3; Family C14: caspase family; ','[Pfam: Peptidase_C14 Erf4 SICA_beta]'),('1581','hsa:1581','[CYP7A1, CP7A, CYP7, CYPVII]','[(RefSeq) cytochrome P450 family 7 subfamily A member 1]','[cholesterol 7alpha-monooxygenase [EC:1.14.14.23]]','[Homo sapiens (human)]','[Primary bile acid biosynthesis, Steroid hormone biosynthesis, Metabolic pathways, PPAR signaling pathway, Bile secretion, Cholesterol metabolism]','[nt06022  Bile acid biosynthesis]','[N00805  Bile acid biosynthesis]','00120 Primary bile acid biosynthesis; 00140 Steroid hormone biosynthesis; 03320 PPAR signaling pathway; 04976 Bile secretion; 04979 Cholesterol metabolism; 00199 Cytochrome P450 [BR:hsa00199]; 1.14.14.23  cholesterol 7alpha-monooxygenase; CYP7 family; ','[Pfam: p450]'),('151306','hsa:151306','[GPBAR1, BG37, GPCR19, GPR131, M-BAR, TGR5]','[(RefSeq) G protein-coupled bile acid receptor 1]','[G protein-coupled bile acid receptor 1]','[Homo sapiens (human)]',NULL,'[]','[]','04030 G protein-coupled receptors [BR:hsa04030]; Bile acid; ','[Pfam: 7tm_1]'),('10333','hsa:10333','[TLR6, CD286]','[(RefSeq) toll like receptor 6]','[toll-like receptor 6]','[Homo sapiens (human)]','[Phagosome, Toll-like receptor signaling pathway, Salmonella infection, Chagas disease, Tuberculosis, Lipid and atherosclerosis]','[]','[]','04145 Phagosome; 04620 Toll-like receptor signaling pathway; 05132 Salmonella infection; 05152 Tuberculosis; 05142 Chagas disease; 05417 Lipid and atherosclerosis; 04131 Membrane trafficking [BR:hsa04131]; 04054 Pattern recognition receptors [BR:hsa04054]; 04090 CD molecules [BR:hsa04090]; Toll-like receptors; Toll-like receptors; Proteins; ','[Pfam: TIR LRR_8 TIR_2 LRR_4 LRR_6 LRRCT LRR_9]'),('7046','hsa:7046','[TGFBR1, AAT5, ACVRLK4, ALK-5, ALK5, ESS1, LDS1, LDS1A, LDS2A, MSSE, SKR4, TBR-i, TBRI, TGFR-1, tbetaR-I]','[(RefSeq) transforming growth factor beta receptor 1]','[TGF-beta receptor type-1 [EC:2.7.11.30]]','[Homo sapiens (human)]','[MAPK signaling pathway, Cytokine-cytokine receptor interaction, FoxO signaling pathway, Endocytosis, Cellular senescence, TGF-beta signaling pathway, Apelin signaling pathway, Osteoclast differentiation, Hippo signaling pathway, Adherens junction, Th17 cell differentiation, Relaxin signaling pathway, AGE-RAGE signaling pathway in diabetic complications, Chagas disease, Hepatitis B, Human T-cell leukemia virus 1 infection, Pathways in cancer, Colorectal cancer, Pancreatic cancer, Chronic myeloid leukemia, Hepatocellular carcinoma, Gastric cancer, Diabetic cardiomyopathy]','[c(\"nt06118  TGFB signaling (viruses)\", \"nt06160  Human T-cell leukemia virus 1 (HTLV-1)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06218  TGFB signaling\", \"nt06260  Colorectal cancer\", \"nt06261  Gastric cancer\", \"nt06263  Hepatocellular carcinoma\", \"nt06276  Chronic myeloid leukemia\")]','[N00063  TGF-beta signaling pathway, N00064  Mutation-inactivated TGFBR2 to TGF-beta signaling pathway, N00065  Mutation-inactivated SMAD2 to TGF-beta signaling pathway, N00241  TGFBR2-reduced expression to TGF-beta signaling pathway, N00256  TGFBR1-reduced expression to TGF-beta signaling pathway, N00531  HBV HBx to TGF-beta signaling pathway]','04010 MAPK signaling pathway; 04350 TGF-beta signaling pathway; 04390 Hippo signaling pathway; 04371 Apelin signaling pathway; 04068 FoxO signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04144 Endocytosis; 04218 Cellular senescence; 04520 Adherens junction; 04659 Th17 cell differentiation; 04926 Relaxin signaling pathway; 04380 Osteoclast differentiation; 05200 Pathways in cancer; 05210 Colorectal cancer; 05212 Pancreatic cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05220 Chronic myeloid leukemia; 05166 Human T-cell leukemia virus 1 infection; 05161 Hepatitis B; 05142 Chagas disease; 05415 Diabetic cardiomyopathy; 04933 AGE-RAGE signaling pathway in diabetic complications; 01001 Protein kinases [BR:hsa01001]; 2.7.11.30  receptor protein serine/threonine kinase; TGFBR1 family [OT]; ','[Pfam: Pkinase PK_Tyr_Ser-Thr TGF_beta_GS Activin_recp Pkinase_fungal APH FTA2 Haspin_kinase]'),('6606','hsa:6606','[SMN1, BCD541, GEMIN1, SMA, SMA1, SMA2, SMA3, SMA4, SMA@, SMN, SMNT, T-BCD541, TDRD16A]','[(RefSeq) survival of motor neuron 1, telomeric]','[survival motor neuron protein]','[Homo sapiens (human)]',NULL,'[]','[]','03041 Spliceosome [BR:hsa03041]; SMN complex factors; ','[Pfam: SMN TUDOR]'),('64750','hsa:64750','[SMURF2]','[(RefSeq) SMAD specific E3 ubiquitin protein ligase 2]','[E3 ubiquitin ligase SMURF1/2 [EC:2.3.2.26]]','[Homo sapiens (human)]','[Ubiquitin mediated proteolysis, Endocytosis, Hedgehog signaling pathway, TGF-beta signaling pathway]','[]','[]','04120 Ubiquitin mediated proteolysis; 04340 Hedgehog signaling pathway; 04350 TGF-beta signaling pathway; 04144 Endocytosis; 04131 Membrane trafficking [BR:hsa04131]; 04121 Ubiquitin system [BR:hsa04121]; 03029 Mitochondrial biogenesis [BR:hsa03029]; 2.3.2.26  HECT-type E3 ubiquitin transferase; Ubiquitin E3 ligases; Other mitophagy associated  proteins; HECT type E3; Parkin-independent mechanism factors; ','[Pfam: HECT WW C2 GpA_nuclease]'),('6751','hsa:6751','[SSTR1, SRIF-2, SS-1-R, SS1-R, SS1R]','[(RefSeq) somatostatin receptor 1]','[somatostatin receptor 1]','[Homo sapiens (human)]','[cAMP signaling pathway, Neuroactive ligand-receptor interaction, Growth hormone synthesis, secretion and action]','[]','[]','04024 cAMP signaling pathway; 04080 Neuroactive ligand-receptor interaction; 04935 Growth hormone synthesis, secretion and action; 04030 G protein-coupled receptors [BR:hsa04030]; Somatostatin; ','[Pfam: 7tm_1 7TM_GPCR_Srsx 7TM_GPCR_Srv 7TM_GPCR_Srx 7tm_4 T2SSM Piezo_RRas_bdg]'),('6753','hsa:6753','[SSTR3, SS-3-R, SS3-R, SS3R, SSR-28]','[(RefSeq) somatostatin receptor 3]','[somatostatin receptor 3]','[Homo sapiens (human)]','[Neuroactive ligand-receptor interaction, Growth hormone synthesis, secretion and action]','[]','[]','04080 Neuroactive ligand-receptor interaction; 04935 Growth hormone synthesis, secretion and action; 04030 G protein-coupled receptors [BR:hsa04030]; Somatostatin; ','[Pfam: 7tm_1 7TM_GPCR_Srsx 7TM_GPCR_Srx 7TM_GPCR_Srv 7TM_GPCR_Srw EMC6]'),('4780','hsa:4780','[NFE2L2, HEBP1, IMDDHH, NRF2, Nrf-2]','[(RefSeq) NFE2 like bZIP transcription factor 2]','[nuclear factor erythroid 2-related factor 2]','[Homo sapiens (human)]','[Protein processing in endoplasmic reticulum, Parkinson disease, Pathways in cancer, Chemical carcinogenesis - reactive oxygen species, Hepatocellular carcinoma, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[c(\"nt06226  KEAP1-NRF2 signaling\", \"nt06263  Hepatocellular carcinoma\", \"nt06440  Transcription\", \"nt06463  Parkinson disease\")]','[N00243  KEAP1-NRF2 signaling pathway, N00244  Mutation-inactivated KEAP1 to KEAP1-NRF2 signaling pathway, N00245  Mutation-activated NRF2 to KEAP1-NRF2 signaling pathway, N01059  Mutation-inactivated DJ1 to KEAP1-NRF2 signaling pathway, N01413  Metals to KEAP1-NRF2 signalig pathway]','04141 Protein processing in endoplasmic reticulum; 05200 Pathways in cancer; 05208 Chemical carcinogenesis - reactive oxygen species; 05225 Hepatocellular carcinoma; 05012 Parkinson disease; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 03000 Transcription factors [BR:hsa03000]; 03029 Mitochondrial biogenesis [BR:hsa03029]; AP-1(-like) components, NF-E2; Nuclear respiratory factors; ','[Pfam: bZIP_Maf bZIP_1 bZIP_2 T3SS_basalb_I]'),('3162','hsa:3162','[HMOX1, HMOX1D, HO-1, HSP32, bK286B10]','[(RefSeq) heme oxygenase 1]','[heme oxygenase 1 [EC:1.14.14.18]]','[Homo sapiens (human)]','[Porphyrin metabolism, Metabolic pathways, HIF-1 signaling pathway, Ferroptosis, Mineral absorption, Pathways in cancer, MicroRNAs in cancer, Chemical carcinogenesis - reactive oxygen species, Hepatocellular carcinoma, Fluid shear stress and atherosclerosis]','[c(\"nt06226  KEAP1-NRF2 signaling\", \"nt06263  Hepatocellular carcinoma\", \"nt06463  Parkinson disease\")]','[N00243  KEAP1-NRF2 signaling pathway, N00244  Mutation-inactivated KEAP1 to KEAP1-NRF2 signaling pathway, N00245  Mutation-activated NRF2 to KEAP1-NRF2 signaling pathway, N01413  Metals to KEAP1-NRF2 signalig pathway]','00860 Porphyrin metabolism; 04066 HIF-1 signaling pathway; 04216 Ferroptosis; 04978 Mineral absorption; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05208 Chemical carcinogenesis - reactive oxygen species; 05225 Hepatocellular carcinoma; 05418 Fluid shear stress and atherosclerosis; 1.14.14.18  heme oxygenase (biliverdin-producing); ','[Pfam: Heme_oxygenase REC1]'),('728','hsa:728','[C5AR1, C5A, C5AR, C5R1, CD88]','[(RefSeq) complement C5a receptor 1]','[C5a anaphylatoxin chemotactic receptor 1]','[Homo sapiens (human)]','[Neuroactive ligand-receptor interaction, Complement and coagulation cascades, Neutrophil extracellular trap formation, Alcoholic liver disease, Staphylococcus aureus infection, Coronavirus disease - COVID-19]','[]','[]','04080 Neuroactive ligand-receptor interaction; 04610 Complement and coagulation cascades; 04613 Neutrophil extracellular trap formation; 05171 Coronavirus disease - COVID-19; 05150 Staphylococcus aureus infection; 04936 Alcoholic liver disease; 04030 G protein-coupled receptors [BR:hsa04030]; 04090 CD molecules [BR:hsa04090]; Anaphylatoxin; Proteins; ','[Pfam: 7tm_1 7TM_GPCR_Srsx 7TM_GPCR_Srv 7TM_GPCR_Srx 7TM_GPCR_Srw]'),('6348','hsa:6348','[CCL3, G0S19-1, LD78ALPHA, MIP-1-alpha, MIP1A, SCYA3]','[(RefSeq) C-C motif chemokine ligand 3]','[C-C motif chemokine 3]','[Homo sapiens (human)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, Chemokine signaling pathway, Toll-like receptor signaling pathway, Chagas disease, Human cytomegalovirus infection, Rheumatoid arthritis, Lipid and atherosclerosis]','[c(\"nt06124  Chemokine signaling (viruses)\", \"nt06150  Cytokine-cytokine receptor interaction (viruses)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06167  Human cytomegalovirus (HCMV)\", \"nt06224  CXCR signaling\")]','[N00153  CCR/CXCR-GNB/G-PI3K-RAC signaling pathway, N00406  HCMV US28 to GNA12/13-Rho signaling pathway, N01166  CCL3/5/7/8/13/14/15/16/23 to CCR1 signaling, N01167  VAV vCCI to CCR signaling, N01168  CCL3/7 to HCMV US28 signaling, N01169  CCL3 to HHV6 U12 signaling, N01196  CCL3/4/5/8 to CCR5 signaling]','04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04620 Toll-like receptor signaling pathway; 04062 Chemokine signaling pathway; 05163 Human cytomegalovirus infection; 05142 Chagas disease; 05323 Rheumatoid arthritis; 05417 Lipid and atherosclerosis; 04052 Cytokines and growth factors [BR:hsa04052]; 00536 Glycosaminoglycan binding proteins [BR:hsa00536]; Chemokines; Chemokines; ','[Pfam: IL8]'),('5743','hsa:5743','[PTGS2, COX-2, COX2, GRIPGHS, PGG/HS, PGHS-2, PHS-2, hCox-2]','[(RefSeq) prostaglandin-endoperoxide synthase 2]','[prostaglandin-endoperoxide synthase 2 [EC:1.14.99.1]]','[Homo sapiens (human)]','[Arachidonic acid metabolism, Metabolic pathways, NF-kappa B signaling pathway, VEGF signaling pathway, C-type lectin receptor signaling pathway, IL-17 signaling pathway, TNF signaling pathway, Retrograde endocannabinoid signaling, Serotonergic synapse, Ovarian steroidogenesis, Oxytocin signaling pathway, Regulation of lipolysis in adipocytes, Alzheimer disease, Pathways of neurodegeneration - multiple diseases, Leishmaniasis, Human cytomegalovirus infection, Human papillomavirus infection, Kaposi sarcoma-associated herpesvirus infection, Pathways in cancer, Chemical carcinogenesis - DNA adducts, MicroRNAs in cancer, Small cell lung cancer]','[c(\"nt06124  Chemokine signaling (viruses)\", \"nt06150  Cytokine-cytokine receptor interaction (viruses)\", \"nt06167  Human cytomegalovirus (HCMV)\", \"nt06227  Nuclear receptor signaling\", \"nt06360  Cushing syndrome\")]','[N00317  AhR signaling pathway, N00402  HCMV US28 to GNAQ-PLCB/G-calcineurin signaling pathway]','00590 Arachidonic acid metabolism; 04370 VEGF signaling pathway; 04064 NF-kappa B signaling pathway; 04668 TNF signaling pathway; 04625 C-type lectin receptor signaling pathway; 04657 IL-17 signaling pathway; 04923 Regulation of lipolysis in adipocytes; 04913 Ovarian steroidogenesis; 04921 Oxytocin signaling pathway; 04726 Serotonergic synapse; 04723 Retrograde endocannabinoid signaling; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05204 Chemical carcinogenesis - DNA adducts; 05222 Small cell lung cancer; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05165 Human papillomavirus infection; 05140 Leishmaniasis; 05010 Alzheimer disease; 05022 Pathways of neurodegeneration - multiple diseases; 1.14.99.1  prostaglandin-endoperoxide synthase; ','[Pfam: An_peroxidase EGF]'),('3606','hsa:3606','[IL18, IGIF, IL-18, IL-1g, IL1F4]','[(RefSeq) interleukin 18]','[interleukin 18]','[Homo sapiens (human)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, NOD-like receptor signaling pathway, Cytosolic DNA-sensing pathway, Pathogenic Escherichia coli infection, Shigellosis, Salmonella infection, Legionellosis, Yersinia infection, African trypanosomiasis, Malaria, Tuberculosis, Influenza A, Inflammatory bowel disease, Rheumatoid arthritis, Lipid and atherosclerosis]','[c(\"nt06139  NLR signaling (viruses and bacteria)\", \"nt06150  Cytokine-cytokine receptor interaction (viruses)\", \"nt06170  Influenza A virus (IAV)\")]','[N00742  NLRP3 Inflammasome signaling pathway, N00867  NLRC4 Inflammasome signaling pathway, N00868  Pyrin Inflammasome signaling pathway, N00934  Non-canonical inflammasome signaling pathway, N01126  Salmonella SipB to Inflammasome signaling pathway, N01127  Salmonella SopE to Inflammasome signaling pathway, N01269  IL18/37 to IL18R signaling, N01270  MOCV1 MC53L/54L to IL18R signaling]','04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04621 NOD-like receptor signaling pathway; 04623 Cytosolic DNA-sensing pathway; 05164 Influenza A; 05130 Pathogenic Escherichia coli infection; 05132 Salmonella infection; 05131 Shigellosis; 05135 Yersinia infection; 05134 Legionellosis; 05152 Tuberculosis; 05144 Malaria; 05143 African trypanosomiasis; 05323 Rheumatoid arthritis; 05321 Inflammatory bowel disease; 05417 Lipid and atherosclerosis; 04052 Cytokines and growth factors [BR:hsa04052]; Interleukins; ','[Pfam: IL1]'),('3565','hsa:3565','[IL4, BCGF-1, BCGF1, BSF-1, BSF1, IL-4]','[(RefSeq) interleukin 4]','[interleukin 4]','[Homo sapiens (human)]','[Cytokine-cytokine receptor interaction, PI3K-Akt signaling pathway, JAK-STAT signaling pathway, Hematopoietic cell lineage, IL-17 signaling pathway, Th1 and Th2 cell differentiation, Th17 cell differentiation, T cell receptor signaling pathway, Fc epsilon RI signaling pathway, Intestinal immune network for IgA production, Leishmaniasis, Pathways in cancer, Asthma, Autoimmune thyroid disease, Inflammatory bowel disease, Allograft rejection]','[c(\"nt06119  JAK-STAT signaling (viruses)\", \"nt06160  Human T-cell leukemia virus 1 (HTLV-1)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06165  Epstein-Barr virus (EBV)\", \"nt06219  JAK-STAT signaling\", \"nt06263  Hepatocellular carcinoma\", \"nt06266  Non-small cell lung cancer\", \"nt06276  Chronic myeloid leukemia\")]','[N00053  Cytokine-Jak-STAT signaling pathway]','04630 JAK-STAT signaling pathway; 04151 PI3K-Akt signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04640 Hematopoietic cell lineage; 04660 T cell receptor signaling pathway; 04658 Th1 and Th2 cell differentiation; 04659 Th17 cell differentiation; 04657 IL-17 signaling pathway; 04664 Fc epsilon RI signaling pathway; 04672 Intestinal immune network for IgA production; 05200 Pathways in cancer; 05140 Leishmaniasis; 05310 Asthma; 05320 Autoimmune thyroid disease; 05321 Inflammatory bowel disease; 05330 Allograft rejection; 04052 Cytokines and growth factors [BR:hsa04052]; 00536 Glycosaminoglycan binding proteins [BR:hsa00536]; Interleukins; Cytokines; ','[Pfam: IL4 Mannosyl_trans3]'),('1047','hsa:1047','[CLGN]','[(RefSeq) calmegin]','[calmegin]','[Homo sapiens (human)]',NULL,'[]','[]','03110 Chaperones and folding catalysts [BR:hsa03110]; 04091 Lectins [BR:hsa04091]; Calnexin / calreticulin; Calnexin / calreticulin; ','[Pfam: Calreticulin CDC45 Folate_carrier SieB V_ATPase_I]'),('27134','hsa:27134','[TJP3, ZO-3, ZO3]','[(RefSeq) tight junction protein 3]','[tight junction protein 3]','[Homo sapiens (human)]','[Tight junction]','[]','[]','04530 Tight junction; ','[Pfam: PDZ PDZ_6 PDZ_2 SH3_2 Peptidase_M50 Guanylate_kin DUF6288]'),('1026','hsa:1026','[CDKN1A, CAP20, CDKN1, CIP1, MDA-6, P21, SDI1, WAF1, p21CIP1]','[(RefSeq) cyclin dependent kinase inhibitor 1A]','[cyclin-dependent kinase inhibitor 1A]','[Homo sapiens (human)]','[Endocrine resistance, Platinum drug resistance, ErbB signaling pathway, HIF-1 signaling pathway, FoxO signaling pathway, Cell cycle, p53 signaling pathway, PI3K-Akt signaling pathway, Cellular senescence, JAK-STAT signaling pathway, Oxytocin signaling pathway, Parathyroid hormone synthesis, secretion and action, Cushing syndrome, Hepatitis C, Hepatitis B, Human cytomegalovirus infection, Human papillomavirus infection, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Epstein-Barr virus infection, Pathways in cancer, Transcriptional misregulation in cancer, Viral carcinogenesis, Proteoglycans in cancer, MicroRNAs in cancer, Colorectal cancer, Renal cell carcinoma, Pancreatic cancer, Endometrial cancer, Glioma, Prostate cancer, Thyroid cancer, Basal cell carcinoma, Melanoma, Bladder cancer, Chronic myeloid leukemia, Small cell lung cancer, Non-small cell lung cancer, Breast cancer, Hepatocellular carcinoma, Gastric cancer]','[c(\"nt06130  Cell cycle (viruses)\", \"nt06140  Transcription (viruses)\", \"nt06160  Human T-cell leukemia virus 1 (HTLV-1)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06163  Hepatitis C virus (HCV)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06165  Epstein-Barr virus (EBV)\", \"nt06166  Human papillomavirus (HPV)\", \"nt06167  Human cytomegalovirus (HCMV)\", \"nt06170  Influenza A virus (IAV)\", \"nt06230  Cell cycle\", \"nt06240  Transcription\", \"nt06260  Colorectal cancer\", \"nt06261  Gastric cancer\", \n\"nt06262  Pancreatic cancer\", \"nt06263  Hepatocellular carcinoma\", \"nt06264  Renal cell carcinoma\", \"nt06265  Bladder cancer\", \"nt06266  Non-small cell lung cancer\", \"nt06267  Small cell lung cancer\", \"nt06268  Melanoma\", \"nt06269  Basal cell carcinoma\", \"nt06270  Breast cancer\", \"nt06271  Endometrial cancer\", \"nt06273  Glioma\", \"nt06274  Thyroid cancer\", \"nt06276  Chronic myeloid leukemia\")]','[N00066  MDM2-p21-Cell cycle G1/S, N00067  Deleted p14(ARF) to p21-cell cycle G1/S, N00068  Amplified MDM2 to p21-cell cycle G1/S, N00076  Mutation-inactivated p14(ARF) to p21-cell cycle G1/S, N00115  Mutation-inactivated TP53 to transcription, N00127  PRCC-TFE3 fusion to transcriptional activation, N00134  EWSR1-FLI1 fusion to transcriptional repression, N00223  EBV EBNA1 to p53-mediated transcription, N00263  EBV EBNA3C to p53-mediated transcription, N00347  p300-p21-Cell cycle G1/S, N00359  HPV E7 to p27-cell cycle G1/S, N00362  HPV E5 to p21-cell cycle G1/S, N00421  HCMV IE2-86 to p21-cell cycle G1/S, N00481  EBV BZLF1 to p53-mediated transcription, N00497  HTLV-1 Tax to p21-cell cycle G1/S, N00498  HTLV-1 Tax to p21-cell cycle G1/S, N00499  ATR-p21-Cell cycle G2/M, N00523  HCV Core to p21-cell cycle G1/S, N00535  HBV HBx to p53-mediated transcription, N00536  MDM2-p21-Cell cycle G1/S]','04012 ErbB signaling pathway; 04630 JAK-STAT signaling pathway; 04066 HIF-1 signaling pathway; 04068 FoxO signaling pathway; 04151 PI3K-Akt signaling pathway; 04110 Cell cycle; 04115 p53 signaling pathway; 04218 Cellular senescence; 04921 Oxytocin signaling pathway; 04928 Parathyroid hormone synthesis, secretion and action; 05200 Pathways in cancer; 05202 Transcriptional misregulation in cancer; 05206 MicroRNAs in cancer; 05205 Proteoglycans in cancer; 05203 Viral carcinogenesis; 05210 Colorectal cancer; 05212 Pancreatic cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05214 Glioma; 05216 Thyroid cancer; 05220 Chronic myeloid leukemia; 05217 Basal cell carcinoma; 05218 Melanoma; 05211 Renal cell carcinoma; 05219 Bladder cancer; 05215 Prostate cancer; 05213 Endometrial cancer; 05224 Breast cancer; 05222 Small cell lung cancer; 05223 Non-small cell lung cancer; 05166 Human T-cell leukemia virus 1 infection; 05161 Hepatitis B; 05160 Hepatitis C; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05165 Human papillomavirus infection; 04934 Cushing syndrome; 01524 Platinum drug resistance; 01522 Endocrine resistance; ','[Pfam: CDI]'),('1021','hsa:1021','[CDK6, MCPH12, PLSTIRE]','[(RefSeq) cyclin dependent kinase 6]','[cyclin-dependent kinase 6 [EC:2.7.11.22]]','[Homo sapiens (human)]','[Cell cycle, p53 signaling pathway, PI3K-Akt signaling pathway, Cellular senescence, Cushing syndrome, Hepatitis C, Measles, Human cytomegalovirus infection, Influenza A, Human papillomavirus infection, Kaposi sarcoma-associated herpesvirus infection, Epstein-Barr virus infection, Pathways in cancer, Viral carcinogenesis, MicroRNAs in cancer, Pancreatic cancer, Glioma, Melanoma, Chronic myeloid leukemia, Small cell lung cancer, Non-small cell lung cancer, Breast cancer, Hepatocellular carcinoma]','[c(\"nt06130  Cell cycle (viruses)\", \"nt06160  Human T-cell leukemia virus 1 (HTLV-1)\", \"nt06163  Hepatitis C virus (HCV)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06165  Epstein-Barr virus (EBV)\", \"nt06166  Human papillomavirus (HPV)\", \"nt06167  Human cytomegalovirus (HCMV)\", \"nt06170  Influenza A virus (IAV)\", \"nt06230  Cell cycle\", \"nt06262  Pancreatic cancer\", \"nt06263  Hepatocellular carcinoma\", \"nt06265  Bladder cancer\", \"nt06266  Non-small cell lung cancer\", \"nt06267  Small cell lung cancer\", \n\"nt06268  Melanoma\", \"nt06270  Breast cancer\", \"nt06273  Glioma\", \"nt06276  Chronic myeloid leukemia\")]','[N00066  MDM2-p21-Cell cycle G1/S, N00067  Deleted p14(ARF) to p21-cell cycle G1/S, N00068  Amplified MDM2 to p21-cell cycle G1/S, N00069  p16-Cell cycle G1/S, N00070  Mutation-inactivated p16(INK4a) to p16-cell cycle G1/S, N00071  Deleted p16(INK4a) to p16-cell cycle G1/S, N00076  Mutation-inactivated p14(ARF) to p21-cell cycle G1/S, N00088  Amplified MYC to p15-cell cycle G1/S, N00089  Amplified MYC to cell cycle G1/S, N00090  p15-Cell cycle G1/S, N00168  KSHV vCyclin to cell cycle G1/S, N00347  p300-p21-Cell cycle G1/S, N00483  EBV EBNA3C to cell cycle G1/S]','04151 PI3K-Akt signaling pathway; 04110 Cell cycle; 04115 p53 signaling pathway; 04218 Cellular senescence; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05203 Viral carcinogenesis; 05212 Pancreatic cancer; 05225 Hepatocellular carcinoma; 05214 Glioma; 05220 Chronic myeloid leukemia; 05218 Melanoma; 05224 Breast cancer; 05222 Small cell lung cancer; 05223 Non-small cell lung cancer; 05160 Hepatitis C; 05164 Influenza A; 05162 Measles; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05165 Human papillomavirus infection; 04934 Cushing syndrome; 01001 Protein kinases [BR:hsa01001]; 2.7.11.22  cyclin-dependent kinase; CDK family; ','[Pfam: Pkinase PK_Tyr_Ser-Thr APH ABC1 Kdo Haspin_kinase Kinase-like RIO1 FTA2]'),('51191','hsa:51191','[HERC5, CEB1, CEBP1]','[(RefSeq) HECT and RLD domain containing E3 ubiquitin protein ligase 5]','[E3 ISG15--protein ligase [EC:2.3.2.-]]','[Homo sapiens (human)]',NULL,'[]','[]','04121 Ubiquitin system [BR:hsa04121]; 2.3.2.-; UBL E3 ligases; ','[Pfam: HECT RCC1 RCC1_2 Peptidase_S51]'),('7157','hsa:7157','[TP53, BCC7, BMFS5, LFS1, P53, TRP53]','[(RefSeq) tumor protein p53]','[tumor protein p53]','[Homo sapiens (human)]','[Endocrine resistance, Platinum drug resistance, MAPK signaling pathway, Sphingolipid signaling pathway, Cell cycle, p53 signaling pathway, Mitophagy - animal, PI3K-Akt signaling pathway, Apoptosis, Longevity regulating pathway, Ferroptosis, Cellular senescence, Wnt signaling pathway, Neurotrophin signaling pathway, Thyroid hormone signaling pathway, Parkinson disease, Amyotrophic lateral sclerosis, Huntington disease, Shigellosis, Hepatitis C, Hepatitis B, Measles, Human cytomegalovirus infection, Human papillomavirus infection, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Herpes simplex virus 1 infection, Epstein-Barr virus infection, Pathways in cancer, Transcriptional misregulation in cancer, Viral carcinogenesis, Proteoglycans in cancer, MicroRNAs in cancer, Colorectal cancer, Pancreatic cancer, Endometrial cancer, Glioma, Prostate cancer, Thyroid cancer, Basal cell carcinoma, Melanoma, Bladder cancer, Chronic myeloid leukemia, Small cell lung cancer, Non-small cell lung cancer, Breast cancer, Hepatocellular carcinoma, Gastric cancer, Central carbon metabolism in cancer, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[c(\"nt06130  Cell cycle (viruses)\", \"nt06140  Transcription (viruses)\", \"nt06160  Human T-cell leukemia virus 1 (HTLV-1)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06163  Hepatitis C virus (HCV)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06165  Epstein-Barr virus (EBV)\", \"nt06166  Human papillomavirus (HPV)\", \"nt06167  Human cytomegalovirus (HCMV)\", \"nt06168  Herpes simplex virus 1 (HSV-1)\", \"nt06169  Measles virus (MV)\", \"nt06170  Influenza A virus (IAV)\", \"nt06230  Cell cycle\", \n\"nt06240  Transcription\", \"nt06260  Colorectal cancer\", \"nt06261  Gastric cancer\", \"nt06262  Pancreatic cancer\", \"nt06263  Hepatocellular carcinoma\", \"nt06265  Bladder cancer\", \"nt06266  Non-small cell lung cancer\", \"nt06267  Small cell lung cancer\", \"nt06268  Melanoma\", \"nt06269  Basal cell carcinoma\", \"nt06270  Breast cancer\", \"nt06271  Endometrial cancer\", \"nt06273  Glioma\", \"nt06274  Thyroid cancer\", \"nt06276  Chronic myeloid leukemia\", \"nt06440  Transcription\", \"nt06461  Huntington disease\", \n\"nt06463  Parkinson disease\")]','[N00066  MDM2-p21-Cell cycle G1/S, N00067  Deleted p14(ARF) to p21-cell cycle G1/S, N00068  Amplified MDM2 to p21-cell cycle G1/S, N00076  Mutation-inactivated p14(ARF) to p21-cell cycle G1/S, N00115  Mutation-inactivated TP53 to transcription, N00131  Amplified MYCN to transcriptional activation, N00167  KSHV vIRF1/3 to p21-cell cycle G1/S, N00169  KSHV LANA to p21-cell cycle G1/S, N00223  EBV EBNA1 to p53-mediated transcription, N00263  EBV EBNA3C to p53-mediated transcription, N00347  p300-p21-Cell cycle G1/S, N00358  HPV E6 to p21-cell cycle G1/S, N00420  HCMV IE2-86 to p21-cell cycle G1/S, N00481  EBV BZLF1 to p53-mediated transcription, N00497  HTLV-1 Tax to p21-cell cycle G1/S, N00499  ATR-p21-Cell cycle G2/M, N00520  HCV NS5A to p21-cell cycle G1/S, N00521  HCV Core to p21-cell cycle G1/S, N00522  HCV NS3 to p21-cell cycle G1/S, N00535  HBV HBx to p53-mediated transcription, N00536  MDM2-p21-Cell cycle G1/S, N00592  HSV ICP0 to p53-mediated transcription, N00697  HV P to p53-mediated transcription, N00982  Mutation-caused aberrant Htt to p53-mediated transcription, N01058  Mutation-inactivated DJ1 to to p53-mediated transcription]','04010 MAPK signaling pathway; 04310 Wnt signaling pathway; 04071 Sphingolipid signaling pathway; 04151 PI3K-Akt signaling pathway; 04137 Mitophagy - animal; 04110 Cell cycle; 04210 Apoptosis; 04216 Ferroptosis; 04115 p53 signaling pathway; 04218 Cellular senescence; 04919 Thyroid hormone signaling pathway; 04722 Neurotrophin signaling pathway; 04211 Longevity regulating pathway; 05200 Pathways in cancer; 05202 Transcriptional misregulation in cancer; 05206 MicroRNAs in cancer; 05205 Proteoglycans in cancer; 05203 Viral carcinogenesis; 05230 Central carbon metabolism in cancer; 05210 Colorectal cancer; 05212 Pancreatic cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05214 Glioma; 05216 Thyroid cancer; 05220 Chronic myeloid leukemia; 05217 Basal cell carcinoma; 05218 Melanoma; 05219 Bladder cancer; 05215 Prostate cancer; 05213 Endometrial cancer; 05224 Breast cancer; 05222 Small cell lung cancer; 05223 Non-small cell lung cancer; 05166 Human T-cell leukemia virus 1 infection; 05161 Hepatitis B; 05160 Hepatitis C; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05165 Human papillomavirus infection; 05131 Shigellosis; 05012 Parkinson disease; 05014 Amyotrophic lateral sclerosis; 05016 Huntington disease; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 01524 Platinum drug resistance; 01522 Endocrine resistance; 03000 Transcription factors [BR:hsa03000]; 03036 Chromosome and associated proteins [BR:hsa03036]; 03400 DNA repair and recombination proteins [BR:hsa03400]; p53; Regulators of Aurora kinases; Other check point factors; ','[Pfam: P53 TAD2 P53_tetramer P53_TAD]'),('23562','hsa:23562','[CLDN14, DFNB29]','[(RefSeq) claudin 14]','[claudin]','[Homo sapiens (human)]','[Cell adhesion molecules, Tight junction, Leukocyte transendothelial migration, Pathogenic Escherichia coli infection, Hepatitis C]','[nt06135  Cytoskeletal regulation (viruses and bacteria)]','[N01287  Tight junction-Actin signaling pathway]','04514 Cell adhesion molecules; 04530 Tight junction; 04670 Leukocyte transendothelial migration; 05160 Hepatitis C; 05130 Pathogenic Escherichia coli infection; 03037 Cilium and associated proteins [BR:hsa03037]; 04147 Exosome [BR:hsa04147]; Stereociliary proteins; Exosomal proteins of ovarian cancer cells; Exosomal proteins of colorectal cancer cells; ','[Pfam: PMP22_Claudin Claudin_2 Amastin Laminin_EGF]'),('2169','hsa:2169','[FABP2, FABPI, I-FABP]','[(RefSeq) fatty acid binding protein 2]','[fatty acid-binding protein 2, intestinal]','[Homo sapiens (human)]','[PPAR signaling pathway, Fat digestion and absorption]','[]','[]','03320 PPAR signaling pathway; 04975 Fat digestion and absorption; ','[Pfam: Lipocalin Lipocalin_7 Lipocalin_4]'),('3708','eco:b3708','[tnaA]','[(RefSeq) tryptophanase]','[tryptophanase [EC:4.1.99.1]]','[Escherichia coli K-12 MG1655]','[Tryptophan metabolism, Metabolic pathways]','[]','[]','','[Pfam: Beta_elim_lyase Aminotran_1_2 SepSecS]'),('5599','hsa:5599','[MAPK8, JNK, JNK-46, JNK1, JNK1A2, JNK21B1/2, PRKM8, SAPK1, SAPK1c]','[(RefSeq) mitogen-activated protein kinase 8]','[mitogen-activated protein kinase 8/9/10 (c-Jun N-terminal kinase) [EC:2.7.11.24]]','[Homo sapiens (human)]','[Endocrine resistance, MAPK signaling pathway, ErbB signaling pathway, Ras signaling pathway, cAMP signaling pathway, FoxO signaling pathway, Sphingolipid signaling pathway, Mitophagy - animal, Autophagy - animal, Protein processing in endoplasmic reticulum, Apoptosis, Apoptosis - multiple species, Necroptosis, Wnt signaling pathway, Osteoclast differentiation, Focal adhesion, Tight junction, Toll-like receptor signaling pathway, NOD-like receptor signaling pathway, RIG-I-like receptor signaling pathway, C-type lectin receptor signaling pathway, IL-17 signaling pathway, Th1 and Th2 cell differentiation, Th17 cell differentiation, T cell receptor signaling pathway, Fc epsilon RI signaling pathway, TNF signaling pathway, Neurotrophin signaling pathway, Retrograde endocannabinoid signaling, Dopaminergic synapse, Inflammatory mediator regulation of TRP channels, Insulin signaling pathway, GnRH signaling pathway, Progesterone-mediated oocyte maturation, Prolactin signaling pathway, Adipocytokine signaling pathway, Relaxin signaling pathway, Type II diabetes mellitus, Insulin resistance, Non-alcoholic fatty liver disease, AGE-RAGE signaling pathway in diabetic complications, Growth hormone synthesis, secretion and action, Alcoholic liver disease, Alzheimer disease, Parkinson disease, Huntington disease, Spinocerebellar ataxia, Prion disease, Pathways of neurodegeneration - multiple diseases, Epithelial cell signaling in Helicobacter pylori infection, Pathogenic Escherichia coli infection, Shigellosis, Salmonella infection, Pertussis, Yersinia infection, Chagas disease, Toxoplasmosis, Tuberculosis, Hepatitis B, Measles, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Epstein-Barr virus infection, Human immunodeficiency virus 1 infection, Coronavirus disease - COVID-19, Pathways in cancer, Chemical carcinogenesis - reactive oxygen species, Colorectal cancer, Pancreatic cancer, Choline metabolism in cancer, Diabetic cardiomyopathy, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[c(\"nt06110  MAPK signaling (viruses and bacteria)\", \"nt06121  TLR signaling (viruses and bacteria)\", \"nt06123  TNF signaling (viruses and bacteria)\", \"nt06124  Chemokine signaling (viruses)\", \"nt06160  Human T-cell leukemia virus 1 (HTLV-1)\", \"nt06161  Human immunodeficiency virus type 1 (HIV-1)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06211  Other MAPK signaling\", \"nt06223  TNF signaling\", \"nt06224  CXCR signaling\", \"nt06263  Hepatocellular carcinoma\", \n\"nt06412  Unfolded protein response (UPR) signaling\", \"nt06414  Apoptosis\", \"nt06416  TNF signaling\", \"nt06460  Alzheimer disease\", \"nt06461  Huntington disease\", \"nt06463  Parkinson disease\")]','[N00178  KSHV vGPCR to GNB/G-PI3K-JNK signaling pathway, N00188  IL1-IL1R-JNK signaling pathway, N00189  KSHV K15 to JNK signaling pathway, N00438  TLR2/4-MAPK signaling pathway, N00446  TNF-JNK signaling pathway, N00447  HIV Vpr/Tat to TNF-JNK signaling pathway, N00503  HTLV-1 Tax to TNF-JNK signaling pathway, N00542  EGF-EGFR-RAS-JNK signaling pathway, N00543  HBV HBx to JNK signaling pathway, N00551  HBV HBs to TLR2/4-MAPK signaling pathway, N00930  Escherichia NleD to TNF-JNK signaling pathway, N00992  Mutation-caused aberrant Htt to TNF-JNK signaling pathway, N01013  IRE1a-JNK signaling pathway, N01014  Mutation-caused aberrant Abeta to IRE1a-JNK signaling pathway, N01056  FAS-JNK signaling pathway, N01057  Mutation-inactivated DJ1 to FAS-JNK signaling patwhay, N01407  Metals to JNK signaling pathway, N01417  Paraquat to FAS-JNK signaling pathway]','04141 Protein processing in endoplasmic reticulum; 04010 MAPK signaling pathway; 04012 ErbB signaling pathway; 04014 Ras signaling pathway; 04310 Wnt signaling pathway; 04668 TNF signaling pathway; 04068 FoxO signaling pathway; 04071 Sphingolipid signaling pathway; 04024 cAMP signaling pathway; 04140 Autophagy - animal; 04137 Mitophagy - animal; 04210 Apoptosis; 04215 Apoptosis - multiple species; 04217 Necroptosis; 04510 Focal adhesion; 04530 Tight junction; 04620 Toll-like receptor signaling pathway; 04621 NOD-like receptor signaling pathway; 04622 RIG-I-like receptor signaling pathway; 04625 C-type lectin receptor signaling pathway; 04660 T cell receptor signaling pathway; 04658 Th1 and Th2 cell differentiation; 04659 Th17 cell differentiation; 04657 IL-17 signaling pathway; 04664 Fc epsilon RI signaling pathway; 04910 Insulin signaling pathway; 04920 Adipocytokine signaling pathway; 04912 GnRH signaling pathway; 04914 Progesterone-mediated oocyte maturation; 04917 Prolactin signaling pathway; 04926 Relaxin signaling pathway; 04935 Growth hormone synthesis, secretion and action; 04728 Dopaminergic synapse; 04723 Retrograde endocannabinoid signaling; 04722 Neurotrophin signaling pathway; 04750 Inflammatory mediator regulation of TRP channels; 04380 Osteoclast differentiation; 05200 Pathways in cancer; 05208 Chemical carcinogenesis - reactive oxygen species; 05231 Choline metabolism in cancer; 05210 Colorectal cancer; 05212 Pancreatic cancer; 05166 Human T-cell leukemia virus 1 infection; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05171 Coronavirus disease - COVID-19; 05162 Measles; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05120 Epithelial cell signaling in Helicobacter pylori infection; 05130 Pathogenic Escherichia coli infection; 05132 Salmonella infection; 05131 Shigellosis; 05135 Yersinia infection; 05133 Pertussis; 05152 Tuberculosis; 05145 Toxoplasmosis; 05142 Chagas disease; 05010 Alzheimer disease; 05012 Parkinson disease; 05016 Huntington disease; 05017 Spinocerebellar ataxia; 05020 Prion disease; 05022 Pathways of neurodegeneration - multiple diseases; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 05415 Diabetic cardiomyopathy; 04930 Type II diabetes mellitus; 04936 Alcoholic liver disease; 04932 Non-alcoholic fatty liver disease; 04931 Insulin resistance; 04933 AGE-RAGE signaling pathway in diabetic complications; 01522 Endocrine resistance; 01001 Protein kinases [BR:hsa01001]; 2.7.11.24  mitogen-activated protein kinase; MAPK family [OT]; ','[Pfam: Pkinase PK_Tyr_Ser-Thr Kdo ABC1 Kinase-like RIO1 APH FTA2 DUF530]'),('150','hsa:150','[ADRA2A, ADRA2, ADRA2R, ADRAR, ALPHA2AAR, ZNF32]','[(RefSeq) adrenoceptor alpha 2A]','[adrenergic receptor alpha-2A]','[Homo sapiens (human)]','[cGMP-PKG signaling pathway, Neuroactive ligand-receptor interaction]','[]','[]','04022 cGMP-PKG signaling pathway; 04080 Neuroactive ligand-receptor interaction; 04030 G protein-coupled receptors [BR:hsa04030]; Adrenaline; ','[Pfam: 7tm_1 7tm_4 7TM_GPCR_Srsx T2SSM]'),('154','hsa:154','[ADRB2, ADRB2R, ADRBR, B2AR, BAR, BETA2AR]','[(RefSeq) adrenoceptor beta 2]','[adrenergic receptor beta-2]','[Homo sapiens (human)]','[Calcium signaling pathway, cGMP-PKG signaling pathway, cAMP signaling pathway, Neuroactive ligand-receptor interaction, Adrenergic signaling in cardiomyocytes, Regulation of lipolysis in adipocytes, Renin secretion, Salivary secretion, Chemical carcinogenesis - receptor activation]','[c(\"nt06214  PI3K signaling\", \"nt06234  cAMP signaling\")]','[N01345  EP/NE-ADRB-cAMP signaling pathway, N01346  Nicotine/NNK to cAMP signaling pathway, N01347  EP/NE-ADRB-PI3K signaling pathway, N01348  Nicotine/NNK to PI3K signaling pathway]','04020 Calcium signaling pathway; 04024 cAMP signaling pathway; 04022 cGMP-PKG signaling pathway; 04080 Neuroactive ligand-receptor interaction; 04923 Regulation of lipolysis in adipocytes; 04924 Renin secretion; 04261 Adrenergic signaling in cardiomyocytes; 04970 Salivary secretion; 05207 Chemical carcinogenesis - receptor activation; 04030 G protein-coupled receptors [BR:hsa04030]; Adrenaline; ','[Pfam: 7tm_1 7TM_GPCR_Srsx 7TM_GPCR_Srx 7TM_GPCR_Srv 7tm_4]'),('151','hsa:151','[ADRA2B, ADRA2L1, ADRA2RL1, ADRARL1, ALPHA2BAR, FAME2, alpha-2BAR]','[(RefSeq) adrenoceptor alpha 2B]','[adrenergic receptor alpha-2B]','[Homo sapiens (human)]','[cGMP-PKG signaling pathway, Neuroactive ligand-receptor interaction]','[]','[]','04022 cGMP-PKG signaling pathway; 04080 Neuroactive ligand-receptor interaction; 04030 G protein-coupled receptors [BR:hsa04030]; Adrenaline; ','[Pfam: 7tm_1 7TM_GPCR_Srsx 7tm_4 7TM_GPCR_Srx Na_Ca_ex DUF4632]'),('5879','hsa:5879','[RAC1, MIG5, MRD48, Rac-1, TC-25, p21-Rac1]','[(RefSeq) Rac family small GTPase 1]','[Ras-related C3 botulinum toxin substrate 1]','[Homo sapiens (human)]','[MAPK signaling pathway, Ras signaling pathway, Rap1 signaling pathway, cAMP signaling pathway, Chemokine signaling pathway, Sphingolipid signaling pathway, Phagosome, PI3K-Akt signaling pathway, Wnt signaling pathway, Axon guidance, VEGF signaling pathway, Osteoclast differentiation, Focal adhesion, Adherens junction, Tight junction, Neutrophil extracellular trap formation, Toll-like receptor signaling pathway, Natural killer cell mediated cytotoxicity, B cell receptor signaling pathway, Fc epsilon RI signaling pathway, Fc gamma R-mediated phagocytosis, Leukocyte transendothelial migration, Neurotrophin signaling pathway, Regulation of actin cytoskeleton, Non-alcoholic fatty liver disease, AGE-RAGE signaling pathway in diabetic complications, Pancreatic secretion, Amyotrophic lateral sclerosis, Prion disease, Pathways of neurodegeneration - multiple diseases, Bacterial invasion of epithelial cells, Epithelial cell signaling in Helicobacter pylori infection, Pathogenic Escherichia coli infection, Shigellosis, Salmonella infection, Yersinia infection, Human cytomegalovirus infection, Kaposi sarcoma-associated herpesvirus infection, Epstein-Barr virus infection, Human immunodeficiency virus 1 infection, Pathways in cancer, Viral carcinogenesis, Proteoglycans in cancer, Chemical carcinogenesis - reactive oxygen species, Colorectal cancer, Renal cell carcinoma, Pancreatic cancer, Choline metabolism in cancer, Diabetic cardiomyopathy, Viral myocarditis, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[c(\"nt06110  MAPK signaling (viruses and bacteria)\", \"nt06124  Chemokine signaling (viruses)\", \"nt06135  Cytoskeletal regulation (viruses and bacteria)\", \"nt06139  NLR signaling (viruses and bacteria)\", \"nt06150  Cytokine-cytokine receptor interaction (viruses)\", \"nt06161  Human immunodeficiency virus type 1 (HIV-1)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06167  Human cytomegalovirus (HCMV)\", \"nt06224  CXCR signaling\", \"nt06415  PI3K signaling\", \"nt06465  Prion disease\")]','[N00153  CCR/CXCR-GNB/G-PI3K-RAC signaling pathway, N00178  KSHV vGPCR to GNB/G-PI3K-JNK signaling pathway, N00394  HCMV gH to ITGA/B-RhoA signaling pathway, N00433  CXCR4-GNB/G-RAC signaling pathway, N00434  HIV gp120 to CXCR4-GNB/G-RAC signaling pathway, N00951  ITGA/B-RHOG-RAC signaling pathway, N00952  Shigella IpgB1 to ITGA/B-RHOG-RAC signaling pathway, N01068  ITGA/B-FAK-RAC signaling pathway, N01069  Shigella IpgB1 to ITGA/B-FAK-RAC signaling pathway, N01077  Shigella IcsB to ITGA/B-FAK-RAC signaling pathway, N01086  Escherichia EspT to RAC signaling pathway, N01087  Escherichia EspW to RAC signaling pathway, N01090  IGG-FCGR-RAC signaling pathway, N01102  Yersinia YopE to ITGA/B-RHOG-RAC signaling pathway, N01103  Yersinia YpkA to IGG-FCGR-RAC signaling pathway, N01119  RAC/CDC42-PAK-ERK signaling pathway, N01120  Salmonella SptP to RAC/CDC42-PAK-ERK signaling pathway, N01127  Salmonella SopE to Inflammasome signaling pathway, N01128  Salmonella SopE to RAC signaling pathway, N01204  PRNP-PI3K-NOX2 signaling pathway, N01205  Scrapie conformation PrPSc to PRNP-PI3K-NOX2 signaling pathway]','04010 MAPK signaling pathway; 04014 Ras signaling pathway; 04015 Rap1 signaling pathway; 04310 Wnt signaling pathway; 04370 VEGF signaling pathway; 04071 Sphingolipid signaling pathway; 04024 cAMP signaling pathway; 04151 PI3K-Akt signaling pathway; 04145 Phagosome; 04510 Focal adhesion; 04520 Adherens junction; 04530 Tight junction; 04810 Regulation of actin cytoskeleton; 04613 Neutrophil extracellular trap formation; 04620 Toll-like receptor signaling pathway; 04650 Natural killer cell mediated cytotoxicity; 04662 B cell receptor signaling pathway; 04664 Fc epsilon RI signaling pathway; 04666 Fc gamma R-mediated phagocytosis; 04670 Leukocyte transendothelial migration; 04062 Chemokine signaling pathway; 04972 Pancreatic secretion; 04722 Neurotrophin signaling pathway; 04360 Axon guidance; 04380 Osteoclast differentiation; 05200 Pathways in cancer; 05205 Proteoglycans in cancer; 05208 Chemical carcinogenesis - reactive oxygen species; 05203 Viral carcinogenesis; 05231 Choline metabolism in cancer; 05210 Colorectal cancer; 05212 Pancreatic cancer; 05211 Renal cell carcinoma; 05170 Human immunodeficiency virus 1 infection; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05120 Epithelial cell signaling in Helicobacter pylori infection; 05130 Pathogenic Escherichia coli infection; 05132 Salmonella infection; 05131 Shigellosis; 05135 Yersinia infection; 05100 Bacterial invasion of epithelial cells; 05014 Amyotrophic lateral sclerosis; 05020 Prion disease; 05022 Pathways of neurodegeneration - multiple diseases; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 05415 Diabetic cardiomyopathy; 05416 Viral myocarditis; 04932 Non-alcoholic fatty liver disease; 04933 AGE-RAGE signaling pathway in diabetic complications; 04131 Membrane trafficking [BR:hsa04131]; 04147 Exosome [BR:hsa04147]; 04031 GTP-binding proteins [BR:hsa04031]; Rho GTPases; Arf6-dependent endocytosis; Ras GTPases; Nox associated proteins; Exosomal proteins of haemopoietic cells  (B-cell, T-cell, DC-cell, reticulocyte, and mast cell); Exosomal proteins of other body fluids (saliva and urine); Exosomal proteins of colorectal cancer cells; Exosomal proteins of bladder cancer cells; Rac/Cdc42 [OT]; ','[Pfam: Ras Roc Arf CagD]'),('5058','hsa:5058','[PAK1, IDDMSSD, PAKalpha, alpha-PAK, p65-PAK]','[(RefSeq) p21 (RAC1) activated kinase 1]','[p21-activated kinase 1 [EC:2.7.11.1]]','[Homo sapiens (human)]','[MAPK signaling pathway, ErbB signaling pathway, Ras signaling pathway, cAMP signaling pathway, Chemokine signaling pathway, Axon guidance, Hippo signaling pathway - multiple species, Focal adhesion, C-type lectin receptor signaling pathway, Natural killer cell mediated cytotoxicity, T cell receptor signaling pathway, Fc gamma R-mediated phagocytosis, Regulation of actin cytoskeleton, Epithelial cell signaling in Helicobacter pylori infection, Pathogenic Escherichia coli infection, Salmonella infection, Human immunodeficiency virus 1 infection, Proteoglycans in cancer, Renal cell carcinoma]','[c(\"nt06110  MAPK signaling (viruses and bacteria)\", \"nt06124  Chemokine signaling (viruses)\", \"nt06150  Cytokine-cytokine receptor interaction (viruses)\", \"nt06161  Human immunodeficiency virus type 1 (HIV-1)\")]','[N00433  CXCR4-GNB/G-RAC signaling pathway, N00434  HIV gp120 to CXCR4-GNB/G-RAC signaling pathway, N01119  RAC/CDC42-PAK-ERK signaling pathway]','04010 MAPK signaling pathway; 04012 ErbB signaling pathway; 04014 Ras signaling pathway; 04392 Hippo signaling pathway - multiple species; 04024 cAMP signaling pathway; 04510 Focal adhesion; 04810 Regulation of actin cytoskeleton; 04625 C-type lectin receptor signaling pathway; 04650 Natural killer cell mediated cytotoxicity; 04660 T cell receptor signaling pathway; 04666 Fc gamma R-mediated phagocytosis; 04062 Chemokine signaling pathway; 04360 Axon guidance; 05205 Proteoglycans in cancer; 05211 Renal cell carcinoma; 05170 Human immunodeficiency virus 1 infection; 05120 Epithelial cell signaling in Helicobacter pylori infection; 05130 Pathogenic Escherichia coli infection; 05132 Salmonella infection; 01001 Protein kinases [BR:hsa01001]; 04131 Membrane trafficking [BR:hsa04131]; 03036 Chromosome and associated proteins [BR:hsa03036]; 2.7.11.1  non-specific serine/threonine protein kinase; STE20 family; Kinases and associated proteins; Kinases; Regulators of Aurora kinases; ','[Pfam: Pkinase PK_Tyr_Ser-Thr PBD ABC1 Kinase-like Pkinase_fungal DSHCT APH Kdo]'),('1366','hsa:1366','[CLDN7, CEPTRL2, CLDN-7, CPETRL2, Hs.84359, claudin-1]','[(RefSeq) claudin 7]','[claudin]','[Homo sapiens (human)]','[Cell adhesion molecules, Tight junction, Leukocyte transendothelial migration, Pathogenic Escherichia coli infection, Hepatitis C]','[nt06135  Cytoskeletal regulation (viruses and bacteria)]','[N01287  Tight junction-Actin signaling pathway]','04514 Cell adhesion molecules; 04530 Tight junction; 04670 Leukocyte transendothelial migration; 05160 Hepatitis C; 05130 Pathogenic Escherichia coli infection; 03037 Cilium and associated proteins [BR:hsa03037]; 04147 Exosome [BR:hsa04147]; Stereociliary proteins; Exosomal proteins of ovarian cancer cells; Exosomal proteins of colorectal cancer cells; ','[Pfam: PMP22_Claudin Claudin_2 SUR7 Clc-like Amastin OppC_N]'),('7098','hsa:7098','[TLR3, CD283, IIAE2, IMD83]','[(RefSeq) toll like receptor 3]','[toll-like receptor 3]','[Homo sapiens (human)]','[Necroptosis, Toll-like receptor signaling pathway, Hepatitis C, Hepatitis B, Influenza A, Human papillomavirus infection, Kaposi sarcoma-associated herpesvirus infection, Herpes simplex virus 1 infection, Coronavirus disease - COVID-19]','[c(\"nt06121  TLR signaling (viruses and bacteria)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06163  Hepatitis C virus (HCV)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06166  Human papillomavirus (HPV)\", \"nt06168  Herpes simplex virus 1 (HSV-1)\", \"nt06221  TLR signaling\", \"nt06263  Hepatocellular carcinoma\")]','[N00148  TLR3-IRF7 signaling pathway, N00149  TLR3-IRF3 signaling pathway, N00563  TLR3-NFKB signaling pathway, N00564  HSV US3 to TLR3-NFKB signaling pathway]','04217 Necroptosis; 04620 Toll-like receptor signaling pathway; 05161 Hepatitis B; 05160 Hepatitis C; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05168 Herpes simplex virus 1 infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05165 Human papillomavirus infection; 04054 Pattern recognition receptors [BR:hsa04054]; 04090 CD molecules [BR:hsa04090]; Toll-like receptors; Proteins; ','[Pfam: LRR_8 LRR_4 LRR_5 Tlr3_TMD LRR_9 TIR TIR_2 LRRNT LRRCT]'),('54106','hsa:54106','[TLR9, CD289]','[(RefSeq) toll like receptor 9]','[toll-like receptor 9]','[Homo sapiens (human)]','[Toll-like receptor signaling pathway, Salmonella infection, Chagas disease, African trypanosomiasis, Malaria, Tuberculosis, Measles, Herpes simplex virus 1 infection, PD-L1 expression and PD-1 checkpoint pathway in cancer]','[c(\"nt06121  TLR signaling (viruses and bacteria)\", \"nt06169  Measles virus (MV)\")]','[N00690  TLR7/9-IRF7 signaling pathway]','04620 Toll-like receptor signaling pathway; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05132 Salmonella infection; 05152 Tuberculosis; 05144 Malaria; 05142 Chagas disease; 05143 African trypanosomiasis; 04054 Pattern recognition receptors [BR:hsa04054]; 04090 CD molecules [BR:hsa04090]; Toll-like receptors; Proteins; ','[Pfam: LRR_8 LRR_4 LRR_12 TIR LRR_9 LRR_5 TIR_2 FNIP]'),('54472','hsa:54472','[TOLLIP, IL-1RAcPIP]','[(RefSeq) toll interacting protein]','[toll-interacting protein]','[Homo sapiens (human)]','[Toll-like receptor signaling pathway]','[]','[]','04620 Toll-like receptor signaling pathway; 04147 Exosome [BR:hsa04147]; Exosomal proteins of colorectal cancer cells; Exosomal proteins of bladder cancer cells; ','[Pfam: C2 CUE UBA HTH_8]'),('59307','hsa:59307','[SIGIRR, IL-1R8, TIR8]','[(RefSeq) single Ig and TIR domain containing]','[single Ig IL-1R-related molecule]','[Homo sapiens (human)]',NULL,'[]','[]','04050 Cytokine receptors [BR:hsa04050]; IL1 receptor family; ','[Pfam: TIR TIR_2 Ig_2 Ig_3 I-set]'),('9655','hsa:9655','[SOCS5, CIS6, CISH6, Cish5, SOCS-5]','[(RefSeq) suppressor of cytokine signaling 5]','[suppressor of cytokine signaling 5]','[Homo sapiens (human)]','[JAK-STAT signaling pathway, Prolactin signaling pathway]','[]','[]','04630 JAK-STAT signaling pathway; 04917 Prolactin signaling pathway; 04121 Ubiquitin system [BR:hsa04121]; Target recognizing subunit; ','[Pfam: SOCS SH2 SOCS_box BRCT_assoc]'),('5290','hsa:5290','[PIK3CA, CCM4, CLAPO, CLOVE, CWS5, MCAP, MCM, MCMTC, PI3K, PI3K-alpha, p110-alpha]','[(RefSeq) phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit alpha]','[phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit alpha/beta/delta [EC:2.7.1.153]]','[Homo sapiens (human)]','[Inositol phosphate metabolism, Metabolic pathways, EGFR tyrosine kinase inhibitor resistance, Endocrine resistance, Platinum drug resistance, ErbB signaling pathway, Ras signaling pathway, Rap1 signaling pathway, cAMP signaling pathway, Chemokine signaling pathway, HIF-1 signaling pathway, FoxO signaling pathway, Phosphatidylinositol signaling system, Sphingolipid signaling pathway, Phospholipase D signaling pathway, Autophagy - animal, mTOR signaling pathway, PI3K-Akt signaling pathway, AMPK signaling pathway, Apoptosis, Longevity regulating pathway, Longevity regulating pathway - multiple species, Cellular senescence, Axon guidance, VEGF signaling pathway, Osteoclast differentiation, Focal adhesion, Signaling pathways regulating pluripotency of stem cells, Platelet activation, Neutrophil extracellular trap formation, Toll-like receptor signaling pathway, C-type lectin receptor signaling pathway, JAK-STAT signaling pathway, Natural killer cell mediated cytotoxicity, T cell receptor signaling pathway, B cell receptor signaling pathway, Fc epsilon RI signaling pathway, Fc gamma R-mediated phagocytosis, TNF signaling pathway, Leukocyte transendothelial migration, Neurotrophin signaling pathway, Cholinergic synapse, Inflammatory mediator regulation of TRP channels, Regulation of actin cytoskeleton, Insulin signaling pathway, Progesterone-mediated oocyte maturation, Estrogen signaling pathway, Prolactin signaling pathway, Thyroid hormone signaling pathway, Regulation of lipolysis in adipocytes, Relaxin signaling pathway, GnRH secretion, Type II diabetes mellitus, Insulin resistance, Non-alcoholic fatty liver disease, AGE-RAGE signaling pathway in diabetic complications, Growth hormone synthesis, secretion and action, Aldosterone-regulated sodium reabsorption, Carbohydrate digestion and absorption, Alzheimer disease, Spinocerebellar ataxia, Prion disease, Bacterial invasion of epithelial cells, Shigellosis, Salmonella infection, Yersinia infection, Chagas disease, Amoebiasis, Hepatitis C, Hepatitis B, Measles, Human cytomegalovirus infection, Influenza A, Human papillomavirus infection, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Herpes simplex virus 1 infection, Epstein-Barr virus infection, Human immunodeficiency virus 1 infection, Coronavirus disease - COVID-19, Pathways in cancer, Viral carcinogenesis, Proteoglycans in cancer, MicroRNAs in cancer, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species, Colorectal cancer, Renal cell carcinoma, Pancreatic cancer, Endometrial cancer, Glioma, Prostate cancer, Melanoma, Chronic myeloid leukemia, Acute myeloid leukemia, Small cell lung cancer, Non-small cell lung cancer, Breast cancer, Hepatocellular carcinoma, Gastric cancer, Central carbon metabolism in cancer, Choline metabolism in cancer, PD-L1 expression and PD-1 checkpoint pathway in cancer, Diabetic cardiomyopathy, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[c(\"nt06114  PI3K signaling (viruses)\", \"nt06124  Chemokine signaling (viruses)\", \"nt06135  Cytoskeletal regulation (viruses and bacteria)\", \"nt06150  Cytokine-cytokine receptor interaction (viruses)\", \"nt06160  Human T-cell leukemia virus 1 (HTLV-1)\", \"nt06161  Human immunodeficiency virus type 1 (HIV-1)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06163  Hepatitis C virus (HCV)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06165  Epstein-Barr virus (EBV)\", \"nt06166  Human papillomavirus (HPV)\", \n\"nt06167  Human cytomegalovirus (HCMV)\", \"nt06168  Herpes simplex virus 1 (HSV-1)\", \"nt06169  Measles virus (MV)\", \"nt06170  Influenza A virus (IAV)\", \"nt06214  PI3K signaling\", \"nt06260  Colorectal cancer\", \"nt06261  Gastric cancer\", \"nt06262  Pancreatic cancer\", \"nt06263  Hepatocellular carcinoma\", \"nt06264  Renal cell carcinoma\", \"nt06266  Non-small cell lung cancer\", \"nt06268  Melanoma\", \"nt06270  Breast cancer\", \"nt06273  Glioma\", \"nt06275  Acute myeloid leukemia\", \"nt06276  Chronic myeloid leukemia\", \n\"nt06415  PI3K signaling\", \"nt06462  Spinocerebellar ataxia\", \"nt06464  Amyotrophic lateral sclerosis\", \"nt06465  Prion disease\")]','[N00030  EGF-EGFR-RAS-PI3K signaling pathway, N00031  Duplication or mutation-activated FLT3 to RAS-PI3K signaling pathway, N00032  Mutation-activated KRAS/NRAS to PI3K signaling pathway, N00033  EGF-EGFR-PI3K signaling pathway, N00034  ERBB2-overexpression to PI3K signaling pathway, N00035  Amplified EGFR to PI3K signaling pathway, N00036  Mutation-activated EGFR to PI3K signaling pathway, N00037  FGF-FGFR-PI3K signaling pathway, N00038  Amplified FGFR to PI3K signaling pathway, N00039  PDGF-PDGFR-PI3K signaling pathway, N00040  Amplified PDGFR to PI3K signaling pathway, N00042  EGFR-overexpression to PI3K signaling pathway, N00043  HGF-MET-PI3K signaling pathway, N00044  Mutation-activated MET to PI3K signaling pathway, N00045  KITLG-KIT-PI3K signaling pathway, N00046  Mutation-activated KIT to PI3K signaling pathway, N00047  EML4-ALK fusion kinase to PI3K signaling pathway, N00048  BCR-ABL fusion kinase to PI3K signaling pathway, N00049  Mutation-activated PI3K to PI3K signaling pathway, N00050  Amplified PI3K to PI3K signaling pathway, N00159  KSHV K1 to PI3K signaling pathway, N00179  KSHV K1 to PI3K-NFKB signaling pathway, N00182  IGF-IGFR-PI3K-NFKB signaling pathway, N00218  FLT3LG-FLT3-RAS-PI3K signaling pathway, N00231  TGFA-EGFR-PI3K signaling pathway, N00232  TGFA-overexpression to PI3K signaling pathway, N00234  IGF2-IGF1R-PI3K signaling pathway, N00236  IGF2-overexpression to PI3K signaling pathway, N00238  IGF1R-overexpression to PI3K signaling pathway, N00247  HGF-overexpression to PI3K signaling pathway, N00249  MET-overexpression to PI3K signaling pathway, N00253  Amplified ERBB2 to PI3K signaling pathway, N00260  Amplified MET to PI3K signaling pathway, N00266  EBV LMP2A to PI3K signaling pathway, N00267  HBV HBx to PI3K signaling pathway, N00272  HCV NS5A to PI3K signaling pathway, N00281  EGF-overexpression to PI3K signaling pathway, N00282  EREG-EGFR-PI3K signaling pathway, N00283  EREG-overexpression to PI3K signaling pathway, N00284  AREG-EGFR-PI3K signaling pathway, N00285  AREG-overexpression to PI3K signaling pathway, N00366  HPV E5 to EGFR-PI3K signaling pathway, N00368  HPV E5 to PDGFR-PI3K signaling pathway, N00385  HCMV gB to PDGFR-PI3K signaling pathway, N00387  HCMV IE1-72/IE2-86 to PI3K signaling pathway, N00390  EGF-EGFR-PI3K-NFKB signaling pathway, N00391  HCMV gB to EGFR-PI3K-NFKB signaling pathway, N00399  CCR2-GNB/G-PI3K-NFKB signaling pathway, N00400  HCMV US28 to GNB/G-PI3K-NFKB signaling pathway, N00430  CXCR4-GNAI-PI3K-BAD signaling pathway, N00431  HIV gp120 to CXCR4-GNAI-PI3K-BAD signaling pathway, N00485  EBV LMP1 to PI3K signaling pathway, N00501  HTLV-1 Tax to EGFR-PI3K-NFKB signaling pathway, N00514  Mutation-activated EGFR to PI3K signaling pathway, N00582  IGF-IGF1R-PI3K signaling pathway, N00583  HSV VP11/12 to PI3K signaling pathway, N00683  CD80/CD86-CD28-PI3K signaling pathway, N00684  MV F/H to CD28-PI3K signaling pathway, N00736  IAV NS1 to PI3K signaling pathway, N00963  RELN-VLDLR-PI3K signaling pathway, N00964  DAB1-overexpression to RELN-VLDLR-PI3K signaling pathway, N01063  Mutation-activated MET to PI3K signaling pathway, N01065  Mutation-activated RET to PI3K signaling pathway, N01067  Shigella IpgD to ARNO-ARF-ACTB_G signaling pathway, N01129  Salmonella SopB to ARNO-ARF-ACTB/G signaling pathway, N01163  NRG-ERBB4-PI3K signaling pathway, N01204  PRNP-PI3K-NOX2 signaling pathway, N01205  Scrapie conformation PrPSc to PRNP-PI3K-NOX2 signaling pathway, N01338  ACH-CHRN-PI3K signaling pathway, N01339  NNK/NNN to PI3K signaling pathway, N01347  EP/NE-ADRB-PI3K signaling pathway, N01348  Nicotine/NNK to PI3K signaling pathway, N01349  ACH-CHRN-PI3K signaling pathway, N01350  NNK/NNN to PI3K signaling pathway, N01355  Arsenic to PI3K signaling pathway, N01356  Membrane-initiated progesterone signaling pathway, N01357  P4/MPA to membrane-initiated progesterone signaling pathway, N01358  P4-PR-PI3K signaling pathway, N01359  P4/MPA to PR-PI3K signaling pathway, N10023  PI3K inhibitor to gain-of-function mutated PIK3CA]','00562 Inositol phosphate metabolism; 04012 ErbB signaling pathway; 04014 Ras signaling pathway; 04015 Rap1 signaling pathway; 04370 VEGF signaling pathway; 04630 JAK-STAT signaling pathway; 04668 TNF signaling pathway; 04066 HIF-1 signaling pathway; 04068 FoxO signaling pathway; 04070 Phosphatidylinositol signaling system; 04072 Phospholipase D signaling pathway; 04071 Sphingolipid signaling pathway; 04024 cAMP signaling pathway; 04151 PI3K-Akt signaling pathway; 04152 AMPK signaling pathway; 04150 mTOR signaling pathway; 04140 Autophagy - animal; 04210 Apoptosis; 04218 Cellular senescence; 04510 Focal adhesion; 04550 Signaling pathways regulating pluripotency of stem cells; 04810 Regulation of actin cytoskeleton; 04611 Platelet activation; 04613 Neutrophil extracellular trap formation; 04620 Toll-like receptor signaling pathway; 04625 C-type lectin receptor signaling pathway; 04650 Natural killer cell mediated cytotoxicity; 04660 T cell receptor signaling pathway; 04662 B cell receptor signaling pathway; 04664 Fc epsilon RI signaling pathway; 04666 Fc gamma R-mediated phagocytosis; 04670 Leukocyte transendothelial migration; 04062 Chemokine signaling pathway; 04910 Insulin signaling pathway; 04923 Regulation of lipolysis in adipocytes; 04929 GnRH secretion; 04915 Estrogen signaling pathway; 04914 Progesterone-mediated oocyte maturation; 04917 Prolactin signaling pathway; 04926 Relaxin signaling pathway; 04935 Growth hormone synthesis, secretion and action; 04919 Thyroid hormone signaling pathway; 04973 Carbohydrate digestion and absorption; 04960 Aldosterone-regulated sodium reabsorption; 04725 Cholinergic synapse; 04722 Neurotrophin signaling pathway; 04750 Inflammatory mediator regulation of TRP channels; 04360 Axon guidance; 04380 Osteoclast differentiation; 04211 Longevity regulating pathway; 04213 Longevity regulating pathway - multiple species; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05205 Proteoglycans in cancer; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 05203 Viral carcinogenesis; 05230 Central carbon metabolism in cancer; 05231 Choline metabolism in cancer; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05210 Colorectal cancer; 05212 Pancreatic cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05214 Glioma; 05221 Acute myeloid leukemia; 05220 Chronic myeloid leukemia; 05218 Melanoma; 05211 Renal cell carcinoma; 05215 Prostate cancer; 05213 Endometrial cancer; 05224 Breast cancer; 05222 Small cell lung cancer; 05223 Non-small cell lung cancer; 05166 Human T-cell leukemia virus 1 infection; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05160 Hepatitis C; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05165 Human papillomavirus infection; 05132 Salmonella infection; 05131 Shigellosis; 05135 Yersinia infection; 05100 Bacterial invasion of epithelial cells; 05146 Amoebiasis; 05142 Chagas disease; 05010 Alzheimer disease; 05017 Spinocerebellar ataxia; 05020 Prion disease; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 05415 Diabetic cardiomyopathy; 04930 Type II diabetes mellitus; 04932 Non-alcoholic fatty liver disease; 04931 Insulin resistance; 04933 AGE-RAGE signaling pathway in diabetic complications; 01521 EGFR tyrosine kinase inhibitor resistance; 01524 Platinum drug resistance; 01522 Endocrine resistance; 04131 Membrane trafficking [BR:hsa04131]; 2.7.1.153  phosphatidylinositol-4,5-bisphosphate 3-kinase; Rab associated proteins; ','[Pfam: PI3Ka PI3_PI4_kinase PI3K_rbd PI3K_C2 PI3K_p85B DUF2321 ASC ABC1 Ribosomal_S15]'),('5291','hsa:5291','[PIK3CB, P110BETA, PI3K, PI3KBETA, PIK3C1]','[(RefSeq) phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit beta]','[phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit alpha/beta/delta [EC:2.7.1.153]]','[Homo sapiens (human)]','[Inositol phosphate metabolism, Metabolic pathways, EGFR tyrosine kinase inhibitor resistance, Endocrine resistance, Platinum drug resistance, ErbB signaling pathway, Ras signaling pathway, Rap1 signaling pathway, cAMP signaling pathway, Chemokine signaling pathway, HIF-1 signaling pathway, FoxO signaling pathway, Phosphatidylinositol signaling system, Sphingolipid signaling pathway, Phospholipase D signaling pathway, Autophagy - animal, mTOR signaling pathway, PI3K-Akt signaling pathway, AMPK signaling pathway, Apoptosis, Longevity regulating pathway, Longevity regulating pathway - multiple species, Cellular senescence, Axon guidance, VEGF signaling pathway, Osteoclast differentiation, Focal adhesion, Signaling pathways regulating pluripotency of stem cells, Platelet activation, Neutrophil extracellular trap formation, Toll-like receptor signaling pathway, C-type lectin receptor signaling pathway, JAK-STAT signaling pathway, Natural killer cell mediated cytotoxicity, T cell receptor signaling pathway, B cell receptor signaling pathway, Fc epsilon RI signaling pathway, Fc gamma R-mediated phagocytosis, TNF signaling pathway, Leukocyte transendothelial migration, Neurotrophin signaling pathway, Cholinergic synapse, Inflammatory mediator regulation of TRP channels, Regulation of actin cytoskeleton, Insulin signaling pathway, Progesterone-mediated oocyte maturation, Estrogen signaling pathway, Prolactin signaling pathway, Thyroid hormone signaling pathway, Regulation of lipolysis in adipocytes, Relaxin signaling pathway, GnRH secretion, Type II diabetes mellitus, Insulin resistance, Non-alcoholic fatty liver disease, AGE-RAGE signaling pathway in diabetic complications, Growth hormone synthesis, secretion and action, Aldosterone-regulated sodium reabsorption, Carbohydrate digestion and absorption, Alzheimer disease, Spinocerebellar ataxia, Prion disease, Bacterial invasion of epithelial cells, Shigellosis, Salmonella infection, Yersinia infection, Chagas disease, Amoebiasis, Hepatitis C, Hepatitis B, Measles, Human cytomegalovirus infection, Influenza A, Human papillomavirus infection, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Herpes simplex virus 1 infection, Epstein-Barr virus infection, Human immunodeficiency virus 1 infection, Coronavirus disease - COVID-19, Pathways in cancer, Viral carcinogenesis, Proteoglycans in cancer, MicroRNAs in cancer, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species, Colorectal cancer, Renal cell carcinoma, Pancreatic cancer, Endometrial cancer, Glioma, Prostate cancer, Melanoma, Chronic myeloid leukemia, Acute myeloid leukemia, Small cell lung cancer, Non-small cell lung cancer, Breast cancer, Hepatocellular carcinoma, Gastric cancer, Central carbon metabolism in cancer, Choline metabolism in cancer, PD-L1 expression and PD-1 checkpoint pathway in cancer, Diabetic cardiomyopathy, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[c(\"nt06114  PI3K signaling (viruses)\", \"nt06124  Chemokine signaling (viruses)\", \"nt06135  Cytoskeletal regulation (viruses and bacteria)\", \"nt06150  Cytokine-cytokine receptor interaction (viruses)\", \"nt06160  Human T-cell leukemia virus 1 (HTLV-1)\", \"nt06161  Human immunodeficiency virus type 1 (HIV-1)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06163  Hepatitis C virus (HCV)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06165  Epstein-Barr virus (EBV)\", \"nt06166  Human papillomavirus (HPV)\", \n\"nt06167  Human cytomegalovirus (HCMV)\", \"nt06168  Herpes simplex virus 1 (HSV-1)\", \"nt06169  Measles virus (MV)\", \"nt06170  Influenza A virus (IAV)\", \"nt06214  PI3K signaling\", \"nt06260  Colorectal cancer\", \"nt06261  Gastric cancer\", \"nt06262  Pancreatic cancer\", \"nt06263  Hepatocellular carcinoma\", \"nt06264  Renal cell carcinoma\", \"nt06266  Non-small cell lung cancer\", \"nt06268  Melanoma\", \"nt06270  Breast cancer\", \"nt06273  Glioma\", \"nt06275  Acute myeloid leukemia\", \"nt06276  Chronic myeloid leukemia\", \n\"nt06415  PI3K signaling\", \"nt06462  Spinocerebellar ataxia\", \"nt06464  Amyotrophic lateral sclerosis\", \"nt06465  Prion disease\")]','[N00030  EGF-EGFR-RAS-PI3K signaling pathway, N00031  Duplication or mutation-activated FLT3 to RAS-PI3K signaling pathway, N00032  Mutation-activated KRAS/NRAS to PI3K signaling pathway, N00033  EGF-EGFR-PI3K signaling pathway, N00034  ERBB2-overexpression to PI3K signaling pathway, N00035  Amplified EGFR to PI3K signaling pathway, N00036  Mutation-activated EGFR to PI3K signaling pathway, N00037  FGF-FGFR-PI3K signaling pathway, N00038  Amplified FGFR to PI3K signaling pathway, N00039  PDGF-PDGFR-PI3K signaling pathway, N00040  Amplified PDGFR to PI3K signaling pathway, N00042  EGFR-overexpression to PI3K signaling pathway, N00043  HGF-MET-PI3K signaling pathway, N00044  Mutation-activated MET to PI3K signaling pathway, N00045  KITLG-KIT-PI3K signaling pathway, N00046  Mutation-activated KIT to PI3K signaling pathway, N00047  EML4-ALK fusion kinase to PI3K signaling pathway, N00048  BCR-ABL fusion kinase to PI3K signaling pathway, N00159  KSHV K1 to PI3K signaling pathway, N00179  KSHV K1 to PI3K-NFKB signaling pathway, N00182  IGF-IGFR-PI3K-NFKB signaling pathway, N00218  FLT3LG-FLT3-RAS-PI3K signaling pathway, N00231  TGFA-EGFR-PI3K signaling pathway, N00232  TGFA-overexpression to PI3K signaling pathway, N00234  IGF2-IGF1R-PI3K signaling pathway, N00236  IGF2-overexpression to PI3K signaling pathway, N00238  IGF1R-overexpression to PI3K signaling pathway, N00247  HGF-overexpression to PI3K signaling pathway, N00249  MET-overexpression to PI3K signaling pathway, N00253  Amplified ERBB2 to PI3K signaling pathway, N00260  Amplified MET to PI3K signaling pathway, N00266  EBV LMP2A to PI3K signaling pathway, N00267  HBV HBx to PI3K signaling pathway, N00272  HCV NS5A to PI3K signaling pathway, N00281  EGF-overexpression to PI3K signaling pathway, N00282  EREG-EGFR-PI3K signaling pathway, N00283  EREG-overexpression to PI3K signaling pathway, N00284  AREG-EGFR-PI3K signaling pathway, N00285  AREG-overexpression to PI3K signaling pathway, N00366  HPV E5 to EGFR-PI3K signaling pathway, N00368  HPV E5 to PDGFR-PI3K signaling pathway, N00385  HCMV gB to PDGFR-PI3K signaling pathway, N00387  HCMV IE1-72/IE2-86 to PI3K signaling pathway, N00390  EGF-EGFR-PI3K-NFKB signaling pathway, N00391  HCMV gB to EGFR-PI3K-NFKB signaling pathway, N00399  CCR2-GNB/G-PI3K-NFKB signaling pathway, N00400  HCMV US28 to GNB/G-PI3K-NFKB signaling pathway, N00430  CXCR4-GNAI-PI3K-BAD signaling pathway, N00431  HIV gp120 to CXCR4-GNAI-PI3K-BAD signaling pathway, N00485  EBV LMP1 to PI3K signaling pathway, N00501  HTLV-1 Tax to EGFR-PI3K-NFKB signaling pathway, N00514  Mutation-activated EGFR to PI3K signaling pathway, N00582  IGF-IGF1R-PI3K signaling pathway, N00583  HSV VP11/12 to PI3K signaling pathway, N00683  CD80/CD86-CD28-PI3K signaling pathway, N00684  MV F/H to CD28-PI3K signaling pathway, N00736  IAV NS1 to PI3K signaling pathway, N00963  RELN-VLDLR-PI3K signaling pathway, N00964  DAB1-overexpression to RELN-VLDLR-PI3K signaling pathway, N01063  Mutation-activated MET to PI3K signaling pathway, N01065  Mutation-activated RET to PI3K signaling pathway, N01067  Shigella IpgD to ARNO-ARF-ACTB_G signaling pathway, N01129  Salmonella SopB to ARNO-ARF-ACTB/G signaling pathway, N01163  NRG-ERBB4-PI3K signaling pathway, N01204  PRNP-PI3K-NOX2 signaling pathway, N01205  Scrapie conformation PrPSc to PRNP-PI3K-NOX2 signaling pathway, N01338  ACH-CHRN-PI3K signaling pathway, N01339  NNK/NNN to PI3K signaling pathway, N01347  EP/NE-ADRB-PI3K signaling pathway, N01348  Nicotine/NNK to PI3K signaling pathway, N01349  ACH-CHRN-PI3K signaling pathway, N01350  NNK/NNN to PI3K signaling pathway, N01355  Arsenic to PI3K signaling pathway, N01356  Membrane-initiated progesterone signaling pathway, N01357  P4/MPA to membrane-initiated progesterone signaling pathway, N01358  P4-PR-PI3K signaling pathway, N01359  P4/MPA to PR-PI3K signaling pathway]','00562 Inositol phosphate metabolism; 04012 ErbB signaling pathway; 04014 Ras signaling pathway; 04015 Rap1 signaling pathway; 04370 VEGF signaling pathway; 04630 JAK-STAT signaling pathway; 04668 TNF signaling pathway; 04066 HIF-1 signaling pathway; 04068 FoxO signaling pathway; 04070 Phosphatidylinositol signaling system; 04072 Phospholipase D signaling pathway; 04071 Sphingolipid signaling pathway; 04024 cAMP signaling pathway; 04151 PI3K-Akt signaling pathway; 04152 AMPK signaling pathway; 04150 mTOR signaling pathway; 04140 Autophagy - animal; 04210 Apoptosis; 04218 Cellular senescence; 04510 Focal adhesion; 04550 Signaling pathways regulating pluripotency of stem cells; 04810 Regulation of actin cytoskeleton; 04611 Platelet activation; 04613 Neutrophil extracellular trap formation; 04620 Toll-like receptor signaling pathway; 04625 C-type lectin receptor signaling pathway; 04650 Natural killer cell mediated cytotoxicity; 04660 T cell receptor signaling pathway; 04662 B cell receptor signaling pathway; 04664 Fc epsilon RI signaling pathway; 04666 Fc gamma R-mediated phagocytosis; 04670 Leukocyte transendothelial migration; 04062 Chemokine signaling pathway; 04910 Insulin signaling pathway; 04923 Regulation of lipolysis in adipocytes; 04929 GnRH secretion; 04915 Estrogen signaling pathway; 04914 Progesterone-mediated oocyte maturation; 04917 Prolactin signaling pathway; 04926 Relaxin signaling pathway; 04935 Growth hormone synthesis, secretion and action; 04919 Thyroid hormone signaling pathway; 04973 Carbohydrate digestion and absorption; 04960 Aldosterone-regulated sodium reabsorption; 04725 Cholinergic synapse; 04722 Neurotrophin signaling pathway; 04750 Inflammatory mediator regulation of TRP channels; 04360 Axon guidance; 04380 Osteoclast differentiation; 04211 Longevity regulating pathway; 04213 Longevity regulating pathway - multiple species; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05205 Proteoglycans in cancer; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 05203 Viral carcinogenesis; 05230 Central carbon metabolism in cancer; 05231 Choline metabolism in cancer; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05210 Colorectal cancer; 05212 Pancreatic cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05214 Glioma; 05221 Acute myeloid leukemia; 05220 Chronic myeloid leukemia; 05218 Melanoma; 05211 Renal cell carcinoma; 05215 Prostate cancer; 05213 Endometrial cancer; 05224 Breast cancer; 05222 Small cell lung cancer; 05223 Non-small cell lung cancer; 05166 Human T-cell leukemia virus 1 infection; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05160 Hepatitis C; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05165 Human papillomavirus infection; 05132 Salmonella infection; 05131 Shigellosis; 05135 Yersinia infection; 05100 Bacterial invasion of epithelial cells; 05146 Amoebiasis; 05142 Chagas disease; 05010 Alzheimer disease; 05017 Spinocerebellar ataxia; 05020 Prion disease; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 05415 Diabetic cardiomyopathy; 04930 Type II diabetes mellitus; 04932 Non-alcoholic fatty liver disease; 04931 Insulin resistance; 04933 AGE-RAGE signaling pathway in diabetic complications; 01521 EGFR tyrosine kinase inhibitor resistance; 01524 Platinum drug resistance; 01522 Endocrine resistance; 04131 Membrane trafficking [BR:hsa04131]; 2.7.1.153  phosphatidylinositol-4,5-bisphosphate 3-kinase; Rab associated proteins; ','[Pfam: PI3Ka PI3_PI4_kinase PI3K_rbd PI3K_p85B PI3K_C2 Tom37_C]'),('5293','hsa:5293','[PIK3CD, APDS, IMD14, IMD14A, IMD14B, P110DELTA, PI3K, ROCHIS, p110D]','[(RefSeq) phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit delta]','[phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit alpha/beta/delta [EC:2.7.1.153]]','[Homo sapiens (human)]','[Inositol phosphate metabolism, Metabolic pathways, EGFR tyrosine kinase inhibitor resistance, Endocrine resistance, Platinum drug resistance, ErbB signaling pathway, Ras signaling pathway, Rap1 signaling pathway, cAMP signaling pathway, Chemokine signaling pathway, HIF-1 signaling pathway, FoxO signaling pathway, Phosphatidylinositol signaling system, Sphingolipid signaling pathway, Phospholipase D signaling pathway, Autophagy - animal, mTOR signaling pathway, PI3K-Akt signaling pathway, AMPK signaling pathway, Apoptosis, Longevity regulating pathway, Longevity regulating pathway - multiple species, Cellular senescence, Axon guidance, VEGF signaling pathway, Osteoclast differentiation, Focal adhesion, Signaling pathways regulating pluripotency of stem cells, Platelet activation, Neutrophil extracellular trap formation, Toll-like receptor signaling pathway, C-type lectin receptor signaling pathway, JAK-STAT signaling pathway, Natural killer cell mediated cytotoxicity, T cell receptor signaling pathway, B cell receptor signaling pathway, Fc epsilon RI signaling pathway, Fc gamma R-mediated phagocytosis, TNF signaling pathway, Leukocyte transendothelial migration, Neurotrophin signaling pathway, Cholinergic synapse, Inflammatory mediator regulation of TRP channels, Regulation of actin cytoskeleton, Insulin signaling pathway, Progesterone-mediated oocyte maturation, Estrogen signaling pathway, Prolactin signaling pathway, Thyroid hormone signaling pathway, Regulation of lipolysis in adipocytes, Relaxin signaling pathway, GnRH secretion, Type II diabetes mellitus, Insulin resistance, Non-alcoholic fatty liver disease, AGE-RAGE signaling pathway in diabetic complications, Growth hormone synthesis, secretion and action, Aldosterone-regulated sodium reabsorption, Carbohydrate digestion and absorption, Alzheimer disease, Spinocerebellar ataxia, Prion disease, Bacterial invasion of epithelial cells, Shigellosis, Salmonella infection, Yersinia infection, Chagas disease, Amoebiasis, Hepatitis C, Hepatitis B, Measles, Human cytomegalovirus infection, Influenza A, Human papillomavirus infection, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Herpes simplex virus 1 infection, Epstein-Barr virus infection, Human immunodeficiency virus 1 infection, Coronavirus disease - COVID-19, Pathways in cancer, Viral carcinogenesis, Proteoglycans in cancer, MicroRNAs in cancer, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species, Colorectal cancer, Renal cell carcinoma, Pancreatic cancer, Endometrial cancer, Glioma, Prostate cancer, Melanoma, Chronic myeloid leukemia, Acute myeloid leukemia, Small cell lung cancer, Non-small cell lung cancer, Breast cancer, Hepatocellular carcinoma, Gastric cancer, Central carbon metabolism in cancer, Choline metabolism in cancer, PD-L1 expression and PD-1 checkpoint pathway in cancer, Diabetic cardiomyopathy, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[c(\"nt06114  PI3K signaling (viruses)\", \"nt06124  Chemokine signaling (viruses)\", \"nt06135  Cytoskeletal regulation (viruses and bacteria)\", \"nt06150  Cytokine-cytokine receptor interaction (viruses)\", \"nt06160  Human T-cell leukemia virus 1 (HTLV-1)\", \"nt06161  Human immunodeficiency virus type 1 (HIV-1)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06163  Hepatitis C virus (HCV)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06165  Epstein-Barr virus (EBV)\", \"nt06166  Human papillomavirus (HPV)\", \n\"nt06167  Human cytomegalovirus (HCMV)\", \"nt06168  Herpes simplex virus 1 (HSV-1)\", \"nt06169  Measles virus (MV)\", \"nt06170  Influenza A virus (IAV)\", \"nt06214  PI3K signaling\", \"nt06260  Colorectal cancer\", \"nt06261  Gastric cancer\", \"nt06262  Pancreatic cancer\", \"nt06263  Hepatocellular carcinoma\", \"nt06264  Renal cell carcinoma\", \"nt06266  Non-small cell lung cancer\", \"nt06268  Melanoma\", \"nt06270  Breast cancer\", \"nt06273  Glioma\", \"nt06275  Acute myeloid leukemia\", \"nt06276  Chronic myeloid leukemia\", \n\"nt06415  PI3K signaling\", \"nt06462  Spinocerebellar ataxia\", \"nt06464  Amyotrophic lateral sclerosis\", \"nt06465  Prion disease\")]','[N00030  EGF-EGFR-RAS-PI3K signaling pathway, N00031  Duplication or mutation-activated FLT3 to RAS-PI3K signaling pathway, N00032  Mutation-activated KRAS/NRAS to PI3K signaling pathway, N00033  EGF-EGFR-PI3K signaling pathway, N00034  ERBB2-overexpression to PI3K signaling pathway, N00035  Amplified EGFR to PI3K signaling pathway, N00036  Mutation-activated EGFR to PI3K signaling pathway, N00037  FGF-FGFR-PI3K signaling pathway, N00038  Amplified FGFR to PI3K signaling pathway, N00039  PDGF-PDGFR-PI3K signaling pathway, N00040  Amplified PDGFR to PI3K signaling pathway, N00042  EGFR-overexpression to PI3K signaling pathway, N00043  HGF-MET-PI3K signaling pathway, N00044  Mutation-activated MET to PI3K signaling pathway, N00045  KITLG-KIT-PI3K signaling pathway, N00046  Mutation-activated KIT to PI3K signaling pathway, N00047  EML4-ALK fusion kinase to PI3K signaling pathway, N00048  BCR-ABL fusion kinase to PI3K signaling pathway, N00159  KSHV K1 to PI3K signaling pathway, N00179  KSHV K1 to PI3K-NFKB signaling pathway, N00182  IGF-IGFR-PI3K-NFKB signaling pathway, N00218  FLT3LG-FLT3-RAS-PI3K signaling pathway, N00231  TGFA-EGFR-PI3K signaling pathway, N00232  TGFA-overexpression to PI3K signaling pathway, N00234  IGF2-IGF1R-PI3K signaling pathway, N00236  IGF2-overexpression to PI3K signaling pathway, N00238  IGF1R-overexpression to PI3K signaling pathway, N00247  HGF-overexpression to PI3K signaling pathway, N00249  MET-overexpression to PI3K signaling pathway, N00253  Amplified ERBB2 to PI3K signaling pathway, N00260  Amplified MET to PI3K signaling pathway, N00266  EBV LMP2A to PI3K signaling pathway, N00267  HBV HBx to PI3K signaling pathway, N00272  HCV NS5A to PI3K signaling pathway, N00281  EGF-overexpression to PI3K signaling pathway, N00282  EREG-EGFR-PI3K signaling pathway, N00283  EREG-overexpression to PI3K signaling pathway, N00284  AREG-EGFR-PI3K signaling pathway, N00285  AREG-overexpression to PI3K signaling pathway, N00366  HPV E5 to EGFR-PI3K signaling pathway, N00368  HPV E5 to PDGFR-PI3K signaling pathway, N00385  HCMV gB to PDGFR-PI3K signaling pathway, N00387  HCMV IE1-72/IE2-86 to PI3K signaling pathway, N00390  EGF-EGFR-PI3K-NFKB signaling pathway, N00391  HCMV gB to EGFR-PI3K-NFKB signaling pathway, N00399  CCR2-GNB/G-PI3K-NFKB signaling pathway, N00400  HCMV US28 to GNB/G-PI3K-NFKB signaling pathway, N00430  CXCR4-GNAI-PI3K-BAD signaling pathway, N00431  HIV gp120 to CXCR4-GNAI-PI3K-BAD signaling pathway, N00485  EBV LMP1 to PI3K signaling pathway, N00501  HTLV-1 Tax to EGFR-PI3K-NFKB signaling pathway, N00514  Mutation-activated EGFR to PI3K signaling pathway, N00582  IGF-IGF1R-PI3K signaling pathway, N00583  HSV VP11/12 to PI3K signaling pathway, N00683  CD80/CD86-CD28-PI3K signaling pathway, N00684  MV F/H to CD28-PI3K signaling pathway, N00736  IAV NS1 to PI3K signaling pathway, N00963  RELN-VLDLR-PI3K signaling pathway, N00964  DAB1-overexpression to RELN-VLDLR-PI3K signaling pathway, N01063  Mutation-activated MET to PI3K signaling pathway, N01065  Mutation-activated RET to PI3K signaling pathway, N01067  Shigella IpgD to ARNO-ARF-ACTB_G signaling pathway, N01129  Salmonella SopB to ARNO-ARF-ACTB/G signaling pathway, N01163  NRG-ERBB4-PI3K signaling pathway, N01204  PRNP-PI3K-NOX2 signaling pathway, N01205  Scrapie conformation PrPSc to PRNP-PI3K-NOX2 signaling pathway, N01338  ACH-CHRN-PI3K signaling pathway, N01339  NNK/NNN to PI3K signaling pathway, N01347  EP/NE-ADRB-PI3K signaling pathway, N01348  Nicotine/NNK to PI3K signaling pathway, N01349  ACH-CHRN-PI3K signaling pathway, N01350  NNK/NNN to PI3K signaling pathway, N01355  Arsenic to PI3K signaling pathway, N01356  Membrane-initiated progesterone signaling pathway, N01357  P4/MPA to membrane-initiated progesterone signaling pathway, N01358  P4-PR-PI3K signaling pathway, N01359  P4/MPA to PR-PI3K signaling pathway]','00562 Inositol phosphate metabolism; 04012 ErbB signaling pathway; 04014 Ras signaling pathway; 04015 Rap1 signaling pathway; 04370 VEGF signaling pathway; 04630 JAK-STAT signaling pathway; 04668 TNF signaling pathway; 04066 HIF-1 signaling pathway; 04068 FoxO signaling pathway; 04070 Phosphatidylinositol signaling system; 04072 Phospholipase D signaling pathway; 04071 Sphingolipid signaling pathway; 04024 cAMP signaling pathway; 04151 PI3K-Akt signaling pathway; 04152 AMPK signaling pathway; 04150 mTOR signaling pathway; 04140 Autophagy - animal; 04210 Apoptosis; 04218 Cellular senescence; 04510 Focal adhesion; 04550 Signaling pathways regulating pluripotency of stem cells; 04810 Regulation of actin cytoskeleton; 04611 Platelet activation; 04613 Neutrophil extracellular trap formation; 04620 Toll-like receptor signaling pathway; 04625 C-type lectin receptor signaling pathway; 04650 Natural killer cell mediated cytotoxicity; 04660 T cell receptor signaling pathway; 04662 B cell receptor signaling pathway; 04664 Fc epsilon RI signaling pathway; 04666 Fc gamma R-mediated phagocytosis; 04670 Leukocyte transendothelial migration; 04062 Chemokine signaling pathway; 04910 Insulin signaling pathway; 04923 Regulation of lipolysis in adipocytes; 04929 GnRH secretion; 04915 Estrogen signaling pathway; 04914 Progesterone-mediated oocyte maturation; 04917 Prolactin signaling pathway; 04926 Relaxin signaling pathway; 04935 Growth hormone synthesis, secretion and action; 04919 Thyroid hormone signaling pathway; 04973 Carbohydrate digestion and absorption; 04960 Aldosterone-regulated sodium reabsorption; 04725 Cholinergic synapse; 04722 Neurotrophin signaling pathway; 04750 Inflammatory mediator regulation of TRP channels; 04360 Axon guidance; 04380 Osteoclast differentiation; 04211 Longevity regulating pathway; 04213 Longevity regulating pathway - multiple species; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05205 Proteoglycans in cancer; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 05203 Viral carcinogenesis; 05230 Central carbon metabolism in cancer; 05231 Choline metabolism in cancer; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05210 Colorectal cancer; 05212 Pancreatic cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05214 Glioma; 05221 Acute myeloid leukemia; 05220 Chronic myeloid leukemia; 05218 Melanoma; 05211 Renal cell carcinoma; 05215 Prostate cancer; 05213 Endometrial cancer; 05224 Breast cancer; 05222 Small cell lung cancer; 05223 Non-small cell lung cancer; 05166 Human T-cell leukemia virus 1 infection; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05160 Hepatitis C; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05165 Human papillomavirus infection; 05132 Salmonella infection; 05131 Shigellosis; 05135 Yersinia infection; 05100 Bacterial invasion of epithelial cells; 05146 Amoebiasis; 05142 Chagas disease; 05010 Alzheimer disease; 05017 Spinocerebellar ataxia; 05020 Prion disease; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 05415 Diabetic cardiomyopathy; 04930 Type II diabetes mellitus; 04932 Non-alcoholic fatty liver disease; 04931 Insulin resistance; 04933 AGE-RAGE signaling pathway in diabetic complications; 01521 EGFR tyrosine kinase inhibitor resistance; 01524 Platinum drug resistance; 01522 Endocrine resistance; 04131 Membrane trafficking [BR:hsa04131]; 2.7.1.153  phosphatidylinositol-4,5-bisphosphate 3-kinase; Rab associated proteins; ','[Pfam: PI3_PI4_kinase PI3Ka PI3K_p85B PI3K_rbd PI3K_C2 DUF4135]'),('11009','hsa:11009','[IL24, C49A, FISP, IL10B, MDA7, MOB5, ST16]','[(RefSeq) interleukin 24]','[interleukin 24]','[Homo sapiens (human)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, JAK-STAT signaling pathway]','[nt06150  Cytokine-cytokine receptor interaction (viruses)]','[N01271  IL19/20/24 to IL20R signaling, N01273  IL20/24 to IL20R/22R signaling]','04630 JAK-STAT signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04052 Cytokines and growth factors [BR:hsa04052]; Interleukins; ','[Pfam: IL22 IL10]'),('282617','hsa:282617','[IFNL3, IFN-lambda-3, IFN-lambda-4, IL-28B, IL-28C, IL28B, IL28C]','[(RefSeq) interferon lambda 3]','[interferon lambda 2/3]','[Homo sapiens (human)]','[Cytokine-cytokine receptor interaction, JAK-STAT signaling pathway]','[]','[]','04630 JAK-STAT signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04052 Cytokines and growth factors [BR:hsa04052]; Interferons; ','[Pfam: IL28A]'),('6370','hsa:6370','[CCL25, Ck_beta-15, Ckb15, SCYA25, TECK]','[(RefSeq) C-C motif chemokine ligand 25]','[C-C motif chemokine 25]','[Homo sapiens (human)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, Chemokine signaling pathway, Intestinal immune network for IgA production]','[nt06150  Cytokine-cytokine receptor interaction (viruses)]','[N01221  CCL25 to CCR9/11 signaling, N01222  HSV1 SgG1 to CCR signaling, N01223  HSV2 SgG2 to CCR signaling, N01224  VAV CrmB to CCR signaling]','04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04672 Intestinal immune network for IgA production; 04062 Chemokine signaling pathway; 04052 Cytokines and growth factors [BR:hsa04052]; Chemokines; ','[Pfam: IL8]'),('6351','hsa:6351','[CCL4, ACT2, AT744.1, G-26, HC21, LAG-1, LAG1, MIP-1-beta, MIP1B, MIP1B1, SCYA2, SCYA4]','[(RefSeq) C-C motif chemokine ligand 4]','[C-C motif chemokine 4]','[Homo sapiens (human)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, Chemokine signaling pathway, NF-kappa B signaling pathway, Toll-like receptor signaling pathway, Cytosolic DNA-sensing pathway, Human cytomegalovirus infection]','[c(\"nt06124  Chemokine signaling (viruses)\", \"nt06150  Cytokine-cytokine receptor interaction (viruses)\", \"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06167  Human cytomegalovirus (HCMV)\", \"nt06224  CXCR signaling\")]','[N00153  CCR/CXCR-GNB/G-PI3K-RAC signaling pathway, N00406  HCMV US28 to GNA12/13-Rho signaling pathway, N01196  CCL3/4/5/8 to CCR5 signaling, N01207  CCL4 to HHV6 U12 signaling, N01212  CCL4 to HCMV US28 signaling]','04064 NF-kappa B signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04620 Toll-like receptor signaling pathway; 04623 Cytosolic DNA-sensing pathway; 04062 Chemokine signaling pathway; 05163 Human cytomegalovirus infection; 04052 Cytokines and growth factors [BR:hsa04052]; Chemokines; ','[Pfam: IL8]'),('6360','hsa:6360','[CCL16, CKb12, HCC-4, ILINCK, LCC-1, LEC, LMC, Mtn-1, NCC-4, NCC4, SCYA16, SCYL4]','[(RefSeq) C-C motif chemokine ligand 16]','[C-C motif chemokine 16]','[Homo sapiens (human)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, Chemokine signaling pathway]','[nt06150  Cytokine-cytokine receptor interaction (viruses)]','[N01166  CCL3/5/7/8/13/14/15/16/23 to CCR1 signaling]','04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04062 Chemokine signaling pathway; 04052 Cytokines and growth factors [BR:hsa04052]; Chemokines; ','[Pfam: IL8]'),('6375','hsa:6375','[XCL1, ATAC, LPTN, LTN, SCM-1, SCM-1a, SCM1, SCM1A, SCYC1]','[(RefSeq) X-C motif chemokine ligand 1]','[C motif chemokine 1]','[Homo sapiens (human)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, Chemokine signaling pathway]','[nt06150  Cytokine-cytokine receptor interaction (viruses)]','[N01258  XCL to XCR1 signaling, N01259  XCL1 to HHV6 U51 signaling]','04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04062 Chemokine signaling pathway; 04052 Cytokines and growth factors [BR:hsa04052]; Chemokines; ','[Pfam: IL8 PHtD_u1]'),('6376','hsa:6376','[CX3CL1, ABCD-3, C3Xkine, CXC3, CXC3C, NTN, NTT, SCYD1, fractalkine, neurotactin]','[(RefSeq) C-X3-C motif chemokine ligand 1]','[C-X3-C motif chemokine 1]','[Homo sapiens (human)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, Chemokine signaling pathway, TNF signaling pathway, Human cytomegalovirus infection]','[c(\"nt06124  Chemokine signaling (viruses)\", \"nt06150  Cytokine-cytokine receptor interaction (viruses)\", \"nt06167  Human cytomegalovirus (HCMV)\")]','[N00403  CX3CR1-GNAI-AC-PKA signaling pathway, N00406  HCMV US28 to GNA12/13-Rho signaling pathway, N01262  CX3CL1 to HCMV US28 signaling]','04668 TNF signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04062 Chemokine signaling pathway; 05163 Human cytomegalovirus infection; 04052 Cytokines and growth factors [BR:hsa04052]; Chemokines; ','[Pfam: IL8]'),('1843','hsa:1843','[DUSP1, CL100, HVH1, MKP-1, MKP1, PTPN10]','[(RefSeq) dual specificity phosphatase 1]','[dual specificity protein phosphatase 1 [EC:3.1.3.16 3.1.3.48]]','[Homo sapiens (human)]','[MAPK signaling pathway, Serotonergic synapse, Parkinson disease, Fluid shear stress and atherosclerosis]','[c(\"nt06440  Transcription\", \"nt06463  Parkinson disease\")]','[N01058  Mutation-inactivated DJ1 to to p53-mediated transcription]','04010 MAPK signaling pathway; 04726 Serotonergic synapse; 05012 Parkinson disease; 05418 Fluid shear stress and atherosclerosis; 01009 Protein phosphatases and associated proteins [BR:hsa01009]; 3.1.3.16  protein-serine/threonine phosphatase; 3.1.3.48  protein-tyrosine-phosphatase; MAPK phosphatases; ','[Pfam: DSPc Rhodanese Y_phosphatase Init_tRNA_PT]'),('1845','hsa:1845','[DUSP3, VHR]','[(RefSeq) dual specificity phosphatase 3]','[dual specificity phosphatase 3 [EC:3.1.3.16 3.1.3.48]]','[Homo sapiens (human)]','[MAPK signaling pathway]','[]','[]','04010 MAPK signaling pathway; 01009 Protein phosphatases and associated proteins [BR:hsa01009]; 3.1.3.16  protein-serine/threonine phosphatase; 3.1.3.48  protein-tyrosine-phosphatase; Atypical DSPs; ','[Pfam: DSPc Y_phosphatase Init_tRNA_PT Y_phosphatase2 BLH_phosphatase]'),('11221','hsa:11221','[DUSP10, MKP-5, MKP5]','[(RefSeq) dual specificity phosphatase 10]','[dual specificity phosphatase 10 [EC:3.1.3.16 3.1.3.48]]','[Homo sapiens (human)]','[MAPK signaling pathway]','[]','[]','04010 MAPK signaling pathway; 01009 Protein phosphatases and associated proteins [BR:hsa01009]; 3.1.3.16  protein-serine/threonine phosphatase; 3.1.3.48  protein-tyrosine-phosphatase; MAPK phosphatases; ','[Pfam: DSPc Rhodanese PTPlike_phytase Y_phosphatase Init_tRNA_PT]'),('150290','hsa:150290','[DUSP18, DSP18, DUSP20, LMWDSP20]','[(RefSeq) dual specificity phosphatase 18]','[atypical dual specificity phosphatase [EC:3.1.3.16 3.1.3.48]]','[Homo sapiens (human)]',NULL,'[]','[]','01009 Protein phosphatases and associated proteins [BR:hsa01009]; 3.1.3.16  protein-serine/threonine phosphatase; 3.1.3.48  protein-tyrosine-phosphatase; Atypical DSPs; ','[Pfam: DSPc Y_phosphatase]'),('56940','hsa:56940','[DUSP22, JKAP, JSP-1, JSP1, LMW-DSP2, LMWDSP2, MKP-x, MKPX, VHX]','[(RefSeq) dual specificity phosphatase 22]','[atypical dual specificity phosphatase [EC:3.1.3.16 3.1.3.48]]','[Homo sapiens (human)]',NULL,'[]','[]','01009 Protein phosphatases and associated proteins [BR:hsa01009]; 3.1.3.16  protein-serine/threonine phosphatase; 3.1.3.48  protein-tyrosine-phosphatase; Atypical DSPs; ','[Pfam: DSPc Y_phosphatase Init_tRNA_PT PTPlike_phytase]'),('3667','hsa:3667','[IRS1, HIRS-1]','[(RefSeq) insulin receptor substrate 1]','[insulin receptor substrate 1]','[Homo sapiens (human)]','[cGMP-PKG signaling pathway, FoxO signaling pathway, Autophagy - animal, mTOR signaling pathway, PI3K-Akt signaling pathway, AMPK signaling pathway, Longevity regulating pathway, Longevity regulating pathway - multiple species, Neurotrophin signaling pathway, Insulin signaling pathway, Adipocytokine signaling pathway, Regulation of lipolysis in adipocytes, Type II diabetes mellitus, Insulin resistance, Non-alcoholic fatty liver disease, Growth hormone synthesis, secretion and action, Aldosterone-regulated sodium reabsorption, Alzheimer disease, MicroRNAs in cancer, Diabetic cardiomyopathy]','[]','[]','04068 FoxO signaling pathway; 04022 cGMP-PKG signaling pathway; 04151 PI3K-Akt signaling pathway; 04152 AMPK signaling pathway; 04150 mTOR signaling pathway; 04140 Autophagy - animal; 04910 Insulin signaling pathway; 04923 Regulation of lipolysis in adipocytes; 04920 Adipocytokine signaling pathway; 04935 Growth hormone synthesis, secretion and action; 04960 Aldosterone-regulated sodium reabsorption; 04722 Neurotrophin signaling pathway; 04211 Longevity regulating pathway; 04213 Longevity regulating pathway - multiple species; 05206 MicroRNAs in cancer; 05010 Alzheimer disease; 05415 Diabetic cardiomyopathy; 04930 Type II diabetes mellitus; 04932 Non-alcoholic fatty liver disease; 04931 Insulin resistance; ','[Pfam: IRS PH PH_11]'),('6009','hsa:6009','[RHEB, RHEB2]','[(RefSeq) Ras homolog, mTORC1 binding]','[Ras homolog enriched in brain]','[Homo sapiens (human)]','[Phospholipase D signaling pathway, Autophagy - animal, mTOR signaling pathway, PI3K-Akt signaling pathway, AMPK signaling pathway, Longevity regulating pathway, Cellular senescence, Thermogenesis, Insulin signaling pathway, Thyroid hormone signaling pathway, Human cytomegalovirus infection, Human papillomavirus infection, Herpes simplex virus 1 infection, Choline metabolism in cancer]','[c(\"nt06114  PI3K signaling (viruses)\", \"nt06166  Human papillomavirus (HPV)\", \"nt06167  Human cytomegalovirus (HCMV)\", \"nt06168  Herpes simplex virus 1 (HSV-1)\", \"nt06415  PI3K signaling\", \"nt06462  Spinocerebellar ataxia\")]','[N00357  HPV E6 to MTOR signaling pathway, N00388  HCMV UL38 to MTOR signaling pathway, N00582  IGF-IGF1R-PI3K signaling pathway, N00583  HSV VP11/12 to PI3K signaling pathway, N00584  HSV US3 to MTOR signaling pathway, N00963  RELN-VLDLR-PI3K signaling pathway]','04072 Phospholipase D signaling pathway; 04151 PI3K-Akt signaling pathway; 04152 AMPK signaling pathway; 04150 mTOR signaling pathway; 04140 Autophagy - animal; 04218 Cellular senescence; 04910 Insulin signaling pathway; 04919 Thyroid hormone signaling pathway; 04211 Longevity regulating pathway; 04714 Thermogenesis; 05231 Choline metabolism in cancer; 05168 Herpes simplex virus 1 infection; 05163 Human cytomegalovirus infection; 05165 Human papillomavirus infection; 04031 GTP-binding proteins [BR:hsa04031]; Rheb; ','[Pfam: Ras Roc Arf MMR_HSR1 GTP_EFTU RsgA_GTPase SRPRB AAA_14]'),('29952','hsa:29952','[DPP7, DPP2, DPPII, QPP]','[(RefSeq) dipeptidyl peptidase 7]','[dipeptidyl-peptidase II [EC:3.4.14.2]]','[Homo sapiens (human)]',NULL,'[]','[]','01002 Peptidases and inhibitors [BR:hsa01002]; 3.4.14.2  dipeptidyl-peptidase II; Family S28; ','[Pfam: Peptidase_S28 Abhydrolase_1 Hydrolase_4 Peptidase_S37 Abhydrolase_6 Peptidase_S9 DUF2920 Abhydrolase_2 COX6B]'),('1613','hsa:1613','[DAPK3, DLK, ZIP, ZIPK]','[(RefSeq) death associated protein kinase 3]','[death-associated protein kinase [EC:2.7.11.1]]','[Homo sapiens (human)]','[Autophagy - animal, Pathways in cancer, Bladder cancer]','[]','[]','04140 Autophagy - animal; 05200 Pathways in cancer; 05219 Bladder cancer; 01001 Protein kinases [BR:hsa01001]; 2.7.11.1  non-specific serine/threonine protein kinase; DAPK family; ','[Pfam: Pkinase PK_Tyr_Ser-Thr Kdo RIO1 ABC1 Choline_kinase TCAD9 APH]'),('57175','hsa:57175','[CORO1B, CORONIN-2]','[(RefSeq) coronin 1B]','[coronin-1B/1C/6]','[Homo sapiens (human)]',NULL,'[]','[]','04812 Cytoskeleton proteins [BR:hsa04812]; Cross-linking proteins; ','[Pfam: DUF1899 WD40_4 WD40 ANAPC4_WD40 Trimer_CC NBCH_WD40]'),('3489','hsa:3489','[IGFBP6, IBP6]','[(RefSeq) insulin like growth factor binding protein 6]','[insulin-like growth factor-binding protein 6]','[Homo sapiens (human)]',NULL,'[]','[]','04131 Membrane trafficking [BR:hsa04131]; Insulin-like growth factor-binding proteins; ','[Pfam: Thyroglobulin_1 IGFBP]'),('3995','hsa:3995','[FADS3, CYB5RP, LLCDL3]','[(RefSeq) fatty acid desaturase 3]','[fatty acid desaturase 3 (Delta-13 desaturase) [EC:1.14.19.-]]','[Homo sapiens (human)]',NULL,'[]','[]','01004 Lipid biosynthesis proteins [BR:hsa01004]; 1.14.19.-; Omega; ','[Pfam: FA_desaturase Cyt-b5]'),('2316','hsa:2316','[FLNA, ABP-280, ABPX, CSBS, CVD1, FGS2, FLN, FLN-A, FLN1, FMD, MNS, NHBP, OPD, OPD1, OPD2, XLVD, XMVD]','[(RefSeq) filamin A]','[filamin]','[Homo sapiens (human)]','[MAPK signaling pathway, Focal adhesion, Salmonella infection, Proteoglycans in cancer]','[]','[]','04010 MAPK signaling pathway; 04510 Focal adhesion; 05205 Proteoglycans in cancer; 05132 Salmonella infection; 04812 Cytoskeleton proteins [BR:hsa04812]; 04147 Exosome [BR:hsa04147]; Cross-linking proteins; Intermediate filament-binding proteins; Exosomal proteins of haemopoietic cells  (B-cell, T-cell, DC-cell, reticulocyte, and mast cell); ','[Pfam: Filamin CH Y_Y_Y Big_3_5 CFSR]'),('2539','hsa:2539','[G6PD, G6PD1]','[(RefSeq) glucose-6-phosphate dehydrogenase]','[glucose-6-phosphate 1-dehydrogenase [EC:1.1.1.49 1.1.1.363]]','[Homo sapiens (human)]','[Pentose phosphate pathway, Glutathione metabolism, Metabolic pathways, Carbon metabolism, Central carbon metabolism in cancer, Diabetic cardiomyopathy]','[nt06026  Glutathione biosynthesis]','[N00905  NADP+ reduction]','00030 Pentose phosphate pathway; 00480 Glutathione metabolism; 05230 Central carbon metabolism in cancer; 05415 Diabetic cardiomyopathy; 04147 Exosome [BR:hsa04147]; 1.1.1.49  glucose-6-phosphate dehydrogenase (NADP+); 1.1.1.363  glucose-6-phosphate dehydrogenase [NAD(P)+]; Exosomal proteins of breast cancer cells; Exosomal proteins of colorectal cancer cells; Exosomal proteins of bladder cancer cells; ','[Pfam: G6PD_C G6PD_N]'),('105375355','hsa:105375355','[UPK3B, P35, UP3B, UPIIIB]','[(RefSeq) uroplakin 3B]',NULL,'[Homo sapiens (human)]',NULL,'[]','[]',NULL,'[]'),('10400','hsa:10400','[PEMT, PEAMT, PEMPT, PEMT2, PLMT, PNMT]','[(RefSeq) phosphatidylethanolamine N-methyltransferase]','[phosphatidylethanolamine/phosphatidyl-N-methylethanolamine N-methyltransferase [EC:2.1.1.17 2.1.1.71]]','[Homo sapiens (human)]','[Glycerophospholipid metabolism, Metabolic pathways]','[]','[]','00564 Glycerophospholipid metabolism; 2.1.1.17  phosphatidylethanolamine N-methyltransferase; 2.1.1.71  phosphatidyl-N-methylethanolamine N-methyltransferase; ','[Pfam: PEMT Herpes_UL74 EptA_B_N]'),('2810','hsa:2810','[SFN, YWHAS]','[(RefSeq) stratifin]','[stratifin]','[Homo sapiens (human)]','[Cell cycle, p53 signaling pathway, Aldosterone-regulated sodium reabsorption]','[]','[]','04110 Cell cycle; 04115 p53 signaling pathway; 04960 Aldosterone-regulated sodium reabsorption; 04147 Exosome [BR:hsa04147]; Exosomal proteins of other body fluids (saliva and urine); Exosomal proteins of colorectal cancer cells; Exosomal proteins of bladder cancer cells; ','[Pfam: 14-3-3 SIKE Peptidase_C98]'),('51337','hsa:51337','[THEM6, C8orf55, DSCD75]','[(RefSeq) thioesterase superfamily member 6]',NULL,'[Homo sapiens (human)]',NULL,'[]','[]',NULL,'[Pfam: 4HBT_2 4HBT Acyl-ACP_TE]'),('54998','hsa:54998','[AURKAIP1, AIP, AKIP, MRP-S38]','[(RefSeq) aurora kinase A interacting protein 1]','[aurora kinase A-interacting protein]','[Homo sapiens (human)]',NULL,'[]','[]','03036 Chromosome and associated proteins [BR:hsa03036]; Regulators of Aurora kinases; ','[Pfam: COX24_C]'),('79094','hsa:79094','[CHAC1]','[(RefSeq) ChaC glutathione specific gamma-glutamylcyclotransferase 1]','[glutathione-specific gamma-glutamylcyclotransferase [EC:4.3.2.7]]','[Homo sapiens (human)]','[Glutathione metabolism, Metabolic pathways]','[]','[]','00480 Glutathione metabolism; 4.3.2.7  glutathione-specific gamma-glutamylcyclotransferase; ','[Pfam: ChaC GGACT]'),('10591','hsa:10591','[DNPH1, C6orf108, RCL, dJ330M21.3]','[(RefSeq) 2\'-deoxynucleoside 5\'-phosphate N-hydrolase 1]','[2\'-deoxynucleoside 5\'-phosphate N-hydrolase [EC:3.2.2.-]]','[Homo sapiens (human)]',NULL,'[]','[]','99980 Enzymes with EC numbers; 3.2.2.-; ','[Pfam: Nuc_deoxyrib_tr]'),('2286','hsa:2286','[FKBP2, FKBP-13, FKBP13, PPIase]','[(RefSeq) FKBP prolyl isomerase 2]','[FK506-binding protein 2 [EC:5.2.1.8]]','[Homo sapiens (human)]',NULL,'[]','[]','03110 Chaperones and folding catalysts [BR:hsa03110]; 5.2.1.8  peptidylprolyl isomerase; FKBP; ','[Pfam: FKBP_C]'),('10791','hsa:10791','[VAMP5]','[(RefSeq) vesicle associated membrane protein 5]','[vesicle-associated membrane protein 5]','[Homo sapiens (human)]','[SNARE interactions in vesicular transport]','[]','[]','04130 SNARE interactions in vesicular transport; 04131 Membrane trafficking [BR:hsa04131]; VAMP; ','[Pfam: Synaptobrevin Tweety Amnionless ODV-E18 RAMP OppC_N DUF2207 Syndecan]'),('648','hsa:648','[BMI1, FLVI2/BMI1, PCGF4, RNF51, flvi-2/bmi-1]','[(RefSeq) BMI1 proto-oncogene, polycomb ring finger]','[polycomb group RING finger protein 4]','[Homo sapiens (human)]','[Signaling pathways regulating pluripotency of stem cells, Transcriptional misregulation in cancer, MicroRNAs in cancer]','[nt06240  Transcription]','[N00131  Amplified MYCN to transcriptional activation]','04550 Signaling pathways regulating pluripotency of stem cells; 05202 Transcriptional misregulation in cancer; 05206 MicroRNAs in cancer; 03036 Chromosome and associated proteins [BR:hsa03036]; PRC1 (polycomb repressive complex 1); ','[Pfam: RAWUL zf-C3HC4_2 zf-C3HC4 zf-RING_2 zf-C3HC4_3 zf-RING_5 zf-rbx1 zf-RING_UBOX Prok-RING_4 zf-C3HC4_4 zf-RING_10 zf-Nse]'),('347735','hsa:347735','[SERINC2, FKSG84, PRO0899, TDE2, TDE2L]','[(RefSeq) serine incorporator 2]','[serine incorporator 2]','[Homo sapiens (human)]',NULL,'[]','[]','04131 Membrane trafficking [BR:hsa04131]; Others; ','[Pfam: Serinc]'),('1509','hsa:1509','[CTSD, CLN10, CPSD, HEL-S-130P]','[(RefSeq) cathepsin D]','[cathepsin D [EC:3.4.23.5]]','[Homo sapiens (human)]','[Sphingolipid signaling pathway, Autophagy - animal, Lysosome, Apoptosis, Estrogen signaling pathway, Tuberculosis, Diabetic cardiomyopathy]','[c(\"nt06227  Nuclear receptor signaling\", \"nt06270  Breast cancer\", \"nt06323  KISS1-GnRH-LH/FSH-E2 signaling\")]','[N00286  Nuclear-initiated estrogen signaling pathway]','04071 Sphingolipid signaling pathway; 04142 Lysosome; 04140 Autophagy - animal; 04210 Apoptosis; 04915 Estrogen signaling pathway; 05152 Tuberculosis; 05415 Diabetic cardiomyopathy; 01002 Peptidases and inhibitors [BR:hsa01002]; 04147 Exosome [BR:hsa04147]; 3.4.23.5  cathepsin D; Family A1: pepsin family; Exosomal proteins of other body fluids (saliva and urine); ','[Pfam: Asp TAXi_N A1_Propeptide TAXi_C]'),('84189','hsa:84189','[SLITRK6, DFNMYP]','[(RefSeq) SLIT and NTRK like family member 6]',NULL,'[Homo sapiens (human)]',NULL,'[]','[]',NULL,'[Pfam: LRR_8 LRR_4 LRR_5 LRR_9 LRRCT_2 LapA_dom LRRCT]'),('93432','hsa:93432','[MGAM2]','[(RefSeq) maltase-glucoamylase 2 (putative)]','[maltase-glucoamylase [EC:3.2.1.20 3.2.1.3]]','[Homo sapiens (human)]','[Galactose metabolism, Starch and sucrose metabolism, Metabolic pathways, Carbohydrate digestion and absorption]','[nt06017  Glycogen metabolism]','[N00720  Glycogen degradation (amylase)]','00052 Galactose metabolism; 00500 Starch and sucrose metabolism; 04973 Carbohydrate digestion and absorption; 3.2.1.3  glucan 1,4-alpha-glucosidase; 3.2.1.20  alpha-glucosidase; ','[Pfam: Glyco_hydro_31 NtCtMGAM_N Trefoil Gal_mutarotas_2 DUF5110]'),('9480','hsa:9480','[ONECUT2, OC-2, OC2]','[(RefSeq) one cut homeobox 2]','[one cut domain family member 2]','[Homo sapiens (human)]',NULL,'[]','[]','03000 Transcription factors [BR:hsa03000]; Homeo domain CUT; ','[Pfam: CUT Homeodomain Homeobox_KN SelP_N]'),('8470','hsa:8470','[SORBS2, ARGBP2, PRO0618]','[(RefSeq) sorbin and SH3 domain containing 2]','[sorbin and SH3 domain-containing protein 2]','[Homo sapiens (human)]',NULL,'[]','[]','04990 Domain-containing proteins not elsewhere classified [BR:hsa04990]; Other SH2 and SH3-containing proteins; ','[Pfam: SH3_1 SH3_9 SH3_2 Sorb]'),('30061','hsa:30061','[SLC40A1, FPN1, HFE4, IREG1, MST079, MSTP079, MTP1, SLC11A3]','[(RefSeq) solute carrier family 40 member 1]','[solute carrier family 40 (iron-regulated transporter), member 1]','[Homo sapiens (human)]','[Ferroptosis, Mineral absorption]','[]','[]','04216 Ferroptosis; 04978 Mineral absorption; 02000 Transporters [BR:hsa02000]; SLC40: Basolateral iron transporter; ','[Pfam: FPN1]'),('10050','hsa:10050','[SLC17A4, KAIA2138]','[(RefSeq) solute carrier family 17 member 4]','[MFS transporter, ACS family, solute carrier family 17 (sodium-dependent inorganic phosphate cotransporter), member 1/2/3/4]','[Homo sapiens (human)]',NULL,'[]','[]','02000 Transporters [BR:hsa02000]; SLC17: Vesicular glutamate transporter; Anion:cation symporter (ACS) family [TC:2.A.1.14]; ','[Pfam: MFS_1 SOSSC]'),('143686','hsa:143686','[SESN3, SEST3]','[(RefSeq) sestrin 3]','[sestrin 1/3]','[Homo sapiens (human)]','[p53 signaling pathway, Longevity regulating pathway]','[]','[]','04115 p53 signaling pathway; 04211 Longevity regulating pathway; ','[Pfam: PA26 CMD PEX11]'),('255488','hsa:255488','[RNF144B, IBRDC2, PIR2, bA528A10.3, p53RFP]','[(RefSeq) ring finger protein 144B]','[E3 ubiquitin-protein ligase RNF144 [EC:2.3.2.31]]','[Homo sapiens (human)]',NULL,'[]','[]','04121 Ubiquitin system [BR:hsa04121]; 2.3.2.31  RBR-type E3 ubiquitin transferase; RBR proteins; ','[Pfam: IBR zf-RING_2]'),('3248','hsa:3248','[HPGD, 15-PGDH, PGDH, PGDH1, PHOAR1, SDR36C1]','[(RefSeq) 15-hydroxyprostaglandin dehydrogenase]','[15-hydroxyprostaglandin dehydrogenase (NAD) [EC:1.1.1.141]]','[Homo sapiens (human)]','[Transcriptional misregulation in cancer]','[nt06240  Transcription]','[N00120  MLL-ENL fusion to transcriptional activation]','05202 Transcriptional misregulation in cancer; 04147 Exosome [BR:hsa04147]; 1.1.1.141  15-hydroxyprostaglandin dehydrogenase (NAD+); Exosomal proteins of other body fluids (saliva and urine); ','[Pfam: adh_short adh_short_C2 KR Epimerase GDP_Man_Dehyd Lectin_leg-like 3Beta_HSD]'),('259266','hsa:259266','[ASPM, ASP, Calmbp1, MCPH5]','[(RefSeq) assembly factor for spindle microtubules]','[abnormal spindle-like microcephaly-associated protein]','[Homo sapiens (human)]',NULL,'[]','[]','03036 Chromosome and associated proteins [BR:hsa03036]; Other centriole associated proteins; ','[Pfam: IQ ASH CH CAMSAP_CH CH_2]'),('27035','hsa:27035','[NOX1, GP91-2, MOX1, NOH-1, NOH-1L, NOH1]','[(RefSeq) NADPH oxidase 1]','[NADPH oxidase 1]','[Homo sapiens (human)]','[Osteoclast differentiation, AGE-RAGE signaling pathway in diabetic complications, Alzheimer disease, Pathways of neurodegeneration - multiple diseases, Chemical carcinogenesis - reactive oxygen species, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[]','[]','04380 Osteoclast differentiation; 05208 Chemical carcinogenesis - reactive oxygen species; 05010 Alzheimer disease; 05022 Pathways of neurodegeneration - multiple diseases; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 04933 AGE-RAGE signaling pathway in diabetic complications; 04131 Membrane trafficking [BR:hsa04131]; NADPH oxidases; ','[Pfam: NAD_binding_6 Ferric_reduct FAD_binding_8 NAD_binding_1 FAD_binding_6]'),('10257','hsa:10257','[ABCC4, MOAT-B, MOATB, MRP4]','[(RefSeq) ATP binding cassette subfamily C member 4]','[ATP-binding cassette, subfamily C (CFTR/MRP), member 4]','[Homo sapiens (human)]','[Antifolate resistance, ABC transporters, cAMP signaling pathway, Bile secretion]','[]','[]','02010 ABC transporters; 04024 cAMP signaling pathway; 04976 Bile secretion; 01523 Antifolate resistance; 02000 Transporters [BR:hsa02000]; ABCC1, 2, 3, 4, 5, 6, 10, 11, 12, 13 subgroups; ','[Pfam: ABC_membrane ABC_tran SMC_N MMR_HSR1 AAA_29 AAA_16 RsgA_GTPase AAA_21 Dynamin_N AAA_30 AAA_23 AAA_7 DUF87 ABC_ATPase]'),('85417','hsa:85417','[CCNB3, CYCB3]','[(RefSeq) cyclin B3]','[G2/mitotic-specific cyclin-B3]','[Homo sapiens (human)]','[FoxO signaling pathway, Cell cycle, Cellular senescence, Progesterone-mediated oocyte maturation, Human immunodeficiency virus 1 infection]','[c(\"nt06130  Cell cycle (viruses)\", \"nt06160  Human T-cell leukemia virus 1 (HTLV-1)\", \"nt06161  Human immunodeficiency virus type 1 (HIV-1)\", \"nt06230  Cell cycle\")]','[N00455  CDC25-Cell cycle G2/M, N00456  HIV Vpr to CDC25-cell cycle G2M, N00457  HIV Vpr to cell cycle G2M, N00459  WEE1-Cell cycle G2/M, N00460  HIV Vpr to WEE1-cell cycle G2M, N00499  ATR-p21-Cell cycle G2/M]','04068 FoxO signaling pathway; 04110 Cell cycle; 04218 Cellular senescence; 04914 Progesterone-mediated oocyte maturation; 05170 Human immunodeficiency virus 1 infection; 03032 DNA replication proteins [BR:hsa03032]; 03036 Chromosome and associated proteins [BR:hsa03036]; Prevention of re-replication factors; Cyclins; ','[Pfam: Cyclin_N Cyclin_C]'),('7166','hsa:7166','[TPH1, TPRH, TRPH, TPH]','[(RefSeq) tryptophan hydroxylase 1]','[tryptophan 5-monooxygenase [EC:1.14.16.4]]','[Homo sapiens (human)]','[Tryptophan metabolism, Folate biosynthesis, Metabolic pathways, Serotonergic synapse]','[]','[]','00380 Tryptophan metabolism; 00790 Folate biosynthesis; 04726 Serotonergic synapse; 1.14.16.4  tryptophan 5-monooxygenase; ','[Pfam: Biopterin_H ACT ACT_4]'),('58985','hsa:58985','[IL22RA1, CRF2-9, IL22R, IL22R1]','[(RefSeq) interleukin 22 receptor subunit alpha 1]','[interleukin 22 receptor alpha 1]','[Homo sapiens (human)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, JAK-STAT signaling pathway]','[nt06150  Cytokine-cytokine receptor interaction (viruses)]','[N01273  IL20/24 to IL20R/22R signaling, N01274  YLDV 134R to IL20R/22R signaling]','04630 JAK-STAT signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04050 Cytokine receptors [BR:hsa04050]; IL10/28 receptor family; ','[Pfam: Tissue_fac Interfer-bind]'),('4582','hsa:4582','[MUC1, ADMCKD, ADMCKD1, ADTKD2, CA_15-3, CD227, Ca15-3, EMA, H23AG, KL-6, MAM6, MCD, MCKD, MCKD1, MUC-1, MUC-1/SEC, MUC-1/X, MUC1/ZD, PEM, PEMT, PUM]','[(RefSeq) mucin 1, cell surface associated]','[mucin-1]','[Homo sapiens (human)]',NULL,'[]','[]','04131 Membrane trafficking [BR:hsa04131]; 04147 Exosome [BR:hsa04147]; 04090 CD molecules [BR:hsa04090]; Membrane bound mucins; Exosomal proteins of breast milk; Exosomal proteins of breast cancer cells; Exosomal proteins of bladder cancer cells; Proteins; ','[Pfam: SEA Stevor Collectrin]'),('4248','hsa:4248','[MGAT3, GNT-III, GNT3]','[(RefSeq) beta-1,4-mannosyl-glycoprotein 4-beta-N-acetylglucosaminyltransferase]','[beta-1,4-mannosyl-glycoprotein beta-1,4-N-acetylglucosaminyltransferase [EC:2.4.1.144]]','[Homo sapiens (human)]','[N-Glycan biosynthesis, Metabolic pathways]','[]','[]','00510 N-Glycan biosynthesis; 01003 Glycosyltransferases [BR:hsa01003]; 2.4.1.144  beta-1,4-mannosyl-glycoprotein 4-beta-N-acetylglucosaminyltransferase; N-Glycan; ','[Pfam: Glyco_transf_17 EGF_2]'),('2683','hsa:2683','[B4GALT1, B4GAL-T1, CDG2D, GGTB2, GT1, GTB, beta4Gal-T1]','[(RefSeq) beta-1,4-galactosyltransferase 1]','[beta-1,4-galactosyltransferase 1 [EC:2.4.1.22 2.4.1.90 2.4.1.38 2.4.1.-]]','[Homo sapiens (human)]','[Galactose metabolism, N-Glycan biosynthesis, Various types of N-glycan biosynthesis, Other types of O-glycan biosynthesis, Mannose type O-glycan biosynthesis, Glycosaminoglycan biosynthesis - keratan sulfate, Glycosphingolipid biosynthesis - lacto and neolacto series, Metabolic pathways]','[nt06015  N-Glycan biosynthesis]','[N00820  N-Glycan biosynthesis]','00052 Galactose metabolism; 00510 N-Glycan biosynthesis; 00513 Various types of N-glycan biosynthesis; 00515 Mannose type O-glycan biosynthesis; 00514 Other types of O-glycan biosynthesis; 00533 Glycosaminoglycan biosynthesis - keratan sulfate; 00601 Glycosphingolipid biosynthesis - lacto and neolacto series; 01003 Glycosyltransferases [BR:hsa01003]; 04131 Membrane trafficking [BR:hsa04131]; 04147 Exosome [BR:hsa04147]; 2.4.1.22  lactose synthase; 2.4.1.38  beta-N-acetylglucosaminylglycopeptide beta-1,4-galactosyltransferase; 2.4.1.90  N-acetyllactosamine synthase; 2.4.1.-; Type 2 chain/Poly-N-acetyllactosamine; Acrosomal exocytosis proteins; Exosomal proteins of breast milk; ','[Pfam: Glyco_transf_7N Glyco_transf_7C Glyco_tranf_2_2]'),('6480','hsa:6480','[ST6GAL1, CDw75, SIAT1, ST6GalI, ST6N]','[(RefSeq) ST6 beta-galactoside alpha-2,6-sialyltransferase 1]','[beta-galactoside alpha-2,6-sialyltransferase (sialyltransferase 1) [EC:2.4.3.1]]','[Homo sapiens (human)]','[N-Glycan biosynthesis, Other types of O-glycan biosynthesis, Metabolic pathways]','[nt06015  N-Glycan biosynthesis]','[N00820  N-Glycan biosynthesis]','00510 N-Glycan biosynthesis; 00514 Other types of O-glycan biosynthesis; 01003 Glycosyltransferases [BR:hsa01003]; 2.4.3.1  beta-galactoside alpha-(2,6)-sialyltransferase; Sialyltransferase; ','[Pfam: Glyco_transf_29]'),('283871','hsa:283871','[PGP, AUM, G3PP, PGPase]','[(RefSeq) phosphoglycolate phosphatase]','[phosphoglycolate phosphatase [EC:3.1.3.18 3.1.3.48]]','[Homo sapiens (human)]','[Glyoxylate and dicarboxylate metabolism, Metabolic pathways, Carbon metabolism]','[]','[]','00630 Glyoxylate and dicarboxylate metabolism; 01009 Protein phosphatases and associated proteins [BR:hsa01009]; 3.1.3.18  phosphoglycolate phosphatase; 3.1.3.48  protein-tyrosine-phosphatase; Other HAD phosphatases; ','[Pfam: Hydrolase_6 Hydrolase_like Hydrolase HAD_2 PGP_phosphatase]'),('10560','hsa:10560','[SLC19A2, TC1, THMD1, THT1, THTR1, TRMA]','[(RefSeq) solute carrier family 19 member 2]','[solute carrier family 19 (thiamine transporter), member 2/3]','[Homo sapiens (human)]','[Vitamin digestion and absorption]','[]','[]','04977 Vitamin digestion and absorption; 02000 Transporters [BR:hsa02000]; SLC19: Folate/thiamine transporter; ','[Pfam: Folate_carrier MFS_1]'),('80704','hsa:80704','[SLC19A3, BBGD, THMD2, THTR2, thTr-2]','[(RefSeq) solute carrier family 19 member 3]','[solute carrier family 19 (thiamine transporter), member 2/3]','[Homo sapiens (human)]','[Vitamin digestion and absorption]','[]','[]','04977 Vitamin digestion and absorption; 02000 Transporters [BR:hsa02000]; SLC19: Folate/thiamine transporter; ','[Pfam: Folate_carrier MFS_1 Cadherin_C_2 DUF2973]'),('6550','hsa:6550','[SLC9A3, DIAR8, NHE-3, NHE3]','[(RefSeq) solute carrier family 9 member A3]','[solute carrier family 9 (sodium/hydrogen exchanger), member 3]','[Homo sapiens (human)]','[Proximal tubule bicarbonate reclamation, Protein digestion and absorption, Bile secretion, Mineral absorption]','[]','[]','04976 Bile secretion; 04974 Protein digestion and absorption; 04978 Mineral absorption; 04964 Proximal tubule bicarbonate reclamation; 02000 Transporters [BR:hsa02000]; SLC9: Na+/H+ exchanger; ','[Pfam: Na_H_Exchanger NEXCaM_BD]'),('5970','hsa:5970','[RELA, CMCU, NFKB3, p65]','[(RefSeq) RELA proto-oncogene, NF-kB subunit]','[transcription factor p65]','[Homo sapiens (human)]','[Antifolate resistance, MAPK signaling pathway, Ras signaling pathway, cAMP signaling pathway, Chemokine signaling pathway, NF-kappa B signaling pathway, HIF-1 signaling pathway, Sphingolipid signaling pathway, Mitophagy - animal, PI3K-Akt signaling pathway, Apoptosis, Longevity regulating pathway, Cellular senescence, Osteoclast differentiation, Neutrophil extracellular trap formation, Toll-like receptor signaling pathway, NOD-like receptor signaling pathway, RIG-I-like receptor signaling pathway, Cytosolic DNA-sensing pathway, C-type lectin receptor signaling pathway, IL-17 signaling pathway, Th1 and Th2 cell differentiation, Th17 cell differentiation, T cell receptor signaling pathway, B cell receptor signaling pathway, TNF signaling pathway, Neurotrophin signaling pathway, Prolactin signaling pathway, Adipocytokine signaling pathway, Relaxin signaling pathway, Insulin resistance, Non-alcoholic fatty liver disease, AGE-RAGE signaling pathway in diabetic complications, Alcoholic liver disease, Alzheimer disease, Pathways of neurodegeneration - multiple diseases, Cocaine addiction, Epithelial cell signaling in Helicobacter pylori infection, Pathogenic Escherichia coli infection, Shigellosis, Salmonella infection, Pertussis, Legionellosis, Yersinia infection, Leishmaniasis, Chagas disease, Toxoplasmosis, Amoebiasis, Tuberculosis, Hepatitis C, Hepatitis B, Measles, Human cytomegalovirus infection, Influenza A, Human papillomavirus infection, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Herpes simplex virus 1 infection, Epstein-Barr virus infection, Human immunodeficiency virus 1 infection, Coronavirus disease - COVID-19, Pathways in cancer, Transcriptional misregulation in cancer, Viral carcinogenesis, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species, Pancreatic cancer, Prostate cancer, Chronic myeloid leukemia, Acute myeloid leukemia, Small cell lung cancer, PD-L1 expression and PD-1 checkpoint pathway in cancer, Inflammatory bowel disease, Diabetic cardiomyopathy, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[c(\"nt06114  PI3K signaling (viruses)\", \"nt06121  TLR signaling (viruses and bacteria)\", \"nt06123  TNF signaling (viruses and bacteria)\", \"nt06124  Chemokine signaling (viruses)\", \"nt06133  RLR signaling (viruses)\", \"nt06139  NLR signaling (viruses and bacteria)\", \"nt06150  Cytokine-cytokine receptor interaction (viruses)\", \"nt06160  Human T-cell leukemia virus 1 (HTLV-1)\", \"nt06161  Human immunodeficiency virus type 1 (HIV-1)\", \"nt06162  Hepatitis B virus (HBV)\", \"nt06163  Hepatitis C virus (HCV)\", \n\"nt06164  Kaposi sarcoma-associated herpesvirus (KSHV)\", \"nt06165  Epstein-Barr virus (EBV)\", \"nt06167  Human cytomegalovirus (HCMV)\", \"nt06168  Herpes simplex virus 1 (HSV-1)\", \"nt06169  Measles virus (MV)\", \"nt06170  Influenza A virus (IAV)\", \"nt06171  Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)\", \"nt06214  PI3K signaling\", \"nt06223  TNF signaling\", \"nt06224  CXCR signaling\", \"nt06240  Transcription\", \"nt06263  Hepatocellular carcinoma\", \"nt06417  AGE-RAGE signaling\", \"nt06460  Alzheimer disease\"\n)]','[N00140  FUS-DDIT3 fusion to NFKB-mediated transcription, N00151  TNF-NFKB signaling pathway, N00171  KSHV vFLIP to NFKB signaling pathway, N00173  KSHV K15 to TNF-NFKB signaling pathway, N00174  KSHV vFLIP to TNF-NFKB signaling pathway, N00178  KSHV vGPCR to GNB/G-PI3K-JNK signaling pathway, N00179  KSHV K1 to PI3K-NFKB signaling pathway, N00182  IGF-IGFR-PI3K-NFKB signaling pathway, N00265  EBV LMP1 to NFKB signaling pathway, N00269  HCV core to TNF-NFKB signaling pathway, N00389  HCMV IE1-72 to transcription, N00390  EGF-EGFR-PI3K-NFKB signaling pathway, N00391  HCMV gB to EGFR-PI3K-NFKB signaling pathway, N00398  HCMV IE2-86 to TNF-NFKB signaling pathway, N00399  CCR2-GNB/G-PI3K-NFKB signaling pathway, N00400  HCMV US28 to GNB/G-PI3K-NFKB signaling pathway, N00435  TLR2/4-NFKB signaling pathway, N00436  HIV Tat to TLR2/4-NFKB signaling pathway, N00437  HIV Vpu to TLR2/4-NFKB signaling pathway, N00441  HIV gp120 to TNF-NFKB signaling pathway, N00442  HIV Nef to TNF-NFKB signaling pathway, N00443  HIV Vpr/Nef/Tat to TNF-NFKB signaling pathway, N00501  HTLV-1 Tax to EGFR-PI3K-NFKB signaling pathway, N00504  HTLV-1 Tax to NFKB signaling pathway, N00550  HBV HBeAg to TLR2/4-NFKB signaling pathway, N00559  LIGHT-HVEM-NFKB signaling pathway, N00560  HSV gD to HVEM-NFKB signaling pathway, N00563  TLR3-NFKB signaling pathway, N00688  RIG-I-NFKB signaling pathway, N00689  MV V/P/C to RIG-I-NFKB signaling pathway, N00730  IAV NS1 to RIG-I-like receptor signaling pathway, N00929  Escherichia NleC to TNF-NFKB signaling pathway, N00932  Escherichia NleH1 to TNF-NFKB signaling pathway, N00940  NOD-NFKB signaling pathway, N00943  Shigella IpaH4.5 to TNF-NFKB signaling pathway, N00994  AGE-RAGE signaling pathway, N00996  Mutation-caused aberrant Abeta to AGE-RAGE signaling pathway, N01112  Salmonella SopE/E2 to NOD-NFKB signaling pathway, N01115  Salmonella SseL to TNF-NFKB signaling pathway, N01118  Salmonella SpvD to TNF-NFKB signaling pathway, N01122  Salmonella PipA/GogA/GtgA to TNF-NFKB signaling pathway, N01306  AngII-AT1R-NOX2 signaling pathway, N01307  SARS-CoV-2 S to AngII-AT1R-NOX2 signaling pathway, N01339  NNK/NNN to PI3K signaling pathway, N01410  Metals to NFKB signaling pathway, N01411  Metals to NFKB signaling pathway]','04010 MAPK signaling pathway; 04014 Ras signaling pathway; 04064 NF-kappa B signaling pathway; 04668 TNF signaling pathway; 04066 HIF-1 signaling pathway; 04071 Sphingolipid signaling pathway; 04024 cAMP signaling pathway; 04151 PI3K-Akt signaling pathway; 04137 Mitophagy - animal; 04210 Apoptosis; 04218 Cellular senescence; 04613 Neutrophil extracellular trap formation; 04620 Toll-like receptor signaling pathway; 04621 NOD-like receptor signaling pathway; 04622 RIG-I-like receptor signaling pathway; 04623 Cytosolic DNA-sensing pathway; 04625 C-type lectin receptor signaling pathway; 04660 T cell receptor signaling pathway; 04658 Th1 and Th2 cell differentiation; 04659 Th17 cell differentiation; 04657 IL-17 signaling pathway; 04662 B cell receptor signaling pathway; 04062 Chemokine signaling pathway; 04920 Adipocytokine signaling pathway; 04917 Prolactin signaling pathway; 04926 Relaxin signaling pathway; 04722 Neurotrophin signaling pathway; 04380 Osteoclast differentiation; 04211 Longevity regulating pathway; 05200 Pathways in cancer; 05202 Transcriptional misregulation in cancer; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 05203 Viral carcinogenesis; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05212 Pancreatic cancer; 05221 Acute myeloid leukemia; 05220 Chronic myeloid leukemia; 05215 Prostate cancer; 05222 Small cell lung cancer; 05166 Human T-cell leukemia virus 1 infection; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05160 Hepatitis C; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05165 Human papillomavirus infection; 05120 Epithelial cell signaling in Helicobacter pylori infection; 05130 Pathogenic Escherichia coli infection; 05132 Salmonella infection; 05131 Shigellosis; 05135 Yersinia infection; 05133 Pertussis; 05134 Legionellosis; 05152 Tuberculosis; 05146 Amoebiasis; 05145 Toxoplasmosis; 05140 Leishmaniasis; 05142 Chagas disease; 05321 Inflammatory bowel disease; 05010 Alzheimer disease; 05022 Pathways of neurodegeneration - multiple diseases; 05030 Cocaine addiction; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 05415 Diabetic cardiomyopathy; 04936 Alcoholic liver disease; 04932 Non-alcoholic fatty liver disease; 04931 Insulin resistance; 04933 AGE-RAGE signaling pathway in diabetic complications; 01523 Antifolate resistance; 03000 Transcription factors [BR:hsa03000]; RHR (Rel homology region) Rel/Ankyrin; ','[Pfam: RHD_DNA_bind RHD_dimer Fimbrial_CS1]'),('8915','hsa:8915','[BCL10, CARMEN, CIPER, CLAP, IMD37, c-E10, mE10]','[(RefSeq) BCL10 immune signaling adaptor]','[B-cell CLL/lymphoma 10]','[Homo sapiens (human)]','[NF-kappa B signaling pathway, C-type lectin receptor signaling pathway, T cell receptor signaling pathway, B cell receptor signaling pathway, Shigellosis, Tuberculosis]','[]','[]','04064 NF-kappa B signaling pathway; 04625 C-type lectin receptor signaling pathway; 04660 T cell receptor signaling pathway; 04662 B cell receptor signaling pathway; 05131 Shigellosis; 05152 Tuberculosis; ','[Pfam: CARD Flu_B_NS1]'),('1811','hsa:1811','[SLC26A3, CLD, DRA]','[(RefSeq) solute carrier family 26 member 3]','[solute carrier family 26 (chloride anion exchanger), member 3]','[Homo sapiens (human)]','[Pancreatic secretion, Mineral absorption]','[]','[]','04972 Pancreatic secretion; 04978 Mineral absorption; 02000 Transporters [BR:hsa02000]; SLC26: Multifunctional anion exchanger; ','[Pfam: Sulfate_transp STAS]'),('11282','hsa:11282','[MGAT4B, GNT-IV, GNT-IVB]','[(RefSeq) alpha-1,3-mannosyl-glycoprotein 4-beta-N-acetylglucosaminyltransferase B]','[alpha-1,3-mannosylglycoprotein beta-1,4-N-acetylglucosaminyltransferase A/B [EC:2.4.1.145]]','[Homo sapiens (human)]','[N-Glycan biosynthesis, Various types of N-glycan biosynthesis, Metabolic pathways]','[]','[]','00510 N-Glycan biosynthesis; 00513 Various types of N-glycan biosynthesis; 01003 Glycosyltransferases [BR:hsa01003]; 2.4.1.145  alpha-1,3-mannosyl-glycoprotein 4-beta-N-acetylglucosaminyltransferase; N-Glycan; ','[Pfam: Glyco_transf_54]'),('11320','hsa:11320','[MGAT4A, GNT-IV, GNT-IVA, GnT-4a]','[(RefSeq) alpha-1,3-mannosyl-glycoprotein 4-beta-N-acetylglucosaminyltransferase A]','[alpha-1,3-mannosylglycoprotein beta-1,4-N-acetylglucosaminyltransferase A/B [EC:2.4.1.145]]','[Homo sapiens (human)]','[N-Glycan biosynthesis, Various types of N-glycan biosynthesis, Metabolic pathways]','[]','[]','00510 N-Glycan biosynthesis; 00513 Various types of N-glycan biosynthesis; 01003 Glycosyltransferases [BR:hsa01003]; 2.4.1.145  alpha-1,3-mannosyl-glycoprotein 4-beta-N-acetylglucosaminyltransferase; N-Glycan; ','[Pfam: Glyco_transf_54]'),('116379','hsa:116379','[IL22RA2, CRF2-10, CRF2-S1, CRF2X, IL-22BP, IL-22R-alpha-2, IL-22RA2, ZCYTOR16]','[(RefSeq) interleukin 22 receptor subunit alpha 2]','[interleukin 22 receptor alpha 2]','[Homo sapiens (human)]','[JAK-STAT signaling pathway]','[]','[]','04630 JAK-STAT signaling pathway; 04050 Cytokine receptors [BR:hsa04050]; IL10/28 receptor family; ','[Pfam: Tissue_fac Interfer-bind Receptor_2B4]'),('7421','hsa:7421','[VDR, NR1I1, PPP1R163]','[(RefSeq) vitamin D receptor]','[vitamin D3 receptor]','[Homo sapiens (human)]','[Parathyroid hormone synthesis, secretion and action, Endocrine and other factor-regulated calcium reabsorption, Mineral absorption, Tuberculosis, Chemical carcinogenesis - receptor activation]','[]','[]','04928 Parathyroid hormone synthesis, secretion and action; 04978 Mineral absorption; 04961 Endocrine and other factor-regulated calcium reabsorption; 05207 Chemical carcinogenesis - receptor activation; 05152 Tuberculosis; 01009 Protein phosphatases and associated proteins [BR:hsa01009]; 03000 Transcription factors [BR:hsa03000]; 03310 Nuclear receptors [BR:hsa03310]; PP1-interacting proteins (PIPs); Cys4 thyroid hormone receptor-like; 1I. Vitamin D3 like receptor; ','[Pfam: zf-C4 Hormone_recep zf-RRPl_C4]'),('6822','hsa:6822','[SULT2A1, DHEA-ST, DHEA-ST8, DHEAS, HST, ST2, ST2A1, ST2A3, STD, SULT2A3, hSTa]','[(RefSeq) sulfotransferase family 2A member 1]','[bile-salt sulfotransferase [EC:2.8.2.14]]','[Homo sapiens (human)]','[Metabolism of xenobiotics by cytochrome P450, Bile secretion, Chemical carcinogenesis - DNA adducts]','[]','[]','00980 Metabolism of xenobiotics by cytochrome P450; 04976 Bile secretion; 05204 Chemical carcinogenesis - DNA adducts; 2.8.2.14  bile-salt sulfotransferase; ','[Pfam: Sulfotransfer_1 Sulfotransfer_3 SIX1_SD]'),('3934','hsa:3934','[LCN2, 24p3, MSFI, NGAL, p25]','[(RefSeq) lipocalin 2]','[lipocalin 2]','[Homo sapiens (human)]','[IL-17 signaling pathway]','[]','[]','04657 IL-17 signaling pathway; ','[Pfam: Lipocalin DUF1419 Lipocalin_2]'),('960','hsa:960','[CD44, CDW44, CSPG8, ECMR-III, HCELL, HUTCH-I, IN, LHR, MC56, MDU2, MDU3, MIC4, Pgp1]','[(RefSeq) CD44 molecule (Indian blood group)]','[CD44 antigen]','[Homo sapiens (human)]','[ECM-receptor interaction, Hematopoietic cell lineage, Shigellosis, Epstein-Barr virus infection, Proteoglycans in cancer, MicroRNAs in cancer]','[]','[]','04512 ECM-receptor interaction; 04640 Hematopoietic cell lineage; 05206 MicroRNAs in cancer; 05205 Proteoglycans in cancer; 05169 Epstein-Barr virus infection; 05131 Shigellosis; 04147 Exosome [BR:hsa04147]; 04090 CD molecules [BR:hsa04090]; 00535 Proteoglycans [BR:hsa00535]; 00536 Glycosaminoglycan binding proteins [BR:hsa00536]; Exosomal proteins of haemopoietic cells  (B-cell, T-cell, DC-cell, reticulocyte, and mast cell); Exosomal proteins of other body fluids (saliva and urine); Exosomal proteins of colorectal cancer cells; Exosomal proteins of bladder cancer cells; Proteins; Others; Cell surface receptors; ','[Pfam: Xlink]'),('3929','hsa:3929','[LBP, BPIFD2]','[(RefSeq) lipopolysaccharide binding protein]','[lipopolysaccharide-binding protein]','[Homo sapiens (human)]','[NF-kappa B signaling pathway, Toll-like receptor signaling pathway, Alcoholic liver disease, Tuberculosis, Lipid and atherosclerosis]','[]','[]','04064 NF-kappa B signaling pathway; 04620 Toll-like receptor signaling pathway; 05152 Tuberculosis; 05417 Lipid and atherosclerosis; 04936 Alcoholic liver disease; 04147 Exosome [BR:hsa04147]; Exosomal proteins of other cancer cells; ','[Pfam: LBP_BPI_CETP_C LBP_BPI_CETP PHA_gran_rgn]'),('12738','mmu:12738','[Cldn2, AL022813]','[(RefSeq) claudin 2]','[claudin]','[Mus musculus (house mouse)]','[Cell adhesion molecules, Tight junction, Leukocyte transendothelial migration, Hepatitis C]','[]','[]','04514 Cell adhesion molecules; 04530 Tight junction; 04670 Leukocyte transendothelial migration; 05160 Hepatitis C; 03037 Cilium and associated proteins [BR:mmu03037]; 04147 Exosome [BR:mmu04147]; Stereociliary proteins; Exosomal proteins of ovarian cancer cells; Exosomal proteins of colorectal cancer cells; ','[Pfam: PMP22_Claudin Claudin_2 Amastin]'),('20848','mmu:20848','[Stat3, 1110034C02Rik, AW109958, Aprf]','[(RefSeq) signal transducer and activator of transcription 3]','[signal transducer and activator of transcription 3]','[Mus musculus (house mouse)]','[EGFR tyrosine kinase inhibitor resistance, Chemokine signaling pathway, HIF-1 signaling pathway, FoxO signaling pathway, Necroptosis, Signaling pathways regulating pluripotency of stem cells, JAK-STAT signaling pathway, Th17 cell differentiation, Prolactin signaling pathway, Adipocytokine signaling pathway, Insulin resistance, AGE-RAGE signaling pathway in diabetic complications, Growth hormone synthesis, secretion and action, Toxoplasmosis, Hepatitis C, Hepatitis B, Measles, Human cytomegalovirus infection, Kaposi sarcoma-associated herpesvirus infection, Epstein-Barr virus infection, Coronavirus disease - COVID-19, Pathways in cancer, Viral carcinogenesis, Proteoglycans in cancer, MicroRNAs in cancer, Chemical carcinogenesis - receptor activation, Pancreatic cancer, Acute myeloid leukemia, Non-small cell lung cancer, PD-L1 expression and PD-1 checkpoint pathway in cancer, Inflammatory bowel disease, Lipid and atherosclerosis]','[]','[]','04630 JAK-STAT signaling pathway; 04066 HIF-1 signaling pathway; 04068 FoxO signaling pathway; 04217 Necroptosis; 04550 Signaling pathways regulating pluripotency of stem cells; 04659 Th17 cell differentiation; 04062 Chemokine signaling pathway; 04920 Adipocytokine signaling pathway; 04917 Prolactin signaling pathway; 04935 Growth hormone synthesis, secretion and action; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05205 Proteoglycans in cancer; 05207 Chemical carcinogenesis - receptor activation; 05203 Viral carcinogenesis; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05212 Pancreatic cancer; 05221 Acute myeloid leukemia; 05223 Non-small cell lung cancer; 05161 Hepatitis B; 05160 Hepatitis C; 05171 Coronavirus disease - COVID-19; 05162 Measles; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05145 Toxoplasmosis; 05321 Inflammatory bowel disease; 05417 Lipid and atherosclerosis; 04931 Insulin resistance; 04933 AGE-RAGE signaling pathway in diabetic complications; 01521 EGFR tyrosine kinase inhibitor resistance; 03000 Transcription factors [BR:mmu03000]; STAT; ','[Pfam: STAT_alpha STAT_bind STAT_int SH2 E2R135 Syntaxin-6_N Fib_alpha]'),('79221','mmu:79221','[Hdac9, AV022454, D030072B18Rik, HD7B, HD9, HDRP, Hdac7b, Mitr, mKIAA0744]','[(RefSeq) histone deacetylase 9]','[histone deacetylase 9 [EC:3.5.1.98]]','[Mus musculus (house mouse)]','[Neutrophil extracellular trap formation, Alcoholism, Viral carcinogenesis]','[]','[]','04613 Neutrophil extracellular trap formation; 05203 Viral carcinogenesis; 05034 Alcoholism; 03036 Chromosome and associated proteins [BR:mmu03036]; 3.5.1.98  histone deacetylase; Class II HDACs; ','[Pfam: HDAC4_Gln]'),('67800','mmu:67800','[Dgat2, 0610010B06Rik, ARAT, DGAT-2]','[(RefSeq) diacylglycerol O-acyltransferase 2]','[diacylglycerol O-acyltransferase 2 [EC:2.3.1.20 2.3.1.75]]','[Mus musculus (house mouse)]','[Glycerolipid metabolism, Metabolic pathways, Fat digestion and absorption]','[]','[]','00561 Glycerolipid metabolism; 04975 Fat digestion and absorption; 2.3.1.20  diacylglycerol O-acyltransferase; 2.3.1.75  long-chain-alcohol O-fatty-acyltransferase; ','[Pfam: DAGAT Acyltransferase]'),('21872','mmu:21872','[Tjp1, ZO1]','[(RefSeq) tight junction protein 1]','[tight junction protein 1]','[Mus musculus (house mouse)]','[Adherens junction, Tight junction, Gap junction]','[]','[]','04520 Adherens junction; 04530 Tight junction; 04540 Gap junction; 04812 Cytoskeleton proteins [BR:mmu04812]; Others; ','[Pfam: PDZ ZU5 PDZ_6 PDZ_2 Peptidase_M50 Guanylate_kin SH3_2 DUF6288 SH3_1 RPOL_N]'),('16153','mmu:16153','[Il10, CSIF, If2a, Il-10]','[(RefSeq) interleukin 10]','[interleukin 10]','[Mus musculus (house mouse)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, FoxO signaling pathway, C-type lectin receptor signaling pathway, JAK-STAT signaling pathway, T cell receptor signaling pathway, Intestinal immune network for IgA production, Pertussis, Yersinia infection, Leishmaniasis, Chagas disease, African trypanosomiasis, Malaria, Toxoplasmosis, Amoebiasis, Staphylococcus aureus infection, Tuberculosis, Asthma, Autoimmune thyroid disease, Inflammatory bowel disease, Systemic lupus erythematosus, Allograft rejection]','[]','[]','04630 JAK-STAT signaling pathway; 04068 FoxO signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04625 C-type lectin receptor signaling pathway; 04660 T cell receptor signaling pathway; 04672 Intestinal immune network for IgA production; 05135 Yersinia infection; 05133 Pertussis; 05150 Staphylococcus aureus infection; 05152 Tuberculosis; 05146 Amoebiasis; 05144 Malaria; 05145 Toxoplasmosis; 05140 Leishmaniasis; 05142 Chagas disease; 05143 African trypanosomiasis; 05310 Asthma; 05322 Systemic lupus erythematosus; 05320 Autoimmune thyroid disease; 05321 Inflammatory bowel disease; 05330 Allograft rejection; 04052 Cytokines and growth factors [BR:mmu04052]; 00536 Glycosaminoglycan binding proteins [BR:mmu00536]; Interleukins; Cytokines; ','[Pfam: IL10 IL22]'),('16176','mmu:16176','[Il1b, IL-1beta, Il-1b]','[(RefSeq) interleukin 1 beta]','[interleukin 1 beta]','[Mus musculus (house mouse)]','[Antifolate resistance, MAPK signaling pathway, Cytokine-cytokine receptor interaction, NF-kappa B signaling pathway, Necroptosis, Osteoclast differentiation, Toll-like receptor signaling pathway, NOD-like receptor signaling pathway, Cytosolic DNA-sensing pathway, C-type lectin receptor signaling pathway, Hematopoietic cell lineage, IL-17 signaling pathway, Th17 cell differentiation, TNF signaling pathway, Inflammatory mediator regulation of TRP channels, Non-alcoholic fatty liver disease, AGE-RAGE signaling pathway in diabetic complications, Alcoholic liver disease, Type I diabetes mellitus, Alzheimer disease, Prion disease, Pathways of neurodegeneration - multiple diseases, Salmonella infection, Pertussis, Legionellosis, Yersinia infection, Leishmaniasis, Chagas disease, African trypanosomiasis, Malaria, Amoebiasis, Tuberculosis, Measles, Human cytomegalovirus infection, Influenza A, Herpes simplex virus 1 infection, Coronavirus disease - COVID-19, Inflammatory bowel disease, Rheumatoid arthritis, Graft-versus-host disease, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[]','[]','04010 MAPK signaling pathway; 04064 NF-kappa B signaling pathway; 04668 TNF signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04217 Necroptosis; 04640 Hematopoietic cell lineage; 04620 Toll-like receptor signaling pathway; 04621 NOD-like receptor signaling pathway; 04623 Cytosolic DNA-sensing pathway; 04625 C-type lectin receptor signaling pathway; 04659 Th17 cell differentiation; 04657 IL-17 signaling pathway; 04750 Inflammatory mediator regulation of TRP channels; 04380 Osteoclast differentiation; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05163 Human cytomegalovirus infection; 05132 Salmonella infection; 05135 Yersinia infection; 05133 Pertussis; 05134 Legionellosis; 05152 Tuberculosis; 05146 Amoebiasis; 05144 Malaria; 05140 Leishmaniasis; 05142 Chagas disease; 05143 African trypanosomiasis; 05323 Rheumatoid arthritis; 05321 Inflammatory bowel disease; 05332 Graft-versus-host disease; 05010 Alzheimer disease; 05020 Prion disease; 05022 Pathways of neurodegeneration - multiple diseases; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 04940 Type I diabetes mellitus; 04936 Alcoholic liver disease; 04932 Non-alcoholic fatty liver disease; 04933 AGE-RAGE signaling pathway in diabetic complications; 01523 Antifolate resistance; 04052 Cytokines and growth factors [BR:mmu04052]; 00536 Glycosaminoglycan binding proteins [BR:mmu00536]; Interleukins; Cytokines; ','[Pfam: IL1 IL1_propep ZinT]'),('16193','mmu:16193','[Il6, Il-6]','[(RefSeq) interleukin 6]','[interleukin 6]','[Mus musculus (house mouse)]','[EGFR tyrosine kinase inhibitor resistance, Antifolate resistance, Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, HIF-1 signaling pathway, FoxO signaling pathway, PI3K-Akt signaling pathway, Cellular senescence, Toll-like receptor signaling pathway, NOD-like receptor signaling pathway, Cytosolic DNA-sensing pathway, C-type lectin receptor signaling pathway, JAK-STAT signaling pathway, Hematopoietic cell lineage, IL-17 signaling pathway, Th17 cell differentiation, TNF signaling pathway, Intestinal immune network for IgA production, Insulin resistance, Non-alcoholic fatty liver disease, AGE-RAGE signaling pathway in diabetic complications, Alcoholic liver disease, Alzheimer disease, Prion disease, Pathways of neurodegeneration - multiple diseases, Salmonella infection, Pertussis, Legionellosis, Yersinia infection, Chagas disease, African trypanosomiasis, Malaria, Amoebiasis, Tuberculosis, Hepatitis B, Measles, Human cytomegalovirus infection, Influenza A, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Herpes simplex virus 1 infection, Epstein-Barr virus infection, Coronavirus disease - COVID-19, Pathways in cancer, Transcriptional misregulation in cancer, Inflammatory bowel disease, Rheumatoid arthritis, Graft-versus-host disease, Hypertrophic cardiomyopathy, Lipid and atherosclerosis]','[]','[]','04630 JAK-STAT signaling pathway; 04668 TNF signaling pathway; 04066 HIF-1 signaling pathway; 04068 FoxO signaling pathway; 04151 PI3K-Akt signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04218 Cellular senescence; 04640 Hematopoietic cell lineage; 04620 Toll-like receptor signaling pathway; 04621 NOD-like receptor signaling pathway; 04623 Cytosolic DNA-sensing pathway; 04625 C-type lectin receptor signaling pathway; 04659 Th17 cell differentiation; 04657 IL-17 signaling pathway; 04672 Intestinal immune network for IgA production; 05200 Pathways in cancer; 05202 Transcriptional misregulation in cancer; 05166 Human T-cell leukemia virus 1 infection; 05161 Hepatitis B; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05132 Salmonella infection; 05135 Yersinia infection; 05133 Pertussis; 05134 Legionellosis; 05152 Tuberculosis; 05146 Amoebiasis; 05144 Malaria; 05142 Chagas disease; 05143 African trypanosomiasis; 05323 Rheumatoid arthritis; 05321 Inflammatory bowel disease; 05332 Graft-versus-host disease; 05010 Alzheimer disease; 05020 Prion disease; 05022 Pathways of neurodegeneration - multiple diseases; 05417 Lipid and atherosclerosis; 05410 Hypertrophic cardiomyopathy; 04936 Alcoholic liver disease; 04932 Non-alcoholic fatty liver disease; 04931 Insulin resistance; 04933 AGE-RAGE signaling pathway in diabetic complications; 01521 EGFR tyrosine kinase inhibitor resistance; 01523 Antifolate resistance; 04052 Cytokines and growth factors [BR:mmu04052]; Interleukins; ','[Pfam: IL6 IL23]'),('21926','mmu:21926','[Tnf, DIF, TNF-a, TNF-alpha, TNFSF2, TNFalpha, Tnfa, Tnfsf1a, Tnlg1f]','[(RefSeq) tumor necrosis factor]','[tumor necrosis factor superfamily, member 2]','[Mus musculus (house mouse)]','[Antifolate resistance, MAPK signaling pathway, Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, NF-kappa B signaling pathway, Sphingolipid signaling pathway, mTOR signaling pathway, Apoptosis, Necroptosis, TGF-beta signaling pathway, Osteoclast differentiation, Antigen processing and presentation, Toll-like receptor signaling pathway, NOD-like receptor signaling pathway, RIG-I-like receptor signaling pathway, C-type lectin receptor signaling pathway, Hematopoietic cell lineage, Natural killer cell mediated cytotoxicity, IL-17 signaling pathway, T cell receptor signaling pathway, Fc epsilon RI signaling pathway, TNF signaling pathway, Adipocytokine signaling pathway, Type II diabetes mellitus, Insulin resistance, Non-alcoholic fatty liver disease, AGE-RAGE signaling pathway in diabetic complications, Alcoholic liver disease, Type I diabetes mellitus, Alzheimer disease, Amyotrophic lateral sclerosis, Prion disease, Pathways of neurodegeneration - multiple diseases, Salmonella infection, Pertussis, Legionellosis, Yersinia infection, Leishmaniasis, Chagas disease, African trypanosomiasis, Malaria, Toxoplasmosis, Amoebiasis, Tuberculosis, Hepatitis C, Hepatitis B, Human cytomegalovirus infection, Influenza A, Human papillomavirus infection, Human T-cell leukemia virus 1 infection, Herpes simplex virus 1 infection, Epstein-Barr virus infection, Human immunodeficiency virus 1 infection, Coronavirus disease - COVID-19, Proteoglycans in cancer, Asthma, Inflammatory bowel disease, Systemic lupus erythematosus, Rheumatoid arthritis, Allograft rejection, Graft-versus-host disease, Hypertrophic cardiomyopathy, Dilated cardiomyopathy, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[]','[]','04010 MAPK signaling pathway; 04350 TGF-beta signaling pathway; 04064 NF-kappa B signaling pathway; 04668 TNF signaling pathway; 04071 Sphingolipid signaling pathway; 04150 mTOR signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04210 Apoptosis; 04217 Necroptosis; 04640 Hematopoietic cell lineage; 04620 Toll-like receptor signaling pathway; 04621 NOD-like receptor signaling pathway; 04622 RIG-I-like receptor signaling pathway; 04625 C-type lectin receptor signaling pathway; 04650 Natural killer cell mediated cytotoxicity; 04612 Antigen processing and presentation; 04660 T cell receptor signaling pathway; 04657 IL-17 signaling pathway; 04664 Fc epsilon RI signaling pathway; 04920 Adipocytokine signaling pathway; 04380 Osteoclast differentiation; 05205 Proteoglycans in cancer; 05166 Human T-cell leukemia virus 1 infection; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05160 Hepatitis C; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05168 Herpes simplex virus 1 infection; 05163 Human cytomegalovirus infection; 05169 Epstein-Barr virus infection; 05165 Human papillomavirus infection; 05132 Salmonella infection; 05135 Yersinia infection; 05133 Pertussis; 05134 Legionellosis; 05152 Tuberculosis; 05146 Amoebiasis; 05144 Malaria; 05145 Toxoplasmosis; 05140 Leishmaniasis; 05142 Chagas disease; 05143 African trypanosomiasis; 05310 Asthma; 05322 Systemic lupus erythematosus; 05323 Rheumatoid arthritis; 05321 Inflammatory bowel disease; 05330 Allograft rejection; 05332 Graft-versus-host disease; 05010 Alzheimer disease; 05014 Amyotrophic lateral sclerosis; 05020 Prion disease; 05022 Pathways of neurodegeneration - multiple diseases; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 05410 Hypertrophic cardiomyopathy; 05414 Dilated cardiomyopathy; 04930 Type II diabetes mellitus; 04940 Type I diabetes mellitus; 04936 Alcoholic liver disease; 04932 Non-alcoholic fatty liver disease; 04931 Insulin resistance; 04933 AGE-RAGE signaling pathway in diabetic complications; 01523 Antifolate resistance; 04052 Cytokines and growth factors [BR:mmu04052]; 00536 Glycosaminoglycan binding proteins [BR:mmu00536]; Tumor necrosis fators; Cytokines; ','[Pfam: TNF DUF956]'),('93759','mmu:93759','[Sirt1, AA673258, SIR2L1, Sir2, Sir2a, Sir2alpha]','[(RefSeq) sirtuin 1]','[NAD+-dependent protein deacetylase sirtuin 1 [EC:2.3.1.286]]','[Mus musculus (house mouse)]','[Nicotinate and nicotinamide metabolism, Metabolic pathways, FoxO signaling pathway, AMPK signaling pathway, Longevity regulating pathway, Longevity regulating pathway - multiple species, Cellular senescence, Glucagon signaling pathway, Alcoholic liver disease, Amphetamine addiction, MicroRNAs in cancer]','[]','[]','00760 Nicotinate and nicotinamide metabolism; 04068 FoxO signaling pathway; 04152 AMPK signaling pathway; 04218 Cellular senescence; 04922 Glucagon signaling pathway; 04211 Longevity regulating pathway; 04213 Longevity regulating pathway - multiple species; 05206 MicroRNAs in cancer; 05031 Amphetamine addiction; 04936 Alcoholic liver disease; 03036 Chromosome and associated proteins [BR:mmu03036]; 03029 Mitochondrial biogenesis [BR:mmu03029]; 2.3.1.286  protein acetyllysine N-acetyltransferase; SIRTs (sirtuins, class III HDACs); Other regulator of mitochondrial biogenesis; ','[Pfam: SIR2 DUF592]'),('387144','mmu:387144','[Mir10b, Mirn10b, mir-10b, mmu-mir-10b]','[(RefSeq) microRNA 10b]','[microRNA 10b]','[Mus musculus (house mouse)]','[Proteoglycans in cancer, MicroRNAs in cancer]','[]','[]','05206 MicroRNAs in cancer; 05205 Proteoglycans in cancer; 03100 Non-coding RNAs [BR:mmu03100]; Micro RNA; ','[]'),('11303','mmu:11303','[Abca1, ABC-1, Abc1]','[(RefSeq) ATP-binding cassette, sub-family A (ABC1), member 1]','[ATP-binding cassette, subfamily A (ABC1), member 1]','[Mus musculus (house mouse)]','[ABC transporters, Fat digestion and absorption, Cholesterol metabolism, Lipid and atherosclerosis]','[]','[]','02010 ABC transporters; 04975 Fat digestion and absorption; 04979 Cholesterol metabolism; 05417 Lipid and atherosclerosis; 02000 Transporters [BR:mmu02000]; ABCA1, 2, 3, 4, 7, 12, 13 subgroups; ','[Pfam: ABC2_membrane_3 ABC_tran AAA_21 DUF4162 AAA_23]'),('11307','mmu:11307','[Abcg1, AW413978, Abc8, White]','[(RefSeq) ATP binding cassette subfamily G member 1]','[ATP-binding cassette, subfamily G (WHITE), member 1]','[Mus musculus (house mouse)]','[ABC transporters, Lipid and atherosclerosis]','[]','[]','02010 ABC transporters; 05417 Lipid and atherosclerosis; 02000 Transporters [BR:mmu02000]; ABCG1, 4 subgroups; ','[Pfam: ABC2_membrane ABC_tran ABC2_membrane_7 SMC_N AAA_21 RsgA_GTPase AAA_29 MMR_HSR1 AAA_16 cobW AAA_13 Rad17 AAA_23 AAA_18 AAA_28 AAA_22 AAA_30 Cytidylate_kin AAA_25 AAA_10]'),('66102','mmu:66102','[Cxcl16, 0910001K24Rik, AV290116, BB024863, CXCL16v1, CXCL16v2, SR-PSOX, Zmynd15, b2b498Clo]','[(RefSeq) chemokine (C-X-C motif) ligand 16]','[C-X-C motif chemokine 16]','[Mus musculus (house mouse)]','[Cytokine-cytokine receptor interaction, Chemokine signaling pathway]','[]','[]','04060 Cytokine-cytokine receptor interaction; 04062 Chemokine signaling pathway; 04052 Cytokines and growth factors [BR:mmu04052]; Chemokines; ','[Pfam: IL8]'),('433759','mmu:433759','[Hdac1, HD1, Hdac1-ps, MommeD5, RPD3]','[(RefSeq) histone deacetylase 1]','[histone deacetylase 1/2 [EC:3.5.1.98]]','[Mus musculus (house mouse)]','[Cell cycle, Longevity regulating pathway - multiple species, Notch signaling pathway, Neutrophil extracellular trap formation, Thyroid hormone signaling pathway, Huntington disease, Amphetamine addiction, Alcoholism, Human papillomavirus infection, Epstein-Barr virus infection, Pathways in cancer, Transcriptional misregulation in cancer, Viral carcinogenesis, MicroRNAs in cancer, Chronic myeloid leukemia]','[]','[]','04330 Notch signaling pathway; 04110 Cell cycle; 04613 Neutrophil extracellular trap formation; 04919 Thyroid hormone signaling pathway; 04213 Longevity regulating pathway - multiple species; 05200 Pathways in cancer; 05202 Transcriptional misregulation in cancer; 05206 MicroRNAs in cancer; 05203 Viral carcinogenesis; 05220 Chronic myeloid leukemia; 05169 Epstein-Barr virus infection; 05165 Human papillomavirus infection; 05016 Huntington disease; 05031 Amphetamine addiction; 05034 Alcoholism; 03036 Chromosome and associated proteins [BR:mmu03036]; 3.5.1.98  histone deacetylase; Class I HDACs; Sin3A-HDAC complex; BRAF-HDAC complex; Other heterochromatin formation proteins; NuRD complex; ','[Pfam: Hist_deacetyl]'),('16171','mmu:16171','[Il17a, Ctla-8, Ctla8, IL-17, IL-17A, Il17]','[(RefSeq) interleukin 17A]','[interleukin 17A]','[Mus musculus (house mouse)]','[Cytokine-cytokine receptor interaction, IL-17 signaling pathway, Th17 cell differentiation, Alcoholic liver disease, Inflammatory bowel disease, Rheumatoid arthritis]','[]','[]','04060 Cytokine-cytokine receptor interaction; 04659 Th17 cell differentiation; 04657 IL-17 signaling pathway; 05323 Rheumatoid arthritis; 05321 Inflammatory bowel disease; 04936 Alcoholic liver disease; 04052 Cytokines and growth factors [BR:mmu04052]; Interleukins; ','[Pfam: IL17]'),('20371','mmu:20371','[Foxp3, JM2, scurfin, sf]','[(RefSeq) forkhead box P3]','[forkhead box P3]','[Mus musculus (house mouse)]','[Th17 cell differentiation, Inflammatory bowel disease]','[]','[]','04659 Th17 cell differentiation; 05321 Inflammatory bowel disease; 03000 Transcription factors [BR:mmu03000]; Fork head/winged helix other regulators; ','[Pfam: FOXP-CC Forkhead]'),('16600','mmu:16600','[Klf4, EZF, Gklf, Zie]','[(RefSeq) Kruppel-like factor 4 (gut)]','[krueppel-like factor 4]','[Mus musculus (house mouse)]','[Signaling pathways regulating pluripotency of stem cells, Chemical carcinogenesis - receptor activation]','[]','[]','04550 Signaling pathways regulating pluripotency of stem cells; 05207 Chemical carcinogenesis - receptor activation; 03000 Transcription factors [BR:mmu03000]; Cys2His2 SP/KLF family and related proteins; ','[Pfam: zf-C2H2 zf-H2C2_2 zf-C2H2_4 zf-C2H2_8]'),('17831','mmu:17831','[Muc2, 2010015E03Rik, MCM, wnn]','[(RefSeq) mucin 2]','[mucin-2]','[Mus musculus (house mouse)]','[Amoebiasis, Gastric cancer]','[]','[]','05226 Gastric cancer; 05146 Amoebiasis; 04131 Membrane trafficking [BR:mmu04131]; Secretory mucins; ','[Pfam: VWD C8 Mucin2_WxxW TIL VWC]'),('140474','mmu:140474','[Muc4, 4933405I11Rik, Asgp]','[(RefSeq) mucin 4]','[mucin-4]','[Mus musculus (house mouse)]',NULL,'[]','[]','04131 Membrane trafficking [BR:mmu04131]; Membrane bound mucins; ','[Pfam: VWD NIDO hEGF]'),('94192','mmu:94192','[C1galt1, 2210410E06Rik, AV284120, T-synthase]','[(RefSeq) core 1 synthase, glycoprotein-N-acetylgalactosamine 3-beta-galactosyltransferase, 1]','[glycoprotein-N-acetylgalactosamine 3-beta-galactosyltransferase [EC:2.4.1.122]]','[Mus musculus (house mouse)]','[Mucin type O-glycan biosynthesis, Other types of O-glycan biosynthesis, Metabolic pathways]','[]','[]','00512 Mucin type O-glycan biosynthesis; 00514 Other types of O-glycan biosynthesis; 01003 Glycosyltransferases [BR:mmu01003]; 2.4.1.122  N-acetylgalactosaminide beta-1,3-galactosyltransferase; Type 3 chain; ','[Pfam: Fringe Galactosyl_T]'),('56375','mmu:56375','[B4galt4, 9130402O08Rik, B4galt-IV, b4Gal-T4, beta4Gal-T4, beta4GalT-IV]','[(RefSeq) UDP-Gal:betaGlcNAc beta 1,4-galactosyltransferase, polypeptide 4]','[beta-1,4-galactosyltransferase 4 [EC:2.4.1.90 2.4.1.275 2.4.1.-]]','[Mus musculus (house mouse)]','[Glycosaminoglycan biosynthesis - keratan sulfate, Glycosphingolipid biosynthesis - lacto and neolacto series, Metabolic pathways]','[]','[]','00533 Glycosaminoglycan biosynthesis - keratan sulfate; 00601 Glycosphingolipid biosynthesis - lacto and neolacto series; 01003 Glycosyltransferases [BR:mmu01003]; 2.4.1.90  N-acetyllactosamine synthase; 2.4.1.275  neolactotriaosylceramide beta-1,4-galactosyltransferase; 2.4.1.-; Type 2 chain/Poly-N-acetyllactosamine; ','[Pfam: Glyco_transf_7N Glyco_transf_7C Glyco_tranf_2_2]'),('20443','mmu:20443','[St3gal4, SIAT4-C, ST3Gal_IV, ST3GalIV, Siat4c]','[(RefSeq) ST3 beta-galactoside alpha-2,3-sialyltransferase 4]','[beta-galactoside alpha-2,3-sialyltransferase (sialyltransferase 4C) [EC:2.4.3.2 2.4.3.4 2.4.3.6]]','[Mus musculus (house mouse)]','[Glycosphingolipid biosynthesis - lacto and neolacto series, Metabolic pathways]','[]','[]','00601 Glycosphingolipid biosynthesis - lacto and neolacto series; 01003 Glycosyltransferases [BR:mmu01003]; 2.4.3.2  beta-D-galactosyl-(1->3)-N-acetyl-beta-D-galactosaminide alpha-2,3-sialyltransferase; 2.4.3.4  beta-galactoside alpha-2,3-sialyltransferase; 2.4.3.6  N-acetyllactosaminide alpha-2,3-sialyltransferase; Sialyltransferase; ','[Pfam: Glyco_transf_29]'),('387140','mmu:387140','[Mir21a, Mir21, Mirn21, mmu-mir-21, mmu-mir-21a]','[(RefSeq) microRNA 21a]','[microRNA 21]','[Mus musculus (house mouse)]','[Proteoglycans in cancer, MicroRNAs in cancer]','[]','[]','05206 MicroRNAs in cancer; 05205 Proteoglycans in cancer; 04147 Exosome [BR:mmu04147]; 03100 Non-coding RNAs [BR:mmu03100]; Exosomal RNA of breast milk; Exosomal RNA of ovarian cancer cells; Exosomal RNA of lung cancer cells; Exosomal RNA of other cancer cells; Micro RNA; ','[]'),('20525','mmu:20525','[Slc2a1, Glut-1, Glut1, M100200, Rgsc200]','[(RefSeq) solute carrier family 2 (facilitated glucose transporter), member 1]','[MFS transporter, SP family, solute carrier family 2 (facilitated glucose transporter), member 1]','[Mus musculus (house mouse)]','[HIF-1 signaling pathway, Insulin secretion, Thyroid hormone signaling pathway, Adipocytokine signaling pathway, Glucagon signaling pathway, Insulin resistance, Bile secretion, Human T-cell leukemia virus 1 infection, Pathways in cancer, Renal cell carcinoma, Central carbon metabolism in cancer, Diabetic cardiomyopathy]','[]','[]','04066 HIF-1 signaling pathway; 04911 Insulin secretion; 04922 Glucagon signaling pathway; 04920 Adipocytokine signaling pathway; 04919 Thyroid hormone signaling pathway; 04976 Bile secretion; 05200 Pathways in cancer; 05230 Central carbon metabolism in cancer; 05211 Renal cell carcinoma; 05166 Human T-cell leukemia virus 1 infection; 05415 Diabetic cardiomyopathy; 04931 Insulin resistance; 02000 Transporters [BR:mmu02000]; SLC2: Facilitative GLUT transporter; Sugar porter (SP) family [TC:2.A.1.1]; ','[Pfam: Sugar_tr MFS_1 2-thiour_desulf TRI12]'),('20528','mmu:20528','[Slc2a4, GT2, Glut-4, Glut4, twgy]','[(RefSeq) solute carrier family 2 (facilitated glucose transporter), member 4]','[MFS transporter, SP family, solute carrier family 2 (facilitated glucose transporter), member 4]','[Mus musculus (house mouse)]','[FoxO signaling pathway, AMPK signaling pathway, Insulin signaling pathway, Adipocytokine signaling pathway, Type II diabetes mellitus, Insulin resistance, Diabetic cardiomyopathy]','[]','[]','04068 FoxO signaling pathway; 04152 AMPK signaling pathway; 04910 Insulin signaling pathway; 04920 Adipocytokine signaling pathway; 05415 Diabetic cardiomyopathy; 04930 Type II diabetes mellitus; 04931 Insulin resistance; 02000 Transporters [BR:mmu02000]; SLC2: Facilitative GLUT transporter; Sugar porter (SP) family [TC:2.A.1.1]; ','[Pfam: Sugar_tr MFS_1 Phage_holin_2_3]'),('11651','mmu:11651','[Akt1, Akt, LTR-akt, PKB, PKB/Akt, PKBalpha, Rac]','[(RefSeq) thymoma viral proto-oncogene 1]','[RAC serine/threonine-protein kinase [EC:2.7.11.1]]','[Mus musculus (house mouse)]','[EGFR tyrosine kinase inhibitor resistance, Endocrine resistance, Platinum drug resistance, MAPK signaling pathway, ErbB signaling pathway, Ras signaling pathway, Rap1 signaling pathway, cGMP-PKG signaling pathway, cAMP signaling pathway, Chemokine signaling pathway, HIF-1 signaling pathway, FoxO signaling pathway, Sphingolipid signaling pathway, Phospholipase D signaling pathway, Autophagy - animal, mTOR signaling pathway, PI3K-Akt signaling pathway, AMPK signaling pathway, Apoptosis, Longevity regulating pathway, Longevity regulating pathway - multiple species, Cellular senescence, Adrenergic signaling in cardiomyocytes, VEGF signaling pathway, Apelin signaling pathway, Osteoclast differentiation, Focal adhesion, Signaling pathways regulating pluripotency of stem cells, Platelet activation, Neutrophil extracellular trap formation, Toll-like receptor signaling pathway, C-type lectin receptor signaling pathway, JAK-STAT signaling pathway, T cell receptor signaling pathway, B cell receptor signaling pathway, Fc epsilon RI signaling pathway, Fc gamma R-mediated phagocytosis, TNF signaling pathway, Neurotrophin signaling pathway, Cholinergic synapse, Dopaminergic synapse, Insulin signaling pathway, Progesterone-mediated oocyte maturation, Estrogen signaling pathway, Prolactin signaling pathway, Thyroid hormone signaling pathway, Adipocytokine signaling pathway, Glucagon signaling pathway, Regulation of lipolysis in adipocytes, Relaxin signaling pathway, GnRH secretion, Insulin resistance, Non-alcoholic fatty liver disease, AGE-RAGE signaling pathway in diabetic complications, Growth hormone synthesis, secretion and action, Alcoholic liver disease, Carbohydrate digestion and absorption, Alzheimer disease, Spinocerebellar ataxia, Salmonella infection, Yersinia infection, Chagas disease, Toxoplasmosis, Tuberculosis, Hepatitis C, Hepatitis B, Measles, Human cytomegalovirus infection, Influenza A, Human papillomavirus infection, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Herpes simplex virus 1 infection, Epstein-Barr virus infection, Human immunodeficiency virus 1 infection, Pathways in cancer, Proteoglycans in cancer, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species, Colorectal cancer, Renal cell carcinoma, Pancreatic cancer, Endometrial cancer, Glioma, Prostate cancer, Melanoma, Chronic myeloid leukemia, Acute myeloid leukemia, Small cell lung cancer, Non-small cell lung cancer, Breast cancer, Hepatocellular carcinoma, Gastric cancer, Central carbon metabolism in cancer, Choline metabolism in cancer, PD-L1 expression and PD-1 checkpoint pathway in cancer, Diabetic cardiomyopathy, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[]','[]','04010 MAPK signaling pathway; 04012 ErbB signaling pathway; 04014 Ras signaling pathway; 04015 Rap1 signaling pathway; 04370 VEGF signaling pathway; 04371 Apelin signaling pathway; 04630 JAK-STAT signaling pathway; 04668 TNF signaling pathway; 04066 HIF-1 signaling pathway; 04068 FoxO signaling pathway; 04072 Phospholipase D signaling pathway; 04071 Sphingolipid signaling pathway; 04024 cAMP signaling pathway; 04022 cGMP-PKG signaling pathway; 04151 PI3K-Akt signaling pathway; 04152 AMPK signaling pathway; 04150 mTOR signaling pathway; 04140 Autophagy - animal; 04210 Apoptosis; 04218 Cellular senescence; 04510 Focal adhesion; 04550 Signaling pathways regulating pluripotency of stem cells; 04611 Platelet activation; 04613 Neutrophil extracellular trap formation; 04620 Toll-like receptor signaling pathway; 04625 C-type lectin receptor signaling pathway; 04660 T cell receptor signaling pathway; 04662 B cell receptor signaling pathway; 04664 Fc epsilon RI signaling pathway; 04666 Fc gamma R-mediated phagocytosis; 04062 Chemokine signaling pathway; 04910 Insulin signaling pathway; 04922 Glucagon signaling pathway; 04923 Regulation of lipolysis in adipocytes; 04920 Adipocytokine signaling pathway; 04929 GnRH secretion; 04915 Estrogen signaling pathway; 04914 Progesterone-mediated oocyte maturation; 04917 Prolactin signaling pathway; 04926 Relaxin signaling pathway; 04935 Growth hormone synthesis, secretion and action; 04919 Thyroid hormone signaling pathway; 04261 Adrenergic signaling in cardiomyocytes; 04973 Carbohydrate digestion and absorption; 04725 Cholinergic synapse; 04728 Dopaminergic synapse; 04722 Neurotrophin signaling pathway; 04380 Osteoclast differentiation; 04211 Longevity regulating pathway; 04213 Longevity regulating pathway - multiple species; 05200 Pathways in cancer; 05205 Proteoglycans in cancer; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 05230 Central carbon metabolism in cancer; 05231 Choline metabolism in cancer; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05210 Colorectal cancer; 05212 Pancreatic cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05214 Glioma; 05221 Acute myeloid leukemia; 05220 Chronic myeloid leukemia; 05218 Melanoma; 05211 Renal cell carcinoma; 05215 Prostate cancer; 05213 Endometrial cancer; 05224 Breast cancer; 05222 Small cell lung cancer; 05223 Non-small cell lung cancer; 05166 Human T-cell leukemia virus 1 infection; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05160 Hepatitis C; 05164 Influenza A; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05165 Human papillomavirus infection; 05132 Salmonella infection; 05135 Yersinia infection; 05152 Tuberculosis; 05145 Toxoplasmosis; 05142 Chagas disease; 05010 Alzheimer disease; 05017 Spinocerebellar ataxia; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 05415 Diabetic cardiomyopathy; 04936 Alcoholic liver disease; 04932 Non-alcoholic fatty liver disease; 04931 Insulin resistance; 04933 AGE-RAGE signaling pathway in diabetic complications; 01521 EGFR tyrosine kinase inhibitor resistance; 01524 Platinum drug resistance; 01522 Endocrine resistance; 01001 Protein kinases [BR:mmu01001]; 2.7.11.1  non-specific serine/threonine protein kinase; AKT/PKB family; ','[Pfam: Pkinase PK_Tyr_Ser-Thr PH Pkinase_C Kinase-like PH_11 PH_20 Amphi-Trp Seadorna_VP7 APH Pkinase_fungal]'),('17395','mmu:17395','[Mmp9, AW743869, B/MMP9, Clg4b, Gel_B, MMP-9, pro-MMP-9]','[(RefSeq) matrix metallopeptidase 9]','[matrix metalloproteinase-9 (gelatinase B) [EC:3.4.24.35]]','[Mus musculus (house mouse)]','[Endocrine resistance, IL-17 signaling pathway, TNF signaling pathway, Leukocyte transendothelial migration, Estrogen signaling pathway, Relaxin signaling pathway, Hepatitis B, Pathways in cancer, Transcriptional misregulation in cancer, Proteoglycans in cancer, MicroRNAs in cancer, Prostate cancer, Bladder cancer, Diabetic cardiomyopathy, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[]','[]','04668 TNF signaling pathway; 04657 IL-17 signaling pathway; 04670 Leukocyte transendothelial migration; 04915 Estrogen signaling pathway; 04926 Relaxin signaling pathway; 05200 Pathways in cancer; 05202 Transcriptional misregulation in cancer; 05206 MicroRNAs in cancer; 05205 Proteoglycans in cancer; 05219 Bladder cancer; 05215 Prostate cancer; 05161 Hepatitis B; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 05415 Diabetic cardiomyopathy; 01522 Endocrine resistance; 01002 Peptidases and inhibitors [BR:mmu01002]; 3.4.24.35  gelatinase B; Family M10; ','[Pfam: Peptidase_M10 fn2 Hemopexin PT PG_binding_1 Reprolysin_3 Reprolysin_4 Peptidase_M66 DUF4953]'),('105787','mmu:105787','[Prkaa1, AI194361, AI450832, AL024255, AMPKalpha1, C130083N04Rik]','[(RefSeq) protein kinase, AMP-activated, alpha 1 catalytic subunit]','[5\'-AMP-activated protein kinase, catalytic alpha subunit [EC:2.7.11.11]]','[Mus musculus (house mouse)]','[FoxO signaling pathway, Autophagy - animal, mTOR signaling pathway, PI3K-Akt signaling pathway, AMPK signaling pathway, Longevity regulating pathway, Longevity regulating pathway - multiple species, Apelin signaling pathway, Tight junction, Circadian rhythm, Thermogenesis, Insulin signaling pathway, Adipocytokine signaling pathway, Oxytocin signaling pathway, Glucagon signaling pathway, Insulin resistance, Non-alcoholic fatty liver disease, Alcoholic liver disease, Hypertrophic cardiomyopathy, Fluid shear stress and atherosclerosis]','[]','[]','04371 Apelin signaling pathway; 04068 FoxO signaling pathway; 04151 PI3K-Akt signaling pathway; 04152 AMPK signaling pathway; 04150 mTOR signaling pathway; 04140 Autophagy - animal; 04530 Tight junction; 04910 Insulin signaling pathway; 04922 Glucagon signaling pathway; 04920 Adipocytokine signaling pathway; 04921 Oxytocin signaling pathway; 04211 Longevity regulating pathway; 04213 Longevity regulating pathway - multiple species; 04710 Circadian rhythm; 04714 Thermogenesis; 05418 Fluid shear stress and atherosclerosis; 05410 Hypertrophic cardiomyopathy; 04936 Alcoholic liver disease; 04932 Non-alcoholic fatty liver disease; 04931 Insulin resistance; 01001 Protein kinases [BR:mmu01001]; 04131 Membrane trafficking [BR:mmu04131]; 03029 Mitochondrial biogenesis [BR:mmu03029]; 2.7.11.11  cAMP-dependent protein kinase; CAMKL family; Others; Other regulator of mitochondrial biogenesis; Other mitophagy factors; ','[Pfam: Pkinase PK_Tyr_Ser-Thr AdenylateSensor ABC1 Kinase-like RIO1 Kdo APH]'),('57875','mmu:57875','[Angptl4, Arp4, Bk89, Fiaf, Hfarp, Ng27, Pgar, Pgarg, Pp1158]','[(RefSeq) angiopoietin-like 4]','[angiopoietin-like 4]','[Mus musculus (house mouse)]','[PPAR signaling pathway, Cholesterol metabolism]','[]','[]','03320 PPAR signaling pathway; 04979 Cholesterol metabolism; 04147 Exosome [BR:mmu04147]; Exosomal proteins of breast milk; ','[Pfam: Fibrinogen_C SlyX HrpJ UPF0242 DUF16 Fib_alpha COG5 DivIC HrpB7]'),('14776','mmu:14776','[Gpx2, GI-GPx, GPx-GI, GSHPx-2, GSHPx-GI]','[(RefSeq) glutathione peroxidase 2]','[glutathione peroxidase [EC:1.11.1.9]]','[Mus musculus (house mouse)]','[Glutathione metabolism, Arachidonic acid metabolism, Metabolic pathways, Thyroid hormone synthesis, Amyotrophic lateral sclerosis, Huntington disease, Pathways of neurodegeneration - multiple diseases]','[]','[]','00590 Arachidonic acid metabolism; 00480 Glutathione metabolism; 04918 Thyroid hormone synthesis; 05014 Amyotrophic lateral sclerosis; 05016 Huntington disease; 05022 Pathways of neurodegeneration - multiple diseases; 1.11.1.9  glutathione peroxidase; ','[Pfam: GSHPx]'),('238055','mmu:238055','[Apob, AI315052, Apo_B-100, apob-100, apob-48]','[(RefSeq) apolipoprotein B]','[apolipoprotein B]','[Mus musculus (house mouse)]','[Fat digestion and absorption, Vitamin digestion and absorption, Cholesterol metabolism, Lipid and atherosclerosis]','[]','[]','04975 Fat digestion and absorption; 04979 Cholesterol metabolism; 04977 Vitamin digestion and absorption; 05417 Lipid and atherosclerosis; 04147 Exosome [BR:mmu04147]; 00536 Glycosaminoglycan binding proteins [BR:mmu00536]; Exosomal proteins of other cancer cells; Others; ','[Pfam: Vitellogenin_N Vit_open_b-sht DUF1081 ApoB100_C DUF6047]'),('11829','mmu:11829','[Aqp4, WCH4]','[(RefSeq) aquaporin 4]','[aquaporin-4]','[Mus musculus (house mouse)]','[Vasopressin-regulated water reabsorption, Bile secretion]','[]','[]','04976 Bile secretion; 04962 Vasopressin-regulated water reabsorption; 02000 Transporters [BR:mmu02000]; Aquaporins; ','[Pfam: MIP]'),('232889','mmu:232889','[Pla2g4c, D7Ertd445e]','[(RefSeq) phospholipase A2, group IVC (cytosolic, calcium-independent)]','[cytosolic phospholipase A2 [EC:3.1.1.4]]','[Mus musculus (house mouse)]','[Glycerophospholipid metabolism, Ether lipid metabolism, Arachidonic acid metabolism, Linoleic acid metabolism, alpha-Linolenic acid metabolism, Metabolic pathways, MAPK signaling pathway, Ras signaling pathway, Phospholipase D signaling pathway, Necroptosis, Vascular smooth muscle contraction, VEGF signaling pathway, Platelet activation, Fc epsilon RI signaling pathway, Fc gamma R-mediated phagocytosis, Glutamatergic synapse, Serotonergic synapse, Long-term depression, Inflammatory mediator regulation of TRP channels, GnRH signaling pathway, Ovarian steroidogenesis, Oxytocin signaling pathway, Choline metabolism in cancer]','[]','[]','00564 Glycerophospholipid metabolism; 00565 Ether lipid metabolism; 00590 Arachidonic acid metabolism; 00591 Linoleic acid metabolism; 00592 alpha-Linolenic acid metabolism; 04010 MAPK signaling pathway; 04014 Ras signaling pathway; 04370 VEGF signaling pathway; 04072 Phospholipase D signaling pathway; 04217 Necroptosis; 04611 Platelet activation; 04664 Fc epsilon RI signaling pathway; 04666 Fc gamma R-mediated phagocytosis; 04912 GnRH signaling pathway; 04913 Ovarian steroidogenesis; 04921 Oxytocin signaling pathway; 04270 Vascular smooth muscle contraction; 04724 Glutamatergic synapse; 04726 Serotonergic synapse; 04730 Long-term depression; 04750 Inflammatory mediator regulation of TRP channels; 05231 Choline metabolism in cancer; 3.1.1.4  phospholipase A2; ','[Pfam: PLA2_B CvfB_WH Gcn1_N DUF1116]'),('108079','mmu:108079','[Prkaa2, 2310008I11Rik, A830082D05, AMPKalpha2]','[(RefSeq) protein kinase, AMP-activated, alpha 2 catalytic subunit]','[5\'-AMP-activated protein kinase, catalytic alpha subunit [EC:2.7.11.11]]','[Mus musculus (house mouse)]','[FoxO signaling pathway, Autophagy - animal, mTOR signaling pathway, PI3K-Akt signaling pathway, AMPK signaling pathway, Longevity regulating pathway, Longevity regulating pathway - multiple species, Apelin signaling pathway, Tight junction, Circadian rhythm, Thermogenesis, Insulin signaling pathway, Adipocytokine signaling pathway, Oxytocin signaling pathway, Glucagon signaling pathway, Insulin resistance, Non-alcoholic fatty liver disease, Alcoholic liver disease, Hypertrophic cardiomyopathy, Fluid shear stress and atherosclerosis]','[]','[]','04371 Apelin signaling pathway; 04068 FoxO signaling pathway; 04151 PI3K-Akt signaling pathway; 04152 AMPK signaling pathway; 04150 mTOR signaling pathway; 04140 Autophagy - animal; 04530 Tight junction; 04910 Insulin signaling pathway; 04922 Glucagon signaling pathway; 04920 Adipocytokine signaling pathway; 04921 Oxytocin signaling pathway; 04211 Longevity regulating pathway; 04213 Longevity regulating pathway - multiple species; 04710 Circadian rhythm; 04714 Thermogenesis; 05418 Fluid shear stress and atherosclerosis; 05410 Hypertrophic cardiomyopathy; 04936 Alcoholic liver disease; 04932 Non-alcoholic fatty liver disease; 04931 Insulin resistance; 01001 Protein kinases [BR:mmu01001]; 04131 Membrane trafficking [BR:mmu04131]; 03029 Mitochondrial biogenesis [BR:mmu03029]; 2.7.11.11  cAMP-dependent protein kinase; CAMKL family; Others; Other regulator of mitochondrial biogenesis; Other mitophagy factors; ','[Pfam: Pkinase PK_Tyr_Ser-Thr AdenylateSensor Kinase-like ABC1 RIO1 Kdo APH]'),('12576','mmu:12576','[Cdkn1b, AA408329, AI843786, Kip1, p27, p27Kip1]','[(RefSeq) cyclin-dependent kinase inhibitor 1B]','[cyclin-dependent kinase inhibitor 1B]','[Mus musculus (house mouse)]','[Endocrine resistance, ErbB signaling pathway, HIF-1 signaling pathway, FoxO signaling pathway, Cell cycle, PI3K-Akt signaling pathway, AGE-RAGE signaling pathway in diabetic complications, Cushing syndrome, Measles, Human papillomavirus infection, Epstein-Barr virus infection, Pathways in cancer, Transcriptional misregulation in cancer, Viral carcinogenesis, MicroRNAs in cancer, Prostate cancer, Chronic myeloid leukemia, Small cell lung cancer, Gastric cancer]','[]','[]','04012 ErbB signaling pathway; 04066 HIF-1 signaling pathway; 04068 FoxO signaling pathway; 04151 PI3K-Akt signaling pathway; 04110 Cell cycle; 05200 Pathways in cancer; 05202 Transcriptional misregulation in cancer; 05206 MicroRNAs in cancer; 05203 Viral carcinogenesis; 05226 Gastric cancer; 05220 Chronic myeloid leukemia; 05215 Prostate cancer; 05222 Small cell lung cancer; 05162 Measles; 05169 Epstein-Barr virus infection; 05165 Human papillomavirus infection; 04933 AGE-RAGE signaling pathway in diabetic complications; 04934 Cushing syndrome; 01522 Endocrine resistance; ','[Pfam: CDI]'),('67443','mmu:67443','[Map1lc3b, 1010001C15Rik, Atg8, LC3b, MAP1A/MAP1B, Map1lc3]','[(RefSeq) microtubule-associated protein 1 light chain 3 beta]','[microtubule-associated protein 1 light chain]','[Mus musculus (house mouse)]','[Mitophagy - animal, Autophagy - animal, Ferroptosis, Apelin signaling pathway, NOD-like receptor signaling pathway, Amyotrophic lateral sclerosis, Pathways of neurodegeneration - multiple diseases, Kaposi sarcoma-associated herpesvirus infection]','[]','[]','04371 Apelin signaling pathway; 04140 Autophagy - animal; 04137 Mitophagy - animal; 04216 Ferroptosis; 04621 NOD-like receptor signaling pathway; 05167 Kaposi sarcoma-associated herpesvirus infection; 05014 Amyotrophic lateral sclerosis; 05022 Pathways of neurodegeneration - multiple diseases; 04131 Membrane trafficking [BR:mmu04131]; 03029 Mitochondrial biogenesis [BR:mmu03029]; 04812 Cytoskeleton proteins [BR:mmu04812]; Other mitophagy associated  proteins; Other mitophagy factors; Microtubule-associated proteins (MAPs); ','[Pfam: ATG8 APG12 Rad60-SLD]'),('20787','mmu:20787','[Srebf1, ADD1, SREBP1, bHLHd1]','[(RefSeq) sterol regulatory element binding transcription factor 1]','[sterol regulatory element-binding transcription factor 1]','[Mus musculus (house mouse)]','[AMPK signaling pathway, Insulin signaling pathway, Insulin resistance, Non-alcoholic fatty liver disease, Alcoholic liver disease]','[]','[]','04152 AMPK signaling pathway; 04910 Insulin signaling pathway; 04936 Alcoholic liver disease; 04932 Non-alcoholic fatty liver disease; 04931 Insulin resistance; 03000 Transcription factors [BR:mmu03000]; 04131 Membrane trafficking [BR:mmu04131]; Ubiquitous bHLH-ZIP factors; Other mitophagy associated  proteins; ','[Pfam: HLH]'),('20249','mmu:20249','[Scd1, AA589638, AI265570, Scd, Scd-1, ab]','[(RefSeq) stearoyl-Coenzyme A desaturase 1]','[stearoyl-CoA desaturase (Delta-9 desaturase) [EC:1.14.19.1]]','[Mus musculus (house mouse)]','[Biosynthesis of unsaturated fatty acids, Metabolic pathways, Fatty acid metabolism, PPAR signaling pathway, AMPK signaling pathway, Alcoholic liver disease]','[]','[]','01040 Biosynthesis of unsaturated fatty acids; 04152 AMPK signaling pathway; 03320 PPAR signaling pathway; 04936 Alcoholic liver disease; 01004 Lipid biosynthesis proteins [BR:mmu01004]; 1.14.19.1  stearoyl-CoA 9-desaturase; First; ','[Pfam: FA_desaturase]'),('14104','mmu:14104','[Fasn, A630082H08Rik, FAS]','[(RefSeq) fatty acid synthase]','[fatty acid synthase, animal type [EC:2.3.1.85]]','[Mus musculus (house mouse)]','[Fatty acid biosynthesis, Metabolic pathways, Fatty acid metabolism, AMPK signaling pathway, Insulin signaling pathway, Alcoholic liver disease]','[]','[]','00061 Fatty acid biosynthesis; 04152 AMPK signaling pathway; 04910 Insulin signaling pathway; 04936 Alcoholic liver disease; 01004 Lipid biosynthesis proteins [BR:mmu01004]; 04147 Exosome [BR:mmu04147]; 2.3.1.85  fatty-acid synthase system; Animal type; Exosomal proteins of haemopoietic cells  (B-cell, T-cell, DC-cell, reticulocyte, and mast cell); Exosomal proteins of breast milk; Exosomal proteins of colorectal cancer cells; Exosomal proteins of bladder cancer cells; ','[Pfam: Acyl_transf_1 ketoacyl-synt Thioesterase KR Ketoacyl-synt_C KAsynt_C_assoc PS-DH ADH_zinc_N ADH_zinc_N_2 PP-binding Methyltransf_12 adh_short Methyltransf_31 Thiolase_N Methyltransf_11 Methyltransf_23 adh_short_C2 YjeF_N ACP_syn_III]'),('11556','mmu:11556','[Adrb3, Adrb-3, beta_3-AR]','[(RefSeq) adrenergic receptor, beta 3]','[adrenergic receptor beta-3]','[Mus musculus (house mouse)]','[Calcium signaling pathway, cGMP-PKG signaling pathway, Neuroactive ligand-receptor interaction, Thermogenesis, Regulation of lipolysis in adipocytes, Renin secretion, Salivary secretion, Chemical carcinogenesis - receptor activation]','[]','[]','04020 Calcium signaling pathway; 04022 cGMP-PKG signaling pathway; 04080 Neuroactive ligand-receptor interaction; 04923 Regulation of lipolysis in adipocytes; 04924 Renin secretion; 04970 Salivary secretion; 04714 Thermogenesis; 05207 Chemical carcinogenesis - receptor activation; 04030 G protein-coupled receptors [BR:mmu04030]; Adrenaline; ','[Pfam: 7tm_1 7TM_GPCR_Srsx 7TM_GPCR_Srx 7tm_4 7TM_GPCR_Srv]'),('66853','mmu:66853','[Pnpla2, 0610039C21Rik, 1110001C14Rik, Atgl, TTS-2.2]','[(RefSeq) patatin-like phospholipase domain containing 2]','[patatin-like phospholipase domain-containing protein 2 [EC:3.1.1.3]]','[Mus musculus (house mouse)]','[Glycerolipid metabolism, Metabolic pathways, Thermogenesis, Regulation of lipolysis in adipocytes]','[]','[]','00561 Glycerolipid metabolism; 04923 Regulation of lipolysis in adipocytes; 04714 Thermogenesis; 3.1.1.3  triacylglycerol lipase; ','[Pfam: Patatin]'),('12896','mmu:12896','[Cpt2, AI323697, CPTII]','[(RefSeq) carnitine palmitoyltransferase 2]','[carnitine O-palmitoyltransferase 2 [EC:2.3.1.21]]','[Mus musculus (house mouse)]','[Fatty acid degradation, Fatty acid metabolism, PPAR signaling pathway, Thermogenesis, Diabetic cardiomyopathy]','[]','[]','00071 Fatty acid degradation; 03320 PPAR signaling pathway; 04714 Thermogenesis; 05415 Diabetic cardiomyopathy; 2.3.1.21  carnitine O-palmitoyltransferase; ','[Pfam: Carn_acyltransf Tri3]'),('11430','mmu:11430','[Acox1, AOX, Acox, D130055E20Rik, Paox]','[(RefSeq) acyl-Coenzyme A oxidase 1, palmitoyl]','[acyl-CoA oxidase [EC:1.3.3.6]]','[Mus musculus (house mouse)]','[Fatty acid degradation, beta-Alanine metabolism, alpha-Linolenic acid metabolism, Propanoate metabolism, Biosynthesis of unsaturated fatty acids, Metabolic pathways, Carbon metabolism, Fatty acid metabolism, PPAR signaling pathway, cAMP signaling pathway, Peroxisome, Alcoholic liver disease]','[]','[]','00640 Propanoate metabolism; 00071 Fatty acid degradation; 00592 alpha-Linolenic acid metabolism; 01040 Biosynthesis of unsaturated fatty acids; 00410 beta-Alanine metabolism; 04024 cAMP signaling pathway; 04146 Peroxisome; 03320 PPAR signaling pathway; 04936 Alcoholic liver disease; 1.3.3.6  acyl-CoA oxidase; ','[Pfam: ACOX Acyl-CoA_ox_N Acyl-CoA_dh_M Acyl-CoA_dh_1]'),('19017','mmu:19017','[Ppargc1a, A830037N07Rik, Gm11133, PGC-1, PPARGC-1-alpha, Pgc-1alpha, Pgc1, Pgco1, Ppargc1]','[(RefSeq) peroxisome proliferative activated receptor, gamma, coactivator 1 alpha]','[alpha peroxisome proliferator-activated receptor gamma coactivator 1-alpha]','[Mus musculus (house mouse)]','[AMPK signaling pathway, Longevity regulating pathway, Apelin signaling pathway, Thermogenesis, Insulin signaling pathway, Adipocytokine signaling pathway, Glucagon signaling pathway, Insulin resistance, Alcoholic liver disease, Huntington disease]','[]','[]','04371 Apelin signaling pathway; 04152 AMPK signaling pathway; 04910 Insulin signaling pathway; 04922 Glucagon signaling pathway; 04920 Adipocytokine signaling pathway; 04211 Longevity regulating pathway; 04714 Thermogenesis; 05016 Huntington disease; 04936 Alcoholic liver disease; 04931 Insulin resistance; 03029 Mitochondrial biogenesis [BR:mmu03029]; Coactivators; ','[Pfam: RRM_1 Tox-ODYAM1]'),('12514','mmu:12514','[Cd68, Lamp4, Scard1, gp110]','[(RefSeq) CD68 antigen]','[CD68 antigen]','[Mus musculus (house mouse)]','[Lysosome]','[]','[]','04142 Lysosome; 04090 CD molecules [BR:mmu04090]; Proteins; ','[Pfam: Lamp]'),('20296','mmu:20296','[Ccl2, AI323594, HC11, JE, MCAF, MCP-1, MCP1, SMC-CF, Scya2, Sigje]','[(RefSeq) chemokine (C-C motif) ligand 2]','[C-C motif chemokine 2]','[Mus musculus (house mouse)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, Chemokine signaling pathway, NOD-like receptor signaling pathway, IL-17 signaling pathway, TNF signaling pathway, AGE-RAGE signaling pathway in diabetic complications, Yersinia infection, Chagas disease, Malaria, Human cytomegalovirus infection, Influenza A, Herpes simplex virus 1 infection, Coronavirus disease - COVID-19, Rheumatoid arthritis, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[]','[]','04668 TNF signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04621 NOD-like receptor signaling pathway; 04657 IL-17 signaling pathway; 04062 Chemokine signaling pathway; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05168 Herpes simplex virus 1 infection; 05163 Human cytomegalovirus infection; 05135 Yersinia infection; 05144 Malaria; 05142 Chagas disease; 05323 Rheumatoid arthritis; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 04933 AGE-RAGE signaling pathway in diabetic complications; 04052 Cytokines and growth factors [BR:mmu04052]; 00536 Glycosaminoglycan binding proteins [BR:mmu00536]; Chemokines; Chemokines; ','[Pfam: IL8]'),('227394','mmu:227394','[Slco4c1, 9530051F04, C330017E21Rik, OATP-H, OATP-M1, OATP4C1, OATPX, PRO2176, SLC21A20, oatp-R]','[(RefSeq) solute carrier organic anion transporter family, member 4C1]','[solute carrier organic anion transporter family, member 4C]','[Mus musculus (house mouse)]',NULL,'[]','[]','02000 Transporters [BR:mmu02000]; SLC21/ASLCO: Organic anion transporter; ','[Pfam: OATP MFS_1 Kazal_2 SLATT_5]'),('21803','mmu:21803','[Tgfb1, TGF-beta1, TGFbeta1, Tgfb, Tgfb-1]','[(RefSeq) transforming growth factor, beta 1]','[transforming growth factor beta-1]','[Mus musculus (house mouse)]','[MAPK signaling pathway, Cytokine-cytokine receptor interaction, FoxO signaling pathway, Cell cycle, Cellular senescence, TGF-beta signaling pathway, Osteoclast differentiation, Hippo signaling pathway, Th17 cell differentiation, Intestinal immune network for IgA production, Relaxin signaling pathway, Non-alcoholic fatty liver disease, AGE-RAGE signaling pathway in diabetic complications, Leishmaniasis, Chagas disease, Malaria, Toxoplasmosis, Amoebiasis, Tuberculosis, Hepatitis B, Human T-cell leukemia virus 1 infection, Pathways in cancer, Proteoglycans in cancer, Colorectal cancer, Renal cell carcinoma, Pancreatic cancer, Chronic myeloid leukemia, Hepatocellular carcinoma, Gastric cancer, Inflammatory bowel disease, Rheumatoid arthritis, Hypertrophic cardiomyopathy, Dilated cardiomyopathy, Diabetic cardiomyopathy]','[]','[]','04010 MAPK signaling pathway; 04350 TGF-beta signaling pathway; 04390 Hippo signaling pathway; 04068 FoxO signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04110 Cell cycle; 04218 Cellular senescence; 04659 Th17 cell differentiation; 04672 Intestinal immune network for IgA production; 04926 Relaxin signaling pathway; 04380 Osteoclast differentiation; 05200 Pathways in cancer; 05205 Proteoglycans in cancer; 05210 Colorectal cancer; 05212 Pancreatic cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05220 Chronic myeloid leukemia; 05211 Renal cell carcinoma; 05166 Human T-cell leukemia virus 1 infection; 05161 Hepatitis B; 05152 Tuberculosis; 05146 Amoebiasis; 05144 Malaria; 05145 Toxoplasmosis; 05140 Leishmaniasis; 05142 Chagas disease; 05323 Rheumatoid arthritis; 05321 Inflammatory bowel disease; 05410 Hypertrophic cardiomyopathy; 05414 Dilated cardiomyopathy; 05415 Diabetic cardiomyopathy; 04932 Non-alcoholic fatty liver disease; 04933 AGE-RAGE signaling pathway in diabetic complications; 03110 Chaperones and folding catalysts [BR:mmu03110]; 04052 Cytokines and growth factors [BR:mmu04052]; 00536 Glycosaminoglycan binding proteins [BR:mmu00536]; Others; Transforming growth factors (RSTK binding); Growth factors/receptors; ','[Pfam: TGFb_propeptide TGF_beta]'),('14268','mmu:14268','[Fn1, E330027I09, Fn, Fn-1]','[(RefSeq) fibronectin 1]','[fibronectin 1]','[Mus musculus (house mouse)]','[PI3K-Akt signaling pathway, Focal adhesion, ECM-receptor interaction, Regulation of actin cytoskeleton, AGE-RAGE signaling pathway in diabetic complications, Bacterial invasion of epithelial cells, Yersinia infection, Amoebiasis, Human papillomavirus infection, Pathways in cancer, Proteoglycans in cancer, Small cell lung cancer]','[]','[]','04151 PI3K-Akt signaling pathway; 04512 ECM-receptor interaction; 04510 Focal adhesion; 04810 Regulation of actin cytoskeleton; 05200 Pathways in cancer; 05205 Proteoglycans in cancer; 05222 Small cell lung cancer; 05165 Human papillomavirus infection; 05135 Yersinia infection; 05100 Bacterial invasion of epithelial cells; 05146 Amoebiasis; 04933 AGE-RAGE signaling pathway in diabetic complications; 04131 Membrane trafficking [BR:mmu04131]; 04147 Exosome [BR:mmu04147]; 00536 Glycosaminoglycan binding proteins [BR:mmu00536]; 04990 Domain-containing proteins not elsewhere classified [BR:mmu04990]; ER-Golgi intermediate compartment (ERGIC) proteins; Exosomal proteins of bladder cancer cells; Extracellular matrix molecules; Fibronectin type I domain-containing proteins; ','[Pfam: fn3 fn1 fn2 Pur_ac_phosph_N Interfer-bind NDNF DUF4959]'),('12842','mmu:12842','[Col1a1, Col1a-1, Cola-1, Cola1, Mov-13, Mov13]','[(RefSeq) collagen, type I, alpha 1]','[collagen type I alpha]','[Mus musculus (house mouse)]','[PI3K-Akt signaling pathway, Focal adhesion, ECM-receptor interaction, Platelet activation, Relaxin signaling pathway, AGE-RAGE signaling pathway in diabetic complications, Protein digestion and absorption, Amoebiasis, Human papillomavirus infection, Proteoglycans in cancer, Diabetic cardiomyopathy]','[]','[]','04151 PI3K-Akt signaling pathway; 04512 ECM-receptor interaction; 04510 Focal adhesion; 04611 Platelet activation; 04926 Relaxin signaling pathway; 04974 Protein digestion and absorption; 05205 Proteoglycans in cancer; 05165 Human papillomavirus infection; 05146 Amoebiasis; 05415 Diabetic cardiomyopathy; 04933 AGE-RAGE signaling pathway in diabetic complications; 00536 Glycosaminoglycan binding proteins [BR:mmu00536]; Extracellular matrix molecules; ','[Pfam: Collagen COLFI VWC]'),('12737','mmu:12737','[Cldn1, AI596271]','[(RefSeq) claudin 1]','[claudin]','[Mus musculus (house mouse)]','[Cell adhesion molecules, Tight junction, Leukocyte transendothelial migration, Hepatitis C]','[]','[]','04514 Cell adhesion molecules; 04530 Tight junction; 04670 Leukocyte transendothelial migration; 05160 Hepatitis C; 03037 Cilium and associated proteins [BR:mmu03037]; 04147 Exosome [BR:mmu04147]; Stereociliary proteins; Exosomal proteins of ovarian cancer cells; Exosomal proteins of colorectal cancer cells; ','[Pfam: PMP22_Claudin Claudin_2 SUR7 Voltage_CLC CD20 Amastin DUF6336 FtsX DUF4134]'),('21786','mmu:21786','[Tff3, ITF, mITF]','[(RefSeq) trefoil factor 3, intestinal]','[trefoil factor 3]','[Mus musculus (house mouse)]',NULL,'[]','[]','99992 Structural proteins; ','[Pfam: Trefoil DUF6666 TEX29]'),('57263','mmu:57263','[Retnlb, 9030012B21Rik, Fizz2, RELMbeta, Relmb, Xcp3]','[(RefSeq) resistin like beta]',NULL,'[Mus musculus (house mouse)]',NULL,'[]','[]',NULL,'[Pfam: Resistin]'),('272411','mmu:272411','[B3gnt6, BC039789, Core-3, Core3]','[(RefSeq) UDP-GlcNAc:betaGal beta-1,3-N-acetylglucosaminyltransferase 6 (core 3 synthase)]','[acetylgalactosaminyl-O-glycosyl-glycoprotein beta-1,3-N-acetylglucosaminyltransferase [EC:2.4.1.147]]','[Mus musculus (house mouse)]','[Mucin type O-glycan biosynthesis, Metabolic pathways]','[]','[]','00512 Mucin type O-glycan biosynthesis; 01003 Glycosyltransferases [BR:mmu01003]; 2.4.1.147  acetylgalactosaminyl-O-glycosyl-glycoprotein beta-1,3-N-acetylglucosaminyltransferase; Type 2 chain/Poly-N-acetyllactosamine; ','[Pfam: Galactosyl_T Fringe]'),('14537','mmu:14537','[Gcnt1, 5630400D21Rik, B130048E03, C2_GlcNAcT, C2GNT, IGnT]','[(RefSeq) glucosaminyl (N-acetyl) transferase 1, core 2]','[beta-1,3-galactosyl-O-glycosyl-glycoprotein beta-1,6-N-acetylglucosaminyltransferase [EC:2.4.1.102]]','[Mus musculus (house mouse)]','[Mucin type O-glycan biosynthesis, Metabolic pathways]','[]','[]','00512 Mucin type O-glycan biosynthesis; 01003 Glycosyltransferases [BR:mmu01003]; 2.4.1.102  beta-1,3-galactosyl-O-glycosyl-glycoprotein beta-1,6-N-acetylglucosaminyltransferase; O-Glycan; ','[Pfam: Branch Exop_C]'),('14344','mmu:14344','[Fut2, MFUT-II]','[(RefSeq) fucosyltransferase 2]','[galactoside 2-L-fucosyltransferase 1/2 [EC:2.4.1.69]]','[Mus musculus (house mouse)]','[Glycosphingolipid biosynthesis - lacto and neolacto series, Glycosphingolipid biosynthesis - globo and isoglobo series, Metabolic pathways]','[]','[]','00601 Glycosphingolipid biosynthesis - lacto and neolacto series; 00603 Glycosphingolipid biosynthesis - globo and isoglobo series; 01003 Glycosyltransferases [BR:mmu01003]; 04131 Membrane trafficking [BR:mmu04131]; 2.4.1.69  type 1 galactoside alpha-(1,2)-fucosyltransferase; ABO blood group; Fucosyltransferase; ','[Pfam: Glyco_transf_11]'),('20440','mmu:20440','[St6gal1, AW742324, Siat1, St6Gal-I, St6gal, St6galI]','[(RefSeq) beta galactoside alpha 2,6 sialyltransferase 1]','[beta-galactoside alpha-2,6-sialyltransferase (sialyltransferase 1) [EC:2.4.3.1]]','[Mus musculus (house mouse)]','[N-Glycan biosynthesis, Other types of O-glycan biosynthesis, Metabolic pathways]','[]','[]','00510 N-Glycan biosynthesis; 00514 Other types of O-glycan biosynthesis; 01003 Glycosyltransferases [BR:mmu01003]; 2.4.3.1  beta-galactoside alpha-(2,6)-sialyltransferase; Sialyltransferase; ','[Pfam: Glyco_transf_29]'),('77040','mmu:77040','[Atg16l1, 1500009K01Rik, Apg16l, Atg16l, WDR30]','[(RefSeq) autophagy related 16-like 1 (S. cerevisiae)]','[autophagy-related protein 16-1]','[Mus musculus (house mouse)]','[Autophagy - other, Autophagy - animal, NOD-like receptor signaling pathway]','[]','[]','04140 Autophagy - animal; 04136 Autophagy - other; 04621 NOD-like receptor signaling pathway; 04131 Membrane trafficking [BR:mmu04131]; 03029 Mitochondrial biogenesis [BR:mmu03029]; Atg12 conjugation proteins; Auto phagosome; Autophagy-related proteins; ','[Pfam: ATG16 WD40 ANAPC4_WD40 eIF2A NBCH_WD40 KASH_CCD TolB_like]'),('11793','mmu:11793','[Atg5, 2010107M05Rik, 3110067M24Rik, AW319544, Apg5l, Atg5l, C88337, Paddy]','[(RefSeq) autophagy related 5]','[autophagy-related protein 5]','[Mus musculus (house mouse)]','[Autophagy - other, Mitophagy - animal, Autophagy - animal, Longevity regulating pathway, Longevity regulating pathway - multiple species, Ferroptosis, NOD-like receptor signaling pathway, RIG-I-like receptor signaling pathway]','[]','[]','04140 Autophagy - animal; 04136 Autophagy - other; 04137 Mitophagy - animal; 04216 Ferroptosis; 04621 NOD-like receptor signaling pathway; 04622 RIG-I-like receptor signaling pathway; 04211 Longevity regulating pathway; 04213 Longevity regulating pathway - multiple species; 04131 Membrane trafficking [BR:mmu04131]; 03029 Mitochondrial biogenesis [BR:mmu03029]; Atg12 conjugation proteins; Autophagy-related proteins; ','[Pfam: APG5 DUF6782]'),('56208','mmu:56208','[Becn1, Atg6]','[(RefSeq) beclin 1, autophagy related]','[beclin]','[Mus musculus (house mouse)]','[Autophagy - other, Mitophagy - animal, Autophagy - animal, Apoptosis - multiple species, Apelin signaling pathway, Alzheimer disease, Amyotrophic lateral sclerosis, Huntington disease, Spinocerebellar ataxia, Pathways of neurodegeneration - multiple diseases, Kaposi sarcoma-associated herpesvirus infection]','[]','[]','04371 Apelin signaling pathway; 04140 Autophagy - animal; 04136 Autophagy - other; 04137 Mitophagy - animal; 04215 Apoptosis - multiple species; 05167 Kaposi sarcoma-associated herpesvirus infection; 05010 Alzheimer disease; 05014 Amyotrophic lateral sclerosis; 05016 Huntington disease; 05017 Spinocerebellar ataxia; 05022 Pathways of neurodegeneration - multiple diseases; 04131 Membrane trafficking [BR:mmu04131]; 03029 Mitochondrial biogenesis [BR:mmu03029]; PI3K complex; Autophagy-related proteins; ','[Pfam: APG6 APG6_N BH3 MT DUF3450 HAUS-augmin3 Exonuc_VII_L V_ATPase_I]'),('16784','mmu:16784','[Lamp2, CD107b, LGP-B, Lamp_II, Lamp-2, Lamp-2a, Lamp-2b, Lamp-2c, Mac3]','[(RefSeq) lysosomal-associated membrane protein 2]','[lysosomal-associated membrane protein 1/2]','[Mus musculus (house mouse)]','[Autophagy - animal, Lysosome, Phagosome, Tuberculosis]','[]','[]','04145 Phagosome; 04142 Lysosome; 04140 Autophagy - animal; 05152 Tuberculosis; 04131 Membrane trafficking [BR:mmu04131]; 04147 Exosome [BR:mmu04147]; 04090 CD molecules [BR:mmu04090]; Cargo receptors; Exosomal proteins of hepatic cells; Proteins; ','[Pfam: Lamp VAS1_LD]'),('66734','mmu:66734','[Map1lc3a, 1010001H21Rik, 4922501H04Rik, LC3, LC3a]','[(RefSeq) microtubule-associated protein 1 light chain 3 alpha]','[microtubule-associated protein 1 light chain]','[Mus musculus (house mouse)]','[Mitophagy - animal, Autophagy - animal, Ferroptosis, Apelin signaling pathway, NOD-like receptor signaling pathway, Amyotrophic lateral sclerosis, Pathways of neurodegeneration - multiple diseases, Kaposi sarcoma-associated herpesvirus infection]','[]','[]','04371 Apelin signaling pathway; 04140 Autophagy - animal; 04137 Mitophagy - animal; 04216 Ferroptosis; 04621 NOD-like receptor signaling pathway; 05167 Kaposi sarcoma-associated herpesvirus infection; 05014 Amyotrophic lateral sclerosis; 05022 Pathways of neurodegeneration - multiple diseases; 04131 Membrane trafficking [BR:mmu04131]; 03029 Mitochondrial biogenesis [BR:mmu03029]; 04812 Cytoskeleton proteins [BR:mmu04812]; Other mitophagy associated  proteins; Other mitophagy factors; Microtubule-associated proteins (MAPs); ','[Pfam: ATG8 APG12]'),('50929','mmu:50929','[Il22, IL-22, IL-22a, ILTIFa, If2b1, Iltif]','[(RefSeq) interleukin 22]','[interleukin 22]','[Mus musculus (house mouse)]','[Cytokine-cytokine receptor interaction, JAK-STAT signaling pathway, Th17 cell differentiation, Inflammatory bowel disease]','[]','[]','04630 JAK-STAT signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04659 Th17 cell differentiation; 05321 Inflammatory bowel disease; 04052 Cytokines and growth factors [BR:mmu04052]; Interleukins; ','[Pfam: IL22 IL10 BshC]'),('77125','mmu:77125','[Il33, 9230117N10Rik, Il-33, Il1f11, NF-HEV]','[(RefSeq) interleukin 33]','[interleukin 33]','[Mus musculus (house mouse)]','[Cytokine-cytokine receptor interaction, Necroptosis, Cytosolic DNA-sensing pathway, Influenza A]','[]','[]','04060 Cytokine-cytokine receptor interaction; 04217 Necroptosis; 04623 Cytosolic DNA-sensing pathway; 05164 Influenza A; 04052 Cytokines and growth factors [BR:mmu04052]; Interleukins; ','[Pfam: IL33]'),('16163','mmu:16163','[Il13, Il-13]','[(RefSeq) interleukin 13]','[interleukin 13]','[Mus musculus (house mouse)]','[Cytokine-cytokine receptor interaction, JAK-STAT signaling pathway, IL-17 signaling pathway, Th1 and Th2 cell differentiation, Fc epsilon RI signaling pathway, Pathways in cancer, Asthma, Inflammatory bowel disease]','[]','[]','04630 JAK-STAT signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04658 Th1 and Th2 cell differentiation; 04657 IL-17 signaling pathway; 04664 Fc epsilon RI signaling pathway; 05200 Pathways in cancer; 05310 Asthma; 05321 Inflammatory bowel disease; 04052 Cytokines and growth factors [BR:mmu04052]; Interleukins; ','[Pfam: IL13]'),('50490','mmu:50490','[Nox4, AI648021]','[(RefSeq) NADPH oxidase 4]','[NADPH oxidase 4 [EC:1.6.3.-]]','[Mus musculus (house mouse)]','[AGE-RAGE signaling pathway in diabetic complications, Alcoholic liver disease, Alzheimer disease, Pathways of neurodegeneration - multiple diseases, Chemical carcinogenesis - reactive oxygen species]','[]','[]','05208 Chemical carcinogenesis - reactive oxygen species; 05010 Alzheimer disease; 05022 Pathways of neurodegeneration - multiple diseases; 04936 Alcoholic liver disease; 04933 AGE-RAGE signaling pathway in diabetic complications; 1.6.3.-; ','[Pfam: Ferric_reduct NAD_binding_6 FAD_binding_8 NAD_binding_1]'),('16173','mmu:16173','[Il18, Igif, Il-18]','[(RefSeq) interleukin 18]','[interleukin 18]','[Mus musculus (house mouse)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, NOD-like receptor signaling pathway, Cytosolic DNA-sensing pathway, Salmonella infection, Legionellosis, Yersinia infection, African trypanosomiasis, Malaria, Tuberculosis, Influenza A, Inflammatory bowel disease, Rheumatoid arthritis, Lipid and atherosclerosis]','[]','[]','04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04621 NOD-like receptor signaling pathway; 04623 Cytosolic DNA-sensing pathway; 05164 Influenza A; 05132 Salmonella infection; 05135 Yersinia infection; 05134 Legionellosis; 05152 Tuberculosis; 05144 Malaria; 05143 African trypanosomiasis; 05323 Rheumatoid arthritis; 05321 Inflammatory bowel disease; 05417 Lipid and atherosclerosis; 04052 Cytokines and growth factors [BR:mmu04052]; Interleukins; ','[Pfam: IL1 DUF273 NCA2]'),('387176','mmu:387176','[Mir181a-2, Mirn181, Mirn181a, Mirn181a-2, miR-181, mir-181a, mir-181a-2]','[(RefSeq) microRNA 181a-2]','[microRNA 181a]','[Mus musculus (house mouse)]','[MicroRNAs in cancer]','[]','[]','05206 MicroRNAs in cancer; 04147 Exosome [BR:mmu04147]; 03100 Non-coding RNAs [BR:mmu03100]; Exosomal RNA of breast milk; Micro RNA; ','[]'),('15978','mmu:15978','[Ifng, IFN-g, If2f, Ifg]','[(RefSeq) interferon gamma]','[interferon gamma]','[Mus musculus (house mouse)]','[Proteasome, Cytokine-cytokine receptor interaction, HIF-1 signaling pathway, Necroptosis, TGF-beta signaling pathway, Osteoclast differentiation, Antigen processing and presentation, JAK-STAT signaling pathway, Natural killer cell mediated cytotoxicity, IL-17 signaling pathway, Th1 and Th2 cell differentiation, Th17 cell differentiation, T cell receptor signaling pathway, Type I diabetes mellitus, Leishmaniasis, Chagas disease, African trypanosomiasis, Malaria, Toxoplasmosis, Amoebiasis, Tuberculosis, Hepatitis C, Influenza A, Herpes simplex virus 1 infection, Pathways in cancer, PD-L1 expression and PD-1 checkpoint pathway in cancer, Inflammatory bowel disease, Systemic lupus erythematosus, Rheumatoid arthritis, Allograft rejection, Graft-versus-host disease, Fluid shear stress and atherosclerosis]','[]','[]','03050 Proteasome; 04350 TGF-beta signaling pathway; 04630 JAK-STAT signaling pathway; 04066 HIF-1 signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04217 Necroptosis; 04650 Natural killer cell mediated cytotoxicity; 04612 Antigen processing and presentation; 04660 T cell receptor signaling pathway; 04658 Th1 and Th2 cell differentiation; 04659 Th17 cell differentiation; 04657 IL-17 signaling pathway; 04380 Osteoclast differentiation; 05200 Pathways in cancer; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05160 Hepatitis C; 05164 Influenza A; 05168 Herpes simplex virus 1 infection; 05152 Tuberculosis; 05146 Amoebiasis; 05144 Malaria; 05145 Toxoplasmosis; 05140 Leishmaniasis; 05142 Chagas disease; 05143 African trypanosomiasis; 05322 Systemic lupus erythematosus; 05323 Rheumatoid arthritis; 05321 Inflammatory bowel disease; 05330 Allograft rejection; 05332 Graft-versus-host disease; 05418 Fluid shear stress and atherosclerosis; 04940 Type I diabetes mellitus; 03051 Proteasome [BR:mmu03051]; 04052 Cytokines and growth factors [BR:mmu04052]; 00536 Glycosaminoglycan binding proteins [BR:mmu00536]; Other assembling factors; Interferons; Cytokines; ','[Pfam: IFN-gamma]'),('14939','mmu:14939','[Gzmb, AI553453, CCP-1/C11, CCP1, Ctla-1, Ctla1, GZB]','[(RefSeq) granzyme B]','[granzyme B [EC:3.4.21.79]]','[Mus musculus (house mouse)]','[Apoptosis, Natural killer cell mediated cytotoxicity, Type I diabetes mellitus, Transcriptional misregulation in cancer, Autoimmune thyroid disease, Allograft rejection, Graft-versus-host disease]','[]','[]','04210 Apoptosis; 04650 Natural killer cell mediated cytotoxicity; 05202 Transcriptional misregulation in cancer; 05320 Autoimmune thyroid disease; 05330 Allograft rejection; 05332 Graft-versus-host disease; 04940 Type I diabetes mellitus; 01002 Peptidases and inhibitors [BR:mmu01002]; 3.4.21.79  granzyme B; Family S1: chymotrypsin family; ','[Pfam: Trypsin Trypsin_2 DUF1986 Pico_P2A]'),('20309','mmu:20309','[Cxcl15, Il8, Scyb15, lungkine, weche]','[(RefSeq) chemokine (C-X-C motif) ligand 15]','[C-X-C motif chemokine 15]','[Mus musculus (house mouse)]','[Cytokine-cytokine receptor interaction, Chemokine signaling pathway]','[]','[]','04060 Cytokine-cytokine receptor interaction; 04062 Chemokine signaling pathway; 04052 Cytokines and growth factors [BR:mmu04052]; Chemokines; ','[Pfam: IL8]'),('213435','mmu:213435','[Mylk3, D830007F02Rik, MLCK]','[(RefSeq) myosin light chain kinase 3]','[myosin-light-chain kinase [EC:2.7.11.18]]','[Mus musculus (house mouse)]','[Calcium signaling pathway, cGMP-PKG signaling pathway, Vascular smooth muscle contraction, Apelin signaling pathway, Focal adhesion, Platelet activation, Regulation of actin cytoskeleton, Oxytocin signaling pathway, Gastric acid secretion]','[]','[]','04371 Apelin signaling pathway; 04020 Calcium signaling pathway; 04022 cGMP-PKG signaling pathway; 04510 Focal adhesion; 04810 Regulation of actin cytoskeleton; 04611 Platelet activation; 04921 Oxytocin signaling pathway; 04270 Vascular smooth muscle contraction; 04971 Gastric acid secretion; 01001 Protein kinases [BR:mmu01001]; 2.7.11.18  myosin-light-chain kinase; MLCK family; ','[Pfam: Pkinase PK_Tyr_Ser-Thr ABC1 Kinase-like Kdo APH TRPM_tetra]'),('14526','mmu:14526','[Gcg, GLP-1, Glu, PPG]','[(RefSeq) glucagon]','[glucagon]','[Mus musculus (house mouse)]','[cAMP signaling pathway, Neuroactive ligand-receptor interaction, Thermogenesis, Insulin secretion, Glucagon signaling pathway]','[]','[]','04024 cAMP signaling pathway; 04080 Neuroactive ligand-receptor interaction; 04911 Insulin secretion; 04922 Glucagon signaling pathway; 04714 Thermogenesis; ','[Pfam: Hormone_2 Lipoprotein_10 MPM1]'),('216799','mmu:216799','[Nlrp3, AGTAVPRL, AII/AVP, Cias1, FCAS, FCU, MWS, Mmig1, NALP3, Pypaf1]','[(RefSeq) NLR family, pyrin domain containing 3]','[NACHT, LRR and PYD domains-containing protein 3]','[Mus musculus (house mouse)]','[Necroptosis, NOD-like receptor signaling pathway, C-type lectin receptor signaling pathway, Salmonella infection, Pertussis, Yersinia infection, Influenza A, Coronavirus disease - COVID-19, Lipid and atherosclerosis]','[]','[]','04217 Necroptosis; 04621 NOD-like receptor signaling pathway; 04625 C-type lectin receptor signaling pathway; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05132 Salmonella infection; 05135 Yersinia infection; 05133 Pertussis; 05417 Lipid and atherosclerosis; 04131 Membrane trafficking [BR:mmu04131]; 04054 Pattern recognition receptors [BR:mmu04054]; TRIM-directed selective autophagy; NOD-like receptors; ','[Pfam: NACHT NLRC4_HD2 PYRIN LRR_6 NOD2_WH FISNA LRR_4 NB-ARC DUF3747 AAA_25]'),('101613','mmu:101613','[Nlrp6, AI504961, Avr, Nalp6, Navr, Navr/Avr, Non-AVR, Pypaf5]','[(RefSeq) NLR family, pyrin domain containing 6]','[NACHT, LRR and PYD domains-containing protein 6]','[Mus musculus (house mouse)]','[NOD-like receptor signaling pathway]','[]','[]','04621 NOD-like receptor signaling pathway; 04054 Pattern recognition receptors [BR:mmu04054]; NOD-like receptors; ','[Pfam: NACHT PYRIN NLRC4_HD2 NOD2_WH AAA_16 LRR_6 DUF3764 NB-ARC]'),('21898','mmu:21898','[Tlr4, Lps, Ly87, Ran/M1, Rasl2-8]','[(RefSeq) toll-like receptor 4]','[toll-like receptor 4]','[Mus musculus (house mouse)]','[NF-kappa B signaling pathway, HIF-1 signaling pathway, Phagosome, PI3K-Akt signaling pathway, Necroptosis, Neutrophil extracellular trap formation, Toll-like receptor signaling pathway, NOD-like receptor signaling pathway, Alcoholic liver disease, Salmonella infection, Pertussis, Legionellosis, Yersinia infection, Leishmaniasis, Chagas disease, Malaria, Toxoplasmosis, Amoebiasis, Tuberculosis, Hepatitis B, Measles, Influenza A, Human immunodeficiency virus 1 infection, Coronavirus disease - COVID-19, Proteoglycans in cancer, PD-L1 expression and PD-1 checkpoint pathway in cancer, Inflammatory bowel disease, Rheumatoid arthritis, Lipid and atherosclerosis]','[]','[]','04064 NF-kappa B signaling pathway; 04066 HIF-1 signaling pathway; 04151 PI3K-Akt signaling pathway; 04145 Phagosome; 04217 Necroptosis; 04613 Neutrophil extracellular trap formation; 04620 Toll-like receptor signaling pathway; 04621 NOD-like receptor signaling pathway; 05205 Proteoglycans in cancer; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05162 Measles; 05132 Salmonella infection; 05135 Yersinia infection; 05133 Pertussis; 05134 Legionellosis; 05152 Tuberculosis; 05146 Amoebiasis; 05144 Malaria; 05145 Toxoplasmosis; 05140 Leishmaniasis; 05142 Chagas disease; 05323 Rheumatoid arthritis; 05321 Inflammatory bowel disease; 05417 Lipid and atherosclerosis; 04936 Alcoholic liver disease; 04131 Membrane trafficking [BR:mmu04131]; 04054 Pattern recognition receptors [BR:mmu04054]; 04090 CD molecules [BR:mmu04090]; 00536 Glycosaminoglycan binding proteins [BR:mmu00536]; Toll-like receptors; Toll-like receptors; Proteins; Cell surface receptors; ','[Pfam: LRR_8 TIR LRR_4 TIR_2 DUF1863]'),('19099','mmu:19099','[Mapk8ip1, IB1, JIP-1, Jip1, Prkm8ip, Skip, mjip-2a]','[(RefSeq) mitogen-activated protein kinase 8 interacting protein 1]','[mitogen-activated protein kinase 8 interacting protein 1]','[Mus musculus (house mouse)]','[MAPK signaling pathway]','[]','[]','04010 MAPK signaling pathway; 04131 Membrane trafficking [BR:mmu04131]; LDLR adaptor proteins; ','[Pfam: PID SH3_9 SH3_1 PID_2 SH3_2]'),('84112','mmu:84112','[Sucnr1, Gpr91]','[(RefSeq) succinate receptor 1]','[succinate receptor 1]','[Mus musculus (house mouse)]','[cAMP signaling pathway]','[]','[]','04024 cAMP signaling pathway; 04030 G protein-coupled receptors [BR:mmu04030]; Purine / pyrimidine; ','[Pfam: 7tm_1]'),('13239','mmu:13239','[Defa5, Defa29, Defcr29, Defcr5]','[(RefSeq) defensin, alpha, 5]','[defensin, alpha]','[Mus musculus (house mouse)]','[NOD-like receptor signaling pathway, Staphylococcus aureus infection, Transcriptional misregulation in cancer]','[]','[]','04621 NOD-like receptor signaling pathway; 05202 Transcriptional misregulation in cancer; 05150 Staphylococcus aureus infection; ','[Pfam: Defensin_propep Defensin_1 MF_alpha_N]'),('11475','mmu:11475','[Acta2, 0610041G09Rik, Actvs, SMAalpha, SMalphaA, a-SMA, alphaSMA]','[(RefSeq) actin alpha 2, smooth muscle, aorta]','[actin, aortic smooth muscle]','[Mus musculus (house mouse)]','[Vascular smooth muscle contraction, Apelin signaling pathway, Relaxin signaling pathway]','[]','[]','04371 Apelin signaling pathway; 04926 Relaxin signaling pathway; 04270 Vascular smooth muscle contraction; 04812 Cytoskeleton proteins [BR:mmu04812]; 04147 Exosome [BR:mmu04147]; Actins; Exosomal proteins of colorectal cancer cells; ','[Pfam: Actin]'),('21857','mmu:21857','[Timp1, Clgi, EPA, TIMP-1, TPA-S1, Timp]','[(RefSeq) tissue inhibitor of metalloproteinase 1]','[metalloproteinase inhibitor 1]','[Mus musculus (house mouse)]','[HIF-1 signaling pathway]','[]','[]','04066 HIF-1 signaling pathway; 01002 Peptidases and inhibitors [BR:mmu01002]; Family I35: timp family; ','[Pfam: TIMP NTR DUF3906]'),('19016','mmu:19016','[Pparg, Nr1c3, PPAR-gamma, PPAR-gamma2, PPARgamma, PPARgamma2]','[(RefSeq) peroxisome proliferator activated receptor gamma]','[peroxisome proliferator-activated receptor gamma]','[Mus musculus (house mouse)]','[PPAR signaling pathway, AMPK signaling pathway, Longevity regulating pathway, Osteoclast differentiation, Thermogenesis, Non-alcoholic fatty liver disease, Huntington disease, Pathways in cancer, Transcriptional misregulation in cancer, Thyroid cancer, Lipid and atherosclerosis]','[]','[]','04152 AMPK signaling pathway; 03320 PPAR signaling pathway; 04380 Osteoclast differentiation; 04211 Longevity regulating pathway; 04714 Thermogenesis; 05200 Pathways in cancer; 05202 Transcriptional misregulation in cancer; 05216 Thyroid cancer; 05016 Huntington disease; 05417 Lipid and atherosclerosis; 04932 Non-alcoholic fatty liver disease; 03000 Transcription factors [BR:mmu03000]; 03310 Nuclear receptors [BR:mmu03310]; Cys4 thyroid hormone receptor-like; 1C. Peroxisome proliferator-activated receptor (PPAR); ','[Pfam: PPARgamma_N zf-C4 Hormone_recep Tmemb_55A]'),('18126','mmu:18126','[Nos2, MAC-NOS, NOS-II, Nos-2, Nos2a, i-NOS, iNOS]','[(RefSeq) nitric oxide synthase 2, inducible]','[nitric-oxide synthase, inducible [EC:1.14.13.39]]','[Mus musculus (house mouse)]','[Arginine biosynthesis, Arginine and proline metabolism, Metabolic pathways, Calcium signaling pathway, HIF-1 signaling pathway, Peroxisome, Apelin signaling pathway, Relaxin signaling pathway, Alzheimer disease, Amyotrophic lateral sclerosis, Pathways of neurodegeneration - multiple diseases, Pertussis, Leishmaniasis, Chagas disease, Toxoplasmosis, Amoebiasis, Tuberculosis, Pathways in cancer, Small cell lung cancer]','[]','[]','00220 Arginine biosynthesis; 00330 Arginine and proline metabolism; 04371 Apelin signaling pathway; 04066 HIF-1 signaling pathway; 04020 Calcium signaling pathway; 04146 Peroxisome; 04926 Relaxin signaling pathway; 05200 Pathways in cancer; 05222 Small cell lung cancer; 05133 Pertussis; 05152 Tuberculosis; 05146 Amoebiasis; 05145 Toxoplasmosis; 05140 Leishmaniasis; 05142 Chagas disease; 05010 Alzheimer disease; 05014 Amyotrophic lateral sclerosis; 05022 Pathways of neurodegeneration - multiple diseases; 1.14.13.39  nitric-oxide synthase (NADPH); ','[Pfam: NO_synthase FAD_binding_1 Flavodoxin_1 NAD_binding_1 NAD_binding_6 KCTD11_21_C]'),('243764','mmu:243764','[Chrm2, AChR-M2, Chrm-2, M2]','[(RefSeq) cholinergic receptor, muscarinic 2, cardiac]','[muscarinic acetylcholine receptor M2]','[Mus musculus (house mouse)]','[Calcium signaling pathway, cAMP signaling pathway, Neuroactive ligand-receptor interaction, PI3K-Akt signaling pathway, Cholinergic synapse, Regulation of actin cytoskeleton]','[]','[]','04020 Calcium signaling pathway; 04024 cAMP signaling pathway; 04151 PI3K-Akt signaling pathway; 04080 Neuroactive ligand-receptor interaction; 04810 Regulation of actin cytoskeleton; 04725 Cholinergic synapse; 04030 G protein-coupled receptors [BR:mmu04030]; Acetylcholine (muscarinic); ','[Pfam: 7tm_1 7tm_4]'),('24088','mmu:24088','[Tlr2, Ly105]','[(RefSeq) toll-like receptor 2]','[toll-like receptor 2]','[Mus musculus (house mouse)]','[Phagosome, PI3K-Akt signaling pathway, Neutrophil extracellular trap formation, Toll-like receptor signaling pathway, Salmonella infection, Legionellosis, Leishmaniasis, Chagas disease, Malaria, Toxoplasmosis, Amoebiasis, Tuberculosis, Hepatitis B, Measles, Herpes simplex virus 1 infection, Epstein-Barr virus infection, Human immunodeficiency virus 1 infection, Coronavirus disease - COVID-19, Proteoglycans in cancer, PD-L1 expression and PD-1 checkpoint pathway in cancer, Inflammatory bowel disease, Rheumatoid arthritis, Lipid and atherosclerosis]','[]','[]','04151 PI3K-Akt signaling pathway; 04145 Phagosome; 04613 Neutrophil extracellular trap formation; 04620 Toll-like receptor signaling pathway; 05205 Proteoglycans in cancer; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05171 Coronavirus disease - COVID-19; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05169 Epstein-Barr virus infection; 05132 Salmonella infection; 05134 Legionellosis; 05152 Tuberculosis; 05146 Amoebiasis; 05144 Malaria; 05145 Toxoplasmosis; 05140 Leishmaniasis; 05142 Chagas disease; 05323 Rheumatoid arthritis; 05321 Inflammatory bowel disease; 05417 Lipid and atherosclerosis; 04131 Membrane trafficking [BR:mmu04131]; 04147 Exosome [BR:mmu04147]; 04054 Pattern recognition receptors [BR:mmu04054]; 04090 CD molecules [BR:mmu04090]; 00536 Glycosaminoglycan binding proteins [BR:mmu00536]; Toll-like receptors; Exosomal proteins of breast milk; Toll-like receptors; Proteins; Cell surface receptors; ','[Pfam: LRR_8 LRR_4 TIR TIR_2 LRR_5 LRR_9 LRR_1 LRRCT DUF1863]'),('11622','mmu:11622','[Ahr, Ah, Ahh, Ahre, In, bHLHe76]','[(RefSeq) aryl-hydrocarbon receptor]','[aryl hydrocarbon receptor]','[Mus musculus (house mouse)]','[Th17 cell differentiation, Cushing syndrome, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species]','[]','[]','04659 Th17 cell differentiation; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 04934 Cushing syndrome; 03000 Transcription factors [BR:mmu03000]; Factors with PAS domain; ','[Pfam: PAS_3 PAS PAS_11 HLH PAS_4 PAS_9]'),('13076','mmu:13076','[Cyp1a1, AHH, AHRR, CP11, CYPIA1, P450-1]','[(RefSeq) cytochrome P450, family 1, subfamily a, polypeptide 1]','[cytochrome P450 family 1 subfamily A1 [EC:1.14.14.1]]','[Mus musculus (house mouse)]','[Steroid hormone biosynthesis, Tryptophan metabolism, Retinol metabolism, Metabolism of xenobiotics by cytochrome P450, Metabolic pathways, Ovarian steroidogenesis, Chemical carcinogenesis - DNA adducts, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species, Lipid and atherosclerosis]','[]','[]','00140 Steroid hormone biosynthesis; 00380 Tryptophan metabolism; 00830 Retinol metabolism; 00980 Metabolism of xenobiotics by cytochrome P450; 04913 Ovarian steroidogenesis; 05204 Chemical carcinogenesis - DNA adducts; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 05417 Lipid and atherosclerosis; 00199 Cytochrome P450 [BR:mmu00199]; 1.14.14.1  unspecific monooxygenase; CYP1 family; ','[Pfam: p450]'),('219990','hsa:219990','[OOSP2, OOSP2A, PLAC1L, TMEM122]','[(RefSeq) oocyte secreted protein 2]','[oocyte-secreted protein]','[Homo sapiens (human)]',NULL,'[]','[]','99994 Others; ','[]'),('15558','mmu:15558','[Htr2a, 5-HT-2, 5-HT-2A, E030013E04, Htr-2, Htr2]','[(RefSeq) 5-hydroxytryptamine (serotonin) receptor 2A]','[5-hydroxytryptamine receptor 2]','[Mus musculus (house mouse)]','[Calcium signaling pathway, Neuroactive ligand-receptor interaction, Gap junction, Serotonergic synapse, Inflammatory mediator regulation of TRP channels]','[]','[]','04020 Calcium signaling pathway; 04080 Neuroactive ligand-receptor interaction; 04540 Gap junction; 04726 Serotonergic synapse; 04750 Inflammatory mediator regulation of TRP channels; 04030 G protein-coupled receptors [BR:mmu04030]; Serotonin; ','[Pfam: 7tm_1 7TM_GPCR_Srsx 7tm_4 7TM_GPCR_Srx]'),('15562','mmu:15562','[Htr4, 5-HT-4, 5-HT4, 5-HT<4L>, 5HTR4]','[(RefSeq) 5 hydroxytryptamine (serotonin) receptor 4]','[5-hydroxytryptamine receptor 4]','[Mus musculus (house mouse)]','[Calcium signaling pathway, cAMP signaling pathway, Neuroactive ligand-receptor interaction, Serotonergic synapse]','[]','[]','04020 Calcium signaling pathway; 04024 cAMP signaling pathway; 04080 Neuroactive ligand-receptor interaction; 04726 Serotonergic synapse; 04030 G protein-coupled receptors [BR:mmu04030]; Serotonin; ','[Pfam: 7tm_1 7TM_GPCR_Srv DUF6249]'),('14160','mmu:14160','[Lgr5, FEX, Gpr49]','[(RefSeq) leucine rich repeat containing G protein coupled receptor 5]','[leucine-rich repeat-containing G protein-coupled receptor 5]','[Mus musculus (house mouse)]','[Wnt signaling pathway]','[]','[]','04310 Wnt signaling pathway; 04030 G protein-coupled receptors [BR:mmu04030]; R-spondin; ','[Pfam: LRR_8 LRR_4 LRR_5 LRR_1 LRR_9 7tm_1 LRRNT]'),('76768','mmu:76768','[Alpi, 2010001C14Rik, Akp6, gIAP]','[(RefSeq) alkaline phosphatase, intestinal]','[alkaline phosphatase [EC:3.1.3.1]]','[Mus musculus (house mouse)]','[Thiamine metabolism, Folate biosynthesis, Metabolic pathways, Biosynthesis of cofactors]','[]','[]','00730 Thiamine metabolism; 00790 Folate biosynthesis; 04147 Exosome [BR:mmu04147]; 00537 Glycosylphosphatidylinositol (GPI)-anchored proteins [BR:mmu00537]; 3.1.3.1  alkaline phosphatase; Exosomal proteins of other body fluids (saliva and urine); Enzymes; ','[Pfam: Alk_phosphatase Metalloenzyme]'),('17110','mmu:17110','[Lyz1, Lyz, Lyzf3, Lzp-s]','[(RefSeq) lysozyme 1]','[lysozyme C [EC:3.2.1.17]]','[Mus musculus (house mouse)]','[Salivary secretion]','[]','[]','04970 Salivary secretion; 04147 Exosome [BR:mmu04147]; 3.2.1.17  lysozyme; Exosomal proteins of other body fluids (saliva and urine); ','[Pfam: Lys SLT SLT_3 Destabilase Phage_lysozyme2]'),('12652','mmu:12652','[Chga, ChrA, cgA]','[(RefSeq) chromogranin A]','[chromogranin-A]','[Mus musculus (house mouse)]',NULL,'[]','[]','04131 Membrane trafficking [BR:mmu04131]; Others; ','[Pfam: Granin PP1_bind]'),('13175','mmu:13175','[Dclk1, 1700113D08Rik, 2810480F11Rik, AI836758, Click-I, Cpg16, Dcamkl1, Dcl, Dclk, mKIAA0369]','[(RefSeq) doublecortin-like kinase 1]','[doublecortin-like kinase 1/2 [EC:2.7.11.1]]','[Mus musculus (house mouse)]',NULL,'[]','[]','01001 Protein kinases [BR:mmu01001]; 2.7.11.1  non-specific serine/threonine protein kinase; DCAMKL family; ','[Pfam: Pkinase DCX PK_Tyr_Ser-Thr Kdo Pkinase_fungal]'),('18489','mmu:18489','[Reg3b, HIP, PAP1, Pap, REG-III]','[(RefSeq) regenerating islet-derived 3 beta]','[regenerating islet-derived protein 3]','[Mus musculus (house mouse)]',NULL,'[]','[]','99995 Signaling proteins; ','[Pfam: Lectin_C]'),('19695','mmu:19695','[Reg3g, AI449515, REG-3-gamma, reg_III-gamma]','[(RefSeq) regenerating islet-derived 3 gamma]','[regenerating islet-derived protein 3]','[Mus musculus (house mouse)]',NULL,'[]','[]','99995 Signaling proteins; ','[Pfam: Lectin_C]'),('13214','mmu:13214','[Defb1, AW260221, BD-1]','[(RefSeq) defensin beta 1]','[beta-defensin 1]','[Mus musculus (house mouse)]','[ABC transporters, Staphylococcus aureus infection]','[]','[]','02010 ABC transporters; 05150 Staphylococcus aureus infection; 02000 Transporters [BR:mmu02000]; 03037 Cilium and associated proteins [BR:mmu03037]; Pores ion channels [TC:1]; Sperm flagellum proteins; ','[Pfam: Defensin_beta]'),('18260','mmu:18260','[Ocln, AI503564, Ocl]','[(RefSeq) occludin]','[occludin]','[Mus musculus (house mouse)]','[Cell adhesion molecules, Tight junction, Leukocyte transendothelial migration, Hepatitis C]','[]','[]','04514 Cell adhesion molecules; 04530 Tight junction; 04670 Leukocyte transendothelial migration; 05160 Hepatitis C; 01009 Protein phosphatases and associated proteins [BR:mmu01009]; PP1-interacting proteins (PIPs); ','[Pfam: MARVEL Occludin_ELL HBM APG6_N GAS DUF4164 TMF_TATA_bd Laminin_II FapA Swi5 Anthrone_oxy]'),('12739','mmu:12739','[Cldn3, AI182374, Cpetr2, mRVP1]','[(RefSeq) claudin 3]','[claudin]','[Mus musculus (house mouse)]','[Cell adhesion molecules, Tight junction, Leukocyte transendothelial migration, Hepatitis C]','[]','[]','04514 Cell adhesion molecules; 04530 Tight junction; 04670 Leukocyte transendothelial migration; 05160 Hepatitis C; 03037 Cilium and associated proteins [BR:mmu03037]; 04147 Exosome [BR:mmu04147]; Stereociliary proteins; Exosomal proteins of ovarian cancer cells; Exosomal proteins of colorectal cancer cells; ','[Pfam: PMP22_Claudin Claudin_2 TMEM37]'),('13112','mmu:13112','[Cyp3a11, AI256190, Cyp3a, IIIAm1, Pcn]','[(RefSeq) cytochrome P450, family 3, subfamily a, polypeptide 11]','[cytochrome P450 family 3 subfamily A [EC:1.14.14.1]]','[Mus musculus (house mouse)]','[Steroid hormone biosynthesis, Linoleic acid metabolism, Retinol metabolism, Metabolic pathways, Chemical carcinogenesis - DNA adducts, Chemical carcinogenesis - receptor activation]','[]','[]','00140 Steroid hormone biosynthesis; 00591 Linoleic acid metabolism; 00830 Retinol metabolism; 05204 Chemical carcinogenesis - DNA adducts; 05207 Chemical carcinogenesis - receptor activation; 00199 Cytochrome P450 [BR:mmu00199]; 1.14.14.1  unspecific monooxygenase; CYP3 family; ','[Pfam: p450]'),('13088','mmu:13088','[Cyp2b10, Cyp2b, Cyp2b20, p16]','[(RefSeq) cytochrome P450, family 2, subfamily b, polypeptide 10]','[cytochrome P450 family 2 subfamily B [EC:1.14.14.1]]','[Mus musculus (house mouse)]','[Steroid hormone biosynthesis, Arachidonic acid metabolism, Retinol metabolism, Metabolic pathways, Chemical carcinogenesis - receptor activation]','[]','[]','00140 Steroid hormone biosynthesis; 00590 Arachidonic acid metabolism; 00830 Retinol metabolism; 05207 Chemical carcinogenesis - receptor activation; 00199 Cytochrome P450 [BR:mmu00199]; 1.14.14.1  unspecific monooxygenase; CYP2 family; ','[Pfam: p450 HRI1]'),('18671','mmu:18671','[Abcb1a, Abcb4, Evi32, Mdr1a, Mdr3, P-gp, Pgp, Pgy-3, Pgy3, mdr-3]','[(RefSeq) ATP-binding cassette, sub-family B (MDR/TAP), member 1A]','[ATP-binding cassette, subfamily B (MDR/TAP), member 1 [EC:7.6.2.2]]','[Mus musculus (house mouse)]','[ABC transporters, Bile secretion, MicroRNAs in cancer, Gastric cancer]','[]','[]','02010 ABC transporters; 04976 Bile secretion; 05206 MicroRNAs in cancer; 05226 Gastric cancer; 02000 Transporters [BR:mmu02000]; 04147 Exosome [BR:mmu04147]; 04090 CD molecules [BR:mmu04090]; 7.6.2.2  ABC-type xenobiotic transporter; ABCB1, 4, 5 subgroups; Exosomal proteins of other body fluids (saliva and urine); Proteins; ','[Pfam: ABC_membrane ABC_tran SMC_N ABC_ATPase AAA_16 AAA_22 Zeta_toxin Rad17 RsgA_GTPase G-alpha AAA AAA_5 AAA_29 AAA_7 AAA_30 APS_kinase PRK DUF815 AAA_15 NTPase_1 DUF2057]'),('13142','mmu:13142','[Dao, AI987963, DAAO, DAMOX, Dao-1, Dao1]','[(RefSeq) D-amino acid oxidase]','[D-amino-acid oxidase [EC:1.4.3.3]]','[Mus musculus (house mouse)]','[Glycine, serine and threonine metabolism, Arginine and proline metabolism, D-Amino acid metabolism, Metabolic pathways, Peroxisome]','[]','[]','00260 Glycine, serine and threonine metabolism; 00330 Arginine and proline metabolism; 00470 D-Amino acid metabolism; 00311 Penicillin and cephalosporin biosynthesis; 04146 Peroxisome; 1.4.3.3  D-amino-acid oxidase; ','[Pfam: DAO ADH_zinc_N_2 UDPG_MGDP_dh_N Glyco_trans_4_5]'),('546644','mmu:546644','[Ly6g, Gr-1, Gr1, Ly-6G]','[(RefSeq) lymphocyte antigen 6 complex, locus G]','[lymphocyte antigen 6 complex locus protein D/E/F/G6/H]','[Mus musculus (house mouse)]',NULL,'[]','[]','00537 Glycosylphosphatidylinositol (GPI)-anchored proteins [BR:mmu00537]; Antigens; ','[Pfam: UPAR_LY6 Toxin_TOLIP PLA2_inh]'),('16412','mmu:16412','[Itgb1, 4633401G24Rik, AA409975, AA960159, CD29, Fnrb, Gm9863, gpIIa]','[(RefSeq) integrin beta 1 (fibronectin receptor beta)]','[integrin beta 1]','[Mus musculus (house mouse)]','[Rap1 signaling pathway, Phagosome, PI3K-Akt signaling pathway, Axon guidance, Focal adhesion, ECM-receptor interaction, Cell adhesion molecules, Tight junction, Platelet activation, Leukocyte transendothelial migration, Regulation of actin cytoskeleton, Bacterial invasion of epithelial cells, Pertussis, Yersinia infection, Leishmaniasis, Toxoplasmosis, Human papillomavirus infection, Pathways in cancer, Proteoglycans in cancer, Small cell lung cancer, Hypertrophic cardiomyopathy, Arrhythmogenic right ventricular cardiomyopathy, Dilated cardiomyopathy]','[]','[]','04015 Rap1 signaling pathway; 04151 PI3K-Akt signaling pathway; 04512 ECM-receptor interaction; 04514 Cell adhesion molecules; 04145 Phagosome; 04510 Focal adhesion; 04530 Tight junction; 04810 Regulation of actin cytoskeleton; 04611 Platelet activation; 04670 Leukocyte transendothelial migration; 04360 Axon guidance; 05200 Pathways in cancer; 05205 Proteoglycans in cancer; 05222 Small cell lung cancer; 05165 Human papillomavirus infection; 05135 Yersinia infection; 05133 Pertussis; 05100 Bacterial invasion of epithelial cells; 05145 Toxoplasmosis; 05140 Leishmaniasis; 05410 Hypertrophic cardiomyopathy; 05412 Arrhythmogenic right ventricular cardiomyopathy; 05414 Dilated cardiomyopathy; 04131 Membrane trafficking [BR:mmu04131]; 04147 Exosome [BR:mmu04147]; 04515 Cell adhesion molecules [BR:mmu04515]; 04090 CD molecules [BR:mmu04090]; Integrins (complement receptors); Proteins found in most exosomes; Integrin beta subunits; Proteins; ','[Pfam: Integrin_beta Integrin_b_cyt Integrin_B_tail PSI_integrin I-EGF_1 EGF_2]'),('12362','mmu:12362','[Casp1, ICE, Il1bc]','[(RefSeq) caspase 1]','[caspase 1 [EC:3.4.22.36]]','[Mus musculus (house mouse)]','[Necroptosis, Neutrophil extracellular trap formation, NOD-like receptor signaling pathway, Cytosolic DNA-sensing pathway, C-type lectin receptor signaling pathway, Amyotrophic lateral sclerosis, Salmonella infection, Pertussis, Legionellosis, Yersinia infection, Influenza A, Coronavirus disease - COVID-19, Lipid and atherosclerosis]','[]','[]','04217 Necroptosis; 04613 Neutrophil extracellular trap formation; 04621 NOD-like receptor signaling pathway; 04623 Cytosolic DNA-sensing pathway; 04625 C-type lectin receptor signaling pathway; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05132 Salmonella infection; 05135 Yersinia infection; 05133 Pertussis; 05134 Legionellosis; 05014 Amyotrophic lateral sclerosis; 05417 Lipid and atherosclerosis; 01002 Peptidases and inhibitors [BR:mmu01002]; 04131 Membrane trafficking [BR:mmu04131]; 3.4.22.36  caspase-1; Family C14: caspase family; TRIM-directed selective autophagy; ','[Pfam: Peptidase_C14 CARD]'),('233079','mmu:233079','[Ffar2, GPCR43, Gpr43]','[(RefSeq) free fatty acid receptor 2]','[free fatty acid receptor 2]','[Mus musculus (house mouse)]','[cAMP signaling pathway]','[]','[]','04024 cAMP signaling pathway; 04030 G protein-coupled receptors [BR:mmu04030]; Free fatty acid; ','[Pfam: 7tm_1 7TM_GPCR_Srv 7TM_GPCR_Srx TAS2R]'),('216225','mmu:216225','[Slc5a8, Ait, SMCT]','[(RefSeq) solute carrier family 5 (iodide transporter), member 8]','[solute carrier family 5 (sodium-coupled monocarboxylate transporter), member 8/12]','[Mus musculus (house mouse)]',NULL,'[]','[]','02000 Transporters [BR:mmu02000]; SLC5: Sodium glucose cotransporter; ','[Pfam: SSF]'),('20186','mmu:20186','[Nr1h4, AI957360, Fxr, HRR1, RIP14, Rxrip14]','[(RefSeq) nuclear receptor subfamily 1, group H, member 4]','[farnesoid X receptor]','[Mus musculus (house mouse)]','[Bile secretion]','[]','[]','04976 Bile secretion; 03000 Transcription factors [BR:mmu03000]; 03310 Nuclear receptors [BR:mmu03310]; Cys4 thyroid hormone receptor-like; 1H. Liver X receptor like receptor; ','[Pfam: zf-C4 Hormone_recep]'),('14170','mmu:14170','[Fgf15, FGF19, Fgf8a]','[(RefSeq) fibroblast growth factor 15]','[fibroblast growth factor 19]','[Mus musculus (house mouse)]','[MAPK signaling pathway, Ras signaling pathway, Rap1 signaling pathway, Calcium signaling pathway, PI3K-Akt signaling pathway, Regulation of actin cytoskeleton, Pathways in cancer, Melanoma, Breast cancer, Gastric cancer]','[]','[]','04010 MAPK signaling pathway; 04014 Ras signaling pathway; 04015 Rap1 signaling pathway; 04020 Calcium signaling pathway; 04151 PI3K-Akt signaling pathway; 04810 Regulation of actin cytoskeleton; 05200 Pathways in cancer; 05226 Gastric cancer; 05218 Melanoma; 05224 Breast cancer; 04052 Cytokines and growth factors [BR:mmu04052]; 00536 Glycosaminoglycan binding proteins [BR:mmu00536]; Growth factors (RTK binding); Growth factors/receptors; ','[Pfam: FGF Fascin DUF3458]'),('14186','mmu:14186','[Fgfr4, Fgfr-4]','[(RefSeq) fibroblast growth factor receptor 4]','[fibroblast growth factor receptor 4 [EC:2.7.10.1]]','[Mus musculus (house mouse)]','[MAPK signaling pathway, Ras signaling pathway, Rap1 signaling pathway, Calcium signaling pathway, Endocytosis, PI3K-Akt signaling pathway, Signaling pathways regulating pluripotency of stem cells, Regulation of actin cytoskeleton, Pathways in cancer]','[]','[]','04010 MAPK signaling pathway; 04014 Ras signaling pathway; 04015 Rap1 signaling pathway; 04020 Calcium signaling pathway; 04151 PI3K-Akt signaling pathway; 04144 Endocytosis; 04550 Signaling pathways regulating pluripotency of stem cells; 04810 Regulation of actin cytoskeleton; 05200 Pathways in cancer; 01001 Protein kinases [BR:mmu01001]; 04090 CD molecules [BR:mmu04090]; 2.7.10.1  receptor protein-tyrosine kinase; FGFR family [OT]; Proteins; ','[Pfam: PK_Tyr_Ser-Thr Pkinase Ig_3 I-set Ig_2 ig V-set C2-set_2 Kinase-like Pkinase_fungal]'),('13122','mmu:13122','[Cyp7a1, CYPVII, CYPVIIc]','[(RefSeq) cytochrome P450, family 7, subfamily a, polypeptide 1]','[cholesterol 7alpha-monooxygenase [EC:1.14.14.23]]','[Mus musculus (house mouse)]','[Primary bile acid biosynthesis, Steroid hormone biosynthesis, Metabolic pathways, PPAR signaling pathway, Bile secretion, Cholesterol metabolism]','[]','[]','00120 Primary bile acid biosynthesis; 00140 Steroid hormone biosynthesis; 03320 PPAR signaling pathway; 04976 Bile secretion; 04979 Cholesterol metabolism; 00199 Cytochrome P450 [BR:mmu00199]; 1.14.14.23  cholesterol 7alpha-monooxygenase; CYP7 family; ','[Pfam: p450 DUF5790 DUF1072]'),('19261','mmu:19261','[Sirpa, AI835480, Bit, CD172a, Idd13.2, P84, Ptpns1, SHP-1, SHPS-1, SIRP]','[(RefSeq) signal-regulatory protein alpha]','[signal-regulatory protein alpha/beta1/gamma]','[Mus musculus (house mouse)]','[Osteoclast differentiation]','[]','[]','04380 Osteoclast differentiation; 04515 Cell adhesion molecules [BR:mmu04515]; 04090 CD molecules [BR:mmu04090]; SIRP family; Proteins; ','[Pfam: C1-set Ig_3 V-set C2-set_2 ig Ig_2 I-set SIT DUF6479]'),('12895','mmu:12895','[Cpt1b, Cpt1, Cpt1-m, Cpti, Cpti-m, M-cpti]','[(RefSeq) carnitine palmitoyltransferase 1b, muscle]','[carnitine O-palmitoyltransferase 1, muscle isoform [EC:2.3.1.21]]','[Mus musculus (house mouse)]','[Fatty acid degradation, Fatty acid metabolism, PPAR signaling pathway, AMPK signaling pathway, Thermogenesis, Adipocytokine signaling pathway, Glucagon signaling pathway, Insulin resistance, Alcoholic liver disease, Diabetic cardiomyopathy]','[]','[]','00071 Fatty acid degradation; 04152 AMPK signaling pathway; 04922 Glucagon signaling pathway; 04920 Adipocytokine signaling pathway; 03320 PPAR signaling pathway; 04714 Thermogenesis; 05415 Diabetic cardiomyopathy; 04936 Alcoholic liver disease; 04931 Insulin resistance; 03029 Mitochondrial biogenesis [BR:mmu03029]; 2.3.1.21  carnitine O-palmitoyltransferase; Other regulator of mitochondrial biogenesis; ','[Pfam: Carn_acyltransf CPT_N DUF3128]'),('14102','mmu:14102','[Fas, AI196731, APO1, APT1, CD95, TNFR6, Tnfrsf6, lpr]','[(RefSeq) Fas (TNF receptor superfamily member 6)]','[tumor necrosis factor receptor superfamily member 6]','[Mus musculus (house mouse)]','[Platinum drug resistance, MAPK signaling pathway, Cytokine-cytokine receptor interaction, p53 signaling pathway, Apoptosis, Necroptosis, Natural killer cell mediated cytotoxicity, TNF signaling pathway, Non-alcoholic fatty liver disease, Alcoholic liver disease, Type I diabetes mellitus, Alzheimer disease, Pathways of neurodegeneration - multiple diseases, Chagas disease, African trypanosomiasis, Hepatitis C, Hepatitis B, Measles, Human cytomegalovirus infection, Influenza A, Human papillomavirus infection, Kaposi sarcoma-associated herpesvirus infection, Herpes simplex virus 1 infection, Epstein-Barr virus infection, Human immunodeficiency virus 1 infection, Pathways in cancer, Proteoglycans in cancer, Autoimmune thyroid disease, Allograft rejection, Graft-versus-host disease, Lipid and atherosclerosis]','[]','[]','04010 MAPK signaling pathway; 04668 TNF signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04210 Apoptosis; 04217 Necroptosis; 04115 p53 signaling pathway; 04650 Natural killer cell mediated cytotoxicity; 05200 Pathways in cancer; 05205 Proteoglycans in cancer; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05160 Hepatitis C; 05164 Influenza A; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05165 Human papillomavirus infection; 05142 Chagas disease; 05143 African trypanosomiasis; 05320 Autoimmune thyroid disease; 05330 Allograft rejection; 05332 Graft-versus-host disease; 05010 Alzheimer disease; 05022 Pathways of neurodegeneration - multiple diseases; 05417 Lipid and atherosclerosis; 04940 Type I diabetes mellitus; 04936 Alcoholic liver disease; 04932 Non-alcoholic fatty liver disease; 01524 Platinum drug resistance; 04050 Cytokine receptors [BR:mmu04050]; 04090 CD molecules [BR:mmu04090]; TNF receptor superfamily; Proteins; ','[Pfam: Death TNFR_c6]'),('233080','mmu:233080','[Ffar3, Gm478, Gpr41]','[(RefSeq) free fatty acid receptor 3]','[free fatty acid receptor 3]','[Mus musculus (house mouse)]',NULL,'[]','[]','04030 G protein-coupled receptors [BR:mmu04030]; Free fatty acid; ','[Pfam: 7tm_1 7TM_GPCR_Srx]'),('18033','mmu:18033','[Nfkb1, NF-KB1, NF-kappaB, NF-kappaB1, p105, p50, p50/p105]','[(RefSeq) nuclear factor of kappa light polypeptide gene enhancer in B cells 1, p105]','[nuclear factor NF-kappa-B p105 subunit]','[Mus musculus (house mouse)]','[Antifolate resistance, MAPK signaling pathway, Ras signaling pathway, cAMP signaling pathway, Chemokine signaling pathway, NF-kappa B signaling pathway, HIF-1 signaling pathway, Sphingolipid signaling pathway, PI3K-Akt signaling pathway, Apoptosis, Longevity regulating pathway, Cellular senescence, Osteoclast differentiation, Neutrophil extracellular trap formation, Toll-like receptor signaling pathway, NOD-like receptor signaling pathway, RIG-I-like receptor signaling pathway, Cytosolic DNA-sensing pathway, C-type lectin receptor signaling pathway, IL-17 signaling pathway, Th1 and Th2 cell differentiation, Th17 cell differentiation, T cell receptor signaling pathway, B cell receptor signaling pathway, TNF signaling pathway, Neurotrophin signaling pathway, Prolactin signaling pathway, Adipocytokine signaling pathway, Relaxin signaling pathway, Insulin resistance, Non-alcoholic fatty liver disease, AGE-RAGE signaling pathway in diabetic complications, Alcoholic liver disease, Alzheimer disease, Pathways of neurodegeneration - multiple diseases, Cocaine addiction, Salmonella infection, Pertussis, Legionellosis, Yersinia infection, Leishmaniasis, Chagas disease, Toxoplasmosis, Amoebiasis, Tuberculosis, Hepatitis C, Hepatitis B, Measles, Human cytomegalovirus infection, Influenza A, Human papillomavirus infection, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Herpes simplex virus 1 infection, Epstein-Barr virus infection, Human immunodeficiency virus 1 infection, Coronavirus disease - COVID-19, Pathways in cancer, Transcriptional misregulation in cancer, Viral carcinogenesis, MicroRNAs in cancer, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species, Pancreatic cancer, Prostate cancer, Chronic myeloid leukemia, Acute myeloid leukemia, Small cell lung cancer, PD-L1 expression and PD-1 checkpoint pathway in cancer, Inflammatory bowel disease, Diabetic cardiomyopathy, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[]','[]','04010 MAPK signaling pathway; 04014 Ras signaling pathway; 04064 NF-kappa B signaling pathway; 04668 TNF signaling pathway; 04066 HIF-1 signaling pathway; 04071 Sphingolipid signaling pathway; 04024 cAMP signaling pathway; 04151 PI3K-Akt signaling pathway; 04210 Apoptosis; 04218 Cellular senescence; 04613 Neutrophil extracellular trap formation; 04620 Toll-like receptor signaling pathway; 04621 NOD-like receptor signaling pathway; 04622 RIG-I-like receptor signaling pathway; 04623 Cytosolic DNA-sensing pathway; 04625 C-type lectin receptor signaling pathway; 04660 T cell receptor signaling pathway; 04658 Th1 and Th2 cell differentiation; 04659 Th17 cell differentiation; 04657 IL-17 signaling pathway; 04662 B cell receptor signaling pathway; 04062 Chemokine signaling pathway; 04920 Adipocytokine signaling pathway; 04917 Prolactin signaling pathway; 04926 Relaxin signaling pathway; 04722 Neurotrophin signaling pathway; 04380 Osteoclast differentiation; 04211 Longevity regulating pathway; 05200 Pathways in cancer; 05202 Transcriptional misregulation in cancer; 05206 MicroRNAs in cancer; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 05203 Viral carcinogenesis; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05212 Pancreatic cancer; 05221 Acute myeloid leukemia; 05220 Chronic myeloid leukemia; 05215 Prostate cancer; 05222 Small cell lung cancer; 05166 Human T-cell leukemia virus 1 infection; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05160 Hepatitis C; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05165 Human papillomavirus infection; 05132 Salmonella infection; 05135 Yersinia infection; 05133 Pertussis; 05134 Legionellosis; 05152 Tuberculosis; 05146 Amoebiasis; 05145 Toxoplasmosis; 05140 Leishmaniasis; 05142 Chagas disease; 05321 Inflammatory bowel disease; 05010 Alzheimer disease; 05022 Pathways of neurodegeneration - multiple diseases; 05030 Cocaine addiction; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 05415 Diabetic cardiomyopathy; 04936 Alcoholic liver disease; 04932 Non-alcoholic fatty liver disease; 04931 Insulin resistance; 04933 AGE-RAGE signaling pathway in diabetic complications; 01523 Antifolate resistance; 03000 Transcription factors [BR:mmu03000]; RHR (Rel homology region) Rel/Ankyrin; ','[Pfam: RHD_DNA_bind RHD_dimer Ank_2 Ank_4 Ank Ank_3 Ank_5 Death]'),('58205','mmu:58205','[Pdcd1lg2, B7-DC, Btdc, F730015O22Rik, PD-L2]','[(RefSeq) programmed cell death 1 ligand 2]','[programmed cell death 1 ligand 2]','[Mus musculus (house mouse)]','[Cell adhesion molecules]','[]','[]','04514 Cell adhesion molecules; 04090 CD molecules [BR:mmu04090]; Proteins; ','[Pfam: Ig_2 Ig_3 I-set ig C2-set_2 C1-set PsbT]'),('12524','mmu:12524','[Cd86, B7, B7-2, B7.2, B70, CLS1, Cd28l2, ETC-1, Ly-58, Ly58, MB7, MB7-2, TS/A-2]','[(RefSeq) CD86 antigen]','[CD86 antigen]','[Mus musculus (house mouse)]','[Cell adhesion molecules, Toll-like receptor signaling pathway, Intestinal immune network for IgA production, Type I diabetes mellitus, Kaposi sarcoma-associated herpesvirus infection, Transcriptional misregulation in cancer, Autoimmune thyroid disease, Systemic lupus erythematosus, Rheumatoid arthritis, Allograft rejection, Graft-versus-host disease, Viral myocarditis]','[]','[]','04514 Cell adhesion molecules; 04620 Toll-like receptor signaling pathway; 04672 Intestinal immune network for IgA production; 05202 Transcriptional misregulation in cancer; 05167 Kaposi sarcoma-associated herpesvirus infection; 05322 Systemic lupus erythematosus; 05323 Rheumatoid arthritis; 05320 Autoimmune thyroid disease; 05330 Allograft rejection; 05332 Graft-versus-host disease; 05416 Viral myocarditis; 04940 Type I diabetes mellitus; 04147 Exosome [BR:mmu04147]; 04090 CD molecules [BR:mmu04090]; Exosomal proteins of haemopoietic cells  (B-cell, T-cell, DC-cell, reticulocyte, and mast cell); Proteins; ','[Pfam: V-set Ig_3 I-set V-set_CD47 C2-set_2 3keto-disac_hyd]'),('21939','mmu:21939','[Cd40, AI326936, Bp50, GP39, HIGM1, IGM, IMD3, T-BAM, TRAP, Tnfrsf5, p50]','[(RefSeq) CD40 antigen]','[tumor necrosis factor receptor superfamily member 5]','[Mus musculus (house mouse)]','[Cytokine-cytokine receptor interaction, NF-kappa B signaling pathway, Cell adhesion molecules, Toll-like receptor signaling pathway, Intestinal immune network for IgA production, Malaria, Toxoplasmosis, Human T-cell leukemia virus 1 infection, Epstein-Barr virus infection, Transcriptional misregulation in cancer, Asthma, Autoimmune thyroid disease, Systemic lupus erythematosus, Allograft rejection, Primary immunodeficiency, Viral myocarditis, Lipid and atherosclerosis]','[]','[]','04064 NF-kappa B signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04514 Cell adhesion molecules; 04620 Toll-like receptor signaling pathway; 04672 Intestinal immune network for IgA production; 05202 Transcriptional misregulation in cancer; 05166 Human T-cell leukemia virus 1 infection; 05169 Epstein-Barr virus infection; 05144 Malaria; 05145 Toxoplasmosis; 05310 Asthma; 05322 Systemic lupus erythematosus; 05320 Autoimmune thyroid disease; 05330 Allograft rejection; 05340 Primary immunodeficiency; 05417 Lipid and atherosclerosis; 05416 Viral myocarditis; 04050 Cytokine receptors [BR:mmu04050]; 04090 CD molecules [BR:mmu04090]; TNF receptor superfamily; Proteins; ','[Pfam: TNFR_c6 DUF3981]'),('16160','mmu:16160','[Il12b, Il-12b, Il-12p40, Il12p40, p40]','[(RefSeq) interleukin 12b]','[interleukin 12B]','[Mus musculus (house mouse)]','[Cytokine-cytokine receptor interaction, Toll-like receptor signaling pathway, RIG-I-like receptor signaling pathway, C-type lectin receptor signaling pathway, JAK-STAT signaling pathway, Th1 and Th2 cell differentiation, Alcoholic liver disease, Type I diabetes mellitus, Pertussis, Legionellosis, Leishmaniasis, Chagas disease, African trypanosomiasis, Toxoplasmosis, Amoebiasis, Tuberculosis, Measles, Influenza A, Herpes simplex virus 1 infection, Coronavirus disease - COVID-19, Pathways in cancer, Proteoglycans in cancer, Inflammatory bowel disease, Allograft rejection, Lipid and atherosclerosis]','[]','[]','04630 JAK-STAT signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04620 Toll-like receptor signaling pathway; 04622 RIG-I-like receptor signaling pathway; 04625 C-type lectin receptor signaling pathway; 04658 Th1 and Th2 cell differentiation; 05200 Pathways in cancer; 05205 Proteoglycans in cancer; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05133 Pertussis; 05134 Legionellosis; 05152 Tuberculosis; 05146 Amoebiasis; 05145 Toxoplasmosis; 05140 Leishmaniasis; 05142 Chagas disease; 05143 African trypanosomiasis; 05321 Inflammatory bowel disease; 05330 Allograft rejection; 05417 Lipid and atherosclerosis; 04940 Type I diabetes mellitus; 04936 Alcoholic liver disease; 04052 Cytokines and growth factors [BR:mmu04052]; 00536 Glycosaminoglycan binding proteins [BR:mmu00536]; Interleukins; Cytokines; ','[Pfam: IL12p40_C Ig_3 Ig_5 fn3 Ig_2 Ig_4 I-set]'),('80885','mmu:80885','[Hcar2, Gpr109a, Gpr109b, HM74, Niacr1, PUMA-G, Pumag, mHM74b]','[(RefSeq) hydroxycarboxylic acid receptor 2]','[hydroxycarboxylic acid receptor 2/3]','[Mus musculus (house mouse)]','[cAMP signaling pathway]','[]','[]','04024 cAMP signaling pathway; 04030 G protein-coupled receptors [BR:mmu04030]; Hydroxy-carboxylic acid; ','[Pfam: 7tm_1 SpoIIM]'),('16154','mmu:16154','[Il10ra, AW553859, CDw210, CDw210a, IL-10R1, IL-10RA, Il10r, mIL-10R]','[(RefSeq) interleukin 10 receptor, alpha]','[interleukin 10 receptor alpha]','[Mus musculus (house mouse)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, JAK-STAT signaling pathway, Toxoplasmosis, Tuberculosis, Human cytomegalovirus infection]','[]','[]','04630 JAK-STAT signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 05163 Human cytomegalovirus infection; 05152 Tuberculosis; 05145 Toxoplasmosis; 04050 Cytokine receptors [BR:mmu04050]; 04090 CD molecules [BR:mmu04090]; IL10/28 receptor family; Proteins; ','[Pfam: Tissue_fac Interfer-bind]'),('26417','mmu:26417','[Mapk3, Erk-1, Erk1, Ert2, Esrk1, Mnk1, Mtap2k, Prkm3, p44, p44erk1, p44mapk]','[(RefSeq) mitogen-activated protein kinase 3]','[mitogen-activated protein kinase 1/3 [EC:2.7.11.24]]','[Mus musculus (house mouse)]','[EGFR tyrosine kinase inhibitor resistance, Endocrine resistance, Platinum drug resistance, MAPK signaling pathway, ErbB signaling pathway, Ras signaling pathway, Rap1 signaling pathway, cGMP-PKG signaling pathway, cAMP signaling pathway, Chemokine signaling pathway, HIF-1 signaling pathway, FoxO signaling pathway, Sphingolipid signaling pathway, Phospholipase D signaling pathway, Oocyte meiosis, Autophagy - animal, mTOR signaling pathway, PI3K-Akt signaling pathway, Apoptosis, Cellular senescence, Adrenergic signaling in cardiomyocytes, Vascular smooth muscle contraction, TGF-beta signaling pathway, Axon guidance, VEGF signaling pathway, Apelin signaling pathway, Osteoclast differentiation, Focal adhesion, Adherens junction, Gap junction, Signaling pathways regulating pluripotency of stem cells, Platelet activation, Neutrophil extracellular trap formation, Toll-like receptor signaling pathway, NOD-like receptor signaling pathway, C-type lectin receptor signaling pathway, Natural killer cell mediated cytotoxicity, IL-17 signaling pathway, Th1 and Th2 cell differentiation, Th17 cell differentiation, T cell receptor signaling pathway, B cell receptor signaling pathway, Fc epsilon RI signaling pathway, Fc gamma R-mediated phagocytosis, TNF signaling pathway, Circadian entrainment, Long-term potentiation, Neurotrophin signaling pathway, Retrograde endocannabinoid signaling, Glutamatergic synapse, Cholinergic synapse, Serotonergic synapse, Long-term depression, Regulation of actin cytoskeleton, Insulin signaling pathway, GnRH signaling pathway, Progesterone-mediated oocyte maturation, Estrogen signaling pathway, Melanogenesis, Prolactin signaling pathway, Thyroid hormone signaling pathway, Oxytocin signaling pathway, Relaxin signaling pathway, Parathyroid hormone synthesis, secretion and action, GnRH secretion, Type II diabetes mellitus, AGE-RAGE signaling pathway in diabetic complications, Cushing syndrome, Growth hormone synthesis, secretion and action, Aldosterone-regulated sodium reabsorption, Alzheimer disease, Prion disease, Pathways of neurodegeneration - multiple diseases, Alcoholism, Salmonella infection, Pertussis, Yersinia infection, Leishmaniasis, Chagas disease, Toxoplasmosis, Tuberculosis, Hepatitis C, Hepatitis B, Human cytomegalovirus infection, Influenza A, Human papillomavirus infection, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Human immunodeficiency virus 1 infection, Coronavirus disease - COVID-19, Pathways in cancer, Viral carcinogenesis, Proteoglycans in cancer, MicroRNAs in cancer, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species, Colorectal cancer, Renal cell carcinoma, Pancreatic cancer, Endometrial cancer, Glioma, Prostate cancer, Thyroid cancer, Melanoma, Bladder cancer, Chronic myeloid leukemia, Acute myeloid leukemia, Non-small cell lung cancer, Breast cancer, Hepatocellular carcinoma, Gastric cancer, Central carbon metabolism in cancer, Choline metabolism in cancer, PD-L1 expression and PD-1 checkpoint pathway in cancer, Lipid and atherosclerosis]','[]','[]','04010 MAPK signaling pathway; 04012 ErbB signaling pathway; 04014 Ras signaling pathway; 04015 Rap1 signaling pathway; 04350 TGF-beta signaling pathway; 04370 VEGF signaling pathway; 04371 Apelin signaling pathway; 04668 TNF signaling pathway; 04066 HIF-1 signaling pathway; 04068 FoxO signaling pathway; 04072 Phospholipase D signaling pathway; 04071 Sphingolipid signaling pathway; 04024 cAMP signaling pathway; 04022 cGMP-PKG signaling pathway; 04151 PI3K-Akt signaling pathway; 04150 mTOR signaling pathway; 04140 Autophagy - animal; 04114 Oocyte meiosis; 04210 Apoptosis; 04218 Cellular senescence; 04510 Focal adhesion; 04520 Adherens junction; 04540 Gap junction; 04550 Signaling pathways regulating pluripotency of stem cells; 04810 Regulation of actin cytoskeleton; 04611 Platelet activation; 04613 Neutrophil extracellular trap formation; 04620 Toll-like receptor signaling pathway; 04621 NOD-like receptor signaling pathway; 04625 C-type lectin receptor signaling pathway; 04650 Natural killer cell mediated cytotoxicity; 04660 T cell receptor signaling pathway; 04658 Th1 and Th2 cell differentiation; 04659 Th17 cell differentiation; 04657 IL-17 signaling pathway; 04662 B cell receptor signaling pathway; 04664 Fc epsilon RI signaling pathway; 04666 Fc gamma R-mediated phagocytosis; 04062 Chemokine signaling pathway; 04910 Insulin signaling pathway; 04929 GnRH secretion; 04912 GnRH signaling pathway; 04915 Estrogen signaling pathway; 04914 Progesterone-mediated oocyte maturation; 04917 Prolactin signaling pathway; 04921 Oxytocin signaling pathway; 04926 Relaxin signaling pathway; 04935 Growth hormone synthesis, secretion and action; 04919 Thyroid hormone signaling pathway; 04928 Parathyroid hormone synthesis, secretion and action; 04916 Melanogenesis; 04261 Adrenergic signaling in cardiomyocytes; 04270 Vascular smooth muscle contraction; 04960 Aldosterone-regulated sodium reabsorption; 04724 Glutamatergic synapse; 04725 Cholinergic synapse; 04726 Serotonergic synapse; 04720 Long-term potentiation; 04730 Long-term depression; 04723 Retrograde endocannabinoid signaling; 04722 Neurotrophin signaling pathway; 04360 Axon guidance; 04380 Osteoclast differentiation; 04713 Circadian entrainment; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05205 Proteoglycans in cancer; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 05203 Viral carcinogenesis; 05230 Central carbon metabolism in cancer; 05231 Choline metabolism in cancer; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05210 Colorectal cancer; 05212 Pancreatic cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05214 Glioma; 05216 Thyroid cancer; 05221 Acute myeloid leukemia; 05220 Chronic myeloid leukemia; 05218 Melanoma; 05211 Renal cell carcinoma; 05219 Bladder cancer; 05215 Prostate cancer; 05213 Endometrial cancer; 05224 Breast cancer; 05223 Non-small cell lung cancer; 05166 Human T-cell leukemia virus 1 infection; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05160 Hepatitis C; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05165 Human papillomavirus infection; 05132 Salmonella infection; 05135 Yersinia infection; 05133 Pertussis; 05152 Tuberculosis; 05145 Toxoplasmosis; 05140 Leishmaniasis; 05142 Chagas disease; 05010 Alzheimer disease; 05020 Prion disease; 05022 Pathways of neurodegeneration - multiple diseases; 05034 Alcoholism; 05417 Lipid and atherosclerosis; 04930 Type II diabetes mellitus; 04933 AGE-RAGE signaling pathway in diabetic complications; 04934 Cushing syndrome; 01521 EGFR tyrosine kinase inhibitor resistance; 01524 Platinum drug resistance; 01522 Endocrine resistance; 01001 Protein kinases [BR:mmu01001]; 03036 Chromosome and associated proteins [BR:mmu03036]; 04147 Exosome [BR:mmu04147]; 2.7.11.24  mitogen-activated protein kinase; MAPK family [OT]; Protein kinases; Exosomal proteins of haemopoietic cells  (B-cell, T-cell, DC-cell, reticulocyte, and mast cell); ','[Pfam: Pkinase PK_Tyr_Ser-Thr ABC1 APH Haspin_kinase FTA2 Choline_kinase Kdo Pkinase_fungal]'),('26413','mmu:26413','[Mapk1, 9030612K14Rik, AA407128, AU018647, C78273, ERK, Erk2, MAPK2, PRKM2, Prkm1, p41mapk, p42mapk]','[(RefSeq) mitogen-activated protein kinase 1]','[mitogen-activated protein kinase 1/3 [EC:2.7.11.24]]','[Mus musculus (house mouse)]','[EGFR tyrosine kinase inhibitor resistance, Endocrine resistance, Platinum drug resistance, MAPK signaling pathway, ErbB signaling pathway, Ras signaling pathway, Rap1 signaling pathway, cGMP-PKG signaling pathway, cAMP signaling pathway, Chemokine signaling pathway, HIF-1 signaling pathway, FoxO signaling pathway, Sphingolipid signaling pathway, Phospholipase D signaling pathway, Oocyte meiosis, Autophagy - animal, mTOR signaling pathway, PI3K-Akt signaling pathway, Apoptosis, Cellular senescence, Adrenergic signaling in cardiomyocytes, Vascular smooth muscle contraction, TGF-beta signaling pathway, Axon guidance, VEGF signaling pathway, Apelin signaling pathway, Osteoclast differentiation, Focal adhesion, Adherens junction, Gap junction, Signaling pathways regulating pluripotency of stem cells, Platelet activation, Neutrophil extracellular trap formation, Toll-like receptor signaling pathway, NOD-like receptor signaling pathway, C-type lectin receptor signaling pathway, Natural killer cell mediated cytotoxicity, IL-17 signaling pathway, Th1 and Th2 cell differentiation, Th17 cell differentiation, T cell receptor signaling pathway, B cell receptor signaling pathway, Fc epsilon RI signaling pathway, Fc gamma R-mediated phagocytosis, TNF signaling pathway, Circadian entrainment, Long-term potentiation, Neurotrophin signaling pathway, Retrograde endocannabinoid signaling, Glutamatergic synapse, Cholinergic synapse, Serotonergic synapse, Long-term depression, Regulation of actin cytoskeleton, Insulin signaling pathway, GnRH signaling pathway, Progesterone-mediated oocyte maturation, Estrogen signaling pathway, Melanogenesis, Prolactin signaling pathway, Thyroid hormone signaling pathway, Oxytocin signaling pathway, Relaxin signaling pathway, Parathyroid hormone synthesis, secretion and action, GnRH secretion, Type II diabetes mellitus, AGE-RAGE signaling pathway in diabetic complications, Cushing syndrome, Growth hormone synthesis, secretion and action, Aldosterone-regulated sodium reabsorption, Alzheimer disease, Prion disease, Pathways of neurodegeneration - multiple diseases, Alcoholism, Salmonella infection, Pertussis, Yersinia infection, Leishmaniasis, Chagas disease, Toxoplasmosis, Tuberculosis, Hepatitis C, Hepatitis B, Human cytomegalovirus infection, Influenza A, Human papillomavirus infection, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Human immunodeficiency virus 1 infection, Coronavirus disease - COVID-19, Pathways in cancer, Viral carcinogenesis, Proteoglycans in cancer, MicroRNAs in cancer, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species, Colorectal cancer, Renal cell carcinoma, Pancreatic cancer, Endometrial cancer, Glioma, Prostate cancer, Thyroid cancer, Melanoma, Bladder cancer, Chronic myeloid leukemia, Acute myeloid leukemia, Non-small cell lung cancer, Breast cancer, Hepatocellular carcinoma, Gastric cancer, Central carbon metabolism in cancer, Choline metabolism in cancer, PD-L1 expression and PD-1 checkpoint pathway in cancer, Lipid and atherosclerosis]','[]','[]','04010 MAPK signaling pathway; 04012 ErbB signaling pathway; 04014 Ras signaling pathway; 04015 Rap1 signaling pathway; 04350 TGF-beta signaling pathway; 04370 VEGF signaling pathway; 04371 Apelin signaling pathway; 04668 TNF signaling pathway; 04066 HIF-1 signaling pathway; 04068 FoxO signaling pathway; 04072 Phospholipase D signaling pathway; 04071 Sphingolipid signaling pathway; 04024 cAMP signaling pathway; 04022 cGMP-PKG signaling pathway; 04151 PI3K-Akt signaling pathway; 04150 mTOR signaling pathway; 04140 Autophagy - animal; 04114 Oocyte meiosis; 04210 Apoptosis; 04218 Cellular senescence; 04510 Focal adhesion; 04520 Adherens junction; 04540 Gap junction; 04550 Signaling pathways regulating pluripotency of stem cells; 04810 Regulation of actin cytoskeleton; 04611 Platelet activation; 04613 Neutrophil extracellular trap formation; 04620 Toll-like receptor signaling pathway; 04621 NOD-like receptor signaling pathway; 04625 C-type lectin receptor signaling pathway; 04650 Natural killer cell mediated cytotoxicity; 04660 T cell receptor signaling pathway; 04658 Th1 and Th2 cell differentiation; 04659 Th17 cell differentiation; 04657 IL-17 signaling pathway; 04662 B cell receptor signaling pathway; 04664 Fc epsilon RI signaling pathway; 04666 Fc gamma R-mediated phagocytosis; 04062 Chemokine signaling pathway; 04910 Insulin signaling pathway; 04929 GnRH secretion; 04912 GnRH signaling pathway; 04915 Estrogen signaling pathway; 04914 Progesterone-mediated oocyte maturation; 04917 Prolactin signaling pathway; 04921 Oxytocin signaling pathway; 04926 Relaxin signaling pathway; 04935 Growth hormone synthesis, secretion and action; 04919 Thyroid hormone signaling pathway; 04928 Parathyroid hormone synthesis, secretion and action; 04916 Melanogenesis; 04261 Adrenergic signaling in cardiomyocytes; 04270 Vascular smooth muscle contraction; 04960 Aldosterone-regulated sodium reabsorption; 04724 Glutamatergic synapse; 04725 Cholinergic synapse; 04726 Serotonergic synapse; 04720 Long-term potentiation; 04730 Long-term depression; 04723 Retrograde endocannabinoid signaling; 04722 Neurotrophin signaling pathway; 04360 Axon guidance; 04380 Osteoclast differentiation; 04713 Circadian entrainment; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05205 Proteoglycans in cancer; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 05203 Viral carcinogenesis; 05230 Central carbon metabolism in cancer; 05231 Choline metabolism in cancer; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05210 Colorectal cancer; 05212 Pancreatic cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05214 Glioma; 05216 Thyroid cancer; 05221 Acute myeloid leukemia; 05220 Chronic myeloid leukemia; 05218 Melanoma; 05211 Renal cell carcinoma; 05219 Bladder cancer; 05215 Prostate cancer; 05213 Endometrial cancer; 05224 Breast cancer; 05223 Non-small cell lung cancer; 05166 Human T-cell leukemia virus 1 infection; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05160 Hepatitis C; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05165 Human papillomavirus infection; 05132 Salmonella infection; 05135 Yersinia infection; 05133 Pertussis; 05152 Tuberculosis; 05145 Toxoplasmosis; 05140 Leishmaniasis; 05142 Chagas disease; 05010 Alzheimer disease; 05020 Prion disease; 05022 Pathways of neurodegeneration - multiple diseases; 05034 Alcoholism; 05417 Lipid and atherosclerosis; 04930 Type II diabetes mellitus; 04933 AGE-RAGE signaling pathway in diabetic complications; 04934 Cushing syndrome; 01521 EGFR tyrosine kinase inhibitor resistance; 01524 Platinum drug resistance; 01522 Endocrine resistance; 01001 Protein kinases [BR:mmu01001]; 03036 Chromosome and associated proteins [BR:mmu03036]; 04147 Exosome [BR:mmu04147]; 2.7.11.24  mitogen-activated protein kinase; MAPK family [OT]; Protein kinases; Exosomal proteins of haemopoietic cells  (B-cell, T-cell, DC-cell, reticulocyte, and mast cell); ','[Pfam: Pkinase PK_Tyr_Ser-Thr ABC1 APH Haspin_kinase FTA2 Choline_kinase Kdo Pkinase_fungal Kinase-like]'),('18571','mmu:18571','[Pdcd6ip, AI480591, AW544830, Aip1, Alix, C76364, Eig2, mKIAA1375]','[(RefSeq) programmed cell death 6 interacting protein]','[programmed cell death 6-interacting protein]','[Mus musculus (house mouse)]','[Viral life cycle - HIV-1, Endocytosis]','[]','[]','03250 Viral life cycle - HIV-1; 04144 Endocytosis; 04131 Membrane trafficking [BR:mmu04131]; 04147 Exosome [BR:mmu04147]; ESCRT associated proteins; Proteins found in most exosomes; ','[Pfam: BRO1 ALIX_LYPXL_bnd AAT]'),('223774','mmu:223774','[Alg12, ECM39]','[(RefSeq) asparagine-linked glycosylation 12 (alpha-1,6-mannosyltransferase)]','[alpha-1,6-mannosyltransferase [EC:2.4.1.260]]','[Mus musculus (house mouse)]','[N-Glycan biosynthesis, Various types of N-glycan biosynthesis, Metabolic pathways]','[]','[]','00510 N-Glycan biosynthesis; 00513 Various types of N-glycan biosynthesis; 01003 Glycosyltransferases [BR:mmu01003]; 2.4.1.260  dolichyl-P-Man:Man7GlcNAc2-PP-dolichol alpha-1,6-mannosyltransferase; Dol-linked oligosaccharide; ','[Pfam: Glyco_transf_22]'),('113868','mmu:113868','[Acaa1a, Acaa, Acaa1, D9Ertd25e, PTL]','[(RefSeq) acetyl-Coenzyme A acyltransferase 1A]','[acetyl-CoA acyltransferase 1 [EC:2.3.1.16]]','[Mus musculus (house mouse)]','[Fatty acid degradation, Valine, leucine and isoleucine degradation, alpha-Linolenic acid metabolism, Biosynthesis of unsaturated fatty acids, Metabolic pathways, Fatty acid metabolism, PPAR signaling pathway, Peroxisome]','[]','[]','00071 Fatty acid degradation; 00592 alpha-Linolenic acid metabolism; 01040 Biosynthesis of unsaturated fatty acids; 00280 Valine, leucine and isoleucine degradation; 04146 Peroxisome; 03320 PPAR signaling pathway; 2.3.1.16  acetyl-CoA C-acyltransferase; ','[Pfam: Thiolase_N Thiolase_C ACP_syn_III_C ketoacyl-synt]'),('235674','mmu:235674','[Acaa1b]','[(RefSeq) acetyl-Coenzyme A acyltransferase 1B]','[acetyl-CoA acyltransferase 1 [EC:2.3.1.16]]','[Mus musculus (house mouse)]','[Fatty acid degradation, Valine, leucine and isoleucine degradation, alpha-Linolenic acid metabolism, Biosynthesis of unsaturated fatty acids, Metabolic pathways, Fatty acid metabolism, PPAR signaling pathway, Peroxisome]','[]','[]','00071 Fatty acid degradation; 00592 alpha-Linolenic acid metabolism; 01040 Biosynthesis of unsaturated fatty acids; 00280 Valine, leucine and isoleucine degradation; 04146 Peroxisome; 03320 PPAR signaling pathway; 2.3.1.16  acetyl-CoA C-acyltransferase; ','[Pfam: Thiolase_N Thiolase_C ACP_syn_III_C ketoacyl-synt SPDY]'),('52538','mmu:52538','[Acaa2, 0610011L04Rik, AI255831, AI265397, D18Ertd240e]','[(RefSeq) acetyl-Coenzyme A acyltransferase 2 (mitochondrial 3-oxoacyl-Coenzyme A thiolase)]','[acetyl-CoA acyltransferase 2 [EC:2.3.1.16]]','[Mus musculus (house mouse)]','[Fatty acid elongation, Fatty acid degradation, Valine, leucine and isoleucine degradation, Metabolic pathways, Fatty acid metabolism]','[]','[]','00062 Fatty acid elongation; 00071 Fatty acid degradation; 00280 Valine, leucine and isoleucine degradation; 2.3.1.16  acetyl-CoA C-acyltransferase; ','[Pfam: Thiolase_N Thiolase_C ketoacyl-synt IreB]'),('11409','mmu:11409','[Acads, AI196007, Bcd-1, Bcd1, Hdlq8, SCAD]','[(RefSeq) acyl-Coenzyme A dehydrogenase, short chain]','[butyryl-CoA dehydrogenase [EC:1.3.8.1]]','[Mus musculus (house mouse)]','[Fatty acid degradation, Valine, leucine and isoleucine degradation, beta-Alanine metabolism, Propanoate metabolism, Butanoate metabolism, Metabolic pathways, Carbon metabolism, Fatty acid metabolism]','[]','[]','00640 Propanoate metabolism; 00650 Butanoate metabolism; 00071 Fatty acid degradation; 00280 Valine, leucine and isoleucine degradation; 00410 beta-Alanine metabolism; 1.3.8.1  short-chain acyl-CoA dehydrogenase; ','[Pfam: Acyl-CoA_dh_1 Acyl-CoA_dh_N Acyl-CoA_dh_M Acyl-CoA_dh_2 HpaB_N]'),('266645','mmu:266645','[Acmsd]','[(RefSeq) amino carboxymuconate semialdehyde decarboxylase]','[aminocarboxymuconate-semialdehyde decarboxylase [EC:4.1.1.45]]','[Mus musculus (house mouse)]','[Tryptophan metabolism, Metabolic pathways]','[]','[]','00380 Tryptophan metabolism; 4.1.1.45  aminocarboxymuconate-semialdehyde decarboxylase; ','[Pfam: Amidohydro_2 SPW]'),('26897','mmu:26897','[Acot1, ACH2, CTE-1, CTE-I, Cte1, D12Ucla1]','[(RefSeq) acyl-CoA thioesterase 1]','[acyl-coenzyme A thioesterase 1/2/4 [EC:3.1.2.2]]','[Mus musculus (house mouse)]','[Fatty acid elongation, Biosynthesis of unsaturated fatty acids, Metabolic pathways, Ovarian steroidogenesis]','[]','[]','00062 Fatty acid elongation; 01040 Biosynthesis of unsaturated fatty acids; 04913 Ovarian steroidogenesis; 01004 Lipid biosynthesis proteins [BR:mmu01004]; 3.1.2.2  palmitoyl-CoA hydrolase; Component type; ','[Pfam: BAAT_C Bile_Hydr_Trans DLH Peptidase_S9 FSH1 AXE1]'),('171281','mmu:171281','[Acot3, AW108394, PTE-Ia, Pte2a]','[(RefSeq) acyl-CoA thioesterase 3]','[acyl-coenzyme A thioesterase 1/2/4 [EC:3.1.2.2]]','[Mus musculus (house mouse)]','[Fatty acid elongation, Biosynthesis of unsaturated fatty acids, Metabolic pathways, Ovarian steroidogenesis]','[]','[]','00062 Fatty acid elongation; 01040 Biosynthesis of unsaturated fatty acids; 04913 Ovarian steroidogenesis; 01004 Lipid biosynthesis proteins [BR:mmu01004]; 3.1.2.2  palmitoyl-CoA hydrolase; Component type; ','[Pfam: BAAT_C Bile_Hydr_Trans DLH Peptidase_S9 AXE1 FSH1]'),('70025','mmu:70025','[Acot7, 2410041A17Rik, Ach1, Act, Bach, CTE-IIa, Cte-II, Lach1]','[(RefSeq) acyl-CoA thioesterase 7]','[acyl-coenzyme A thioesterase 7 [EC:3.1.2.2]]','[Mus musculus (house mouse)]','[Fatty acid elongation, Biosynthesis of unsaturated fatty acids]','[]','[]','00062 Fatty acid elongation; 01040 Biosynthesis of unsaturated fatty acids; 01004 Lipid biosynthesis proteins [BR:mmu01004]; 04147 Exosome [BR:mmu04147]; 3.1.2.2  palmitoyl-CoA hydrolase; Component type; Exosomal proteins of other body fluids (saliva and urine); ','[Pfam: 4HBT tRNA_anti-codon NINJA_B TBCC]'),('170789','mmu:170789','[Acot8, PTE-1, PTE-2, Pte1]','[(RefSeq) acyl-CoA thioesterase 8]','[acyl-CoA thioesterase 8 [EC:3.1.2.27]]','[Mus musculus (house mouse)]','[Primary bile acid biosynthesis, Metabolic pathways, Peroxisome]','[]','[]','00120 Primary bile acid biosynthesis; 04146 Peroxisome; 3.1.2.27  choloyl-CoA hydrolase; ','[Pfam: 4HBT_3 Acyl_CoA_thio]'),('50790','mmu:50790','[Acsl4, 9430020A05Rik, ACS4, AU018108, Facl4, Lacs4]','[(RefSeq) acyl-CoA synthetase long-chain family member 4]','[long-chain acyl-CoA synthetase [EC:6.2.1.3]]','[Mus musculus (house mouse)]','[Fatty acid biosynthesis, Fatty acid degradation, Metabolic pathways, Fatty acid metabolism, PPAR signaling pathway, Peroxisome, Ferroptosis, Thermogenesis, Adipocytokine signaling pathway]','[]','[]','00061 Fatty acid biosynthesis; 00071 Fatty acid degradation; 04146 Peroxisome; 04216 Ferroptosis; 04920 Adipocytokine signaling pathway; 03320 PPAR signaling pathway; 04714 Thermogenesis; 01004 Lipid biosynthesis proteins [BR:mmu01004]; 04147 Exosome [BR:mmu04147]; 6.2.1.3  long-chain-fatty-acid---CoA ligase; Long/very long chain acyl-CoA synthetase; Exosomal proteins of breast milk; Exosomal proteins of breast cancer cells; ','[Pfam: AMP-binding AMP-binding_C]'),('60525','mmu:60525','[Acss2, 1110017C11Rik, ACAS, ACS, Acas1, Acas2, AceCS1, Acs1, aceCS]','[(RefSeq) acyl-CoA synthetase short-chain family member 2]','[acetyl-CoA synthetase [EC:6.2.1.1]]','[Mus musculus (house mouse)]','[Glycolysis / Gluconeogenesis, Pyruvate metabolism, Glyoxylate and dicarboxylate metabolism, Propanoate metabolism, Metabolic pathways, Carbon metabolism]','[]','[]','00010 Glycolysis / Gluconeogenesis; 00620 Pyruvate metabolism; 00630 Glyoxylate and dicarboxylate metabolism; 00640 Propanoate metabolism; 01004 Lipid biosynthesis proteins [BR:mmu01004]; 6.2.1.1  acetate---CoA ligase; Short/medium chain acyl-CoA synthetase; ','[Pfam: AMP-binding AMP-binding_C ACAS_N]'),('75572','mmu:75572','[Acyp2, 2310004B09Rik]','[(RefSeq) acylphosphatase 2, muscle type]','[acylphosphatase [EC:3.6.1.7]]','[Mus musculus (house mouse)]','[Pyruvate metabolism, Metabolic pathways]','[]','[]','00620 Pyruvate metabolism; 3.6.1.7  acylphosphatase; ','[Pfam: Acylphosphatase]'),('11486','mmu:11486','[Ada]','[(RefSeq) adenosine deaminase]','[adenosine deaminase [EC:3.5.4.4]]','[Mus musculus (house mouse)]','[Purine metabolism, Metabolic pathways, Nucleotide metabolism, Primary immunodeficiency]','[]','[]','00230 Purine metabolism; 05340 Primary immunodeficiency; 3.5.4.4  adenosine deaminase; ','[Pfam: A_deaminase]'),('67512','mmu:67512','[Agpat2, 2510002J07Rik, AV000834, BSCL, BSCL1, LPAAB, LPAAT-beta]','[(RefSeq) 1-acylglycerol-3-phosphate O-acyltransferase 2 (lysophosphatidic acid acyltransferase, beta)]','[lysophosphatidate acyltransferase [EC:2.3.1.51]]','[Mus musculus (house mouse)]','[Glycerolipid metabolism, Glycerophospholipid metabolism, Metabolic pathways, Phospholipase D signaling pathway, Fat digestion and absorption]','[]','[]','00561 Glycerolipid metabolism; 00564 Glycerophospholipid metabolism; 04072 Phospholipase D signaling pathway; 04975 Fat digestion and absorption; 01004 Lipid biosynthesis proteins [BR:mmu01004]; 2.3.1.51  1-acylglycerol-3-phosphate O-acyltransferase; LPAAT/LPLAT; ','[Pfam: Acyltransferase DUF4131]'),('269378','mmu:269378','[Ahcy, AA987153, AL024110, CuBP, SAHH]','[(RefSeq) S-adenosylhomocysteine hydrolase]','[adenosylhomocysteinase [EC:3.3.1.1]]','[Mus musculus (house mouse)]','[Cysteine and methionine metabolism, Metabolic pathways]','[]','[]','00270 Cysteine and methionine metabolism; 01009 Protein phosphatases and associated proteins [BR:mmu01009]; 04147 Exosome [BR:mmu04147]; 3.3.1.1  adenosylhomocysteinase; PP1-interacting proteins (PIPs); Exosomal proteins of other body fluids (saliva and urine); ','[Pfam: AdoHcyase AdoHcyase_NAD 2-Hacid_dh_C IlvN TrkA_N 3HCDH_N]'),('11997','mmu:11997','[Akr1b7, AR, Avdp, MVDP]','[(RefSeq) aldo-keto reductase family 1, member B7]','[aldehyde reductase [EC:1.1.1.21]]','[Mus musculus (house mouse)]','[Pentose and glucuronate interconversions, Fructose and mannose metabolism, Galactose metabolism, Glycerolipid metabolism, Folate biosynthesis, Metabolic pathways]','[]','[]','00040 Pentose and glucuronate interconversions; 00051 Fructose and mannose metabolism; 00052 Galactose metabolism; 00561 Glycerolipid metabolism; 00790 Folate biosynthesis; 1.1.1.21  aldose reductase; ','[Pfam: Aldo_ket_red HU-CCDC81_euk_1]'),('11655','mmu:11655','[Alas1, ALAS, ALAS-N, Alas-1, Alas-h]','[(RefSeq) aminolevulinic acid synthase 1]','[5-aminolevulinate synthase [EC:2.3.1.37]]','[Mus musculus (house mouse)]','[Glycine, serine and threonine metabolism, Porphyrin metabolism, Metabolic pathways, Biosynthesis of cofactors]','[]','[]','00260 Glycine, serine and threonine metabolism; 00860 Porphyrin metabolism; 01007 Amino acid related enzymes [BR:mmu01007]; 2.3.1.37  5-aminolevulinate synthase; Class II; ','[Pfam: Aminotran_1_2 Preseq_ALAS Cys_Met_Meta_PP Aminotran_5 Beta_elim_lyase]'),('11676','mmu:11676','[Aldoc, AI847350, AU040929, Aldo3, Scrg2]','[(RefSeq) aldolase C, fructose-bisphosphate]','[fructose-bisphosphate aldolase, class I [EC:4.1.2.13]]','[Mus musculus (house mouse)]','[Glycolysis / Gluconeogenesis, Pentose phosphate pathway, Fructose and mannose metabolism, Metabolic pathways, Carbon metabolism, Biosynthesis of amino acids, HIF-1 signaling pathway]','[]','[]','00010 Glycolysis / Gluconeogenesis; 00030 Pentose phosphate pathway; 00051 Fructose and mannose metabolism; 04066 HIF-1 signaling pathway; 04131 Membrane trafficking [BR:mmu04131]; 03036 Chromosome and associated proteins [BR:mmu03036]; 04147 Exosome [BR:mmu04147]; 4.1.2.13  fructose-bisphosphate aldolase; Selective cargos; Other centriole associated proteins; Proteins found in most exosomes; ','[Pfam: Glycolytic]'),('11761','mmu:11761','[Aox1, AI196512, AI255253, Ao, Aox-1, Aox-2, Aox2, Moro, Ro]','[(RefSeq) aldehyde oxidase 1]','[aldehyde oxidase [EC:1.2.3.1]]','[Mus musculus (house mouse)]','[Valine, leucine and isoleucine degradation, Tyrosine metabolism, Tryptophan metabolism, Vitamin B6 metabolism, Nicotinate and nicotinamide metabolism, Retinol metabolism, Drug metabolism - cytochrome P450, Metabolic pathways, JAK-STAT signaling pathway]','[]','[]','00280 Valine, leucine and isoleucine degradation; 00350 Tyrosine metabolism; 00380 Tryptophan metabolism; 00750 Vitamin B6 metabolism; 00760 Nicotinate and nicotinamide metabolism; 00830 Retinol metabolism; 00982 Drug metabolism - cytochrome P450; 04630 JAK-STAT signaling pathway; 1.2.3.1  aldehyde oxidase; ','[Pfam: MoCoBD_1 MoCoBD_2 FAD_binding_5 Ald_Xan_dh_C Fer2_2 CO_deh_flav_C Fer2 RhoGAP]'),('11883','mmu:11883','[Arsa, AS-A, ASA, AW212749, As-2, As2, TISP73]','[(RefSeq) arylsulfatase A]','[arylsulfatase A [EC:3.1.6.8]]','[Mus musculus (house mouse)]','[Sphingolipid metabolism, Metabolic pathways, Lysosome]','[]','[]','00600 Sphingolipid metabolism; 04142 Lysosome; 3.1.6.8  cerebroside-sulfatase; ','[Pfam: Sulfatase Sulfatase_C Phosphodiest DUF1501]'),('109900','mmu:109900','[Asl, 2510006M18Rik, ASAL]','[(RefSeq) argininosuccinate lyase]','[argininosuccinate lyase [EC:4.3.2.1]]','[Mus musculus (house mouse)]','[Arginine biosynthesis, Alanine, aspartate and glutamate metabolism, Metabolic pathways, Biosynthesis of amino acids]','[]','[]','00250 Alanine, aspartate and glutamate metabolism; 00220 Arginine biosynthesis; 04147 Exosome [BR:mmu04147]; 4.3.2.1  argininosuccinate lyase; Exosomal proteins of other body fluids (saliva and urine); ','[Pfam: Lyase_1 ASL_C2 T4SS_CagC]'),('27053','mmu:27053','[Asns]','[(RefSeq) asparagine synthetase]','[asparagine synthase (glutamine-hydrolysing) [EC:6.3.5.4]]','[Mus musculus (house mouse)]','[Alanine, aspartate and glutamate metabolism, Metabolic pathways, Biosynthesis of amino acids]','[]','[]','00250 Alanine, aspartate and glutamate metabolism; 01002 Peptidases and inhibitors [BR:mmu01002]; 6.3.5.4  asparagine synthase (glutamine-hydrolysing); Family C44; ','[Pfam: Asn_synthase GATase_7 GATase_6 DUF3700 tRNA_Me_trans QueC NAD_synthase]'),('68352','mmu:68352','[Aspdh, 0610012D14Rik]','[(RefSeq) aspartate dehydrogenase domain containing]','[aspartate dehydrogenase [EC:1.4.1.21]]','[Mus musculus (house mouse)]','[Nicotinate and nicotinamide metabolism, Metabolic pathways, Biosynthesis of cofactors]','[]','[]','00760 Nicotinate and nicotinamide metabolism; 1.4.1.21  aspartate dehydrogenase; ','[Pfam: Asp_DH_C NAD_binding_3 F420_oxidored 2-Hacid_dh_C]'),('26877','mmu:26877','[B3galt1, beta3Gal-T1]','[(RefSeq) UDP-Gal:betaGlcNAc beta 1,3-galactosyltransferase, polypeptide 1]','[beta-1,3-galactosyltransferase 1 [EC:2.4.1.86]]','[Mus musculus (house mouse)]','[Glycosphingolipid biosynthesis - lacto and neolacto series, Metabolic pathways]','[]','[]','00601 Glycosphingolipid biosynthesis - lacto and neolacto series; 01003 Glycosyltransferases [BR:mmu01003]; 2.4.1.86  N-acetyl-beta-D-glucosaminide beta-(1,3)-galactosyltransferase; Type 1 chain; ','[Pfam: Galactosyl_T B3GALT2_N]'),('72727','mmu:72727','[B3gat3, 2810405M13Rik]','[(RefSeq) beta-1,3-glucuronyltransferase 3 (glucuronosyltransferase I)]','[galactosylgalactosylxylosylprotein 3-beta-glucuronosyltransferase 3 [EC:2.4.1.135]]','[Mus musculus (house mouse)]','[Glycosaminoglycan biosynthesis - chondroitin sulfate / dermatan sulfate, Glycosaminoglycan biosynthesis - heparan sulfate / heparin, Metabolic pathways]','[]','[]','00532 Glycosaminoglycan biosynthesis - chondroitin sulfate / dermatan sulfate; 00534 Glycosaminoglycan biosynthesis - heparan sulfate / heparin; 01003 Glycosyltransferases [BR:mmu01003]; 2.4.1.135  galactosylgalactosylxylosylprotein 3-beta-glucuronosyltransferase; O-Glycan; ','[Pfam: Glyco_transf_43]'),('72297','mmu:72297','[B3gnt3, 2210008L19Rik]','[(RefSeq) UDP-GlcNAc:betaGal beta-1,3-N-acetylglucosaminyltransferase 3]','[beta-1,3-N-acetylglucosaminyltransferase 3 [EC:2.4.1.146 2.4.1.149]]','[Mus musculus (house mouse)]','[Mucin type O-glycan biosynthesis, Glycosphingolipid biosynthesis - lacto and neolacto series, Metabolic pathways]','[]','[]','00512 Mucin type O-glycan biosynthesis; 00601 Glycosphingolipid biosynthesis - lacto and neolacto series; 01003 Glycosyltransferases [BR:mmu01003]; 2.4.1.146  beta-1,3-galactosyl-O-glycosyl-glycoprotein beta-1,3-N-acetylglucosaminyltransferase; 2.4.1.149  N-acetyllactosaminide beta-1,3-N-acetylglucosaminyltransferase; Type 2 chain/Poly-N-acetyllactosamine; ','[Pfam: Galactosyl_T Fringe]'),('14421','mmu:14421','[B4galnt1, 4933429D13Rik, Gal-NAc-T, GalNAc-T, GalNAcT, Galgt1, Ggm-2, Ggm2]','[(RefSeq) beta-1,4-N-acetyl-galactosaminyl transferase 1]','[(N-Acetylneuraminyl)-galactosylglucosylceramide N-acetylgalactosaminyltransferase [EC:2.4.1.92]]','[Mus musculus (house mouse)]','[Sphingolipid metabolism, Glycosphingolipid biosynthesis - ganglio series, Metabolic pathways]','[]','[]','00600 Sphingolipid metabolism; 00604 Glycosphingolipid biosynthesis - ganglio series; 01003 Glycosyltransferases [BR:mmu01003]; 2.4.1.92  (N-acetylneuraminyl)-galactosylglucosylceramide N-acetylgalactosaminyltransferase; Glycolipid; ','[Pfam: Glycos_transf_2 Glyco_transf_64]'),('12660','mmu:12660','[Chka, CK, CK/EK-alpha, Chetk-alpha, Chk, ChoK, EK, EtnK-alpha]','[(RefSeq) choline kinase alpha]','[choline/ethanolamine kinase [EC:2.7.1.32 2.7.1.82]]','[Mus musculus (house mouse)]','[Glycerophospholipid metabolism, Metabolic pathways, Choline metabolism in cancer]','[]','[]','00564 Glycerophospholipid metabolism; 05231 Choline metabolism in cancer; 2.7.1.32  choline kinase; 2.7.1.82  ethanolamine kinase; ','[Pfam: Choline_kinase APH EcKL Kdo Pkinase_fungal Helo_like_N]'),('74241','mmu:74241','[Chpf, 1700028N03Rik, AI414328, D1Bwg1363e]','[(RefSeq) chondroitin polymerizing factor]','[chondroitin polymerizing factor [EC:2.4.1.175 2.4.1.226]]','[Mus musculus (house mouse)]','[Glycosaminoglycan biosynthesis - chondroitin sulfate / dermatan sulfate, Metabolic pathways]','[]','[]','00532 Glycosaminoglycan biosynthesis - chondroitin sulfate / dermatan sulfate; 01003 Glycosyltransferases [BR:mmu01003]; 2.4.1.175  glucuronosyl-N-acetylgalactosaminyl-proteoglycan 4-beta-N-acetylgalactosaminyltransferase; 2.4.1.226  N-acetylgalactosaminyl-proteoglycan 3-beta-glucuronosyltransferase; O-Glycan; ','[Pfam: CHGN Fringe]'),('60322','mmu:60322','[Chst7, 2600013M07Rik, C6ST-2, GST5, Gn6st-4, glcNAc6ST-4]','[(RefSeq) carbohydrate (N-acetylglucosamino) sulfotransferase 7]','[chondroitin 6-sulfotransferase 7 [EC:2.8.2.17]]','[Mus musculus (house mouse)]','[Glycosaminoglycan biosynthesis - chondroitin sulfate / dermatan sulfate]','[]','[]','00532 Glycosaminoglycan biosynthesis - chondroitin sulfate / dermatan sulfate; 01003 Glycosyltransferases [BR:mmu01003]; 2.8.2.17  chondroitin 6-sulfotransferase; Sulfotransferases; ','[Pfam: Sulfotransfer_1 Sulfotransfer_3]'),('12709','mmu:12709','[Ckb, B-CK, Bck, CPK-B, Ck-3, Ck3, Ckbb]','[(RefSeq) creatine kinase, brain]','[creatine kinase [EC:2.7.3.2]]','[Mus musculus (house mouse)]','[Arginine and proline metabolism, Metabolic pathways]','[]','[]','00330 Arginine and proline metabolism; 04147 Exosome [BR:mmu04147]; 2.7.3.2  creatine kinase; Exosomal proteins of epithelial cells; ','[Pfam: ATP-gua_Ptrans ATP-gua_PtransN]'),('246277','mmu:246277','[Csad, Csd]','[(RefSeq) cysteine sulfinic acid decarboxylase]','[sulfinoalanine decarboxylase / aspartate 1-decarboxylase [EC:4.1.1.29 4.1.1.11]]','[Mus musculus (house mouse)]','[beta-Alanine metabolism, Taurine and hypotaurine metabolism, Pantothenate and CoA biosynthesis, Metabolic pathways]','[]','[]','00410 beta-Alanine metabolism; 00430 Taurine and hypotaurine metabolism; 00770 Pantothenate and CoA biosynthesis; 4.1.1.11  aspartate 1-decarboxylase; 4.1.1.29  sulfinoalanine decarboxylase; ','[Pfam: Pyridoxal_deC Aminotran_5 Beta_elim_lyase SepSecS]'),('13077','mmu:13077','[Cyp1a2, CP12, CYPIA2, P450-3]','[(RefSeq) cytochrome P450, family 1, subfamily a, polypeptide 2]','[cytochrome P450 family 1 subfamily A2 [EC:1.14.14.1]]','[Mus musculus (house mouse)]','[Steroid hormone biosynthesis, Caffeine metabolism, Tryptophan metabolism, Linoleic acid metabolism, Retinol metabolism, Metabolism of xenobiotics by cytochrome P450, Drug metabolism - cytochrome P450, Metabolic pathways, Chemical carcinogenesis - DNA adducts, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species]','[]','[]','00140 Steroid hormone biosynthesis; 00591 Linoleic acid metabolism; 00380 Tryptophan metabolism; 00830 Retinol metabolism; 00232 Caffeine metabolism; 00980 Metabolism of xenobiotics by cytochrome P450; 00982 Drug metabolism - cytochrome P450; 05204 Chemical carcinogenesis - DNA adducts; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 00199 Cytochrome P450 [BR:mmu00199]; 1.14.14.1  unspecific monooxygenase; CYP1 family; ','[Pfam: p450 Sec8_exocyst COG7]'),('13086','mmu:13086','[Cyp2a4, AI893559, Cyp15a1, D7Ucla4]','[(RefSeq) cytochrome P450, family 2, subfamily a, polypeptide 4]','[cytochrome P450 family 2 subfamily A [EC:1.14.14.1]]','[Mus musculus (house mouse)]','[Retinol metabolism, Metabolic pathways]','[]','[]','00830 Retinol metabolism; 00199 Cytochrome P450 [BR:mmu00199]; 1.14.14.1  unspecific monooxygenase; CYP2 family; ','[Pfam: p450]'),('13087','mmu:13087','[Cyp2a5, CYPIIA5, Coh, Cyp15a2]','[(RefSeq) cytochrome P450, family 2, subfamily a, polypeptide 5]','[cytochrome P450 family 2 subfamily A [EC:1.14.14.1]]','[Mus musculus (house mouse)]','[Retinol metabolism, Metabolic pathways]','[]','[]','00830 Retinol metabolism; 00199 Cytochrome P450 [BR:mmu00199]; 1.14.14.1  unspecific monooxygenase; CYP2 family; ','[Pfam: p450]'),('13089','mmu:13089','[Cyp2b13]','[(RefSeq) cytochrome P450, family 2, subfamily b, polypeptide 13]','[cytochrome P450 family 2 subfamily B [EC:1.14.14.1]]','[Mus musculus (house mouse)]','[Steroid hormone biosynthesis, Arachidonic acid metabolism, Retinol metabolism, Metabolic pathways, Chemical carcinogenesis - receptor activation]','[]','[]','00140 Steroid hormone biosynthesis; 00590 Arachidonic acid metabolism; 00830 Retinol metabolism; 05207 Chemical carcinogenesis - receptor activation; 00199 Cytochrome P450 [BR:mmu00199]; 1.14.14.1  unspecific monooxygenase; CYP2 family; ','[Pfam: p450]'),('13094','mmu:13094','[Cyp2b9, Cyp2b]','[(RefSeq) cytochrome P450, family 2, subfamily b, polypeptide 9]','[cytochrome P450 family 2 subfamily B [EC:1.14.14.1]]','[Mus musculus (house mouse)]','[Steroid hormone biosynthesis, Arachidonic acid metabolism, Retinol metabolism, Metabolic pathways, Chemical carcinogenesis - receptor activation]','[]','[]','00140 Steroid hormone biosynthesis; 00590 Arachidonic acid metabolism; 00830 Retinol metabolism; 05207 Chemical carcinogenesis - receptor activation; 00199 Cytochrome P450 [BR:mmu00199]; 1.14.14.1  unspecific monooxygenase; CYP2 family; ','[Pfam: p450]'),('13095','mmu:13095','[Cyp2c29, AHOH, AHOHase, AI159681, AI196010, Ah-2, Ahh-1, Cyp2c, P450-2C]','[(RefSeq) cytochrome P450, family 2, subfamily c, polypeptide 29]','[cytochrome P450 family 2 subfamily C [EC:1.14.14.1]]','[Mus musculus (house mouse)]','[Steroid hormone biosynthesis, Arachidonic acid metabolism, Linoleic acid metabolism, Retinol metabolism, Metabolic pathways, Serotonergic synapse, Inflammatory mediator regulation of TRP channels, Chemical carcinogenesis - DNA adducts]','[]','[]','00140 Steroid hormone biosynthesis; 00590 Arachidonic acid metabolism; 00591 Linoleic acid metabolism; 00830 Retinol metabolism; 04726 Serotonergic synapse; 04750 Inflammatory mediator regulation of TRP channels; 05204 Chemical carcinogenesis - DNA adducts; 00199 Cytochrome P450 [BR:mmu00199]; 1.14.14.1  unspecific monooxygenase; CYP2 family; ','[Pfam: p450]'),('13096','mmu:13096','[Cyp2c37]','[(RefSeq) cytochrome P450, family 2. subfamily c, polypeptide 37]','[cytochrome P450 family 2 subfamily C [EC:1.14.14.1]]','[Mus musculus (house mouse)]','[Steroid hormone biosynthesis, Arachidonic acid metabolism, Linoleic acid metabolism, Retinol metabolism, Metabolic pathways, Serotonergic synapse, Inflammatory mediator regulation of TRP channels, Chemical carcinogenesis - DNA adducts]','[]','[]','00140 Steroid hormone biosynthesis; 00590 Arachidonic acid metabolism; 00591 Linoleic acid metabolism; 00830 Retinol metabolism; 04726 Serotonergic synapse; 04750 Inflammatory mediator regulation of TRP channels; 05204 Chemical carcinogenesis - DNA adducts; 00199 Cytochrome P450 [BR:mmu00199]; 1.14.14.1  unspecific monooxygenase; CYP2 family; ','[Pfam: p450 SMI1_KNR4 TetR_C_17]'),('13099','mmu:13099','[Cyp2c40, AI265721, AI662255]','[(RefSeq) cytochrome P450, family 2, subfamily c, polypeptide 40]','[cytochrome P450 family 2 subfamily C [EC:1.14.14.1]]','[Mus musculus (house mouse)]','[Steroid hormone biosynthesis, Arachidonic acid metabolism, Linoleic acid metabolism, Retinol metabolism, Metabolic pathways, Serotonergic synapse, Inflammatory mediator regulation of TRP channels, Chemical carcinogenesis - DNA adducts]','[]','[]','00140 Steroid hormone biosynthesis; 00590 Arachidonic acid metabolism; 00591 Linoleic acid metabolism; 00830 Retinol metabolism; 04726 Serotonergic synapse; 04750 Inflammatory mediator regulation of TRP channels; 05204 Chemical carcinogenesis - DNA adducts; 00199 Cytochrome P450 [BR:mmu00199]; 1.14.14.1  unspecific monooxygenase; CYP2 family; ','[Pfam: p450]'),('72082','mmu:72082','[Cyp2c55, 2010318C06Rik]','[(RefSeq) cytochrome P450, family 2, subfamily c, polypeptide 55]','[cytochrome P450 family 2 subfamily C [EC:1.14.14.1]]','[Mus musculus (house mouse)]','[Steroid hormone biosynthesis, Arachidonic acid metabolism, Linoleic acid metabolism, Retinol metabolism, Metabolic pathways, Serotonergic synapse, Inflammatory mediator regulation of TRP channels, Chemical carcinogenesis - DNA adducts]','[]','[]','00140 Steroid hormone biosynthesis; 00590 Arachidonic acid metabolism; 00591 Linoleic acid metabolism; 00830 Retinol metabolism; 04726 Serotonergic synapse; 04750 Inflammatory mediator regulation of TRP channels; 05204 Chemical carcinogenesis - DNA adducts; 00199 Cytochrome P450 [BR:mmu00199]; 1.14.14.1  unspecific monooxygenase; CYP2 family; ','[Pfam: p450 Peptidase_M99_m]'),('76279','mmu:76279','[Cyp2d26, 1300006E06Rik]','[(RefSeq) cytochrome P450, family 2, subfamily d, polypeptide 26]','[cytochrome P450 family 2 subfamily D [EC:1.14.14.1]]','[Mus musculus (house mouse)]','[Steroid hormone biosynthesis, Serotonergic synapse]','[]','[]','00140 Steroid hormone biosynthesis; 04726 Serotonergic synapse; 00199 Cytochrome P450 [BR:mmu00199]; 1.14.14.1  unspecific monooxygenase; CYP2 family; ','[Pfam: p450 Herpes_IE68]'),('223706','mmu:223706','[Cyp2d34]','[(RefSeq) cytochrome P450, family 2, subfamily d, polypeptide 34]','[cytochrome P450 family 2 subfamily D [EC:1.14.14.1]]','[Mus musculus (house mouse)]','[Steroid hormone biosynthesis, Serotonergic synapse]','[]','[]','00140 Steroid hormone biosynthesis; 04726 Serotonergic synapse; 00199 Cytochrome P450 [BR:mmu00199]; 1.14.14.1  unspecific monooxygenase; CYP2 family; ','[Pfam: p450]'),('71754','mmu:71754','[Cyp2d40, 1300013D18Rik, CYPIID3, P450-DB3]','[(RefSeq) cytochrome P450, family 2, subfamily d, polypeptide 40]','[cytochrome P450 family 2 subfamily D [EC:1.14.14.1]]','[Mus musculus (house mouse)]','[Steroid hormone biosynthesis, Serotonergic synapse]','[]','[]','00140 Steroid hormone biosynthesis; 04726 Serotonergic synapse; 00199 Cytochrome P450 [BR:mmu00199]; 1.14.14.1  unspecific monooxygenase; CYP2 family; ','[Pfam: p450]'),('74519','mmu:74519','[Cyp2j9, 8430417E17Rik]','[(RefSeq) cytochrome P450, family 2, subfamily j, polypeptide 9]','[cytochrome P450 family 2 subfamily J [EC:1.14.14.1 1.14.14.73 1.14.14.74 1.14.14.75]]','[Mus musculus (house mouse)]','[Arachidonic acid metabolism, Linoleic acid metabolism, Metabolic pathways, Serotonergic synapse, Inflammatory mediator regulation of TRP channels, Ovarian steroidogenesis, Lipid and atherosclerosis]','[]','[]','00590 Arachidonic acid metabolism; 00591 Linoleic acid metabolism; 04913 Ovarian steroidogenesis; 04726 Serotonergic synapse; 04750 Inflammatory mediator regulation of TRP channels; 05417 Lipid and atherosclerosis; 00199 Cytochrome P450 [BR:mmu00199]; 1.14.14.1  unspecific monooxygenase; 1.14.14.73  albendazole monooxygenase (sulfoxide-forming); 1.14.14.74  albendazole monooxygenase (hydroxylating); 1.14.14.75  fenbendazole monooxygenase (4\'-hydroxylating); CYP2 family; ','[Pfam: p450]'),('56388','mmu:56388','[Cyp3a25, AI327008]','[(RefSeq) cytochrome P450, family 3, subfamily a, polypeptide 25]','[cytochrome P450 family 3 subfamily A [EC:1.14.14.1]]','[Mus musculus (house mouse)]','[Steroid hormone biosynthesis, Linoleic acid metabolism, Retinol metabolism, Metabolic pathways, Chemical carcinogenesis - DNA adducts, Chemical carcinogenesis - receptor activation]','[]','[]','00140 Steroid hormone biosynthesis; 00591 Linoleic acid metabolism; 00830 Retinol metabolism; 05204 Chemical carcinogenesis - DNA adducts; 05207 Chemical carcinogenesis - receptor activation; 00199 Cytochrome P450 [BR:mmu00199]; 1.14.14.1  unspecific monooxygenase; CYP3 family; ','[Pfam: p450]'),('337924','mmu:337924','[Cyp3a44]','[(RefSeq) cytochrome P450, family 3, subfamily a, polypeptide 44]','[cytochrome P450 family 3 subfamily A [EC:1.14.14.1]]','[Mus musculus (house mouse)]','[Steroid hormone biosynthesis, Linoleic acid metabolism, Retinol metabolism, Metabolic pathways, Chemical carcinogenesis - DNA adducts, Chemical carcinogenesis - receptor activation]','[]','[]','00140 Steroid hormone biosynthesis; 00591 Linoleic acid metabolism; 00830 Retinol metabolism; 05204 Chemical carcinogenesis - DNA adducts; 05207 Chemical carcinogenesis - receptor activation; 00199 Cytochrome P450 [BR:mmu00199]; 1.14.14.1  unspecific monooxygenase; CYP3 family; ','[Pfam: p450]'),('13117','mmu:13117','[Cyp4a10, AI647584, Cyp4a, D4Rp1, Msp-3, RP1]','[(RefSeq) cytochrome P450, family 4, subfamily a, polypeptide 10]','[long-chain fatty acid omega-monooxygenase [EC:1.14.14.80]]','[Mus musculus (house mouse)]','[Fatty acid degradation, Arachidonic acid metabolism, Retinol metabolism, Metabolic pathways, PPAR signaling pathway, Vascular smooth muscle contraction, Inflammatory mediator regulation of TRP channels]','[]','[]','00071 Fatty acid degradation; 00590 Arachidonic acid metabolism; 00830 Retinol metabolism; 03320 PPAR signaling pathway; 04270 Vascular smooth muscle contraction; 04750 Inflammatory mediator regulation of TRP channels; 00199 Cytochrome P450 [BR:mmu00199]; 1.14.14.80  long-chain fatty acid omega-monooxygenase; CYP4 family; ','[Pfam: p450]'),('13118','mmu:13118','[Cyp4a12b, BC060945, Cyp4a12]','[(RefSeq) cytochrome P450, family 4, subfamily a, polypeptide 12B]','[long-chain fatty acid omega-monooxygenase [EC:1.14.14.80]]','[Mus musculus (house mouse)]','[Fatty acid degradation, Arachidonic acid metabolism, Retinol metabolism, Metabolic pathways, PPAR signaling pathway, Vascular smooth muscle contraction, Inflammatory mediator regulation of TRP channels]','[]','[]','00071 Fatty acid degradation; 00590 Arachidonic acid metabolism; 00830 Retinol metabolism; 03320 PPAR signaling pathway; 04270 Vascular smooth muscle contraction; 04750 Inflammatory mediator regulation of TRP channels; 00199 Cytochrome P450 [BR:mmu00199]; 1.14.14.80  long-chain fatty acid omega-monooxygenase; CYP4 family; ','[Pfam: p450 YihI]'),('13119','mmu:13119','[Cyp4a14, AI314743]','[(RefSeq) cytochrome P450, family 4, subfamily a, polypeptide 14]','[long-chain fatty acid omega-monooxygenase [EC:1.14.14.80]]','[Mus musculus (house mouse)]','[Fatty acid degradation, Arachidonic acid metabolism, Retinol metabolism, Metabolic pathways, PPAR signaling pathway, Vascular smooth muscle contraction, Inflammatory mediator regulation of TRP channels]','[]','[]','00071 Fatty acid degradation; 00590 Arachidonic acid metabolism; 00830 Retinol metabolism; 03320 PPAR signaling pathway; 04270 Vascular smooth muscle contraction; 04750 Inflammatory mediator regulation of TRP channels; 00199 Cytochrome P450 [BR:mmu00199]; 1.14.14.80  long-chain fatty acid omega-monooxygenase; CYP4 family; ','[Pfam: p450 Choline_sulf_C]'),('13120','mmu:13120','[Cyp4b1, CYPIVB1]','[(RefSeq) cytochrome P450, family 4, subfamily b, polypeptide 1]','[cytochrome P450 family 4 subfamily B1 [EC:1.14.14.1]]','[Mus musculus (house mouse)]',NULL,'[]','[]','00199 Cytochrome P450 [BR:mmu00199]; 1.14.14.1  unspecific monooxygenase; CYP4 family; ','[Pfam: p450]'),('13123','mmu:13123','[Cyp7b1, AW261589, D3Ertd552e, hct-1]','[(RefSeq) cytochrome P450, family 7, subfamily b, polypeptide 1]','[25/26-hydroxycholesterol 7alpha-hydroxylase [EC:1.14.14.29]]','[Mus musculus (house mouse)]','[Primary bile acid biosynthesis, Steroid hormone biosynthesis]','[]','[]','00120 Primary bile acid biosynthesis; 00140 Steroid hormone biosynthesis; 00199 Cytochrome P450 [BR:mmu00199]; 1.14.14.29  25/26-hydroxycholesterol 7alpha-hydroxylase; CYP7 family; ','[Pfam: p450]'),('13124','mmu:13124','[Cyp8b1]','[(RefSeq) cytochrome P450, family 8, subfamily b, polypeptide 1]','[sterol 12-alpha-hydroxylase [EC:1.14.14.139]]','[Mus musculus (house mouse)]','[Primary bile acid biosynthesis, Metabolic pathways, PPAR signaling pathway]','[]','[]','00120 Primary bile acid biosynthesis; 03320 PPAR signaling pathway; 00199 Cytochrome P450 [BR:mmu00199]; 1.14.14.139  5beta-cholestane-3alpha,7alpha-diol 12alpha-hydroxylase; CYP8 family; ','[Pfam: p450]'),('13171','mmu:13171','[Dbt, BCKAD-E2, D3Wsu60e]','[(RefSeq) dihydrolipoamide branched chain transacylase E2]','[2-oxoisovalerate dehydrogenase E2 component (dihydrolipoyl transacylase) [EC:2.3.1.168]]','[Mus musculus (house mouse)]','[Valine, leucine and isoleucine degradation, Propanoate metabolism, Metabolic pathways]','[]','[]','00640 Propanoate metabolism; 00280 Valine, leucine and isoleucine degradation; 2.3.1.168  dihydrolipoyllysine-residue (2-methylpropanoyl)transferase; ','[Pfam: 2-oxoacid_dh Biotin_lipoyl E3_binding HlyD_D23]'),('13190','mmu:13190','[Dct, DT, TRP-2, TRP2, Tyrp-2, Tyrp2, slaty, slt]','[(RefSeq) dopachrome tautomerase]','[dopachrome tautomerase [EC:5.3.3.12]]','[Mus musculus (house mouse)]','[Tyrosine metabolism, Metabolic pathways, Melanogenesis]','[]','[]','00350 Tyrosine metabolism; 04916 Melanogenesis; 5.3.3.12  L-dopachrome isomerase; ','[Pfam: Tyrosinase]'),('66422','mmu:66422','[Dctpp1, 2410015N17Rik, AI854235, RS21-C6]','[(RefSeq) dCTP pyrophosphatase 1]','[dCTP diphosphatase [EC:3.6.1.12]]','[Mus musculus (house mouse)]','[Pyrimidine metabolism, Metabolic pathways, Nucleotide metabolism]','[]','[]','00240 Pyrimidine metabolism; 3.6.1.12  dCTP diphosphatase; ','[Pfam: MazG-like MazG dUTPase_2]'),('67880','mmu:67880','[Dcxr, 0610038K04Rik, 1810027P18Rik, XR]','[(RefSeq) dicarbonyl L-xylulose reductase]','[L-xylulose reductase [EC:1.1.1.10]]','[Mus musculus (house mouse)]','[Pentose and glucuronate interconversions, Metabolic pathways]','[]','[]','00040 Pentose and glucuronate interconversions; 1.1.1.10  L-xylulose reductase; ','[Pfam: adh_short_C2 adh_short KR Epimerase NAD_binding_10 ADH_zinc_N Shikimate_DH RmlD_sub_bind Sacchrp_dh_NADP THF_DHG_CYH_C NAD_binding_7 2-Hacid_dh_C]'),('13195','mmu:13195','[Ddc, Aadc]','[(RefSeq) dopa decarboxylase]','[aromatic-L-amino-acid/L-tryptophan decarboxylase [EC:4.1.1.28 4.1.1.105]]','[Mus musculus (house mouse)]','[Tyrosine metabolism, Phenylalanine metabolism, Tryptophan metabolism, Metabolic pathways, Serotonergic synapse, Dopaminergic synapse, Cocaine addiction, Amphetamine addiction, Alcoholism]','[]','[]','00350 Tyrosine metabolism; 00360 Phenylalanine metabolism; 00380 Tryptophan metabolism; 04728 Dopaminergic synapse; 04726 Serotonergic synapse; 05030 Cocaine addiction; 05031 Amphetamine addiction; 05034 Alcoholism; 04147 Exosome [BR:mmu04147]; 4.1.1.28  aromatic-L-amino-acid decarboxylase; 4.1.1.105  L-tryptophan decarboxylase; Exosomal proteins of other body fluids (saliva and urine); ','[Pfam: Pyridoxal_deC Beta_elim_lyase Aminotran_1_2 Autoind_bind]'),('13382','mmu:13382','[Dld]','[(RefSeq) dihydrolipoamide dehydrogenase]','[dihydrolipoamide dehydrogenase [EC:1.8.1.4]]','[Mus musculus (house mouse)]','[Glycolysis / Gluconeogenesis, Citrate cycle (TCA cycle), Glycine, serine and threonine metabolism, Valine, leucine and isoleucine degradation, Lysine degradation, Tryptophan metabolism, Pyruvate metabolism, Glyoxylate and dicarboxylate metabolism, Propanoate metabolism, Metabolic pathways, Carbon metabolism, Biosynthesis of cofactors]','[]','[]','00010 Glycolysis / Gluconeogenesis; 00020 Citrate cycle (TCA cycle); 00620 Pyruvate metabolism; 00630 Glyoxylate and dicarboxylate metabolism; 00640 Propanoate metabolism; 00260 Glycine, serine and threonine metabolism; 00280 Valine, leucine and isoleucine degradation; 00310 Lysine degradation; 00380 Tryptophan metabolism; 04147 Exosome [BR:mmu04147]; 1.8.1.4  dihydrolipoyl dehydrogenase; Exosomal proteins of breast cancer cells; ','[Pfam: Pyr_redox_2 Pyr_redox_dim Pyr_redox Pyr_redox_3 FAD_oxidored NAD_binding_8 FAD_binding_2 GIDA AlaDh_PNT_C HI0933_like FAD_binding_3 Lys_Orn_oxgnase Thi4 DUF6635]'),('52665','mmu:52665','[Echdc1, 1700028A24Rik, AI314462, AI930038, D10Ertd667e, MMCD]','[(RefSeq) enoyl Coenzyme A hydratase domain containing 1]','[ethylmalonyl-CoA/methylmalonyl-CoA decarboxylase [EC:4.1.1.94 4.1.1.-]]','[Mus musculus (house mouse)]','[Propanoate metabolism, Metabolic pathways]','[]','[]','00640 Propanoate metabolism; 4.1.1.94  ethylmalonyl-CoA decarboxylase; 4.1.1.-; ','[Pfam: ECH_1 ECH_2 DUF4478]'),('12686','mmu:12686','[Elovl3, CIN-2, Cig30]','[(RefSeq) elongation of very long chain fatty acids (FEN1/Elo2, SUR4/Elo3, yeast)-like 3]','[elongation of very long chain fatty acids protein 3 [EC:2.3.1.199]]','[Mus musculus (house mouse)]','[Fatty acid elongation, Biosynthesis of unsaturated fatty acids, Metabolic pathways, Fatty acid metabolism]','[]','[]','00062 Fatty acid elongation; 01040 Biosynthesis of unsaturated fatty acids; 01004 Lipid biosynthesis proteins [BR:mmu01004]; 2.3.1.199  very-long-chain 3-oxoacyl-CoA synthase; Unclassified; ','[Pfam: ELO Rota_Capsid_VP6 DUF3464]'),('68801','mmu:68801','[Elovl5, 1110059L23Rik, AI747313, AU043003, HELO1]','[(RefSeq) ELOVL family member 5, elongation of long chain fatty acids (yeast)]','[elongation of very long chain fatty acids protein 5 [EC:2.3.1.199]]','[Mus musculus (house mouse)]','[Fatty acid elongation, Biosynthesis of unsaturated fatty acids, Metabolic pathways, Fatty acid metabolism]','[]','[]','00062 Fatty acid elongation; 01040 Biosynthesis of unsaturated fatty acids; 01004 Lipid biosynthesis proteins [BR:mmu01004]; 2.3.1.199  very-long-chain 3-oxoacyl-CoA synthase; PUFA; ','[Pfam: ELO]'),('433182','mmu:433182','[Eno1b, EG433182, Eno-1, Eno1, Gm5506, NNE]','[(RefSeq) enolase 1B, retrotransposed]','[enolase [EC:4.2.1.11]]','[Mus musculus (house mouse)]','[Glycolysis / Gluconeogenesis, Metabolic pathways, Carbon metabolism, Biosynthesis of amino acids, RNA degradation, HIF-1 signaling pathway]','[]','[]','00010 Glycolysis / Gluconeogenesis; 03018 RNA degradation; 04066 HIF-1 signaling pathway; 03019 Messenger RNA biogenesis [BR:mmu03019]; 04147 Exosome [BR:mmu04147]; 4.2.1.11  phosphopyruvate hydratase; Other RNA degradosome components; Proteins found in most exosomes; ','[Pfam: Enolase_C Enolase_N MR_MLE_C MAAL_C]'),('13850','mmu:13850','[Ephx2, CEH, Eph2, SEH, sEP]','[(RefSeq) epoxide hydrolase 2, cytoplasmic]','[soluble epoxide hydrolase / lipid-phosphate phosphatase [EC:3.3.2.10 3.1.3.76]]','[Mus musculus (house mouse)]','[Arachidonic acid metabolism, Metabolic pathways, Peroxisome, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species]','[]','[]','00590 Arachidonic acid metabolism; 04146 Peroxisome; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 01009 Protein phosphatases and associated proteins [BR:mmu01009]; 01002 Peptidases and inhibitors [BR:mmu01002]; 3.1.3.76  lipid-phosphate phosphatase; 3.3.2.10  soluble epoxide hydrolase; Other HAD phosphatases; Family S33; ','[Pfam: Abhydrolase_1 Abhydrolase_6 HAD_2 Hydrolase_4 Hydrolase Hydrolase_like]'),('71760','mmu:71760','[Etnppl, 1300019H02Rik, AI195447, Agxt2l1]','[(RefSeq) ethanolamine phosphate phospholyase]','[ethanolamine-phosphate phospho-lyase [EC:4.2.3.2]]','[Mus musculus (house mouse)]','[Glycerophospholipid metabolism, Metabolic pathways]','[]','[]','00564 Glycerophospholipid metabolism; 4.2.3.2  ethanolamine-phosphate phospho-lyase; ','[Pfam: Aminotran_3]'),('56473','mmu:56473','[Fads2, 2900042M13Rik, Fads2a, Fadsd2]','[(RefSeq) fatty acid desaturase 2]','[acyl-CoA 6-desaturase (Delta-6 desaturase) [EC:1.14.19.3]]','[Mus musculus (house mouse)]','[alpha-Linolenic acid metabolism, Biosynthesis of unsaturated fatty acids, Metabolic pathways, Fatty acid metabolism, PPAR signaling pathway]','[]','[]','00592 alpha-Linolenic acid metabolism; 01040 Biosynthesis of unsaturated fatty acids; 03320 PPAR signaling pathway; 01004 Lipid biosynthesis proteins [BR:mmu01004]; 1.14.19.3  acyl-CoA 6-desaturase; Front-end; ','[Pfam: FA_desaturase Cyt-b5]'),('71665','mmu:71665','[Fuca1, 0610006A03Rik, 9530055J05Rik, Afuc, Fuca]','[(RefSeq) fucosidase, alpha-L- 1, tissue]','[alpha-L-fucosidase [EC:3.2.1.51]]','[Mus musculus (house mouse)]','[Other glycan degradation, Lysosome]','[]','[]','00511 Other glycan degradation; 04142 Lysosome; 04147 Exosome [BR:mmu04147]; 3.2.1.51  alpha-L-fucosidase; Exosomal proteins of other body fluids (saliva and urine); ','[Pfam: Alpha_L_fucos Fucosidase_C GHL6]'),('14297','mmu:14297','[Fxn, FA, FARR, Frda, X25]','[(RefSeq) frataxin]','[frataxin [EC:1.16.3.1]]','[Mus musculus (house mouse)]','[Porphyrin metabolism]','[]','[]','00860 Porphyrin metabolism; 03029 Mitochondrial biogenesis [BR:mmu03029]; 1.16.3.1  ferroxidase; Proteins involved in Fe-S cluster biogenesis; ','[Pfam: Frataxin_Cyay ORF_12_N]'),('14377','mmu:14377','[G6pc, AW107337, G6Pase, G6pc1, G6pt, Glc-6-Pase]','[(RefSeq) glucose-6-phosphatase, catalytic]','[glucose-6-phosphatase [EC:3.1.3.9]]','[Mus musculus (house mouse)]','[Glycolysis / Gluconeogenesis, Galactose metabolism, Starch and sucrose metabolism, Metabolic pathways, FoxO signaling pathway, PI3K-Akt signaling pathway, AMPK signaling pathway, Insulin signaling pathway, Adipocytokine signaling pathway, Glucagon signaling pathway, Insulin resistance, Carbohydrate digestion and absorption]','[]','[]','00010 Glycolysis / Gluconeogenesis; 00052 Galactose metabolism; 00500 Starch and sucrose metabolism; 04068 FoxO signaling pathway; 04151 PI3K-Akt signaling pathway; 04152 AMPK signaling pathway; 04910 Insulin signaling pathway; 04922 Glucagon signaling pathway; 04920 Adipocytokine signaling pathway; 04973 Carbohydrate digestion and absorption; 04931 Insulin resistance; 3.1.3.9  glucose-6-phosphatase; ','[Pfam: PAP2]'),('53897','mmu:53897','[Gal3st1, Cst, Gcst]','[(RefSeq) galactose-3-O-sulfotransferase 1]','[galactosylceramide sulfotransferase [EC:2.8.2.11]]','[Mus musculus (house mouse)]','[Ether lipid metabolism, Sphingolipid metabolism, Metabolic pathways]','[]','[]','00565 Ether lipid metabolism; 00600 Sphingolipid metabolism; 01003 Glycosyltransferases [BR:mmu01003]; 2.8.2.11  galactosylceramide sulfotransferase; Sulfotransferases; ','[Pfam: Gal-3-0_sulfotr Sulfotransfer_3]'),('74246','mmu:74246','[Gale, 2310002A12Rik, AI323962]','[(RefSeq) galactose-4-epimerase, UDP]','[UDP-glucose 4-epimerase [EC:5.1.3.2]]','[Mus musculus (house mouse)]','[Galactose metabolism, Amino sugar and nucleotide sugar metabolism, Metabolic pathways, Biosynthesis of nucleotide sugars]','[]','[]','00052 Galactose metabolism; 00520 Amino sugar and nucleotide sugar metabolism; 5.1.3.2  UDP-glucose 4-epimerase; ','[Pfam: GDP_Man_Dehyd Epimerase 3Beta_HSD RmlD_sub_bind Polysacc_synt_2 KR NAD_binding_4 NAD_binding_10 adh_short NmrA]'),('14635','mmu:14635','[Galk1, AA409894, GALK, Gk, Glk, Glk1]','[(RefSeq) galactokinase 1]','[galactokinase [EC:2.7.1.6]]','[Mus musculus (house mouse)]','[Galactose metabolism, Amino sugar and nucleotide sugar metabolism, Metabolic pathways, Biosynthesis of nucleotide sugars]','[]','[]','00052 Galactose metabolism; 00520 Amino sugar and nucleotide sugar metabolism; 04147 Exosome [BR:mmu04147]; 2.7.1.6  galactokinase; Exosomal proteins of other body fluids (saliva and urine); ','[Pfam: GalKase_gal_bdg GHMP_kinases_C GHMP_kinases_N]'),('26912','mmu:26912','[Gcat, AI526977, Kbl]','[(RefSeq) glycine C-acetyltransferase (2-amino-3-ketobutyrate-coenzyme A ligase)]','[glycine C-acetyltransferase [EC:2.3.1.29]]','[Mus musculus (house mouse)]','[Glycine, serine and threonine metabolism, Metabolic pathways]','[]','[]','00260 Glycine, serine and threonine metabolism; 01007 Amino acid related enzymes [BR:mmu01007]; 2.3.1.29  glycine C-acetyltransferase; Class II; ','[Pfam: Aminotran_1_2 Aminotran_5 Cys_Met_Meta_PP Beta_elim_lyase DegT_DnrJ_EryC1]'),('103988','mmu:103988','[Gck, GLK, Gk, Gls006, HK4, HKIV, HXKP, Hlb62, MODY2]','[(RefSeq) glucokinase]','[glucokinase [EC:2.7.1.2]]','[Mus musculus (house mouse)]','[Glycolysis / Gluconeogenesis, Galactose metabolism, Starch and sucrose metabolism, Amino sugar and nucleotide sugar metabolism, Neomycin, kanamycin and gentamicin biosynthesis, Metabolic pathways, Carbon metabolism, Biosynthesis of nucleotide sugars, Insulin signaling pathway, Insulin secretion, Prolactin signaling pathway, Glucagon signaling pathway, Type II diabetes mellitus, Maturity onset diabetes of the young, Central carbon metabolism in cancer]','[]','[]','00010 Glycolysis / Gluconeogenesis; 00052 Galactose metabolism; 00500 Starch and sucrose metabolism; 00520 Amino sugar and nucleotide sugar metabolism; 00524 Neomycin, kanamycin and gentamicin biosynthesis; 04911 Insulin secretion; 04910 Insulin signaling pathway; 04922 Glucagon signaling pathway; 04917 Prolactin signaling pathway; 05230 Central carbon metabolism in cancer; 04930 Type II diabetes mellitus; 04950 Maturity onset diabetes of the young; 2.7.1.2  glucokinase; ','[Pfam: Hexokinase_2 Hexokinase_1]'),('14629','mmu:14629','[Gclc, D9Wsu168e, GLCL-H, Ggcs-hs, Glclc]','[(RefSeq) glutamate-cysteine ligase, catalytic subunit]','[glutamate--cysteine ligase catalytic subunit [EC:6.3.2.2]]','[Mus musculus (house mouse)]','[Cysteine and methionine metabolism, Glutathione metabolism, Metabolic pathways, Biosynthesis of cofactors, Ferroptosis]','[]','[]','00270 Cysteine and methionine metabolism; 00480 Glutathione metabolism; 04216 Ferroptosis; 6.3.2.2  glutamate---cysteine ligase; ','[Pfam: GCS GCS2]'),('14583','mmu:14583','[Gfpt1, 2810423A18Rik, AI324119, AI449986, GFA, GFAT, GFAT1, GFAT1m, Gfpt]','[(RefSeq) glutamine fructose-6-phosphate transaminase 1]','[glutamine---fructose-6-phosphate transaminase (isomerizing) [EC:2.6.1.16]]','[Mus musculus (house mouse)]','[Alanine, aspartate and glutamate metabolism, Amino sugar and nucleotide sugar metabolism, Metabolic pathways, Biosynthesis of nucleotide sugars, Insulin resistance, Diabetic cardiomyopathy]','[]','[]','00520 Amino sugar and nucleotide sugar metabolism; 00250 Alanine, aspartate and glutamate metabolism; 05415 Diabetic cardiomyopathy; 04931 Insulin resistance; 01002 Peptidases and inhibitors [BR:mmu01002]; 2.6.1.16  glutamine---fructose-6-phosphate transaminase (isomerizing); Family C44; ','[Pfam: SIS GATase_6 GATase_7 GATase_4 GATase_2 SIS_2 DUF6287 Acetyltransf_6 ZapA]'),('218138','mmu:218138','[Gmds, BC031788, C87208]','[(RefSeq) GDP-mannose 4, 6-dehydratase]','[GDPmannose 4,6-dehydratase [EC:4.2.1.47]]','[Mus musculus (house mouse)]','[Fructose and mannose metabolism, Amino sugar and nucleotide sugar metabolism, Metabolic pathways, Biosynthesis of nucleotide sugars]','[]','[]','00051 Fructose and mannose metabolism; 00520 Amino sugar and nucleotide sugar metabolism; 4.2.1.47  GDP-mannose 4,6-dehydratase; ','[Pfam: GDP_Man_Dehyd Epimerase RmlD_sub_bind Polysacc_synt_2 NmrA]'),('66355','mmu:66355','[Gmpr, 2310004P21Rik, AV028449, GMPR_1]','[(RefSeq) guanosine monophosphate reductase]','[GMP reductase [EC:1.7.1.7]]','[Mus musculus (house mouse)]','[Purine metabolism, Metabolic pathways, Nucleotide metabolism]','[]','[]','00230 Purine metabolism; 1.7.1.7  GMP reductase; ','[Pfam: IMPDH FMN_dh Aldolase QRPTase_C]'),('14711','mmu:14711','[Gnmt]','[(RefSeq) glycine N-methyltransferase]','[glycine N-methyltransferase [EC:2.1.1.20]]','[Mus musculus (house mouse)]','[Glycine, serine and threonine metabolism, Metabolic pathways]','[]','[]','00260 Glycine, serine and threonine metabolism; 2.1.1.20  glycine N-methyltransferase; ','[Pfam: Methyltransf_31 Methyltransf_25 Methyltransf_12 Methyltransf_11 Methyltransf_23 Ubie_methyltran TehB TPMT MTS PCMT CMAS Methyltransf_15]'),('67980','mmu:67980','[Gnpda2, 4921523I18Rik, 4933412A11Rik, BB189630, GNPDA, Gnp2, Sb52, mKIAA4008]','[(RefSeq) glucosamine-6-phosphate deaminase 2]','[glucosamine-6-phosphate deaminase [EC:3.5.99.6]]','[Mus musculus (house mouse)]','[Amino sugar and nucleotide sugar metabolism, Metabolic pathways]','[]','[]','00520 Amino sugar and nucleotide sugar metabolism; 3.5.99.6  glucosamine-6-phosphate deaminase; ','[Pfam: Glucosamine_iso]'),('74182','mmu:74182','[Gpcpd1, 2310004G06Rik, 2310032D16Rik, AU015213, Gde5, Prei4, mKIAA1434]','[(RefSeq) glycerophosphocholine phosphodiesterase 1]','[glycerophosphocholine phosphodiesterase GPCPD1 [EC:3.1.4.2]]','[Mus musculus (house mouse)]','[Glycerophospholipid metabolism, Choline metabolism in cancer]','[]','[]','00564 Glycerophospholipid metabolism; 05231 Choline metabolism in cancer; 3.1.4.2  glycerophosphocholine phosphodiesterase; ','[Pfam: GDPD CBM_20 PI-PLC-X]'),('14555','mmu:14555','[Gpd1, AI747587, GPD-C, GPDH-C, Gdc-1, Gdc1]','[(RefSeq) glycerol-3-phosphate dehydrogenase 1 (soluble)]','[glycerol-3-phosphate dehydrogenase (NAD+) [EC:1.1.1.8]]','[Mus musculus (house mouse)]','[Glycerophospholipid metabolism]','[]','[]','00564 Glycerophospholipid metabolism; 1.1.1.8  glycerol-3-phosphate dehydrogenase (NAD+); ','[Pfam: NAD_Gly3P_dh_N NAD_Gly3P_dh_C GPD_NAD_C_bact]'),('14778','mmu:14778','[Gpx3, AA960521, EGPx, GPx, GSHPx-3, GSHPx-P]','[(RefSeq) glutathione peroxidase 3]','[glutathione peroxidase [EC:1.11.1.9]]','[Mus musculus (house mouse)]','[Glutathione metabolism, Arachidonic acid metabolism, Metabolic pathways, Thyroid hormone synthesis, Amyotrophic lateral sclerosis, Huntington disease, Pathways of neurodegeneration - multiple diseases]','[]','[]','00590 Arachidonic acid metabolism; 00480 Glutathione metabolism; 04918 Thyroid hormone synthesis; 05014 Amyotrophic lateral sclerosis; 05016 Huntington disease; 05022 Pathways of neurodegeneration - multiple diseases; 1.11.1.9  glutathione peroxidase; ','[Pfam: GSHPx]'),('75512','mmu:75512','[Gpx6, 1700020G18Rik, Ry2d1, olfactory]','[(RefSeq) glutathione peroxidase 6]','[glutathione peroxidase [EC:1.11.1.9]]','[Mus musculus (house mouse)]','[Glutathione metabolism, Arachidonic acid metabolism, Metabolic pathways, Thyroid hormone synthesis, Amyotrophic lateral sclerosis, Huntington disease, Pathways of neurodegeneration - multiple diseases]','[]','[]','00590 Arachidonic acid metabolism; 00480 Glutathione metabolism; 04918 Thyroid hormone synthesis; 05014 Amyotrophic lateral sclerosis; 05016 Huntington disease; 05022 Pathways of neurodegeneration - multiple diseases; 1.11.1.9  glutathione peroxidase; ','[Pfam: GSHPx]'),('14857','mmu:14857','[Gsta1, Gst2-1]','[(RefSeq) glutathione S-transferase, alpha 1 (Ya)]','[glutathione S-transferase [EC:2.5.1.18]]','[Mus musculus (house mouse)]','[Glutathione metabolism, Metabolism of xenobiotics by cytochrome P450, Drug metabolism - cytochrome P450, Drug metabolism - other enzymes, Metabolic pathways, Platinum drug resistance, Pathways in cancer, Chemical carcinogenesis - DNA adducts, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species, Hepatocellular carcinoma, Fluid shear stress and atherosclerosis]','[]','[]','00480 Glutathione metabolism; 00980 Metabolism of xenobiotics by cytochrome P450; 00982 Drug metabolism - cytochrome P450; 00983 Drug metabolism - other enzymes; 05200 Pathways in cancer; 05204 Chemical carcinogenesis - DNA adducts; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 05225 Hepatocellular carcinoma; 05418 Fluid shear stress and atherosclerosis; 01524 Platinum drug resistance; 02000 Transporters [BR:mmu02000]; 2.5.1.18  glutathione transferase; Pores ion channels [TC:1]; ','[Pfam: GST_N GST_C GST_C_3 GST_N_3]'),('14858','mmu:14858','[Gsta2, Gst2-2, Gstc-2, Gstc2]','[(RefSeq) glutathione S-transferase, alpha 2 (Yc2)]','[glutathione S-transferase [EC:2.5.1.18]]','[Mus musculus (house mouse)]','[Glutathione metabolism, Metabolism of xenobiotics by cytochrome P450, Drug metabolism - cytochrome P450, Drug metabolism - other enzymes, Metabolic pathways, Platinum drug resistance, Pathways in cancer, Chemical carcinogenesis - DNA adducts, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species, Hepatocellular carcinoma, Fluid shear stress and atherosclerosis]','[]','[]','00480 Glutathione metabolism; 00980 Metabolism of xenobiotics by cytochrome P450; 00982 Drug metabolism - cytochrome P450; 00983 Drug metabolism - other enzymes; 05200 Pathways in cancer; 05204 Chemical carcinogenesis - DNA adducts; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 05225 Hepatocellular carcinoma; 05418 Fluid shear stress and atherosclerosis; 01524 Platinum drug resistance; 02000 Transporters [BR:mmu02000]; 2.5.1.18  glutathione transferase; Pores ion channels [TC:1]; ','[Pfam: GST_N GST_C GST_C_3 GST_N_3]'),('14862','mmu:14862','[Gstm1, Gstb-1, Gstb1]','[(RefSeq) glutathione S-transferase, mu 1]','[glutathione S-transferase [EC:2.5.1.18]]','[Mus musculus (house mouse)]','[Glutathione metabolism, Metabolism of xenobiotics by cytochrome P450, Drug metabolism - cytochrome P450, Drug metabolism - other enzymes, Metabolic pathways, Platinum drug resistance, Pathways in cancer, Chemical carcinogenesis - DNA adducts, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species, Hepatocellular carcinoma, Fluid shear stress and atherosclerosis]','[]','[]','00480 Glutathione metabolism; 00980 Metabolism of xenobiotics by cytochrome P450; 00982 Drug metabolism - cytochrome P450; 00983 Drug metabolism - other enzymes; 05200 Pathways in cancer; 05204 Chemical carcinogenesis - DNA adducts; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 05225 Hepatocellular carcinoma; 05418 Fluid shear stress and atherosclerosis; 01524 Platinum drug resistance; 02000 Transporters [BR:mmu02000]; 2.5.1.18  glutathione transferase; Pores ion channels [TC:1]; ','[Pfam: GST_N GST_C_3 GST_C GST_C_2 GST_C_6 GST_N_3 GST_N_4]'),('14863','mmu:14863','[Gstm2, Gstb-2, Gstb2]','[(RefSeq) glutathione S-transferase, mu 2]','[glutathione S-transferase [EC:2.5.1.18]]','[Mus musculus (house mouse)]','[Glutathione metabolism, Metabolism of xenobiotics by cytochrome P450, Drug metabolism - cytochrome P450, Drug metabolism - other enzymes, Metabolic pathways, Platinum drug resistance, Pathways in cancer, Chemical carcinogenesis - DNA adducts, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species, Hepatocellular carcinoma, Fluid shear stress and atherosclerosis]','[]','[]','00480 Glutathione metabolism; 00980 Metabolism of xenobiotics by cytochrome P450; 00982 Drug metabolism - cytochrome P450; 00983 Drug metabolism - other enzymes; 05200 Pathways in cancer; 05204 Chemical carcinogenesis - DNA adducts; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 05225 Hepatocellular carcinoma; 05418 Fluid shear stress and atherosclerosis; 01524 Platinum drug resistance; 02000 Transporters [BR:mmu02000]; 2.5.1.18  glutathione transferase; Pores ion channels [TC:1]; ','[Pfam: GST_N GST_C GST_C_3 GST_C_6 GST_N_3 GST_C_2 Tom37 GST_N_2]'),('14864','mmu:14864','[Gstm3, Fsc2, mGSTM5]','[(RefSeq) glutathione S-transferase, mu 3]','[glutathione S-transferase [EC:2.5.1.18]]','[Mus musculus (house mouse)]','[Glutathione metabolism, Metabolism of xenobiotics by cytochrome P450, Drug metabolism - cytochrome P450, Drug metabolism - other enzymes, Metabolic pathways, Platinum drug resistance, Pathways in cancer, Chemical carcinogenesis - DNA adducts, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species, Hepatocellular carcinoma, Fluid shear stress and atherosclerosis]','[]','[]','00480 Glutathione metabolism; 00980 Metabolism of xenobiotics by cytochrome P450; 00982 Drug metabolism - cytochrome P450; 00983 Drug metabolism - other enzymes; 05200 Pathways in cancer; 05204 Chemical carcinogenesis - DNA adducts; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 05225 Hepatocellular carcinoma; 05418 Fluid shear stress and atherosclerosis; 01524 Platinum drug resistance; 02000 Transporters [BR:mmu02000]; 2.5.1.18  glutathione transferase; Pores ion channels [TC:1]; ','[Pfam: GST_N GST_C GST_C_3 GST_C_2 GST_C_6 GST_N_3]'),('14865','mmu:14865','[Gstm4, 1110004G14Rik, GSTM4-4, GSTM7-7, Gstb-4, Gstb4]','[(RefSeq) glutathione S-transferase, mu 4]','[glutathione S-transferase [EC:2.5.1.18]]','[Mus musculus (house mouse)]','[Glutathione metabolism, Metabolism of xenobiotics by cytochrome P450, Drug metabolism - cytochrome P450, Drug metabolism - other enzymes, Metabolic pathways, Platinum drug resistance, Pathways in cancer, Chemical carcinogenesis - DNA adducts, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species, Hepatocellular carcinoma, Fluid shear stress and atherosclerosis]','[]','[]','00480 Glutathione metabolism; 00980 Metabolism of xenobiotics by cytochrome P450; 00982 Drug metabolism - cytochrome P450; 00983 Drug metabolism - other enzymes; 05200 Pathways in cancer; 05204 Chemical carcinogenesis - DNA adducts; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 05225 Hepatocellular carcinoma; 05418 Fluid shear stress and atherosclerosis; 01524 Platinum drug resistance; 02000 Transporters [BR:mmu02000]; 2.5.1.18  glutathione transferase; Pores ion channels [TC:1]; ','[Pfam: GST_N GST_C_3 GST_C GST_C_6 GST_N_3 GST_C_2 GST_N_2]'),('103140','mmu:103140','[Gstt3, AI118089]','[(RefSeq) glutathione S-transferase, theta 3]','[glutathione S-transferase [EC:2.5.1.18]]','[Mus musculus (house mouse)]','[Glutathione metabolism, Metabolism of xenobiotics by cytochrome P450, Drug metabolism - cytochrome P450, Drug metabolism - other enzymes, Metabolic pathways, Platinum drug resistance, Pathways in cancer, Chemical carcinogenesis - DNA adducts, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species, Hepatocellular carcinoma, Fluid shear stress and atherosclerosis]','[]','[]','00480 Glutathione metabolism; 00980 Metabolism of xenobiotics by cytochrome P450; 00982 Drug metabolism - cytochrome P450; 00983 Drug metabolism - other enzymes; 05200 Pathways in cancer; 05204 Chemical carcinogenesis - DNA adducts; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 05225 Hepatocellular carcinoma; 05418 Fluid shear stress and atherosclerosis; 01524 Platinum drug resistance; 02000 Transporters [BR:mmu02000]; 2.5.1.18  glutathione transferase; Pores ion channels [TC:1]; ','[Pfam: GST_N GST_N_3 GST_N_2 GST_C GST_C_3 GST_N_4 GST_C_2 Glutaredoxin]'),('268756','mmu:268756','[Gulo, 5730581M22, AU018375, BC028822, GLO, LGO, sfx, unh, unhip]','[(RefSeq) gulonolactone (L-) oxidase]','[L-gulonolactone oxidase [EC:1.1.3.8]]','[Mus musculus (house mouse)]','[Ascorbate and aldarate metabolism, Metabolic pathways, Biosynthesis of cofactors]','[]','[]','00053 Ascorbate and aldarate metabolism; 1.1.3.8  L-gulonolactone oxidase; ','[Pfam: ALO FAD_binding_4 zf_C2H2_ZHX B12-binding]'),('107766','mmu:107766','[Haao, 0610007K21Rik, 0610012J07Rik, 3-HAO, 3-HAOxase, 3HAO]','[(RefSeq) 3-hydroxyanthranilate 3,4-dioxygenase]','[3-hydroxyanthranilate 3,4-dioxygenase [EC:1.13.11.6]]','[Mus musculus (house mouse)]','[Tryptophan metabolism, Metabolic pathways, Biosynthesis of cofactors]','[]','[]','00380 Tryptophan metabolism; 1.13.11.6  3-hydroxyanthranilate 3,4-dioxygenase; ','[Pfam: 3-HAO Cupin_2 Cupin_3 AraC_binding Cupin_1]'),('56185','mmu:56185','[Hao2, AI325478, Hao-2, Hao3, Haox3]','[(RefSeq) hydroxyacid oxidase 2]','[(S)-2-hydroxy-acid oxidase [EC:1.1.3.15]]','[Mus musculus (house mouse)]','[Glyoxylate and dicarboxylate metabolism, Metabolic pathways, Carbon metabolism, Peroxisome]','[]','[]','00630 Glyoxylate and dicarboxylate metabolism; 04146 Peroxisome; 1.1.3.15  (S)-2-hydroxy-acid oxidase; ','[Pfam: FMN_dh IMPDH Glu_synthase DHO_dh His_biosynth Ribul_P_3_epim]'),('208715','mmu:208715','[Hmgcs1, B130032C06Rik]','[(RefSeq) 3-hydroxy-3-methylglutaryl-Coenzyme A synthase 1]','[hydroxymethylglutaryl-CoA synthase [EC:2.3.3.10]]','[Mus musculus (house mouse)]','[Valine, leucine and isoleucine degradation, Butanoate metabolism, Terpenoid backbone biosynthesis, Metabolic pathways, PPAR signaling pathway]','[]','[]','00650 Butanoate metabolism; 00280 Valine, leucine and isoleucine degradation; 00900 Terpenoid backbone biosynthesis; 03320 PPAR signaling pathway; 2.3.3.10  hydroxymethylglutaryl-CoA synthase; ','[Pfam: HMG_CoA_synt_C HMG_CoA_synt_N]'),('15445','mmu:15445','[Hpd, 4HPPD, Fla, Flp, Hppd, Laf]','[(RefSeq) 4-hydroxyphenylpyruvic acid dioxygenase]','[4-hydroxyphenylpyruvate dioxygenase [EC:1.13.11.27]]','[Mus musculus (house mouse)]','[Ubiquinone and other terpenoid-quinone biosynthesis, Tyrosine metabolism, Phenylalanine metabolism, Metabolic pathways, Biosynthesis of cofactors]','[]','[]','00350 Tyrosine metabolism; 00360 Phenylalanine metabolism; 00130 Ubiquinone and other terpenoid-quinone biosynthesis; 04147 Exosome [BR:mmu04147]; 1.13.11.27  4-hydroxyphenylpyruvate dioxygenase; Exosomal proteins of other body fluids (saliva and urine); ','[Pfam: Glyoxalase Glyoxalase_4 Glyoxalase_3 Glyoxalase_5]'),('15108','mmu:15108','[Hsd17b10, 17bHSD10, Ads9, ERAB, Hadh2, MHBD]','[(RefSeq) hydroxysteroid (17-beta) dehydrogenase 10]','[3-hydroxyacyl-CoA dehydrogenase / 3-hydroxy-2-methylbutyryl-CoA dehydrogenase [EC:1.1.1.35 1.1.1.178]]','[Mus musculus (house mouse)]','[Valine, leucine and isoleucine degradation, Metabolic pathways, Alzheimer disease, Pathways of neurodegeneration - multiple diseases]','[]','[]','00280 Valine, leucine and isoleucine degradation; 05010 Alzheimer disease; 05022 Pathways of neurodegeneration - multiple diseases; 03029 Mitochondrial biogenesis [BR:mmu03029]; 04147 Exosome [BR:mmu04147]; 1.1.1.35  3-hydroxyacyl-CoA dehydrogenase; 1.1.1.178  3-hydroxy-2-methylbutyryl-CoA dehydrogenase; Mitochondrial ribonuclease P; Exosomal proteins of breast cancer cells; ','[Pfam: adh_short adh_short_C2 KR Epimerase Ribosomal_L12]'),('15496','mmu:15496','[Hsd3b5]','[(RefSeq) hydroxy-delta-5-steroid dehydrogenase, 3 beta- and steroid delta-isomerase 5]','[3beta-hydroxy-Delta5-steroid dehydrogenase / steroid Delta-isomerase [EC:1.1.1.145 5.3.3.1]]','[Mus musculus (house mouse)]','[Steroid hormone biosynthesis, Metabolic pathways, Ovarian steroidogenesis, Aldosterone synthesis and secretion, Cortisol synthesis and secretion, Cushing syndrome]','[]','[]','00140 Steroid hormone biosynthesis; 04913 Ovarian steroidogenesis; 04925 Aldosterone synthesis and secretion; 04927 Cortisol synthesis and secretion; 04934 Cushing syndrome; 1.1.1.145  3beta-hydroxy-Delta5-steroid dehydrogenase; 5.3.3.1  steroid Delta-isomerase; ','[Pfam: 3Beta_HSD Epimerase NAD_binding_4 NAD_binding_10 GDP_Man_Dehyd Polysacc_synt_2 NmrA RmlD_sub_bind KR Ldh_1_N Semialdhyde_dh]'),('15587','mmu:15587','[Hyal2, AI256841, AU020858, AW555733]','[(RefSeq) hyaluronoglucosaminidase 2]','[hyaluronoglucosaminidase [EC:3.2.1.35]]','[Mus musculus (house mouse)]','[Glycosaminoglycan degradation, Metabolic pathways, Lysosome]','[]','[]','00531 Glycosaminoglycan degradation; 04142 Lysosome; 02042 Bacterial toxins [BR:mmu02042]; 00536 Glycosaminoglycan binding proteins [BR:mmu00536]; 00537 Glycosylphosphatidylinositol (GPI)-anchored proteins [BR:mmu00537]; 3.2.1.35  hyaluronoglucosaminidase; Hyaluronidases/Collagenases; Enzymes; Enzymes; ','[Pfam: Glyco_hydro_56]'),('114663','mmu:114663','[Impa2, 2210415D20Rik, AI326924, AW259601, IMP_2, IMPase_2]','[(RefSeq) inositol monophosphatase 2]','[myo-inositol-1(or 4)-monophosphatase [EC:3.1.3.25]]','[Mus musculus (house mouse)]','[Inositol phosphate metabolism, Metabolic pathways, Phosphatidylinositol signaling system]','[]','[]','00562 Inositol phosphate metabolism; 04070 Phosphatidylinositol signaling system; 3.1.3.25  inositol-phosphate phosphatase; ','[Pfam: Inositol_P]'),('71780','mmu:71780','[Isyna1, 1300017C10Rik, AU018670, IPS_1]','[(RefSeq) myo-inositol 1-phosphate synthase A1]','[myo-inositol-1-phosphate synthase [EC:5.5.1.4]]','[Mus musculus (house mouse)]','[Inositol phosphate metabolism, Metabolic pathways]','[]','[]','00562 Inositol phosphate metabolism; 5.5.1.4  inositol-3-phosphate synthase; ','[Pfam: NAD_binding_5 Inos-1-P_synth]'),('16548','mmu:16548','[Khk]','[(RefSeq) ketohexokinase]','[ketohexokinase [EC:2.7.1.3]]','[Mus musculus (house mouse)]','[Fructose and mannose metabolism, Metabolic pathways]','[]','[]','00051 Fructose and mannose metabolism; 04147 Exosome [BR:mmu04147]; 2.7.1.3  ketohexokinase; Exosomal proteins of other body fluids (saliva and urine); ','[Pfam: PfkB]'),('214162','mmu:214162','[Kmt2a, 6430520K01, ALL-1, All1, Cxxc7, HRX, HTRX1, Mll, Mll1, mKIAA4050]','[(RefSeq) lysine (K)-specific methyltransferase 2A]','[[histone H3]-lysine4 N-trimethyltransferase MLL1 [EC:2.1.1.354]]','[Mus musculus (house mouse)]','[Lysine degradation, Metabolic pathways, Cushing syndrome, Transcriptional misregulation in cancer]','[]','[]','00310 Lysine degradation; 05202 Transcriptional misregulation in cancer; 04934 Cushing syndrome; 03000 Transcription factors [BR:mmu03000]; 03036 Chromosome and associated proteins [BR:mmu03036]; 2.1.1.354  [histone H3]-lysine4 N-trimethyltransferase; CXXC CpG-binding proteins; HKMTs (histone lysine methyltransferases); MLL-HCF complex; ','[Pfam: SET FYRC PHD FYRN zf-CXXC zf-HC5HC2H PHD_2 Bromodomain PHD_4]'),('16816','mmu:16816','[Lcat, AI046659, D8Wsu61e]','[(RefSeq) lecithin cholesterol acyltransferase]','[lecithin-cholesterol acyltransferase [EC:2.3.1.43]]','[Mus musculus (house mouse)]','[Glycerophospholipid metabolism, Cholesterol metabolism]','[]','[]','00564 Glycerophospholipid metabolism; 04979 Cholesterol metabolism; 2.3.1.43  phosphatidylcholine---sterol O-acyltransferase; ','[Pfam: LCAT Abhydrolase_6 Hydrolase_4 Ser_hydrolase Abhydrolase_1 DUF1749 LD_cluster2 PGAP1 Esterase]'),('16828','mmu:16828','[Ldha, Ldh1, Ldhm, l7R2]','[(RefSeq) lactate dehydrogenase A]','[L-lactate dehydrogenase [EC:1.1.1.27]]','[Mus musculus (house mouse)]','[Glycolysis / Gluconeogenesis, Cysteine and methionine metabolism, Pyruvate metabolism, Propanoate metabolism, Metabolic pathways, HIF-1 signaling pathway, Glucagon signaling pathway, Central carbon metabolism in cancer]','[]','[]','00010 Glycolysis / Gluconeogenesis; 00620 Pyruvate metabolism; 00640 Propanoate metabolism; 00270 Cysteine and methionine metabolism; 04066 HIF-1 signaling pathway; 04922 Glucagon signaling pathway; 05230 Central carbon metabolism in cancer; 04147 Exosome [BR:mmu04147]; 1.1.1.27  L-lactate dehydrogenase; Exosomal proteins of epithelial cells; Exosomal proteins of breast milk; ','[Pfam: Ldh_1_N Ldh_1_C Pyr_redox_2 Glyco_hydro_4 Glyco_hydro_4C]'),('16956','mmu:16956','[Lpl]','[(RefSeq) lipoprotein lipase]','[lipoprotein lipase [EC:3.1.1.34]]','[Mus musculus (house mouse)]','[Glycerolipid metabolism, PPAR signaling pathway, Cholesterol metabolism, Alzheimer disease]','[]','[]','00561 Glycerolipid metabolism; 03320 PPAR signaling pathway; 04979 Cholesterol metabolism; 05010 Alzheimer disease; 04147 Exosome [BR:mmu04147]; 00536 Glycosaminoglycan binding proteins [BR:mmu00536]; 00537 Glycosylphosphatidylinositol (GPI)-anchored proteins [BR:mmu00537]; 3.1.1.34  lipoprotein lipase; Exosomal proteins of microglial cells; Exosomal proteins of breast milk; Enzymes; Enzymes; ','[Pfam: Lipase PLAT Abhydrolase_1 Abhydrolase_6 BD-FAE]'),('17160','mmu:17160','[Man2b2, mKIAA0935]','[(RefSeq) mannosidase 2, alpha B2]','[epididymis-specific alpha-mannosidase [EC:3.2.1.24]]','[Mus musculus (house mouse)]','[Other glycan degradation]','[]','[]','00511 Other glycan degradation; 3.2.1.24  alpha-mannosidase; ','[Pfam: Glyco_hydro_38N Glyco_hydro_38C Alpha-mann_mid Glyco_hydro38C2]'),('223722','mmu:223722','[Mcat, AI225907, BC025519]','[(RefSeq) malonyl CoA:ACP acyltransferase (mitochondrial)]','[[acyl-carrier-protein] S-malonyltransferase [EC:2.3.1.39]]','[Mus musculus (house mouse)]','[Fatty acid biosynthesis, Metabolic pathways, Fatty acid metabolism]','[]','[]','00061 Fatty acid biosynthesis; 00333 Prodigiosin biosynthesis; 01004 Lipid biosynthesis proteins [BR:mmu01004]; 2.3.1.39  [acyl-carrier-protein] S-malonyltransferase; Component type; ','[Pfam: Acyl_transf_1]'),('103534','mmu:103534','[Mgat4b, AA407995, GnTIVb]','[(RefSeq) mannoside acetylglucosaminyltransferase 4, isoenzyme B]','[alpha-1,3-mannosylglycoprotein beta-1,4-N-acetylglucosaminyltransferase A/B [EC:2.4.1.145]]','[Mus musculus (house mouse)]','[N-Glycan biosynthesis, Various types of N-glycan biosynthesis, Metabolic pathways]','[]','[]','00510 N-Glycan biosynthesis; 00513 Various types of N-glycan biosynthesis; 01003 Glycosyltransferases [BR:mmu01003]; 2.4.1.145  alpha-1,3-mannosyl-glycoprotein 4-beta-N-acetylglucosaminyltransferase; N-Glycan; ','[Pfam: Glyco_transf_54]'),('23945','mmu:23945','[Mgll, AA589436, Magl, Mgl]','[(RefSeq) monoglyceride lipase]','[acylglycerol lipase [EC:3.1.1.23]]','[Mus musculus (house mouse)]','[Glycerolipid metabolism, Metabolic pathways, Thermogenesis, Retrograde endocannabinoid signaling, Regulation of lipolysis in adipocytes]','[]','[]','00561 Glycerolipid metabolism; 04923 Regulation of lipolysis in adipocytes; 04723 Retrograde endocannabinoid signaling; 04714 Thermogenesis; 01002 Peptidases and inhibitors [BR:mmu01002]; 3.1.1.23  acylglycerol lipase; Family S33; ','[Pfam: Hydrolase_4 Abhydrolase_1 Abhydrolase_6 Abhydrolase_11 Lipase_3 Esterase LIDHydrolase Abhydrolase_3 PGAP1 Abhydrolase_5 Peptidase_S9 Thioesterase DUF1057 DUF2920 Chlorophyllase2 Abhydrolase_2 DUF1350]'),('17319','mmu:17319','[Mif, DER6, GIF, Glif]','[(RefSeq) macrophage migration inhibitory factor (glycosylation-inhibiting factor)]','[phenylpyruvate tautomerase [EC:5.3.2.1]]','[Mus musculus (house mouse)]','[Tyrosine metabolism, Phenylalanine metabolism, Metabolic pathways]','[]','[]','00350 Tyrosine metabolism; 00360 Phenylalanine metabolism; 04147 Exosome [BR:mmu04147]; 5.3.2.1  phenylpyruvate tautomerase; Exosomal proteins of breast milk; ','[Pfam: MIF]'),('56690','mmu:56690','[Mlycd, AI324784, Mcd]','[(RefSeq) malonyl-CoA decarboxylase]','[malonyl-CoA decarboxylase [EC:4.1.1.9]]','[Mus musculus (house mouse)]','[beta-Alanine metabolism, Propanoate metabolism, Metabolic pathways, Peroxisome, AMPK signaling pathway, Alcoholic liver disease]','[]','[]','00640 Propanoate metabolism; 00410 beta-Alanine metabolism; 04152 AMPK signaling pathway; 04146 Peroxisome; 04936 Alcoholic liver disease; 4.1.1.9  malonyl-CoA decarboxylase; ','[Pfam: MCD MCD_N]'),('246221','mmu:246221','[Mpst, Mst]','[(RefSeq) mercaptopyruvate sulfurtransferase]','[thiosulfate/3-mercaptopyruvate sulfurtransferase [EC:2.8.1.1 2.8.1.2]]','[Mus musculus (house mouse)]','[Cysteine and methionine metabolism, Sulfur metabolism, Metabolic pathways, Sulfur relay system]','[]','[]','00920 Sulfur metabolism; 00270 Cysteine and methionine metabolism; 04122 Sulfur relay system; 2.8.1.1  thiosulfate sulfurtransferase; 2.8.1.2  3-mercaptopyruvate sulfurtransferase; ','[Pfam: Rhodanese]'),('17769','mmu:17769','[Mthfr, AI323986]','[(RefSeq) methylenetetrahydrofolate reductase]','[methylenetetrahydrofolate reductase (NADPH) [EC:1.5.1.53]]','[Mus musculus (house mouse)]','[One carbon pool by folate, Metabolic pathways, Antifolate resistance]','[]','[]','00670 One carbon pool by folate; 01523 Antifolate resistance; 1.5.1.53  methylenetetrahydrofolate reductase (NADPH); ','[Pfam: MTHFR]'),('68396','mmu:68396','[Nat8, 0610037O16Rik, ATase2, CCNAT, Cml4]','[(RefSeq) N-acetyltransferase 8 (GCN5-related)]','[N-acetyltransferase 8 [EC:2.3.1.80 2.3.1.-]]','[Mus musculus (house mouse)]','[Glutathione metabolism, Metabolic pathways]','[]','[]','00480 Glutathione metabolism; 2.3.1.80  cysteine-S-conjugate N-acetyltransferase; 2.3.1.-; ','[Pfam: Acetyltransf_1 Acetyltransf_7 Acetyltransf_10 Acetyltransf_9 Acetyltransf_CG FR47 DUF4131 Acetyltransf_4 Acetyltransf_3]'),('68349','mmu:68349','[Ndufs3, 0610010M09Rik, CI-30kD]','[(RefSeq) NADH:ubiquinone oxidoreductase core subunit S3]','[NADH dehydrogenase (ubiquinone) Fe-S protein 3 [EC:7.1.1.2]]','[Mus musculus (house mouse)]','[Oxidative phosphorylation, Metabolic pathways, Thermogenesis, Retrograde endocannabinoid signaling, Non-alcoholic fatty liver disease, Alzheimer disease, Parkinson disease, Amyotrophic lateral sclerosis, Huntington disease, Prion disease, Pathways of neurodegeneration - multiple diseases, Chemical carcinogenesis - reactive oxygen species, Diabetic cardiomyopathy]','[]','[]','00190 Oxidative phosphorylation; 04723 Retrograde endocannabinoid signaling; 04714 Thermogenesis; 05208 Chemical carcinogenesis - reactive oxygen species; 05010 Alzheimer disease; 05012 Parkinson disease; 05014 Amyotrophic lateral sclerosis; 05016 Huntington disease; 05020 Prion disease; 05022 Pathways of neurodegeneration - multiple diseases; 05415 Diabetic cardiomyopathy; 04932 Non-alcoholic fatty liver disease; 7.1.1.2  NADH:ubiquinone reductase (H+-translocating); ','[Pfam: Complex1_30kDa]'),('225887','mmu:225887','[Ndufs8, BC021616, CI-23kD, TYKY]','[(RefSeq) NADH:ubiquinone oxidoreductase core subunit S8]','[NADH dehydrogenase (ubiquinone) Fe-S protein 8 [EC:7.1.1.2]]','[Mus musculus (house mouse)]','[Oxidative phosphorylation, Metabolic pathways, Thermogenesis, Retrograde endocannabinoid signaling, Non-alcoholic fatty liver disease, Alzheimer disease, Parkinson disease, Amyotrophic lateral sclerosis, Huntington disease, Prion disease, Pathways of neurodegeneration - multiple diseases, Chemical carcinogenesis - reactive oxygen species, Diabetic cardiomyopathy]','[]','[]','00190 Oxidative phosphorylation; 04723 Retrograde endocannabinoid signaling; 04714 Thermogenesis; 05208 Chemical carcinogenesis - reactive oxygen species; 05010 Alzheimer disease; 05012 Parkinson disease; 05014 Amyotrophic lateral sclerosis; 05016 Huntington disease; 05020 Prion disease; 05022 Pathways of neurodegeneration - multiple diseases; 05415 Diabetic cardiomyopathy; 04932 Non-alcoholic fatty liver disease; 7.1.1.2  NADH:ubiquinone reductase (H+-translocating); ','[Pfam: Fer4 Fer4_7 Fer4_16 Fer4_21 Fer4_10 Fer4_9 Fer4_8 Fer4_6 Fer4_2 Fer4_4 Fer4_17 Fer4_3]'),('18127','mmu:18127','[Nos3, 2310065A03Rik, Nos-3, eNOS, ecNOS]','[(RefSeq) nitric oxide synthase 3, endothelial cell]','[nitric-oxide synthase, endothelial [EC:1.14.13.39]]','[Mus musculus (house mouse)]','[Arginine biosynthesis, Arginine and proline metabolism, Metabolic pathways, Calcium signaling pathway, cGMP-PKG signaling pathway, HIF-1 signaling pathway, Sphingolipid signaling pathway, PI3K-Akt signaling pathway, VEGF signaling pathway, Apelin signaling pathway, Platelet activation, Estrogen signaling pathway, Oxytocin signaling pathway, Relaxin signaling pathway, Insulin resistance, AGE-RAGE signaling pathway in diabetic complications, Diabetic cardiomyopathy, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[]','[]','00220 Arginine biosynthesis; 00330 Arginine and proline metabolism; 04370 VEGF signaling pathway; 04371 Apelin signaling pathway; 04066 HIF-1 signaling pathway; 04020 Calcium signaling pathway; 04071 Sphingolipid signaling pathway; 04022 cGMP-PKG signaling pathway; 04151 PI3K-Akt signaling pathway; 04611 Platelet activation; 04915 Estrogen signaling pathway; 04921 Oxytocin signaling pathway; 04926 Relaxin signaling pathway; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 05415 Diabetic cardiomyopathy; 04931 Insulin resistance; 04933 AGE-RAGE signaling pathway in diabetic complications; 1.14.13.39  nitric-oxide synthase (NADPH); ','[Pfam: NO_synthase FAD_binding_1 Flavodoxin_1 NAD_binding_1]'),('18104','mmu:18104','[Nqo1, AV001255, Dia4, Dtd, Nmo-1, Nmo1, Nmor1, Ox-1, Ox1, Qr1]','[(RefSeq) NAD(P)H dehydrogenase, quinone 1]','[NAD(P)H dehydrogenase (quinone) [EC:1.6.5.2]]','[Mus musculus (house mouse)]','[Ubiquinone and other terpenoid-quinone biosynthesis, Metabolic pathways, Biosynthesis of cofactors, Pathways in cancer, Chemical carcinogenesis - reactive oxygen species, Hepatocellular carcinoma, Fluid shear stress and atherosclerosis]','[]','[]','00130 Ubiquinone and other terpenoid-quinone biosynthesis; 05200 Pathways in cancer; 05208 Chemical carcinogenesis - reactive oxygen species; 05225 Hepatocellular carcinoma; 05418 Fluid shear stress and atherosclerosis; 1.6.5.2  NAD(P)H dehydrogenase (quinone); ','[Pfam: Flavodoxin_2 FMN_red]'),('67993','mmu:67993','[Nudt12, 0610016O18Rik]','[(RefSeq) nudix (nucleoside diphosphate linked moiety X)-type motif 12]','[NAD+ diphosphatase [EC:3.6.1.22]]','[Mus musculus (house mouse)]','[Nicotinate and nicotinamide metabolism, Metabolic pathways, Peroxisome]','[]','[]','00760 Nicotinate and nicotinamide metabolism; 04146 Peroxisome; 3.6.1.22  NAD+ diphosphatase; ','[Pfam: NUDIX Ank_2 Ank_5 Ank_4 Ank_3 Ank zf-NADH-PPase NUDIX-like DNA_ligase_ZBD]'),('108155','mmu:108155','[Ogt, 1110038P24Rik, 4831420N21Rik, AI115525, Ogtl]','[(RefSeq) O-linked N-acetylglucosamine (GlcNAc) transferase (UDP-N-acetylglucosamine:polypeptide-N-acetylglucosaminyl transferase)]','[protein O-GlcNAc transferase [EC:2.4.1.255]]','[Mus musculus (house mouse)]','[Other types of O-glycan biosynthesis, Insulin resistance]','[]','[]','00514 Other types of O-glycan biosynthesis; 04931 Insulin resistance; 01003 Glycosyltransferases [BR:mmu01003]; 03036 Chromosome and associated proteins [BR:mmu03036]; 2.4.1.255  protein O-GlcNAc transferase; O-linked GlcNAc type; NSL complex; ','[Pfam: Glyco_transf_41 TPR_1 TPR_11 TPR_2 TPR_8 TPR_17 TPR_12 TPR_14 TPR_19 TPR_16 TPR_7 TPR_10 TPR_6 TPR_9 TPR-S NatA_aux_su TPR_MalT TPR_15 FAT TPR_21 MIT Wzy_C_2 SNAP BTAD EST1_DNA_bind ANAPC3 Tcf25]'),('23972','mmu:23972','[Papss2, 1810018P12Rik, AI159688, AtpsU2, Atpsk2, Sk2, bm]','[(RefSeq) 3\'-phosphoadenosine 5\'-phosphosulfate synthase 2]','[3\'-phosphoadenosine 5\'-phosphosulfate synthase [EC:2.7.7.4 2.7.1.25]]','[Mus musculus (house mouse)]','[Purine metabolism, Selenocompound metabolism, Sulfur metabolism, Metabolic pathways]','[]','[]','00920 Sulfur metabolism; 00230 Purine metabolism; 00450 Selenocompound metabolism; 00261 Monobactam biosynthesis; 2.7.1.25  adenylyl-sulfate kinase; 2.7.7.4  sulfate adenylyltransferase; ','[Pfam: APS_kinase ATP-sulfurylase PUA_2 AAA_33 AAA_18 KTI12]'),('207728','mmu:207728','[Pde2a, CGS-PDE, cGSPDE]','[(RefSeq) phosphodiesterase 2A, cGMP-stimulated]','[cGMP-dependent 3\',5\'-cyclic phosphodiesterase [EC:3.1.4.17]]','[Mus musculus (house mouse)]','[Purine metabolism, Metabolic pathways, cGMP-PKG signaling pathway, Olfactory transduction, Aldosterone synthesis and secretion, Morphine addiction]','[]','[]','00230 Purine metabolism; 04022 cGMP-PKG signaling pathway; 04925 Aldosterone synthesis and secretion; 04740 Olfactory transduction; 05032 Morphine addiction; 3.1.4.17  3\',5\'-cyclic-nucleotide phosphodiesterase; ','[Pfam: PDEase_I GAF GAF_2 GAF_3]'),('238871','mmu:238871','[Pde4d, 9630011N22Rik, Dpde3]','[(RefSeq) phosphodiesterase 4D, cAMP specific]','[cAMP-specific phosphodiesterase 4 [EC:3.1.4.53]]','[Mus musculus (house mouse)]','[Purine metabolism, Metabolic pathways, cAMP signaling pathway, Parathyroid hormone synthesis, secretion and action, Morphine addiction]','[]','[]','00230 Purine metabolism; 04024 cAMP signaling pathway; 04928 Parathyroid hormone synthesis, secretion and action; 05032 Morphine addiction; 3.1.4.53  3\',5\'-cyclic-AMP phosphodiesterase; ','[Pfam: PDEase_I PDE4_UCR]'),('18618','mmu:18618','[Pemt, PEAMT, PEMT2, PLMT, Pempt, Pempt2]','[(RefSeq) phosphatidylethanolamine N-methyltransferase]','[phosphatidylethanolamine/phosphatidyl-N-methylethanolamine N-methyltransferase [EC:2.1.1.17 2.1.1.71]]','[Mus musculus (house mouse)]','[Glycerophospholipid metabolism, Metabolic pathways]','[]','[]','00564 Glycerophospholipid metabolism; 2.1.1.17  phosphatidylethanolamine N-methyltransferase; 2.1.1.71  phosphatidyl-N-methylethanolamine N-methyltransferase; ','[Pfam: PEMT]'),('170768','mmu:170768','[Pfkfb3, E330010H22Rik, iPFK-2, uPFK-2]','[(RefSeq) 6-phosphofructo-2-kinase/fructose-2,6-biphosphatase 3]','[6-phosphofructo-2-kinase / fructose-2,6-biphosphatase 3 [EC:2.7.1.105 3.1.3.46]]','[Mus musculus (house mouse)]','[Fructose and mannose metabolism, Metabolic pathways, HIF-1 signaling pathway, AMPK signaling pathway]','[]','[]','00051 Fructose and mannose metabolism; 04066 HIF-1 signaling pathway; 04152 AMPK signaling pathway; 2.7.1.105  6-phosphofructo-2-kinase; 3.1.3.46  fructose-2,6-bisphosphate 2-phosphatase; ','[Pfam: 6PF2K His_Phos_1 AAA_33 KTI12 Zeta_toxin SRP54 AAA_18 APS_kinase AAA_17]'),('66171','mmu:66171','[Pgls, 1110030K05Rik, AI447866, Plgs]','[(RefSeq) 6-phosphogluconolactonase]','[6-phosphogluconolactonase [EC:3.1.1.31]]','[Mus musculus (house mouse)]','[Pentose phosphate pathway, Metabolic pathways, Carbon metabolism]','[]','[]','00030 Pentose phosphate pathway; 3.1.1.31  6-phosphogluconolactonase; ','[Pfam: Glucosamine_iso]'),('67078','mmu:67078','[Pgp, 1700012G19Rik, AI481330, AUM, G3PP]','[(RefSeq) phosphoglycolate phosphatase]','[phosphoglycolate phosphatase [EC:3.1.3.18 3.1.3.48]]','[Mus musculus (house mouse)]','[Glyoxylate and dicarboxylate metabolism, Metabolic pathways, Carbon metabolism]','[]','[]','00630 Glyoxylate and dicarboxylate metabolism; 01009 Protein phosphatases and associated proteins [BR:mmu01009]; 3.1.3.18  phosphoglycolate phosphatase; 3.1.3.48  protein-tyrosine-phosphatase; Other HAD phosphatases; ','[Pfam: Hydrolase_6 Hydrolase_like Hydrolase HAD_2 PGP_phosphatase]'),('18770','mmu:18770','[Pklr, Pk-1, Pk1, R-PK]','[(RefSeq) pyruvate kinase liver and red blood cell]','[pyruvate kinase isozymes R/L [EC:2.7.1.40]]','[Mus musculus (house mouse)]','[Glycolysis / Gluconeogenesis, Pyruvate metabolism, Metabolic pathways, Carbon metabolism, Biosynthesis of amino acids, Insulin signaling pathway, Type II diabetes mellitus, Non-alcoholic fatty liver disease, Maturity onset diabetes of the young]','[]','[]','00010 Glycolysis / Gluconeogenesis; 00620 Pyruvate metabolism; 04910 Insulin signaling pathway; 04930 Type II diabetes mellitus; 04950 Maturity onset diabetes of the young; 04932 Non-alcoholic fatty liver disease; 04147 Exosome [BR:mmu04147]; 2.7.1.40  pyruvate kinase; Exosomal proteins of other body fluids (saliva and urine); ','[Pfam: PK PK_C IMPDH]'),('18795','mmu:18795','[Plcb1, 3110043I21Rik, AI132408, Plcb, mKIAA0581]','[(RefSeq) phospholipase C, beta 1]','[phosphatidylinositol phospholipase C, beta [EC:3.1.4.11]]','[Mus musculus (house mouse)]','[Inositol phosphate metabolism, Metabolic pathways, Rap1 signaling pathway, Calcium signaling pathway, cGMP-PKG signaling pathway, Chemokine signaling pathway, Phosphatidylinositol signaling system, Sphingolipid signaling pathway, Phospholipase D signaling pathway, Adrenergic signaling in cardiomyocytes, Vascular smooth muscle contraction, Wnt signaling pathway, Apelin signaling pathway, Gap junction, Platelet activation, Neutrophil extracellular trap formation, NOD-like receptor signaling pathway, Circadian entrainment, Long-term potentiation, Retrograde endocannabinoid signaling, Glutamatergic synapse, Cholinergic synapse, Serotonergic synapse, Dopaminergic synapse, Long-term depression, Taste transduction, Inflammatory mediator regulation of TRP channels, Insulin secretion, GnRH signaling pathway, Estrogen signaling pathway, Melanogenesis, Thyroid hormone synthesis, Thyroid hormone signaling pathway, Oxytocin signaling pathway, Glucagon signaling pathway, Renin secretion, Aldosterone synthesis and secretion, Relaxin signaling pathway, Cortisol synthesis and secretion, Parathyroid hormone synthesis, secretion and action, GnRH secretion, AGE-RAGE signaling pathway in diabetic complications, Cushing syndrome, Growth hormone synthesis, secretion and action, Endocrine and other factor-regulated calcium reabsorption, Salivary secretion, Gastric acid secretion, Pancreatic secretion, Carbohydrate digestion and absorption, Alzheimer disease, Huntington disease, Spinocerebellar ataxia, Pathways of neurodegeneration - multiple diseases, Chagas disease, African trypanosomiasis, Amoebiasis, Human cytomegalovirus infection, Pathways in cancer, Diabetic cardiomyopathy, Lipid and atherosclerosis]','[]','[]','00562 Inositol phosphate metabolism; 04015 Rap1 signaling pathway; 04310 Wnt signaling pathway; 04371 Apelin signaling pathway; 04020 Calcium signaling pathway; 04070 Phosphatidylinositol signaling system; 04072 Phospholipase D signaling pathway; 04071 Sphingolipid signaling pathway; 04022 cGMP-PKG signaling pathway; 04540 Gap junction; 04611 Platelet activation; 04613 Neutrophil extracellular trap formation; 04621 NOD-like receptor signaling pathway; 04062 Chemokine signaling pathway; 04911 Insulin secretion; 04922 Glucagon signaling pathway; 04929 GnRH secretion; 04912 GnRH signaling pathway; 04915 Estrogen signaling pathway; 04921 Oxytocin signaling pathway; 04926 Relaxin signaling pathway; 04935 Growth hormone synthesis, secretion and action; 04918 Thyroid hormone synthesis; 04919 Thyroid hormone signaling pathway; 04928 Parathyroid hormone synthesis, secretion and action; 04916 Melanogenesis; 04924 Renin secretion; 04925 Aldosterone synthesis and secretion; 04927 Cortisol synthesis and secretion; 04261 Adrenergic signaling in cardiomyocytes; 04270 Vascular smooth muscle contraction; 04970 Salivary secretion; 04971 Gastric acid secretion; 04972 Pancreatic secretion; 04973 Carbohydrate digestion and absorption; 04961 Endocrine and other factor-regulated calcium reabsorption; 04724 Glutamatergic synapse; 04725 Cholinergic synapse; 04728 Dopaminergic synapse; 04726 Serotonergic synapse; 04720 Long-term potentiation; 04730 Long-term depression; 04723 Retrograde endocannabinoid signaling; 04742 Taste transduction; 04750 Inflammatory mediator regulation of TRP channels; 04713 Circadian entrainment; 05200 Pathways in cancer; 05163 Human cytomegalovirus infection; 05146 Amoebiasis; 05142 Chagas disease; 05143 African trypanosomiasis; 05010 Alzheimer disease; 05016 Huntington disease; 05017 Spinocerebellar ataxia; 05022 Pathways of neurodegeneration - multiple diseases; 05417 Lipid and atherosclerosis; 05415 Diabetic cardiomyopathy; 04933 AGE-RAGE signaling pathway in diabetic complications; 04934 Cushing syndrome; 04131 Membrane trafficking [BR:mmu04131]; 3.1.4.11  phosphoinositide phospholipase C; Phospholipases; Phospholipase C; ','[Pfam: PI-PLC-X PH_14 PLC-beta_C PI-PLC-Y EF-hand_like DUF1154]'),('104759','mmu:104759','[Pld4, AI132321, thss]','[(RefSeq) phospholipase D family, member 4]','[phospholipase D3/4 [EC:3.1.4.4]]','[Mus musculus (house mouse)]','[Glycerophospholipid metabolism, Ether lipid metabolism, Metabolic pathways]','[]','[]','00564 Glycerophospholipid metabolism; 00565 Ether lipid metabolism; 04131 Membrane trafficking [BR:mmu04131]; 3.1.4.4  phospholipase D; Phospholipases; ','[Pfam: PLDc_3 PLDc_2 PLDc]'),('29858','mmu:29858','[Pmm1, C77612]','[(RefSeq) phosphomannomutase 1]','[phosphomannomutase [EC:5.4.2.8]]','[Mus musculus (house mouse)]','[Fructose and mannose metabolism, Amino sugar and nucleotide sugar metabolism, Metabolic pathways, Biosynthesis of cofactors, Biosynthesis of nucleotide sugars]','[]','[]','00051 Fructose and mannose metabolism; 00520 Amino sugar and nucleotide sugar metabolism; 5.4.2.8  phosphomannomutase; ','[Pfam: PMM Hydrolase_3]'),('69060','mmu:69060','[Pnlip, 1810007A24Rik, PL, PTL]','[(RefSeq) pancreatic lipase]','[pancreatic triacylglycerol lipase [EC:3.1.1.3]]','[Mus musculus (house mouse)]','[Glycerolipid metabolism, Metabolic pathways, Pancreatic secretion, Fat digestion and absorption, Vitamin digestion and absorption]','[]','[]','00561 Glycerolipid metabolism; 04972 Pancreatic secretion; 04975 Fat digestion and absorption; 04977 Vitamin digestion and absorption; 3.1.1.3  triacylglycerol lipase; ','[Pfam: Lipase PLAT Abhydrolase_6 Palm_thioest LIDHydrolase Hydrolase_4 DUF900]'),('18946','mmu:18946','[Pnliprp1, Plrp1]','[(RefSeq) pancreatic lipase related protein 1]','[pancreatic lipase-related protein 1 [EC:3.1.1.3]]','[Mus musculus (house mouse)]','[Glycerolipid metabolism, Metabolic pathways, Pancreatic secretion, Fat digestion and absorption]','[]','[]','00561 Glycerolipid metabolism; 04972 Pancreatic secretion; 04975 Fat digestion and absorption; 3.1.1.3  triacylglycerol lipase; ','[Pfam: Lipase PLAT Abhydrolase_6 Palm_thioest LIDHydrolase Abhydrolase_3 DUF676 DUF900 Brr6_like_C_C Hydrolase_4 Lipase_3]'),('71701','mmu:71701','[Pnpt1, 1200003F12Rik, Old35, PNPase, Pnptl1]','[(RefSeq) polyribonucleotide nucleotidyltransferase 1]','[polyribonucleotide nucleotidyltransferase [EC:2.7.7.8]]','[Mus musculus (house mouse)]','[RNA degradation]','[]','[]','03018 RNA degradation; 03019 Messenger RNA biogenesis [BR:mmu03019]; 03016 Transfer RNA biogenesis [BR:mmu03016]; 2.7.7.8  polyribonucleotide nucleotidyltransferase; Other RNA degradosome components; tRNA degradation factors; ','[Pfam: RNase_PH RNase_PH_C KH_1 PNPase S1]'),('215494','mmu:215494','[Pomgnt2, Ago61, C85492, Gtdc2]','[(RefSeq) protein O-linked mannose beta 1,4-N-acetylglucosaminyltransferase 2]','[protein O-mannose beta-1,4-N-acetylglucosaminyltransferase [EC:2.4.1.312]]','[Mus musculus (house mouse)]','[Mannose type O-glycan biosynthesis, Metabolic pathways]','[]','[]','00515 Mannose type O-glycan biosynthesis; 01003 Glycosyltransferases [BR:mmu01003]; 2.4.1.312  protein O-mannose beta-1,4-N-acetylglucosaminyltransferase; O-Glycan; ','[Pfam: Glyco_transf_61]'),('54397','mmu:54397','[Ppt2, 0610007M19Rik, AA672937]','[(RefSeq) palmitoyl-protein thioesterase 2]','[palmitoyl-protein thioesterase [EC:3.1.2.22]]','[Mus musculus (house mouse)]','[Fatty acid elongation, Metabolic pathways, Fatty acid metabolism, Lysosome]','[]','[]','00062 Fatty acid elongation; 04142 Lysosome; 01004 Lipid biosynthesis proteins [BR:mmu01004]; 3.1.2.22  palmitoyl[protein] hydrolase; Component type; ','[Pfam: Palm_thioest Abhydrolase_1 Abhydrolase_6 DUF676 PGAP1 Lipase_2 Hydrolase_4]'),('21672','mmu:21672','[Prdx2, AL022839, Band-8, NkefB, PRP, PrxII, TDX1, TPx, TPx-B, TR, TSA, Tdpx1, Torin]','[(RefSeq) peroxiredoxin 2]','[peroxiredoxin 2/4 [EC:1.11.1.24]]','[Mus musculus (house mouse)]',NULL,'[]','[]','04147 Exosome [BR:mmu04147]; 1.11.1.24  thioredoxin-dependent peroxiredoxin; Proteins found in most exosomes; ','[Pfam: AhpC-TSA Redoxin 1-cysPrx_C SCO1-SenC]'),('53381','mmu:53381','[Prdx4, AOE372, Prx-iv, Prx4, TRANK]','[(RefSeq) peroxiredoxin 4]','[peroxiredoxin 2/4 [EC:1.11.1.24]]','[Mus musculus (house mouse)]',NULL,'[]','[]','04147 Exosome [BR:mmu04147]; 1.11.1.24  thioredoxin-dependent peroxiredoxin; Proteins found in most exosomes; ','[Pfam: AhpC-TSA Redoxin 1-cysPrx_C]'),('56189','mmu:56189','[Prodh2, 2510028N04Rik, 2510038B11Rik, HYPDH, MmPOX, MmPOX1, POX1]','[(RefSeq) proline dehydrogenase (oxidase) 2]','[hydroxyproline dehydrogenase [EC:1.5.5.3]]','[Mus musculus (house mouse)]','[Arginine and proline metabolism, Metabolic pathways]','[]','[]','00330 Arginine and proline metabolism; 1.5.5.3  hydroxyproline dehydrogenase; ','[Pfam: Pro_dh]'),('19215','mmu:19215','[Ptgds, 21kDa, L-PGDS, PGD2, PGDS, PGDS2, Ptgs3]','[(RefSeq) prostaglandin D2 synthase (brain)]','[prostaglandin-H2 D-isomerase [EC:5.3.99.2]]','[Mus musculus (house mouse)]','[Arachidonic acid metabolism, Metabolic pathways]','[]','[]','00590 Arachidonic acid metabolism; 5.3.99.2  prostaglandin-D synthase; ','[Pfam: Lipocalin Triabin VDE Viral_helicase1]'),('96979','mmu:96979','[Ptges2, 0610038H10Rik, C79137, Gbf1, Mpges2, Pges2]','[(RefSeq) prostaglandin E synthase 2]','[microsomal prostaglandin-E synthase 2 [EC:5.3.99.3]]','[Mus musculus (house mouse)]','[Arachidonic acid metabolism, Metabolic pathways]','[]','[]','00590 Arachidonic acid metabolism; 5.3.99.3  prostaglandin-E synthase; ','[Pfam: GST_N_3 Glutaredoxin GST_N_2 GST_N_4 GST_C_2 GST_C GST_N]'),('19223','mmu:19223','[Ptgis, Cyp8, Cyp8a1, Pgis]','[(RefSeq) prostaglandin I2 (prostacyclin) synthase]','[prostacyclin synthase [EC:5.3.99.4]]','[Mus musculus (house mouse)]','[Arachidonic acid metabolism, Metabolic pathways]','[]','[]','00590 Arachidonic acid metabolism; 00199 Cytochrome P450 [BR:mmu00199]; 5.3.99.4  prostaglandin-I synthase; CYP8 family; ','[Pfam: p450]'),('19703','mmu:19703','[Renbp, Age, Rnbp]','[(RefSeq) renin binding protein]','[N-acylglucosamine 2-epimerase [EC:5.1.3.8]]','[Mus musculus (house mouse)]','[Amino sugar and nucleotide sugar metabolism, Metabolic pathways]','[]','[]','00520 Amino sugar and nucleotide sugar metabolism; 5.1.3.8  N-acylglucosamine 2-epimerase; ','[Pfam: GlcNAc_2-epim]'),('67442','mmu:67442','[Retsat, 0610039N19Rik, C80029, MMT-7, Ppsig]','[(RefSeq) retinol saturase (all trans retinol 13,14 reductase)]','[all-trans-retinol 13,14-reductase [EC:1.3.99.23]]','[Mus musculus (house mouse)]','[Retinol metabolism]','[]','[]','00830 Retinol metabolism; 1.3.99.23  all-trans-retinol 13,14-reductase; ','[Pfam: NAD_binding_8 Amino_oxidase FAD_binding_2 DAO FAD_oxidored Thi4 FAD_binding_3 Pyr_redox_2 HI0933_like MCRA GMC_oxred_N GIDA Pyr_redox AlaDh_PNT_C Pyr_redox_3]'),('19714','mmu:19714','[Rev3l, Rev, Rev3, Sez4]','[(RefSeq) REV3 like, DNA directed polymerase zeta catalytic subunit]','[DNA polymerase zeta [EC:2.7.7.7]]','[Mus musculus (house mouse)]','[Platinum drug resistance, Fanconi anemia pathway]','[]','[]','03460 Fanconi anemia pathway; 01524 Platinum drug resistance; 03400 DNA repair and recombination proteins [BR:mmu03400]; 2.7.7.7  DNA-directed DNA polymerase; B-family DNA polymerases; ','[Pfam: DUF4683 DNA_pol_B DNA_pol_B_exo1 zf-C4pol]'),('67680','mmu:67680','[Sdhb, 0710008N11Rik]','[(RefSeq) succinate dehydrogenase complex, subunit B, iron sulfur (Ip)]','[succinate dehydrogenase (ubiquinone) iron-sulfur subunit [EC:1.3.5.1]]','[Mus musculus (house mouse)]','[Citrate cycle (TCA cycle), Oxidative phosphorylation, Metabolic pathways, Carbon metabolism, Thermogenesis, Non-alcoholic fatty liver disease, Alzheimer disease, Parkinson disease, Amyotrophic lateral sclerosis, Huntington disease, Prion disease, Pathways of neurodegeneration - multiple diseases, Chemical carcinogenesis - reactive oxygen species, Diabetic cardiomyopathy]','[]','[]','00020 Citrate cycle (TCA cycle); 00190 Oxidative phosphorylation; 04714 Thermogenesis; 05208 Chemical carcinogenesis - reactive oxygen species; 05010 Alzheimer disease; 05012 Parkinson disease; 05014 Amyotrophic lateral sclerosis; 05016 Huntington disease; 05020 Prion disease; 05022 Pathways of neurodegeneration - multiple diseases; 05415 Diabetic cardiomyopathy; 04932 Non-alcoholic fatty liver disease; 1.3.5.1  succinate dehydrogenase; ','[Pfam: Fer2_3 Fer4_17 Fer4_8 Fer4_10 Fer2 Fer4_9 Fer4_18 Fer4_7]'),('58994','mmu:58994','[Smpd3, 4631433G07Rik, AI427456, AW537966, fro, nSMase2]','[(RefSeq) sphingomyelin phosphodiesterase 3, neutral]','[sphingomyelin phosphodiesterase 3 [EC:3.1.4.12]]','[Mus musculus (house mouse)]','[Sphingolipid metabolism, Metabolic pathways]','[]','[]','00600 Sphingolipid metabolism; 3.1.4.12  sphingomyelin phosphodiesterase; ','[Pfam: Exo_endo_phos]'),('20860','mmu:20860','[Sult1e1, EST, ST1E1, Ste]','[(RefSeq) sulfotransferase family 1E, member 1]','[estrone sulfotransferase [EC:2.8.2.4]]','[Mus musculus (house mouse)]','[Steroid hormone biosynthesis, Metabolic pathways]','[]','[]','00140 Steroid hormone biosynthesis; 2.8.2.4  estrone sulfotransferase; ','[Pfam: Sulfotransfer_1 Sulfotransfer_3 DUF6090]'),('21391','mmu:21391','[Tbxas1, CYP5, CYP5A1, THAS, TS, TXAS, TXS]','[(RefSeq) thromboxane A synthase 1, platelet]','[thromboxane-A synthase [EC:5.3.99.5]]','[Mus musculus (house mouse)]','[Arachidonic acid metabolism, Metabolic pathways, Platelet activation]','[]','[]','00590 Arachidonic acid metabolism; 04611 Platelet activation; 00199 Cytochrome P450 [BR:mmu00199]; 5.3.99.5  thromboxane-A synthase; CYP5 family; ','[Pfam: p450]'),('56720','mmu:56720','[Tdo2, AA407491, TDO, TO, chky]','[(RefSeq) tryptophan 2,3-dioxygenase]','[tryptophan 2,3-dioxygenase [EC:1.13.11.11]]','[Mus musculus (house mouse)]','[Tryptophan metabolism, Metabolic pathways, Biosynthesis of cofactors]','[]','[]','00380 Tryptophan metabolism; 1.13.11.11  tryptophan 2,3-dioxygenase; ','[Pfam: Trp_dioxygenase]'),('106529','mmu:106529','[Tecr, 2410016D23Rik, A230102P12Rik, AI173355, D17Ertd178e, Gpsn2, SC2]','[(RefSeq) trans-2,3-enoyl-CoA reductase]','[very-long-chain enoyl-CoA reductase [EC:1.3.1.93]]','[Mus musculus (house mouse)]','[Fatty acid elongation, Biosynthesis of unsaturated fatty acids, Metabolic pathways, Fatty acid metabolism]','[]','[]','00062 Fatty acid elongation; 01040 Biosynthesis of unsaturated fatty acids; 01004 Lipid biosynthesis proteins [BR:mmu01004]; 1.3.1.93  very-long-chain enoyl-CoA reductase; Component type; ','[Pfam: Steroid_dh ubiquitin]'),('73166','mmu:73166','[Tm7sf2, 3110041O18Rik, ANG1, C14SR, Dhcr14]','[(RefSeq) transmembrane 7 superfamily member 2]','[Delta14-sterol reductase [EC:1.3.1.70]]','[Mus musculus (house mouse)]','[Steroid biosynthesis, Metabolic pathways]','[]','[]','00100 Steroid biosynthesis; 1.3.1.70  Delta14-sterol reductase; ','[Pfam: ERG4_ERG24 DUF1295 ICMT PEMT]'),('58866','mmu:58866','[Treh, 2210412M19Rik]','[(RefSeq) trehalase (brush-border membrane glycoprotein)]','[alpha,alpha-trehalase [EC:3.2.1.28]]','[Mus musculus (house mouse)]','[Starch and sucrose metabolism, Metabolic pathways]','[]','[]','00500 Starch and sucrose metabolism; 00537 Glycosylphosphatidylinositol (GPI)-anchored proteins [BR:mmu00537]; 3.2.1.28  alpha,alpha-trehalase; Enzymes; ','[Pfam: Trehalase Glyco_hydro_63]'),('22122','mmu:22122','[Gfus, AI256181, FX, Tsta3, Tstap35b]','[(RefSeq) GDP-L-fucose synthase]','[GDP-L-fucose synthase [EC:1.1.1.271]]','[Mus musculus (house mouse)]','[Fructose and mannose metabolism, Amino sugar and nucleotide sugar metabolism, Metabolic pathways, Biosynthesis of nucleotide sugars]','[]','[]','00051 Fructose and mannose metabolism; 00520 Amino sugar and nucleotide sugar metabolism; 1.1.1.271  GDP-L-fucose synthase; ','[Pfam: Epimerase GDP_Man_Dehyd RmlD_sub_bind 3Beta_HSD Polysacc_synt_2 NICE-3]'),('227620','mmu:227620','[Uap1l1, 5730445F03Rik]','[(RefSeq) UDP-N-acteylglucosamine pyrophosphorylase 1-like 1]','[UDP-N-acetylglucosamine/UDP-N-acetylgalactosamine diphosphorylase [EC:2.7.7.23 2.7.7.83]]','[Mus musculus (house mouse)]','[Amino sugar and nucleotide sugar metabolism, Metabolic pathways, Biosynthesis of nucleotide sugars]','[]','[]','00520 Amino sugar and nucleotide sugar metabolism; 2.7.7.23  UDP-N-acetylglucosamine diphosphorylase; 2.7.7.83  UDP-N-acetylgalactosamine diphosphorylase; ','[Pfam: UDPGP NTP_transf_3 TetR_C_11]'),('22235','mmu:22235','[Ugdh, Udpgdh]','[(RefSeq) UDP-glucose dehydrogenase]','[UDPglucose 6-dehydrogenase [EC:1.1.1.22]]','[Mus musculus (house mouse)]','[Pentose and glucuronate interconversions, Ascorbate and aldarate metabolism, Amino sugar and nucleotide sugar metabolism, Metabolic pathways, Biosynthesis of cofactors, Biosynthesis of nucleotide sugars]','[]','[]','00040 Pentose and glucuronate interconversions; 00053 Ascorbate and aldarate metabolism; 00520 Amino sugar and nucleotide sugar metabolism; 1.1.1.22  UDP-glucose 6-dehydrogenase; ','[Pfam: UDPG_MGDP_dh_N UDPG_MGDP_dh_C UDPG_MGDP_dh 3HCDH_N DUF4853 Sacchrp_dh_NADP]'),('71773','mmu:71773','[Ugt2b1, 1300012D20Rik]','[(RefSeq) UDP glucuronosyltransferase 2 family, polypeptide B1]','[glucuronosyltransferase [EC:2.4.1.17]]','[Mus musculus (house mouse)]','[Pentose and glucuronate interconversions, Ascorbate and aldarate metabolism, Steroid hormone biosynthesis, Retinol metabolism, Porphyrin metabolism, Metabolism of xenobiotics by cytochrome P450, Drug metabolism - cytochrome P450, Drug metabolism - other enzymes, Metabolic pathways, Biosynthesis of cofactors, Bile secretion, Chemical carcinogenesis - DNA adducts, Chemical carcinogenesis - receptor activation]','[]','[]','00040 Pentose and glucuronate interconversions; 00053 Ascorbate and aldarate metabolism; 00140 Steroid hormone biosynthesis; 00830 Retinol metabolism; 00860 Porphyrin metabolism; 00980 Metabolism of xenobiotics by cytochrome P450; 00982 Drug metabolism - cytochrome P450; 00983 Drug metabolism - other enzymes; 04976 Bile secretion; 05204 Chemical carcinogenesis - DNA adducts; 05207 Chemical carcinogenesis - receptor activation; 01003 Glycosyltransferases [BR:mmu01003]; 2.4.1.17  glucuronosyltransferase; Hydrophobic molecule; ','[Pfam: UDPGT Glyco_tran_28_C Pro_3_hydrox_C]'),('100727','mmu:100727','[Ugt2b34, AI788959]','[(RefSeq) UDP glucuronosyltransferase 2 family, polypeptide B34]','[glucuronosyltransferase [EC:2.4.1.17]]','[Mus musculus (house mouse)]','[Pentose and glucuronate interconversions, Ascorbate and aldarate metabolism, Steroid hormone biosynthesis, Retinol metabolism, Porphyrin metabolism, Metabolism of xenobiotics by cytochrome P450, Drug metabolism - cytochrome P450, Drug metabolism - other enzymes, Metabolic pathways, Biosynthesis of cofactors, Bile secretion, Chemical carcinogenesis - DNA adducts, Chemical carcinogenesis - receptor activation]','[]','[]','00040 Pentose and glucuronate interconversions; 00053 Ascorbate and aldarate metabolism; 00140 Steroid hormone biosynthesis; 00830 Retinol metabolism; 00860 Porphyrin metabolism; 00980 Metabolism of xenobiotics by cytochrome P450; 00982 Drug metabolism - cytochrome P450; 00983 Drug metabolism - other enzymes; 04976 Bile secretion; 05204 Chemical carcinogenesis - DNA adducts; 05207 Chemical carcinogenesis - receptor activation; 01003 Glycosyltransferases [BR:mmu01003]; 2.4.1.17  glucuronosyltransferase; Hydrophobic molecule; ','[Pfam: UDPGT Glyco_tran_28_C]'),('243085','mmu:243085','[Ugt2b35, C730031G17]','[(RefSeq) UDP glucuronosyltransferase 2 family, polypeptide B35]','[glucuronosyltransferase [EC:2.4.1.17]]','[Mus musculus (house mouse)]','[Pentose and glucuronate interconversions, Ascorbate and aldarate metabolism, Steroid hormone biosynthesis, Retinol metabolism, Porphyrin metabolism, Metabolism of xenobiotics by cytochrome P450, Drug metabolism - cytochrome P450, Drug metabolism - other enzymes, Metabolic pathways, Biosynthesis of cofactors, Bile secretion, Chemical carcinogenesis - DNA adducts, Chemical carcinogenesis - receptor activation]','[]','[]','00040 Pentose and glucuronate interconversions; 00053 Ascorbate and aldarate metabolism; 00140 Steroid hormone biosynthesis; 00830 Retinol metabolism; 00860 Porphyrin metabolism; 00980 Metabolism of xenobiotics by cytochrome P450; 00982 Drug metabolism - cytochrome P450; 00983 Drug metabolism - other enzymes; 04976 Bile secretion; 05204 Chemical carcinogenesis - DNA adducts; 05207 Chemical carcinogenesis - receptor activation; 01003 Glycosyltransferases [BR:mmu01003]; 2.4.1.17  glucuronosyltransferase; Hydrophobic molecule; ','[Pfam: UDPGT Glyco_tran_28_C Pro_3_hydrox_C]'),('22271','mmu:22271','[Upp1, AI325217, UPase, UdRPase, Up, Upp]','[(RefSeq) uridine phosphorylase 1]','[uridine phosphorylase [EC:2.4.2.3]]','[Mus musculus (house mouse)]','[Pyrimidine metabolism, Drug metabolism - other enzymes, Metabolic pathways, Nucleotide metabolism]','[]','[]','00240 Pyrimidine metabolism; 00983 Drug metabolism - other enzymes; 2.4.2.3  uridine phosphorylase; ','[Pfam: PNP_UDP_1 UL97]'),('76654','mmu:76654','[Upp2, 1700124F02Rik, AI266885, UDRPASE2, UP2, UPASE2]','[(RefSeq) uridine phosphorylase 2]','[uridine phosphorylase [EC:2.4.2.3]]','[Mus musculus (house mouse)]','[Pyrimidine metabolism, Drug metabolism - other enzymes, Metabolic pathways, Nucleotide metabolism]','[]','[]','00240 Pyrimidine metabolism; 00983 Drug metabolism - other enzymes; 2.4.2.3  uridine phosphorylase; ','[Pfam: PNP_UDP_1]'),('70008','mmu:70008','[Ace2, 2010305L05Rik]','[(RefSeq) angiotensin I converting enzyme (peptidyl-dipeptidase A) 2]','[angiotensin-converting enzyme 2 [EC:3.4.17.23]]','[Mus musculus (house mouse)]','[Renin-angiotensin system, Protein digestion and absorption, Coronavirus disease - COVID-19]','[]','[]','04614 Renin-angiotensin system; 04974 Protein digestion and absorption; 05171 Coronavirus disease - COVID-19; 01002 Peptidases and inhibitors [BR:mmu01002]; 04147 Exosome [BR:mmu04147]; 3.4.17.23  angiotensin-converting enzyme 2; Family M2; Exosomal proteins of other body fluids (saliva and urine); ','[Pfam: Peptidase_M2 Collectrin Glycophorin_A DUF3827 AbiEi_2]'),('16159','mmu:16159','[Il12a, IL-12p35, Il-12a, Ll12a, p35]','[(RefSeq) interleukin 12a]','[interleukin 12A]','[Mus musculus (house mouse)]','[Cytokine-cytokine receptor interaction, Toll-like receptor signaling pathway, RIG-I-like receptor signaling pathway, C-type lectin receptor signaling pathway, JAK-STAT signaling pathway, Th1 and Th2 cell differentiation, Alcoholic liver disease, Type I diabetes mellitus, Pertussis, Legionellosis, Leishmaniasis, Chagas disease, African trypanosomiasis, Malaria, Toxoplasmosis, Amoebiasis, Tuberculosis, Measles, Influenza A, Herpes simplex virus 1 infection, Coronavirus disease - COVID-19, Pathways in cancer, Inflammatory bowel disease, Allograft rejection, Lipid and atherosclerosis]','[]','[]','04630 JAK-STAT signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04620 Toll-like receptor signaling pathway; 04622 RIG-I-like receptor signaling pathway; 04625 C-type lectin receptor signaling pathway; 04658 Th1 and Th2 cell differentiation; 05200 Pathways in cancer; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05133 Pertussis; 05134 Legionellosis; 05152 Tuberculosis; 05146 Amoebiasis; 05144 Malaria; 05145 Toxoplasmosis; 05140 Leishmaniasis; 05142 Chagas disease; 05143 African trypanosomiasis; 05321 Inflammatory bowel disease; 05330 Allograft rejection; 05417 Lipid and atherosclerosis; 04940 Type I diabetes mellitus; 04936 Alcoholic liver disease; 04052 Cytokines and growth factors [BR:mmu04052]; 00536 Glycosaminoglycan binding proteins [BR:mmu00536]; Interleukins; Cytokines; ','[Pfam: IL12 DUF2564]'),('14825','mmu:14825','[Cxcl1, Fsp, Gro1, KC, Mgsa, N51, Scyb1, gro]','[(RefSeq) chemokine (C-X-C motif) ligand 1]','[C-X-C motif chemokine 1/2/3]','[Mus musculus (house mouse)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, Chemokine signaling pathway, NF-kappa B signaling pathway, NOD-like receptor signaling pathway, IL-17 signaling pathway, TNF signaling pathway, Alcoholic liver disease, Legionellosis, Amoebiasis, Kaposi sarcoma-associated herpesvirus infection, Rheumatoid arthritis, Lipid and atherosclerosis]','[]','[]','04064 NF-kappa B signaling pathway; 04668 TNF signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04621 NOD-like receptor signaling pathway; 04657 IL-17 signaling pathway; 04062 Chemokine signaling pathway; 05167 Kaposi sarcoma-associated herpesvirus infection; 05134 Legionellosis; 05146 Amoebiasis; 05323 Rheumatoid arthritis; 05417 Lipid and atherosclerosis; 04936 Alcoholic liver disease; 04052 Cytokines and growth factors [BR:mmu04052]; Chemokines; ','[Pfam: IL8]'),('20310','mmu:20310','[Cxcl2, CINC-2a, GROb, Gro2, MIP-2, MIP-2a, Mgsa-b, Mip2, Scyb, Scyb2]','[(RefSeq) chemokine (C-X-C motif) ligand 2]','[C-X-C motif chemokine 1/2/3]','[Mus musculus (house mouse)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, Chemokine signaling pathway, NF-kappa B signaling pathway, NOD-like receptor signaling pathway, IL-17 signaling pathway, TNF signaling pathway, Alcoholic liver disease, Legionellosis, Amoebiasis, Kaposi sarcoma-associated herpesvirus infection, Rheumatoid arthritis, Lipid and atherosclerosis]','[]','[]','04064 NF-kappa B signaling pathway; 04668 TNF signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04621 NOD-like receptor signaling pathway; 04657 IL-17 signaling pathway; 04062 Chemokine signaling pathway; 05167 Kaposi sarcoma-associated herpesvirus infection; 05134 Legionellosis; 05146 Amoebiasis; 05323 Rheumatoid arthritis; 05417 Lipid and atherosclerosis; 04936 Alcoholic liver disease; 04052 Cytokines and growth factors [BR:mmu04052]; Chemokines; ','[Pfam: IL8]'),('20859','mmu:20859','[Sult2a1, ST2A1, Std, Sth1, mSTa1]','[(RefSeq) sulfotransferase family 2A, dehydroepiandrosterone (DHEA)-preferring, member 1]','[bile-salt sulfotransferase [EC:2.8.2.14]]','[Mus musculus (house mouse)]','[Metabolism of xenobiotics by cytochrome P450, Bile secretion, Chemical carcinogenesis - DNA adducts]','[]','[]','00980 Metabolism of xenobiotics by cytochrome P450; 04976 Bile secretion; 05204 Chemical carcinogenesis - DNA adducts; 2.8.2.14  bile-salt sulfotransferase; ','[Pfam: Sulfotransfer_1 Sulfotransfer_3]'),('100043194','mmu:100043194','[Sult2a2, C730007P19Rik, ST, ST2A2, Sth2, mSTa2]','[(RefSeq) sulfotransferase family 2A, dehydroepiandrosterone (DHEA)-preferring, member 2]',NULL,'[Mus musculus (house mouse)]',NULL,'[]','[]',NULL,'[Pfam: Sulfotransfer_1]'),('629203','mmu:629203','[Sult2a3, Gm6955]','[(RefSeq) sulfotransferase family 2A, dehydroepiandrosterone (DHEA)-preferring, member 3]','[bile-salt sulfotransferase [EC:2.8.2.14]]','[Mus musculus (house mouse)]','[Metabolism of xenobiotics by cytochrome P450, Bile secretion, Chemical carcinogenesis - DNA adducts]','[]','[]','00980 Metabolism of xenobiotics by cytochrome P450; 04976 Bile secretion; 05204 Chemical carcinogenesis - DNA adducts; 2.8.2.14  bile-salt sulfotransferase; ','[Pfam: Sulfotransfer_1 Sulfotransfer_3]'),('16189','mmu:16189','[Il4, BSF-1, Il-4]','[(RefSeq) interleukin 4]','[interleukin 4]','[Mus musculus (house mouse)]','[Cytokine-cytokine receptor interaction, PI3K-Akt signaling pathway, JAK-STAT signaling pathway, Hematopoietic cell lineage, IL-17 signaling pathway, Th1 and Th2 cell differentiation, Th17 cell differentiation, T cell receptor signaling pathway, Fc epsilon RI signaling pathway, Intestinal immune network for IgA production, Leishmaniasis, Pathways in cancer, Asthma, Autoimmune thyroid disease, Inflammatory bowel disease, Allograft rejection]','[]','[]','04630 JAK-STAT signaling pathway; 04151 PI3K-Akt signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04640 Hematopoietic cell lineage; 04660 T cell receptor signaling pathway; 04658 Th1 and Th2 cell differentiation; 04659 Th17 cell differentiation; 04657 IL-17 signaling pathway; 04664 Fc epsilon RI signaling pathway; 04672 Intestinal immune network for IgA production; 05200 Pathways in cancer; 05140 Leishmaniasis; 05310 Asthma; 05320 Autoimmune thyroid disease; 05321 Inflammatory bowel disease; 05330 Allograft rejection; 04052 Cytokines and growth factors [BR:mmu04052]; 00536 Glycosaminoglycan binding proteins [BR:mmu00536]; Interleukins; Cytokines; ','[Pfam: IL4]'),('19013','mmu:19013','[Ppara, 4933429D07Rik, AW742785, Nr1c1, PPAR-alpha, PPARalpha, Ppar]','[(RefSeq) peroxisome proliferator activated receptor alpha]','[peroxisome proliferator-activated receptor alpha]','[Mus musculus (house mouse)]','[PPAR signaling pathway, cAMP signaling pathway, Adipocytokine signaling pathway, Glucagon signaling pathway, Insulin resistance, Non-alcoholic fatty liver disease, Alcoholic liver disease, Hepatitis C, Chemical carcinogenesis - receptor activation, Diabetic cardiomyopathy]','[]','[]','04024 cAMP signaling pathway; 04922 Glucagon signaling pathway; 04920 Adipocytokine signaling pathway; 03320 PPAR signaling pathway; 05207 Chemical carcinogenesis - receptor activation; 05160 Hepatitis C; 05415 Diabetic cardiomyopathy; 04936 Alcoholic liver disease; 04932 Non-alcoholic fatty liver disease; 04931 Insulin resistance; 03000 Transcription factors [BR:mmu03000]; 03310 Nuclear receptors [BR:mmu03310]; Cys4 thyroid hormone receptor-like; 1C. Peroxisome proliferator-activated receptor (PPAR); ','[Pfam: zf-C4 Hormone_recep]'),('16183','mmu:16183','[Il2, Il-2]','[(RefSeq) interleukin 2]','[interleukin 2]','[Mus musculus (house mouse)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, PI3K-Akt signaling pathway, C-type lectin receptor signaling pathway, JAK-STAT signaling pathway, Th1 and Th2 cell differentiation, Th17 cell differentiation, T cell receptor signaling pathway, Intestinal immune network for IgA production, Type I diabetes mellitus, Yersinia infection, Chagas disease, Measles, Human T-cell leukemia virus 1 infection, Coronavirus disease - COVID-19, Pathways in cancer, Autoimmune thyroid disease, Inflammatory bowel disease, Allograft rejection, Graft-versus-host disease]','[]','[]','04630 JAK-STAT signaling pathway; 04151 PI3K-Akt signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04625 C-type lectin receptor signaling pathway; 04660 T cell receptor signaling pathway; 04658 Th1 and Th2 cell differentiation; 04659 Th17 cell differentiation; 04672 Intestinal immune network for IgA production; 05200 Pathways in cancer; 05166 Human T-cell leukemia virus 1 infection; 05171 Coronavirus disease - COVID-19; 05162 Measles; 05135 Yersinia infection; 05142 Chagas disease; 05320 Autoimmune thyroid disease; 05321 Inflammatory bowel disease; 05330 Allograft rejection; 05332 Graft-versus-host disease; 04940 Type I diabetes mellitus; 04052 Cytokines and growth factors [BR:mmu04052]; 00536 Glycosaminoglycan binding proteins [BR:mmu00536]; Interleukins; Cytokines; ','[Pfam: IL2 Spt20 DivIC CDK5RAP3 FlaG]'),('140781','mmu:140781','[Myh7, B-MHC, MYH-beta/slow, MyHC-I, Myhc-b, Myhcb, beta-MHC, betaMHC, myHC-beta, myHC-slow]','[(RefSeq) myosin, heavy polypeptide 7, cardiac muscle, beta]','[myosin heavy chain 6/7]','[Mus musculus (house mouse)]','[cGMP-PKG signaling pathway, Cardiac muscle contraction, Adrenergic signaling in cardiomyocytes, Thyroid hormone signaling pathway, Hypertrophic cardiomyopathy, Dilated cardiomyopathy, Viral myocarditis]','[]','[]','04022 cGMP-PKG signaling pathway; 04919 Thyroid hormone signaling pathway; 04260 Cardiac muscle contraction; 04261 Adrenergic signaling in cardiomyocytes; 05410 Hypertrophic cardiomyopathy; 05414 Dilated cardiomyopathy; 05416 Viral myocarditis; 04812 Cytoskeleton proteins [BR:mmu04812]; Myosins; ','[Pfam: Myosin_head Myosin_tail_1 Myosin_N IQ AAA_22 Filament Crescentin]'),('230899','mmu:230899','[Nppa, ANP, Anf, CDD, Pnd]','[(RefSeq) natriuretic peptide type A]','[natriuretic peptide A]','[Mus musculus (house mouse)]','[cGMP-PKG signaling pathway, cAMP signaling pathway, HIF-1 signaling pathway, Vascular smooth muscle contraction, Thermogenesis, Oxytocin signaling pathway, Regulation of lipolysis in adipocytes, Renin secretion, Aldosterone synthesis and secretion, African trypanosomiasis]','[]','[]','04066 HIF-1 signaling pathway; 04024 cAMP signaling pathway; 04022 cGMP-PKG signaling pathway; 04923 Regulation of lipolysis in adipocytes; 04921 Oxytocin signaling pathway; 04924 Renin secretion; 04925 Aldosterone synthesis and secretion; 04270 Vascular smooth muscle contraction; 04714 Thermogenesis; 05143 African trypanosomiasis; ','[Pfam: ANP]'),('7442','hsa:7442','[TRPV1, VR1]','[(RefSeq) transient receptor potential cation channel subfamily V member 1]','[transient receptor potential cation channel subfamily V member 1]','[Homo sapiens (human)]','[Neuroactive ligand-receptor interaction, Inflammatory mediator regulation of TRP channels]','[]','[]','04080 Neuroactive ligand-receptor interaction; 04750 Inflammatory mediator regulation of TRP channels; 04040 Ion channels [BR:hsa04040]; Transient receptor potential channel (TRP); ','[Pfam: Ion_trans Ank_2 Ank Ank_3 Ank_5 Ank_4]'),('7350','hsa:7350','[UCP1, SLC25A7, UCP]','[(RefSeq) uncoupling protein 1]','[solute carrier family 25 (mitochondrial uncoupling protein), member 7]','[Homo sapiens (human)]','[PPAR signaling pathway, Apelin signaling pathway, Thermogenesis, Huntington disease]','[]','[]','04371 Apelin signaling pathway; 03320 PPAR signaling pathway; 04714 Thermogenesis; 05016 Huntington disease; 02000 Transporters [BR:hsa02000]; SLC25: Mitochondrial carrier; ','[Pfam: Mito_carr Utp8]'),('10891','hsa:10891','[PPARGC1A, LEM6, PGC-1(alpha), PGC-1alpha, PGC-1v, PGC1, PGC1A, PPARGC1]','[(RefSeq) PPARG coactivator 1 alpha]','[alpha peroxisome proliferator-activated receptor gamma coactivator 1-alpha]','[Homo sapiens (human)]','[AMPK signaling pathway, Longevity regulating pathway, Apelin signaling pathway, Thermogenesis, Insulin signaling pathway, Adipocytokine signaling pathway, Glucagon signaling pathway, Insulin resistance, Alcoholic liver disease, Huntington disease]','[c(\"nt06440  Transcription\", \"nt06461  Huntington disease\")]','[N00981  Mutation-caused aberrant Htt to CREB-mediated transcription]','04371 Apelin signaling pathway; 04152 AMPK signaling pathway; 04910 Insulin signaling pathway; 04922 Glucagon signaling pathway; 04920 Adipocytokine signaling pathway; 04211 Longevity regulating pathway; 04714 Thermogenesis; 05016 Huntington disease; 04936 Alcoholic liver disease; 04931 Insulin resistance; 03029 Mitochondrial biogenesis [BR:hsa03029]; Coactivators; ','[Pfam: RRM_1 Tox-ODYAM1]'),('1375','hsa:1375','[CPT1B, CPT1-M, CPT1M, CPTI, CPTI-M, M-CPT1, MCCPT1, MCPT1]','[(RefSeq) carnitine palmitoyltransferase 1B]','[carnitine O-palmitoyltransferase 1, muscle isoform [EC:2.3.1.21]]','[Homo sapiens (human)]','[Fatty acid degradation, Fatty acid metabolism, PPAR signaling pathway, AMPK signaling pathway, Thermogenesis, Adipocytokine signaling pathway, Glucagon signaling pathway, Insulin resistance, Alcoholic liver disease, Diabetic cardiomyopathy]','[nt06020  beta-Oxidation in mitochondria]','[N00765  beta-Oxidation, acyl-CoA synthesis]','00071 Fatty acid degradation; 04152 AMPK signaling pathway; 04922 Glucagon signaling pathway; 04920 Adipocytokine signaling pathway; 03320 PPAR signaling pathway; 04714 Thermogenesis; 05415 Diabetic cardiomyopathy; 04936 Alcoholic liver disease; 04931 Insulin resistance; 03029 Mitochondrial biogenesis [BR:hsa03029]; 2.3.1.21  carnitine O-palmitoyltransferase; Other regulator of mitochondrial biogenesis; ','[Pfam: Carn_acyltransf CPT_N DUF3128]'),('63976','hsa:63976','[PRDM16, CMD1LL, KMT8F, LVNC8, MEL1, PFM13]','[(RefSeq) PR/SET domain 16]','[[histone H3]-lysine9 N-methyltransferase [EC:2.1.1.367]]','[Homo sapiens (human)]','[Lysine degradation, Metabolic pathways, Thermogenesis]','[]','[]','00310 Lysine degradation; 04714 Thermogenesis; 03000 Transcription factors [BR:hsa03000]; 03036 Chromosome and associated proteins [BR:hsa03036]; 2.1.1.367  [histone H3]-lysine9 N-methyltransferase; Cys2His2 PRDM; HKMTs (histone lysine methyltransferases); ','[Pfam: zf-C2H2 zf-C2H2_4 zf-H2C2_2 zf-C2H2_6 zf-C2H2_jaz zf-H2C2_5 zf-C2H2_11 SET zf-BED zinc_ribbon_15]'),('6899','hsa:6899','[TBX1, CAFS, CATCH22, CTHM, DGCR, DGS, DORV, TBX1C, TGA, VCF, VCFS]','[(RefSeq) T-box transcription factor 1]','[T-box protein 1]','[Homo sapiens (human)]',NULL,'[]','[]','03000 Transcription factors [BR:hsa03000]; T-box; ','[Pfam: T-box]'),('15183','mmu:15183','[Hdac3, AW537363]','[(RefSeq) histone deacetylase 3]','[histone deacetylase 3 [EC:3.5.1.98]]','[Mus musculus (house mouse)]','[Neutrophil extracellular trap formation, Thyroid hormone signaling pathway, Alcoholism, Viral carcinogenesis]','[]','[]','04613 Neutrophil extracellular trap formation; 04919 Thyroid hormone signaling pathway; 05203 Viral carcinogenesis; 05034 Alcoholism; 03036 Chromosome and associated proteins [BR:mmu03036]; 3.5.1.98  histone deacetylase; Class I HDACs; ','[Pfam: Hist_deacetyl]'),('15184','mmu:15184','[Hdac5, AI426555, HD5, Hdac4, mHDA1, mKIAA0600]','[(RefSeq) histone deacetylase 5]','[histone deacetylase 4/5 [EC:3.5.1.98]]','[Mus musculus (house mouse)]','[Apelin signaling pathway, Neutrophil extracellular trap formation, Alcoholism, Viral carcinogenesis, MicroRNAs in cancer]','[]','[]','04371 Apelin signaling pathway; 04613 Neutrophil extracellular trap formation; 05206 MicroRNAs in cancer; 05203 Viral carcinogenesis; 05034 Alcoholism; 03036 Chromosome and associated proteins [BR:mmu03036]; 3.5.1.98  histone deacetylase; Class II HDACs; ','[Pfam: Hist_deacetyl HDAC4_Gln TFIIA]'),('20464','mmu:20464','[Sim1, bHLHe14, mSIM1]','[(RefSeq) single-minded family bHLH transcription factor 1]','[single-minded]','[Mus musculus (house mouse)]',NULL,'[]','[]','03000 Transcription factors [BR:mmu03000]; Factors with PAS domain; ','[Pfam: SIM_C PAS_3 PAS_11 PAS HLH PAS_9 PAS_4]'),('20852','mmu:20852','[Stat6]','[(RefSeq) signal transducer and activator of transcription 6]','[signal transducer and activator of transcription 6]','[Mus musculus (house mouse)]','[Necroptosis, JAK-STAT signaling pathway, Th1 and Th2 cell differentiation, Th17 cell differentiation, Hepatitis B, Pathways in cancer, Inflammatory bowel disease]','[]','[]','04630 JAK-STAT signaling pathway; 04217 Necroptosis; 04658 Th1 and Th2 cell differentiation; 04659 Th17 cell differentiation; 05200 Pathways in cancer; 05161 Hepatitis B; 05321 Inflammatory bowel disease; 03000 Transcription factors [BR:mmu03000]; STAT; ','[Pfam: STAT6_C STAT_bind STAT_int STAT_alpha SH2]'),('20482','mmu:20482','[Skil, Skir, SnoN, sno]','[(RefSeq) SKI-like]','[Ski-like protein]','[Mus musculus (house mouse)]','[Signaling pathways regulating pluripotency of stem cells]','[]','[]','04550 Signaling pathways regulating pluripotency of stem cells; ','[Pfam: Ski_Sno c-SKI_SMAD_bind DUF489 HlyD_2]'),('19213','mmu:19213','[Ptf1a, PTF1-p48, PTF1p48, bHLHa29]','[(RefSeq) pancreas specific transcription factor, 1a]','[pancreas-specific transcription factor 1a]','[Mus musculus (house mouse)]',NULL,'[]','[]','03000 Transcription factors [BR:mmu03000]; Tal/Twist/Atonal/Hen, Mesodermal Twist-like factors; ','[Pfam: HLH]'),('19401','mmu:19401','[Rara, Nr1b1, RAR, RARalpha1]','[(RefSeq) retinoic acid receptor, alpha]','[retinoic acid receptor alpha]','[Mus musculus (house mouse)]','[Th17 cell differentiation, Estrogen signaling pathway, Pathways in cancer, Transcriptional misregulation in cancer, Acute myeloid leukemia]','[]','[]','04659 Th17 cell differentiation; 04915 Estrogen signaling pathway; 05200 Pathways in cancer; 05202 Transcriptional misregulation in cancer; 05221 Acute myeloid leukemia; 03000 Transcription factors [BR:mmu03000]; 03310 Nuclear receptors [BR:mmu03310]; Cys4 thyroid hormone receptor-like; 1B. Retinoic acid receptor (RAR); ','[Pfam: zf-C4 Hormone_recep]'),('11864','mmu:11864','[Arnt2, Hif-2b, bHLHe1, mKIAA0307]','[(RefSeq) aryl hydrocarbon receptor nuclear translocator 2]','[aryl hydrocarbon receptor nuclear translocator 2]','[Mus musculus (house mouse)]','[Pathways in cancer, Transcriptional misregulation in cancer, Renal cell carcinoma]','[]','[]','05200 Pathways in cancer; 05202 Transcriptional misregulation in cancer; 05211 Renal cell carcinoma; 03000 Transcription factors [BR:mmu03000]; Factors with PAS domain; ','[Pfam: PAS_3 PAS PAS_11 HLH PAS_9 PAS_4 PAS_2]'),('16392','mmu:16392','[Isl1]','[(RefSeq) ISL1 transcription factor, LIM/homeodomain]','[insulin gene enhancer protein ISL-1]','[Mus musculus (house mouse)]','[Signaling pathways regulating pluripotency of stem cells]','[]','[]','04550 Signaling pathways regulating pluripotency of stem cells; 03000 Transcription factors [BR:mmu03000]; Homeo domain LIM; ','[Pfam: LIM Homeodomain Homeobox_KN MerR_1]'),('14815','mmu:14815','[Nr3c1, GR, Grl-1, Grl1]','[(RefSeq) nuclear receptor subfamily 3, group C, member 1]','[glucocorticoid receptor]','[Mus musculus (house mouse)]','[Neuroactive ligand-receptor interaction]','[]','[]','04080 Neuroactive ligand-receptor interaction; 03000 Transcription factors [BR:mmu03000]; 03310 Nuclear receptors [BR:mmu03310]; Cys4 estrogen receptor-like; 3C. 3-Ketosteroid receptor; ','[Pfam: GCR Hormone_recep zf-C4 Med3 RAB3GAP2_N]'),('21848','mmu:21848','[Trim24, A130082H20Rik, AI447469, D430004I05Rik, TIF1, TIF1-alpha, TIF1alpha, Tif1a]','[(RefSeq) tripartite motif-containing 24]','[tripartite motif-containing protein 24 [EC:2.3.2.27]]','[Mus musculus (house mouse)]',NULL,'[]','[]','01001 Protein kinases [BR:mmu01001]; 04121 Ubiquitin system [BR:mmu04121]; 2.3.2.27  RING-type E3 ubiquitin transferase; TIF1 family [OT]; TRIM/RBCC proteins; ','[Pfam: Bromodomain zf-B_box PHD zf-RING_UBOX zf-RING_5 zf-RING_2 zf-C3HC4 zf-C3HC4_2 CRIC_ras_sig]'),('75605','mmu:75605','[Kdm5b, 2010009J12Rik, 2210016I17Rik, AW556288, D1Ertd202e, Jarid1b, PLU-1, PUT1, Plu1, RBBP2H1A, Rb-Bp2, mKIAA4034]','[(RefSeq) lysine (K)-specific demethylase 5B]','[[histone H3]-trimethyl-L-lysine4 demethylase [EC:1.14.11.67]]','[Mus musculus (house mouse)]',NULL,'[]','[]','01009 Protein phosphatases and associated proteins [BR:mmu01009]; 03036 Chromosome and associated proteins [BR:mmu03036]; 1.14.11.67  [histone H3]-trimethyl-L-lysine4 demethylase; PP1-interacting proteins (PIPs); Histone demethylases; ','[Pfam: PLU-1 JmjC PHD ARID JmjN zf-C5HC2 PHD_2 zf-PHD-like FYVE_2]'),('18012','mmu:18012','[Neurod1, BETA2, BHF-1, Nd1, Neurod, bHLHa3]','[(RefSeq) neurogenic differentiation 1]','[neurogenic differentiation factor 1]','[Mus musculus (house mouse)]','[Maturity onset diabetes of the young]','[]','[]','04950 Maturity onset diabetes of the young; 03000 Transcription factors [BR:mmu03000]; Tal/Twist/Atonal/Hen, Atonal; ','[Pfam: Neuro_bHLH HLH CCDC106 YL1 Nop14 CDC45 Nop53 Drc1-Sld2]'),('20674','mmu:20674','[Sox2, Sox-2, lcc, ysb]','[(RefSeq) SRY (sex determining region Y)-box 2]','[transcription factor SOX2 (SOX group B)]','[Mus musculus (house mouse)]','[Hippo signaling pathway, Signaling pathways regulating pluripotency of stem cells]','[]','[]','04390 Hippo signaling pathway; 04550 Signaling pathways regulating pluripotency of stem cells; 03000 Transcription factors [BR:mmu03000]; HMG SOX; ','[Pfam: SOXp HMG_box HMG_box_2]'),('14462','mmu:14462','[Gata3, Gata-3, jal]','[(RefSeq) GATA binding protein 3]','[GATA-binding protein 3]','[Mus musculus (house mouse)]','[Th1 and Th2 cell differentiation, Th17 cell differentiation, Parathyroid hormone synthesis, secretion and action, Inflammatory bowel disease]','[]','[]','04658 Th1 and Th2 cell differentiation; 04659 Th17 cell differentiation; 04928 Parathyroid hormone synthesis, secretion and action; 05321 Inflammatory bowel disease; 03000 Transcription factors [BR:mmu03000]; Cys4 GATA-factors; ','[Pfam: GATA TF_Zn_Ribbon DZR_2 ArfGap Prim_Zn_Ribbon]'),('11863','mmu:11863','[Arnt, D3Ertd557e, Drnt, ESTM42, Hif1b, W08714, bHLHe2, mKIAA4051]','[(RefSeq) aryl hydrocarbon receptor nuclear translocator]','[aryl hydrocarbon receptor nuclear translocator]','[Mus musculus (house mouse)]','[HIF-1 signaling pathway, Cushing syndrome, Pathways in cancer, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species, Renal cell carcinoma]','[]','[]','04066 HIF-1 signaling pathway; 05200 Pathways in cancer; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 05211 Renal cell carcinoma; 04934 Cushing syndrome; 03000 Transcription factors [BR:mmu03000]; Factors with PAS domain; ','[Pfam: PAS_3 PAS_11 PAS HLH PAS_9 PAS_4]'),('15379','mmu:15379','[Onecut1, D9Ertd423e, HNF-6, Hfh12, Hnf6, Hnf6b, OC-1, Oc1]','[(RefSeq) one cut domain, family member 1]','[one cut domain, family member 1 (hepatocyte nuclear factor 6)]','[Mus musculus (house mouse)]','[Signaling pathways regulating pluripotency of stem cells, Maturity onset diabetes of the young]','[]','[]','04550 Signaling pathways regulating pluripotency of stem cells; 04950 Maturity onset diabetes of the young; 03000 Transcription factors [BR:mmu03000]; Homeo domain CUT; ','[Pfam: CUT Homeodomain Homeobox_KN]'),('20666','mmu:20666','[Sox11, 1110038H03Rik, 6230403H02Rik, AI836553, end1]','[(RefSeq) SRY (sex determining region Y)-box 11]','[transcription factor SOX11/12 (SOX group C)]','[Mus musculus (house mouse)]',NULL,'[]','[]','03000 Transcription factors [BR:mmu03000]; HMG SOX; ','[Pfam: HMG_box HMG_box_2 PRR18]'),('19698','mmu:19698','[Relb, shep]','[(RefSeq) avian reticuloendotheliosis viral (v-rel) oncogene related B]','[transcription factor RelB]','[Mus musculus (house mouse)]','[MAPK signaling pathway, NF-kappa B signaling pathway, Osteoclast differentiation, C-type lectin receptor signaling pathway, Human T-cell leukemia virus 1 infection, Epstein-Barr virus infection]','[]','[]','04010 MAPK signaling pathway; 04064 NF-kappa B signaling pathway; 04625 C-type lectin receptor signaling pathway; 04380 Osteoclast differentiation; 05166 Human T-cell leukemia virus 1 infection; 05169 Epstein-Barr virus infection; 03000 Transcription factors [BR:mmu03000]; RHR (Rel homology region) Rel/Ankyrin; ','[Pfam: RelB_transactiv RHD_DNA_bind RelB_leu_zip RHD_dimer]'),('14463','mmu:14463','[Gata4, Gata-4]','[(RefSeq) GATA binding protein 4]','[GATA-binding protein 4]','[Mus musculus (house mouse)]','[cGMP-PKG signaling pathway, Cellular senescence, Tight junction, Thyroid hormone signaling pathway]','[]','[]','04022 cGMP-PKG signaling pathway; 04218 Cellular senescence; 04530 Tight junction; 04919 Thyroid hormone signaling pathway; 03000 Transcription factors [BR:mmu03000]; Cys4 GATA-factors; ','[Pfam: GATA-N GATA TF_Zn_Ribbon Zn-ribbon_8 DUF2322 ArfGap Apidaecin Prim_Zn_Ribbon]'),('12591','mmu:12591','[Cdx2, Cdx-2]','[(RefSeq) caudal type homeobox 2]','[homeobox protein CDX2]','[Mus musculus (house mouse)]','[Gastric cancer]','[]','[]','05226 Gastric cancer; 03000 Transcription factors [BR:mmu03000]; Homeo domain ANTP: HOXL; ','[Pfam: Caudal_act Homeodomain Homeobox_KN SUIM_assoc]'),('23872','mmu:23872','[Ets2, AU022856, Ets-2]','[(RefSeq) E26 avian leukemia oncogene 2, 3\' domain]','[C-ets-2]','[Mus musculus (house mouse)]','[Ras signaling pathway, Human T-cell leukemia virus 1 infection]','[]','[]','04014 Ras signaling pathway; 05166 Human T-cell leukemia virus 1 infection; 03000 Transcription factors [BR:mmu03000]; Tryptophan clusters Ets-type; ','[Pfam: Ets SAM_PNT Ets1_N_flank HSF_DNA-bind]'),('21410','mmu:21410','[Hnf1b, HNF-1-beta, HNF-1B, HNF-1Beta, Hnf1beta, LFB3, Tcf-2, Tcf2, vHNF1]','[(RefSeq) HNF1 homeobox B]','[transcription factor 2, hepatocyte nuclear factor 1-beta]','[Mus musculus (house mouse)]','[Maturity onset diabetes of the young]','[]','[]','04950 Maturity onset diabetes of the young; 03000 Transcription factors [BR:mmu03000]; Homeo domain POU; ','[Pfam: HNF-1B_C HNF-1_N Homeodomain Homeobox_KN APP1_cat]'),('18128','mmu:18128','[Notch1, 9930111A19Rik, Mis6, N1, Tan1, lin-12]','[(RefSeq) notch 1]','[Notch 1]','[Mus musculus (house mouse)]','[Endocrine resistance, Notch signaling pathway, Th1 and Th2 cell differentiation, Thyroid hormone signaling pathway, Prion disease, Human papillomavirus infection, Pathways in cancer, MicroRNAs in cancer, Breast cancer]','[]','[]','04330 Notch signaling pathway; 04658 Th1 and Th2 cell differentiation; 04919 Thyroid hormone signaling pathway; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05224 Breast cancer; 05165 Human papillomavirus infection; 05020 Prion disease; 01522 Endocrine resistance; ','[Pfam: EGF hEGF EGF_CA EGF_2 Ank_2 NOD Notch Ank NODP Ank_5 Ank_4 Ank_3 EGF_3 cEGF]'),('54139','mmu:54139','[Irf6, AI876454, E230028I05Rik, mirf6]','[(RefSeq) interferon regulatory factor 6]','[interferon regulatory factor 6]','[Mus musculus (house mouse)]',NULL,'[]','[]','03000 Transcription factors [BR:mmu03000]; 04147 Exosome [BR:mmu04147]; Tryptophan clusters interferon-regulating factors; Exosomal proteins of other body fluids (saliva and urine); ','[Pfam: IRF-3 IRF]'),('20466','mmu:20466','[Sin3a, AW553200, Sin3, mKIAA4126, mSin3A]','[(RefSeq) transcriptional regulator, SIN3A (yeast)]','[paired amphipathic helix protein Sin3a]','[Mus musculus (house mouse)]','[Thyroid hormone signaling pathway, Huntington disease, Epstein-Barr virus infection, Transcriptional misregulation in cancer]','[]','[]','04919 Thyroid hormone signaling pathway; 05202 Transcriptional misregulation in cancer; 05169 Epstein-Barr virus infection; 05016 Huntington disease; 03036 Chromosome and associated proteins [BR:mmu03036]; Sin3A-HDAC complex; Other heterochromatin formation proteins; ','[Pfam: Sin3a_C PAH Sin3_corepress]'),('26992','mmu:26992','[Brd7, BP75, CELTIX1, Ptpn13ip]','[(RefSeq) bromodomain containing 7]','[bromodomain-containing protein 7]','[Mus musculus (house mouse)]','[Hepatocellular carcinoma]','[]','[]','05225 Hepatocellular carcinoma; 03036 Chromosome and associated proteins [BR:mmu03036]; PBAF complex; ','[Pfam: DUF3512 Bromodomain GntR]'),('21413','mmu:21413','[Tcf4, 5730422P05Rik, ASP-I2, E2-2, E2.2, ITF-2, ITF-2b, ITF2, ME2, MITF-2A, MITF-2B, SEF-2, SEF2, SEF2-1, TFE, Tcf-4, bHLHb19]','[(RefSeq) transcription factor 4]','[transcription factor 4/12]','[Mus musculus (house mouse)]',NULL,'[]','[]','03000 Transcription factors [BR:mmu03000]; Ubiquitous (class A) factors; ','[Pfam: HLH]'),('17865','mmu:17865','[Mybl2, B-Myb, Bmyb]','[(RefSeq) myeloblastosis oncogene-like 2]','[myb-related protein B]','[Mus musculus (house mouse)]','[Cellular senescence]','[]','[]','04218 Cellular senescence; 03000 Transcription factors [BR:mmu03000]; Tryptophan clusters Myb, Myb-factors; ','[Pfam: Cmyb_C Myb_DNA-binding Myb_DNA-bind_6 Myb_DNA-bind_7 Spore_III_AB DUF1348]'),('14489','mmu:14489','[Mtpn, 5033418D15Rik, Gcdp, V1]','[(RefSeq) myotrophin]',NULL,'[Mus musculus (house mouse)]',NULL,'[]','[]',NULL,'[Pfam: Ank_4 Ank_2 Ank_5 Ank_3 Ank DUF655]'),('13982','mmu:13982','[Esr1, ER, ER-alpha, ERa, ERalpha, ESR, Estr, Estra, Nr3a1]','[(RefSeq) estrogen receptor 1 (alpha)]','[estrogen receptor alpha]','[Mus musculus (house mouse)]','[Endocrine resistance, Estrogen signaling pathway, Prolactin signaling pathway, Thyroid hormone signaling pathway, Endocrine and other factor-regulated calcium reabsorption, Pathways in cancer, Proteoglycans in cancer, Chemical carcinogenesis - receptor activation, Breast cancer]','[]','[]','04915 Estrogen signaling pathway; 04917 Prolactin signaling pathway; 04919 Thyroid hormone signaling pathway; 04961 Endocrine and other factor-regulated calcium reabsorption; 05200 Pathways in cancer; 05205 Proteoglycans in cancer; 05207 Chemical carcinogenesis - receptor activation; 05224 Breast cancer; 01522 Endocrine resistance; 03000 Transcription factors [BR:mmu03000]; 03310 Nuclear receptors [BR:mmu03310]; Cys4 estrogen receptor-like; 3A. Estrogen receptor; ','[Pfam: Oest_recep Hormone_recep zf-C4 ESR1_C]'),('14235','mmu:14235','[Foxm1, AA408308, AW554517, BB238854, Fkh16, Foxm1b, HFH-11B, MPHOSPH2, Mpm2, WIN]','[(RefSeq) forkhead box M1]','[forkhead box protein M]','[Mus musculus (house mouse)]','[Cellular senescence]','[]','[]','04218 Cellular senescence; 03000 Transcription factors [BR:mmu03000]; Fork head/winged helix other regulators; ','[Pfam: Forkhead]'),('242705','mmu:242705','[E2f2, E2F-2]','[(RefSeq) E2F transcription factor 2]','[transcription factor E2F2]','[Mus musculus (house mouse)]','[Endocrine resistance, Cell cycle, Cellular senescence, Cushing syndrome, Hepatitis C, Hepatitis B, Human cytomegalovirus infection, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Epstein-Barr virus infection, Pathways in cancer, MicroRNAs in cancer, Pancreatic cancer, Glioma, Prostate cancer, Melanoma, Bladder cancer, Chronic myeloid leukemia, Small cell lung cancer, Non-small cell lung cancer, Breast cancer, Hepatocellular carcinoma, Gastric cancer]','[]','[]','04110 Cell cycle; 04218 Cellular senescence; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05212 Pancreatic cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05214 Glioma; 05220 Chronic myeloid leukemia; 05218 Melanoma; 05219 Bladder cancer; 05215 Prostate cancer; 05224 Breast cancer; 05222 Small cell lung cancer; 05223 Non-small cell lung cancer; 05166 Human T-cell leukemia virus 1 infection; 05161 Hepatitis B; 05160 Hepatitis C; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 04934 Cushing syndrome; 01522 Endocrine resistance; 03000 Transcription factors [BR:mmu03000]; Fork head/winged helix cell cycle-controlling factors, E2F; ','[Pfam: E2F_CC-MB E2F_TDP]'),('13557','mmu:13557','[E2f3, E2F3b, E2f3a]','[(RefSeq) E2F transcription factor 3]','[transcription factor E2F3]','[Mus musculus (house mouse)]','[Endocrine resistance, Cell cycle, Cellular senescence, Cushing syndrome, Hepatitis C, Hepatitis B, Human cytomegalovirus infection, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Epstein-Barr virus infection, Pathways in cancer, MicroRNAs in cancer, Pancreatic cancer, Glioma, Prostate cancer, Melanoma, Bladder cancer, Chronic myeloid leukemia, Small cell lung cancer, Non-small cell lung cancer, Breast cancer, Hepatocellular carcinoma, Gastric cancer]','[]','[]','04110 Cell cycle; 04218 Cellular senescence; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05212 Pancreatic cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05214 Glioma; 05220 Chronic myeloid leukemia; 05218 Melanoma; 05219 Bladder cancer; 05215 Prostate cancer; 05224 Breast cancer; 05222 Small cell lung cancer; 05223 Non-small cell lung cancer; 05166 Human T-cell leukemia virus 1 infection; 05161 Hepatitis B; 05160 Hepatitis C; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 04934 Cushing syndrome; 01522 Endocrine resistance; 03000 Transcription factors [BR:mmu03000]; Fork head/winged helix cell cycle-controlling factors, E2F; ','[Pfam: E2F_CC-MB E2F_TDP]'),('22642','mmu:22642','[Zbtb17, AA589413, Lp-1, Miz1, Zfp100, mZ13]','[(RefSeq) zinc finger and BTB domain containing 17]','[zinc finger and BTB domain-containing protein 17]','[Mus musculus (house mouse)]','[Cell cycle, Pathways in cancer, Transcriptional misregulation in cancer, Small cell lung cancer]','[]','[]','04110 Cell cycle; 05200 Pathways in cancer; 05202 Transcriptional misregulation in cancer; 05222 Small cell lung cancer; 03000 Transcription factors [BR:mmu03000]; 04121 Ubiquitin system [BR:mmu04121]; Cys2His2 BTB-ZF factors; Adoptor and target recognizing subunit (BTB); ','[Pfam: zf-H2C2_2 zf-C2H2 zf-C2H2_4 BTB zf-C2H2_jaz zf-C2H2_11 zf-met zf-H2C2_5 zf-C2H2_6 Zn-ribbon_8 zf-Di19 zf-C2HC_2 zf-BED zinc_ribbon_15 zf-C2HE zf_UBZ Packaging_FI HypA]'),('12151','mmu:12151','[Bmi1, AW546694, Bmi-1, Pcgf4]','[(RefSeq) Bmi1 polycomb ring finger oncogene]','[polycomb group RING finger protein 4]','[Mus musculus (house mouse)]','[Signaling pathways regulating pluripotency of stem cells, Transcriptional misregulation in cancer, MicroRNAs in cancer]','[]','[]','04550 Signaling pathways regulating pluripotency of stem cells; 05202 Transcriptional misregulation in cancer; 05206 MicroRNAs in cancer; 03036 Chromosome and associated proteins [BR:mmu03036]; PRC1 (polycomb repressive complex 1); ','[Pfam: RAWUL zf-C3HC4_2 zf-C3HC4 zf-RING_2 zf-C3HC4_3 zf-RING_5 zf-rbx1 zf-RING_UBOX Prok-RING_4 zf-C3HC4_4 zf-RING_10 zf-Nse]'),('116870','mmu:116870','[Mta1]','[(RefSeq) metastasis associated 1]','[metastasis-associated protein MTA]','[Mus musculus (house mouse)]',NULL,'[]','[]','03000 Transcription factors [BR:mmu03000]; 03036 Chromosome and associated proteins [BR:mmu03036]; Cys4 GATA-factors; NuRD complex; ','[Pfam: MTA_R1 BAH ELM2 GATA Myb_DNA-binding Myb_DNA-bind_7]'),('21652','mmu:21652','[Phf1, AW557215, D17Ertd455e, Pcl1, Phf2, Tctex3]','[(RefSeq) PHD finger protein 1]','[PHD finger protein 1]','[Mus musculus (house mouse)]',NULL,'[]','[]','03000 Transcription factors [BR:mmu03000]; 03036 Chromosome and associated proteins [BR:mmu03036]; Other zinc fingers; Other polycomb group proteins; ','[Pfam: Tudor_2 PHD Mtf2_C PHD_2 DUF4537 Chromo C1_1]'),('15375','mmu:15375','[Foxa1, Hnf-3a, Hnf3a, Tcf-3a, Tcf3a]','[(RefSeq) forkhead box A1]','[forkhead box protein A1, hepatocyte nuclear factor 3-alpha]','[Mus musculus (house mouse)]',NULL,'[]','[]','03000 Transcription factors [BR:mmu03000]; Fork head/winged helix tissue-specific regulators; ','[Pfam: Forkhead HNF_C Forkhead_N]'),('17126','mmu:17126','[Smad2, 7120426M23Rik, Madh2, Madr2, Smad-2, mMad2]','[(RefSeq) SMAD family member 2]','[mothers against decapentaplegic homolog 2]','[Mus musculus (house mouse)]','[Cell cycle, Endocytosis, Cellular senescence, TGF-beta signaling pathway, Apelin signaling pathway, Hippo signaling pathway, Signaling pathways regulating pluripotency of stem cells, Th17 cell differentiation, Relaxin signaling pathway, AGE-RAGE signaling pathway in diabetic complications, Chagas disease, Human T-cell leukemia virus 1 infection, Pathways in cancer, Proteoglycans in cancer, Colorectal cancer, Pancreatic cancer, Hepatocellular carcinoma, Gastric cancer, Inflammatory bowel disease, Diabetic cardiomyopathy]','[]','[]','04350 TGF-beta signaling pathway; 04390 Hippo signaling pathway; 04371 Apelin signaling pathway; 04144 Endocytosis; 04110 Cell cycle; 04218 Cellular senescence; 04550 Signaling pathways regulating pluripotency of stem cells; 04659 Th17 cell differentiation; 04926 Relaxin signaling pathway; 05200 Pathways in cancer; 05205 Proteoglycans in cancer; 05210 Colorectal cancer; 05212 Pancreatic cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05166 Human T-cell leukemia virus 1 infection; 05142 Chagas disease; 05321 Inflammatory bowel disease; 05415 Diabetic cardiomyopathy; 04933 AGE-RAGE signaling pathway in diabetic complications; ','[Pfam: MH2 MH1 IRF-3]'),('17127','mmu:17127','[Smad3, AU022421, Madh3]','[(RefSeq) SMAD family member 3]','[mothers against decapentaplegic homolog 3]','[Mus musculus (house mouse)]','[FoxO signaling pathway, Cell cycle, Endocytosis, Cellular senescence, Wnt signaling pathway, TGF-beta signaling pathway, Apelin signaling pathway, Hippo signaling pathway, Adherens junction, Signaling pathways regulating pluripotency of stem cells, Th17 cell differentiation, AGE-RAGE signaling pathway in diabetic complications, Hepatitis B, Human T-cell leukemia virus 1 infection, Pathways in cancer, Colorectal cancer, Pancreatic cancer, Chronic myeloid leukemia, Hepatocellular carcinoma, Gastric cancer, Inflammatory bowel disease, Diabetic cardiomyopathy]','[]','[]','04310 Wnt signaling pathway; 04350 TGF-beta signaling pathway; 04390 Hippo signaling pathway; 04371 Apelin signaling pathway; 04068 FoxO signaling pathway; 04144 Endocytosis; 04110 Cell cycle; 04218 Cellular senescence; 04520 Adherens junction; 04550 Signaling pathways regulating pluripotency of stem cells; 04659 Th17 cell differentiation; 05200 Pathways in cancer; 05210 Colorectal cancer; 05212 Pancreatic cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05220 Chronic myeloid leukemia; 05166 Human T-cell leukemia virus 1 infection; 05161 Hepatitis B; 05321 Inflammatory bowel disease; 05415 Diabetic cardiomyopathy; 04933 AGE-RAGE signaling pathway in diabetic complications; ','[Pfam: MH2 MH1 IRF-3]'),('17128','mmu:17128','[Smad4, AW743858, D18Wsu70e, DPC4, Madh4]','[(RefSeq) SMAD family member 4]','[mothers against decapentaplegic homolog 4]','[Mus musculus (house mouse)]','[FoxO signaling pathway, Cell cycle, Wnt signaling pathway, TGF-beta signaling pathway, Apelin signaling pathway, Hippo signaling pathway, Adherens junction, Signaling pathways regulating pluripotency of stem cells, Th17 cell differentiation, AGE-RAGE signaling pathway in diabetic complications, Hepatitis B, Human T-cell leukemia virus 1 infection, Pathways in cancer, Colorectal cancer, Pancreatic cancer, Chronic myeloid leukemia, Hepatocellular carcinoma, Gastric cancer]','[]','[]','04310 Wnt signaling pathway; 04350 TGF-beta signaling pathway; 04390 Hippo signaling pathway; 04371 Apelin signaling pathway; 04068 FoxO signaling pathway; 04110 Cell cycle; 04520 Adherens junction; 04550 Signaling pathways regulating pluripotency of stem cells; 04659 Th17 cell differentiation; 05200 Pathways in cancer; 05210 Colorectal cancer; 05212 Pancreatic cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05220 Chronic myeloid leukemia; 05166 Human T-cell leukemia virus 1 infection; 05161 Hepatitis B; 04933 AGE-RAGE signaling pathway in diabetic complications; ','[Pfam: MH2 MH1 IRF-3]'),('21781','mmu:21781','[Tfdp1, Dp1, Drtf1]','[(RefSeq) transcription factor Dp 1]','[transcription factor Dp-1]','[Mus musculus (house mouse)]','[Cell cycle, TGF-beta signaling pathway]','[]','[]','04350 TGF-beta signaling pathway; 04110 Cell cycle; 03000 Transcription factors [BR:mmu03000]; Fork head/winged helix cell cycle-controlling factors, DP; ','[Pfam: DP E2F_TDP]'),('18667','mmu:18667','[Pgr, 9930019P03Rik, BB114106, NR3C3, PR, PR-A, PR-B]','[(RefSeq) progesterone receptor]','[progesterone receptor]','[Mus musculus (house mouse)]','[Oocyte meiosis, Progesterone-mediated oocyte maturation, Estrogen signaling pathway, Chemical carcinogenesis - receptor activation, Breast cancer]','[]','[]','04114 Oocyte meiosis; 04915 Estrogen signaling pathway; 04914 Progesterone-mediated oocyte maturation; 05207 Chemical carcinogenesis - receptor activation; 05224 Breast cancer; 03000 Transcription factors [BR:mmu03000]; 03310 Nuclear receptors [BR:mmu03310]; Cys4 estrogen receptor-like; 3C. 3-Ketosteroid receptor; ','[Pfam: Prog_receptor Hormone_recep zf-C4]'),('12753','mmu:12753','[Clock, 5330400M04Rik, KAT13D]','[(RefSeq) circadian locomotor output cycles kaput]','[circadian locomoter output cycles kaput protein [EC:2.3.1.48]]','[Mus musculus (house mouse)]','[Circadian rhythm, Dopaminergic synapse]','[]','[]','04728 Dopaminergic synapse; 04710 Circadian rhythm; 03000 Transcription factors [BR:mmu03000]; 03036 Chromosome and associated proteins [BR:mmu03036]; 2.3.1.48  histone acetyltransferase; Factors with PAS domain; HATs (histone acetyltransferases); ','[Pfam: PAS_11 PAS_3 HLH PAS PAS_9 DUF6080]'),('15213','mmu:15213','[Hey1, AI316788, AI414254, CHF2, HRT1, Herp2, Hesr1, bHLHb31, hesr-1]','[(RefSeq) hairy/enhancer-of-split related with YRPW motif 1]','[hairy and enhancer of split related with YRPW motif]','[Mus musculus (house mouse)]','[Notch signaling pathway, Human papillomavirus infection, Pathways in cancer, Breast cancer]','[]','[]','04330 Notch signaling pathway; 05200 Pathways in cancer; 05224 Breast cancer; 05165 Human papillomavirus infection; 03000 Transcription factors [BR:mmu03000]; Hairy, Hairy/E(SPL); ','[Pfam: HLH Hairy_orange HTH_Crp_2 dCache_2]'),('12590','mmu:12590','[Cdx1, Cdx, Cdx-1]','[(RefSeq) caudal type homeobox 1]','[homeobox protein CDX1/4]','[Mus musculus (house mouse)]',NULL,'[]','[]','03000 Transcription factors [BR:mmu03000]; Homeo domain ANTP: HOXL; ','[Pfam: Caudal_act Homeodomain Homeobox_KN SUIM_assoc YdaS_antitoxin]'),('67154','mmu:67154','[Mtdh, 2610103J23Rik, 3D3, 3D3/Lyric, AEG-1, AV353288, D8Bwg1112e, Lyric]','[(RefSeq) metadherin]',NULL,'[Mus musculus (house mouse)]',NULL,'[]','[]',NULL,'[Pfam: LYRIC]'),('17344','mmu:17344','[Pias2, AI462206, ARIP3, AU018068, DIP, Dib, Miz1, PIASxalpha, PIASxalpha6, PIASxb, PIASxbeta, SIZ2]','[(RefSeq) protein inhibitor of activated STAT 2]','[E3 SUMO-protein ligase PIAS2 [EC:2.3.2.-]]','[Mus musculus (house mouse)]','[Ubiquitin mediated proteolysis, JAK-STAT signaling pathway]','[]','[]','04120 Ubiquitin mediated proteolysis; 04630 JAK-STAT signaling pathway; 04121 Ubiquitin system [BR:mmu04121]; 2.3.2.-; UBL E3 ligases; ','[Pfam: PINIT zf-MIZ zf-Nse PHD]'),('14581','mmu:14581','[Gfi1, AW495828, Gfi-1, Pal-1, Pal1]','[(RefSeq) growth factor independent 1 transcription repressor]','[growth factor independent 1]','[Mus musculus (house mouse)]',NULL,'[]','[]','03000 Transcription factors [BR:mmu03000]; Cys2His2 others; ','[Pfam: zf-C2H2 zf-H2C2_2 zf-C2H2_4 zf-C2H2_jaz zf-C2H2_6 zf-met zf-BED zf-Di19 zf-C2H2_3rep zf-CHCC]'),('13170','mmu:13170','[Dbp]','[(RefSeq) D site albumin promoter binding protein]','[albumin D box-binding protein]','[Mus musculus (house mouse)]',NULL,'[]','[]','03000 Transcription factors [BR:mmu03000]; bZIP/PAR; ','[Pfam: bZIP_2 bZIP_1]'),('11910','mmu:11910','[Atf3, LRG-21]','[(RefSeq) activating transcription factor 3]','[activating transcription factor 3]','[Mus musculus (house mouse)]',NULL,'[]','[]','03000 Transcription factors [BR:mmu03000]; AP-1(-like) components, Fos; AP-1(-like) components, CRE-BP/ATF; ','[Pfam: bZIP_1 bZIP_2 bZIP_Maf Vfa1 Afi1 Ribosomal_L6e DivIC Fungal_TACC]'),('50754','mmu:50754','[Fbxw7, 1110001A17Rik, AGO, Cdc4, Fbw7, Fbwd6, Fbx30, Fbxo30, Fbxw6, SEL-10]','[(RefSeq) F-box and WD-40 domain protein 7]','[F-box and WD-40 domain protein 7]','[Mus musculus (house mouse)]','[Ubiquitin mediated proteolysis]','[]','[]','04120 Ubiquitin mediated proteolysis; 04131 Membrane trafficking [BR:mmu04131]; 04121 Ubiquitin system [BR:mmu04121]; Other mitophagy associated  proteins; Target recognizing subunit (F-box); ','[Pfam: WD40 Nup160 ANAPC4_WD40 F-box-like NBCH_WD40 F-box Ge1_WD40 PQQ_2 WD40_like eIF2A CDtoxinA PALB2_WD40 TFIIF_alpha]'),('66830','mmu:66830','[Nacc1, 2010001H03Rik, 4930511N13Rik, Btbd14b, Nac1]','[(RefSeq) nucleus accumbens associated 1, BEN and BTB (POZ) domain containing]','[BTB/POZ domain-containing protein 14]','[Mus musculus (house mouse)]',NULL,'[]','[]','04121 Ubiquitin system [BR:mmu04121]; Adoptor and target recognizing subunit (BTB); ','[Pfam: BTB BEN P_C]'),('19650','mmu:19650','[Rbl1, AW547426, PRB1, p107]','[(RefSeq) RB transcriptional corepressor like 1]','[retinoblastoma-like protein 1]','[Mus musculus (house mouse)]','[Cell cycle, Cellular senescence, TGF-beta signaling pathway, Human papillomavirus infection, Viral carcinogenesis]','[]','[]','04350 TGF-beta signaling pathway; 04110 Cell cycle; 04218 Cellular senescence; 05203 Viral carcinogenesis; 05165 Human papillomavirus infection; 03000 Transcription factors [BR:mmu03000]; Pocket domain Rb; ','[Pfam: RB_A RB_B DUF3452 Rb_C TFIIB]'),('21417','mmu:21417','[Zeb1, 3110032K11Rik, AREB6, BZP, MEB1, Nil2, TCF-8, Tcf18, Tcf8, Tw, ZEB, Zfhep, Zfhx1a, Zfx1a, Zfx1ha, [delta]EF1]','[(RefSeq) zinc finger E-box binding homeobox 1]','[zinc finger homeobox protein 1]','[Mus musculus (house mouse)]','[Transcriptional misregulation in cancer, MicroRNAs in cancer, Prostate cancer]','[]','[]','05202 Transcriptional misregulation in cancer; 05206 MicroRNAs in cancer; 05215 Prostate cancer; 03000 Transcription factors [BR:mmu03000]; Homeo domain ZF; ','[Pfam: zf-H2C2_2 zf-C2H2 zf-C2H2_4 zf-C2H2_jaz zf-Di19 zf-met Homeodomain zf-C2HE zf-C2H2_6]'),('24136','mmu:24136','[Zeb2, 9130203F04Rik, D130016B08Rik, SIP1, Zfhx1b, Zfx1b, Zfxh1b]','[(RefSeq) zinc finger E-box binding homeobox 2]','[zinc finger homeobox protein 2]','[Mus musculus (house mouse)]','[MicroRNAs in cancer]','[]','[]','05206 MicroRNAs in cancer; 03000 Transcription factors [BR:mmu03000]; Homeo domain ZF; ','[Pfam: zf-H2C2_2 zf-C2H2 zf-C2H2_4 zf-Di19 zf-C2H2_jaz zf-met Homeodomain zf-H2C2_5 zf-C2H2_6]'),('19821','mmu:19821','[Rnf2, AI326319, AI450156, AU019207, Ring1B, dinG]','[(RefSeq) ring finger protein 2]','[E3 ubiquitin-protein ligase RNF1/2 [EC:2.3.2.27]]','[Mus musculus (house mouse)]',NULL,'[]','[]','04121 Ubiquitin system [BR:mmu04121]; 03036 Chromosome and associated proteins [BR:mmu03036]; 2.3.2.27  RING-type E3 ubiquitin transferase; Other single Ring-finger type E3; PRC1 (polycomb repressive complex 1); AUTS2-polycomb complex; ','[Pfam: RAWUL zf-C3HC4_2 zf-C3HC4 zf-RING_2 zf-C3HC4_3 zf-RING_UBOX zf-RING_5 zf-C3HC4_4 zf-Nse zf-RING_4 zf-RING_6 zf-rbx1 zf-UDP zf-C3HC4_5 zf-ANAPC11]'),('19697','mmu:19697','[Rela, p65, p65_NF-kappa_B, p65_NFkB]','[(RefSeq) v-rel reticuloendotheliosis viral oncogene homolog A (avian)]','[transcription factor p65]','[Mus musculus (house mouse)]','[Antifolate resistance, MAPK signaling pathway, Ras signaling pathway, cAMP signaling pathway, Chemokine signaling pathway, NF-kappa B signaling pathway, HIF-1 signaling pathway, Sphingolipid signaling pathway, Mitophagy - animal, PI3K-Akt signaling pathway, Apoptosis, Longevity regulating pathway, Cellular senescence, Osteoclast differentiation, Neutrophil extracellular trap formation, Toll-like receptor signaling pathway, NOD-like receptor signaling pathway, RIG-I-like receptor signaling pathway, Cytosolic DNA-sensing pathway, C-type lectin receptor signaling pathway, IL-17 signaling pathway, Th1 and Th2 cell differentiation, Th17 cell differentiation, T cell receptor signaling pathway, B cell receptor signaling pathway, TNF signaling pathway, Neurotrophin signaling pathway, Prolactin signaling pathway, Adipocytokine signaling pathway, Relaxin signaling pathway, Insulin resistance, Non-alcoholic fatty liver disease, AGE-RAGE signaling pathway in diabetic complications, Alcoholic liver disease, Alzheimer disease, Pathways of neurodegeneration - multiple diseases, Cocaine addiction, Salmonella infection, Pertussis, Legionellosis, Yersinia infection, Leishmaniasis, Chagas disease, Toxoplasmosis, Amoebiasis, Tuberculosis, Hepatitis C, Hepatitis B, Measles, Human cytomegalovirus infection, Influenza A, Human papillomavirus infection, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Herpes simplex virus 1 infection, Epstein-Barr virus infection, Human immunodeficiency virus 1 infection, Coronavirus disease - COVID-19, Pathways in cancer, Transcriptional misregulation in cancer, Viral carcinogenesis, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species, Pancreatic cancer, Prostate cancer, Chronic myeloid leukemia, Acute myeloid leukemia, Small cell lung cancer, PD-L1 expression and PD-1 checkpoint pathway in cancer, Inflammatory bowel disease, Diabetic cardiomyopathy, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[]','[]','04010 MAPK signaling pathway; 04014 Ras signaling pathway; 04064 NF-kappa B signaling pathway; 04668 TNF signaling pathway; 04066 HIF-1 signaling pathway; 04071 Sphingolipid signaling pathway; 04024 cAMP signaling pathway; 04151 PI3K-Akt signaling pathway; 04137 Mitophagy - animal; 04210 Apoptosis; 04218 Cellular senescence; 04613 Neutrophil extracellular trap formation; 04620 Toll-like receptor signaling pathway; 04621 NOD-like receptor signaling pathway; 04622 RIG-I-like receptor signaling pathway; 04623 Cytosolic DNA-sensing pathway; 04625 C-type lectin receptor signaling pathway; 04660 T cell receptor signaling pathway; 04658 Th1 and Th2 cell differentiation; 04659 Th17 cell differentiation; 04657 IL-17 signaling pathway; 04662 B cell receptor signaling pathway; 04062 Chemokine signaling pathway; 04920 Adipocytokine signaling pathway; 04917 Prolactin signaling pathway; 04926 Relaxin signaling pathway; 04722 Neurotrophin signaling pathway; 04380 Osteoclast differentiation; 04211 Longevity regulating pathway; 05200 Pathways in cancer; 05202 Transcriptional misregulation in cancer; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 05203 Viral carcinogenesis; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05212 Pancreatic cancer; 05221 Acute myeloid leukemia; 05220 Chronic myeloid leukemia; 05215 Prostate cancer; 05222 Small cell lung cancer; 05166 Human T-cell leukemia virus 1 infection; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05160 Hepatitis C; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05165 Human papillomavirus infection; 05132 Salmonella infection; 05135 Yersinia infection; 05133 Pertussis; 05134 Legionellosis; 05152 Tuberculosis; 05146 Amoebiasis; 05145 Toxoplasmosis; 05140 Leishmaniasis; 05142 Chagas disease; 05321 Inflammatory bowel disease; 05010 Alzheimer disease; 05022 Pathways of neurodegeneration - multiple diseases; 05030 Cocaine addiction; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 05415 Diabetic cardiomyopathy; 04936 Alcoholic liver disease; 04932 Non-alcoholic fatty liver disease; 04931 Insulin resistance; 04933 AGE-RAGE signaling pathway in diabetic complications; 01523 Antifolate resistance; 03000 Transcription factors [BR:mmu03000]; RHR (Rel homology region) Rel/Ankyrin; ','[Pfam: RHD_DNA_bind RHD_dimer Fimbrial_CS1]'),('18035','mmu:18035','[Nfkbia, AI462015, Nfkbi]','[(RefSeq) nuclear factor of kappa light polypeptide gene enhancer in B cells inhibitor, alpha]','[NF-kappa-B inhibitor alpha]','[Mus musculus (house mouse)]','[cAMP signaling pathway, Chemokine signaling pathway, NF-kappa B signaling pathway, Apoptosis, Osteoclast differentiation, Toll-like receptor signaling pathway, NOD-like receptor signaling pathway, RIG-I-like receptor signaling pathway, Cytosolic DNA-sensing pathway, C-type lectin receptor signaling pathway, IL-17 signaling pathway, Th1 and Th2 cell differentiation, Th17 cell differentiation, T cell receptor signaling pathway, B cell receptor signaling pathway, TNF signaling pathway, Neurotrophin signaling pathway, Adipocytokine signaling pathway, Relaxin signaling pathway, Insulin resistance, Alcoholic liver disease, Salmonella infection, Legionellosis, Yersinia infection, Leishmaniasis, Chagas disease, Toxoplasmosis, Hepatitis C, Hepatitis B, Measles, Human cytomegalovirus infection, Influenza A, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Herpes simplex virus 1 infection, Epstein-Barr virus infection, Human immunodeficiency virus 1 infection, Coronavirus disease - COVID-19, Pathways in cancer, Viral carcinogenesis, Chemical carcinogenesis - reactive oxygen species, Prostate cancer, Chronic myeloid leukemia, Small cell lung cancer, PD-L1 expression and PD-1 checkpoint pathway in cancer, Lipid and atherosclerosis]','[]','[]','04064 NF-kappa B signaling pathway; 04668 TNF signaling pathway; 04024 cAMP signaling pathway; 04210 Apoptosis; 04620 Toll-like receptor signaling pathway; 04621 NOD-like receptor signaling pathway; 04622 RIG-I-like receptor signaling pathway; 04623 Cytosolic DNA-sensing pathway; 04625 C-type lectin receptor signaling pathway; 04660 T cell receptor signaling pathway; 04658 Th1 and Th2 cell differentiation; 04659 Th17 cell differentiation; 04657 IL-17 signaling pathway; 04662 B cell receptor signaling pathway; 04062 Chemokine signaling pathway; 04920 Adipocytokine signaling pathway; 04926 Relaxin signaling pathway; 04722 Neurotrophin signaling pathway; 04380 Osteoclast differentiation; 05200 Pathways in cancer; 05208 Chemical carcinogenesis - reactive oxygen species; 05203 Viral carcinogenesis; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05220 Chronic myeloid leukemia; 05215 Prostate cancer; 05222 Small cell lung cancer; 05166 Human T-cell leukemia virus 1 infection; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05160 Hepatitis C; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05132 Salmonella infection; 05135 Yersinia infection; 05134 Legionellosis; 05145 Toxoplasmosis; 05140 Leishmaniasis; 05142 Chagas disease; 05417 Lipid and atherosclerosis; 04936 Alcoholic liver disease; 04931 Insulin resistance; 03000 Transcription factors [BR:mmu03000]; 04131 Membrane trafficking [BR:mmu04131]; RHR (Rel homology region) Ankyrin only; Selective cargos; ','[Pfam: Ank Ank_2 Ank_3 Ank_5 Ank_4]'),('72749','mmu:72749','[Tonsl, 2810439M11Rik, Nfkbil2]','[(RefSeq) tonsoku-like, DNA repair protein]','[NF-kappa-B inhibitor-like protein 2]','[Mus musculus (house mouse)]',NULL,'[]','[]','03000 Transcription factors [BR:mmu03000]; RHR (Rel homology region) Ankyrin only; ','[Pfam: Ank_2 Ank Ank_5 Ank_3 Ank_4 LRR_6 TPR_12 TPR_7 LRR_4 TPR_1 Par3_HAL_N_term]'),('56484','mmu:56484','[Foxo3, 1110048B16Rik, 2010203A17Rik, C76856, FKHRL1, Fkhr2, Foxo3a]','[(RefSeq) forkhead box O3]','[forkhead box protein O3]','[Mus musculus (house mouse)]','[EGFR tyrosine kinase inhibitor resistance, Chemokine signaling pathway, FoxO signaling pathway, Mitophagy - animal, PI3K-Akt signaling pathway, AMPK signaling pathway, Longevity regulating pathway, Longevity regulating pathway - multiple species, Cellular senescence, Neurotrophin signaling pathway, Prolactin signaling pathway, Alcoholic liver disease, Chemical carcinogenesis - reactive oxygen species, Endometrial cancer, Non-small cell lung cancer]','[]','[]','04068 FoxO signaling pathway; 04151 PI3K-Akt signaling pathway; 04152 AMPK signaling pathway; 04137 Mitophagy - animal; 04218 Cellular senescence; 04062 Chemokine signaling pathway; 04917 Prolactin signaling pathway; 04722 Neurotrophin signaling pathway; 04211 Longevity regulating pathway; 04213 Longevity regulating pathway - multiple species; 05208 Chemical carcinogenesis - reactive oxygen species; 05213 Endometrial cancer; 05223 Non-small cell lung cancer; 04936 Alcoholic liver disease; 01521 EGFR tyrosine kinase inhibitor resistance; 03000 Transcription factors [BR:mmu03000]; Fork head/winged helix other regulators; ','[Pfam: FOXO_KIX_bdg Forkhead FOXO-TAD]'),('14461','mmu:14461','[Gata2, Gata-2]','[(RefSeq) GATA binding protein 2]','[GATA-binding protein 2]','[Mus musculus (house mouse)]',NULL,'[]','[]','03000 Transcription factors [BR:mmu03000]; Cys4 GATA-factors; ','[Pfam: GATA TF_Zn_Ribbon]'),('12914','mmu:12914','[Crebbp, AW558298, CBP, CBP/p300, KAT3A, p300/CBP]','[(RefSeq) CREB binding protein]','[E1A/CREB-binding protein [EC:2.3.1.48]]','[Mus musculus (house mouse)]','[Viral life cycle - HIV-1, cAMP signaling pathway, HIF-1 signaling pathway, FoxO signaling pathway, Cell cycle, Wnt signaling pathway, Notch signaling pathway, TGF-beta signaling pathway, Adherens junction, JAK-STAT signaling pathway, Long-term potentiation, Melanogenesis, Thyroid hormone signaling pathway, Glucagon signaling pathway, Growth hormone synthesis, secretion and action, Huntington disease, Tuberculosis, Hepatitis B, Influenza A, Human papillomavirus infection, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Pathways in cancer, Viral carcinogenesis, MicroRNAs in cancer, Renal cell carcinoma, Prostate cancer]','[]','[]','03250 Viral life cycle - HIV-1; 04310 Wnt signaling pathway; 04330 Notch signaling pathway; 04350 TGF-beta signaling pathway; 04630 JAK-STAT signaling pathway; 04066 HIF-1 signaling pathway; 04068 FoxO signaling pathway; 04024 cAMP signaling pathway; 04110 Cell cycle; 04520 Adherens junction; 04922 Glucagon signaling pathway; 04935 Growth hormone synthesis, secretion and action; 04919 Thyroid hormone signaling pathway; 04916 Melanogenesis; 04720 Long-term potentiation; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05203 Viral carcinogenesis; 05211 Renal cell carcinoma; 05215 Prostate cancer; 05166 Human T-cell leukemia virus 1 infection; 05161 Hepatitis B; 05164 Influenza A; 05167 Kaposi sarcoma-associated herpesvirus infection; 05165 Human papillomavirus infection; 05152 Tuberculosis; 05016 Huntington disease; 03000 Transcription factors [BR:mmu03000]; 03036 Chromosome and associated proteins [BR:mmu03036]; 03029 Mitochondrial biogenesis [BR:mmu03029]; 2.3.1.48  histone acetyltransferase; Pocket domain CBP; HATs (histone acetyltransferases); Coactivators; ','[Pfam: HAT_KAT11 KIX Creb_binding zf-TAZ RING_CBP-p300 Bromodomain ZZ KIX_2 Npa1]'),('12013','mmu:12013','[Bach1, 6230421P05Rik, AI323795]','[(RefSeq) BTB and CNC homology 1, basic leucine zipper transcription factor 1]','[transcription regulator BACH]','[Mus musculus (house mouse)]',NULL,'[]','[]','03000 Transcription factors [BR:mmu03000]; AP-1(-like) components, NF-E2; ','[Pfam: BTB bZIP_Maf bZIP_1 bZIP_2 Cep57_CLD_2]'),('16476','mmu:16476','[Jun, AP-1, Junc, c-jun]','[(RefSeq) jun proto-oncogene]','[transcription factor AP-1]','[Mus musculus (house mouse)]','[Endocrine resistance, MAPK signaling pathway, ErbB signaling pathway, cAMP signaling pathway, Mitophagy - animal, Apoptosis, Wnt signaling pathway, Osteoclast differentiation, Focal adhesion, Tight junction, Toll-like receptor signaling pathway, NOD-like receptor signaling pathway, C-type lectin receptor signaling pathway, IL-17 signaling pathway, Th1 and Th2 cell differentiation, Th17 cell differentiation, T cell receptor signaling pathway, B cell receptor signaling pathway, TNF signaling pathway, Neurotrophin signaling pathway, GnRH signaling pathway, Estrogen signaling pathway, Oxytocin signaling pathway, Relaxin signaling pathway, Non-alcoholic fatty liver disease, AGE-RAGE signaling pathway in diabetic complications, Cocaine addiction, Amphetamine addiction, Salmonella infection, Pertussis, Yersinia infection, Leishmaniasis, Chagas disease, Hepatitis B, Measles, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Epstein-Barr virus infection, Human immunodeficiency virus 1 infection, Coronavirus disease - COVID-19, Pathways in cancer, Viral carcinogenesis, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species, Colorectal cancer, Renal cell carcinoma, Breast cancer, Choline metabolism in cancer, PD-L1 expression and PD-1 checkpoint pathway in cancer, Inflammatory bowel disease, Rheumatoid arthritis, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[]','[]','04010 MAPK signaling pathway; 04012 ErbB signaling pathway; 04310 Wnt signaling pathway; 04668 TNF signaling pathway; 04024 cAMP signaling pathway; 04137 Mitophagy - animal; 04210 Apoptosis; 04510 Focal adhesion; 04530 Tight junction; 04620 Toll-like receptor signaling pathway; 04621 NOD-like receptor signaling pathway; 04625 C-type lectin receptor signaling pathway; 04660 T cell receptor signaling pathway; 04658 Th1 and Th2 cell differentiation; 04659 Th17 cell differentiation; 04657 IL-17 signaling pathway; 04662 B cell receptor signaling pathway; 04912 GnRH signaling pathway; 04915 Estrogen signaling pathway; 04921 Oxytocin signaling pathway; 04926 Relaxin signaling pathway; 04722 Neurotrophin signaling pathway; 04380 Osteoclast differentiation; 05200 Pathways in cancer; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 05203 Viral carcinogenesis; 05231 Choline metabolism in cancer; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05210 Colorectal cancer; 05211 Renal cell carcinoma; 05224 Breast cancer; 05166 Human T-cell leukemia virus 1 infection; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05171 Coronavirus disease - COVID-19; 05162 Measles; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05132 Salmonella infection; 05135 Yersinia infection; 05133 Pertussis; 05140 Leishmaniasis; 05142 Chagas disease; 05323 Rheumatoid arthritis; 05321 Inflammatory bowel disease; 05030 Cocaine addiction; 05031 Amphetamine addiction; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 04932 Non-alcoholic fatty liver disease; 04933 AGE-RAGE signaling pathway in diabetic complications; 01522 Endocrine resistance; 03000 Transcription factors [BR:mmu03000]; AP-1(-like) components, Jun; ','[Pfam: Jun bZIP_1 bZIP_2 bZIP_Maf ZapB PspA_IM30 FlaC_arch TFIIA NBP1 Fib_alpha DUF3450]'),('13983','mmu:13983','[Esr2, ER[b], ERbeta, Estrb]','[(RefSeq) estrogen receptor 2 (beta)]','[estrogen receptor beta]','[Mus musculus (house mouse)]','[Endocrine resistance, Estrogen signaling pathway, Prolactin signaling pathway, GnRH secretion, Pathways in cancer, Chemical carcinogenesis - receptor activation, Breast cancer]','[]','[]','04929 GnRH secretion; 04915 Estrogen signaling pathway; 04917 Prolactin signaling pathway; 05200 Pathways in cancer; 05207 Chemical carcinogenesis - receptor activation; 05224 Breast cancer; 01522 Endocrine resistance; 03000 Transcription factors [BR:mmu03000]; 03310 Nuclear receptors [BR:mmu03310]; Cys4 estrogen receptor-like; 3A. Estrogen receptor; ','[Pfam: ERbeta_N Hormone_recep zf-C4]'),('15951','mmu:15951','[Ifi204, Ifi16, p204]','[(RefSeq) interferon activated gene 204]','[gamma-interferon-inducible protein 16]','[Mus musculus (house mouse)]','[NOD-like receptor signaling pathway]','[]','[]','04621 NOD-like receptor signaling pathway; ','[Pfam: HIN PYRIN tRNA_anti-codon DUF4551]'),('12608','mmu:12608','[Cebpb, C/EBPbeta, CRP2, IL-6DBP, LAP, LIP, NF-IL6, NF-M, Nfil6]','[(RefSeq) CCAAT/enhancer binding protein (C/EBP), beta]','[CCAAT/enhancer binding protein (C/EBP), beta]','[Mus musculus (house mouse)]','[IL-17 signaling pathway, TNF signaling pathway, Tuberculosis, Transcriptional misregulation in cancer]','[]','[]','04668 TNF signaling pathway; 04657 IL-17 signaling pathway; 05202 Transcriptional misregulation in cancer; 05152 Tuberculosis; 03000 Transcription factors [BR:mmu03000]; C/EBP-like factors; ','[Pfam: bZIP_2 bZIP_1 CCD48 Taxilin DUF3450 Nup54 Uds1 UPF0242 bZIP_Maf]'),('20851','mmu:20851','[Stat5b]','[(RefSeq) signal transducer and activator of transcription 5B]','[signal transducer and activator of transcription 5B]','[Mus musculus (house mouse)]','[ErbB signaling pathway, Chemokine signaling pathway, Necroptosis, JAK-STAT signaling pathway, Th1 and Th2 cell differentiation, Th17 cell differentiation, Prolactin signaling pathway, AGE-RAGE signaling pathway in diabetic complications, Growth hormone synthesis, secretion and action, Hepatitis B, Measles, Human T-cell leukemia virus 1 infection, Pathways in cancer, Viral carcinogenesis, Chemical carcinogenesis - receptor activation, Chronic myeloid leukemia, Acute myeloid leukemia, Non-small cell lung cancer]','[]','[]','04012 ErbB signaling pathway; 04630 JAK-STAT signaling pathway; 04217 Necroptosis; 04658 Th1 and Th2 cell differentiation; 04659 Th17 cell differentiation; 04062 Chemokine signaling pathway; 04917 Prolactin signaling pathway; 04935 Growth hormone synthesis, secretion and action; 05200 Pathways in cancer; 05207 Chemical carcinogenesis - receptor activation; 05203 Viral carcinogenesis; 05221 Acute myeloid leukemia; 05220 Chronic myeloid leukemia; 05223 Non-small cell lung cancer; 05166 Human T-cell leukemia virus 1 infection; 05161 Hepatitis B; 05162 Measles; 04933 AGE-RAGE signaling pathway in diabetic complications; 03000 Transcription factors [BR:mmu03000]; STAT; ','[Pfam: STAT_bind STAT_alpha STAT_int SH2]'),('66277','mmu:66277','[Klf15, 1810013I09Rik, AV048136, AW494632, CKLF, KKLF, hlb444]','[(RefSeq) Kruppel-like factor 15]','[krueppel-like factor 15]','[Mus musculus (house mouse)]',NULL,'[]','[]','03000 Transcription factors [BR:mmu03000]; Cys2His2 SP/KLF family and related proteins; ','[Pfam: zf-C2H2 zf-H2C2_2 zf-C2H2_4 FOXP-CC DUF5810 zf-met zf-C2H2_9 IFT43 DUF2225 zf-C2H2_8 zf-Di19 zf-C2HC_2]'),('17286','mmu:17286','[Meox2, AI528662, Gax, Mox-2, Mox2]','[(RefSeq) mesenchyme homeobox 2]','[homeobox protein MOX]','[Mus musculus (house mouse)]',NULL,'[]','[]','03000 Transcription factors [BR:mmu03000]; Homeo domain ANTP: HOXL; ','[Pfam: Homeodomain]'),('20682','mmu:20682','[Sox9, 2010306G03Rik, AV220920, mKIAA4243, mSox9]','[(RefSeq) SRY (sex determining region Y)-box 9]','[transcription factor SOX9 (SOX group E)]','[Mus musculus (house mouse)]','[cAMP signaling pathway]','[]','[]','04024 cAMP signaling pathway; 03000 Transcription factors [BR:mmu03000]; HMG SOX; ','[Pfam: HMG_box Sox_N HMG_box_2]'),('17131','mmu:17131','[Smad7, Madh7]','[(RefSeq) SMAD family member 7]','[mothers against decapentaplegic homolog 7]','[Mus musculus (house mouse)]','[TGF-beta signaling pathway, Hippo signaling pathway]','[]','[]','04350 TGF-beta signaling pathway; 04390 Hippo signaling pathway; ','[Pfam: MH2 MH1]'),('20850','mmu:20850','[Stat5a, AA959963, STAT5]','[(RefSeq) signal transducer and activator of transcription 5A]','[signal transducer and activator of transcription 5A]','[Mus musculus (house mouse)]','[ErbB signaling pathway, Necroptosis, JAK-STAT signaling pathway, Th1 and Th2 cell differentiation, Th17 cell differentiation, Prolactin signaling pathway, AGE-RAGE signaling pathway in diabetic complications, Growth hormone synthesis, secretion and action, Hepatitis B, Measles, Human T-cell leukemia virus 1 infection, Pathways in cancer, Viral carcinogenesis, Chemical carcinogenesis - receptor activation, Chronic myeloid leukemia, Acute myeloid leukemia, Non-small cell lung cancer]','[]','[]','04012 ErbB signaling pathway; 04630 JAK-STAT signaling pathway; 04217 Necroptosis; 04658 Th1 and Th2 cell differentiation; 04659 Th17 cell differentiation; 04917 Prolactin signaling pathway; 04935 Growth hormone synthesis, secretion and action; 05200 Pathways in cancer; 05207 Chemical carcinogenesis - receptor activation; 05203 Viral carcinogenesis; 05221 Acute myeloid leukemia; 05220 Chronic myeloid leukemia; 05223 Non-small cell lung cancer; 05166 Human T-cell leukemia virus 1 infection; 05161 Hepatitis B; 05162 Measles; 04933 AGE-RAGE signaling pathway in diabetic complications; 03000 Transcription factors [BR:mmu03000]; STAT; ','[Pfam: STAT_bind STAT_alpha STAT_int SH2 Flu_B_NS1 Caldesmon]'),('18412','mmu:18412','[Sqstm1, A170, OSF-6, Osi, STAP, STONE14, p62]','[(RefSeq) sequestosome 1]','[sequestosome 1]','[Mus musculus (house mouse)]','[Mitophagy - animal, Autophagy - animal, Necroptosis, Cellular senescence, Osteoclast differentiation, Amyotrophic lateral sclerosis, Pathways of neurodegeneration - multiple diseases, Fluid shear stress and atherosclerosis]','[]','[]','04140 Autophagy - animal; 04137 Mitophagy - animal; 04217 Necroptosis; 04218 Cellular senescence; 04380 Osteoclast differentiation; 05014 Amyotrophic lateral sclerosis; 05022 Pathways of neurodegeneration - multiple diseases; 05418 Fluid shear stress and atherosclerosis; 04131 Membrane trafficking [BR:mmu04131]; 03029 Mitochondrial biogenesis [BR:mmu03029]; 04147 Exosome [BR:mmu04147]; Cargo receptors; Pexophagy; Aggrephagy; Xenophagy; Midbophagy associated proteins; Parkin-dependent mechanism factors; Exosomal proteins of other body fluids (saliva and urine); ','[Pfam: UBA_5 PB1 ZZ C1_2]'),('22337','mmu:22337','[Vdr, Nr1i1]','[(RefSeq) vitamin D (1,25-dihydroxyvitamin D3) receptor]','[vitamin D3 receptor]','[Mus musculus (house mouse)]','[Parathyroid hormone synthesis, secretion and action, Endocrine and other factor-regulated calcium reabsorption, Mineral absorption, Tuberculosis, Chemical carcinogenesis - receptor activation]','[]','[]','04928 Parathyroid hormone synthesis, secretion and action; 04978 Mineral absorption; 04961 Endocrine and other factor-regulated calcium reabsorption; 05207 Chemical carcinogenesis - receptor activation; 05152 Tuberculosis; 01009 Protein phosphatases and associated proteins [BR:mmu01009]; 03000 Transcription factors [BR:mmu03000]; 03310 Nuclear receptors [BR:mmu03310]; PP1-interacting proteins (PIPs); Cys4 thyroid hormone receptor-like; 1I. Vitamin D3 like receptor; ','[Pfam: zf-C4 Hormone_recep zf-RRPl_C4]'),('26423','mmu:26423','[Nr5a1, Ad4BP, ELP, ELP-3, Ftz-F1, Ftzf1, SF-1, SF1, STF-1]','[(RefSeq) nuclear receptor subfamily 5, group A, member 1]','[steroidogenic factor 1]','[Mus musculus (house mouse)]','[Cortisol synthesis and secretion, Cushing syndrome]','[]','[]','04927 Cortisol synthesis and secretion; 04934 Cushing syndrome; 03000 Transcription factors [BR:mmu03000]; 03310 Nuclear receptors [BR:mmu03310]; Cys4 Fushi tarazu-F1-like; 5A. Fushi tarazu-F1 like receptor; ','[Pfam: zf-C4 Hormone_recep]'),('19712','mmu:19712','[Rest, 2610008J04Rik, AA407358, NRSF, REST4]','[(RefSeq) RE1-silencing transcription factor]','[RE1-silencing transcription factor]','[Mus musculus (house mouse)]','[Signaling pathways regulating pluripotency of stem cells, Huntington disease]','[]','[]','04550 Signaling pathways regulating pluripotency of stem cells; 05016 Huntington disease; 03000 Transcription factors [BR:mmu03000]; Cys2His2 others; ','[Pfam: zf-H2C2_2 zf-H2C2_5 zf-C2H2 zf-C2H2_4 zf-C2H2_jaz zf-C2HE]'),('14390','mmu:14390','[Gabpa, GABPalpha]','[(RefSeq) GA repeat binding protein, alpha]','[GA-binding protein transcription factor, alpha]','[Mus musculus (house mouse)]',NULL,'[]','[]','03000 Transcription factors [BR:mmu03000]; Tryptophan clusters Ets-type; ','[Pfam: Ets GABP-alpha SAM_PNT AAA_25]'),('13198','mmu:13198','[Ddit3, AltDDIT3, CHOP-10, CHOP10, chop, gadd153]','[(RefSeq) DNA-damage inducible transcript 3]','[DNA damage-inducible transcript 3]','[Mus musculus (house mouse)]','[MAPK signaling pathway, Protein processing in endoplasmic reticulum, Apoptosis, Non-alcoholic fatty liver disease, Alzheimer disease, Parkinson disease, Amyotrophic lateral sclerosis, Prion disease, Pathways of neurodegeneration - multiple diseases, Transcriptional misregulation in cancer, Lipid and atherosclerosis]','[]','[]','04141 Protein processing in endoplasmic reticulum; 04010 MAPK signaling pathway; 04210 Apoptosis; 05202 Transcriptional misregulation in cancer; 05010 Alzheimer disease; 05012 Parkinson disease; 05014 Amyotrophic lateral sclerosis; 05020 Prion disease; 05022 Pathways of neurodegeneration - multiple diseases; 05417 Lipid and atherosclerosis; 04932 Non-alcoholic fatty liver disease; 03000 Transcription factors [BR:mmu03000]; C/EBP-like factors; ','[Pfam: GAS bZIP_2 bZIP_1 DivIC Gcd10p Fungal_TACC DASH_Hsk3 DASH_Spc19 WDCP RR_TM4-6 DUF5595 DUF5094 Phlebovirus_NSM DUF4407 Neur_chan_memb]'),('18181','mmu:18181','[Nrf1, C87038, D6Ertd415e]','[(RefSeq) nuclear respiratory factor 1]','[nuclear respiratory factor 1]','[Mus musculus (house mouse)]','[Apelin signaling pathway, Huntington disease]','[]','[]','04371 Apelin signaling pathway; 05016 Huntington disease; 03029 Mitochondrial biogenesis [BR:mmu03029]; Nuclear respiratory factors; ','[Pfam: Nrf1_DNA-bind DUF5008 DUF3459 Peptidase_M23]'),('15370','mmu:15370','[Nr4a1, GFRP1, Gfrp, Hbr-1, Hbr1, Hmr, N10, NGFI-B, NGFIB, NP10, NUR77-1, NUR77-2, TIS1, TR3, nur77]','[(RefSeq) nuclear receptor subfamily 4, group A, member 1]','[nuclear receptor subfamily 4 group A member 1]','[Mus musculus (house mouse)]','[MAPK signaling pathway, PI3K-Akt signaling pathway, Aldosterone synthesis and secretion, Cortisol synthesis and secretion, Cushing syndrome]','[]','[]','04010 MAPK signaling pathway; 04151 PI3K-Akt signaling pathway; 04925 Aldosterone synthesis and secretion; 04927 Cortisol synthesis and secretion; 04934 Cushing syndrome; 03000 Transcription factors [BR:mmu03000]; 03310 Nuclear receptors [BR:mmu03310]; Cys4 nerve growth factor IB-like; 4A. Nerve growth factor IB like receptor; ','[Pfam: Hormone_recep zf-C4]'),('18171','mmu:18171','[Nr1i2, PXR, PXR.1, PXR.2, PXR1, SXR, mPXR]','[(RefSeq) nuclear receptor subfamily 1, group I, member 2]','[pregnane X receptor]','[Mus musculus (house mouse)]',NULL,'[]','[]','03000 Transcription factors [BR:mmu03000]; 03310 Nuclear receptors [BR:mmu03310]; Cys4 thyroid hormone receptor-like; 1I. Vitamin D3 like receptor; ','[Pfam: zf-C4 Hormone_recep]'),('29560','rno:29560','[Hif1a]','[(RefSeq) hypoxia-inducible factor 1-alpha]','[hypoxia-inducible factor 1 alpha]','[Rattus norvegicus (rat)]','[HIF-1 signaling pathway, Mitophagy - animal, Autophagy - animal, Th17 cell differentiation, Thyroid hormone signaling pathway, Kaposi sarcoma-associated herpesvirus infection, Pathways in cancer, Proteoglycans in cancer, Chemical carcinogenesis - reactive oxygen species, Renal cell carcinoma, Central carbon metabolism in cancer, Choline metabolism in cancer, PD-L1 expression and PD-1 checkpoint pathway in cancer]','[]','[]','04066 HIF-1 signaling pathway; 04140 Autophagy - animal; 04137 Mitophagy - animal; 04659 Th17 cell differentiation; 04919 Thyroid hormone signaling pathway; 05200 Pathways in cancer; 05205 Proteoglycans in cancer; 05208 Chemical carcinogenesis - reactive oxygen species; 05230 Central carbon metabolism in cancer; 05231 Choline metabolism in cancer; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05211 Renal cell carcinoma; 05167 Kaposi sarcoma-associated herpesvirus infection; 03000 Transcription factors [BR:rno03000]; Factors with PAS domain; ','[Pfam: PAS_3 HIF-1a_CTAD PAS_11 PAS HIF-1 PAS_9 PAS_4 DDRGK HLH]'),('22346','mmu:22346','[Vhl, Vhlh, pVHL]','[(RefSeq) von Hippel-Lindau tumor suppressor]','[von Hippel-Lindau disease tumor supressor]','[Mus musculus (house mouse)]','[HIF-1 signaling pathway, Ubiquitin mediated proteolysis, Pathways in cancer, Renal cell carcinoma]','[]','[]','04120 Ubiquitin mediated proteolysis; 04066 HIF-1 signaling pathway; 05200 Pathways in cancer; 05211 Renal cell carcinoma; 04121 Ubiquitin system [BR:mmu04121]; Target recognizing subunit; ','[Pfam: VHL VHL_C Arm_vescicular]'),('22632','mmu:22632','[Yy1, NF-E1, YY-1]','[(RefSeq) YY1 transcription factor]','[transcription factor YY]','[Mus musculus (house mouse)]',NULL,'[]','[]','03000 Transcription factors [BR:mmu03000]; 03036 Chromosome and associated proteins [BR:mmu03036]; 03029 Mitochondrial biogenesis [BR:mmu03029]; Cys2His2 others; Other polycomb group proteins; Ubiquitas transcription factors; ','[Pfam: zf-C2H2 zf-H2C2_2 zf-C2H2_4 IBR Raffinose_syn]'),('22059','mmu:22059','[Trp53, Tp53, bbl, bfy, bhy, p44, p53]','[(RefSeq) transformation related protein 53]','[tumor protein p53]','[Mus musculus (house mouse)]','[Endocrine resistance, Platinum drug resistance, MAPK signaling pathway, Sphingolipid signaling pathway, Cell cycle, p53 signaling pathway, Mitophagy - animal, PI3K-Akt signaling pathway, Apoptosis, Longevity regulating pathway, Ferroptosis, Cellular senescence, Wnt signaling pathway, Neurotrophin signaling pathway, Thyroid hormone signaling pathway, Parkinson disease, Amyotrophic lateral sclerosis, Huntington disease, Hepatitis C, Hepatitis B, Measles, Human cytomegalovirus infection, Human papillomavirus infection, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Herpes simplex virus 1 infection, Epstein-Barr virus infection, Pathways in cancer, Transcriptional misregulation in cancer, Viral carcinogenesis, Proteoglycans in cancer, MicroRNAs in cancer, Colorectal cancer, Pancreatic cancer, Endometrial cancer, Glioma, Prostate cancer, Thyroid cancer, Basal cell carcinoma, Melanoma, Bladder cancer, Chronic myeloid leukemia, Small cell lung cancer, Non-small cell lung cancer, Breast cancer, Hepatocellular carcinoma, Gastric cancer, Central carbon metabolism in cancer, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[]','[]','04010 MAPK signaling pathway; 04310 Wnt signaling pathway; 04071 Sphingolipid signaling pathway; 04151 PI3K-Akt signaling pathway; 04137 Mitophagy - animal; 04110 Cell cycle; 04210 Apoptosis; 04216 Ferroptosis; 04115 p53 signaling pathway; 04218 Cellular senescence; 04919 Thyroid hormone signaling pathway; 04722 Neurotrophin signaling pathway; 04211 Longevity regulating pathway; 05200 Pathways in cancer; 05202 Transcriptional misregulation in cancer; 05206 MicroRNAs in cancer; 05205 Proteoglycans in cancer; 05203 Viral carcinogenesis; 05230 Central carbon metabolism in cancer; 05210 Colorectal cancer; 05212 Pancreatic cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05214 Glioma; 05216 Thyroid cancer; 05220 Chronic myeloid leukemia; 05217 Basal cell carcinoma; 05218 Melanoma; 05219 Bladder cancer; 05215 Prostate cancer; 05213 Endometrial cancer; 05224 Breast cancer; 05222 Small cell lung cancer; 05223 Non-small cell lung cancer; 05166 Human T-cell leukemia virus 1 infection; 05161 Hepatitis B; 05160 Hepatitis C; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05165 Human papillomavirus infection; 05012 Parkinson disease; 05014 Amyotrophic lateral sclerosis; 05016 Huntington disease; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 01524 Platinum drug resistance; 01522 Endocrine resistance; 03000 Transcription factors [BR:mmu03000]; 03036 Chromosome and associated proteins [BR:mmu03036]; 03400 DNA repair and recombination proteins [BR:mmu03400]; p53; Regulators of Aurora kinases; Other check point factors; ','[Pfam: P53 P53_tetramer P53_TAD]'),('30939','mmu:30939','[Pttg1, AW555095, C87862, Pttg, Pttg3]','[(RefSeq) pituitary tumor-transforming gene 1]','[securin]','[Mus musculus (house mouse)]','[Cell cycle, Oocyte meiosis, Human T-cell leukemia virus 1 infection]','[]','[]','04110 Cell cycle; 04114 Oocyte meiosis; 05166 Human T-cell leukemia virus 1 infection; 03036 Chromosome and associated proteins [BR:mmu03036]; Securin; ','[Pfam: Securin]'),('15248','mmu:15248','[Hic1, AA408311, HIC-1]','[(RefSeq) hypermethylated in cancer 1]','[hypermethylated in cancer protein]','[Mus musculus (house mouse)]',NULL,'[]','[]','03000 Transcription factors [BR:mmu03000]; Cys2His2 BTB-ZF factors; ','[Pfam: BTB zf-C2H2 zf-C2H2_4 zf-H2C2_2 zf-C2H2_jaz zf-C2H2_6 zf-met zf-C2H2_3rep zf-C2H2_11 zf-Di19 Zn-ribbon_8 LIM]'),('22062','mmu:22062','[Trp73, TAp73, Tp73, deltaNp73, p73]','[(RefSeq) transformation related protein 73]','[tumor protein p73]','[Mus musculus (house mouse)]','[p53 signaling pathway, Hippo signaling pathway, Neurotrophin signaling pathway, Measles]','[]','[]','04390 Hippo signaling pathway; 04115 p53 signaling pathway; 04722 Neurotrophin signaling pathway; 05162 Measles; 03000 Transcription factors [BR:mmu03000]; p53; ','[Pfam: P53 P53_tetramer SAM_2 SAM_1]'),('19645','mmu:19645','[Rb1, Rb, Rb-1, p110-RB1, pRb, pp105]','[(RefSeq) RB transcriptional corepressor 1]','[retinoblastoma-associated protein]','[Mus musculus (house mouse)]','[Endocrine resistance, Cell cycle, Cellular senescence, Cushing syndrome, Hepatitis C, Hepatitis B, Human cytomegalovirus infection, Human papillomavirus infection, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Epstein-Barr virus infection, Pathways in cancer, Viral carcinogenesis, Chemical carcinogenesis - receptor activation, Pancreatic cancer, Glioma, Prostate cancer, Melanoma, Bladder cancer, Chronic myeloid leukemia, Small cell lung cancer, Non-small cell lung cancer, Breast cancer, Hepatocellular carcinoma, Gastric cancer]','[]','[]','04110 Cell cycle; 04218 Cellular senescence; 05200 Pathways in cancer; 05207 Chemical carcinogenesis - receptor activation; 05203 Viral carcinogenesis; 05212 Pancreatic cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05214 Glioma; 05220 Chronic myeloid leukemia; 05218 Melanoma; 05219 Bladder cancer; 05215 Prostate cancer; 05224 Breast cancer; 05222 Small cell lung cancer; 05223 Non-small cell lung cancer; 05166 Human T-cell leukemia virus 1 infection; 05161 Hepatitis B; 05160 Hepatitis C; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05165 Human papillomavirus infection; 04934 Cushing syndrome; 01522 Endocrine resistance; 01009 Protein phosphatases and associated proteins [BR:mmu01009]; 03000 Transcription factors [BR:mmu03000]; PP1-interacting proteins (PIPs); Pocket domain Rb; ','[Pfam: Rb_C RB_A RB_B DUF3452 RuvC_1 TFIIB]'),('54601','mmu:54601','[Foxo4, Afxh, Mllt7, afx]','[(RefSeq) forkhead box O4]','[forkhead box protein O4]','[Mus musculus (house mouse)]','[Ras signaling pathway, FoxO signaling pathway]','[]','[]','04014 Ras signaling pathway; 04068 FoxO signaling pathway; 03000 Transcription factors [BR:mmu03000]; Fork head/winged helix other regulators; ','[Pfam: Forkhead FOXO-TAD FOXO_KIX_bdg SAND]'),('228980','mmu:228980','[Taf4, 135kDa, AI450312, TAFII130, TAFII135, Taf2c1, Taf4a]','[(RefSeq) TATA-box binding protein associated factor 4]','[transcription initiation factor TFIID subunit 4]','[Mus musculus (house mouse)]','[Basal transcription factors, Huntington disease]','[]','[]','03022 Basal transcription factors; 05016 Huntington disease; 03021 Transcription machinery [BR:mmu03021]; 03036 Chromosome and associated proteins [BR:mmu03036]; TFIID; SAGA/TFTC/STAGA complex; TFTC complex; ','[Pfam: TAF4 TAFH tRNA-synt_1c]'),('17095','mmu:17095','[Lyl1, Lyl-1, bHLHa18]','[(RefSeq) lymphoblastomic leukemia 1]','[protein lyl-1]','[Mus musculus (house mouse)]','[Transcriptional misregulation in cancer]','[]','[]','05202 Transcriptional misregulation in cancer; 03000 Transcription factors [BR:mmu03000]; Tal/Twist/Atonal/Hen, Lymphoid factors; ','[Pfam: HLH COE1_HLH]'),('12916','mmu:12916','[Crem, ICER, ICERI]','[(RefSeq) cAMP responsive element modulator]','[cAMP response element modulator]','[Mus musculus (house mouse)]','[Adrenergic signaling in cardiomyocytes]','[]','[]','04261 Adrenergic signaling in cardiomyocytes; 03000 Transcription factors [BR:mmu03000]; CREB; ','[Pfam: bZIP_1 pKID bZIP_2 bZIP_Maf]'),('16842','mmu:16842','[Lef1, 3000002B05, AI451430, Lef-1]','[(RefSeq) lymphoid enhancer binding factor 1]','[lymphoid enhancer-binding factor 1]','[Mus musculus (house mouse)]','[Wnt signaling pathway, Hippo signaling pathway, Adherens junction, Melanogenesis, Cushing syndrome, Alcoholic liver disease, Salmonella infection, Kaposi sarcoma-associated herpesvirus infection, Pathways in cancer, Colorectal cancer, Endometrial cancer, Prostate cancer, Thyroid cancer, Basal cell carcinoma, Acute myeloid leukemia, Breast cancer, Hepatocellular carcinoma, Gastric cancer, Arrhythmogenic right ventricular cardiomyopathy]','[]','[]','04310 Wnt signaling pathway; 04390 Hippo signaling pathway; 04520 Adherens junction; 04916 Melanogenesis; 05200 Pathways in cancer; 05210 Colorectal cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05216 Thyroid cancer; 05221 Acute myeloid leukemia; 05217 Basal cell carcinoma; 05215 Prostate cancer; 05213 Endometrial cancer; 05224 Breast cancer; 05167 Kaposi sarcoma-associated herpesvirus infection; 05132 Salmonella infection; 05412 Arrhythmogenic right ventricular cardiomyopathy; 04936 Alcoholic liver disease; 04934 Cushing syndrome; 03000 Transcription factors [BR:mmu03000]; HMG TCF-1; ','[Pfam: CTNNB1_binding HMG_box HMG_box_2]'),('17869','mmu:17869','[Myc, AU016757, Myc2, Niard, Nird, bHLHe39]','[(RefSeq) myelocytomatosis oncogene]','[Myc proto-oncogene protein]','[Mus musculus (house mouse)]','[MAPK signaling pathway, ErbB signaling pathway, Cell cycle, PI3K-Akt signaling pathway, Cellular senescence, Wnt signaling pathway, TGF-beta signaling pathway, Hippo signaling pathway, Signaling pathways regulating pluripotency of stem cells, JAK-STAT signaling pathway, Thyroid hormone signaling pathway, Salmonella infection, Hepatitis C, Hepatitis B, Human cytomegalovirus infection, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Epstein-Barr virus infection, Pathways in cancer, Transcriptional misregulation in cancer, Proteoglycans in cancer, MicroRNAs in cancer, Chemical carcinogenesis - receptor activation, Colorectal cancer, Endometrial cancer, Thyroid cancer, Bladder cancer, Chronic myeloid leukemia, Acute myeloid leukemia, Small cell lung cancer, Breast cancer, Hepatocellular carcinoma, Gastric cancer, Central carbon metabolism in cancer]','[]','[]','04010 MAPK signaling pathway; 04012 ErbB signaling pathway; 04310 Wnt signaling pathway; 04350 TGF-beta signaling pathway; 04390 Hippo signaling pathway; 04630 JAK-STAT signaling pathway; 04151 PI3K-Akt signaling pathway; 04110 Cell cycle; 04218 Cellular senescence; 04550 Signaling pathways regulating pluripotency of stem cells; 04919 Thyroid hormone signaling pathway; 05200 Pathways in cancer; 05202 Transcriptional misregulation in cancer; 05206 MicroRNAs in cancer; 05205 Proteoglycans in cancer; 05207 Chemical carcinogenesis - receptor activation; 05230 Central carbon metabolism in cancer; 05210 Colorectal cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05216 Thyroid cancer; 05221 Acute myeloid leukemia; 05220 Chronic myeloid leukemia; 05219 Bladder cancer; 05213 Endometrial cancer; 05224 Breast cancer; 05222 Small cell lung cancer; 05166 Human T-cell leukemia virus 1 infection; 05161 Hepatitis B; 05160 Hepatitis C; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05132 Salmonella infection; 03000 Transcription factors [BR:mmu03000]; 03019 Messenger RNA biogenesis [BR:mmu03019]; Cell cycle controlling factors, Myc; Factors that have mRNA localization signals; ','[Pfam: Myc_N Myc-LZ HLH TRH]'),('14281','mmu:14281','[Fos, D12Rfj1, c-fos, cFos]','[(RefSeq) FBJ osteosarcoma oncogene]','[proto-oncogene protein c-fos]','[Mus musculus (house mouse)]','[Endocrine resistance, MAPK signaling pathway, cAMP signaling pathway, Apoptosis, Osteoclast differentiation, Toll-like receptor signaling pathway, IL-17 signaling pathway, Th1 and Th2 cell differentiation, Th17 cell differentiation, T cell receptor signaling pathway, B cell receptor signaling pathway, TNF signaling pathway, Circadian entrainment, Cholinergic synapse, Dopaminergic synapse, Estrogen signaling pathway, Prolactin signaling pathway, Oxytocin signaling pathway, Relaxin signaling pathway, Parathyroid hormone synthesis, secretion and action, Non-alcoholic fatty liver disease, Growth hormone synthesis, secretion and action, Amphetamine addiction, Salmonella infection, Pertussis, Yersinia infection, Leishmaniasis, Chagas disease, Hepatitis B, Measles, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Human immunodeficiency virus 1 infection, Coronavirus disease - COVID-19, Pathways in cancer, Chemical carcinogenesis - receptor activation, Chemical carcinogenesis - reactive oxygen species, Colorectal cancer, Breast cancer, Choline metabolism in cancer, PD-L1 expression and PD-1 checkpoint pathway in cancer, Rheumatoid arthritis, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[]','[]','04010 MAPK signaling pathway; 04668 TNF signaling pathway; 04024 cAMP signaling pathway; 04210 Apoptosis; 04620 Toll-like receptor signaling pathway; 04660 T cell receptor signaling pathway; 04658 Th1 and Th2 cell differentiation; 04659 Th17 cell differentiation; 04657 IL-17 signaling pathway; 04662 B cell receptor signaling pathway; 04915 Estrogen signaling pathway; 04917 Prolactin signaling pathway; 04921 Oxytocin signaling pathway; 04926 Relaxin signaling pathway; 04935 Growth hormone synthesis, secretion and action; 04928 Parathyroid hormone synthesis, secretion and action; 04725 Cholinergic synapse; 04728 Dopaminergic synapse; 04380 Osteoclast differentiation; 04713 Circadian entrainment; 05200 Pathways in cancer; 05207 Chemical carcinogenesis - receptor activation; 05208 Chemical carcinogenesis - reactive oxygen species; 05231 Choline metabolism in cancer; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05210 Colorectal cancer; 05224 Breast cancer; 05166 Human T-cell leukemia virus 1 infection; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05171 Coronavirus disease - COVID-19; 05162 Measles; 05167 Kaposi sarcoma-associated herpesvirus infection; 05132 Salmonella infection; 05135 Yersinia infection; 05133 Pertussis; 05140 Leishmaniasis; 05142 Chagas disease; 05323 Rheumatoid arthritis; 05031 Amphetamine addiction; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 04932 Non-alcoholic fatty liver disease; 01522 Endocrine resistance; 03000 Transcription factors [BR:mmu03000]; 04131 Membrane trafficking [BR:mmu04131]; AP-1(-like) components, Fos; Selective cargos; ','[Pfam: bZIP_1 bZIP_2 bZIP_Maf Macoilin DUF3450 HALZ]'),('21420','mmu:21420','[Tfap2c, AA409384, AP2gamma, Ap-2.2, Stra2, Tcfap2c]','[(RefSeq) transcription factor AP-2, gamma]','[transcription factor AP-2 gamma]','[Mus musculus (house mouse)]',NULL,'[]','[]','03000 Transcription factors [BR:mmu03000]; AP-2 (bHSH); ','[Pfam: TF_AP-2]'),('20846','mmu:20846','[Stat1, 2010005J02Rik, AA408197]','[(RefSeq) signal transducer and activator of transcription 1]','[signal transducer and activator of transcription 1]','[Mus musculus (house mouse)]','[Chemokine signaling pathway, Necroptosis, Osteoclast differentiation, Toll-like receptor signaling pathway, NOD-like receptor signaling pathway, C-type lectin receptor signaling pathway, JAK-STAT signaling pathway, Th1 and Th2 cell differentiation, Th17 cell differentiation, Prolactin signaling pathway, Thyroid hormone signaling pathway, AGE-RAGE signaling pathway in diabetic complications, Growth hormone synthesis, secretion and action, Leishmaniasis, Toxoplasmosis, Tuberculosis, Hepatitis C, Hepatitis B, Measles, Influenza A, Human papillomavirus infection, Kaposi sarcoma-associated herpesvirus infection, Herpes simplex virus 1 infection, Epstein-Barr virus infection, Coronavirus disease - COVID-19, Pathways in cancer, Pancreatic cancer, PD-L1 expression and PD-1 checkpoint pathway in cancer, Inflammatory bowel disease]','[]','[]','04630 JAK-STAT signaling pathway; 04217 Necroptosis; 04620 Toll-like receptor signaling pathway; 04621 NOD-like receptor signaling pathway; 04625 C-type lectin receptor signaling pathway; 04658 Th1 and Th2 cell differentiation; 04659 Th17 cell differentiation; 04062 Chemokine signaling pathway; 04917 Prolactin signaling pathway; 04935 Growth hormone synthesis, secretion and action; 04919 Thyroid hormone signaling pathway; 04380 Osteoclast differentiation; 05200 Pathways in cancer; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05212 Pancreatic cancer; 05161 Hepatitis B; 05160 Hepatitis C; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05165 Human papillomavirus infection; 05152 Tuberculosis; 05145 Toxoplasmosis; 05140 Leishmaniasis; 05321 Inflammatory bowel disease; 04933 AGE-RAGE signaling pathway in diabetic complications; 03000 Transcription factors [BR:mmu03000]; STAT; ','[Pfam: STAT_alpha STAT_int STAT_bind STAT1_TAZ2bind SH2 BLI1 BORCS7 NPV_P10]'),('12224','mmu:12224','[Klf5, 4930520J07Rik, Bteb2, CKLF, IKLF]','[(RefSeq) Kruppel-like factor 5]','[krueppel-like factor 5]','[Mus musculus (house mouse)]','[Chemical carcinogenesis - receptor activation]','[]','[]','05207 Chemical carcinogenesis - receptor activation; 03000 Transcription factors [BR:mmu03000]; Cys2His2 SP/KLF family and related proteins; ','[Pfam: zf-C2H2 zf-H2C2_2 zf-C2H2_4 zf-met]'),('14056','mmu:14056','[Ezh2, Enx-1, Enx1h, KMT6, mKIAA4065]','[(RefSeq) enhancer of zeste 2 polycomb repressive complex 2 subunit]','[[histone H3]-lysine27 N-trimethyltransferase EZH2 [EC:2.1.1.356]]','[Mus musculus (house mouse)]','[Lysine degradation, Metabolic pathways, MicroRNAs in cancer]','[]','[]','00310 Lysine degradation; 05206 MicroRNAs in cancer; 03036 Chromosome and associated proteins [BR:mmu03036]; 2.1.1.356  [histone H3]-lysine27 N-trimethyltransferase; HKMTs (histone lysine methyltransferases); PRC2-EZH2 complex; ','[Pfam: PRC2_HTH_1 SET EZH2_WD-Binding preSET_CXC]'),('230233','mmu:230233','[Elp1, 3110040G09Rik, 6030413P05, C78473, IKAP, Ikbkap]','[(RefSeq) elongator complex protein 1]','[elongator complex protein 1]','[Mus musculus (house mouse)]',NULL,'[]','[]','03016 Transfer RNA biogenesis [BR:mmu03016]; 03036 Chromosome and associated proteins [BR:mmu03036]; Elongator complex; Elongator complex; ','[Pfam: IKI3 ANAPC4_WD40]'),('12912','mmu:12912','[Creb1, 2310001E10Rik, 3526402H21Rik, AV083133, Creb, Creb-1]','[(RefSeq) cAMP responsive element binding protein 1]','[cyclic AMP-responsive element-binding protein 1]','[Mus musculus (house mouse)]','[cGMP-PKG signaling pathway, cAMP signaling pathway, PI3K-Akt signaling pathway, AMPK signaling pathway, Longevity regulating pathway, Adrenergic signaling in cardiomyocytes, Osteoclast differentiation, Antigen processing and presentation, TNF signaling pathway, Circadian rhythm, Circadian entrainment, Thermogenesis, Cholinergic synapse, Dopaminergic synapse, Insulin secretion, Estrogen signaling pathway, Melanogenesis, Thyroid hormone synthesis, Glucagon signaling pathway, Renin secretion, Aldosterone synthesis and secretion, Relaxin signaling pathway, Cortisol synthesis and secretion, Parathyroid hormone synthesis, secretion and action, Insulin resistance, Cushing syndrome, Growth hormone synthesis, secretion and action, Vasopressin-regulated water reabsorption, Huntington disease, Prion disease, Cocaine addiction, Amphetamine addiction, Alcoholism, Tuberculosis, Hepatitis B, Human cytomegalovirus infection, Human papillomavirus infection, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Viral carcinogenesis, Chemical carcinogenesis - receptor activation, Prostate cancer]','[]','[]','04668 TNF signaling pathway; 04024 cAMP signaling pathway; 04022 cGMP-PKG signaling pathway; 04151 PI3K-Akt signaling pathway; 04152 AMPK signaling pathway; 04612 Antigen processing and presentation; 04911 Insulin secretion; 04922 Glucagon signaling pathway; 04915 Estrogen signaling pathway; 04926 Relaxin signaling pathway; 04935 Growth hormone synthesis, secretion and action; 04918 Thyroid hormone synthesis; 04928 Parathyroid hormone synthesis, secretion and action; 04916 Melanogenesis; 04924 Renin secretion; 04925 Aldosterone synthesis and secretion; 04927 Cortisol synthesis and secretion; 04261 Adrenergic signaling in cardiomyocytes; 04962 Vasopressin-regulated water reabsorption; 04725 Cholinergic synapse; 04728 Dopaminergic synapse; 04380 Osteoclast differentiation; 04211 Longevity regulating pathway; 04710 Circadian rhythm; 04713 Circadian entrainment; 04714 Thermogenesis; 05207 Chemical carcinogenesis - receptor activation; 05203 Viral carcinogenesis; 05215 Prostate cancer; 05166 Human T-cell leukemia virus 1 infection; 05161 Hepatitis B; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05165 Human papillomavirus infection; 05152 Tuberculosis; 05016 Huntington disease; 05020 Prion disease; 05030 Cocaine addiction; 05031 Amphetamine addiction; 05034 Alcoholism; 04931 Insulin resistance; 04934 Cushing syndrome; 03000 Transcription factors [BR:mmu03000]; 03029 Mitochondrial biogenesis [BR:mmu03029]; CREB; Ubiquitas transcription factors; ','[Pfam: pKID bZIP_1 bZIP_2 bZIP_Maf]'),('18044','mmu:18044','[Nfya, AA407810, Cbf-b, SEZ-10, SEZ10]','[(RefSeq) nuclear transcription factor-Y alpha]','[nuclear transcription factor Y, alpha]','[Mus musculus (house mouse)]','[Antigen processing and presentation, Spinocerebellar ataxia, Tuberculosis]','[]','[]','04612 Antigen processing and presentation; 05152 Tuberculosis; 05017 Spinocerebellar ataxia; 03000 Transcription factors [BR:mmu03000]; Heteromeric CCAAT factors; ','[Pfam: CBFB_NFYA]'),('13555','mmu:13555','[E2f1, E2F-1, Tg(Wnt1-cre)2Sor, mKIAA4009]','[(RefSeq) E2F transcription factor 1]','[transcription factor E2F1]','[Mus musculus (house mouse)]','[Endocrine resistance, Cell cycle, Mitophagy - animal, Cellular senescence, Cushing syndrome, Hepatitis C, Hepatitis B, Human cytomegalovirus infection, Human papillomavirus infection, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Epstein-Barr virus infection, Pathways in cancer, MicroRNAs in cancer, Chemical carcinogenesis - receptor activation, Pancreatic cancer, Glioma, Prostate cancer, Melanoma, Bladder cancer, Chronic myeloid leukemia, Small cell lung cancer, Non-small cell lung cancer, Breast cancer, Hepatocellular carcinoma, Gastric cancer]','[]','[]','04137 Mitophagy - animal; 04110 Cell cycle; 04218 Cellular senescence; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05207 Chemical carcinogenesis - receptor activation; 05212 Pancreatic cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05214 Glioma; 05220 Chronic myeloid leukemia; 05218 Melanoma; 05219 Bladder cancer; 05215 Prostate cancer; 05224 Breast cancer; 05222 Small cell lung cancer; 05223 Non-small cell lung cancer; 05166 Human T-cell leukemia virus 1 infection; 05161 Hepatitis B; 05160 Hepatitis C; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05165 Human papillomavirus infection; 04934 Cushing syndrome; 01522 Endocrine resistance; 03000 Transcription factors [BR:mmu03000]; Fork head/winged helix cell cycle-controlling factors, E2F; ','[Pfam: E2F_CC-MB E2F_TDP TP1]'),('22061','mmu:22061','[Trp63, AI462811, Ket, P51/P63, P63, P73l, Tp63, Trp53rp1]','[(RefSeq) transformation related protein 63]','[tumor protein p63]','[Mus musculus (house mouse)]','[MicroRNAs in cancer]','[]','[]','05206 MicroRNAs in cancer; 03000 Transcription factors [BR:mmu03000]; p53; ','[Pfam: P53 P53_tetramer SAM_2 Forkhead_N]'),('56458','mmu:56458','[Foxo1, AI876417, Afxh, FKHR, Fkhr1, Foxo1a]','[(RefSeq) forkhead box O1]','[forkhead box protein O1]','[Mus musculus (house mouse)]','[FoxO signaling pathway, AMPK signaling pathway, Longevity regulating pathway, Longevity regulating pathway - multiple species, Cellular senescence, Insulin signaling pathway, Thyroid hormone signaling pathway, Glucagon signaling pathway, Insulin resistance, AGE-RAGE signaling pathway in diabetic complications, Alcoholic liver disease, Human papillomavirus infection, Pathways in cancer, Transcriptional misregulation in cancer, Prostate cancer]','[]','[]','04068 FoxO signaling pathway; 04152 AMPK signaling pathway; 04218 Cellular senescence; 04910 Insulin signaling pathway; 04922 Glucagon signaling pathway; 04919 Thyroid hormone signaling pathway; 04211 Longevity regulating pathway; 04213 Longevity regulating pathway - multiple species; 05200 Pathways in cancer; 05202 Transcriptional misregulation in cancer; 05215 Prostate cancer; 05165 Human papillomavirus infection; 04936 Alcoholic liver disease; 04931 Insulin resistance; 04933 AGE-RAGE signaling pathway in diabetic complications; 03000 Transcription factors [BR:mmu03000]; Fork head/winged helix other regulators; ','[Pfam: FOXO_KIX_bdg Forkhead FOXO-TAD]'),('16598','mmu:16598','[Klf2, Lklf]','[(RefSeq) Kruppel-like factor 2 (lung)]','[krueppel-like factor 2]','[Mus musculus (house mouse)]','[FoxO signaling pathway, Apelin signaling pathway, Fluid shear stress and atherosclerosis]','[]','[]','04371 Apelin signaling pathway; 04068 FoxO signaling pathway; 05418 Fluid shear stress and atherosclerosis; 03000 Transcription factors [BR:mmu03000]; Cys2His2 SP/KLF family and related proteins; ','[Pfam: zf-C2H2 zf-H2C2_2 zf-C2H2_4]'),('22057','mmu:22057','[Tob1, Tob, Trob]','[(RefSeq) transducer of ErbB-2.1]','[protein Tob/BTG]','[Mus musculus (house mouse)]','[RNA degradation]','[]','[]','03018 RNA degradation; 03019 Messenger RNA biogenesis [BR:mmu03019]; P-body specific factors; Other 3\'-5\' decay factors; ','[Pfam: BTG PAM2 TFIIA CCDC53 YjbH]'),('21423','mmu:21423','[Tcf3, A1, AA408400, ALF2, AW209082, E12, E12/E47, E2A, E47, KA1, ME2, Pan1, Pan2, TCF-3, Tcfe2a, VDIR, bHLHb21]','[(RefSeq) transcription factor 3]','[transcription factor E2-alpha]','[Mus musculus (house mouse)]','[Signaling pathways regulating pluripotency of stem cells, Human T-cell leukemia virus 1 infection, Transcriptional misregulation in cancer]','[]','[]','04550 Signaling pathways regulating pluripotency of stem cells; 05202 Transcriptional misregulation in cancer; 05166 Human T-cell leukemia virus 1 infection; 03000 Transcription factors [BR:mmu03000]; Ubiquitous (class A) factors; ','[Pfam: HLH Response_reg_2]'),('20849','mmu:20849','[Stat4]','[(RefSeq) signal transducer and activator of transcription 4]','[signal transducer and activator of transcription 4]','[Mus musculus (house mouse)]','[Necroptosis, JAK-STAT signaling pathway, Th1 and Th2 cell differentiation, Hepatitis B, Pathways in cancer, Inflammatory bowel disease]','[]','[]','04630 JAK-STAT signaling pathway; 04217 Necroptosis; 04658 Th1 and Th2 cell differentiation; 05200 Pathways in cancer; 05161 Hepatitis B; 05321 Inflammatory bowel disease; 03000 Transcription factors [BR:mmu03000]; STAT; ','[Pfam: STAT_alpha STAT_int STAT_bind SH2 NPV_P10 Prp19]'),('12053','mmu:12053','[Bcl6, Bcl5]','[(RefSeq) B cell leukemia/lymphoma 6]','[B-cell lymphoma 6 protein]','[Mus musculus (house mouse)]','[FoxO signaling pathway, Transcriptional misregulation in cancer, Chemical carcinogenesis - receptor activation]','[]','[]','04068 FoxO signaling pathway; 05202 Transcriptional misregulation in cancer; 05207 Chemical carcinogenesis - receptor activation; 03000 Transcription factors [BR:mmu03000]; Cys2His2 BTB-ZF factors; ','[Pfam: BTB zf-H2C2_2 zf-C2H2 zf-met zf-C2H2_4 zf-C2H2_jaz DZR zf-C2HE zf-C2H2_6 Zn-ribbon_8]'),('21405','mmu:21405','[Hnf1a, AI323641, HNF1, HNF1-alpha, HNF1[a], Hnf-1, Hnf1alpha, LFB1, Tcf-1, Tcf1]','[(RefSeq) HNF1 homeobox A]','[transcription factor 1, hepatocyte nuclear factor 1-alpha]','[Mus musculus (house mouse)]','[Maturity onset diabetes of the young]','[]','[]','04950 Maturity onset diabetes of the young; 03000 Transcription factors [BR:mmu03000]; Homeo domain POU; ','[Pfam: HNF-1B_C HNF-1_N HNF-1A_C Homeodomain Telethonin Homeobox_KN]'),('18024','mmu:18024','[Nfe2l2, Nrf2]','[(RefSeq) nuclear factor, erythroid derived 2, like 2]','[nuclear factor erythroid 2-related factor 2]','[Mus musculus (house mouse)]','[Protein processing in endoplasmic reticulum, Parkinson disease, Pathways in cancer, Chemical carcinogenesis - reactive oxygen species, Hepatocellular carcinoma, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[]','[]','04141 Protein processing in endoplasmic reticulum; 05200 Pathways in cancer; 05208 Chemical carcinogenesis - reactive oxygen species; 05225 Hepatocellular carcinoma; 05012 Parkinson disease; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 03000 Transcription factors [BR:mmu03000]; 03029 Mitochondrial biogenesis [BR:mmu03029]; AP-1(-like) components, NF-E2; Nuclear respiratory factors; ','[Pfam: bZIP_Maf bZIP_1 bZIP_2]'),('14632','mmu:14632','[Gli1, AV235269, Zfp-5, Zfp5]','[(RefSeq) GLI-Kruppel family member GLI1]','[zinc finger protein GLI1]','[Mus musculus (house mouse)]','[cAMP signaling pathway, Hedgehog signaling pathway, Pathways in cancer, Basal cell carcinoma]','[]','[]','04340 Hedgehog signaling pathway; 04024 cAMP signaling pathway; 05200 Pathways in cancer; 05217 Basal cell carcinoma; 03000 Transcription factors [BR:mmu03000]; Cys2His2 GLI-like; ','[Pfam: zf-C2H2 zf-H2C2_2 FOXP-CC zf-C2H2_4 IL17]'),('11925','mmu:11925','[Neurog3, Atoh5, Math4B, bHLHa7, ngn3]','[(RefSeq) neurogenin 3]','[neurogenin 3]','[Mus musculus (house mouse)]','[Maturity onset diabetes of the young]','[]','[]','04950 Maturity onset diabetes of the young; 03000 Transcription factors [BR:mmu03000]; Tal/Twist/Atonal/Hen, Atonal; ','[Pfam: HLH]'),('14247','mmu:14247','[Fli1, EWSR2, Fli-1, SIC-1, Sic1]','[(RefSeq) Friend leukemia integration 1]','[friend leukemia integration 1 transcription factor]','[Mus musculus (house mouse)]','[Transcriptional misregulation in cancer]','[]','[]','05202 Transcriptional misregulation in cancer; 03000 Transcription factors [BR:mmu03000]; Tryptophan clusters Ets-type; ','[Pfam: Ets SAM_PNT]'),('12387','mmu:12387','[Ctnnb1, Bfc, Catnb, Mesc]','[(RefSeq) catenin (cadherin associated protein), beta 1]','[catenin beta 1]','[Mus musculus (house mouse)]','[Rap1 signaling pathway, Wnt signaling pathway, Hippo signaling pathway, Focal adhesion, Adherens junction, Signaling pathways regulating pluripotency of stem cells, Leukocyte transendothelial migration, Melanogenesis, Thyroid hormone signaling pathway, Cushing syndrome, Alcoholic liver disease, Alzheimer disease, Pathways of neurodegeneration - multiple diseases, Bacterial invasion of epithelial cells, Salmonella infection, Hepatitis C, Human cytomegalovirus infection, Human papillomavirus infection, Kaposi sarcoma-associated herpesvirus infection, Pathways in cancer, Proteoglycans in cancer, Colorectal cancer, Endometrial cancer, Prostate cancer, Thyroid cancer, Basal cell carcinoma, Breast cancer, Hepatocellular carcinoma, Gastric cancer, Arrhythmogenic right ventricular cardiomyopathy, Fluid shear stress and atherosclerosis]','[]','[]','04015 Rap1 signaling pathway; 04310 Wnt signaling pathway; 04390 Hippo signaling pathway; 04510 Focal adhesion; 04520 Adherens junction; 04550 Signaling pathways regulating pluripotency of stem cells; 04670 Leukocyte transendothelial migration; 04919 Thyroid hormone signaling pathway; 04916 Melanogenesis; 05200 Pathways in cancer; 05205 Proteoglycans in cancer; 05210 Colorectal cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05216 Thyroid cancer; 05217 Basal cell carcinoma; 05215 Prostate cancer; 05213 Endometrial cancer; 05224 Breast cancer; 05160 Hepatitis C; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05165 Human papillomavirus infection; 05132 Salmonella infection; 05100 Bacterial invasion of epithelial cells; 05010 Alzheimer disease; 05022 Pathways of neurodegeneration - multiple diseases; 05418 Fluid shear stress and atherosclerosis; 05412 Arrhythmogenic right ventricular cardiomyopathy; 04936 Alcoholic liver disease; 04934 Cushing syndrome; ','[Pfam: Arm Cnd1 HEAT_2 Adaptin_N HEAT_EZ KAP Atx10homo_assoc Arm_2 Queuosine_synth Phage_T7_Capsid]'),('13653','mmu:13653','[Egr1, A530045N19Rik, ETR103, Egr-1, Krox-1, Krox-24, Krox24, NGF1-A, NGFI-A, NGFIA, TIS8, Zenk, Zfp-6, Zif268, egr]','[(RefSeq) early growth response 1]','[early growth response protein 1]','[Mus musculus (house mouse)]','[Apelin signaling pathway, GnRH signaling pathway, Parathyroid hormone synthesis, secretion and action, AGE-RAGE signaling pathway in diabetic complications, Prion disease, Human T-cell leukemia virus 1 infection]','[]','[]','04371 Apelin signaling pathway; 04912 GnRH signaling pathway; 04928 Parathyroid hormone synthesis, secretion and action; 05166 Human T-cell leukemia virus 1 infection; 05020 Prion disease; 04933 AGE-RAGE signaling pathway in diabetic complications; 03000 Transcription factors [BR:mmu03000]; Cys2His2 SP/KLF family and related proteins; ','[Pfam: DUF3432 DUF3446 zf-H2C2_2 zf-C2H2 zf-C2H2_4 zf-C2HC_2 zf-RING_15 zf-BED zf-H2C2_5]'),('328572','mmu:328572','[Ep300, A430090G16, A730011L11, KAT3B, p300, p300_HAT]','[(RefSeq) E1A binding protein p300]','[E1A/CREB-binding protein [EC:2.3.1.48]]','[Mus musculus (house mouse)]','[Viral life cycle - HIV-1, cAMP signaling pathway, HIF-1 signaling pathway, FoxO signaling pathway, Cell cycle, Wnt signaling pathway, Notch signaling pathway, TGF-beta signaling pathway, Adherens junction, JAK-STAT signaling pathway, Long-term potentiation, Melanogenesis, Thyroid hormone signaling pathway, Glucagon signaling pathway, Growth hormone synthesis, secretion and action, Huntington disease, Tuberculosis, Hepatitis B, Influenza A, Human papillomavirus infection, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Pathways in cancer, Viral carcinogenesis, MicroRNAs in cancer, Renal cell carcinoma, Prostate cancer]','[]','[]','03250 Viral life cycle - HIV-1; 04310 Wnt signaling pathway; 04330 Notch signaling pathway; 04350 TGF-beta signaling pathway; 04630 JAK-STAT signaling pathway; 04066 HIF-1 signaling pathway; 04068 FoxO signaling pathway; 04024 cAMP signaling pathway; 04110 Cell cycle; 04520 Adherens junction; 04922 Glucagon signaling pathway; 04935 Growth hormone synthesis, secretion and action; 04919 Thyroid hormone signaling pathway; 04916 Melanogenesis; 04720 Long-term potentiation; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05203 Viral carcinogenesis; 05211 Renal cell carcinoma; 05215 Prostate cancer; 05166 Human T-cell leukemia virus 1 infection; 05161 Hepatitis B; 05164 Influenza A; 05167 Kaposi sarcoma-associated herpesvirus infection; 05165 Human papillomavirus infection; 05152 Tuberculosis; 05016 Huntington disease; 03000 Transcription factors [BR:mmu03000]; 03036 Chromosome and associated proteins [BR:mmu03036]; 03029 Mitochondrial biogenesis [BR:mmu03029]; 2.3.1.48  histone acetyltransferase; Pocket domain CBP; HATs (histone acetyltransferases); Coactivators; ','[Pfam: HAT_KAT11 KIX Creb_binding zf-TAZ RING_CBP-p300 Bromodomain ZZ KIX_2 Npa1]'),('18609','mmu:18609','[Pdx1, IDX-1, IPF-1, Ipf1, Mody4, STF-1, pdx-1]','[(RefSeq) pancreatic and duodenal homeobox 1]','[insulin promoter factor 1]','[Mus musculus (house mouse)]','[Insulin secretion, Type II diabetes mellitus, Maturity onset diabetes of the young]','[]','[]','04911 Insulin secretion; 04930 Type II diabetes mellitus; 04950 Maturity onset diabetes of the young; 03000 Transcription factors [BR:mmu03000]; Homeo domain ANTP: HOXL; ','[Pfam: Homeodomain]'),('56711','mmu:56711','[Plag1]','[(RefSeq) pleiomorphic adenoma gene 1]','[zinc finger protein PLAG1]','[Mus musculus (house mouse)]',NULL,'[]','[]','03000 Transcription factors [BR:mmu03000]; Cys2His2 others; ','[Pfam: zf-C2H2_4 zf-C2H2 zf-H2C2_2 zf-C2H2_jaz zf-met zf-C2H2_6 Acyltransf_C zf-C2H2_11 zf-H2C2_5]'),('17428','mmu:17428','[Mnt, Rox, bHLHd3]','[(RefSeq) max binding protein]','[MAX-binding protein]','[Mus musculus (house mouse)]',NULL,'[]','[]','03000 Transcription factors [BR:mmu03000]; Cell cycle controlling factors, Mad/Max; ','[Pfam: HLH Sipho_Gp157]'),('56505','mmu:56505','[Ruvbl1, 2510009G06Rik, Pontin52, Tip49a]','[(RefSeq) RuvB-like protein 1]','[RuvB-like protein 1 [EC:5.6.2.4]]','[Mus musculus (house mouse)]','[Wnt signaling pathway]','[]','[]','04310 Wnt signaling pathway; 03036 Chromosome and associated proteins [BR:mmu03036]; 03037 Cilium and associated proteins [BR:mmu03037]; 5.6.2.4  DNA 3\'-5\' helicase; TIP60 complex; SRCAP complex; INO80 complex; INO80 complex (yeast); SWR1 complex; ASTRA complex; Dynein assembly factors; ','[Pfam: TIP49 TIP49_C AAA RuvB_N AAA_22 AAA_5 AAA_16 Sigma54_activat bpMoxR Mg_chelatase AAA_25 AAA_19 AAA_18 AAA_28 IstB_IS21 DnaB_C Zeta_toxin TsaE AAA_7 ResIII AAA_33 NACHT Parvo_NS1 PhoH AAA_30 AAA_11 NTPase_1 ATPase DUF2075 RNA_helicase]'),('15251','mmu:15251','[Hif1a, AA959795, HIF-1-alpha, HIF1-alpha, HIF1alpha, MOP1, bHLHe78]','[(RefSeq) hypoxia inducible factor 1, alpha subunit]','[hypoxia-inducible factor 1 alpha]','[Mus musculus (house mouse)]','[HIF-1 signaling pathway, Mitophagy - animal, Autophagy - animal, Th17 cell differentiation, Thyroid hormone signaling pathway, Kaposi sarcoma-associated herpesvirus infection, Pathways in cancer, Proteoglycans in cancer, Chemical carcinogenesis - reactive oxygen species, Renal cell carcinoma, Central carbon metabolism in cancer, Choline metabolism in cancer, PD-L1 expression and PD-1 checkpoint pathway in cancer]','[]','[]','04066 HIF-1 signaling pathway; 04140 Autophagy - animal; 04137 Mitophagy - animal; 04659 Th17 cell differentiation; 04919 Thyroid hormone signaling pathway; 05200 Pathways in cancer; 05205 Proteoglycans in cancer; 05208 Chemical carcinogenesis - reactive oxygen species; 05230 Central carbon metabolism in cancer; 05231 Choline metabolism in cancer; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05211 Renal cell carcinoma; 05167 Kaposi sarcoma-associated herpesvirus infection; 03000 Transcription factors [BR:mmu03000]; Factors with PAS domain; ','[Pfam: PAS_3 HIF-1a_CTAD PAS_11 PAS HIF-1 PAS_9 PAS_4 HLH]'),('13819','mmu:13819','[Epas1, HIF-2alpha, HIF2A, HLF, HRF, MOP2, bHLHe73]','[(RefSeq) endothelial PAS domain protein 1]','[hypoxia-inducible factor 2 alpha]','[Mus musculus (house mouse)]','[Pathways in cancer, Renal cell carcinoma]','[]','[]','05200 Pathways in cancer; 05211 Renal cell carcinoma; 03000 Transcription factors [BR:mmu03000]; 04131 Membrane trafficking [BR:mmu04131]; Factors with PAS domain; Other pexophagy associated proteins; ','[Pfam: PAS_3 PAS_11 HIF-1a_CTAD HIF-1 PAS PAS_9 PAS_4 Corona_I HLH]'),('11865','mmu:11865','[Arntl, Arnt3, BMAL1b, Bmal1, MOP3, bHLHe5, bmal1b\']','[(RefSeq) aryl hydrocarbon receptor nuclear translocator-like]','[aryl hydrocarbon receptor nuclear translocator-like protein 1]','[Mus musculus (house mouse)]','[Circadian rhythm, Dopaminergic synapse]','[]','[]','04728 Dopaminergic synapse; 04710 Circadian rhythm; 03000 Transcription factors [BR:mmu03000]; Factors with PAS domain; ','[Pfam: PAS_11 PAS PAS_3 HLH PAS_9 PAS_4 PAS_2]'),('12578','mmu:12578','[Cdkn2a, ARF-INK4a, Arf, INK4a-ARF, Ink4a/Arf, MTS1, Pctr1, p16, p16(INK4a), p16INK4a, p19<ARF>, p19ARF]','[(RefSeq) cyclin dependent kinase inhibitor 2A]','[cyclin-dependent kinase inhibitor 2A]','[Mus musculus (house mouse)]','[Endocrine resistance, Platinum drug resistance, Cell cycle, p53 signaling pathway, Cellular senescence, Cushing syndrome, Human cytomegalovirus infection, Human T-cell leukemia virus 1 infection, Pathways in cancer, Viral carcinogenesis, MicroRNAs in cancer, Pancreatic cancer, Glioma, Melanoma, Bladder cancer, Chronic myeloid leukemia, Non-small cell lung cancer, Hepatocellular carcinoma]','[]','[]','04110 Cell cycle; 04115 p53 signaling pathway; 04218 Cellular senescence; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05203 Viral carcinogenesis; 05212 Pancreatic cancer; 05225 Hepatocellular carcinoma; 05214 Glioma; 05220 Chronic myeloid leukemia; 05218 Melanoma; 05219 Bladder cancer; 05223 Non-small cell lung cancer; 05166 Human T-cell leukemia virus 1 infection; 05163 Human cytomegalovirus infection; 04934 Cushing syndrome; 01524 Platinum drug resistance; 01522 Endocrine resistance; ','[Pfam: Ank_4 Ank_5 Ank_2]'),('18129','mmu:18129','[Notch2, AI853703, N2]','[(RefSeq) notch 2]','[Notch 2]','[Mus musculus (house mouse)]','[Endocrine resistance, Notch signaling pathway, Th1 and Th2 cell differentiation, Thyroid hormone signaling pathway, Human papillomavirus infection, Pathways in cancer, MicroRNAs in cancer, Chemical carcinogenesis - receptor activation, Breast cancer]','[]','[]','04330 Notch signaling pathway; 04658 Th1 and Th2 cell differentiation; 04919 Thyroid hormone signaling pathway; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05207 Chemical carcinogenesis - receptor activation; 05224 Breast cancer; 05165 Human papillomavirus infection; 01522 Endocrine resistance; ','[Pfam: EGF hEGF EGF_CA EGF_2 Notch Ank_2 NOD Ank Ank_5 NODP Ank_3 Ank_4 EGF_3 cEGF]'),('18131','mmu:18131','[Notch3, AW229011, N3, hpbk]','[(RefSeq) notch 3]','[Notch 3]','[Mus musculus (house mouse)]','[Endocrine resistance, Notch signaling pathway, Apelin signaling pathway, Th1 and Th2 cell differentiation, Thyroid hormone signaling pathway, Human papillomavirus infection, Pathways in cancer, MicroRNAs in cancer, Breast cancer]','[]','[]','04330 Notch signaling pathway; 04371 Apelin signaling pathway; 04658 Th1 and Th2 cell differentiation; 04919 Thyroid hormone signaling pathway; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05224 Breast cancer; 05165 Human papillomavirus infection; 01522 Endocrine resistance; ','[Pfam: EGF hEGF EGF_CA Notch EGF_2 Ank Ank_2 NODP Ank_5 NOD Ank_4 Ank_3 EGF_3 cEGF]'),('18132','mmu:18132','[Notch4, Int-3, Int3, N4]','[(RefSeq) notch 4]','[Notch 4]','[Mus musculus (house mouse)]','[Endocrine resistance, Notch signaling pathway, Thyroid hormone signaling pathway, Human papillomavirus infection, Pathways in cancer, MicroRNAs in cancer, Breast cancer]','[]','[]','04330 Notch signaling pathway; 04919 Thyroid hormone signaling pathway; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05224 Breast cancer; 05165 Human papillomavirus infection; 01522 Endocrine resistance; ','[Pfam: EGF hEGF EGF_CA Ank Ank_5 Ank_2 EGF_3 Notch Ank_3 Ank_4 EGF_2 NODP NOD EGF_MSP1_1 cEGF]'),('11911','mmu:11911','[Atf4, Atf-4, C/ATF, CREB-2, CREB2, TAXREB67]','[(RefSeq) activating transcription factor 4]','[cyclic AMP-dependent transcription factor ATF-4]','[Mus musculus (house mouse)]','[MAPK signaling pathway, cGMP-PKG signaling pathway, Mitophagy - animal, Protein processing in endoplasmic reticulum, PI3K-Akt signaling pathway, Apoptosis, Longevity regulating pathway, Adrenergic signaling in cardiomyocytes, TNF signaling pathway, Long-term potentiation, Neurotrophin signaling pathway, Cholinergic synapse, Dopaminergic synapse, Insulin secretion, GnRH signaling pathway, Estrogen signaling pathway, Thyroid hormone synthesis, Glucagon signaling pathway, Aldosterone synthesis and secretion, Relaxin signaling pathway, Cortisol synthesis and secretion, Parathyroid hormone synthesis, secretion and action, Non-alcoholic fatty liver disease, Cushing syndrome, Growth hormone synthesis, secretion and action, Alzheimer disease, Parkinson disease, Amyotrophic lateral sclerosis, Prion disease, Pathways of neurodegeneration - multiple diseases, Cocaine addiction, Amphetamine addiction, Alcoholism, Hepatitis B, Human cytomegalovirus infection, Human T-cell leukemia virus 1 infection, Viral carcinogenesis, Chemical carcinogenesis - receptor activation, Prostate cancer, Lipid and atherosclerosis]','[]','[]','04141 Protein processing in endoplasmic reticulum; 04010 MAPK signaling pathway; 04668 TNF signaling pathway; 04022 cGMP-PKG signaling pathway; 04151 PI3K-Akt signaling pathway; 04137 Mitophagy - animal; 04210 Apoptosis; 04911 Insulin secretion; 04922 Glucagon signaling pathway; 04912 GnRH signaling pathway; 04915 Estrogen signaling pathway; 04926 Relaxin signaling pathway; 04935 Growth hormone synthesis, secretion and action; 04918 Thyroid hormone synthesis; 04928 Parathyroid hormone synthesis, secretion and action; 04925 Aldosterone synthesis and secretion; 04927 Cortisol synthesis and secretion; 04261 Adrenergic signaling in cardiomyocytes; 04725 Cholinergic synapse; 04728 Dopaminergic synapse; 04720 Long-term potentiation; 04722 Neurotrophin signaling pathway; 04211 Longevity regulating pathway; 05207 Chemical carcinogenesis - receptor activation; 05203 Viral carcinogenesis; 05215 Prostate cancer; 05166 Human T-cell leukemia virus 1 infection; 05161 Hepatitis B; 05163 Human cytomegalovirus infection; 05010 Alzheimer disease; 05012 Parkinson disease; 05014 Amyotrophic lateral sclerosis; 05020 Prion disease; 05022 Pathways of neurodegeneration - multiple diseases; 05030 Cocaine addiction; 05031 Amphetamine addiction; 05034 Alcoholism; 05417 Lipid and atherosclerosis; 04932 Non-alcoholic fatty liver disease; 04934 Cushing syndrome; 03000 Transcription factors [BR:mmu03000]; AP-1(-like) components, CRE-BP/ATF; ','[Pfam: bZIP_1 bZIP_2 DUF4407 Fez1 Laminin_II DUF3450 AAA_13]'),('12189','mmu:12189','[Brca1]','[(RefSeq) breast cancer 1, early onset]','[breast cancer type 1 susceptibility protein [EC:2.3.2.27]]','[Mus musculus (house mouse)]','[Platinum drug resistance, Homologous recombination, Fanconi anemia pathway, Ubiquitin mediated proteolysis, PI3K-Akt signaling pathway, MicroRNAs in cancer, Breast cancer]','[]','[]','04120 Ubiquitin mediated proteolysis; 03440 Homologous recombination; 03460 Fanconi anemia pathway; 04151 PI3K-Akt signaling pathway; 05206 MicroRNAs in cancer; 05224 Breast cancer; 01524 Platinum drug resistance; 01009 Protein phosphatases and associated proteins [BR:mmu01009]; 04121 Ubiquitin system [BR:mmu04121]; 03036 Chromosome and associated proteins [BR:mmu03036]; 03400 DNA repair and recombination proteins [BR:mmu03400]; 2.3.2.27  RING-type E3 ubiquitin transferase; PP1-interacting proteins (PIPs); Other single Ring-finger type E3; BRCC complex; Regulators of centriole replication; BRCA1-core complex; BRCA1-A complex; BRCA1-B complex; BRCA1-C complex; BRCA complex; Downstream FA components; ','[Pfam: BRCT_assoc BRCT zf-C3HC4 zf-C3HC4_2 BRCT_2 zf-C3HC4_3 zf-RING_UBOX zf-C3HC4_4 zf-RING_2 zf-RING_5 zf-Nse zf-RING_6 Prok-RING_4 zf-ANAPC11 zf-RING_10 Baculo_IE-1]'),('15378','mmu:15378','[Hnf4a, HNF-4, Hnf4, Hnf4alpha, MODY1, Nr2a1, TCF-14, Tcf14]','[(RefSeq) hepatic nuclear factor 4, alpha]','[hepatocyte nuclear factor 4-alpha]','[Mus musculus (house mouse)]','[AMPK signaling pathway, Maturity onset diabetes of the young]','[]','[]','04152 AMPK signaling pathway; 04950 Maturity onset diabetes of the young; 03000 Transcription factors [BR:mmu03000]; 03310 Nuclear receptors [BR:mmu03310]; Cys4 hepatocyte nuclear factor 4-like; 2A. Hepatocyte nuclear factor 4 receptor; ','[Pfam: Hormone_recep zf-C4]'),('21385','mmu:21385','[Tbx2]','[(RefSeq) T-box 2]','[T-box protein 2]','[Mus musculus (house mouse)]',NULL,'[]','[]','03000 Transcription factors [BR:mmu03000]; T-box; ','[Pfam: T-box TBX]'),('56312','mmu:56312','[Nupr1, 2310032H04Rik, Com1, p8]','[(RefSeq) nuclear protein transcription regulator 1]','[nuclear protein 1]','[Mus musculus (house mouse)]','[Transcriptional misregulation in cancer]','[]','[]','05202 Transcriptional misregulation in cancer; 03000 Transcription factors [BR:mmu03000]; NUPR; ','[Pfam: Phospho_p8]'),('18089','mmu:18089','[Nkx2-3, Nkx-2.3, Nkx2.3, nkx2-C, tinman]','[(RefSeq) NK2 homeobox 3]','[homeobox protein Nkx-2.3]','[Mus musculus (house mouse)]',NULL,'[]','[]','03000 Transcription factors [BR:mmu03000]; Homeo domain ANTP: NKL; ','[Pfam: Homeodomain Homeobox_KN]'),('22431','mmu:22431','[Wt1, D630046I19Rik, Wt-1]','[(RefSeq) WT1 transcription factor]','[Wilms tumor protein 1]','[Mus musculus (house mouse)]','[Transcriptional misregulation in cancer]','[]','[]','05202 Transcriptional misregulation in cancer; 03000 Transcription factors [BR:mmu03000]; Cys2His2 SP/KLF family and related proteins; ','[Pfam: WT1 zf-H2C2_2 zf-C2H2 zf-C2H2_4 zf-met zf-C2H2_aberr zf-C2H2_8 FOXP-CC]'),('20198','mmu:20198','[S100a4, 18A2, 42a, Capl, FSp1, Mts1, PeL98, metastasin, pk9a]','[(RefSeq) S100 calcium binding protein A4]','[protein S100-A4]','[Mus musculus (house mouse)]',NULL,'[]','[]','04990 Domain-containing proteins not elsewhere classified [BR:mmu04990]; S100 proteins; ','[Pfam: S_100 EF-hand_7 EF-hand_1 EF-hand_4 EF-hand_6 EF-hand_8 EF-hand_5]'),('12826','mmu:12826','[Col4a1, Bru, Col4a-1, Raw, Svc]','[(RefSeq) collagen, type IV, alpha 1]','[collagen type IV alpha]','[Mus musculus (house mouse)]','[PI3K-Akt signaling pathway, Focal adhesion, ECM-receptor interaction, Relaxin signaling pathway, AGE-RAGE signaling pathway in diabetic complications, Protein digestion and absorption, Amoebiasis, Human papillomavirus infection, Pathways in cancer, Small cell lung cancer]','[]','[]','04151 PI3K-Akt signaling pathway; 04512 ECM-receptor interaction; 04510 Focal adhesion; 04926 Relaxin signaling pathway; 04974 Protein digestion and absorption; 05200 Pathways in cancer; 05222 Small cell lung cancer; 05165 Human papillomavirus infection; 05146 Amoebiasis; 04933 AGE-RAGE signaling pathway in diabetic complications; 04147 Exosome [BR:mmu04147]; 00536 Glycosaminoglycan binding proteins [BR:mmu00536]; Exosomal proteins of other cancer cells; Extracellular matrix molecules; ','[Pfam: Collagen C4]'),('20302','mmu:20302','[Ccl3, AI323804, G0S19-1, LD78alpha, MIP-1alpha, MIP1-(a), MIP1-alpha, Mip1a, Scya3]','[(RefSeq) chemokine (C-C motif) ligand 3]','[C-C motif chemokine 3]','[Mus musculus (house mouse)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, Chemokine signaling pathway, Toll-like receptor signaling pathway, Chagas disease, Human cytomegalovirus infection, Rheumatoid arthritis, Lipid and atherosclerosis]','[]','[]','04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04620 Toll-like receptor signaling pathway; 04062 Chemokine signaling pathway; 05163 Human cytomegalovirus infection; 05142 Chagas disease; 05323 Rheumatoid arthritis; 05417 Lipid and atherosclerosis; 04052 Cytokines and growth factors [BR:mmu04052]; 00536 Glycosaminoglycan binding proteins [BR:mmu00536]; Chemokines; Chemokines; ','[Pfam: IL8]'),('228714','mmu:228714','[Kat14, 2510008M08Rik, ATAC2, AU023459, Csrp2bp, D2Ertd473e, D2Wsu131e, E430020F17]','[(RefSeq) lysine acetyltransferase 14]','[cysteine-rich protein 2-binding protein]','[Mus musculus (house mouse)]',NULL,'[]','[]','03036 Chromosome and associated proteins [BR:mmu03036]; ATAC complex; ','[Pfam: Acetyltransf_1 Acetyltransf_10 Acetyltransf_7 FR47 Acetyltransf_9 GNAT_acetyltran]'),('20871','mmu:20871','[Aurkc, AIE1, AIK3, ARK-3, IAK3, Stk13]','[(RefSeq) aurora kinase C]','[aurora kinase C [EC:2.7.11.1]]','[Mus musculus (house mouse)]',NULL,'[]','[]','01001 Protein kinases [BR:mmu01001]; 03036 Chromosome and associated proteins [BR:mmu03036]; 2.7.11.1  non-specific serine/threonine protein kinase; AUR family [OT]; Protein kinases; Aurora kinases; ','[Pfam: Pkinase PK_Tyr_Ser-Thr ABC1 Kinase-like APH Kdo YrbL-PhoP_reg]'),('208266','mmu:208266','[Dot1l, A630076O07, AW907654, Dot1, KMT4, mDot1]','[(RefSeq) DOT1-like, histone H3 methyltransferase (S. cerevisiae)]','[[histone H3]-lysine79 N-trimethyltransferase [EC:2.1.1.360]]','[Mus musculus (house mouse)]','[Lysine degradation, Metabolic pathways, Transcriptional misregulation in cancer]','[]','[]','00310 Lysine degradation; 05202 Transcriptional misregulation in cancer; 03036 Chromosome and associated proteins [BR:mmu03036]; 2.1.1.360  [histone H3]-lysine79 N-trimethyltransferase; HKMTs (histone lysine methyltransferases); Dot1-containing complex; ','[Pfam: DOT1]'),('81601','mmu:81601','[Kat5, AI839539, CPLA2, Htatip, Htatip1, PLIP, Tip55, Tip60]','[(RefSeq) K(lysine) acetyltransferase 5]','[histone acetyltransferase HTATIP [EC:2.3.1.48]]','[Mus musculus (house mouse)]','[Spinocerebellar ataxia, Human T-cell leukemia virus 1 infection]','[]','[]','05166 Human T-cell leukemia virus 1 infection; 05017 Spinocerebellar ataxia; 03000 Transcription factors [BR:mmu03000]; 03036 Chromosome and associated proteins [BR:mmu03036]; 2.3.1.48  histone acetyltransferase; Cys2HisCys zinc factors; HATs (histone acetyltransferases); TIP60 complex; NuA4 complex; ','[Pfam: MOZ_SAS zf-MYST Tudor-knot Acetyltransf_7 Acetyltransf_1]'),('15185','mmu:15185','[Hdac6, Hd6, Hdac5, Sfc6, mHDA2]','[(RefSeq) histone deacetylase 6]','[histone deacetylase 6 [EC:3.5.1.98]]','[Mus musculus (house mouse)]','[Neutrophil extracellular trap formation, Amyotrophic lateral sclerosis, Alcoholism, Viral carcinogenesis]','[]','[]','04613 Neutrophil extracellular trap formation; 05203 Viral carcinogenesis; 05014 Amyotrophic lateral sclerosis; 05034 Alcoholism; 01009 Protein phosphatases and associated proteins [BR:mmu01009]; 03019 Messenger RNA biogenesis [BR:mmu03019]; 04131 Membrane trafficking [BR:mmu04131]; 03036 Chromosome and associated proteins [BR:mmu03036]; 3.5.1.98  histone deacetylase; PP1-interacting proteins (PIPs); Stress granule specific factors; Other aggrephagy associated proteins; Class II HDACs; ','[Pfam: Hist_deacetyl zf-UBP zf-UBP_var]'),('54631','mmu:54631','[Nphs1, NephrinB, nephrin]','[(RefSeq) nephrosis 1, nephrin]','[nephron]','[Mus musculus (house mouse)]',NULL,'[]','[]','04990 Domain-containing proteins not elsewhere classified [BR:mmu04990]; Fibronectin type III domain-containing proteins; ','[Pfam: C2-set_2 Ig_3 I-set Ig_2 ig V-set fn3 DUF4643]'),('170643','mmu:170643','[Kirrel, 6720469N11Rik, Kirrel1, Neph1]','[(RefSeq) kirre like nephrin family adhesion molecule 1]',NULL,'[Mus musculus (house mouse)]',NULL,'[]','[]',NULL,'[Pfam: Ig_3 I-set Ig_2 ig V-set C2-set_2 Ig_5 C1-set Ig_4 DUF1180]'),('104027','mmu:104027','[Synpo, 9030217H17Rik, 9130229N11, 9330140I15Rik, AW046661]','[(RefSeq) synaptopodin]','[synaptopodin]','[Mus musculus (house mouse)]','[Tight junction]','[]','[]','04530 Tight junction; ','[Pfam: Calsarcin]'),('22067','mmu:22067','[Trpc5, CCE2, TRP-5, TRP5, Trrp5]','[(RefSeq) transient receptor potential cation channel, subfamily C, member 5]','[transient receptor potential cation channel subfamily C member 5]','[Mus musculus (house mouse)]','[Axon guidance, GnRH secretion]','[]','[]','04929 GnRH secretion; 04360 Axon guidance; 01009 Protein phosphatases and associated proteins [BR:mmu01009]; 04040 Ion channels [BR:mmu04040]; PP1-interacting proteins (PIPs); Transient receptor potential channel (TRP); ','[Pfam: Ion_trans TRP_2 Ank_2 Ank_4 Ank_3 PKD_channel Ank]'),('22068','mmu:22068','[Trpc6, AV025995, LLHWJM002, LLHWJM003, LLHWJM004, TRP-6, Trrp6, mtrp6]','[(RefSeq) transient receptor potential cation channel, subfamily C, member 6]','[transient receptor potential cation channel subfamily C member 6]','[Mus musculus (house mouse)]','[cGMP-PKG signaling pathway, Axon guidance]','[]','[]','04022 cGMP-PKG signaling pathway; 04360 Axon guidance; 04040 Ion channels [BR:mmu04040]; Transient receptor potential channel (TRP); ','[Pfam: Ion_trans TRP_2 PKD_channel Ank_2 Ank_4 Ank_5 CBFB_NFYA ING]'),('19353','mmu:19353','[Rac1, AL023026, D5Ertd559e]','[(RefSeq) Rac family small GTPase 1]','[Ras-related C3 botulinum toxin substrate 1]','[Mus musculus (house mouse)]','[MAPK signaling pathway, Ras signaling pathway, Rap1 signaling pathway, cAMP signaling pathway, Chemokine signaling pathway, Sphingolipid signaling pathway, Phagosome, PI3K-Akt signaling pathway, Wnt signaling pathway, Axon guidance, VEGF signaling pathway, Osteoclast differentiation, Focal adhesion, Adherens junction, Tight junction, Neutrophil extracellular trap formation, Toll-like receptor signaling pathway, Natural killer cell mediated cytotoxicity, B cell receptor signaling pathway, Fc epsilon RI signaling pathway, Fc gamma R-mediated phagocytosis, Leukocyte transendothelial migration, Neurotrophin signaling pathway, Regulation of actin cytoskeleton, Non-alcoholic fatty liver disease, AGE-RAGE signaling pathway in diabetic complications, Pancreatic secretion, Amyotrophic lateral sclerosis, Prion disease, Pathways of neurodegeneration - multiple diseases, Bacterial invasion of epithelial cells, Salmonella infection, Yersinia infection, Human cytomegalovirus infection, Kaposi sarcoma-associated herpesvirus infection, Epstein-Barr virus infection, Human immunodeficiency virus 1 infection, Pathways in cancer, Viral carcinogenesis, Proteoglycans in cancer, Chemical carcinogenesis - reactive oxygen species, Colorectal cancer, Renal cell carcinoma, Pancreatic cancer, Choline metabolism in cancer, Diabetic cardiomyopathy, Viral myocarditis, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[]','[]','04010 MAPK signaling pathway; 04014 Ras signaling pathway; 04015 Rap1 signaling pathway; 04310 Wnt signaling pathway; 04370 VEGF signaling pathway; 04071 Sphingolipid signaling pathway; 04024 cAMP signaling pathway; 04151 PI3K-Akt signaling pathway; 04145 Phagosome; 04510 Focal adhesion; 04520 Adherens junction; 04530 Tight junction; 04810 Regulation of actin cytoskeleton; 04613 Neutrophil extracellular trap formation; 04620 Toll-like receptor signaling pathway; 04650 Natural killer cell mediated cytotoxicity; 04662 B cell receptor signaling pathway; 04664 Fc epsilon RI signaling pathway; 04666 Fc gamma R-mediated phagocytosis; 04670 Leukocyte transendothelial migration; 04062 Chemokine signaling pathway; 04972 Pancreatic secretion; 04722 Neurotrophin signaling pathway; 04360 Axon guidance; 04380 Osteoclast differentiation; 05200 Pathways in cancer; 05205 Proteoglycans in cancer; 05208 Chemical carcinogenesis - reactive oxygen species; 05203 Viral carcinogenesis; 05231 Choline metabolism in cancer; 05210 Colorectal cancer; 05212 Pancreatic cancer; 05211 Renal cell carcinoma; 05170 Human immunodeficiency virus 1 infection; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05132 Salmonella infection; 05135 Yersinia infection; 05100 Bacterial invasion of epithelial cells; 05014 Amyotrophic lateral sclerosis; 05020 Prion disease; 05022 Pathways of neurodegeneration - multiple diseases; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 05415 Diabetic cardiomyopathy; 05416 Viral myocarditis; 04932 Non-alcoholic fatty liver disease; 04933 AGE-RAGE signaling pathway in diabetic complications; 04131 Membrane trafficking [BR:mmu04131]; 04147 Exosome [BR:mmu04147]; 04031 GTP-binding proteins [BR:mmu04031]; Rho GTPases; Arf6-dependent endocytosis; Ras GTPases; Nox associated proteins; Exosomal proteins of haemopoietic cells  (B-cell, T-cell, DC-cell, reticulocyte, and mast cell); Exosomal proteins of other body fluids (saliva and urine); Exosomal proteins of colorectal cancer cells; Exosomal proteins of bladder cancer cells; Rac/Cdc42 [OT]; ','[Pfam: Ras Roc Arf CagD]'),('12540','mmu:12540','[Cdc42, AI747189, AU018915]','[(RefSeq) cell division cycle 42]','[cell division control protein 42]','[Mus musculus (house mouse)]','[MAPK signaling pathway, Ras signaling pathway, Rap1 signaling pathway, Chemokine signaling pathway, Endocytosis, Axon guidance, VEGF signaling pathway, Focal adhesion, Adherens junction, Tight junction, T cell receptor signaling pathway, Fc gamma R-mediated phagocytosis, Leukocyte transendothelial migration, Neurotrophin signaling pathway, Regulation of actin cytoskeleton, GnRH signaling pathway, Non-alcoholic fatty liver disease, AGE-RAGE signaling pathway in diabetic complications, Bacterial invasion of epithelial cells, Salmonella infection, Yersinia infection, Human papillomavirus infection, Pathways in cancer, Viral carcinogenesis, Proteoglycans in cancer, Renal cell carcinoma, Pancreatic cancer, Lipid and atherosclerosis]','[]','[]','04010 MAPK signaling pathway; 04014 Ras signaling pathway; 04015 Rap1 signaling pathway; 04370 VEGF signaling pathway; 04144 Endocytosis; 04510 Focal adhesion; 04520 Adherens junction; 04530 Tight junction; 04810 Regulation of actin cytoskeleton; 04660 T cell receptor signaling pathway; 04666 Fc gamma R-mediated phagocytosis; 04670 Leukocyte transendothelial migration; 04062 Chemokine signaling pathway; 04912 GnRH signaling pathway; 04722 Neurotrophin signaling pathway; 04360 Axon guidance; 05200 Pathways in cancer; 05205 Proteoglycans in cancer; 05203 Viral carcinogenesis; 05212 Pancreatic cancer; 05211 Renal cell carcinoma; 05165 Human papillomavirus infection; 05132 Salmonella infection; 05135 Yersinia infection; 05100 Bacterial invasion of epithelial cells; 05417 Lipid and atherosclerosis; 04932 Non-alcoholic fatty liver disease; 04933 AGE-RAGE signaling pathway in diabetic complications; 04131 Membrane trafficking [BR:mmu04131]; 03036 Chromosome and associated proteins [BR:mmu03036]; 04147 Exosome [BR:mmu04147]; 04031 GTP-binding proteins [BR:mmu04031]; Rho GTPases; GRAF1-dependent endocytosis; Ras GTPases; Other centriole associated proteins; Exosomal proteins of microglial cells; Exosomal proteins of breast milk; Rac/Cdc42 [OT]; ','[Pfam: Ras Roc Arf PET117 GTP_EFTU]'),('109305','mmu:109305','[Orai1, D730049H07Rik, Tmem142a, orai-1]','[(RefSeq) ORAI calcium release-activated calcium modulator 1]','[calcium release-activated calcium channel protein 1]','[Mus musculus (house mouse)]','[Calcium signaling pathway, cAMP signaling pathway, Platelet activation, Renin secretion, Aldosterone synthesis and secretion, Cortisol synthesis and secretion, Cushing syndrome, Primary immunodeficiency]','[]','[]','04020 Calcium signaling pathway; 04024 cAMP signaling pathway; 04611 Platelet activation; 04924 Renin secretion; 04925 Aldosterone synthesis and secretion; 04927 Cortisol synthesis and secretion; 05340 Primary immunodeficiency; 04934 Cushing syndrome; 04040 Ion channels [BR:mmu04040]; Calcium channel, calcium release-activated (Orai); ','[Pfam: Orai-1 DUF687 DUF4153]'),('20866','mmu:20866','[Stim1, SIM]','[(RefSeq) stromal interaction molecule 1]','[stromal interaction molecule 1]','[Mus musculus (house mouse)]','[Calcium signaling pathway, Platelet activation]','[]','[]','04020 Calcium signaling pathway; 04611 Platelet activation; 02000 Transporters [BR:mmu02000]; Pores ion channels [TC:1]; ','[Pfam: SOAR SAM_2 SAM_1 SAM_Ste50p]'),('116873','mmu:116873','[Stim2]','[(RefSeq) stromal interaction molecule 2]','[stromal interaction molecule 2]','[Mus musculus (house mouse)]','[Calcium signaling pathway]','[]','[]','04020 Calcium signaling pathway; 02000 Transporters [BR:mmu02000]; Pores ion channels [TC:1]; ','[Pfam: SOAR SAM_2 SAM_1 SAM_Ste50p TfuA DUF6572]'),('723893','mmu:723893','[Mir10a, Mirn10a, miR-10a, mmu-mir-10a]','[(RefSeq) microRNA 10a]','[microRNA 10a]','[Mus musculus (house mouse)]','[Proteoglycans in cancer, MicroRNAs in cancer, Fluid shear stress and atherosclerosis]','[]','[]','05206 MicroRNAs in cancer; 05205 Proteoglycans in cancer; 05418 Fluid shear stress and atherosclerosis; 03100 Non-coding RNAs [BR:mmu03100]; Micro RNA; ','[]'),('21938','mmu:21938','[Tnfrsf1b, CD120b, TNF-R-II, TNF-R2, TNF-R75, TNF-alphaR2, TNFBR, TNFR80, TNFRII, TNFalpha-R2, Tnfr-1, Tnfr2, p75]','[(RefSeq) tumor necrosis factor receptor superfamily, member 1b]','[tumor necrosis factor receptor superfamily member 1B]','[Mus musculus (house mouse)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, TNF signaling pathway, Adipocytokine signaling pathway, Amyotrophic lateral sclerosis, Pathways of neurodegeneration - multiple diseases, Human immunodeficiency virus 1 infection]','[]','[]','04668 TNF signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04920 Adipocytokine signaling pathway; 05170 Human immunodeficiency virus 1 infection; 05014 Amyotrophic lateral sclerosis; 05022 Pathways of neurodegeneration - multiple diseases; 04050 Cytokine receptors [BR:mmu04050]; 04090 CD molecules [BR:mmu04090]; TNF receptor superfamily; Proteins; ','[Pfam: TNFR_c6 Casein_kappa]'),('233081','mmu:233081','[Ffar1, Gpr40]','[(RefSeq) free fatty acid receptor 1]','[free fatty acid receptor 1]','[Mus musculus (house mouse)]','[Insulin secretion]','[]','[]','04911 Insulin secretion; 04030 G protein-coupled receptors [BR:mmu04030]; Free fatty acid; ','[Pfam: 7tm_1 Cytochrome_B]'),('227288','mmu:227288','[Cxcr1, Il8ra]','[(RefSeq) chemokine (C-X-C motif) receptor 1]','[C-X-C chemokine receptor type 1]','[Mus musculus (house mouse)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, Chemokine signaling pathway, Phospholipase D signaling pathway, Endocytosis]','[]','[]','04072 Phospholipase D signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04144 Endocytosis; 04062 Chemokine signaling pathway; 04030 G protein-coupled receptors [BR:mmu04030]; 04050 Cytokine receptors [BR:mmu04050]; 04090 CD molecules [BR:mmu04090]; CXC Chemokine; CXC Chemokine; Proteins; ','[Pfam: 7tm_1 7TM_GPCR_Srsx]'),('12765','mmu:12765','[Cxcr2, CD128, CDw128, Cmkar2, Gpcr16, IL-8Rh, IL-8rb, IL8RA, Il8rb, mIL-8RH]','[(RefSeq) chemokine (C-X-C motif) receptor 2]','[C-X-C chemokine receptor type 2]','[Mus musculus (house mouse)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, Chemokine signaling pathway, Phospholipase D signaling pathway, Endocytosis, Human cytomegalovirus infection]','[]','[]','04072 Phospholipase D signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04144 Endocytosis; 04062 Chemokine signaling pathway; 05163 Human cytomegalovirus infection; 04030 G protein-coupled receptors [BR:mmu04030]; 04050 Cytokine receptors [BR:mmu04050]; 04090 CD molecules [BR:mmu04090]; CXC Chemokine; CXC Chemokine; Proteins; ','[Pfam: 7tm_1]'),('12575','mmu:12575','[Cdkn1a, CAP20, CDKI, CIP1, Cdkn1, P21, SDI1, Waf1, mda6, p21Cip1, p21WAF]','[(RefSeq) cyclin-dependent kinase inhibitor 1A (P21)]','[cyclin-dependent kinase inhibitor 1A]','[Mus musculus (house mouse)]','[Endocrine resistance, Platinum drug resistance, ErbB signaling pathway, HIF-1 signaling pathway, FoxO signaling pathway, Cell cycle, p53 signaling pathway, PI3K-Akt signaling pathway, Cellular senescence, JAK-STAT signaling pathway, Oxytocin signaling pathway, Parathyroid hormone synthesis, secretion and action, Cushing syndrome, Hepatitis C, Hepatitis B, Human cytomegalovirus infection, Human papillomavirus infection, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Epstein-Barr virus infection, Pathways in cancer, Transcriptional misregulation in cancer, Viral carcinogenesis, Proteoglycans in cancer, MicroRNAs in cancer, Colorectal cancer, Renal cell carcinoma, Pancreatic cancer, Endometrial cancer, Glioma, Prostate cancer, Thyroid cancer, Basal cell carcinoma, Melanoma, Bladder cancer, Chronic myeloid leukemia, Small cell lung cancer, Non-small cell lung cancer, Breast cancer, Hepatocellular carcinoma, Gastric cancer]','[]','[]','04012 ErbB signaling pathway; 04630 JAK-STAT signaling pathway; 04066 HIF-1 signaling pathway; 04068 FoxO signaling pathway; 04151 PI3K-Akt signaling pathway; 04110 Cell cycle; 04115 p53 signaling pathway; 04218 Cellular senescence; 04921 Oxytocin signaling pathway; 04928 Parathyroid hormone synthesis, secretion and action; 05200 Pathways in cancer; 05202 Transcriptional misregulation in cancer; 05206 MicroRNAs in cancer; 05205 Proteoglycans in cancer; 05203 Viral carcinogenesis; 05210 Colorectal cancer; 05212 Pancreatic cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05214 Glioma; 05216 Thyroid cancer; 05220 Chronic myeloid leukemia; 05217 Basal cell carcinoma; 05218 Melanoma; 05211 Renal cell carcinoma; 05219 Bladder cancer; 05215 Prostate cancer; 05213 Endometrial cancer; 05224 Breast cancer; 05222 Small cell lung cancer; 05223 Non-small cell lung cancer; 05166 Human T-cell leukemia virus 1 infection; 05161 Hepatitis B; 05160 Hepatitis C; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05165 Human papillomavirus infection; 04934 Cushing syndrome; 01524 Platinum drug resistance; 01522 Endocrine resistance; ','[Pfam: CDI]'),('19225','mmu:19225','[Ptgs2, COX2, Cox-2, PES-2, PGHS-2, PHS_II, PHS-2, Pghs2, TIS10, gripghs]','[(RefSeq) prostaglandin-endoperoxide synthase 2]','[prostaglandin-endoperoxide synthase 2 [EC:1.14.99.1]]','[Mus musculus (house mouse)]','[Arachidonic acid metabolism, Metabolic pathways, NF-kappa B signaling pathway, VEGF signaling pathway, C-type lectin receptor signaling pathway, IL-17 signaling pathway, TNF signaling pathway, Retrograde endocannabinoid signaling, Serotonergic synapse, Ovarian steroidogenesis, Oxytocin signaling pathway, Regulation of lipolysis in adipocytes, Alzheimer disease, Pathways of neurodegeneration - multiple diseases, Leishmaniasis, Human cytomegalovirus infection, Human papillomavirus infection, Kaposi sarcoma-associated herpesvirus infection, Pathways in cancer, Chemical carcinogenesis - DNA adducts, MicroRNAs in cancer, Small cell lung cancer]','[]','[]','00590 Arachidonic acid metabolism; 04370 VEGF signaling pathway; 04064 NF-kappa B signaling pathway; 04668 TNF signaling pathway; 04625 C-type lectin receptor signaling pathway; 04657 IL-17 signaling pathway; 04923 Regulation of lipolysis in adipocytes; 04913 Ovarian steroidogenesis; 04921 Oxytocin signaling pathway; 04726 Serotonergic synapse; 04723 Retrograde endocannabinoid signaling; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05204 Chemical carcinogenesis - DNA adducts; 05222 Small cell lung cancer; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05165 Human papillomavirus infection; 05140 Leishmaniasis; 05010 Alzheimer disease; 05022 Pathways of neurodegeneration - multiple diseases; 1.14.99.1  prostaglandin-endoperoxide synthase; ','[Pfam: An_peroxidase EGF]'),('19219','mmu:19219','[Ptger4, EP4, Ptgerep4]','[(RefSeq) prostaglandin E receptor 4 (subtype EP4)]','[prostaglandin E receptor 4]','[Mus musculus (house mouse)]','[Neuroactive ligand-receptor interaction, Inflammatory mediator regulation of TRP channels, Renin secretion, Human cytomegalovirus infection, Human papillomavirus infection, Pathways in cancer]','[]','[]','04080 Neuroactive ligand-receptor interaction; 04924 Renin secretion; 04750 Inflammatory mediator regulation of TRP channels; 05200 Pathways in cancer; 05163 Human cytomegalovirus infection; 05165 Human papillomavirus infection; 04030 G protein-coupled receptors [BR:mmu04030]; Prostaglandin; ','[Pfam: 7tm_1 7TM_GPCR_Srx 7TM_GPCR_Srsx NdhL]'),('18669','mmu:18669','[Abcb1b, Abcb1, Mdr1, Mdr1b, Pgy-1, Pgy1, mdr]','[(RefSeq) ATP-binding cassette, sub-family B (MDR/TAP), member 1B]','[ATP-binding cassette, subfamily B (MDR/TAP), member 1 [EC:7.6.2.2]]','[Mus musculus (house mouse)]','[ABC transporters, Bile secretion, MicroRNAs in cancer, Gastric cancer]','[]','[]','02010 ABC transporters; 04976 Bile secretion; 05206 MicroRNAs in cancer; 05226 Gastric cancer; 02000 Transporters [BR:mmu02000]; 04147 Exosome [BR:mmu04147]; 04090 CD molecules [BR:mmu04090]; 7.6.2.2  ABC-type xenobiotic transporter; ABCB1, 4, 5 subgroups; Exosomal proteins of other body fluids (saliva and urine); Proteins; ','[Pfam: ABC_membrane ABC_tran SMC_N ABC_ATPase AAA_16 Zeta_toxin AAA_22 Rad17 G-alpha RsgA_GTPase AAA AAA_5 AAA_29 AAA_7 APS_kinase AAA_30 PRK AAA_15 DUF815 NTPase_1]'),('394436','mmu:394436','[Ugt1a1, Gnt1, UDPGT_1-1, UGT1A01, Udpgt-1a, UgtBr1]','[(RefSeq) UDP glucuronosyltransferase 1 family, polypeptide A1]','[glucuronosyltransferase [EC:2.4.1.17]]','[Mus musculus (house mouse)]','[Pentose and glucuronate interconversions, Ascorbate and aldarate metabolism, Steroid hormone biosynthesis, Retinol metabolism, Porphyrin metabolism, Metabolism of xenobiotics by cytochrome P450, Drug metabolism - cytochrome P450, Drug metabolism - other enzymes, Metabolic pathways, Biosynthesis of cofactors, Bile secretion, Chemical carcinogenesis - DNA adducts, Chemical carcinogenesis - receptor activation]','[]','[]','00040 Pentose and glucuronate interconversions; 00053 Ascorbate and aldarate metabolism; 00140 Steroid hormone biosynthesis; 00830 Retinol metabolism; 00860 Porphyrin metabolism; 00980 Metabolism of xenobiotics by cytochrome P450; 00982 Drug metabolism - cytochrome P450; 00983 Drug metabolism - other enzymes; 04976 Bile secretion; 05204 Chemical carcinogenesis - DNA adducts; 05207 Chemical carcinogenesis - receptor activation; 01003 Glycosyltransferases [BR:mmu01003]; 2.4.1.17  glucuronosyltransferase; Hydrophobic molecule; ','[Pfam: UDPGT Glyco_tran_28_C Glyco_transf_4 Pox_A14]'),('64564','rno:64564','[Bbox1]','[(RefSeq) gamma-butyrobetaine dioxygenase]','[gamma-butyrobetaine dioxygenase [EC:1.14.11.1]]','[Rattus norvegicus (rat)]','[Lysine degradation, Metabolic pathways]','[]','[]','00310 Lysine degradation; 1.14.11.1  gamma-butyrobetaine dioxygenase; ','[Pfam: TauD GBBH-like_N CsiD Raf1_N]'),('15368','mmu:15368','[Hmox1, D8Wsu38e, HO-1, HO1, Hemox, Hmox, Hsp32]','[(RefSeq) heme oxygenase 1]','[heme oxygenase 1 [EC:1.14.14.18]]','[Mus musculus (house mouse)]','[Porphyrin metabolism, Metabolic pathways, HIF-1 signaling pathway, Ferroptosis, Mineral absorption, Pathways in cancer, MicroRNAs in cancer, Chemical carcinogenesis - reactive oxygen species, Hepatocellular carcinoma, Fluid shear stress and atherosclerosis]','[]','[]','00860 Porphyrin metabolism; 04066 HIF-1 signaling pathway; 04216 Ferroptosis; 04978 Mineral absorption; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05208 Chemical carcinogenesis - reactive oxygen species; 05225 Hepatocellular carcinoma; 05418 Fluid shear stress and atherosclerosis; 1.14.14.18  heme oxygenase (biliverdin-producing); ','[Pfam: Heme_oxygenase]'),('14870','mmu:14870','[Gstp1, GstpiB]','[(RefSeq) glutathione S-transferase, pi 1]','[glutathione S-transferase P [EC:2.5.1.18]]','[Mus musculus (house mouse)]','[Glutathione metabolism, Metabolism of xenobiotics by cytochrome P450, Drug metabolism - cytochrome P450, Drug metabolism - other enzymes, Metabolic pathways, Platinum drug resistance, Pathways in cancer, Chemical carcinogenesis - DNA adducts, Prostate cancer, Hepatocellular carcinoma, Fluid shear stress and atherosclerosis]','[]','[]','00480 Glutathione metabolism; 00980 Metabolism of xenobiotics by cytochrome P450; 00982 Drug metabolism - cytochrome P450; 00983 Drug metabolism - other enzymes; 05200 Pathways in cancer; 05204 Chemical carcinogenesis - DNA adducts; 05225 Hepatocellular carcinoma; 05215 Prostate cancer; 05418 Fluid shear stress and atherosclerosis; 01524 Platinum drug resistance; 02000 Transporters [BR:mmu02000]; 2.5.1.18  glutathione transferase; Pores ion channels [TC:1]; ','[Pfam: GST_C_3 GST_C GST_N GST_C_2]'),('12322','mmu:12322','[Camk2a, CaMKII, R74975, mKIAA0968]','[(RefSeq) calcium/calmodulin-dependent protein kinase II alpha]','[calcium/calmodulin-dependent protein kinase (CaM kinase) II [EC:2.7.11.17]]','[Mus musculus (house mouse)]','[ErbB signaling pathway, Calcium signaling pathway, cAMP signaling pathway, HIF-1 signaling pathway, Oocyte meiosis, Necroptosis, Adrenergic signaling in cardiomyocytes, Wnt signaling pathway, Axon guidance, Circadian entrainment, Long-term potentiation, Neurotrophin signaling pathway, Cholinergic synapse, Dopaminergic synapse, Olfactory transduction, Inflammatory mediator regulation of TRP channels, Insulin secretion, GnRH signaling pathway, Melanogenesis, Oxytocin signaling pathway, Glucagon signaling pathway, Aldosterone synthesis and secretion, Cushing syndrome, Gastric acid secretion, Parkinson disease, Pathways of neurodegeneration - multiple diseases, Amphetamine addiction, Tuberculosis, Pathways in cancer, Proteoglycans in cancer, Glioma, Diabetic cardiomyopathy, Lipid and atherosclerosis]','[]','[]','04012 ErbB signaling pathway; 04310 Wnt signaling pathway; 04066 HIF-1 signaling pathway; 04020 Calcium signaling pathway; 04024 cAMP signaling pathway; 04114 Oocyte meiosis; 04217 Necroptosis; 04911 Insulin secretion; 04922 Glucagon signaling pathway; 04912 GnRH signaling pathway; 04921 Oxytocin signaling pathway; 04916 Melanogenesis; 04925 Aldosterone synthesis and secretion; 04261 Adrenergic signaling in cardiomyocytes; 04971 Gastric acid secretion; 04725 Cholinergic synapse; 04728 Dopaminergic synapse; 04720 Long-term potentiation; 04722 Neurotrophin signaling pathway; 04740 Olfactory transduction; 04750 Inflammatory mediator regulation of TRP channels; 04360 Axon guidance; 04713 Circadian entrainment; 05200 Pathways in cancer; 05205 Proteoglycans in cancer; 05214 Glioma; 05152 Tuberculosis; 05012 Parkinson disease; 05022 Pathways of neurodegeneration - multiple diseases; 05031 Amphetamine addiction; 05417 Lipid and atherosclerosis; 05415 Diabetic cardiomyopathy; 04934 Cushing syndrome; 01001 Protein kinases [BR:mmu01001]; 2.7.11.17  Ca2+/calmodulin-dependent protein kinase; CAMK2 family; ','[Pfam: Pkinase CaMKII_AD PK_Tyr_Ser-Thr DUF4440 SnoaL_3 APH Kdo Pkinase_fungal SnoaL_2 SnoaL_4]'),('12307','mmu:12307','[Calb1, Brain-2, CB, Calb, Calb-1]','[(RefSeq) calbindin 1]','[calbindin]','[Mus musculus (house mouse)]','[Endocrine and other factor-regulated calcium reabsorption]','[]','[]','04961 Endocrine and other factor-regulated calcium reabsorption; 04990 Domain-containing proteins not elsewhere classified [BR:mmu04990]; Other EF-hand domain-containing proteins; ','[Pfam: EF-hand_1 EF-hand_6 EF-hand_7 EF-hand_5 EF-hand_8]'),('56717','mmu:56717','[Mtor, 2610315D21Rik, AI327068, FRAP, FRAP2, Frap1, RAFT1, RAPT1, flat]','[(RefSeq) mechanistic target of rapamycin kinase]','[serine/threonine-protein kinase mTOR [EC:2.7.11.1]]','[Mus musculus (house mouse)]','[EGFR tyrosine kinase inhibitor resistance, Endocrine resistance, ErbB signaling pathway, HIF-1 signaling pathway, Phospholipase D signaling pathway, Autophagy - other, Autophagy - animal, mTOR signaling pathway, PI3K-Akt signaling pathway, AMPK signaling pathway, Longevity regulating pathway, Longevity regulating pathway - multiple species, Cellular senescence, Apelin signaling pathway, Neutrophil extracellular trap formation, JAK-STAT signaling pathway, Th17 cell differentiation, Thermogenesis, Insulin signaling pathway, Thyroid hormone signaling pathway, Adipocytokine signaling pathway, Type II diabetes mellitus, Insulin resistance, Growth hormone synthesis, secretion and action, Alzheimer disease, Amyotrophic lateral sclerosis, Huntington disease, Spinocerebellar ataxia, Pathways of neurodegeneration - multiple diseases, Human cytomegalovirus infection, Human papillomavirus infection, Kaposi sarcoma-associated herpesvirus infection, Herpes simplex virus 1 infection, Human immunodeficiency virus 1 infection, Pathways in cancer, Proteoglycans in cancer, MicroRNAs in cancer, Chemical carcinogenesis - receptor activation, Colorectal cancer, Pancreatic cancer, Glioma, Prostate cancer, Acute myeloid leukemia, Breast cancer, Hepatocellular carcinoma, Gastric cancer, Central carbon metabolism in cancer, Choline metabolism in cancer, PD-L1 expression and PD-1 checkpoint pathway in cancer, Diabetic cardiomyopathy]','[]','[]','04012 ErbB signaling pathway; 04371 Apelin signaling pathway; 04630 JAK-STAT signaling pathway; 04066 HIF-1 signaling pathway; 04072 Phospholipase D signaling pathway; 04151 PI3K-Akt signaling pathway; 04152 AMPK signaling pathway; 04150 mTOR signaling pathway; 04140 Autophagy - animal; 04136 Autophagy - other; 04218 Cellular senescence; 04613 Neutrophil extracellular trap formation; 04659 Th17 cell differentiation; 04910 Insulin signaling pathway; 04920 Adipocytokine signaling pathway; 04935 Growth hormone synthesis, secretion and action; 04919 Thyroid hormone signaling pathway; 04211 Longevity regulating pathway; 04213 Longevity regulating pathway - multiple species; 04714 Thermogenesis; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05205 Proteoglycans in cancer; 05207 Chemical carcinogenesis - receptor activation; 05230 Central carbon metabolism in cancer; 05231 Choline metabolism in cancer; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05210 Colorectal cancer; 05212 Pancreatic cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05214 Glioma; 05221 Acute myeloid leukemia; 05215 Prostate cancer; 05224 Breast cancer; 05170 Human immunodeficiency virus 1 infection; 05168 Herpes simplex virus 1 infection; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05165 Human papillomavirus infection; 05010 Alzheimer disease; 05014 Amyotrophic lateral sclerosis; 05016 Huntington disease; 05017 Spinocerebellar ataxia; 05022 Pathways of neurodegeneration - multiple diseases; 05415 Diabetic cardiomyopathy; 04930 Type II diabetes mellitus; 04931 Insulin resistance; 01521 EGFR tyrosine kinase inhibitor resistance; 01522 Endocrine resistance; 01001 Protein kinases [BR:mmu01001]; 04131 Membrane trafficking [BR:mmu04131]; 2.7.11.1  non-specific serine/threonine protein kinase; PIKK family [OT]; mTORC1 complex; mTORC2 complex; ','[Pfam: FAT PI3_PI4_kinase DUF3385 FRB_dom FATC HEAT_2 UME HEAT_EZ EPSP_synthase Nipped-B_C]'),('12142','mmu:12142','[Prdm1, Blimp-1, Blimp1, PRDI-BF1, ZNFPR1A1, b2b1765Clo]','[(RefSeq) PR domain containing 1, with ZNF domain]','[PR domain zinc finger protein 1 [EC:2.1.1.-]]','[Mus musculus (house mouse)]',NULL,'[]','[]','03000 Transcription factors [BR:mmu03000]; 2.1.1.-; Cys2His2 PRDM; ','[Pfam: zf-H2C2_2 zf-C2H2 SET zf-C2H2_4 zf-C2H2_6 zf-C2H2_jaz zf-C2HE zf-met]'),('11846','mmu:11846','[Arg1, AI, AI256583, Arg-1, PGIF]','[(RefSeq) arginase, liver]','[arginase [EC:3.5.3.1]]','[Mus musculus (house mouse)]','[Arginine biosynthesis, Arginine and proline metabolism, Metabolic pathways, Biosynthesis of amino acids, Amoebiasis]','[]','[]','00220 Arginine biosynthesis; 00330 Arginine and proline metabolism; 05146 Amoebiasis; 3.5.3.1  arginase; ','[Pfam: Arginase]'),('12367','mmu:12367','[Casp3, A830040C14Rik, AC-3, CASP-3, CC3, CPP-32, CPP32, Caspase-3, Lice, SCA-1, Yama, mldy]','[(RefSeq) caspase 3]','[caspase 3 [EC:3.4.22.56]]','[Mus musculus (house mouse)]','[Platinum drug resistance, MAPK signaling pathway, p53 signaling pathway, Apoptosis, Apoptosis - multiple species, Natural killer cell mediated cytotoxicity, IL-17 signaling pathway, TNF signaling pathway, Serotonergic synapse, Non-alcoholic fatty liver disease, AGE-RAGE signaling pathway in diabetic complications, Alcoholic liver disease, Alzheimer disease, Parkinson disease, Amyotrophic lateral sclerosis, Huntington disease, Prion disease, Pathways of neurodegeneration - multiple diseases, Salmonella infection, Pertussis, Legionellosis, Toxoplasmosis, Amoebiasis, Tuberculosis, Hepatitis C, Hepatitis B, Measles, Human cytomegalovirus infection, Influenza A, Human papillomavirus infection, Kaposi sarcoma-associated herpesvirus infection, Herpes simplex virus 1 infection, Epstein-Barr virus infection, Human immunodeficiency virus 1 infection, Pathways in cancer, Viral carcinogenesis, Proteoglycans in cancer, MicroRNAs in cancer, Colorectal cancer, Small cell lung cancer, Viral myocarditis, Lipid and atherosclerosis]','[]','[]','04010 MAPK signaling pathway; 04668 TNF signaling pathway; 04210 Apoptosis; 04215 Apoptosis - multiple species; 04115 p53 signaling pathway; 04650 Natural killer cell mediated cytotoxicity; 04657 IL-17 signaling pathway; 04726 Serotonergic synapse; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05205 Proteoglycans in cancer; 05203 Viral carcinogenesis; 05210 Colorectal cancer; 05222 Small cell lung cancer; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05160 Hepatitis C; 05164 Influenza A; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05165 Human papillomavirus infection; 05132 Salmonella infection; 05133 Pertussis; 05134 Legionellosis; 05152 Tuberculosis; 05146 Amoebiasis; 05145 Toxoplasmosis; 05010 Alzheimer disease; 05012 Parkinson disease; 05014 Amyotrophic lateral sclerosis; 05016 Huntington disease; 05020 Prion disease; 05022 Pathways of neurodegeneration - multiple diseases; 05417 Lipid and atherosclerosis; 05416 Viral myocarditis; 04936 Alcoholic liver disease; 04932 Non-alcoholic fatty liver disease; 04933 AGE-RAGE signaling pathway in diabetic complications; 01524 Platinum drug resistance; 01002 Peptidases and inhibitors [BR:mmu01002]; 3.4.22.56  caspase-3; Family C14: caspase family; ','[Pfam: Peptidase_C14 SICA_beta]'),('12064','mmu:12064','[Bdnf]','[(RefSeq) brain derived neurotrophic factor]','[brain-derived neurotrophic factor]','[Mus musculus (house mouse)]','[MAPK signaling pathway, Ras signaling pathway, cAMP signaling pathway, PI3K-Akt signaling pathway, Neurotrophin signaling pathway, Huntington disease, Pathways of neurodegeneration - multiple diseases, Cocaine addiction, Alcoholism]','[]','[]','04010 MAPK signaling pathway; 04014 Ras signaling pathway; 04024 cAMP signaling pathway; 04151 PI3K-Akt signaling pathway; 04722 Neurotrophin signaling pathway; 05016 Huntington disease; 05022 Pathways of neurodegeneration - multiple diseases; 05030 Cocaine addiction; 05034 Alcoholism; 04052 Cytokines and growth factors [BR:mmu04052]; Growth factors (RTK binding); ','[Pfam: NGF NTF3_N Spaetzle]'),('20655','mmu:20655','[Sod1, B430204E11Rik, Cu/Zn-SOD, CuZnSOD, Ipo-1, Ipo1, SODC, Sod-1]','[(RefSeq) superoxide dismutase 1, soluble]','[superoxide dismutase, Cu-Zn family [EC:1.15.1.1]]','[Mus musculus (house mouse)]','[Peroxisome, Longevity regulating pathway - multiple species, Parkinson disease, Amyotrophic lateral sclerosis, Huntington disease, Prion disease, Pathways of neurodegeneration - multiple diseases, Chemical carcinogenesis - reactive oxygen species]','[]','[]','04146 Peroxisome; 04213 Longevity regulating pathway - multiple species; 05208 Chemical carcinogenesis - reactive oxygen species; 05012 Parkinson disease; 05014 Amyotrophic lateral sclerosis; 05016 Huntington disease; 05020 Prion disease; 05022 Pathways of neurodegeneration - multiple diseases; 1.15.1.1  superoxide dismutase; ','[Pfam: Sod_Cu]'),('12359','mmu:12359','[Cat, 2210418N07, Cas-1, Cas1, Cs-1]','[(RefSeq) catalase]','[catalase [EC:1.11.1.6]]','[Mus musculus (house mouse)]','[Tryptophan metabolism, Glyoxylate and dicarboxylate metabolism, Metabolic pathways, Carbon metabolism, FoxO signaling pathway, Peroxisome, Longevity regulating pathway, Longevity regulating pathway - multiple species, Amyotrophic lateral sclerosis, Pathways of neurodegeneration - multiple diseases, Chemical carcinogenesis - reactive oxygen species]','[]','[]','00630 Glyoxylate and dicarboxylate metabolism; 00380 Tryptophan metabolism; 04068 FoxO signaling pathway; 04146 Peroxisome; 04211 Longevity regulating pathway; 04213 Longevity regulating pathway - multiple species; 05208 Chemical carcinogenesis - reactive oxygen species; 05014 Amyotrophic lateral sclerosis; 05022 Pathways of neurodegeneration - multiple diseases; 1.11.1.6  catalase; ','[Pfam: Catalase Catalase-rel]'),('16409','mmu:16409','[Itgam, CD11b/CD18, CR3, CR3A, Cd11b, F730045J24Rik, Ly-40, MAC1, Mac-1, Mac-1a]','[(RefSeq) integrin alpha M]','[integrin alpha M]','[Mus musculus (house mouse)]','[Rap1 signaling pathway, Phagosome, Cell adhesion molecules, Complement and coagulation cascades, Neutrophil extracellular trap formation, Hematopoietic cell lineage, Leukocyte transendothelial migration, Regulation of actin cytoskeleton, Pertussis, Legionellosis, Leishmaniasis, Amoebiasis, Staphylococcus aureus infection, Tuberculosis, Transcriptional misregulation in cancer, Acute myeloid leukemia]','[]','[]','04015 Rap1 signaling pathway; 04514 Cell adhesion molecules; 04145 Phagosome; 04810 Regulation of actin cytoskeleton; 04640 Hematopoietic cell lineage; 04610 Complement and coagulation cascades; 04613 Neutrophil extracellular trap formation; 04670 Leukocyte transendothelial migration; 05202 Transcriptional misregulation in cancer; 05221 Acute myeloid leukemia; 05133 Pertussis; 05134 Legionellosis; 05150 Staphylococcus aureus infection; 05152 Tuberculosis; 05146 Amoebiasis; 05140 Leishmaniasis; 04131 Membrane trafficking [BR:mmu04131]; 04147 Exosome [BR:mmu04147]; 04515 Cell adhesion molecules [BR:mmu04515]; 04090 CD molecules [BR:mmu04090]; 00536 Glycosaminoglycan binding proteins [BR:mmu00536]; Integrins (complement receptors); Exosomal proteins of haemopoietic cells  (B-cell, T-cell, DC-cell, reticulocyte, and mast cell); Integrin alpha subunits; Proteins; Adhesion molecules of cell surface; ','[Pfam: VWA Integrin_alpha2 FG-GAP VWA_2 FG-GAP_3 Integrin_alpha FG-GAP_2 DUF11 AbfS_sensor]'),('16191','mmu:16191','[Il5, Il-5]','[(RefSeq) interleukin 5]','[interleukin 5]','[Mus musculus (house mouse)]','[Cytokine-cytokine receptor interaction, JAK-STAT signaling pathway, Hematopoietic cell lineage, IL-17 signaling pathway, Th1 and Th2 cell differentiation, T cell receptor signaling pathway, Fc epsilon RI signaling pathway, Intestinal immune network for IgA production, Pathways in cancer, Asthma, Autoimmune thyroid disease, Inflammatory bowel disease, Allograft rejection]','[]','[]','04630 JAK-STAT signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04640 Hematopoietic cell lineage; 04660 T cell receptor signaling pathway; 04658 Th1 and Th2 cell differentiation; 04657 IL-17 signaling pathway; 04664 Fc epsilon RI signaling pathway; 04672 Intestinal immune network for IgA production; 05200 Pathways in cancer; 05310 Asthma; 05320 Autoimmune thyroid disease; 05321 Inflammatory bowel disease; 05330 Allograft rejection; 04052 Cytokines and growth factors [BR:mmu04052]; 00536 Glycosaminoglycan binding proteins [BR:mmu00536]; Interleukins; Cytokines; ','[Pfam: IL5]'),('21990','mmu:21990','[Tph1, Tph]','[(RefSeq) tryptophan hydroxylase 1]','[tryptophan 5-monooxygenase [EC:1.14.16.4]]','[Mus musculus (house mouse)]','[Tryptophan metabolism, Folate biosynthesis, Metabolic pathways, Serotonergic synapse]','[]','[]','00380 Tryptophan metabolism; 00790 Folate biosynthesis; 04726 Serotonergic synapse; 1.14.16.4  tryptophan 5-monooxygenase; ','[Pfam: Biopterin_H ACT ACT_4]'),('15930','mmu:15930','[Ido1, Ido, Indo]','[(RefSeq) indoleamine 2,3-dioxygenase 1]','[indoleamine 2,3-dioxygenase [EC:1.13.11.52]]','[Mus musculus (house mouse)]','[Tryptophan metabolism, Metabolic pathways, Biosynthesis of cofactors, African trypanosomiasis]','[]','[]','00380 Tryptophan metabolism; 05143 African trypanosomiasis; 1.13.11.52  indoleamine 2,3-dioxygenase; ','[Pfam: IDO DUF1864 MecA]'),('12894','mmu:12894','[Cpt1a, C730027G07, CPTI, Cpt1]','[(RefSeq) carnitine palmitoyltransferase 1a, liver]','[carnitine O-palmitoyltransferase 1, liver isoform [EC:2.3.1.21]]','[Mus musculus (house mouse)]','[Fatty acid degradation, Fatty acid metabolism, PPAR signaling pathway, AMPK signaling pathway, Thermogenesis, Adipocytokine signaling pathway, Glucagon signaling pathway, Insulin resistance, Alcoholic liver disease]','[]','[]','00071 Fatty acid degradation; 04152 AMPK signaling pathway; 04922 Glucagon signaling pathway; 04920 Adipocytokine signaling pathway; 03320 PPAR signaling pathway; 04714 Thermogenesis; 04936 Alcoholic liver disease; 04931 Insulin resistance; 03029 Mitochondrial biogenesis [BR:mmu03029]; 2.3.1.21  carnitine O-palmitoyltransferase; Other regulator of mitochondrial biogenesis; ','[Pfam: Carn_acyltransf CPT_N]'),('22227','mmu:22227','[Ucp1, AI385626, Slc25a7, Ucp]','[(RefSeq) uncoupling protein 1 (mitochondrial, proton carrier)]','[solute carrier family 25 (mitochondrial uncoupling protein), member 7]','[Mus musculus (house mouse)]','[PPAR signaling pathway, Apelin signaling pathway, Thermogenesis, Huntington disease]','[]','[]','04371 Apelin signaling pathway; 03320 PPAR signaling pathway; 04714 Thermogenesis; 05016 Huntington disease; 02000 Transporters [BR:mmu02000]; SLC25: Mitochondrial carrier; ','[Pfam: Mito_carr]'),('246779','mmu:246779','[Il27, IL-27, IL-27-A, IL-27p28, IL27-A, Il30, p28]','[(RefSeq) interleukin 27]','[interleukin 27 subunit alpha]','[Mus musculus (house mouse)]','[Cytokine-cytokine receptor interaction, Th17 cell differentiation]','[]','[]','04060 Cytokine-cytokine receptor interaction; 04659 Th17 cell differentiation; 04052 Cytokines and growth factors [BR:mmu04052]; Interleukins; ','[Pfam: XK-related]'),('60533','mmu:60533','[Cd274, A530045L16Rik, B7h1, Pdcd1l1, Pdcd1lg1, Pdl1]','[(RefSeq) CD274 antigen]','[programmed cell death 1 ligand 1]','[Mus musculus (house mouse)]','[Cell adhesion molecules, PD-L1 expression and PD-1 checkpoint pathway in cancer]','[]','[]','04514 Cell adhesion molecules; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 04090 CD molecules [BR:mmu04090]; Proteins; ','[Pfam: Ig_3 V-set I-set Ig_2 ig C2-set_2 C1-set]'),('12495','mmu:12495','[Entpd1, 2610206B08Rik, AA408691, Cd39, E130009M23Rik, NTPDase-1]','[(RefSeq) ectonucleoside triphosphate diphosphohydrolase 1]','[apyrase [EC:3.6.1.5]]','[Mus musculus (house mouse)]','[Purine metabolism, Pyrimidine metabolism, Metabolic pathways, Nucleotide metabolism, Epstein-Barr virus infection]','[]','[]','00230 Purine metabolism; 00240 Pyrimidine metabolism; 05169 Epstein-Barr virus infection; 04090 CD molecules [BR:mmu04090]; 3.6.1.5  apyrase; Proteins; ','[Pfam: GDA1_CD39]'),('21873','mmu:21873','[Tjp2, ZO-2]','[(RefSeq) tight junction protein 2]','[tight junction protein 2]','[Mus musculus (house mouse)]','[Tight junction]','[]','[]','04530 Tight junction; ','[Pfam: PDZ PDZ_6 PDZ_2 Peptidase_M50 Guanylate_kin SH3_2 SH3_1]'),('16456','mmu:16456','[F11r, 9130004G24, AA638916, ESTM33, JAM, JAM-1, JAM-A, Jcam, Jcam1, Ly106]','[(RefSeq) F11 receptor]','[junctional adhesion molecule 1]','[Mus musculus (house mouse)]','[Cell adhesion molecules, Tight junction, Leukocyte transendothelial migration]','[]','[]','04514 Cell adhesion molecules; 04530 Tight junction; 04670 Leukocyte transendothelial migration; 04090 CD molecules [BR:mmu04090]; Proteins; ','[Pfam: Ig_3 ig I-set Ig_2 V-set C1-set C2-set_2 Adeno_E3_CR1 Ig_C17orf99 Ig_6 Invasin_D3]'),('17874','mmu:17874','[Myd88]','[(RefSeq) myeloid differentiation primary response gene 88]','[myeloid differentiation primary response protein MyD88]','[Mus musculus (house mouse)]','[MAPK signaling pathway, NF-kappa B signaling pathway, Toll-like receptor signaling pathway, NOD-like receptor signaling pathway, Alcoholic liver disease, Salmonella infection, Pertussis, Legionellosis, Yersinia infection, Leishmaniasis, Chagas disease, African trypanosomiasis, Malaria, Toxoplasmosis, Tuberculosis, Hepatitis B, Measles, Influenza A, Herpes simplex virus 1 infection, Epstein-Barr virus infection, Human immunodeficiency virus 1 infection, Coronavirus disease - COVID-19, PD-L1 expression and PD-1 checkpoint pathway in cancer, Lipid and atherosclerosis]','[]','[]','04010 MAPK signaling pathway; 04064 NF-kappa B signaling pathway; 04620 Toll-like receptor signaling pathway; 04621 NOD-like receptor signaling pathway; 05235 PD-L1 expression and PD-1 checkpoint pathway in cancer; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05171 Coronavirus disease - COVID-19; 05164 Influenza A; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05169 Epstein-Barr virus infection; 05132 Salmonella infection; 05135 Yersinia infection; 05133 Pertussis; 05134 Legionellosis; 05152 Tuberculosis; 05144 Malaria; 05145 Toxoplasmosis; 05140 Leishmaniasis; 05142 Chagas disease; 05143 African trypanosomiasis; 05417 Lipid and atherosclerosis; 04936 Alcoholic liver disease; ','[Pfam: Death TIR TIR_2 LPD11]'),('14598','mmu:14598','[Ggt1, CD224, GGT, GGT_1, GGT-1, Ggtp, dwg]','[(RefSeq) gamma-glutamyltransferase 1]','[gamma-glutamyltranspeptidase / glutathione hydrolase / leukotriene-C4 hydrolase [EC:2.3.2.2 3.4.19.13 3.4.19.14]]','[Mus musculus (house mouse)]','[Taurine and hypotaurine metabolism, Glutathione metabolism, Arachidonic acid metabolism, Metabolic pathways]','[]','[]','00590 Arachidonic acid metabolism; 00430 Taurine and hypotaurine metabolism; 00480 Glutathione metabolism; 01002 Peptidases and inhibitors [BR:mmu01002]; 04090 CD molecules [BR:mmu04090]; 2.3.2.2  gamma-glutamyltransferase; 3.4.19.13  glutathione gamma-glutamate hydrolase; 3.4.19.14  leukotriene-C4 hydrolase; Family T3: gamma-glutamyltransferase family; Proteins; ','[Pfam: G_glu_transpept DUF6568 Thiamine_BP]'),('235504','mmu:235504','[Slc17a5, 4631416G20Rik, 4732491M05, AST, ISSD, NSD, SD, SIALIN, SIASD, SLD]','[(RefSeq) solute carrier family 17 (anion/sugar transporter), member 5]','[MFS transporter, ACS family, solute carrier family 17 (sodium-dependent inorganic phosphate cotransporter), member 5]','[Mus musculus (house mouse)]','[Lysosome]','[]','[]','04142 Lysosome; 02000 Transporters [BR:mmu02000]; SLC17: Vesicular glutamate transporter; Anion:cation symporter (ACS) family [TC:2.A.1.14]; ','[Pfam: MFS_1]'),('76282','mmu:76282','[Gpt, 1300007J06Rik, 2310022B03Rik, ALT, ALT1, Gpt-1, Gpt1]','[(RefSeq) glutamic pyruvic transaminase, soluble]','[alanine transaminase [EC:2.6.1.2]]','[Mus musculus (house mouse)]','[Arginine biosynthesis, Alanine, aspartate and glutamate metabolism, Metabolic pathways, Carbon metabolism, 2-Oxocarboxylic acid metabolism, Biosynthesis of amino acids]','[]','[]','00250 Alanine, aspartate and glutamate metabolism; 00220 Arginine biosynthesis; 01007 Amino acid related enzymes [BR:mmu01007]; 2.6.1.2  alanine transaminase; Class I; ','[Pfam: Aminotran_1_2]'),('12491','mmu:12491','[Cd36, FAT, GPIV, Scarb3]','[(RefSeq) CD36 molecule]','[CD36 antigen]','[Mus musculus (house mouse)]','[PPAR signaling pathway, Phagosome, AMPK signaling pathway, ECM-receptor interaction, Hematopoietic cell lineage, Adipocytokine signaling pathway, Insulin resistance, Fat digestion and absorption, Cholesterol metabolism, Malaria, Diabetic cardiomyopathy, Lipid and atherosclerosis]','[]','[]','04152 AMPK signaling pathway; 04512 ECM-receptor interaction; 04145 Phagosome; 04640 Hematopoietic cell lineage; 04920 Adipocytokine signaling pathway; 03320 PPAR signaling pathway; 04975 Fat digestion and absorption; 04979 Cholesterol metabolism; 05144 Malaria; 05417 Lipid and atherosclerosis; 05415 Diabetic cardiomyopathy; 04931 Insulin resistance; 04131 Membrane trafficking [BR:mmu04131]; 04147 Exosome [BR:mmu04147]; 04090 CD molecules [BR:mmu04090]; Scavenger receptors; Exosomal proteins of breast milk; Proteins; ','[Pfam: CD36]'),('25598','rno:25598','[Fabp2]','[(RefSeq) fatty acid-binding protein, intestinal]','[fatty acid-binding protein 2, intestinal]','[Rattus norvegicus (rat)]','[PPAR signaling pathway, Fat digestion and absorption]','[]','[]','03320 PPAR signaling pathway; 04975 Fat digestion and absorption; ','[Pfam: Lipocalin Lipocalin_7 Lipocalin_4]'),('12768','mmu:12768','[Ccr1, Cmkbr1, Mip-1a-R]','[(RefSeq) chemokine (C-C motif) receptor 1]','[C-C chemokine receptor type 1]','[Mus musculus (house mouse)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, Chemokine signaling pathway, Human cytomegalovirus infection, Kaposi sarcoma-associated herpesvirus infection]','[]','[]','04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04062 Chemokine signaling pathway; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 04030 G protein-coupled receptors [BR:mmu04030]; 04050 Cytokine receptors [BR:mmu04050]; 04090 CD molecules [BR:mmu04090]; CC Chemokine; CC Chemokine; Proteins; ','[Pfam: 7tm_1 7TM_GPCR_Srsx DUF1240 7TM_GPCR_Srx]'),('20303','mmu:20303','[Ccl4, AT744.1, Act-2, MIP-1B, Mip1b, Scya4]','[(RefSeq) chemokine (C-C motif) ligand 4]','[C-C motif chemokine 4]','[Mus musculus (house mouse)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, Chemokine signaling pathway, NF-kappa B signaling pathway, Toll-like receptor signaling pathway, Cytosolic DNA-sensing pathway, Human cytomegalovirus infection]','[]','[]','04064 NF-kappa B signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04620 Toll-like receptor signaling pathway; 04623 Cytosolic DNA-sensing pathway; 04062 Chemokine signaling pathway; 05163 Human cytomegalovirus infection; 04052 Cytokines and growth factors [BR:mmu04052]; Chemokines; ','[Pfam: IL8]'),('12043','mmu:12043','[Bcl2, AW986256, Bcl-2, C430015F12Rik, D630044D05Rik, D830018M01Rik]','[(RefSeq) B cell leukemia/lymphoma 2]','[apoptosis regulator Bcl-2]','[Mus musculus (house mouse)]','[EGFR tyrosine kinase inhibitor resistance, Endocrine resistance, Platinum drug resistance, NF-kappa B signaling pathway, HIF-1 signaling pathway, Sphingolipid signaling pathway, p53 signaling pathway, Autophagy - animal, Protein processing in endoplasmic reticulum, PI3K-Akt signaling pathway, Apoptosis, Apoptosis - multiple species, Necroptosis, Adrenergic signaling in cardiomyocytes, Hedgehog signaling pathway, Focal adhesion, NOD-like receptor signaling pathway, JAK-STAT signaling pathway, Neurotrophin signaling pathway, Cholinergic synapse, Estrogen signaling pathway, Parathyroid hormone synthesis, secretion and action, AGE-RAGE signaling pathway in diabetic complications, Amyotrophic lateral sclerosis, Pathways of neurodegeneration - multiple diseases, Salmonella infection, Toxoplasmosis, Tuberculosis, Hepatitis B, Measles, Herpes simplex virus 1 infection, Epstein-Barr virus infection, Human immunodeficiency virus 1 infection, Pathways in cancer, MicroRNAs in cancer, Chemical carcinogenesis - receptor activation, Colorectal cancer, Prostate cancer, Small cell lung cancer, Gastric cancer, Lipid and atherosclerosis, Fluid shear stress and atherosclerosis]','[]','[]','04141 Protein processing in endoplasmic reticulum; 04340 Hedgehog signaling pathway; 04630 JAK-STAT signaling pathway; 04064 NF-kappa B signaling pathway; 04066 HIF-1 signaling pathway; 04071 Sphingolipid signaling pathway; 04151 PI3K-Akt signaling pathway; 04140 Autophagy - animal; 04210 Apoptosis; 04215 Apoptosis - multiple species; 04217 Necroptosis; 04115 p53 signaling pathway; 04510 Focal adhesion; 04621 NOD-like receptor signaling pathway; 04915 Estrogen signaling pathway; 04928 Parathyroid hormone synthesis, secretion and action; 04261 Adrenergic signaling in cardiomyocytes; 04725 Cholinergic synapse; 04722 Neurotrophin signaling pathway; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05207 Chemical carcinogenesis - receptor activation; 05210 Colorectal cancer; 05226 Gastric cancer; 05215 Prostate cancer; 05222 Small cell lung cancer; 05170 Human immunodeficiency virus 1 infection; 05161 Hepatitis B; 05162 Measles; 05168 Herpes simplex virus 1 infection; 05169 Epstein-Barr virus infection; 05132 Salmonella infection; 05152 Tuberculosis; 05145 Toxoplasmosis; 05014 Amyotrophic lateral sclerosis; 05022 Pathways of neurodegeneration - multiple diseases; 05417 Lipid and atherosclerosis; 05418 Fluid shear stress and atherosclerosis; 04933 AGE-RAGE signaling pathway in diabetic complications; 01521 EGFR tyrosine kinase inhibitor resistance; 01524 Platinum drug resistance; 01522 Endocrine resistance; 01009 Protein phosphatases and associated proteins [BR:mmu01009]; PP1-interacting proteins (PIPs); ','[Pfam: Bcl-2 BH4 Bcl-2_3]'),('21933','mmu:21933','[Tnfrsf10b, DR5, KILLER, Ly98, MK, TRAILR2, TRICK2A, TRICK2B, TRICKB]','[(RefSeq) tumor necrosis factor receptor superfamily, member 10b]','[tumor necrosis factor receptor superfamily member 10A/B]','[Mus musculus (house mouse)]','[Cytokine-cytokine receptor interaction, Viral protein interaction with cytokine and cytokine receptor, p53 signaling pathway, Apoptosis, Necroptosis, Natural killer cell mediated cytotoxicity, Salmonella infection, Influenza A, Lipid and atherosclerosis]','[]','[]','04060 Cytokine-cytokine receptor interaction; 04061 Viral protein interaction with cytokine and cytokine receptor; 04210 Apoptosis; 04217 Necroptosis; 04115 p53 signaling pathway; 04650 Natural killer cell mediated cytotoxicity; 05164 Influenza A; 05132 Salmonella infection; 05417 Lipid and atherosclerosis; 04050 Cytokine receptors [BR:mmu04050]; 04090 CD molecules [BR:mmu04090]; TNF receptor superfamily; Proteins; ','[Pfam: Death TNFR_c6]'),('12801','mmu:12801','[Cnr1, CB-R, CB1, CB1A, CB1B, CB1R]','[(RefSeq) cannabinoid receptor 1 (brain)]','[cannabinoid receptor 1]','[Mus musculus (house mouse)]','[Rap1 signaling pathway, Neuroactive ligand-receptor interaction, Thermogenesis, Retrograde endocannabinoid signaling]','[]','[]','04015 Rap1 signaling pathway; 04080 Neuroactive ligand-receptor interaction; 04723 Retrograde endocannabinoid signaling; 04714 Thermogenesis; 04030 G protein-coupled receptors [BR:mmu04030]; Cannabinoid; ','[Pfam: 7tm_1 7TM_GPCR_Srsx 7tm_4 7TM_GPCR_Srx]'),('57314','mmu:57314','[Nelfcd, 2410003I03Rik, C77797, Th1, Th1l]','[(RefSeq) negative elongation factor complex member C/D, Th1l]','[negative elongation factor C/D]','[Mus musculus (house mouse)]','[Viral life cycle - HIV-1]','[]','[]','03250 Viral life cycle - HIV-1; 03021 Transcription machinery [BR:mmu03021]; NELF complex; ','[Pfam: TH1]'),('105243','mmu:105243','[Slc9a3, 9030624O13Rik, AI930210, NHE-3, NHE3]','[(RefSeq) solute carrier family 9 (sodium/hydrogen exchanger), member 3]','[solute carrier family 9 (sodium/hydrogen exchanger), member 3]','[Mus musculus (house mouse)]','[Proximal tubule bicarbonate reclamation, Protein digestion and absorption, Bile secretion, Mineral absorption]','[]','[]','04976 Bile secretion; 04974 Protein digestion and absorption; 04978 Mineral absorption; 04964 Proximal tubule bicarbonate reclamation; 02000 Transporters [BR:mmu02000]; SLC9: Na+/H+ exchanger; ','[Pfam: Na_H_Exchanger NEXCaM_BD]'),('13487','mmu:13487','[Slc26a3, 9030623B18Rik, 9130013M11Rik, AV376035, Dra]','[(RefSeq) solute carrier family 26, member 3]','[solute carrier family 26 (chloride anion exchanger), member 3]','[Mus musculus (house mouse)]','[Pancreatic secretion, Mineral absorption]','[]','[]','04972 Pancreatic secretion; 04978 Mineral absorption; 02000 Transporters [BR:mmu02000]; SLC26: Multifunctional anion exchanger; ','[Pfam: Sulfate_transp STAS]'),('12304','mmu:12304','[Pdia4, AI987846, Cai, ERp-72, Erp72]','[(RefSeq) protein disulfide isomerase associated 4]','[protein disulfide-isomerase A4 [EC:5.3.4.1]]','[Mus musculus (house mouse)]','[Protein processing in endoplasmic reticulum, Thyroid hormone synthesis]','[]','[]','04141 Protein processing in endoplasmic reticulum; 04918 Thyroid hormone synthesis; 03110 Chaperones and folding catalysts [BR:mmu03110]; 04131 Membrane trafficking [BR:mmu04131]; 5.3.4.1  protein disulfide-isomerase; Protein disulfide isomerase; PDI family proteins; ','[Pfam: Thioredoxin Thioredoxin_6 OST3_OST6 Thioredoxin_2 Thioredoxin_8 Thioredoxin_7 Calsequestrin AhpC-TSA TraF Ribosomal_60s RPA43_OB PMC2NT]'),('12349','mmu:12349','[Car2, AI131712, CAII, Ca2, Car-2, Ltw-5, Lvtw-5]','[(RefSeq) carbonic anhydrase 2]','[carbonic anhydrase 2 [EC:4.2.1.1]]','[Mus musculus (house mouse)]','[Nitrogen metabolism, Metabolic pathways, Proximal tubule bicarbonate reclamation, Collecting duct acid secretion, Gastric acid secretion, Pancreatic secretion, Bile secretion]','[]','[]','00910 Nitrogen metabolism; 04971 Gastric acid secretion; 04972 Pancreatic secretion; 04976 Bile secretion; 04964 Proximal tubule bicarbonate reclamation; 04966 Collecting duct acid secretion; 04147 Exosome [BR:mmu04147]; 4.2.1.1  carbonic anhydrase; Exosomal proteins of other body fluids (saliva and urine); ','[Pfam: Carb_anhydrase]'),('59020','mmu:59020','[Pdzk1, 1700023D20Rik, 2610507N21Rik, 4921513F16Rik, AI267131, AI314638, AL022680, D3Ertd537e, NHERF-3, Pdzd1, mPDZK1, naPi-Cap1]','[(RefSeq) PDZ domain containing 1]','[DZ domain-containing protein 1]','[Mus musculus (house mouse)]',NULL,'[]','[]','04990 Domain-containing proteins not elsewhere classified [BR:mmu04990]; Other PDZ domain-containing proteins; ','[Pfam: PDZ PDZ_6 PDZ_2 Peptidase_M50 GRASP55_65 Tricorn_PDZ DUF6288]'),('12006','mmu:12006','[Axin2, Axi1, Axil, Conductin]','[(RefSeq) axin 2]','[axin 2]','[Mus musculus (house mouse)]','[Wnt signaling pathway, Hippo signaling pathway, Signaling pathways regulating pluripotency of stem cells, Cushing syndrome, Alzheimer disease, Pathways of neurodegeneration - multiple diseases, Human papillomavirus infection, Pathways in cancer, Colorectal cancer, Endometrial cancer, Basal cell carcinoma, Breast cancer, Hepatocellular carcinoma, Gastric cancer]','[]','[]','04310 Wnt signaling pathway; 04390 Hippo signaling pathway; 04550 Signaling pathways regulating pluripotency of stem cells; 05200 Pathways in cancer; 05210 Colorectal cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05217 Basal cell carcinoma; 05213 Endometrial cancer; 05224 Breast cancer; 05165 Human papillomavirus infection; 05010 Alzheimer disease; 05022 Pathways of neurodegeneration - multiple diseases; 04934 Cushing syndrome; 03036 Chromosome and associated proteins [BR:mmu03036]; Other centriole associated proteins; ','[Pfam: RGS DIX AXIN1_TNKS_BD Axin_b-cat_bind]'),('12505','mmu:12505','[Cd44, AU023126, AW121933, AW146109, HERMES, Ly-24, Pgp-1]','[(RefSeq) CD44 antigen]','[CD44 antigen]','[Mus musculus (house mouse)]','[ECM-receptor interaction, Hematopoietic cell lineage, Epstein-Barr virus infection, Proteoglycans in cancer, MicroRNAs in cancer]','[]','[]','04512 ECM-receptor interaction; 04640 Hematopoietic cell lineage; 05206 MicroRNAs in cancer; 05205 Proteoglycans in cancer; 05169 Epstein-Barr virus infection; 04147 Exosome [BR:mmu04147]; 04090 CD molecules [BR:mmu04090]; 00535 Proteoglycans [BR:mmu00535]; 00536 Glycosaminoglycan binding proteins [BR:mmu00536]; Exosomal proteins of haemopoietic cells  (B-cell, T-cell, DC-cell, reticulocyte, and mast cell); Exosomal proteins of other body fluids (saliva and urine); Exosomal proteins of colorectal cancer cells; Exosomal proteins of bladder cancer cells; Proteins; Others; Cell surface receptors; ','[Pfam: Xlink]'),('13844','mmu:13844','[Ephb2, Cek5, Drt, ETECK, Erk, Hek5, Nuk, Prkm5, Qek5, Sek3, Tyro5]','[(RefSeq) Eph receptor B2]','[Eph receptor B2 [EC:2.7.10.1]]','[Mus musculus (house mouse)]','[Axon guidance]','[]','[]','04360 Axon guidance; 01001 Protein kinases [BR:mmu01001]; 2.7.10.1  receptor protein-tyrosine kinase; EPH family [OT]; ','[Pfam: PK_Tyr_Ser-Thr Ephrin_lbd Pkinase fn3 EphA2_TM SAM_1 SAM_2 Ephrin_rec_like Pur_ac_phosph_N ABC1 SusE cEGF Kdo Haspin_kinase]'),('12443','mmu:12443','[Ccnd1, AI327039, CycD1, Cyl-1, PRAD1, bcl-1, cD1]','[(RefSeq) cyclin D1]','[G1/S-specific cyclin-D1]','[Mus musculus (house mouse)]','[Endocrine resistance, FoxO signaling pathway, Cell cycle, p53 signaling pathway, PI3K-Akt signaling pathway, AMPK signaling pathway, Cellular senescence, Wnt signaling pathway, Hedgehog signaling pathway, Apelin signaling pathway, Hippo signaling pathway, Focal adhesion, Tight junction, JAK-STAT signaling pathway, Prolactin signaling pathway, Thyroid hormone signaling pathway, Oxytocin signaling pathway, AGE-RAGE signaling pathway in diabetic complications, Cushing syndrome, Alcoholic liver disease, Hepatitis C, Measles, Human cytomegalovirus infection, Human papillomavirus infection, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Epstein-Barr virus infection, Pathways in cancer, Viral carcinogenesis, Proteoglycans in cancer, MicroRNAs in cancer, Chemical carcinogenesis - receptor activation, Colorectal cancer, Pancreatic cancer, Endometrial cancer, Glioma, Prostate cancer, Thyroid cancer, Melanoma, Bladder cancer, Chronic myeloid leukemia, Acute myeloid leukemia, Small cell lung cancer, Non-small cell lung cancer, Breast cancer, Hepatocellular carcinoma, Gastric cancer, Viral myocarditis]','[]','[]','04310 Wnt signaling pathway; 04340 Hedgehog signaling pathway; 04390 Hippo signaling pathway; 04371 Apelin signaling pathway; 04630 JAK-STAT signaling pathway; 04068 FoxO signaling pathway; 04151 PI3K-Akt signaling pathway; 04152 AMPK signaling pathway; 04110 Cell cycle; 04115 p53 signaling pathway; 04218 Cellular senescence; 04510 Focal adhesion; 04530 Tight junction; 04917 Prolactin signaling pathway; 04921 Oxytocin signaling pathway; 04919 Thyroid hormone signaling pathway; 05200 Pathways in cancer; 05206 MicroRNAs in cancer; 05205 Proteoglycans in cancer; 05207 Chemical carcinogenesis - receptor activation; 05203 Viral carcinogenesis; 05210 Colorectal cancer; 05212 Pancreatic cancer; 05225 Hepatocellular carcinoma; 05226 Gastric cancer; 05214 Glioma; 05216 Thyroid cancer; 05221 Acute myeloid leukemia; 05220 Chronic myeloid leukemia; 05218 Melanoma; 05219 Bladder cancer; 05215 Prostate cancer; 05213 Endometrial cancer; 05224 Breast cancer; 05222 Small cell lung cancer; 05223 Non-small cell lung cancer; 05166 Human T-cell leukemia virus 1 infection; 05160 Hepatitis C; 05162 Measles; 05163 Human cytomegalovirus infection; 05167 Kaposi sarcoma-associated herpesvirus infection; 05169 Epstein-Barr virus infection; 05165 Human papillomavirus infection; 05416 Viral myocarditis; 04936 Alcoholic liver disease; 04933 AGE-RAGE signaling pathway in diabetic complications; 04934 Cushing syndrome; 01522 Endocrine resistance; ','[Pfam: Cyclin_N Cyclin_C]'),('14810','mmu:14810','[Grin1, GluN1, GluRdelta1, GluRzeta1, M100174, NMD-R1, NMDAR1, NR1, Nmdar, Rgsc174]','[(RefSeq) glutamate receptor, ionotropic, NMDA1 (zeta 1)]','[glutamate receptor ionotropic, NMDA 1]','[Mus musculus (house mouse)]','[Ras signaling pathway, Rap1 signaling pathway, Calcium signaling pathway, cAMP signaling pathway, Neuroactive ligand-receptor interaction, Circadian entrainment, Long-term potentiation, Glutamatergic synapse, Alzheimer disease, Amyotrophic lateral sclerosis, Huntington disease, Spinocerebellar ataxia, Prion disease, Pathways of neurodegeneration - multiple diseases, Cocaine addiction, Amphetamine addiction, Nicotine addiction, Alcoholism]','[]','[]','04014 Ras signaling pathway; 04015 Rap1 signaling pathway; 04020 Calcium signaling pathway; 04024 cAMP signaling pathway; 04080 Neuroactive ligand-receptor interaction; 04724 Glutamatergic synapse; 04720 Long-term potentiation; 04713 Circadian entrainment; 05010 Alzheimer disease; 05014 Amyotrophic lateral sclerosis; 05016 Huntington disease; 05017 Spinocerebellar ataxia; 05020 Prion disease; 05022 Pathways of neurodegeneration - multiple diseases; 05030 Cocaine addiction; 05031 Amphetamine addiction; 05033 Nicotine addiction; 05034 Alcoholism; 04040 Ion channels [BR:mmu04040]; Glutamate (ionotropic); ','[Pfam: Lig_chan ANF_receptor Lig_chan-Glu_bd SBP_bac_3 CaM_bdg_C0 Peripla_BP_6 Ion_trans_2 DUF6724]'),('230405','mmu:230405','[Ifne, If1ea, Ifn-tau-1, Ifne1, Ifnt1, Infe1]','[(RefSeq) interferon epsilon]','[interferon epsilon]','[Mus musculus (house mouse)]','[Cytokine-cytokine receptor interaction, RIG-I-like receptor signaling pathway, JAK-STAT signaling pathway]','[]','[]','04630 JAK-STAT signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04622 RIG-I-like receptor signaling pathway; 04052 Cytokines and growth factors [BR:mmu04052]; Interferons; ','[Pfam: Interferon EBV-NA3]'),('13070','mmu:13070','[Cyp11a1, Cyp11a, Cypxia1, D9Ertd411e, P450scc, Scc, cscc]','[(RefSeq) cytochrome P450, family 11, subfamily a, polypeptide 1]','[cholesterol monooxygenase (side-chain-cleaving) [EC:1.14.15.6]]','[Mus musculus (house mouse)]','[Steroid hormone biosynthesis, Metabolic pathways, Ovarian steroidogenesis, Aldosterone synthesis and secretion, Cortisol synthesis and secretion, Cushing syndrome]','[]','[]','00140 Steroid hormone biosynthesis; 04913 Ovarian steroidogenesis; 04925 Aldosterone synthesis and secretion; 04927 Cortisol synthesis and secretion; 04934 Cushing syndrome; 00199 Cytochrome P450 [BR:mmu00199]; 1.14.15.6  cholesterol monooxygenase (side-chain-cleaving); CYP11 family; ','[Pfam: p450]'),('110115','mmu:110115','[Cyp11b1, AA389061, CPN1, Cyp11b, Cyp11b-1, FHI]','[(RefSeq) cytochrome P450, family 11, subfamily b, polypeptide 1]','[steroid 11beta-monooxygenase [EC:1.14.15.4]]','[Mus musculus (house mouse)]','[Steroid hormone biosynthesis, Metabolic pathways, Cortisol synthesis and secretion, Cushing syndrome]','[]','[]','00140 Steroid hormone biosynthesis; 04927 Cortisol synthesis and secretion; 04934 Cushing syndrome; 00199 Cytochrome P450 [BR:mmu00199]; 1.14.15.4  steroid 11beta-monooxygenase; CYP11 family; ','[Pfam: p450]'),('15483','mmu:15483','[Hsd11b1]','[(RefSeq) hydroxysteroid 11-beta dehydrogenase 1]','[corticosteroid 11-beta-dehydrogenase isozyme 1 [EC:1.1.1.146]]','[Mus musculus (house mouse)]','[Steroid hormone biosynthesis, Metabolism of xenobiotics by cytochrome P450, Metabolic pathways, Chemical carcinogenesis - DNA adducts]','[]','[]','00140 Steroid hormone biosynthesis; 00980 Metabolism of xenobiotics by cytochrome P450; 05204 Chemical carcinogenesis - DNA adducts; 1.1.1.146  11beta-hydroxysteroid dehydrogenase; ','[Pfam: adh_short adh_short_C2 KR DUF1776 THF_DHG_CYH_C Pyr_redox_2 2-Hacid_dh_C]'),('15484','mmu:15484','[Hsd11b2, 11HSD2]','[(RefSeq) hydroxysteroid 11-beta dehydrogenase 2]','[corticosteroid 11-beta-dehydrogenase isozyme 2 [EC:1.1.1.-]]','[Mus musculus (house mouse)]','[Steroid hormone biosynthesis, Metabolic pathways, Aldosterone-regulated sodium reabsorption]','[]','[]','00140 Steroid hormone biosynthesis; 04960 Aldosterone-regulated sodium reabsorption; 1.1.1.-; ','[Pfam: adh_short adh_short_C2 DUF1776 KR]'),('227289','mmu:227289','[Gpbar1, BG37, GPCR, GPR131, M-BAR, TGR5]','[(RefSeq) G protein-coupled bile acid receptor 1]','[G protein-coupled bile acid receptor 1]','[Mus musculus (house mouse)]',NULL,'[]','[]','04030 G protein-coupled receptors [BR:mmu04030]; Bile acid; ','[Pfam: 7tm_1]'),('100009600','mmu:100009600','[Zglp1, Glp1]','[(RefSeq) zinc finger, GATA-like protein 1]','[GATA-type zinc finger protein 1]','[Mus musculus (house mouse)]',NULL,'[]','[]','03000 Transcription factors [BR:mmu03000]; Cys4 GATA-factors; ','[Pfam: GATA ArfGap DZR]'),('20494','mmu:20494','[Slc10a2, 9130221J18Rik, AI605518, ASBT, IBAT, ISBT]','[(RefSeq) solute carrier family 10, member 2]','[solute carrier family 10 (sodium/bile acid cotransporter), member 2]','[Mus musculus (house mouse)]','[Bile secretion]','[]','[]','04976 Bile secretion; 02000 Transporters [BR:mmu02000]; SLC10: Sodium bile salt cotransporter; ','[Pfam: SBF]'),('106407','mmu:106407','[Slc51a, AV001382, AW261577, D630035O19Rik, OSTalpha, Osta]','[(RefSeq) solute carrier family 51, alpha subunit]','[organic solute transporter subunit alpha]','[Mus musculus (house mouse)]','[Bile secretion]','[]','[]','04976 Bile secretion; 02000 Transporters [BR:mmu02000]; SLC51: Transporter of steroid-derived molecules; ','[Pfam: Solute_trans_a]'),('16819','mmu:16819','[Lcn2, 24p3, AW212229, NRL, Sip24]','[(RefSeq) lipocalin 2]','[lipocalin 2]','[Mus musculus (house mouse)]','[IL-17 signaling pathway]','[]','[]','04657 IL-17 signaling pathway; ','[Pfam: Lipocalin VDE Lipocalin_2]'),('12985','mmu:12985','[Csf3, Csfg, G-CSF, MGI-IG]','[(RefSeq) colony stimulating factor 3 (granulocyte)]','[granulocyte colony-stimulating factor]','[Mus musculus (house mouse)]','[Cytokine-cytokine receptor interaction, PI3K-Akt signaling pathway, JAK-STAT signaling pathway, Hematopoietic cell lineage, IL-17 signaling pathway, Malaria, Coronavirus disease - COVID-19]','[]','[]','04630 JAK-STAT signaling pathway; 04151 PI3K-Akt signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04640 Hematopoietic cell lineage; 04657 IL-17 signaling pathway; 05171 Coronavirus disease - COVID-19; 05144 Malaria; 04052 Cytokines and growth factors [BR:mmu04052]; CSF and other factors; ','[Pfam: GCSF IL6 IL11]'),('12981','mmu:12981','[Csf2, CSF, Csfgm, GMCSF, Gm-CSf, MGI-IGM]','[(RefSeq) colony stimulating factor 2 (granulocyte-macrophage)]','[granulocyte-macrophage colony-stimulating factor]','[Mus musculus (house mouse)]','[Cytokine-cytokine receptor interaction, JAK-STAT signaling pathway, Hematopoietic cell lineage, Natural killer cell mediated cytotoxicity, IL-17 signaling pathway, T cell receptor signaling pathway, Fc epsilon RI signaling pathway, TNF signaling pathway, Amoebiasis, Human T-cell leukemia virus 1 infection, Kaposi sarcoma-associated herpesvirus infection, Coronavirus disease - COVID-19, Transcriptional misregulation in cancer, Acute myeloid leukemia, Rheumatoid arthritis]','[]','[]','04630 JAK-STAT signaling pathway; 04668 TNF signaling pathway; 04060 Cytokine-cytokine receptor interaction; 04640 Hematopoietic cell lineage; 04650 Natural killer cell mediated cytotoxicity; 04660 T cell receptor signaling pathway; 04657 IL-17 signaling pathway; 04664 Fc epsilon RI signaling pathway; 05202 Transcriptional misregulation in cancer; 05221 Acute myeloid leukemia; 05166 Human T-cell leukemia virus 1 infection; 05171 Coronavirus disease - COVID-19; 05167 Kaposi sarcoma-associated herpesvirus infection; 05146 Amoebiasis; 05323 Rheumatoid arthritis; 03019 Messenger RNA biogenesis [BR:mmu03019]; 04052 Cytokines and growth factors [BR:mmu04052]; 00536 Glycosaminoglycan binding proteins [BR:mmu00536]; Other 3\'-5\' decay factors; CSF and other factors; Cytokines; ','[Pfam: GM_CSF]'),('18948','mmu:18948','[Pnmt, Pent]','[(RefSeq) phenylethanolamine-N-methyltransferase]','[phenylethanolamine N-methyltransferase [EC:2.1.1.28]]','[Mus musculus (house mouse)]','[Tyrosine metabolism, Metabolic pathways]','[]','[]','00350 Tyrosine metabolism; 2.1.1.28  phenylethanolamine N-methyltransferase; ','[Pfam: NNMT_PNMT_TEMT Methyltransf_23 Methyltransf_11 Methyltransf_12]'),('14732','mmu:14732','[Gpam, GPAT, GPAT-1, GPAT1, P90]','[(RefSeq) glycerol-3-phosphate acyltransferase, mitochondrial]','[glycerol-3-phosphate O-acyltransferase 1/2 [EC:2.3.1.15]]','[Mus musculus (house mouse)]','[Glycerolipid metabolism, Glycerophospholipid metabolism, Metabolic pathways]','[]','[]','00561 Glycerolipid metabolism; 00564 Glycerophospholipid metabolism; 01004 Lipid biosynthesis proteins [BR:mmu01004]; 2.3.1.15  glycerol-3-phosphate 1-O-acyltransferase; GPAT; ','[Pfam: GPAT_C Acyltransferase DUF4400]'),('58805','mmu:58805','[Mlxipl, ChREBP, Mlx, WS-bHLH, Wbscr14, bHLHd14]','[(RefSeq) MLX interacting protein-like]','[MAX-like protein X]','[Mus musculus (house mouse)]','[Insulin resistance, Non-alcoholic fatty liver disease]','[]','[]','04932 Non-alcoholic fatty liver disease; 04931 Insulin resistance; 03000 Transcription factors [BR:mmu03000]; Cell cycle controlling factors, Mad/Max; ','[Pfam: HLH ATG14 TolA_bind_tri ZapB NUDE_C]'),('20788','mmu:20788','[Srebf2, AI608257, SREBP-2, SREBP2, SREBP2gc, bHLHd2, lop13, nuc]','[(RefSeq) sterol regulatory element binding factor 2]','[sterol regulatory element-binding transcription factor 2]','[Mus musculus (house mouse)]',NULL,'[]','[]','03000 Transcription factors [BR:mmu03000]; Ubiquitous bHLH-ZIP factors; ','[Pfam: HLH]'),('15357','mmu:15357','[Hmgcr, HMG-CoAR, Red]','[(RefSeq) 3-hydroxy-3-methylglutaryl-Coenzyme A reductase]','[hydroxymethylglutaryl-CoA reductase (NADPH) [EC:1.1.1.34]]','[Mus musculus (house mouse)]','[Terpenoid backbone biosynthesis, Metabolic pathways, AMPK signaling pathway, Bile secretion]','[]','[]','00900 Terpenoid backbone biosynthesis; 04152 AMPK signaling pathway; 04976 Bile secretion; 1.1.1.34  hydroxymethylglutaryl-CoA reductase (NADPH); ','[Pfam: HMG-CoA_red Sterol-sensing Patched RNA_pol_Rpb4]');
/*!40000 ALTER TABLE `details` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `gene`
--

DROP TABLE IF EXISTS `gene`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `gene` (
  `gene_symbol` longtext CHARACTER SET utf8mb4 COLLATE utf8mb4_bin,
  `ncbi_gene_id` mediumtext COLLATE utf8mb4_bin
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `gene`
--

LOCK TABLES `gene` WRITE;
/*!40000 ALTER TABLE `gene` DISABLE KEYS */;
INSERT INTO `gene` VALUES ('Bacteroidetes','gm0090'),('Christensenellaceae','gm0174'),('Enterobacteriaceae','gm0282'),('Blautia','gm0117'),('Eggerthella','gm0271'),('Sarcina','gm0596'),('CXCL6','6372'),('CCL20','6364'),('ANXA10','11199'),('DUOX2','50506'),('EGFR','1956'),('FFAR3','2865'),('FFAR2','2867'),('CCL2','6347'),('IL10','3586'),('IL12B','3593'),('IL12A','3592'),('CCR6','1235'),('CXCR6','10663'),('CXCR3','2833'),('TJP2','9414'),('PRKCZ','5590'),('IL1B','3553'),('TNF','7124'),('MAPK3','5595'),('MAPK1','5594'),('CXCL8','3576'),('FOS','2353'),('JUN','3725'),('MUC2','4583'),('HDAC1','3065'),('HDAC2','3066'),('HDAC3','8841'),('HDAC4','9759'),('HDAC5','10014'),('HDAC7','51564'),('HDAC8','55869'),('HDAC9','9734'),('HDAC11','79885'),('TLR5','7100'),('TLR2','7097'),('TLR4','7099'),('TJP1','7082'),('CLDN1','9076'),('OCLN','100506658'),('CGN','57530'),('DEFB4A','1673'),('SLC22A6','9356'),('CD14','929'),('CD163','9332'),('ANGPTL4','51129'),('CLDN4','1364'),('F11R','50848'),('TGFB1','7040'),('FOXP3','50943'),('AHR','196'),('CYP1A1','1543'),('Htr1a','15550'),('LGR5','8549'),('ALPI','248'),('LYZ','4069'),('CHGA','1113'),('DCLK1','9201'),('REG3B',NULL),('REG3G','130120'),('DEFB1','1672'),('CLDN3','1365'),('GHRL','51738'),('IL4R','3556'),('CXCR2','3579'),('HCAR2','338442'),('HCAR3','8843'),('KITLG','4254'),('SLC5A8','160728'),('MAPK14','1432'),('IKBKE','9641'),('MIR146A','406938'),('MIR155','406947'),('IL6','3569'),('GSK3B','2932'),('TCF7L2','6934'),('CCND1','595'),('MYC','4609'),('MTOR','2475'),('CXCL10','3627'),('PDCD6IP','10015'),('ALG12','79087'),('MCCC1','56922'),('DDC','1644'),('NQO1','1728'),('GNMT','27232'),('CYP1A2','1544'),('CYP3A5','1577'),('GCLC','2729'),('CYP2D6','1565'),('CYP3A4','1576'),('NFKB1','4790'),('IL1A','3552'),('IL2','3558'),('IL5','3567'),('IL22','50616'),('IFNG','3458'),('IL17A','3605'),('RORA','6095'),('CXCL1','2919'),('CXCL5','6374'),('PPARG','5468'),('PLIN2','123'),('NR1H4','9971'),('GLP1R','2740'),('GJC1','10052'),('NLRP6','171389'),('LEP','3952'),('AKT1','207'),('SOD1','6647'),('CAT','847'),('GPX3','2878'),('BCL2','596'),('BAX','581'),('CASP3','836'),('CYP7A1','1581'),('GPBAR1','151306'),('TLR6','10333'),('TGFBR1','7046'),('SMN1','6606'),('SMURF2','64750'),('SSTR1','6751'),('SSTR3','6753'),('IGAD1','10986'),('NFE2L2','4780'),('HMOX1','3162'),('C5AR1','728'),('CCL3','6348'),('PTGS2','5743'),('IL18','3606'),('IL4','3565'),('CLGN','1047'),('TJP3','27134'),('CDKN1A','1026'),('CDK6','1021'),('HERC5','51191'),('TP53','7157'),('CLDN14','23562'),('FABP2','2169'),('tnaA','948221'),('MAPK8','5599'),('ADRA2A','150'),('ADRB2','154'),('ADRA2B','151'),('RAC1','5879'),('PAK1','5058'),('CLDN7','1366'),('TLR3','7098'),('TLR9','54106'),('TOLLIP','54472'),('SIGIRR','59307'),('SOCS5','9655'),('PIK3CA','5290'),('PIK3CB','5291'),('PIK3CD','5293'),('IL24','11009'),('IFNL3','282617'),('CCL25','6370'),('CCL4','6351'),('CCL16','6360'),('XCL1','6375'),('CX3CL1','6376'),('DUSP1','1843'),('DUSP3','1845'),('DUSP10','11221'),('DUSP18','150290'),('DUSP22','56940'),('IRS1','3667'),('RHEB','6009'),('DPP7','29952'),('DAPK3','1613'),('CORO1B','57175'),('IGFBP6','3489'),('FADS3','3995'),('FLNA','2316'),('G6PD','2539'),('UPK3B','105375355'),('PEMT','10400'),('SFN','2810'),('THEM6','51337'),('AURKAIP1','54998'),('CHAC1','79094'),('DNPH1','10591'),('FKBP2','2286'),('VAMP5','10791'),('BLVRB','648'),('SERINC2','347735'),('CTSD','1509'),('SLITRK6','84189'),('MGAM2','93432'),('ONECUT2','9480'),('SORBS2','8470'),('SLC40A1','30061'),('SLC17A4','10050'),('PFN1P2','767846'),('SESN3','143686'),('RNF144B','255488'),('HPGD','3248'),('ASPM','259266'),('NOX1','27035'),('ABCC4','10257'),('LINC00504','201853'),('CCNB3','85417'),('TPH1','7166'),('IL22RA1','58985'),('MUC1','4582'),('MGAT3','4248'),('B4GALT1','2683'),('ST6GAL1','6480'),('PGP','283871'),('SLC19A2','10560'),('SLC19A3','80704'),('SLC9A3','6550'),('RELA','5970'),('BCL10','8915'),('SLC26A3','1811'),('MGAT4B','11282'),('MGAT4A','11320'),('IL22RA2','116379'),('VDR','7421'),('SULT2A1','6822'),('LCN2','3934'),('CD44','960'),('LBP','3929'),('Cldn2','12738'),('Stat3','20848'),('Hdac9','79221'),('Dgat2','67800'),('Tjp1','21872'),('Il10','16153'),('Il1b','16176'),('Il6','16193'),('Tnf','21926'),('Sirt1','93759'),('Mir10b','387144'),('Abca1','11303'),('Abcg1','11307'),('Cxcl16','66102'),('Hdac1','433759'),('Il17a','16171'),('Foxp3','20371'),('Klf4','16600'),('Muc2','17831'),('Muc4','140474'),('C1galt1','94192'),('B4galt4','56375'),('St3gal4','20443'),('Mir21a','387140'),('Slc2a1','20525'),('Slc2a4','20528'),('Akt1','11651'),('Mmp9','17395'),('Ighe','380792'),('Ighg1','16017'),('Il12p70',NULL),('Prkaa1','105787'),('Angptl4','57875'),('Gpx2','14776'),('Apob','238055'),('Aqp4','11829'),('Pla2g4c','232889'),('Prkaa2','108079'),('Cdkn1b','12576'),('Map1lc3b','67443'),('Srebf1','20787'),('Scd1','20249'),('Fasn','14104'),('Adrb3','11556'),('Pnpla2','66853'),('Cpt2','12896'),('Acox1','11430'),('Ppargc1a','19017'),('Cd68','12514'),('Ccl2','20296'),('Slco4c1','227394'),('Tgfb1','21803'),('Fn1','14268'),('Col1a1','12842'),('Cldn1','12737'),('Tff3','21786'),('Retnlb','57263'),('B3gnt6','272411'),('Gcnt1','14537'),('Fut2','14344'),('St6gal1','20440'),('Atg16l1','77040'),('Atg5','11793'),('Becn1','56208'),('Lamp2','16784'),('Map1lc3a','66734'),('Il22','50929'),('Il33','77125'),('Il13','16163'),('Nox4','50490'),('Il18','16173'),('Mir181a-2','387176'),('Ifng','15978'),('Gzmb','14939'),('Cxcl15','20309'),('Mylk3','213435'),('Gcg','14526'),('Nlrp3','216799'),('Nlrp6','101613'),('Tlr4','21898'),('Mapk8ip1','19099'),('Sucnr1','84112'),('Defa5','13239'),('Acta2','11475'),('Timp1','21857'),('Pparg','19016'),('Nos2','18126'),('Chrm2','243764'),('Tlr2','24088'),('Ahr','11622'),('Cyp1a1','13076'),('Tph1','219990'),('Htr2a','15558'),('Htr4','15562'),('Lgr5','14160'),('Alpi','76768'),('Lyz1','17110'),('Chga','12652'),('Dclk1','13175'),('Reg3b','18489'),('Reg3g','19695'),('Defb1','13214'),('Ocln','18260'),('Cldn3','12739'),('Cyp3a11','13112'),('Cyp2b10','13088'),('Abcb1a','18671'),('Dao','13142'),('Ly6g','546644'),('Itgb1','16412'),('Casp1','12362'),('Ffar2','233079'),('Slc5a8','216225'),('Nr1h4','20186'),('Fgf15','14170'),('Fgfr4','14186'),('Cyp7a1','13122'),('Sirpa','19261'),('Cpt1b','12895'),('Fas','14102'),('Ffar3','233080'),('Nfkb1','18033'),('Pdcd1lg2','58205'),('Cd86','12524'),('Cd40','21939'),('Il12b','16160'),('Hcar2','80885'),('Il10ra','16154'),('Mapk3','26417'),('Mapk1','26413'),('Pdcd6ip','18571'),('Alg12','223774'),('Acaa1a','113868'),('Acaa1b','235674'),('Acaa2','52538'),('Acads','11409'),('Acmsd','266645'),('Acot1','26897'),('Acot3','171281'),('Acot7','70025'),('Acot8','170789'),('Acsl4','50790'),('Acss2','60525'),('Acyp2','75572'),('Ada','11486'),('Agpat2','67512'),('Ahcy','269378'),('Akr1b7','11997'),('Alas1','11655'),('Aldoc','11676'),('Aox1','11761'),('Arsa','11883'),('Asl','109900'),('Asns','27053'),('Aspdh','68352'),('B3galt1','26877'),('B3gat3','72727'),('B3gnt3','72297'),('B4galnt1','14421'),('Btd','26363'),('Cel','12613'),('Chka','12660'),('Chpf','74241'),('Chst7','60322'),('Ckb','12709'),('Csad','246277'),('Cyp1a2','13077'),('Cyp2a4','13086'),('Cyp2a5','13087'),('Cyp2b13','13089'),('Cyp2b9','13094'),('Cyp2c29','13095'),('Cyp2c37','13096'),('Cyp2c40','13099'),('Cyp2c55','72082'),('Cyp2d26','76279'),('Cyp2d34','223706'),('Cyp2d40','71754'),('Cyp2j9','74519'),('Cyp3a25','56388'),('Cyp3a44','337924'),('Cyp4a10','13117'),('Cyp4a12b','13118'),('Cyp4a14','13119'),('Cyp4b1','13120'),('Cyp7b1','13123'),('Cyp8b1','13124'),('Dbt','13171'),('Dct','13190'),('Dctpp1','66422'),('Dcxr','67880'),('Ddc','13195'),('Dld','13382'),('Echdc1','52665'),('Elovl3','12686'),('Elovl5','68801'),('Eno1b','433182'),('Ephx2','13850'),('Etnppl','71760'),('Fads2','56473'),('Fuca1','71665'),('Fxn','14297'),('G6pc','14377'),('Gal3st1','53897'),('Gale','74246'),('Galk1','14635'),('Gcat','26912'),('Gck','103988'),('Gclc','14629'),('Gfpt1','14583'),('Gmds','218138'),('Gmpr','66355'),('Gnmt','14711'),('Gnpda2','67980'),('Gpcpd1','74182'),('Gpd1','14555'),('Gpx3','14778'),('Gpx6','75512'),('Gsta1','14857'),('Gsta2','14858'),('Gstm1','14862'),('Gstm2','14863'),('Gstm3','14864'),('Gstm4','14865'),('Gstt3','103140'),('Gulo','268756'),('Haao','107766'),('Hao2','56185'),('Hmgcs1','208715'),('Hpd','15445'),('Hsd17b10','15108'),('Hsd3b5','15496'),('Hyal2','15587'),('Impa2','114663'),('Isyna1','71780'),('Khk','16548'),('Kmt2a','214162'),('Lcat','16816'),('Ldha','16828'),('Lpl','16956'),('Man2b2','17160'),('Mcat','223722'),('Mgat4b','103534'),('Mgll','23945'),('Mif','17319'),('Mlycd','56690'),('Mpst','246221'),('Mthfr','17769'),('Nat8','68396'),('Ndufs3','68349'),('Ndufs8','225887'),('Nos3','18127'),('Nqo1','18104'),('Nudt12','67993'),('Ogt','108155'),('Papss2','23972'),('Pde2a','207728'),('Pde4d','238871'),('Pemt','18618'),('Pfkfb3','170768'),('Pgls','66171'),('Pgp','67078'),('Pklr','18770'),('Plcb1','18795'),('Pld4','104759'),('Pmm1','29858'),('Pnlip','69060'),('Pnliprp1','18946'),('Pnpt1','71701'),('Pomgnt2','215494'),('Ppt2','54397'),('Prdx2','21672'),('Prdx4','53381'),('Prodh2','56189'),('Ptgds','19215'),('Ptges2','96979'),('Ptgis','19223'),('Renbp','19703'),('Retsat','67442'),('Rev3l','19714'),('Sdhb','67680'),('Smpd3','58994'),('Sult1e1','20860'),('Tbxas1','21391'),('Tdo2','56720'),('Tecr','106529'),('Tm7sf2','73166'),('Treh','58866'),('Gfus','22122'),('Uap1l1','227620'),('Ugdh','22235'),('Ugt2b1','71773'),('Ugt2b34','100727'),('Ugt2b35','243085'),('Upp1','22271'),('Upp2','76654'),('Ace2','70008'),('Il12a','16159'),('Cxcl1','14825'),('Cxcl2','20310'),('Sult2a1','20859'),('Sult2a2','100043194'),('Sult2a3','629203'),('Il4','16189'),('Ppara','19013'),('Il2','16183'),('Myh7','140781'),('Nppa','230899'),('Trpv1','7442'),('Ucp1','7350'),('Ppargc1a','10891'),('Cpt1b','1375'),('Prdm16','63976'),('Tbx1','6899'),('Hdac3','15183'),('Hdac5','15184'),('Sim1','20464'),('Stat6','20852'),('Skil','20482'),('Ptf1a','19213'),('Rara','19401'),('Arnt2','11864'),('Isl1','16392'),('Nr3c1','14815'),('Trim24','21848'),('Kdm5b','75605'),('Neurod1','18012'),('Sox2','20674'),('Gata3','14462'),('Arnt','11863'),('Onecut1','15379'),('Sox11','20666'),('Relb','19698'),('Gata4','14463'),('Cdx2','12591'),('Ets2','23872'),('Hnf1b','21410'),('Notch1','18128'),('Irf6','54139'),('Sin3a','20466'),('Brd7','26992'),('Tcf4','21413'),('Mybl2','17865'),('Mtpn','14489'),('Esr1','13982'),('Foxm1','14235'),('E2f2','242705'),('E2f3','13557'),('Zbtb17','22642'),('Bmi1','12151'),('Mta1','116870'),('Phf1','21652'),('Foxa1','15375'),('Smad2','17126'),('Smad3','17127'),('Smad4','17128'),('Tfdp1','21781'),('Pgr','18667'),('Clock','12753'),('Hey1','15213'),('Cdx1','12590'),('Mtdh','67154'),('Pias2','17344'),('Gfi1','14581'),('Dbp','13170'),('Atf3','11910'),('Fbxw7','50754'),('Nacc1','66830'),('Rbl1','19650'),('Zeb1','21417'),('Zeb2','24136'),('Rnf2','19821'),('Rela','19697'),('Nfkbia','18035'),('Tonsl','72749'),('Foxo3','56484'),('Gata2','14461'),('Crebbp','12914'),('Bach1','12013'),('Jun','16476'),('Esr2','13983'),('Ifi204','15951'),('Cebpb','12608'),('Stat5b','20851'),('Klf15','66277'),('Meox2','17286'),('Sox9','20682'),('Smad7','17131'),('Stat5a','20850'),('Sqstm1','18412'),('Vdr','22337'),('Nr5a1','26423'),('Rest','19712'),('Gabpa','14390'),('Ddit3','13198'),('Nrf1','18181'),('Nr4a1','15370'),('Nr1i2','18171'),('Hif1a','29560'),('Vhl','22346'),('Yy1','22632'),('Trp53','22059'),('Pttg1','30939'),('Hic1','15248'),('Trp73','22062'),('Rb1','19645'),('Foxo4','54601'),('Taf4','228980'),('Lyl1','17095'),('Crem','12916'),('Lef1','16842'),('Myc','17869'),('Fos','14281'),('Tfap2c','21420'),('Stat1','20846'),('Klf5','12224'),('Ezh2','14056'),('Elp1','230233'),('Creb1','12912'),('Nfya','18044'),('E2f1','13555'),('Trp63','22061'),('Foxo1','56458'),('Klf2','16598'),('Tob1','22057'),('Tcf3','21423'),('Stat4','20849'),('Bcl6','12053'),('Hnf1a','21405'),('Nfe2l2','18024'),('Gli1','14632'),('Neurog3','11925'),('Fli1','14247'),('Ctnnb1','12387'),('Egr1','13653'),('Ep300','328572'),('Pdx1','18609'),('Plag1','56711'),('Mnt','17428'),('Ruvbl1','56505'),('Hif1a','15251'),('Epas1','13819'),('Arntl','11865'),('Arntl','11866'),('Cdkn2a','12578'),('Notch2','18129'),('Notch3','18131'),('Notch4','18132'),('Atf4','11911'),('Brca1','12189'),('Hnf4a','15378'),('Tbx2','21385'),('Nupr1','56312'),('Nkx2-3','18089'),('Wt1','22431'),('S100a4','20198'),('Col4a1','12826'),('Ccl3','20302'),('Kat14','228714'),('Aurkc','20871'),('Dot1l','208266'),('Kat5','81601'),('Hdac6','15185'),('Nphs1','54631'),('Kirrel','170643'),('Synpo','104027'),('Trpc5','22067'),('Trpc6','22068'),('Rac1','19353'),('Cdc42','12540'),('Orai1','109305'),('Stim1','20866'),('Stim2','116873'),('Mir10a','723893'),('Tnfrsf1b','21938'),('Ffar1','233081'),('Cxcr1','227288'),('Cxcr2','12765'),('Cdkn1a','12575'),('Ptgs2','19225'),('Ptger4','19219'),('Abcb1b','18669'),('Ugt1a1','394436'),('Bbox1','64564'),('Ighm','10619'),('Hmox1','15368'),('Gstp1','14870'),('Camk2a','12322'),('Calb1','12307'),('H3','109741'),('Mtor','56717'),('Prdm1','12142'),('Arg1','11846'),('Casp3','12367'),('Bdnf','12064'),('Sod1','20655'),('Cat','12359'),('Itgam','16409'),('Il5','16191'),('Tph1','21990'),('Ido1','15930'),('Cpt1a','12894'),('Ucp1','22227'),('Il27','246779'),('Cd274','60533'),('Entpd1','12495'),('Tjp2','21873'),('F11r','16456'),('Myd88','17874'),('mct1','17236'),('Ggt1','14598'),('Slc17a5','235504'),('Gpt','76282'),('Cd36','12491'),('Fabp2','25598'),('Ccr1','12768'),('Ccl4','20303'),('Bcl2','12043'),('Tnfrsf10b','21933'),('Cnr1','12801'),('Nelfcd','57314'),('Slc9a3','105243'),('Slc26a3','13487'),('Pdia4','12304'),('Car2','12349'),('Pdzk1','59020'),('Axin2','12006'),('Cd44','12505'),('Ephb2','13844'),('Ccnd1','12443'),('Grin1','14810'),('Ifne','230405'),('Cyp11a1','13070'),('Cyp11b1','110115'),('Hsd11b1','15483'),('Hsd11b2','15484'),('Gpbar1','227289'),('Zglp1','100009600'),('Slc10a2','20494'),('Slc51a','106407'),('Lcn2','16819'),('Csf3','12985'),('Csf2','12981'),('Pnmt','18948'),('Acc','104371'),('Gpam','14732'),('Mlxipl','58805'),('Srebf2','20788'),('Hmgcr','15357'),('Il1','111343');
/*!40000 ALTER TABLE `gene` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `gut_microbiota_gene_change_results`
--

DROP TABLE IF EXISTS `gut_microbiota_gene_change_results`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `gut_microbiota_gene_change_results` (
  `index` int DEFAULT NULL,
  `host_type` text COLLATE utf8mb4_bin,
  `Alteration` text COLLATE utf8mb4_bin,
  `throughput` text COLLATE utf8mb4_bin,
  `description` text COLLATE utf8mb4_bin,
  `experimental_method` text COLLATE utf8mb4_bin,
  `sequencing_technology` text COLLATE utf8mb4_bin,
  `Evidence` int DEFAULT NULL
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `gut_microbiota_gene_change_results`
--

LOCK TABLES `gut_microbiota_gene_change_results` WRITE;
/*!40000 ALTER TABLE `gut_microbiota_gene_change_results` DISABLE KEYS */;
INSERT INTO `gut_microbiota_gene_change_results` VALUES (3078,'human','decrease','low-throughput','The expression of several chemokine genes, some of which have reported antimicrobial properties37 (CXCL6, CCL20), were negatively correlated with the relative abundance of Eubacterium rectale (OTU 120) in the ileum, and Streptococcus (OTU 37) and Eikenella (OTU 39) in the rectum, suggesting that these species are the most susceptible to the activity of these chemokines.','cell culture','HPLC analysis,qPCR,LC/MS analysis',31142855),(3079,'human','decrease','low-throughput','The expression of several chemokine genes, some of which have reported antimicrobial properties37 (CXCL6, CCL20), were negatively correlated with the relative abundance of Eubacterium rectale (OTU 120) in the ileum, and Streptococcus (OTU 37) and Eikenella (OTU 39) in the rectum, suggesting that these species are the most susceptible to the activity of these chemokines.','cell culture','HPLC analysis,qPCR,LC/MS analysis',31142855),(3080,'human','increase','low-throughput','Calprotectin, as the sole feature in its own measurement type, was weakly associated with a number of metabolites that were not differentially abundant in dysbiosis, as well as with the metagenomic abundance of several dysbiosis-associated ECs. Three host genes appeared in this high-significance subnetwork: ileal expression of GIP, NXPE4, and ANXA10. Expression of RNA polymerase was also a prominent node in the network, though not a hub, that was upregulated in dysbiosis. The regulation of this essential enzyme class is growth-rate-dependent, suggesting that microbial communities as a whole are more often in higher growth conditions in dysbiotic IBD.','cell culture','HPLC analysis,qPCR,LC/MS analysis',31142855),(3081,'human','increase','low-throughput','Calprotectin, as the sole feature in its own measurement type, was weakly associated with a number of metabolites that were not differentially abundant in dysbiosis, as well as with the metagenomic abundance of several dysbiosis-associated ECs. Three host genes appeared in this high-significance subnetwork: ileal expression of GIP, NXPE4, and ANXA10. Expression of RNA polymerase was also a prominent node in the network, though not a hub, that was upregulated in dysbiosis. The regulation of this essential enzyme class is growth-rate-dependent, suggesting that microbial communities as a whole are more often in higher growth conditions in dysbiotic IBD.','cell culture','HPLC analysis,qPCR,LC/MS analysis',31142855),(3082,'human','increase','low-throughput','Calprotectin, as the sole feature in its own measurement type, was weakly associated with a number of metabolites that were not differentially abundant in dysbiosis, as well as with the metagenomic abundance of several dysbiosis-associated ECs. Three host genes appeared in this high-significance subnetwork: ileal expression of GIP, NXPE4, and ANXA10. Expression of RNA polymerase was also a prominent node in the network, though not a hub, that was upregulated in dysbiosis. The regulation of this essential enzyme class is growth-rate-dependent, suggesting that microbial communities as a whole are more often in higher growth conditions in dysbiotic IBD.','cell culture','HPLC analysis,qPCR,LC/MS analysis',31142855),(3084,'human','decrease','low-throughput','The expression of several chemokine genes, some of which have reported antimicrobial properties37 (CXCL6, CCL20), were negatively correlated with the relative abundance of Eubacterium rectale (OTU 120) in the ileum, and Streptococcus (OTU 37) and Eikenella (OTU 39) in the rectum, suggesting that these species are the most susceptible to the activity of these chemokines.','cell culture','HPLC analysis,qPCR,LC/MS analysis',31142855),(3085,'human','decrease','low-throughput','The expression of several chemokine genes, some of which have reported antimicrobial properties37 (CXCL6, CCL20), were negatively correlated with the relative abundance of Eubacterium rectale (OTU 120) in the ileum, and Streptococcus (OTU 37) and Eikenella (OTU 39) in the rectum, suggesting that these species are the most susceptible to the activity of these chemokines.','cell culture','HPLC analysis,qPCR,LC/MS analysis',31142855),(3086,'human','decrease','low-throughput','The genes involved included known IBD-associated host–microbial interaction factors, including DUOX2 and its maturation factor DUOXA231,36, both of which were negatively associated with the abundance of Ruminococcaceae UCG 005 (OTU 89) in the ileum.','cell culture','HPLC analysis,qPCR,LC/MS analysis',31142855),(3087,'human','decrease','low-throughput','The expression of several chemokine genes, some of which have reported antimicrobial properties37 (CXCL6, CCL20), were negatively correlated with the relative abundance of Eubacterium rectale (OTU 120) in the ileum, and Streptococcus (OTU 37) and Eikenella (OTU 39) in the rectum, suggesting that these species are the most susceptible to the activity of these chemokines.','cell culture','HPLC analysis,qPCR,LC/MS analysis',31142855),(3109,'human','decrease','low-throughput','Indoxyl-sulfate (Beta(SE) = -2.74(0.24), P = 8.8x10-29),p-cresyl-sulfate (-1.99(0.24), P = 4.6x10-16), and phenylacetylglutamine (-2.73(0.25),P = 1.2x10-25) were significantly and negatively associated with eGFR after adjusting for age, sex, body mass index (BMI), metabolite batch, type 2 diabetes, family relatedness and multiple testing using Bonferroni correction (Table 2).','in vitro experiments','GC/MS analysis',26241311),(3110,'human','decrease','low-throughput','Indoxyl-sulfate (Beta(SE) = -2.74(0.24), P = 8.8x10-29),p-cresyl-sulfate (-1.99(0.24), P = 4.6x10-16), and phenylacetylglutamine (-2.73(0.25),P = 1.2x10-25) were significantly and negatively associated with eGFR after adjusting for age, sex, body mass index (BMI), metabolite batch, type 2 diabetes, family relatedness and multiple testing using Bonferroni correction (Table 2).','in vitro experiments','GC/MS analysis',26241311),(3111,'human','decrease','low-throughput','Indoxyl-sulfate (Beta(SE) = -2.74(0.24), P = 8.8x10-29),p-cresyl-sulfate (-1.99(0.24), P = 4.6x10-16), and phenylacetylglutamine (-2.73(0.25),P = 1.2x10-25) were significantly and negatively associated with eGFR after adjusting for age, sex, body mass index (BMI), metabolite batch, type 2 diabetes, family relatedness and multiple testing using Bonferroni correction (Table 2).','in vitro experiments','GC/MS analysis',26241311),(3141,'human','increase','low-throughput','In the evaluation of our study, a lower count of A.muciniphila and F. prausnitzii was observed in the micro biota of allergic children. F. prausnitzii and A.muciniphila may have induced anti-inflammatory cytokine IL-10 and may prevent the secretion of pro-inflammatory cytokines like IL-12. These findings suggest that A. muciniphila and F.prausnitzii may suppress inflammation through its secreted metabolites.29 We suggest that all of the aforementioned mechanisms that result from the decrease of F.prausnitzii and A. muciniphila may contribute to allergic asthma in our patient group.','cell culture','qPCR',30765132),(3142,'human','decrease','low-throughput','In the evaluation of our study, a lower count of A.muciniphila and F. prausnitzii was observed in the micro biota of allergic children. F. prausnitzii and A.muciniphila may have induced anti-inflammatory cytokine IL-10 and may prevent the secretion of pro-inflammatory cytokines like IL-12. These findings suggest that A. muciniphila and F.prausnitzii may suppress inflammation through its secreted metabolites.29 We suggest that all of the aforementioned mechanisms that result from the decrease of F.prausnitzii and A. muciniphila may contribute to allergic asthma in our patient group.','cell culture','qPCR',30765132),(3143,'human','decrease','low-throughput','In the evaluation of our study, a lower count of A.muciniphila and F. prausnitzii was observed in the micro biota of allergic children. F. prausnitzii and A.muciniphila may have induced anti-inflammatory cytokine IL-10 and may prevent the secretion of pro-inflammatory cytokines like IL-12. These findings suggest that A. muciniphila and F.prausnitzii may suppress inflammation through its secreted metabolites.29 We suggest that all of the aforementioned mechanisms that result from the decrease of F.prausnitzii and A. muciniphila may contribute to allergic asthma in our patient group.','cell culture','qPCR',30765132),(3144,'human','increase','low-throughput','In the evaluation of our study, a lower count of A.muciniphila and F. prausnitzii was observed in the micro biota of allergic children. F. prausnitzii and A.muciniphila may have induced anti-inflammatory cytokine IL-10 and may prevent the secretion of pro-inflammatory cytokines like IL-12. These findings suggest that A. muciniphila and F.prausnitzii may suppress inflammation through its secreted metabolites.29 We suggest that all of the aforementioned mechanisms that result from the decrease of F.prausnitzii and A. muciniphila may contribute to allergic asthma in our patient group.','cell culture','qPCR',30765132),(3145,'human','decrease','low-throughput','In the evaluation of our study, a lower count of A.muciniphila and F. prausnitzii was observed in the micro biota of allergic children. F. prausnitzii and A.muciniphila may have induced anti-inflammatory cytokine IL-10 and may prevent the secretion of pro-inflammatory cytokines like IL-12. These findings suggest that A. muciniphila and F.prausnitzii may suppress inflammation through its secreted metabolites.29 We suggest that all of the aforementioned mechanisms that result from the decrease of F.prausnitzii and A. muciniphila may contribute to allergic asthma in our patient group.','cell culture','qPCR',30765132),(3146,'human','decrease','low-throughput','In the evaluation of our study, a lower count of A.muciniphila and F. prausnitzii was observed in the micro biota of allergic children. F. prausnitzii and A.muciniphila may have induced anti-inflammatory cytokine IL-10 and may prevent the secretion of pro-inflammatory cytokines like IL-12. These findings suggest that A. muciniphila and F.prausnitzii may suppress inflammation through its secreted metabolites.29 We suggest that all of the aforementioned mechanisms that result from the decrease of F.prausnitzii and A. muciniphila may contribute to allergic asthma in our patient group.','cell culture','qPCR',30765132),(3149,'human','increase','low-throughput','Based on these results and on the frequent expression of CCR6, CXCR3, and CXCR6 by colon-derived DP8a cells, we then further focused on CCR6, CXCR3, and CXCR6 as potential markers for F prausnitzii–specific DP8a cells: in comparison to LPLs, CCR6þ, CXCR6þ, and CXCR3þ cells represented only 55%, 32%, and 37%, respectively, of DP8a cells derived from PBMCs, and almost 18% of the latter cells expressed the 3 molecules.','cell culture','ELISA',29981781),(3150,'human','increase','low-throughput','Based on these results and on the frequent expression of CCR6, CXCR3, and CXCR6 by colon-derived DP8a cells, we then further focused on CCR6, CXCR3, and CXCR6 as potential markers for F prausnitzii–specific DP8a cells: in comparison to LPLs, CCR6þ, CXCR6þ, and CXCR3þ cells represented only 55%, 32%, and 37%, respectively, of DP8a cells derived from PBMCs, and almost 18% of the latter cells expressed the 3 molecules.','cell culture','ELISA',29981781),(3151,'human','increase','low-throughput','Based on these results and on the frequent expression of CCR6, CXCR3, and CXCR6 by colon-derived DP8a cells, we then further focused on CCR6, CXCR3, and CXCR6 as potential markers for F prausnitzii–specific DP8a cells: in comparison to LPLs, CCR6þ, CXCR6þ, and CXCR3þ cells represented only 55%, 32%, and 37%, respectively, of DP8a cells derived from PBMCs, and almost 18% of the latter cells expressed the 3 molecules.','cell culture','ELISA',29981781),(3152,'human','increase','low-throughput','For the TJ protein ZO-2 we found a slight upregulation of mRNA after incubation with EcN.','cell culture','RT-qPCR,immunofluorescence staining,Western blot assay',17087734),(3153,'human','increase','low-throughput','For the TJ protein ZO-2 we found a slight upregulation of mRNA after incubation with EcN.','cell culture','RT-qPCR,immunofluorescence staining,Western blot assay',17087734),(3154,'human','decrease','low-throughput','The activity of PKCz is reduced during incubation with EcN as well as during co-incubation with EcN and EPEC.','cell culture','RT-qPCR,immunofluorescence staining,Western blot assay',17087734),(3155,'human','decrease','low-throughput','The activity of PKCz is reduced during incubation with EcN as well as during co-incubation with EcN and EPEC.','cell culture','RT-qPCR,immunofluorescence staining,Western blot assay',17087734),(3162,'human','increase','low-throughput','butyrate enhanced the PMA-induced expression of c-fos and ERK1/2 phosphorylation, but not p38 and JNK.','cell culture','HPLC analysis,real-time qPCR,LC/MS analysis,Luciferase Reporter Assay',23300800),(3163,'human','increase','low-throughput','butyrate enhanced the PMA-induced expression of c-fos and ERK1/2 phosphorylation, but not p38 and JNK.','cell culture','HPLC analysis,real-time qPCR,LC/MS analysis,Luciferase Reporter Assay',23300800),(3164,'human','increase','low-throughput','butyrate enhanced the PMA-induced expression of c-fos and ERK1/2 phosphorylation, but not p38 and JNK.','cell culture','HPLC analysis,real-time qPCR,LC/MS analysis,Luciferase Reporter Assay',23300800),(3165,'human','increase','low-throughput','The activation of AP-1 was strongly correlated with the presence of butyrate in the CM.','cell culture','HPLC analysis,real-time qPCR,LC/MS analysis,Luciferase Reporter Assay',23300800),(3166,'human','increase','low-throughput','butyrate enhanced the PMA-induced expression of c-fos and ERK1/2 phosphorylation, but not p38 and JNK.','cell culture','HPLC analysis,real-time qPCR,LC/MS analysis,Luciferase Reporter Assay',23300800),(3167,'human','increase','low-throughput','butyrate enhanced the PMA-induced expression of c-fos and ERK1/2 phosphorylation, but not p38 and JNK.','cell culture','HPLC analysis,real-time qPCR,LC/MS analysis,Luciferase Reporter Assay',23300800),(3168,'human','increase','low-throughput','butyrate enhanced the PMA-induced expression of c-fos and ERK1/2 phosphorylation, but not p38 and JNK.','cell culture','HPLC analysis,real-time qPCR,LC/MS analysis,Luciferase Reporter Assay',23300800),(3169,'human','increase','low-throughput','The activation of AP-1 was strongly correlated with the presence of butyrate in the CM.','cell culture','HPLC analysis,real-time qPCR,LC/MS analysis,Luciferase Reporter Assay',23300800),(3180,'human','decrease','low-throughput','Both JIM8772 and CIP102503T supernatants and their corresponding <3-kDa fractions provoked a drastic decrease of IL-8 secretion induced by TNF-α.','cell culture','HPLC analysis',21602373),(3181,'human','decrease','low-throughput','We showed that the supernatant of S. salivarius also inhibited NF-κB activity induced in HT-29 cells with another proinflammatory cytokine, IL-1β, and the TLR5 ligand, flagellin.','cell culture','HPLC analysis',21602373),(3182,'human','decrease','low-throughput','We showed that the supernatant of S. salivarius also inhibited NF-κB activity induced in HT-29 cells with another proinflammatory cytokine, IL-1β, and the TLR5 ligand, flagellin.','cell culture','HPLC analysis',21602373),(3183,'human','decrease','low-throughput','Both JIM8772 and CIP102503T supernatants and their corresponding <3-kDa fractions provoked a drastic decrease of IL-8 secretion induced by TNF-α.','cell culture','HPLC analysis',21602373),(3184,'human','decrease','low-throughput','We showed that the supernatant of S. salivarius also inhibited NF-κB activity induced in HT-29 cells with another proinflammatory cytokine, IL-1β, and the TLR5 ligand, flagellin.','cell culture','HPLC analysis',21602373),(3185,'human','decrease','low-throughput','We showed that the supernatant of S. salivarius also inhibited NF-κB activity induced in HT-29 cells with another proinflammatory cytokine, IL-1β, and the TLR5 ligand, flagellin.','cell culture','HPLC analysis',21602373),(3192,'human','increase','low-throughput','A. muciniphila activates the NF-κB pathway through TLR4 and TLR2 receptors.','cell culture','LC/MS/MS analysis',28249045),(3193,'human','increase','low-throughput','A. muciniphila activates the NF-κB pathway through TLR4 and TLR2 receptors.','cell culture','LC/MS/MS analysis',28249045),(3237,'human','increase','low-throughput','The expression of barrier-forming genes, viz. ZO-1, Claudin-1, Occludin and Cingulin were elevated significantly (p < 0.05) by 2.33 ± 0.4, 2.47 ± 0.3, 2.70 ± 0.5 and 3.13 ± 0.6 folds, respectively, after exposure of Caco-2 cells to L. rhamnosus for 6 h as compared to untreated cells (Fig. 2A-D).','control experiment,in vitro bacterial culture,cell culture','real-time qPCR,real-time PCR,qPCR',31732863),(3238,'human','increase','low-throughput','The expression of barrier-forming genes, viz. ZO-1, Claudin-1, Occludin and Cingulin were elevated significantly (p < 0.05) by 2.33 ± 0.4, 2.47 ± 0.3, 2.70 ± 0.5 and 3.13 ± 0.6 folds, respectively, after exposure of Caco-2 cells to L. rhamnosus for 6 h as compared to untreated cells (Fig. 2A-D).','control experiment,in vitro bacterial culture,cell culture','real-time qPCR,real-time PCR,qPCR',31732863),(3239,'human','increase','low-throughput','The expression of barrier-forming genes, viz. ZO-1, Claudin-1, Occludin and Cingulin were elevated significantly (p < 0.05) by 2.33 ± 0.4, 2.47 ± 0.3, 2.70 ± 0.5 and 3.13 ± 0.6 folds, respectively, after exposure of Caco-2 cells to L. rhamnosus for 6 h as compared to untreated cells (Fig. 2A-D).','control experiment,in vitro bacterial culture,cell culture','real-time qPCR,real-time PCR,qPCR',31732863),(3240,'human','increase','low-throughput','The expression of barrier-forming genes, viz. ZO-1, Claudin-1, Occludin and Cingulin were elevated significantly (p < 0.05) by 2.33 ± 0.4, 2.47 ± 0.3, 2.70 ± 0.5 and 3.13 ± 0.6 folds, respectively, after exposure of Caco-2 cells to L. rhamnosus for 6 h as compared to untreated cells (Fig. 2A-D).','control experiment,in vitro bacterial culture,cell culture','real-time qPCR,real-time PCR,qPCR',31732863),(3241,'human','decrease','low-throughput','In contrast to the remarkable elevated levels of hBD-2 mRNA after stimulation with E. coli individually, exclusion, competition and displacement of E. coli by L. rhamnosus considerably reduced its expression which was indicative of the induction of healthy conditions in presence of L. rhamnosus under the pathogenic milieu set in by E. coli alone.','control experiment,in vitro bacterial culture,cell culture','real-time qPCR,real-time PCR,qPCR',31732863),(3345,'human','increase','low-throughput','For the PBMC assay, although all the strains tend to increase the production of IL-10 cytokine, only four strains were able to induce statistically significant increase production of this cytokine .The two most performing strains (A2-165 and 4543) belong to the phylogroup II, group B.','in vitro bacterial culture','16S rRNA sequence,ELISA',28713353),(3346,'human','increase','low-throughput','For the PBMC assay, although all the strains tend to increase the production of IL-10 cytokine, only four strains were able to induce statistically significant increase production of this cytokine .The two most performing strains (A2-165 and 4543) belong to the phylogroup II, group B.','in vitro bacterial culture','16S rRNA sequence,ELISA',28713353),(3347,'human','increase','low-throughput','For the PBMC assay, although all the strains tend to increase the production of IL-10 cytokine, only four strains were able to induce statistically significant increase production of this cytokine .The two most performing strains (A2-165 and 4543) belong to the phylogroup II, group B.','in vitro bacterial culture','16S rRNA sequence,ELISA',28713353),(3348,'human','increase','low-throughput','For the PBMC assay, although all the strains tend to increase the production of IL-10 cytokine, only four strains were able to induce statistically significant increase production of this cytokine .The two most performing strains (A2-165 and 4543) belong to the phylogroup II, group B.','in vitro bacterial culture','16S rRNA sequence,ELISA',28713353),(3349,'human','decrease','low-throughput','As shown in Figure 6A, all the strains tend to decrease IL-8 concentrations. However, this decrease was not equivalent in all the strains and does not correlate either with growth ratio (r = ±0.2857, p = 0.344) or butyrate production (r = �?.3357, p =0.2869).','in vitro bacterial culture','16S rRNA sequence,ELISA',28713353),(3350,'human','decrease','low-throughput','As shown in Figure 6A, all the strains tend to decrease IL-8 concentrations. However, this decrease was not equivalent in all the strains and does not correlate either with growth ratio (r = ±0.2857, p = 0.344) or butyrate production (r = �?.3357, p =0.2869).','in vitro bacterial culture','16S rRNA sequence,ELISA',28713353),(3351,'human','decrease','low-throughput','As shown in Figure 6A, all the strains tend to decrease IL-8 concentrations. However, this decrease was not equivalent in all the strains and does not correlate either with growth ratio (r = ±0.2857, p = 0.344) or butyrate production (r = �?.3357, p =0.2869).','in vitro bacterial culture','16S rRNA sequence,ELISA',28713353),(3352,'human','decrease','low-throughput','As shown in Figure 6A, all the strains tend to decrease IL-8 concentrations. However, this decrease was not equivalent in all the strains and does not correlate either with growth ratio (r = ±0.2857, p = 0.344) or butyrate production (r = �?.3357, p =0.2869).','in vitro bacterial culture','16S rRNA sequence,ELISA',28713353),(3353,'human','decrease','low-throughput','As shown in Figure 6A, all the strains tend to decrease IL-8 concentrations. However, this decrease was not equivalent in all the strains and does not correlate either with growth ratio (r = ±0.2857, p = 0.344) or butyrate production (r = �?.3357, p =0.2869).','in vitro bacterial culture','16S rRNA sequence,ELISA',28713353),(3354,'human','decrease','low-throughput','As shown in Figure 6A, all the strains tend to decrease IL-8 concentrations. However, this decrease was not equivalent in all the strains and does not correlate either with growth ratio (r = ±0.2857, p = 0.344) or butyrate production (r = �?.3357, p =0.2869).','in vitro bacterial culture','16S rRNA sequence,ELISA',28713353),(3355,'human','decrease','low-throughput','As shown in Figure 6A, all the strains tend to decrease IL-8 concentrations. However, this decrease was not equivalent in all the strains and does not correlate either with growth ratio (r = ±0.2857, p = 0.344) or butyrate production (r = �?.3357, p =0.2869).','in vitro bacterial culture','16S rRNA sequence,ELISA',28713353),(3356,'human','decrease','low-throughput','As shown in Figure 6A, all the strains tend to decrease IL-8 concentrations. However, this decrease was not equivalent in all the strains and does not correlate either with growth ratio (r = ±0.2857, p = 0.344) or butyrate production (r = �?.3357, p =0.2869).','in vitro bacterial culture','16S rRNA sequence,ELISA',28713353),(3357,'human','decrease','low-throughput','As shown in Figure 6A, all the strains tend to decrease IL-8 concentrations. However, this decrease was not equivalent in all the strains and does not correlate either with growth ratio (r = ±0.2857, p = 0.344) or butyrate production (r = �?.3357, p =0.2869).','in vitro bacterial culture','16S rRNA sequence,ELISA',28713353),(3367,'human','increase','low-throughput','In the present study, we focused on the impact of SCFAs and the role of PPAR-γin intestinal ANGPTL4 expression. Here, we show that ANGPTL4 expression can be regulated by SCFAs both in vitro and in vivo, with butyrate being the most efficient activator.','control experiment','real-time qPCR，Western blot assay，ELISA',23518684),(3417,'human','decrease','low-throughput','In the present study, LP mitigated the UCB-induced redistribution of occludin, ZO-1, claudin-1, claudin-4, JAM-1 and F-actin.','control experiment,in vitro bacterial culture,cell culture','Western blot assay, real-time qPCR',20412608),(3418,'human','decrease','low-throughput','In the present study, LP mitigated the UCB-induced redistribution of occludin, ZO-1, claudin-1, claudin-4, JAM-1 and F-actin.','control experiment,in vitro bacterial culture,cell culture','Western blot assay, real-time qPCR',20412608),(3419,'human','decrease','low-throughput','In the present study, LP mitigated the UCB-induced redistribution of occludin, ZO-1, claudin-1, claudin-4, JAM-1 and F-actin.','control experiment,in vitro bacterial culture,cell culture','Western blot assay, real-time qPCR',20412608),(3420,'human','decrease','low-throughput','In the present study, LP mitigated the UCB-induced redistribution of occludin, ZO-1, claudin-1, claudin-4, JAM-1 and F-actin.','control experiment,in vitro bacterial culture,cell culture','Western blot assay, real-time qPCR',20412608),(3421,'human','decrease','low-throughput','In the present study, LP mitigated the UCB-induced redistribution of occludin, ZO-1, claudin-1, claudin-4, JAM-1 and F-actin.','control experiment,in vitro bacterial culture,cell culture','Western blot assay, real-time qPCR',20412608),(3422,'human','increase','low-throughput','F. prausnitzii, B. longum and F. prausnitzii supernatant clearly facilitated the induction of IL-10 and TGF-β1, while induced relatively mild production of IL-12p70 in both cellular and animal models. The F. prausnitzii, B. longum and supernatant differed in their capacity to induce T-bet, GATA-3 and ROR-γt mRNA expression in human PBMCs (both bacterial strains inhibited the expression of ROR-γt while supernatant inhibited the T-bet and GATA-3). However, all of them induced the Foxp3 and Treg production and ameliorated the TNBS-induced colitis. In addition, F. prausnitzii supernatant exhibited the supreme anti-inflammatory capacity.','control experiment,in vitro bacterial culture','ELISA,real-time qPCR',23643066),(3423,'human','increase','low-throughput','F. prausnitzii, B. longum and F. prausnitzii supernatant clearly facilitated the induction of IL-10 and TGF-β1, while induced relatively mild production of IL-12p70 in both cellular and animal models. The F. prausnitzii, B. longum and supernatant differed in their capacity to induce T-bet, GATA-3 and ROR-γt mRNA expression in human PBMCs (both bacterial strains inhibited the expression of ROR-γt while supernatant inhibited the T-bet and GATA-3). However, all of them induced the Foxp3 and Treg production and ameliorated the TNBS-induced colitis. In addition, F. prausnitzii supernatant exhibited the supreme anti-inflammatory capacity.','control experiment,in vitro bacterial culture','ELISA,real-time qPCR',23643066),(3424,'human','increase','low-throughput','F. prausnitzii, B. longum and F. prausnitzii supernatant clearly facilitated the induction of IL-10 and TGF-β1, while induced relatively mild production of IL-12p70 in both cellular and animal models. The F. prausnitzii, B. longum and supernatant differed in their capacity to induce T-bet, GATA-3 and ROR-γt mRNA expression in human PBMCs (both bacterial strains inhibited the expression of ROR-γt while supernatant inhibited the T-bet and GATA-3). However, all of them induced the Foxp3 and Treg production and ameliorated the TNBS-induced colitis. In addition, F. prausnitzii supernatant exhibited the supreme anti-inflammatory capacity.','control experiment,in vitro bacterial culture','ELISA,real-time qPCR',23643066),(3451,'human','increase','low-throughput','These data show that B dentium–secreted products, including acetate, can modulate 5-HT production by host enterochromaffin cells.','in vitro bacterial culture','immunostaining, qPCR，LC-MS/MS analysis',32795610),(3504,'human','decrease','low-throughput','A Gram-negative anaerobic bacterium, strain 4, capable of metabolizing diosmetin-7-O-glucoside was newly isolated.Its 16S rRNA sequence displayed 99% similarity with that of Escherichia. Then strain 4 was identified as a species of thegenus Escherichia and was named Escherichia sp. 4. Additionally,  an ultraperformance liquid chromatography/quadrupole-time-of-flight mass spectrometry  technique combined with Metabolynx software method was established to screenthe metabolites of diosmetin-7-O-glucoside.','in vitro bacterial culture,control experiment','HPLC analysis,16S rRNA sequence',25382172),(3511,'human','decrease','low-throughput','Inactivated probiotic strains can affect TLR4 expression in a down-regulatory direction as with lipopolysaccharides  after 12 hours. LGG significantly down-regulated expression of p38 while IκB expression was significantly reduced  in L.del-treated DCs. Moreover, we found these Lactobacillus strains could even modify the immune response at  post-transcriptional level by modifying miRNAs expression. Based on our results LGG induced a significant down_regulatory effect on miR-146a expression which is known as a novel fine negative regulator of immune response  targeting NFκB. On the other hand, miR-155 was up-regulated by LGG which is consistent with down-regulation  of p38 and in LGG-treated DCs.','cell culture','fluorescence microscopy,RT-PCR',22476320),(3512,'human','decrease','low-throughput','Inactivated probiotic strains can affect TLR4 expression in a down-regulatory direction as with lipopolysaccharides  after 12 hours. LGG significantly down-regulated expression of p38 while IκB expression was significantly reduced  in L.del-treated DCs. Moreover, we found these Lactobacillus strains could even modify the immune response at  post-transcriptional level by modifying miRNAs expression. Based on our results LGG induced a significant down_regulatory effect on miR-146a expression which is known as a novel fine negative regulator of immune response  targeting NFκB. On the other hand, miR-155 was up-regulated by LGG which is consistent with down-regulation  of p38 and in LGG-treated DCs.','cell culture','fluorescence microscopy,RT-PCR',22476320),(3513,'human','decrease','low-throughput','Inactivated probiotic strains can affect TLR4 expression in a down-regulatory direction as with lipopolysaccharides  after 12 hours. LGG significantly down-regulated expression of p38 while IκB expression was significantly reduced  in L.del-treated DCs. Moreover, we found these Lactobacillus strains could even modify the immune response at  post-transcriptional level by modifying miRNAs expression. Based on our results LGG induced a significant down_regulatory effect on miR-146a expression which is known as a novel fine negative regulator of immune response  targeting NFκB. On the other hand, miR-155 was up-regulated by LGG which is consistent with down-regulation  of p38 and in LGG-treated DCs.','cell culture','fluorescence microscopy,RT-PCR',22476320),(3514,'human','decrease','low-throughput','Inactivated probiotic strains can affect TLR4 expression in a down-regulatory direction as with lipopolysaccharides  after 12 hours. LGG significantly down-regulated expression of p38 while IκB expression was significantly reduced  in L.del-treated DCs. Moreover, we found these Lactobacillus strains could even modify the immune response at  post-transcriptional level by modifying miRNAs expression. Based on our results LGG induced a significant down_regulatory effect on miR-146a expression which is known as a novel fine negative regulator of immune response  targeting NFκB. On the other hand, miR-155 was up-regulated by LGG which is consistent with down-regulation  of p38 and in LGG-treated DCs.','cell culture','fluorescence microscopy,RT-PCR',22476320),(3515,'human','increase','low-throughput','Inactivated probiotic strains can affect TLR4 expression in a down-regulatory direction as with lipopolysaccharides  after 12 hours. LGG significantly down-regulated expression of p38 while IκB expression was significantly reduced  in L.del-treated DCs. Moreover, we found these Lactobacillus strains could even modify the immune response at  post-transcriptional level by modifying miRNAs expression. Based on our results LGG induced a significant down_regulatory effect on miR-146a expression which is known as a novel fine negative regulator of immune response  targeting NFκB. On the other hand, miR-155 was up-regulated by LGG which is consistent with down-regulation  of p38 and in LGG-treated DCs.','cell culture','fluorescence microscopy,RT-PCR',22476320),(3529,'human','increase','low-throughput','DIF-3 induces proteolysis of cyclin D1 and c-Myc in HCT-116 cells via activation of GSK-3b.','cell culture','Western blot assay,RT-PCR',25913757),(3530,'human','decrease','low-throughput','The amount of TCF7L2 increased gradually over time in control cells, but it was markedly reduced by DIF-3 in a time and dose-dependent manner.','cell culture','Western blot assay,RT-PCR',25913757),(3531,'human','decrease','low-throughput','DIF-3 decreased the expression of cyclin D1 and cMyc to a level comparable to the reduction induced by DIF-1.','cell culture','Western blot assay,RT-PCR',25913757),(3532,'human','decrease','low-throughput','DIF-3 decreased the expression of cyclin D1 and cMyc to a level comparable to the reduction induced by DIF-1.','cell culture','Western blot assay,RT-PCR',25913757),(3533,'human','decrease','low-throughput','Butyrate-induced antimicrobial activity was associated with a shift in macrophage metabolism, a reduction in mTOR kinase activity, increased LC3-associated host defense and anti-microbial peptide production in the absence of an increased inflammatory cytokine response. Butyrate drove this monocyte to macrophage differentiation program through histone deacetylase 3 (HDAC3) inhibition. Administration of butyrate induced antimicrobial activity in intestinal macrophages in vivo and increased resistance to enteropathogens.','in vitro bacterial culture,cell culture','LC/MS analysis,qPCR',30683619),(3534,'human','decrease','low-throughput','Butyrate-induced antimicrobial activity was associated with a shift in macrophage metabolism, a reduction in mTOR kinase activity, increased LC3-associated host defense and anti-microbial peptide production in the absence of an increased inflammatory cytokine response. Butyrate drove this monocyte to macrophage differentiation program through histone deacetylase 3 (HDAC4) inhibition. Administration of butyrate induced antimicrobial activity in intestinal macrophages in vivo and increased resistance to enteropathogens.','in vitro bacterial culture,cell culture','LC/MS analysis,qPCR',30683619),(3535,'human','decrease','low-throughput','Butyrate-induced antimicrobial activity was associated with a shift in macrophage metabolism, a reduction in mTOR kinase activity, increased LC3-associated host defense and anti-microbial peptide production in the absence of an increased inflammatory cytokine response. Butyrate drove this monocyte to macrophage differentiation program through histone deacetylase 3 (HDAC5) inhibition. Administration of butyrate induced antimicrobial activity in intestinal macrophages in vivo and increased resistance to enteropathogens.','in vitro bacterial culture,cell culture','LC/MS analysis,qPCR',30683619),(3540,'human','increase','low-throughput','In summary, our results clearly demonstrate that different E. faecalis strains differentially induce colitis in monoassociated IL-10-/- mice and that severe E. faecalis-induced colitis resulted in decreased Alix/AIP1 protein expression as well as ERK1/2 phosphorylation. Functional studies revealed no effect of Alix/AIP1 on bacteria- and EGF-mediated effects on the epithelium. The 13 differentially regulated proteins might be ofinterest aspotentialmarkers for severeintestinalinflammation.','in vitro bacterial culture,cell culture','qPCR,ELISA,Western blot assay',19166300),(3541,'human','decrease','low-throughput','In summary, our results clearly demonstrate that different E. faecalis strains differentially induce colitis in monoassociated IL-10-/- mice and that severe E. faecalis-induced colitis resulted in decreased Alix/AIP1 protein expression as well as ERK1/2 phosphorylation. Functional studies revealed no effect of Alix/AIP1 on bacteria- and EGF-mediated effects on the epithelium. The 13 differentially regulated proteins might be ofinterest aspotentialmarkers for severeintestinalinflammation.','in vitro bacterial culture,cell culture','qPCR,ELISA,Western blot assay',19166300),(3566,'human','increase','low-throughput','Regulatory T cells (Tregs) that express the transcription factor Foxp3 are critical for regulating intestinal inflammation. Candidate microbe approaches have identified bacterial species and strain-specific molecules that can affect intestinal immune responses, including species that modulate Treg responses. Because neither all humans nor mice harbor the same bacterial strains, we posited that more prevalent factors exist that regulate the number and function of colonic Tregs. We determined that short chain fatty acids (SCFA), gut microbiota-derived bacterial fermentation products, regulate the size and function of the colonic Treg pool and protect against colitis in a Ffar2(GPR43)-dependent manner in mice. Our study reveals that a class of abundant microbial metabolites underlies adaptive immune microbiota co-adaptation and promotes colonic homeostasis and health.','cell culture','null',23828891),(3567,'human','increase','low-throughput','Regulatory T cells (Tregs) that express the transcription factor Foxp3 are critical for regulating intestinal inflammation. Candidate microbe approaches have identified bacterial species and strain-specific molecules that can affect intestinal immune responses, including species that modulate Treg responses. Because neither all humans nor mice harbor the same bacterial strains, we posited that more prevalent factors exist that regulate the number and function of colonic Tregs. We determined that short chain fatty acids (SCFA), gut microbiota-derived bacterial fermentation products, regulate the size and function of the colonic Treg pool and protect against colitis in a Ffar2(GPR43)-dependent manner in mice. Our study reveals that a class of abundant microbial metabolites underlies adaptive immune microbiota co-adaptation and promotes colonic homeostasis and health.','cell culture','null',23828891),(3568,'human','increase','low-throughput','Regulatory T cells (Tregs) that express the transcription factor Foxp3 are critical for regulating intestinal inflammation. Candidate microbe approaches have identified bacterial species and strain-specific molecules that can affect intestinal immune responses, including species that modulate Treg responses. Because neither all humans nor mice harbor the same bacterial strains, we posited that more prevalent factors exist that regulate the number and function of colonic Tregs. We determined that short chain fatty acids (SCFA), gut microbiota-derived bacterial fermentation products, regulate the size and function of the colonic Treg pool and protect against colitis in a Ffar2(GPR43)-dependent manner in mice. Our study reveals that a class of abundant microbial metabolites underlies adaptive immune microbiota co-adaptation and promotes colonic homeostasis and health.','cell culture','null',23828891),(3569,'human','decrease','low-throughput','F. prausnitzii exhibits anti-inflammatory effects, partly due to secreted metabolites able to block nuclear factor-κB activation and the secretion of proinflammatory mediators (41). Oral administration of F. prausnitzii or of supernatant from F. prausnitzii cultures increased the production of IL-10 by blood mononuclear cells and reduced the production of the proinflammatory mediator-like IL-12 in the colon.','in vitro bacterial culture','RT-qPCR',20876719),(3570,'human','decrease','low-throughput','F. prausnitzii exhibits anti-inflammatory effects, partly due to secreted metabolites able to block nuclear factor-κB activation and the secretion of proinflammatory mediators (41). Oral administration of F. prausnitzii or of supernatant from F. prausnitzii cultures increased the production of IL-10 by blood mononuclear cells and reduced the production of the proinflammatory mediator-like IL-12 in the colon.','in vitro bacterial culture','RT-qPCR',20876719),(3571,'human','decrease','low-throughput','F. prausnitzii exhibits anti-inflammatory effects, partly due to secreted metabolites able to block nuclear factor-κB activation and the secretion of proinflammatory mediators (41). Oral administration of F. prausnitzii or of supernatant from F. prausnitzii cultures increased the production of IL-10 by blood mononuclear cells and reduced the production of the proinflammatory mediator-like IL-12 in the colon.','in vitro bacterial culture','RT-qPCR',20876719),(3572,'human','increase','low-throughput','F. prausnitzii exhibits anti-inflammatory effects, partly due to secreted metabolites able to block nuclear factor-κB activation and the secretion of proinflammatory mediators (41). Oral administration of F. prausnitzii or of supernatant from F. prausnitzii cultures increased the production of IL-10 by blood mononuclear cells and reduced the production of the proinflammatory mediator-like IL-12 in the colon.','in vitro bacterial culture','RT-qPCR',20876719),(3586,'human','decrease','low-throughput','F. prausnitzii modulated the metabolome of the host blood by decreasing IL-1α, IL-2, and IL-5 concentrations.','in vivo experiment,in vitro bacterial culture','null',25900655),(3587,'human','decrease','low-throughput','F. prausnitzii modulated the metabolome of the host blood by decreasing IL-1α, IL-2, and IL-5 concentrations.','in vivo experiment,in vitro bacterial culture','null',25900655),(3588,'human','decrease','low-throughput','F. prausnitzii modulated the metabolome of the host blood by decreasing IL-1α, IL-2, and IL-5 concentrations.','in vivo experiment,in vitro bacterial culture','null',25900655),(3589,'human','decrease','low-throughput','Retinoic acid (RA), a vitamin A metabolite, regulates transcriptional programs that drive protective or pathogenic immune responses in the intestine, in a manner dependent on RA concentration. Vitamin A is obtained from diet and is metabolized by intestinal epithelial cells (IECs), which operate in intimate association with microbes and immune cells. Here we found that commensal bacteria belonging to class Clostridia modulate RA concentration in the gut by suppressing the expression of retinol dehydrogenase 7 (Rdh7) in IECs. Rdh7 expression and associated RA amounts were lower in the intestinal tissue of conventional mice, as compared to germ-free mice. Deletion of Rdh7 in IECs diminished RA signaling in immune cells, reduced the IL-22-dependent antimicrobial response, and enhanced resistance to colonization by Salmonella Typhimurium. Our findings define a regulatory circuit wherein bacterial regulation of IEC-intrinsic RA synthesis protects microbial communities in the gut from excessive immune activity, achieving a balance that prevents colonization by enteric pathogens.','in vitro bacterial culture','16S rRNA sequence,Western blot assay,RT-PCR',30566883),(3590,'human','increase','low-throughput','Retinoic acid (RA), a vitamin A metabolite, regulates transcriptional programs that drive protective or pathogenic immune responses in the intestine, in a manner dependent on RA concentration. Vitamin A is obtained from diet and is metabolized by intestinal epithelial cells (IECs), which operate in intimate association with microbes and immune cells. Here we found that commensal bacteria belonging to class Clostridia modulate RA concentration in the gut by suppressing the expression of retinol dehydrogenase 7 (Rdh7) in IECs. Rdh7 expression and associated RA amounts were lower in the intestinal tissue of conventional mice, as compared to germ-free mice. Deletion of Rdh7 in IECs diminished RA signaling in immune cells, reduced the IL-22-dependent antimicrobial response, and enhanced resistance to colonization by Salmonella Typhimurium. Our findings define a regulatory circuit wherein bacterial regulation of IEC-intrinsic RA synthesis protects microbial communities in the gut from excessive immune activity, achieving a balance that prevents colonization by enteric pathogens.','in vitro bacterial culture','16S rRNA sequence,Western blot assay,RT-PCR',30566883),(3591,'human','increase','low-throughput','Retinoic acid (RA), a vitamin A metabolite, regulates transcriptional programs that drive protective or pathogenic immune responses in the intestine, in a manner dependent on RA concentration. Vitamin A is obtained from diet and is metabolized by intestinal epithelial cells (IECs), which operate in intimate association with microbes and immune cells. Here we found that commensal bacteria belonging to class Clostridia modulate RA concentration in the gut by suppressing the expression of retinol dehydrogenase 7 (Rdh7) in IECs. Rdh7 expression and associated RA amounts were lower in the intestinal tissue of conventional mice, as compared to germ-free mice. Deletion of Rdh7 in IECs diminished RA signaling in immune cells, reduced the IL-22-dependent antimicrobial response, and enhanced resistance to colonization by Salmonella Typhimurium. Our findings define a regulatory circuit wherein bacterial regulation of IEC-intrinsic RA synthesis protects microbial communities in the gut from excessive immune activity, achieving a balance that prevents colonization by enteric pathogens.','in vitro bacterial culture','16S rRNA sequence,Western blot assay,RT-PCR',30566883),(3592,'human','increase','low-throughput','Our data demonstrate that a dysbiotic gut microbiota in RA patients, characterized by a decrease in Faecalibacterium and expansion of C. aerofaciens and E. lenta, could trigger inflammatory conditions in the gut that depend on the production of chemokines and IL-17A and compromise the gut epithelium integrity. It is possible that the inflammatory conditions can be modulated by prebiotics or probiotics. The therapeutic potential of the only probiotics, lactobacilli, used as treatment for RA is inconclusive, with some studies in favor of their use while others did not show significant improvement with lactobacilli using the American College of Rheumatology (ACR) response criteria for RA .','cell culture','16S rRNA sequence,RT-qPCR',27102666),(3593,'human','increase','low-throughput','Our data demonstrate that a dysbiotic gut microbiota in RA patients, characterized by a decrease in Faecalibacterium and expansion of C. aerofaciens and E. lenta, could trigger inflammatory conditions in the gut that depend on the production of chemokines and IL-17A and compromise the gut epithelium integrity. It is possible that the inflammatory conditions can be modulated by prebiotics or probiotics. The therapeutic potential of the only probiotics, lactobacilli, used as treatment for RA is inconclusive, with some studies in favor of their use while others did not show significant improvement with lactobacilli using the American College of Rheumatology (ACR) response criteria for RA .','cell culture','16S rRNA sequence,RT-qPCR',27102666),(3594,'human','increase','low-throughput','Our data demonstrate that a dysbiotic gut microbiota in RA patients, characterized by a decrease in Faecalibacterium and expansion of C. aerofaciens and E. lenta, could trigger inflammatory conditions in the gut that depend on the production of chemokines and IL-17A and compromise the gut epithelium integrity. It is possible that the inflammatory conditions can be modulated by prebiotics or probiotics. The therapeutic potential of the only probiotics, lactobacilli, used as treatment for RA is inconclusive, with some studies in favor of their use while others did not show significant improvement with lactobacilli using the American College of Rheumatology (ACR) response criteria for RA .','cell culture','16S rRNA sequence,RT-qPCR',27102666),(3595,'human','increase','low-throughput','Our data demonstrate that a dysbiotic gut microbiota in RA patients, characterized by a decrease in Faecalibacterium and expansion of C. aerofaciens and E. lenta, could trigger inflammatory conditions in the gut that depend on the production of chemokines and IL-17A and compromise the gut epithelium integrity. It is possible that the inflammatory conditions can be modulated by prebiotics or probiotics. The therapeutic potential of the only probiotics, lactobacilli, used as treatment for RA is inconclusive, with some studies in favor of their use while others did not show significant improvement with lactobacilli using the American College of Rheumatology (ACR) response criteria for RA .','cell culture','16S rRNA sequence,RT-qPCR',27102666),(3598,'human','increase','low-throughput','Here, we show that in colonic cell lines and in primary colonic cells, Enterococcus faecalis isolated from newborn babies possess the ability to regulate peroxisome proliferator-activated receptor-γ1 (PPARγ1) activity through phosphorylation. This results in elevated DNA binding and transcriptional activation of downstream target genes, including IL-10, a cytokine known to modulate innate immune function. Furthermore, phosphorylation appears tightly regulated as phospho-PPARγ1 becomes an immediate substrate for degradation possibly to curtail any extended transactivation. The involvement of PPARγ1 in a myriad of physiological processes further confirms that microflora-driven regulation might be important for a number of homeostatic strategies in the gut.','cell culture','Western blot assay,RT-PCR',18234854),(3599,'human','increase','low-throughput','Here, we show that in colonic cell lines and in primary colonic cells, Enterococcus faecalis isolated from newborn babies possess the ability to regulate peroxisome proliferator-activated receptor-γ1 (PPARγ1) activity through phosphorylation. This results in elevated DNA binding and transcriptional activation of downstream target genes, including IL-10, a cytokine known to modulate innate immune function. Furthermore, phosphorylation appears tightly regulated as phospho-PPARγ1 becomes an immediate substrate for degradation possibly to curtail any extended transactivation. The involvement of PPARγ1 in a myriad of physiological processes further confirms that microflora-driven regulation might be important for a number of homeostatic strategies in the gut.','cell culture','Western blot assay,RT-PCR',18234854),(3600,'human','increase','low-throughput','Here, we show that in colonic cell lines and in primary colonic cells, Enterococcus faecalis isolated from newborn babies possess the ability to regulate peroxisome proliferator-activated receptor-γ1 (PPARγ1) activity through phosphorylation. This results in elevated DNA binding and transcriptional activation of downstream target genes, including IL-10, a cytokine known to modulate innate immune function. Furthermore, phosphorylation appears tightly regulated as phospho-PPARγ1 becomes an immediate substrate for degradation possibly to curtail any extended transactivation. The involvement of PPARγ1 in a myriad of physiological processes further confirms that microflora-driven regulation might be important for a number of homeostatic strategies in the gut.','cell culture','Western blot assay,RT-PCR',18234854),(3601,'human','increase','low-throughput','Here, we show that in colonic cell lines and in primary colonic cells, Enterococcus faecalis isolated from newborn babies possess the ability to regulate peroxisome proliferator-activated receptor-γ1 (PPARγ1) activity through phosphorylation. This results in elevated DNA binding and transcriptional activation of downstream target genes, including IL-10, a cytokine known to modulate innate immune function. Furthermore, phosphorylation appears tightly regulated as phospho-PPARγ1 becomes an immediate substrate for degradation possibly to curtail any extended transactivation. The involvement of PPARγ1 in a myriad of physiological processes further confirms that microflora-driven regulation might be important for a number of homeostatic strategies in the gut.','cell culture','Western blot assay,RT-PCR',18234854),(3602,'human','decrease','low-throughput','Human and murine pregnancy were associated with reduced intestinal FXR signaling, with lower FGF19/15 and resultant increased hepatic bile acid synthesis. Terminal ileal ASBT protein was reduced in murine pregnancy. Cecal bile acid conjugation was reduced in pregnancy because of elevated bile salt hydrolase‐producing Bacteroidetes. CA supplementation induced intestinal FXR signaling, which was not abrogated by pregnancy, with strikingly similar changes to the microbiota and metabonome as identified in pregnancy. Conclusion: The altered intestinal microbiota of pregnancy enhance bile acid deconjugation, reducing ileal bile acid uptake and lowering FXR induction in enterocytes. This exacerbates the effects mediated by reduced bile acid uptake transporters in pregnancy. Thus, in pregnant women and mice, there is reduced FGF19/15‐mediated hepatic repression of hepatic bile acid synthesis, resulting in hypercholanemia.','in vivo experiment','RT-PCR,16S rRNA sequence',30983011),(3603,'human','increase','low-throughput','Human and murine pregnancy were associated with reduced intestinal FXR signaling, with lower FGF19/15 and resultant increased hepatic bile acid synthesis. Terminal ileal ASBT protein was reduced in murine pregnancy. Cecal bile acid conjugation was reduced in pregnancy because of elevated bile salt hydrolase‐producing Bacteroidetes. CA supplementation induced intestinal FXR signaling, which was not abrogated by pregnancy, with strikingly similar changes to the microbiota and metabonome as identified in pregnancy. Conclusion: The altered intestinal microbiota of pregnancy enhance bile acid deconjugation, reducing ileal bile acid uptake and lowering FXR induction in enterocytes. This exacerbates the effects mediated by reduced bile acid uptake transporters in pregnancy. Thus, in pregnant women and mice, there is reduced FGF19/15‐mediated hepatic repression of hepatic bile acid synthesis, resulting in hypercholanemia.','in vivo experiment','RT-PCR,16S rRNA sequence',30983011),(3604,'human','increase','low-throughput','The effect of Lactobacillus plantarum MB452 on tight junction integrity was determined by measuring trans-epithelial electrical resistance (TEER) across Caco-2 cell layers. L. plantarum MB452 caused a dose-dependent TEER increase across Caco-2 cell monolayers compared to control medium. Gene expression was compared in Caco-2 cells untreated or treated with L. plantarum MB452 for 10 hours. Caco-2 cell RNA was hybridised to human oligonucleotide arrays. Data was analysed using linear models and differently expressed genes were examined using pathway analysis tools. Nineteen tight junction-related genes had altered expression levels in response to L. plantarum MB452 (modified-P < 0.05, fold-change > 1.2), including those encoding occludin and its associated plaque proteins that anchor it to the cytoskeleton. L. plantarum MB452 also caused changes in tubulin and proteasome gene expression levels which may be linked to intestinal barrier function.','cell culture','null',21143932),(3605,'human','increase','low-throughput','The effect of Lactobacillus plantarum MB452 on tight junction integrity was determined by measuring trans-epithelial electrical resistance (TEER) across Caco-2 cell layers. L. plantarum MB452 caused a dose-dependent TEER increase across Caco-2 cell monolayers compared to control medium. Gene expression was compared in Caco-2 cells untreated or treated with L. plantarum MB452 for 10 hours. Caco-2 cell RNA was hybridised to human oligonucleotide arrays. Data was analysed using linear models and differently expressed genes were examined using pathway analysis tools. Nineteen tight junction-related genes had altered expression levels in response to L. plantarum MB452 (modified-P < 0.05, fold-change > 1.2), including those encoding occludin and its associated plaque proteins that anchor it to the cytoskeleton. L. plantarum MB452 also caused changes in tubulin and proteasome gene expression levels which may be linked to intestinal barrier function.','cell culture','null',21143932),(3606,'human','increase','low-throughput','The effect of Lactobacillus plantarum MB452 on tight junction integrity was determined by measuring trans-epithelial electrical resistance (TEER) across Caco-2 cell layers. L. plantarum MB452 caused a dose-dependent TEER increase across Caco-2 cell monolayers compared to control medium. Gene expression was compared in Caco-2 cells untreated or treated with L. plantarum MB452 for 10 hours. Caco-2 cell RNA was hybridised to human oligonucleotide arrays. Data was analysed using linear models and differently expressed genes were examined using pathway analysis tools. Nineteen tight junction-related genes had altered expression levels in response to L. plantarum MB452 (modified-P < 0.05, fold-change > 1.2), including those encoding occludin and its associated plaque proteins that anchor it to the cytoskeleton. L. plantarum MB452 also caused changes in tubulin and proteasome gene expression levels which may be linked to intestinal barrier function.','cell culture','null',21143932),(3607,'human','increase','low-throughput','The effect of Lactobacillus plantarum MB452 on tight junction integrity was determined by measuring trans-epithelial electrical resistance (TEER) across Caco-2 cell layers. L. plantarum MB452 caused a dose-dependent TEER increase across Caco-2 cell monolayers compared to control medium. Gene expression was compared in Caco-2 cells untreated or treated with L. plantarum MB452 for 10 hours. Caco-2 cell RNA was hybridised to human oligonucleotide arrays. Data was analysed using linear models and differently expressed genes were examined using pathway analysis tools. Nineteen tight junction-related genes had altered expression levels in response to L. plantarum MB452 (modified-P < 0.05, fold-change > 1.2), including those encoding occludin and its associated plaque proteins that anchor it to the cytoskeleton. L. plantarum MB452 also caused changes in tubulin and proteasome gene expression levels which may be linked to intestinal barrier function.','cell culture','null',21143932),(3608,'human','decrease','low-throughput','The effect of Lactobacillus plantarum MB452 on tight junction integrity was determined by measuring trans-epithelial electrical resistance (TEER) across Caco-2 cell layers. L. plantarum MB452 caused a dose-dependent TEER increase across Caco-2 cell monolayers compared to control medium. Gene expression was compared in Caco-2 cells untreated or treated with L. plantarum MB452 for 10 hours. Caco-2 cell RNA was hybridised to human oligonucleotide arrays. Data was analysed using linear models and differently expressed genes were examined using pathway analysis tools. Nineteen tight junction-related genes had altered expression levels in response to L. plantarum MB452 (modified-P < 0.05, fold-change > 1.2), including those encoding occludin and its associated plaque proteins that anchor it to the cytoskeleton. L. plantarum MB452 also caused changes in tubulin and proteasome gene expression levels which may be linked to intestinal barrier function.','cell culture','null',21143932),(3609,'human','increase','low-throughput','In this study, we investigated the effects of LA and LA-derived fatty acids on the activation of peroxisome proliferator-activated receptors (PPARs) which regulate whole-body energy metabolism. None of the fatty acids activated PPARd, whereas almost all activated PPARa in luciferase assays. Two fatty acids potently activated PPARg, a master regulator of adipocyte differentiation, with 10-oxo-12(Z)-octadecenoic acid (KetoA) having the most potency. In 3T3- L1 cells, KetoA induced adipocyte differentiation via the activation of PPARg, and increased adiponectin production and insulin-stimulated glucose uptake. These findings suggest that fatty acids, including KetoA, generated in gut by lactic acid bacteria may be involved in the regulation of host energy metabolism.','cell culture','RT-PCR',25749343),(3611,'human','decrease','low-throughput','The addition of the EPM had no significant effect on transcription of IL-1β, TNF-α or IL-10 in hDCs stimulated with L. plantarum (not shown) but significantly decreased the transcript levels of IL-12 at 20h by about 2 fold.','in vitro bacterial culture','RT-qPCR',25910186),(3612,'human','decrease','low-throughput','The addition of the EPM had no significant effect on transcription of IL-1β, TNF-α or IL-10 in hDCs stimulated with L. plantarum (not shown) but significantly decreased the transcript levels of IL-12 at 20h by about 2 fold.','in vitro bacterial culture','RT-qPCR',25910186),(3613,'human','decrease','low-throughput','The EPM of F. prausnitzii HTF-F decreases transcription and production of pro-inflammatory IL-12p70 in L. plantarum-activated hDCs','in vitro bacterial culture','RT-qPCR',25910186),(3623,'human','decrease','low-throughput','F. prausnitzii was found to be underrepresented in the gut microbiotas of patients with Crohn\'s disease, active ulcerative colitis, alternating-type irritable bowel syndrome (IBS-A), and diabetes type II. One of its main fermentation products, the short-chain fatty acid butyrate, serves as the main energy source for colonocytes and has anti-inflammatory properties . F. prausnitzii has been proposed as a potential probiotic for treatment of gut inflammation, as the microbe stimulates the expression of the anti-inflammatory cytokine interleukin 10 (IL-10) in peripheral blood mononuclear cells in vitro. Furthermore, cell supernatants of F. prausnitzii culture reduced the secretion of proinflammatory IL-8 and strongly inhibited NF-κB activation in cancer cells . Butyrate alone did not provoke the observed inhibitory effect , demonstrating that F. prausnitzii likely secretes an unknown anti-inflammatory metabolite apart from butyrate.','in vitro bacterial culture','LC/MS analysis',25002542),(3624,'human','decrease','low-throughput','F. prausnitzii was found to be underrepresented in the gut microbiotas of patients with Crohn\'s disease, active ulcerative colitis, alternating-type irritable bowel syndrome (IBS-A), and diabetes type II. One of its main fermentation products, the short-chain fatty acid butyrate, serves as the main energy source for colonocytes and has anti-inflammatory properties . F. prausnitzii has been proposed as a potential probiotic for treatment of gut inflammation, as the microbe stimulates the expression of the anti-inflammatory cytokine interleukin 10 (IL-10) in peripheral blood mononuclear cells in vitro. Furthermore, cell supernatants of F. prausnitzii culture reduced the secretion of proinflammatory IL-8 and strongly inhibited NF-κB activation in cancer cells . Butyrate alone did not provoke the observed inhibitory effect , demonstrating that F. prausnitzii likely secretes an unknown anti-inflammatory metabolite apart from butyrate.','in vitro bacterial culture','LC/MS analysis',25002542),(3625,'human','increase','low-throughput','F. prausnitzii was found to be underrepresented in the gut microbiotas of patients with Crohn\'s disease, active ulcerative colitis, alternating-type irritable bowel syndrome (IBS-A), and diabetes type II. One of its main fermentation products, the short-chain fatty acid butyrate, serves as the main energy source for colonocytes and has anti-inflammatory properties . F. prausnitzii has been proposed as a potential probiotic for treatment of gut inflammation, as the microbe stimulates the expression of the anti-inflammatory cytokine interleukin 10 (IL-10) in peripheral blood mononuclear cells in vitro. Furthermore, cell supernatants of F. prausnitzii culture reduced the secretion of proinflammatory IL-8 and strongly inhibited NF-κB activation in cancer cells . Butyrate alone did not provoke the observed inhibitory effect , demonstrating that F. prausnitzii likely secretes an unknown anti-inflammatory metabolite apart from butyrate.','in vitro bacterial culture','LC/MS analysis',25002542),(3629,'human','increase','low-throughput','Here we reveal a relationship between gut microbiota and caspase recruitment domain family member 9 (CARD9), a susceptibility gene for inflammatory bowel disease (IBD) that functions in the immune response against microorganisms. CARD9 promotes recovery from colitis by promoting interleukin (IL)-22 production, and Card9�?�?mice are more susceptible to colitis. The microbiota is altered in Card9�?�?mice, and transfer of the microbiota from Card9�?�?to wild-type, germ-free recipients increases their susceptibility to colitis. The microbiota from Card9�?�?mice fails to metabolize tryptophan into metabolites that act as aryl hydrocarbon receptor (AHR) ligands. Intestinal inflammation is attenuated after inoculation of mice with three Lactobacillus strains capable of metabolizing tryptophan or by treatment with an AHR agonist. Reduced production of AHR ligands is also observed in the microbiota from individuals with IBD, particularly in those with CARD9 risk alleles associated with IBD. Our findings reveal that host genes affect the composition and function of the gut microbiota, altering the production of microbial metabolites and intestinal inflammation','in vitro bacterial culture','qPCR,16S rDNA sequence,HPLC analysis',27158904),(3630,'human','increase','low-throughput','Here we reveal a relationship between gut microbiota and caspase recruitment domain family member 9 (CARD9), a susceptibility gene for inflammatory bowel disease (IBD) that functions in the immune response against microorganisms. CARD9 promotes recovery from colitis by promoting interleukin (IL)-22 production, and Card9�?�?mice are more susceptible to colitis. The microbiota is altered in Card9�?�?mice, and transfer of the microbiota from Card9�?�?to wild-type, germ-free recipients increases their susceptibility to colitis. The microbiota from Card9�?�?mice fails to metabolize tryptophan into metabolites that act as aryl hydrocarbon receptor (AHR) ligands. Intestinal inflammation is attenuated after inoculation of mice with three Lactobacillus strains capable of metabolizing tryptophan or by treatment with an AHR agonist. Reduced production of AHR ligands is also observed in the microbiota from individuals with IBD, particularly in those with CARD9 risk alleles associated with IBD. Our findings reveal that host genes affect the composition and function of the gut microbiota, altering the production of microbial metabolites and intestinal inflammation','in vitro bacterial culture','qPCR,16S rDNA sequence,HPLC analysis',27158904),(3631,'human','decrease','low-throughput','Plasma from rats treated for seven days with the non-absorbed antibiotic vancomycin or a mixture of streptomycin, neomycin, polymyxin B and bacitracin was analyzed using mass spectrometry-based metabolite profiling platforms. Antibiotic-induced changes in the abundance of individual groups of intestinal microbiota dramatically altered the host\'s metabolism. Hierarchical clustering of dissimilarities separated the levels of 284 identified metabolites from treated vs. untreated rats; 193 were altered by the antibiotic treatments with a tendency towards decreased metabolite levels. Catabolism of the aromatic amino acids phenylalanine, tryptophan and tyrosine was the most affected pathway comprising 33 affected metabolites. Vancomycin-treated rats were administered amino acid metabolites prior to ischemia/reperfusion studies. Oral or intravenous pretreatment of rats with these amino acid metabolites abolished the decrease in infarct size conferred by vancomycin. Cardioprotection by vancomycin is mediated by activation of pro-survival kinases Akt, p42/44 MAPK and p38 MAPK, as well as activation of KATP channels, classical mediators and effectors of pharmacologic cardioprotection. These proteins are components of known intersecting and interdependent signaling pathways in the myocardium that determine severity of myocardial infarction.','in vivo experiment','RT-PCR',27505423),(3645,'human','decrease','low-throughput','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.','cell culture、in vitro bacterial culture、control experiment','Western blot assay、RT-PCR、HPLC analysis',21889493),(3646,'human','increase','low-throughput','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.','cell culture、in vitro bacterial culture、control experiment','Western blot assay、RT-PCR、HPLC analysis',21889493),(3647,'human','increase','low-throughput','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.','cell culture、in vitro bacterial culture、control experiment','Western blot assay、RT-PCR、HPLC analysis',21889493),(3648,'human','decrease','low-throughput','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.','cell culture、in vitro bacterial culture、control experiment','Western blot assay、RT-PCR、HPLC analysis',21889493),(3649,'human','increase','low-throughput','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.','cell culture、in vitro bacterial culture、control experiment','Western blot assay、RT-PCR、HPLC analysis',21889493),(3650,'human','increase','low-throughput','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.','cell culture、in vitro bacterial culture、control experiment','Western blot assay、RT-PCR、HPLC analysis',21889493),(3651,'human','decrease','low-throughput','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.','cell culture、in vitro bacterial culture、control experiment','Western blot assay、RT-PCR、HPLC analysis',21889493),(3652,'human','increase','low-throughput','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.','cell culture、in vitro bacterial culture、control experiment','Western blot assay、RT-PCR、HPLC analysis',21889493),(3653,'human','increase','low-throughput','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.','cell culture、in vitro bacterial culture、control experiment','Western blot assay、RT-PCR、HPLC analysis',21889493),(3654,'human','decrease','low-throughput','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.','cell culture、in vitro bacterial culture、control experiment','Western blot assay、RT-PCR、HPLC analysis',21889493),(3655,'human','increase','low-throughput','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.','cell culture、in vitro bacterial culture、control experiment','Western blot assay、RT-PCR、HPLC analysis',21889493),(3656,'human','increase','low-throughput','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.','cell culture、in vitro bacterial culture、control experiment','Western blot assay、RT-PCR、HPLC analysis',21889493),(3657,'human','increase','low-throughput','S. macedonicus ACA-DC 198 was the strongest inducer of the three pro inflammatory cytokines IL-12 (337 pg mL�?1), TNF-α (36721 pg mL�?) and IFN-γ (37684 pgmL�?). L. plantarum ACA-DC 287 induced lower, but still considerable, amounts of IL-12, TNF-α and IFN-γ (153, 19307 and 28694 pg mL�?, respectively), with higher levels of the anti-inflammatory cytokine IL-10 (1498 pg mL�?) compared to S. macedonicus ACA-DC 198 (777 pg mL�?). The highest secretion of IL-10 (2815 pg mL�?) was displayed when PBMCs were incubated with L. fermentum ACA-DC 179, which also induced notable levels of TNF-α (3579 pg mL�?), but low levels of IFN-γ (115 pg mL�?) and no secretion for IL-12.','cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment','null',18077037),(3658,'human','increase','low-throughput','S. macedonicus ACA-DC 198 was the strongest inducer of the three pro inflammatory cytokines IL-12 (337 pg mL�?1), TNF-α (36721 pg mL�?) and IFN-γ (37684 pgmL�?). L. plantarum ACA-DC 287 induced lower, but still considerable, amounts of IL-12, TNF-α and IFN-γ (153, 19307 and 28694 pg mL�?, respectively), with higher levels of the anti-inflammatory cytokine IL-10 (1498 pg mL�?) compared to S. macedonicus ACA-DC 198 (777 pg mL�?). The highest secretion of IL-10 (2815 pg mL�?) was displayed when PBMCs were incubated with L. fermentum ACA-DC 179, which also induced notable levels of TNF-α (3579 pg mL�?), but low levels of IFN-γ (115 pg mL�?) and no secretion for IL-12.','cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment','null',18077037),(3659,'human','increase','low-throughput','S. macedonicus ACA-DC 198 was the strongest inducer of the three pro inflammatory cytokines IL-12 (337 pg mL�?1), TNF-α (36721 pg mL�?) and IFN-γ (37684 pgmL�?). L. plantarum ACA-DC 287 induced lower, but still considerable, amounts of IL-12, TNF-α and IFN-γ (153, 19307 and 28694 pg mL�?, respectively), with higher levels of the anti-inflammatory cytokine IL-10 (1498 pg mL�?) compared to S. macedonicus ACA-DC 198 (777 pg mL�?). The highest secretion of IL-10 (2815 pg mL�?) was displayed when PBMCs were incubated with L. fermentum ACA-DC 179, which also induced notable levels of TNF-α (3579 pg mL�?), but low levels of IFN-γ (115 pg mL�?) and no secretion for IL-12.','cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment','null',18077037),(3660,'human','increase','low-throughput','S. macedonicus ACA-DC 198 was the strongest inducer of the three pro inflammatory cytokines IL-12 (337 pg mL�?1), TNF-α (36721 pg mL�?) and IFN-γ (37684 pgmL�?). L. plantarum ACA-DC 287 induced lower, but still considerable, amounts of IL-12, TNF-α and IFN-γ (153, 19307 and 28694 pg mL�?, respectively), with higher levels of the anti-inflammatory cytokine IL-10 (1498 pg mL�?) compared to S. macedonicus ACA-DC 198 (777 pg mL�?). The highest secretion of IL-10 (2815 pg mL�?) was displayed when PBMCs were incubated with L. fermentum ACA-DC 179, which also induced notable levels of TNF-α (3579 pg mL�?), but low levels of IFN-γ (115 pg mL�?) and no secretion for IL-12.','cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment','null',18077037),(3661,'human','increase','low-throughput','S. macedonicus ACA-DC 198 was the strongest inducer of the three pro inflammatory cytokines IL-12 (337 pg mL�?1), TNF-α (36721 pg mL�?) and IFN-γ (37684 pgmL�?). L. plantarum ACA-DC 287 induced lower, but still considerable, amounts of IL-12, TNF-α and IFN-γ (153, 19307 and 28694 pg mL�?, respectively), with higher levels of the anti-inflammatory cytokine IL-10 (1498 pg mL�?) compared to S. macedonicus ACA-DC 198 (777 pg mL�?). The highest secretion of IL-10 (2815 pg mL�?) was displayed when PBMCs were incubated with L. fermentum ACA-DC 179, which also induced notable levels of TNF-α (3579 pg mL�?), but low levels of IFN-γ (115 pg mL�?) and no secretion for IL-12.','cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment','null',18077037),(3662,'human','increase','low-throughput','S. macedonicus ACA-DC 198 was the strongest inducer of the three pro inflammatory cytokines IL-12 (337 pg mL�?1), TNF-α (36721 pg mL�?) and IFN-γ (37684 pgmL�?). L. plantarum ACA-DC 287 induced lower, but still considerable, amounts of IL-12, TNF-α and IFN-γ (153, 19307 and 28694 pg mL�?, respectively), with higher levels of the anti-inflammatory cytokine IL-10 (1498 pg mL�?) compared to S. macedonicus ACA-DC 198 (777 pg mL�?). The highest secretion of IL-10 (2815 pg mL�?) was displayed when PBMCs were incubated with L. fermentum ACA-DC 179, which also induced notable levels of TNF-α (3579 pg mL�?), but low levels of IFN-γ (115 pg mL�?) and no secretion for IL-12.','cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment','null',18077037),(3663,'human','increase','low-throughput','S. macedonicus ACA-DC 198 was the strongest inducer of the three pro inflammatory cytokines IL-12 (337 pg mL�?1), TNF-α (36721 pg mL�?) and IFN-γ (37684 pgmL�?). L. plantarum ACA-DC 287 induced lower, but still considerable, amounts of IL-12, TNF-α and IFN-γ (153, 19307 and 28694 pg mL�?, respectively), with higher levels of the anti-inflammatory cytokine IL-10 (1498 pg mL�?) compared to S. macedonicus ACA-DC 198 (777 pg mL�?). The highest secretion of IL-10 (2815 pg mL�?) was displayed when PBMCs were incubated with L. fermentum ACA-DC 179, which also induced notable levels of TNF-α (3579 pg mL�?), but low levels of IFN-γ (115 pg mL�?) and no secretion for IL-12.','cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment','null',18077037),(3664,'human','increase','low-throughput','S. macedonicus ACA-DC 198 was the strongest inducer of the three pro inflammatory cytokines IL-12 (337 pg mL�?1), TNF-α (36721 pg mL�?) and IFN-γ (37684 pgmL�?). L. plantarum ACA-DC 287 induced lower, but still considerable, amounts of IL-12, TNF-α and IFN-γ (153, 19307 and 28694 pg mL�?, respectively), with higher levels of the anti-inflammatory cytokine IL-10 (1498 pg mL�?) compared to S. macedonicus ACA-DC 198 (777 pg mL�?). The highest secretion of IL-10 (2815 pg mL�?) was displayed when PBMCs were incubated with L. fermentum ACA-DC 179, which also induced notable levels of TNF-α (3579 pg mL�?), but low levels of IFN-γ (115 pg mL�?) and no secretion for IL-12.','cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment','null',18077037),(3665,'human','increase','low-throughput','S. macedonicus ACA-DC 198 was the strongest inducer of the three pro inflammatory cytokines IL-12 (337 pg mL�?1), TNF-α (36721 pg mL�?) and IFN-γ (37684 pgmL�?). L. plantarum ACA-DC 287 induced lower, but still considerable, amounts of IL-12, TNF-α and IFN-γ (153, 19307 and 28694 pg mL�?, respectively), with higher levels of the anti-inflammatory cytokine IL-10 (1498 pg mL�?) compared to S. macedonicus ACA-DC 198 (777 pg mL�?). The highest secretion of IL-10 (2815 pg mL�?) was displayed when PBMCs were incubated with L. fermentum ACA-DC 179, which also induced notable levels of TNF-α (3579 pg mL�?), but low levels of IFN-γ (115 pg mL�?) and no secretion for IL-12.','cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment','null',18077037),(3666,'human','decrease','low-throughput','B. infantis and MO decreased hepatic CYP7A1 and induced Sult2a1, Sult2a2, and Sult2a3 suggesting reduced BA synthesis and increased detoxification. Furthermore, B. infantis and MO increased ileal BA membrane receptor TGR5-regulated signaling. Together, via BA-regulated signaling, synbiotics B. infantis and MO have their unique and combined effects in reversing NASH.','in vitro bacterial culture','qPCR,LC/MS analysis',29800811),(3676,'human','increase','low-throughput','B. infantis and MO decreased hepatic CYP7A1 and induced Sult2a1, Sult2a2, and Sult2a3 suggesting reduced BA synthesis and increased detoxification. Furthermore, B. infantis and MO increased ileal BA membrane receptor TGR5-regulated signaling. Together, via BA-regulated signaling, synbiotics B. infantis and MO have their unique and combined effects in reversing NASH.','in vitro bacterial culture','qPCR,LC/MS analysis',11040423),(3677,'human','increase','low-throughput','There was a significant difference in TLR2 signalling induced by live and UV‐killed bacteria. However, the activation by live bacteria was only 1.1‐fold higher than UV‐killed bacteria; therefore, the difference was less than that seen in apical anaerobic conditions and may not be biologically relevant.','in vitro bacterial culture,cell culture,control experiment','null',29112296),(3678,'human','increase','low-throughput','For the activation of TLR2/6, the interaction between the environment and live/dead status was significant as shown in Figure 5b(p < .001). In apical anaerobic conditions, TLR2/6 activation by live F. prausnitzii was 1.6‐fold higher than by UV‐killed bacteria.','in vitro bacterial culture,cell culture,control experiment','null',29112296),(3679,'human','decrease','low-throughput','There was no TLR4 activation by either live or UV‐killed F. prausnitzii in both conventional and apical anaerobic conditions. The raw luminescence values for the activation of TLR2, TLR2/6,TLR4, and the activation of the control cell line (TLR‐null) are shown in Figure S3.','in vitro bacterial culture,cell culture,control experiment','null',29112296),(3712,'human','decrease','low-throughput','The E.limosum produced butyrate, acetate, propionate, and lactate at 0.25, 1.0, 0.025 and 0.07 mmol/L, respectively in medium. At this concentration, each acid had no growth stimulating activity on T84 cells; however, when these acids were mixed together at the above levels, it showed significantly high activity than control. Except for lactate, these acids significantly attenuated IL-6 production at just 0.1 mmol/L. In addition, under TNF-a stimulation, butyrate attenuated the production of TLR4 mRNA. The treatment with E.limosum significantly attenuated clinical and histological scores of colitis with an increase of cecal butyrate levels, compared with the control group.','in vitro bacterial culture,cell culture','RT-PCR,fluorescence microscopy',16534848),(3713,'human','decrease','low-throughput','The E.limosum produced butyrate, acetate, propionate, and lactate at 0.25, 1.0, 0.025 and 0.07 mmol/L, respectively in medium. At this concentration, each acid had no growth stimulating activity on T84 cells; however, when these acids were mixed together at the above levels, it showed significantly high activity than control. Except for lactate, these acids significantly attenuated IL-6 production at just 0.1 mmol/L. In addition, under TNF-a stimulation, butyrate attenuated the production of TLR4 mRNA. The treatment with E.limosum significantly attenuated clinical and histological scores of colitis with an increase of cecal butyrate levels, compared with the control group.','in vitro bacterial culture,cell culture','RT-PCR,fluorescence microscopy',16534848),(3714,'human','decrease','low-throughput','The E.limosum produced butyrate, acetate, propionate, and lactate at 0.25, 1.0, 0.025 and 0.07 mmol/L, respectively in medium. At this concentration, each acid had no growth stimulating activity on T84 cells; however, when these acids were mixed together at the above levels, it showed significantly high activity than control. Except for lactate, these acids significantly attenuated IL-6 production at just 0.1 mmol/L. In addition, under TNF-a stimulation, butyrate attenuated the production of TLR4 mRNA. The treatment with E.limosum significantly attenuated clinical and histological scores of colitis with an increase of cecal butyrate levels, compared with the control group.','in vitro bacterial culture,cell culture','RT-PCR,fluorescence microscopy',16534848),(3715,'human','decrease','low-throughput','The E.limosum produced butyrate, acetate, propionate, and lactate at 0.25, 1.0, 0.025 and 0.07 mmol/L, respectively in medium. At this concentration, each acid had no growth stimulating activity on T84 cells; however, when these acids were mixed together at the above levels, it showed significantly high activity than control. Except for lactate, these acids significantly attenuated IL-6 production at just 0.1 mmol/L. In addition, under TNF-a stimulation, butyrate attenuated the production of TLR4 mRNA. The treatment with E.limosum significantly attenuated clinical and histological scores of colitis with an increase of cecal butyrate levels, compared with the control group.','in vitro bacterial culture,cell culture','RT-PCR,fluorescence microscopy',16534848),(3727,'human','decrease','low-throughput','The commensal bacteria Escherichia coli, Bifidobacterium adolescentis, Bacteroides vulgatus, Bacteroides distasonis, and Streptococcus salivarius were screened for adhesion to, invasion of, and toxicity for host epithelial cells (EC), and the strains were tested for their ability to inhibit Y. enterocolitica-induced NF-kappaB activation. Additionally, B. adolescentis was administered to mice orally infected with Y. enterocolitica and to mice with mucosae impaired by DSS treatment. None of the commensal bacteria tested was toxic for or invaded the EC. B. adolescentis, B. distasonis, B. vulgatus, and S. salivarius inhibited the Y. enterocolitica-induced NF-kappaB activation and interleukin-8 production in EC. In line with these findings, B. adolescentis-fed mice had significantly lower results for mean pathogen burden in the visceral organs, intestinal tumor necrosis factor alpha mRNA expression, and loss of body weight upon oral infection with Y. enterocolitica. In addition, the administration of B. adolescentis decelerated inflammation upon DSS treatment in mice.','in vitro bacterial culture,cell culture','RT-PCR',17485456),(3728,'human','decrease','low-throughput','The commensal bacteria Escherichia coli, Bifidobacterium adolescentis, Bacteroides vulgatus, Bacteroides distasonis, and Streptococcus salivarius were screened for adhesion to, invasion of, and toxicity for host epithelial cells (EC), and the strains were tested for their ability to inhibit Y. enterocolitica-induced NF-kappaB activation. Additionally, B. adolescentis was administered to mice orally infected with Y. enterocolitica and to mice with mucosae impaired by DSS treatment. None of the commensal bacteria tested was toxic for or invaded the EC. B. adolescentis, B. distasonis, B. vulgatus, and S. salivarius inhibited the Y. enterocolitica-induced NF-kappaB activation and interleukin-8 production in EC. In line with these findings, B. adolescentis-fed mice had significantly lower results for mean pathogen burden in the visceral organs, intestinal tumor necrosis factor alpha mRNA expression, and loss of body weight upon oral infection with Y. enterocolitica. In addition, the administration of B. adolescentis decelerated inflammation upon DSS treatment in mice.','in vitro bacterial culture,cell culture','RT-PCR',17485456),(3729,'human','decrease','low-throughput','The commensal bacteria Escherichia coli, Bifidobacterium adolescentis, Bacteroides vulgatus, Bacteroides distasonis, and Streptococcus salivarius were screened for adhesion to, invasion of, and toxicity for host epithelial cells (EC), and the strains were tested for their ability to inhibit Y. enterocolitica-induced NF-kappaB activation. Additionally, B. adolescentis was administered to mice orally infected with Y. enterocolitica and to mice with mucosae impaired by DSS treatment. None of the commensal bacteria tested was toxic for or invaded the EC. B. adolescentis, B. distasonis, B. vulgatus, and S. salivarius inhibited the Y. enterocolitica-induced NF-kappaB activation and interleukin-8 production in EC. In line with these findings, B. adolescentis-fed mice had significantly lower results for mean pathogen burden in the visceral organs, intestinal tumor necrosis factor alpha mRNA expression, and loss of body weight upon oral infection with Y. enterocolitica. In addition, the administration of B. adolescentis decelerated inflammation upon DSS treatment in mice.','in vitro bacterial culture,cell culture','RT-PCR',17485456),(3730,'human','decrease','low-throughput','The commensal bacteria Escherichia coli, Bifidobacterium adolescentis, Bacteroides vulgatus, Bacteroides distasonis, and Streptococcus salivarius were screened for adhesion to, invasion of, and toxicity for host epithelial cells (EC), and the strains were tested for their ability to inhibit Y. enterocolitica-induced NF-kappaB activation. Additionally, B. adolescentis was administered to mice orally infected with Y. enterocolitica and to mice with mucosae impaired by DSS treatment. None of the commensal bacteria tested was toxic for or invaded the EC. B. adolescentis, B. distasonis, B. vulgatus, and S. salivarius inhibited the Y. enterocolitica-induced NF-kappaB activation and interleukin-8 production in EC. In line with these findings, B. adolescentis-fed mice had significantly lower results for mean pathogen burden in the visceral organs, intestinal tumor necrosis factor alpha mRNA expression, and loss of body weight upon oral infection with Y. enterocolitica. In addition, the administration of B. adolescentis decelerated inflammation upon DSS treatment in mice.','in vitro bacterial culture,cell culture','RT-PCR',17485456),(3731,'human','decrease','low-throughput','The commensal bacteria Escherichia coli, Bifidobacterium adolescentis, Bacteroides vulgatus, Bacteroides distasonis, and Streptococcus salivarius were screened for adhesion to, invasion of, and toxicity for host epithelial cells (EC), and the strains were tested for their ability to inhibit Y. enterocolitica-induced NF-kappaB activation. Additionally, B. adolescentis was administered to mice orally infected with Y. enterocolitica and to mice with mucosae impaired by DSS treatment. None of the commensal bacteria tested was toxic for or invaded the EC. B. adolescentis, B. distasonis, B. vulgatus, and S. salivarius inhibited the Y. enterocolitica-induced NF-kappaB activation and interleukin-8 production in EC. In line with these findings, B. adolescentis-fed mice had significantly lower results for mean pathogen burden in the visceral organs, intestinal tumor necrosis factor alpha mRNA expression, and loss of body weight upon oral infection with Y. enterocolitica. In addition, the administration of B. adolescentis decelerated inflammation upon DSS treatment in mice.','in vitro bacterial culture,cell culture','RT-PCR',17485456),(3732,'human','decrease','low-throughput','The commensal bacteria Escherichia coli, Bifidobacterium adolescentis, Bacteroides vulgatus, Bacteroides distasonis, and Streptococcus salivarius were screened for adhesion to, invasion of, and toxicity for host epithelial cells (EC), and the strains were tested for their ability to inhibit Y. enterocolitica-induced NF-kappaB activation. Additionally, B. adolescentis was administered to mice orally infected with Y. enterocolitica and to mice with mucosae impaired by DSS treatment. None of the commensal bacteria tested was toxic for or invaded the EC. B. adolescentis, B. distasonis, B. vulgatus, and S. salivarius inhibited the Y. enterocolitica-induced NF-kappaB activation and interleukin-8 production in EC. In line with these findings, B. adolescentis-fed mice had significantly lower results for mean pathogen burden in the visceral organs, intestinal tumor necrosis factor alpha mRNA expression, and loss of body weight upon oral infection with Y. enterocolitica. In addition, the administration of B. adolescentis decelerated inflammation upon DSS treatment in mice.','in vitro bacterial culture,cell culture','RT-PCR',17485456),(3733,'human','decrease','low-throughput','The commensal bacteria Escherichia coli, Bifidobacterium adolescentis, Bacteroides vulgatus, Bacteroides distasonis, and Streptococcus salivarius were screened for adhesion to, invasion of, and toxicity for host epithelial cells (EC), and the strains were tested for their ability to inhibit Y. enterocolitica-induced NF-kappaB activation. Additionally, B. adolescentis was administered to mice orally infected with Y. enterocolitica and to mice with mucosae impaired by DSS treatment. None of the commensal bacteria tested was toxic for or invaded the EC. B. adolescentis, B. distasonis, B. vulgatus, and S. salivarius inhibited the Y. enterocolitica-induced NF-kappaB activation and interleukin-8 production in EC. In line with these findings, B. adolescentis-fed mice had significantly lower results for mean pathogen burden in the visceral organs, intestinal tumor necrosis factor alpha mRNA expression, and loss of body weight upon oral infection with Y. enterocolitica. In addition, the administration of B. adolescentis decelerated inflammation upon DSS treatment in mice.','in vitro bacterial culture,cell culture','RT-PCR',17485456),(3734,'human','decrease','low-throughput','The commensal bacteria Escherichia coli, Bifidobacterium adolescentis, Bacteroides vulgatus, Bacteroides distasonis, and Streptococcus salivarius were screened for adhesion to, invasion of, and toxicity for host epithelial cells (EC), and the strains were tested for their ability to inhibit Y. enterocolitica-induced NF-kappaB activation. Additionally, B. adolescentis was administered to mice orally infected with Y. enterocolitica and to mice with mucosae impaired by DSS treatment. None of the commensal bacteria tested was toxic for or invaded the EC. B. adolescentis, B. distasonis, B. vulgatus, and S. salivarius inhibited the Y. enterocolitica-induced NF-kappaB activation and interleukin-8 production in EC. In line with these findings, B. adolescentis-fed mice had significantly lower results for mean pathogen burden in the visceral organs, intestinal tumor necrosis factor alpha mRNA expression, and loss of body weight upon oral infection with Y. enterocolitica. In addition, the administration of B. adolescentis decelerated inflammation upon DSS treatment in mice.','in vitro bacterial culture,cell culture','RT-PCR',17485456),(3735,'human','decrease','low-throughput','The commensal bacteria Escherichia coli, Bifidobacterium adolescentis, Bacteroides vulgatus, Bacteroides distasonis, and Streptococcus salivarius were screened for adhesion to, invasion of, and toxicity for host epithelial cells (EC), and the strains were tested for their ability to inhibit Y. enterocolitica-induced NF-kappaB activation. Additionally, B. adolescentis was administered to mice orally infected with Y. enterocolitica and to mice with mucosae impaired by DSS treatment. None of the commensal bacteria tested was toxic for or invaded the EC. B. adolescentis, B. distasonis, B. vulgatus, and S. salivarius inhibited the Y. enterocolitica-induced NF-kappaB activation and interleukin-8 production in EC. In line with these findings, B. adolescentis-fed mice had significantly lower results for mean pathogen burden in the visceral organs, intestinal tumor necrosis factor alpha mRNA expression, and loss of body weight upon oral infection with Y. enterocolitica. In addition, the administration of B. adolescentis decelerated inflammation upon DSS treatment in mice.','in vitro bacterial culture,cell culture','RT-PCR',17485456),(3736,'human','decrease','low-throughput','L. plantarum prevents EIEC-induced expression of Claudin-1, Occludin, JAM-1 and ZO-1 proteins.','in vitro bacterial culture','fluorescence microscopy',19331693),(3737,'human','decrease','low-throughput','L. plantarum prevents EIEC-induced expression of Claudin-1, Occludin, JAM-1 and ZO-1 proteins.','in vitro bacterial culture','fluorescence microscopy',19331693),(3738,'human','decrease','low-throughput','L. plantarum prevents EIEC-induced expression of Claudin-1, Occludin, JAM-1 and ZO-1 proteins.','in vitro bacterial culture','fluorescence microscopy',19331693),(3739,'human','decrease','low-throughput','L. plantarum prevents EIEC-induced expression of Claudin-1, Occludin, JAM-1 and ZO-1 proteins.','in vitro bacterial culture','fluorescence microscopy',19331693),(3748,'human','increase','low-throughput','Dietary and bacterial metabolites influence immune responses. This raises the question whether the increased incidence of allergies, asthma, some autoimmune diseases, cardiovascular disease, and others might relate to intake of unhealthy foods, and the decreased intake of dietary fiber. In recent years, new knowledge on the molecular mechanisms underpinning a \'diet-gut microbiota-physiology axis\' has emerged to substantiate this idea. Fiber is fermented to short chain fatty acids (SCFAs), particularly acetate, butyrate, and propionate. These metabolites bind \'metabolite-sensing\' G-protein-coupled receptors such as GPR43, GPR41, and GPR109A. These receptors play fundamental roles in the promotion of gut homeostasis and the regulation of inflammatory responses. For instance, these receptors and their metabolites influence Treg biology, epithelial integrity, gut homeostasis, DC biology, and IgA antibody responses. The SCFAs also influence gene transcription in many cells and tissues, through their inhibition of histone deacetylase expression or function.','in vitro bacterial culture','RT-PCR',28658542),(3749,'human','increase','low-throughput','In this study, we investigated the effects of novel fatty acid metabolite derivatives of linoleic acid generated by the gut lactic acid bacteria Lactobacillus plantarum on the Nrf2-ARE pathway. 10-Oxo-trans-11-octadecenoic acid (KetoC) protected HepG2 cells from cytotoxicity induced by hydrogen peroxide. KetoC also significantly increased cellular Nrf2 protein levels, ARE-dependent transcription, and the gene expression of antioxidative enzymes such as heme oxygenase-1 (HO-1), glutamate-cysteine ligase modifier subunit (GCLM), and Nad(p)h: quinone oxidoreductase 1 (NQO1) in HepG2 cells.','cell culture','null',26879219),(3750,'human','increase','low-throughput','In this study, we investigated the effects of novel fatty acid metabolite derivatives of linoleic acid generated by the gut lactic acid bacteria Lactobacillus plantarum on the Nrf2-ARE pathway. 10-Oxo-trans-11-octadecenoic acid (KetoC) protected HepG2 cells from cytotoxicity induced by hydrogen peroxide. KetoC also significantly increased cellular Nrf2 protein levels, ARE-dependent transcription, and the gene expression of antioxidative enzymes such as heme oxygenase-1 (HO-1), glutamate-cysteine ligase modifier subunit (GCLM), and Nad(p)h: quinone oxidoreductase 1 (NQO1) in HepG2 cells.','cell culture','null',26879219),(3753,'human','decrease','low-throughput','A feature of human ulcerative colitis and other colitic diseases is a change in \'healthy\' microbiota such as Bifidobacterium and Bacteriodes, and a concurrent reduction in SCFAs. Moreover, increased intake of fermentable dietary fibre, or SCFAs, seems to be clinically beneficial in the treatment of colitis. SCFAs bind the G-protein-coupled receptor 43 (GPR43, also known as FFAR2), and here we show that SCFA-GPR43 interactions profoundly affect inflammatory responses. Stimulation of GPR43 by SCFAs was necessary for the normal resolution of certain inflammatory responses, because GPR43-deficient (Gpr43(-/-)) mice showed exacerbated or unresolving inflammation in models of colitis, arthritis and asthma. This seemed to relate to increased production of inflammatory mediators by Gpr43(-/-) immune cells, and increased immune cell recruitment. Germ-free mice, which are devoid of bacteria and express little or no SCFAs, showed a similar dysregulation of certain inflammatory responses. GPR43 binding of SCFAs potentially provides a molecular link between diet, gastrointestinal bacterial metabolism, and immune and inflammatory responses.','in vitro bacterial culture','null',19865172),(3754,'human','decrease','low-throughput','A feature of human ulcerative colitis and other colitic diseases is a change in \'healthy\' microbiota such as Bifidobacterium and Bacteriodes, and a concurrent reduction in SCFAs. Moreover, increased intake of fermentable dietary fibre, or SCFAs, seems to be clinically beneficial in the treatment of colitis. SCFAs bind the G-protein-coupled receptor 43 (GPR43, also known as FFAR2), and here we show that SCFA-GPR43 interactions profoundly affect inflammatory responses. Stimulation of GPR43 by SCFAs was necessary for the normal resolution of certain inflammatory responses, because GPR43-deficient (Gpr43(-/-)) mice showed exacerbated or unresolving inflammation in models of colitis, arthritis and asthma. This seemed to relate to increased production of inflammatory mediators by Gpr43(-/-) immune cells, and increased immune cell recruitment. Germ-free mice, which are devoid of bacteria and express little or no SCFAs, showed a similar dysregulation of certain inflammatory responses. GPR43 binding of SCFAs potentially provides a molecular link between diet, gastrointestinal bacterial metabolism, and immune and inflammatory responses.','in vitro bacterial culture','null',19865172),(3755,'human','decrease','low-throughput','A feature of human ulcerative colitis and other colitic diseases is a change in \'healthy\' microbiota such as Bifidobacterium and Bacteriodes, and a concurrent reduction in SCFAs. Moreover, increased intake of fermentable dietary fibre, or SCFAs, seems to be clinically beneficial in the treatment of colitis. SCFAs bind the G-protein-coupled receptor 43 (GPR43, also known as FFAR2), and here we show that SCFA-GPR43 interactions profoundly affect inflammatory responses. Stimulation of GPR43 by SCFAs was necessary for the normal resolution of certain inflammatory responses, because GPR43-deficient (Gpr43(-/-)) mice showed exacerbated or unresolving inflammation in models of colitis, arthritis and asthma. This seemed to relate to increased production of inflammatory mediators by Gpr43(-/-) immune cells, and increased immune cell recruitment. Germ-free mice, which are devoid of bacteria and express little or no SCFAs, showed a similar dysregulation of certain inflammatory responses. GPR43 binding of SCFAs potentially provides a molecular link between diet, gastrointestinal bacterial metabolism, and immune and inflammatory responses.','in vitro bacterial culture','null',19865172),(3756,'human','decrease','low-throughput','A feature of human ulcerative colitis and other colitic diseases is a change in \'healthy\' microbiota such as Bifidobacterium and Bacteriodes, and a concurrent reduction in SCFAs. Moreover, increased intake of fermentable dietary fibre, or SCFAs, seems to be clinically beneficial in the treatment of colitis. SCFAs bind the G-protein-coupled receptor 43 (GPR43, also known as FFAR2), and here we show that SCFA-GPR43 interactions profoundly affect inflammatory responses. Stimulation of GPR43 by SCFAs was necessary for the normal resolution of certain inflammatory responses, because GPR43-deficient (Gpr43(-/-)) mice showed exacerbated or unresolving inflammation in models of colitis, arthritis and asthma. This seemed to relate to increased production of inflammatory mediators by Gpr43(-/-) immune cells, and increased immune cell recruitment. Germ-free mice, which are devoid of bacteria and express little or no SCFAs, showed a similar dysregulation of certain inflammatory responses. GPR43 binding of SCFAs potentially provides a molecular link between diet, gastrointestinal bacterial metabolism, and immune and inflammatory responses.','in vitro bacterial culture','null',19865172),(3757,'human','decrease','low-throughput','The genome of new equol-producing bacteria, Lactobacillus paracasei JS1, was analyzed. Also, the effect of P. lobata extract fermented with L. paracasei JS1 (FPE) on the skin and intestinal immune response was examined. With gene expression analysis, it was proven that seven skin-related proteins, hyaluronan synthase-1, -2, -3, collagen, elastin, epidermal growth factor, and epidermal growth factor receptor were differentially expressed upon FPE treatment. The messenger RNA expression increased with treatment with the FPE, and a skin moisturizing effect was confirmed by a hematoxylin-eosin staining experiment. In addition, such an experiment showed that proinflammatory cytokines, tumor necrosis factor-α, cyclooxygenase-2, inducible nitric oxide synthase, interleukin-1β, -4, and -6, were reduced in large intestine when treated with FPE.','in vitro bacterial culture,cell culture','HPLC analysis,RT-PCR',30767231),(3758,'human','decrease','low-throughput','The genome of new equol-producing bacteria, Lactobacillus paracasei JS1, was analyzed. Also, the effect of P. lobata extract fermented with L. paracasei JS1 (FPE) on the skin and intestinal immune response was examined. With gene expression analysis, it was proven that seven skin-related proteins, hyaluronan synthase-1, -2, -3, collagen, elastin, epidermal growth factor, and epidermal growth factor receptor were differentially expressed upon FPE treatment. The messenger RNA expression increased with treatment with the FPE, and a skin moisturizing effect was confirmed by a hematoxylin-eosin staining experiment. In addition, such an experiment showed that proinflammatory cytokines, tumor necrosis factor-α, cyclooxygenase-2, inducible nitric oxide synthase, interleukin-1β, -4, and -6, were reduced in large intestine when treated with FPE.','in vitro bacterial culture,cell culture','HPLC analysis,RT-PCR',30767231),(3759,'human','decrease','low-throughput','The genome of new equol-producing bacteria, Lactobacillus paracasei JS1, was analyzed. Also, the effect of P. lobata extract fermented with L. paracasei JS1 (FPE) on the skin and intestinal immune response was examined. With gene expression analysis, it was proven that seven skin-related proteins, hyaluronan synthase-1, -2, -3, collagen, elastin, epidermal growth factor, and epidermal growth factor receptor were differentially expressed upon FPE treatment. The messenger RNA expression increased with treatment with the FPE, and a skin moisturizing effect was confirmed by a hematoxylin-eosin staining experiment. In addition, such an experiment showed that proinflammatory cytokines, tumor necrosis factor-α, cyclooxygenase-2, inducible nitric oxide synthase, interleukin-1β, -4, and -6, were reduced in large intestine when treated with FPE.','in vitro bacterial culture,cell culture','HPLC analysis,RT-PCR',30767231),(3760,'human','decrease','low-throughput','The genome of new equol-producing bacteria, Lactobacillus paracasei JS1, was analyzed. Also, the effect of P. lobata extract fermented with L. paracasei JS1 (FPE) on the skin and intestinal immune response was examined. With gene expression analysis, it was proven that seven skin-related proteins, hyaluronan synthase-1, -2, -3, collagen, elastin, epidermal growth factor, and epidermal growth factor receptor were differentially expressed upon FPE treatment. The messenger RNA expression increased with treatment with the FPE, and a skin moisturizing effect was confirmed by a hematoxylin-eosin staining experiment. In addition, such an experiment showed that proinflammatory cytokines, tumor necrosis factor-α, cyclooxygenase-2, inducible nitric oxide synthase, interleukin-1β, -4, and -6, were reduced in large intestine when treated with FPE.','in vitro bacterial culture,cell culture','HPLC analysis,RT-PCR',30767231),(3761,'human','decrease','low-throughput','The genome of new equol-producing bacteria, Lactobacillus paracasei JS1, was analyzed. Also, the effect of P. lobata extract fermented with L. paracasei JS1 (FPE) on the skin and intestinal immune response was examined. With gene expression analysis, it was proven that seven skin-related proteins, hyaluronan synthase-1, -2, -3, collagen, elastin, epidermal growth factor, and epidermal growth factor receptor were differentially expressed upon FPE treatment. The messenger RNA expression increased with treatment with the FPE, and a skin moisturizing effect was confirmed by a hematoxylin-eosin staining experiment. In addition, such an experiment showed that proinflammatory cytokines, tumor necrosis factor-α, cyclooxygenase-2, inducible nitric oxide synthase, interleukin-1β, -4, and -6, were reduced in large intestine when treated with FPE.','in vitro bacterial culture,cell culture','HPLC analysis,RT-PCR',30767231),(3762,'human','decrease','low-throughput','The genome of new equol-producing bacteria, Lactobacillus paracasei JS1, was analyzed. Also, the effect of P. lobata extract fermented with L. paracasei JS1 (FPE) on the skin and intestinal immune response was examined. With gene expression analysis, it was proven that seven skin-related proteins, hyaluronan synthase-1, -2, -3, collagen, elastin, epidermal growth factor, and epidermal growth factor receptor were differentially expressed upon FPE treatment. The messenger RNA expression increased with treatment with the FPE, and a skin moisturizing effect was confirmed by a hematoxylin-eosin staining experiment. In addition, such an experiment showed that proinflammatory cytokines, tumor necrosis factor-α, cyclooxygenase-2, inducible nitric oxide synthase, interleukin-1β, -4, and -6, were reduced in large intestine when treated with FPE.','in vitro bacterial culture,cell culture','HPLC analysis,RT-PCR',30767231),(3765,'human','increase','low-throughput','A Prevotella and an Atopobium strain were chosen for further study, and shown to up-regulate two PPARγ-target genes, ANGPTL4 and ADRP. The molecular mechanisms of these activations involved the phosphorylation of PPARγ through ERK1/2. The responsible metabolites were shown to be heat sensitive but markedly diverged in size, emphasizing the diversity of bioactive compounds found in the intestine. Here we describe different mechanisms by which single intestinal bacteria can directly impact their host\'s health through transcriptional regulation.','in vitro bacterial culture','HPLC analysis,GC/MS analysis',28266623),(3766,'human','increase','low-throughput','A Prevotella and an Atopobium strain were chosen for further study, and shown to up-regulate two PPARγ-target genes, ANGPTL4 and ADRP. The molecular mechanisms of these activations involved the phosphorylation of PPARγ through ERK1/2. The responsible metabolites were shown to be heat sensitive but markedly diverged in size, emphasizing the diversity of bioactive compounds found in the intestine. Here we describe different mechanisms by which single intestinal bacteria can directly impact their host\'s health through transcriptional regulation.','in vitro bacterial culture','HPLC analysis,GC/MS analysis',28266623),(3767,'human','increase','low-throughput','A Prevotella and an Atopobium strain were chosen for further study, and shown to up-regulate two PPARγ-target genes, ANGPTL4 and ADRP. The molecular mechanisms of these activations involved the phosphorylation of PPARγ through ERK1/2. The responsible metabolites were shown to be heat sensitive but markedly diverged in size, emphasizing the diversity of bioactive compounds found in the intestine. Here we describe different mechanisms by which single intestinal bacteria can directly impact their host\'s health through transcriptional regulation.','in vitro bacterial culture','HPLC analysis,GC/MS analysis',28266623),(3768,'human','increase','low-throughput','A Prevotella and an Atopobium strain were chosen for further study, and shown to up-regulate two PPARγ-target genes, ANGPTL4 and ADRP. The molecular mechanisms of these activations involved the phosphorylation of PPARγ through ERK1/2. The responsible metabolites were shown to be heat sensitive but markedly diverged in size, emphasizing the diversity of bioactive compounds found in the intestine. Here we describe different mechanisms by which single intestinal bacteria can directly impact their host\'s health through transcriptional regulation.','in vitro bacterial culture','HPLC analysis,GC/MS analysis',28266623),(3779,'human','increase','low-throughput','A. muciniphila increased the expression of tlr2 and tlr4. However, EVs at all of the concentrations showed a decrease in tlr4 expression. EVs at the concentrations of 0.1 and 0.5 µg/ml decreased the expression of tlr2. A. muciniphila significantly increased the expression of ocldn and cldn4. Both this bacterium and EVs increased the expression of zo2 in the cell line. Furthermore, this data show that A. muciniphila derived EVs have a dose-independent effect on Caco-2 cells.','in vitro bacterial culture,cell culture','RT-PCR',31191842),(3780,'human','increase','low-throughput','A. muciniphila increased the expression of tlr2 and tlr4. However, EVs at all of the concentrations showed a decrease in tlr4 expression. EVs at the concentrations of 0.1 and 0.5 µg/ml decreased the expression of tlr2. A. muciniphila significantly increased the expression of ocldn and cldn4. Both this bacterium and EVs increased the expression of zo2 in the cell line. Furthermore, this data show that A. muciniphila derived EVs have a dose-independent effect on Caco-2 cells.','in vitro bacterial culture,cell culture','RT-PCR',31191842),(3781,'human','increase','low-throughput','A. muciniphila increased the expression of tlr2 and tlr4. However, EVs at all of the concentrations showed a decrease in tlr4 expression. EVs at the concentrations of 0.1 and 0.5 µg/ml decreased the expression of tlr2. A. muciniphila significantly increased the expression of ocldn and cldn4. Both this bacterium and EVs increased the expression of zo2 in the cell line. Furthermore, this data show that A. muciniphila derived EVs have a dose-independent effect on Caco-2 cells.','in vitro bacterial culture,cell culture','RT-PCR',31191842),(3782,'human','increase','low-throughput','A. muciniphila increased the expression of tlr2 and tlr4. However, EVs at all of the concentrations showed a decrease in tlr4 expression. EVs at the concentrations of 0.1 and 0.5 µg/ml decreased the expression of tlr2. A. muciniphila significantly increased the expression of ocldn and cldn4. Both this bacterium and EVs increased the expression of zo2 in the cell line. Furthermore, this data show that A. muciniphila derived EVs have a dose-independent effect on Caco-2 cells.','in vitro bacterial culture,cell culture','RT-PCR',31191842),(3783,'human','increase','low-throughput','A. muciniphila increased the expression of tlr2 and tlr4. However, EVs at all of the concentrations showed a decrease in tlr4 expression. EVs at the concentrations of 0.1 and 0.5 µg/ml decreased the expression of tlr2. A. muciniphila significantly increased the expression of ocldn and cldn4. Both this bacterium and EVs increased the expression of zo2 in the cell line. Furthermore, this data show that A. muciniphila derived EVs have a dose-independent effect on Caco-2 cells.','in vitro bacterial culture,cell culture','RT-PCR',31191842),(3784,'human','increase','low-throughput','A. muciniphila increased the expression of tlr2 and tlr4. However, EVs at all of the concentrations showed a decrease in tlr4 expression. EVs at the concentrations of 0.1 and 0.5 µg/ml decreased the expression of tlr2. A. muciniphila significantly increased the expression of ocldn and cldn4. Both this bacterium and EVs increased the expression of zo2 in the cell line. Furthermore, this data show that A. muciniphila derived EVs have a dose-independent effect on Caco-2 cells.','in vitro bacterial culture,cell culture','RT-PCR',31191842),(3785,'human','increase','low-throughput','On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.','in vitro bacterial culture,cell culture','RT-PCR',28239378),(3786,'human','increase','low-throughput','On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.','in vitro bacterial culture,cell culture','RT-PCR',28239378),(3787,'human','increase','low-throughput','On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.','in vitro bacterial culture,cell culture','RT-PCR',28239378),(3788,'human','increase','low-throughput','On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.','in vitro bacterial culture,cell culture','RT-PCR',28239378),(3789,'human','increase','low-throughput','On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.','in vitro bacterial culture,cell culture','RT-PCR',28239378),(3790,'human','increase','low-throughput','On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.','in vitro bacterial culture,cell culture','RT-PCR',28239378),(3791,'human','increase','low-throughput','On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.','in vitro bacterial culture,cell culture','RT-PCR',28239378),(3792,'human','increase','low-throughput','On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.','in vitro bacterial culture,cell culture','RT-PCR',28239378),(3793,'human','increase','low-throughput','On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.','in vitro bacterial culture,cell culture','RT-PCR',28239378),(3794,'human','increase','low-throughput','On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.','in vitro bacterial culture,cell culture','RT-PCR',28239378),(3795,'human','increase','low-throughput','On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.','in vitro bacterial culture,cell culture','RT-PCR',28239378),(3796,'human','increase','low-throughput','On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.','in vitro bacterial culture,cell culture','RT-PCR',28239378),(3797,'human','increase','low-throughput','On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.','in vitro bacterial culture,cell culture','RT-PCR',28239378),(3798,'human','increase','low-throughput','On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.','in vitro bacterial culture,cell culture','RT-PCR',28239378),(3799,'human','increase','low-throughput','On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.','in vitro bacterial culture,cell culture','RT-PCR',28239378),(3800,'human','increase','low-throughput','On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.','in vitro bacterial culture,cell culture','RT-PCR',28239378),(3801,'human','increase','low-throughput','On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.','in vitro bacterial culture,cell culture','RT-PCR',28239378),(3802,'human','increase','low-throughput','On PBMC, quantification of secreted cytokines in supernatant showed a similar pattern for the three strains under study.All of them increased IL-1β, IL-6, IL-12p70, IFN-γ, and TNF-α as well as IL-10.','in vitro bacterial culture,cell culture','RT-PCR',28239378),(3811,'human','increase','low-throughput','Here we analyse the ability of OMVs and soluble secreted factors to protect epithelial barrier function in polarized T-84 and Caco-2 cells infected with enteropathogenic Escherichia coli (EPEC). Transepithelial electrical resistance, paracellular permeability, mRNA levels and subcellular distribution of tight junction proteins were monitored in the absence or presence of EcN and ECOR63 extracellular fractions. EPEC downregulated expression of ZO-1 ZO-2, occludin and claudin-14 and altered the subcellular localization of ZO-1, occludin and F-actin cytoskeleton. OMVs and soluble factors secreted by EcN and ECOR63 counteracted EPEC-altered transepithelial resistance and paracellular permeability, preserved occludin and claudin-14 mRNA levels, retained ZO-1 and occludin at tight junctions in the cell boundaries and ameliorated F-actin disorganization. Redistribution of ZO-1 was not accompanied by changes at mRNA level.','in vitro bacterial culture,cell culture','null',31315566),(3812,'human','increase','low-throughput','Here we analyse the ability of OMVs and soluble secreted factors to protect epithelial barrier function in polarized T-84 and Caco-2 cells infected with enteropathogenic Escherichia coli (EPEC). Transepithelial electrical resistance, paracellular permeability, mRNA levels and subcellular distribution of tight junction proteins were monitored in the absence or presence of EcN and ECOR63 extracellular fractions. EPEC downregulated expression of ZO-1 ZO-2, occludin and claudin-14 and altered the subcellular localization of ZO-1, occludin and F-actin cytoskeleton. OMVs and soluble factors secreted by EcN and ECOR63 counteracted EPEC-altered transepithelial resistance and paracellular permeability, preserved occludin and claudin-14 mRNA levels, retained ZO-1 and occludin at tight junctions in the cell boundaries and ameliorated F-actin disorganization. Redistribution of ZO-1 was not accompanied by changes at mRNA level.','in vitro bacterial culture,cell culture','null',31315566),(3813,'human','increase','low-throughput','Here we analyse the ability of OMVs and soluble secreted factors to protect epithelial barrier function in polarized T-84 and Caco-2 cells infected with enteropathogenic Escherichia coli (EPEC). Transepithelial electrical resistance, paracellular permeability, mRNA levels and subcellular distribution of tight junction proteins were monitored in the absence or presence of EcN and ECOR63 extracellular fractions. EPEC downregulated expression of ZO-1 ZO-2, occludin and claudin-14 and altered the subcellular localization of ZO-1, occludin and F-actin cytoskeleton. OMVs and soluble factors secreted by EcN and ECOR63 counteracted EPEC-altered transepithelial resistance and paracellular permeability, preserved occludin and claudin-14 mRNA levels, retained ZO-1 and occludin at tight junctions in the cell boundaries and ameliorated F-actin disorganization. Redistribution of ZO-1 was not accompanied by changes at mRNA level.','in vitro bacterial culture,cell culture','null',31315566),(3814,'human','increase','low-throughput','Here we analyse the ability of OMVs and soluble secreted factors to protect epithelial barrier function in polarized T-84 and Caco-2 cells infected with enteropathogenic Escherichia coli (EPEC). Transepithelial electrical resistance, paracellular permeability, mRNA levels and subcellular distribution of tight junction proteins were monitored in the absence or presence of EcN and ECOR63 extracellular fractions. EPEC downregulated expression of ZO-1 ZO-2, occludin and claudin-14 and altered the subcellular localization of ZO-1, occludin and F-actin cytoskeleton. OMVs and soluble factors secreted by EcN and ECOR63 counteracted EPEC-altered transepithelial resistance and paracellular permeability, preserved occludin and claudin-14 mRNA levels, retained ZO-1 and occludin at tight junctions in the cell boundaries and ameliorated F-actin disorganization. Redistribution of ZO-1 was not accompanied by changes at mRNA level.','in vitro bacterial culture,cell culture','null',31315566),(3815,'human','increase','low-throughput','Here we analyse the ability of OMVs and soluble secreted factors to protect epithelial barrier function in polarized T-84 and Caco-2 cells infected with enteropathogenic Escherichia coli (EPEC). Transepithelial electrical resistance, paracellular permeability, mRNA levels and subcellular distribution of tight junction proteins were monitored in the absence or presence of EcN and ECOR63 extracellular fractions. EPEC downregulated expression of ZO-1 ZO-2, occludin and claudin-14 and altered the subcellular localization of ZO-1, occludin and F-actin cytoskeleton. OMVs and soluble factors secreted by EcN and ECOR63 counteracted EPEC-altered transepithelial resistance and paracellular permeability, preserved occludin and claudin-14 mRNA levels, retained ZO-1 and occludin at tight junctions in the cell boundaries and ameliorated F-actin disorganization. Redistribution of ZO-1 was not accompanied by changes at mRNA level.','in vitro bacterial culture,cell culture','null',31315566),(3816,'human','increase','low-throughput','Here we analyse the ability of OMVs and soluble secreted factors to protect epithelial barrier function in polarized T-84 and Caco-2 cells infected with enteropathogenic Escherichia coli (EPEC). Transepithelial electrical resistance, paracellular permeability, mRNA levels and subcellular distribution of tight junction proteins were monitored in the absence or presence of EcN and ECOR63 extracellular fractions. EPEC downregulated expression of ZO-1 ZO-2, occludin and claudin-14 and altered the subcellular localization of ZO-1, occludin and F-actin cytoskeleton. OMVs and soluble factors secreted by EcN and ECOR63 counteracted EPEC-altered transepithelial resistance and paracellular permeability, preserved occludin and claudin-14 mRNA levels, retained ZO-1 and occludin at tight junctions in the cell boundaries and ameliorated F-actin disorganization. Redistribution of ZO-1 was not accompanied by changes at mRNA level.','in vitro bacterial culture,cell culture','null',31315566),(3824,'human','decrease','low-throughput','It has been shown to down-regulate nuclear transcription factor (NF-кB) in human intestinal cells, a central regulator of the host mucosal immune system response to the microbiota.','cell culture,in vitro bacterial culture,control experiment','RT-PCR,Western blot assay',25946041),(3825,'human','decrease','low-throughput','In order to evaluate its impact on a further important transcription factor shown to link metabolism and inflammation in the intestine, namely PPARγ (peroxisome proliferator-activated receptor), we used human intestinal epithelial cell-lines engineered to monitor PPARγ transcriptional activity in response to a wide range of S. salivarius strains.','cell culture,in vitro bacterial culture,control experiment','RT-PCR,Western blot assay',25946041),(3826,'human','decrease','low-throughput','To assess the impact of S. salivarius-dependent inhibition on PPARγ target genes, we quantified by RT-qPCR the expression of Angptl4 and I-FABP (intestinal fatty acid binding protein).The expression of these two well-characterized PPARγ-regulated genes were strongly reduced by S. salivarius supernatant in the presence of the PPARγ-specific ligand, rosiglitazone.','cell culture,in vitro bacterial culture,control experiment','RT-PCR,Western blot assay',25946041),(3827,'human','decrease','low-throughput','To assess the impact of S. salivarius-dependent inhibition on PPARγ target genes, we quantified by RT-qPCR the expression of Angptl4 and I-FABP (intestinal fatty acid binding protein).The expression of these two well-characterized PPARγ-regulated genes were strongly reduced by S. salivarius supernatant in the presence of the PPARγ-specific ligand, rosiglitazone.','cell culture,in vitro bacterial culture,control experiment','RT-PCR,Western blot assay',25946041),(3828,'human','decrease','low-throughput','It has been shown to down-regulate nuclear transcription factor (NF-кB) in human intestinal cells, a central regulator of the host mucosal immune system response to the microbiota.','cell culture,in vitro bacterial culture,control experiment','RT-PCR,Western blot assay',25946041),(3829,'human','decrease','low-throughput','In order to evaluate its impact on a further important transcription factor shown to link metabolism and inflammation in the intestine, namely PPARγ (peroxisome proliferator-activated receptor), we used human intestinal epithelial cell-lines engineered to monitor PPARγ transcriptional activity in response to a wide range of S. salivarius strains.','cell culture,in vitro bacterial culture,control experiment','RT-PCR,Western blot assay',25946041),(3830,'human','decrease','low-throughput','To assess the impact of S. salivarius-dependent inhibition on PPARγ target genes, we quantified by RT-qPCR the expression of Angptl4 and I-FABP (intestinal fatty acid binding protein).The expression of these two well-characterized PPARγ-regulated genes were strongly reduced by S. salivarius supernatant in the presence of the PPARγ-specific ligand, rosiglitazone.','cell culture,in vitro bacterial culture,control experiment','RT-PCR,Western blot assay',25946041),(3831,'human','decrease','low-throughput','To assess the impact of S. salivarius-dependent inhibition on PPARγ target genes, we quantified by RT-qPCR the expression of Angptl4 and I-FABP (intestinal fatty acid binding protein).The expression of these two well-characterized PPARγ-regulated genes were strongly reduced by S. salivarius supernatant in the presence of the PPARγ-specific ligand, rosiglitazone.','cell culture,in vitro bacterial culture,control experiment','RT-PCR,Western blot assay',25946041),(3832,'human','decrease','low-throughput','In the coculture model, treatment with Enterococcus durans and with butyrate reduced basal as well as E coli stimulated secretion of IL-8, IL-6, and  TNF and increased secretion of IL-10.','cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment,Supplementation of metabolite','RT-PCR',21178864),(3833,'human','decrease','low-throughput','In the coculture model, treatment with Enterococcus durans and with butyrate reduced basal as well as E coli stimulated secretion of IL-8, IL-6, and  TNF and increased secretion of IL-10.','cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment,Supplementation of metabolite','RT-PCR',21178864),(3834,'human','decrease','low-throughput','In the coculture model, treatment with Enterococcus durans and with butyrate reduced basal as well as E coli stimulated secretion of IL-8, IL-6, and  TNF and increased secretion of IL-10.','cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment,Supplementation of metabolite','RT-PCR',21178864),(3835,'human','increase','low-throughput','In the coculture model, treatment with Enterococcus durans and with butyrate reduced basal as well as E coli stimulated secretion of IL-8, IL-6, and  TNF and increased secretion of IL-10.','cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment,Supplementation of metabolite','RT-PCR',21178864),(3836,'human','decrease','low-throughput','In the coculture model, treatment with Enterococcus durans and with butyrate reduced basal as well as E coli stimulated secretion of IL-8, IL-6, and  TNF and increased secretion of IL-10.','cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment,Supplementation of metabolite','RT-PCR',21178864),(3837,'human','decrease','low-throughput','Incubation with LGG significantly decreased the concentration of IL-8 compared with the control, but no other significant effects were observed for LGG or its genomic DNA isolate.','cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment,Supplementation of metabolite','RT-PCR',21178864),(3839,'human','increase','low-throughput','To examine how C. difficile induces indole production, the expression of the tryptophanase gene (tnaA) in the E. coli strains was measured by quantitative real-time PCR. E. coli strains exposed to C. difficile stationary phase supernatant expressed significantly (P   0.0025) higher tnaA levels ( 4-fold) than when cultured alone (Fig. 4A). The increased expression of tnaA in the presence of the stationary-phase culture supernatant suggested that C. difficile likely induces indole at the level of transcription.','in vitro bacterial culture、in vitro bacterial culture','RT-PCR',30944877),(3840,'human','increase','low-throughput','To examine how C. difficile induces indole production, the expression of the tryptophanase gene (tnaA) in the E. coli strains was measured by quantitative real-time PCR. E. coli strains exposed to C. difficile stationary phase supernatant expressed significantly (P   0.0025) higher tnaA levels ( 4-fold) than when cultured alone (Fig. 4A). The increased expression of tnaA in the presence of the stationary-phase culture supernatant suggested that C. difficile likely induces indole at the level of transcription.','in vitro bacterial culture、in vitro bacterial culture','RT-PCR',30944877),(3845,'human','decrease','low-throughput','We examined the effects of UV-killed F. prausnitzii, bacterial fractions,supernatants, or sterile medium on IL-8 secretion in Caco-2 cells,with or without IL-1 stimulation (SI Materials and Methods and Fig. 3A). Either UV-killed F. prausnitzii, membrane extracts, or cytoplasmic extracts, but not DNA or sterile medium, induce low levels of IL-8 secretion. UV-killed F. prausnitzii, bacterial fractions,and sterile medium did not inhibit IL-8 secretion induced by IL-1,compared with negative control. In contrast, supernatant from F.prausnitzii cultures significantly reduces IL-8 secretion induced by IL-1.','cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment','qPCR',18936492),(3846,'human','decrease','low-throughput','We examined the effects of UV-killed F. prausnitzii, bacterial fractions,supernatants, or sterile medium on IL-8 secretion in Caco-2 cells,with or without IL-1 stimulation (SI Materials and Methods and Fig. 3A). Either UV-killed F. prausnitzii, membrane extracts, or cytoplasmic extracts, but not DNA or sterile medium, induce low levels of IL-8 secretion. UV-killed F. prausnitzii, bacterial fractions,and sterile medium did not inhibit IL-8 secretion induced by IL-1,compared with negative control. In contrast, supernatant from F.prausnitzii cultures significantly reduces IL-8 secretion induced by IL-1.','cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment','qPCR',18936492),(3847,'human','decrease','low-throughput','In vitro peripheral blood mononuclear cell stimulation by F. prausnitzii led to significantly lower IL-12 and IFN-γ production levels and higher secretion of IL-10.','cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment','qPCR',18936492),(3848,'human','decrease','low-throughput','In vitro peripheral blood mononuclear cell stimulation by F. prausnitzii led to significantly lower IL-12 and IFN-γ production levels and higher secretion of IL-10.','cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment','qPCR',18936492),(3849,'human','increase','low-throughput','In vitro peripheral blood mononuclear cell stimulation by F. prausnitzii led to significantly lower IL-12 and IFN-γ production levels and higher secretion of IL-10.','cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment','qPCR',18936492),(3870,'human','increase','low-throughput','Compared with the control, the stimulation of HCT116 cells with C. butyricum supernatant and sodium butyrate induced 4.5-fold and 4.2-fold upregulation of the GPR43 mRNA levels, 2.6-fold and 7.0-fold increases in the GPR109A mRNA levels.','cell culture,in vitro bacterial culture','null',31734354),(3871,'human','increase','low-throughput','Compared with the control, the stimulation of HCT116 cells with C. butyricum supernatant and sodium butyrate induced 4.5-fold and 4.2-fold upregulation of the GPR43 mRNA levels, 2.6-fold and 7.0-fold increases in the GPR109A mRNA levels.','cell culture,in vitro bacterial culture','null',31734354),(3872,'human','increase','low-throughput','The mRNA level of P21WAF1 was increased in HCT116 cells treated with C. butyricum.','cell culture,in vitro bacterial culture','null',31734354),(3882,'human','increase','low-throughput','Exposure to 1 mM indole led to an increase in the expression of two Toll-like receptor (TLR) genes—TLR-3 and TLR-9.','cell culture','RT-qPCR',19966295),(3883,'human','increase','low-throughput','Exposure to 1 mM indole led to an increase in the expression of two Toll-like receptor (TLR) genes—TLR-3 and TLR-9.','cell culture','RT-qPCR',19966295),(3884,'human','increase','low-throughput','Additionally, genes involved in the modulation of TLR signaling—TOLLIP, SIGIRR, and SOCS5—were also induced by indole.','cell culture','RT-qPCR',19966295),(3885,'human','increase','low-throughput','Additionally, genes involved in the modulation of TLR signaling—TOLLIP, SIGIRR, and SOCS5—were also induced by indole.','cell culture','RT-qPCR',19966295),(3886,'human','increase','low-throughput','Additionally, genes involved in the modulation of TLR signaling—TOLLIP, SIGIRR, and SOCS5—were also induced by indole.','cell culture','RT-qPCR',19966295),(3887,'human','increase','low-throughput','Similarly, genes involved in the phosphoinositide-3-kinase (PI3K) pathway, which modulates TLR signaling and leads to antiinflammatory cytokine IL-10 induction, were also induced by indole.','cell culture','RT-qPCR',19966295),(3888,'human','increase','low-throughput','Three genes involved in PI3K signaling (PIK3CA, PIK3CB, and PIK3CD) were induced, as was the transcription factor AKT1 that is activated by signaling through the PI3K pathway.','cell culture','RT-qPCR',19966295),(3889,'human','increase','low-throughput','Three genes involved in PI3K signaling (PIK3CA, PIK3CB, and PIK3CD) were induced, as was the transcription factor AKT1 that is activated by signaling through the PI3K pathway.','cell culture','RT-qPCR',19966295),(3890,'human','increase','low-throughput','Three genes involved in PI3K signaling (PIK3CA, PIK3CB, and PIK3CD) were induced, as was the transcription factor AKT1 that is activated by signaling through the PI3K pathway.','cell culture','RT-qPCR',19966295),(3891,'human','increase','low-throughput','Downstream target genes of AKT1, including the cAMP response element-binding protein and IL-10, were also induced upon addition of indole.','cell culture','RT-qPCR',19966295),(3892,'human','increase','low-throughput','In addition, the receptor for the proinflammatory cytokine IL-2, IL-4, IL-18 binding protein, IL-24, and IL-28 were all induced, suggesting that indole exposure leads to an increase in cytokine expression and signaling in HCT-8 intestinal epithelial cells.','cell culture','RT-qPCR',19966295),(3893,'human','increase','low-throughput','In addition, the receptor for the proinflammatory cytokine IL-2, IL-4, IL-18 binding protein, IL-24, and IL-28 were all induced, suggesting that indole exposure leads to an increase in cytokine expression and signaling in HCT-8 intestinal epithelial cells.','cell culture','RT-qPCR',19966295),(3894,'human','increase','low-throughput','In addition, the receptor for the proinflammatory cytokine IL-2, IL-4, IL-18 binding protein, IL-24, and IL-28 were all induced, suggesting that indole exposure leads to an increase in cytokine expression and signaling in HCT-8 intestinal epithelial cells.','cell culture','RT-qPCR',19966295),(3895,'human','increase','low-throughput','In addition, the receptor for the proinflammatory cytokine IL-2, IL-4, IL-18 binding protein, IL-24, and IL-28 were all induced, suggesting that indole exposure leads to an increase in cytokine expression and signaling in HCT-8 intestinal epithelial cells.','cell culture','RT-qPCR',19966295),(3896,'human','increase','low-throughput','In addition, the receptor for the proinflammatory cytokine IL-2, IL-4, IL-18 binding protein, IL-24, and IL-28 were all induced, suggesting that indole exposure leads to an increase in cytokine expression and signaling in HCT-8 intestinal epithelial cells.','cell culture','RT-qPCR',19966295),(3897,'human','increase','low-throughput','In addition to altering the expression of cytokines, indole also induced the expression of several chemokines (CCL25, CCL4, CCL16, CXCL3, XCL1, and CX3CL1) in HCT-8 cells.','cell culture','RT-qPCR',19966295),(3898,'human','increase','low-throughput','In addition to altering the expression of cytokines, indole also induced the expression of several chemokines (CCL25, CCL4, CCL16, CXCL3, XCL1, and CX3CL1) in HCT-8 cells.','cell culture','RT-qPCR',19966295),(3899,'human','increase','low-throughput','In addition to altering the expression of cytokines, indole also induced the expression of several chemokines (CCL25, CCL4, CCL16, CXCL3, XCL1, and CX3CL1) in HCT-8 cells.','cell culture','RT-qPCR',19966295),(3900,'human','increase','low-throughput','In addition to altering the expression of cytokines, indole also induced the expression of several chemokines (CCL25, CCL4, CCL16, CXCL3, XCL1, and CX3CL1) in HCT-8 cells.','cell culture','RT-qPCR',19966295),(3901,'human','increase','low-throughput','In addition to altering the expression of cytokines, indole also induced the expression of several chemokines (CCL25, CCL4, CCL16, CXCL3, XCL1, and CX3CL1) in HCT-8 cells.','cell culture','RT-qPCR',19966295),(3902,'human','increase','low-throughput','In addition to altering the expression of cytokines, indole also induced the expression of several chemokines (CCL25, CCL4, CCL16, CXCL3, XCL1, and CX3CL1) in HCT-8 cells.','cell culture','RT-qPCR',19966295),(3903,'human','increase','low-throughput','Specifically, the expression of dual-specificity phosphatases (DUSPs)—Dusp1, Dusp3, Dusp10, Dusp16, Dusp18, and Dusp22, which function as inhibitors of MAPK signaling —was increased upon indole exposure.','cell culture','RT-qPCR',19966295),(3904,'human','increase','low-throughput','Specifically, the expression of dual-specificity phosphatases (DUSPs)—Dusp1, Dusp3, Dusp10, Dusp16, Dusp18, and Dusp22, which function as inhibitors of MAPK signaling —was increased upon indole exposure.','cell culture','RT-qPCR',19966295),(3905,'human','increase','low-throughput','Specifically, the expression of dual-specificity phosphatases (DUSPs)—Dusp1, Dusp3, Dusp10, Dusp16, Dusp18, and Dusp22, which function as inhibitors of MAPK signaling —was increased upon indole exposure.','cell culture','RT-qPCR',19966295),(3906,'human','increase','low-throughput','Specifically, the expression of dual-specificity phosphatases (DUSPs)—Dusp1, Dusp3, Dusp10, Dusp16, Dusp18, and Dusp22, which function as inhibitors of MAPK signaling —was increased upon indole exposure.','cell culture','RT-qPCR',19966295),(3907,'human','increase','low-throughput','Specifically, the expression of dual-specificity phosphatases (DUSPs)—Dusp1, Dusp3, Dusp10, Dusp16, Dusp18, and Dusp22, which function as inhibitors of MAPK signaling —was increased upon indole exposure.','cell culture','RT-qPCR',19966295),(3936,'human','decrease','low-throughput','We also noted that IRS1 levels were reduced by 8 hr treatment of imidazole propionate in the human embryonic kidney cell line HEK293.','cell culture','UPLC-MS/MS',30401435),(3937,'human','increase','low-throughput','Imidazole propionate activates mTORC1.','cell culture','UPLC-MS/MS',30401435),(3938,'human','decrease','low-throughput','We also noted that IRS1 levels were reduced by 8 hr treatment of imidazole propionate in the human embryonic kidney cell line HEK293.','cell culture','UPLC-MS/MS',30401435),(3939,'human','increase','low-throughput','Imidazole propionate activates mTORC1.','cell culture','UPLC-MS/MS',30401435),(3944,'human','increase','low-throughput','EcN positively modulates the intestinal epithelial barrier through upregulation and redistribution of the tight junction proteins ZO-1, ZO-2 and claudin-14.','cell culture','RT-qPCR',28018313),(3945,'human','increase','low-throughput','EcN positively modulates the intestinal epithelial barrier through upregulation and redistribution of the tight junction proteins ZO-1, ZO-2 and claudin-14.','cell culture','RT-qPCR',28018313),(3946,'human','increase','low-throughput','EcN positively modulates the intestinal epithelial barrier through upregulation and redistribution of the tight junction proteins ZO-1, ZO-2 and claudin-14.','cell culture','RT-qPCR',28018313),(3947,'human','increase','low-throughput','There was a strong correlation between the amount of Firmicutes and TLR2 gene expression levels.','in vivo experiment','qPCR',31677982),(3948,'human','decrease','low-throughput','Preincubation with BCM significantly reduced the occludin expression compared with the control and LCM group.','cell culture','RT-qPCR',28230606),(4017,'human','increase','low-throughput','B. lactis augmented TNF-α compared to controlas time and dose dependent.','cell culture','qPCR',31780030),(4018,'human','decrease','low-throughput','B.fragilis significantly decreased TLR2 gene expression.TLR4 gene expression was slightly increased by this bacterium.','cell culture','RT-qPCR',30507089),(4019,'human','increase','low-throughput','B.fragilis significantly decreased TLR2 gene expression.TLR4 gene expression was slightly increased by this bacterium.','cell culture','RT-qPCR',30507089),(4020,'human','increase','low-throughput','B. fragilis signifcantly elevated IFNγ concentration.','cell culture','RT-qPCR',30507089),(4021,'human','increase','low-throughput','B. fragilis was able to increase IL-4 and IL-10 concentrations.','cell culture','RT-qPCR',30507089),(4022,'human','increase','low-throughput','B. fragilis was able to increase IL-4 and IL-10 concentrations.','cell culture','RT-qPCR',30507089),(4074,'human','increase','low-throughput','CS of LP increases SMCT1 mRNA and protein levels in IEC-6 cells.','cell culture','RT-PCR',20671196),(4078,'human','increase','low-throughput','The results showed a significant increase in the expression of MUC2 and MUC1 genes in HT29-MTX cells with RSN.','cell culture','RT-PCR',26868655),(4079,'human','increase','low-throughput','The results showed a significant increase in the expression of MUC2 and MUC1 genes in HT29-MTX cells with RSN.','cell culture','RT-PCR',26868655),(4080,'human','increase','low-throughput','The results showed an increase in MGAT3, B4GALT1 and ST6GAL1 expression in RSN-BHI compared to BHI-treated HT29-MTX cells.','cell culture','RT-PCR',26868655),(4081,'human','increase','low-throughput','The results showed an increase in MGAT3, B4GALT1 and ST6GAL1 expression in RSN-BHI compared to BHI-treated HT29-MTX cells.','cell culture','RT-PCR',26868655),(4082,'human','increase','low-throughput','The results showed an increase in MGAT3, B4GALT1 and ST6GAL1 expression in RSN-BHI compared to BHI-treated HT29-MTX cells.','cell culture','RT-PCR',26868655),(4086,'human','decrease','low-throughput','Secretion of TNF-α-triggered IL-8 was significantly reduced in cells pre-incubated with L. plantarum L9.','cell culture','ELISA',28441885),(4087,'human','decrease','low-throughput','Secretion of TNF-α-triggered IL-8 was significantly reduced in cells pre-incubated with L. plantarum L9.','cell culture','ELISA',28441885),(4088,'human','increase','low-throughput','LA CS induces Erk1/2 MAPK activation and c-Fos or c-Jun protein expression in Caco-2 cells.','cell culture','Western blot assay',26867563),(4089,'human','increase','low-throughput','LA CS induces Erk1/2 MAPK activation and c-Fos or c-Jun protein expression in Caco-2 cells.','cell culture','Western blot assay',26867563),(4090,'human','increase','low-throughput','These results further suggest that c-Fos and c-Jun play an important role in stimulation of Pgp gene expression in Caco-2 cells in response to LA CS.','cell culture','Western blot assay',26867563),(4097,'human','decrease','low-throughput','The results showed the mRNA levels of both hTHTR-1 and hTHTR-2 were significantly (P < 0.01 for both) lower in EPEC-treated cells compared with controls.','cell culture','RT-PCR',19628653),(4098,'human','decrease','low-throughput','The results showed the mRNA levels of both hTHTR-1 and hTHTR-2 were significantly (P < 0.01 for both) lower in EPEC-treated cells compared with controls.','cell culture','RT-PCR',19628653),(4105,'human','increase','low-throughput','Treatment of Caco2 cells with LA or its conditioned culture supernatant (CS) for 8�?4 h resulted in a significant increase in Na+/H+ exchange activity, mRNA, and protein levels of NHE3.','in vitro bacterial culture','RT-PCR',23086913),(4108,'human','decrease','low-throughput','L. acidophilus blocks PAF-induced NF-kB activation and IL-8 production in human intestinal epithelial cells.','in vitro bacterial culture','ELISA,RT-PCR',24130731),(4109,'human','decrease','low-throughput','L. acidophilus blocks PAF-induced NF-kB activation and IL-8 production in human intestinal epithelial cells.','in vitro bacterial culture','ELISA,RT-PCR',24130731),(4110,'human','decrease','low-throughput','LA-CS also attenuated PAF-induced increase in Bcl10 mRNA and protein levels and Bcl10 promoter activity.','in vitro bacterial culture','ELISA,RT-PCR',24130731),(4111,'human','increase','low-throughput','LA-CS Increases DRA mRNA Levels and Blocks IFN-γ-Induced Decrease in DRA','in vitro bacterial culture','ELISA,RT-PCR',25059823),(4128,'human','decrease','low-throughput','Using a publically available database (GSE75214) 25, we found that the expression of host glycosyltransferases, such as MGTA4A and MGAT4B, was significantly impaired in UC patients compared to normal controls (Fig. 4a). The expression of IL22RA2 (also known as IL-22 binding protein (IL-22BP), a soluble IL-22 receptor that blocks IL-22 signaling, was markedly up-regulated in patients with UC, while IL-22 expression was not affected (Fig. 4a and Extended Data Fig. 7a).','cell culture,in vitro bacterial culture,control experiment','RT-PCR,qPCR',32066975),(4129,'human','decrease','low-throughput','Using a publically available database (GSE75214) 25, we found that the expression of host glycosyltransferases, such as MGTA4A and MGAT4B, was significantly impaired in UC patients compared to normal controls (Fig. 4a). The expression of IL22RA2 (also known as IL-22 binding protein (IL-22BP), a soluble IL-22 receptor that blocks IL-22 signaling, was markedly up-regulated in patients with UC, while IL-22 expression was not affected (Fig. 4a and Extended Data Fig. 7a).','cell culture,in vitro bacterial culture,control experiment','RT-PCR,qPCR',32066975),(4130,'human','increase','low-throughput','Using a publically available database (GSE75214) 25, we found that the expression of host glycosyltransferases, such as MGTA4A and MGAT4B, was significantly impaired in UC patients compared to normal controls (Fig. 4a). The expression of IL22RA2 (also known as IL-22 binding protein (IL-22BP), a soluble IL-22 receptor that blocks IL-22 signaling, was markedly up-regulated in patients with UC, while IL-22 expression was not affected (Fig. 4a and Extended Data Fig. 7a).','cell culture,in vitro bacterial culture,control experiment','RT-PCR,qPCR',32066975),(4230,'mouse','decrease','low-throughput','Using human intestinal epithelial cells (IECs), we discovered that SCFAs, particularly butyrate, enhanced IEC barrier formation, induced IL10RA mRNA, IL-10RA protein, and transactivation through activated Stat3 and HDAC inhibition. Loss and gain of IL10RA expression directly correlates with IEC barrier formation and butyrate represses permeability-promoting claudin-2 (Cldn2) tight-junction protein expression through an IL-10RA-dependent mechanism. Our findings provide a novel mechanism by which microbial derived butyrate promotes barrier through IL-10RA-dependent repression of Cldn2.','cell culture','real-time PCR,qPCR,Western blot assay',28893958),(4231,'mouse','increase','low-throughput','Using human intestinal epithelial cells (IECs), we discovered that SCFAs, particularly butyrate, enhanced IEC barrier formation, induced IL10RA mRNA, IL-10RA protein, and transactivation through activated Stat3 and HDAC inhibition. Loss and gain of IL10RA expression directly correlates with IEC barrier formation and butyrate represses permeability-promoting claudin-2 (Cldn2) tight-junction protein expression through an IL-10RA-dependent mechanism. Our findings provide a novel mechanism by which microbial derived butyrate promotes barrier through IL-10RA-dependent repression of Cldn2.','cell culture','real-time PCR,qPCR,Western blot assay',28893958),(4232,'mouse','decrease','low-throughput','Using human intestinal epithelial cells (IECs), we discovered that SCFAs, particularly butyrate, enhanced IEC barrier formation, induced IL10RA mRNA, IL-10RA protein, and transactivation through activated Stat3 and HDAC inhibition. Loss and gain of IL10RA expression directly correlates with IEC barrier formation and butyrate represses permeability-promoting claudin-2 (Cldn2) tight-junction protein expression through an IL-10RA-dependent mechanism. Our findings provide a novel mechanism by which microbial derived butyrate promotes barrier through IL-10RA-dependent repression of Cldn2.','cell culture','real-time PCR,qPCR,Western blot assay',28893958),(4243,'mouse','decrease','low-throughput','Ratelimited enzymes in the process of TG synthesis of the DGAT1/2 mRNA level in the HFD group were signifificantly increased compared with the LFD group. The DGAT2 mRNA level in the HFD-CBM group decreased in the same range as the LFD group, whereas signifificant differences of the DGAT1 mRNA level were not found between the HFD group and the HFD-CBM group.','cell culture','Western blot assay,GC/MS Analysis, real-time PCR',24166662),(4244,'mouse','decrease','low-throughput','several candidate strains were screened to find some probiotics as therapeutics using dextran sodium sulfate (DSS)-induced colitis in mice, and China Center for Type Culture Collection (CCTCC) M206119 strain led to an exacerbated phenotype of DSS-colitis mice. Expression of protective factors zonula occludens-1 and β-defensin 2 was downregulated after CCTCC M206119 treatment. There was an increase in the nuclear translocation of nuclear factor-κB in epithelial cells. Then, intestinal proinflammatory and anti-inflammatory cytokine responses were evaluated. Proinflammatory colonic cytokines [interleukin (IL)-1β, IL-6 and tumor necrosis factor-α] levels were clearly increased in CCTCC-M206119-treated animals, whereas anti-inflammatory colonic cytokine (IL-10) level was lowered compared with saline- or 5-ASA-treated DSS-colitis mice. Next, CCTCC M206119 strain was identified as Lactobacillus crispatus, so it was concluded that not all lactobacilli strains have beneficial effects on intestinal inflammation and that Lactobacillus crispatus (L. crispatus) CCTCC M206119 is involved in exacerbation of intestinal inflammation in DSS-colitis mice.','cell culture','RT-PCR,HPLC analysis',22654425),(4245,'mouse','decrease','low-throughput','several candidate strains were screened to find some probiotics as therapeutics using dextran sodium sulfate (DSS)-induced colitis in mice, and China Center for Type Culture Collection (CCTCC) M206119 strain led to an exacerbated phenotype of DSS-colitis mice. Expression of protective factors zonula occludens-1 and β-defensin 2 was downregulated after CCTCC M206119 treatment. There was an increase in the nuclear translocation of nuclear factor-κB in epithelial cells. Then, intestinal proinflammatory and anti-inflammatory cytokine responses were evaluated. Proinflammatory colonic cytokines [interleukin (IL)-1β, IL-6 and tumor necrosis factor-α] levels were clearly increased in CCTCC-M206119-treated animals, whereas anti-inflammatory colonic cytokine (IL-10) level was lowered compared with saline- or 5-ASA-treated DSS-colitis mice. Next, CCTCC M206119 strain was identified as Lactobacillus crispatus, so it was concluded that not all lactobacilli strains have beneficial effects on intestinal inflammation and that Lactobacillus crispatus (L. crispatus) CCTCC M206119 is involved in exacerbation of intestinal inflammation in DSS-colitis mice.','cell culture','RT-PCR,HPLC analysis',22654425),(4246,'mouse','increase','low-throughput','several candidate strains were screened to find some probiotics as therapeutics using dextran sodium sulfate (DSS)-induced colitis in mice, and China Center for Type Culture Collection (CCTCC) M206119 strain led to an exacerbated phenotype of DSS-colitis mice. Expression of protective factors zonula occludens-1 and β-defensin 2 was downregulated after CCTCC M206119 treatment. There was an increase in the nuclear translocation of nuclear factor-κB in epithelial cells. Then, intestinal proinflammatory and anti-inflammatory cytokine responses were evaluated. Proinflammatory colonic cytokines [interleukin (IL)-1β, IL-6 and tumor necrosis factor-α] levels were clearly increased in CCTCC-M206119-treated animals, whereas anti-inflammatory colonic cytokine (IL-10) level was lowered compared with saline- or 5-ASA-treated DSS-colitis mice. Next, CCTCC M206119 strain was identified as Lactobacillus crispatus, so it was concluded that not all lactobacilli strains have beneficial effects on intestinal inflammation and that Lactobacillus crispatus (L. crispatus) CCTCC M206119 is involved in exacerbation of intestinal inflammation in DSS-colitis mice.','cell culture','RT-PCR,HPLC analysis',22654425),(4247,'mouse','increase','low-throughput','several candidate strains were screened to find some probiotics as therapeutics using dextran sodium sulfate (DSS)-induced colitis in mice, and China Center for Type Culture Collection (CCTCC) M206119 strain led to an exacerbated phenotype of DSS-colitis mice. Expression of protective factors zonula occludens-1 and β-defensin 2 was downregulated after CCTCC M206119 treatment. There was an increase in the nuclear translocation of nuclear factor-κB in epithelial cells. Then, intestinal proinflammatory and anti-inflammatory cytokine responses were evaluated. Proinflammatory colonic cytokines [interleukin (IL)-1β, IL-6 and tumor necrosis factor-α] levels were clearly increased in CCTCC-M206119-treated animals, whereas anti-inflammatory colonic cytokine (IL-10) level was lowered compared with saline- or 5-ASA-treated DSS-colitis mice. Next, CCTCC M206119 strain was identified as Lactobacillus crispatus, so it was concluded that not all lactobacilli strains have beneficial effects on intestinal inflammation and that Lactobacillus crispatus (L. crispatus) CCTCC M206119 is involved in exacerbation of intestinal inflammation in DSS-colitis mice.','cell culture','RT-PCR,HPLC analysis',22654425),(4248,'mouse','increase','low-throughput','several candidate strains were screened to find some probiotics as therapeutics using dextran sodium sulfate (DSS)-induced colitis in mice, and China Center for Type Culture Collection (CCTCC) M206119 strain led to an exacerbated phenotype of DSS-colitis mice. Expression of protective factors zonula occludens-1 and β-defensin 2 was downregulated after CCTCC M206119 treatment. There was an increase in the nuclear translocation of nuclear factor-κB in epithelial cells. Then, intestinal proinflammatory and anti-inflammatory cytokine responses were evaluated. Proinflammatory colonic cytokines [interleukin (IL)-1β, IL-6 and tumor necrosis factor-α] levels were clearly increased in CCTCC-M206119-treated animals, whereas anti-inflammatory colonic cytokine (IL-10) level was lowered compared with saline- or 5-ASA-treated DSS-colitis mice. Next, CCTCC M206119 strain was identified as Lactobacillus crispatus, so it was concluded that not all lactobacilli strains have beneficial effects on intestinal inflammation and that Lactobacillus crispatus (L. crispatus) CCTCC M206119 is involved in exacerbation of intestinal inflammation in DSS-colitis mice.','cell culture','RT-PCR,HPLC analysis',22654425),(4253,'mouse','decrease','low-throughput','We extended our study to human samples to determine the effect of bile acids on CXCL16 mRNA expression of human liver sinusoidal endothelial cells. Similar to the mouse study, primary bile acids CDCA and TCA induced CXCL16 mRNA expression in SK-HEP1 cells (Fig. 5H). Next, the correlation between bile acids and CXCL16 expression was tested in nontumor liver tissues from HCC or cholangiocarcinoma patients of the Thailand Initiative for Genomics and Expression Research in Liver Cancer (TIGER-LC) cohort (34). Primary bile acid CDCA levels correlated with CXCL16 expression (Fig. 5I), whereas inverse correlation was seen for the secondary bile acid glycolithocholate (GLCA) (fig. S6I). The primary/secondary ratio was associated with CXCL16 increase (fig. S6J), indicating that the opposing effect of bile acids on CXCL16 expression also exists in humans.','cell culture','real-time PCR',29798856),(4254,'mouse','decrease','low-throughput','In summary,this study showed that it is butyrate, in stead of other substances produced by F.prausnitzii,inhibiting HDAC1 to activate Foxp3 and block the IL-6/STAT3/IL-17 pathway,thus regulating Th17/Treg cells and exerting an anti-inflammatory effect in IBD animal models.','cell culture','GC/MS analysis',29796620),(4255,'mouse','decrease','low-throughput','In summary,this study showed that it is butyrate, in stead of other substances produced by F.prausnitzii,inhibiting HDAC1 to activate Foxp3 and block the IL-6/STAT3/IL-17 pathway,thus regulating Th17/Treg cells and exerting an anti-inflammatory effect in IBD animal models.','cell culture','GC/MS analysis',29796620),(4256,'mouse','decrease','low-throughput','In summary,this study showed that it is butyrate, in stead of other substances produced by F.prausnitzii,inhibiting HDAC1 to activate Foxp3 and block the IL-6/STAT3/IL-17 pathway,thus regulating Th17/Treg cells and exerting an anti-inflammatory effect in IBD animal models.','cell culture','GC/MS analysis',29796620),(4257,'mouse','decrease','low-throughput','In summary,this study showed that it is butyrate, in stead of other substances produced by F.prausnitzii,inhibiting HDAC1 to activate Foxp3 and block the IL-6/STAT3/IL-17 pathway,thus regulating Th17/Treg cells and exerting an anti-inflammatory effect in IBD animal models.','cell culture','GC/MS analysis',29796620),(4258,'mouse','increase','low-throughput','In summary,this study showed that it is butyrate, in stead of other substances produced by F.prausnitzii,inhibiting HDAC1 to activate Foxp3 and block the IL-6/STAT3/IL-17 pathway,thus regulating Th17/Treg cells and exerting an anti-inflammatory effect in IBD animal models.','cell culture','GC/MS analysis',29796620),(4350,'mouse','increase','low-throughput','The main metabolites produced by B. thetaiotaomicron in the GIT were acetate (up to 6 mM) and propionate (up to 2 mM) (Figure 1D). We evaluated the effects of these bacterial metabolites on the amount of KLF4 protein produced by the mucus producing cell line HT29-MTX [31]. The amount of KLF4 protein was higher in the presence than in the absence of acetate (10 and 20 mM) (Figure 3A)whereas propionate (5 and 10 mM) and butyrate (1 and 5 mM) had no effect (Figure 3B, C).','cell culture','Western blot assay,real-time qPC,immunohistochemisty assay',23692866),(4352,'mouse','increase','low-throughput','The amount of KLF4 protein, a transcription factor involved in goblet cell terminal differentiation [30], was higher in Bt-2d and Bt-30d than GF rats (Figure 2D). Also, muc2, muc4, c1galt1 and b4galt4 mRNAs, encoding proteins involved in mucin synthesis and glycosylation, were significantly more abundant in Bt-2d and Bt-30d rats than in GF rats(Table 1). The klf4 mRNA level was higher in Bt-2d, but not in Bt-30d, than in controls (Table 1).','cell culture','Western blot assay,real-time qPC,immunohistochemisty assay',23692866),(4353,'mouse','increase','low-throughput','The amount of KLF4 protein, a transcription factor involved in goblet cell terminal differentiation [30], was higher in Bt-2d and Bt-30d than GF rats (Figure 2D). Also, muc2, muc4, c1galt1 and b4galt4 mRNAs, encoding proteins involved in mucin synthesis and glycosylation, were significantly more abundant in Bt-2d and Bt-30d rats than in GF rats(Table 1). The klf4 mRNA level was higher in Bt-2d, but not in Bt-30d, than in controls (Table 1).','cell culture','Western blot assay,real-time qPC,immunohistochemisty assay',23692866),(4354,'mouse','increase','low-throughput','The amount of KLF4 protein, a transcription factor involved in goblet cell terminal differentiation [30], was higher in Bt-2d and Bt-30d than GF rats (Figure 2D). Also, muc2, muc4, c1galt1 and b4galt4 mRNAs, encoding proteins involved in mucin synthesis and glycosylation, were significantly more abundant in Bt-2d and Bt-30d rats than in GF rats(Table 1). The klf4 mRNA level was higher in Bt-2d, but not in Bt-30d, than in controls (Table 1).','cell culture','Western blot assay,real-time qPC,immunohistochemisty assay',23692866),(4355,'mouse','increase','low-throughput','The amount of KLF4 protein, a transcription factor involved in goblet cell terminal differentiation [30], was higher in Bt-2d and Bt-30d than GF rats (Figure 2D). Also, muc2, muc4, c1galt1 and b4galt4 mRNAs, encoding proteins involved in mucin synthesis and glycosylation, were significantly more abundant in Bt-2d and Bt-30d rats than in GF rats(Table 1). The klf4 mRNA level was higher in Bt-2d, but not in Bt-30d, than in controls (Table 1).','cell culture','Western blot assay,real-time qPC,immunohistochemisty assay',23692866),(4356,'mouse','increase','low-throughput','We observed an increase in the expression of the sialyltransferase st3gal4.','cell culture','Western blot assay,real-time qPC,immunohistochemisty assay',23692866),(4357,'mouse','increase','low-throughput','We first checked  in vitro  that  B. thetaiotaomicron produced acetate and propionate and that  F. prausnitzii consumed acetate and produced butyrate (Figure 4A). When both bacteria were grown together, butyrate was produced rather than acetate. The butyrate production by F. prausnitzii as higher in the presence than absence of B. thetaiotaomicron  (10.9 ± 0.3 mM versus 7.8 ± 0.5 mM) suggesting that  F. prausnitzii  synthesized butyrate from the acetate produced by  B. thetaiotaomicron .','cell culture','Western blot assay,real-time qPC,immunohistochemisty assay',23692866),(4358,'mouse','increase','low-throughput','In fact,miR-21 was reportedly induced by pathogenic microbes such as Helicobacter pylori, Salmonella Typhimurium, and Mycobacterium species, leading to excessive immune responses (24, 25). Neither LPS nor P3CSK4 acted in a dose-dependent manner, suggesting that cellular mechanisms suppress the excessive induction of miR-21-5p.','cell culture','RT-qPCR,Western blot assay',28760826),(4359,'mouse','increase','low-throughput','In fact,miR-21 was reportedly induced by pathogenic microbes such as Helicobacter pylori, Salmonella Typhimurium, and Mycobacterium species, leading to excessive immune responses (24, 25). Neither LPS nor P3CSK4 acted in a dose-dependent manner, suggesting that cellular mechanisms suppress the excessive induction of miR-21-5p.','cell culture','RT-qPCR,Western blot assay',28760826),(4360,'mouse','increase','low-throughput','In fact,miR-21 was reportedly induced by pathogenic microbes such as Helicobacter pylori, Salmonella Typhimurium, and Mycobacterium species, leading to excessive immune responses (24, 25). Neither LPS nor P3CSK4 acted in a dose-dependent manner, suggesting that cellular mechanisms suppress the excessive induction of miR-21-5p.','cell culture','RT-qPCR,Western blot assay',28760826),(4421,'mouse','decrease','low-throughput','C. butyricum suppresses the expression of MMP-9 in asthmatic mice.','cell culture','ELISA',28122397),(4422,'mouse','decrease','low-throughput','A recent study suggests that for an OVA-sensitized/ challenged mouse, oral L. rhamnosus GG (LGG) can not only reduce lung resistance and pulmonary inflam mation, but also suppress the expression of T helper 2 (Th2) cytokines, MMP-9 and OVA-specific IgE.16 Sim ilarly, oral administration of C. butyricum to asthmatic mice is effective in reducing allergic airway inflamma tion and remodelling in our study.','cell culture','ELISA',28122397),(4423,'mouse','decrease','low-throughput','Both INF-γ(Fig. S2E, Supplementary Information) and IL-10 (Fig. S2F,I, Supplementary Information) in the PC group were decreased compared with the counterparts of the NC group. They increased in the groups that received C. butyricum orally except for BALF IL-10 in the Tre group.','cell culture','ELISA',28122397),(4424,'mouse','decrease','low-throughput','Compared with the NC group, IgE (Fig. 4A) and IgG1(Fig. 4B) were signifificantly elevated in the PC group (P < 0.001). However, they decreased dramatically in the Pre and Tre groups, especially the level of IgG1 in the Tre group.','cell culture','ELISA',28122397),(4462,'mouse','increase','low-throughput','The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.','cell culture','null',26725514),(4463,'mouse','increase','low-throughput','The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.','cell culture','null',26725514),(4464,'mouse','increase','low-throughput','The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.','cell culture','null',26725514),(4465,'mouse','increase','low-throughput','The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.','cell culture','null',26725514),(4466,'mouse','increase','low-throughput','The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.','cell culture','null',26725514),(4467,'mouse','increase','low-throughput','The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.','cell culture','null',26725514),(4468,'mouse','increase','low-throughput','The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.','cell culture','null',26725514),(4469,'mouse','increase','low-throughput','The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.','cell culture','null',26725514),(4470,'mouse','increase','low-throughput','The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.','cell culture','null',26725514),(4471,'mouse','increase','low-throughput','The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.','cell culture','null',26725514),(4472,'mouse','increase','low-throughput','The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.','cell culture','null',26725514),(4473,'mouse','increase','low-throughput','The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.','cell culture','null',26725514),(4474,'mouse','increase','low-throughput','The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.','cell culture','null',26725514),(4475,'mouse','increase','low-throughput','The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.','cell culture','null',26725514),(4476,'mouse','increase','low-throughput','The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.','cell culture','null',26725514),(4477,'mouse','increase','low-throughput','The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.','cell culture','null',26725514),(4478,'mouse','increase','low-throughput','The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.','cell culture','null',26725514),(4479,'mouse','increase','low-throughput','The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.','cell culture','null',26725514),(4480,'mouse','increase','low-throughput','The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.','cell culture','null',26725514),(4481,'mouse','increase','low-throughput','The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.','cell culture','null',26725514),(4482,'mouse','increase','low-throughput','The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.','cell culture','null',26725514),(4483,'mouse','increase','low-throughput','The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.','cell culture','null',26725514),(4484,'mouse','increase','low-throughput','The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.','cell culture','null',26725514),(4485,'mouse','increase','low-throughput','The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.','cell culture','null',26725514),(4486,'mouse','increase','low-throughput','The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.','cell culture','null',26725514),(4487,'mouse','increase','low-throughput','The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.','cell culture','null',26725514),(4488,'mouse','increase','low-throughput','The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.','cell culture','null',26725514),(4489,'mouse','increase','low-throughput','The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.','cell culture','null',26725514),(4490,'mouse','increase','low-throughput','The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.','cell culture','null',26725514),(4491,'mouse','increase','low-throughput','The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.','cell culture','null',26725514),(4492,'mouse','increase','low-throughput','The largest amounts of IL-10 were induced by the Fp strains, Eubacterium hallii (Eh) L2-7 and Megamonas rupellenesis (Mr) Mag1B. In addition, these strains induced the largest amounts of TNF-α and IL-6. The Fp strains, Eh L2-7 and Eubacterium rectale (Er) A1-86 also induced relatively large amounts of IL-1β. Lactobacillus plantarum (Lp) WCFS1, Mr Mag1B and Eh L2-7 were among the most potent inducers of IL-12p70, whereas the Fp strains were among the poorer inducers (Fig. 1). Surprisingly, Ruminococcus bromii (Rb) L2-63, Ch 82-B and Clostridium xylanovorans (Cx) Lac1D induced small or undetectable amounts of some cytokines (Fig. 1) despite having effectively induced hDCs maturation (Supplementary Fig. S1). The five Fp strains had very different cytokines profiles. The highest IL-10 producer was strain A2-165, strains L2-6 and M21/2 induced high amounts of IL-10 but also IL-12p70 while strains S3L/3 and HTF-F induced intermediate to low amounts of both cytokines.','cell culture','null',26725514),(4493,'mouse','increase','low-throughput','The metalloprotease GelE, produced by commensal strains of E faecalis, contributes to development of chronic intestinal inflflammation in mice that are susceptible to intestinal inflflammation by impairing epithelial barrier in tegrity.','cell culture','LC-MS/MS analysis',21699778),(4496,'mouse','increase','low-throughput','Our results suggest this is due to GF colonocytes adapting to their suboptimal energetic state by undergoing autophagy. Reduced ATP levels activate AMPK, which, in turn, activates p27 to prevent apoptosis and induce autophagy in these cells.','cell culture','RT-qPCR,Western blot assay,HPLC analysis,LC/MS analysis',21531334),(4497,'mouse','increase','low-throughput','we performed transcriptome profiling and quantitative mass spectrometry on freshly isolated colonocytes from GF and CONV-R mice. For the transcriptome profiling experiments, we identified 624 genes that are upregulated and 813 genes are downregulated in GF compared to CONV-R. All of these gene expression changes are statistically significant with false discovery rates below 0.01. The entire data set and statistical values are included as Table S1. Our transcriptome data includes genes previously shown to be regulated by microbiota in various tissues of zebrafish, mice, and piglets. For example, Angpt14/Fiaf is upregulated in GF compared to CONV while Gpx2, ApoB, Aqp4, and Pla2g4c are downregulated.','cell culture','RT-qPCR,Western blot assay,HPLC analysis,LC/MS analysis',21531334),(4498,'mouse','decrease','low-throughput','we performed transcriptome profiling and quantitative mass spectrometry on freshly isolated colonocytes from GF and CONV-R mice. For the transcriptome profiling experiments, we identified 624 genes that are upregulated and 813 genes are downregulated in GF compared to CONV-R. All of these gene expression changes are statistically significant with false discovery rates below 0.01. The entire data set and statistical values are included as Table S1. Our transcriptome data includes genes previously shown to be regulated by microbiota in various tissues of zebrafish, mice, and piglets. For example, Angpt14/Fiaf is upregulated in GF compared to CONV while Gpx2, ApoB, Aqp4, and Pla2g4c are downregulated.','cell culture','RT-qPCR,Western blot assay,HPLC analysis,LC/MS analysis',21531334),(4499,'mouse','decrease','low-throughput','we performed transcriptome profiling and quantitative mass spectrometry on freshly isolated colonocytes from GF and CONV-R mice. For the transcriptome profiling experiments, we identified 624 genes that are upregulated and 813 genes are downregulated in GF compared to CONV-R. All of these gene expression changes are statistically significant with false discovery rates below 0.01. The entire data set and statistical values are included as Table S1. Our transcriptome data includes genes previously shown to be regulated by microbiota in various tissues of zebrafish, mice, and piglets. For example, Angpt14/Fiaf is upregulated in GF compared to CONV while Gpx2, ApoB, Aqp4, and Pla2g4c are downregulated.','cell culture','RT-qPCR,Western blot assay,HPLC analysis,LC/MS analysis',21531334),(4500,'mouse','decrease','low-throughput','we performed transcriptome profiling and quantitative mass spectrometry on freshly isolated colonocytes from GF and CONV-R mice. For the transcriptome profiling experiments, we identified 624 genes that are upregulated and 813 genes are downregulated in GF compared to CONV-R. All of these gene expression changes are statistically significant with false discovery rates below 0.01. The entire data set and statistical values are included as Table S1. Our transcriptome data includes genes previously shown to be regulated by microbiota in various tissues of zebrafish, mice, and piglets. For example, Angpt14/Fiaf is upregulated in GF compared to CONV while Gpx2, ApoB, Aqp4, and Pla2g4c are downregulated.','cell culture','RT-qPCR,Western blot assay,HPLC analysis,LC/MS analysis',21531334),(4501,'mouse','decrease','low-throughput','we performed transcriptome profiling and quantitative mass spectrometry on freshly isolated colonocytes from GF and CONV-R mice. For the transcriptome profiling experiments, we identified 624 genes that are upregulated and 813 genes are downregulated in GF compared to CONV-R. All of these gene expression changes are statistically significant with false discovery rates below 0.01. The entire data set and statistical values are included as Table S1. Our transcriptome data includes genes previously shown to be regulated by microbiota in various tissues of zebrafish, mice, and piglets. For example, Angpt14/Fiaf is upregulated in GF compared to CONV while Gpx2, ApoB, Aqp4, and Pla2g4c are downregulated.','cell culture','RT-qPCR,Western blot assay,HPLC analysis,LC/MS analysis',21531334),(4502,'mouse','increase','low-throughput','Our results suggest this is due to GF colonocytes adapting to their suboptimal energetic state by undergoing autophagy. Reduced ATP levels activate AMPK, which, in turn, activates p27 to prevent apoptosis and induce autophagy in these cells.','cell culture','RT-qPCR,Western blot assay,HPLC analysis,LC/MS analysis',21531334),(4503,'mouse','increase','low-throughput','Our results suggest this is due to GF colonocytes adapting to their suboptimal energetic state by undergoing autophagy. Reduced ATP levels activate AMPK, which, in turn, activates p27 to prevent apoptosis and induce autophagy in these cells.','cell culture','RT-qPCR,Western blot assay,HPLC analysis,LC/MS analysis',21531334),(4504,'mouse','increase','low-throughput','Our results suggest this is due to GF colonocytes adapting to their suboptimal energetic state by undergoing autophagy. Reduced ATP levels activate AMPK, which, in turn, activates p27 to prevent apoptosis and induce autophagy in these cells.','cell culture','RT-qPCR,Western blot assay,HPLC analysis,LC/MS analysis',21531334),(4505,'mouse','increase','low-throughput','Our results suggest this is due to GF colonocytes adapting to their suboptimal energetic state by undergoing autophagy. Reduced ATP levels activate AMPK, which, in turn, activates p27 to prevent apoptosis and induce autophagy in these cells.','cell culture','RT-qPCR,Western blot assay,HPLC analysis,LC/MS analysis',21531334),(4506,'mouse','increase','low-throughput','Our results suggest this is due to GF colonocytes adapting to their suboptimal energetic state by undergoing autophagy. Reduced ATP levels activate AMPK, which, in turn, activates p27 to prevent apoptosis and induce autophagy in these cells.','cell culture','RT-qPCR,Western blot assay,HPLC analysis,LC/MS analysis',21531334),(4509,'mouse','decrease','low-throughput','mRNA levels of genes encod ing proteins involved in lipogenesis, including Srebf1, Scd1 and Fasn,  were lower in white adipose tissue of HFD mice gavaged with live  B. thetaiotaomicron, whereas expression of genes involved in lipolysis  and fatty acid oxidation, including Adrb3, Pnpla2, Cpt2, Acox1 and  Ppargc1a was higher (Fig. 4k). In addition, expression of markers of  inflammation, Cd68, Ccl2, Il6 and Il1b, in adipose tissue of HFD mice  was attenuated in response to gavage with live B. thetaiotaomicron,  indicating an improved inflammatory status.','cell culture','RT-qPCR,Western blot assay,HPLC analysis,LC/MS analysis',28628112),(4510,'mouse','decrease','low-throughput','mRNA levels of genes encod ing proteins involved in lipogenesis, including Srebf1, Scd1 and Fasn,  were lower in white adipose tissue of HFD mice gavaged with live  B. thetaiotaomicron, whereas expression of genes involved in lipolysis  and fatty acid oxidation, including Adrb3, Pnpla2, Cpt2, Acox1 and  Ppargc1a was higher (Fig. 4k). In addition, expression of markers of  inflammation, Cd68, Ccl2, Il6 and Il1b, in adipose tissue of HFD mice  was attenuated in response to gavage with live B. thetaiotaomicron,  indicating an improved inflammatory status.','cell culture','real-time PCR,Western blot assay,HPLC analysis,LC/MS analysis,GC/MS analysis',28628112),(4511,'mouse','decrease','low-throughput','mRNA levels of genes encod ing proteins involved in lipogenesis, including Srebf1, Scd1 and Fasn,  were lower in white adipose tissue of HFD mice gavaged with live  B. thetaiotaomicron, whereas expression of genes involved in lipolysis  and fatty acid oxidation, including Adrb3, Pnpla2, Cpt2, Acox1 and  Ppargc1a was higher (Fig. 4k). In addition, expression of markers of  inflammation, Cd68, Ccl2, Il6 and Il1b, in adipose tissue of HFD mice  was attenuated in response to gavage with live B. thetaiotaomicron,  indicating an improved inflammatory status.','cell culture','real-time PCR,Western blot assay,HPLC analysis,LC/MS analysis,GC/MS analysis',28628112),(4514,'mouse','increase','low-throughput','mRNA levels of genes encod ing proteins involved in lipogenesis, including Srebf1, Scd1 and Fasn,  were lower in white adipose tissue of HFD mice gavaged with live  B. thetaiotaomicron, whereas expression of genes involved in lipolysis  and fatty acid oxidation, including Adrb3, Pnpla2, Cpt2, Acox1 and  Ppargc1a was higher (Fig. 4k). In addition, expression of markers of  inflammation, Cd68, Ccl2, Il6 and Il1b, in adipose tissue of HFD mice  was attenuated in response to gavage with live B. thetaiotaomicron,  indicating an improved inflammatory status.','cell culture','real-time PCR,Western blot assay,HPLC analysis,LC/MS analysis,GC/MS analysis',28628112),(4515,'mouse','increase','low-throughput','mRNA levels of genes encod ing proteins involved in lipogenesis, including Srebf1, Scd1 and Fasn,  were lower in white adipose tissue of HFD mice gavaged with live  B. thetaiotaomicron, whereas expression of genes involved in lipolysis  and fatty acid oxidation, including Adrb3, Pnpla2, Cpt2, Acox1 and  Ppargc1a was higher (Fig. 4k). In addition, expression of markers of  inflammation, Cd68, Ccl2, Il6 and Il1b, in adipose tissue of HFD mice  was attenuated in response to gavage with live B. thetaiotaomicron,  indicating an improved inflammatory status.','cell culture','real-time PCR,Western blot assay,HPLC analysis,LC/MS analysis,GC/MS analysis',28628112),(4516,'mouse','increase','low-throughput','mRNA levels of genes encod ing proteins involved in lipogenesis, including Srebf1, Scd1 and Fasn,  were lower in white adipose tissue of HFD mice gavaged with live  B. thetaiotaomicron, whereas expression of genes involved in lipolysis  and fatty acid oxidation, including Adrb3, Pnpla2, Cpt2, Acox1 and  Ppargc1a was higher (Fig. 4k). In addition, expression of markers of  inflammation, Cd68, Ccl2, Il6 and Il1b, in adipose tissue of HFD mice  was attenuated in response to gavage with live B. thetaiotaomicron,  indicating an improved inflammatory status.','cell culture','real-time PCR,Western blot assay,HPLC analysis,LC/MS analysis,GC/MS analysis',28628112),(4517,'mouse','increase','low-throughput','mRNA levels of genes encod ing proteins involved in lipogenesis, including Srebf1, Scd1 and Fasn,  were lower in white adipose tissue of HFD mice gavaged with live  B. thetaiotaomicron, whereas expression of genes involved in lipolysis  and fatty acid oxidation, including Adrb3, Pnpla2, Cpt2, Acox1 and  Ppargc1a was higher (Fig. 4k). In addition, expression of markers of  inflammation, Cd68, Ccl2, Il6 and Il1b, in adipose tissue of HFD mice  was attenuated in response to gavage with live B. thetaiotaomicron,  indicating an improved inflammatory status.','cell culture','real-time PCR,Western blot assay,HPLC analysis,LC/MS analysis,GC/MS analysis',28628112),(4518,'mouse','increase','low-throughput','mRNA levels of genes encod ing proteins involved in lipogenesis, including Srebf1, Scd1 and Fasn,  were lower in white adipose tissue of HFD mice gavaged with live  B. thetaiotaomicron, whereas expression of genes involved in lipolysis  and fatty acid oxidation, including Adrb3, Pnpla2, Cpt2, Acox1 and  Ppargc1a was higher (Fig. 4k). In addition, expression of markers of  inflammation, Cd68, Ccl2, Il6 and Il1b, in adipose tissue of HFD mice  was attenuated in response to gavage with live B. thetaiotaomicron,  indicating an improved inflammatory status.','cell culture','real-time PCR,Western blot assay,HPLC analysis,LC/MS analysis,GC/MS analysis',28628112),(4519,'mouse','decrease','low-throughput','mRNA levels of genes encod ing proteins involved in lipogenesis, including Srebf1, Scd1 and Fasn,  were lower in white adipose tissue of HFD mice gavaged with live  B. thetaiotaomicron, whereas expression of genes involved in lipolysis  and fatty acid oxidation, including Adrb3, Pnpla2, Cpt2, Acox1 and  Ppargc1a was higher (Fig. 4k). In addition, expression of markers of  inflammation, Cd68, Ccl2, Il6 and Il1b, in adipose tissue of HFD mice  was attenuated in response to gavage with live B. thetaiotaomicron,  indicating an improved inflammatory status.','cell culture','real-time PCR,Western blot assay,HPLC analysis,LC/MS analysis,GC/MS analysis',28628112),(4520,'mouse','decrease','low-throughput','mRNA levels of genes encod ing proteins involved in lipogenesis, including Srebf1, Scd1 and Fasn,  were lower in white adipose tissue of HFD mice gavaged with live  B. thetaiotaomicron, whereas expression of genes involved in lipolysis  and fatty acid oxidation, including Adrb3, Pnpla2, Cpt2, Acox1 and  Ppargc1a was higher (Fig. 4k). In addition, expression of markers of  inflammation, Cd68, Ccl2, Il6 and Il1b, in adipose tissue of HFD mice  was attenuated in response to gavage with live B. thetaiotaomicron,  indicating an improved inflammatory status.','cell culture','real-time PCR,Western blot assay,HPLC analysis,LC/MS analysis,GC/MS analysis',28628112),(4521,'mouse','decrease','low-throughput','mRNA levels of genes encod ing proteins involved in lipogenesis, including Srebf1, Scd1 and Fasn,  were lower in white adipose tissue of HFD mice gavaged with live  B. thetaiotaomicron, whereas expression of genes involved in lipolysis  and fatty acid oxidation, including Adrb3, Pnpla2, Cpt2, Acox1 and  Ppargc1a was higher (Fig. 4k). In addition, expression of markers of  inflammation, Cd68, Ccl2, Il6 and Il1b, in adipose tissue of HFD mice  was attenuated in response to gavage with live B. thetaiotaomicron,  indicating an improved inflammatory status.','cell culture','real-time PCR,Western blot assay,HPLC analysis,LC/MS analysis,GC/MS analysis',28628112),(4522,'mouse','decrease','low-throughput','mRNA levels of genes encod ing proteins involved in lipogenesis, including Srebf1, Scd1 and Fasn,  were lower in white adipose tissue of HFD mice gavaged with live  B. thetaiotaomicron, whereas expression of genes involved in lipolysis  and fatty acid oxidation, including Adrb3, Pnpla2, Cpt2, Acox1 and  Ppargc1a was higher (Fig. 4k). In addition, expression of markers of  inflammation, Cd68, Ccl2, Il6 and Il1b, in adipose tissue of HFD mice  was attenuated in response to gavage with live B. thetaiotaomicron,  indicating an improved inflammatory status.','cell culture','real-time PCR,Western blot assay,HPLC analysis,LC/MS analysis,GC/MS analysis',28628112),(4523,'mouse','increase','low-throughput','Here we show using untargeted metabolomics that levels of phenyl sulfate, a gut microbiota-derived metabolite, increase with the progression of diabetes in rats overexpressing human uremic toxin transporter SLCO4C1 in the kidney, and are decreased in rats with limited proteinuria.','cell culture','UHPLC analysis,UHPLC-QTOF/MS analysis,LC/MS analysis',31015435),(4524,'mouse','increase','low-throughput','although there was no difference between in the preserved tubular area and fibrotic area at the light microscopic level, the expression levels of representative inflammatory genes, TNF-α (Tnfa) and MCP-1 (Ccl2), the fibrotic gene TGF-α1 (Tgfb1), fibronectin (Fn1), and collagen I (Col1a1) were increased in PS-treated db/db kidney. It was reported that db/db mice develop renal fibrosis.','cell culture','UHPLC analysis,UHPLC-QTOF/MS analysis,LC/MS analysis',31015435),(4525,'mouse','increase','low-throughput','although there was no difference between in the preserved tubular area and fibrotic area at the light microscopic level, the expression levels of representative inflammatory genes, TNF-α (Tnfa) and MCP-1 (Ccl2), the fibrotic gene TGF-α1 (Tgfb1), fibronectin (Fn1), and collagen I (Col1a1) were increased in PS-treated db/db kidney. It was reported that db/db mice develop renal fibrosis.','cell culture','UHPLC analysis,UHPLC-QTOF/MS analysis,LC/MS analysis',31015435),(4526,'mouse','increase','low-throughput','although there was no difference between in the preserved tubular area and fibrotic area at the light microscopic level, the expression levels of representative inflammatory genes, TNF-α (Tnfa) and MCP-1 (Ccl2), the fibrotic gene TGF-α1 (Tgfb1), fibronectin (Fn1), and collagen I (Col1a1) were increased in PS-treated db/db kidney. It was reported that db/db mice develop renal fibrosis.','cell culture','UHPLC analysis,UHPLC-QTOF/MS analysis,LC/MS analysis',31015435),(4527,'mouse','increase','low-throughput','although there was no difference between in the preserved tubular area and fibrotic area at the light microscopic level, the expression levels of representative inflammatory genes, TNF-α (Tnfa) and MCP-1 (Ccl2), the fibrotic gene TGF-α1 (Tgfb1), fibronectin (Fn1), and collagen I (Col1a1) were increased in PS-treated db/db kidney. It was reported that db/db mice develop renal fibrosis.','cell culture','UHPLC analysis,UHPLC-QTOF/MS analysis,LC/MS analysis',31015435),(4528,'mouse','increase','low-throughput','although there was no difference between in the preserved tubular area and fibrotic area at the light microscopic level, the expression levels of representative inflammatory genes, TNF-α (Tnfa) and MCP-1 (Ccl2), the fibrotic gene TGF-α1 (Tgfb1), fibronectin (Fn1), and collagen I (Col1a1) were increased in PS-treated db/db kidney. It was reported that db/db mice develop renal fibrosis.','cell culture','UHPLC analysis,UHPLC-QTOF/MS analysis,LC/MS analysis',31015435),(4529,'mouse','increase','low-throughput','The expression of tight junction protein ZO-1, Occludin and Claudin-1 was measured by western blot. As shown in Fig. 2A, the expression of Claudin-1 was significantly decreased in irinotecan treated mice compared with control group. However, ZO-1 and Oc cludin expression was not modified by irinotecan. Interestingly, EcN significantly restored the decreased Claudin-1 expression.','cell culture','Western blot assay',31173781),(4531,'mouse','increase','low-throughput','Live B. dentium-monoassociated mice exhibited increased colonic expression of goblet cell markers Krüppel-like factor 4 (Klf4), Trefoil factor 3 (Tff3), Relm-β, Muc2, and several glycosyltransferases compared to both heat-killed B. dentium and germfree counterparts.','cell culture','real-time qPCR',31213556),(4532,'mouse','increase','low-throughput','Live B. dentium-monoassociated mice exhibited increased colonic expression of goblet cell markers Krüppel-like factor 4 (Klf4), Trefoil factor 3 (Tff3), Relm-β, Muc2, and several glycosyltransferases compared to both heat-killed B. dentium and germfree counterparts.','cell culture','real-time qPCR',31213556),(4533,'mouse','increase','low-throughput','Live B. dentium-monoassociated mice exhibited increased colonic expression of goblet cell markers Krüppel-like factor 4 (Klf4), Trefoil factor 3 (Tff3), Relm-β, Muc2, and several glycosyltransferases compared to both heat-killed B. dentium and germfree counterparts.','cell culture','real-time qPCR',31213556),(4534,'mouse','increase','low-throughput','Live B. dentium-monoassociated mice exhibited increased colonic expression of goblet cell markers Krüppel-like factor 4 (Klf4), Trefoil factor 3 (Tff3), Relm-β, Muc2, and several glycosyltransferases compared to both heat-killed B. dentium and germfree counterparts.','cell culture','real-time qPCR',31213556),(4535,'mouse','increase','low-throughput','Increases in B. dentium-monoassociated colons compared with germfree controls were observed in the following glycosyltransferases: B3gnt6 (β-1,3-N-acetylglucosaminyltransferase 6), C1galt1 (β-1,3-galactosyltransferase 1), C2gnt [glucosaminyl (N-acetyl)transferase 1], Fut2 (fucosyltransferase 2), C2GnT1 [glucosaminyl (N-acetyl)transferase 1], and St6gal1 (ST6 N-acetylgalactosaminide α-2,6-sialyltransferase).','cell culture','real-time qPCR',31213556),(4536,'mouse','increase','low-throughput','Increases in B. dentium-monoassociated colons compared with germfree controls were observed in the following glycosyltransferases: B3gnt6 (β-1,3-N-acetylglucosaminyltransferase 6), C1galt1 (β-1,3-galactosyltransferase 1), C2gnt [glucosaminyl (N-acetyl)transferase 1], Fut2 (fucosyltransferase 2), C2GnT1 [glucosaminyl (N-acetyl)transferase 1], and St6gal1 (ST6 N-acetylgalactosaminide α-2,6-sialyltransferase).','cell culture','real-time qPCR',31213556),(4537,'mouse','increase','low-throughput','Increases in B. dentium-monoassociated colons compared with germfree controls were observed in the following glycosyltransferases: B3gnt6 (β-1,3-N-acetylglucosaminyltransferase 6), C1galt1 (β-1,3-galactosyltransferase 1), C2gnt [glucosaminyl (N-acetyl)transferase 1], Fut2 (fucosyltransferase 2), C2GnT1 [glucosaminyl (N-acetyl)transferase 1], and St6gal1 (ST6 N-acetylgalactosaminide α-2,6-sialyltransferase).','cell culture','real-time qPCR',31213556),(4538,'mouse','increase','low-throughput','Increases in B. dentium-monoassociated colons compared with germfree controls were observed in the following glycosyltransferases: B3gnt6 (β-1,3-N-acetylglucosaminyltransferase 6), C1galt1 (β-1,3-galactosyltransferase 1), C2gnt [glucosaminyl (N-acetyl)transferase 1], Fut2 (fucosyltransferase 2), C2GnT1 [glucosaminyl (N-acetyl)transferase 1], and St6gal1 (ST6 N-acetylgalactosaminide α-2,6-sialyltransferase).','cell culture','real-time qPCR',31213556),(4539,'mouse','increase','low-throughput','Increases in B. dentium-monoassociated colons compared with germfree controls were observed in the following glycosyltransferases: B3gnt6 (β-1,3-N-acetylglucosaminyltransferase 6), C1galt1 (β-1,3-galactosyltransferase 1), C2gnt [glucosaminyl (N-acetyl)transferase 1], Fut2 (fucosyltransferase 2), C2GnT1 [glucosaminyl (N-acetyl)transferase 1], and St6gal1 (ST6 N-acetylgalactosaminide α-2,6-sialyltransferase).','cell culture','real-time qPCR',31213556),(4540,'mouse','increase','low-throughput','We examined a number of known autophagy markers by qPCR and found that B. dentium colonization was associated with increased expression of ATG16L, ATG5, Beclin1, LAMP2, and LC3.','cell culture','real-time qPCR',31213556),(4541,'mouse','increase','low-throughput','We examined a number of known autophagy markers by qPCR and found that B. dentium colonization was associated with increased expression of ATG16L, ATG5, Beclin1, LAMP2, and LC3.','cell culture','real-time qPCR',31213556),(4542,'mouse','increase','low-throughput','We examined a number of known autophagy markers by qPCR and found that B. dentium colonization was associated with increased expression of ATG16L, ATG5, Beclin1, LAMP2, and LC3.','cell culture','real-time qPCR',31213556),(4543,'mouse','increase','low-throughput','We examined a number of known autophagy markers by qPCR and found that B. dentium colonization was associated with increased expression of ATG16L, ATG5, Beclin1, LAMP2, and LC3.','cell culture','real-time qPCR',31213556),(4544,'mouse','increase','low-throughput','We examined a number of known autophagy markers by qPCR and found that B. dentium colonization was associated with increased expression of ATG16L, ATG5, Beclin1, LAMP2, and LC3.','cell culture','real-time qPCR',31213556),(4545,'mouse','increase','low-throughput','B. dentium can secrete products, such as acetate, that are capable of increasing MUC2 synthesis.','cell culture','real-time qPCR',31213556),(4546,'mouse','increase','low-throughput','Of note, a trend toward increased expression of Relm-β and Muc2 was observed in heat-killed B. dentium-treated mice. Although not significant, this may point to a complementary role of B. dentium surface proteins in modulating goblet cells. Several cytokines are associated with increased mucus production, most notably interleukin-22 (IL-22), IL-33, and IL-13.','cell culture','real-time qPCR',31213556),(4547,'mouse','increase','low-throughput','Of note, a trend toward increased expression of Relm-β and Muc2 was observed in heat-killed B. dentium-treated mice. Although not significant, this may point to a complementary role of B. dentium surface proteins in modulating goblet cells. Several cytokines are associated with increased mucus production, most notably interleukin-22 (IL-22), IL-33, and IL-13.','cell culture','real-time qPCR',31213556),(4548,'mouse','increase','low-throughput','Of note, a trend toward increased expression of Relm-β and Muc2 was observed in heat-killed B. dentium-treated mice. Although not significant, this may point to a complementary role of B. dentium surface proteins in modulating goblet cells. Several cytokines are associated with increased mucus production, most notably interleukin-22 (IL-22), IL-33, and IL-13.','cell culture','real-time qPCR',31213556),(4557,'mouse','decrease','low-throughput','the secretion of NF-kB-dependent cytokines (TNF-a, IL-1b and IL-6) was dramatically inhibited by treatment with live EC-M17.','control experiment,in vitro bacterial culture','Western blot analysis',18279557),(4558,'mouse','decrease','low-throughput','the secretion of NF-kB-dependent cytokines (TNF-a, IL-1b and IL-6) was dramatically inhibited by treatment with live EC-M17.','control experiment,in vitro bacterial culture','Western blot analysis',18279557),(4559,'mouse','decrease','low-throughput','the secretion of NF-kB-dependent cytokines (TNF-a, IL-1b and IL-6) was dramatically inhibited by treatment with live EC-M17.','control experiment,in vitro bacterial culture','Western blot analysis',18279557),(4560,'mouse','increase','low-throughput','D8 up-regulates IL-22 expression ex vivo and in vivo.','control experiment,in vitro bacterial culture','RT-qPCR,real-time PCR,real-time qPCR,16s rRNA sequence',29459771),(4565,'mouse','decrease','low-throughput','The OLL2838 strain significantly suppressed TNF-α-induced IL-8 production.','control experiment,in vitro bacterial culture','ELISA,Western blot assay',19447972),(4566,'mouse','increase','low-throughput','The OLL2838 strain prevented the loss of ZO-1 and MLCK and suppressed the increase in mucosal permeability.','control experiment,in vitro bacterial culture','ELISA,Western blot assay',19447972),(4567,'mouse','increase','low-throughput','The OLL2838 strain prevented the loss of ZO-1 and MLCK and suppressed the increase in mucosal permeability.','control experiment,in vitro bacterial culture','ELISA,Western blot assay',19447972),(4570,'mouse','decrease','low-throughput','The mRNA expression of IL-1β, IL-6, IL-17A and IFN-γ decreased significantly in the DSS plus TN-3 mice as compared to the DSS mice.','control experiment','real-time PCR,HPLC analysis,T-RFLP',27438072),(4571,'mouse','decrease','low-throughput','The mRNA expression of IL-1β, IL-6, IL-17A and IFN-γ decreased significantly in the DSS plus TN-3 mice as compared to the DSS mice.','control experiment','real-time PCR,HPLC analysis,T-RFLP',27438072),(4572,'mouse','decrease','low-throughput','The mRNA expression of IL-1β, IL-6, IL-17A and IFN-γ decreased significantly in the DSS plus TN-3 mice as compared to the DSS mice.','control experiment','real-time PCR,HPLC analysis,T-RFLP',27438072),(4573,'mouse','decrease','low-throughput','The mRNA expression of IL-1β, IL-6, IL-17A and IFN-γ decreased significantly in the DSS plus TN-3 mice as compared to the DSS mice.','control experiment','real-time PCR,HPLC analysis,T-RFLP',27438072),(4574,'mouse','increase','low-throughput','The expression of IL-10 increased in the colon tissues of the DSS plus TN-3 mice as compared to the DSS mice using immunohistochemistry.','control experiment','real-time PCR,HPLC analysis,T-RFLP',27438072),(4576,'mouse','decrease','low-throughput','Indole, the most prevalent metabolite of tryptophan, is produced by a wide range of bacterial species, including those belonging to the genera Escherichia, Bacteroides, and ClostridiumProlonged exposure to indole inhibits ATP production and thus GLP-1 secretion.','cell culture','null',25456122),(4577,'mouse','decrease','low-throughput','Indole, the most prevalent metabolite of tryptophan, is produced by a wide range of bacterial species, including those belonging to the genera Escherichia, Bacteroides, and ClostridiumProlonged exposure to indole inhibits ATP production and thus GLP-1 secretion.','cell culture','null',25456122),(4578,'mouse','decrease','low-throughput','Indole, the most prevalent metabolite of tryptophan, is produced by a wide range of bacterial species, including those belonging to the genera Escherichia, Bacteroides, and ClostridiumProlonged exposure to indole inhibits ATP production and thus GLP-1 secretion.','cell culture','null',25456122),(4588,'mouse','increase','low-throughput','NaB corrected the HFD-induced gut microbiota imbalance in mice, while it considerably elevated the abundances of the beneficial bacteria Christensenellaceae, Blautia and Lactobacillus . These bacteria can produce butyric acid in what seems like a virtuous circle. And butyrate restored HFD induced intestinal mucosa damage, increased the expression of ZO-1 in small intestine, further decreased the levels of gut endotoxin in serum and liver compared with HF group.','control experiment','ELISA�?real-time PCR',28104981),(4589,'mouse','increase','low-throughput','NaB corrected the HFD-induced gut microbiota imbalance in mice, while it considerably elevated the abundances of the beneficial bacteria Christensenellaceae, Blautia and Lactobacillus . These bacteria can produce butyric acid in what seems like a virtuous circle. And butyrate restored HFD induced intestinal mucosa damage, increased the expression of ZO-1 in small intestine, further decreased the levels of gut endotoxin in serum and liver compared with HF group.','control experiment','ELISA�?real-time PCR',28104981),(4590,'mouse','increase','low-throughput','NaB corrected the HFD-induced gut microbiota imbalance in mice, while it considerably elevated the abundances of the beneficial bacteria Christensenellaceae, Blautia and Lactobacillus . These bacteria can produce butyric acid in what seems like a virtuous circle. And butyrate restored HFD induced intestinal mucosa damage, increased the expression of ZO-1 in small intestine, further decreased the levels of gut endotoxin in serum and liver compared with HF group.','control experiment','ELISA�?real-time PCR',28104981),(4597,'mouse','increase','low-throughput','The helminth Nippostrongylus brasiliensis and a tritrichomonad protist both secreted succinate as a metabolite. In vivo sensing of the tritrichomonad required SUCNR1, whereas N. brasiliensis was SUCNR1 independent. These findings define a paradigm wherein tuft cells monitor microbial metabolites to initiate type 2 immunity and suggest the existence of other sensing pathways triggering the response to helminths.','Cell culture,in vitro bacterial culture','RT-qPCR,RNA-seq,',30021144),(4598,'mouse','increase','low-throughput','The helminth Nippostrongylus brasiliensis and a tritrichomonad protist both secreted succinate as a metabolite. In vivo sensing of the tritrichomonad required SUCNR1, whereas N. brasiliensis was SUCNR1 independent. These findings define a paradigm wherein tuft cells monitor microbial metabolites to initiate type 2 immunity and suggest the existence of other sensing pathways triggering the response to helminths.','Cell culture,in vitro bacterial culture','RT-qPCR,RNA-seq,',30021144),(4600,'mouse','increase','low-throughput','Our study shows that MAM from F. prausnitzii can restore the intestinal barrier structure and function in DM conditions via the regulation of the tight junction pathway and ZO-1 expression.Collectively, our data reveal that MAM protein upregulates ZO-1 expression in intestinal epithelial cells.','control experiment,cell culture','16S high-throughput sequencing,qRT-PCR,Western blot assay',31503404),(4612,'mouse','decrease','low-throughput','Faecalibacterium prausnitzii produces butyrate to decrease c-Myc-related metabolism and Th17 differentiation by inhibiting histone deacetylase 3.','control experiment,in vitro bacterial culture','qPCR,real-time PCR,Western blot assay,GC-MS analysis',30809639),(4614,'mouse','increase','low-throughput','F. prausnitzii, B. longum and F. prausnitzii supernatant clearly facilitated the induction of IL-10 and TGF-β1, while induced relatively mild production of IL-12p70 in both cellular and animal models. The F. prausnitzii, B. longum and supernatant differed in their capacity to induce T-bet, GATA-3 and ROR-γt mRNA expression in human PBMCs (both bacterial strains inhibited the expression of ROR-γt while supernatant inhibited the T-bet and GATA-3). However, all of them induced the Foxp3 and Treg production and ameliorated the TNBS-induced colitis. In addition, F. prausnitzii supernatant exhibited the supreme anti-inflammatory capacity.','control experiment,in vitro bacterial culture','ELISA,real-time qPCR',23643066),(4615,'mouse','increase','low-throughput','F. prausnitzii, B. longum and F. prausnitzii supernatant clearly facilitated the induction of IL-10 and TGF-β1, while induced relatively mild production of IL-12p70 in both cellular and animal models. The F. prausnitzii, B. longum and supernatant differed in their capacity to induce T-bet, GATA-3 and ROR-γt mRNA expression in human PBMCs (both bacterial strains inhibited the expression of ROR-γt while supernatant inhibited the T-bet and GATA-3). However, all of them induced the Foxp3 and Treg production and ameliorated the TNBS-induced colitis. In addition, F. prausnitzii supernatant exhibited the supreme anti-inflammatory capacity.','control experiment,in vitro bacterial culture','ELISA,real-time qPCR',23643066),(4619,'mouse','decrease','low-throughput','Colonic cDNA concentrations of IL-1b and TNF-a were significantly down regulated in DSS-treated E. durans-fed mice but not in control or DSS-treated L.delbrueckii- fed mice.','in vitro bacterial culture,control experiment','qPCR,real-time PCR',17349085),(4620,'mouse','decrease','low-throughput','Colonic cDNA concentrations of IL-1b and TNF-a were significantly down regulated in DSS-treated E. durans-fed mice but not in control or DSS-treated L.delbrueckii- fed mice.','in vitro bacterial culture,control experiment','qPCR,real-time PCR',17349085),(4623,'mouse','increase','low-throughput','Lactobacilli Induce IL-22 via Iald.','control experiment,in vitro bacterial culture','16S rRNA sequence,qPCR ,HPLC-HRMS Analysis',23973224),(4631,'mouse','decrease','low-throughput','The mRNA levels of  -SMA and TIMP-1 were revealed to be markedly elevated in the livers of the MCD diet-fed mice, and these high expressions were normalized by LcS intervention (Fig. 3B). These data suggest LcS intervention to have suppressed HSC activation and liver fibrosis.','control experiment,in vitro bacterial culture','real-time RT-qPCR',24113768),(4632,'mouse','decrease','low-throughput','The mRNA levels of  -SMA and TIMP-1 were revealed to be markedly elevated in the livers of the MCD diet-fed mice, and these high expressions were normalized by LcS intervention (Fig. 3B). These data suggest LcS intervention to have suppressed HSC activation and liver fibrosis.','control experiment,in vitro bacterial culture','real-time RT-qPCR',24113768),(4633,'mouse','decrease','low-throughput','To investigate the molecular mechanisms underlying the resistance to NASH development conferred by LcS intervention, the mRNA expression levels of lipogenic enzyme genes such as SREBP-1c and FAS, oxidation enzyme genes, CPT-1, and CD36 were examined by quantitative real-time PCR. While MCD diet feeding signifi- cantly elevated the expression of SREBP-1c but not that of FAS, LcS intervention suppressed these mRNA levels compared with the MCD diet alone (Fig. 4A).','control experiment,in vitro bacterial culture','real-time RT-qPCR',24113768),(4634,'mouse','decrease','low-throughput','To investigate the molecular mechanisms underlying the resistance to NASH development conferred by LcS intervention, the mRNA expression levels of lipogenic enzyme genes such as SREBP-1c and FAS, oxidation enzyme genes, CPT-1, and CD36 were examined by quantitative real-time PCR. While MCD diet feeding signifi- cantly elevated the expression of SREBP-1c but not that of FAS, LcS intervention suppressed these mRNA levels compared with the MCD diet alone (Fig. 4A).','control experiment,in vitro bacterial culture','real-time RT-qPCR',24113768),(4635,'mouse','increase','low-throughput','To this end, DCA feeding was performed in HFD-fed mice treated with vancomycin (Figure 2(a,b)). Intriguingly, DCA feeding alone in VCM-treated HFD-fed mice was observed to induce substantial proinflammatory macrophages (iNOS+) infiltration and increase the expression of multiple proinflammatory factors (IL-6, TNF-α, IL-1β, iNOS) in the colon tissue (Figure 2(c–f)), mimicking the inflammatory intestinal damage induced by HFD.','control experiment','16S rDNA sequence,LC-MS/MS analysis,RT-PCR,Western blot assay',33006494),(4636,'mouse','increase','low-throughput','To this end, DCA feeding was performed in HFD-fed mice treated with vancomycin (Figure 2(a,b)). Intriguingly, DCA feeding alone in VCM-treated HFD-fed mice was observed to induce substantial proinflammatory macrophages (iNOS+) infiltration and increase the expression of multiple proinflammatory factors (IL-6, TNF-α, IL-1β, iNOS) in the colon tissue (Figure 2(c–f)), mimicking the inflammatory intestinal damage induced by HFD.','control experiment','16S rDNA sequence,LC-MS/MS analysis,RT-PCR,Western blot assay',33006494),(4637,'mouse','increase','low-throughput','To this end, DCA feeding was performed in HFD-fed mice treated with vancomycin (Figure 2(a,b)). Intriguingly, DCA feeding alone in VCM-treated HFD-fed mice was observed to induce substantial proinflammatory macrophages (iNOS+) infiltration and increase the expression of multiple proinflammatory factors (IL-6, TNF-α, IL-1β, iNOS) in the colon tissue (Figure 2(c–f)), mimicking the inflammatory intestinal damage induced by HFD.','control experiment','16S rDNA sequence,LC-MS/MS analysis,RT-PCR,Western blot assay',33006494),(4638,'mouse','increase','low-throughput','To this end, DCA feeding was performed in HFD-fed mice treated with vancomycin (Figure 2(a,b)). Intriguingly, DCA feeding alone in VCM-treated HFD-fed mice was observed to induce substantial proinflammatory macrophages (iNOS+) infiltration and increase the expression of multiple proinflammatory factors (IL-6, TNF-α, IL-1β, iNOS) in the colon tissue (Figure 2(c–f)), mimicking the inflammatory intestinal damage induced by HFD.','control experiment','16S rDNA sequence,LC-MS/MS analysis,RT-PCR,Western blot assay',33006494),(4639,'mouse','increase','low-throughput','Indeed, DCA stimulation could induce potent increase of mRNA expression level of M2-mAchR in macrophages, and selective inhibition of M2-mAchR significantly reduced theproduction of TNF-α, NO and IL-6 triggered by DCA (Figure 4(d,e) and Supplementary Figure 4c).','control experiment','16S rDNA sequence,LC-MS/MS analysis,RT-PCR,Western blot assay',33006494),(4640,'mouse','increase','low-throughput','Toll-like receptor 2 (TLR2) is up-regulated upon DCA stimulation and involved in DCA-induced macrophage polarization.','control experiment','RT-PCR,Western blot assay,ChIP',33006494),(4652,'mouse','increase','low-throughput','When the complex carbohydrates of breast milk interact with B. fragilis, it produces short-chain fatty acids (SCFAs). We know from animal studies that SCFAs interacting with G-protein-coupled receptors can stimulate regulatory T cells via the GPR 43 receptor to activate the original cells and to produce IL-10, which inhibits inflammation [44].','null','null',32229728),(4653,'mouse','increase','low-throughput','To determine the functional impact of B dentium colonization on enterochromaffin-produced serotonin, we assessed 5-HT concentrations by immunostaining, qPCR,and LC-MS/MS. Increased numbers of 5-HT–positive cells were observed in both the ileum and colon of B dentium mono-associated mice relative to germ-free and heat-killed B dentium–treated mice (Figure 3A and B). Increases in 5-HT were observed in enteroendocrine cells directly above enteric neurons (Figure 3C). Tph-1 is the rate limiting enzyme for 5-HT production in enterochromaffin cells. The results of the immunostaining were corroborated by the significant increase in Tph-1 expression in both the ileum and colon of B dentium mono-associated mice relative to germ-free and heat-killed B dentium–treated mice(Figure 3D). Increased Tph-1 gene expression also was accompanied by significantly increased 5-HT concentrations in the intestinal lumen and feces of mice, as measured by LC-MS/MS (Figure 3E).','in vitro bacterial culture','immunostaining, qPCR，LC-MS/MS analysis',32795610),(4654,'mouse','increase','low-throughput','We found that expression of the colonic 5-HT receptors 2a and 4 (HTR2A and HTR4) were increased significantly in B dentium mono-associated mice (Figure 7A).','in vitro bacterial culture','qPCR，immunostaining',32795610),(4655,'mouse','increase','low-throughput','We found that expression of the colonic 5-HT receptors 2a and 4 (HTR2A and HTR4) were increased significantly in B dentium mono-associated mice (Figure 7A).','in vitro bacterial culture','qPCR，immunostaining',32795610),(4683,'mouse','increase','low-throughput','Analysis of the expression of mRNAs of three TJ proteins (ZO-1, claudin-1, and occludin) indicated greater expression in the Sham Group than in the three other groups (P < 0.05), and greater expression in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4A,B,D). In addition, expression of claudin-2, MMP9 was lower in the Sham Group than in the three other groups (P < 0.05) and lower in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4C,E).','in vitro bacterial culture,control experiment','RT-PCR,Western blot assay',33192557),(4684,'mouse','decrease','low-throughput','Analysis of the expression of mRNAs of three TJ proteins (ZO-1, claudin-1, and occludin) indicated greater expression in the Sham Group than in the three other groups (P < 0.05), and greater expression in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4A,B,D). In addition, expression of claudin-2, MMP9 was lower in the Sham Group than in the three other groups (P < 0.05) and lower in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4C,E).','in vitro bacterial culture,control experiment','RT-PCR,Western blot assay',33192557),(4685,'mouse','increase','low-throughput','Analysis of the expression of mRNAs of three TJ proteins (ZO-1, claudin-1, and occludin) indicated greater expression in the Sham Group than in the three other groups (P < 0.05), and greater expression in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4A,B,D). In addition, expression of claudin-2, MMP9 was lower in the Sham Group than in the three other groups (P < 0.05) and lower in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4C,E).','in vitro bacterial culture,control experiment','RT-PCR,Western blot assay',33192557),(4686,'mouse','increase','low-throughput','Analysis of the expression of mRNAs of three TJ proteins (ZO-1, claudin-1, and occludin) indicated greater expression in the Sham Group than in the three other groups (P < 0.05), and greater expression in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4A,B,D). In addition, expression of claudin-2, MMP9 was lower in the Sham Group than in the three other groups (P < 0.05) and lower in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4C,E).','in vitro bacterial culture,control experiment','RT-PCR,Western blot assay',33192557),(4687,'mouse','decrease','low-throughput','Analysis of the expression of mRNAs of three TJ proteins (ZO-1, claudin-1, and occludin) indicated greater expression in the Sham Group than in the three other groups (P < 0.05), and greater expression in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4A,B,D). In addition, expression of claudin-2, MMP9 was lower in the Sham Group than in the three other groups (P < 0.05) and lower in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4C,E).','in vitro bacterial culture,control experiment','RT-PCR,Western blot assay',33192557),(4688,'mouse','decrease','low-throughput','In addition, the DAO, LPS, FITC-dextran, and inflammatory cytokine levels of the two treatment groups were similar to each other (P > 0.05) and significantly lower than that of the SAP + IAH Group (P < 0.05; Figures 3B–E).','in vitro bacterial culture,control experiment','RT-PCR,Western blot assay',33192557),(4689,'mouse','decrease','low-throughput','In addition, the DAO, LPS, FITC-dextran, and inflammatory cytokine levels of the two treatment groups were similar to each other (P > 0.05) and significantly lower than that of the SAP + IAH Group (P < 0.05; Figures 3B–E).','in vitro bacterial culture,control experiment','RT-PCR,Western blot assay',33192557),(4690,'mouse','decrease','low-throughput','In addition, the DAO, LPS, FITC-dextran, and inflammatory cytokine levels of the two treatment groups were similar to each other (P > 0.05) and significantly lower than that of the SAP + IAH Group (P < 0.05; Figures 3B–E).','in vitro bacterial culture,control experiment','RT-PCR,Western blot assay',33192557),(4691,'mouse','decrease','low-throughput','Relative to the SAP + IAH group, the levels of claudin-2, MMP9, and TNF-α were significantly lower in the two treatment groups (P < 0.05; Figures 5A,B).','in vitro bacterial culture,control experiment','RT-PCR,Western blot assay',33192557),(4701,'mouse','increase','low-throughput','SLC5A8 and GPR43 expressions in butyrate‑treated CRC cells were quantified (Fig. 5B and C for SLC5A8 and GPR43, respectively). In comparison with untreated cells, the NaB‑treated SW480 cells only presented higher GPR43 (P<0.05), but the NaB‑treated SW620 cells expressed the most SLC5A8 (P<0.01) and GPR43 (P<0.05).','cell culture,control experiment','GC/MS analysis',33101496),(4702,'mouse','increase','low-throughput','SLC5A8 and GPR43 expressions in butyrate‑treated CRC cells were quantified (Fig. 5B and C for SLC5A8 and GPR43, respectively). In comparison with untreated cells, the NaB‑treated SW480 cells only presented higher GPR43 (P<0.05), but the NaB‑treated SW620 cells expressed the most SLC5A8 (P<0.01) and GPR43 (P<0.05).','cell culture,control experiment','GC/MS analysis',33101496),(4703,'mouse','increase','low-throughput','SLC5A8 and GPR43 expressions in butyrate‑treated CRC cells were quantified (Fig. 5B and C for SLC5A8 and GPR43, respectively). In comparison with untreated cells, the NaB‑treated SW480 cells only presented higher GPR43 (P<0.05), but the NaB‑treated SW620 cells expressed the most SLC5A8 (P<0.01) and GPR43 (P<0.05).','cell culture,control experiment','GC/MS analysis',33101496),(4704,'mouse','increase','low-throughput','Intestinal expression of farnesoid X receptor (FXR) mRNA was significantly higher, whereas hepatic expression of cholesterol-7a-hydroxylase 1 (CYP7A1) and small heterodimer partner (SHP) were significantly lower in ob/ob mice compared to those in control mice.','control experiment','RT-PCR,16S rRNA',27111464),(4705,'mouse','increase','low-throughput','FGF15 mRNA expression was also increased in ob/ob mice compared to that of the lean control mice.','control experiment','RT-PCR,16S rRNA',27111464),(4706,'mouse','increase','low-throughput','In the present study, hepatic mRNA expression of FGFR4 was significantly increased, while CYP7A1 was greatly decreased in ob/ob mice.','control experiment','RT-PCR,16S rRNA',27111464),(4707,'mouse','decrease','low-throughput','In the presentstudy, hepatic mRNA expression of FGFR4 was significantly increased, while CYP7A1 was greatly decreased in ob/ob mice.','control experiment','RT-PCR,16S rRNA',27111464),(4708,'mouse','decrease','low-throughput','Decreased intestinal taurine-conjugated BAs may also affect enterohepatic circulation of BAs and facilitate hepatic de novo lipogenesis and down-regulate fatty acid oxidation in ob/ob mice through intestinal FXR-FGF15 and hepatic FXR-SHP pathways.','control experiment','RT-PCR,16S rRNA',27111464),(4709,'mouse','decrease','low-throughput','Also, ob/ob mice showed significantly downregulated CPT-1 mRNA expression compared to their lean control mice.','control experiment','RT-PCR,16S rRNA',27111464),(4710,'mouse','increase','low-throughput','Moreover, hepatic FAS gene expression was up-regulated while both SREBP-1c and ACC were slightly increased without significant differences between two groups.','control experiment','RT-PCR,16S rRNA',27111464),(4711,'mouse','increase','low-throughput','Intestinal expression of farnesoid X receptor (FXR) mRNA was significantly higher, whereas hepatic expression of cholesterol-7a-hydroxylase 1 (CYP7A1) and small heterodimer partner (SHP) were significantly lower in ob/ob mice compared to those in control mice.','control experiment','RT-PCR,16S rRNA',27111464),(4712,'mouse','increase','low-throughput','FGF15 mRNA expression was also increased in ob/ob mice compared to that of the lean control mice.','control experiment','RT-PCR,16S rRNA',27111464),(4713,'mouse','increase','low-throughput','In the present study, hepatic mRNA expression of FGFR4 was significantly increased, while CYP7A1 was greatly decreased in ob/ob mice.','control experiment','RT-PCR,16S rRNA',27111464),(4714,'mouse','decrease','low-throughput','In the presentstudy, hepatic mRNA expression of FGFR4 was significantly increased, while CYP7A1 was greatly decreased in ob/ob mice.','control experiment','RT-PCR,16S rRNA',27111464),(4715,'mouse','decrease','low-throughput','Decreased intestinal taurine-conjugated BAs may also affect enterohepatic circulation of BAs and facilitate hepatic de novo lipogenesis and down-regulate fatty acid oxidation in ob/ob mice through intestinal FXR-FGF15 and hepatic FXR-SHP pathways.','control experiment','RT-PCR,16S rRNA',27111464),(4716,'mouse','decrease','low-throughput','Also, ob/ob mice showed significantly downregulated CPT-1 mRNA expression compared to their lean control mice.','control experiment','RT-PCR,16S rRNA',27111464),(4717,'mouse','increase','low-throughput','Moreover, hepatic FAS gene expression was up-regulated while both SREBP-1c and ACC were slightly increased without significant differences between two groups.','control experiment','RT-PCR,16S rRNA',27111464),(4718,'mouse','increase','low-throughput','Intestinal expression of farnesoid X receptor (FXR) mRNA was significantly higher, whereas hepatic expression of cholesterol-7a-hydroxylase 1 (CYP7A1) and small heterodimer partner (SHP) were significantly lower in ob/ob mice compared to those in control mice.','control experiment','RT-PCR,16S rRNA',27111464),(4719,'mouse','increase','low-throughput','FGF15 mRNA expression was also increased in ob/ob mice compared to that of the lean control mice.','control experiment','RT-PCR,16S rRNA',27111464),(4720,'mouse','increase','low-throughput','In the present study, hepatic mRNA expression of FGFR4 was significantly increased, while CYP7A1 was greatly decreased in ob/ob mice.','control experiment','RT-PCR,16S rRNA',27111464),(4721,'mouse','decrease','low-throughput','In the presentstudy, hepatic mRNA expression of FGFR4 was significantly increased, while CYP7A1 was greatly decreased in ob/ob mice.','control experiment','RT-PCR,16S rRNA',27111464),(4722,'mouse','decrease','low-throughput','Decreased intestinal taurine-conjugated BAs may also affect enterohepatic circulation of BAs and facilitate hepatic de novo lipogenesis and down-regulate fatty acid oxidation in ob/ob mice through intestinal FXR-FGF15 and hepatic FXR-SHP pathways.','control experiment','RT-PCR,16S rRNA',27111464),(4723,'mouse','decrease','low-throughput','Also, ob/ob mice showed significantly downregulated CPT-1 mRNA expression compared to their lean control mice.','control experiment','RT-PCR,16S rRNA',27111464),(4724,'mouse','increase','low-throughput','Moreover, hepatic FAS gene expression was up-regulated while both SREBP-1c and ACC were slightly increased without significant differences between two groups.','control experiment','RT-PCR,16S rRNA',27111464),(4739,'mouse','increase','low-throughput','B. intestinalis produced the highest levels of colonic lamina propria Tregs [36.6% ±4.39 (mean ± SEM) FoxP3+ among CD4+ T cells], and reached high densities of colonization. Collinsella aerofaciens, which colonized to low density, did not induce an increase in Tregs in this initial screen.','in vitro bacterial culture,control experiment,in vivo infection experiment','GC/MS analysis,LC/MS analysis',24452263),(4740,'mouse','increase','low-throughput','B. intestinalis produced the highest levels of colonic lamina propria Tregs [36.6% ±4.39 (mean ± SEM) FoxP3+ among CD4+ T cells], and reached high densities of colonization. Collinsella aerofaciens, which colonized to low density, did not induce an increase in Tregs in this initial screen.','in vitro bacterial culture,control experiment,in vivo infection experiment','GC/MS analysis,LC/MS analysis',24452263),(4741,'mouse','increase','low-throughput','Each Bacteroides strain resulted in significant increases in the proportions of FoxP3+ Treg cells among CD4+ T cells in the colon lamina propria (Fig. 4A and fig. S5A; see table S9 for P-values).','in vitro bacterial culture,control experiment,in vivo infection experiment','GC/MS analysis,LC/MS analysis',24452263),(4742,'mouse','increase','low-throughput','Each Bacteroides strain resulted in significant increases in the proportions of FoxP3+ Treg cells among CD4+ T cells in the colon lamina propria (Fig. 4A and fig. S5A; see table S9 for P-values).','in vitro bacterial culture,control experiment,in vivo infection experiment','GC/MS analysis,LC/MS analysis',24452263),(4743,'mouse','increase','low-throughput','Each Bacteroides strain resulted in significant increases in the proportions of FoxP3+ Treg cells among CD4+ T cells in the colon lamina propria (Fig. 4A and fig. S5A; see table S9 for P-values).','in vitro bacterial culture,control experiment,in vivo infection experiment','GC/MS analysis,LC/MS analysis',24452263),(4744,'mouse','increase','low-throughput','Each Bacteroides strain resulted in significant increases in the proportions of FoxP3+ Treg cells among CD4+ T cells in the colon lamina propria (Fig. 4A and fig. S5A; see table S9 for P-values).','in vitro bacterial culture,control experiment,in vivo infection experiment','GC/MS analysis,LC/MS analysis',24452263),(4745,'mouse','increase','low-throughput','Each Bacteroides strain resulted in significant increases in the proportions of FoxP3+ Treg cells among CD4+ T cells in the colon lamina propria (Fig. 4A and fig. S5A; see table S9 for P-values).','in vitro bacterial culture,control experiment,in vivo infection experiment','GC/MS analysis,LC/MS analysis',24452263),(4746,'mouse','increase','low-throughput','Each Bacteroides strain resulted in significant increases in the proportions of FoxP3+ Treg cells among CD4+ T cells in the colon lamina propria (Fig. 4A and fig. S5A; see table S9 for P-values).','in vitro bacterial culture,control experiment,in vivo infection experiment','GC/MS analysis,LC/MS analysis',24452263),(4760,'mouse','increase','low-throughput','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.','null','16S rRNA sequence',27448578),(4761,'mouse','increase','low-throughput','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.','null','16S rRNA sequence',27448578),(4762,'mouse','increase','low-throughput','SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.','null','16S rRNA sequence',27448578),(4763,'mouse','decrease','low-throughput','Furthermore, DCs from mice treated with propionate, characterized by reduced expression of CD40, PD-L2 and CD86, exhibited the impaired ability to initiate Th2 effector function and further to promote Th2-mediated allergic airway inflammation.','null','16S rRNA sequence',27448578),(4764,'mouse','decrease','low-throughput','Furthermore, DCs from mice treated with propionate, characterized by reduced expression of CD40, PD-L2 and CD86, exhibited the impaired ability to initiate Th2 effector function and further to promote Th2-mediated allergic airway inflammation.','null','16S rRNA sequence',27448578),(4765,'mouse','decrease','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.','null','16S rRNA sequence',27448578),(4766,'mouse','decrease','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.','null','16S rRNA sequence',27448578),(4767,'mouse','decrease','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.','null','16S rRNA sequence',27448578),(4768,'mouse','increase','low-throughput','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.','null','16S rRNA sequence',27448578),(4769,'mouse','increase','low-throughput','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.','null','16S rRNA sequence',27448578),(4770,'mouse','increase','low-throughput','SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.','null','16S rRNA sequence',27448578),(4771,'mouse','increase','low-throughput','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.','null','16S rRNA sequence',27448578),(4772,'mouse','increase','low-throughput','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.','null','16S rRNA sequence',27448578),(4773,'mouse','increase','low-throughput','SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.','null','16S rRNA sequence',27448578),(4774,'mouse','increase','low-throughput','GPR41 and GPR43 can be efficiently activated by acetate, propionate, butyrate and other SCFAs [39], while GPR109a can be activated mainly by butyrate and niacin.','null','16S rRNA sequence',27448578),(4775,'mouse','increase','low-throughput','Butyrate upregulated the histone H3 acetylation of Foxp3 and promoted the differentiation of Treg, which acts as a key anti-inflammatory effector.','null','16S rRNA sequence',27448578),(4776,'mouse','decrease','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.','null','16S rRNA sequence',27448578),(4777,'mouse','decrease','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.','null','16S rRNA sequence',27448578),(4778,'mouse','decrease','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.','null','16S rRNA sequence',27448578),(4779,'mouse','increase','low-throughput','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.','null','16S rRNA sequence',27448578),(4780,'mouse','increase','low-throughput','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.','null','16S rRNA sequence',27448578),(4781,'mouse','increase','low-throughput','SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.','null','16S rRNA sequence',27448578),(4782,'mouse','increase','low-throughput','GPR41 and GPR43 can be efficiently activated by acetate, propionate, butyrate and other SCFAs [39], while GPR109a can be activated mainly by butyrate and niacin.','null','16S rRNA sequence',27448578),(4783,'mouse','increase','low-throughput','Butyrate upregulated the histone H3 acetylation of Foxp3 and promoted the differentiation of Treg, which acts as a key anti-inflammatory effector.','null','16S rRNA sequence',27448578),(4784,'mouse','decrease','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.','null','16S rRNA sequence',27448578),(4785,'mouse','decrease','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.','null','16S rRNA sequence',27448578),(4786,'mouse','decrease','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.','null','16S rRNA sequence',27448578),(4787,'mouse','increase','low-throughput','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.','null','16S rRNA sequence',27448578),(4788,'mouse','increase','low-throughput','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.','null','16S rRNA sequence',27448578),(4789,'mouse','increase','low-throughput','SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.','null','16S rRNA sequence',27448578),(4790,'mouse','increase','low-throughput','GPR41 and GPR43 can be efficiently activated by acetate, propionate, butyrate and other SCFAs [39], while GPR109a can be activated mainly by butyrate and niacin.','null','16S rRNA sequence',27448578),(4791,'mouse','increase','low-throughput','Butyrate upregulated the histone H3 acetylation of Foxp3 and promoted the differentiation of Treg, which acts as a key anti-inflammatory effector.','null','16S rRNA sequence',27448578),(4792,'mouse','decrease','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.','null','16S rRNA sequence',27448578),(4793,'mouse','decrease','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.','null','16S rRNA sequence',27448578),(4794,'mouse','decrease','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.','null','16S rRNA sequence',27448578),(4795,'mouse','increase','low-throughput','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.','null','16S rRNA sequence',27448578),(4796,'mouse','increase','low-throughput','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.','null','16S rRNA sequence',27448578),(4797,'mouse','increase','low-throughput','SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.','null','16S rRNA sequence',27448578),(4798,'mouse','increase','low-throughput','GPR41 and GPR43 can be efficiently activated by acetate, propionate, butyrate and other SCFAs [39], while GPR109a can be activated mainly by butyrate and niacin.','null','16S rRNA sequence',27448578),(4799,'mouse','increase','low-throughput','Butyrate upregulated the histone H3 acetylation of Foxp3 and promoted the differentiation of Treg, which acts as a key anti-inflammatory effector.','null','16S rRNA sequence',27448578),(4800,'mouse','decrease','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.','null','16S rRNA sequence',27448578),(4801,'mouse','decrease','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.','null','16S rRNA sequence',27448578),(4802,'mouse','decrease','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.','null','16S rRNA sequence',27448578),(4803,'mouse','increase','low-throughput','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.','null','16S rRNA sequence',27448578),(4804,'mouse','increase','low-throughput','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.','null','16S rRNA sequence',27448578),(4805,'mouse','increase','low-throughput','SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.','null','16S rRNA sequence',27448578),(4806,'mouse','decrease','low-throughput','Furthermore, DCs from mice treated with propionate, characterized by reduced expression of CD40, PD-L2 and CD86, exhibited the impaired ability to initiate Th2 effector function and further to promote Th2-mediated allergic airway inflammation.','null','16S rRNA sequence',27448578),(4807,'mouse','decrease','low-throughput','Furthermore, DCs from mice treated with propionate, characterized by reduced expression of CD40, PD-L2 and CD86, exhibited the impaired ability to initiate Th2 effector function and further to promote Th2-mediated allergic airway inflammation.','null','16S rRNA sequence',27448578),(4808,'mouse','decrease','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.','null','16S rRNA sequence',27448578),(4809,'mouse','decrease','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.','null','16S rRNA sequence',27448578),(4810,'mouse','decrease','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.','null','16S rRNA sequence',27448578),(4811,'mouse','increase','low-throughput','Colonization of germ-free mice with wild-type Escherichia coli, but not E. coli mutants unable to generate indole, induced colonic epithelial IL-10R1.','cell culture,in vitro bacterial culture,control experiment','RT-PCR,Western blot assay,HPLC analysis',29454749),(4812,'mouse','increase','low-throughput','Colonization of germ-free mice with wild-type Escherichia coli, but not E. coli mutants unable to generate indole, induced colonic epithelial IL-10R1.','cell culture,in vitro bacterial culture,control experiment','RT-PCR,Western blot assay,HPLC analysis',29454749),(4813,'mouse','increase','low-throughput','IAld has recently been identified as an aryl hydrocarbon receptor (AHR) ligand.','cell culture,in vitro bacterial culture,control experiment','RT-PCR,Western blot assay,HPLC analysis',29454749),(4814,'mouse','increase','low-throughput','Colonization of germ-free mice with wild-type Escherichia coli, but not E. coli mutants unable to generate indole, induced colonic epithelial IL-10R1.','cell culture,in vitro bacterial culture,control experiment','RT-PCR,Western blot assay,HPLC analysis',29454749),(4815,'mouse','increase','low-throughput','Colonization of germ-free mice with wild-type Escherichia coli, but not E. coli mutants unable to generate indole, induced colonic epithelial IL-10R1.','cell culture,in vitro bacterial culture,control experiment','RT-PCR,Western blot assay,HPLC analysis',29454749),(4816,'mouse','increase','low-throughput','IAld has recently been identified as an aryl hydrocarbon receptor (AHR) ligand.','cell culture,in vitro bacterial culture,control experiment','RT-PCR,Western blot assay,HPLC analysis',29454749),(4822,'mouse','decrease','low-throughput','In summary, our results clearly demonstrate that different E. faecalis strains differentially induce colitis in monoassociated IL-10-/- mice and that severe E. faecalis-induced colitis resulted in decreased Alix/AIP1 protein expression as well as ERK1/2 phosphorylation. Functional studies revealed no effect of Alix/AIP1 on bacteria- and EGF-mediated effects on the epithelium. The 13 differentially regulated proteins might be ofinterest aspotentialmarkers for severeintestinalinflammation.','in vitro bacterial culture,cell culture','qPCR,ELISA,Western blot assay',19166300),(4823,'mouse','decrease','low-throughput','In summary, our results clearly demonstrate that different E. faecalis strains differentially induce colitis in monoassociated IL-10-/- mice and that severe E. faecalis-induced colitis resulted in decreased Alix/AIP1 protein expression as well as ERK1/2 phosphorylation. Functional studies revealed no effect of Alix/AIP1 on bacteria- and EGF-mediated effects on the epithelium. The 13 differentially regulated proteins might be ofinterest aspotentialmarkers for severeintestinalinflammation.','in vitro bacterial culture,cell culture','qPCR,ELISA,Western blot assay',19166300),(4824,'mouse','decrease','low-throughput','In summary, our results clearly demonstrate that different E. faecalis strains differentially induce colitis in monoassociated IL-10-/- mice and that severe E. faecalis-induced colitis resulted in decreased Alix/AIP1 protein expression as well as ERK1/2 phosphorylation. Functional studies revealed no effect of Alix/AIP1 on bacteria- and EGF-mediated effects on the epithelium. The 13 differentially regulated proteins might be ofinterest aspotentialmarkers for severeintestinalinflammation.','in vitro bacterial culture,cell culture','qPCR,ELISA,Western blot assay',19166300),(5026,'mouse','increase','low-throughput','Human and murine pregnancy were associated with reduced intestinal FXR signaling, with lower FGF19/15 and resultant increased hepatic bile acid synthesis. Terminal ileal ASBT protein was reduced in murine pregnancy. Cecal bile acid conjugation was reduced in pregnancy because of elevated bile salt hydrolase‐producing Bacteroidetes. CA supplementation induced intestinal FXR signaling, which was not abrogated by pregnancy, with strikingly similar changes to the microbiota and metabonome as identified in pregnancy. Conclusion: The altered intestinal microbiota of pregnancy enhance bile acid deconjugation, reducing ileal bile acid uptake and lowering FXR induction in enterocytes. This exacerbates the effects mediated by reduced bile acid uptake transporters in pregnancy. Thus, in pregnant women and mice, there is reduced FGF19/15‐mediated hepatic repression of hepatic bile acid synthesis, resulting in hypercholanemia.','in vivo experiment','RT-PCR,16S rRNA sequence',30983011),(5031,'mouse','decrease','low-throughput','Inhibitory activity of angiotensin I-converting enzyme (ACE) was examined with (�?-epigallocatechin gallate (EGCG) metabolites produced by intestinal bacteria, together with tea catechins. All of the metabolites showed ACE inhibitory activities.','in vitro bacterial culture,in vivo infection experiment','HPLC analysis',26323573),(5032,'mouse','decrease','low-throughput','Inhibitory activity of angiotensin I-converting enzyme (ACE) was examined with (�?-epigallocatechin gallate (EGCG) metabolites produced by intestinal bacteria, together with tea catechins. All of the metabolites showed ACE inhibitory activities.','in vitro bacterial culture,in vivo infection experiment','HPLC analysis',26323573),(5033,'mouse','decrease','low-throughput','Inhibitory activity of angiotensin I-converting enzyme (ACE) was examined with (�?-epigallocatechin gallate (EGCG) metabolites produced by intestinal bacteria, together with tea catechins. All of the metabolites showed ACE inhibitory activities.','in vitro bacterial culture,in vivo infection experiment','HPLC analysis',26323573),(5034,'mouse','decrease','low-throughput','Inhibitory activity of angiotensin I-converting enzyme (ACE) was examined with (�?-epigallocatechin gallate (EGCG) metabolites produced by intestinal bacteria, together with tea catechins. All of the metabolites showed ACE inhibitory activities.','in vitro bacterial culture,in vivo infection experiment','HPLC analysis',26323573),(5035,'mouse','decrease','low-throughput','Inhibitory activity of angiotensin I-converting enzyme (ACE) was examined with (�?-epigallocatechin gallate (EGCG) metabolites produced by intestinal bacteria, together with tea catechins. All of the metabolites showed ACE inhibitory activities.','in vitro bacterial culture,in vivo infection experiment','HPLC analysis',26323573),(5036,'mouse','decrease','low-throughput','Inhibitory activity of angiotensin I-converting enzyme (ACE) was examined with (�?-epigallocatechin gallate (EGCG) metabolites produced by intestinal bacteria, together with tea catechins. All of the metabolites showed ACE inhibitory activities.','in vitro bacterial culture,in vivo infection experiment','HPLC analysis',26323573),(5040,'mouse','decrease','low-throughput','Inflammatory responses were induced in GPR41(-/-), GPR43(-/-), and C57BL6 (control) mice by administration of ethanol; 2, 4, 6 trinitrobenzene sulfonic-acid (TNBS); or infection with Citrobacter rodentium. We examined the effects of C rodentium infection on control mice fed SCFAs and/or given injections of antibodies that delay the immune response. We also studied the kinetics of cytokine and chemokine production, leukocyte recruitment, intestinal permeability, and T-cell responses. Primary colon epithelial cells were isolated from GPR41(-/-), GPR43(-/-), and control mice; signaling pathways regulated by SCFAs were identified using immunohistochemical, enzyme-linked immunosorbent assay, and flow cytometry analyses.','in vivo experiment,cell culture','qPCR',23665276),(5041,'mouse','decrease','low-throughput','Inflammatory responses were induced in GPR41(-/-), GPR43(-/-), and C57BL6 (control) mice by administration of ethanol; 2, 4, 6 trinitrobenzene sulfonic-acid (TNBS); or infection with Citrobacter rodentium. We examined the effects of C rodentium infection on control mice fed SCFAs and/or given injections of antibodies that delay the immune response. We also studied the kinetics of cytokine and chemokine production, leukocyte recruitment, intestinal permeability, and T-cell responses. Primary colon epithelial cells were isolated from GPR41(-/-), GPR43(-/-), and control mice; signaling pathways regulated by SCFAs were identified using immunohistochemical, enzyme-linked immunosorbent assay, and flow cytometry analyses.','in vivo experiment,cell culture','qPCR',23665276),(5042,'mouse','decrease','low-throughput','Inflammatory responses were induced in GPR41(-/-), GPR43(-/-), and C57BL6 (control) mice by administration of ethanol; 2, 4, 6 trinitrobenzene sulfonic-acid (TNBS); or infection with Citrobacter rodentium. We examined the effects of C rodentium infection on control mice fed SCFAs and/or given injections of antibodies that delay the immune response. We also studied the kinetics of cytokine and chemokine production, leukocyte recruitment, intestinal permeability, and T-cell responses. Primary colon epithelial cells were isolated from GPR41(-/-), GPR43(-/-), and control mice; signaling pathways regulated by SCFAs were identified using immunohistochemical, enzyme-linked immunosorbent assay, and flow cytometry analyses.','in vivo experiment,cell culture','qPCR',23665276),(5043,'mouse','decrease','low-throughput','Inflammatory responses were induced in GPR41(-/-), GPR43(-/-), and C57BL6 (control) mice by administration of ethanol; 2, 4, 6 trinitrobenzene sulfonic-acid (TNBS); or infection with Citrobacter rodentium. We examined the effects of C rodentium infection on control mice fed SCFAs and/or given injections of antibodies that delay the immune response. We also studied the kinetics of cytokine and chemokine production, leukocyte recruitment, intestinal permeability, and T-cell responses. Primary colon epithelial cells were isolated from GPR41(-/-), GPR43(-/-), and control mice; signaling pathways regulated by SCFAs were identified using immunohistochemical, enzyme-linked immunosorbent assay, and flow cytometry analyses.','in vivo experiment,cell culture','qPCR',23665276),(5044,'mouse','decrease','low-throughput','Inflammatory responses were induced in GPR41(-/-), GPR43(-/-), and C57BL6 (control) mice by administration of ethanol; 2, 4, 6 trinitrobenzene sulfonic-acid (TNBS); or infection with Citrobacter rodentium. We examined the effects of C rodentium infection on control mice fed SCFAs and/or given injections of antibodies that delay the immune response. We also studied the kinetics of cytokine and chemokine production, leukocyte recruitment, intestinal permeability, and T-cell responses. Primary colon epithelial cells were isolated from GPR41(-/-), GPR43(-/-), and control mice; signaling pathways regulated by SCFAs were identified using immunohistochemical, enzyme-linked immunosorbent assay, and flow cytometry analyses.','in vivo experiment,cell culture','qPCR',23665276),(5045,'mouse','decrease','low-throughput','Inflammatory responses were induced in GPR41(-/-), GPR43(-/-), and C57BL6 (control) mice by administration of ethanol; 2, 4, 6 trinitrobenzene sulfonic-acid (TNBS); or infection with Citrobacter rodentium. We examined the effects of C rodentium infection on control mice fed SCFAs and/or given injections of antibodies that delay the immune response. We also studied the kinetics of cytokine and chemokine production, leukocyte recruitment, intestinal permeability, and T-cell responses. Primary colon epithelial cells were isolated from GPR41(-/-), GPR43(-/-), and control mice; signaling pathways regulated by SCFAs were identified using immunohistochemical, enzyme-linked immunosorbent assay, and flow cytometry analyses.','in vivo experiment,cell culture','qPCR',23665276),(5046,'mouse','decrease','low-throughput','Inflammatory responses were induced in GPR41(-/-), GPR43(-/-), and C57BL6 (control) mice by administration of ethanol; 2, 4, 6 trinitrobenzene sulfonic-acid (TNBS); or infection with Citrobacter rodentium. We examined the effects of C rodentium infection on control mice fed SCFAs and/or given injections of antibodies that delay the immune response. We also studied the kinetics of cytokine and chemokine production, leukocyte recruitment, intestinal permeability, and T-cell responses. Primary colon epithelial cells were isolated from GPR41(-/-), GPR43(-/-), and control mice; signaling pathways regulated by SCFAs were identified using immunohistochemical, enzyme-linked immunosorbent assay, and flow cytometry analyses.','in vivo experiment,cell culture','qPCR',23665276),(5049,'mouse','increase','low-throughput','B. infantis and MO decreased hepatic CYP7A1 and induced Sult2a1, Sult2a2, and Sult2a3 suggesting reduced BA synthesis and increased detoxification. Furthermore, B. infantis and MO increased ileal BA membrane receptor TGR5-regulated signaling. Together, via BA-regulated signaling, synbiotics B. infantis and MO have their unique and combined effects in reversing NASH.','in vitro bacterial culture','qPCR,LC/MS analysis',15640190),(5050,'mouse','increase','low-throughput','B. infantis and MO decreased hepatic CYP7A1 and induced Sult2a1, Sult2a2, and Sult2a3 suggesting reduced BA synthesis and increased detoxification. Furthermore, B. infantis and MO increased ileal BA membrane receptor TGR5-regulated signaling. Together, via BA-regulated signaling, synbiotics B. infantis and MO have their unique and combined effects in reversing NASH.','in vitro bacterial culture','qPCR,LC/MS analysis',31295342),(5051,'mouse','increase','low-throughput','B. infantis and MO decreased hepatic CYP7A1 and induced Sult2a1, Sult2a2, and Sult2a3 suggesting reduced BA synthesis and increased detoxification. Furthermore, B. infantis and MO increased ileal BA membrane receptor TGR5-regulated signaling. Together, via BA-regulated signaling, synbiotics B. infantis and MO have their unique and combined effects in reversing NASH.','in vitro bacterial culture','qPCR,LC/MS analysis',11040423),(5052,'mouse','decrease','low-throughput','A 15kDa protein with anti-inflammatory properties is produced by F. prausnitzii, a commensal bacterium involved in CD pathogenesis. This protein is able to inhibit the NF-κB pathway in intestinal epithelial cells and to prevent colitis in an animal model.','in vitro bacterial culture,in vitro bacterial culture,cell culture','fluorescence microscopy',26045134),(5060,'mouse','decrease','low-throughput','L. rhamnosus strain CNCM I-3690 reduces NF-kB activation by approximately 40% confirming the previous results, whereas the L. paracasei CNCM I-3689 reduces this activation only by 20%','null','RT-PCR',25517879),(5061,'mouse','decrease','low-throughput','L. rhamnosus strain CNCM I-3690 reduces NF-kB activation by approximately 40% confirming the previous results, whereas the L. paracasei CNCM I-3689 reduces this activation only by 20%.','null','RT-PCR',25517879),(5062,'mouse','decrease','low-throughput','IL-4 and IFN-g levels were significantly reduced in mice treated with either L. rhamnosus CNCM I-3690 or F. prausnitzii but not with the L. paracasei CNCM I-3689 strain (P < 0.05). A significant reduction in IL-6 levels was observed after treatment with all bacteri (P < 0.05).','null','RT-PCR',25517879),(5063,'mouse','decrease','low-throughput','IL-4 and IFN-g levels were significantly reduced in mice treated with either L. rhamnosus CNCM I-3690 or F. prausnitzii but not with the L. paracasei CNCM I-3689 strain (P < 0.05). A significant reduction in IL-6 levels was observed after treatment with all bacteri (P < 0.05).','null','RT-PCR',25517879),(5064,'mouse','decrease','low-throughput','IL-4 and IFN-g levels were significantly reduced in mice treated with either L. rhamnosus CNCM I-3690 or F. prausnitzii but not with the L. paracasei CNCM I-3689 strain (P < 0.05). A significant reduction in IL-6 levels was observed after treatment with all bacteri (P < 0.05).','null','RT-PCR',25517879),(5065,'mouse','decrease','low-throughput','IL-4 and IFN-g levels were significantly reduced in mice treated with either L. rhamnosus CNCM I-3690 or F. prausnitzii but not with the L. paracasei CNCM I-3689 strain (P < 0.05). A significant reduction in IL-6 levels was observed after treatment with all bacteri (P < 0.05).','null','RT-PCR',25517879),(5066,'mouse','decrease','low-throughput','IL-4 and IFN-g levels were significantly reduced in mice treated with either L. rhamnosus CNCM I-3690 or F. prausnitzii but not with the L. paracasei CNCM I-3689 strain (P < 0.05). A significant reduction in IL-6 levels was observed after treatment with all bacteri (P < 0.05).','null','RT-PCR',25517879),(5067,'mouse','decrease','low-throughput','IL-4 and IFN-g levels were significantly reduced in mice treated with either L. rhamnosus CNCM I-3690 or F. prausnitzii but not with the L. paracasei CNCM I-3689 strain (P < 0.05). A significant reduction in IL-6 levels was observed after treatment with all bacteri (P < 0.05).','null','RT-PCR',25517879),(5068,'mouse','decrease','low-throughput','The analysis  of cytokines�?and chemokines�?expression showed a decrease  in mRNA amounts of the proinflammatory molecules il6, il1b, il12p70, and tnf-a after 7 days, but no differences in expression after 21 days of administration.','in vivo bacterial administration','RT-PCR',28239378),(5069,'mouse','decrease','low-throughput','The analysis  of cytokines�?and chemokines�?expression showed a decrease  in mRNA amounts of the proinflammatory molecules il6, il1b, il12p70, and tnf-a after 7 days, but no differences in expression after 21 days of administration.','in vivo bacterial administration','RT-PCR',28239378),(5070,'mouse','decrease','low-throughput','The analysis  of cytokines�?and chemokines�?expression showed a decrease  in mRNA amounts of the proinflammatory molecules il6, il1b, il12p70, and tnf-a after 7 days, but no differences in expression after 21 days of administration.','in vivo bacterial administration','RT-PCR',28239378),(5071,'mouse','decrease','low-throughput','The analysis  of cytokines�?and chemokines�?expression showed a decrease  in mRNA amounts of the proinflammatory molecules il6, il1b, il12p70, and tnf-a after 7 days, but no differences in expression after 21 days of administration.','in vivo bacterial administration','RT-PCR',28239378),(5072,'mouse','decrease','low-throughput','EP1 consumption decreased il6, il1b, and cxcl-1 expression after 7 days, and this downregulation was persistent for the first two genes after 21 days of probiotic treatment.','in vivo bacterial administration','RT-PCR',28239378),(5073,'mouse','decrease','low-throughput','Treatment with EP1 during 21 days decreased the expression of the proinflammatory molecules il1b, il6, ifng, and cxcl-1 only in PP while il 17 was reduced in both MLN and PP.','in vivo bacterial administration','RT-PCR',28239378),(5074,'mouse','decrease','low-throughput','Treatment with EP1 during 21 days decreased the expression of the proinflammatory molecules il1b, il6, ifng, and cxcl-1 only in PP while il 17 was reduced in both MLN and PP.','in vivo bacterial administration','RT-PCR',28239378),(5075,'mouse','increase','low-throughput','PPAR-α and PPAR-γ mRNAs in the epididymal fat tissue were significantly downregulated in the HFD group compared with the control group and significantly upregulated after CB intervention.','in vivo infection experiment','RT-PCR',28109017),(5076,'mouse','increase','low-throughput','PPAR-α and PPAR-γ mRNAs in the epididymal fat tissue were significantly downregulated in the HFD group compared with the control group and significantly upregulated after CB intervention.','in vivo infection experiment','RT-PCR',28109017),(5077,'mouse','decrease','low-throughput','The mRNA levels of TNF-α, MCP-1, IL-1β, IL-2, IL-6 and IL-10 were upregulated in the HFD group compared with the control group, all of which were reduced by CB  intervention.','in vivo infection experiment','RT-PCR',28109017),(5078,'mouse','decrease','low-throughput','The mRNA levels of TNF-α, MCP-1, IL-1β, IL-2, IL-6 and IL-10 were upregulated in the HFD group compared with the control group, all of which were reduced by CB  intervention.','in vivo infection experiment','RT-PCR',28109017),(5079,'mouse','decrease','low-throughput','The mRNA levels of TNF-α, MCP-1, IL-1β, IL-2, IL-6 and IL-10 were upregulated in the HFD group compared with the control group, all of which were reduced by CB  intervention.','in vivo infection experiment','RT-PCR',28109017),(5080,'mouse','decrease','low-throughput','The mRNA levels of TNF-α, MCP-1, IL-1β, IL-2, IL-6 and IL-10 were upregulated in the HFD group compared with the control group, all of which were reduced by CB  intervention.','in vivo infection experiment','RT-PCR',28109017),(5081,'mouse','decrease','low-throughput','The mRNA levels of TNF-α, MCP-1, IL-1β, IL-2, IL-6 and IL-10 were upregulated in the HFD group compared with the control group, all of which were reduced by CB  intervention.','in vivo infection experiment','RT-PCR',28109017),(5082,'mouse','decrease','low-throughput','The mRNA levels of TNF-α, MCP-1, IL-1β, IL-2, IL-6 and IL-10 were upregulated in the HFD group compared with the control group, all of which were reduced by CB  intervention.','in vivo infection experiment','RT-PCR',28109017),(5083,'mouse','increase','low-throughput','CB intervention effectively reversed these imbalanced immune factors, which increased the levels of Foxp3, IL-4 and IL-22 and decreased IFN-γ and IL-17 in the liver and ileum.','in vivo infection experiment','RT-PCR',28109017),(5084,'mouse','increase','low-throughput','CB intervention effectively reversed these imbalanced immune factors, which increased the levels of Foxp3, IL-4 and IL-22 and decreased IFN-γ and IL-17 in the liver and ileum.','in vivo infection experiment','RT-PCR',28109017),(5085,'mouse','increase','low-throughput','CB intervention effectively reversed these imbalanced immune factors, which increased the levels of Foxp3, IL-4 and IL-22 and decreased IFN-γ and IL-17 in the liver and ileum.','in vivo infection experiment','RT-PCR',28109017),(5086,'mouse','decrease','low-throughput','CB intervention effectively reversed these imbalanced immune factors, which increased the levels of Foxp3, IL-4 and IL-22 and decreased IFN-γ and IL-17 in the liver and ileum.','in vivo infection experiment','RT-PCR',28109017),(5087,'mouse','decrease','low-throughput','CB intervention effectively reversed these imbalanced immune factors, which increased the levels of Foxp3, IL-4 and IL-22 and decreased IFN-γ and IL-17 in the liver and ileum.','in vivo infection experiment','RT-PCR',28109017),(5090,'mouse','increase','low-throughput','Therefore, in this study, we investigated the effects of KetoA [10-oxo-12(Z)-octadecenoic acid]—a linoleic acid metabolite produced by gut lactic acid bacteria—on whole-body energy metabolism and found that dietary intake of KetoA could enhance energy expenditure in mice, thereby protecting mice from diet-induced obesity.','cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment','RT-PCR,Western blot assay',28754711),(5091,'mouse','increase','low-throughput','Ucp1 expression levels were also observed to have increased after administration of KetoA.','cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment','RT-PCR,Western blot assay',28754711),(5092,'mouse','increase','low-throughput','We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT\nmice.','cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment','RT-PCR,Western blot assay',28754711),(5093,'mouse','increase','low-throughput','We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT\nmice.','cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment','RT-PCR,Western blot assay',28754711),(5094,'mouse','increase','low-throughput','We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT\nmice.','cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment','RT-PCR,Western blot assay',28754711),(5095,'mouse','increase','low-throughput','We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT\nmice.','cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment','RT-PCR,Western blot assay',28754711),(5096,'mouse','increase','low-throughput','We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT\nmice.','cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment','RT-PCR,Western blot assay',28754711),(5097,'mouse','increase','low-throughput','Therefore, in this study, we investigated the effects of KetoA [10-oxo-12(Z)-octadecenoic acid]—a linoleic acid metabolite produced by gut lactic acid bacteria—on whole-body energy metabolism and found that dietary intake of KetoA could enhance energy expenditure in mice, thereby protecting mice from diet-induced obesity.','cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment','RT-PCR,Western blot assay',28754711),(5098,'mouse','increase','low-throughput','Ucp1 expression levels were also observed to have increased after administration of KetoA.','cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment','RT-PCR,Western blot assay',28754711),(5099,'mouse','increase','low-throughput','We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT mice.','cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment','RT-PCR,Western blot assay',28754711),(5100,'mouse','increase','low-throughput','We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT mice.','cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment','RT-PCR,Western blot assay',28754711),(5101,'mouse','increase','low-throughput','We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT mice.','cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment','RT-PCR,Western blot assay',28754711),(5102,'mouse','increase','low-throughput','We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT mice.','cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment','RT-PCR,Western blot assay',28754711),(5103,'mouse','increase','low-throughput','We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT mice.','cell culture,in vitro bacterial culture,in vivo infection experiment,control experiment','RT-PCR,Western blot assay',28754711),(5104,'mouse','increase','high-throughput','It should be noted that the observed Akkermansia muciniphila-induced decrease of Fiaf is not significant. These results are partially supported by the effects of both butyrate and propionate stimulation, leading to significant increase in Fiaf expression, while the expression of Gpr43 and Pparis significantly decreased .','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5105,'mouse','decrease','high-throughput','Akkermansia muciniphila leads to significantly decreased expression of Gpr43 and Ppar and increased expression of Hdac3 and Hdac5.','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5106,'mouse','increase','high-throughput','It should be noted that the observed Akkermansia muciniphila-induced decrease of Fiaf is not significant. These results are partially supported by the effects of both butyrate and propionate stimulation, leading to significant increase in Fiaf expression, while the expression of Gpr43 and Pparis significantly decreased .','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5107,'mouse','increase','high-throughput','It should be noted that the observed Akkermansia muciniphila-induced decrease of Fiaf is not significant. These results are partially supported by the effects of both butyrate and propionate stimulation, leading to significant increase in Fiaf expression, while the expression of Gpr43 and Pparis significantly decreased .','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5108,'mouse','decrease','high-throughput','Akkermansia muciniphila leads to significantly decreased expression of Gpr43 and Ppar and increased expression of Hdac3 and Hdac5.','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5109,'mouse','increase','high-throughput','Acetate had no significant effect on the expression of Fiaf, Gpr43, Hdac5, or Ppar, while the expression of Hdac3 was significantly increased by acetate.','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5110,'mouse','increase','high-throughput','It should be noted that the observed Akkermansia muciniphila-induced decrease of Fiaf is not significant. These results are partially supported by the effects of both butyrate and propionate stimulation, leading to significant increase in Fiaf expression, while the expression of Gpr43 and Pparis significantly decreased .','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5111,'mouse','decrease','high-throughput','Akkermansia muciniphila leads to significantly decreased expression of Gpr43 and Ppar and increased expression of Hdac3 and Hdac5.','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5112,'mouse','increase','high-throughput','It should be noted that the observed Akkermansia muciniphila-induced decrease of Fiaf is not significant. These results are partially supported by the effects of both butyrate and propionate stimulation, leading to significant increase in Fiaf expression, while the expression of Gpr43 and Pparis significantly decreased .','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5113,'mouse','increase','high-throughput','It should be noted that the observed Akkermansia muciniphila-induced decrease of Fiaf is not significant. These results are partially supported by the effects of both butyrate and propionate stimulation, leading to significant increase in Fiaf expression, while the expression of Gpr43 and Pparis significantly decreased .','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5114,'mouse','decrease','high-throughput','Akkermansia muciniphila leads to significantly decreased expression of Gpr43 and Ppar and increased expression of Hdac3 and Hdac5.','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5115,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5116,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5117,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5118,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5119,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5120,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5121,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5122,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5123,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5124,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5125,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5126,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5127,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5128,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5129,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5130,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5131,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5132,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5133,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5134,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5135,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5136,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5137,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5138,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5139,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5140,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5141,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5142,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5143,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5144,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5145,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5146,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5147,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5148,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5149,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5150,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5151,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5152,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5153,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5154,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5155,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5156,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5157,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5158,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5159,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5160,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5161,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5162,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5163,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5164,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5165,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5166,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5167,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5168,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5169,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5170,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5171,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5172,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5173,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5174,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5175,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5176,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5177,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5178,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5179,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5180,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5181,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5182,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5183,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5184,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5185,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5186,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5187,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5188,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5189,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5190,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5191,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5192,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5193,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5194,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5195,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5196,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5197,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5198,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5199,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5200,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5201,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5202,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5203,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5204,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5205,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5206,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5207,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5208,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5209,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5210,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5211,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5212,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5213,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5214,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5215,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5216,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5217,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5218,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5219,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5220,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5221,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5222,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5223,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5224,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5225,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5226,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5227,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5228,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5229,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5230,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5231,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5232,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5233,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5234,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5235,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5236,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5237,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5238,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5239,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5240,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5241,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5242,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5243,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5244,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5245,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5246,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5247,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5248,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5249,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5250,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5251,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5252,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5253,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5254,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5255,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5256,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5257,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5258,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5259,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5260,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5261,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5262,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5263,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5264,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5265,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5266,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5267,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5268,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5269,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5270,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5271,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5272,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5273,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5274,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5275,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5276,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5277,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5278,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5279,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5280,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5281,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5282,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5283,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5284,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5285,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5286,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5287,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5288,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5289,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5290,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5291,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5292,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5293,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5294,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5295,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5296,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5297,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5298,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5299,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5300,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5301,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5302,'mouse','decrease','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5303,'mouse','increase','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).','cell culture,in vitro bacterial culture,control experiment','null',25118238),(5334,'mouse','increase','low-throughput','Forty-eight hours after TNBS instillation,colonic proinflammatory TNF and IL-12 cytokines and anti-inflammatory IL-10 were quantified by ELISA (SI Materials and Methods and Fig. 5). Compared with the no-colitis control group,TNF-α and IL-12 levels were increased, whereas IL-10 levels were barely modified. In the two TNBS groups treated with either F.prausnitzii or its supernatant, the secretion of TNF and IL-12 was significantly lower than in the colitis control group. Interestingly, IL-10 secretion was induced in the colon of mice treated with F.prausnitzii or its supernatant.','cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment','qPCR',18936492),(5335,'mouse','decrease','low-throughput','Forty-eight hours after TNBS instillation,colonic proinflammatory TNF and IL-12 cytokines and anti-inflammatory IL-10 were quantified by ELISA (SI Materials and Methods and Fig. 5). Compared with the no-colitis control group,TNF-α and IL-12 levels were increased, whereas IL-10 levels were barely modified. In the two TNBS groups treated with either F.prausnitzii or its supernatant, the secretion of TNF and IL-12 was significantly lower than in the colitis control group. Interestingly, IL-10 secretion was induced in the colon of mice treated with F.prausnitzii or its supernatant.','cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment','qPCR',18936492),(5336,'mouse','decrease','low-throughput','Forty-eight hours after TNBS instillation,colonic proinflammatory TNF and IL-12 cytokines and anti-inflammatory IL-10 were quantified by ELISA (SI Materials and Methods and Fig. 5). Compared with the no-colitis control group,TNF-α and IL-12 levels were increased, whereas IL-10 levels were barely modified. In the two TNBS groups treated with either F.prausnitzii or its supernatant, the secretion of TNF and IL-12 was significantly lower than in the colitis control group. Interestingly, IL-10 secretion was induced in the colon of mice treated with F.prausnitzii or its supernatant.','cell culture、in vitro bacterial culture、in vivo infection experiment、control experiment','qPCR',18936492),(5337,'mouse','decrease','low-throughput','Altered substrate oxidation and fecal fat excretion in LHFD-fed GF mice is linked to decreased hepatic Cyp7a1 and Nr1h4 expression.','control experiment','RT-PCR�?6S rRNA sequence',27900259),(5338,'mouse','decrease','low-throughput','Altered substrate oxidation and fecal fat excretion in LHFD-fed GF mice is linked to decreased hepatic Cyp7a1 and Nr1h4 expression.','control experiment','RT-PCR�?6S rRNA sequence',27900259),(5339,'mouse','decrease','low-throughput','10-Hydroxy-cis-12-octadecenoic acid (HYA), a gut microbial metabolite of linoleic acid, suppressed TNF and dextran sulfate sodium-induced changes in the expression of TJ-related molecules, occludin, zonula occludens-1, and myosin light chain kinase. HYA also suppressed the expression of TNF receptor 2 (TNFR2) mRNA and protein expression in Caco-2 cells and colonic tissue. In addition, HYA suppressed the protein expression of TNFR2 in murine intestinal epithelial cells. Furthermore, HYA significantly up-regulated G protein-coupled receptor (GPR) 40 expression in Caco-2 cells.','cell culture、control experiment','Western blot assay、RT-PCR',25505251),(5340,'mouse','decrease','low-throughput','10-Hydroxy-cis-12-octadecenoic acid (HYA), a gut microbial metabolite of linoleic acid, suppressed TNF and dextran sulfate sodium-induced changes in the expression of TJ-related molecules, occludin, zonula occludens-1, and myosin light chain kinase. HYA also suppressed the expression of TNF receptor 2 (TNFR2) mRNA and protein expression in Caco-2 cells and colonic tissue. In addition, HYA suppressed the protein expression of TNFR2 in murine intestinal epithelial cells. Furthermore, HYA significantly up-regulated G protein-coupled receptor (GPR) 40 expression in Caco-2 cells.','cell culture、control experiment','Western blot assay、RT-PCR',25505251),(5341,'mouse','increase','low-throughput','10-Hydroxy-cis-12-octadecenoic acid (HYA), a gut microbial metabolite of linoleic acid, suppressed TNF and dextran sulfate sodium-induced changes in the expression of TJ-related molecules, occludin, zonula occludens-1, and myosin light chain kinase. HYA also suppressed the expression of TNF receptor 2 (TNFR2) mRNA and protein expression in Caco-2 cells and colonic tissue. In addition, HYA suppressed the protein expression of TNFR2 in murine intestinal epithelial cells. Furthermore, HYA significantly up-regulated G protein-coupled receptor (GPR) 40 expression in Caco-2 cells.','cell culture、control experiment','Western blot assay、RT-PCR',25505251),(5342,'mouse','decrease','low-throughput','IFN-TNF-induced an increase in FD-4 permeability;however, the pretreatment of HYA and linoleic acid signififcantly suppressed this increase (Fig. 2C). In contrast, the pretreatment of HYB did not alter IFN-TNF-induced changes in TER and FD-4 permeability. As shown in Fig. 2D,IFN-TNF-induced a marked increase in the secretion of IL-8, which is a major pro-inflammatory cytokine. However,the pretreatment of HYA significantly suppressed IL-8 secretion.','cell culture、control experiment','Western blot assay、RT-PCR',25505251),(5343,'mouse','increase','low-throughput','Pretreatment of HYA, but not HYB, significantly restored IFN-TNF-induced changes in the expression of occludin, ZO-1, claudin-1, and MLCK.','cell culture、control experiment','Western blot assay、RT-PCR',25505251),(5344,'mouse','increase','low-throughput','Pretreatment of HYA, but not HYB, significantly restored IFN-TNF-induced changes in the expression of occludin, ZO-1, claudin-1, and MLCK.','cell culture、control experiment','Western blot assay、RT-PCR',25505251),(5345,'mouse','increase','low-throughput','Pretreatment of HYA, but not HYB, significantly restored IFN-TNF-induced changes in the expression of occludin, ZO-1, claudin-1, and MLCK.','cell culture、control experiment','Western blot assay、RT-PCR',25505251),(5346,'mouse','decrease','low-throughput','Pretreatment of HYA, but not HYB, significantly restored IFN-TNF-induced changes in the expression of occludin, ZO-1, claudin-1, and MLCK.','cell culture、control experiment','Western blot assay、RT-PCR',25505251),(5347,'mouse','decrease','low-throughput','The pretreatment of HYA suppressed IFN-induced up-regulation of TNFR2 (but not TNFR1), and it subsequently decreased the ratio of phospho-INF-B p65 expression induced by cytokines (Fig. 3, B and C). In contrast, pretreatment of HYB did not show the same inhibitory effects as HYA.','cell culture、control experiment','Western blot assay、RT-PCR',25505251),(5348,'mouse','increase','low-throughput','It was noteworthy that the pretreatment of HYA augmented the mRNA expression of GPR40 and GPR43 but not GPR120, PPAR, and GPR41 in Caco-2 cells (Fig. 4A). However, the pretreatment of HYB did not up-regulate GPR40 and GPR43 expression (Fig. 4A).','cell culture、control experiment','Western blot assay、RT-PCR',25505251),(5363,'mouse','increase','low-throughput','Indole 3-propionic acid (IPA), in the context of indole, is a ligand for PXR in vivo, and IPA downregulated enterocyte TNF-a while it upregulated junctional protein-coding mRNAs. PXR_deficient mice showed a distinctly ‘‘leaky’’gut physiology coupled with upregulation of the Toll-like receptor (TLR) signaling pathway.','in vitro bacterial culture、in vivo infection experiment、control experiment','qPCR',25065623),(5364,'mouse','decrease','low-throughput','Indole 3-propionic acid (IPA), in the context of indole, is a ligand for PXR in vivo, and IPA downregulated enterocyte TNF-a while it upregulated junctional protein-coding mRNAs. PXR_deficient mice showed a distinctly ‘‘leaky’’gut physiology coupled with upregulation of the Toll-like receptor (TLR) signaling pathway.','in vitro bacterial culture、in vivo infection experiment、control experiment','qPCR',25065623),(5365,'mouse','increase','low-throughput','Indole 3-propionic acid (IPA), in the context of indole, is a ligand for PXR in vivo, and IPA downregulated enterocyte TNF-a while it upregulated junctional protein-coding mRNAs. PXR_deficient mice showed a distinctly ‘‘leaky’’gut physiology coupled with upregulation of the Toll-like receptor (TLR) signaling pathway.','in vitro bacterial culture、in vivo infection experiment、control experiment','qPCR',25065623),(5366,'mouse','increase','low-throughput','For critical validation of the ex vivo experiments demonstrating IPA effects on junctional regulators, we coadministered C. sporogenes to germ-free mice in the presence or absence of\nL-tryptophan (Figure 1D). We verified that C. sporogenes inoculation led to production of IPA in vivo (thus, it was assumed that indoles were present) (Figure 1E). Germ-free mice exposed to\nC. sporogenes had a significant reduction in FITC-dextran recovery from the serum and this was further reduced in the presence of L-tryptophan dosing (Figure 1F). The mice intestinal mucosa\nexposed to C. sporogenes demonstrated significant induction of PXR target genes (Mdr1, Cyp3a11, Ugt1a1), which was further augmented in the presence of L-tryptophan (Figure 1G).','in vitro bacterial culture、in vivo infection experiment、control experiment','qPCR',25065623),(5367,'mouse','increase','low-throughput','For critical validation of the ex vivo experiments demonstrating IPA effects on junctional regulators, we coadministered C. sporogenes to germ-free mice in the presence or absence of\nL-tryptophan (Figure 1D). We verified that C. sporogenes inoculation led to production of IPA in vivo (thus, it was assumed that indoles were present) (Figure 1E). Germ-free mice exposed to\nC. sporogenes had a significant reduction in FITC-dextran recovery from the serum and this was further reduced in the presence of L-tryptophan dosing (Figure 1F). The mice intestinal mucosa\nexposed to C. sporogenes demonstrated significant induction of PXR target genes (Mdr1, Cyp3a11, Ugt1a1), which was further augmented in the presence of L-tryptophan (Figure 1G).','in vitro bacterial culture、in vivo infection experiment、control experiment','qPCR',25065623),(5368,'mouse','increase','low-throughput','For critical validation of the ex vivo experiments demonstrating IPA effects on junctional regulators, we coadministered C. sporogenes to germ-free mice in the presence or absence of\nL-tryptophan (Figure 1D). We verified that C. sporogenes inoculation led to production of IPA in vivo (thus, it was assumed that indoles were present) (Figure 1E). Germ-free mice exposed to\nC. sporogenes had a significant reduction in FITC-dextran recovery from the serum and this was further reduced in the presence of L-tryptophan dosing (Figure 1F). The mice intestinal mucosa\nexposed to C. sporogenes demonstrated significant induction of PXR target genes (Mdr1, Cyp3a11, Ugt1a1), which was further augmented in the presence of L-tryptophan (Figure 1G).','in vitro bacterial culture、in vivo infection experiment、control experiment','qPCR',25065623),(5369,'mouse','decrease','low-throughput','We found TMAVA to bind and inhibit BBOX, reducing synthesis of carnitine.We found the intestinal bacteria Enterococcus faecalis and Pseudomonas aeruginosa to metabolize trimethyllysine to TMAVA.','in vivo experiment','qPCR',32105727),(5370,'mouse','decrease','low-throughput','We found TMAVA to bind and inhibit BBOX, reducing synthesis of carnitine.We found the intestinal bacteria Enterococcus faecalis and Pseudomonas aeruginosa to metabolize trimethyllysine to TMAVA.','in vivo experiment','qPCR',32105727),(5372,'mouse','increase','low-throughput','In addition, Parabacteroides_sp_HGS0025 positively correlated with butyrate, IgM, IL-10, and TNF-α products in intestine and mice blood, respectively.','in vivo experiment','ELISA',32014483),(5373,'mouse','increase','low-throughput','In addition, Parabacteroides_sp_HGS0025 positively correlated with butyrate, IgM, IL-10, and TNF-α products in intestine and mice blood, respectively.','in vivo experiment','ELISA',32014483),(5374,'mouse','increase','low-throughput','In addition, Parabacteroides_sp_HGS0025 positively correlated with butyrate, IgM, IL-10, and TNF-α products in intestine and mice blood, respectively.','in vivo experiment','ELISA',32014483),(5392,'mouse','increase','low-throughput','Treatments of BMDCs with the fatty acids significantly increased mRNA expression of Arg1 in a dose-dependent manner in comparison with that of IL-4 treatment alone.','in vivo experiment','RT-qPCR',28904023),(5393,'mouse','increase','low-throughput','Treatments of BMDCs with the fatty acids significantly increased mRNA expression of Arg1 in a dose-dependent manner in comparison with that of IL-4 treatment alone.','in vivo experiment','RT-qPCR',28904023),(5394,'mouse','increase','low-throughput','The ratio of p-Akt/Akt in the C. butyricum-treated mice was significantly increased compared with the D-I/R group, suggesting activating Akt by C. butyricum treatment.','in vivo experiment','Western blot assay',27037183),(5395,'mouse','decrease','low-throughput','C. butyricum treatment resulted in decreases of caspase-3 protein.','in vivo experiment','Western blot assay',27037183),(5396,'mouse','increase','low-throughput','BDNF in the CUMS group was markedly decreased compared to that in the Con group, and Cb treatment significantly reversed this decrease.','in vivo experiment','ELISA',30040410),(5397,'mouse','increase','low-throughput','Cb treatment resulted in a significant increase in GLP-1.','in vivo experiment','ELISA',30040410),(5398,'mouse','decrease','low-throughput','Microbial suppression of Gpr41 expression.','in vivo experiment','RT-qPCR',18931303),(5399,'mouse','increase','low-throughput','AM attenuates ALS in mice by increasing NAM levels','in vivo experiment','RNA-seq',31330533),(5400,'mouse','increase','low-throughput','AM attenuates ALS in mice by increasing NAM levels','in vivo experiment','RNA-seq',31330533),(5401,'mouse','increase','low-throughput','Additionally, prophylactic treatment with C. butyricum increased the activity of both superoxide dismutase (SOD) and catalase (CAT).','in vivo experiment','Western blot assay',28933492),(5402,'mouse','increase','low-throughput','Additionally, prophylactic treatment with C. butyricum increased the activity of both superoxide dismutase (SOD) and catalase (CAT).','in vivo experiment','Western blot assay',28933492),(5403,'mouse','increase','low-throughput','Furthermore, C. butyricum up-regulated the nuclear factor (erythroid-derived 2)-like 2 (NRF2) content.','in vivo experiment','Western blot assay',28933492),(5404,'mouse','decrease','low-throughput','The levels of IL-1β and TNF-α in the brains of APP/PS1 mice were remarkably increased, while the levels in the CB-treated mice were significantly reduced.','in vivo experiment,cell culture','Western blot assay,immunofluorescence',31835282),(5405,'mouse','decrease','low-throughput','The levels of IL-1β and TNF-α in the brains of APP/PS1 mice were remarkably increased, while the levels in the CB-treated mice were significantly reduced.','in vivo experiment,cell culture','Western blot assay,immunofluorescence',31835282),(5406,'mouse','decrease','low-throughput','Butyrate treatment significantly decreased expression of the CD11b.','in vivo experiment,cell culture','Western blot assay,immunofluorescence',31835282),(5407,'mouse','increase','low-throughput','Here, we show that the biological effects of tryptamine are mediated through the 5-HT4 receptor (5-HT4R), a G-protein-coupled receptor (GPCR) uniquely expressed in the colonic epithelium.','in vivo experiment','LC/MS analysis,RT-PCR',29902441),(5408,'mouse','increase','low-throughput','Here, we show that the biological effects of tryptamine are mediated through the 5-HT4 receptor (5-HT4R), a G-protein-coupled receptor (GPCR) uniquely expressed in the colonic epithelium.','in vivo experiment','LC/MS analysis,RT-PCR',29902441),(5413,'mouse','decrease','low-throughput','A significant decrease in IL17, IFNγ and IL5 was found in supernatant of lymphocytes from LL-pILMAM treated mice compared to LL-pILEMPTY and control mice.','in vivo experiment','ELISA',28203226),(5414,'mouse','decrease','low-throughput','A significant decrease in IL17, IFNγ and IL5 was found in supernatant of lymphocytes from LL-pILMAM treated mice compared to LL-pILEMPTY and control mice.','in vivo experiment','ELISA',28203226),(5415,'mouse','decrease','low-throughput','A significant decrease in IL17, IFNγ and IL5 was found in supernatant of lymphocytes from LL-pILMAM treated mice compared to LL-pILEMPTY and control mice.','in vivo experiment','ELISA',28203226),(5419,'mouse','increase','low-throughput','The expression of colonic IDO was increased by Bifidobacterium.','in vivo experiment','Western blot assay',29226383),(5420,'mouse','decrease','low-throughput','Furthermore, CB0313.1-treated mice exhibited signifcantly lower serum fructosamine and glucose-6-phosphatase (G6Pase) levels.','in vivo experiment','ELISA,Western blot analysis,RT-PCR,GC-MS',28765642),(5421,'mouse','increase','low-throughput','In CB0313.1-treated mice, we observed an increase of mitochondrial function marker PPARγ (1.50-fold, p = 0.0339), carnitine palmitoyltransferase-1α (CPT1α) (1.85-fold, p = 0.0057), nuclear factor-like 2 (NRF2) (1.39-fold, p = 0.0012) in epididymal adipose tissue.','in vivo experiment','ELISA,Western blot analysis,RT-PCR,GC-MS',28765642),(5422,'mouse','increase','low-throughput','In CB0313.1-treated mice, we observed an increase of mitochondrial function marker PPARγ (1.50-fold, p = 0.0339), carnitine palmitoyltransferase-1α (CPT1α) (1.85-fold, p = 0.0057), nuclear factor-like 2 (NRF2) (1.39-fold, p = 0.0012) in epididymal adipose tissue.','in vivo experiment','ELISA,Western blot analysis,RT-PCR,GC-MS',28765642),(5423,'mouse','increase','low-throughput','In CB0313.1-treated mice, we observed an increase of mitochondrial function marker PPARγ (1.50-fold, p = 0.0339), carnitine palmitoyltransferase-1α (CPT1α) (1.85-fold, p = 0.0057), nuclear factor-like 2 (NRF2) (1.39-fold, p = 0.0012) in epididymal adipose tissue.','in vivo experiment','ELISA,Western blot analysis,RT-PCR,GC-MS',28765642),(5424,'mouse','increase','low-throughput','Notably, glucose transporter 4 (Glut4) mRNA levels in CB0313.1 and NaB-fed mice were increased','in vivo experiment','ELISA,Western blot analysis,RT-PCR,GC-MS',28765642),(5425,'mouse','increase','low-throughput','Ucp-1 is a key gene capable of retarding oxidative phosphorylation and thereby hindering the production of ATP17. Its mRNA levels in CB0313.1 and NaB-fed mice increased.','in vivo experiment','ELISA,Western blot analysis,RT-PCR,GC-MS',28765642),(5427,'mouse','increase','low-throughput','Similarly, F. prausnitziiexposed DC systematically secreted IL-27 and IL-6.','cell culture','ELISA',30787928),(5428,'mouse','increase','low-throughput','Similarly, F. prausnitziiexposed DC systematically secreted IL-27 and IL-6.','cell culture','ELISA',30787928),(5429,'mouse','increase','low-throughput','F. prausnitzii-exposed DC upregulated PDL-1,CD39,and IDO1.','cell culture','ELISA',30787928),(5430,'mouse','increase','low-throughput','F. prausnitzii-exposed DC upregulated PDL-1,CD39,and IDO1.','cell culture','ELISA',30787928),(5431,'mouse','increase','low-throughput','F. prausnitzii-exposed DC upregulated PDL-1,CD39,and IDO1.','cell culture','ELISA',30787928),(5432,'mouse','increase','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5433,'mouse','increase','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5434,'mouse','increase','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5435,'mouse','increase','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5436,'mouse','increase','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5437,'mouse','decrease','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5438,'mouse','decrease','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5439,'mouse','decrease','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5440,'mouse','decrease','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5441,'mouse','decrease','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5442,'mouse','increase','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5443,'mouse','increase','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5444,'mouse','increase','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5445,'mouse','increase','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5446,'mouse','increase','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5447,'mouse','decrease','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5448,'mouse','decrease','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5449,'mouse','decrease','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5450,'mouse','decrease','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5451,'mouse','decrease','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5452,'mouse','increase','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5453,'mouse','increase','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5454,'mouse','increase','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5455,'mouse','increase','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5456,'mouse','increase','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5457,'mouse','decrease','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5458,'mouse','decrease','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5459,'mouse','decrease','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5460,'mouse','decrease','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5461,'mouse','decrease','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5462,'mouse','increase','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5463,'mouse','increase','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5464,'mouse','increase','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5465,'mouse','increase','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5466,'mouse','increase','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5467,'mouse','decrease','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5468,'mouse','decrease','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5469,'mouse','decrease','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5470,'mouse','decrease','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5471,'mouse','decrease','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5472,'mouse','increase','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5473,'mouse','increase','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5474,'mouse','increase','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5475,'mouse','increase','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5476,'mouse','increase','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5477,'mouse','decrease','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5478,'mouse','decrease','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5479,'mouse','decrease','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5480,'mouse','decrease','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5481,'mouse','decrease','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5482,'mouse','increase','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5483,'mouse','increase','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5484,'mouse','increase','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5485,'mouse','increase','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5486,'mouse','increase','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5487,'mouse','decrease','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5488,'mouse','decrease','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5489,'mouse','decrease','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5490,'mouse','decrease','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5491,'mouse','decrease','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.','in vivo experiment','HPLC-ESI-MS analysis,RT-qPCR,Western blot assay',31863867),(5494,'mouse','decrease','low-throughput','In contrast, in livers of fructose + Lcs fed mice expression of TLR-4 and MyD88 was almost at the level of controls.','in vivo experiment','RNA-seq,Western blot assay',22749137),(5495,'mouse','decrease','low-throughput','In contrast, in livers of fructose + Lcs fed mice expression of TLR-4 and MyD88 was almost at the level of controls.','in vivo experiment','RNA-seq,Western blot assay',22749137),(5496,'mouse','increase','low-throughput','Lactobacillus casei Shirota treatment up-regulates peroxisome proliferator-activated receptor γ activity in the liver.','in vivo experiment','RNA-seq,Western blot assay',22749137),(5497,'mouse','decrease','low-throughput','Membrane expression of MCT1 is decreased by EPEC infection.','in vivo experiment','Western blot assay',16150873),(5499,'mouse','increase','low-throughput','Colonization with A. onderdonkii, B. fragilis, or B. thetaiotamicron led to variable IL-6 production by IELs that was substantially higher than that produced by IELs isolated from E. coli or PBS-gavaged germ-free mice.','in vivo experiment','qPCR',28812548),(5500,'mouse','increase','low-throughput','Colonization with A. onderdonkii, B. fragilis, or B. thetaiotamicron led to variable IL-6 production by IELs that was substantially higher than that produced by IELs isolated from E. coli or PBS-gavaged germ-free mice.','in vivo experiment','qPCR',28812548),(5501,'mouse','increase','low-throughput','Colonization with A. onderdonkii, B. fragilis, or B. thetaiotamicron led to variable IL-6 production by IELs that was substantially higher than that produced by IELs isolated from E. coli or PBS-gavaged germ-free mice.','in vivo experiment','qPCR',28812548),(5502,'mouse','increase','low-throughput','Expression levels of claudin-1, occludin and zonula occludens-1 were also elevated by the administration of C. butyricum. In addition, C. butyricum increased the expression of the anti-inflammatory cytokine, IL-10, and decreased levels of pro-inflammatory cytokines, including IL-1β, TNF, and IL-13, as well as NF-κB.','in vivo experiment','Western blot assay',30141701),(5503,'mouse','increase','low-throughput','Expression levels of claudin-1, occludin and zonula occludens-1 were also elevated by the administration of C. butyricum. In addition, C. butyricum increased the expression of the anti-inflammatory cytokine, IL-10, and decreased levels of pro-inflammatory cytokines, including IL-1β, TNF, and IL-13, as well as NF-κB.','in vivo experiment','Western blot assay',30141701),(5504,'mouse','increase','low-throughput','Expression levels of claudin-1, occludin and zonula occludens-1 were also elevated by the administration of C. butyricum. In addition, C. butyricum increased the expression of the anti-inflammatory cytokine, IL-10, and decreased levels of pro-inflammatory cytokines, including IL-1β, TNF, and IL-13, as well as NF-κB.','in vivo experiment','Western blot assay',30141701),(5505,'mouse','increase','low-throughput','Expression levels of claudin-1, occludin and zonula occludens-1 were also elevated by the administration of C. butyricum. In addition, C. butyricum increased the expression of the anti-inflammatory cytokine, IL-10, and decreased levels of pro-inflammatory cytokines, including IL-1β, TNF, and IL-13, as well as NF-κB.','in vivo experiment','Western blot assay',30141701),(5506,'mouse','decrease','low-throughput','Expression levels of claudin-1, occludin and zonula occludens-1 were also elevated by the administration of C. butyricum. In addition, C. butyricum increased the expression of the anti-inflammatory cytokine, IL-10, and decreased levels of pro-inflammatory cytokines, including IL-1β, TNF, and IL-13, as well as NF-κB.','in vivo experiment','Western blot assay',30141701),(5507,'mouse','decrease','low-throughput','Expression levels of claudin-1, occludin and zonula occludens-1 were also elevated by the administration of C. butyricum. In addition, C. butyricum increased the expression of the anti-inflammatory cytokine, IL-10, and decreased levels of pro-inflammatory cytokines, including IL-1β, TNF, and IL-13, as well as NF-κB.','in vivo experiment','Western blot assay',30141701),(5508,'mouse','decrease','low-throughput','Expression levels of claudin-1, occludin and zonula occludens-1 were also elevated by the administration of C. butyricum. In addition, C. butyricum increased the expression of the anti-inflammatory cytokine, IL-10, and decreased levels of pro-inflammatory cytokines, including IL-1β, TNF, and IL-13, as well as NF-κB.','in vivo experiment','Western blot assay',30141701),(5509,'mouse','decrease','low-throughput','Expression levels of claudin-1, occludin and zonula occludens-1 were also elevated by the administration of C. butyricum. In addition, C. butyricum increased the expression of the anti-inflammatory cytokine, IL-10, and decreased levels of pro-inflammatory cytokines, including IL-1β, TNF, and IL-13, as well as NF-κB.','in vivo experiment','Western blot assay',30141701),(5510,'mouse','decrease','low-throughput','Table 3 revealed that γ-GT, AST and ALT were signifcantly elevated in the M group, while these alterations were obviously reversed by treatment with L. rhamnosus LS-8 and L. crustorum MN047.','in vivo experiment','qPCR',31659451),(5511,'mouse','decrease','low-throughput','Table 3 revealed that γ-GT, AST and ALT were signifcantly elevated in the M group, while these alterations were obviously reversed by treatment with L. rhamnosus LS-8 and L. crustorum MN047.','in vivo experiment','qPCR',31659451),(5512,'mouse','decrease','low-throughput','Table 3 revealed that γ-GT, AST and ALT were signifcantly elevated in the M group, while these alterations were obviously reversed by treatment with L. rhamnosus LS-8 and L. crustorum MN047.','in vivo experiment','qPCR',31659451),(5513,'mouse','decrease','low-throughput','On the contrary, the increasing trend of these pro-infammatory factors in HFFD-fed mice were apparently down-regulated by the administration with L. rhamnosus LS-8 and L. crustorum MN047.','in vivo experiment','qPCR',31659451),(5514,'mouse','decrease','low-throughput','On the contrary, the increasing trend of these pro-infammatory factors in HFFD-fed mice were apparently down-regulated by the administration with L. rhamnosus LS-8 and L. crustorum MN047.','in vivo experiment','qPCR',31659451),(5515,'mouse','decrease','low-throughput','On the contrary, the increasing trend of these pro-infammatory factors in HFFD-fed mice were apparently down-regulated by the administration with L. rhamnosus LS-8 and L. crustorum MN047.','in vivo experiment','qPCR',31659451),(5516,'mouse','decrease','low-throughput','Results indicated that the mRNA expressions of hepatic lipogenic genes (Pparγ and Srebp-1c) were higher in the HFFD-fed mice than that in the C group mice, while the expression of these genes were signifcantly reduced via supplementation of L. rhamnosus LS-8 and L. crustorum MN047.','in vivo experiment','qPCR',31659451),(5517,'mouse','decrease','low-throughput','Results indicated that the mRNA expressions of hepatic lipogenic genes (Pparγ and Srebp-1c) were higher in the HFFD-fed mice than that in the C group mice, while the expression of these genes were signifcantly reduced via supplementation of L. rhamnosus LS-8 and L. crustorum MN047.','in vivo experiment','qPCR',31659451),(5518,'mouse','decrease','low-throughput','Moreover, the expressions of fatty acid metabolic genes, including CD36 and Fabp2, were also dramatically increased in the M group compared to the C group and showed a down-regulated expression in L. rhamnosus LS-8 and L. crustorum MN047 treatment group.','in vivo experiment','qPCR',31659451),(5519,'mouse','decrease','low-throughput','Moreover, the expressions of fatty acid metabolic genes, including CD36 and Fabp2, were also dramatically increased in the M group compared to the C group and showed a down-regulated expression in L. rhamnosus LS-8 and L. crustorum MN047 treatment group.','in vivo experiment','qPCR',31659451),(5520,'mouse','decrease','low-throughput','Table 3 revealed that γ-GT, AST and ALT were signifcantly elevated in the M group, while these alterations were obviously reversed by treatment with L. rhamnosus LS-8 and L. crustorum MN047.','in vivo experiment','qPCR',31659451),(5521,'mouse','decrease','low-throughput','Table 3 revealed that γ-GT, AST and ALT were signifcantly elevated in the M group, while these alterations were obviously reversed by treatment with L. rhamnosus LS-8 and L. crustorum MN047.','in vivo experiment','qPCR',31659451),(5522,'mouse','decrease','low-throughput','Table 3 revealed that γ-GT, AST and ALT were signifcantly elevated in the M group, while these alterations were obviously reversed by treatment with L. rhamnosus LS-8 and L. crustorum MN047.','in vivo experiment','qPCR',31659451),(5523,'mouse','decrease','low-throughput','On the contrary, the increasing trend of these pro-infammatory factors in HFFD-fed mice were apparently down-regulated by the administration with L. rhamnosus LS-8 and L. crustorum MN047.','in vivo experiment','qPCR',31659451),(5524,'mouse','decrease','low-throughput','On the contrary, the increasing trend of these pro-infammatory factors in HFFD-fed mice were apparently down-regulated by the administration with L. rhamnosus LS-8 and L. crustorum MN047.','in vivo experiment','qPCR',31659451),(5525,'mouse','decrease','low-throughput','On the contrary, the increasing trend of these pro-infammatory factors in HFFD-fed mice were apparently down-regulated by the administration with L. rhamnosus LS-8 and L. crustorum MN047.','in vivo experiment','qPCR',31659451),(5526,'mouse','decrease','low-throughput','Results indicated that the mRNA expressions of hepatic lipogenic genes (Pparγ and Srebp-1c) were higher in the HFFD-fed mice than that in the C group mice, while the expression of these genes were signifcantly reduced via supplementation of L. rhamnosus LS-8 and L. crustorum MN047.','in vivo experiment','qPCR',31659451),(5527,'mouse','decrease','low-throughput','Results indicated that the mRNA expressions of hepatic lipogenic genes (Pparγ and Srebp-1c) were higher in the HFFD-fed mice than that in the C group mice, while the expression of these genes were signifcantly reduced via supplementation of L. rhamnosus LS-8 and L. crustorum MN047.','in vivo experiment','qPCR',31659451),(5528,'mouse','decrease','low-throughput','Moreover, the expressions of fatty acid metabolic genes, including CD36 and Fabp2, were also dramatically increased in the M group compared to the C group and showed a down-regulated expression in L. rhamnosus LS-8 and L. crustorum MN047 treatment group.','in vivo experiment','qPCR',31659451),(5529,'mouse','decrease','low-throughput','Moreover, the expressions of fatty acid metabolic genes, including CD36 and Fabp2, were also dramatically increased in the M group compared to the C group and showed a down-regulated expression in L. rhamnosus LS-8 and L. crustorum MN047 treatment group.','in vivo experiment','qPCR',31659451),(5530,'mouse','increase','low-throughput','The culture supernatant of F. prausnitzii increased plasma anti-Th17 cytokines (IL-10 and IL-12)and suppressed IL-17 levels in both plasma and colonic mucosa, with ameliorated colonic colitis lesions.','cell culture,in vitro bacterial culture','ELISA',25275569),(5531,'mouse','increase','low-throughput','The culture supernatant of F. prausnitzii increased plasma anti-Th17 cytokines (IL-10 and IL-12)and suppressed IL-17 levels in both plasma and colonic mucosa, with ameliorated colonic colitis lesions.','cell culture,in vitro bacterial culture','ELISA',25275569),(5532,'mouse','increase','low-throughput','The culture supernatant of F. prausnitzii increased plasma anti-Th17 cytokines (IL-10 and IL-12)and suppressed IL-17 levels in both plasma and colonic mucosa, with ameliorated colonic colitis lesions.','cell culture,in vitro bacterial culture','ELISA',25275569),(5533,'mouse','decrease','low-throughput','The culture supernatant of F. prausnitzii increased plasma anti-Th17 cytokines (IL-10 and IL-12)and suppressed IL-17 levels in both plasma and colonic mucosa, with ameliorated colonic colitis lesions.','cell culture,in vitro bacterial culture','ELISA',25275569),(5534,'mouse','decrease','low-throughput','A. muciniphila treatment significantly reduced the Con A-induced elevation of serum ALT and AST activities.','in vivo experiment','qPCR',29033903),(5535,'mouse','decrease','low-throughput','A. muciniphila treatment significantly reduced the Con A-induced elevation of serum ALT and AST activities.','in vivo experiment','qPCR',29033903),(5536,'mouse','decrease','low-throughput','Serum levels of pro-inflammatory cytokines and chemokines (IL-2, IFN-γ, IL-12p40, MCP-1, MIP-1a, MIP-1b) were substantially attenuated.','in vivo experiment','qPCR',29033903),(5537,'mouse','decrease','low-throughput','Serum levels of pro-inflammatory cytokines and chemokines (IL-2, IFN-γ, IL-12p40, MCP-1, MIP-1a, MIP-1b) were substantially attenuated.','in vivo experiment','qPCR',29033903),(5538,'mouse','decrease','low-throughput','Serum levels of pro-inflammatory cytokines and chemokines (IL-2, IFN-γ, IL-12p40, MCP-1, MIP-1a, MIP-1b) were substantially attenuated.','in vivo experiment','qPCR',29033903),(5539,'mouse','decrease','low-throughput','Serum levels of pro-inflammatory cytokines and chemokines (IL-2, IFN-γ, IL-12p40, MCP-1, MIP-1a, MIP-1b) were substantially attenuated.','in vivo experiment','qPCR',29033903),(5540,'mouse','decrease','low-throughput','Serum levels of pro-inflammatory cytokines and chemokines (IL-2, IFN-γ, IL-12p40, MCP-1, MIP-1a, MIP-1b) were substantially attenuated.','in vivo experiment','qPCR',29033903),(5541,'mouse','decrease','low-throughput','A. muciniphila significantly reduced the increase in IFN-g, IL-2, IL-1b, and IL-12p40.','in vivo experiment','qPCR',29033903),(5542,'mouse','decrease','low-throughput','A. muciniphila significantly reduced the increase in IFN-g, IL-2, IL-1b, and IL-12p40.','in vivo experiment','qPCR',29033903),(5543,'mouse','increase','low-throughput','A. muciniphila significantly decreased hepatocellular apoptosis; Bcl-2 expression increased, but Fas and DR5 decreased.','in vivo experiment','qPCR',29033903),(5544,'mouse','decrease','low-throughput','A. muciniphila significantly decreased hepatocellular apoptosis; Bcl-2 expression increased, but Fas and DR5 decreased.','in vivo experiment','qPCR',29033903),(5545,'mouse','decrease','low-throughput','A. muciniphila significantly decreased hepatocellular apoptosis; Bcl-2 expression increased, but Fas and DR5 decreased.','in vivo experiment','qPCR',29033903),(5546,'mouse','increase','low-throughput','Further investigation showed that A. muciniphila enhanced expression of Occludin and Tjp-1 and inhibited CB1 receptor.','in vivo experiment','qPCR',29033903),(5547,'mouse','increase','low-throughput','Further investigation showed that A. muciniphila enhanced expression of Occludin and Tjp-1 and inhibited CB1 receptor.','in vivo experiment','qPCR',29033903),(5548,'mouse','decrease','low-throughput','Further investigation showed that A. muciniphila enhanced expression of Occludin and Tjp-1 and inhibited CB1 receptor.','in vivo experiment','qPCR',29033903),(5549,'mouse','increase','low-throughput','Besides this, not only Th1 cytokines but also regulatory T (Treg) cytokines including IL-12, IFN-g and TGF-b were significantly increased in the LGG pre- and posttreatment groups.','in vivo experiment','ELISA',25440975),(5550,'mouse','increase','low-throughput','Besides this, not only Th1 cytokines but also regulatory T (Treg) cytokines including IL-12, IFN-g and TGF-b were significantly increased in the LGG pre- and posttreatment groups.','in vivo experiment','ELISA',25440975),(5551,'mouse','increase','low-throughput','Besides this, not only Th1 cytokines but also regulatory T (Treg) cytokines including IL-12, IFN-g and TGF-b were significantly increased in the LGG pre- and posttreatment groups.','in vivo experiment','ELISA',25440975),(5552,'mouse','increase','low-throughput','Besides this, not only Th1 cytokines but also regulatory T (Treg) cytokines including IL-12, IFN-g and TGF-b were significantly increased in the LGG pre- and posttreatment groups.','in vivo experiment','ELISA',25440975),(5553,'mouse','increase','low-throughput','Besides this, not only Th1 cytokines but also regulatory T (Treg) cytokines including IL-12, IFN-g and TGF-b were significantly increased in the LGG pre- and posttreatment groups.','in vivo experiment','ELISA',25440975),(5554,'mouse','decrease','low-throughput','After the LGG treatment, in both of the groups treated with LGG in the allergic asthma models, there was significantly less MMP9 expression.','in vivo experiment','ELISA',25440975),(5555,'mouse','increase','low-throughput','LA counteracts CR-induced decrease in NHE3 and DRA expression.','in vivo experiment','RT-PCR',27634011),(5556,'mouse','increase','low-throughput','LA counteracts CR-induced decrease in NHE3 and DRA expression.','in vivo experiment','RT-PCR',27634011),(5557,'mouse','increase','low-throughput','LA alone had no effect but significantly attenuated CR-induced decrease in CAI and CAIV mRNA.','in vivo experiment','RT-PCR',27634011),(5558,'mouse','increase','low-throughput','LA alone had no effect but significantly attenuated CR-induced decrease in CAI and CAIV mRNA.','in vivo experiment','RT-PCR',27634011),(5559,'mouse','increase','low-throughput','LA alone had no effect on NHERF 1 and 2 but significantly increased NHERF3 mRNA level.','in vivo experiment','RT-PCR',27634011),(5560,'mouse','decrease','low-throughput','LA alone showed significantly blocked the CR-induced increase in their mRNA levels in the CR+LA group.','in vivo experiment','RT-PCR',27634011),(5561,'mouse','decrease','low-throughput','LA alone showed significantly blocked the CR-induced increase in their mRNA levels in the CR+LA group.','in vivo experiment','RT-PCR',27634011),(5562,'mouse','decrease','low-throughput','LA alone showed significantly blocked the CR-induced increase in their mRNA levels in the CR+LA group.','in vivo experiment','RT-PCR',27634011),(5563,'mouse','increase','low-throughput','Similar to the in vitro studies, in vivo studies using mice gavaged with LA also showed significantly increased DRA mRNA ( approximately 4-fold) and protein expression in the colonic regions as assessed by Western blot analysis and immunofluorescence.','in vivo experiment','Western blot assay',20044511),(5564,'mouse','decrease','low-throughput','Intrarectal administration of C. tyrobutyricum prevented appearance of clinical symptoms of DSS-colitis, restored normal MUC-2 production, unaltered expression of TJ protein ZO-1 and decreased levels of TNF-α and IL-18 in the descending colon of SCID and BALB/c mice, respectively.','in vivo experiment','ELISA,RT-PCR',22236013),(5565,'mouse','decrease','low-throughput','Intrarectal administration of C. tyrobutyricum prevented appearance of clinical symptoms of DSS-colitis, restored normal MUC-2 production, unaltered expression of TJ protein ZO-1 and decreased levels of TNF-α and IL-18 in the descending colon of SCID and BALB/c mice, respectively.','in vivo experiment','ELISA,RT-PCR',22236013),(5566,'mouse','increase','low-throughput','Intrarectal administration of C. tyrobutyricum prevented appearance of clinical symptoms of DSS-colitis, restored normal MUC-2 production, unaltered expression of TJ protein ZO-1 and decreased levels of TNF-α and IL-18 in the descending colon of SCID and BALB/c mice, respectively.','in vivo experiment','ELISA,RT-PCR',22236013),(5567,'mouse','increase','low-throughput','Intrarectal administration of C. tyrobutyricum prevented appearance of clinical symptoms of DSS-colitis, restored normal MUC-2 production, unaltered expression of TJ protein ZO-1 and decreased levels of TNF-α and IL-18 in the descending colon of SCID and BALB/c mice, respectively.','in vivo experiment','ELISA,RT-PCR',22236013),(5573,'mouse','increase','low-throughput','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.','control experiment,Supplementation of metabolite','16S rRNA sequence,Western blot assay,RT–qPCR',32728212),(5574,'mouse','increase','low-throughput','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.','control experiment,Supplementation of metabolite','16S rRNA sequence,Western blot assay,RT–qPCR',32728212),(5575,'mouse','increase','low-throughput','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.','control experiment,Supplementation of metabolite','16S rRNA sequence,Western blot assay,RT–qPCR',32728212),(5576,'mouse','increase','low-throughput','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.','control experiment,Supplementation of metabolite','16S rRNA sequence,Western blot assay,RT–qPCR',32728212),(5577,'mouse','increase','low-throughput','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.','control experiment,Supplementation of metabolite','16S rRNA sequence,Western blot assay,RT–qPCR',32728212),(5578,'mouse','increase','low-throughput','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.','control experiment,Supplementation of metabolite','16S rRNA sequence,Western blot assay,RT–qPCR',32728212),(5579,'mouse','increase','low-throughput','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.','control experiment,Supplementation of metabolite','16S rRNA sequence,Western blot assay,RT–qPCR',32728212),(5580,'mouse','increase','low-throughput','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.','control experiment,Supplementation of metabolite','16S rRNA sequence,Western blot assay,RT–qPCR',32728212),(5581,'mouse','increase','low-throughput','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.','control experiment,Supplementation of metabolite','16S rRNA sequence,Western blot assay,RT–qPCR',32728212),(5582,'mouse','increase','low-throughput','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.','control experiment,Supplementation of metabolite','16S rRNA sequence,Western blot assay,RT–qPCR',32728212),(5583,'mouse','increase','low-throughput','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.','control experiment,Supplementation of metabolite','16S rRNA sequence,Western blot assay,RT–qPCR',32728212),(5584,'mouse','increase','low-throughput','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.','control experiment,Supplementation of metabolite','16S rRNA sequence,Western blot assay,RT–qPCR',32728212),(5585,'mouse','increase','low-throughput','Taken together, these data demonstrate bacterial-intrinsic phytate metabolism and provoke the hypothesis that breakdown of phytate by microbiota promotes intestinal HDAC activity.','in vitro bacterial culture,control experiment','qPCR',32731255),(5586,'mouse','decrease','low-throughput','Further, colonization of germ-free (GF) mice with butyrate producing bacteria,Faecalibacterium prausnitzii, inhibited IEC-intrinsic HDAC activity.','in vitro bacterial culture,control experiment','qPCR',32731255),(5587,'mouse','increase','low-throughput','CNV mice exhibited significantly higher IP3 in both the intestinal lumen (Fig. 1e) and colonic IECs (Fig. 1f), supporting that microbiota promote increased IP3. To assess the effect of IP3 on epithelial HDAC3 activity, HDAC3 was purified by immunoprecipitation from IECs and incubated with IP3. IP3 increased endogenous HDAC3 activity (Fig. 1g), supporting that IP3 directly activates HDAC3 in IECs.','in vitro bacterial culture,control experiment','qPCR',32731255),(5588,'mouse','increase','low-throughput','Although less potent than IP3, IP4 also increased HDAC activity in IECs (Extended Data Fig. 5f), suggesting that other inositol phosphates simultaneously generated by phytate metabolism may provide synergistic effects.','in vitro bacterial culture,control experiment','qPCR',32731255),(5589,'mouse','increase','low-throughput','Glutamate, the agonist of NMDAR and a major excitatory neurotransmitter in the mammalian central nervous system (CNS) [6], is intracellularly located and extensively distributed in the brain.','null','null',32290475),(5590,'mouse','increase','low-throughput','Glutamate, the agonist of NMDAR and a major excitatory neurotransmitter in the mammalian central nervous system (CNS) [6], is intracellularly located and extensively distributed in the brain.','null','null',32290475),(5591,'mouse','increase','low-throughput','Non-pathogenic bacteria with glutamate racemase including Corynebacterium glutamicum, Brevibacterium lactofermentum, Brevibacterium avium [121], Mycobacterium smegmatis[122], and Bacillus subtilis[123] convert l-glutamate to d-glutamate.','null','null',32290475),(5592,'mouse','increase','low-throughput','Non-pathogenic bacteria with glutamate racemase including Corynebacterium glutamicum, Brevibacterium lactofermentum, Brevibacterium avium [121], Mycobacterium smegmatis[122], and Bacillus subtilis[124] convert l-glutamate to d-glutamate.','null','null',32290475),(5593,'mouse','increase','low-throughput','Coryneform bacteria are mainly used to produce glutamate in industry. LAB strains such as Lactobacillus plantarum, Lactococcus lactis, and Lactobacillus paracasei can synthesize glutamate [113,114]. ','null','null',32290475),(5594,'mouse','increase','low-throughput','Coryneform bacteria are mainly used to produce glutamate in industry. LAB strains such as Lactobacillus plantarum, Lactococcus lactis, and Lactobacillus paracasei can synthesize glutamate [113,114]. ','null','null',32290475),(5595,'mouse','increase','low-throughput','Coryneform bacteria are mainly used to produce glutamate in industry. LAB strains such as Lactobacillus plantarum, Lactococcus lactis, and Lactobacillus paracasei can synthesize glutamate [113,114]. ','null','null',32290475),(5596,'mouse','increase','low-throughput','Coryneform bacteria are mainly used to produce glutamate in industry. LAB strains such as Lactobacillus plantarum, Lactococcus lactis, and Lactobacillus paracasei can synthesize glutamate [113,114]. ','null','null',32290475),(5598,'mouse','increase','low-throughput','Reconstitution with a single bacterial species, Clostridium scindens, or its derived metabolite, the secondary bile acid deoxycholic acid, can restore pDC- and MyD88-dependent type I IFN responses to restrict systemic CHIKV infection and transmission back to vector mosquitoes. ','cell culture,control experiment,in vitro bacterial culture','16S rDNA sequence,qPCR',32668198),(5599,'mouse','increase','low-throughput','Reconstitution with a single bacterial species, Clostridium scindens, or its derived metabolite, the secondary bile acid deoxycholic acid, can restore pDC- and MyD88-dependent type I IFN responses to restrict systemic CHIKV infection and transmission back to vector mosquitoes. ','cell culture,control experiment,in vitro bacterial culture','16S rDNA sequence,qPCR',32668198),(5601,'mouse','decrease','low-throughput','Moreover, the culture supernatant of B fra gilis YCH46 could significantly reduce the corticoste rone level and reverse the abnormal expressions of Mr, Cyp11a1, Cyp11b1, Hsd11b1, and Hsd11b2 in high salt–fed mice colon tissues (Figure 5E through 5J). Namely, B fragilis effectively counteracted HSD effects on intesti nal corticosterone metabolism and dramatically reduced intestinal corticosterone production level.','in vitro bacterial culture,control experiment','16S rRNA sequence,HPLC analysis',32078445),(5602,'mouse','decrease','low-throughput','Moreover, the culture supernatant of B fra gilis YCH46 could significantly reduce the corticoste rone level and reverse the abnormal expressions of Mr, Cyp11a1, Cyp11b1, Hsd11b1, and Hsd11b2 in high salt–fed mice colon tissues (Figure 5E through 5J). Namely, B fragilis effectively counteracted HSD effects on intesti nal corticosterone metabolism and dramatically reduced intestinal corticosterone production level.','in vitro bacterial culture,control experiment','16S rRNA sequence,HPLC analysis',32078445),(5603,'mouse','decrease','low-throughput','Moreover, the culture supernatant of B fra gilis YCH46 could significantly reduce the corticoste rone level and reverse the abnormal expressions of Mr, Cyp11a1, Cyp11b1, Hsd11b1, and Hsd11b2 in high salt–fed mice colon tissues (Figure 5E through 5J). Namely, B fragilis effectively counteracted HSD effects on intesti nal corticosterone metabolism and dramatically reduced intestinal corticosterone production level.','in vitro bacterial culture,control experiment','16S rRNA sequence,HPLC analysis',32078445),(5604,'mouse','increase','low-throughput','Moreover, the culture supernatant of B fra gilis YCH46 could significantly reduce the corticoste rone level and reverse the abnormal expressions of Mr, Cyp11a1, Cyp11b1, Hsd11b1, and Hsd11b2 in high salt–fed mice colon tissues (Figure 5E through 5J). Namely, B fragilis effectively counteracted HSD effects on intesti nal corticosterone metabolism and dramatically reduced intestinal corticosterone production level.','in vitro bacterial culture,control experiment','16S rRNA sequence,HPLC analysis',32078445),(5617,'mouse','increase','low-throughput','In this study, we observed that in mice exposed to HFD or without intestinal epithelial cell AhR expression, the efect of diet was more signifcant than the absence of AhR on the fecal microbiome composition. On the other hand, the absence of AhR had a more pronounced efect on the fecal metabolome than diet. Tryptophan metabolites were among those that were signifcantly altered in the metabolome, and integrated microbiome and metabolome analysis predicted three taxa—unclas sifed Clostridiales, unclassifed Desulfovibrionaceae, and Akkermansia—as key contributors to the synthesis and/or degradation of tryptophan metabolites. Our study highlights the use of multi-omic analysis to investigate the relationship between microbiome and metabolome and suggests possible taxa that can be targeted to manip ulate the microbiome for CRC prevention.','control experiment','16S rRNA sequence,LC/MS analysis',33256731),(5618,'mouse','increase','low-throughput','In this study, we observed that in mice exposed to HFD or without intestinal epithelial cell AhR expression, the efect of diet was more signifcant than the absence of AhR on the fecal microbiome composition. On the other hand, the absence of AhR had a more pronounced efect on the fecal metabolome than diet. Tryptophan metabolites were among those that were signifcantly altered in the metabolome, and integrated microbiome and metabolome analysis predicted three taxa—unclas sifed Clostridiales, unclassifed Desulfovibrionaceae, and Akkermansia—as key contributors to the synthesis and/or degradation of tryptophan metabolites. Our study highlights the use of multi-omic analysis to investigate the relationship between microbiome and metabolome and suggests possible taxa that can be targeted to manip ulate the microbiome for CRC prevention.','control experiment','16S rRNA sequence,LC/MS analysis',33256731),(5619,'mouse','increase','low-throughput','In this study, we observed that in mice exposed to HFD or without intestinal epithelial cell AhR expression, the efect of diet was more signifcant than the absence of AhR on the fecal microbiome composition. On the other hand, the absence of AhR had a more pronounced efect on the fecal metabolome than diet. Tryptophan metabolites were among those that were signifcantly altered in the metabolome, and integrated microbiome and metabolome analysis predicted three taxa—unclas sifed Clostridiales, unclassifed Desulfovibrionaceae, and Akkermansia—as key contributors to the synthesis and/or degradation of tryptophan metabolites. Our study highlights the use of multi-omic analysis to investigate the relationship between microbiome and metabolome and suggests possible taxa that can be targeted to manip ulate the microbiome for CRC prevention.','control experiment','16S rRNA sequence,LC/MS analysis',33256731),(5632,'mouse','increase','low-throughput','Interestingly, in the same mice, indole induced a dramatic increase of the expression of the adrenomedullary Pnmt gene, which is involved in catecholamine biosynthesis.','control experiment','HPLC analysis,ELISA,RT-PCR',32569990),(5641,'mouse','decrease','low-throughput','Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).','cell culture,control experiment,Supplementation of metabolite','16S rRNA sequence,RT-PCR',33024047),(5642,'mouse','decrease','low-throughput','Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).','cell culture,control experiment,Supplementation of metabolite','16S rRNA sequence,RT-PCR',33024047),(5643,'mouse','decrease','low-throughput','Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).','cell culture,control experiment,Supplementation of metabolite','16S rRNA sequence,RT-PCR',33024047),(5644,'mouse','decrease','low-throughput','Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).','cell culture,control experiment,Supplementation of metabolite','16S rRNA sequence,RT-PCR',33024047),(5645,'mouse','decrease','low-throughput','Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).','cell culture,control experiment,Supplementation of metabolite','16S rRNA sequence,RT-PCR',33024047),(5646,'mouse','decrease','low-throughput','Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).','cell culture,control experiment,Supplementation of metabolite','16S rRNA sequence,RT-PCR',33024047),(5647,'mouse','decrease','low-throughput','Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).','cell culture,control experiment,Supplementation of metabolite','16S rRNA sequence,RT-PCR',33024047),(5648,'mouse','decrease','low-throughput','Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).','cell culture,control experiment,Supplementation of metabolite','16S rRNA sequence,RT-PCR',33024047);
/*!40000 ALTER TABLE `gut_microbiota_gene_change_results` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `gut_microbiota_metabolite_generation_results`
--

DROP TABLE IF EXISTS `gut_microbiota_metabolite_generation_results`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `gut_microbiota_metabolite_generation_results` (
  `index` int DEFAULT NULL,
  `host_type` text COLLATE utf8mb4_bin,
  `substrate_name` text COLLATE utf8mb4_bin,
  `description` text COLLATE utf8mb4_bin,
  `Evidence` int DEFAULT NULL
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `gut_microbiota_metabolite_generation_results`
--

LOCK TABLES `gut_microbiota_metabolite_generation_results` WRITE;
/*!40000 ALTER TABLE `gut_microbiota_metabolite_generation_results` DISABLE KEYS */;
INSERT INTO `gut_microbiota_metabolite_generation_results` VALUES (3059,'human','D-Glucose','Major end products of metabolism of glucose are acetic and butyric acids.',21357455),(3060,'human','Salicin','Acid is produced from glucose, salicin, D-xylose, L-arabinose and L-rhamnose. Acid is produced weakly from D-mannose.',21357455),(3061,'human','D-Xylose','Acid is produced from glucose, salicin, D-xylose, L-arabinose and L-rhamnose. Acid is produced weakly from D-mannose.',21357455),(3062,'human','L-Arabinose','Acid is produced from glucose, salicin, D-xylose, L-arabinose and L-rhamnose. Acid is produced weakly from D-mannose.',21357455),(3063,'human','L-Rhamnose','Acid is produced from glucose, salicin, D-xylose, L-arabinose and L-rhamnose. Acid is produced weakly from D-mannose.',21357455),(3064,'human','D-Mannose','Acid is produced from glucose, salicin, D-xylose, L-arabinose and L-rhamnose. Acid is produced weakly from D-mannose.',21357455),(3065,'human','D-Glucose','Major end products of metabolism of glucose are acetic and butyric acids.',21357455),(3066,'human','Salicin','Acid is produced from glucose, salicin, D-xylose, L-arabinose and L-rhamnose. Acid is produced weakly from D-mannose.',21357455),(3067,'human','D-Xylose','Acid is produced from glucose, salicin, D-xylose, L-arabinose and L-rhamnose. Acid is produced weakly from D-mannose.',21357455),(3068,'human','L-Arabinose','Acid is produced from glucose, salicin, D-xylose, L-arabinose and L-rhamnose. Acid is produced weakly from D-mannose.',21357455),(3069,'human','L-Rhamnose','Acid is produced from glucose, salicin, D-xylose, L-arabinose and L-rhamnose. Acid is produced weakly from D-mannose.',21357455),(3070,'human','D-Mannose','Acid is produced from glucose, salicin, D-xylose, L-arabinose and L-rhamnose. Acid is produced weakly from D-mannose.',21357455),(3071,'human','Orientin','This strain, which was able to cleave the C-glycoside of orientin to produce luteolin and generate some other metabolites.',24316522),(3072,'human','Bile acid','Deoxycholic acid (DCA) and lithocholic acid (LCA) are secondary bile acids that are generated in the gut by bacterial transformation of primary bile acids, cholic acid(CA), and chenodeoxycholic acid (CDCA), respectively.The proposed pathway for this transformation by Clostridium scindens VPI 12708.',23836456),(3073,'human','Cholic acid','Deoxycholic acid (DCA) and lithocholic acid (LCA) are secondary bile acids that are generated in the gut by bacterial transformation of primary bile acids, cholic acid(CA), and chenodeoxycholic acid (CDCA), respectively.The proposed pathway for this transformation by Clostridium scindens VPI 12708.',23836456),(3074,'human','Chenodeoxycholic acid','Deoxycholic acid (DCA) and lithocholic acid (LCA) are secondary bile acids that are generated in the gut by bacterial transformation of primary bile acids, cholic acid(CA), and chenodeoxycholic acid (CDCA), respectively.The proposed pathway for this transformation by Clostridium scindens VPI 12708.',23836456),(3075,'human','Bile acid','Deoxycholic acid (DCA) and lithocholic acid (LCA) are secondary bile acids that are generated in the gut by bacterial transformation of primary bile acids, cholic acid(CA), and chenodeoxycholic acid (CDCA), respectively.The proposed pathway for this transformation by Clostridium scindens VPI 12708.',23836456),(3076,'human','Cholic acid','Deoxycholic acid (DCA) and lithocholic acid (LCA) are secondary bile acids that are generated in the gut by bacterial transformation of primary bile acids, cholic acid(CA), and chenodeoxycholic acid (CDCA), respectively.The proposed pathway for this transformation by Clostridium scindens VPI 12708.',23836456),(3077,'human','Chenodeoxycholic acid','Deoxycholic acid (DCA) and lithocholic acid (LCA) are secondary bile acids that are generated in the gut by bacterial transformation of primary bile acids, cholic acid(CA), and chenodeoxycholic acid (CDCA), respectively.The proposed pathway for this transformation by Clostridium scindens VPI 12708.',23836456),(3080,'human','Cholesterol','Calprotectin, as the sole feature in its own measurement type, was weakly associated with a number of metabolites that were not differentially abundant in dysbiosis, as well as with the metagenomic abundance of several dysbiosis-associated ECs. Three host genes appeared in this high-significance subnetwork: ileal expression of GIP, NXPE4, and ANXA10. Expression of RNA polymerase was also a prominent node in the network, though not a hub, that was upregulated in dysbiosis. The regulation of this essential enzyme class is growth-rate-dependent, suggesting that microbial communities as a whole are more often in higher growth conditions in dysbiotic IBD.',31142855),(3081,'human','Cholesterol','Calprotectin, as the sole feature in its own measurement type, was weakly associated with a number of metabolites that were not differentially abundant in dysbiosis, as well as with the metagenomic abundance of several dysbiosis-associated ECs. Three host genes appeared in this high-significance subnetwork: ileal expression of GIP, NXPE4, and ANXA10. Expression of RNA polymerase was also a prominent node in the network, though not a hub, that was upregulated in dysbiosis. The regulation of this essential enzyme class is growth-rate-dependent, suggesting that microbial communities as a whole are more often in higher growth conditions in dysbiotic IBD.',31142855),(3082,'human','Cholesterol','Calprotectin, as the sole feature in its own measurement type, was weakly associated with a number of metabolites that were not differentially abundant in dysbiosis, as well as with the metagenomic abundance of several dysbiosis-associated ECs. Three host genes appeared in this high-significance subnetwork: ileal expression of GIP, NXPE4, and ANXA10. Expression of RNA polymerase was also a prominent node in the network, though not a hub, that was upregulated in dysbiosis. The regulation of this essential enzyme class is growth-rate-dependent, suggesting that microbial communities as a whole are more often in higher growth conditions in dysbiotic IBD.',31142855),(3083,'human','Cholesterol','Calprotectin, as the sole feature in its own measurement type, was weakly associated with a number of metabolites that were not differentially abundant in dysbiosis, as well as with the metagenomic abundance of several dysbiosis-associated ECs. Three host genes appeared in this high-significance subnetwork: ileal expression of GIP, NXPE4, and ANXA10. Expression of RNA polymerase was also a prominent node in the network, though not a hub, that was upregulated in dysbiosis. The regulation of this essential enzyme class is growth-rate-dependent, suggesting that microbial communities as a whole are more often in higher growth conditions in dysbiotic IBD.',31142855),(3088,'human','Cynaroside','Eubacterium ramulus, a quercetin-3-glucoside degrading anaerobic microorganism that occurs at num bers of approximately 108/g dry feces in humans, was tested for its ability to transform other flavonoids. The organism degraded luteolin-7-glucoside, rutin, quercetin, kaempferol, luteolin, eriodictyol, naringenin, taxifolin, and phloretin to phenolic acids. It hydrolyzed kaempferol-3-sorphoro side-7-glucoside to kaempferol-3-sorphoroside and trans formed 3,4-dihydroxyphenylacetic acid, a product of anaer obic quercetin degradation, very slowly to non-aromatic fermentation products.',10648107),(3089,'human','Taxifolin','Eubacterium ramulus, a quercetin-3-glucoside degrading anaerobic microorganism that occurs at num bers of approximately 108/g dry feces in humans, was tested for its ability to transform other flavonoids. The organism degraded luteolin-7-glucoside, rutin, quercetin, kaempferol, luteolin, eriodictyol, naringenin, taxifolin, and phloretin to phenolic acids. It hydrolyzed kaempferol-3-sorphoro side-7-glucoside to kaempferol-3-sorphoroside and trans formed 3,4-dihydroxyphenylacetic acid, a product of anaer obic quercetin degradation, very slowly to non-aromatic fermentation products.',10648107),(3090,'human','Phloretin','Eubacterium ramulus, a quercetin-3-glucoside degrading anaerobic microorganism that occurs at num bers of approximately 108/g dry feces in humans, was tested for its ability to transform other flavonoids. The organism degraded luteolin-7-glucoside, rutin, quercetin, kaempferol, luteolin, eriodictyol, naringenin, taxifolin, and phloretin to phenolic acids. It hydrolyzed kaempferol-3-sorphoro side-7-glucoside to kaempferol-3-sorphoroside and trans formed 3,4-dihydroxyphenylacetic acid, a product of anaer obic quercetin degradation, very slowly to non-aromatic fermentation products.',10648107),(3091,'human','Luteolin','Eubacterium ramulus, a quercetin-3-glucoside degrading anaerobic microorganism that occurs at num bers of approximately 108/g dry feces in humans, was tested for its ability to transform other flavonoids. The organism degraded luteolin-7-glucoside, rutin, quercetin, kaempferol, luteolin, eriodictyol, naringenin, taxifolin, and phloretin to phenolic acids. It hydrolyzed kaempferol-3-sorphoro side-7-glucoside to kaempferol-3-sorphoroside and trans formed 3,4-dihydroxyphenylacetic acid, a product of anaer obic quercetin degradation, very slowly to non-aromatic fermentation products.',10648107),(3092,'human','5,7-Dihydroxy-2-(4-hydroxyphenyl)chroman-4-one','Eubacterium ramulus, a quercetin-3-glucoside degrading anaerobic microorganism that occurs at num bers of approximately 108/g dry feces in humans, was tested for its ability to transform other flavonoids. The organism degraded luteolin-7-glucoside, rutin, quercetin, kaempferol, luteolin, eriodictyol, naringenin, taxifolin, and phloretin to phenolic acids. It hydrolyzed kaempferol-3-sorphoro side-7-glucoside to kaempferol-3-sorphoroside and trans formed 3,4-dihydroxyphenylacetic acid, a product of anaer obic quercetin degradation, very slowly to non-aromatic fermentation products.',10648107),(3093,'human','Eriodictyol','Eubacterium ramulus, a quercetin-3-glucoside degrading anaerobic microorganism that occurs at num bers of approximately 108/g dry feces in humans, was tested for its ability to transform other flavonoids. The organism degraded luteolin-7-glucoside, rutin, quercetin, kaempferol, luteolin, eriodictyol, naringenin, taxifolin, and phloretin to phenolic acids. It hydrolyzed kaempferol-3-sorphoro side-7-glucoside to kaempferol-3-sorphoroside and trans formed 3,4-dihydroxyphenylacetic acid, a product of anaer obic quercetin degradation, very slowly to non-aromatic fermentation products.',10648107),(3094,'human','Kaempferol','Eubacterium ramulus, a quercetin-3-glucoside degrading anaerobic microorganism that occurs at num bers of approximately 108/g dry feces in humans, was tested for its ability to transform other flavonoids. The organism degraded luteolin-7-glucoside, rutin, quercetin, kaempferol, luteolin, eriodictyol, naringenin, taxifolin, and phloretin to phenolic acids. It hydrolyzed kaempferol-3-sorphoro side-7-glucoside to kaempferol-3-sorphoroside and trans formed 3,4-dihydroxyphenylacetic acid, a product of anaer obic quercetin degradation, very slowly to non-aromatic fermentation products.',10648107),(3095,'human','Rutin','Eubacterium ramulus, a quercetin-3-glucoside degrading anaerobic microorganism that occurs at num bers of approximately 108/g dry feces in humans, was tested for its ability to transform other flavonoids. The organism degraded luteolin-7-glucoside, rutin, quercetin, kaempferol, luteolin, eriodictyol, naringenin, taxifolin, and phloretin to phenolic acids. It hydrolyzed kaempferol-3-sorphoro side-7-glucoside to kaempferol-3-sorphoroside and trans formed 3,4-dihydroxyphenylacetic acid, a product of anaer obic quercetin degradation, very slowly to non-aromatic fermentation products.',10648107),(3096,'human','Quercetin','Eubacterium ramulus, a quercetin-3-glucoside degrading anaerobic microorganism that occurs at num bers of approximately 108/g dry feces in humans, was tested for its ability to transform other flavonoids. The organism degraded luteolin-7-glucoside, rutin, quercetin, kaempferol, luteolin, eriodictyol, naringenin, taxifolin, and phloretin to phenolic acids. It hydrolyzed kaempferol-3-sorphoro side-7-glucoside to kaempferol-3-sorphoroside and trans formed 3,4-dihydroxyphenylacetic acid, a product of anaer obic quercetin degradation, very slowly to non-aromatic fermentation products.',10648107),(3097,'human','Isoquercitrin','Schneider et al. (1999) isolated strains of Eubacterium ramulus from human feces that are capable of degrading quercetin-3-glucoside to 3,4-dihydroxyphenylacetic acid (3,4-DIOHA), butyrate, and acetate.Phloroglucinol was formed as an intermediate.',10648107),(3098,'human','Isoquercitrin','Schneider et al. (1999) isolated strains of Eubacterium ramulus from human feces that are capable of degrading quercetin-3-glucoside to 3,4-dihydroxyphenylacetic acid (3,4-DIOHA), butyrate, and acetate.Phloroglucinol was formed as an intermediate.',10648107),(3099,'human','Isoquercitrin','Schneider et al. (1999) isolated strains of Eubacterium ramulus from human feces that are capable of degrading quercetin-3-glucoside to 3,4-dihydroxyphenylacetic acid (3,4-DIOHA), butyrate, and acetate.Phloroglucinol was formed as an intermediate.',10648107),(3100,'human','Isoquercitrin','In addition,the identity of fermentation products 3,4-dihydroxyphenylacetic acid, formed from quercetin-3-glucosdide, or taxifolin, 3-(3,4 dihydroxyphenyl)propionic acid, formed from luteolin or eriodictyol, and 4-hydroxyphenylacetic acid, formed from kaempferol, were confirmed by LC-MS/MS. The most intensive ion peaks in the mass spectra of these compounds are listed in Table 2.',10648107),(3101,'human','Taxifolin','In addition,the identity of fermentation products 3,4-dihydroxyphenylacetic acid, formed from quercetin-3-glucosdide, or taxifolin, 3-(3,4 dihydroxyphenyl)propionic acid, formed from luteolin or eriodictyol, and 4-hydroxyphenylacetic acid, formed from kaempferol, were confirmed by LC-MS/MS. The most intensive ion peaks in the mass spectra of these compounds are listed in Table 2.',10648107),(3102,'human','Luteolin','In addition,the identity of fermentation products 3,4-dihydroxyphenylacetic acid, formed from quercetin-3-glucosdide, or taxifolin, 3-(3,4 dihydroxyphenyl)propionic acid, formed from luteolin or eriodictyol, and 4-hydroxyphenylacetic acid, formed from kaempferol, were confirmed by LC-MS/MS. The most intensive ion peaks in the mass spectra of these compounds are listed in Table 2.',10648107),(3103,'human','Eriodictyol','In addition,the identity of fermentation products 3,4-dihydroxyphenylacetic acid, formed from quercetin-3-glucosdide, or taxifolin, 3-(3,4 dihydroxyphenyl)propionic acid, formed from luteolin or eriodictyol, and 4-hydroxyphenylacetic acid, formed from kaempferol, were confirmed by LC-MS/MS. The most intensive ion peaks in the mass spectra of these compounds are listed in Table 2.',10648107),(3104,'human','Kaempferol','In addition,the identity of fermentation products 3,4-dihydroxyphenylacetic acid, formed from quercetin-3-glucosdide, or taxifolin, 3-(3,4 dihydroxyphenyl)propionic acid, formed from luteolin or eriodictyol, and 4-hydroxyphenylacetic acid, formed from kaempferol, were confirmed by LC-MS/MS. The most intensive ion peaks in the mass spectra of these compounds are listed in Table 2.',10648107),(3105,'human','5,7-Dihydroxy-2-(4-hydroxyphenyl)chroman-4-one','This assumption is supported by the degradation of eriodictyol and naringenin (IX) to 3-(3,4-dihydroxyphenyl)propionic acid and 3-(4-hydroxyphenyl)propionic acid (XII), respectively.',10648107),(3106,'human','Kaempferol-3-sorphoroside-7-glucoside','This assumption is supported by the degradation of eriodictyol and naringenin (IX) to 3-(3,4-dihydroxyphenyl)propionic acid and 3-(4-hydroxyphenyl)propionic acid (XII), respectively.',10648107),(3107,'human','Quercetin','Luteolin-5-glucoside, diosmetin-7rutinoside, naringenin-7-neohesperidoside, (+)-catechin, and (�?-epicatechin were not degraded. Cell extracts of E. ramulus contained α- and β-D-glucosidase activities, but were devoid of α-L-rhamnosidase activity.',10648107),(3108,'human','null','Schneider et al. (1999) isolated strains of Eubacterium ramulus from human feces that are capable of degrading quercetin-3-glucoside to 3,4-dihydroxyphenylacetic acid (3,4-DIOHA), butyrate, and acetate.Phloroglucinol was formed as an intermediate.',10648107),(3109,'human','null','Indoxyl-sulfate (Beta(SE) = -2.74(0.24), P = 8.8x10-29),p-cresyl-sulfate (-1.99(0.24), P = 4.6x10-16), and phenylacetylglutamine (-2.73(0.25),P = 1.2x10-25) were significantly and negatively associated with eGFR after adjusting for age, sex, body mass index (BMI), metabolite batch, type 2 diabetes, family relatedness and multiple testing using Bonferroni correction (Table 2).',26241311),(3110,'human','null','Indoxyl-sulfate (Beta(SE) = -2.74(0.24), P = 8.8x10-29),p-cresyl-sulfate (-1.99(0.24), P = 4.6x10-16), and phenylacetylglutamine (-2.73(0.25),P = 1.2x10-25) were significantly and negatively associated with eGFR after adjusting for age, sex, body mass index (BMI), metabolite batch, type 2 diabetes, family relatedness and multiple testing using Bonferroni correction (Table 2).',26241311),(3111,'human','null','Indoxyl-sulfate (Beta(SE) = -2.74(0.24), P = 8.8x10-29),p-cresyl-sulfate (-1.99(0.24), P = 4.6x10-16), and phenylacetylglutamine (-2.73(0.25),P = 1.2x10-25) were significantly and negatively associated with eGFR after adjusting for age, sex, body mass index (BMI), metabolite batch, type 2 diabetes, family relatedness and multiple testing using Bonferroni correction (Table 2).',26241311),(3112,'human','null','we demonstrated that GPR41 and GPR43 were modestly expressed in human distal renal tubules and col lecting tubules. We also found that SCFAs (acetate, propionate, and butyrate) inhibited TNF-a induced expression of a pro inflammatory factor, MCP-1, in HRCEs by reducing p38 and JNK phosphorylation via GPR41 and/or GPR43 activation involving Gi/o, particularly Gb/g. To the best of our knowledge, this is the first report to describe the anti-inflammatory effects of SCFAs through a GPR41/43/Gb/g pathway in renal parenchymal cells.',28322790),(3113,'human','null','we demonstrated that GPR41 and GPR43 were modestly expressed in human distal renal tubules and col lecting tubules. We also found that SCFAs (acetate, propionate, and butyrate) inhibited TNF-a induced expression of a pro inflammatory factor, MCP-1, in HRCEs by reducing p38 and JNK phosphorylation via GPR41 and/or GPR43 activation involving Gi/o, particularly Gb/g. To the best of our knowledge, this is the first report to describe the anti-inflammatory effects of SCFAs through a GPR41/43/Gb/g pathway in renal parenchymal cells.',28322790),(3114,'human','null','we demonstrated that GPR41 and GPR43 were modestly expressed in human distal renal tubules and col lecting tubules. We also found that SCFAs (acetate, propionate, and butyrate) inhibited TNF-a induced expression of a pro inflammatory factor, MCP-1, in HRCEs by reducing p38 and JNK phosphorylation via GPR41 and/or GPR43 activation involving Gi/o, particularly Gb/g. To the best of our knowledge, this is the first report to describe the anti-inflammatory effects of SCFAs through a GPR41/43/Gb/g pathway in renal parenchymal cells.',28322790),(3115,'human','null','we demonstrated that GPR41 and GPR43 were modestly expressed in human distal renal tubules and col lecting tubules. We also found that SCFAs (acetate, propionate, and butyrate) inhibited TNF-a induced expression of a pro inflammatory factor, MCP-1, in HRCEs by reducing p38 and JNK phosphorylation via GPR41 and/or GPR43 activation involving Gi/o, particularly Gb/g. To the best of our knowledge, this is the first report to describe the anti-inflammatory effects of SCFAs through a GPR41/43/Gb/g pathway in renal parenchymal cells.',28322790),(3116,'human','null','we demonstrated that GPR41 and GPR43 were modestly expressed in human distal renal tubules and col lecting tubules. We also found that SCFAs (acetate, propionate, and butyrate) inhibited TNF-a induced expression of a pro inflammatory factor, MCP-1, in HRCEs by reducing p38 and JNK phosphorylation via GPR41 and/or GPR43 activation involving Gi/o, particularly Gb/g. To the best of our knowledge, this is the first report to describe the anti-inflammatory effects of SCFAs through a GPR41/43/Gb/g pathway in renal parenchymal cells.',28322790),(3117,'human','null','we demonstrated that GPR41 and GPR43 were modestly expressed in human distal renal tubules and col lecting tubules. We also found that SCFAs (acetate, propionate, and butyrate) inhibited TNF-a induced expression of a pro inflammatory factor, MCP-1, in HRCEs by reducing p38 and JNK phosphorylation via GPR41 and/or GPR43 activation involving Gi/o, particularly Gb/g. To the best of our knowledge, this is the first report to describe the anti-inflammatory effects of SCFAs through a GPR41/43/Gb/g pathway in renal parenchymal cells.',28322790),(3118,'human','null','we demonstrated that GPR41 and GPR43 were modestly expressed in human distal renal tubules and col lecting tubules. We also found that SCFAs (acetate, propionate, and butyrate) inhibited TNF-a induced expression of a pro inflammatory factor, MCP-1, in HRCEs by reducing p38 and JNK phosphorylation via GPR41 and/or GPR43 activation involving Gi/o, particularly Gb/g. To the best of our knowledge, this is the first report to describe the anti-inflammatory effects of SCFAs through a GPR41/43/Gb/g pathway in renal parenchymal cells.',28322790),(3119,'human','null','we demonstrated that GPR41 and GPR43 were modestly expressed in human distal renal tubules and col lecting tubules. We also found that SCFAs (acetate, propionate, and butyrate) inhibited TNF-a induced expression of a pro inflammatory factor, MCP-1, in HRCEs by reducing p38 and JNK phosphorylation via GPR41 and/or GPR43 activation involving Gi/o, particularly Gb/g. To the best of our knowledge, this is the first report to describe the anti-inflammatory effects of SCFAs through a GPR41/43/Gb/g pathway in renal parenchymal cells.',28322790),(3120,'human','null','we demonstrated that GPR41 and GPR43 were modestly expressed in human distal renal tubules and col lecting tubules. We also found that SCFAs (acetate, propionate, and butyrate) inhibited TNF-a induced expression of a pro inflammatory factor, MCP-1, in HRCEs by reducing p38 and JNK phosphorylation via GPR41 and/or GPR43 activation involving Gi/o, particularly Gb/g. To the best of our knowledge, this is the first report to describe the anti-inflammatory effects of SCFAs through a GPR41/43/Gb/g pathway in renal parenchymal cells.',28322790),(3121,'human','Daidzein','One strain (strain TM-40) produced dihydrodaidzein both from daidzein and daidzin.',17113326),(3122,'human','Daidzin','One strain (strain TM-40) produced dihydrodaidzein both from daidzein and daidzin.',17113326),(3123,'human','Daidzein','A newly identified equol-producing bacterium, Slackia sp.strain NATTS, with a high equol-producing activity was isolated from human feces in Japanese adults.',24761887),(3124,'human','D-Glucose','endogenous capability of Escherichia coli for 2,3-butanediol (2,3-BD) synthesis was explored using the anaerobic selection platform based on redox balance. To address the issue of NADH excess associated with the 2,3-BD pathway, we devised a substrate-decoupled system where a pathway intermediate is externally supplied in addition to the carbon source to decouple NADH recycling ratio from the intrinsic pathway stoichiometry.',27931827),(3125,'human','Pyruvate','endogenous capability of Escherichia coli for 2,3-butanediol (2,3-BD) synthesis was explored using the anaerobic selection platform based on redox balance. To address the issue of NADH excess associated with the 2,3-BD pathway, we devised a substrate-decoupled system where a pathway intermediate is externally supplied in addition to the carbon source to decouple NADH recycling ratio from the intrinsic pathway stoichiometry.',27931827),(3126,'human','D-Glucose','endogenous capability of Escherichia coli for 2,3-butanediol (2,3-BD) synthesis was explored using the anaerobic selection platform based on redox balance. To address the issue of NADH excess associated with the 2,3-BD pathway, we devised a substrate-decoupled system where a pathway intermediate is externally supplied in addition to the carbon source to decouple NADH recycling ratio from the intrinsic pathway stoichiometry.',27931827),(3127,'human','D-Glucose','endogenous capability of Escherichia coli for 2,3-butanediol (2,3-BD) synthesis was explored using the anaerobic selection platform based on redox balance. To address the issue of NADH excess associated with the 2,3-BD pathway, we devised a substrate-decoupled system where a pathway intermediate is externally supplied in addition to the carbon source to decouple NADH recycling ratio from the intrinsic pathway stoichiometry.',27931827),(3128,'human','D-Glucose','endogenous capability of Escherichia coli for 2,3-butanediol (2,3-BD) synthesis was explored using the anaerobic selection platform based on redox balance. To address the issue of NADH excess associated with the 2,3-BD pathway, we devised a substrate-decoupled system where a pathway intermediate is externally supplied in addition to the carbon source to decouple NADH recycling ratio from the intrinsic pathway stoichiometry.',27931827),(3129,'human','Pyruvate','endogenous capability of Escherichia coli for 2,3-butanediol (2,3-BD) synthesis was explored using the anaerobic selection platform based on redox balance. To address the issue of NADH excess associated with the 2,3-BD pathway, we devised a substrate-decoupled system where a pathway intermediate is externally supplied in addition to the carbon source to decouple NADH recycling ratio from the intrinsic pathway stoichiometry.',27931827),(3130,'human','D-Glucose','endogenous capability of Escherichia coli for 2,3-butanediol (2,3-BD) synthesis was explored using the anaerobic selection platform based on redox balance. To address the issue of NADH excess associated with the 2,3-BD pathway, we devised a substrate-decoupled system where a pathway intermediate is externally supplied in addition to the carbon source to decouple NADH recycling ratio from the intrinsic pathway stoichiometry.',27931827),(3131,'human','D-Glucose','endogenous capability of Escherichia coli for 2,3-butanediol (2,3-BD) synthesis was explored using the anaerobic selection platform based on redox balance. To address the issue of NADH excess associated with the 2,3-BD pathway, we devised a substrate-decoupled system where a pathway intermediate is externally supplied in addition to the carbon source to decouple NADH recycling ratio from the intrinsic pathway stoichiometry.',27931827),(3132,'human','Choline','Klebsiella and Escherichia harboured three of the four potential TMA-production pathways (choline, carnitine and TMAO).',28785417),(3133,'human','Carnitine','Klebsiella and Escherichia harboured three of the four potential TMA-production pathways (choline, carnitine and TMAO).',28785417),(3134,'human','Trimethylamine oxide','Klebsiella and Escherichia harboured three of the four potential TMA-production pathways (choline, carnitine and TMAO).',28785417),(3135,'human','Choline','Klebsiella and Escherichia harboured three of the four potential TMA-production pathways (choline, carnitine and TMAO).',28785417),(3136,'human','Carnitine','Klebsiella and Escherichia harboured three of the four potential TMA-production pathways (choline, carnitine and TMAO).',28785417),(3137,'human','Trimethylamine oxide','Klebsiella and Escherichia harboured three of the four potential TMA-production pathways (choline, carnitine and TMAO).',28785417),(3138,'human','Hesperetin dihydrochalcone','C.orbiscindens was not able to convert hesperetindihydrochalcone 4�?â-D-glucoside within a period of 43 h, whereas the aglycon was cleaved by this species. C. orbiscindens formed 0.18 mM 3-(3-hydroxy-4-methoxyphenyl)propionic acid from 0.26 mM hesperetin dihydrochalcone within 23 h of incubation.',15740074),(3139,'human','beta-D-Glucopyranoside, 3,5-dihydroxy-4-(3-(3-hydroxy-4-methoxyphenyl)-1-oxopropyl)phenyl','However, hesperetin dihydrochalcone 4�?â-D-glucoside was converted by E. ramulus to hesperetin dihydrochalcone and further to 3-(3-hydroxy-4-methoxyphenyl)propionic acid, but not by C. orbiscindens.',15740074),(3140,'human','Hesperetin dihydrochalcone','However, hesperetin dihydrochalcone 4�?â-D-glucoside was converted by E. ramulus to hesperetin dihydrochalcone and further to 3-(3-hydroxy-4-methoxyphenyl)propionic acid, but not by C. orbiscindens.',15740074),(3147,'human','Daidzein','Strain YY7918 converted substrates daidzein and dihydrodaidzein into S-equol.',18838805),(3148,'human','Dihydrodaidzein','Strain YY7918 converted substrates daidzein and dihydrodaidzein into S-equol.',18838805),(3156,'human','null','As Erk is a ‘downstream�?signaling component of TLR pathways, we investigated whether succinate alone or in combination with TLR ligands was able to induce proinflammatory cytokine production by human DCs. Succinate alone did not have any great effect on a variety of cytokines and chemokines produced by activated human DCs. However, succinate augmented the expression of TNF and IL-1b by iMoDCs when used in combination with the TLR3 ligand poly(I:C) or the TLR7 agonist imiquimod.',18820681),(3157,'human','null','As Erk is a ‘downstream�?signaling component of TLR pathways, we investigated whether succinate alone or in combination with TLR ligands was able to induce proinflammatory cytokine production by human DCs. Succinate alone did not have any great effect on a variety of cytokines and chemokines produced by activated human DCs. However, succinate augmented the expression of TNF and IL-1b by iMoDCs when used in combination with the TLR3 ligand poly(I:C) or the TLR7 agonist imiquimod.',18820681),(3158,'human','null','Succinate alone was able to induce phosphorylation of Erk1/2 in human iMoDCs,in agreement with the reported activity of succinate on GPR91 transfectants. Polyinosinic-polycytidylic acid (poly(I:C)) alone showed a very modest activation of this signaling pathway, but when both poly(I:C) and succinate were present, Erk1/2 phosphorylation was synergistically induced.',18820681),(3159,'human','null','Succinate alone was able to induce phosphorylation of Erk1/2 in human iMoDCs,in agreement with the reported activity of succinate on GPR91 transfectants. Polyinosinic-polycytidylic acid (poly(I:C)) alone showed a very modest activation of this signaling pathway, but when both poly(I:C) and succinate were present, Erk1/2 phosphorylation was synergistically induced.',18820681),(3160,'human','null','In addition, activation of iMoDCs with poly(I:C) or imiquimod in the presence of succinate induced substantial enhancement of the expression of CCL2 mRNA and CXCL8 mRNA (data not shown). Therefore, we concluded that succinate potentiates the production and secretion of proinflam matory mediators by iMoDCs when the cells are stimulated through some TLRs.',18820681),(3161,'human','null','In addition, activation of iMoDCs with poly(I:C) or imiquimod in the presence of succinate induced substantial enhancement of the expression of CCL2 mRNA and CXCL8 mRNA (data not shown). Therefore, we concluded that succinate potentiates the production and secretion of proinflam matory mediators by iMoDCs when the cells are stimulated through some TLRs.',18820681),(3162,'human','null','butyrate enhanced the PMA-induced expression of c-fos and ERK1/2 phosphorylation, but not p38 and JNK.',23300800),(3163,'human','null','butyrate enhanced the PMA-induced expression of c-fos and ERK1/2 phosphorylation, but not p38 and JNK.',23300800),(3164,'human','null','butyrate enhanced the PMA-induced expression of c-fos and ERK1/2 phosphorylation, but not p38 and JNK.',23300800),(3165,'human','null','The activation of AP-1 was strongly correlated with the presence of butyrate in the CM.',23300800),(3166,'human','null','butyrate enhanced the PMA-induced expression of c-fos and ERK1/2 phosphorylation, but not p38 and JNK.',23300800),(3167,'human','null','butyrate enhanced the PMA-induced expression of c-fos and ERK1/2 phosphorylation, but not p38 and JNK.',23300800),(3168,'human','null','butyrate enhanced the PMA-induced expression of c-fos and ERK1/2 phosphorylation, but not p38 and JNK.',23300800),(3169,'human','null','The activation of AP-1 was strongly correlated with the presence of butyrate in the CM.',23300800),(3170,'human','null','We also measured MUC2 mRNA levels in the butyrate-treated cells, and found that 1 mM butyrate treatment for 36 h increased the cellular MUC2 mRNA levels, although the fold increase was less than that for MUC2 proteins (Fig. 3B). These results show that buty rate treatment inhibits the proliferation of LS174T cells but concomitantly increases MUC2 production per cell.',17374366),(3171,'human','null','Butyrate, which has inhibitory activity on many HDACs except the class III sirtuin family of HDACs, HDAC6, and HDAC10.',17374366),(3172,'human','null','Butyrate, which has inhibitory activity on many HDACs except the class III sirtuin family of HDACs, HDAC6, and HDAC10.',17374366),(3173,'human','null','Butyrate, which has inhibitory activity on many HDACs except the class III sirtuin family of HDACs, HDAC6, and HDAC10.',17374366),(3174,'human','null','Butyrate, which has inhibitory activity on many HDACs except the class III sirtuin family of HDACs, HDAC6, and HDAC10.',17374366),(3175,'human','null','Butyrate, which has inhibitory activity on many HDACs except the class III sirtuin family of HDACs, HDAC6, and HDAC10.',17374366),(3176,'human','null','Butyrate, which has inhibitory activity on many HDACs except the class III sirtuin family of HDACs, HDAC6, and HDAC10.',17374366),(3177,'human','null','Butyrate, which has inhibitory activity on many HDACs except the class III sirtuin family of HDACs, HDAC6, and HDAC10.',17374366),(3178,'human','null','Butyrate, which has inhibitory activity on many HDACs except the class III sirtuin family of HDACs, HDAC6, and HDAC10.',17374366),(3179,'human','null','Butyrate, which has inhibitory activity on many HDACs except the class III sirtuin family of HDACs, HDAC6, and HDAC10.',17374366),(3186,'human','Daidzein','One culture produced the metabolites dihydrodaidzein and O-desmethylangolensin, another produced dihydrodaidzein and equol. From the latter, a stable and transferable mixed culture transforming daidzein into equol was obtained. Molecular fingerprinting analysis (denaturing gradient gel electrophoresis) showed the presence of four bacterial species of which only the first three strains could be brought into pure culture. These strains were identified as Lactobacillus mucosae EPI2, Enterococcus faecium EPI1 and Finegoldia magna EPI3, and did not produce equol in pure culture. The fourth species was tentatively identified as Veillonella sp strain EP.',15578160),(3187,'human','Daidzein','One culture produced the metabolites dihydrodaidzein and O-desmethylangolensin, another produced dihydrodaidzein and equol. From the latter, a stable and transferable mixed culture transforming daidzein into equol was obtained. Molecular fingerprinting analysis (denaturing gradient gel electrophoresis) showed the presence of four bacterial species of which only the first three strains could be brought into pure culture. These strains were identified as Lactobacillus mucosae EPI2, Enterococcus faecium EPI1 and Finegoldia magna EPI3, and did not produce equol in pure culture. The fourth species was tentatively identified as Veillonella sp strain EP.',15578160),(3188,'human','Daidzein','One culture produced the metabolites dihydrodaidzein and O-desmethylangolensin, another produced dihydrodaidzein and equol. From the latter, a stable and transferable mixed culture transforming daidzein into equol was obtained. Molecular fingerprinting analysis (denaturing gradient gel electrophoresis) showed the presence of four bacterial species of which only the first three strains could be brought into pure culture. These strains were identified as Lactobacillus mucosae EPI2, Enterococcus faecium EPI1 and Finegoldia magna EPI3, and did not produce equol in pure culture. The fourth species was tentatively identified as Veillonella sp strain EP.',15578160),(3189,'human','Daidzein','One culture produced the metabolites dihydrodaidzein and O-desmethylangolensin, another produced dihydrodaidzein and equol. From the latter, a stable and transferable mixed culture transforming daidzein into equol was obtained. Molecular fingerprinting analysis (denaturing gradient gel electrophoresis) showed the presence of four bacterial species of which only the first three strains could be brought into pure culture. These strains were identified as Lactobacillus mucosae EPI2, Enterococcus faecium EPI1 and Finegoldia magna EPI3, and did not produce equol in pure culture. The fourth species was tentatively identified as Veillonella sp strain EP.',15578160),(3190,'human','Daidzein','One culture produced the metabolites dihydrodaidzein and O-desmethylangolensin, another produced dihydrodaidzein and equol. From the latter, a stable and transferable mixed culture transforming daidzein into equol was obtained. Molecular fingerprinting analysis (denaturing gradient gel electrophoresis) showed the presence of four bacterial species of which only the first three strains could be brought into pure culture. These strains were identified as Lactobacillus mucosae EPI2, Enterococcus faecium EPI1 and Finegoldia magna EPI3, and did not produce equol in pure culture. The fourth species was tentatively identified as Veillonella sp strain EP.',15578160),(3191,'human','Daidzein','One culture produced the metabolites dihydrodaidzein and O-desmethylangolensin, another produced dihydrodaidzein and equol. From the latter, a stable and transferable mixed culture transforming daidzein into equol was obtained. Molecular fingerprinting analysis (denaturing gradient gel electrophoresis) showed the presence of four bacterial species of which only the first three strains could be brought into pure culture. These strains were identified as Lactobacillus mucosae EPI2, Enterococcus faecium EPI1 and Finegoldia magna EPI3, and did not produce equol in pure culture. The fourth species was tentatively identified as Veillonella sp strain EP.',15578160),(3194,'human','null','Strains of Roseburia sp. and F. prausnitzii possessed butyryl coenzyme A (CoA):acetate-CoA transferase and acetate kinase activities, but butyrate kinase activity was not detectable either in growing or in stationary-phase cultures.',12324374),(3195,'human','null','Strains of Roseburia sp. and F. prausnitzii possessed butyryl coenzyme A (CoA):acetate-CoA transferase and acetate kinase activities, but butyrate kinase activity was not detectable either in growing or in stationary-phase cultures.',12324374),(3196,'human','null','Strains of Roseburia sp. and F. prausnitzii possessed butyryl coenzyme A (CoA):acetate-CoA transferase and acetate kinase activities, but butyrate kinase activity was not detectable either in growing or in stationary-phase cultures.',12324374),(3197,'human','null','On the other hand, Coprococcus sp. strain L2-50 produces acetate and possesses both butyrate kinase and butyryl-CoA transferase.',12324374),(3198,'human','null','A. muciniphila has previously been shown to produce acetate and propionate during its growth on mucin (24).',28687644),(3199,'human','null','A. muciniphila has previously been shown to produce acetate and propionate during its growth on mucin (24).',28687644),(3200,'human','Cianidanol','The conversion of both epicatechin and catechin resulted in the formation of metabolite M1 subsequently characterized as 1�?3,4‐dihydroxyphenyl)�?�?2,4,6‐trihydroxyphenyl)propan�?‐ol (3,4‐diHPP�?‐ol, structure in Fig. 3). In accordance with the initial conversion of the catechins, 3,4‐diHPP�?‐ol was formed more rapidly from catechin than from epicatechin.',21457417),(3201,'human','(-)-Epicatechin','The conversion of both epicatechin and catechin resulted in the formation of metabolite M1 subsequently characterized as 1�?3,4‐dihydroxyphenyl)�?�?2,4,6‐trihydroxyphenyl)propan�?‐ol (3,4‐diHPP�?‐ol, structure in Fig. 3). In accordance with the initial conversion of the catechins, 3,4‐diHPP�?‐ol was formed more rapidly from catechin than from epicatechin.',21457417),(3202,'human','1-(3,4-Dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl)-2-propanol','Two metabolites, M2 and M3, were first detected after 4 and 6 h, respectively, of incubation with F. plautii aK2. The concentration of M2 was maximal after 6 h and slightly decreased afterwards. The concentration of M3 increased rapidly until 8 h followed by a minor increase during further incubation. Characterization of the observed metabolites by MS analyses and taking literature data into account led to the assignment of M2 as 5�?3,4‐dihydroxyphenyl)‐γ‐valerolactone (3,4‐diHPVL) and M3 as 4‐hydroxy�?�?3,4‐dihydroxyphenyl)valeric acid (4‐H�?,4‐diHPVA) (structures in Fig. 3).',21457417),(3203,'human','1-(3,4-Dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl)-2-propanol','Two metabolites, M2 and M3, were first detected after 4 and 6 h, respectively, of incubation with F. plautii aK2. The concentration of M2 was maximal after 6 h and slightly decreased afterwards. The concentration of M3 increased rapidly until 8 h followed by a minor increase during further incubation. Characterization of the observed metabolites by MS analyses and taking literature data into account led to the assignment of M2 as 5�?3,4‐dihydroxyphenyl)‐γ‐valerolactone (3,4‐diHPVL) and M3 as 4‐hydroxy�?�?3,4‐dihydroxyphenyl)valeric acid (4‐H�?,4‐diHPVA) (structures in Fig. 3).',21457417),(3204,'human','null','Additionally,the abundances of genera Blautia (r = 0.307, p = 0.004), Bifidobacterium (r = 0.36, p = 0.001), Collinsella(r = 0.254, p = 0.018), and Lactobacillus (r = 0.428, p < 0.001) (Figure 2C) were positively correlated withthe urinary TMAO level.',30469463),(3205,'human','null','Additionally,the abundances of genera Blautia (r = 0.307, p = 0.004), Bifidobacterium (r = 0.36, p = 0.001), Collinsella(r = 0.254, p = 0.018), and Lactobacillus (r = 0.428, p < 0.001) (Figure 2C) were positively correlated withthe urinary TMAO level.',30469463),(3206,'human','null','Additionally,the abundances of genera Blautia (r = 0.307, p = 0.004), Bifidobacterium (r = 0.36, p = 0.001), Collinsella(r = 0.254, p = 0.018), and Lactobacillus (r = 0.428, p < 0.001) (Figure 2C) were positively correlated withthe urinary TMAO level.',30469463),(3207,'human','null','Additionally,the abundances of genera Blautia (r = 0.307, p = 0.004), Bifidobacterium (r = 0.36, p = 0.001), Collinsella(r = 0.254, p = 0.018), and Lactobacillus (r = 0.428, p < 0.001) (Figure 2C) were positively correlated withthe urinary TMAO level.',30469463),(3208,'human','null','The present analyses indicate that Faecalibacterium (formerly Fusobacterium) prausnitzii may make a significant contribution to  D-lactate and butyrate formation in the large intestine.',12508881),(3209,'human','null','The present analyses indicate that Faecalibacterium (formerly Fusobacterium) prausnitzii may make a significant contribution to  D-lactate and butyrate formation in the large intestine.',12508881),(3210,'human','null','The present analyses indicate that Faecalibacterium (formerly Fusobacterium) prausnitzii may make a significant contribution to  D-lactate and butyrate formation in the large intestine.',12508881),(3211,'human','null','The present analyses indicate that Faecalibacterium (formerly Fusobacterium) prausnitzii may make a significant contribution to  D-lactate and butyrate formation in the large intestine.',12508881),(3212,'human','null','The present analyses indicate that Faecalibacterium (formerly Fusobacterium) prausnitzii may make a significant contribution to  D-lactate and butyrate formation in the large intestine.',12508881),(3213,'human','null','The present analyses indicate that Faecalibacterium (formerly Fusobacterium) prausnitzii may make a significant contribution to  D-lactate and butyrate formation in the large intestine.',12508881),(3214,'human','null','The present analyses indicate that Faecalibacterium (formerly Fusobacterium) prausnitzii may make a significant contribution to  D-lactate and butyrate formation in the large intestine.',12508881),(3215,'human','null','The present analyses indicate that Faecalibacterium (formerly Fusobacterium) prausnitzii may make a significant contribution to  D-lactate and butyrate formation in the large intestine.',12508881),(3216,'human','null','A higher relative abundance of succinate-producing Prevotellaceae (P) and Veillonellaceae (V), and a lower relative abundance of succinate-consuming Odoribacteraceae (O) and Clostridaceae (C) in obese individuals, with the (P + V/O + C) ratio being a main determinant of plasma succinate.',29434314),(3217,'human','null','A higher relative abundance of succinate-producing Prevotellaceae (P) and Veillonellaceae (V), and a lower relative abundance of succinate-consuming Odoribacteraceae (O) and Clostridaceae (C) in obese individuals, with the (P + V/O + C) ratio being a main determinant of plasma succinate.',29434314),(3218,'human','null','We also found positive correlations with the butyrate producing Faecalibacterium prausnitzii, the genus Coprococcus,the class Mollicutes and several OTUs affiliated with the Ruminococcaceae as well as one from the family Chistensenellaceae.',31030641),(3219,'human','null','We also found positive correlations with the butyrate producing Faecalibacterium prausnitzii, the genus Coprococcus,the class Mollicutes and several OTUs affiliated with the Ruminococcaceae as well as one from the family Chistensenellaceae.',31030641),(3220,'human','null','We also found positive correlations with the butyrate producing Faecalibacterium prausnitzii, the genus Coprococcus,the class Mollicutes and several OTUs affiliated with the Ruminococcaceae as well as one from the family Chistensenellaceae.',31030641),(3221,'human','null','We also found positive correlations with the butyrate producing Faecalibacterium prausnitzii, the genus Coprococcus,the class Mollicutes and several OTUs affiliated with the Ruminococcaceae as well as one from the family Chistensenellaceae.',31030641),(3222,'human','null','We also found positive correlations with the butyrate producing Faecalibacterium prausnitzii, the genus Coprococcus,the class Mollicutes and several OTUs affiliated with the Ruminococcaceae as well as one from the family Chistensenellaceae.',31030641),(3223,'human','null','The strongest associations seen were positive associations with RF39 assigned to the Tenericutes phylum and negative associations with OTUs from the Lachnospiraceae family.',31030641),(3224,'human','null','The strongest associations seen were positive associations with RF39 assigned to the Tenericutes phylum and negative associations with OTUs from the Lachnospiraceae family.',31030641),(3225,'human','null','Negative correlations instead were observed with several Eubacterium species including E. dolichum that has been previously reported to be associated with increased visceral fat (Figure 1).',31030641),(3226,'human','null','Negative correlations instead were observed with several Eubacterium species including E. dolichum that has been previously reported to be associated with increased visceral fat (Figure 1).',31030641),(3227,'human','Chlorogenic acid','The substrate and nine metabolites were observed after the incubation of C-QA with the resting cells of human gut microbiota from nine different volunteers.',24550206),(3228,'human','Chlorogenic acid','The substrate and nine metabolites were observed after the incubation of C-QA with the resting cells of human gut microbiota from nine different volunteers.',24550206),(3229,'human','Chlorogenic acid','The substrate and nine metabolites were observed after the incubation of C-QA with the resting cells of human gut microbiota from nine different volunteers.',24550206),(3230,'human','Chlorogenic acid','The substrate and nine metabolites were observed after the incubation of C-QA with the resting cells of human gut microbiota from nine different volunteers.',24550206),(3231,'human','Chlorogenic acid','The substrate and nine metabolites were observed after the incubation of C-QA with the resting cells of human gut microbiota from nine different volunteers.',24550206),(3232,'human','Chlorogenic acid','The substrate and nine metabolites were observed after the incubation of C-QA with the resting cells of human gut microbiota from nine different volunteers.',24550206),(3233,'human','Chlorogenic acid','The substrate and nine metabolites were observed after the incubation of C-QA with the resting cells of human gut microbiota from nine different volunteers.',24550206),(3234,'human','Chlorogenic acid','The substrate and nine metabolites were observed after the incubation of C-QA with the resting cells of human gut microbiota from nine different volunteers.',24550206),(3235,'human','Chlorogenic acid','The substrate and nine metabolites were observed after the incubation of C-QA with the resting cells of human gut microbiota from nine different volunteers.',24550206),(3236,'human','Rutin','α-L-rhamnosidase from Bifidobacterium breve was produced by using Escherichia coli BL21 for biotransformation of rutin to isoquercitrin.',25724715),(3242,'human','Taxifolin','The isolated C. orbiscindens strain converted quercetin and taxifolin to 3,4-dihydroxyphenylacetic acid; luteolin and eriodictyol to 3-(3,4-dihydroxyphenyl)propionic acid; and apigenin, naringenin, and phloretin to 3-(4-hydroxyphenyl)propionic acid, respectively.',14532034),(3243,'human','Quercetin','The isolated C. orbiscindens strain converted quercetin and taxifolin to 3,4-dihydroxyphenylacetic acid; luteolin and eriodictyol to 3-(3,4-dihydroxyphenyl)propionic acid; and apigenin, naringenin, and phloretin to 3-(4-hydroxyphenyl)propionic acid, respectively.',14532034),(3244,'human','Eriodictyol','The isolated C. orbiscindens strain converted quercetin and taxifolin to 3,4-dihydroxyphenylacetic acid; luteolin and eriodictyol to 3-(3,4-dihydroxyphenyl)propionic acid; and apigenin, naringenin, and phloretin to 3-(4-hydroxyphenyl)propionic acid, respectively.',14532034),(3245,'human','Luteolin','The isolated C. orbiscindens strain converted quercetin and taxifolin to 3,4-dihydroxyphenylacetic acid; luteolin and eriodictyol to 3-(3,4-dihydroxyphenyl)propionic acid; and apigenin, naringenin, and phloretin to 3-(4-hydroxyphenyl)propionic acid, respectively.',14532034),(3246,'human','Apigenin','The isolated C. orbiscindens strain converted quercetin and taxifolin to 3,4-dihydroxyphenylacetic acid; luteolin and eriodictyol to 3-(3,4-dihydroxyphenyl)propionic acid; and apigenin, naringenin, and phloretin to 3-(4-hydroxyphenyl)propionic acid, respectively.',14532034),(3247,'human','5,7-Dihydroxy-2-(4-hydroxyphenyl)chroman-4-one','The isolated C. orbiscindens strain converted quercetin and taxifolin to 3,4-dihydroxyphenylacetic acid; luteolin and eriodictyol to 3-(3,4-dihydroxyphenyl)propionic acid; and apigenin, naringenin, and phloretin to 3-(4-hydroxyphenyl)propionic acid, respectively.',14532034),(3248,'human','Phloretin','The isolated C. orbiscindens strain converted quercetin and taxifolin to 3,4-dihydroxyphenylacetic acid; luteolin and eriodictyol to 3-(3,4-dihydroxyphenyl)propionic acid; and apigenin, naringenin, and phloretin to 3-(4-hydroxyphenyl)propionic acid, respectively.',14532034),(3249,'human','Daidzein','Daidzein was converted to equol by strain DZE.',18670101),(3250,'human','Succinate','PY broth tube tests indicated that strains YIT 12067T and YIT 12068 grew well only in PY medium supplemented with succinate. HPLC analysis showed that succinate was quantitatively converted to propionate. These results indicate that succinate decarboxylation stimulated the growth of strains YIT 12067T and YIT 12068.',22081579),(3251,'human','Succinate','PY broth tube tests indicated that strains YIT 12067T and YIT 12068 grew well only in PY medium supplemented with succinate. HPLC analysis showed that succinate was quantitatively converted to propionate. These results indicate that succinate decarboxylation stimulated the growth of strains YIT 12067T and YIT 12068.',22081579),(3252,'human','Daidzein','In this study, we isolated a human intestinal bacterium capable of transforming daidzein to equol and genistein to 5-hydroxy-equol.',19139227),(3253,'human','Genistein','In this study, we isolated a human intestinal bacterium capable of transforming daidzein to equol and genistein to 5-hydroxy-equol.',19139227),(3254,'human','Genistein 7-O-glucoside','Genistein-7-O-glucoside was partially transformed by way of genistein to the product 2-(4-hydroxyphenyl)-propionic acid.',11959436),(3255,'human','Genistein','Genistein was completely degraded by E. ramulus via 6\'-hydroxy-O-desmethylangolensin to 2-(4-hydroxyphenyl)-propionic acid.',11959436),(3256,'human','Daidzein','Daidzein was in part degraded to O-desmethylangolensin, the corresponding metabolite to 6\'-hydroxy-O-desmethylangolensin.',11959436),(3257,'human','Isoxanthohumol','The intestinal bacterium E. limosum was able to perform this conversion, and after strain selection, the bacterium rapidly converted all IX into 8-PN.',16076107),(3258,'human','null','In reactions using CF faecal inocula, butyrate levels were strongly correlated with the relative abundance of Clostridium sensu stricto cluster 1.',30359203),(3259,'human','null','Propionate concentrations in CF reactions were significantly correlated with the relative abundance of the Veillonella genus.',30359203),(3260,'human','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3261,'human','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3262,'human','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3263,'human','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3264,'human','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3265,'human','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3266,'human','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3267,'human','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3268,'human','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3269,'human','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3270,'human','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3271,'human','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3272,'human','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3273,'human','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3274,'human','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3275,'human','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3276,'human','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3277,'human','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3278,'human','null','Genus Turicibacter and species Bifdobacterium longum,Lactobacillus mucosae, Lactobacillus zeae were positively correlated with 3,4 dihydroxyhydrocinnmaic acid.',31235863),(3279,'human','null','Genus Turicibacter and species Bifdobacterium longum,Lactobacillus mucosae, Lactobacillus zeae were positively correlated with 3,4 dihydroxyhydrocinnmaic acid.',31235863),(3280,'human','null','Genus Turicibacter and species Bifdobacterium longum,Lactobacillus mucosae, Lactobacillus zeae were positively correlated with 3,4 dihydroxyhydrocinnmaic acid.',31235863),(3281,'human','null','Genus Turicibacter and species Bifdobacterium longum,Lactobacillus mucosae, Lactobacillus zeae were positively correlated with 3,4 dihydroxyhydrocinnmaic acid.',31235863),(3282,'human','null','Similarly, a positive correlation of Eubacterium with 3-hydroxybenzoic acid and 4-hydroxybenzoic acid concurs with the previous studies.',31235863),(3283,'human','null','Similarly, a positive correlation of Eubacterium with 3-hydroxybenzoic acid and 4-hydroxybenzoic acid concurs with the previous studies.',31235863),(3284,'human','null','Collinsella aerofaciens showed a positive correlation with sedoheptulose, citric acid, tricarballylic acid.',31235863),(3285,'human','null','Collinsella aerofaciens showed a positive correlation with sedoheptulose, citric acid, tricarballylic acid.',31235863),(3286,'human','null','Collinsella aerofaciens showed a positive correlation with sedoheptulose, citric acid, tricarballylic acid.',31235863),(3287,'human','null','E. coli was positively correlated with pipecolic acid.',31235863),(3288,'human','null','Genus Faecalibacterium showed a negative correlation with L-leucine and serine and species Faecalibacterium prausnitzii showed a negative correlation with serine, L-leucine, and malic acid.',31235863),(3289,'human','null','Genus Faecalibacterium showed a negative correlation with L-leucine and serine and species Faecalibacterium prausnitzii showed a negative correlation with serine, L-leucine, and malic acid.',31235863),(3290,'human','null','Genus Faecalibacterium showed a negative correlation with L-leucine and serine and species Faecalibacterium prausnitzii showed a negative correlation with serine, L-leucine, and malic acid.',31235863),(3291,'human','null','Genus Faecalibacterium showed a negative correlation with L-leucine and serine and species Faecalibacterium prausnitzii showed a negative correlation with serine, L-leucine, and malic acid.',31235863),(3292,'human','null','Genus Faecalibacterium showed a negative correlation with L-leucine and serine and species Faecalibacterium prausnitzii showed a negative correlation with serine, L-leucine, and malic acid.',31235863),(3293,'human','null','Lactobacillus ruminis showed a negative correlation with glycine.',31235863),(3294,'human','null','Genera Dialister, Catenibacterium,Turicibacter showed a positive correlation with tartaric acid and species Ruminococcus bromii, Parabacteroides distasonis, Bacteroides caccae were negatively correlated with tartaric acid.',31235863),(3295,'human','null','Genera Dialister, Catenibacterium,Turicibacter showed a positive correlation with tartaric acid and species Ruminococcus bromii, Parabacteroides distasonis, Bacteroides caccae were negatively correlated with tartaric acid.',31235863),(3296,'human','null','Genera Dialister, Catenibacterium,Turicibacter showed a positive correlation with tartaric acid and species Ruminococcus bromii, Parabacteroides distasonis, Bacteroides caccae were negatively correlated with tartaric acid.',31235863),(3297,'human','null','Genera Dialister, Catenibacterium,Turicibacter showed a positive correlation with tartaric acid and species Ruminococcus bromii, Parabacteroides distasonis, Bacteroides caccae were negatively correlated with tartaric acid.',31235863),(3298,'human','null','Genera Dialister, Catenibacterium,Turicibacter showed a positive correlation with tartaric acid and species Ruminococcus bromii, Parabacteroides distasonis, Bacteroides caccae were negatively correlated with tartaric acid.',31235863),(3299,'human','null','Genera Dialister, Catenibacterium,Turicibacter showed a positive correlation with tartaric acid and species Ruminococcus bromii, Parabacteroides distasonis, Bacteroides caccae were negatively correlated with tartaric acid.',31235863),(3300,'human','null','Genera Akkermansia, species Coprococcus catus were positively correlated with creatine and genera Mitsuokella, Weissella, Lactobacillus, species Mitsuokella multacida, Lactobacillus ruminis were negatively correlated with creatine.',31235863),(3301,'human','null','Genera Akkermansia, species Coprococcus catus were positively correlated with creatine and genera Mitsuokella, Weissella, Lactobacillus, species Mitsuokella multacida, Lactobacillus ruminis were negatively correlated with creatine.',31235863),(3302,'human','null','Genera Akkermansia, species Coprococcus catus were positively correlated with creatine and genera Mitsuokella, Weissella, Lactobacillus, species Mitsuokella multacida, Lactobacillus ruminis were negatively correlated with creatine.',31235863),(3303,'human','null','Genera Akkermansia, species Coprococcus catus were positively correlated with creatine and genera Mitsuokella, Weissella, Lactobacillus, species Mitsuokella multacida, Lactobacillus ruminis were negatively correlated with creatine.',31235863),(3304,'human','null','Genera Akkermansia, species Coprococcus catus were positively correlated with creatine and genera Mitsuokella, Weissella, Lactobacillus, species Mitsuokella multacida, Lactobacillus ruminis were negatively correlated with creatine.',31235863),(3305,'human','null','Genera Akkermansia, species Coprococcus catus were positively correlated with creatine and genera Mitsuokella, Weissella, Lactobacillus, species Mitsuokella multacida, Lactobacillus ruminis were negatively correlated with creatine.',31235863),(3306,'human','null','Genera Akkermansia, species Coprococcus catus were positively correlated with creatine and genera Mitsuokella, Weissella, Lactobacillus, species Mitsuokella multacida, Lactobacillus ruminis were negatively correlated with creatine.',31235863),(3307,'human','null','Genera Faecalibacterium, Succinivibrio, Macrococcus are negatively correlated with creatinine.',31235863),(3308,'human','null','Genera Faecalibacterium, Succinivibrio, Macrococcus are negatively correlated with creatinine.',31235863),(3309,'human','null','Genera Faecalibacterium, Succinivibrio, Macrococcus are negatively correlated with creatinine.',31235863),(3310,'human','null','Genera Bilophila, Enterococcus, Dorea,Clostridium, Phascolarctobacter and species Faecalibacterium prausnitzii, Photobacterium angustum are positively correlated with creatinine.',31235863),(3311,'human','null','Genera Bilophila, Enterococcus, Dorea,Clostridium, Phascolarctobacter and species Faecalibacterium prausnitzii, Photobacterium angustum are positively correlated with creatinine.',31235863),(3312,'human','null','Genera Bilophila, Enterococcus, Dorea,Clostridium, Phascolarctobacter and species Faecalibacterium prausnitzii, Photobacterium angustum are positively correlated with creatinine.',31235863),(3313,'human','null','Genera Bilophila, Enterococcus, Dorea,Clostridium, Phascolarctobacter and species Faecalibacterium prausnitzii, Photobacterium angustum are positively correlated with creatinine.',31235863),(3314,'human','null','Genera Bilophila, Enterococcus, Dorea,Clostridium, Phascolarctobacter and species Faecalibacterium prausnitzii, Photobacterium angustum are positively correlated with creatinine.',31235863),(3315,'human','null','Genera Bilophila, Enterococcus, Dorea,Clostridium, Phascolarctobacter and species Faecalibacterium prausnitzii, Photobacterium angustum are positively correlated with creatinine.',31235863),(3316,'human','null','Genera Escherichia, Paraprevotella, Akkermansia have a positive correlation with glycocholic acid.',31235863),(3317,'human','null','Genera Escherichia, Paraprevotella, Akkermansia have a positive correlation with glycocholic acid.',31235863),(3318,'human','null','Genera Escherichia, Paraprevotella, Akkermansia have a positive correlation with glycocholic acid.',31235863),(3319,'human','null','Genus Carnobacterium and species Ruminococcus favefaciens, Butyricicoccus pullicaecorum,Mitsuokella multacida, Bacteroides fragilis have a negative correlation with glycocholic acid.',31235863),(3320,'human','null','Genus Carnobacterium and species Ruminococcus favefaciens, Butyricicoccus pullicaecorum,Mitsuokella multacida, Bacteroides fragilis have a negative correlation with glycocholic acid.',31235863),(3321,'human','null','Genus Carnobacterium and species Ruminococcus favefaciens, Butyricicoccus pullicaecorum,Mitsuokella multacida, Bacteroides fragilis have a negative correlation with glycocholic acid.',31235863),(3322,'human','null','Genus Carnobacterium and species Ruminococcus favefaciens, Butyricicoccus pullicaecorum,Mitsuokella multacida, Bacteroides fragilis have a negative correlation with glycocholic acid.',31235863),(3323,'human','null','Genus Carnobacterium and species Ruminococcus favefaciens, Butyricicoccus pullicaecorum,Mitsuokella multacida, Bacteroides fragilis have a negative correlation with glycocholic acid.',31235863),(3324,'human','null','Genera Enterococcus,Dermacoccus, Kocuria, Roseburia and species Eggerthella lenta, Bacteroides fragilis showed a negative correlation with 3-phenylpropionic acid.',31235863),(3325,'human','null','Genera Enterococcus,Dermacoccus, Kocuria, Roseburia and species Eggerthella lenta, Bacteroides fragilis showed a negative correlation with 3-phenylpropionic acid.',31235863),(3326,'human','null','Genera Enterococcus,Dermacoccus, Kocuria, Roseburia and species Eggerthella lenta, Bacteroides fragilis showed a negative correlation with 3-phenylpropionic acid.',31235863),(3327,'human','null','Genera Enterococcus,Dermacoccus, Kocuria, Roseburia and species Eggerthella lenta, Bacteroides fragilis showed a negative correlation with 3-phenylpropionic acid.',31235863),(3328,'human','null','Genera Enterococcus,Dermacoccus, Kocuria, Roseburia and species Eggerthella lenta, Bacteroides fragilis showed a negative correlation with 3-phenylpropionic acid.',31235863),(3329,'human','null','Genera Enterococcus,Dermacoccus, Kocuria, Roseburia and species Eggerthella lenta, Bacteroides fragilis showed a negative correlation with 3-phenylpropionic acid.',31235863),(3330,'human','null','Genus Oscillospira is positively correlated with citric acid.',31235863),(3331,'human','null','Genera Acinetobacter, Microbacterium, Dysgonomonas, Bulleidia, Oscillospira and species Bacteroides caccae, Kocuria rhizophila, Pseudomonas fragi, Ruminococcus bromii were positively correlated with 4-pyridoxic acid.',31235863),(3332,'human','null','Genera Acinetobacter, Microbacterium, Dysgonomonas, Bulleidia, Oscillospira and species Bacteroides caccae, Kocuria rhizophila, Pseudomonas fragi, Ruminococcus bromii were positively correlated with 4-pyridoxic acid.',31235863),(3333,'human','null','Genera Acinetobacter, Microbacterium, Dysgonomonas, Bulleidia, Oscillospira and species Bacteroides caccae, Kocuria rhizophila, Pseudomonas fragi, Ruminococcus bromii were positively correlated with 4-pyridoxic acid.',31235863),(3334,'human','null','Genera Acinetobacter, Microbacterium, Dysgonomonas, Bulleidia, Oscillospira and species Bacteroides caccae, Kocuria rhizophila, Pseudomonas fragi, Ruminococcus bromii were positively correlated with 4-pyridoxic acid.',31235863),(3335,'human','null','Genera Acinetobacter, Microbacterium, Dysgonomonas, Bulleidia, Oscillospira and species Bacteroides caccae, Kocuria rhizophila, Pseudomonas fragi, Ruminococcus bromii were positively correlated with 4-pyridoxic acid.',31235863),(3336,'human','null','Genera Acinetobacter, Microbacterium, Dysgonomonas, Bulleidia, Oscillospira and species Bacteroides caccae, Kocuria rhizophila, Pseudomonas fragi, Ruminococcus bromii were positively correlated with 4-pyridoxic acid.',31235863),(3337,'human','null','Genera Acinetobacter, Microbacterium, Dysgonomonas, Bulleidia, Oscillospira and species Bacteroides caccae, Kocuria rhizophila, Pseudomonas fragi, Ruminococcus bromii were positively correlated with 4-pyridoxic acid.',31235863),(3338,'human','null','Genera Acinetobacter, Microbacterium, Dysgonomonas, Bulleidia, Oscillospira and species Bacteroides caccae, Kocuria rhizophila, Pseudomonas fragi, Ruminococcus bromii were positively correlated with 4-pyridoxic acid.',31235863),(3339,'human','null','Genera Acinetobacter, Microbacterium, Dysgonomonas, Bulleidia, Oscillospira and species Bacteroides caccae, Kocuria rhizophila, Pseudomonas fragi, Ruminococcus bromii were positively correlated with 4-pyridoxic acid.',31235863),(3340,'human','null','We cultured fecal specimens from healthy young adults in media containing defatted flaxseeds and detected END from the culture supernatant. Following selection through successive subcultures of the fecal microbiota with defatted flaxseeds as the only carbon source, we obtained a bacterial consortium, designated as END-49, which contained the smallest number of bacterial types still capable of metabolizing defatted flaxseeds to produce END.',20398397),(3341,'human','null','We provide direct evidence that IS activates EGFR and downstream MAPK signaling, which stimulates AhR−ARNT nuclear translocation and results in enhanced OAT1 expression and function in the kidney.',31341083),(3342,'human','null','We provide direct evidence that IS activates EGFR and downstream MAPK signaling, which stimulates AhR−ARNT nuclear translocation and results in enhanced OAT1 expression and function in the kidney.',31341083),(3343,'human','Choline','Interestingly, we found that plasma TMAO was positively correlated with serum sCD14 and sCD163 levels which are biomarkers of monocyte activation and macrophage inflammation and have been reported to be associated with carotid artery atherosclerosis and CVD risk in HIV infected individuals.',29912352),(3344,'human','Choline','Interestingly, we found that plasma TMAO was positively correlated with serum sCD14 and sCD163 levels which are biomarkers of monocyte activation and macrophage inflammation and have been reported to be associated with carotid artery atherosclerosis and CVD risk in HIV infected individuals.',29912352),(3345,'human','null','For the PBMC assay, although all the strains tend to increase the production of IL-10 cytokine, only four strains were able to induce statistically significant increase production of this cytokine .The two most performing strains (A2-165 and 4543) belong to the phylogroup II, group B.',28713353),(3346,'human','null','For the PBMC assay, although all the strains tend to increase the production of IL-10 cytokine, only four strains were able to induce statistically significant increase production of this cytokine .The two most performing strains (A2-165 and 4543) belong to the phylogroup II, group B.',28713353),(3347,'human','null','For the PBMC assay, although all the strains tend to increase the production of IL-10 cytokine, only four strains were able to induce statistically significant increase production of this cytokine .The two most performing strains (A2-165 and 4543) belong to the phylogroup II, group B.',28713353),(3348,'human','null','For the PBMC assay, although all the strains tend to increase the production of IL-10 cytokine, only four strains were able to induce statistically significant increase production of this cytokine .The two most performing strains (A2-165 and 4543) belong to the phylogroup II, group B.',28713353),(3358,'human','Choline','Further, in vitro conversion of choline to TMA was checked in presence/absence of trigonelline and in presence of trigonelline there was significant inhibition of growth of C. freundii and hence significantly reduced levels of TMA was estimated.',29980011),(3359,'human','Trimethylamine','Butin presence oftrigonelline, significantly reduced levels of TMAO was estimated which was, of course, due to flavin monooxygenase (FMO3) enzyme inhibition by trigonelline.',29980011),(3360,'human','Quercetin','The degradation of the flavonol quercetin and the flavone luteolin by Eubacterium ramulus, a strict anaerobe of the human intestinal tract, was studied. Resting cells converted these flavonoids to 3,4-dihydroxyphenylacetic acid and 3-(3,4-dihydroxyphenyl)propionic acid, respectively.',11722907),(3361,'human','Luteolin','The degradation of the flavonol quercetin and the flavone luteolin by Eubacterium ramulus, a strict anaerobe of the human intestinal tract, was studied. Resting cells converted these flavonoids to 3,4-dihydroxyphenylacetic acid and 3-(3,4-dihydroxyphenyl)propionic acid, respectively.',11722907),(3362,'human','Hesperidin','The data herein presented demonstrated that B. pseudocatenulatum can hydrolyze certain rutinose-conjugated polyphenols, such as hesperidin, thus releasing the aglycone form. Therefore, B. pseudocatenulatum may potentially contribute to the bioavailability of this class of molecules.',25875120),(3363,'human','Hesperidin','The data herein presented demonstrated that B. pseudocatenulatum can hydrolyze certain rutinose-conjugated polyphenols, such as hesperidin, thus releasing the aglycone form. Therefore, B. pseudocatenulatum may potentially contribute to the bioavailability of this class of molecules.',25875120),(3364,'human','Hesperidin','The data herein presented demonstrated that B. pseudocatenulatum can hydrolyze certain rutinose-conjugated polyphenols, such as hesperidin, thus releasing the aglycone form. Therefore, B. pseudocatenulatum may potentially contribute to the bioavailability of this class of molecules.',25875120),(3365,'human','Hesperidin','The data herein presented demonstrated that B. pseudocatenulatum can hydrolyze certain rutinose-conjugated polyphenols, such as hesperidin, thus releasing the aglycone form. Therefore, B. pseudocatenulatum may potentially contribute to the bioavailability of this class of molecules.',25875120),(3366,'human','Hesperidin','The data herein presented demonstrated that B. pseudocatenulatum can hydrolyze certain rutinose-conjugated polyphenols, such as hesperidin, thus releasing the aglycone form. Therefore, B. pseudocatenulatum may potentially contribute to the bioavailability of this class of molecules.',25875120),(3367,'human','null','In the present study, we focused on the impact of SCFAs and the role of PPAR-γin intestinal ANGPTL4 expression. Here, we show that ANGPTL4 expression can be regulated by SCFAs both in vitro and in vivo, with butyrate being the most efficient activator.',23518684),(3368,'human','null','Faecalibacterium prausnitzii is one of the most abundant bacteria in the human gut ecosystem and it is an important supplier of butyrate to the colonic epithelium.',22357539),(3369,'human','Cianidanol','When (+)-catechin was anaerobically incubated with Eg. sp. CAT-1, and the incubation mixture was analyzed by LC/MS, two metabolites, designated CAM1 and CAM2, were detected.',23207778),(3370,'human','Cianidanol','When (+)-catechin was anaerobically incubated with Eg. sp. CAT-1, and the incubation mixture was analyzed by LC/MS, two metabolites, designated CAM1 and CAM2, were detected.',23207778),(3371,'human','null','On the other hand, we confirmed in both ex vivo epithelial cultures and whole mucosal samples that the inflammatory milieu, in particular the presence of TNFɑ, does impact butyrate uptake and oxidation. Indeed, TNFɑ may be responsible for the downregulation of butyrate transporters in the human intestinal epithelium observed in patient biopsies, and that may result in decreased butyrate consumption.',31211831),(3372,'human','Exopolysaccharides','In conclusion, this study revealed that relevant changes in the populations of some major anaerobic intestinal bacterial groups as well as noticeable shifts in SCFA and lactic acid production can result from the presence of bifidobacterial EPS. Microbial population dynamics correlated well and supported the SCFA and lactic acid variations detected. EPS could reach the colon by two different ways: they could be either synthesized by probiotics in food products or by intestinal bacteria in the intestinal environment, thus contributing to the modulation of the intestinal microbiota.',19735956),(3373,'human','null','These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.',18820905),(3374,'human','null','These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.',18820905),(3375,'human','null','These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.',18820905),(3376,'human','null','These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.',18820905),(3377,'human','null','These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.',18820905),(3378,'human','null','These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.',18820905),(3379,'human','null','These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.',18820905),(3380,'human','null','These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.',18820905),(3381,'human','null','These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.',18820905),(3382,'human','null','These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.',18820905),(3383,'human','null','These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.',18820905),(3384,'human','null','These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.',18820905),(3385,'human','null','We also show that E. lenta further metabolizes the dopamine produced by L-dopa decarboxylation using a distinctly microbial reaction, catechol dehydroxylation.',31196984),(3386,'human','null','Figure 3 shows the metabolites produced by pure cultures of two propionate-producing LUB strains (P-LUB; V. ratti and P.avidum), two butyrate-producing LUB strains (B-LUB; E. hallii and E. limosum); and one LUB SRB strain (D. piger) in YCFA  medium supplemented with lactose, DL-lactate, L-lactate or glucose as a sole carbon source. H2 production and  OD measurements of these pure cultures are shown in Supplementary Figs S3 and S4, respectively.',28894218),(3387,'human','null','Figure 3 shows the metabolites produced by pure cultures of two propionate-producing LUB strains (P-LUB; V. ratti and P.avidum), two butyrate-producing LUB strains (B-LUB; E. hallii and E. limosum); and one LUB SRB strain (D. piger) in YCFA  medium supplemented with lactose, DL-lactate, L-lactate or glucose as a sole carbon source. H2 production and  OD measurements of these pure cultures are shown in Supplementary Figs S3 and S4, respectively.',28894218),(3388,'human','null','Figure 3 shows the metabolites produced by pure cultures of two propionate-producing LUB strains (P-LUB; V. ratti and P.avidum), two butyrate-producing LUB strains (B-LUB; E. hallii and E. limosum); and one LUB SRB strain (D. piger) in YCFA  medium supplemented with lactose, DL-lactate, L-lactate or glucose as a sole carbon source. H2 production and  OD measurements of these pure cultures are shown in Supplementary Figs S3 and S4, respectively.',28894218),(3389,'human','null','Figure 3 shows the metabolites produced by pure cultures of two propionate-producing LUB strains (P-LUB; V. ratti and P.avidum), two butyrate-producing LUB strains (B-LUB; E. hallii and E. limosum); and one LUB SRB strain (D. piger) in YCFA  medium supplemented with lactose, DL-lactate, L-lactate or glucose as a sole carbon source. H2 production and  OD measurements of these pure cultures are shown in Supplementary Figs S3 and S4, respectively.',28894218),(3390,'human','null','Figure 3 shows the metabolites produced by pure cultures of two propionate-producing LUB strains (P-LUB; V. ratti and P.avidum), two butyrate-producing LUB strains (B-LUB; E. hallii and E. limosum); and one LUB SRB strain (D. piger) in YCFA  medium supplemented with lactose, DL-lactate, L-lactate or glucose as a sole carbon source. H2 production and  OD measurements of these pure cultures are shown in Supplementary Figs S3 and S4, respectively.',28894218),(3391,'human','null','Streptococcus sp. MRG-ICA-B isolated from human intestine were reacted with icariin under anaerobic conditions. The MRG-ICA-B and E strains hydrolyzed only the glucose moiety of icariin, and icariside II was the only metabolite.',27589718),(3392,'human','null','Enterococcus sp. MRG-ICA-E isolated from human intestine were reacted with icariin under anaerobic conditions. The MRG-ICA-B and E strains hydrolyzed only the glucose moiety of icariin, and icariside II was the only metabolite.',27589718),(3393,'human','null','Blautia sp. MRG-PMF-1 isolated from human intestine were reacted with icariin under anaerobic conditions. It not only produced icariside II  but strain metabolized icariin further to desmethylicaritin via icariside II and icaritin.',27589718),(3394,'human','null','Blautia sp. MRG-PMF-1 isolated from human intestine were reacted with icariin under anaerobic conditions. It not only produced icariside II  but strain metabolized icariin further to desmethylicaritin via icariside II and icaritin.',27589718),(3395,'human','Puerarin','Puerarin was completely converted to daidzein within 100 min by Lactococcus sp. MRG-IFC-1 and Enterococcus sp. MRG-IFC-2, which is the fastest conversion among the reported human intestinal bacteria.',25199800),(3396,'human','Puerarin','Puerarin was completely converted to daidzein within 100 min by Lactococcus sp. MRG-IFC-1 and Enterococcus sp. MRG-IFC-2, which is the fastest conversion among the reported human intestinal bacteria.',25199800),(3397,'human','Isoquercitrin','From human feces two phenotypically different types of bacteria were isolated on quercetin-3-gluco_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x0002_side as carbon and energy source. Isolates of one type were identified as strains of Enterococcus casseliflavus. The sugar moiety (4 mM) was fermented to 5.5 ± 2.1 mM formate, 2.1 ± 0.7 mM acetate,1.6 ± 0.3 mM L-lactate, and 1.3 ± 0.4 mM ethanol. The second type of isolate was identified as Eubacterium ramulus. This organism was capable of degrading the aromatic ring system. Growing cultures of Eubacterium ramulus converted 5 mM quercetin-3-glucoside to 1.7 ± 0.6 mM 3,4-dihydroxyphenylacetic acid, 7.6 ± 1.0 mM acetate, and 4.0 ± 0.4 mM butyrate.',9914304),(3398,'human','Isoquercitrin','From human feces two phenotypically different types of bacteria were isolated on quercetin-3-gluco_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x0002_side as carbon and energy source. Isolates of one type were identified as strains of Enterococcus casseliflavus. The sugar moiety (4 mM) was fermented to 5.5 ± 2.1 mM formate, 2.1 ± 0.7 mM acetate,1.6 ± 0.3 mM L-lactate, and 1.3 ± 0.4 mM ethanol. The second type of isolate was identified as Eubacterium ramulus. This organism was capable of degrading the aromatic ring system. Growing cultures of Eubacterium ramulus converted 5 mM quercetin-3-glucoside to 1.7 ± 0.6 mM 3,4-dihydroxyphenylacetic acid, 7.6 ± 1.0 mM acetate, and 4.0 ± 0.4 mM butyrate.',9914304),(3399,'human','Isoquercitrin','From human feces two phenotypically different types of bacteria were isolated on quercetin-3-gluco_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x0002_side as carbon and energy source. Isolates of one type were identified as strains of Enterococcus casseliflavus. The sugar moiety (4 mM) was fermented to 5.5 ± 2.1 mM formate, 2.1 ± 0.7 mM acetate,1.6 ± 0.3 mM L-lactate, and 1.3 ± 0.4 mM ethanol. The second type of isolate was identified as Eubacterium ramulus. This organism was capable of degrading the aromatic ring system. Growing cultures of Eubacterium ramulus converted 5 mM quercetin-3-glucoside to 1.7 ± 0.6 mM 3,4-dihydroxyphenylacetic acid, 7.6 ± 1.0 mM acetate, and 4.0 ± 0.4 mM butyrate.',9914304),(3400,'human','Isoquercitrin','From human feces two phenotypically different types of bacteria were isolated on quercetin-3-gluco_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x0002_side as carbon and energy source. Isolates of one type were identified as strains of Enterococcus casseliflavus. The sugar moiety (4 mM) was fermented to 5.5 ± 2.1 mM formate, 2.1 ± 0.7 mM acetate,1.6 ± 0.3 mM L-lactate, and 1.3 ± 0.4 mM ethanol. The second type of isolate was identified as Eubacterium ramulus. This organism was capable of degrading the aromatic ring system. Growing cultures of Eubacterium ramulus converted 5 mM quercetin-3-glucoside to 1.7 ± 0.6 mM 3,4-dihydroxyphenylacetic acid, 7.6 ± 1.0 mM acetate, and 4.0 ± 0.4 mM butyrate.',9914304),(3401,'human','Isoquercitrin','From human feces two phenotypically different types of bacteria were isolated on quercetin-3-gluco_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x0002_side as carbon and energy source. Isolates of one type were identified as strains of Enterococcus casseliflavus. The sugar moiety (4 mM) was fermented to 5.5 ± 2.1 mM formate, 2.1 ± 0.7 mM acetate,1.6 ± 0.3 mM L-lactate, and 1.3 ± 0.4 mM ethanol. The second type of isolate was identified as Eubacterium ramulus. This organism was capable of degrading the aromatic ring system. Growing cultures of Eubacterium ramulus converted 5 mM quercetin-3-glucoside to 1.7 ± 0.6 mM 3,4-dihydroxyphenylacetic acid, 7.6 ± 1.0 mM acetate, and 4.0 ± 0.4 mM butyrate.',9914304),(3402,'human','Isoquercitrin','From human feces two phenotypically different types of bacteria were isolated on quercetin-3-gluco_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x0002_side as carbon and energy source. Isolates of one type were identified as strains of Enterococcus casseliflavus. The sugar moiety (4 mM) was fermented to 5.5 ± 2.1 mM formate, 2.1 ± 0.7 mM acetate,1.6 ± 0.3 mM L-lactate, and 1.3 ± 0.4 mM ethanol. The second type of isolate was identified as Eubacterium ramulus. This organism was capable of degrading the aromatic ring system. Growing cultures of Eubacterium ramulus converted 5 mM quercetin-3-glucoside to 1.7 ± 0.6 mM 3,4-dihydroxyphenylacetic acid, 7.6 ± 1.0 mM acetate, and 4.0 ± 0.4 mM butyrate.',9914304),(3403,'human','Isoquercitrin','From human feces two phenotypically different types of bacteria were isolated on quercetin-3-gluco_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x0002_side as carbon and energy source. Isolates of one type were identified as strains of Enterococcus casseliflavus. The sugar moiety (4 mM) was fermented to 5.5 ± 2.1 mM formate, 2.1 ± 0.7 mM acetate,1.6 ± 0.3 mM L-lactate, and 1.3 ± 0.4 mM ethanol. The second type of isolate was identified as Eubacterium ramulus. This organism was capable of degrading the aromatic ring system. Growing cultures of Eubacterium ramulus converted 5 mM quercetin-3-glucoside to 1.7 ± 0.6 mM 3,4-dihydroxyphenylacetic acid, 7.6 ± 1.0 mM acetate, and 4.0 ± 0.4 mM butyrate.',9914304),(3404,'human','null','Finally, a total of seven metabolites were detected and tentatively identified, such as the demethylation metabolite (M1), deoxidation metabolites (M3, M7) and hydrolysis at the C-28 oligosaccharide metabolites (M5,M6), which were first discovered in this experiment. The results indicate that hydrolysis, demethylation, dehydroxylation, and acetylation were the major metabolic pathways of PDin vitro. Additionally, four bacterial strains from human feces including Enterococcus sp.41, Bacillus sp.46, Escherichia sp.49A and Escherichia sp.64 were detected and further identified with 16S rRNA gene sequencing due to their relatively strong metabolic capacity',27222071),(3405,'human','null','Finally, a total of seven metabolites were detected and tentatively identified, such as the demethylation metabolite (M1), deoxidation metabolites (M3, M7) and hydrolysis at the C-28 oligosaccharide metabolites (M5,M6), which were first discovered in this experiment. The results indicate that hydrolysis, demethylation, dehydroxylation, and acetylation were the major metabolic pathways of PDin vitro. Additionally, four bacterial strains from human feces including Enterococcus sp.41, Bacillus sp.46, Escherichia sp.49A and Escherichia sp.64 were detected and further identified with 16S rRNA gene sequencing due to their relatively strong metabolic capacity',27222071),(3406,'human','null','Finally, a total of seven metabolites were detected and tentatively identified, such as the demethylation metabolite (M1), deoxidation metabolites (M3, M7) and hydrolysis at the C-28 oligosaccharide metabolites (M5,M6), which were first discovered in this experiment. The results indicate that hydrolysis, demethylation, dehydroxylation, and acetylation were the major metabolic pathways of PDin vitro. Additionally, four bacterial strains from human feces including Enterococcus sp.41, Bacillus sp.46, Escherichia sp.49A and Escherichia sp.64 were detected and further identified with 16S rRNA gene sequencing due to their relatively strong metabolic capacity',27222071),(3407,'human','null','Finally, a total of seven metabolites were detected and tentatively identified, such as the demethylation metabolite (M1), deoxidation metabolites (M3, M7) and hydrolysis at the C-28 oligosaccharide metabolites (M5,M6), which were first discovered in this experiment. The results indicate that hydrolysis, demethylation, dehydroxylation, and acetylation were the major metabolic pathways of PDin vitro. Additionally, four bacterial strains from human feces including Enterococcus sp.41, Bacillus sp.46, Escherichia sp.49A and Escherichia sp.64 were detected and further identified with 16S rRNA gene sequencing due to their relatively strong metabolic capacity',27222071),(3408,'human','Daidzein','We isolated nine strains capable of metabolizing isoflavones to equol from human faeces. Seven of these strains could metabolize daidzein via dihydrodaidzein to equol, while the other two isolates could metabolize only dihydrodaidzein to equol.',18450717),(3409,'human','Daidzein','We isolated nine strains capable of metabolizing isoflavones to equol from human faeces. Seven of these strains could metabolize daidzein via dihydrodaidzein to equol, while the other two isolates could metabolize only dihydrodaidzein to equol.',18450717),(3410,'human','Daidzein','We isolated nine strains capable of metabolizing isoflavones to equol from human faeces. Seven of these strains could metabolize daidzein via dihydrodaidzein to equol, while the other two isolates could metabolize only dihydrodaidzein to equol.',18450717),(3411,'human','Daidzein','We isolated nine strains capable of metabolizing isoflavones to equol from human faeces. Seven of these strains could metabolize daidzein via dihydrodaidzein to equol, while the other two isolates could metabolize only dihydrodaidzein to equol.',18450717),(3412,'human','Daidzein','We isolated nine strains capable of metabolizing isoflavones to equol from human faeces. Seven of these strains could metabolize daidzein via dihydrodaidzein to equol, while the other two isolates could metabolize only dihydrodaidzein to equol.',18450717),(3413,'human','Daidzein','We isolated nine strains capable of metabolizing isoflavones to equol from human faeces. Seven of these strains could metabolize daidzein via dihydrodaidzein to equol, while the other two isolates could metabolize only dihydrodaidzein to equol.',18450717),(3414,'human','Daidzein','We isolated nine strains capable of metabolizing isoflavones to equol from human faeces. Seven of these strains could metabolize daidzein via dihydrodaidzein to equol, while the other two isolates could metabolize only dihydrodaidzein to equol.',18450717),(3415,'human','Dihydrodaidzein','We isolated nine strains capable of metabolizing isoflavones to equol from human faeces. Seven of these strains could metabolize daidzein via dihydrodaidzein to equol, while the other two isolates could metabolize only dihydrodaidzein to equol.',18450717),(3416,'human','Dihydrodaidzein','We isolated nine strains capable of metabolizing isoflavones to equol from human faeces. Seven of these strains could metabolize daidzein via dihydrodaidzein to equol, while the other two isolates could metabolize only dihydrodaidzein to equol.',18450717),(3425,'human','D-Glucose','Acetate is readily absorbed from the rumen and subsequently used in fatty acid metabolism.',8481001),(3426,'human','D-Glucose','Succinate and formate, on the other hand, are considered intermediates of rumen fermentation which are further metabolized by other organisms. Succinate is thought to be an important intermediate for propionate formation , with the latter being readily absorbed from the rumen for gluconeogenesis.',8481001),(3427,'human','D-Glucose','Succinate and formate, on the other hand, are considered intermediates of rumen fermentation which are further metabolized by other organisms. Formate has been shown to be metabolized to methane by rumen methanogens.',8481001),(3428,'human','D-Glucose','Lactate is also an intermediate of the overall rumen fermentation and, when present in excess, has been correlated with acidosis.',8481001),(3429,'human','D-Glucose','Lactic acid bacteria, because the bifidobacterial pathway yields 2.5 ATP molecules from 1 mol of fermented glucose, as well as 1.5 mol of acetate and 1 mol of lactate [94]. In contrast, the homofermenta tive group of lactic acid bacteria (LAB) produces 2 mol of ATP and 2 mol of lactic acid from 1 mol of glucose, whereas heterofermentative LAB produce 1 mol each of lactic acid, ethanol and ATP per 1 mol of fermented glu cose.',21484167),(3430,'human','D-Glucose','Lactic acid bacteria, because the bifidobacterial pathway yields 2.5 ATP molecules from 1 mol of fermented glucose, as well as 1.5 mol of acetate and 1 mol of lactate [94]. In contrast, the homofermenta tive group of lactic acid bacteria (LAB) produces 2 mol of ATP and 2 mol of lactic acid from 1 mol of glucose, whereas heterofermentative LAB produce 1 mol each of lactic acid, ethanol and ATP per 1 mol of fermented glu cose.',21484167),(3431,'human','D-Glucose','Lactic acid bacteria, because the bifidobacterial pathway yields 2.5 ATP molecules from 1 mol of fermented glucose, as well as 1.5 mol of acetate and 1 mol of lactate [94]. In contrast, the homofermenta tive group of lactic acid bacteria (LAB) produces 2 mol of ATP and 2 mol of lactic acid from 1 mol of glucose, whereas heterofermentative LAB produce 1 mol each of lactic acid, ethanol and ATP per 1 mol of fermented glu cose.',21484167),(3432,'human','null','Due to the fact that bifidobacteria produce lactic acid as one of their main fermentation end products, they are often included in the lactic acid bacteria (LAB).',21484167),(3433,'human','5,7-Dimethoxyflavone','Activity-guided screening for PMF-metabolizing intestinal bacteria under anaerobic conditions resulted in the isolation of a strict anaerobic bacterium, which was identified as Blautia sp. MRG-PMF1. The isolated MRGPMF1 was able to metabolize various PMFs to the corresponding demethylated flavones.The microbial conversion of bioactive 5,7-dimethoxyflavone (5,7-DMF) and 5,7,4�?trimethoxyflavone (5,7,4�?TMF) was studied in detail. 5,7-DMF and 5,7,4�?TMF were completely metabolized to 5,7-dihydroxyflavone (chrysin) and 5,7,4�?trihydroxyflavone (apigenin), respectively.',25437273),(3434,'human','4\',5,7-Trimethoxyflavone','Activity-guided screening for PMF-metabolizing intestinal bacteria under anaerobic conditions resulted in the isolation of a strict anaerobic bacterium, which was identified as Blautia sp. MRG-PMF1. The isolated MRGPMF1 was able to metabolize various PMFs to the corresponding demethylated flavones.The microbial conversion of bioactive 5,7-dimethoxyflavone (5,7-DMF) and 5,7,4�?trimethoxyflavone (5,7,4�?TMF) was studied in detail. 5,7-DMF and 5,7,4�?TMF were completely metabolized to 5,7-dihydroxyflavone (chrysin) and 5,7,4�?trihydroxyflavone (apigenin), respectively.',25437273),(3435,'human','Ellagic Acid','Four gut bacteria strains (CEBAS 4A1, 4A2, 4A3, 4A4)isolated from a healthy donor and able to produce urolithins M6, C and isourolithin A, are described in the present study.Strains were found at high concentrations (�?07 cfu g�?feces).Comparison of the 16S rRNA gene sequences of the strains showed that the four isolates belong to the same species and that they are phylogenetically members of the family Eggerthellaceae.',28824607),(3436,'human','Ellagic Acid','Four gut bacteria strains (CEBAS 4A1, 4A2, 4A3, 4A4)isolated from a healthy donor and able to produce urolithins M6, C and isourolithin A, are described in the present study.Strains were found at high concentrations (�?07 cfu g�?feces).Comparison of the 16S rRNA gene sequences of the strains showed that the four isolates belong to the same species and that they are phylogenetically members of the family Eggerthellaceae.',28824607),(3437,'human','Ellagic Acid','Four gut bacteria strains (CEBAS 4A1, 4A2, 4A3, 4A4)isolated from a healthy donor and able to produce urolithins M6, C and isourolithin A, are described in the present study.Strains were found at high concentrations (�?07 cfu g�?feces).Comparison of the 16S rRNA gene sequences of the strains showed that the four isolates belong to the same species and that they are phylogenetically members of the family Eggerthellaceae.',28824607),(3438,'human','Ellagic Acid','Four gut bacteria strains (CEBAS 4A1, 4A2, 4A3, 4A4)isolated from a healthy donor and able to produce urolithins M6, C and isourolithin A, are described in the present study.Strains were found at high concentrations (�?07 cfu g�?feces).Comparison of the 16S rRNA gene sequences of the strains showed that the four isolates belong to the same species and that they are phylogenetically members of the family Eggerthellaceae.',28824607),(3439,'human','null','Recently, we isolated a dihydrodaidzein (DHD)-producing Clostridium-like bacterium, strain TM-40, from human feces.',25045313),(3440,'human','DL-Alanine','For example, if R1 is a methyl group (-CH3), then the donor amino acid is an alanine, and the product of the oxidative deamination would be pyruvate, a central compounds of the intermediary metabolism.',22325238),(3441,'human','Glycine','For example when R2 is a hydrogen (thus the acceptor amino acid is a glycine) this product corresponds to acetyl-phosphate.',22325238),(3442,'human','null','Having detected (ex vivo from mouse cecal contents and human stool) and identified 3-methyl-2-oxindole and 5-hydroxyindole-3-acetic acid, 3-indoleacrylic acid and indole-3 carboxylic acid as novel in vitro AHR activators.',32783770),(3443,'human','null','Having detected (ex vivo from mouse cecal contents and human stool) and identified 3-methyl-2-oxindole and 5-hydroxyindole-3-acetic acid, 3-indoleacrylic acid and indole-3 carboxylic acid as novel in vitro AHR activators.',32783770),(3444,'human','null','Having detected (ex vivo from mouse cecal contents and human stool) and identified 3-methyl-2-oxindole and 5-hydroxyindole-3-acetic acid, 3-indoleacrylic acid and indole-3 carboxylic acid as novel in vitro AHR activators.',32783770),(3445,'human','null','Having detected (ex vivo from mouse cecal contents and human stool) and identified 3-methyl-2-oxindole and 5-hydroxyindole-3-acetic acid, 3-indoleacrylic acid and indole-3 carboxylic acid as novel in vitro AHR activators.',32783770),(3446,'human','null','And yet, in Caco2 cells, indole still elicited a significant saturation of CYP1A1 expression (~15-fold, p < .001), while the remaining tryptophan metabolites elicited nonsaturable yet variable levels of CYP1A1 induction. 2-oxindole, indole-3-acetic acid, 3-methyl indole and kynurenic acid prompted significant CYP1A1 inducibility. The remaining tryptophan metabolites, however, failed to elicit significant induction.',32783770),(3447,'human','null','And yet, in Caco2 cells, indole still elicited a significant saturation of CYP1A1 expression (~15-fold, p < .001), while the remaining tryptophan metabolites elicited nonsaturable yet variable levels of CYP1A1 induction. 2-oxindole, indole-3-acetic acid, 3-methyl indole and kynurenic acid prompted significant CYP1A1 inducibility. The remaining tryptophan metabolites, however, failed to elicit significant induction.',32783770),(3448,'human','null','And yet, in Caco2 cells, indole still elicited a significant saturation of CYP1A1 expression (~15-fold, p < .001), while the remaining tryptophan metabolites elicited nonsaturable yet variable levels of CYP1A1 induction. 2-oxindole, indole-3-acetic acid, 3-methyl indole and kynurenic acid prompted significant CYP1A1 inducibility. The remaining tryptophan metabolites, however, failed to elicit significant induction.',32783770),(3449,'human','null','And yet, in Caco2 cells, indole still elicited a significant saturation of CYP1A1 expression (~15-fold, p < .001), while the remaining tryptophan metabolites elicited nonsaturable yet variable levels of CYP1A1 induction. 2-oxindole, indole-3-acetic acid, 3-methyl indole and kynurenic acid prompted significant CYP1A1 inducibility. The remaining tryptophan metabolites, however, failed to elicit significant induction.',32783770),(3450,'human','null','And yet, in Caco2 cells, indole still elicited a significant saturation of CYP1A1 expression (~15-fold, p < .001), while the remaining tryptophan metabolites elicited nonsaturable yet variable levels of CYP1A1 induction. 2-oxindole, indole-3-acetic acid, 3-methyl indole and kynurenic acid prompted significant CYP1A1 inducibility. The remaining tryptophan metabolites, however, failed to elicit significant induction.',32783770),(3451,'human','null','These data show that B dentium–secreted products, including acetate, can modulate 5-HT production by host enterochromaffin cells.',32795610),(3452,'human','null','Gene expression of LGR5 was increased more than 2-fold in human duodenal enteroids treated with 10 mM ACET',32240190),(3453,'human','null','Conversely, enteroids treated with 10 mM BUT and PROP had a greater than 10-fold decrease in LGR5 compared to controls (CON; P < 0.05; Fig 4b). Only 1mM treatment of BUT was able to significantly decrease LGR5 expression (P < 0.05; Fig 4a).',32240190),(3454,'human','null','Conversely, enteroids treated with 10 mM BUT and PROP had a greater than 10-fold decrease in LGR5 compared to controls (CON; P < 0.05; Fig 4b). Only 1mM treatment of BUT was able to significantly decrease LGR5 expression (P < 0.05; Fig 4a).',32240190),(3455,'human','null','Treatment with 10 mM SCFA significantly increased ALPI expression treatments but was highest (11-fold) in enteroids treated with PROP. There was no difference in ALPI expression in enteroids treated with 1 mM SCFAs (P > 0.05).',32240190),(3456,'human','null','Treatment with 10 mM SCFA significantly increased ALPI expression treatments but was highest (11-fold) in enteroids treated with PROP. There was no difference in ALPI expression in enteroids treated with 1 mM SCFAs (P > 0.05).',32240190),(3457,'human','null','Treatment with 10 mM SCFA significantly increased ALPI expression treatments but was highest (11-fold) in enteroids treated with PROP. There was no difference in ALPI expression in enteroids treated with 1 mM SCFAs (P > 0.05).',32240190),(3458,'human','null','Gene expression of MUC2 was increased 2-fold due to ACET treatment and 5-fold due to BUT treatment (P < 0.05; Fig 4b) but was not increased due to PROP treatment. There was no difference in MUC2 expression in enteroids treated with 1 mM SCFAs (P > 0.05).',32240190),(3459,'human','null','Gene expression of MUC2 was increased 2-fold due to ACET treatment and 5-fold due to BUT treatment (P < 0.05; Fig 4b) but was not increased due to PROP treatment. There was no difference in MUC2 expression in enteroids treated with 1 mM SCFAs (P > 0.05).',32240190),(3460,'human','null','Lysozyme (LYZ) increased due to BUT alone (P < 0.05)however there was no difference in ALPI expression in enteroids treated with 1 mM SCFAs(P > 0.05).',32240190),(3461,'human','null','Enteroendocrine cell marker CHGA was significantly increased (170-fold) in BUT treated enteroids. There was no difference in CHGA expression in enteroids treated with 1\nmM SCFAs (P > 0.05).',32240190),(3462,'human','null','Tuft cell marker DCLK1 was significantly elevated due to ACET(7-fold) and BUT (14-fold) treatment compared to CON (P < 0.05; Fig 4b). 1 mM ACET,BUT and PROP significantly increased DCLK1 expression (5-fold, 3.6-fold, and 5.8-fold,respectively; P < 0.05).',32240190),(3463,'human','null','Tuft cell marker DCLK1 was significantly elevated due to ACET(7-fold) and BUT (14-fold) treatment compared to CON (P < 0.05; Fig 4b). 1 mM ACET,BUT and PROP significantly increased DCLK1 expression (5-fold, 3.6-fold, and 5.8-fold,respectively; P < 0.05).',32240190),(3464,'human','null','Tuft cell marker DCLK1 was significantly elevated due to ACET(7-fold) and BUT (14-fold) treatment compared to CON (P < 0.05; Fig 4b). 1 mM ACET,BUT and PROP significantly increased DCLK1 expression (5-fold, 3.6-fold, and 5.8-fold,respectively; P < 0.05).',32240190),(3465,'human','null','Anti-microbial peptide gene expression of REG3β and REG3γ (Fig 4b)were not increased due to ACET treatment, however, were significantly increased due to BUT(2.5-fold and 3-fold, respectively) compared to CON. There was no difference in REG3β and REG3γ expression in enteroids treated with 1 mM SCFAs (P > 0.05).',32240190),(3466,'human','null','Anti-microbial peptide gene expression of REG3β and REG3γ (Fig 4b)were not increased due to ACET treatment, however, were significantly increased due to BUT(2.5-fold and 3-fold, respectively) compared to CON. There was no difference in REG3β and REG3γ expression in enteroids treated with 1 mM SCFAs (P > 0.05).',32240190),(3467,'human','null','β-defensin 1 (DEFB1)was increased 75-fold and 66-fold respectively in BUT and PROP treated enteroids (P < 0.05; Fig4b).',32240190),(3468,'human','null','β-defensin 1 (DEFB1)was increased 75-fold and 66-fold respectively in BUT and PROP treated enteroids (P < 0.05; Fig4b).',32240190),(3469,'human','null','Tight junction marker expression of OCLN was only significantly elevated in ACET treated enteroids (4-fold; P < 0.05,Fig 4b).There was no difference in OCLN expression in enteroids treated with 1 mM SCFAs(P > 0.05).',32240190),(3470,'human','null','CLDN3 and TJP1 were increased due to both ACET and BUT treatments(P < 0.05, Fig 4b). There was no difference in CLDN3 expression in enteroids treated with 1 mM SCFAs (P > 0.05), however 1 mM BUT decreased TJP1 expression more than 2-fold (P < 0.05).',32240190),(3471,'human','null','CLDN3 and TJP1 were increased due to both ACET and BUT treatments(P < 0.05, Fig 4b). There was no difference in CLDN3 expression in enteroids treated with 1 mM SCFAs (P > 0.05), however 1 mM BUT decreased TJP1 expression more than 2-fold (P < 0.05).',32240190),(3472,'human','null','CLDN3 and TJP1 were increased due to both ACET and BUT treatments(P < 0.05, Fig 4b). There was no difference in CLDN3 expression in enteroids treated with 1 mM SCFAs (P > 0.05), however 1 mM BUT decreased TJP1 expression more than 2-fold (P < 0.05).',32240190),(3473,'human','null','CLDN3 and TJP1 were increased due to both ACET and BUT treatments(P < 0.05, Fig 4b). There was no difference in CLDN3 expression in enteroids treated with 1 mM SCFAs (P > 0.05), however 1 mM BUT decreased TJP1 expression more than 2-fold (P < 0.05).',32240190),(3474,'human','null','Tight junction related marker claudin-3 (CLDN3) expression was significantly increased by 10 mM BUT and 10 mM PROP (1.5 and 1.4-fold, respectively; P < 0.05).',32240190),(3475,'human','null','Occludin (OCLN) expression was significantly increased by 10 mM BUT treatment alone (P < 0.05).',32240190),(3476,'human','null','Ghrelin was significantly increased by all SCFAs in cell supernatants, with BUT and PROP showing highest concentrations (P < 0.05). Ghrelin was also significantly increased by 10 mM BUT alone in basolateral media (P < 0.05; Table 1).',32240190),(3477,'human','null','Ghrelin was significantly increased by all SCFAs in cell supernatants, with BUT and PROP showing highest concentrations (P < 0.05). Ghrelin was also significantly increased by 10 mM BUT alone in basolateral media (P < 0.05; Table 1).',32240190),(3478,'human','null','Ghrelin was significantly increased by all SCFAs in cell supernatants, with BUT and PROP showing highest concentrations (P < 0.05). Ghrelin was also significantly increased by 10 mM BUT alone in basolateral media (P < 0.05; Table 1).',32240190),(3479,'human','null','Interleukin 4 was significantly increased in 10 mM BUT and PROP treated 2D enteroids(P < 0.05) and was only increased significantly in basolateral media by 10 mM BUT (P < 0.05;Table 1).',32240190),(3480,'human','null','Interleukin 4 was significantly increased in 10 mM BUT and PROP treated 2D enteroids(P < 0.05) and was only increased significantly in basolateral media by 10 mM BUT (P < 0.05;Table 1).',32240190),(3481,'human','null','Interleukin 8 was significantly increased in cell supernatants of 10 mM BUT treated enteroids as well as basolateral media (P < 0.05). Interleukin 18 was not significantly different in cell supernatant, however, there was a tendency for 10 mM BUT to increase expression (P = 0.08;Table 1). Interleukin 18, however was increased by 10 mM BUT treatment in basolateral media(P < 0.05).',32240190),(3482,'human','D-Glucose','B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).',32873608),(3483,'human','D-Glucose','B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).',32873608),(3484,'human','D-Glucose','B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).',32873608),(3485,'human','D-Glucose','B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).',32873608),(3486,'human','D-Glucose','B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).',32873608),(3487,'human','D-Glucose','B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).',32873608),(3488,'human','D-Glucose','B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).',32873608),(3489,'human','D-Glucose','B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).',32873608),(3490,'human','D-Glucose','B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).',32873608),(3491,'human','D-Glucose','B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).',32873608),(3492,'human','D-Glucose','B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).',32873608),(3493,'human','Levodopa','Here, we show that C. sporogenes deaminates levodopa, the main treatment in Parkinson’s disease, and identify the aromatic aminotransferase responsible for the initiation of the deamination pathway. The deaminate metabolite from levodopa, 3-(3,4-dihydroxyphenyl)propionic acid, elicits an inhibitory effect on ileal motility in an ex vivo model.',33076930),(3494,'human','5-Hydroxytryptophan','5-HTP was converted into two new unknown peaks (Additional File1: Fig. S2A, S2B), albeit to a much lesser extent compared to levodopa. Only the first peak could be detected and assigned by LC-MS as 5-hydroxyindole-3-lactic acid (5-HILA) by its predicted exact mass (Additional File 1:Fig. S2C).',33076930),(3495,'human','Tryptophan','Tryptophan and tyrosine were converted to their intermediates ILA (indole-3-lactic acid) and 4-HPLA (3-(4-hydroxyphenyl)lactic acid), respectively, as previously shown [11].',33076930),(3496,'human','Tyrosine','Tryptophan and tyrosine were converted to their intermediates ILA (indole-3-lactic acid) and 4-HPLA (3-(4-hydroxyphenyl)lactic acid), respectively, as previously shown [11].',33076930),(3497,'human','Levodopa','Analogous to tryptophan and tyrosine, levodopa was no longer deaminated to DHPPA but to its intermediate product 3-(3,4-dihydroxy phenyl)lactic acid (DHPLA) (Fig. 1d, e). ',33076930),(3498,'human','null','The human ortholog HM74a is also a nicotinic acid receptor and likely has a similar role in anti-lipolysis.',15929991),(3499,'human','null','Here we show that the fatty acid-derived ketone body (D)-OHB specifically activates PUMA-G/HM74a at concentrations observed in serum during fasting.',15929991),(3500,'human','null','Here we show that the fatty acid-derived ketone body (D)-OHB specifically activates PUMA-G/HM74a at concentrations observed in serum during fasting.',15929991),(3501,'human','null','Here we show that the fatty acid-derived ketone body (D)-OHB specifically activates PUMA-G/HM74a at concentrations observed in serum during fasting.',15929991),(3502,'human','null','Both (D)- and (L)-OHB, but not acetoacetate or free acetone, elicited a Ca2 response in cells expressing HM74a but not HM74.',15929991),(3503,'human','null','Racemic (DL)-OHB also showed some degree of activity on all receptors; however, it was more potent on HM74a and its ortholog PUMA-G  than on HM74.',15929991),(3504,'human','Diosmetin 7-O-beta-D-glucopyranoside','A Gram-negative anaerobic bacterium, strain 4, capable of metabolizing diosmetin-7-O-glucoside was newly isolated.Its 16S rRNA sequence displayed 99% similarity with that of Escherichia. Then strain 4 was identified as a species of thegenus Escherichia and was named Escherichia sp. 4. Additionally,  an ultraperformance liquid chromatography/quadrupole-time-of-flight mass spectrometry  technique combined with Metabolynx software method was established to screenthe metabolites of diosmetin-7-O-glucoside.',25382172),(3505,'human','Diosmetin 7-O-beta-D-glucopyranoside','A Gram-negative anaerobic bacterium, strain 4, capable of metabolizing diosmetin-7-O-glucoside was newly isolated.Its 16S rRNA sequence displayed 99% similarity with that of Escherichia. Then strain 4 was identified as a species of thegenus Escherichia and was named Escherichia sp. 4. Additionally,  an ultraperformance liquid chromatography/quadrupole-time-of-flight mass spectrometry  technique combined with Metabolynx software method was established to screenthe metabolites of diosmetin-7-O-glucoside.',25382172),(3506,'human','null','We analyzed 50 colon adenomas from AA patients for both microsatellite instability (MSI) and the methylation status of SLC5A8 gene. This gene\'s product is involved in the transport of butyrate that has anti-proliferative properties through its effects on histone acetylation and gene expression. A proteomic analysis to check the expressed histones in adenoma and normal tissues was also performed.No association was found between SLC5A8 methylation and the MSI status. Also, there was no association between SLC5A8 methylation and the sex and age of the patients. However, there were more right sided adenomas with SLC5A8 methylation than the left sided ones. The proteomic analysis revealed distinct histone expression profiles between normal and adenoma tissues.SLC5A8 is highly methylated in AA colon adenomas which points to its potential use as a marker for early detection. The MSI rate is similar to that found in colon cancer tumors in AAs. These findings suggest that both processes stem from the same epigenetic and genetic events occurring at an early stage in colon carcinogenesis in AAs.',21687703),(3507,'human','null','We analyzed 50 colon adenomas from AA patients for both microsatellite instability (MSI) and the methylation status of SLC5A8 gene. This gene\'s product is involved in the transport of butyrate that has anti-proliferative properties through its effects on histone acetylation and gene expression. A proteomic analysis to check the expressed histones in adenoma and normal tissues was also performed.No association was found between SLC5A8 methylation and the MSI status. Also, there was no association between SLC5A8 methylation and the sex and age of the patients. However, there were more right sided adenomas with SLC5A8 methylation than the left sided ones. The proteomic analysis revealed distinct histone expression profiles between normal and adenoma tissues.SLC5A8 is highly methylated in AA colon adenomas which points to its potential use as a marker for early detection. The MSI rate is similar to that found in colon cancer tumors in AAs. These findings suggest that both processes stem from the same epigenetic and genetic events occurring at an early stage in colon carcinogenesis in AAs.',21687703),(3508,'human','null','We analyzed 50 colon adenomas from AA patients for both microsatellite instability (MSI) and the methylation status of SLC5A8 gene. This gene\'s product is involved in the transport of butyrate that has anti-proliferative properties through its effects on histone acetylation and gene expression. A proteomic analysis to check the expressed histones in adenoma and normal tissues was also performed.No association was found between SLC5A8 methylation and the MSI status. Also, there was no association between SLC5A8 methylation and the sex and age of the patients. However, there were more right sided adenomas with SLC5A8 methylation than the left sided ones. The proteomic analysis revealed distinct histone expression profiles between normal and adenoma tissues.SLC5A8 is highly methylated in AA colon adenomas which points to its potential use as a marker for early detection. The MSI rate is similar to that found in colon cancer tumors in AAs. These findings suggest that both processes stem from the same epigenetic and genetic events occurring at an early stage in colon carcinogenesis in AAs.',21687703),(3509,'human','null','In vitro cultivation of PD demonstrated its capacity to transform bile acids and production of succinate. Succinate supplementation in the diet decreased hyperglycemia in ob/ob mice via the activation of intestinal gluconeogenesis (IGN). Gavage with a mixture of LCA and UDCA reduced hyperlipidemia by activating the FXR pathway and repairing gut barrier integrity. Co-treatment with succinate and LCA/UDCA mirrored the benefits of LPD. The binding target of succinate was identified as fructose-1,6-bisphosphatase, the rate-limiting enzyme in IGN. The succinate and secondary bile acids produced by P. distasonis played key roles in the modulation of host metabolism.',30605678),(3510,'human','null','In vitro cultivation of PD demonstrated its capacity to transform bile acids and production of succinate. Succinate supplementation in the diet decreased hyperglycemia in ob/ob mice via the activation of intestinal gluconeogenesis (IGN). Gavage with a mixture of LCA and UDCA reduced hyperlipidemia by activating the FXR pathway and repairing gut barrier integrity. Co-treatment with succinate and LCA/UDCA mirrored the benefits of LPD. The binding target of succinate was identified as fructose-1,6-bisphosphatase, the rate-limiting enzyme in IGN. The succinate and secondary bile acids produced by P. distasonis played key roles in the modulation of host metabolism.',30605678),(3516,'human','Clindamycin','In the CDI chemostat model, clindamycin decreased valerate and deoxycholic acid concentrations and increased C difficile total viable counts (TVC) and valerate precursors, taurocholic acid, and succinate concentrations. After we stopped adding clindamycin, levels of bile acids and succinate recovered, whereas levels of valerate and valerate precursors did not. In the CDI chemostat model FMT increased valerate concentrations and decreased C difficile TVC (94% reduction), spore counts (86% reduction), and valerate precursor concentrations—concentrations of bile acids were unchanged.',30025704),(3517,'human','Clindamycin','In the CDI chemostat model, clindamycin decreased valerate and deoxycholic acid concentrations and increased C difficile total viable counts (TVC) and valerate precursors, taurocholic acid, and succinate concentrations. After we stopped adding clindamycin, levels of bile acids and succinate recovered, whereas levels of valerate and valerate precursors did not. In the CDI chemostat model FMT increased valerate concentrations and decreased C difficile TVC (94% reduction), spore counts (86% reduction), and valerate precursor concentrations—concentrations of bile acids were unchanged.',30025704),(3518,'human','Clindamycin','In the CDI chemostat model, clindamycin decreased valerate and deoxycholic acid concentrations and increased C difficile total viable counts (TVC) and valerate precursors, taurocholic acid, and succinate concentrations. After we stopped adding clindamycin, levels of bile acids and succinate recovered, whereas levels of valerate and valerate precursors did not. In the CDI chemostat model FMT increased valerate concentrations and decreased C difficile TVC (94% reduction), spore counts (86% reduction), and valerate precursor concentrations—concentrations of bile acids were unchanged.',30025704),(3519,'human','Clindamycin','In the CDI chemostat model, clindamycin decreased valerate and deoxycholic acid concentrations and increased C difficile total viable counts (TVC) and valerate precursors, taurocholic acid, and succinate concentrations. After we stopped adding clindamycin, levels of bile acids and succinate recovered, whereas levels of valerate and valerate precursors did not. In the CDI chemostat model FMT increased valerate concentrations and decreased C difficile TVC (94% reduction), spore counts (86% reduction), and valerate precursor concentrations—concentrations of bile acids were unchanged.',30025704),(3520,'human','null','In our study, we demonstrate that n-butyrate, a bacterial metabolite found in high abundance in the colon, regulates macrophage function through the inhibition of HDACs. n-Butyrate may behave as a microbial signal to inform the host immune system that the relative levels of n-butyrate–producing bacteria are within the desired range. In this model, HDACs act as sensors for n-butyrate, and their inhibition by high amounts of this metabolite causes intestinal macrophages to reduce production of proinflammatory mediators such as NO, IL-6, and IL-12.',24390544),(3521,'human','null','In our study, we demonstrate that n-butyrate, a bacterial metabolite found in high abundance in the colon, regulates macrophage function through the inhibition of HDACs. n-Butyrate may behave as a microbial signal to inform the host immune system that the relative levels of n-butyrate–producing bacteria are within the desired range. In this model, HDACs act as sensors for n-butyrate, and their inhibition by high amounts of this metabolite causes intestinal macrophages to reduce production of proinflammatory mediators such as NO, IL-6, and IL-12.',24390544),(3522,'human','null','In our study, we demonstrate that n-butyrate, a bacterial metabolite found in high abundance in the colon, regulates macrophage function through the inhibition of HDACs. n-Butyrate may behave as a microbial signal to inform the host immune system that the relative levels of n-butyrate–producing bacteria are within the desired range. In this model, HDACs act as sensors for n-butyrate, and their inhibition by high amounts of this metabolite causes intestinal macrophages to reduce production of proinflammatory mediators such as NO, IL-6, and IL-12.',24390544),(3523,'human','Resveratrol','In conclusion, we identified 3 microbial trans-resveratrol metabolites, 2 of which were previously unknown and we also identified 2 intestinal bacterial species that were previously not known to produce dihydroresveratrol. In addition, we showed that there can be pronounced interindividual differences in resveratrol conversion by human gut microbiota.',23283496),(3524,'human','Resveratrol','In conclusion, we identified 3 microbial trans-resveratrol metabolites, 2 of which were previously unknown and we also identified 2 intestinal bacterial species that were previously not known to produce dihydroresveratrol. In addition, we showed that there can be pronounced interindividual differences in resveratrol conversion by human gut microbiota.',23283496),(3525,'human','Resveratrol','In conclusion, we identified 3 microbial trans-resveratrol metabolites, 2 of which were previously unknown and we also identified 2 intestinal bacterial species that were previously not known to produce dihydroresveratrol. In addition, we showed that there can be pronounced interindividual differences in resveratrol conversion by human gut microbiota.',23283496),(3526,'human','Resveratrol','In conclusion, we identified 3 microbial trans-resveratrol metabolites, 2 of which were previously unknown and we also identified 2 intestinal bacterial species that were previously not known to produce dihydroresveratrol. In addition, we showed that there can be pronounced interindividual differences in resveratrol conversion by human gut microbiota.',23283496),(3527,'human','Resveratrol','In conclusion, we identified 3 microbial trans-resveratrol metabolites, 2 of which were previously unknown and we also identified 2 intestinal bacterial species that were previously not known to produce dihydroresveratrol. In addition, we showed that there can be pronounced interindividual differences in resveratrol conversion by human gut microbiota.',23283496),(3528,'human','Resveratrol','In conclusion, we identified 3 microbial trans-resveratrol metabolites, 2 of which were previously unknown and we also identified 2 intestinal bacterial species that were previously not known to produce dihydroresveratrol. In addition, we showed that there can be pronounced interindividual differences in resveratrol conversion by human gut microbiota.',23283496),(3529,'human','1-((3,5-Dichloro)-2,6-dihydroxy-4-methoxyphenyl)-1-hexanone','DIF-3 induces proteolysis of cyclin D1 and c-Myc in HCT-116 cells via activation of GSK-3b.',25913757),(3530,'human','1-((3,5-Dichloro)-2,6-dihydroxy-4-methoxyphenyl)-1-hexanone','The amount of TCF7L2 increased gradually over time in control cells, but it was markedly reduced by DIF-3 in a time and dose-dependent manner.',25913757),(3531,'human','1-((3,5-Dichloro)-2,6-dihydroxy-4-methoxyphenyl)-1-hexanone','DIF-3 decreased the expression of cyclin D1 and cMyc to a level comparable to the reduction induced by DIF-1.',25913757),(3532,'human','1-((3,5-Dichloro)-2,6-dihydroxy-4-methoxyphenyl)-1-hexanone','DIF-3 decreased the expression of cyclin D1 and cMyc to a level comparable to the reduction induced by DIF-1.',25913757),(3533,'human','null','Butyrate-induced antimicrobial activity was associated with a shift in macrophage metabolism, a reduction in mTOR kinase activity, increased LC3-associated host defense and anti-microbial peptide production in the absence of an increased inflammatory cytokine response. Butyrate drove this monocyte to macrophage differentiation program through histone deacetylase 3 (HDAC3) inhibition. Administration of butyrate induced antimicrobial activity in intestinal macrophages in vivo and increased resistance to enteropathogens.',30683619),(3534,'human','null','Butyrate-induced antimicrobial activity was associated with a shift in macrophage metabolism, a reduction in mTOR kinase activity, increased LC3-associated host defense and anti-microbial peptide production in the absence of an increased inflammatory cytokine response. Butyrate drove this monocyte to macrophage differentiation program through histone deacetylase 3 (HDAC4) inhibition. Administration of butyrate induced antimicrobial activity in intestinal macrophages in vivo and increased resistance to enteropathogens.',30683619),(3535,'human','null','Butyrate-induced antimicrobial activity was associated with a shift in macrophage metabolism, a reduction in mTOR kinase activity, increased LC3-associated host defense and anti-microbial peptide production in the absence of an increased inflammatory cytokine response. Butyrate drove this monocyte to macrophage differentiation program through histone deacetylase 3 (HDAC5) inhibition. Administration of butyrate induced antimicrobial activity in intestinal macrophages in vivo and increased resistance to enteropathogens.',30683619),(3536,'human','Xyloglucan Heptasaccharide','Commensal bifidobacteria colonize the human gastrointestinal tract and catabolize glycans that are impervious to host digestion. Accordingly, Bifidobacterium longum typically secretes acetate and lactate as fermentative end products. This study tested the hypothesis that B. longum utilizes cranberry-derived xyloglucans in a strain-dependent manner. Interestingly, the B. longum strain that efficiently utilizes cranberry xyloglucans secretes 2.0 to 2.5 mol of acetate-lactate. The 1.5 acetate:lactate ratio theoretical yield obtained in hexose fermentations shifts during xyloglucan metabolism. Accordingly, this metabolic shift is characterized by increased acetate and formate production at the expense of lactate. α-l-Arabinofuranosidase, an arabinan endo-1,5-α-l-arabinosidase, and a β-xylosidase with a carbohydrate substrate-binding protein and carbohydrate ABC transporter membrane proteins are upregulated (>2-fold change), which suggests carbon flux through this catabolic pathway. Finally, syntrophic interactions occurred with strains that utilize carbohydrate products derived from initial degradation from heterologous bacteria.',28667113),(3537,'human','Xyloglucan Heptasaccharide','Commensal bifidobacteria colonize the human gastrointestinal tract and catabolize glycans that are impervious to host digestion. Accordingly, Bifidobacterium longum typically secretes acetate and lactate as fermentative end products. This study tested the hypothesis that B. longum utilizes cranberry-derived xyloglucans in a strain-dependent manner. Interestingly, the B. longum strain that efficiently utilizes cranberry xyloglucans secretes 2.0 to 2.5 mol of acetate-lactate. The 1.5 acetate:lactate ratio theoretical yield obtained in hexose fermentations shifts during xyloglucan metabolism. Accordingly, this metabolic shift is characterized by increased acetate and formate production at the expense of lactate. α-l-Arabinofuranosidase, an arabinan endo-1,5-α-l-arabinosidase, and a β-xylosidase with a carbohydrate substrate-binding protein and carbohydrate ABC transporter membrane proteins are upregulated (>2-fold change), which suggests carbon flux through this catabolic pathway. Finally, syntrophic interactions occurred with strains that utilize carbohydrate products derived from initial degradation from heterologous bacteria.',28667113),(3538,'human','null','We have presented direct evidence that the gut bacterial genotoxin colibactin alkylates DNA in vivo. The ability of pks+ E. coli to generate DNA adducts in mammalian cells and in mice strengthens support for the involvement of colibactin in cancer development or progression since misrepaired mono- and cross-linked adducts may generate mutations in oncogenes or tumor suppressor genes, contributing to tumorigenesis . Our findings will enable efforts to decipher the molecular details of this process. Importantly, this work has also uncovered a candidate metabolite biomarker of colibactin exposure and cancer risk. The ability to directly assess whether exposure to colibactin has occurred in animal models and human patients will help address the critical question of whether pks+ E. coli contribute to colorectal carcinogenesis in patient cohorts .',30765538),(3539,'human','null','We have presented direct evidence that the gut bacterial genotoxin colibactin alkylates DNA in vivo. The ability of pks+ E. coli to generate DNA adducts in mammalian cells and in mice strengthens support for the involvement of colibactin in cancer development or progression since misrepaired mono- and cross-linked adducts may generate mutations in oncogenes or tumor suppressor genes, contributing to tumorigenesis . Our findings will enable efforts to decipher the molecular details of this process. Importantly, this work has also uncovered a candidate metabolite biomarker of colibactin exposure and cancer risk. The ability to directly assess whether exposure to colibactin has occurred in animal models and human patients will help address the critical question of whether pks+ E. coli contribute to colorectal carcinogenesis in patient cohorts .',30765538),(3542,'human','Soybean Isoflavones','An equol-producing bacterium, strain DZET , which was isolated from human faeces, was characterized by morphological, biochemical and molecular methods. The isolate was Grampositive, obligately anaerobic, non-spore forming, asaccharolytic and rod-shaped. 16S rRNA sequence analysis showed 92.8, 91.0, 91.1 and 90.6 % similarities with Slackia faecicanis, Slackia exigua, Slackia heliotrinireducens and Slackia isoflavoniconvertens, respectively. Basedon these data, we propose a novel species of the genus Slackia, Slackia equolifaciens sp. nov.The major cellular fatty acids are C14 : 0, C18 : 1v9c and C18 : 1v9c DMA (dimethyl acetal). The DNA G+C content of the strain is 60.8 mol%. The type strain of S.equolifaciens sp. nov. is DZE.',19734283),(3543,'human','Daidzein','Equol and 5-hydroxy-equol were found in intestinal contents, feces, and urine of SIA rats but not in the corresponding samples of the control rats. 5-Hydroxy-equol was present at much lower concentrations than equol and the main metabolite produced from genistein was the intermediate dihydrogenistein. The plasma of SIA rats contained equol but no 5-hydroxy-equol. Equol formation had no effect on plasma concentrations of the insulin-like growth factor I. The concentrations of daidzein and genistein were considerably lower in all samples of the SIA rats than in those of the control rats. Male SIA rats had higher intestinal and fecal concentrations of the isoflavones and their metabolites than female SIA rats. The observed activity in the rat model indicates that S. isoflavoniconvertens is capable of contributing in vivo to the bioactivation of daidzein and genistein by formation of equol and 5-hydroxy-equol.',22113864),(3544,'human','Genistein','Equol and 5-hydroxy-equol were found in intestinal contents, feces, and urine of SIA rats but not in the corresponding samples of the control rats. 5-Hydroxy-equol was present at much lower concentrations than equol and the main metabolite produced from genistein was the intermediate dihydrogenistein. The plasma of SIA rats contained equol but no 5-hydroxy-equol. Equol formation had no effect on plasma concentrations of the insulin-like growth factor I. The concentrations of daidzein and genistein were considerably lower in all samples of the SIA rats than in those of the control rats. Male SIA rats had higher intestinal and fecal concentrations of the isoflavones and their metabolites than female SIA rats. The observed activity in the rat model indicates that S. isoflavoniconvertens is capable of contributing in vivo to the bioactivation of daidzein and genistein by formation of equol and 5-hydroxy-equol.',22113864),(3545,'human','null','The alignment results showed that homologous proteins for FOS transporters and glycosidases were distributed in 237 of the 453 strains of gut bacteria. La506 msmK encoding the ATP-binding protein and Aec45 fosGH1 encoding glycoside hydrolase were most widely distributed, in 155 and 55 strains, respectively. Seven of eight strains with both transporters and glycosidases were proven to be capable of metabolizing FOS, while five strains without either transporters or glycosidases were not. Fifteen species isolated from human feces and 11 species from the alignment search were identified to be FOS-metabolizing, of which Cronobacter sakazakii, Marvinbryantia formatexigens, Ruminococcus gnavus, and Weissella paramesenteroides are reported here for the first time.',25653214),(3546,'human','null','The alignment results showed that homologous proteins for FOS transporters and glycosidases were distributed in 237 of the 453 strains of gut bacteria. La506 msmK encoding the ATP-binding protein and Aec45 fosGH1 encoding glycoside hydrolase were most widely distributed, in 155 and 55 strains, respectively. Seven of eight strains with both transporters and glycosidases were proven to be capable of metabolizing FOS, while five strains without either transporters or glycosidases were not. Fifteen species isolated from human feces and 11 species from the alignment search were identified to be FOS-metabolizing, of which Cronobacter sakazakii, Marvinbryantia formatexigens, Ruminococcus gnavus, and Weissella paramesenteroides are reported here for the first time.',25653214),(3547,'human','null','The alignment results showed that homologous proteins for FOS transporters and glycosidases were distributed in 237 of the 453 strains of gut bacteria. La506 msmK encoding the ATP-binding protein and Aec45 fosGH1 encoding glycoside hydrolase were most widely distributed, in 155 and 55 strains, respectively. Seven of eight strains with both transporters and glycosidases were proven to be capable of metabolizing FOS, while five strains without either transporters or glycosidases were not. Fifteen species isolated from human feces and 11 species from the alignment search were identified to be FOS-metabolizing, of which Cronobacter sakazakii, Marvinbryantia formatexigens, Ruminococcus gnavus, and Weissella paramesenteroides are reported here for the first time.',25653214),(3548,'human','null','The alignment results showed that homologous proteins for FOS transporters and glycosidases were distributed in 237 of the 453 strains of gut bacteria. La506 msmK encoding the ATP-binding protein and Aec45 fosGH1 encoding glycoside hydrolase were most widely distributed, in 155 and 55 strains, respectively. Seven of eight strains with both transporters and glycosidases were proven to be capable of metabolizing FOS, while five strains without either transporters or glycosidases were not. Fifteen species isolated from human feces and 11 species from the alignment search were identified to be FOS-metabolizing, of which Cronobacter sakazakii, Marvinbryantia formatexigens, Ruminococcus gnavus, and Weissella paramesenteroides are reported here for the first time.',25653214),(3549,'human','Daidzin','Fecal bacteria from a healthy individual were screened for the specific bacteria involved in the metabolism of dietary isoflavonoids. Two strains of bacteria capable of producing primary and secondary metabolites from the natural isoflavone glycosides daidzin and genistin were detected. The metabolites were identified by comparison of their HPLC/mass, 1H NMR and UV spectra with those of standard and synthetic compounds. Both Escherichia coli HGH21 and the gram-positive strain HGH6 converted daidzin and genistin to the their respective aglycones daidzein and genistein. Under anoxic conditions, strain HGH6 further metabolized the isoflavones daidzein and genistein to dihydrodaidzein and dihydrogenistein, respectively.',11195098),(3550,'human','Genistin','Fecal bacteria from a healthy individual were screened for the specific bacteria involved in the metabolism of dietary isoflavonoids. Two strains of bacteria capable of producing primary and secondary metabolites from the natural isoflavone glycosides daidzin and genistin were detected. The metabolites were identified by comparison of their HPLC/mass, 1H NMR and UV spectra with those of standard and synthetic compounds. Both Escherichia coli HGH21 and the gram-positive strain HGH6 converted daidzin and genistin to the their respective aglycones daidzein and genistein. Under anoxic conditions, strain HGH6 further metabolized the isoflavones daidzein and genistein to dihydrodaidzein and dihydrogenistein, respectively.',11195098),(3551,'human','Daidzin','Fecal bacteria from a healthy individual were screened for the specific bacteria involved in the metabolism of dietary isoflavonoids. Two strains of bacteria capable of producing primary and secondary metabolites from the natural isoflavone glycosides daidzin and genistin were detected. The metabolites were identified by comparison of their HPLC/mass, 1H NMR and UV spectra with those of standard and synthetic compounds. Both Escherichia coli HGH21 and the gram-positive strain HGH6 converted daidzin and genistin to the their respective aglycones daidzein and genistein. Under anoxic conditions, strain HGH6 further metabolized the isoflavones daidzein and genistein to dihydrodaidzein and dihydrogenistein, respectively.',11195098),(3552,'human','Genistin','Fecal bacteria from a healthy individual were screened for the specific bacteria involved in the metabolism of dietary isoflavonoids. Two strains of bacteria capable of producing primary and secondary metabolites from the natural isoflavone glycosides daidzin and genistin were detected. The metabolites were identified by comparison of their HPLC/mass, 1H NMR and UV spectra with those of standard and synthetic compounds. Both Escherichia coli HGH21 and the gram-positive strain HGH6 converted daidzin and genistin to the their respective aglycones daidzein and genistein. Under anoxic conditions, strain HGH6 further metabolized the isoflavones daidzein and genistein to dihydrodaidzein and dihydrogenistein, respectively.',11195098),(3553,'human','Daidzein','Fecal bacteria from a healthy individual were screened for the specific bacteria involved in the metabolism of dietary isoflavonoids. Two strains of bacteria capable of producing primary and secondary metabolites from the natural isoflavone glycosides daidzin and genistin were detected. The metabolites were identified by comparison of their HPLC/mass, 1H NMR and UV spectra with those of standard and synthetic compounds. Both Escherichia coli HGH21 and the gram-positive strain HGH6 converted daidzin and genistin to the their respective aglycones daidzein and genistein. Under anoxic conditions, strain HGH6 further metabolized the isoflavones daidzein and genistein to dihydrodaidzein and dihydrogenistein, respectively.',11195098),(3554,'human','Genistein','Fecal bacteria from a healthy individual were screened for the specific bacteria involved in the metabolism of dietary isoflavonoids. Two strains of bacteria capable of producing primary and secondary metabolites from the natural isoflavone glycosides daidzin and genistin were detected. The metabolites were identified by comparison of their HPLC/mass, 1H NMR and UV spectra with those of standard and synthetic compounds. Both Escherichia coli HGH21 and the gram-positive strain HGH6 converted daidzin and genistin to the their respective aglycones daidzein and genistein. Under anoxic conditions, strain HGH6 further metabolized the isoflavones daidzein and genistein to dihydrodaidzein and dihydrogenistein, respectively.',11195098),(3555,'human','null','Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.',31123037),(3556,'human','null','Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.',31123037),(3557,'human','null','Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.',31123037),(3558,'human','null','Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.',31123037),(3559,'human','null','Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.',31123037),(3560,'human','null','Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.',31123037),(3561,'human','null','Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.',31123037),(3562,'human','null','Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.',31123037),(3563,'human','null','Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.',31123037),(3564,'human','null','Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.',31123037),(3565,'human','null','Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.',31123037),(3573,'human','D-Glucose','Two anaerobic, non-spore-forming, non-motile, Gram-negative-staining bacteria, strains YIT 12060T and YIT 12061T, were isolated from human faeces. Cells of strain YIT 12060T were coccoid to rod-shaped with round ends, positive for catalase, negative for indole and oxidase production, produced succinic and acetic acids as end products of glucose metabolism in peptone/yeast extract/glucose medium and had a DNA G+C content of 55.2 mol%. Phylogenetic analysis and biochemical data supported the affiliation of strain YIT 12060T to the genus Alistipes of the family ‘Rikenellaceae�? Strain YIT 12060T therefore represents a novel species of the genus Alistipes for which the name Alistipes indistinctus sp. nov. is proposed; the type strain is YIT 12060T.',19667375),(3574,'human','D-Glucose','Two anaerobic, non-spore-forming, non-motile, Gram-negative-staining bacteria, strains YIT 12060T and YIT 12061T, were isolated from human faeces. Cells of strain YIT 12060T were coccoid to rod-shaped with round ends, positive for catalase, negative for indole and oxidase production, produced succinic and acetic acids as end products of glucose metabolism in peptone/yeast extract/glucose medium and had a DNA G+C content of 55.2 mol%. Phylogenetic analysis and biochemical data supported the affiliation of strain YIT 12060T to the genus Alistipes of the family ‘Rikenellaceae�? Strain YIT 12060T therefore represents a novel species of the genus Alistipes for which the name Alistipes indistinctus sp. nov. is proposed; the type strain is YIT 12060T.',19667375),(3575,'human','D-Glucose','Two anaerobic, non-spore-forming, non-motile, Gram-negative-staining bacteria, strains YIT 12060T and YIT 12061T, were isolated from human faeces. Cells of strain YIT 12060T were coccoid to rod-shaped with round ends, positive for catalase, negative for indole and oxidase production, produced succinic and acetic acids as end products of glucose metabolism in peptone/yeast extract/glucose medium and had a DNA G+C content of 55.2 mol%. Phylogenetic analysis and biochemical data supported the affiliation of strain YIT 12060T to the genus Alistipes of the family ‘Rikenellaceae�? Strain YIT 12060T therefore represents a novel species of the genus Alistipes for which the name Alistipes indistinctus sp. nov. is proposed; the type strain is YIT 12060T.',19667375),(3576,'human','D-Glucose','Two anaerobic, non-spore-forming, non-motile, Gram-negative-staining bacteria, strains YIT 12060T and YIT 12061T, were isolated from human faeces. Cells of strain YIT 12060T were coccoid to rod-shaped with round ends, positive for catalase, negative for indole and oxidase production, produced succinic and acetic acids as end products of glucose metabolism in peptone/yeast extract/glucose medium and had a DNA G+C content of 55.2 mol%. Phylogenetic analysis and biochemical data supported the affiliation of strain YIT 12060T to the genus Alistipes of the family ‘Rikenellaceae�? Strain YIT 12060T therefore represents a novel species of the genus Alistipes for which the name Alistipes indistinctus sp. nov. is proposed; the type strain is YIT 12060T.',19667375),(3577,'human','D-Glucose','Two anaerobic, non-spore-forming, non-motile, Gram-negative-staining bacteria, strains YIT 12060T and YIT 12061T, were isolated from human faeces. Cells of strain YIT 12060T were coccoid to rod-shaped with round ends, positive for catalase, negative for indole and oxidase production, produced succinic and acetic acids as end products of glucose metabolism in peptone/yeast extract/glucose medium and had a DNA G+C content of 55.2 mol%. Phylogenetic analysis and biochemical data supported the affiliation of strain YIT 12060T to the genus Alistipes of the family ‘Rikenellaceae�? Strain YIT 12060T therefore represents a novel species of the genus Alistipes for which the name Alistipes indistinctus sp. nov. is proposed; the type strain is YIT 12060T.',19667375),(3578,'human','D-Glucose','Two anaerobic, non-spore-forming, non-motile, Gram-negative-staining bacteria, strains YIT 12060T and YIT 12061T, were isolated from human faeces. Cells of strain YIT 12060T were coccoid to rod-shaped with round ends, positive for catalase, negative for indole and oxidase production, produced succinic and acetic acids as end products of glucose metabolism in peptone/yeast extract/glucose medium and had a DNA G+C content of 55.2 mol%. Phylogenetic analysis and biochemical data supported the affiliation of strain YIT 12060T to the genus Alistipes of the family ‘Rikenellaceae�? Strain YIT 12060T therefore represents a novel species of the genus Alistipes for which the name Alistipes indistinctus sp. nov. is proposed; the type strain is YIT 12060T.',19667375),(3579,'human','D-Glucose','Two anaerobic, non-spore-forming, pleomorphic, Gram-negative rods, designated YIT 11840T and YIT 11841T, were isolated from human faeces. The organisms were catalase-negative, produced succinic and acetic acids as end products of glucose metabolism and had DNA G+C contents of approximately 48�?9 mol%. Although the phenotypic characteristics of these two strains were very similar, analysis of their 16S rRNA sequence showed that they are only distantly related (93.8 %), indicating that they represent two different species. A comparative sequence analysis revealed that these two species are members of the family ‘Prevotellaceae�?but are phylogenetically distant (<88 % sequence similarity) from the known genera belonging to this family, including Prevotella, Hallela and Xylanibacter.',19567577),(3580,'human','D-Glucose','Two anaerobic, non-spore-forming, pleomorphic, Gram-negative rods, designated YIT 11840T and YIT 11841T, were isolated from human faeces. The organisms were catalase-negative, produced succinic and acetic acids as end products of glucose metabolism and had DNA G+C contents of approximately 48�?9 mol%. Although the phenotypic characteristics of these two strains were very similar, analysis of their 16S rRNA sequence showed that they are only distantly related (93.8 %), indicating that they represent two different species. A comparative sequence analysis revealed that these two species are members of the family ‘Prevotellaceae�?but are phylogenetically distant (<88 % sequence similarity) from the known genera belonging to this family, including Prevotella, Hallela and Xylanibacter.',19567577),(3581,'human','D-Glucose','Two anaerobic, non-spore-forming, pleomorphic, Gram-negative rods, designated YIT 11840T and YIT 11841T, were isolated from human faeces. The organisms were catalase-negative, produced succinic and acetic acids as end products of glucose metabolism and had DNA G+C contents of approximately 48�?9 mol%. Although the phenotypic characteristics of these two strains were very similar, analysis of their 16S rRNA sequence showed that they are only distantly related (93.8 %), indicating that they represent two different species. A comparative sequence analysis revealed that these two species are members of the family ‘Prevotellaceae�?but are phylogenetically distant (<88 % sequence similarity) from the known genera belonging to this family, including Prevotella, Hallela and Xylanibacter.',19567577),(3582,'human','D-Glucose','Two anaerobic, non-spore-forming, pleomorphic, Gram-negative rods, designated YIT 11840T and YIT 11841T, were isolated from human faeces. The organisms were catalase-negative, produced succinic and acetic acids as end products of glucose metabolism and had DNA G+C contents of approximately 48�?9 mol%. Although the phenotypic characteristics of these two strains were very similar, analysis of their 16S rRNA sequence showed that they are only distantly related (93.8 %), indicating that they represent two different species. A comparative sequence analysis revealed that these two species are members of the family ‘Prevotellaceae�?but are phylogenetically distant (<88 % sequence similarity) from the known genera belonging to this family, including Prevotella, Hallela and Xylanibacter.',19567577),(3583,'human','null','F. prausnitzii modulated the metabolome of the host blood by decreasing IL-1α, IL-2, and IL-5 concentrations.',25900655),(3584,'human','null','F. prausnitzii modulated the metabolome of the host blood by decreasing IL-1α, IL-2, and IL-5 concentrations.',25900655),(3585,'human','null','F. prausnitzii modulated the metabolome of the host blood by decreasing IL-1α, IL-2, and IL-5 concentrations.',25900655),(3589,'human','Retinol','Retinoic acid (RA), a vitamin A metabolite, regulates transcriptional programs that drive protective or pathogenic immune responses in the intestine, in a manner dependent on RA concentration. Vitamin A is obtained from diet and is metabolized by intestinal epithelial cells (IECs), which operate in intimate association with microbes and immune cells. Here we found that commensal bacteria belonging to class Clostridia modulate RA concentration in the gut by suppressing the expression of retinol dehydrogenase 7 (Rdh7) in IECs. Rdh7 expression and associated RA amounts were lower in the intestinal tissue of conventional mice, as compared to germ-free mice. Deletion of Rdh7 in IECs diminished RA signaling in immune cells, reduced the IL-22-dependent antimicrobial response, and enhanced resistance to colonization by Salmonella Typhimurium. Our findings define a regulatory circuit wherein bacterial regulation of IEC-intrinsic RA synthesis protects microbial communities in the gut from excessive immune activity, achieving a balance that prevents colonization by enteric pathogens.',30566883),(3590,'human','Retinol','Retinoic acid (RA), a vitamin A metabolite, regulates transcriptional programs that drive protective or pathogenic immune responses in the intestine, in a manner dependent on RA concentration. Vitamin A is obtained from diet and is metabolized by intestinal epithelial cells (IECs), which operate in intimate association with microbes and immune cells. Here we found that commensal bacteria belonging to class Clostridia modulate RA concentration in the gut by suppressing the expression of retinol dehydrogenase 7 (Rdh7) in IECs. Rdh7 expression and associated RA amounts were lower in the intestinal tissue of conventional mice, as compared to germ-free mice. Deletion of Rdh7 in IECs diminished RA signaling in immune cells, reduced the IL-22-dependent antimicrobial response, and enhanced resistance to colonization by Salmonella Typhimurium. Our findings define a regulatory circuit wherein bacterial regulation of IEC-intrinsic RA synthesis protects microbial communities in the gut from excessive immune activity, achieving a balance that prevents colonization by enteric pathogens.',30566883),(3591,'human','Retinol','Retinoic acid (RA), a vitamin A metabolite, regulates transcriptional programs that drive protective or pathogenic immune responses in the intestine, in a manner dependent on RA concentration. Vitamin A is obtained from diet and is metabolized by intestinal epithelial cells (IECs), which operate in intimate association with microbes and immune cells. Here we found that commensal bacteria belonging to class Clostridia modulate RA concentration in the gut by suppressing the expression of retinol dehydrogenase 7 (Rdh7) in IECs. Rdh7 expression and associated RA amounts were lower in the intestinal tissue of conventional mice, as compared to germ-free mice. Deletion of Rdh7 in IECs diminished RA signaling in immune cells, reduced the IL-22-dependent antimicrobial response, and enhanced resistance to colonization by Salmonella Typhimurium. Our findings define a regulatory circuit wherein bacterial regulation of IEC-intrinsic RA synthesis protects microbial communities in the gut from excessive immune activity, achieving a balance that prevents colonization by enteric pathogens.',30566883),(3596,'human','Trimethylamine','We now show gut microbes, through generation of trimethylamine N-oxide (TMAO), directly contribute to platelet hyperreactivity and enhanced thrombosis potential. Plasma TMAO levels in subjects (n > 4,000) independently predicted incident (3 years) thrombosis (heart attack, stroke) risk. Direct exposure of platelets to TMAO enhanced sub maximal stimulus-dependent platelet activation from multiple agonists through augmented Ca2+ release from intracellular stores. Animal model studies employing dietary choline or TMAO, germ-free mice, and microbial transplantation collectively confirm a role for gut microbiota and TMAO in modulating platelet hyperresponsiveness and thrombosis potential and identify microbial taxa associated with plasma TMAO and thrombosis potential.',26972052),(3597,'human','Trimethylamine','We now show gut microbes, through generation of trimethylamine N-oxide (TMAO), directly contribute to platelet hyperreactivity and enhanced thrombosis potential. Plasma TMAO levels in subjects (n > 4,000) independently predicted incident (3 years) thrombosis (heart attack, stroke) risk. Direct exposure of platelets to TMAO enhanced sub maximal stimulus-dependent platelet activation from multiple agonists through augmented Ca2+ release from intracellular stores. Animal model studies employing dietary choline or TMAO, germ-free mice, and microbial transplantation collectively confirm a role for gut microbiota and TMAO in modulating platelet hyperresponsiveness and thrombosis potential and identify microbial taxa associated with plasma TMAO and thrombosis potential.',26972052),(3609,'human','Linoleic acid','In this study, we investigated the effects of LA and LA-derived fatty acids on the activation of peroxisome proliferator-activated receptors (PPARs) which regulate whole-body energy metabolism. None of the fatty acids activated PPARd, whereas almost all activated PPARa in luciferase assays. Two fatty acids potently activated PPARg, a master regulator of adipocyte differentiation, with 10-oxo-12(Z)-octadecenoic acid (KetoA) having the most potency. In 3T3- L1 cells, KetoA induced adipocyte differentiation via the activation of PPARg, and increased adiponectin production and insulin-stimulated glucose uptake. These findings suggest that fatty acids, including KetoA, generated in gut by lactic acid bacteria may be involved in the regulation of host energy metabolism.',25749343),(3610,'human','D-Fructose','A high-fructose diet caused the hippocampal neuroinflammatory response, reactive gliosis, and neuronal loss in C57BL/6N mice. Depletion of the gut microbiota using broad-spectrum antibiotics suppressed the hippocampal neuroinflammatory response in fructose-fed mice, but these animals still exhibited neuronal loss. Gut microbiota compositional alteration, short-chain fatty acids (SCFAs) reduction, intestinal epithelial barrier impairment, NOD-like receptor family pyrin domain-containing 6 (NLRP6) inflammasome dysfunction, high levels of serum endotoxin, and FITC-dextran were observed in fructose-fed mice. Of note, SCFAs, as well as pioglitazone (a selective peroxisome proliferator-activated receptor gamma (PPAR-γ) agonist), shaped the gut microbiota and ameliorated intestinal epithelial barrier impairment and NLRP6 inflammasome dysfunction in fructose-fed mice. Moreover, SCFAs-mediated NLRP6 inflammasome activation was inhibited by histamine (a bacterial metabolite) in ex vivo colonic explants and suppressed in murine CT26 colon carcinoma cells transfected with NLRP6 siRNA. However, pioglitazone and GW9662 (a PPAR-γ antagonist) exerted no impact on SCFAs-mediated NLRP6 inflammasome activation in ex vivo colonic explants, suggesting that SCFAs may stimulate NLRP6 inflammasome independently of PPAR-γ activation. SCFAs and pioglitazone prevented fructose-induced hippocampal neuroinflammatory response and neuronal loss in mice. Additionally, SCFAs activated colonic NLRP6 inflammasome and increased DCX+ newborn neurons in the hippocampal DG of control mice.',31255176),(3614,'human','Tryptophan','Indole-3-propionic acid (IPA), a tryptophan metabolite produced by gut bacteria, is a potent anti-non-alcoholic steatohepatitis (NASH) microbial metabolite. Here, we demonstrate that administration of IPA modulates the microbiota composition in the gut and inhibits microbial dysbiosis in rats fed a high-fat diet. IPA induces the expression of tight junction proteins, such as ZO-1 and Occludin, and maintains intestinal epithelium homeostasis, leading to a reduction in plasma endotoxin levels. Interestingly, IPA inhibits NF-κB signaling and reduces the levels of proinflammatory cytokines, such as TNFα, IL-1β, and IL-6, in response to endotoxin in macrophages to repress hepatic inflammation and liver injury. Moreover, IPA is sufficient to inhibit the expression of fibrogenic and collagen genes and attenuate diet-induced NASH phenotypes. The beneficial effects of IPA on the liver are likely mediated through inhibiting the production of endotoxin in the gut.',31506421),(3615,'human','Tryptophan','Indole-3-propionic acid (IPA), a tryptophan metabolite produced by gut bacteria, is a potent anti-non-alcoholic steatohepatitis (NASH) microbial metabolite. Here, we demonstrate that administration of IPA modulates the microbiota composition in the gut and inhibits microbial dysbiosis in rats fed a high-fat diet. IPA induces the expression of tight junction proteins, such as ZO-1 and Occludin, and maintains intestinal epithelium homeostasis, leading to a reduction in plasma endotoxin levels. Interestingly, IPA inhibits NF-κB signaling and reduces the levels of proinflammatory cytokines, such as TNFα, IL-1β, and IL-6, in response to endotoxin in macrophages to repress hepatic inflammation and liver injury. Moreover, IPA is sufficient to inhibit the expression of fibrogenic and collagen genes and attenuate diet-induced NASH phenotypes. The beneficial effects of IPA on the liver are likely mediated through inhibiting the production of endotoxin in the gut.',31506421),(3616,'human','Tryptophan','Indole-3-propionic acid (IPA), a tryptophan metabolite produced by gut bacteria, is a potent anti-non-alcoholic steatohepatitis (NASH) microbial metabolite. Here, we demonstrate that administration of IPA modulates the microbiota composition in the gut and inhibits microbial dysbiosis in rats fed a high-fat diet. IPA induces the expression of tight junction proteins, such as ZO-1 and Occludin, and maintains intestinal epithelium homeostasis, leading to a reduction in plasma endotoxin levels. Interestingly, IPA inhibits NF-κB signaling and reduces the levels of proinflammatory cytokines, such as TNFα, IL-1β, and IL-6, in response to endotoxin in macrophages to repress hepatic inflammation and liver injury. Moreover, IPA is sufficient to inhibit the expression of fibrogenic and collagen genes and attenuate diet-induced NASH phenotypes. The beneficial effects of IPA on the liver are likely mediated through inhibiting the production of endotoxin in the gut.',31506421),(3617,'human','Tryptophan','Indole-3-propionic acid (IPA), a tryptophan metabolite produced by gut bacteria, is a potent anti-non-alcoholic steatohepatitis (NASH) microbial metabolite. Here, we demonstrate that administration of IPA modulates the microbiota composition in the gut and inhibits microbial dysbiosis in rats fed a high-fat diet. IPA induces the expression of tight junction proteins, such as ZO-1 and Occludin, and maintains intestinal epithelium homeostasis, leading to a reduction in plasma endotoxin levels. Interestingly, IPA inhibits NF-κB signaling and reduces the levels of proinflammatory cytokines, such as TNFα, IL-1β, and IL-6, in response to endotoxin in macrophages to repress hepatic inflammation and liver injury. Moreover, IPA is sufficient to inhibit the expression of fibrogenic and collagen genes and attenuate diet-induced NASH phenotypes. The beneficial effects of IPA on the liver are likely mediated through inhibiting the production of endotoxin in the gut.',31506421),(3618,'human','Tryptophan','Indole-3-propionic acid (IPA), a tryptophan metabolite produced by gut bacteria, is a potent anti-non-alcoholic steatohepatitis (NASH) microbial metabolite. Here, we demonstrate that administration of IPA modulates the microbiota composition in the gut and inhibits microbial dysbiosis in rats fed a high-fat diet. IPA induces the expression of tight junction proteins, such as ZO-1 and Occludin, and maintains intestinal epithelium homeostasis, leading to a reduction in plasma endotoxin levels. Interestingly, IPA inhibits NF-κB signaling and reduces the levels of proinflammatory cytokines, such as TNFα, IL-1β, and IL-6, in response to endotoxin in macrophages to repress hepatic inflammation and liver injury. Moreover, IPA is sufficient to inhibit the expression of fibrogenic and collagen genes and attenuate diet-induced NASH phenotypes. The beneficial effects of IPA on the liver are likely mediated through inhibiting the production of endotoxin in the gut.',31506421),(3619,'human','null','Gut flora and plasma microbiome-related metabolites were evaluated by amplicon sequencing of the bacterial 16S ribosomal RNA gene and capillary electrophoresis time-of-flight mass spectrometry, respectively. HF patients were evaluated in both the Decomp and Comp phases during hospitalization. The phylum Actinobacteria was enriched in HF patients compared with control subjects. At the genus level, Bifiodobacterium was abundant while Megamonas was depleted in HF patients. Meanwhile, plasma concentration of trimethylamine N-oxide (TMAO), a gut microbiome-derived metabolite, was increased in HF patients. A correlation analysis revealed positive correlations between the abundance of the genus Escherichia/Shigella and levels of TMAO and indoxyl sulfate (IS, a microbe-dependent uremic toxin) in Comp HF . Escherichia/Shigella was more abundant in Decomp than in Comp HF (P=0.030).Conclusions: Our results suggest that gut microbiome composition and microbiome-related metabolites are altered in HF patient.',30487369),(3620,'human','null','Gut flora and plasma microbiome-related metabolites were evaluated by amplicon sequencing of the bacterial 16S ribosomal RNA gene and capillary electrophoresis time-of-flight mass spectrometry, respectively. HF patients were evaluated in both the Decomp and Comp phases during hospitalization. The phylum Actinobacteria was enriched in HF patients compared with control subjects. At the genus level, Bifiodobacterium was abundant while Megamonas was depleted in HF patients. Meanwhile, plasma concentration of trimethylamine N-oxide (TMAO), a gut microbiome-derived metabolite, was increased in HF patients. A correlation analysis revealed positive correlations between the abundance of the genus Escherichia/Shigella and levels of TMAO and indoxyl sulfate (IS, a microbe-dependent uremic toxin) in Comp HF . Escherichia/Shigella was more abundant in Decomp than in Comp HF (P=0.030).Conclusions: Our results suggest that gut microbiome composition and microbiome-related metabolites are altered in HF patient.',30487369),(3621,'human','null','Gut flora and plasma microbiome-related metabolites were evaluated by amplicon sequencing of the bacterial 16S ribosomal RNA gene and capillary electrophoresis time-of-flight mass spectrometry, respectively. HF patients were evaluated in both the Decomp and Comp phases during hospitalization. The phylum Actinobacteria was enriched in HF patients compared with control subjects. At the genus level, Bifiodobacterium was abundant while Megamonas was depleted in HF patients. Meanwhile, plasma concentration of trimethylamine N-oxide (TMAO), a gut microbiome-derived metabolite, was increased in HF patients. A correlation analysis revealed positive correlations between the abundance of the genus Escherichia/Shigella and levels of TMAO and indoxyl sulfate (IS, a microbe-dependent uremic toxin) in Comp HF . Escherichia/Shigella was more abundant in Decomp than in Comp HF (P=0.030).Conclusions: Our results suggest that gut microbiome composition and microbiome-related metabolites are altered in HF patient.',30487369),(3622,'human','null','Gut flora and plasma microbiome-related metabolites were evaluated by amplicon sequencing of the bacterial 16S ribosomal RNA gene and capillary electrophoresis time-of-flight mass spectrometry, respectively. HF patients were evaluated in both the Decomp and Comp phases during hospitalization. The phylum Actinobacteria was enriched in HF patients compared with control subjects. At the genus level, Bifiodobacterium was abundant while Megamonas was depleted in HF patients. Meanwhile, plasma concentration of trimethylamine N-oxide (TMAO), a gut microbiome-derived metabolite, was increased in HF patients. A correlation analysis revealed positive correlations between the abundance of the genus Escherichia/Shigella and levels of TMAO and indoxyl sulfate (IS, a microbe-dependent uremic toxin) in Comp HF . Escherichia/Shigella was more abundant in Decomp than in Comp HF (P=0.030).Conclusions: Our results suggest that gut microbiome composition and microbiome-related metabolites are altered in HF patient.',30487369),(3626,'human','Quercitrin','Ultra performance liquid chromatography/quadrupole-time-of-flight mass spectrometry (UPLC/Q-TOF MS) technique combined with MetabolynxTM software was used for analysis of the metabolites of quercitrin by the isolated human intestinal bacteria from the human feces. Four metabolites of quercitrin were detected and tentatively identified based on the characteristics of their protonated ions. The metabolites were metabolized by four main metabolic pathwaysincluding hydroxylation, demethylation, deglycosylation and ring-cleavage. Quercitrin was metabolized to the hydroxyquercitrin and desmethylquercitrin by the majority of the isolated intestinal bacteria such as Bacteroidessp. 54, and was degraded to the deglycosylated product quercetin by rhamnosidase and further ring cleavage metabolite 3,4-dihydroxybenzoic acid by the minority of the isolated bacteria such as Bacteroides sp. 45. The metabolic pathways and most of the metabolites of quercitrin were reported for the first time.',23754166),(3627,'human','Quercitrin','Ultra performance liquid chromatography/quadrupole-time-of-flight mass spectrometry (UPLC/Q-TOF MS) technique combined with MetabolynxTM software was used for analysis of the metabolites of quercitrin by the isolated human intestinal bacteria from the human feces. Four metabolites of quercitrin were detected and tentatively identified based on the characteristics of their protonated ions. The metabolites were metabolized by four main metabolic pathwaysincluding hydroxylation, demethylation, deglycosylation and ring-cleavage. Quercitrin was metabolized to the hydroxyquercitrin and desmethylquercitrin by the majority of the isolated intestinal bacteria such as Bacteroidessp. 54, and was degraded to the deglycosylated product quercetin by rhamnosidase and further ring cleavage metabolite 3,4-dihydroxybenzoic acid by the minority of the isolated bacteria such as Bacteroides sp. 45. The metabolic pathways and most of the metabolites of quercitrin were reported for the first time.',23754166),(3628,'human','Quercitrin','Ultra performance liquid chromatography/quadrupole-time-of-flight mass spectrometry (UPLC/Q-TOF MS) technique combined with MetabolynxTM software was used for analysis of the metabolites of quercitrin by the isolated human intestinal bacteria from the human feces. Four metabolites of quercitrin were detected and tentatively identified based on the characteristics of their protonated ions. The metabolites were metabolized by four main metabolic pathwaysincluding hydroxylation, demethylation, deglycosylation and ring-cleavage. Quercitrin was metabolized to the hydroxyquercitrin and desmethylquercitrin by the majority of the isolated intestinal bacteria such as Bacteroidessp. 54, and was degraded to the deglycosylated product quercetin by rhamnosidase and further ring cleavage metabolite 3,4-dihydroxybenzoic acid by the minority of the isolated bacteria such as Bacteroides sp. 45. The metabolic pathways and most of the metabolites of quercitrin were reported for the first time.',23754166),(3632,'human','null','Plasma from rats treated for seven days with the non-absorbed antibiotic vancomycin or a mixture of streptomycin, neomycin, polymyxin B and bacitracin was analyzed using mass spectrometry-based metabolite profiling platforms. Antibiotic-induced changes in the abundance of individual groups of intestinal microbiota dramatically altered the host\'s metabolism. Hierarchical clustering of dissimilarities separated the levels of 284 identified metabolites from treated vs. untreated rats; 193 were altered by the antibiotic treatments with a tendency towards decreased metabolite levels. Catabolism of the aromatic amino acids phenylalanine, tryptophan and tyrosine was the most affected pathway comprising 33 affected metabolites. Vancomycin-treated rats were administered amino acid metabolites prior to ischemia/reperfusion studies. Oral or intravenous pretreatment of rats with these amino acid metabolites abolished the decrease in infarct size conferred by vancomycin. Cardioprotection by vancomycin is mediated by activation of pro-survival kinases Akt, p42/44 MAPK and p39 MAPK, as well as activation of KATP channels, classical mediators and effectors of pharmacologic cardioprotection. These proteins are components of known intersecting and interdependent signaling pathways in the myocardium that determine severity of myocardial infarction.',27505423),(3633,'human','null','Plasma from rats treated for seven days with the non-absorbed antibiotic vancomycin or a mixture of streptomycin, neomycin, polymyxin B and bacitracin was analyzed using mass spectrometry-based metabolite profiling platforms. Antibiotic-induced changes in the abundance of individual groups of intestinal microbiota dramatically altered the host\'s metabolism. Hierarchical clustering of dissimilarities separated the levels of 284 identified metabolites from treated vs. untreated rats; 193 were altered by the antibiotic treatments with a tendency towards decreased metabolite levels. Catabolism of the aromatic amino acids phenylalanine, tryptophan and tyrosine was the most affected pathway comprising 33 affected metabolites. Vancomycin-treated rats were administered amino acid metabolites prior to ischemia/reperfusion studies. Oral or intravenous pretreatment of rats with these amino acid metabolites abolished the decrease in infarct size conferred by vancomycin. Cardioprotection by vancomycin is mediated by activation of pro-survival kinases Akt, p42/44 MAPK and p40 MAPK, as well as activation of KATP channels, classical mediators and effectors of pharmacologic cardioprotection. These proteins are components of known intersecting and interdependent signaling pathways in the myocardium that determine severity of myocardial infarction.',27505423),(3634,'human','null','Plasma from rats treated for seven days with the non-absorbed antibiotic vancomycin or a mixture of streptomycin, neomycin, polymyxin B and bacitracin was analyzed using mass spectrometry-based metabolite profiling platforms. Antibiotic-induced changes in the abundance of individual groups of intestinal microbiota dramatically altered the host\'s metabolism. Hierarchical clustering of dissimilarities separated the levels of 284 identified metabolites from treated vs. untreated rats; 193 were altered by the antibiotic treatments with a tendency towards decreased metabolite levels. Catabolism of the aromatic amino acids phenylalanine, tryptophan and tyrosine was the most affected pathway comprising 33 affected metabolites. Vancomycin-treated rats were administered amino acid metabolites prior to ischemia/reperfusion studies. Oral or intravenous pretreatment of rats with these amino acid metabolites abolished the decrease in infarct size conferred by vancomycin. Cardioprotection by vancomycin is mediated by activation of pro-survival kinases Akt, p42/44 MAPK and p41 MAPK, as well as activation of KATP channels, classical mediators and effectors of pharmacologic cardioprotection. These proteins are components of known intersecting and interdependent signaling pathways in the myocardium that determine severity of myocardial infarction.',27505423),(3635,'human','null','Plasma from rats treated for seven days with the non-absorbed antibiotic vancomycin or a mixture of streptomycin, neomycin, polymyxin B and bacitracin was analyzed using mass spectrometry-based metabolite profiling platforms. Antibiotic-induced changes in the abundance of individual groups of intestinal microbiota dramatically altered the host\'s metabolism. Hierarchical clustering of dissimilarities separated the levels of 284 identified metabolites from treated vs. untreated rats; 193 were altered by the antibiotic treatments with a tendency towards decreased metabolite levels. Catabolism of the aromatic amino acids phenylalanine, tryptophan and tyrosine was the most affected pathway comprising 33 affected metabolites. Vancomycin-treated rats were administered amino acid metabolites prior to ischemia/reperfusion studies. Oral or intravenous pretreatment of rats with these amino acid metabolites abolished the decrease in infarct size conferred by vancomycin. Cardioprotection by vancomycin is mediated by activation of pro-survival kinases Akt, p42/44 MAPK and p42 MAPK, as well as activation of KATP channels, classical mediators and effectors of pharmacologic cardioprotection. These proteins are components of known intersecting and interdependent signaling pathways in the myocardium that determine severity of myocardial infarction.',27505423),(3636,'human','null','Plasma from rats treated for seven days with the non-absorbed antibiotic vancomycin or a mixture of streptomycin, neomycin, polymyxin B and bacitracin was analyzed using mass spectrometry-based metabolite profiling platforms. Antibiotic-induced changes in the abundance of individual groups of intestinal microbiota dramatically altered the host\'s metabolism. Hierarchical clustering of dissimilarities separated the levels of 284 identified metabolites from treated vs. untreated rats; 193 were altered by the antibiotic treatments with a tendency towards decreased metabolite levels. Catabolism of the aromatic amino acids phenylalanine, tryptophan and tyrosine was the most affected pathway comprising 33 affected metabolites. Vancomycin-treated rats were administered amino acid metabolites prior to ischemia/reperfusion studies. Oral or intravenous pretreatment of rats with these amino acid metabolites abolished the decrease in infarct size conferred by vancomycin. Cardioprotection by vancomycin is mediated by activation of pro-survival kinases Akt, p42/44 MAPK and p43 MAPK, as well as activation of KATP channels, classical mediators and effectors of pharmacologic cardioprotection. These proteins are components of known intersecting and interdependent signaling pathways in the myocardium that determine severity of myocardial infarction.',27505423),(3637,'human','D-Sorbitol','Three sequential fermentative batches were carried out with cell recycle in four simultaneously operating bioreactors maintained at pH 6.5, 30°C, and 100 rpm. P. acidipropionici ATCC 4875 was able to produce propionic and succinic acid from sorbitol. The concentration of propionic acid decreased slightly from 39.5 ± 5.2 g L�? to 34.4 ± 1.9 g L�?,and that of succinic acid increased significantly from 6.1 ± 2.1 g L�? to 14.8 ± 0.9 g L�? through the sequential batches. In addition, a small amount of acetic acid was produced that decreased from 3.3 ± 0.4 g L�? to 2.0 ± 0.3 g L�? through the batches. The major yield for propionic acid was 0.613 g g�? in the first batch and succinic acid it was 0.212 g g�? in the third batch. The minor yield of acetic acid was 0.029 g g�? ,in the second and third batches.',25852990),(3638,'human','D-Sorbitol','Three sequential fermentative batches were carried out with cell recycle in four simultaneously operating bioreactors maintained at pH 6.5, 30°C, and 100 rpm. P. acidipropionici ATCC 4875 was able to produce propionic and succinic acid from sorbitol. The concentration of propionic acid decreased slightly from 39.5 ± 5.2 g L�? to 34.4 ± 1.9 g L�?,and that of succinic acid increased significantly from 6.1 ± 2.1 g L�? to 14.8 ± 0.9 g L�? through the sequential batches. In addition, a small amount of acetic acid was produced that decreased from 3.3 ± 0.4 g L�? to 2.0 ± 0.3 g L�? through the batches. The major yield for propionic acid was 0.613 g g�? in the first batch and succinic acid it was 0.212 g g�? in the third batch. The minor yield of acetic acid was 0.029 g g�? ,in the second and third batches.',25852990),(3639,'human','null','In the present study, we found that TMAO significantly upregulated the expression of proinflammatory markers, such as IL-1β, IL-6, and TNF-α in TH positive neurons, as well as NMDARs, which might further result in sympathetic hyperactivation within the LSG and promote the incidence of VA.',30954457),(3640,'human','null','In the present study, we found that TMAO significantly upregulated the expression of proinflammatory markers, such as IL-1β, IL-6, and TNF-α in TH positive neurons, as well as NMDARs, which might further result in sympathetic hyperactivation within the LSG and promote the incidence of VA.',30954457),(3641,'human','null','In the present study, we found that TMAO significantly upregulated the expression of proinflammatory markers, such as IL-1β, IL-6, and TNF-α in TH positive neurons, as well as NMDARs, which might further result in sympathetic hyperactivation within the LSG and promote the incidence of VA.',30954457),(3642,'human','4-N-(3-Chloro-4-fluorophenyl)-6-N-[1-(iodomethyl)piperidin-4-yl]pyrimido[5,4-d]pyrimidine-4,6-diamine','The present study identified the metabolites from DN through analysis of organ-specific biotransformation and validated the cardioprotective effects of the screened metabolites using an isoprenaline-induced myocardial ischemia rat model. The findings of the present study provide evidence that, first, diosgenin is generated from DN by intestinal microflora; second, diosgenin can protect the myocardium against ischemic insult in a dose-dependent manner, almost comparable to the effect of DN extract; and, third, DN\'s protective effect can be attributed to the increase of enzymatic and nonenzymatic antioxidants (SOD, CAT, GPx, and T-AOC) in vivo and to a decrease in lipid peroxidation. These phenomena help explain the clinical efficacy of DN as an anti-MI drug.',29085554),(3643,'human','4-N-(3-Chloro-4-fluorophenyl)-6-N-[1-(iodomethyl)piperidin-4-yl]pyrimido[5,4-d]pyrimidine-4,6-diamine','The present study identified the metabolites from DN through analysis of organ-specific biotransformation and validated the cardioprotective effects of the screened metabolites using an isoprenaline-induced myocardial ischemia rat model. The findings of the present study provide evidence that, first, diosgenin is generated from DN by intestinal microflora; second, diosgenin can protect the myocardium against ischemic insult in a dose-dependent manner, almost comparable to the effect of DN extract; and, third, DN\'s protective effect can be attributed to the increase of enzymatic and nonenzymatic antioxidants (SOD, CAT, GPx, and T-AOC) in vivo and to a decrease in lipid peroxidation. These phenomena help explain the clinical efficacy of DN as an anti-MI drug.',29085554),(3644,'human','4-N-(3-Chloro-4-fluorophenyl)-6-N-[1-(iodomethyl)piperidin-4-yl]pyrimido[5,4-d]pyrimidine-4,6-diamine','The present study identified the metabolites from DN through analysis of organ-specific biotransformation and validated the cardioprotective effects of the screened metabolites using an isoprenaline-induced myocardial ischemia rat model. The findings of the present study provide evidence that, first, diosgenin is generated from DN by intestinal microflora; second, diosgenin can protect the myocardium against ischemic insult in a dose-dependent manner, almost comparable to the effect of DN extract; and, third, DN\'s protective effect can be attributed to the increase of enzymatic and nonenzymatic antioxidants (SOD, CAT, GPx, and T-AOC) in vivo and to a decrease in lipid peroxidation. These phenomena help explain the clinical efficacy of DN as an anti-MI drug.',29085554),(3645,'human','Arbutin','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.',21889493),(3646,'human','Arbutin','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.',21889493),(3647,'human','Arbutin','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.',21889493),(3648,'human','Arbutin','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.',21889493),(3649,'human','Arbutin','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.',21889493),(3650,'human','Arbutin','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.',21889493),(3651,'human','Arbutin','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.',21889493),(3652,'human','Arbutin','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.',21889493),(3653,'human','Arbutin','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.',21889493),(3654,'human','Arbutin','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.',21889493),(3655,'human','Arbutin','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.',21889493),(3656,'human','Arbutin','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.',21889493),(3667,'human','Dihydrodaidzein','In this study, chemically synthesized dihydrodaidzein was used as the substrate administered to a newly isolated rodshaped, gram-negative, anaerobic bacterium from human feces which can convert dihydrodaidzein to S-equol.',29112296),(3668,'human','daidzein','Clostridium sp. strain HGH6, that is capable of reducing daidzein to dihydrodaidzein, but further metabolism to tetrahydrodaidzein or equol did not occur.',15640190),(3669,'human','null','Our result also demonstrated that butyrate intervention enhanced fecal SCFAs concentrations, and inhibited endotoxin levels in feces and serum. Correlation analysis indicated positive relation between endotoxin level and Desulfovibrionaceae abundance. Furthermore,  butyrate intervention inhibited expressions of IL-1β, IL-6 and MCP1/CCL2 in liver, as well as TLR4 in adipose tissue.',15640190),(3670,'human','null','Our result also demonstrated that butyrate intervention enhanced fecal SCFAs concentrations, and inhibited endotoxin levels in feces and serum. Correlation analysis indicated positive relation between endotoxin level and Desulfovibrionaceae abundance. Furthermore,  butyrate intervention inhibited expressions of IL-1β, IL-6 and MCP1/CCL2 in liver, as well as TLR4 in adipose tissue.',31295342),(3671,'human','null','Our result also demonstrated that butyrate intervention enhanced fecal SCFAs concentrations, and inhibited endotoxin levels in feces and serum. Correlation analysis indicated positive relation between endotoxin level and Desulfovibrionaceae abundance. Furthermore,  butyrate intervention inhibited expressions of IL-1β, IL-6 and MCP1/CCL2 in liver, as well as TLR4 in adipose tissue.',31295342),(3672,'human','null','Our result also demonstrated that butyrate intervention enhanced fecal SCFAs concentrations, and inhibited endotoxin levels in feces and serum. Correlation analysis indicated positive relation between endotoxin level and Desulfovibrionaceae abundance. Furthermore,  butyrate intervention inhibited expressions of IL-1β, IL-6 and MCP1/CCL2 in liver, as well as TLR4 in adipose tissue.',31295342),(3673,'human','Biochanin A','Eubacterium limosum (ATCC 8486), a strict anaerobe from the human intestinal tract that is capable of O-demethylation of several compounds, was tested for the ability to metabolize three methoxylated isoflavonoids, biochanin A, formononetin, and glycitein. High performance liquid chromatography elution profiles of metabolites produced from biochanin A, formononetin, and glycitein showed peaks that had identical retention times to authentic genistein, daidzein, and 6,7,4P-trihydroxyisoflavone, respectively.',31295342),(3674,'human','Formononetin','Eubacterium limosum (ATCC 8486), a strict anaerobe from the human intestinal tract that is capable of O-demethylation of several compounds, was tested for the ability to metabolize three methoxylated isoflavonoids, biochanin A, formononetin, and glycitein. High performance liquid chromatography elution profiles of metabolites produced from biochanin A, formononetin, and glycitein showed peaks that had identical retention times to authentic genistein, daidzein, and 6,7,4P-trihydroxyisoflavone, respectively.',11040423),(3675,'human','Glycitein','Eubacterium limosum (ATCC 8486), a strict anaerobe from the human intestinal tract that is capable of O-demethylation of several compounds, was tested for the ability to metabolize three methoxylated isoflavonoids, biochanin A, formononetin, and glycitein. High performance liquid chromatography elution profiles of metabolites produced from biochanin A, formononetin, and glycitein showed peaks that had identical retention times to authentic genistein, daidzein, and 6,7,4P-trihydroxyisoflavone, respectively.',11040423),(3680,'human','(-)-Epigallocatechin gallate','The results showed that L. plantarum IFPL935 exhibited higher resistance to nongalloylated monomeric flavan-3-ols, A-type dimeric procyanidins, and cranberry extract than to (�?-epicatechin-3-O-gallate and B-type dimeric procyanidins. Despite these findings,the strain was capable of rapidly degrading (�?-epicatechin-3-O-gallate, but not A- or B-type dimeric procyanidins. However, it was able to produce large changes in the phenolic profile of the cranberry extract mainly due to the catabolism of hydroxycinnamic and hydroxybenzoic acids. Of most relevance was the fact that L. plantarum IFPL935 cleaved the heterocyclic ring of monomeric flavan-3-ols, giving rise to 1-(3�?4�?dihydroxyphenyl)-3-(2�?4�?6�?trihydroxyphenyl)propan-2-ol, activity exhibited by only a few human intestinal bacteria.',22646528),(3681,'human','(-)-Epigallocatechin gallate','The results showed that L. plantarum IFPL935 exhibited higher resistance to nongalloylated monomeric flavan-3-ols, A-type dimeric procyanidins, and cranberry extract than to (�?-epicatechin-3-O-gallate and B-type dimeric procyanidins. Despite these findings,the strain was capable of rapidly degrading (�?-epicatechin-3-O-gallate, but not A- or B-type dimeric procyanidins. However, it was able to produce large changes in the phenolic profile of the cranberry extract mainly due to the catabolism of hydroxycinnamic and hydroxybenzoic acids. Of most relevance was the fact that L. plantarum IFPL935 cleaved the heterocyclic ring of monomeric flavan-3-ols, giving rise to 1-(3�?4�?dihydroxyphenyl)-3-(2�?4�?6�?trihydroxyphenyl)propan-2-ol, activity exhibited by only a few human intestinal bacteria.',22646528),(3682,'human','null','One important secondary metabolite of bifidobacteria is folate, which contributes to the one-carbon metabolism of the host and therefore enables maintenance of DNA methylation marks as explained below (41).',23594611),(3683,'human','null','Furthermore, TMAO increased TGF-β receptor I expression, which promoted the phosphorylation of Smad2 and up-regulated the expression of α-SMA and collagen I. The ubiquitination of TGF-βRI was decreased in neonatal mouse fibroblasts after TMAO treatment. TMAO also inhibited the expression of smurf2. Inhibition of TGF-β1 receptor with the small molecule inhibitor SB431542 decreased TGF-β receptor I expression, reduced the phosphorylation of Smad2, down-regulated TMAO-induced α-SMA and collagen I expression in cardiac fibroblasts. Conclusions: Cardiac function and cardiac fibrosis were significantly exacerbated in mice fed diets supplemented with either choline or TMAO, probably through accelerating the transformation of fibroblasts into myofibroblasts, indicating activation of TGF-βRI/Smad2 pathway',31299216),(3684,'human','null','Furthermore, TMAO increased TGF-β receptor I expression, which promoted the phosphorylation of Smad2 and up-regulated the expression of α-SMA and collagen I. The ubiquitination of TGF-βRI was decreased in neonatal mouse fibroblasts after TMAO treatment. TMAO also inhibited the expression of smurf2. Inhibition of TGF-β1 receptor with the small molecule inhibitor SB431542 decreased TGF-β receptor I expression, reduced the phosphorylation of Smad2, down-regulated TMAO-induced α-SMA and collagen I expression in cardiac fibroblasts. Conclusions: Cardiac function and cardiac fibrosis were significantly exacerbated in mice fed diets supplemented with either choline or TMAO, probably through accelerating the transformation of fibroblasts into myofibroblasts, indicating activation of TGF-βRI/Smad2 pathway',31299216),(3685,'human','null','Furthermore, TMAO increased TGF-β receptor I expression, which promoted the phosphorylation of Smad2 and up-regulated the expression of α-SMA and collagen I. The ubiquitination of TGF-βRI was decreased in neonatal mouse fibroblasts after TMAO treatment. TMAO also inhibited the expression of smurf2. Inhibition of TGF-β1 receptor with the small molecule inhibitor SB431542 decreased TGF-β receptor I expression, reduced the phosphorylation of Smad2, down-regulated TMAO-induced α-SMA and collagen I expression in cardiac fibroblasts. Conclusions: Cardiac function and cardiac fibrosis were significantly exacerbated in mice fed diets supplemented with either choline or TMAO, probably through accelerating the transformation of fibroblasts into myofibroblasts, indicating activation of TGF-βRI/Smad2 pathway',31299216),(3686,'human','Glycitein','The dichotomy between the biotransformation of polyphenols into their metabolites by gut microbiota and the modulation of gut microbiota composition by polyphenols contributes to positive health outcomes. Although there are many studies on the in vivo bioavailability of polyphenols, the mutual relationship between polyphenols and gut microbiota is not fully understood. This review focuses on the biotransformation of polyphenols by gut microbiota, modulation of gut microbiota by polyphenols, and the effects of these two-way mutual interactions on polyphenol bioavailability, and ultimately, human health.',26861391),(3687,'human','Glycitein','The dichotomy between the biotransformation of polyphenols into their metabolites by gut microbiota and the modulation of gut microbiota composition by polyphenols contributes to positive health outcomes. Although there are many studies on the in vivo bioavailability of polyphenols, the mutual relationship between polyphenols and gut microbiota is not fully understood. This review focuses on the biotransformation of polyphenols by gut microbiota, modulation of gut microbiota by polyphenols, and the effects of these two-way mutual interactions on polyphenol bioavailability, and ultimately, human health.',26861391),(3688,'human','Chlorogenic acid','Chlorogenic acid hydrolysing activities were essentially intracellular. These cinnamoyl esterase-producing organisms were devoid of other phenolic-degrading activities. The results show that certain gut bacteria, including some already recognized as potentially health-promoting (i.e. species belonging to the genera Bifidobacterium and Lactobacillus), are involved in the release of bioactive hydroxycinnamic acids in the human colon.',11412317),(3689,'human','Chlorogenic acid','Chlorogenic acid hydrolysing activities were essentially intracellular. These cinnamoyl esterase-producing organisms were devoid of other phenolic-degrading activities. The results show that certain gut bacteria, including some already recognized as potentially health-promoting (i.e. species belonging to the genera Bifidobacterium and Lactobacillus), are involved in the release of bioactive hydroxycinnamic acids in the human colon.',11412317),(3690,'human','Chlorogenic acid','Chlorogenic acid hydrolysing activities were essentially intracellular. These cinnamoyl esterase-producing organisms were devoid of other phenolic-degrading activities. The results show that certain gut bacteria, including some already recognized as potentially health-promoting (i.e. species belonging to the genera Bifidobacterium and Lactobacillus), are involved in the release of bioactive hydroxycinnamic acids in the human colon.',11412317),(3691,'human','Chlorogenic acid','Chlorogenic acid hydrolysing activities were essentially intracellular. These cinnamoyl esterase-producing organisms were devoid of other phenolic-degrading activities. The results show that certain gut bacteria, including some already recognized as potentially health-promoting (i.e. species belonging to the genera Bifidobacterium and Lactobacillus), are involved in the release of bioactive hydroxycinnamic acids in the human colon.',11412317),(3692,'human','Chlorogenic acid','Chlorogenic acid hydrolysing activities were essentially intracellular. These cinnamoyl esterase-producing organisms were devoid of other phenolic-degrading activities. The results show that certain gut bacteria, including some already recognized as potentially health-promoting (i.e. species belonging to the genera Bifidobacterium and Lactobacillus), are involved in the release of bioactive hydroxycinnamic acids in the human colon.',11412317),(3693,'human','Chlorogenic acid','Chlorogenic acid hydrolysing activities were essentially intracellular. These cinnamoyl esterase-producing organisms were devoid of other phenolic-degrading activities. The results show that certain gut bacteria, including some already recognized as potentially health-promoting (i.e. species belonging to the genera Bifidobacterium and Lactobacillus), are involved in the release of bioactive hydroxycinnamic acids in the human colon.',11412317),(3694,'human','D-Glucose','To gain some specific insight into the roles microorganisms might play in non-alcoholic fatty liver disease (NAFLD), some intestinal and lactic acid bacteria and one yeast  were characterized by high performance liquid chromatography for production of ethanol when grown on different carbohydrates. Highest amounts of ethanol were produced by S. cerevisiae, L. fermentum, and W. confusa on glucose and by S. cerevisiae and W. confusa on fructose.',26858714),(3695,'human','D-Glucose','To gain some specific insight into the roles microorganisms might play in non-alcoholic fatty liver disease (NAFLD), some intestinal and lactic acid bacteria and one yeast  were characterized by high performance liquid chromatography for production of ethanol when grown on different carbohydrates. Highest amounts of ethanol were produced by S. cerevisiae, L. fermentum, and W. confusa on glucose and by S. cerevisiae and W. confusa on fructose.',26858714),(3696,'human','D-Fructose','To gain some specific insight into the roles microorganisms might play in non-alcoholic fatty liver disease (NAFLD), some intestinal and lactic acid bacteria and one yeast  were characterized by high performance liquid chromatography for production of ethanol when grown on different carbohydrates. Highest amounts of ethanol were produced by S. cerevisiae, L. fermentum, and W. confusa on glucose and by S. cerevisiae and W. confusa on fructose.',26858714),(3697,'human','D-Glucose','To gain some specific insight into the roles microorganisms might play in non-alcoholic fatty liver disease (NAFLD), some intestinal and lactic acid bacteria and one yeast  were characterized by high performance liquid chromatography for production of ethanol when grown on different carbohydrates. Highest amounts of ethanol were produced by S. cerevisiae, L. fermentum, and W. confusa on glucose and by S. cerevisiae and W. confusa on fructose.',26858714),(3698,'human','D-Fructose','To gain some specific insight into the roles microorganisms might play in non-alcoholic fatty liver disease (NAFLD), some intestinal and lactic acid bacteria and one yeast  were characterized by high performance liquid chromatography for production of ethanol when grown on different carbohydrates. Highest amounts of ethanol were produced by S. cerevisiae, L. fermentum, and W. confusa on glucose and by S. cerevisiae and W. confusa on fructose.',26858714),(3699,'human','Phosphoenolpyruvate','Our attempt to revisit classic literature has revealed many gaps in our knowledge that can now be examined in the light of modern systems approaches. At the same time, descriptive sequence-based studies are serving an important complementary role in defining priorities for pursuing mechanism-level understanding of this otherwise exceedingly complex interplay between the microbial species that inhabit humans and the resulting interactions that occur between these species and the host.',22018234),(3700,'human','Fumarate','Our attempt to revisit classic literature has revealed many gaps in our knowledge that can now be examined in the light of modern systems approaches. At the same time, descriptive sequence-based studies are serving an important complementary role in defining priorities for pursuing mechanism-level understanding of this otherwise exceedingly complex interplay between the microbial species that inhabit humans and the resulting interactions that occur between these species and the host.',22018234),(3701,'human','Succinate','Our attempt to revisit classic literature has revealed many gaps in our knowledge that can now be examined in the light of modern systems approaches. At the same time, descriptive sequence-based studies are serving an important complementary role in defining priorities for pursuing mechanism-level understanding of this otherwise exceedingly complex interplay between the microbial species that inhabit humans and the resulting interactions that occur between these species and the host.',22018234),(3702,'human','Acetate','Our attempt to revisit classic literature has revealed many gaps in our knowledge that can now be examined in the light of modern systems approaches. At the same time, descriptive sequence-based studies are serving an important complementary role in defining priorities for pursuing mechanism-level understanding of this otherwise exceedingly complex interplay between the microbial species that inhabit humans and the resulting interactions that occur between these species and the host.',22018234),(3703,'human','D-Glucose','Our attempt to revisit classic literature has revealed many gaps in our knowledge that can now be examined in the light of modern systems approaches. At the same time, descriptive sequence-based studies are serving an important complementary role in defining priorities for pursuing mechanism-level understanding of this otherwise exceedingly complex interplay between the microbial species that inhabit humans and the resulting interactions that occur between these species and the host.',22018234),(3704,'human','D-Glucose','Our attempt to revisit classic literature has revealed many gaps in our knowledge that can now be examined in the light of modern systems approaches. At the same time, descriptive sequence-based studies are serving an important complementary role in defining priorities for pursuing mechanism-level understanding of this otherwise exceedingly complex interplay between the microbial species that inhabit humans and the resulting interactions that occur between these species and the host.',22018234),(3705,'human','Daidzein','Equol is a metabolite produced from daidzein by enteric microflora, and it has attracted a great deal of attention because of its protective or ameliorative ability against several sex hormone-dependent diseases (e.g., menopausal disorder and lower bone density), which is more potent than that of other isoflavonoids. We purified a novel NADP(H)-dependent daidzein reductase (L-DZNR) from Lactococcus strain 20-92 (Lactococcus 20-92; S. Uchiyama, T. Ueno, and T. Suzuki, international patent WO2005/000042) that is involved in the metabolism of soy isoflavones and equol production and converts daidzein to dihydrodaidzein.',20639368),(3706,'human','Daidzein','Equol is a metabolite produced from daidzein by enteric microflora, and it has attracted a great deal of attention because of its protective or ameliorative ability against several sex hormone-dependent diseases (e.g., menopausal disorder and lower bone density), which is more potent than that of other isoflavonoids. We purified a novel NADP(H)-dependent daidzein reductase (L-DZNR) from Lactococcus strain 20-92 (Lactococcus 20-92; S. Uchiyama, T. Ueno, and T. Suzuki, international patent WO2005/000042) that is involved in the metabolism of soy isoflavones and equol production and converts daidzein to dihydrodaidzein.',20639368),(3707,'human','Mangiferin','The C-glucosyl bond of C-glucosides generally tolerates acid and enzymatic hydrolysis. Many C-glucosides are cleaved by human intestinal bacteria. We isolated the specific bacterium involved in the metabolism of mangiferin (2-b-D-glucopyranosyl-1,3,6,7-tetrahydroxyxanthone), C-glucosyl xanthone, from a mixture of human fecal bacteria. The anaerobic Bacteroides species named MANG, transformed mangiferin to the aglycone, norathyriol, suggesting cleavage of a C-glucosyl bond.',16141538),(3708,'human','Mangiferin','The C-glucosyl bond of C-glucosides generally tolerates acid and enzymatic hydrolysis. Many C-glucosides are cleaved by human intestinal bacteria. We isolated the specific bacterium involved in the metabolism of mangiferin (2-b-D-glucopyranosyl-1,3,6,7-tetrahydroxyxanthone), C-glucosyl xanthone, from a mixture of human fecal bacteria. The anaerobic Bacteroides species named MANG, transformed mangiferin to the aglycone, norathyriol, suggesting cleavage of a C-glucosyl bond.',16141538),(3709,'human','null','5-HT storing enterochromaffin (EC) cells are believed to respond to nutrient and gut microbial components, and 5-HT receptor-expressing afferent vagal neurons have been described to be the major sensors of nutrients in the GI-tract. However, the molecular mechanism through which EC cells sense nutrients and gut microbiota is still unclear.Nutrient metabolites apparently do not stimulate EC cells of the small intestine directly but through a paracrine mechanism involving GLP-1 secreted from neighboring enteroendocrine cells. In contrast, colonic EC cells are able to sense a multitude of different metabolites generated by the gut microbiota as well as gut hormones, including GLP-1.',29576437),(3710,'human','null','5-HT storing enterochromaffin (EC) cells are believed to respond to nutrient and gut microbial components, and 5-HT receptor-expressing afferent vagal neurons have been described to be the major sensors of nutrients in the GI-tract. However, the molecular mechanism through which EC cells sense nutrients and gut microbiota is still unclear.Nutrient metabolites apparently do not stimulate EC cells of the small intestine directly but through a paracrine mechanism involving GLP-1 secreted from neighboring enteroendocrine cells. In contrast, colonic EC cells are able to sense a multitude of different metabolites generated by the gut microbiota as well as gut hormones, including GLP-1.',29576437),(3711,'human','null','5-HT storing enterochromaffin (EC) cells are believed to respond to nutrient and gut microbial components, and 5-HT receptor-expressing afferent vagal neurons have been described to be the major sensors of nutrients in the GI-tract. However, the molecular mechanism through which EC cells sense nutrients and gut microbiota is still unclear.Nutrient metabolites apparently do not stimulate EC cells of the small intestine directly but through a paracrine mechanism involving GLP-1 secreted from neighboring enteroendocrine cells. In contrast, colonic EC cells are able to sense a multitude of different metabolites generated by the gut microbiota as well as gut hormones, including GLP-1.',29576437),(3712,'human','null','The E.limosum produced butyrate, acetate, propionate, and lactate at 0.25, 1.0, 0.025 and 0.07 mmol/L, respectively in medium. At this concentration, each acid had no growth stimulating activity on T84 cells; however, when these acids were mixed together at the above levels, it showed significantly high activity than control. Except for lactate, these acids significantly attenuated IL-6 production at just 0.1 mmol/L. In addition, under TNF-a stimulation, butyrate attenuated the production of TLR4 mRNA. The treatment with E.limosum significantly attenuated clinical and histological scores of colitis with an increase of cecal butyrate levels, compared with the control group.',16534848),(3713,'human','null','The E.limosum produced butyrate, acetate, propionate, and lactate at 0.25, 1.0, 0.025 and 0.07 mmol/L, respectively in medium. At this concentration, each acid had no growth stimulating activity on T84 cells; however, when these acids were mixed together at the above levels, it showed significantly high activity than control. Except for lactate, these acids significantly attenuated IL-6 production at just 0.1 mmol/L. In addition, under TNF-a stimulation, butyrate attenuated the production of TLR4 mRNA. The treatment with E.limosum significantly attenuated clinical and histological scores of colitis with an increase of cecal butyrate levels, compared with the control group.',16534848),(3714,'human','null','The E.limosum produced butyrate, acetate, propionate, and lactate at 0.25, 1.0, 0.025 and 0.07 mmol/L, respectively in medium. At this concentration, each acid had no growth stimulating activity on T84 cells; however, when these acids were mixed together at the above levels, it showed significantly high activity than control. Except for lactate, these acids significantly attenuated IL-6 production at just 0.1 mmol/L. In addition, under TNF-a stimulation, butyrate attenuated the production of TLR4 mRNA. The treatment with E.limosum significantly attenuated clinical and histological scores of colitis with an increase of cecal butyrate levels, compared with the control group.',16534848),(3715,'human','null','The E.limosum produced butyrate, acetate, propionate, and lactate at 0.25, 1.0, 0.025 and 0.07 mmol/L, respectively in medium. At this concentration, each acid had no growth stimulating activity on T84 cells; however, when these acids were mixed together at the above levels, it showed significantly high activity than control. Except for lactate, these acids significantly attenuated IL-6 production at just 0.1 mmol/L. In addition, under TNF-a stimulation, butyrate attenuated the production of TLR4 mRNA. The treatment with E.limosum significantly attenuated clinical and histological scores of colitis with an increase of cecal butyrate levels, compared with the control group.',16534848),(3716,'human','Rutin','In this paper, rutin was metabolized by human intestinal bacteria and five isolated strains including Bacillus sp. 52, Bacteroides sp. 45, 42, 22 and Veillonella sp. 32, the metabolites were identified using ultra performance liquid chromatography/quadrupole-time-of-flight mass spectrometry (UPLC–Q-TOF/MS). As a result, Bacillus sp. 52 and Bacteroides sp. 45 could metabolize rutin to quercetin 3-O-glucoside and leucocyanidin. Bacteroides sp. 42 and Veillonella sp. 32 could convert rutin to leucocyanidin. Bacteroides sp. 22 could hydrolyze rutin to quercetin-3-O-glucoside. In order to further explain the metabolism pathway of rutin,the -d-glucosidase and -l-rhamnosidase activities offive strains weredetermined.',22583754),(3717,'human','Rutin','In this paper, rutin was metabolized by human intestinal bacteria and five isolated strains including Bacillus sp. 52, Bacteroides sp. 45, 42, 22 and Veillonella sp. 32, the metabolites were identified using ultra performance liquid chromatography/quadrupole-time-of-flight mass spectrometry (UPLC–Q-TOF/MS). As a result, Bacillus sp. 52 and Bacteroides sp. 45 could metabolize rutin to quercetin 3-O-glucoside and leucocyanidin. Bacteroides sp. 42 and Veillonella sp. 32 could convert rutin to leucocyanidin. Bacteroides sp. 22 could hydrolyze rutin to quercetin-3-O-glucoside. In order to further explain the metabolism pathway of rutin,the -d-glucosidase and -l-rhamnosidase activities offive strains weredetermined.',22583754),(3718,'human','Rutin','In this paper, rutin was metabolized by human intestinal bacteria and five isolated strains including Bacillus sp. 52, Bacteroides sp. 45, 42, 22 and Veillonella sp. 32, the metabolites were identified using ultra performance liquid chromatography/quadrupole-time-of-flight mass spectrometry (UPLC–Q-TOF/MS). As a result, Bacillus sp. 52 and Bacteroides sp. 45 could metabolize rutin to quercetin 3-O-glucoside and leucocyanidin. Bacteroides sp. 42 and Veillonella sp. 32 could convert rutin to leucocyanidin. Bacteroides sp. 22 could hydrolyze rutin to quercetin-3-O-glucoside. In order to further explain the metabolism pathway of rutin,the -d-glucosidase and -l-rhamnosidase activities offive strains weredetermined.',22583754),(3719,'human','Rutin','In this paper, rutin was metabolized by human intestinal bacteria and five isolated strains including Bacillus sp. 52, Bacteroides sp. 45, 42, 22 and Veillonella sp. 32, the metabolites were identified using ultra performance liquid chromatography/quadrupole-time-of-flight mass spectrometry (UPLC–Q-TOF/MS). As a result, Bacillus sp. 52 and Bacteroides sp. 45 could metabolize rutin to quercetin 3-O-glucoside and leucocyanidin. Bacteroides sp. 42 and Veillonella sp. 32 could convert rutin to leucocyanidin. Bacteroides sp. 22 could hydrolyze rutin to quercetin-3-O-glucoside. In order to further explain the metabolism pathway of rutin,the -d-glucosidase and -l-rhamnosidase activities offive strains weredetermined.',22583754),(3720,'human','Rutin','In this paper, rutin was metabolized by human intestinal bacteria and five isolated strains including Bacillus sp. 52, Bacteroides sp. 45, 42, 22 and Veillonella sp. 32, the metabolites were identified using ultra performance liquid chromatography/quadrupole-time-of-flight mass spectrometry (UPLC–Q-TOF/MS). As a result, Bacillus sp. 52 and Bacteroides sp. 45 could metabolize rutin to quercetin 3-O-glucoside and leucocyanidin. Bacteroides sp. 42 and Veillonella sp. 32 could convert rutin to leucocyanidin. Bacteroides sp. 22 could hydrolyze rutin to quercetin-3-O-glucoside. In order to further explain the metabolism pathway of rutin,the -d-glucosidase and -l-rhamnosidase activities offive strains weredetermined.',22583754),(3721,'human','Rutin','In this paper, rutin was metabolized by human intestinal bacteria and five isolated strains including Bacillus sp. 52, Bacteroides sp. 45, 42, 22 and Veillonella sp. 32, the metabolites were identified using ultra performance liquid chromatography/quadrupole-time-of-flight mass spectrometry (UPLC–Q-TOF/MS). As a result, Bacillus sp. 52 and Bacteroides sp. 45 could metabolize rutin to quercetin 3-O-glucoside and leucocyanidin. Bacteroides sp. 42 and Veillonella sp. 32 could convert rutin to leucocyanidin. Bacteroides sp. 22 could hydrolyze rutin to quercetin-3-O-glucoside. In order to further explain the metabolism pathway of rutin,the -d-glucosidase and -l-rhamnosidase activities offive strains weredetermined.',22583754),(3722,'human','Rutin','In this paper, rutin was metabolized by human intestinal bacteria and five isolated strains including Bacillus sp. 52, Bacteroides sp. 45, 42, 22 and Veillonella sp. 32, the metabolites were identified using ultra performance liquid chromatography/quadrupole-time-of-flight mass spectrometry (UPLC–Q-TOF/MS). As a result, Bacillus sp. 52 and Bacteroides sp. 45 could metabolize rutin to quercetin 3-O-glucoside and leucocyanidin. Bacteroides sp. 42 and Veillonella sp. 32 could convert rutin to leucocyanidin. Bacteroides sp. 22 could hydrolyze rutin to quercetin-3-O-glucoside. In order to further explain the metabolism pathway of rutin,the -d-glucosidase and -l-rhamnosidase activities offive strains weredetermined.',22583754),(3723,'human','Phytoestrogens','Nine healthy male adult subjects ingested 0.3 g/kg/day flaxseeds during 1 week. Gut bacteria as well as blood and fecal metabolites were analyzed. Ingestion of flaxseeds triggered a significant increase in the blood concentration of enterolignans, accompanied by fecal excretion of propionate and glycerol. Overall diversity and composition of dominant fecal bacteria remained individual specific throughout the study.',25988339),(3724,'human','Phytoestrogens','Nine healthy male adult subjects ingested 0.3 g/kg/day flaxseeds during 2 week. Gut bacteria as well as blood and fecal metabolites were analyzed. Ingestion of flaxseeds triggered a significant increase in the blood concentration of enterolignans, accompanied by fecal excretion of propionate and glycerol. Overall diversity and composition of dominant fecal bacteria remained individual specific throughout the study.',25988339),(3725,'human','Geniposide','Genipin, a deglycosylated form of geniposide, was cytotoxic at the concentrations that geniposide was not. As metabolic activation systems for geniposide, human intestinal bacterial cultures, fecal preparation (fecalase) and intestinal microbial enzyme mix were employed in the present study. When geniposide was incubated with human intestinal bacteria, either Bifidobacterium longum HY8001 or Bacteroides fragilis, for 24 h, the cultured media caused cytotoxicity in HepG3 cells.',22553067),(3726,'human','Geniposide','Genipin, a deglycosylated form of geniposide, was cytotoxic at the concentrations that geniposide was not. As metabolic activation systems for geniposide, human intestinal bacterial cultures, fecal preparation (fecalase) and intestinal microbial enzyme mix were employed in the present study. When geniposide was incubated with human intestinal bacteria, either Bifidobacterium longum HY8001 or Bacteroides fragilis, for 24 h, the cultured media caused cytotoxicity in HepG3 cells.',22553067),(3740,'human','Retinol','We reported here that microbiota metabolite short-chain fatty acid acetate promoted intestinal IgA responses, which was mediated by \"metabolite-sensing\" GPR43. GPR43-/- mice demonstrated lower levels of intestinal IgA and IgA+ gut bacteria compared with those in wild type (WT) mice. Feeding WT but not GPR43-/- mice acetate but not butyrate promoted intestinal IgA response independent of T cells.',27966553),(3741,'human','Bile acid','The most important bile acid transformation is 7α-dehydroxylation, producing deoxycholic acid (DCA) and lithocholic acid (LCA). The enzymatic pathway carrying out 7α-dehydroxylation is restricted to a narrow phylogenetic group of commensal bacteria, the best-characterized of which is Clostridium scindens.we used a recently established, standardized gnotobiotic mouse model that is stably associated with a simplified murine 12-species \"oligo-mouse microbiota\" (Oligo-MM12).',28066726),(3742,'human','Bile acid','The most important bile acid transformation is 7α-dehydroxylation, producing deoxycholic acid (DCA) and lithocholic acid (LCA). The enzymatic pathway carrying out 7α-dehydroxylation is restricted to a narrow phylogenetic group of commensal bacteria, the best-characterized of which is Clostridium scindens.we used a recently established, standardized gnotobiotic mouse model that is stably associated with a simplified murine 12-species \"oligo-mouse microbiota\" (Oligo-MM12).',28066726),(3743,'human','null','Dietary and bacterial metabolites influence immune responses. This raises the question whether the increased incidence of allergies, asthma, some autoimmune diseases, cardiovascular disease, and others might relate to intake of unhealthy foods, and the decreased intake of dietary fiber. In recent years, new knowledge on the molecular mechanisms underpinning a \'diet-gut microbiota-physiology axis\' has emerged to substantiate this idea. Fiber is fermented to short chain fatty acids (SCFAs), particularly acetate, butyrate, and propionate. These metabolites bind \'metabolite-sensing\' G-protein-coupled receptors such as GPR43, GPR41, and GPR109A. These receptors play fundamental roles in the promotion of gut homeostasis and the regulation of inflammatory responses. For instance, these receptors and their metabolites influence Treg biology, epithelial integrity, gut homeostasis, DC biology, and IgA antibody responses. The SCFAs also influence gene transcription in many cells and tissues, through their inhibition of histone deacetylase expression or function.',28658542),(3744,'human','null','Dietary and bacterial metabolites influence immune responses. This raises the question whether the increased incidence of allergies, asthma, some autoimmune diseases, cardiovascular disease, and others might relate to intake of unhealthy foods, and the decreased intake of dietary fiber. In recent years, new knowledge on the molecular mechanisms underpinning a \'diet-gut microbiota-physiology axis\' has emerged to substantiate this idea. Fiber is fermented to short chain fatty acids (SCFAs), particularly acetate, butyrate, and propionate. These metabolites bind \'metabolite-sensing\' G-protein-coupled receptors such as GPR43, GPR41, and GPR109A. These receptors play fundamental roles in the promotion of gut homeostasis and the regulation of inflammatory responses. For instance, these receptors and their metabolites influence Treg biology, epithelial integrity, gut homeostasis, DC biology, and IgA antibody responses. The SCFAs also influence gene transcription in many cells and tissues, through their inhibition of histone deacetylase expression or function.',28658542),(3745,'human','null','Dietary and bacterial metabolites influence immune responses. This raises the question whether the increased incidence of allergies, asthma, some autoimmune diseases, cardiovascular disease, and others might relate to intake of unhealthy foods, and the decreased intake of dietary fiber. In recent years, new knowledge on the molecular mechanisms underpinning a \'diet-gut microbiota-physiology axis\' has emerged to substantiate this idea. Fiber is fermented to short chain fatty acids (SCFAs), particularly acetate, butyrate, and propionate. These metabolites bind \'metabolite-sensing\' G-protein-coupled receptors such as GPR43, GPR41, and GPR109A. These receptors play fundamental roles in the promotion of gut homeostasis and the regulation of inflammatory responses. For instance, these receptors and their metabolites influence Treg biology, epithelial integrity, gut homeostasis, DC biology, and IgA antibody responses. The SCFAs also influence gene transcription in many cells and tissues, through their inhibition of histone deacetylase expression or function.',28658542),(3746,'human','null','Dietary and bacterial metabolites influence immune responses. This raises the question whether the increased incidence of allergies, asthma, some autoimmune diseases, cardiovascular disease, and others might relate to intake of unhealthy foods, and the decreased intake of dietary fiber. In recent years, new knowledge on the molecular mechanisms underpinning a \'diet-gut microbiota-physiology axis\' has emerged to substantiate this idea. Fiber is fermented to short chain fatty acids (SCFAs), particularly acetate, butyrate, and propionate. These metabolites bind \'metabolite-sensing\' G-protein-coupled receptors such as GPR43, GPR41, and GPR109A. These receptors play fundamental roles in the promotion of gut homeostasis and the regulation of inflammatory responses. For instance, these receptors and their metabolites influence Treg biology, epithelial integrity, gut homeostasis, DC biology, and IgA antibody responses. The SCFAs also influence gene transcription in many cells and tissues, through their inhibition of histone deacetylase expression or function.',28658542),(3747,'human','null','Dietary and bacterial metabolites influence immune responses. This raises the question whether the increased incidence of allergies, asthma, some autoimmune diseases, cardiovascular disease, and others might relate to intake of unhealthy foods, and the decreased intake of dietary fiber. In recent years, new knowledge on the molecular mechanisms underpinning a \'diet-gut microbiota-physiology axis\' has emerged to substantiate this idea. Fiber is fermented to short chain fatty acids (SCFAs), particularly acetate, butyrate, and propionate. These metabolites bind \'metabolite-sensing\' G-protein-coupled receptors such as GPR43, GPR41, and GPR109A. These receptors play fundamental roles in the promotion of gut homeostasis and the regulation of inflammatory responses. For instance, these receptors and their metabolites influence Treg biology, epithelial integrity, gut homeostasis, DC biology, and IgA antibody responses. The SCFAs also influence gene transcription in many cells and tissues, through their inhibition of histone deacetylase expression or function.',28658542),(3748,'human','Tryptophan','Dietary and bacterial metabolites influence immune responses. This raises the question whether the increased incidence of allergies, asthma, some autoimmune diseases, cardiovascular disease, and others might relate to intake of unhealthy foods, and the decreased intake of dietary fiber. In recent years, new knowledge on the molecular mechanisms underpinning a \'diet-gut microbiota-physiology axis\' has emerged to substantiate this idea. Fiber is fermented to short chain fatty acids (SCFAs), particularly acetate, butyrate, and propionate. These metabolites bind \'metabolite-sensing\' G-protein-coupled receptors such as GPR43, GPR41, and GPR109A. These receptors play fundamental roles in the promotion of gut homeostasis and the regulation of inflammatory responses. For instance, these receptors and their metabolites influence Treg biology, epithelial integrity, gut homeostasis, DC biology, and IgA antibody responses. The SCFAs also influence gene transcription in many cells and tissues, through their inhibition of histone deacetylase expression or function.',28658542),(3749,'human','Linoleic Acid','In this study, we investigated the effects of novel fatty acid metabolite derivatives of linoleic acid generated by the gut lactic acid bacteria Lactobacillus plantarum on the Nrf2-ARE pathway. 10-Oxo-trans-11-octadecenoic acid (KetoC) protected HepG2 cells from cytotoxicity induced by hydrogen peroxide. KetoC also significantly increased cellular Nrf2 protein levels, ARE-dependent transcription, and the gene expression of antioxidative enzymes such as heme oxygenase-1 (HO-1), glutamate-cysteine ligase modifier subunit (GCLM), and Nad(p)h: quinone oxidoreductase 1 (NQO1) in HepG2 cells.',26879219),(3750,'human','Linoleic Acid','In this study, we investigated the effects of novel fatty acid metabolite derivatives of linoleic acid generated by the gut lactic acid bacteria Lactobacillus plantarum on the Nrf2-ARE pathway. 10-Oxo-trans-11-octadecenoic acid (KetoC) protected HepG2 cells from cytotoxicity induced by hydrogen peroxide. KetoC also significantly increased cellular Nrf2 protein levels, ARE-dependent transcription, and the gene expression of antioxidative enzymes such as heme oxygenase-1 (HO-1), glutamate-cysteine ligase modifier subunit (GCLM), and Nad(p)h: quinone oxidoreductase 1 (NQO1) in HepG2 cells.',26879219),(3751,'human','Rutin','The average α-Lrhamnosidase activities on the p-nitrophenyl-α-L-rhamnopyranoside, rutin, and poncirin subtrates were 0.10 ± 0.07, 0.25 ± 0.08, and 0.15 ± 0.09 pmol/min/mg, respectively. To investigate the enzymatic properties, α-L-rhamnosidase-producing bacteria were isolated from the specimens, and the α-L-rhamnosidase gene was cloned from a selected organism, Bifidobacterium dentium, and expressed in E. coli. The cloned α-L-rhamnosidase gene contained a 2,673 bp sequcence encoding 890 amino acid residues. The cloned gene was expressed using the pET 26b(+) vector in E. coli BL21, and the expressed enzyme was purified using Ni(2+)-NTA and Q-HP column chromatography. The specific activity of the purified α-L-rhamnosidase was 23.3 μmol/min/mg.',25179902),(3752,'human','Poncirin','The average α-Lrhamnosidase activities on the p-nitrophenyl-α-L-rhamnopyranoside, rutin, and poncirin subtrates were 0.10 ± 0.07, 0.25 ± 0.08, and 0.15 ± 0.09 pmol/min/mg, respectively. To investigate the enzymatic properties, α-L-rhamnosidase-producing bacteria were isolated from the specimens, and the α-L-rhamnosidase gene was cloned from a selected organism, Bifidobacterium dentium, and expressed in E. coli. The cloned α-L-rhamnosidase gene contained a 2,673 bp sequcence encoding 890 amino acid residues. The cloned gene was expressed using the pET 26b(+) vector in E. coli BL21, and the expressed enzyme was purified using Ni(2+)-NTA and Q-HP column chromatography. The specific activity of the purified α-L-rhamnosidase was 23.3 μmol/min/mg.',25179902),(3753,'human','null','A feature of human ulcerative colitis and other colitic diseases is a change in \'healthy\' microbiota such as Bifidobacterium and Bacteriodes, and a concurrent reduction in SCFAs. Moreover, increased intake of fermentable dietary fibre, or SCFAs, seems to be clinically beneficial in the treatment of colitis. SCFAs bind the G-protein-coupled receptor 43 (GPR43, also known as FFAR2), and here we show that SCFA-GPR43 interactions profoundly affect inflammatory responses. Stimulation of GPR43 by SCFAs was necessary for the normal resolution of certain inflammatory responses, because GPR43-deficient (Gpr43(-/-)) mice showed exacerbated or unresolving inflammation in models of colitis, arthritis and asthma. This seemed to relate to increased production of inflammatory mediators by Gpr43(-/-) immune cells, and increased immune cell recruitment. Germ-free mice, which are devoid of bacteria and express little or no SCFAs, showed a similar dysregulation of certain inflammatory responses. GPR43 binding of SCFAs potentially provides a molecular link between diet, gastrointestinal bacterial metabolism, and immune and inflammatory responses.',19865172),(3754,'human','null','A feature of human ulcerative colitis and other colitic diseases is a change in \'healthy\' microbiota such as Bifidobacterium and Bacteriodes, and a concurrent reduction in SCFAs. Moreover, increased intake of fermentable dietary fibre, or SCFAs, seems to be clinically beneficial in the treatment of colitis. SCFAs bind the G-protein-coupled receptor 43 (GPR43, also known as FFAR2), and here we show that SCFA-GPR43 interactions profoundly affect inflammatory responses. Stimulation of GPR43 by SCFAs was necessary for the normal resolution of certain inflammatory responses, because GPR43-deficient (Gpr43(-/-)) mice showed exacerbated or unresolving inflammation in models of colitis, arthritis and asthma. This seemed to relate to increased production of inflammatory mediators by Gpr43(-/-) immune cells, and increased immune cell recruitment. Germ-free mice, which are devoid of bacteria and express little or no SCFAs, showed a similar dysregulation of certain inflammatory responses. GPR43 binding of SCFAs potentially provides a molecular link between diet, gastrointestinal bacterial metabolism, and immune and inflammatory responses.',19865172),(3755,'human','null','A feature of human ulcerative colitis and other colitic diseases is a change in \'healthy\' microbiota such as Bifidobacterium and Bacteriodes, and a concurrent reduction in SCFAs. Moreover, increased intake of fermentable dietary fibre, or SCFAs, seems to be clinically beneficial in the treatment of colitis. SCFAs bind the G-protein-coupled receptor 43 (GPR43, also known as FFAR2), and here we show that SCFA-GPR43 interactions profoundly affect inflammatory responses. Stimulation of GPR43 by SCFAs was necessary for the normal resolution of certain inflammatory responses, because GPR43-deficient (Gpr43(-/-)) mice showed exacerbated or unresolving inflammation in models of colitis, arthritis and asthma. This seemed to relate to increased production of inflammatory mediators by Gpr43(-/-) immune cells, and increased immune cell recruitment. Germ-free mice, which are devoid of bacteria and express little or no SCFAs, showed a similar dysregulation of certain inflammatory responses. GPR43 binding of SCFAs potentially provides a molecular link between diet, gastrointestinal bacterial metabolism, and immune and inflammatory responses.',19865172),(3756,'human','null','A feature of human ulcerative colitis and other colitic diseases is a change in \'healthy\' microbiota such as Bifidobacterium and Bacteriodes, and a concurrent reduction in SCFAs. Moreover, increased intake of fermentable dietary fibre, or SCFAs, seems to be clinically beneficial in the treatment of colitis. SCFAs bind the G-protein-coupled receptor 43 (GPR43, also known as FFAR2), and here we show that SCFA-GPR43 interactions profoundly affect inflammatory responses. Stimulation of GPR43 by SCFAs was necessary for the normal resolution of certain inflammatory responses, because GPR43-deficient (Gpr43(-/-)) mice showed exacerbated or unresolving inflammation in models of colitis, arthritis and asthma. This seemed to relate to increased production of inflammatory mediators by Gpr43(-/-) immune cells, and increased immune cell recruitment. Germ-free mice, which are devoid of bacteria and express little or no SCFAs, showed a similar dysregulation of certain inflammatory responses. GPR43 binding of SCFAs potentially provides a molecular link between diet, gastrointestinal bacterial metabolism, and immune and inflammatory responses.',19865172),(3757,'human','Isoflavone','The genome of new equol-producing bacteria, Lactobacillus paracasei JS1, was analyzed. Also, the effect of P. lobata extract fermented with L. paracasei JS1 (FPE) on the skin and intestinal immune response was examined. With gene expression analysis, it was proven that seven skin-related proteins, hyaluronan synthase-1, -2, -3, collagen, elastin, epidermal growth factor, and epidermal growth factor receptor were differentially expressed upon FPE treatment. The messenger RNA expression increased with treatment with the FPE, and a skin moisturizing effect was confirmed by a hematoxylin-eosin staining experiment. In addition, such an experiment showed that proinflammatory cytokines, tumor necrosis factor-α, cyclooxygenase-2, inducible nitric oxide synthase, interleukin-1β, -4, and -6, were reduced in large intestine when treated with FPE.',30767231),(3758,'human','Isoflavone','The genome of new equol-producing bacteria, Lactobacillus paracasei JS1, was analyzed. Also, the effect of P. lobata extract fermented with L. paracasei JS1 (FPE) on the skin and intestinal immune response was examined. With gene expression analysis, it was proven that seven skin-related proteins, hyaluronan synthase-1, -2, -3, collagen, elastin, epidermal growth factor, and epidermal growth factor receptor were differentially expressed upon FPE treatment. The messenger RNA expression increased with treatment with the FPE, and a skin moisturizing effect was confirmed by a hematoxylin-eosin staining experiment. In addition, such an experiment showed that proinflammatory cytokines, tumor necrosis factor-α, cyclooxygenase-2, inducible nitric oxide synthase, interleukin-1β, -4, and -6, were reduced in large intestine when treated with FPE.',30767231),(3759,'human','Isoflavone','The genome of new equol-producing bacteria, Lactobacillus paracasei JS1, was analyzed. Also, the effect of P. lobata extract fermented with L. paracasei JS1 (FPE) on the skin and intestinal immune response was examined. With gene expression analysis, it was proven that seven skin-related proteins, hyaluronan synthase-1, -2, -3, collagen, elastin, epidermal growth factor, and epidermal growth factor receptor were differentially expressed upon FPE treatment. The messenger RNA expression increased with treatment with the FPE, and a skin moisturizing effect was confirmed by a hematoxylin-eosin staining experiment. In addition, such an experiment showed that proinflammatory cytokines, tumor necrosis factor-α, cyclooxygenase-2, inducible nitric oxide synthase, interleukin-1β, -4, and -6, were reduced in large intestine when treated with FPE.',30767231),(3760,'human','Isoflavone','The genome of new equol-producing bacteria, Lactobacillus paracasei JS1, was analyzed. Also, the effect of P. lobata extract fermented with L. paracasei JS1 (FPE) on the skin and intestinal immune response was examined. With gene expression analysis, it was proven that seven skin-related proteins, hyaluronan synthase-1, -2, -3, collagen, elastin, epidermal growth factor, and epidermal growth factor receptor were differentially expressed upon FPE treatment. The messenger RNA expression increased with treatment with the FPE, and a skin moisturizing effect was confirmed by a hematoxylin-eosin staining experiment. In addition, such an experiment showed that proinflammatory cytokines, tumor necrosis factor-α, cyclooxygenase-2, inducible nitric oxide synthase, interleukin-1β, -4, and -6, were reduced in large intestine when treated with FPE.',30767231),(3761,'human','Isoflavone','The genome of new equol-producing bacteria, Lactobacillus paracasei JS1, was analyzed. Also, the effect of P. lobata extract fermented with L. paracasei JS1 (FPE) on the skin and intestinal immune response was examined. With gene expression analysis, it was proven that seven skin-related proteins, hyaluronan synthase-1, -2, -3, collagen, elastin, epidermal growth factor, and epidermal growth factor receptor were differentially expressed upon FPE treatment. The messenger RNA expression increased with treatment with the FPE, and a skin moisturizing effect was confirmed by a hematoxylin-eosin staining experiment. In addition, such an experiment showed that proinflammatory cytokines, tumor necrosis factor-α, cyclooxygenase-2, inducible nitric oxide synthase, interleukin-1β, -4, and -6, were reduced in large intestine when treated with FPE.',30767231),(3762,'human','Isoflavone','The genome of new equol-producing bacteria, Lactobacillus paracasei JS1, was analyzed. Also, the effect of P. lobata extract fermented with L. paracasei JS1 (FPE) on the skin and intestinal immune response was examined. With gene expression analysis, it was proven that seven skin-related proteins, hyaluronan synthase-1, -2, -3, collagen, elastin, epidermal growth factor, and epidermal growth factor receptor were differentially expressed upon FPE treatment. The messenger RNA expression increased with treatment with the FPE, and a skin moisturizing effect was confirmed by a hematoxylin-eosin staining experiment. In addition, such an experiment showed that proinflammatory cytokines, tumor necrosis factor-α, cyclooxygenase-2, inducible nitric oxide synthase, interleukin-1β, -4, and -6, were reduced in large intestine when treated with FPE.',30767231),(3763,'human','Galbeta1,3GlcNAc','In the current study, we isolated a novel strain of Bifidobacterium kashiwanohense, named APCKJ1, from the faeces of a four-week old breastfed infant, based on the ability of the strain to utilise the HMO component fucosyllactose. We then determined the full genome sequence of this strain, and employed the generated data to analyze fucosyllactose metabolism in B. kashiwanohense APCKJ1. Transcriptomic and growth analyses, combined with metabolite analysis, in vitro hydrolysis assays and heterologous expression, allowed us to elucidate the pathway for fucosyllactose metabolism in B. kashiwanohense APCKJ1. Homologs of the key genes for this metabolic pathway were identified in particular in infant-derived members of the Bifdobacterium genus, revealing the apparent niche-specific nature of this pathway, and allowing a broad perspective on bifidobacterial fucosyllactose and L-fucose metabolism.',31659215),(3764,'human','N-Acetyl-D-lactosamine','In the current study, we isolated a novel strain of Bifidobacterium kashiwanohense, named APCKJ1, from the faeces of a four-week old breastfed infant, based on the ability of the strain to utilise the HMO component fucosyllactose. We then determined the full genome sequence of this strain, and employed the generated data to analyze fucosyllactose metabolism in B. kashiwanohense APCKJ1. Transcriptomic and growth analyses, combined with metabolite analysis, in vitro hydrolysis assays and heterologous expression, allowed us to elucidate the pathway for fucosyllactose metabolism in B. kashiwanohense APCKJ1. Homologs of the key genes for this metabolic pathway were identified in particular in infant-derived members of the Bifdobacterium genus, revealing the apparent niche-specific nature of this pathway, and allowing a broad perspective on bifidobacterial fucosyllactose and L-fucose metabolism.',31659215),(3765,'human','null','A Prevotella and an Atopobium strain were chosen for further study, and shown to up-regulate two PPARγ-target genes, ANGPTL4 and ADRP. The molecular mechanisms of these activations involved the phosphorylation of PPARγ through ERK1/2. The responsible metabolites were shown to be heat sensitive but markedly diverged in size, emphasizing the diversity of bioactive compounds found in the intestine. Here we describe different mechanisms by which single intestinal bacteria can directly impact their host\'s health through transcriptional regulation.',28266623),(3766,'human','null','A Prevotella and an Atopobium strain were chosen for further study, and shown to up-regulate two PPARγ-target genes, ANGPTL4 and ADRP. The molecular mechanisms of these activations involved the phosphorylation of PPARγ through ERK1/2. The responsible metabolites were shown to be heat sensitive but markedly diverged in size, emphasizing the diversity of bioactive compounds found in the intestine. Here we describe different mechanisms by which single intestinal bacteria can directly impact their host\'s health through transcriptional regulation.',28266623),(3767,'human','null','A Prevotella and an Atopobium strain were chosen for further study, and shown to up-regulate two PPARγ-target genes, ANGPTL4 and ADRP. The molecular mechanisms of these activations involved the phosphorylation of PPARγ through ERK1/2. The responsible metabolites were shown to be heat sensitive but markedly diverged in size, emphasizing the diversity of bioactive compounds found in the intestine. Here we describe different mechanisms by which single intestinal bacteria can directly impact their host\'s health through transcriptional regulation.',28266623),(3768,'human','null','A Prevotella and an Atopobium strain were chosen for further study, and shown to up-regulate two PPARγ-target genes, ANGPTL4 and ADRP. The molecular mechanisms of these activations involved the phosphorylation of PPARγ through ERK1/2. The responsible metabolites were shown to be heat sensitive but markedly diverged in size, emphasizing the diversity of bioactive compounds found in the intestine. Here we describe different mechanisms by which single intestinal bacteria can directly impact their host\'s health through transcriptional regulation.',28266623),(3769,'human','Daidzein','Several kinds of carbohydrates such as sorbose, adonitol, and melezitose were found to enhance equol production from daidzein in an in vitro human fecal culture. Sorbose, one of the most effective carbohydrates, was used as a carbohydrate source for isolating the NATTS strain, which was a Gram-positive, non-spore-forming rod bacterium with high ability to convert daidzein to equol isolated from the 7th maintenance culture. The strain was found to belong to the genus Slackia family Coriobacteriaceae by 16S rRNA sequence-based analysis, and the prevalence of the Slackia sp. in Japanese adults was examined by reverse transcription-quantitative PCR (RT-qPCR), which was found to be 40% at a mean population level of 10(6) cells per gram of feces.',20237913),(3770,'human','Daidzin','Several kinds of carbohydrates such as sorbose, adonitol, and melezitose were found to enhance equol production from daidzein in an in vitro human fecal culture. Sorbose, one of the most effective carbohydrates, was used as a carbohydrate source for isolating the NATTS strain, which was a Gram-positive, non-spore-forming rod bacterium with high ability to convert daidzein to equol isolated from the 7th maintenance culture. The strain was found to belong to the genus Slackia family Coriobacteriaceae by 16S rRNA sequence-based analysis, and the prevalence of the Slackia sp. in Japanese adults was examined by reverse transcription-quantitative PCR (RT-qPCR), which was found to be 40% at a mean population level of 10(6) cells per gram of feces.',20237913),(3771,'human','Daidzein','Several kinds of carbohydrates such as sorbose, adonitol, and melezitose were found to enhance equol production from daidzein in an in vitro human fecal culture. Sorbose, one of the most effective carbohydrates, was used as a carbohydrate source for isolating the NATTS strain, which was a Gram-positive, non-spore-forming rod bacterium with high ability to convert daidzein to equol isolated from the 7th maintenance culture. The strain was found to belong to the genus Slackia family Coriobacteriaceae by 16S rRNA sequence-based analysis, and the prevalence of the Slackia sp. in Japanese adults was examined by reverse transcription-quantitative PCR (RT-qPCR), which was found to be 40% at a mean population level of 10(6) cells per gram of feces.',20237913),(3772,'human','Daidzein','Several kinds of carbohydrates such as sorbose, adonitol, and melezitose were found to enhance equol production from daidzein in an in vitro human fecal culture. Sorbose, one of the most effective carbohydrates, was used as a carbohydrate source for isolating the NATTS strain, which was a Gram-positive, non-spore-forming rod bacterium with high ability to convert daidzein to equol isolated from the 7th maintenance culture. The strain was found to belong to the genus Slackia family Coriobacteriaceae by 16S rRNA sequence-based analysis, and the prevalence of the Slackia sp. in Japanese adults was examined by reverse transcription-quantitative PCR (RT-qPCR), which was found to be 40% at a mean population level of 10(6) cells per gram of feces.',20237913),(3773,'human','Puerarin','A rod-shaped Gram-positive bacterium, strain CG19-1, capable of deglycosylating puerarin to daidzein was isolated from the corresponding suspension. However, the strictly anaerobic isolate was unable to utilize puerarin as sole carbon and energy source nor any of the tested carbohydrates. Comparative 16S rRNA sequence analysis indicated that strain CG19-1 is a new species of the Lachnospiraceae. Strain CG19-1 also converted other aromatic C-glucosides in addition to puerarin. The xanthone C-glucoside mangiferin was deglycosylated to norathyriol. The flavone C-glucosides homoorientin and vitexin were degraded to 3-(3,4-dihydroxyphenyl)propionic acid via luteolin and 3-(4-hydroxyphenyl)propionic acid respectively. In addition, strain CG19-1 converted flavonoid O-glucosides, but at rates that were lower than those of the C-glucosides tested. The isolate deglycosylated the isoflavone O-glucosides daidzin and genistin to daidzein and genistein respectively. Several O-glucosides of the flavones luteolin and apigenin undergoing deglycosylation were subsequently cleaved to 3-(3,4-dihydroxyphenyl)propionic acid and 3-(4-hydroxyphenyl)propionic acid respectively. Moreover, strain CG19-1 cleaved both O-desmethylangolensin and 6\'-hydroxy-O-desmethylangolensin to yield 2-(4-dihydroxyphenyl)propionic acid. The corresponding cleavage product, resorcinol, was only observed for O-desmethylangolensin.',20946528),(3774,'human','Vitexin','A rod-shaped Gram-positive bacterium, strain CG19-1, capable of deglycosylating puerarin to daidzein was isolated from the corresponding suspension. However, the strictly anaerobic isolate was unable to utilize puerarin as sole carbon and energy source nor any of the tested carbohydrates. Comparative 16S rRNA sequence analysis indicated that strain CG19-1 is a new species of the Lachnospiraceae. Strain CG19-1 also converted other aromatic C-glucosides in addition to puerarin. The xanthone C-glucoside mangiferin was deglycosylated to norathyriol. The flavone C-glucosides homoorientin and vitexin were degraded to 3-(3,4-dihydroxyphenyl)propionic acid via luteolin and 3-(4-hydroxyphenyl)propionic acid respectively. In addition, strain CG19-1 converted flavonoid O-glucosides, but at rates that were lower than those of the C-glucosides tested. The isolate deglycosylated the isoflavone O-glucosides daidzin and genistin to daidzein and genistein respectively. Several O-glucosides of the flavones luteolin and apigenin undergoing deglycosylation were subsequently cleaved to 3-(3,4-dihydroxyphenyl)propionic acid and 3-(4-hydroxyphenyl)propionic acid respectively. Moreover, strain CG19-1 cleaved both O-desmethylangolensin and 6\'-hydroxy-O-desmethylangolensin to yield 2-(4-dihydroxyphenyl)propionic acid. The corresponding cleavage product, resorcinol, was only observed for O-desmethylangolensin.',20946528),(3775,'human','O-Desmethylangolensin','A rod-shaped Gram-positive bacterium, strain CG19-1, capable of deglycosylating puerarin to daidzein was isolated from the corresponding suspension. However, the strictly anaerobic isolate was unable to utilize puerarin as sole carbon and energy source nor any of the tested carbohydrates. Comparative 16S rRNA sequence analysis indicated that strain CG19-1 is a new species of the Lachnospiraceae. Strain CG19-1 also converted other aromatic C-glucosides in addition to puerarin. The xanthone C-glucoside mangiferin was deglycosylated to norathyriol. The flavone C-glucosides homoorientin and vitexin were degraded to 3-(3,4-dihydroxyphenyl)propionic acid via luteolin and 3-(4-hydroxyphenyl)propionic acid respectively. In addition, strain CG19-1 converted flavonoid O-glucosides, but at rates that were lower than those of the C-glucosides tested. The isolate deglycosylated the isoflavone O-glucosides daidzin and genistin to daidzein and genistein respectively. Several O-glucosides of the flavones luteolin and apigenin undergoing deglycosylation were subsequently cleaved to 3-(3,4-dihydroxyphenyl)propionic acid and 3-(4-hydroxyphenyl)propionic acid respectively. Moreover, strain CG19-1 cleaved both O-desmethylangolensin and 6\'-hydroxy-O-desmethylangolensin to yield 2-(4-dihydroxyphenyl)propionic acid. The corresponding cleavage product, resorcinol, was only observed for O-desmethylangolensin.',20946528),(3776,'human','6\'-Hydroxy-O-desmethylangolensin','A rod-shaped Gram-positive bacterium, strain CG19-1, capable of deglycosylating puerarin to daidzein was isolated from the corresponding suspension. However, the strictly anaerobic isolate was unable to utilize puerarin as sole carbon and energy source nor any of the tested carbohydrates. Comparative 16S rRNA sequence analysis indicated that strain CG19-1 is a new species of the Lachnospiraceae. Strain CG19-1 also converted other aromatic C-glucosides in addition to puerarin. The xanthone C-glucoside mangiferin was deglycosylated to norathyriol. The flavone C-glucosides homoorientin and vitexin were degraded to 3-(3,4-dihydroxyphenyl)propionic acid via luteolin and 3-(4-hydroxyphenyl)propionic acid respectively. In addition, strain CG19-1 converted flavonoid O-glucosides, but at rates that were lower than those of the C-glucosides tested. The isolate deglycosylated the isoflavone O-glucosides daidzin and genistin to daidzein and genistein respectively. Several O-glucosides of the flavones luteolin and apigenin undergoing deglycosylation were subsequently cleaved to 3-(3,4-dihydroxyphenyl)propionic acid and 3-(4-hydroxyphenyl)propionic acid respectively. Moreover, strain CG19-1 cleaved both O-desmethylangolensin and 6\'-hydroxy-O-desmethylangolensin to yield 2-(4-dihydroxyphenyl)propionic acid. The corresponding cleavage product, resorcinol, was only observed for O-desmethylangolensin.',20946528),(3777,'human','(3R)-flavan-3-ols','A human intestinal bacterium, Eubacterium (E.) sp. strain SDG-2, was tested for its ability to metabolize various (3R)- and (3S)-flavan-3-ols and their 3-O-gallates. This bacterium cleaved the C-ring of (3R)- and (3S)-flavan-3-ols to give 1,3-diphenylpropan-2-ol derivatives, but not their 3-O-gallates. Furthermore, E. sp. strain SDG-2 had the ability of p-dehydroxylation in the B-ring of (3R)-flavan-3-ols, such as (-)-catechin, (-)-epicatechin, (-)-gallocatechin and (-)-epigallocatechin, but not of (3S)-flavan-3-ols, such as (+)-catechin and (+)-epicatechin.',11767089),(3778,'human','(3S)-flavan-3-ols','A human intestinal bacterium, Eubacterium (E.) sp. strain SDG-2, was tested for its ability to metabolize various (3R)- and (3S)-flavan-3-ols and their 3-O-gallates. This bacterium cleaved the C-ring of (3R)- and (3S)-flavan-3-ols to give 1,3-diphenylpropan-2-ol derivatives, but not their 3-O-gallates. Furthermore, E. sp. strain SDG-2 had the ability of p-dehydroxylation in the B-ring of (3R)-flavan-3-ols, such as (-)-catechin, (-)-epicatechin, (-)-gallocatechin and (-)-epigallocatechin, but not of (3S)-flavan-3-ols, such as (+)-catechin and (+)-epicatechin.',11767089),(3803,'human','Ginsenoside rb1','Compound K significantly up‑regulated expression of p21 (CDKN1A), and downregulated the expressions of CDK6, cyclin D and cyclin E, which are G1 phase arrest regulators.',29805567),(3804,'human','Ginsenoside rb1','Compound K significantly up‑regulated expression of p21 (CDKN1A), and downregulated the expressions of CDK6, cyclin D and cyclin E, which are G1 phase arrest regulators.',29805567),(3805,'human','Ginsenoside rb1','Compound K significantly up‑regulated expression of p21 (CDKN1A), and downregulated the expressions of CDK6, cyclin D and cyclin E, which are G1 phase arrest regulators.',29805567),(3806,'human','Ginsenoside rb1','Compound K significantly up‑regulated expression of p21 (CDKN1A), and downregulated the expressions of CDK6, cyclin D and cyclin E, which are G1 phase arrest regulators.',29805567),(3807,'human','Ginsenoside rb1','Compound K up-regulated the expressions of p53 and Bax,  which are pro-apoptotic genes. In contrast, compound K down-regulated the expression of Bcl2, which is an anti-apoptotic gene.',29805567),(3808,'human','Ginsenoside rb1','Compound K up-regulated the expressions of p53 and Bax,  which are pro-apoptotic genes. In contrast, compound K down-regulated the expression of Bcl2, which is an anti-apoptotic gene.',29805567),(3809,'human','Ginsenoside rb1','Compound K up-regulated the expressions of p53 and Bax,  which are pro-apoptotic genes. In contrast, compound K down-regulated the expression of Bcl2, which is an anti-apoptotic gene.',29805567),(3810,'human','Ginsenoside rb1','Compound K significantly reduced the IL-8 production in HT-29 cells.',29805567),(3817,'human','Levodopa','Gut-associated bacteria have the abality to decarboxylate levodopa to dopamine via tyrosine decarboxylases.',30659181),(3818,'human','PhIP-N2-β-D-glucuronide','B-GUS of Faecalibacterium prausnitzii converted PhIP-G to PhIP and GDH of Flavonifractor plautii, Blautia obeum, Eubacterium hallii, and Lactobacillus reuteri converted PhIP to PhIP-M1 in the presence of glycerol. In addition, B-GUS- and GDH-positive bacteria cooperatively converted PhIP-G to PhIP-M1.',31096909),(3819,'human','2-Amino-1-methyl-6-phenylimidazo[4,5-b]pyridine','B-GUS of Faecalibacterium prausnitzii converted PhIP-G to PhIP and GDH of Flavonifractor plautii, Blautia obeum, Eubacterium hallii, and Lactobacillus reuteri converted PhIP to PhIP-M1 in the presence of glycerol. In addition, B-GUS- and GDH-positive bacteria cooperatively converted PhIP-G to PhIP-M1.',31096909),(3820,'human','2-Amino-1-methyl-6-phenylimidazo[4,5-b]pyridine','B-GUS of Faecalibacterium prausnitzii converted PhIP-G to PhIP and GDH of Flavonifractor plautii, Blautia obeum, Eubacterium hallii, and Lactobacillus reuteri converted PhIP to PhIP-M1 in the presence of glycerol. In addition, B-GUS- and GDH-positive bacteria cooperatively converted PhIP-G to PhIP-M1.',31096909),(3821,'human','2-Amino-1-methyl-6-phenylimidazo[4,5-b]pyridine','B-GUS of Faecalibacterium prausnitzii converted PhIP-G to PhIP and GDH of Flavonifractor plautii, Blautia obeum, Eubacterium hallii, and Lactobacillus reuteri converted PhIP to PhIP-M1 in the presence of glycerol. In addition, B-GUS- and GDH-positive bacteria cooperatively converted PhIP-G to PhIP-M1.',31096909),(3822,'human','2-Amino-1-methyl-6-phenylimidazo[4,5-b]pyridine','B-GUS of Faecalibacterium prausnitzii converted PhIP-G to PhIP and GDH of Flavonifractor plautii, Blautia obeum, Eubacterium hallii, and Lactobacillus reuteri converted PhIP to PhIP-M1 in the presence of glycerol. In addition, B-GUS- and GDH-positive bacteria cooperatively converted PhIP-G to PhIP-M1.',31096909),(3823,'human','Rutin','Enterococcus avium EFEL009 is capable of converting rutin to quercetin.',26505733),(3832,'human','null','In the coculture model, treatment with Enterococcus durans and with butyrate reduced basal as well as E coli stimulated secretion of IL-8, IL-6, and  TNF and increased secretion of IL-10.',21178864),(3833,'human','null','In the coculture model, treatment with Enterococcus durans and with butyrate reduced basal as well as E coli stimulated secretion of IL-8, IL-6, and  TNF and increased secretion of IL-10.',21178864),(3834,'human','null','In the coculture model, treatment with Enterococcus durans and with butyrate reduced basal as well as E coli stimulated secretion of IL-8, IL-6, and  TNF and increased secretion of IL-10.',21178864),(3835,'human','null','In the coculture model, treatment with Enterococcus durans and with butyrate reduced basal as well as E coli stimulated secretion of IL-8, IL-6, and  TNF and increased secretion of IL-10.',21178864),(3836,'human','null','In the coculture model, treatment with Enterococcus durans and with butyrate reduced basal as well as E coli stimulated secretion of IL-8, IL-6, and  TNF and increased secretion of IL-10.',21178864),(3838,'human','null','Lactobacillus casei strain Shirota intake increased defecation frequency, normalized stool consistency, and improved constipation related symptoms in Chinese adults with constipation. Through the use of metabolomic approaches, PIPA, which is a non-volatile metabolite in the gut, was identified as a potential target mediator in LcS-mediated constipation alleviation. Furthermore, PIPA could improve constipation symptoms, as shown by increases in the small intestine propulsive rate and the number of fecal pellets in a constipation mouse model, raising its possibility in the constipation therapy. Future studies should verify the anti-constipation function of PIPA and elucidate its potential mechanisms.',30846982),(3841,'human','Geniposide','Western blot analysis revealed that the activated GS inhibited Bcl-2 expression with a subsequent increase in Bax expression. Likewise, GS activation by IM stimulated caspase-3 and the production of reactive oxygen species (ROS). In addition, activated GS-induced apoptosis was confirmed by apoptosis and ROS assays; N-acetyl-l-cysteine (NAC) suppressed ROS production and apoptotic cell death. Activated GS induced sustained JNK phosphorylation. Moreover, activated GS-induced cell death was reversed by SP600125. Taken together, these findings suggest that human IM is able to metabolize GS into GP, and the related biological activities induce apoptosis through ROS/JNK signaling.',22245672),(3842,'human','Geniposide','Western blot analysis revealed that the activated GS inhibited Bcl-2 expression with a subsequent increase in Bax expression. Likewise, GS activation by IM stimulated caspase-3 and the production of reactive oxygen species (ROS). In addition, activated GS-induced apoptosis was confirmed by apoptosis and ROS assays; N-acetyl-l-cysteine (NAC) suppressed ROS production and apoptotic cell death. Activated GS induced sustained JNK phosphorylation. Moreover, activated GS-induced cell death was reversed by SP600125. Taken together, these findings suggest that human IM is able to metabolize GS into GP, and the related biological activities induce apoptosis through ROS/JNK signaling.',22245672),(3843,'human','Geniposide','Western blot analysis revealed that the activated GS inhibited Bcl-2 expression with a subsequent increase in Bax expression. Likewise, GS activation by IM stimulated caspase-3 and the production of reactive oxygen species (ROS). In addition, activated GS-induced apoptosis was confirmed by apoptosis and ROS assays; N-acetyl-l-cysteine (NAC) suppressed ROS production and apoptotic cell death. Activated GS induced sustained JNK phosphorylation. Moreover, activated GS-induced cell death was reversed by SP600125. Taken together, these findings suggest that human IM is able to metabolize GS into GP, and the related biological activities induce apoptosis through ROS/JNK signaling.',22245672),(3844,'human','Geniposide','Western blot analysis revealed that the activated GS inhibited Bcl-2 expression with a subsequent increase in Bax expression. Likewise, GS activation by IM stimulated caspase-3 and the production of reactive oxygen species (ROS). In addition, activated GS-induced apoptosis was confirmed by apoptosis and ROS assays; N-acetyl-l-cysteine (NAC) suppressed ROS production and apoptotic cell death. Activated GS induced sustained JNK phosphorylation. Moreover, activated GS-induced cell death was reversed by SP600125. Taken together, these findings suggest that human IM is able to metabolize GS into GP, and the related biological activities induce apoptosis through ROS/JNK signaling.',22245672),(3845,'human','null','We examined the effects of UV-killed F. prausnitzii, bacterial fractions,supernatants, or sterile medium on IL-8 secretion in Caco-2 cells,with or without IL-1 stimulation (SI Materials and Methods and Fig. 3A). Either UV-killed F. prausnitzii, membrane extracts, or cytoplasmic extracts, but not DNA or sterile medium, induce low levels of IL-8 secretion. UV-killed F. prausnitzii, bacterial fractions,and sterile medium did not inhibit IL-8 secretion induced by IL-1,compared with negative control. In contrast, supernatant from F.prausnitzii cultures significantly reduces IL-8 secretion induced by IL-1.',18936492),(3846,'human','null','We examined the effects of UV-killed F. prausnitzii, bacterial fractions,supernatants, or sterile medium on IL-8 secretion in Caco-2 cells,with or without IL-1 stimulation (SI Materials and Methods and Fig. 3A). Either UV-killed F. prausnitzii, membrane extracts, or cytoplasmic extracts, but not DNA or sterile medium, induce low levels of IL-8 secretion. UV-killed F. prausnitzii, bacterial fractions,and sterile medium did not inhibit IL-8 secretion induced by IL-1,compared with negative control. In contrast, supernatant from F.prausnitzii cultures significantly reduces IL-8 secretion induced by IL-1.',18936492),(3850,'human','3,8,9-Trihydroxy-6H-benzo[c]chromen-6-one','The expression of p53 was significantly increased in the HCT- 116 cells (wild type p53) by Uro-A and UM-A treatments.',32179165),(3851,'human','3,8,9-Trihydroxy-6H-benzo[c]chromen-6-one','A significant induction of the of p21Cip1/Waf1 in the cells treated with Uro-A (2.2 ± 0.2) and UM-A (2.2 ± 0.5) was observed.',32179165),(3852,'human','null','Both gain- and loss-of-function studies employing genetic and pharmacological tools reveal PAGln mediates cellular events through G-protein coupled receptors, including α2A, α2B, and β2-adrenergic receptors.',32142679),(3853,'human','null','Both gain- and loss-of-function studies employing genetic and pharmacological tools reveal PAGln mediates cellular events through G-protein coupled receptors, including α2A, α2B, and β2-adrenergic receptors.',32142679),(3854,'human','null','Both gain- and loss-of-function studies employing genetic and pharmacological tools reveal PAGln mediates cellular events through G-protein coupled receptors, including α2A, α2B, and β2-adrenergic receptors.',32142679),(3855,'human','Ellagic acid','As a result, a 24 h treatment with UA (20 µm) significantly decreased Rac1 and PAK1 transcript levels and activity, depolymerized actin and wound healing.',31976617),(3856,'human','Ellagic acid','As a result, a 24 h treatment with UA (20 µm) significantly decreased Rac1 and PAK1 transcript levels and activity, depolymerized actin and wound healing.',31976617),(3857,'human','(-)-Epicatechin','All the isoflavone-metabolizing bacteria catalyzed C-ring cleavage of –EC to produce (2S)-1-(3, 4-dihydroxyphenyl)-3-(2, 4, 6-trihydroxyphenyl)propan-2-ol (1S), and the formation was accelerated in the presence of hydrogen.',26312950),(3858,'human','(-)-Epicatechin','All the isoflavone-metabolizing bacteria catalyzed C-ring cleavage of –EC to produce (2S)-1-(3, 4-dihydroxyphenyl)-3-(2, 4, 6-trihydroxyphenyl)propan-2-ol (1S), and the formation was accelerated in the presence of hydrogen.',26312950),(3859,'human','(-)-Epicatechin','All the isoflavone-metabolizing bacteria catalyzed C-ring cleavage of –EC to produce (2S)-1-(3, 4-dihydroxyphenyl)-3-(2, 4, 6-trihydroxyphenyl)propan-2-ol (1S), and the formation was accelerated in the presence of hydrogen.',26312950),(3860,'human','(-)-Epicatechin','All the isoflavone-metabolizing bacteria catalyzed C-ring cleavage of –EC to produce (2S)-1-(3, 4-dihydroxyphenyl)-3-(2, 4, 6-trihydroxyphenyl)propan-2-ol (1S), and the formation was accelerated in the presence of hydrogen.',26312950),(3861,'human','Cianidanol','The bacterial strains also showed ring-cleaving ability on +C to yield (2R)-1-(3, 4-dihydroxyphenyl)-3-(2, 4, 6-trihydroxyphenyl)propan-2-ol (1R), and this reaction was enhanced by the presence of hydrogen, with the exception of S. equolifaciens JCM 16059.',26312950),(3862,'human','Cianidanol','The bacterial strains also showed ring-cleaving ability on +C to yield (2R)-1-(3, 4-dihydroxyphenyl)-3-(2, 4, 6-trihydroxyphenyl)propan-2-ol (1R), and this reaction was enhanced by the presence of hydrogen, with the exception of S. equolifaciens JCM 16059.',26312950),(3863,'human','Cianidanol','The bacterial strains also showed ring-cleaving ability on +C to yield (2R)-1-(3, 4-dihydroxyphenyl)-3-(2, 4, 6-trihydroxyphenyl)propan-2-ol (1R), and this reaction was enhanced by the presence of hydrogen, with the exception of S. equolifaciens JCM 16059.',26312950),(3864,'human','Daidzein','Equol producing-related species Adlercreutzia equolifaciens and Bifdobacterium bifdum, showed higher relative abundance in EP than in NP.',31168326),(3865,'human','Daidzein','Equol producing-related species Adlercreutzia equolifaciens and Bifdobacterium bifdum, showed higher relative abundance in EP than in NP.',31168326),(3866,'human','Ginsenoside Rb1','The enzyme potently biotransformed ginsenoside Rb1, loganin, arctiin and arbutin to ginsenoside Rd, loganetin, arctigenin and hydroquinone, respectively, but was not active in the case of hesperidin, and kakkalide.',22466563),(3867,'human','Loganin','The enzyme potently biotransformed ginsenoside Rb1, loganin, arctiin and arbutin to ginsenoside Rd, loganetin, arctigenin and hydroquinone, respectively, but was not active in the case of hesperidin, and kakkalide.',22466563),(3868,'human','Arctiin','The enzyme potently biotransformed ginsenoside Rb1, loganin, arctiin and arbutin to ginsenoside Rd, loganetin, arctigenin and hydroquinone, respectively, but was not active in the case of hesperidin, and kakkalide.',22466563),(3869,'human','Arbutin','The enzyme potently biotransformed ginsenoside Rb1, loganin, arctiin and arbutin to ginsenoside Rd, loganetin, arctigenin and hydroquinone, respectively, but was not active in the case of hesperidin, and kakkalide.',22466563),(3870,'human','null','Compared with the control, the stimulation of HCT116 cells with C. butyricum supernatant and sodium butyrate induced 4.5-fold and 4.2-fold upregulation of the GPR43 mRNA levels, 2.6-fold and 7.0-fold increases in the GPR109A mRNA levels.',31734354),(3871,'human','null','Compared with the control, the stimulation of HCT116 cells with C. butyricum supernatant and sodium butyrate induced 4.5-fold and 4.2-fold upregulation of the GPR43 mRNA levels, 2.6-fold and 7.0-fold increases in the GPR109A mRNA levels.',31734354),(3872,'human','null','The mRNA level of P21WAF1 was increased in HCT116 cells treated with C. butyricum.',31734354),(3873,'human','null','Indole promoted mRNA expression of TJ-associated molecules, such as Cldn7, Ocln, and Tjp1 in the human adenocarcinoma cell line, Caco-2.',24278294),(3874,'human','null','Indole promoted mRNA expression of TJ-associated molecules, such as Cldn7, Ocln, and Tjp1 in the human adenocarcinoma cell line, Caco-2.',24278294),(3875,'human','null','Indole promoted mRNA expression of TJ-associated molecules, such as Cldn7, Ocln, and Tjp1 in the human adenocarcinoma cell line, Caco-2.',24278294),(3876,'human','null','The function of butyrate as a GPR109A agonist was investigated using a heterologous expression system in which the coupling of pertussis toxin-sensitive Gi proteins to the receptor was monitored with the activity of inwardly-rectifying potassium (GIRK) channels as the read-out.',19276343),(3877,'human','null','The concentration of ursodesoxycholic acid in stool samples was highly positively correlated with the level of Ruminococcus.',28587349),(3878,'human','null','The concentrations of phenylalanine and glutamic acid were highly positively correlated with the levels of Phascolarctobacterium and Acidiminobacter.',28587349),(3879,'human','null','The concentrations of phenylalanine and glutamic acid were highly positively correlated with the levels of Phascolarctobacterium and Acidiminobacter.',28587349),(3880,'human','null','The concentrations of serine and threonine were moderately positively correlated with the levels of Phascolarctobacterium and Acidiminobacter.',28587349),(3881,'human','null','The concentrations of serine and threonine were moderately positively correlated with the levels of Phascolarctobacterium and Acidiminobacter.',28587349),(3882,'human','Tryptophan','Exposure to 1 mM indole led to an increase in the expression of two Toll-like receptor (TLR) genes—TLR-3 and TLR-9.',19966295),(3883,'human','Tryptophan','Exposure to 1 mM indole led to an increase in the expression of two Toll-like receptor (TLR) genes—TLR-3 and TLR-9.',19966295),(3884,'human','Tryptophan','Additionally, genes involved in the modulation of TLR signaling—TOLLIP, SIGIRR, and SOCS5—were also induced by indole.',19966295),(3885,'human','Tryptophan','Additionally, genes involved in the modulation of TLR signaling—TOLLIP, SIGIRR, and SOCS5—were also induced by indole.',19966295),(3886,'human','Tryptophan','Additionally, genes involved in the modulation of TLR signaling—TOLLIP, SIGIRR, and SOCS5—were also induced by indole.',19966295),(3887,'human','Tryptophan','Similarly, genes involved in the phosphoinositide-3-kinase (PI3K) pathway, which modulates TLR signaling and leads to antiinflammatory cytokine IL-10 induction, were also induced by indole.',19966295),(3888,'human','Tryptophan','Three genes involved in PI3K signaling (PIK3CA, PIK3CB, and PIK3CD) were induced, as was the transcription factor AKT1 that is activated by signaling through the PI3K pathway.',19966295),(3889,'human','Tryptophan','Three genes involved in PI3K signaling (PIK3CA, PIK3CB, and PIK3CD) were induced, as was the transcription factor AKT1 that is activated by signaling through the PI3K pathway.',19966295),(3890,'human','Tryptophan','Three genes involved in PI3K signaling (PIK3CA, PIK3CB, and PIK3CD) were induced, as was the transcription factor AKT1 that is activated by signaling through the PI3K pathway.',19966295),(3891,'human','Tryptophan','Downstream target genes of AKT1, including the cAMP response element-binding protein and IL-10, were also induced upon addition of indole.',19966295),(3892,'human','Tryptophan','In addition, the receptor for the proinflammatory cytokine IL-2, IL-4, IL-18 binding protein, IL-24, and IL-28 were all induced, suggesting that indole exposure leads to an increase in cytokine expression and signaling in HCT-8 intestinal epithelial cells.',19966295),(3893,'human','Tryptophan','In addition, the receptor for the proinflammatory cytokine IL-2, IL-4, IL-18 binding protein, IL-24, and IL-28 were all induced, suggesting that indole exposure leads to an increase in cytokine expression and signaling in HCT-8 intestinal epithelial cells.',19966295),(3894,'human','Tryptophan','In addition, the receptor for the proinflammatory cytokine IL-2, IL-4, IL-18 binding protein, IL-24, and IL-28 were all induced, suggesting that indole exposure leads to an increase in cytokine expression and signaling in HCT-8 intestinal epithelial cells.',19966295),(3895,'human','Tryptophan','In addition, the receptor for the proinflammatory cytokine IL-2, IL-4, IL-18 binding protein, IL-24, and IL-28 were all induced, suggesting that indole exposure leads to an increase in cytokine expression and signaling in HCT-8 intestinal epithelial cells.',19966295),(3896,'human','Tryptophan','In addition, the receptor for the proinflammatory cytokine IL-2, IL-4, IL-18 binding protein, IL-24, and IL-28 were all induced, suggesting that indole exposure leads to an increase in cytokine expression and signaling in HCT-8 intestinal epithelial cells.',19966295),(3897,'human','Tryptophan','In addition to altering the expression of cytokines, indole also induced the expression of several chemokines (CCL25, CCL4, CCL16, CXCL3, XCL1, and CX3CL1) in HCT-8 cells.',19966295),(3898,'human','Tryptophan','In addition to altering the expression of cytokines, indole also induced the expression of several chemokines (CCL25, CCL4, CCL16, CXCL3, XCL1, and CX3CL1) in HCT-8 cells.',19966295),(3899,'human','Tryptophan','In addition to altering the expression of cytokines, indole also induced the expression of several chemokines (CCL25, CCL4, CCL16, CXCL3, XCL1, and CX3CL1) in HCT-8 cells.',19966295),(3900,'human','Tryptophan','In addition to altering the expression of cytokines, indole also induced the expression of several chemokines (CCL25, CCL4, CCL16, CXCL3, XCL1, and CX3CL1) in HCT-8 cells.',19966295),(3901,'human','Tryptophan','In addition to altering the expression of cytokines, indole also induced the expression of several chemokines (CCL25, CCL4, CCL16, CXCL3, XCL1, and CX3CL1) in HCT-8 cells.',19966295),(3902,'human','Tryptophan','In addition to altering the expression of cytokines, indole also induced the expression of several chemokines (CCL25, CCL4, CCL16, CXCL3, XCL1, and CX3CL1) in HCT-8 cells.',19966295),(3903,'human','Tryptophan','Specifically, the expression of dual-specificity phosphatases (DUSPs)—Dusp1, Dusp3, Dusp10, Dusp16, Dusp18, and Dusp22, which function as inhibitors of MAPK signaling —was increased upon indole exposure.',19966295),(3904,'human','Tryptophan','Specifically, the expression of dual-specificity phosphatases (DUSPs)—Dusp1, Dusp3, Dusp10, Dusp16, Dusp18, and Dusp22, which function as inhibitors of MAPK signaling —was increased upon indole exposure.',19966295),(3905,'human','Tryptophan','Specifically, the expression of dual-specificity phosphatases (DUSPs)—Dusp1, Dusp3, Dusp10, Dusp16, Dusp18, and Dusp22, which function as inhibitors of MAPK signaling —was increased upon indole exposure.',19966295),(3906,'human','Tryptophan','Specifically, the expression of dual-specificity phosphatases (DUSPs)—Dusp1, Dusp3, Dusp10, Dusp16, Dusp18, and Dusp22, which function as inhibitors of MAPK signaling —was increased upon indole exposure.',19966295),(3907,'human','Tryptophan','Specifically, the expression of dual-specificity phosphatases (DUSPs)—Dusp1, Dusp3, Dusp10, Dusp16, Dusp18, and Dusp22, which function as inhibitors of MAPK signaling —was increased upon indole exposure.',19966295),(3908,'human','Daidzin','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3909,'human','Daidzin','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3910,'human','Daidzin','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3911,'human','Daidzin','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3912,'human','Daidzin','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3913,'human','Genistin','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3914,'human','Genistin','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3915,'human','Genistin','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3916,'human','Genistin','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3917,'human','Genistin','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3918,'human','Glycitin','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3919,'human','Glycitin','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3920,'human','Glycitin','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3921,'human','Glycitin','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3922,'human','Glycitin','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3923,'human','Astragalin','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3924,'human','Astragalin','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3925,'human','Astragalin','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3926,'human','Astragalin','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3927,'human','Astragalin','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3928,'human','gamma-Butyrobetaine hydrochloride','Only 1 Clostridiales bacterium, Emergencia timonensis, that converted γBB to TMA.',30530985),(3929,'human','Phenylalanine','After incubation of wild-type C. sporogenes with phenylalanine, tyrosine and tryptophan, we observed robust production of PPA, 4-OH-PPA and IPA.',29168502),(3930,'human','Tyrosine','After incubation of wild-type C. sporogenes with phenylalanine, tyrosine and tryptophan, we observed robust production of PPA, 4-OH-PPA and IPA.',29168502),(3931,'human','Tryptophan','After incubation of wild-type C. sporogenes with phenylalanine, tyrosine and tryptophan, we observed robust production of PPA, 4-OH-PPA and IPA.',29168502),(3932,'human','Tryptophan','Notably, we identified four new IPA-producing gut bacteria: Peptostreptococcus anaerobius CC14N and three strains of Clostridium cadaveris.',29168502),(3933,'human','Tryptophan','Notably, we identified four new IPA-producing gut bacteria: Peptostreptococcus anaerobius CC14N and three strains of Clostridium cadaveris.',29168502),(3934,'human','Tryptophan','Notably, we identified four new IPA-producing gut bacteria: Peptostreptococcus anaerobius CC14N and three strains of Clostridium cadaveris.',29168502),(3935,'human','Tryptophan','Notably, we identified four new IPA-producing gut bacteria: Peptostreptococcus anaerobius CC14N and three strains of Clostridium cadaveris.',29168502),(3936,'human','Urocanic acid','We also noted that IRS1 levels were reduced by 8 hr treatment of imidazole propionate in the human embryonic kidney cell line HEK293.',30401435),(3937,'human','Urocanic acid','Imidazole propionate activates mTORC1.',30401435),(3938,'human','Urocanic acid','We also noted that IRS1 levels were reduced by 8 hr treatment of imidazole propionate in the human embryonic kidney cell line HEK293.',30401435),(3939,'human','Urocanic acid','Imidazole propionate activates mTORC1.',30401435),(3940,'human','null','Propionate was the most potent agonist for both GPR41 and GPR43.',12711604),(3941,'human','null','Propionate was the most potent agonist for both GPR41 and GPR43.',12711604),(3942,'human','null','Acetate was more selective for GPR43, whereas butyrate and isobutyrate were more active on GPR41',12711604),(3943,'human','null','Acetate was more selective for GPR43, whereas butyrate and isobutyrate were more active on GPR41',12711604),(3949,'human','Ginsenoside Rg3','Among the bacteria isolated from human fecal microflora, Bacteroides sp., Eubacterium sp., and Bifidobacterium sp. metabolized ginsenoside Rg3 to protopanaxadiol via ginsenoside Rh2.',11824558),(3950,'human','Ginsenoside Rg3','Among the bacteria isolated from human fecal microflora, Bacteroides sp., Eubacterium sp., and Bifidobacterium sp. metabolized ginsenoside Rg3 to protopanaxadiol via ginsenoside Rh2.',11824558),(3951,'human','Ginsenoside Rg3','Among the bacteria isolated from human fecal microflora, Bacteroides sp., Eubacterium sp., and Bifidobacterium sp. metabolized ginsenoside Rg3 to protopanaxadiol via ginsenoside Rh2.',11824558),(3952,'human','Ginsenoside Rh2','Among the bacteria isolated from human fecal microflora, Bacteroides sp., Eubacterium sp., and Bifidobacterium sp. metabolized ginsenoside Rg3 to protopanaxadiol via ginsenoside Rh2.',11824558),(3953,'human','Ginsenoside Rh2','Among the bacteria isolated from human fecal microflora, Bacteroides sp., Eubacterium sp., and Bifidobacterium sp. metabolized ginsenoside Rg3 to protopanaxadiol via ginsenoside Rh2.',11824558),(3954,'human','(20R)-Ginsenoside Rg3','Among the bacteria isolated from human fecal microflora, Bacteroides sp., Eubacterium sp., and Bifidobacterium sp. metabolized ginsenoside Rg3 to protopanaxadiol via ginsenoside Rh2.',11824558),(3955,'human','(20R)-Ginsenoside Rg3','Among the bacteria isolated from human fecal microflora, Bacteroides sp., Eubacterium sp., and Bifidobacterium sp. metabolized ginsenoside Rg3 to protopanaxadiol via ginsenoside Rh2.',11824558),(3956,'human','20(R)-Ginsenoside Rh2','Among the bacteria isolated from human fecal microflora, Bacteroides sp., Eubacterium sp., and Bifidobacterium sp. metabolized ginsenoside Rg3 to protopanaxadiol via ginsenoside Rh2.',11824558),(3957,'human','Arctigenin','In the present study, we isolated one single bacterium, which we named Blautia sp. AUH-JLD56, capable of solely biotransforming arctiin or arctigenin to (-)-3\'-DMAG.',24236649),(3958,'human','null','Ruminococcus, showed positive correlations with plasma glycerophospholipids.',31748797),(3959,'human','null','The RNA-Seq analysis showed that cytochrome P450 1A1 (Cyp1A1) is among the top 3 UroA upregulated genes.',30626868),(3960,'human','null','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3961,'human','null','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3962,'human','null','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3963,'human','null','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3964,'human','null','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3965,'human','null','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3966,'human','null','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3967,'human','null','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3968,'human','null','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3969,'human','null','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3970,'human','null','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3971,'human','null','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3972,'human','null','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3973,'human','null','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3974,'human','null','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3975,'human','null','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3976,'human','null','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3977,'human','null','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3978,'human','null','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3979,'human','null','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3980,'human','null','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3981,'human','null','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3982,'human','null','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3983,'human','null','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3984,'human','null','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3985,'human','null','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3986,'human','null','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3987,'human','null','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3988,'human','null','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3989,'human','null','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3990,'human','null','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3991,'human','null','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3992,'human','null','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3993,'human','null','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3994,'human','null','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3995,'human','null','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3996,'human','null','The fold changes in mRNA levels of claudin 4 (Cldn4), occludin (Ocln), and Zona occludens 1 (ZO1) in HT29 cells were determined by RT PCR method.',30626868),(3997,'human','null','The fold changes in mRNA levels of claudin 4 (Cldn4), occludin (Ocln), and Zona occludens 1 (ZO1) in HT29 cells were determined by RT PCR method.',30626868),(3998,'human','null','The fold changes in mRNA levels of claudin 4 (Cldn4), occludin (Ocln), and Zona occludens 1 (ZO1) in HT29 cells were determined by RT PCR method.',30626868),(3999,'human','null','UroA/UAS03 induced NQO1 expression was also confirmed in HT29 cells (Supplementary Fig. 9d).',30626868),(4000,'human','L-Fucose','R. inulinivorans A2-194 is also able to grow on the host-derived sugar fucose, following a lag period, producing propionate and propanol as additional fermentation products.',16740940),(4001,'human','L-Fucose','R. inulinivorans A2-194 is also able to grow on the host-derived sugar fucose, following a lag period, producing propionate and propanol as additional fermentation products.',16740940),(4002,'human','L-Fucose','R. inulinivorans A2-194 is also able to grow on the host-derived sugar fucose, following a lag period, producing propionate and propanol as additional fermentation products.',16740940),(4003,'human','null','Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.',29572891),(4004,'human','null','Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.',29572891),(4005,'human','null','Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.',29572891),(4006,'human','null','Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.',29572891),(4007,'human','null','Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.',29572891),(4008,'human','null','Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.',29572891),(4009,'human','null','Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.',29572891),(4010,'human','null','Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.',29572891),(4011,'human','null','Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.',29572891),(4012,'human','null','Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.',29572891),(4013,'human','null','Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.',29572891),(4014,'human','null','Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.',29572891),(4015,'human','null','Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.',29572891),(4016,'human','null','Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.',29572891),(4023,'human','D-Glucose','The major end products of glucose metabolism are lactic, acetic and formic acids.',20023061),(4024,'human','D-Glucose','The major end products of glucose metabolism are lactic, acetic and formic acids.',20023061),(4025,'human','D-Glucose','The major end products of glucose metabolism are lactic, acetic and formic acids.',20023061),(4026,'human','D-Glucose','The major end products of glucose metabolism are lactic, acetic and formic acids.',20023061),(4027,'human','D-Glucose','The major end products of glucose metabolism were succinate and acetate.',19749029),(4028,'human','D-Glucose','The major end products of glucose metabolism were succinate and acetate.',19749029),(4029,'human','Avicularin','Avicularin (M1) was anaerobic metabolized to its aglycone quercetin (M2) and methoxylated avicularin (M3, M4), then quercetin was converted to quercetin glycosides: quercetin-3-O-rhamnoside (M5), quercetin-3-O-glucoside (M6) and quercetin-7-O-glucoside (M7) by Bacillus sp. 46.',24463398),(4030,'human','Quercetin','Avicularin (M1) was anaerobic metabolized to its aglycone quercetin (M2) and methoxylated avicularin (M3, M4), then quercetin was converted to quercetin glycosides: quercetin-3-O-rhamnoside (M5), quercetin-3-O-glucoside (M6) and quercetin-7-O-glucoside (M7) by Bacillus sp. 46.',24463398),(4031,'human','Quercetin','Avicularin (M1) was anaerobic metabolized to its aglycone quercetin (M2) and methoxylated avicularin (M3, M4), then quercetin was converted to quercetin glycosides: quercetin-3-O-rhamnoside (M5), quercetin-3-O-glucoside (M6) and quercetin-7-O-glucoside (M7) by Bacillus sp. 46.',24463398),(4032,'human','Quercetin','Avicularin (M1) was anaerobic metabolized to its aglycone quercetin (M2) and methoxylated avicularin (M3, M4), then quercetin was converted to quercetin glycosides: quercetin-3-O-rhamnoside (M5), quercetin-3-O-glucoside (M6) and quercetin-7-O-glucoside (M7) by Bacillus sp. 46.',24463398),(4033,'human','Avicularin','Avicularin (M1) was anaerobic metabolized to its aglycone quercetin (M2) and methoxylated avicularin (M3, M4), then quercetin was converted to quercetin glycosides: quercetin-3-O-rhamnoside (M5), quercetin-3-O-glucoside (M6) and quercetin-7-O-glucoside (M7) by Bacillus sp. 46.',24463398),(4034,'human','null','Sodium acetate (10�?0 mM) significantly increased TPH1mRNA expression.',25550456),(4035,'human','D-Glucose','C. scindens fermented glucose mainly to ethanol, acetate, formate, and H2.',30737348),(4036,'human','D-Glucose','C. scindens fermented glucose mainly to ethanol, acetate, formate, and H2.',30737348),(4037,'human','D-Glucose','C. scindens fermented glucose mainly to ethanol, acetate, formate, and H2.',30737348),(4038,'human','D-Glucose','C. scindens fermented glucose mainly to ethanol, acetate, formate, and H2.',30737348),(4039,'human','null','A trace amount of propionate was detected as an end product of metabolism.',19542131),(4040,'human','D-Glucose','The two strains produced significant levels of butyric and isobutyric acids as end products from glucose.',19542124),(4041,'human','D-Glucose','The two strains produced significant levels of butyric and isobutyric acids as end products from glucose.',19542124),(4042,'human','D-Glucose','The two strains produced significant levels of butyric and isobutyric acids as end products from glucose.',19542124),(4043,'human','D-Glucose','The two strains produced significant levels of butyric and isobutyric acids as end products from glucose.',19542124),(4044,'human','null','Butyrate stimulated a rapid twofold increase of IL- 22R1 mRNA expression that, after a total 8 h incubation period, translated to the protein level.',27801948),(4045,'human','(-)-Epicatechin','Strain MT4s-5 readily converted −EC and −C into (2S)-1-(3,4-dihydroxyphenyl)-3-(2,4,6- trihydroxyphenyl) propan-2-ol (1S) and +C into (2R)-1-(3,4- dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol (1R).',25947926),(4046,'human','(-)-Catechin','Strain MT4s-5 readily converted −EC and −C into (2S)-1-(3,4-dihydroxyphenyl)-3-(2,4,6- trihydroxyphenyl) propan-2-ol (1S) and +C into (2R)-1-(3,4- dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol (1R).',25947926),(4047,'human','Cianidanol','Strain MT4s-5 readily converted −EC and −C into (2S)-1-(3,4-dihydroxyphenyl)-3-(2,4,6- trihydroxyphenyl) propan-2-ol (1S) and +C into (2R)-1-(3,4- dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol (1R).',25947926),(4048,'human','(-)-Epicatechin','Strain JCM 9979 easily converted −EC and +C into 1S and 1R, respectively.',25947926),(4049,'human','Cianidanol','Strain JCM 9979 easily converted −EC and +C into 1S and 1R, respectively.',25947926),(4050,'human','(2S)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','F. plautii MT42 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4051,'human','(2S)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','F. plautii MT42 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4052,'human','(2R)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','F. plautii MT42 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4053,'human','(2R)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','F. plautii MT42 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4054,'human','(2S)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','Similarly, strain MT42 could convert 2S and 2R to their corresponding 4-hydroxy-5-(3-hydroxyphenyl) valeric acids (5R from 2S, 5S from 2R) and 5-(3-hydroxyphenyl)-γ-valerolactones (6R from 2S, 6S from 2R).',25947926),(4055,'human','(2R)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','Similarly, strain MT42 could convert 2S and 2R to their corresponding 4-hydroxy-5-(3-hydroxyphenyl) valeric acids (5R from 2S, 5S from 2R) and 5-(3-hydroxyphenyl)-γ-valerolactones (6R from 2S, 6S from 2R).',25947926),(4056,'human','(2S)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','Similarly, strain MT42 could convert 2S and 2R to their corresponding 4-hydroxy-5-(3-hydroxyphenyl) valeric acids (5R from 2S, 5S from 2R) and 5-(3-hydroxyphenyl)-γ-valerolactones (6R from 2S, 6S from 2R).',25947926),(4057,'human','(2R)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','Similarly, strain MT42 could convert 2S and 2R to their corresponding 4-hydroxy-5-(3-hydroxyphenyl) valeric acids (5R from 2S, 5S from 2R) and 5-(3-hydroxyphenyl)-γ-valerolactones (6R from 2S, 6S from 2R).',25947926),(4058,'human','(2S)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','F. plautii ATCC 29863 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4059,'human','(2S)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','F. plautii ATCC 29863 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4060,'human','(2R)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','F. plautii ATCC 29863 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4061,'human','(2R)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','F. plautii ATCC 29863 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4062,'human','(2S)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','F. plautii ATCC 29863 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4063,'human','(2R)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','F. plautii ATCC 29863 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4064,'human','(2S)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','F. plautii ATCC 29863 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4065,'human','(2R)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','F. plautii ATCC 29863 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4066,'human','(2S)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','F. plautii ATCC 49531  was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4067,'human','(2S)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','F. plautii ATCC 49531  was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4068,'human','(2R)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','F. plautii ATCC 49531  was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4069,'human','(2R)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','F. plautii ATCC 49531  was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4070,'human','(2S)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','F. plautii ATCC 49531  was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4071,'human','(2R)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','F. plautii ATCC 49531  was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4072,'human','(2S)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','F. plautii ATCC 49531  was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4073,'human','(2R)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','F. plautii ATCC 49531  was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4075,'human','Daidzein','The bacterium cleaved the C-ring of daidzein to produce O-demethylangolensin ( O-Dma).',12070764),(4076,'human','Daidzein','The human intestinal bacterium Slackia isoflavoniconvertens converts the main soybean isoflavones daidzein and genistein to equol and 5-hydroxy-equol, respectively.',23542626),(4077,'human','Genistein','The human intestinal bacterium Slackia isoflavoniconvertens converts the main soybean isoflavones daidzein and genistein to equol and 5-hydroxy-equol, respectively.',23542626),(4083,'human','D-Glucose','The organism produced acetate, lactate, and ethanol as the major products of glucose fermentation.',16337765),(4084,'human','D-Glucose','The organism produced acetate, lactate, and ethanol as the major products of glucose fermentation.',16337765),(4085,'human','D-Glucose','The organism produced acetate, lactate, and ethanol as the major products of glucose fermentation.',16337765),(4091,'human','Myricitrin','Specifically, the majority of the strains such as Escherichia sp. 12 could metabolize myricitrin (M1) to quercetin-3-O-rhamnoside (M2) and myricetin (M3) (Figure 1C), and the metabolites of some strains like Enterococcus sp. 45 were large amounts of myricetin (M3) and quercetin (M4) (Figure 1D), while the other strains, for example, Escherichia sp. 33, could degrade myricitrin completely to myricetin and quercetin.',24882500),(4092,'human','Myricitrin','Specifically, the majority of the strains such as Escherichia sp. 12 could metabolize myricitrin (M1) to quercetin-3-O-rhamnoside (M2) and myricetin (M3) (Figure 1C), and the metabolites of some strains like Enterococcus sp. 45 were large amounts of myricetin (M3) and quercetin (M4) (Figure 1D), while the other strains, for example, Escherichia sp. 33, could degrade myricitrin completely to myricetin and quercetin.',24882500),(4093,'human','Myricitrin','Specifically, the majority of the strains such as Escherichia sp. 12 could metabolize myricitrin (M1) to quercetin-3-O-rhamnoside (M2) and myricetin (M3) (Figure 1C), and the metabolites of some strains like Enterococcus sp. 45 were large amounts of myricetin (M3) and quercetin (M4) (Figure 1D), while the other strains, for example, Escherichia sp. 33, could degrade myricitrin completely to myricetin and quercetin.',24882500),(4094,'human','Myricitrin','Specifically, the majority of the strains such as Escherichia sp. 12 could metabolize myricitrin (M1) to quercetin-3-O-rhamnoside (M2) and myricetin (M3) (Figure 1C), and the metabolites of some strains like Enterococcus sp. 45 were large amounts of myricetin (M3) and quercetin (M4) (Figure 1D), while the other strains, for example, Escherichia sp. 33, could degrade myricitrin completely to myricetin and quercetin.',24882500),(4095,'human','Myricitrin','Specifically, the majority of the strains such as Escherichia sp. 12 could metabolize myricitrin (M1) to quercetin-3-O-rhamnoside (M2) and myricetin (M3) (Figure 1C), and the metabolites of some strains like Enterococcus sp. 45 were large amounts of myricetin (M3) and quercetin (M4) (Figure 1D), while the other strains, for example, Escherichia sp. 33, could degrade myricitrin completely to myricetin and quercetin.',24882500),(4096,'human','Myricitrin','Specifically, the majority of the strains such as Escherichia sp. 12 could metabolize myricitrin (M1) to quercetin-3-O-rhamnoside (M2) and myricetin (M3) (Figure 1C), and the metabolites of some strains like Enterococcus sp. 45 were large amounts of myricetin (M3) and quercetin (M4) (Figure 1D), while the other strains, for example, Escherichia sp. 33, could degrade myricitrin completely to myricetin and quercetin.',24882500),(4099,'human','D-Glucose','Strain M25(T) was negative for both catalase and oxidase activity, utilized carbohydrates as fermentable substrates, produced lactate and acetate as the major end products of glucose fermentation in PYG medium.',22544782),(4100,'human','D-Glucose','Strain M25(T) was negative for both catalase and oxidase activity, utilized carbohydrates as fermentable substrates, produced lactate and acetate as the major end products of glucose fermentation in PYG medium.',22544782),(4101,'human','Succinate','However, strain YIT 11850(T) was able to decarboxylate succinate to propionate.',19060046),(4102,'human','D-Glucose','Cells of the other isolate, strain YIT 11860(T), were non-motile, rod-shaped, positive for aesculin hydrolysis, negative for indole production, produced succinic and acetic acids as end products of glucose metabolism and possessed a DNA G+C content of 45.5 mol%.',19060046),(4103,'human','D-Glucose','Cells of the other isolate, strain YIT 11860(T), were non-motile, rod-shaped, positive for aesculin hydrolysis, negative for indole production, produced succinic and acetic acids as end products of glucose metabolism and possessed a DNA G+C content of 45.5 mol%.',19060046),(4104,'human','Cynaroside','Three metabolites (acetylated luteoloside, luteolin and deoxygenated luteolin) were detected in the fractions isolated from the bacterial samples.',25899973),(4106,'human','D-(+)-Cellobiose','Acetate and succinate were the major end products of cellulose and cellobiose fermentation.',21357460),(4107,'human','D-(+)-Cellobiose','Acetate and succinate were the major end products of cellulose and cellobiose fermentation.',21357460),(4112,'human','null','Acetate and butyrate were produced by strain 3BBH22T as metabolic end-products.',29745868),(4113,'human','null','Acetate and butyrate were produced by strain 3BBH22T as metabolic end-products.',29745868),(4114,'human','Isoquercitrin','Fermentation of quercetin-3-glucoside by growing cultures of Eubacterium ramulus and Enterococcus casseliflavus.',12747214),(4115,'human','Isoquercitrin','Fermentation of quercetin-3-glucoside by growing cultures of Eubacterium ramulus and Enterococcus casseliflavus.',12747214),(4116,'human','null','We also observed both P1NP and CTX-1 to be positively associated with osteopenia-enriched Allisonella, Klebsiella and Megasphaera.',32392181),(4117,'human','null','We also observed both P1NP and CTX-1 to be positively associated with osteopenia-enriched Allisonella, Klebsiella and Megasphaera.',32392181),(4118,'human','null','We also observed both P1NP and CTX-1 to be positively associated with osteopenia-enriched Allisonella, Klebsiella and Megasphaera.',32392181),(4119,'human','null','We also observed both P1NP and CTX-1 to be positively associated with osteopenia-enriched Allisonella, Klebsiella and Megasphaera.',32392181),(4120,'human','null','We also observed both P1NP and CTX-1 to be positively associated with osteopenia-enriched Allisonella, Klebsiella and Megasphaera.',32392181),(4121,'human','null','We also observed both P1NP and CTX-1 to be positively associated with osteopenia-enriched Allisonella, Klebsiella and Megasphaera.',32392181),(4122,'human','Daidzein','In conclusion, the results obtained in this study suggest that the presence of Bacteroides genus in the fecal samples of control subjects seems to be involved in the daidzein transformation to equol, an antioxidative, anti-carcinogenic molecule, which shows the most active estrogenic properties of soy isoflavones.\n',32218321),(4123,'human','Daidzein','In conclusion, the results obtained in this study suggest that the presence of Bacteroides genus in the fecal samples of control subjects seems to be involved in the daidzein transformation to equol, an antioxidative, anti-carcinogenic molecule, which shows the most active estrogenic properties of soy isoflavones.\n',32218321),(4124,'human','Daidzein','In conclusion, the results obtained in this study suggest that the presence of Bacteroides genus in the fecal samples of control subjects seems to be involved in the daidzein transformation to equol, an antioxidative, anti-carcinogenic molecule, which shows the most active estrogenic properties of soy isoflavones.\n',32218321),(4125,'human','Daidzein','In conclusion, the results obtained in this study suggest that the presence of Bacteroides genus in the fecal samples of control subjects seems to be involved in the daidzein transformation to equol, an antioxidative, anti-carcinogenic molecule, which shows the most active estrogenic properties of soy isoflavones.\n',32218321),(4126,'human','Daidzein','In conclusion, the results obtained in this study suggest that the presence of Bacteroides genus in the fecal samples of control subjects seems to be involved in the daidzein transformation to equol, an antioxidative, anti-carcinogenic molecule, which shows the most active estrogenic properties of soy isoflavones.\n',32218321),(4127,'human','Daidzein','In conclusion, the results obtained in this study suggest that the presence of Bacteroides genus in the fecal samples of control subjects seems to be involved in the daidzein transformation to equol, an antioxidative, anti-carcinogenic molecule, which shows the most active estrogenic properties of soy isoflavones.\n',32218321),(4131,'human','Pyruvate','According to the FDR-adjusted p-values (q-values) in multivariable linear regression, p-cresyl sulfate showed a significant association with six major genera, with positive correlations observed with Alistipes, Oscillibacter, and Subdoligranulum (q < 0.001, q < 0.001, and q = 0.023, respectively) and negative correlations with Lachnospira, Veillonella, and Megamonas (q = 0.039, q = 0.014, and q = 0.034, respectively). Serum p-cresyl glucuronide and indoxyl sulfate levels were positively correlated with Alistipes (q = 0.010 and q = 0.035, respectively) and Oscillibacter (q = 0.001 and q = 0.037, respectively). Serum TMAO levels were associated only with Oscillibacter (q = 0.006). Unexpectedly, we found Oscillibacter to be associated with all four measured uremic toxins, although each of these metabolites is derived from different parent compounds.',32560104),(4132,'human','Pyruvate','According to the FDR-adjusted p-values (q-values) in multivariable linear regression, p-cresyl sulfate showed a significant association with six major genera, with positive correlations observed with Alistipes, Oscillibacter, and Subdoligranulum (q < 0.001, q < 0.001, and q = 0.023, respectively) and negative correlations with Lachnospira, Veillonella, and Megamonas (q = 0.039, q = 0.014, and q = 0.034, respectively). Serum p-cresyl glucuronide and indoxyl sulfate levels were positively correlated with Alistipes (q = 0.010 and q = 0.035, respectively) and Oscillibacter (q = 0.001 and q = 0.037, respectively). Serum TMAO levels were associated only with Oscillibacter (q = 0.006). Unexpectedly, we found Oscillibacter to be associated with all four measured uremic toxins, although each of these metabolites is derived from different parent compounds.',32560104),(4133,'human','Pyruvate','According to the FDR-adjusted p-values (q-values) in multivariable linear regression, p-cresyl sulfate showed a significant association with six major genera, with positive correlations observed with Alistipes, Oscillibacter, and Subdoligranulum (q < 0.001, q < 0.001, and q = 0.023, respectively) and negative correlations with Lachnospira, Veillonella, and Megamonas (q = 0.039, q = 0.014, and q = 0.034, respectively). Serum p-cresyl glucuronide and indoxyl sulfate levels were positively correlated with Alistipes (q = 0.010 and q = 0.035, respectively) and Oscillibacter (q = 0.001 and q = 0.037, respectively). Serum TMAO levels were associated only with Oscillibacter (q = 0.006). Unexpectedly, we found Oscillibacter to be associated with all four measured uremic toxins, although each of these metabolites is derived from different parent compounds.',32560104),(4134,'human','Pyruvate','According to the FDR-adjusted p-values (q-values) in multivariable linear regression, p-cresyl sulfate showed a significant association with six major genera, with positive correlations observed with Alistipes, Oscillibacter, and Subdoligranulum (q < 0.001, q < 0.001, and q = 0.023, respectively) and negative correlations with Lachnospira, Veillonella, and Megamonas (q = 0.039, q = 0.014, and q = 0.034, respectively). Serum p-cresyl glucuronide and indoxyl sulfate levels were positively correlated with Alistipes (q = 0.010 and q = 0.035, respectively) and Oscillibacter (q = 0.001 and q = 0.037, respectively). Serum TMAO levels were associated only with Oscillibacter (q = 0.006). Unexpectedly, we found Oscillibacter to be associated with all four measured uremic toxins, although each of these metabolites is derived from different parent compounds.',32560104),(4135,'human','null','Here, we elucidate a micro biome-dependent pathway by which SG increases CA7S production. We show that a microbial metabolite,lithocholic acid (LCA), is increased in murine portal veins post-SG and by activating the vitamin D receptor, induces hepatic mSult2A1/hSULT2A expression to drive CA7S production. An SG-induced shift in the micro biome increases gut expression of the bile acid transporters Asbt and Osta, which in turn facilitate selective transport of LCA across the gut epithelium. Cecal microbiota transplant from SG animals is sufficient to recreate the pathway in germ-free (GF) animals. \nand GLP-1 secretion, causally connecting a microbial metabolite with the improvement of diabetic phe notypes.',33434516),(4136,'human','null','Here, we elucidate a micro biome-dependent pathway by which SG increases CA7S production. We show that a microbial metabolite,lithocholic acid (LCA), is increased in murine portal veins post-SG and by activating the vitamin D receptor, induces hepatic mSult2A1/hSULT2A expression to drive CA7S production. An SG-induced shift in the micro biome increases gut expression of the bile acid transporters Asbt and Osta, which in turn facilitate selective transport of LCA across the gut epithelium. Cecal microbiota transplant from SG animals is sufficient to recreate the pathway in germ-free (GF) animals. \nand GLP-1 secretion, causally connecting a microbial metabolite with the improvement of diabetic phe notypes.',33434516),(4137,'human','Trimethylamine','We found that TMAO treatment activated astrocytes, as indicated by a ~ 2-fold increase in LCN2 (Fig. 6a) and a ~ 3-fold increase in CD44 (Fig. 6b), a pro-inflammatory intercellular adhesion pro tein characteristic of reactive astrocytes [10].',32862276),(4138,'human','Trimethylamine','We found that TMAO treatment activated astrocytes, as indicated by a ~ 2-fold increase in LCN2 (Fig. 6a) and a ~ 3-fold increase in CD44 (Fig. 6b), a pro-inflammatory intercellular adhesion pro tein characteristic of reactive astrocytes [10].',32862276),(4139,'human','Baicalin','As shown in Fig. 5a, treatment with 10�?0 μM of BE signifcantly reduced LPS-induced IL-8 secretion. Similar inhibitory efects were observed for BI at 15�?0 μM; however, BE showed more potent efects.',31650468),(4140,'human','null','As shown in Fig. 5a, treatment with 10�?0 μM of BE signifcantly reduced LPS-induced IL-8 secretion. Similar inhibitory efects were observed for BI at 15�?0 μM; however, BE showed more potent efects.',31650468),(4141,'human','Trimethylamine','Plasma TMAO was positively associated with plasma levels of TNFα and IL-12 (Figures 3A,B), but not with IL-6 (p = 0.263, rho = 0.11) or IL-8.',33042155),(4142,'human','Trimethylamine','Plasma TMAO was positively associated with plasma levels of TNFα and IL-12 (Figures 3A,B), but not with IL-6 (p = 0.263, rho = 0.11) or IL-8.',33042155),(4143,'human','Trimethylamine','Herein, we found that plasma levels of LPS were significantly correlated with TMAOand the TMA precursors carnitine and y-BB in CVID patients (Figures 6A–C), indicating a relation between gut leakage and TMAO pathway metabolites.',33042155),(4144,'human','Secoisolariciresinol diglucoside','Over the first 30 min at 37 1C, the amount of SDG converted to SECO by resting cells of Clostridium sp. SDG Mt85-3Db was approximately 8- and 30-fold higher than the amount converted by B. distasonis DSM 20701T and B.ovatus SDG-Mt85-3Cy, respectively.',16765860),(4145,'human','Secoisolariciresinol diglucoside','Over the first 30 min at 37 1C, the amount of SDG converted to SECO by resting cells of Clostridium sp. SDG Mt85-3Db was approximately 8- and 30-fold higher than the amount converted by B. distasonis DSM 20701T and B.ovatus SDG-Mt85-3Cy, respectively.',16765860),(4146,'human','Secoisolariciresinol diglucoside','Over the first 30 min at 37 1C, the amount of SDG converted to SECO by resting cells of Clostridium sp. SDG Mt85-3Db was approximately 8- and 30-fold higher than the amount converted by B. distasonis DSM 20701T and B.ovatus SDG-Mt85-3Cy, respectively.',16765860),(4147,'human','Secoisolariciresinol diglucoside','Over the first 30 min at 37 1C, the amount of SDG converted to SECO by resting cells of Clostridium sp. SDG Mt85-3Db was approximately 8- and 30-fold higher than the amount converted by B. distasonis DSM 20701T and B.ovatus SDG-Mt85-3Cy, respectively.',16765860),(4148,'human','Secoisolariciresinol diglucoside','Over the first 30 min at 37 1C, the amount of SDG converted to SECO by resting cells of Clostridium sp. SDG Mt85-3Db was approximately 8- and 30-fold higher than the amount converted by B. distasonis DSM 20701T and B.ovatus SDG-Mt85-3Cy, respectively.',16765860),(4149,'human','Secoisolariciresinol diglucoside','Over the first 30 min at 37 1C, the amount of SDG converted to SECO by resting cells of Clostridium sp. SDG Mt85-3Db was approximately 8- and 30-fold higher than the amount converted by B. distasonis DSM 20701T and B.ovatus SDG-Mt85-3Cy, respectively.',16765860),(4150,'human','Lariciresinol','In addition,the previously isolated strain E. lenta SECO-Mt75m3 catalysed the reduction of pinoresinol to SECO via lariciresinol.',16765860),(4151,'human','Pinoresinol','In addition,the previously isolated strain E. lenta SECO-Mt75m3 catalysed the reduction of pinoresinol to SECO via lariciresinol.',16765860),(4152,'human','Matairesinol','However, methoxy or hydroxyl groups adjacent to the targeted methoxy groups may be required by P. productus SECO-Mt75m3 to catalyse demethylation, as evident from the inability of this organism to demethylate m-anisic acid. Incubation of P.productus SECO-Mt75m3 with ferulic acid led to complete conversion of the substrate without detection of any products by HPLC, suggesting that P. productus not only catalyses the demethylation of ferulic acid but also a number of additional reactions. More interestingly, P. productus SECO-Mt75m3 catalysed as well the demethyla tion of other lignans, namely lariciresinol (Fig. 2),\nmatairesinol and pinoresinol (data not shown). In addition, the previously isolated strain E. lenta SECO-Mt75m3 catalysed the reduction of pinoresinol to SECO via lariciresinol (Fig. 3).',16765860),(4153,'human','Secoisolariciresinol','In conclusion, SDG metabolising bacteria are expected to play a major role in the activation of various dietary lignans with ensuing consequences on the bioavailability and biological activity of enterolignans. Fig. 4 summarises the new data reported above and the present knowledge about the bacterial pathways of enterolignan production.',16765860),(4154,'human','Secoisolariciresinol','In conclusion, SDG metabolising bacteria are expected to play a major role in the activation of various dietary lignans with ensuing consequences on the bioavailability and biological activity of enterolignans. Fig. 4 summarises the new data reported above and the present knowledge about the bacterial pathways of enterolignan production.',16765860),(4155,'human','Secoisolariciresinol','In conclusion, SDG metabolising bacteria are expected to play a major role in the activation of various dietary lignans with ensuing consequences on the bioavailability and biological activity of enterolignans. Fig. 4 summarises the new data reported above and the present knowledge about the bacterial pathways of enterolignan production.',16765860),(4156,'human','Secoisolariciresinol','In conclusion, SDG metabolising bacteria are expected to play a major role in the activation of various dietary lignans with ensuing consequences on the bioavailability and biological activity of enterolignans. Fig. 4 summarises the new data reported above and the present knowledge about the bacterial pathways of enterolignan production.',16765860),(4157,'human','Pinoresinol','In conclusion, SDG metabolising bacteria are expected to play a major role in the activation of various dietary lignans with ensuing consequences on the bioavailability and biological activity of enterolignans. Fig. 4 summarises the new data reported above and the present knowledge about the bacterial pathways of enterolignan production.',16765860),(4158,'human','5,7-Dihydroxy-2-(4-hydroxyphenyl)chroman-4-one','A hypothetical pathway for the degradation of narin genin by E. ramulus has been proposed by Schneider and Blaut (2000) (Fig. 1a). The first step in this pathway is the isomerization of naringenin to naringenin chalcone, which is subsequently reduced to phloretin. Phloretin is further\nhydrolyzed to phloroglucinol and 3(4-hydroxyphenyl)pro pionic acid.',15127184),(4159,'human','Naringenin chalcone','A hypothetical pathway for the degradation of narin genin by E. ramulus has been proposed by Schneider and Blaut (2000) (Fig. 1a). The first step in this pathway is the isomerization of naringenin to naringenin chalcone, which is subsequently reduced to phloretin. Phloretin is further\nhydrolyzed to phloroglucinol and 3(4-hydroxyphenyl)pro pionic acid.',15127184),(4160,'human','Phloretin','A hypothetical pathway for the degradation of narin genin by E. ramulus has been proposed by Schneider and Blaut (2000) (Fig. 1a). The first step in this pathway is the isomerization of naringenin to naringenin chalcone, which is subsequently reduced to phloretin. Phloretin is further\nhydrolyzed to phloroglucinol and 3(4-hydroxyphenyl)pro pionic acid.',15127184),(4161,'human','Phloretin','A hypothetical pathway for the degradation of narin genin by E. ramulus has been proposed by Schneider and Blaut (2000) (Fig. 1a). The first step in this pathway is the isomerization of naringenin to naringenin chalcone, which is subsequently reduced to phloretin. Phloretin is further\nhydrolyzed to phloroglucinol and 3(4-hydroxyphenyl)pro pionic acid.',15127184),(4162,'human','Apigenin','The flavone apigenin is presum ably reduced to the flavanone naringenin in a first step which is followed by an isomerization to the corresponding chalcone structure. The chalcone is reduced to the dihydrochalcone phloretin, which is subsequently hydrolyzed to phloroglucinol and 3-(4-hydroxyphenyl)propionic acid (23). Although the degradation pathway of flavones by E. ramulus has successfully been elucidated, the enzymes involved have so far escaped purification and characterization.',15466559),(4163,'human','5,7-Dihydroxy-2-(4-hydroxyphenyl)chroman-4-one','The flavone apigenin is presum ably reduced to the flavanone naringenin in a first step which\nis followed by an isomerization to the corresponding chalcone structure. The chalcone is reduced to the dihydrochalcone phloretin, which is subsequently hydrolyzed to phloroglucinol and 3-(4-hydroxyphenyl)propionic acid (23). Although the degradation pathway of flavones by E. ramulus has successfully been elucidated, the enzymes involved have so far escaped purification and characterization.',15466559),(4164,'human','Naringenin chalcone','The flavone apigenin is presum ably reduced to the flavanone naringenin in a first step which\nis followed by an isomerization to the corresponding chalcone structure. The chalcone is reduced to the dihydrochalcone phloretin, which is subsequently hydrolyzed to phloroglucinol and 3-(4-hydroxyphenyl)propionic acid (23). Although the degradation pathway of flavones by E. ramulus has successfully been elucidated, the enzymes involved have so far escaped purification and characterization.',15466559),(4165,'human','Phloretin','The flavone apigenin is presum ably reduced to the flavanone naringenin in a first step which\nis followed by an isomerization to the corresponding chalcone structure. The chalcone is reduced to the dihydrochalcone phloretin, which is subsequently hydrolyzed to phloroglucinol and 3-(4-hydroxyphenyl)propionic acid (23). Although the degradation pathway of flavones by E. ramulus has successfully been elucidated, the enzymes involved have so far escaped purification and characterization.',15466559),(4166,'human','Phloretin','The flavone apigenin is presum ably reduced to the flavanone naringenin in a first step which\nis followed by an isomerization to the corresponding chalcone structure. The chalcone is reduced to the dihydrochalcone phloretin, which is subsequently hydrolyzed to phloroglucinol and 3-(4-hydroxyphenyl)propionic acid (23). Although the degradation pathway of flavones by E. ramulus has successfully been elucidated, the enzymes involved have so far escaped purification and characterization.',15466559),(4167,'human','Kaempferol-3-O-gal-rham-7-O-rham','Bacteroides distasonis hydrolyzed robinin to kaempferol, and Bacteroides uniformis and Bacteroides ovatus converted rutin to quercetin.',19580283),(4168,'human','Rutin','Bacteroides distasonis hydrolyzed robinin to kaempferol, and Bacteroides uniformis and Bacteroides ovatus converted rutin to quercetin.',19580283),(4169,'human','Rutin','Bacteroides distasonis hydrolyzed robinin to kaempferol, and Bacteroides uniformis and Bacteroides ovatus converted rutin to quercetin.',19580283),(4170,'human','Glycitein','Glycitein is demethoxylated in vitro by Eubacterium limosum to 6,7,40-trihydroxyisoflavone .',19580283),(4171,'human','Daidzein','Strains of Bacteroide ovatus spp., Streptococcus intermedius spp., and Ru minococcus productus SNU-Julong 732 (AY310748) as well as a stable mixed culture consisting of Enterococcus faecium EPI1, Lactobacillus mucosae EPI2, Finegoldia magna EPI3, and Veillo nella sp. EP have been isolated from human feces and reported to be able to convert daidzein into equol (16, 119-122) .',19580283),(4172,'human','Daidzein','Strains of Bacteroide ovatus spp., Streptococcus intermedius spp., and Ru minococcus productus SNU-Julong 732 (AY310748) as well as a stable mixed culture consisting of Enterococcus faecium EPI1, Lactobacillus mucosae EPI2, Finegoldia magna EPI3, and Veillo nella sp. EP have been isolated from human feces and reported to be able to convert daidzein into equol (16, 119-122) .',19580283),(4173,'human','Daidzein','Strains of Bacteroide ovatus spp., Streptococcus intermedius spp., and Ru minococcus productus SNU-Julong 732 (AY310748) as well as a stable mixed culture consisting of Enterococcus faecium EPI1, Lactobacillus mucosae EPI2, Finegoldia magna EPI3, and Veillo nella sp. EP have been isolated from human feces and reported to be able to convert daidzein into equol (16, 119-122) .',19580283),(4174,'human','Daidzein','Strains of Bacteroide ovatus spp., Streptococcus intermedius spp., and Ru minococcus productus SNU-Julong 732 (AY310748) as well as a stable mixed culture consisting of Enterococcus faecium EPI1, Lactobacillus mucosae EPI2, Finegoldia magna EPI3, and Veillo nella sp. EP have been isolated from human feces and reported to be able to convert daidzein into equol (16, 119-122) .',19580283),(4175,'human','Daidzein','Strains of Bacteroide ovatus spp., Streptococcus intermedius spp., and Ru minococcus productus SNU-Julong 732 (AY310748) as well as a stable mixed culture consisting of Enterococcus faecium EPI1, Lactobacillus mucosae EPI2, Finegoldia magna EPI3, and Veillo nella sp. EP have been isolated from human feces and reported to be able to convert daidzein into equol (16, 119-122) .',19580283),(4176,'human','Daidzein','Strains of Bacteroide ovatus spp., Streptococcus intermedius spp., and Ru minococcus productus SNU-Julong 732 (AY310748) as well as a stable mixed culture consisting of Enterococcus faecium EPI1, Lactobacillus mucosae EPI2, Finegoldia magna EPI3, and Veillo nella sp. EP have been isolated from human feces and reported to be able to convert daidzein into equol (16, 119-122) .',19580283),(4177,'human','Daidzein','Strains of Bacteroide ovatus spp., Streptococcus intermedius spp., and Ru minococcus productus SNU-Julong 732 (AY310748) as well as a stable mixed culture consisting of Enterococcus faecium EPI1, Lactobacillus mucosae EPI2, Finegoldia magna EPI3, and Veillo nella sp. EP have been isolated from human feces and reported to be able to convert daidzein into equol (16, 119-122) .',19580283),(4178,'human','Dihydrodaidzein','The bacterial strain named Julong 732 is also able to metabolize dihydrodaidzein to S-equol (123). Thus, other bacteria have to be involved in the previous steps of the conversion of daidzein to equol .',19580283),(4179,'human','Isoxanthohumol','Isoxanthohumol is a prenylated flavanone from hops and beer that has interesting biological activities. It can be metabo lized in vivo by colonic microbiota to render 8-prenylnaringenin (133, 134). This metabolic O-demethylation is carried out by Eubacterium species and particularly by Eubacterium limosum.',19580283),(4180,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4181,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4182,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4183,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4184,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4185,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4186,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4187,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4188,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4189,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4190,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4191,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4192,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4193,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4194,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4195,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4196,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4197,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4198,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4199,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4200,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4201,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4202,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4203,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4204,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4205,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4206,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4207,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4208,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4209,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4210,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4211,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4212,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4213,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4214,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4215,'mouse','Cholesterol','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4216,'mouse','Cholesterol','Prevotella and Paraprevotella presented negative correlations with the same nine  metabolites 93-phenylpropanoic acid, cholic acid, retinol, L-glutamate, L-phenylalanine, 7-dehydrocholesterol,  chenodeoxycholic acid, (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid and 11-deoxycorticosterone) and positive  correlations with four metabolites (tetradecanoic acid, L-isoleucine, benzoic acid, and indole-3-acetic acid). Our  results also revealed clear correlations among some metabolites.',31780788),(4217,'mouse','Cholesterol','Prevotella and Paraprevotella presented negative correlations with the same nine  metabolites 93-phenylpropanoic acid, cholic acid, retinol, L-glutamate, L-phenylalanine, 7-dehydrocholesterol,  chenodeoxycholic acid, (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid and 11-deoxycorticosterone) and positive  correlations with four metabolites (tetradecanoic acid, L-isoleucine, benzoic acid, and indole-3-acetic acid). Our  results also revealed clear correlations among some metabolites.',31780788),(4218,'mouse','Cholesterol','Prevotella and Paraprevotella presented negative correlations with the same nine  metabolites 93-phenylpropanoic acid, cholic acid, retinol, L-glutamate, L-phenylalanine, 7-dehydrocholesterol,  chenodeoxycholic acid, (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid and 11-deoxycorticosterone) and positive  correlations with four metabolites (tetradecanoic acid, L-isoleucine, benzoic acid, and indole-3-acetic acid). Our  results also revealed clear correlations among some metabolites.',31780788),(4219,'mouse','Cholesterol','Prevotella and Paraprevotella presented negative correlations with the same nine  metabolites 93-phenylpropanoic acid, cholic acid, retinol, L-glutamate, L-phenylalanine, 7-dehydrocholesterol,  chenodeoxycholic acid, (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid and 11-deoxycorticosterone) and positive  correlations with four metabolites (tetradecanoic acid, L-isoleucine, benzoic acid, and indole-3-acetic acid). Our  results also revealed clear correlations among some metabolites.',31780788),(4220,'mouse','Cholesterol','Prevotella and Paraprevotella presented negative correlations with the same nine  metabolites 93-phenylpropanoic acid, cholic acid, retinol, L-glutamate, L-phenylalanine, 7-dehydrocholesterol,  chenodeoxycholic acid, (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid and 11-deoxycorticosterone) and positive  correlations with four metabolites (tetradecanoic acid, L-isoleucine, benzoic acid, and indole-3-acetic acid). Our  results also revealed clear correlations among some metabolites.',31780788),(4221,'mouse','Cholesterol','Prevotella and Paraprevotella presented negative correlations with the same nine  metabolites 93-phenylpropanoic acid, cholic acid, retinol, L-glutamate, L-phenylalanine, 7-dehydrocholesterol,  chenodeoxycholic acid, (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid and 11-deoxycorticosterone) and positive  correlations with four metabolites (tetradecanoic acid, L-isoleucine, benzoic acid, and indole-3-acetic acid). Our  results also revealed clear correlations among some metabolites.',31780788),(4222,'mouse','Cholesterol','Prevotella and Paraprevotella presented negative correlations with the same nine  metabolites 93-phenylpropanoic acid, cholic acid, retinol, L-glutamate, L-phenylalanine, 7-dehydrocholesterol,  chenodeoxycholic acid, (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid and 11-deoxycorticosterone) and positive  correlations with four metabolites (tetradecanoic acid, L-isoleucine, benzoic acid, and indole-3-acetic acid). Our  results also revealed clear correlations among some metabolites.',31780788),(4223,'mouse','Cholesterol','Prevotella and Paraprevotella presented negative correlations with the same nine  metabolites 93-phenylpropanoic acid, cholic acid, retinol, L-glutamate, L-phenylalanine, 7-dehydrocholesterol,  chenodeoxycholic acid, (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid and 11-deoxycorticosterone) and positive  correlations with four metabolites (tetradecanoic acid, L-isoleucine, benzoic acid, and indole-3-acetic acid). Our  results also revealed clear correlations among some metabolites.',31780788),(4224,'mouse','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4225,'mouse','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4226,'mouse','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4227,'mouse','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4228,'mouse','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4229,'mouse','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4233,'mouse','Epigallocatechin','Biotransformation of both EGC and GC was observed with Adlercreutzia equolifaciens JCM 14793, Asaccharobacter celatus JCM 14811, and Slackia equolifaciens JCM  16059, but not Slackia isoflavoniconvertens JCM 16137. With respect to the degradation of EGC, strain JCM  14793 only catalyzed 4�?dehydroxylation to produce 4�?dehydroxylated EGC (7). Strain JCM 14811 mainly  produced 7, along with a slight formation of the C ring-cleaving product 1-(3,4,5-trihydroxyphenyl)-3-(2,4,6trihydroxyphenyl)propan-2-ol (1). Strain JCM 16059 catalyzed only C ring cleavage to form 1. Interestingly,  the presence of hydrogen promoted the bioconversion of EGC by these bacteria. In addition, strain JCM  14811 revealed the ability to catalyze 4�?dehydroxylation of 1 to yield 1-(3,5-dihydroxyphenyl)-3-(2,4,6-trihy droxyphenyl)propan-2-ol (2) in the presence of hydrogen. In the case of GC, strain JCM 14793 mainly pro duced C ring-cleaving product (1) with only a very small amount of 4�?dehydroxylated GC (8), while Strain  JCM 14811 only catalyzed 4�?dehydroxylation to form 8. Strain JCM 16059 formed 1. The bioconversion of  GC by the three strains was stimulated by hydrogen. Strain JCM 14793 showed the ability to convert 1 into  2 in the presence of hydrogen as did strain JCM 14811. Furthermore, strains JCM 14793 and JCM 14811  were found to have the ability to catalyze p-dehydroxylation of the pyrogallol moiety in the EGC metabolites  4-hydroxy-5-(3,4,5-trihydroxyphenyl)valeric acid (3) and 5-(3,4,5-trihydroxyphenyl)-γ-valerolactone (4), and  this ability was enhanced by the presence of hydrogen.',25747993),(4234,'mouse','Epigallocatechin','Biotransformation of both EGC and GC was observed with  Adlercreutzia equolifaciens JCM 14793, Asaccharobacter celatus JCM 14811, and Slackia equolifaciens JCM  16059, but not Slackia isoflavoniconvertens JCM 16137. With respect to the degradation of EGC, strain JCM  14793 only catalyzed 4�?dehydroxylation to produce 4�?dehydroxylated EGC (7). Strain JCM 14811 mainly  produced 7, along with a slight formation of the C ring-cleaving product 1-(3,4,5-trihydroxyphenyl)-3-(2,4,6trihydroxyphenyl)propan-2-ol (1). Strain JCM 16059 catalyzed only C ring cleavage to form 1. Interestingly,  the presence of hydrogen promoted the bioconversion of EGC by these bacteria. In addition, strain JCM  14811 revealed the ability to catalyze 4�?dehydroxylation of 1 to yield 1-(3,5-dihydroxyphenyl)-3-(2,4,6-trihy droxyphenyl)propan-2-ol (2) in the presence of hydrogen. In the case of GC, strain JCM 14793 mainly pro duced C ring-cleaving product (1) with only a very small amount of 4�?dehydroxylated GC (8), while Strain  JCM 14811 only catalyzed 4�?dehydroxylation to form 8. Strain JCM 16059 formed 1. The bioconversion of  GC by the three strains was stimulated by hydrogen. Strain JCM 14793 showed the ability to convert 1 into  2 in the presence of hydrogen as did strain JCM 14811. Furthermore, strains JCM 14793 and JCM 14811  were found to have the ability to catalyze p-dehydroxylation of the pyrogallol moiety in the EGC metabolites  4-hydroxy-5-(3,4,5-trihydroxyphenyl)valeric acid (3) and 5-(3,4,5-trihydroxyphenyl)-γ-valerolactone (4), and  this ability was enhanced by the presence of hydrogen.',25747993),(4235,'mouse','Epigallocatechin','Biotransformation of both EGC and GC was observed with  Adlercreutzia equolifaciens JCM 14793, Asaccharobacter celatus JCM 14811, and Slackia equolifaciens JCM  16059, but not Slackia isoflavoniconvertens JCM 16137. With respect to the degradation of EGC, strain JCM  14793 only catalyzed 4�?dehydroxylation to produce 4�?dehydroxylated EGC (7). Strain JCM 14811 mainly  produced 7, along with a slight formation of the C ring-cleaving product 1-(3,4,5-trihydroxyphenyl)-3-(2,4,6trihydroxyphenyl)propan-2-ol (1). Strain JCM 16059 catalyzed only C ring cleavage to form 1. Interestingly,  the presence of hydrogen promoted the bioconversion of EGC by these bacteria. In addition, strain JCM  14811 revealed the ability to catalyze 4�?dehydroxylation of 1 to yield 1-(3,5-dihydroxyphenyl)-3-(2,4,6-trihy droxyphenyl)propan-2-ol (2) in the presence of hydrogen. In the case of GC, strain JCM 14793 mainly pro duced C ring-cleaving product (1) with only a very small amount of 4�?dehydroxylated GC (8), while Strain  JCM 14811 only catalyzed 4�?dehydroxylation to form 8. Strain JCM 16059 formed 1. The bioconversion of  GC by the three strains was stimulated by hydrogen. Strain JCM 14793 showed the ability to convert 1 into  2 in the presence of hydrogen as did strain JCM 14811. Furthermore, strains JCM 14793 and JCM 14811  were found to have the ability to catalyze p-dehydroxylation of the pyrogallol moiety in the EGC metabolites  4-hydroxy-5-(3,4,5-trihydroxyphenyl)valeric acid (3) and 5-(3,4,5-trihydroxyphenyl)-γ-valerolactone (4), and  this ability was enhanced by the presence of hydrogen.',25747993),(4236,'mouse','4\'-dehydroxylated (�?-epigallocatechin','Biotransformation of both EGC and GC was observed with  Adlercreutzia equolifaciens JCM 14793, Asaccharobacter celatus JCM 14811, and Slackia equolifaciens JCM  16059, but not Slackia isoflavoniconvertens JCM 16137. With respect to the degradation of EGC, strain JCM  14793 only catalyzed 4�?dehydroxylation to produce 4�?dehydroxylated EGC (7). Strain JCM 14811 mainly  produced 7, along with a slight formation of the C ring-cleaving product 1-(3,4,5-trihydroxyphenyl)-3-(2,4,6trihydroxyphenyl)propan-2-ol (1). Strain JCM 16059 catalyzed only C ring cleavage to form 1. Interestingly,  the presence of hydrogen promoted the bioconversion of EGC by these bacteria. In addition, strain JCM  14811 revealed the ability to catalyze 4�?dehydroxylation of 1 to yield 1-(3,5-dihydroxyphenyl)-3-(2,4,6-trihy droxyphenyl)propan-2-ol (2) in the presence of hydrogen. In the case of GC, strain JCM 14793 mainly pro duced C ring-cleaving product (1) with only a very small amount of 4�?dehydroxylated GC (8), while Strain  JCM 14811 only catalyzed 4�?dehydroxylation to form 8. Strain JCM 16059 formed 1. The bioconversion of  GC by the three strains was stimulated by hydrogen. Strain JCM 14793 showed the ability to convert 1 into  2 in the presence of hydrogen as did strain JCM 14811. Furthermore, strains JCM 14793 and JCM 14811  were found to have the ability to catalyze p-dehydroxylation of the pyrogallol moiety in the EGC metabolites  4-hydroxy-5-(3,4,5-trihydroxyphenyl)valeric acid (3) and 5-(3,4,5-trihydroxyphenyl)-γ-valerolactone (4), and  this ability was enhanced by the presence of hydrogen.',25747993),(4237,'mouse','null','Biotransformation of both EGC and GC was observed with  Adlercreutzia equolifaciens JCM 14793, Asaccharobacter celatus JCM 14811, and Slackia equolifaciens JCM  16059, but not Slackia isoflavoniconvertens JCM 16137. With respect to the degradation of EGC, strain JCM  14793 only catalyzed 4�?dehydroxylation to produce 4�?dehydroxylated EGC (7). Strain JCM 14811 mainly  produced 7, along with a slight formation of the C ring-cleaving product 1-(3,4,5-trihydroxyphenyl)-3-(2,4,6trihydroxyphenyl)propan-2-ol (1). Strain JCM 16059 catalyzed only C ring cleavage to form 1. Interestingly,  the presence of hydrogen promoted the bioconversion of EGC by these bacteria. In addition, strain JCM  14811 revealed the ability to catalyze 4�?dehydroxylation of 1 to yield 1-(3,5-dihydroxyphenyl)-3-(2,4,6-trihy droxyphenyl)propan-2-ol (2) in the presence of hydrogen. In the case of GC, strain JCM 14793 mainly pro duced C ring-cleaving product (1) with only a very small amount of 4�?dehydroxylated GC (8), while Strain  JCM 14811 only catalyzed 4�?dehydroxylation to form 8. Strain JCM 16059 formed 1. The bioconversion of  GC by the three strains was stimulated by hydrogen. Strain JCM 14793 showed the ability to convert 1 into  2 in the presence of hydrogen as did strain JCM 14811. Furthermore, strains JCM 14793 and JCM 14811  were found to have the ability to catalyze p-dehydroxylation of the pyrogallol moiety in the EGC metabolites  4-hydroxy-5-(3,4,5-trihydroxyphenyl)valeric acid (3) and 5-(3,4,5-trihydroxyphenyl)-γ-valerolactone (4), and  this ability was enhanced by the presence of hydrogen.',25747993),(4238,'mouse','null','Biotransformation of both EGC and GC was observed with  Adlercreutzia equolifaciens JCM 14793, Asaccharobacter celatus JCM 14811, and Slackia equolifaciens JCM  16059, but not Slackia isoflavoniconvertens JCM 16137. With respect to the degradation of EGC, strain JCM  14793 only catalyzed 4�?dehydroxylation to produce 4�?dehydroxylated EGC (7). Strain JCM 14811 mainly  produced 7, along with a slight formation of the C ring-cleaving product 1-(3,4,5-trihydroxyphenyl)-3-(2,4,6trihydroxyphenyl)propan-2-ol (1). Strain JCM 16059 catalyzed only C ring cleavage to form 1. Interestingly,  the presence of hydrogen promoted the bioconversion of EGC by these bacteria. In addition, strain JCM  14811 revealed the ability to catalyze 4�?dehydroxylation of 1 to yield 1-(3,5-dihydroxyphenyl)-3-(2,4,6-trihy droxyphenyl)propan-2-ol (2) in the presence of hydrogen. In the case of GC, strain JCM 14793 mainly pro duced C ring-cleaving product (1) with only a very small amount of 4�?dehydroxylated GC (8), while Strain  JCM 14811 only catalyzed 4�?dehydroxylation to form 8. Strain JCM 16059 formed 1. The bioconversion of  GC by the three strains was stimulated by hydrogen. Strain JCM 14793 showed the ability to convert 1 into  2 in the presence of hydrogen as did strain JCM 14811. Furthermore, strains JCM 14793 and JCM 14811  were found to have the ability to catalyze p-dehydroxylation of the pyrogallol moiety in the EGC metabolites  4-hydroxy-5-(3,4,5-trihydroxyphenyl)valeric acid (3) and 5-(3,4,5-trihydroxyphenyl)-γ-valerolactone (4), and  this ability was enhanced by the presence of hydrogen.',25747993),(4239,'mouse','1-(3,4,5-trihydroxyphenyl)-3-(2,4,6trihydroxyphenyl)propan-2-ol','Biotransformation of both EGC and GC was observed with  Adlercreutzia equolifaciens JCM 14793, Asaccharobacter celatus JCM 14811, and Slackia equolifaciens JCM  16059, but not Slackia isoflavoniconvertens JCM 16137. With respect to the degradation of EGC, strain JCM  14793 only catalyzed 4�?dehydroxylation to produce 4�?dehydroxylated EGC (7). Strain JCM 14811 mainly  produced 7, along with a slight formation of the C ring-cleaving product 1-(3,4,5-trihydroxyphenyl)-3-(2,4,6trihydroxyphenyl)propan-2-ol (1). Strain JCM 16059 catalyzed only C ring cleavage to form 1. Interestingly,  the presence of hydrogen promoted the bioconversion of EGC by these bacteria. In addition, strain JCM  14811 revealed the ability to catalyze 4�?dehydroxylation of 1 to yield 1-(3,5-dihydroxyphenyl)-3-(2,4,6-trihy droxyphenyl)propan-2-ol (2) in the presence of hydrogen. In the case of GC, strain JCM 14793 mainly pro duced C ring-cleaving product (1) with only a very small amount of 4�?dehydroxylated GC (8), while Strain  JCM 14811 only catalyzed 4�?dehydroxylation to form 8. Strain JCM 16059 formed 1. The bioconversion of  GC by the three strains was stimulated by hydrogen. Strain JCM 14793 showed the ability to convert 1 into  2 in the presence of hydrogen as did strain JCM 14811. Furthermore, strains JCM 14793 and JCM 14811  were found to have the ability to catalyze p-dehydroxylation of the pyrogallol moiety in the EGC metabolites  4-hydroxy-5-(3,4,5-trihydroxyphenyl)valeric acid (3) and 5-(3,4,5-trihydroxyphenyl)-γ-valerolactone (4), and  this ability was enhanced by the presence of hydrogen.',25747993),(4240,'mouse','(-)-Gallocatechin','The�?bioconversion�?of�?GC�?was�?examined�?here.�? Strain MT4s-5 was�?observed to convert GC into 1 and 8  with�?slight production of 2. In the presence of hydrogen, strain MT4s-5�?accelerated�?the�?formation�?of�?2�?(data�?not�?shown).�?The�?results�?were�?very�?similar�?to�?those�?obtained�?in�?the�?bioconversion of EGC by strain  MT4s-5.',25747993),(4241,'mouse','(-)-Gallocatechin','The�?bioconversion�?of�?GC�?was�?examined�?here.�? Strain MT4s-5 was�?observed to convert GC into 1 and 8  with�?slight production of 2. In the presence of hydrogen, strain MT4s-5�?accelerated�?the�?formation�?of�?2�?(data�?not�?shown).�?The�?results�?were�?very�?similar�?to�?those�?obtained�?in�?the�?bioconversion of EGC by strain  MT4s-5.',25747993),(4242,'mouse','(-)-Gallocatechin','The�?bioconversion�?of�?GC�?was�?examined�?here.�? Strain MT4s-5 was�?observed to convert GC into 1 and 8  with�?slight production of 2. In the presence of hydrogen, strain MT4s-5�?accelerated�?the�?formation�?of�?2�?(data�?not�?shown).�?The�?results�?were�?very�?similar�?to�?those�?obtained�?in�?the�?bioconversion of EGC by strain  MT4s-5.',25747993),(4243,'mouse','null','Ratelimited enzymes in the process of TG synthesis of the DGAT1/2 mRNA level in the HFD group were signifificantly increased compared with the LFD group. The DGAT2 mRNA level in the HFD-CBM group decreased in the same range as the LFD group, whereas signifificant differences of the DGAT1 mRNA level were not found between the HFD group and the HFD-CBM group.',24166662),(4249,'mouse','null','TMAO inhibits SIRT1 expression, induces oxidative stress, and impairs NO production both in vivo and in vitro.',29325896),(4250,'mouse','null','By conducting a miRNA microarray screening, we revealed that expression of miRNA-10b in macrophages can be reduced by PCA. Functional analyses demonstrated that miRNA-10b directly represses ABCA1 and ABCG1 and negatively regulates cholesterol efflux from murine- and human-derived macrophages. Further in vitro and ex vivo analyses verified that PCA accelerates macrophage cholesterol efflux, correlating with the regulation of miRNA-10b-ABCA1/ABCG1 cascade, whereas Cy-3-G consumption promoted macrophage RCT and regressed atherosclerotic lesion in a gut microbiotaendependent manner.',22821931),(4251,'mouse','null','By conducting a miRNA microarray screening, we revealed that expression of miRNA-10b in macrophages can be reduced by PCA. Functional analyses demonstrated that miRNA-10b directly represses ABCA1 and ABCG1 and negatively regulates cholesterol efflux from murine- and human-derived macrophages. Further in vitro and ex vivo analyses verified that PCA accelerates macrophage cholesterol efflux, correlating with the regulation of miRNA-10b-ABCA1/ABCG1 cascade, whereas Cy-3-G consumption promoted macrophage RCT and regressed atherosclerotic lesion in a gut microbiotaendependent manner.',22821931),(4252,'mouse','null','By conducting a miRNA microarray screening, we revealed that expression of miRNA-10b in macrophages can be reduced by PCA. Functional analyses demonstrated that miRNA-10b directly represses ABCA1 and ABCG1 and negatively regulates cholesterol efflux from murine- and human-derived macrophages. Further in vitro and ex vivo analyses verified that PCA accelerates macrophage cholesterol efflux, correlating with the regulation of miRNA-10b-ABCA1/ABCG1 cascade, whereas Cy-3-G consumption promoted macrophage RCT and regressed atherosclerotic lesion in a gut microbiotaendependent manner.',22821931),(4253,'mouse','null','We extended our study to human samples to determine the effect of bile acids on CXCL16 mRNA expression of human liver sinusoidal endothelial cells. Similar to the mouse study, primary bile acids CDCA and TCA induced CXCL16 mRNA expression in SK-HEP1 cells (Fig. 5H). Next, the correlation between bile acids and CXCL16 expression was tested in nontumor liver tissues from HCC or cholangiocarcinoma patients of the Thailand Initiative for Genomics and Expression Research in Liver Cancer (TIGER-LC) cohort (34). Primary bile acid CDCA levels correlated with CXCL16 expression (Fig. 5I), whereas inverse correlation was seen for the secondary bile acid glycolithocholate (GLCA) (fig. S6I). The primary/secondary ratio was associated with CXCL16 increase (fig. S6J), indicating that the opposing effect of bile acids on CXCL16 expression also exists in humans.',29798856),(4254,'mouse','null','In summary,this study showed that it is butyrate, in stead of other substances produced by F.prausnitzii,inhibiting HDAC1 to activate Foxp3 and block the IL-6/STAT3/IL-17 pathway,thus regulating Th17/Treg cells and exerting an anti-inflammatory effect in IBD animal models.',29796620),(4255,'mouse','null','In summary,this study showed that it is butyrate, in stead of other substances produced by F.prausnitzii,inhibiting HDAC1 to activate Foxp3 and block the IL-6/STAT3/IL-17 pathway,thus regulating Th17/Treg cells and exerting an anti-inflammatory effect in IBD animal models.',29796620),(4256,'mouse','null','In summary,this study showed that it is butyrate, in stead of other substances produced by F.prausnitzii,inhibiting HDAC1 to activate Foxp3 and block the IL-6/STAT3/IL-17 pathway,thus regulating Th17/Treg cells and exerting an anti-inflammatory effect in IBD animal models.',29796620),(4257,'mouse','null','In summary,this study showed that it is butyrate, in stead of other substances produced by F.prausnitzii,inhibiting HDAC1 to activate Foxp3 and block the IL-6/STAT3/IL-17 pathway,thus regulating Th17/Treg cells and exerting an anti-inflammatory effect in IBD animal models.',29796620),(4258,'mouse','null','In summary,this study showed that it is butyrate, in stead of other substances produced by F.prausnitzii,inhibiting HDAC1 to activate Foxp3 and block the IL-6/STAT3/IL-17 pathway,thus regulating Th17/Treg cells and exerting an anti-inflammatory effect in IBD animal models.',29796620),(4259,'mouse','4-Hydroxyphenylacetic acid','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4260,'mouse','4-Hydroxyphenylacetic acid','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4261,'mouse','4-Hydroxyphenylacetic acid','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4262,'mouse','4-Hydroxyphenylacetic acid','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4263,'mouse','4-Hydroxyphenylacetic acid','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4264,'mouse','4-Hydroxyphenylacetic acid','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4265,'mouse','4-Hydroxyphenylacetic acid','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4266,'mouse','4-Hydroxyphenylacetic acid','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4267,'mouse','4-Hydroxyphenylacetic acid','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4268,'mouse','D-Glucose','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4269,'mouse','D-Glucose','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4270,'mouse','D-Glucose','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4271,'mouse','D-Glucose','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4272,'mouse','D-Glucose','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4273,'mouse','D-Glucose','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4274,'mouse','D-Glucose','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4275,'mouse','D-Glucose','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4276,'mouse','D-Glucose','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4277,'mouse','D-Glucose','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4278,'mouse','D-Glucose','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4279,'mouse','D-Glucose','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4280,'mouse','Indole-3-acetic acid','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4281,'mouse','Indole-3-acetic acid','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4282,'mouse','Indole-3-acetic acid','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4283,'mouse','Indole-3-acetic acid','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4284,'mouse','Indole-3-acetic acid','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4285,'mouse','Indole-3-acetic acid','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4286,'mouse','Indole-3-acetic acid','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4287,'mouse','Indole-3-acetic acid','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4288,'mouse','Indole-3-acetic acid','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4289,'mouse','Phenylacetic acid','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4290,'mouse','Phenylacetic acid','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4291,'mouse','Phenylacetic acid','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4292,'mouse','Phenylacetic acid','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4293,'mouse','Phenylacetic acid','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4294,'mouse','Phenylacetic acid','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4295,'mouse','Phenylacetic acid','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4296,'mouse','Phenylacetic acid','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4297,'mouse','Phenylacetic acid','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4298,'mouse','Phenylacetic acid','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4299,'mouse','Phenylacetic acid','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4300,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4301,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4302,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4303,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4304,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4305,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4306,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4307,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4308,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4309,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4310,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4311,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4312,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4313,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4314,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4315,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4316,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4317,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4318,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4319,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4320,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4321,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4322,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4323,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4324,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4325,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4326,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4327,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4328,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4329,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4330,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4331,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4332,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4333,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4334,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4335,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4336,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4337,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4338,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4339,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4340,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4341,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4342,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4343,'mouse','Tryptophan','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4344,'mouse','phenyalanine','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4345,'mouse','phenyalanine','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4346,'mouse','phenyalanine','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4347,'mouse','phenyalanine','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4348,'mouse','phenyalanine','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4349,'mouse','phenyalanine','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4350,'mouse','null','The main metabolites produced by B. thetaiotaomicron in the GIT were acetate (up to 6 mM) and propionate (up to 2 mM) (Figure 1D). We evaluated the effects of these bacterial metabolites on the amount of KLF4 protein produced by the mucus producing cell line HT29-MTX [31]. The amount of KLF4 protein was higher in the presence than in the absence of acetate (10 and 20 mM) (Figure 3A)whereas propionate (5 and 10 mM) and butyrate (1 and 5 mM) had no effect (Figure 3B, C).',23692866),(4351,'mouse','null','The main metabolites produced by B. thetaiotaomicron in the GIT were acetate (up to 6 mM) and propionate (up to 2 mM) (Figure 1D). We evaluated the effects of these bacterial metabolites on the amount of KLF4 protein produced by the mucus producing cell line HT29-MTX [31]. The amount of KLF4 protein was higher in the presence than in the absence of acetate (10 and 20 mM) (Figure 3A)whereas propionate (5 and 10 mM) and butyrate (1 and 5 mM) had no effect (Figure 3B, C).',23692866),(4357,'mouse','Acetate','We first checked  in vitro  that  B. thetaiotaomicron produced acetate and propionate and that  F. prausnitzii consumed acetate and produced butyrate (Figure 4A). When both bacteria were grown together, butyrate was produced rather than acetate. The butyrate production by F. prausnitzii as higher in the presence than absence of B. thetaiotaomicron  (10.9 ± 0.3 mM versus 7.8 ± 0.5 mM) suggesting that  F. prausnitzii  synthesized butyrate from the acetate produced by  B. thetaiotaomicron .',23692866),(4361,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4362,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4363,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4364,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4365,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4366,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4367,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4368,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4369,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4370,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4371,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4372,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4373,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4374,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4375,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4376,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4377,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4378,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4379,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4380,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4381,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4382,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4383,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4384,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4385,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4386,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4387,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4388,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4389,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4390,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4391,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4392,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4393,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4394,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4395,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4396,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4397,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4398,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4399,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4400,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4401,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4402,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4403,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4404,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4405,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4406,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4407,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4408,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4409,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4410,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4411,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4412,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4413,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4414,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4415,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4416,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4417,'mouse','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4418,'mouse','null','IVAS stimulated a dose-dependent increase in glucose transport through glucose transporter GLUT4- and PI3K-dependent mechanisms. IVAS also up regulated GLUT1, GLUT4, and PI3K p85a protein, and increased phosphorylation of Akt.',30183376),(4419,'mouse','null','IVAS stimulated a dose-dependent increase in glucose transport through glucose transporter GLUT4- and PI3K-dependent mechanisms. IVAS also up regulated GLUT1, GLUT4, and PI3K p85a protein, and increased phosphorylation of Akt.',30183376),(4420,'mouse','null','IVAS stimulated a dose-dependent increase in glucose transport through glucose transporter GLUT4- and PI3K-dependent mechanisms. IVAS also up regulated GLUT1, GLUT4, and PI3K p85a protein, and increased phosphorylation of Akt.',30183376),(4425,'mouse','D-Glucose','R. obeum produces acetate, R. torques produces lactate and acetate and R. gnavus produces acetate and formate on PYG medium.',24553467),(4426,'mouse','D-Glucose','R. obeum produces acetate, R. torques produces lactate and acetate and R. gnavus produces acetate and formate on PYG medium.',24553467),(4427,'mouse','D-Glucose','R. obeum produces acetate, R. torques produces lactate and acetate and R. gnavus produces acetate and formate on PYG medium.',24553467),(4428,'mouse','D-Glucose','R. obeum produces acetate, R. torques produces lactate and acetate and R. gnavus produces acetate and formate on PYG medium.',24553467),(4429,'mouse','D-Glucose','R. obeum produces acetate, R. torques produces lactate and acetate and R. gnavus produces acetate and formate on PYG medium.',24553467),(4430,'mouse','D-Glucose','R. obeum produces acetate, R. torques produces lactate and acetate and R. gnavus produces acetate and formate on PYG medium.',24553467),(4431,'mouse','D-Glucose','Propionate formation from fucose via propanediol has also been described in the human gut anaerobe Roseburia inulinivorans, which produces butyrate rather than propionate when grown on glucose.',24553467),(4432,'mouse','L-Fucose','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4433,'mouse','L-Fucose','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4434,'mouse','L-Fucose','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4435,'mouse','Lactic acid','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4436,'mouse','Lactic acid','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4437,'mouse','Lactic acid','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4438,'mouse','Lactic acid','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4439,'mouse','Lactic acid','We tested different Coprococcus strains for their ability to utilise lactate for growth. C. eutactus ATCC 27759 and C. comes ATCC 27758 could not grow on lactate, but C. catus GD/7 was able to grow with 25 mM lactate on YCFA medium containing 30 mM acetate. Propionate became the main product and substantial lactate utilisation also occurred in the presence of 10 mM fructose.',24553467),(4440,'mouse','Lactic acid','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4441,'mouse','Lactic acid','We tested different Coprococcus strains for their ability to utilise lactate for growth. C. eutactus ATCC 27759 and C. comes ATCC 27758 could not grow on lactate, but C. catus GD/7 was able to grow with 25 mM lactate on YCFA medium containing 30 mM acetate. Propionate became the main product and substantial lactate utilisation also occurred in the presence of 10 mM fructose.',24553467),(4442,'mouse','Lactic acid','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4443,'mouse','Lactic acid','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4444,'mouse','Lactic acid','We tested different Coprococcus strains for their ability to utilise lactate for growth. C. eutactus ATCC 27759 and C. comes ATCC 27758 could not grow on lactate, but C. catus GD/7 was able to grow with 25 mM lactate on YCFA medium containing 30 mM acetate. Propionate became the main product and substantial lactate utilisation also occurred in the presence of 10 mM fructose.',24553467),(4445,'mouse','Lactic acid','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4446,'mouse','Formate','We tested different Coprococcus strains for their ability to utilise lactate for growth. C. eutactus ATCC 27759 and C. comes ATCC 27758 could not grow on lactate, but C. catus GD/7 was able to grow with 25 mM lactate on YCFA medium containing 30 mM acetate. Propionate became the main product and substantial lactate utilisation also occurred in the presence of 10 mM fructose.',24553467),(4447,'mouse','Formate','We tested different Coprococcus strains for their ability to utilise lactate for growth. C. eutactus ATCC 27759 and C. comes ATCC 27758 could not grow on lactate, but C. catus GD/7 was able to grow with 25 mM lactate on YCFA medium containing 30 mM acetate. Propionate became the main product and substantial lactate utilisation also occurred in the presence of 10 mM fructose.',24553467),(4448,'mouse','Acrylate','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4449,'mouse','Acrylate','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4450,'mouse','Acrylate','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4451,'mouse','Butyrate','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4452,'mouse','Butyrate','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4453,'mouse','Propanediol','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4454,'mouse','Propanediol','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4455,'mouse','Succinate','The succinate pathway of propionate formation has mainly been studied in bacteria belonging to the class Negativicutes.',24553467),(4456,'mouse','Succinate','The succinate pathway of propionate formation has mainly been studied in bacteria belonging to the class Negativicutes.',24553467),(4457,'mouse','Succinate','The succinate pathway of propionate formation has mainly been studied in bacteria belonging to the class Negativicutes.',24553467),(4458,'mouse','Succinate','The succinate pathway of propionate formation has mainly been studied in bacteria belonging to the class Negativicutes.',24553467),(4459,'mouse','Succinate','The succinate pathway of propionate formation has mainly been studied in bacteria belonging to the class Negativicutes.',24553467),(4460,'mouse','Succinate','The succinate pathway of propionate formation has mainly been studied in bacteria belonging to the class Negativicutes.',24553467),(4461,'mouse','Succinate','The succinate pathway of propionate formation has mainly been studied in bacteria belonging to the class Negativicutes.',24553467),(4494,'mouse','null','succinate augmented the expression of TNF and IL-1b by iMoDCs when used in combination with the TLR3 ligand poly(I:C) or the TLR7 agonist imiquimod.',18820681),(4495,'mouse','null','succinate augmented the expression of TNF and IL-1b by iMoDCs when used in combination with the TLR3 ligand poly(I:C) or the TLR7 agonist imiquimod.',18820681),(4496,'mouse','null','Our results suggest this is due to GF colonocytes adapting to their suboptimal energetic state by undergoing autophagy. Reduced ATP levels activate AMPK, which, in turn, activates p27 to prevent apoptosis and induce autophagy in these cells.',21531334),(4502,'mouse','null','Our results suggest this is due to GF colonocytes adapting to their suboptimal energetic state by undergoing autophagy. Reduced ATP levels activate AMPK, which, in turn, activates p27 to prevent apoptosis and induce autophagy in these cells.',21531334),(4503,'mouse','null','Our results suggest this is due to GF colonocytes adapting to their suboptimal energetic state by undergoing autophagy. Reduced ATP levels activate AMPK, which, in turn, activates p27 to prevent apoptosis and induce autophagy in these cells.',21531334),(4504,'mouse','null','Our results suggest this is due to GF colonocytes adapting to their suboptimal energetic state by undergoing autophagy. Reduced ATP levels activate AMPK, which, in turn, activates p27 to prevent apoptosis and induce autophagy in these cells.',21531334),(4505,'mouse','null','Our results suggest this is due to GF colonocytes adapting to their suboptimal energetic state by undergoing autophagy. Reduced ATP levels activate AMPK, which, in turn, activates p27 to prevent apoptosis and induce autophagy in these cells.',21531334),(4506,'mouse','null','Our results suggest this is due to GF colonocytes adapting to their suboptimal energetic state by undergoing autophagy. Reduced ATP levels activate AMPK, which, in turn, activates p27 to prevent apoptosis and induce autophagy in these cells.',21531334),(4507,'mouse','null','We also tested whether butyrate inhibits autophagy in GF colonocytes. As expected, it decreases p27 phosphorylation and LC3-II compared to untreated GF colonocytes.',21531334),(4508,'mouse','null','We also tested whether butyrate inhibits autophagy in GF colonocytes. As expected, it decreases p27 phosphorylation and LC3-II compared to untreated GF colonocytes.',21531334),(4512,'mouse','null','It has been reported that B. thetaiotaomicron and B. ovatus have the ability to ferment AAA  to produce phenylacetic acid.',28628112),(4513,'mouse','null','It has been reported that B. thetaiotaomicron and B. ovatus have the ability to ferment AAA  to produce phenylacetic acid.',28628112),(4523,'mouse','Tyrosine','Here we show using untargeted metabolomics that levels of phenyl sulfate, a gut microbiota-derived metabolite, increase with the progression of diabetes in rats overexpressing human uremic toxin transporter SLCO4C1 in the kidney, and are decreased in rats with limited proteinuria.',31015435),(4524,'mouse','Tyrosine','although there was no difference between in the preserved tubular area and fibrotic area at the light microscopic level, the expression levels of representative inflammatory genes, TNF-α (Tnfa) and MCP-1 (Ccl2), the fibrotic gene TGF-α1 (Tgfb1), fibronectin (Fn1), and collagen I (Col1a1) were increased in PS-treated db/db kidney. It was reported that db/db mice develop renal fibrosis.',31015435),(4525,'mouse','Tyrosine','although there was no difference between in the preserved tubular area and fibrotic area at the light microscopic level, the expression levels of representative inflammatory genes, TNF-α (Tnfa) and MCP-1 (Ccl2), the fibrotic gene TGF-α1 (Tgfb1), fibronectin (Fn1), and collagen I (Col1a1) were increased in PS-treated db/db kidney. It was reported that db/db mice develop renal fibrosis.',31015435),(4526,'mouse','Tyrosine','although there was no difference between in the preserved tubular area and fibrotic area at the light microscopic level, the expression levels of representative inflammatory genes, TNF-α (Tnfa) and MCP-1 (Ccl2), the fibrotic gene TGF-α1 (Tgfb1), fibronectin (Fn1), and collagen I (Col1a1) were increased in PS-treated db/db kidney. It was reported that db/db mice develop renal fibrosis.',31015435),(4527,'mouse','Tyrosine','although there was no difference between in the preserved tubular area and fibrotic area at the light microscopic level, the expression levels of representative inflammatory genes, TNF-α (Tnfa) and MCP-1 (Ccl2), the fibrotic gene TGF-α1 (Tgfb1), fibronectin (Fn1), and collagen I (Col1a1) were increased in PS-treated db/db kidney. It was reported that db/db mice develop renal fibrosis.',31015435),(4528,'mouse','Tyrosine','although there was no difference between in the preserved tubular area and fibrotic area at the light microscopic level, the expression levels of representative inflammatory genes, TNF-α (Tnfa) and MCP-1 (Ccl2), the fibrotic gene TGF-α1 (Tgfb1), fibronectin (Fn1), and collagen I (Col1a1) were increased in PS-treated db/db kidney. It was reported that db/db mice develop renal fibrosis.',31015435),(4530,'mouse','Daidzein','Equol is a metabolite of daidzein that is produced by intestinal microbiota.The rod-shaped and Gram-positive anaerobic equol-producing intestinal bacterium Slackia TM-30 was isolated from healthy human faeces.',26455424),(4534,'mouse','null','Live B. dentium-monoassociated mice exhibited increased colonic expression of goblet cell markers Krüppel-like factor 4 (Klf4), Trefoil factor 3 (Tff3), Relm-β, Muc2, and several glycosyltransferases compared to both heat-killed B. dentium and germfree counterparts.',31213556),(4540,'mouse','null','We examined a number of known autophagy markers by qPCR and found that B. dentium colonization was associated with increased expression of ATG16L, ATG5, Beclin1, LAMP2, and LC3.',31213556),(4541,'mouse','null','We examined a number of known autophagy markers by qPCR and found that B. dentium colonization was associated with increased expression of ATG16L, ATG5, Beclin1, LAMP2, and LC3.',31213556),(4542,'mouse','null','We examined a number of known autophagy markers by qPCR and found that B. dentium colonization was associated with increased expression of ATG16L, ATG5, Beclin1, LAMP2, and LC3.',31213556),(4543,'mouse','null','We examined a number of known autophagy markers by qPCR and found that B. dentium colonization was associated with increased expression of ATG16L, ATG5, Beclin1, LAMP2, and LC3.',31213556),(4544,'mouse','null','We examined a number of known autophagy markers by qPCR and found that B. dentium colonization was associated with increased expression of ATG16L, ATG5, Beclin1, LAMP2, and LC3.',31213556),(4545,'mouse','null','B. dentium can secrete products, such as acetate, that are capable of increasing MUC2 synthesis.',31213556),(4549,'mouse','null','A specific human-associated gut microbe, Clostridium orbiscindens,produced DAT and rescued antibiotic-treated influenza-infected mice.Consistent with our hypothesis that DAT protects from influenza infection by enhancing type I IFN signaling before infection.',28774928),(4550,'mouse','null','Oxidative stress and inflammation in the kidney have been demonstrated to play a pivotal role in the initiation and progression of obesity-related renal injury [20e22]. To further determine the possible molecular mechanisms by which elevated TMAO levels induce renal interstitial fibrosis and dysfunction, we measured expression of NOX-4 (a constitutive active enzyme responsible for the production of reactive oxygen species) and proinflammatory cytokines in the kidney. As presented in Fig. 4, expression of NOX-4(Fig. 4A and B) and pro-inflammatory cytokines TNF-a (Fig. 4A and C) and IL-1 b (Fig. 4A and D) was significantly increased in mice fed a HFD, compared with mice fed a LFD, but was reduced by DMB treatment. Expression of NOX-4, TNF-a and IL-1 b in mice fed a LFD was not affected by DMB treatment.',28942145),(4551,'mouse','null','Oxidative stress and inflammation in the kidney have been demonstrated to play a pivotal role in the initiation and progression of obesity-related renal injury [20e22]. To further determine the possible molecular mechanisms by which elevated TMAO levels induce renal interstitial fibrosis and dysfunction, we measured expression of NOX-4 (a constitutive active enzyme responsible for the production of reactive oxygen species) and proinflammatory cytokines in the kidney. As presented in Fig. 4, expression of NOX-4(Fig. 4A and B) and pro-inflammatory cytokines TNF-a (Fig. 4A and C) and IL-1 b (Fig. 4A and D) was significantly increased in mice fed a HFD, compared with mice fed a LFD, but was reduced by DMB treatment. Expression of NOX-4, TNF-a and IL-1 b in mice fed a LFD was not affected by DMB treatment.',28942145),(4552,'mouse','null','Oxidative stress and inflammation in the kidney have been demonstrated to play a pivotal role in the initiation and progression of obesity-related renal injury [20e22]. To further determine the possible molecular mechanisms by which elevated TMAO levels induce renal interstitial fibrosis and dysfunction, we measured expression of NOX-4 (a constitutive active enzyme responsible for the production of reactive oxygen species) and proinflammatory cytokines in the kidney. As presented in Fig. 4, expression of NOX-4(Fig. 4A and B) and pro-inflammatory cytokines TNF-a (Fig. 4A and C) and IL-1 b (Fig. 4A and D) was significantly increased in mice fed a HFD, compared with mice fed a LFD, but was reduced by DMB treatment. Expression of NOX-4, TNF-a and IL-1 b in mice fed a LFD was not affected by DMB treatment.',28942145),(4553,'mouse','null','Butyrate or niacin induces IL-18 expression in colonic epithelium in a Gpr109a-dependent manner.',24412617),(4554,'mouse','null','Butyrate or niacin induces IL-18 expression in colonic epithelium in a Gpr109a-dependent manner.',24412617),(4555,'mouse','null','Our data indicate that WAT miR-181 expression is controlled by the relative abundance of microbiota-derived metabolites acting as positive and negative regulators of miR-181 expression and that  indole and indole-derived metabolites  including indoxyl sulfate  are key negative regulators of miR-181 expression in WAT.',31189717),(4556,'mouse','null','Our data indicate that WAT miR-181 expression is controlled by the relative abundance of microbiota-derived metabolites acting as positive and negative regulators of miR-181 expression and that  indole and indole-derived metabolites  including indoxyl sulfate  are key negative regulators of miR-181 expression in WAT.',31189717),(4560,'mouse','null','D8 up-regulates IL-22 expression ex vivo and in vivo.',29459771),(4561,'mouse','null','Thus, exposure to S. cerevisiae in the gut led to enhanced purine metabolism and promoted increased uric acid.',28275154),(4562,'mouse','null','Thus, exposure to S. cerevisiae in the gut led to enhanced purine metabolism and promoted increased uric acid.',28275154),(4563,'mouse','null','Butyrate promotes the increased expression of IFN-γ and granzyme B in CTLs and Tc17 cells.',30258117),(4564,'mouse','null','Butyrate promotes the increased expression of IFN-γ and granzyme B in CTLs and Tc17 cells.',30258117),(4568,'mouse','Sennoside','We demonstrated that intestinal peristalsis was promoted by rheinanthrone produced by hydrolysis of sennoside by strain LKM512 and LKM10070.',22384059),(4569,'mouse','Sennoside','We demonstrated that intestinal peristalsis was promoted by rheinanthrone produced by hydrolysis of sennoside by strain LKM512 and LKM10070.',22384059),(4575,'mouse','null','Fecal butyrate levels significantly decreased in the DSS mice as compared to the control mice, but increased significantly in the DSS plus TN-3 mice as compared to the DSS mice.',27438072),(4576,'mouse','null','Indole, the most prevalent metabolite of tryptophan, is produced by a wide range of bacterial species, including those belonging to the genera Escherichia, Bacteroides, and ClostridiumProlonged exposure to indole inhibits ATP production and thus GLP-1 secretion.',25456122),(4577,'mouse','null','Indole, the most prevalent metabolite of tryptophan, is produced by a wide range of bacterial species, including those belonging to the genera Escherichia, Bacteroides, and ClostridiumProlonged exposure to indole inhibits ATP production and thus GLP-1 secretion.',25456122),(4578,'mouse','null','Indole, the most prevalent metabolite of tryptophan, is produced by a wide range of bacterial species, including those belonging to the genera Escherichia, Bacteroides, and ClostridiumProlonged exposure to indole inhibits ATP production and thus GLP-1 secretion.',25456122),(4579,'mouse','null','TL133 also increasing the caecal concentration of acetate, propionate and butyrate.',17349084),(4580,'mouse','null','TL134 also increasing the caecal concentration of acetate, propionate and butyrate.',17349084),(4581,'mouse','null','TL135 also increasing the caecal concentration of acetate, propionate and butyrate.',17349084),(4582,'mouse','L-Fucose','Accordingly, the mouse model provided L-fucose, enabling B. thetaiotaomicron to synthesize L-fucose containing polysaccharides.',23022739),(4583,'mouse','L-Fucose','Accordingly, the mouse model provided L-fucose, enabling B. thetaiotaomicron to synthesize L-fucose containing polysaccharides.',23022739),(4584,'mouse','L-Fucose','Accordingly, the mouse model provided L-fucose, enabling B. thetaiotaomicron to synthesize L-fucose containing polysaccharides.',23022739),(4585,'mouse','null','We were able to identify a few metabolites specific to the humanized Salmonella-infected group, including two metabolites in the glutathione pathway. In particular, we identified glutathione disulfide and glutathione cysteine disulfide in higher abundance in humanized Salmonella-infected mice. Salmonella infection triggers vast amounts of oxidative stress, and glutathione metabolism is important for protection against oxidative stress.',30401779),(4586,'mouse','null','We were able to identify a few metabolites specific to the humanized Salmonella-infected group, including two metabolites in the glutathione pathway. In particular, we identified glutathione disulfide and glutathione cysteine disulfide in higher abundance in humanized Salmonella-infected mice. Salmonella infection triggers vast amounts of oxidative stress, and glutathione metabolism is important for protection against oxidative stress.',30401779),(4587,'mouse','null','SCFAs binding to GPR43 on colonic epithelial cells stimulates K þ efflux and hyperpolarization,which lead to NLRP3 inflammasome activation.',25828455),(4588,'mouse','null','NaB corrected the HFD-induced gut microbiota imbalance in mice, while it considerably elevated the abundances of the beneficial bacteria Christensenellaceae, Blautia and Lactobacillus . These bacteria can produce butyric acid in what seems like a virtuous circle. And butyrate restored HFD induced intestinal mucosa damage, increased the expression of ZO-1 in small intestine, further decreased the levels of gut endotoxin in serum and liver compared with HF group.',28104981),(4589,'mouse','null','NaB corrected the HFD-induced gut microbiota imbalance in mice, while it considerably elevated the abundances of the beneficial bacteria Christensenellaceae, Blautia and Lactobacillus . These bacteria can produce butyric acid in what seems like a virtuous circle. And butyrate restored HFD induced intestinal mucosa damage, increased the expression of ZO-1 in small intestine, further decreased the levels of gut endotoxin in serum and liver compared with HF group.',28104981),(4590,'mouse','null','NaB corrected the HFD-induced gut microbiota imbalance in mice, while it considerably elevated the abundances of the beneficial bacteria Christensenellaceae, Blautia and Lactobacillus . These bacteria can produce butyric acid in what seems like a virtuous circle. And butyrate restored HFD induced intestinal mucosa damage, increased the expression of ZO-1 in small intestine, further decreased the levels of gut endotoxin in serum and liver compared with HF group.',28104981),(4591,'mouse','null','Taken together, these experiments suggest that RYGB restores intestinal NLRP6 inflammasome expression by increasing taurine and decreasing histamine in the gut.',31602628),(4592,'mouse','null','Taken together, these experiments suggest that RYGB restores intestinal NLRP6 inflammasome expression by increasing taurine and decreasing histamine in the gut.',31602628),(4593,'mouse','Lipopolysaccharide','Equol inhibited the lipopolysaccharide (LPS)-induced TLR4 activation, MAPK activation,NF-kB-mediated transcription of inflammatory mediators, production of nitric oxide (NO), release of prostaglandin E2 (PGE-2), secretion of tumor necrosis factor-α (TNF-α) and interleukin 6 (IL-6),in Lipopolysaccharide (LPS)-activated murine microglia cells.',28264445),(4594,'mouse','Lipopolysaccharide','Equol inhibited the lipopolysaccharide (LPS)-induced TLR4 activation, MAPK activation,NF-kB-mediated transcription of inflammatory mediators, production of nitric oxide (NO), release of prostaglandin E2 (PGE-2), secretion of tumor necrosis factor-α (TNF-α) and interleukin 6 (IL-6),in Lipopolysaccharide (LPS)-activated murine microglia cells.',28264445),(4595,'mouse','Lipopolysaccharide','Equol inhibited the lipopolysaccharide (LPS)-induced TLR4 activation, MAPK activation,NF-kB-mediated transcription of inflammatory mediators, production of nitric oxide (NO), release of prostaglandin E2 (PGE-2), secretion of tumor necrosis factor-α (TNF-α) and interleukin 6 (IL-6),in Lipopolysaccharide (LPS)-activated murine microglia cells.',28264445),(4596,'mouse','Lipopolysaccharide','Equol inhibited the lipopolysaccharide (LPS)-induced TLR4 activation, MAPK activation,NF-kB-mediated transcription of inflammatory mediators, production of nitric oxide (NO), release of prostaglandin E2 (PGE-2), secretion of tumor necrosis factor-α (TNF-α) and interleukin 6 (IL-6),in Lipopolysaccharide (LPS)-activated murine microglia cells.',28264445),(4597,'mouse','null','The helminth Nippostrongylus brasiliensis and a tritrichomonad protist both secreted succinate as a metabolite. In vivo sensing of the tritrichomonad required SUCNR1, whereas N. brasiliensis was SUCNR1 independent. These findings define a paradigm wherein tuft cells monitor microbial metabolites to initiate type 2 immunity and suggest the existence of other sensing pathways triggering the response to helminths.',30021144),(4598,'mouse','null','The helminth Nippostrongylus brasiliensis and a tritrichomonad protist both secreted succinate as a metabolite. In vivo sensing of the tritrichomonad required SUCNR1, whereas N. brasiliensis was SUCNR1 independent. These findings define a paradigm wherein tuft cells monitor microbial metabolites to initiate type 2 immunity and suggest the existence of other sensing pathways triggering the response to helminths.',30021144),(4599,'mouse','omega-6 Fatty Acids','Here, we demonstrated that gut microbiota exerted protective effects against HFD-induced obesity and improved glucose homeostasis associated with several mechanisms involving gut microbial metabolites of dietary PUFAs: (1) gut microbiota converted LA to HYA, reducing the adipose inflammation otherwise induced by excessive dietary LA via the AA cascade; (2) HYA activated GPR40 and GPR120 and promoted GLP-1 secretion; and (3) HYA suppressed lipid absorption by promoting intestinal peristalsis via EP3.',31488836),(4600,'mouse','null','Our study shows that MAM from F. prausnitzii can restore the intestinal barrier structure and function in DM conditions via the regulation of the tight junction pathway and ZO-1 expression.Collectively, our data reveal that MAM protein upregulates ZO-1 expression in intestinal epithelial cells.',31503404),(4601,'mouse','Melamine','We further demonstrated that melamine is converted to cyanuric acid in vitro by bacteria cultured from normal rat feces;Klebsiella was subsequently identified in fecal samples by 16S ribosomal DNA sequencing. In culture, Klebsiella terrigena was shown to convert melamine to cyanuric acid directly.',23408055),(4602,'mouse','null','We demonstrate that commensal Bacteroides species limit S. Typhimurium infection in the distalgut. We find that Bacteroides spp. production of the short-chain fatty acid (SCFA) metabolic byproduct propionate directly inhibits S. Typhimurium growth in vitro by disrupting intracellular pH homeostasis and limits distal intestinal expansion and fecal shedding in vivo.',30057174),(4603,'mouse','null','We screened a collection of 79 sequenced human intestinal isolates encompassing the major phyla found in the human gut and identified nine strains capable of producing TMA from choline in vitro.',25784704),(4604,'mouse','null','We screened a collection of 79 sequenced human intestinal isolates encompassing the major phyla found in the human gut and identified nine strains capable of producing TMA from choline in vitro.',25784704),(4605,'mouse','null','We screened a collection of 79 sequenced human intestinal isolates encompassing the major phyla found in the human gut and identified nine strains capable of producing TMA from choline in vitro.',25784704),(4606,'mouse','null','We screened a collection of 79 sequenced human intestinal isolates encompassing the major phyla found in the human gut and identified nine strains capable of producing TMA from choline in vitro.',25784704),(4607,'mouse','null','We screened a collection of 79 sequenced human intestinal isolates encompassing the major phyla found in the human gut and identified nine strains capable of producing TMA from choline in vitro.',25784704),(4608,'mouse','null','We screened a collection of 79 sequenced human intestinal isolates encompassing the major phyla found in the human gut and identified nine strains capable of producing TMA from choline in vitro.',25784704),(4609,'mouse','null','We screened a collection of 79 sequenced human intestinal isolates encompassing the major phyla found in the human gut and identified nine strains capable of producing TMA from choline in vitro.',25784704),(4610,'mouse','null','We screened a collection of 79 sequenced human intestinal isolates encompassing the major phyla found in the human gut and identified nine strains capable of producing TMA from choline in vitro.',25784704),(4611,'mouse','null','We screened a collection of 79 sequenced human intestinal isolates encompassing the major phyla found in the human gut and identified nine strains capable of producing TMA from choline in vitro.',25784704),(4612,'mouse','null','Faecalibacterium prausnitzii produces butyrate to decrease c-Myc-related metabolism and Th17 differentiation by inhibiting histone deacetylase 3.',30809639),(4613,'mouse','null','However, PPD  synergistically enhanced APAP induced hepatic injury in vivo and in vitro. Finally, we  found Saccharomyces cerevisiae, which could reduce intestinal PPD levels, was able  to markedly alleviate APAP induced liver damage at ZT12.',29524531),(4616,'mouse','null','Accordingly, dietary succinate improved glucose and insulin tolerance in wild-type mice, but those effects were absent in mice deficient in IGN. Conventional mice colonized with the succinate producer Prevotella copri exhibited metabolic benefits, which could be related to succinate-activated IGN. Thus, microbiota-produced succinate is a previously unsuspected bacterial metabolite improving glycemic control through activation of IGN.',27411015),(4617,'mouse','null','Butyrate can induce AMPK activation and GLUT4 expression in the adipose tissue, improving cardiovascular disease (CVD)-related metabolic disorder, resisting HFD-induced gut microbiome dysbiosis, and promoting resolvin E1 and lipoxin biosynthesis. Oral supplement of the natural fermentation product butyrate can be a potential strategy for preventing CVD.',31607907),(4618,'mouse','null','Butyrate can induce AMPK activation and GLUT4 expression in the adipose tissue, improving cardiovascular disease (CVD)-related metabolic disorder, resisting HFD-induced gut microbiome dysbiosis, and promoting resolvin E1 and lipoxin biosynthesis. Oral supplement of the natural fermentation product butyrate can be a potential strategy for preventing CVD.',31607907),(4619,'mouse','null','Colonic cDNA concentrations of IL-1b and TNF-a were significantly down regulated in DSS-treated E. durans-fed mice but not in control or DSS-treated L.delbrueckii- fed mice.',17349085),(4620,'mouse','null','Colonic cDNA concentrations of IL-1b and TNF-a were significantly down regulated in DSS-treated E. durans-fed mice but not in control or DSS-treated L.delbrueckii- fed mice.',17349085),(4621,'mouse','null','we demonstrate here that α-defensin 5 expression can also be regulated by microbial metabolites. Among these, lactate appeared to significantly suppress α-defensin 5 gene transcription.',27760493),(4622,'mouse','null','We report that the human gut bacterium Bacteroides thetaiotaomicron relies on the stringent response, an intracellular signaling pathway that allocates resources away from growth, to survive carbon starvation and persist in the gut. Genome-scale transcriptomics, 13C-labeling, and metabolomics analyses reveal that B. thetaiotaomicron uses the alarmone (p)ppGpp to repress multiple biosynthetic pathways and upregulate tricarboxylic acid (TCA) cycle genes in these conditions.',30008292),(4623,'mouse','null','Lactobacilli Induce IL-22 via Iald.',23973224),(4624,'mouse','Ferulic acid','Taxonomic analysis of the cecal microbiota showed an increased abundance of Bacteroidetes, along with a reduction in the population of Firmicutes with HMPA supplementation (Figure 4B). Notably, HMPA intervention induced a drastic increase in the population of Actinobacteria, while the population of Deferribacteres in the cecal microbiota of HMPA-supplemented HFD-fed mice were decreased (Figure 4B). These results were similar to those observed with HMCA supplementation. Hierarchical clustering of individual families also confirmed the effect of HMPA intake on gut microbiota (Figure 4C). The HMPA intervention significantly increased the Bacteroidales S24-7 group and Coriobacteriaceae, which are families within the Bacteroidetes and Actinobacteria, respectively (Figure 4C). These families were also significantly increased in HMCA-supplemented HFD-fed mice (Figure 2D). Meanwhile, significant increases in Prevotellaceae and decreases in Deferribacteraceae were observed in HMPA-supplemented HFD-fed mice, but not in HMCA-supplemented HFD-fed mice (Figures 2D and 4C). These findings demonstrated that HMPA intake changed the composition of mouse gut microbiota.',31075850),(4625,'mouse','Ferulic acid','Taxonomic analysis of the cecal microbiota showed an increased abundance of Bacteroidetes, along with a reduction in the population of Firmicutes with HMPA supplementation (Figure 4B). Notably, HMPA intervention induced a drastic increase in the population of Actinobacteria, while the population of Deferribacteres in the cecal microbiota of HMPA-supplemented HFD-fed mice were decreased (Figure 4B). These results were similar to those observed with HMCA supplementation. Hierarchical clustering of individual families also confirmed the effect of HMPA intake on gut microbiota (Figure 4C). The HMPA intervention significantly increased the Bacteroidales S24-7 group and Coriobacteriaceae, which are families within the Bacteroidetes and Actinobacteria, respectively (Figure 4C). These families were also significantly increased in HMCA-supplemented HFD-fed mice (Figure 2D). Meanwhile, significant increases in Prevotellaceae and decreases in Deferribacteraceae were observed in HMPA-supplemented HFD-fed mice, but not in HMCA-supplemented HFD-fed mice (Figures 2D and 4C). These findings demonstrated that HMPA intake changed the composition of mouse gut microbiota.',31075850),(4626,'mouse','Ferulic acid','Taxonomic analysis of the cecal microbiota showed an increased abundance of Bacteroidetes, along with a reduction in the population of Firmicutes with HMPA supplementation (Figure 4B). Notably, HMPA intervention induced a drastic increase in the population of Actinobacteria, while the population of Deferribacteres in the cecal microbiota of HMPA-supplemented HFD-fed mice were decreased (Figure 4B). These results were similar to those observed with HMCA supplementation. Hierarchical clustering of individual families also confirmed the effect of HMPA intake on gut microbiota (Figure 4C). The HMPA intervention significantly increased the Bacteroidales S24-7 group and Coriobacteriaceae, which are families within the Bacteroidetes and Actinobacteria, respectively (Figure 4C). These families were also significantly increased in HMCA-supplemented HFD-fed mice (Figure 2D). Meanwhile, significant increases in Prevotellaceae and decreases in Deferribacteraceae were observed in HMPA-supplemented HFD-fed mice, but not in HMCA-supplemented HFD-fed mice (Figures 2D and 4C). These findings demonstrated that HMPA intake changed the composition of mouse gut microbiota.',31075850),(4627,'mouse','Ferulic acid','Taxonomic analysis of the cecal microbiota showed an increased abundance of Bacteroidetes, along with a reduction in the population of Firmicutes with HMPA supplementation (Figure 4B). Notably, HMPA intervention induced a drastic increase in the population of Actinobacteria, while the population of Deferribacteres in the cecal microbiota of HMPA-supplemented HFD-fed mice were decreased (Figure 4B). These results were similar to those observed with HMCA supplementation. Hierarchical clustering of individual families also confirmed the effect of HMPA intake on gut microbiota (Figure 4C). The HMPA intervention significantly increased the Bacteroidales S24-7 group and Coriobacteriaceae, which are families within the Bacteroidetes and Actinobacteria, respectively (Figure 4C). These families were also significantly increased in HMCA-supplemented HFD-fed mice (Figure 2D). Meanwhile, significant increases in Prevotellaceae and decreases in Deferribacteraceae were observed in HMPA-supplemented HFD-fed mice, but not in HMCA-supplemented HFD-fed mice (Figures 2D and 4C). These findings demonstrated that HMPA intake changed the composition of mouse gut microbiota.',31075850),(4628,'mouse','Ferulic acid','Taxonomic analysis of the cecal microbiota showed an increased abundance of Bacteroidetes, along with a reduction in the population of Firmicutes with HMPA supplementation (Figure 4B). Notably, HMPA intervention induced a drastic increase in the population of Actinobacteria, while the population of Deferribacteres in the cecal microbiota of HMPA-supplemented HFD-fed mice were decreased (Figure 4B). These results were similar to those observed with HMCA supplementation. Hierarchical clustering of individual families also confirmed the effect of HMPA intake on gut microbiota (Figure 4C). The HMPA intervention significantly increased the Bacteroidales S24-7 group and Coriobacteriaceae, which are families within the Bacteroidetes and Actinobacteria, respectively (Figure 4C). These families were also significantly increased in HMCA-supplemented HFD-fed mice (Figure 2D). Meanwhile, significant increases in Prevotellaceae and decreases in Deferribacteraceae were observed in HMPA-supplemented HFD-fed mice, but not in HMCA-supplemented HFD-fed mice (Figures 2D and 4C). These findings demonstrated that HMPA intake changed the composition of mouse gut microbiota.',31075850),(4629,'mouse','Ferulic acid','Taxonomic analysis of the cecal microbiota showed an increased abundance of Bacteroidetes, along with a reduction in the population of Firmicutes with HMPA supplementation (Figure 4B). Notably, HMPA intervention induced a drastic increase in the population of Actinobacteria, while the population of Deferribacteres in the cecal microbiota of HMPA-supplemented HFD-fed mice were decreased (Figure 4B). These results were similar to those observed with HMCA supplementation. Hierarchical clustering of individual families also confirmed the effect of HMPA intake on gut microbiota (Figure 4C). The HMPA intervention significantly increased the Bacteroidales S24-7 group and Coriobacteriaceae, which are families within the Bacteroidetes and Actinobacteria, respectively (Figure 4C). These families were also significantly increased in HMCA-supplemented HFD-fed mice (Figure 2D). Meanwhile, significant increases in Prevotellaceae and decreases in Deferribacteraceae were observed in HMPA-supplemented HFD-fed mice, but not in HMCA-supplemented HFD-fed mice (Figures 2D and 4C). These findings demonstrated that HMPA intake changed the composition of mouse gut microbiota.',31075850),(4630,'mouse','Ferulic acid','Taxonomic analysis of the cecal microbiota showed an increased abundance of Bacteroidetes, along with a reduction in the population of Firmicutes with HMPA supplementation (Figure 4B). Notably, HMPA intervention induced a drastic increase in the population of Actinobacteria, while the population of Deferribacteres in the cecal microbiota of HMPA-supplemented HFD-fed mice were decreased (Figure 4B). These results were similar to those observed with HMCA supplementation. Hierarchical clustering of individual families also confirmed the effect of HMPA intake on gut microbiota (Figure 4C). The HMPA intervention significantly increased the Bacteroidales S24-7 group and Coriobacteriaceae, which are families within the Bacteroidetes and Actinobacteria, respectively (Figure 4C). These families were also significantly increased in HMCA-supplemented HFD-fed mice (Figure 2D). Meanwhile, significant increases in Prevotellaceae and decreases in Deferribacteraceae were observed in HMPA-supplemented HFD-fed mice, but not in HMCA-supplemented HFD-fed mice (Figures 2D and 4C). These findings demonstrated that HMPA intake changed the composition of mouse gut microbiota.',31075850),(4635,'mouse','null','To this end, DCA feeding was performed in HFD-fed mice treated with vancomycin (Figure 2(a,b)). Intriguingly, DCA feeding alone in VCM-treated HFD-fed mice was observed to induce substantial proinflammatory macrophages (iNOS+) infiltration and increase the expression of multiple proinflammatory factors (IL-6, TNF-α, IL-1β, iNOS) in the colon tissue (Figure 2(c–f)), mimicking the inflammatory intestinal damage induced by HFD.',33006494),(4636,'mouse','null','To this end, DCA feeding was performed in HFD-fed mice treated with vancomycin (Figure 2(a,b)). Intriguingly, DCA feeding alone in VCM-treated HFD-fed mice was observed to induce substantial proinflammatory macrophages (iNOS+) infiltration and increase the expression of multiple proinflammatory factors (IL-6, TNF-α, IL-1β, iNOS) in the colon tissue (Figure 2(c–f)), mimicking the inflammatory intestinal damage induced by HFD.',33006494),(4637,'mouse','null','To this end, DCA feeding was performed in HFD-fed mice treated with vancomycin (Figure 2(a,b)). Intriguingly, DCA feeding alone in VCM-treated HFD-fed mice was observed to induce substantial proinflammatory macrophages (iNOS+) infiltration and increase the expression of multiple proinflammatory factors (IL-6, TNF-α, IL-1β, iNOS) in the colon tissue (Figure 2(c–f)), mimicking the inflammatory intestinal damage induced by HFD.',33006494),(4638,'mouse','null','To this end, DCA feeding was performed in HFD-fed mice treated with vancomycin (Figure 2(a,b)). Intriguingly, DCA feeding alone in VCM-treated HFD-fed mice was observed to induce substantial proinflammatory macrophages (iNOS+) infiltration and increase the expression of multiple proinflammatory factors (IL-6, TNF-α, IL-1β, iNOS) in the colon tissue (Figure 2(c–f)), mimicking the inflammatory intestinal damage induced by HFD.',33006494),(4639,'mouse','null','Indeed, DCA stimulation could induce potent increase of mRNA expression level of M2-mAchR in macrophages, and selective inhibition of M2-mAchR significantly reduced theproduction of TNF-α, NO and IL-6 triggered by DCA (Figure 4(d,e) and Supplementary Figure 4c).',33006494),(4640,'mouse','null','Toll-like receptor 2 (TLR2) is up-regulated upon DCA stimulation and involved in DCA-induced macrophage polarization.',33006494),(4641,'mouse','null','Having detected (ex vivo from mouse cecal contents and human stool) and identified 3-methyl-2-oxindole and 5-hydroxyindole-3-acetic acid, 3-indoleacrylic acid and indole-3 carboxylic acid as novel in vitro AHR activators.',32783770),(4642,'mouse','null','Having detected (ex vivo from mouse cecal contents and human stool) and identified 3-methyl-2-oxindole and 5-hydroxyindole-3-acetic acid, 3-indoleacrylic acid and indole-3 carboxylic acid as novel in vitro AHR activators.',32783770),(4643,'mouse','null','Having detected (ex vivo from mouse cecal contents and human stool) and identified 3-methyl-2-oxindole and 5-hydroxyindole-3-acetic acid, 3-indoleacrylic acid and indole-3 carboxylic acid as novel in vitro AHR activators.',32783770),(4644,'mouse','null','Having detected (ex vivo from mouse cecal contents and human stool) and identified 3-methyl-2-oxindole and 5-hydroxyindole-3-acetic acid, 3-indoleacrylic acid and indole-3 carboxylic acid as novel in vitro AHR activators.',32783770),(4645,'mouse','null','At the concentrations tested, indole, 2-oxindole, kynurenic acid and indole-3-acetic acid elicited significant Cyp1a1 induction relative to vehicle (~5-fold, p < .01). 2, 8-dihydroxyquinoline induced Cyp1a1 to a lesser yet significant extent (1.3-fold, p < .01). None of the remaining tryptophan metabolites elicited significant Cyp1a1 inducibility.',32783770),(4646,'mouse','null','At the concentrations tested, indole, 2-oxindole, kynurenic acid and indole-3-acetic acid elicited significant Cyp1a1 induction relative to vehicle (~5-fold, p < .01). 2, 8-dihydroxyquinoline induced Cyp1a1 to a lesser yet significant extent (1.3-fold, p < .01). None of the remaining tryptophan metabolites elicited significant Cyp1a1 inducibility.',32783770),(4647,'mouse','null','At the concentrations tested, indole, 2-oxindole, kynurenic acid and indole-3-acetic acid elicited significant Cyp1a1 induction relative to vehicle (~5-fold, p < .01). 2, 8-dihydroxyquinoline induced Cyp1a1 to a lesser yet significant extent (1.3-fold, p < .01). None of the remaining tryptophan metabolites elicited significant Cyp1a1 inducibility.',32783770),(4648,'mouse','null','At the concentrations tested, indole, 2-oxindole, kynurenic acid and indole-3-acetic acid elicited significant Cyp1a1 induction relative to vehicle (~5-fold, p < .01). 2, 8-dihydroxyquinoline induced Cyp1a1 to a lesser yet significant extent (1.3-fold, p < .01). None of the remaining tryptophan metabolites elicited significant Cyp1a1 inducibility.',32783770),(4649,'mouse','null','At the concentrations tested, indole, 2-oxindole, kynurenic acid and indole-3-acetic acid elicited significant Cyp1a1 induction relative to vehicle (~5-fold, p < .01). 2, 8-dihydroxyquinoline induced Cyp1a1 to a lesser yet significant extent (1.3-fold, p < .01). None of the remaining tryptophan metabolites elicited significant Cyp1a1 inducibility.',32783770),(4650,'mouse','null','Furthermore, TGP failed to blunt the increased of colonic IL-1β in colitis mice receiving ILA (Fig. 6E).',33002829),(4651,'mouse','Tryptophan','We have shown that breast milk tryptophan and Bifidobacterium infantis result in a metabolite (indole-3-lactic acid) response, which is anti-inflammatory via inhibition of the aryl hydrocarbon receptor transcription factor which stimulates an IL-8 response.',32229728),(4656,'mouse','null','Duodenal murine enteroids were exposed to 1 and 5 mM SCFAs for 24 h. Expression of stem cell marker leucine-rich repeat-containing G-protein coupled receptor 5 (Lgr5) was increased more than 2-fold in murine enteroids treated with 5 mM ACET (P < 0.05). Conversely, enteroids treated with 5 mM BUT, or PROP had greater than 5-fold decreases in Lgr5 compared to CON (P < 0.05; Fig 2b). There was no difference in Lgr5 expression in murine enteroids treated with 1 mM SCFAs (P > 0.05; Fig 2a).',32240190),(4657,'mouse','null','Duodenal murine enteroids were exposed to 1 and 5 mM SCFAs for 24 h. Expression of stem cell marker leucine-rich repeat-containing G-protein coupled receptor 5 (Lgr5) was increased more than 2-fold in murine enteroids treated with 5 mM ACET (P < 0.05). Conversely, enteroids treated with 5 mM BUT, or PROP had greater than 5-fold decreases in Lgr5 compared to CON (P < 0.05; Fig 2b). There was no difference in Lgr5 expression in murine enteroids treated with 1 mM SCFAs (P > 0.05; Fig 2a).',32240190),(4658,'mouse','null','Duodenal murine enteroids were exposed to 1 and 5 mM SCFAs for 24 h. Expression of stem cell marker leucine-rich repeat-containing G-protein coupled receptor 5 (Lgr5) was increased more than 2-fold in murine enteroids treated with 5 mM ACET (P < 0.05). Conversely, enteroids treated with 5 mM BUT, or PROP had greater than 5-fold decreases in Lgr5 compared to CON (P < 0.05; Fig 2b). There was no difference in Lgr5 expression in murine enteroids treated with 1 mM SCFAs (P > 0.05; Fig 2a).',32240190),(4659,'mouse','null','Enterocyte biomarker alkaline phosphatase (Alpi) was significantly increased in response to all SCFA treatments but was highest (3.5-fold;P < 0.05) in enteroids treated with PROP. There was no difference in Alpi expression in murine enteroids treated with 1 mM SCFAs (P > 0.05; Fig 2a).',32240190),(4660,'mouse','null','Enterocyte biomarker alkaline phosphatase (Alpi) was significantly increased in response to all SCFA treatments but was highest (3.5-fold;P < 0.05) in enteroids treated with PROP. There was no difference in Alpi expression in murine enteroids treated with 1 mM SCFAs (P > 0.05; Fig 2a).',32240190),(4661,'mouse','null','Enterocyte biomarker alkaline phosphatase (Alpi) was significantly increased in response to all SCFA treatments but was highest (3.5-fold;P < 0.05) in enteroids treated with PROP. There was no difference in Alpi expression in murine enteroids treated with 1 mM SCFAs (P > 0.05; Fig 2a).',32240190),(4662,'mouse','null','Gene expression of goblet cell marker mucin 2 (Muc2) was not significantly affected by ACET treatment, however, Muc2 was significantly decreased due to both BUT and PROP treatment (3-fold and 10-fold respectively;P < 0.05). Muc2 expression was not altered in murine enteroids treated with 1 mM SCFAs(P > 0.05; Fig 2a).',32240190),(4663,'mouse','null','Gene expression of goblet cell marker mucin 2 (Muc2) was not significantly affected by ACET treatment, however, Muc2 was significantly decreased due to both BUT and PROP treatment (3-fold and 10-fold respectively;P < 0.05). Muc2 expression was not altered in murine enteroids treated with 1 mM SCFAs(P > 0.05; Fig 2a).',32240190),(4664,'mouse','null','Paneth cell marker lysozyme (Lyz) had a tendency (P = 0.08) to increase due to BUT treatment but there was no effect observed when treated with 1 mM SCFAs (P > 0.05;Fig 2a).',32240190),(4665,'mouse','null','Enteroendocrine cell marker chromogranin A (Chga) was only significantly increased in BUT treated enteroids at both 1 mM and 5 mM concentrations (P < 0.05).',32240190),(4666,'mouse','null','Tuft cell marker doublecortin-like kinase 1 (Dclk1) was significantly elevated in response to 5 mM ACET (3-fold) and PROP (2-fold) treatments compared to CON (P < 0.05; Fig 2b). There was no difference in Dclk1 expression in murine enteroids treated with 1 mM SCFAs (P > 0.05; Fig 2a).',32240190),(4667,'mouse','null','Tuft cell marker doublecortin-like kinase 1 (Dclk1) was significantly elevated in response to 5 mM ACET (3-fold) and PROP (2-fold) treatments compared to CON (P < 0.05; Fig 2b). There was no difference in Dclk1 expression in murine enteroids treated with 1 mM SCFAs (P > 0.05; Fig 2a).',32240190),(4668,'mouse','null','Antimicrobial peptide gene expression of c-type lectins regenerating islet-derived protein 3β (Reg3β) and 3γ (Reg3γ; Fig 2b) were increased moderately due to ACET treatment (20-fold and 40-fold), but were much more significantly increased due to BUT (150-fold and 350-fold,respectively) and PROP (250-fold, and 480-fold, respectively, P < 0.05) compared to CON.Only 1 mM BUT increased Reg3β and Reg3γ expression (80-fold and 100-fold respectively;P < 0.05).',32240190),(4669,'mouse','null','Antimicrobial peptide gene expression of c-type lectins regenerating islet-derived protein 3β (Reg3β) and 3γ (Reg3γ; Fig 2b) were increased moderately due to ACET treatment (20-fold and 40-fold), but were much more significantly increased due to BUT (150-fold and 350-fold,respectively) and PROP (250-fold, and 480-fold, respectively, P < 0.05) compared to CON.Only 1 mM BUT increased Reg3β and Reg3γ expression (80-fold and 100-fold respectively;P < 0.05).',32240190),(4670,'mouse','null','Antimicrobial peptide, β-defensin 1 (Defb1), was increased 3.5-fold due to ACET and BUT treatment, and 6-fold due to PROP treatment (P < 0.05); Fig 2b). 1 mM PROP was sufficient to increase Defb1 expression 2-fold compared to CON (P < 0.05).',32240190),(4671,'mouse','null','Antimicrobial peptide, β-defensin 1 (Defb1), was increased 3.5-fold due to ACET and BUT treatment, and 6-fold due to PROP treatment (P < 0.05); Fig 2b). 1 mM PROP was sufficient to increase Defb1 expression 2-fold compared to CON (P < 0.05).',32240190),(4672,'mouse','null','Antimicrobial peptide, β-defensin 1 (Defb1), was increased 3.5-fold due to ACET and BUT treatment, and 6-fold due to PROP treatment (P < 0.05); Fig 2b). 1 mM PROP was sufficient to increase Defb1 expression 2-fold compared to CON (P < 0.05).',32240190),(4673,'mouse','null','Tight junction markers occludin (Ocln), and claudin-3 (Cldn3) were increased nearly 2-fold in enteroids treated with 5 mM ACET (P < 0.05).',32240190),(4674,'mouse','null','Tight junction markers occludin (Ocln), and claudin-3 (Cldn3) were increased nearly 2-fold in enteroids treated with 5 mM ACET (P < 0.05).',32240190),(4675,'mouse','null','BUT decreased Cldn3 expression at 5 mM but Ocln was not altered (P > 0.05).',32240190),(4676,'mouse','null','5 mM PROP significantly decreased Cldn3 (P < 0.05).',32240190),(4677,'mouse','null','Tight junction protein 1 (Tjp1) was not altered by ACET treatment, but was decreased 1.7 and 2-fold respectively due to BUT and PROP treatment (P < 0.05; Fig 2b). There was no\ndifference in Tjp1 expression in murine enteroids treated with 1 mM SCFAs (P > 0.05; Fig 2a).',32240190),(4678,'mouse','null','Tight junction protein 1 (Tjp1) was not altered by ACET treatment, but was decreased 1.7 and 2-fold respectively due to BUT and PROP treatment (P < 0.05; Fig 2b). There was no\ndifference in Tjp1 expression in murine enteroids treated with 1 mM SCFAs (P > 0.05; Fig 2a).',32240190),(4679,'mouse','null','Ergothioneine, a metabolite of the gut bacterium Lactobacillus reuteri, protects against stress-induced sleep disturbances.',32467627),(4680,'mouse','null','Moreover, the expression of Cyp3a11 in GF mice also remained exten sively downregulated after butyrate supplementation rela tive to the corresponding conventional group (P < 0.01;P < 0.001, respectively).',32337713),(4681,'mouse','null','In the presence of a complex microbiota, butyrate only had a significant effect on the hepatic expression of Cyp2b10 (2.85-fold higher relative to conventional mice; P < 0.05).',32337713),(4682,'mouse','null','Butyrate decreased the expression of Mdr1a in GF mice(P < 0.05) relative to the butyrate-treated conventional group, despite no evident changes in this isoform under GF conditions or by colonisation.',32337713),(4683,'mouse','null','Analysis of the expression of mRNAs of three TJ proteins (ZO-1, claudin-1, and occludin) indicated greater expression in the Sham Group than in the three other groups (P < 0.05), and greater expression in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4A,B,D). In addition, expression of claudin-2, MMP9 was lower in the Sham Group than in the three other groups (P < 0.05) and lower in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4C,E).',33192557),(4684,'mouse','null','Analysis of the expression of mRNAs of three TJ proteins (ZO-1, claudin-1, and occludin) indicated greater expression in the Sham Group than in the three other groups (P < 0.05), and greater expression in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4A,B,D). In addition, expression of claudin-2, MMP9 was lower in the Sham Group than in the three other groups (P < 0.05) and lower in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4C,E).',33192557),(4685,'mouse','null','Analysis of the expression of mRNAs of three TJ proteins (ZO-1, claudin-1, and occludin) indicated greater expression in the Sham Group than in the three other groups (P < 0.05), and greater expression in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4A,B,D). In addition, expression of claudin-2, MMP9 was lower in the Sham Group than in the three other groups (P < 0.05) and lower in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4C,E).',33192557),(4686,'mouse','null','Analysis of the expression of mRNAs of three TJ proteins (ZO-1, claudin-1, and occludin) indicated greater expression in the Sham Group than in the three other groups (P < 0.05), and greater expression in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4A,B,D). In addition, expression of claudin-2, MMP9 was lower in the Sham Group than in the three other groups (P < 0.05) and lower in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4C,E).',33192557),(4687,'mouse','null','Analysis of the expression of mRNAs of three TJ proteins (ZO-1, claudin-1, and occludin) indicated greater expression in the Sham Group than in the three other groups (P < 0.05), and greater expression in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4A,B,D). In addition, expression of claudin-2, MMP9 was lower in the Sham Group than in the three other groups (P < 0.05) and lower in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4C,E).',33192557),(4688,'mouse','null','In addition, the DAO, LPS, FITC-dextran, and inflammatory cytokine levels of the two treatment groups were similar to each other (P > 0.05) and significantly lower than that of the SAP + IAH Group (P < 0.05; Figures 3B–E).',33192557),(4689,'mouse','null','In addition, the DAO, LPS, FITC-dextran, and inflammatory cytokine levels of the two treatment groups were similar to each other (P > 0.05) and significantly lower than that of the SAP + IAH Group (P < 0.05; Figures 3B–E).',33192557),(4690,'mouse','null','In addition, the DAO, LPS, FITC-dextran, and inflammatory cytokine levels of the two treatment groups were similar to each other (P > 0.05) and significantly lower than that of the SAP + IAH Group (P < 0.05; Figures 3B–E).',33192557),(4691,'mouse','null','Relative to the SAP + IAH group, the levels of claudin-2, MMP9, and TNF-α were significantly lower in the two treatment groups (P < 0.05; Figures 5A,B).',33192557),(4692,'mouse','null','LPS significantly decreased the level of TEER (P < 0.01), while treatment with PA moderated the reduced TEER (P < 0.05). Secondly, we utilized immunoblotting to analyze the protein expression levels of ZO-1, occludin, and claudin-1. As shown in Figures 2B–E, the protein expression levels of ZO-1, occludin, and claudin-1 were lower in the LPS group than in the control group (P < 0.01). This LPS-induced alteration of tight junction protein expression was inhibited by PA (P < 0.05). These results suggest that PA can enhance and protect intestinal barrier function against LPS in IEC-6 cells.',33041816),(4693,'mouse','null','LPS significantly decreased the level of TEER (P < 0.01), while treatment with PA moderated the reduced TEER (P < 0.05). Secondly, we utilized immunoblotting to analyze the protein expression levels of ZO-1, occludin, and claudin-1. As shown in Figures 2B–E, the protein expression levels of ZO-1, occludin, and claudin-1 were lower in the LPS group than in the control group (P < 0.01). This LPS-induced alteration of tight junction protein expression was inhibited by PA (P < 0.05). These results suggest that PA can enhance and protect intestinal barrier function against LPS in IEC-6 cells.',33041816),(4694,'mouse','null','LPS significantly decreased the level of TEER (P < 0.01), while treatment with PA moderated the reduced TEER (P < 0.05). Secondly, we utilized immunoblotting to analyze the protein expression levels of ZO-1, occludin, and claudin-1. As shown in Figures 2B–E, the protein expression levels of ZO-1, occludin, and claudin-1 were lower in the LPS group than in the control group (P < 0.01). This LPS-induced alteration of tight junction protein expression was inhibited by PA (P < 0.05). These results suggest that PA can enhance and protect intestinal barrier function against LPS in IEC-6 cells.',33041816),(4695,'mouse','null','These data indicate that PA promotes epithelial cell migration via inhibition of expression of b1-integrin induced by LPS.',33041816),(4696,'mouse','null','Treatment with PA rescued the obvious increases in protein expression in NLRP3, caspase-1, IL-1b, and IL-18 induced by LPS.',33041816),(4697,'mouse','null','Treatment with PA rescued the obvious increases in protein expression in NLRP3, caspase-1, IL-1b, and IL-18 induced by LPS.',33041816),(4698,'mouse','null','Treatment with PA rescued the obvious increases in protein expression in NLRP3, caspase-1, IL-1b, and IL-18 induced by LPS.',33041816),(4699,'mouse','null','Treatment with PA rescued the obvious increases in protein expression in NLRP3, caspase-1, IL-1b, and IL-18 induced by LPS.',33041816),(4700,'mouse','null','The research results show that LPS strongly stimulated TLR4, the phosphorylation of IkBa and p65 in whole cell extracts and IkBa degradation in the cytoplasm. It also caused the translocation of p65 into the nucleus. However, these effects were significantly inhibited by PA (Figures 5A–G).',33041816),(4701,'mouse','null','SLC5A8 and GPR43 expressions in butyrate‑treated CRC cells were quantified (Fig. 5B and C for SLC5A8 and GPR43, respectively). In comparison with untreated cells, the NaB‑treated SW480 cells only presented higher GPR43 (P<0.05), but the NaB‑treated SW620 cells expressed the most SLC5A8 (P<0.01) and GPR43 (P<0.05).',33101496),(4704,'mouse','Cholic acid','Intestinal expression of farnesoid X receptor (FXR) mRNA was significantly higher, whereas hepatic expression of cholesterol-7a-hydroxylase 1 (CYP7A1) and small heterodimer partner (SHP) were significantly lower in ob/ob mice compared to those in control mice.',27111464),(4705,'mouse','Cholic acid','FGF15 mRNA expression was also increased in ob/ob mice compared to that of the lean control mice.',27111464),(4706,'mouse','Cholic acid','In the present study, hepatic mRNA expression of FGFR4 was significantly increased, while CYP7A1 was greatly decreased in ob/ob mice.',27111464),(4707,'mouse','Cholic acid','In the presentstudy, hepatic mRNA expression of FGFR4 was significantly increased, while CYP7A1 was greatly decreased in ob/ob mice.',27111464),(4708,'mouse','Cholic acid','Decreased intestinal taurine-conjugated BAs may also affect enterohepatic circulation of BAs and facilitate hepatic de novo lipogenesis and down-regulate fatty acid oxidation in ob/ob mice through intestinal FXR-FGF15 and hepatic FXR-SHP pathways.',27111464),(4709,'mouse','Cholic acid','Also, ob/ob mice showed significantly downregulated CPT-1 mRNA expression compared to their lean control mice.',27111464),(4710,'mouse','Cholic acid','Moreover, hepatic FAS gene expression was up-regulated while both SREBP-1c and ACC were slightly increased without significant differences between two groups.',27111464),(4711,'mouse','Cholic acid','Intestinal expression of farnesoid X receptor (FXR) mRNA was significantly higher, whereas hepatic expression of cholesterol-7a-hydroxylase 1 (CYP7A1) and small heterodimer partner (SHP) were significantly lower in ob/ob mice compared to those in control mice.',27111464),(4712,'mouse','Cholic acid','FGF15 mRNA expression was also increased in ob/ob mice compared to that of the lean control mice.',27111464),(4713,'mouse','Cholic acid','In the present study, hepatic mRNA expression of FGFR4 was significantly increased, while CYP7A1 was greatly decreased in ob/ob mice.',27111464),(4714,'mouse','Cholic acid','In the presentstudy, hepatic mRNA expression of FGFR4 was significantly increased, while CYP7A1 was greatly decreased in ob/ob mice.',27111464),(4715,'mouse','Cholic acid','Decreased intestinal taurine-conjugated BAs may also affect enterohepatic circulation of BAs and facilitate hepatic de novo lipogenesis and down-regulate fatty acid oxidation in ob/ob mice through intestinal FXR-FGF15 and hepatic FXR-SHP pathways.',27111464),(4716,'mouse','Cholic acid','Also, ob/ob mice showed significantly downregulated CPT-1 mRNA expression compared to their lean control mice.',27111464),(4717,'mouse','Cholic acid','Moreover, hepatic FAS gene expression was up-regulated while both SREBP-1c and ACC were slightly increased without significant differences between two groups.',27111464),(4718,'mouse','null','Intestinal expression of farnesoid X receptor (FXR) mRNA was significantly higher, whereas hepatic expression of cholesterol-7a-hydroxylase 1 (CYP7A1) and small heterodimer partner (SHP) were significantly lower in ob/ob mice compared to those in control mice.',27111464),(4719,'mouse','null','FGF15 mRNA expression was also increased in ob/ob mice compared to that of the lean control mice.',27111464),(4720,'mouse','null','In the present study, hepatic mRNA expression of FGFR4 was significantly increased, while CYP7A1 was greatly decreased in ob/ob mice.',27111464),(4721,'mouse','null','In the presentstudy, hepatic mRNA expression of FGFR4 was significantly increased, while CYP7A1 was greatly decreased in ob/ob mice.',27111464),(4722,'mouse','null','Decreased intestinal taurine-conjugated BAs may also affect enterohepatic circulation of BAs and facilitate hepatic de novo lipogenesis and down-regulate fatty acid oxidation in ob/ob mice through intestinal FXR-FGF15 and hepatic FXR-SHP pathways.',27111464),(4723,'mouse','null','Also, ob/ob mice showed significantly downregulated CPT-1 mRNA expression compared to their lean control mice.',27111464),(4724,'mouse','null','Moreover, hepatic FAS gene expression was up-regulated while both SREBP-1c and ACC were slightly increased without significant differences between two groups.',27111464),(4725,'mouse','Indole','Production of IPA was shown to be completely dependent on the presence of gut microflora and could be established by colonization with the bacterium Clostridium sporogenes.',19234110),(4726,'mouse','Rutin','LY1 also metabolized rutin to quercetin and 3,4-DPA, although the amount of 3,4-DPA formed was minute compared to that of quercetin.',14629841),(4727,'mouse','Rutin','LY1 also metabolized rutin to quercetin and 3,4-DPA, although the amount of 3,4-DPA formed was minute compared to that of quercetin.',14629841),(4728,'mouse','Quercitrin','A significant portion of flavonoids in EGb 761 was converted to the main aglycone forms (quercetin and kaempferol) (Fig. 4B) after 36 hours.',14629841),(4729,'mouse','Quercitrin','A significant portion of flavonoids in EGb 761 was converted to the main aglycone forms (quercetin and kaempferol) (Fig. 4B) after 36 hours.',14629841),(4730,'mouse','Quercitrin','When enrichment cultures derived from either rat fecal materials was used, glycosides in EGb 761 were completely consumed but quercetin, kaempferol, and the possible isorhamnetin were only transiently detected during the growth of the bacteria in enrichment cultures.',14629841),(4731,'mouse','Isoquercitrin','A significant portion of flavonoids in EGb 761 was converted to the main aglycone forms (quercetin and kaempferol) (Fig. 4B) after 36 hours.',14629841),(4732,'mouse','Isoquercitrin','A significant portion of flavonoids in EGb 761 was converted to the main aglycone forms (quercetin and kaempferol) (Fig. 4B) after 36 hours.',14629841),(4733,'mouse','Isoquercitrin','When enrichment cultures derived from either rat fecal materials was used, glycosides in EGb 761 were completely consumed but quercetin, kaempferol, and the possible isorhamnetin were only transiently detected during the growth of the bacteria in enrichment cultures.',14629841),(4734,'mouse','null','For example, quinic acid, a cyclic polyol obtained from plant-derived products, reached its highest levels when Odoribacter splanchnicus was present but Escherichia coli was not.',24452263),(4735,'mouse','null','One such example is our observation that concentrations of a lysophosphatidylethanolamine (LysoPE15) can be increased by either B. ovatus or B. vulgatus, but reaches maximal cecal concentrations when both strains were present in a community (Fig. 3B).',24452263),(4736,'mouse','null','One such example is our observation that concentrations of a lysophosphatidylethanolamine (LysoPE15) can be increased by either B. ovatus or B. vulgatus, but reaches maximal cecal concentrations when both strains were present in a community (Fig. 3B).',24452263),(4737,'mouse','null','While another metabolite (assignment, 4�?hydroxy-3�?5,6,7,8-pentamethoxyflavone) required both B. ovatus and O.splanchnicus to consistently reach high levels.',24452263),(4738,'mouse','null','While another metabolite (assignment, 4�?hydroxy-3�?5,6,7,8-pentamethoxyflavone) required both B. ovatus and O.splanchnicus to consistently reach high levels.',24452263),(4747,'mouse','null','Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.',24452263),(4748,'mouse','null','Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.',24452263),(4749,'mouse','null','Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.',24452263),(4750,'mouse','null','Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.',24452263),(4751,'mouse','null','Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.',24452263),(4752,'mouse','null','Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.',24452263),(4753,'mouse','null','Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.',24452263),(4754,'mouse','null','Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.',24452263),(4755,'mouse','null','Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.',24452263),(4756,'mouse','null','Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.',24452263),(4757,'mouse','null','Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.',24452263),(4758,'mouse','null','Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.',24452263),(4759,'mouse','null','Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.',24452263),(4760,'mouse','Lactic acid','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.',27448578),(4761,'mouse','Lactic acid','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.',27448578),(4762,'mouse','Lactic acid','SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.',27448578),(4763,'mouse','Lactic acid','Furthermore, DCs from mice treated with propionate, characterized by reduced expression of CD40, PD-L2 and CD86, exhibited the impaired ability to initiate Th2 effector function and further to promote Th2-mediated allergic airway inflammation.',27448578),(4764,'mouse','Lactic acid','Furthermore, DCs from mice treated with propionate, characterized by reduced expression of CD40, PD-L2 and CD86, exhibited the impaired ability to initiate Th2 effector function and further to promote Th2-mediated allergic airway inflammation.',27448578),(4765,'mouse','Lactic acid','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4766,'mouse','Lactic acid','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4767,'mouse','Lactic acid','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4768,'mouse','Pyruvate','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.',27448578),(4769,'mouse','Pyruvate','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.',27448578),(4770,'mouse','Pyruvate','SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.',27448578),(4771,'mouse','null','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.',27448578),(4772,'mouse','null','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.',27448578),(4773,'mouse','null','SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.',27448578),(4774,'mouse','null','GPR41 and GPR43 can be efficiently activated by acetate, propionate, butyrate and other SCFAs [39], while GPR109a can be activated mainly by butyrate and niacin.',27448578),(4775,'mouse','null','Butyrate upregulated the histone H3 acetylation of Foxp3 and promoted the differentiation of Treg, which acts as a key anti-inflammatory effector.',27448578),(4776,'mouse','null','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4777,'mouse','null','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4778,'mouse','null','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4779,'mouse','null','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.',27448578),(4780,'mouse','null','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.',27448578),(4781,'mouse','null','SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.',27448578),(4782,'mouse','null','GPR41 and GPR43 can be efficiently activated by acetate, propionate, butyrate and other SCFAs [39], while GPR109a can be activated mainly by butyrate and niacin.',27448578),(4783,'mouse','null','Butyrate upregulated the histone H3 acetylation of Foxp3 and promoted the differentiation of Treg, which acts as a key anti-inflammatory effector.',27448578),(4784,'mouse','null','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4785,'mouse','null','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4786,'mouse','null','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4787,'mouse','null','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.',27448578),(4788,'mouse','null','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.',27448578),(4789,'mouse','null','SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.',27448578),(4790,'mouse','null','GPR41 and GPR43 can be efficiently activated by acetate, propionate, butyrate and other SCFAs [39], while GPR109a can be activated mainly by butyrate and niacin.',27448578),(4791,'mouse','null','Butyrate upregulated the histone H3 acetylation of Foxp3 and promoted the differentiation of Treg, which acts as a key anti-inflammatory effector.',27448578),(4792,'mouse','null','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4793,'mouse','null','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4794,'mouse','null','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4795,'mouse','Starch','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.',27448578),(4796,'mouse','Starch','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.',27448578),(4797,'mouse','Starch','SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.',27448578),(4798,'mouse','Starch','GPR41 and GPR43 can be efficiently activated by acetate, propionate, butyrate and other SCFAs [39], while GPR109a can be activated mainly by butyrate and niacin.',27448578),(4799,'mouse','Starch','Butyrate upregulated the histone H3 acetylation of Foxp3 and promoted the differentiation of Treg, which acts as a key anti-inflammatory effector.',27448578),(4800,'mouse','Starch','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4801,'mouse','Starch','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4802,'mouse','Starch','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4803,'mouse','Succinate','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.',27448578),(4804,'mouse','Succinate','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.',27448578),(4805,'mouse','Succinate','SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.',27448578),(4806,'mouse','Succinate','Furthermore, DCs from mice treated with propionate, characterized by reduced expression of CD40, PD-L2 and CD86, exhibited the impaired ability to initiate Th2 effector function and further to promote Th2-mediated allergic airway inflammation.',27448578),(4807,'mouse','Succinate','Furthermore, DCs from mice treated with propionate, characterized by reduced expression of CD40, PD-L2 and CD86, exhibited the impaired ability to initiate Th2 effector function and further to promote Th2-mediated allergic airway inflammation.',27448578),(4808,'mouse','Succinate','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4809,'mouse','Succinate','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4810,'mouse','Succinate','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4811,'mouse','Tryptophan','Colonization of germ-free mice with wild-type Escherichia coli, but not E. coli mutants unable to generate indole, induced colonic epithelial IL-10R1.',29454749),(4812,'mouse','Tryptophan','Colonization of germ-free mice with wild-type Escherichia coli, but not E. coli mutants unable to generate indole, induced colonic epithelial IL-10R1.',29454749),(4813,'mouse','Tryptophan','IAld has recently been identified as an aryl hydrocarbon receptor (AHR) ligand.',29454749),(4814,'mouse','Tryptophan','Colonization of germ-free mice with wild-type Escherichia coli, but not E. coli mutants unable to generate indole, induced colonic epithelial IL-10R1.',29454749),(4815,'mouse','Tryptophan','Colonization of germ-free mice with wild-type Escherichia coli, but not E. coli mutants unable to generate indole, induced colonic epithelial IL-10R1.',29454749),(4816,'mouse','Tryptophan','IAld has recently been identified as an aryl hydrocarbon receptor (AHR) ligand.',29454749),(4817,'mouse','null','In mice, nicotinic acid inhibits lipolysis via PUMA-G, a Gi/o-coupled seven-transmembrane receptor expressed in adipocytes and activated macrophages.',15929991),(4818,'mouse','null','Here we show that the fatty acid-derived ketone body (D)-OHB specifically activates PUMA-G/HM74a at concentrations observed in serum during fasting.',15929991),(4819,'mouse','null','Here we show that the fatty acid-derived ketone body (D)-OHB specifically activates PUMA-G/HM74a at concentrations observed in serum during fasting.',15929991),(4820,'mouse','null','Both (D)- and (L)-OHB, but not acetoacetate or free acetone, elicited a Ca2 response in cells expressing HM74a but not HM74.',15929991),(4821,'mouse','null','Racemic (DL)-OHB also showed some degree of activity on all receptors; however, it was more potent on HM74a and its ortholog PUMA-G  than on HM74.',15929991),(4825,'mouse','Tryptophan','Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.',31123037),(4826,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4827,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4828,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4829,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4830,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4831,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4832,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4833,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4834,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4835,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4836,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4837,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4838,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4839,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4840,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4841,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4842,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4843,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4844,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4845,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4846,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4847,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4848,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4849,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4850,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4851,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4852,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4853,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4854,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4855,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4856,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4857,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4858,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4859,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4860,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4861,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4862,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4863,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4864,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4865,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4866,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4867,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4868,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4869,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4870,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4871,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4872,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4873,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4874,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4875,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4876,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4877,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4878,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4879,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4880,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4881,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4882,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4883,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4884,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4885,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4886,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4887,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4888,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4889,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4890,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4891,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4892,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4893,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4894,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4895,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4896,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4897,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4898,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4899,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4900,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4901,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4902,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4903,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4904,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4905,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4906,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4907,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4908,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4909,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4910,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4911,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4912,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4913,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4914,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4915,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4916,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4917,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4918,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4919,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4920,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4921,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4922,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4923,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4924,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4925,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4926,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4927,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4928,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4929,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4930,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4931,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4932,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4933,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4934,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4935,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4936,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4937,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4938,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4939,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4940,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4941,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4942,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4943,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4944,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4945,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4946,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4947,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4948,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4949,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4950,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4951,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4952,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4953,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4954,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4955,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4956,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4957,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4958,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4959,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4960,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4961,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4962,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4963,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4964,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4965,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4966,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4967,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4968,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4969,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4970,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4971,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4972,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4973,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4974,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4975,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4976,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4977,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4978,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4979,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4980,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4981,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4982,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4983,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4984,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4985,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4986,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4987,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4988,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4989,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4990,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4991,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4992,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4993,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4994,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4995,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4996,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4997,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4998,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4999,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5000,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5001,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5002,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5003,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5004,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5005,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5006,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5007,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5008,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5009,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5010,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5011,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5012,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5013,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5014,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5015,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5016,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5017,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5018,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5019,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5020,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5021,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5022,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5023,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5024,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5025,'mouse','null','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5027,'mouse','D-Fructose','A high-fructose diet caused the hippocampal neuroinflammatory response, reactive gliosis, and neuronal loss in C57BL/6N mice. Depletion of the gut microbiota using broad-spectrum antibiotics suppressed the hippocampal neuroinflammatory response in fructose-fed mice, but these animals still exhibited neuronal loss. Gut microbiota compositional alteration, short-chain fatty acids (SCFAs) reduction, intestinal epithelial barrier impairment, NOD-like receptor family pyrin domain-containing 6 (NLRP6) inflammasome dysfunction, high levels of serum endotoxin, and FITC-dextran were observed in fructose-fed mice. Of note, SCFAs, as well as pioglitazone (a selective peroxisome proliferator-activated receptor gamma (PPAR-γ) agonist), shaped the gut microbiota and ameliorated intestinal epithelial barrier impairment and NLRP6 inflammasome dysfunction in fructose-fed mice. Moreover, SCFAs-mediated NLRP6 inflammasome activation was inhibited by histamine (a bacterial metabolite) in ex vivo colonic explants and suppressed in murine CT26 colon carcinoma cells transfected with NLRP6 siRNA. However, pioglitazone and GW9662 (a PPAR-γ antagonist) exerted no impact on SCFAs-mediated NLRP6 inflammasome activation in ex vivo colonic explants, suggesting that SCFAs may stimulate NLRP6 inflammasome independently of PPAR-γ activation. SCFAs and pioglitazone prevented fructose-induced hippocampal neuroinflammatory response and neuronal loss in mice. Additionally, SCFAs activated colonic NLRP6 inflammasome and increased DCX+ newborn neurons in the hippocampal DG of control mice',31255176),(5028,'mouse','D-Fructose','A high-fructose diet caused the hippocampal neuroinflammatory response, reactive gliosis, and neuronal loss in C57BL/6N mice. Depletion of the gut microbiota using broad-spectrum antibiotics suppressed the hippocampal neuroinflammatory response in fructose-fed mice, but these animals still exhibited neuronal loss. Gut microbiota compositional alteration, short-chain fatty acids (SCFAs) reduction, intestinal epithelial barrier impairment, NOD-like receptor family pyrin domain-containing 6 (NLRP6) inflammasome dysfunction, high levels of serum endotoxin, and FITC-dextran were observed in fructose-fed mice. Of note, SCFAs, as well as pioglitazone (a selective peroxisome proliferator-activated receptor gamma (PPAR-γ) agonist), shaped the gut microbiota and ameliorated intestinal epithelial barrier impairment and NLRP6 inflammasome dysfunction in fructose-fed mice. Moreover, SCFAs-mediated NLRP6 inflammasome activation was inhibited by histamine (a bacterial metabolite) in ex vivo colonic explants and suppressed in murine CT26 colon carcinoma cells transfected with NLRP6 siRNA. However, pioglitazone and GW9662 (a PPAR-γ antagonist) exerted no impact on SCFAs-mediated NLRP6 inflammasome activation in ex vivo colonic explants, suggesting that SCFAs may stimulate NLRP6 inflammasome independently of PPAR-γ activation. SCFAs and pioglitazone prevented fructose-induced hippocampal neuroinflammatory response and neuronal loss in mice. Additionally, SCFAs activated colonic NLRP6 inflammasome and increased DCX+ newborn neurons in the hippocampal DG of control mice',31255176),(5029,'mouse','D-Fructose','A high-fructose diet caused the hippocampal neuroinflammatory response, reactive gliosis, and neuronal loss in C57BL/6N mice. Depletion of the gut microbiota using broad-spectrum antibiotics suppressed the hippocampal neuroinflammatory response in fructose-fed mice, but these animals still exhibited neuronal loss. Gut microbiota compositional alteration, short-chain fatty acids (SCFAs) reduction, intestinal epithelial barrier impairment, NOD-like receptor family pyrin domain-containing 6 (NLRP6) inflammasome dysfunction, high levels of serum endotoxin, and FITC-dextran were observed in fructose-fed mice. Of note, SCFAs, as well as pioglitazone (a selective peroxisome proliferator-activated receptor gamma (PPAR-γ) agonist), shaped the gut microbiota and ameliorated intestinal epithelial barrier impairment and NLRP6 inflammasome dysfunction in fructose-fed mice. Moreover, SCFAs-mediated NLRP6 inflammasome activation was inhibited by histamine (a bacterial metabolite) in ex vivo colonic explants and suppressed in murine CT26 colon carcinoma cells transfected with NLRP6 siRNA. However, pioglitazone and GW9662 (a PPAR-γ antagonist) exerted no impact on SCFAs-mediated NLRP6 inflammasome activation in ex vivo colonic explants, suggesting that SCFAs may stimulate NLRP6 inflammasome independently of PPAR-γ activation. SCFAs and pioglitazone prevented fructose-induced hippocampal neuroinflammatory response and neuronal loss in mice. Additionally, SCFAs activated colonic NLRP6 inflammasome and increased DCX+ newborn neurons in the hippocampal DG of control mice',31255176),(5030,'mouse','D-Fructose','A high-fructose diet caused the hippocampal neuroinflammatory response, reactive gliosis, and neuronal loss in C57BL/6N mice. Depletion of the gut microbiota using broad-spectrum antibiotics suppressed the hippocampal neuroinflammatory response in fructose-fed mice, but these animals still exhibited neuronal loss. Gut microbiota compositional alteration, short-chain fatty acids (SCFAs) reduction, intestinal epithelial barrier impairment, NOD-like receptor family pyrin domain-containing 6 (NLRP6) inflammasome dysfunction, high levels of serum endotoxin, and FITC-dextran were observed in fructose-fed mice. Of note, SCFAs, as well as pioglitazone (a selective peroxisome proliferator-activated receptor gamma (PPAR-γ) agonist), shaped the gut microbiota and ameliorated intestinal epithelial barrier impairment and NLRP6 inflammasome dysfunction in fructose-fed mice. Moreover, SCFAs-mediated NLRP6 inflammasome activation was inhibited by histamine (a bacterial metabolite) in ex vivo colonic explants and suppressed in murine CT26 colon carcinoma cells transfected with NLRP6 siRNA. However, pioglitazone and GW9662 (a PPAR-γ antagonist) exerted no impact on SCFAs-mediated NLRP6 inflammasome activation in ex vivo colonic explants, suggesting that SCFAs may stimulate NLRP6 inflammasome independently of PPAR-γ activation. SCFAs and pioglitazone prevented fructose-induced hippocampal neuroinflammatory response and neuronal loss in mice. Additionally, SCFAs activated colonic NLRP6 inflammasome and increased DCX+ newborn neurons in the hippocampal DG of control mice',31255176),(5031,'mouse','(-)-Epigallocatechin gallate','Inhibitory activity of angiotensin I-converting enzyme (ACE) was examined with (�?-epigallocatechin gallate (EGCG) metabolites produced by intestinal bacteria, together with tea catechins. All of the metabolites showed ACE inhibitory activities.',26323573),(5032,'mouse','(-)-Epigallocatechin gallate','Inhibitory activity of angiotensin I-converting enzyme (ACE) was examined with (�?-epigallocatechin gallate (EGCG) metabolites produced by intestinal bacteria, together with tea catechins. All of the metabolites showed ACE inhibitory activities.',26323573),(5033,'mouse','(-)-Epigallocatechin gallate','Inhibitory activity of angiotensin I-converting enzyme (ACE) was examined with (�?-epigallocatechin gallate (EGCG) metabolites produced by intestinal bacteria, together with tea catechins. All of the metabolites showed ACE inhibitory activities.',26323573),(5034,'mouse','(-)-Epigallocatechin gallate','Inhibitory activity of angiotensin I-converting enzyme (ACE) was examined with (�?-epigallocatechin gallate (EGCG) metabolites produced by intestinal bacteria, together with tea catechins. All of the metabolites showed ACE inhibitory activities.',26323573),(5035,'mouse','(-)-Epigallocatechin gallate','Inhibitory activity of angiotensin I-converting enzyme (ACE) was examined with (�?-epigallocatechin gallate (EGCG) metabolites produced by intestinal bacteria, together with tea catechins. All of the metabolites showed ACE inhibitory activities.',26323573),(5036,'mouse','(-)-Epigallocatechin gallate','Inhibitory activity of angiotensin I-converting enzyme (ACE) was examined with (�?-epigallocatechin gallate (EGCG) metabolites produced by intestinal bacteria, together with tea catechins. All of the metabolites showed ACE inhibitory activities.',26323573),(5037,'mouse','null','Firstly, studies have shown that both Treponema.spp and glycerophospholipids are  associated with brain development and diseases. Iron metabolism might provide  substantial contribution to the cross talk of Treponema.spp and glycerophospholipids .  Secondly, in this report, the Facklamia genus was negatively correlated with 4 saturated  and 15 unsaturated fatty acids. Among them, linoleic acid , eicosapentaenoic acid, arachidonic acid  and docosahexaenoic acid  are  well-known dietary nutrients for neural development and brain functions especially in infants and the elderly. Facklamia.spp was associated with invasive diseases although  the pathogenic potential is still unclear. Thirdly, the close correlation of Ruminococcus  gnavus (a bile salt hydrolase (BSH) bacterium) and ursodeoxycholic acid (UDCA), deoxycholic acid (DCA) and chenodeoxycholic acid (CDCA) were in line with the  consensus that BSH bacteria participant directly in the connection between brain and gut and that modulation of BSH activity may be of benefit to human health, Additionally, the links of UDCA, DCA, CDCA and Ruminococcus genuswith  typical intestinal, brain and neuropathic diseases were demonstrated separately by numerous reports which were circumstantial evidence of these association pairs. Fourthly,  oral administration of glutamic acid (the dominant neurotransmitter in brain) , GABA  and glycinewere proposed as new therapeutic strategies for  the regulation of both neurobehaviors and gastrointestinal tract health, although further  studies are needed to define their possible roles in the gut-brain axis. We here observed that these neurotransmitters were positively correlated with Haliscomenobacter.spp,  Pedobacter.spp and Dyadobacter.spp, suggesting new mechanisms underlying healthy  diet and antibiotic interventions.',29353471),(5038,'mouse','null','Firstly, studies have shown that both Treponema.spp and glycerophospholipids are  associated with brain development and diseases. Iron metabolism might provide  substantial contribution to the cross talk of Treponema.spp and glycerophospholipids .  Secondly, in this report, the Facklamia genus was negatively correlated with 4 saturated  and 15 unsaturated fatty acids. Among them, linoleic acid , eicosapentaenoic acid, arachidonic acid  and docosahexaenoic acid  are  well-known dietary nutrients for neural development and brain functions especially in infants and the elderly. Facklamia.spp was associated with invasive diseases although  the pathogenic potential is still unclear. Thirdly, the close correlation of Ruminococcus  gnavus (a bile salt hydrolase (BSH) bacterium) and ursodeoxycholic acid (UDCA), deoxycholic acid (DCA) and chenodeoxycholic acid (CDCA) were in line with the  consensus that BSH bacteria participant directly in the connection between brain and gut and that modulation of BSH activity may be of benefit to human health, Additionally, the links of UDCA, DCA, CDCA and Ruminococcus genuswith  typical intestinal, brain and neuropathic diseases were demonstrated separately by numerous reports which were circumstantial evidence of these association pairs. Fourthly,  oral administration of glutamic acid (the dominant neurotransmitter in brain) , GABA  and glycinewere proposed as new therapeutic strategies for  the regulation of both neurobehaviors and gastrointestinal tract health, although further  studies are needed to define their possible roles in the gut-brain axis. We here observed that these neurotransmitters were positively correlated with Haliscomenobacter.spp,  Pedobacter.spp and Dyadobacter.spp, suggesting new mechanisms underlying healthy  diet and antibiotic interventions.',29353471),(5039,'mouse','null','Firstly, studies have shown that both Treponema.spp and glycerophospholipids are  associated with brain development and diseases. Iron metabolism might provide  substantial contribution to the cross talk of Treponema.spp and glycerophospholipids .  Secondly, in this report, the Facklamia genus was negatively correlated with 4 saturated  and 15 unsaturated fatty acids. Among them, linoleic acid , eicosapentaenoic acid, arachidonic acid  and docosahexaenoic acid  are  well-known dietary nutrients for neural development and brain functions especially in infants and the elderly. Facklamia.spp was associated with invasive diseases although  the pathogenic potential is still unclear. Thirdly, the close correlation of Ruminococcus  gnavus (a bile salt hydrolase (BSH) bacterium) and ursodeoxycholic acid (UDCA), deoxycholic acid (DCA) and chenodeoxycholic acid (CDCA) were in line with the  consensus that BSH bacteria participant directly in the connection between brain and gut and that modulation of BSH activity may be of benefit to human health, Additionally, the links of UDCA, DCA, CDCA and Ruminococcus genuswith  typical intestinal, brain and neuropathic diseases were demonstrated separately by numerous reports which were circumstantial evidence of these association pairs. Fourthly,  oral administration of glutamic acid (the dominant neurotransmitter in brain) , GABA  and glycinewere proposed as new therapeutic strategies for  the regulation of both neurobehaviors and gastrointestinal tract health, although further  studies are needed to define their possible roles in the gut-brain axis. We here observed that these neurotransmitters were positively correlated with Haliscomenobacter.spp,  Pedobacter.spp and Dyadobacter.spp, suggesting new mechanisms underlying healthy  diet and antibiotic interventions.',29353471),(5047,'mouse','Vorinostat','Metabolite profiles of P. mallochii CCH01 were obviously changed by SAHA treatment. Four metabolites (1�?), including two new natural sclerotioramine derivatives, isochromophilone XIV (1) and isochromophilone XV (2), and two known compounds, sclerotioramine (3) and (+)- sclerotiorin (4), were isolated and purified from P. mallochii CCH01 treated with SAHA.',31632360),(5048,'mouse','Vorinostat','Metabolite profiles of P. mallochii CCH01 were obviously changed by SAHA treatment. Four metabolites (1�?), including two new natural sclerotioramine derivatives, isochromophilone XIV (1) and isochromophilone XV (2), and two known compounds, sclerotioramine (3) and (+)- sclerotiorin (4), were isolated and purified from P. mallochii CCH01 treated with SAHA.',31632360),(5053,'mouse','Isoxanthohumol','Recently, it was shown that the exposure to the potent hop phytoestrogen 8-prenylnaringenin (8-PN) depends on intestinal bacterial activation of isoxanthohumol (IX), but this occurs in only one-third of tested individuals. As the butyrate-producing Eubacterium limosum can produce 8-PN from IX, a probiotic strategy was applied to investigate whether 8-PN production could be increased in low 8-PN producers, thus balancing phytoestrogen exposure.',18567753),(5054,'mouse','P-Cresol','In the present study, we investigated the immunological effects of pCS derived from intestinal bacteria in mice. For this purpose, we fed mice a tyrosine-rich diet that causes the accumulation of pCS in their blood. The mice were shown to exhibit decreased Th1-driven 2, 4-dinitrofluorobenzene-induced contact hypersensitivity response. The concentration of pCS in blood was negatively correlated with the degree of the contact hypersensitivity response. In contrast, the T cell-dependent antibody response was not influenced by the accumulated pCS. We also examined the in vitro cytokine responses by T cells in the presence of pCS. The production of IFN-γ was suppressed by pCS. Further, pCS decreased the percentage of IFN-γ-producing Th1 cells. Our results suggest that intestinal bacteria-derived pCS suppressesTh1-type cellular immune responses',24161588),(5055,'mouse','P-Cresol','In the present study, we investigated the immunological effects of pCS derived from intestinal bacteria in mice. For this purpose, we fed mice a tyrosine-rich diet that causes the accumulation of pCS in their blood. The mice were shown to exhibit decreased Th1-driven 2, 4-dinitrofluorobenzene-induced contact hypersensitivity response. The concentration of pCS in blood was negatively correlated with the degree of the contact hypersensitivity response. In contrast, the T cell-dependent antibody response was not influenced by the accumulated pCS. We also examined the in vitro cytokine responses by T cells in the presence of pCS. The production of IFN-γ was suppressed by pCS. Further, pCS decreased the percentage of IFN-γ-producing Th1 cells. Our results suggest that intestinal bacteria-derived pCS suppressesTh1-type cellular immune responses',24161588),(5056,'mouse','Pectin','The purpose of this study was to elucidate the effect of pectin on the polyamine production by defined bacterial species in vivo. Germfree male Wistar rats (n = 18) were randomly assigned to one of three treatments: (i) monoassociation with Bacteroides thetaiotaomicron + fiber-free diet; (ii) diassociation with B. thetaiotaomicron + Fusobacterium varium + fiber-free diet or (iii) diassociation with B. thetaiotaomicron + F. varium + fiber-free diet + 10% pectin. The cecal contents of monoassociated rats fed fiber-free diet contained large amounts (1.51+/-0.21 micromol/dry total cecum content) of spermidine which was the major polyamine. The cecum of diassociated rats fed the fiber-free diet contained even higher concentrations of spermidine (2.53+/-0.21 micromol/dry total cecum content) and also putrescine, which was now the dominant polyamine (putrescine 0.32+/-0.28 vs. 3.01+/-0.28 micromol/dry total cecum content; monoassociation vs. diassociation).',10801923),(5057,'mouse','Pectin','The purpose of this study was to elucidate the effect of pectin on the polyamine production by defined bacterial species in vivo. Germfree male Wistar rats (n = 18) were randomly assigned to one of three treatments: (i) monoassociation with Bacteroides thetaiotaomicron + fiber-free diet; (ii) diassociation with B. thetaiotaomicron + Fusobacterium varium + fiber-free diet or (iii) diassociation with B. thetaiotaomicron + F. varium + fiber-free diet + 10% pectin. The cecal contents of monoassociated rats fed fiber-free diet contained large amounts (1.51+/-0.21 micromol/dry total cecum content) of spermidine which was the major polyamine. The cecum of diassociated rats fed the fiber-free diet contained even higher concentrations of spermidine (2.53+/-0.21 micromol/dry total cecum content) and also putrescine, which was now the dominant polyamine (putrescine 0.32+/-0.28 vs. 3.01+/-0.28 micromol/dry total cecum content; monoassociation vs. diassociation).',10801923),(5058,'mouse','Pectin','The purpose of this study was to elucidate the effect of pectin on the polyamine production by defined bacterial species in vivo. Germfree male Wistar rats (n = 18) were randomly assigned to one of three treatments: (i) monoassociation with Bacteroides thetaiotaomicron + fiber-free diet; (ii) diassociation with B. thetaiotaomicron + Fusobacterium varium + fiber-free diet or (iii) diassociation with B. thetaiotaomicron + F. varium + fiber-free diet + 10% pectin. The cecal contents of monoassociated rats fed fiber-free diet contained large amounts (1.51+/-0.21 micromol/dry total cecum content) of spermidine which was the major polyamine. The cecum of diassociated rats fed the fiber-free diet contained even higher concentrations of spermidine (2.53+/-0.21 micromol/dry total cecum content) and also putrescine, which was now the dominant polyamine (putrescine 0.32+/-0.28 vs. 3.01+/-0.28 micromol/dry total cecum content; monoassociation vs. diassociation).',10801923),(5059,'mouse','Pectin','The purpose of this study was to elucidate the effect of pectin on the polyamine production by defined bacterial species in vivo. Germfree male Wistar rats (n = 18) were randomly assigned to one of three treatments: (i) monoassociation with Bacteroides thetaiotaomicron + fiber-free diet; (ii) diassociation with B. thetaiotaomicron + Fusobacterium varium + fiber-free diet or (iii) diassociation with B. thetaiotaomicron + F. varium + fiber-free diet + 10% pectin. The cecal contents of monoassociated rats fed fiber-free diet contained large amounts (1.51+/-0.21 micromol/dry total cecum content) of spermidine which was the major polyamine. The cecum of diassociated rats fed the fiber-free diet contained even higher concentrations of spermidine (2.53+/-0.21 micromol/dry total cecum content) and also putrescine, which was now the dominant polyamine (putrescine 0.32+/-0.28 vs. 3.01+/-0.28 micromol/dry total cecum content; monoassociation vs. diassociation).',10801923),(5088,'mouse','null','Transverse aortic constriction (TAC) was performed to induce cardiac hypertrophy in Sprague-Dawley (SD) rats. We observed that TMAO levels were significantly elevated in SD rats after 6 weeks of TAC, suggesting the potential role of TMAO in regulating cardiac hypertrophy. In cultured cardiomyocytes, TMAO treatment stimulated cardiac hypertrophy, as indicated by increased cell area of cardiomyocytes and expression of hypertrophic markers including atrial natriuretic peptide (ANP) and beta-myosin heavy chain (β-MHC). Additionally, TMAO treatment induced cardiac hypertrophy and cardiac fibrosis in SD rats. Reducing TMAO synthesis by antibiotics (Abs) attenuated TAC-induced cardiac hypertrophy and fibrosis. Furthermore, pharmacological inhibition of Smad3 by SIS3 significantly reduced the expression of ANP and β-MHC, and cardiomyocyte cell size in TMAO-treated group. These data for the first time demonstrate that gut microbe-derived metabolite TMAO induces cardiac hypertrophy and fibrosis involving Smad3 signaling, suggesting that inhibition of gut microbes or generation of TMAO may become a potential target for the prevention and treatment of cardiac hypertrophy.',30068915),(5089,'mouse','null','Transverse aortic constriction (TAC) was performed to induce cardiac hypertrophy in Sprague-Dawley (SD) rats. We observed that TMAO levels were significantly elevated in SD rats after 6 weeks of TAC, suggesting the potential role of TMAO in regulating cardiac hypertrophy. In cultured cardiomyocytes, TMAO treatment stimulated cardiac hypertrophy, as indicated by increased cell area of cardiomyocytes and expression of hypertrophic markers including atrial natriuretic peptide (ANP) and beta-myosin heavy chain (β-MHC). Additionally, TMAO treatment induced cardiac hypertrophy and cardiac fibrosis in SD rats. Reducing TMAO synthesis by antibiotics (Abs) attenuated TAC-induced cardiac hypertrophy and fibrosis. Furthermore, pharmacological inhibition of Smad3 by SIS3 significantly reduced the expression of ANP and β-MHC, and cardiomyocyte cell size in TMAO-treated group. These data for the first time demonstrate that gut microbe-derived metabolite TMAO induces cardiac hypertrophy and fibrosis involving Smad3 signaling, suggesting that inhibition of gut microbes or generation of TMAO may become a potential target for the prevention and treatment of cardiac hypertrophy.',30068915),(5090,'mouse','Linoelaidic acid','Therefore, in this study, we investigated the effects of KetoA [10-oxo-12(Z)-octadecenoic acid]—a linoleic acid metabolite produced by gut lactic acid bacteria—on whole-body energy metabolism and found that dietary intake of KetoA could enhance energy expenditure in mice, thereby protecting mice from diet-induced obesity.',28754711),(5091,'mouse','Linoelaidic acid','Ucp1 expression levels were also observed to have increased after administration of KetoA.',28754711),(5092,'mouse','Linoelaidic acid','We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT\nmice.',28754711),(5093,'mouse','Linoelaidic acid','We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT\nmice.',28754711),(5094,'mouse','Linoelaidic acid','We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT\nmice.',28754711),(5095,'mouse','Linoelaidic acid','We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT\nmice.',28754711),(5096,'mouse','Linoelaidic acid','We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT\nmice.',28754711),(5097,'mouse','(Z)-10-Hydroxy-12-octadecenoic acid','Therefore, in this study, we investigated the effects of KetoA [10-oxo-12(Z)-octadecenoic acid]—a linoleic acid metabolite produced by gut lactic acid bacteria—on whole-body energy metabolism and found that dietary intake of KetoA could enhance energy expenditure in mice, thereby protecting mice from diet-induced obesity.',28754711),(5098,'mouse','(Z)-10-Hydroxy-12-octadecenoic acid','Ucp1 expression levels were also observed to have increased after administration of KetoA.',28754711),(5099,'mouse','(Z)-10-Hydroxy-12-octadecenoic acid','We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT mice.',28754711),(5100,'mouse','(Z)-10-Hydroxy-12-octadecenoic acid','We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT mice.',28754711),(5101,'mouse','(Z)-10-Hydroxy-12-octadecenoic acid','We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT mice.',28754711),(5102,'mouse','(Z)-10-Hydroxy-12-octadecenoic acid','We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT mice.',28754711),(5103,'mouse','(Z)-10-Hydroxy-12-octadecenoic acid','We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT mice.',28754711),(5104,'mouse','null','It should be noted that the observed Akkermansia muciniphila-induced decrease of Fiaf is not significant. These results are partially supported by the effects of both butyrate and propionate stimulation, leading to significant increase in Fiaf expression, while the expression of Gpr43 and Pparis significantly decreased .',25118238),(5105,'mouse','null','Akkermansia muciniphila leads to significantly decreased expression of Gpr43 and Ppar and increased expression of Hdac3 and Hdac5.',25118238),(5106,'mouse','null','It should be noted that the observed Akkermansia muciniphila-induced decrease of Fiaf is not significant. These results are partially supported by the effects of both butyrate and propionate stimulation, leading to significant increase in Fiaf expression, while the expression of Gpr43 and Pparis significantly decreased .',25118238),(5107,'mouse','null','It should be noted that the observed Akkermansia muciniphila-induced decrease of Fiaf is not significant. These results are partially supported by the effects of both butyrate and propionate stimulation, leading to significant increase in Fiaf expression, while the expression of Gpr43 and Pparis significantly decreased .',25118238),(5108,'mouse','null','Akkermansia muciniphila leads to significantly decreased expression of Gpr43 and Ppar and increased expression of Hdac3 and Hdac5.',25118238),(5109,'mouse','null','Acetate had no significant effect on the expression of Fiaf, Gpr43, Hdac5, or Ppar, while the expression of Hdac3 was significantly increased by acetate.',25118238),(5110,'mouse','null','It should be noted that the observed Akkermansia muciniphila-induced decrease of Fiaf is not significant. These results are partially supported by the effects of both butyrate and propionate stimulation, leading to significant increase in Fiaf expression, while the expression of Gpr43 and Pparis significantly decreased .',25118238),(5111,'mouse','null','Akkermansia muciniphila leads to significantly decreased expression of Gpr43 and Ppar and increased expression of Hdac3 and Hdac5.',25118238),(5112,'mouse','null','It should be noted that the observed Akkermansia muciniphila-induced decrease of Fiaf is not significant. These results are partially supported by the effects of both butyrate and propionate stimulation, leading to significant increase in Fiaf expression, while the expression of Gpr43 and Pparis significantly decreased .',25118238),(5113,'mouse','null','It should be noted that the observed Akkermansia muciniphila-induced decrease of Fiaf is not significant. These results are partially supported by the effects of both butyrate and propionate stimulation, leading to significant increase in Fiaf expression, while the expression of Gpr43 and Pparis significantly decreased .',25118238),(5114,'mouse','null','Akkermansia muciniphila leads to significantly decreased expression of Gpr43 and Ppar and increased expression of Hdac3 and Hdac5.',25118238),(5115,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5116,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5117,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5118,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5119,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5120,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5121,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5122,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5123,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5124,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5125,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5126,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5127,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5128,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5129,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5130,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5131,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5132,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5133,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5134,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5135,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5136,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5137,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5138,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5139,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5140,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5141,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5142,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5143,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5144,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5145,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5146,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5147,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5148,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5149,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5150,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5151,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5152,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5153,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5154,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5155,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5156,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5157,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5158,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5159,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5160,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5161,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5162,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5163,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5164,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5165,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5166,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5167,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5168,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5169,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5170,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5171,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5172,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5173,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5174,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5175,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5203,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5204,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5205,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5206,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5207,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5208,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5209,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5210,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5211,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5212,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5213,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5214,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5215,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5216,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5217,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5218,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5219,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5220,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5221,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5222,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5223,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5224,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5225,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5226,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5227,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5228,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5229,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5230,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5231,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5232,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5233,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5234,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5235,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5236,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5237,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5238,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5239,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5240,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5241,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5242,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5243,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5244,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5245,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5246,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5247,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5248,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5249,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5250,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5252,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5253,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5254,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5255,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5256,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5257,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5258,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5260,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5262,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5264,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5265,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5266,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5267,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5268,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5269,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5270,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5272,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5273,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5274,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5275,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5276,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5277,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5278,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5280,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5281,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5282,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5283,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5284,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5285,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5286,'mouse','null','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5304,'mouse','null','The results clearly showed that the high dose of PIPA could improve the intestinal movement and alleviate constipation symptoms in AD induced constipation mice.',30846982),(5305,'mouse','null','The results showed that the AD groups had lower levels of ACh and 5-HT in colonic tissue and that the H-PIPA dose induced an increase in both colonic ACh and 5-HT but without significant differences from those in the AD group.',30846982),(5306,'mouse','null','The results showed that the AD groups had lower levels of ACh and 5-HT in colonic tissue and that the H-PIPA dose induced an increase in both colonic ACh and 5-HT but without significant differences from those in the AD group.',30846982),(5307,'mouse','null','Butyrate is a short-chain fatty acid derived from the metabolism of indigestible carbohydrates by the gut microbiota. Butyrate contributes to gut homeostasis, but it may also control inflammatory responses and host physiology in other tissues. Butyrate inhibits histone deacetylases, thereby affecting gene transcription, and also signals through the metabolite-sensing G protein receptor (GPR)109a.',31366236),(5308,'mouse','null','The protective effect of butyrate was further confirmed in situ by enhanced expression of WT-1 (Fig. 1D),podocin, and podoplanin (Supplemental Fig. S3). In ad_x0002_dition, analysis of glomeruli ultrastructure revealed a well-preserved podocytelayer surrounding the GBM with presence of slit diaphragm and reduced FP effacement in butyrate-treated mice at early and later time points (Fig.1E). Over 60 d, all butyrate-treated mice survived ADRinduced nephropathy, compared with only about 20% of nontreated mice (Fig. 1F).',31366236),(5309,'mouse','null','We also assessed the gene expression levels of important fibrotic markers such as Tgf-b, Fsp1, ActaII, col4a1, and the enzymes involved in matrix remodeling, Mmp9 and Timp1 (Fig. 3C). Butyrate-treated mice showed lower levels of fibrotic markers both at d 7 and at d 14 compared with nontreated mice.',31366236),(5310,'mouse','null','We also assessed the gene expression levels of important fibrotic markers such as Tgf-b, Fsp1, ActaII, col4a1, and the enzymes involved in matrix remodeling, Mmp9 and Timp1 (Fig. 3C). Butyrate-treated mice showed lower levels of fibrotic markers both at d 7 and at d 14 compared with nontreated mice.',31366236),(5311,'mouse','null','We also assessed the gene expression levels of important fibrotic markers such as Tgf-b, Fsp1, ActaII, col4a1, and the enzymes involved in matrix remodeling, Mmp9 and Timp1 (Fig. 3C). Butyrate-treated mice showed lower levels of fibrotic markers both at d 7 and at d 14 compared with nontreated mice.',31366236),(5312,'mouse','null','We also assessed the gene expression levels of important fibrotic markers such as Tgf-b, Fsp1, ActaII, col4a1, and the enzymes involved in matrix remodeling, Mmp9 and Timp1 (Fig. 3C). Butyrate-treated mice showed lower levels of fibrotic markers both at d 7 and at d 14 compared with nontreated mice.',31366236),(5313,'mouse','null','We also assessed the gene expression levels of important fibrotic markers such as Tgf-b, Fsp1, ActaII, col4a1, and the enzymes involved in matrix remodeling, Mmp9 and Timp1 (Fig. 3C). Butyrate-treated mice showed lower levels of fibrotic markers both at d 7 and at d 14 compared with nontreated mice.',31366236),(5314,'mouse','null','Renal inflammation is a common finding in ADR nephropathy. Injured resident kidney cells release chemokines that promote the recruitment ofmacrophages, which play a pivotal role as disease progresses. CD68 staining in renal cortex showed, at early stages of the disease, a slightly positive staining in nontreated mice, whereby we identified several foci of infiltrating CD68-positive cells at d 14 surrounding the glomeruli (Fig. 4A). Butyrate treatment significantly reduced the infiltration of CD68-positive cells, as well as gene expression of chemokines such as ccl2 and ccl3 and important proinflammatory cytokines such as Tnf-a and Il-1b in renal tissue (Fig. 4B). Protein levels for the proinflammatory cytokines in renal tissue were also significantly reduced by butyrate treatment (Fig. 4C).',31366236),(5315,'mouse','null','Renal inflammation is a common finding in ADR nephropathy. Injured resident kidney cells release chemokines that promote the recruitment ofmacrophages, which play a pivotal role as disease progresses. CD68 staining in renal cortex showed, at early stages of the disease, a slightly positive staining in nontreated mice, whereby we identified several foci of infiltrating CD68-positive cells at d 14 surrounding the glomeruli (Fig. 4A). Butyrate treatment significantly reduced the infiltration of CD68-positive cells, as well as gene expression of chemokines such as ccl2 and ccl3 and important proinflammatory cytokines such as Tnf-a and Il-1b in renal tissue (Fig. 4B). Protein levels for the proinflammatory cytokines in renal tissue were also significantly reduced by butyrate treatment (Fig. 4C).',31366236),(5316,'mouse','null','Renal inflammation is a common finding in ADR nephropathy. Injured resident kidney cells release chemokines that promote the recruitment ofmacrophages, which play a pivotal role as disease progresses. CD68 staining in renal cortex showed, at early stages of the disease, a slightly positive staining in nontreated mice, whereby we identified several foci of infiltrating CD68-positive cells at d 14 surrounding the glomeruli (Fig. 4A). Butyrate treatment significantly reduced the infiltration of CD68-positive cells, as well as gene expression of chemokines such as ccl2 and ccl3 and important proinflammatory cytokines such as Tnf-a and Il-1b in renal tissue (Fig. 4B). Protein levels for the proinflammatory cytokines in renal tissue were also significantly reduced by butyrate treatment (Fig. 4C).',31366236),(5317,'mouse','null','Renal inflammation is a common finding in ADR nephropathy. Injured resident kidney cells release chemokines that promote the recruitment ofmacrophages, which play a pivotal role as disease progresses. CD68 staining in renal cortex showed, at early stages of the disease, a slightly positive staining in nontreated mice, whereby we identified several foci of infiltrating CD68-positive cells at d 14 surrounding the glomeruli (Fig. 4A). Butyrate treatment significantly reduced the infiltration of CD68-positive cells, as well as gene expression of chemokines such as ccl2 and ccl3 and important proinflammatory cytokines such as Tnf-a and Il-1b in renal tissue (Fig. 4B). Protein levels for the proinflammatory cytokines in renal tissue were also significantly reduced by butyrate treatment (Fig. 4C).',31366236),(5318,'mouse','null','In terestingly, we identified 5 genes that had their expression switched after treatment with butyrate: Csrp2bp,Aurkc, Dot1L, Kat5, and Hdac6.',31366236),(5319,'mouse','null','In terestingly, we identified 5 genes that had their expression switched after treatment with butyrate: Csrp2bp,Aurkc, Dot1L, Kat5, and Hdac6.',31366236),(5320,'mouse','null','In terestingly, we identified 5 genes that had their expression switched after treatment with butyrate: Csrp2bp,Aurkc, Dot1L, Kat5, and Hdac6.',31366236),(5321,'mouse','null','In terestingly, we identified 5 genes that had their expression switched after treatment with butyrate: Csrp2bp,Aurkc, Dot1L, Kat5, and Hdac6.',31366236),(5322,'mouse','null','In terestingly, we identified 5 genes that had their expression switched after treatment with butyrate: Csrp2bp,Aurkc, Dot1L, Kat5, and Hdac6.',31366236),(5323,'mouse','null','To test this hypothesis, we evaluated the histone modifications H3K27Ac and H3K4me3, which are typically regarded as markers that lead to transcriptional activation, andH3K9me3,which correlates to transcriptional repression (Fig. 5F). ADR led to significant reductions of the activation marker H3K27Ac at the promoter region of both Synpo and Nphs1 genes, which was recovered in the presence of butyrate. In parallel, the increase of the repressive marker H3K9me3 that is due to ADR at the promoter region on both Synpo and Kirrel genes was significantly reduced in the presence of butyrate. Indeed, podocytes grown in vitro showed preserved expression of synaptopodin (Fig. 5G) after treatmentwith butyrate in the presence of ADR.',31366236),(5324,'mouse','null','To test this hypothesis, we evaluated the histone modifications H3K27Ac and H3K4me3, which are typically regarded as markers that lead to transcriptional activation, andH3K9me3,which correlates to transcriptional repression (Fig. 5F). ADR led to significant reductions of the activation marker H3K27Ac at the promoter region of both Synpo and Nphs1 genes, which was recovered in the presence of butyrate. In parallel, the increase of the repressive marker H3K9me3 that is due to ADR at the promoter region on both Synpo and Kirrel genes was significantly reduced in the presence of butyrate. Indeed, podocytes grown in vitro showed preserved expression of synaptopodin (Fig. 5G) after treatmentwith butyrate in the presence of ADR.',31366236),(5325,'mouse','null','To test this hypothesis, we evaluated the histone modifications H3K27Ac and H3K4me3, which are typically regarded as markers that lead to transcriptional activation, andH3K9me3,which correlates to transcriptional repression (Fig. 5F). ADR led to significant reductions of the activation marker H3K27Ac at the promoter region of both Synpo and Nphs1 genes, which was recovered in the presence of butyrate. In parallel, the increase of the repressive marker H3K9me3 that is due to ADR at the promoter region on both Synpo and Kirrel genes was significantly reduced in the presence of butyrate. Indeed, podocytes grown in vitro showed preserved expression of synaptopodin (Fig. 5G) after treatmentwith butyrate in the presence of ADR.',31366236),(5326,'mouse','null','Podocytes grown in vitro in the presence of ADR exhibited significantly increased TRPC5 and TRPC6 gene transcriptions, which were abolished when podocytes were incubated with buty rate.',31366236),(5327,'mouse','null','Podocytes grown in vitro in the presence of ADR exhibited significantly increased TRPC5 and TRPC6 gene transcriptions, which were abolished when podocytes were incubated with buty rate.',31366236),(5328,'mouse','null','Here, the presence of ADR in podocytes grown in vitro decreased the expression levels of RHOA, RAC1, and CDC42, whereas treatmentwith butyrate significantly recovered the levels of both RAC1 and CDC42 but not RHOA.',31366236),(5329,'mouse','null','Here, the presence of ADR in podocytes grown in vitro decreased the expression levels of RHOA, RAC1, and CDC42, whereas treatmentwith butyrate significantly recovered the levels of both RAC1 and CDC42 but not RHOA.',31366236),(5330,'mouse','Ellagic acid','Thus, the gut bacterial metabolite UA increases miR-10a-5p levels thereby downregulating Orai1/STIM1/STIM2 expression, store operated Ca2+ entry, and proliferation of murine CD4+ T cells.',31417547),(5331,'mouse','Ellagic acid','Thus, the gut bacterial metabolite UA increases miR-10a-5p levels thereby downregulating Orai1/STIM1/STIM2 expression, store operated Ca2+ entry, and proliferation of murine CD4+ T cells.',31417547),(5332,'mouse','Ellagic acid','Thus, the gut bacterial metabolite UA increases miR-10a-5p levels thereby downregulating Orai1/STIM1/STIM2 expression, store operated Ca2+ entry, and proliferation of murine CD4+ T cells.',31417547),(5333,'mouse','Ellagic acid','Thus, the present observations uncover a novel action of UA, i.e., the upregulation of miR-10a-5p with subsequent downregulation of store operated Ca2+ influx in CD4+ T cells.',31417547),(5337,'mouse','Cholesterol','Altered substrate oxidation and fecal fat excretion in LHFD-fed GF mice is linked to decreased hepatic Cyp7a1 and Nr1h4 expression.',27900259),(5338,'mouse','Cholesterol','Altered substrate oxidation and fecal fat excretion in LHFD-fed GF mice is linked to decreased hepatic Cyp7a1 and Nr1h4 expression.',27900259),(5339,'mouse','Linoleic acid','10-Hydroxy-cis-12-octadecenoic acid (HYA), a gut microbial metabolite of linoleic acid, suppressed TNF and dextran sulfate sodium-induced changes in the expression of TJ-related molecules, occludin, zonula occludens-1, and myosin light chain kinase. HYA also suppressed the expression of TNF receptor 2 (TNFR2) mRNA and protein expression in Caco-2 cells and colonic tissue. In addition, HYA suppressed the protein expression of TNFR2 in murine intestinal epithelial cells. Furthermore, HYA significantly up-regulated G protein-coupled receptor (GPR) 40 expression in Caco-2 cells.',25505251),(5340,'mouse','Linoleic acid','10-Hydroxy-cis-12-octadecenoic acid (HYA), a gut microbial metabolite of linoleic acid, suppressed TNF and dextran sulfate sodium-induced changes in the expression of TJ-related molecules, occludin, zonula occludens-1, and myosin light chain kinase. HYA also suppressed the expression of TNF receptor 2 (TNFR2) mRNA and protein expression in Caco-2 cells and colonic tissue. In addition, HYA suppressed the protein expression of TNFR2 in murine intestinal epithelial cells. Furthermore, HYA significantly up-regulated G protein-coupled receptor (GPR) 40 expression in Caco-2 cells.',25505251),(5341,'mouse','Linoleic acid','10-Hydroxy-cis-12-octadecenoic acid (HYA), a gut microbial metabolite of linoleic acid, suppressed TNF and dextran sulfate sodium-induced changes in the expression of TJ-related molecules, occludin, zonula occludens-1, and myosin light chain kinase. HYA also suppressed the expression of TNF receptor 2 (TNFR2) mRNA and protein expression in Caco-2 cells and colonic tissue. In addition, HYA suppressed the protein expression of TNFR2 in murine intestinal epithelial cells. Furthermore, HYA significantly up-regulated G protein-coupled receptor (GPR) 40 expression in Caco-2 cells.',25505251),(5342,'mouse','Linoleic acid','IFN-TNF-induced an increase in FD-4 permeability;however, the pretreatment of HYA and linoleic acid signififcantly suppressed this increase (Fig. 2C). In contrast, the pretreatment of HYB did not alter IFN-TNF-induced changes in TER and FD-4 permeability. As shown in Fig. 2D,IFN-TNF-induced a marked increase in the secretion of IL-8, which is a major pro-inflammatory cytokine. However,the pretreatment of HYA significantly suppressed IL-8 secretion.',25505251),(5343,'mouse','Linoleic acid','Pretreatment of HYA, but not HYB, significantly restored IFN-TNF-induced changes in the expression of occludin, ZO-1, claudin-1, and MLCK.',25505251),(5344,'mouse','Linoleic acid','Pretreatment of HYA, but not HYB, significantly restored IFN-TNF-induced changes in the expression of occludin, ZO-1, claudin-1, and MLCK.',25505251),(5345,'mouse','Linoleic acid','Pretreatment of HYA, but not HYB, significantly restored IFN-TNF-induced changes in the expression of occludin, ZO-1, claudin-1, and MLCK.',25505251),(5346,'mouse','Linoleic acid','Pretreatment of HYA, but not HYB, significantly restored IFN-TNF-induced changes in the expression of occludin, ZO-1, claudin-1, and MLCK.',25505251),(5347,'mouse','Linoleic acid','The pretreatment of HYA suppressed IFN-induced up-regulation of TNFR2 (but not TNFR1), and it subsequently decreased the ratio of phospho-INF-B p65 expression induced by cytokines (Fig. 3, B and C). In contrast, pretreatment of HYB did not show the same inhibitory effects as HYA.',25505251),(5348,'mouse','Linoleic acid','It was noteworthy that the pretreatment of HYA augmented the mRNA expression of GPR40 and GPR43 but not GPR120, PPAR, and GPR41 in Caco-2 cells (Fig. 4A). However, the pretreatment of HYB did not up-regulate GPR40 and GPR43 expression (Fig. 4A).',25505251),(5349,'mouse','null','The expression levels of IL�? (Fig. 5A and B) and its receptors CXCR1 (Fig. 5A and C) and CXCR2 (Fig. 5A and D) were significantly increased in the non‑infarct LV of MI + VEH rats compared with the LV of SHAM + VEH rats; however, the expression of these proteins was normalized by DMB treatment.',31180562),(5350,'mouse','null','The expression levels of IL�? (Fig. 5A and B) and its receptors CXCR1 (Fig. 5A and C) and CXCR2 (Fig. 5A and D) were significantly increased in the non‑infarct LV of MI + VEH rats compared with the LV of SHAM + VEH rats; however, the expression of these proteins was normalized by DMB treatment.',31180562),(5351,'mouse','null','The expression levels of IL�? (Fig. 5A and B) and its receptors CXCR1 (Fig. 5A and C) and CXCR2 (Fig. 5A and D) were significantly increased in the non‑infarct LV of MI + VEH rats compared with the LV of SHAM + VEH rats; however, the expression of these proteins was normalized by DMB treatment.',31180562),(5352,'mouse','null','Accordingly, in livers from TLR2-defcient mice, the expression of SASP factors, such as IL1β,  Groα, and IL6, was significantly reduced in HSCs, suggesting that TLR2-mediated signaling induced the expression of SASP factors (Fig. 2F). Notably, however, the presence of p21 and 53BP1 foci was still observed in activated HSCs from TLR2-deficient mice (Fig. 2F). The majority of senescent \nsignals are detected in the HSCs but not in HCC cells in the tumor region (Supplementary Fig. S2). Taken together, these results suggest that TLR2-mediated signaling is important for the upregulation of SASP factors in senescent HSCs but not for the induction of senescent cell-cycle arrest.',28202625),(5353,'mouse','null','Accordingly, in livers from TLR2-defcient mice, the expression of SASP factors, such as IL1β,  Groα, and IL6, was significantly reduced in HSCs, suggesting that TLR2-mediated signaling induced the expression of SASP factors (Fig. 2F). Notably, however, the presence of p21 and 53BP1 foci was still observed in activated HSCs from TLR2-deficient mice (Fig. 2F). The majority of senescent \nsignals are detected in the HSCs but not in HCC cells in the tumor region (Supplementary Fig. S2). Taken together, these results suggest that TLR2-mediated signaling is important for the upregulation of SASP factors in senescent HSCs but not for the induction of senescent cell-cycle arrest.',28202625),(5354,'mouse','null','Accordingly, in livers from TLR2-defcient mice, the expression of SASP factors, such as IL1β,  Groα, and IL6, was significantly reduced in HSCs, suggesting that TLR2-mediated signaling induced the expression of SASP factors (Fig. 2F). Notably, however, the presence of p21 and 53BP1 foci was still observed in activated HSCs from TLR2-deficient mice (Fig. 2F). The majority of senescent \nsignals are detected in the HSCs but not in HCC cells in the tumor region (Supplementary Fig. S2). Taken together, these results suggest that TLR2-mediated signaling is important for the upregulation of SASP factors in senescent HSCs but not for the induction of senescent cell-cycle arrest.',28202625),(5355,'mouse','null','Consistent with our previous report, the expression of the senescence inducers p21 (encoded by Cdkn1a) and p16 (encoded by Cdkn2a) was upregulated by DCA treatment (Fig. 3B). Notably, the expression levels of TLR2 and SASP factors, such as IL1β, Groα, and IL6, were significantly enhanced by LTA in DCA-induced senescent HSCs (Fig. 3C).',28202625),(5356,'mouse','null','Consistent with our previous report, the expression of the senescence inducers p21 (encoded by Cdkn1a) and p16 (encoded by Cdkn2a) was upregulated by DCA treatment (Fig. 3B). Notably, the expression levels of TLR2 and SASP factors, such as IL1β, Groα, and IL6, were significantly enhanced by LTA in DCA-induced senescent HSCs (Fig. 3C).',28202625),(5357,'mouse','null','The expression of COX2, but not COX1, in DCA-treated primary HSCs was significantly induced by increasing amounts of LTA (Fig. 4C), and the COX2 expression was dependent on TLR2 (Supplementary \nFig. S3). Indeed, we confirmed that TLR2 was expressed in HSCs and this expression was enhanced by LTA alone or DCA plus LTA treatment (Supplementary Fig. S8A).',28202625),(5358,'mouse','null','The production of IFNγ and TNFα, which are known to activate antitumor immune responses and inhibit cancer cell growth, was significantly and specifically attenuated by PGE2.',28202625),(5359,'mouse','null','The production of IFNγ and TNFα, which are known to activate antitumor immune responses and inhibit cancer cell growth, was significantly and specifically attenuated by PGE2.',28202625),(5360,'mouse','null','In contrast, the production of IL10, a potent anti-inflammatory cytokine, and IL6, a known tumor-promoting cytokine, in immune cells was slightly but significantly enhanced (Supplementary Fig. S8D).',28202625),(5361,'mouse','null','In contrast, the production of IL10, a potent anti-inflammatory cytokine, and IL6, a known tumor-promoting cytokine, in immune cells was slightly but significantly enhanced (Supplementary Fig. S8D).',28202625),(5362,'mouse','null','PGE2 is known to be recognized by four types of receptors: PTGER1, PTGER2, PTGER3, and PTGER4 (37, 38). In our mouse model, the expression of PTGER4, but not other PGE2 receptors, was significantly upregulated in tumor tissues (Fig. 5C),',28202625),(5363,'mouse','null','Indole 3-propionic acid (IPA), in the context of indole, is a ligand for PXR in vivo, and IPA downregulated enterocyte TNF-a while it upregulated junctional protein-coding mRNAs. PXR_deficient mice showed a distinctly ‘‘leaky’’gut physiology coupled with upregulation of the Toll-like receptor (TLR) signaling pathway.',25065623),(5364,'mouse','null','Indole 3-propionic acid (IPA), in the context of indole, is a ligand for PXR in vivo, and IPA downregulated enterocyte TNF-a while it upregulated junctional protein-coding mRNAs. PXR_deficient mice showed a distinctly ‘‘leaky’’gut physiology coupled with upregulation of the Toll-like receptor (TLR) signaling pathway.',25065623),(5365,'mouse','null','Indole 3-propionic acid (IPA), in the context of indole, is a ligand for PXR in vivo, and IPA downregulated enterocyte TNF-a while it upregulated junctional protein-coding mRNAs. PXR_deficient mice showed a distinctly ‘‘leaky’’gut physiology coupled with upregulation of the Toll-like receptor (TLR) signaling pathway.',25065623),(5366,'mouse','null','For critical validation of the ex vivo experiments demonstrating IPA effects on junctional regulators, we coadministered C. sporogenes to germ-free mice in the presence or absence of\nL-tryptophan (Figure 1D). We verified that C. sporogenes inoculation led to production of IPA in vivo (thus, it was assumed that indoles were present) (Figure 1E). Germ-free mice exposed to\nC. sporogenes had a significant reduction in FITC-dextran recovery from the serum and this was further reduced in the presence of L-tryptophan dosing (Figure 1F). The mice intestinal mucosa\nexposed to C. sporogenes demonstrated significant induction of PXR target genes (Mdr1, Cyp3a11, Ugt1a1), which was further augmented in the presence of L-tryptophan (Figure 1G).',25065623),(5367,'mouse','null','For critical validation of the ex vivo experiments demonstrating IPA effects on junctional regulators, we coadministered C. sporogenes to germ-free mice in the presence or absence of\nL-tryptophan (Figure 1D). We verified that C. sporogenes inoculation led to production of IPA in vivo (thus, it was assumed that indoles were present) (Figure 1E). Germ-free mice exposed to\nC. sporogenes had a significant reduction in FITC-dextran recovery from the serum and this was further reduced in the presence of L-tryptophan dosing (Figure 1F). The mice intestinal mucosa\nexposed to C. sporogenes demonstrated significant induction of PXR target genes (Mdr1, Cyp3a11, Ugt1a1), which was further augmented in the presence of L-tryptophan (Figure 1G).',25065623),(5368,'mouse','null','For critical validation of the ex vivo experiments demonstrating IPA effects on junctional regulators, we coadministered C. sporogenes to germ-free mice in the presence or absence of\nL-tryptophan (Figure 1D). We verified that C. sporogenes inoculation led to production of IPA in vivo (thus, it was assumed that indoles were present) (Figure 1E). Germ-free mice exposed to\nC. sporogenes had a significant reduction in FITC-dextran recovery from the serum and this was further reduced in the presence of L-tryptophan dosing (Figure 1F). The mice intestinal mucosa\nexposed to C. sporogenes demonstrated significant induction of PXR target genes (Mdr1, Cyp3a11, Ugt1a1), which was further augmented in the presence of L-tryptophan (Figure 1G).',25065623),(5369,'mouse','N-Trimethyllysine','We found TMAVA to bind and inhibit BBOX, reducing synthesis of carnitine.We found the intestinal bacteria Enterococcus faecalis and Pseudomonas aeruginosa to metabolize trimethyllysine to TMAVA.',32105727),(5370,'mouse','N-Trimethyllysine','We found TMAVA to bind and inhibit BBOX, reducing synthesis of carnitine.We found the intestinal bacteria Enterococcus faecalis and Pseudomonas aeruginosa to metabolize trimethyllysine to TMAVA.',32105727),(5371,'mouse','null','In addition, Parabacteroides_sp_HGS0025 positively correlated with butyrate, IgM, IL-10, and TNF-α products in intestine and mice blood, respectively.',32014483),(5375,'mouse','beta-D-Glucopyranose, 1,3,6-tris(3,4,5-trihydroxybenzoate)','We found that UB treatment resulted in significant p62 accumulation.',31996327),(5376,'mouse','beta-D-Glucopyranose, 1,3,6-tris(3,4,5-trihydroxybenzoate)','The nuclear Nrf2 level was increased with UB treatment.',31996327),(5377,'mouse','beta-D-Glucopyranose, 1,3,6-tris(3,4,5-trihydroxybenzoate)','UB treatment significantly enhanced nuclear Nrf2 level and its downstream genes, HO-1, NQO1 and GSTP1 in the rat heart subjected to IR.',31996327),(5378,'mouse','beta-D-Glucopyranose, 1,3,6-tris(3,4,5-trihydroxybenzoate)','UB treatment significantly enhanced nuclear Nrf2 level and its downstream genes, HO-1, NQO1 and GSTP1 in the rat heart subjected to IR.',31996327),(5379,'mouse','beta-D-Glucopyranose, 1,3,6-tris(3,4,5-trihydroxybenzoate)','UB treatment significantly enhanced nuclear Nrf2 level and its downstream genes, HO-1, NQO1 and GSTP1 in the rat heart subjected to IR.',31996327),(5380,'mouse','null','The amino acid metabolite, N-delta-acetylornithine, was notably increased by B. longum rice bran fermentation when compared to non-fermented rice bran in food, colon, and blood.',31965839),(5381,'mouse','Mogroside V','Only 11-oxo-mogrol could increase the mRNA expression of both Camk2α and Calbindin genes',31900523),(5382,'mouse','Mogroside V','Only 11-oxo-mogrol could increase the mRNA expression of both Camk2α and Calbindin genes',31900523),(5383,'mouse','null','Treatment of naive T cells under the Treg-cell-polarizing conditions with butyrate enhanced histone H3 acetylation in the promoter and conserved non-coding sequence regions of the Foxp3 locus, suggesting a possible mechanism for how microbial-derived butyrate regulates the differentiation of Treg cells.',24226770),(5384,'mouse','null','Treatment of naive T cells under the Treg-cell-polarizing conditions with butyrate enhanced histone H3 acetylation in the promoter and conserved non-coding sequence regions of the Foxp3 locus, suggesting a possible mechanism for how microbial-derived butyrate regulates the differentiation of Treg cells.',24226770),(5385,'mouse','Tryptophan','Bt and B. ovatus produced indole in both media conditions.',27916477),(5386,'mouse','Tryptophan','Bt and B. ovatus produced indole in both media conditions.',27916477),(5387,'mouse','null','We found that butyrate enhanced phosphorylated-STAT3, mTOR, and ERK1/2 as evidenced by Western blots.',30177845),(5388,'mouse','null','We found that butyrate enhanced phosphorylated-STAT3, mTOR, and ERK1/2 as evidenced by Western blots.',30177845),(5389,'mouse','null','Mechanistically, SCFAs activate Th1 cell STAT3 and mTOR, and consequently upregulate transcription factor B lymphocyte-induced maturation protein 1 (Blimp-1), which mediates SCFA-induction of IL-10.',30177845),(5390,'mouse','null','Mechanistically, SCFAs activate Th1 cell STAT3 and mTOR, and consequently upregulate transcription factor B lymphocyte-induced maturation protein 1 (Blimp-1), which mediates SCFA-induction of IL-10.',30177845),(5391,'mouse','null','SCFA Propionate Induced Sympathetic Activation via GPR41-Gβγ in Sympathetic Neurons.',21518883),(5392,'mouse','Linolenic acid','Treatments of BMDCs with the fatty acids significantly increased mRNA expression of Arg1 in a dose-dependent manner in comparison with that of IL-4 treatment alone.',28904023),(5393,'mouse','Linolenic acid','Treatments of BMDCs with the fatty acids significantly increased mRNA expression of Arg1 in a dose-dependent manner in comparison with that of IL-4 treatment alone.',28904023),(5399,'mouse','null','AM attenuates ALS in mice by increasing NAM levels',31330533),(5400,'mouse','null','AM attenuates ALS in mice by increasing NAM levels',31330533),(5406,'mouse','null','Butyrate treatment significantly decreased expression of the CD11b.',31835282),(5407,'mouse','Tryptophan','Here, we show that the biological effects of tryptamine are mediated through the 5-HT4 receptor (5-HT4R), a G-protein-coupled receptor (GPCR) uniquely expressed in the colonic epithelium.',29902441),(5408,'mouse','Tryptophan','Here, we show that the biological effects of tryptamine are mediated through the 5-HT4 receptor (5-HT4R), a G-protein-coupled receptor (GPCR) uniquely expressed in the colonic epithelium.',29902441),(5409,'mouse','null','UroA or UAS03 treatment significantly reduced the LPS-induced increase in serum IL-6 and TNF-α levels (Fig. 1c). These results suggest that UAS03 is a potent structural analogue of UroA with increased anti-inflammatory activities.',30626868),(5410,'mouse','null','UroA or UAS03 treatment significantly reduced the LPS-induced increase in serum IL-6 and TNF-α levels (Fig. 1c). These results suggest that UAS03 is a potent structural analogue of UroA with increased anti-inflammatory activities.',30626868),(5411,'mouse','null','UroA/UAS03 treatment attenuates TNBS-induced colitis in mice. Colitis was induced by intrarectal administration of TNBS (2.5 mg/mouse) in C57BL/6 (8 week age old, n = 5/group) mice. Mice were orally treated with vehicle or UroA (20 mg/kg) or UAS03 (20 mg/kg body weight) every 12 h post-TNBS instillation for 60 h and the experiment terminated at 72 h. Representative data from one of three independent experiments is shown. a Percent body weight loss (No TNBS- Solid black line; Veh + TNBS- Solid red line; UroA + TNBS- Solid blue line; UAS03 + TNBS- Solid purple line). b disease activity index, c intestinal permeability, d colon lengths were measured. e Gross morphological changes of colon, f ratio of colon weight/length, g colonic myeloperoxidase (MPO) levels, h serum IL-6, TNF-α, CXCL1, and IL-1β levels, i microphotographs of hematoxylin and eosin (H&E) stained sections of colons and inflammation scores are shown. Scale bar indicates 300 μm. j Cldn4 expression in the colons of these mice (n = 3) was measured by immunoblots and quantified. Statistical analysis was performed (unpaired t-test) using Graphpad Prism software. Error bars, ±SEM ***p < 0.001; **p < 0.01\n*p < 0.05. Source Data are provided as a Source Data File.',30626868),(5412,'mouse','null','UroA/UAS03 treatment attenuates TNBS-induced colitis in mice. Colitis was induced by intrarectal administration of TNBS (2.5 mg/mouse) in C57BL/6 (8 week age old, n = 5/group) mice. Mice were orally treated with vehicle or UroA (20 mg/kg) or UAS03 (20 mg/kg body weight) every 12 h post-TNBS instillation for 60 h and the experiment terminated at 72 h. Representative data from one of three independent experiments is shown. a Percent body weight loss (No TNBS- Solid black line; Veh + TNBS- Solid red line; UroA + TNBS- Solid blue line; UAS03 + TNBS- Solid purple line). b disease activity index, c intestinal permeability, d colon lengths were measured. e Gross morphological changes of colon, f ratio of colon weight/length, g colonic myeloperoxidase (MPO) levels, h serum IL-6, TNF-α, CXCL1, and IL-1β levels, i microphotographs of hematoxylin and eosin (H&E) stained sections of colons and inflammation scores are shown. Scale bar indicates 300 μm. j Cldn4 expression in the colons of these mice (n = 3) was measured by immunoblots and quantified. Statistical analysis was performed (unpaired t-test) using Graphpad Prism software. Error bars, ±SEM ***p < 0.001; **p < 0.01\n*p < 0.05. Source Data are provided as a Source Data File.',30626868),(5416,'mouse','null','Interestingly, metabolites produced by gut microbiota, including propionate, butyrate, and the secondary bile acid deoxycolate directly induced expression of tryptophan hydroxylase 1 (Tph1).',28961041),(5417,'mouse','null','Interestingly, metabolites produced by gut microbiota, including propionate, butyrate, and the secondary bile acid deoxycolate directly induced expression of tryptophan hydroxylase 1 (Tph1).',28961041),(5418,'mouse','null','Interestingly, metabolites produced by gut microbiota, including propionate, butyrate, and the secondary bile acid deoxycolate directly induced expression of tryptophan hydroxylase 1 (Tph1).',28961041),(5426,'mouse','null','CB0313.1 upregulates butyrate production.',28765642),(5432,'mouse','Berberine','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5433,'mouse','Berberine','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5434,'mouse','Berberine','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5435,'mouse','Berberine','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5436,'mouse','Berberine','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5437,'mouse','Berberine','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5438,'mouse','Berberine','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5439,'mouse','Berberine','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5440,'mouse','Berberine','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5441,'mouse','Berberine','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5442,'mouse','Berberine','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5443,'mouse','Berberine','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5444,'mouse','Berberine','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5445,'mouse','Berberine','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5446,'mouse','Berberine','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5447,'mouse','Berberine','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5448,'mouse','Berberine','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5449,'mouse','Berberine','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5450,'mouse','Berberine','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5451,'mouse','Berberine','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5452,'mouse','Berberine','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5453,'mouse','Berberine','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5454,'mouse','Berberine','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5455,'mouse','Berberine','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5456,'mouse','Berberine','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5457,'mouse','Berberine','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5458,'mouse','Berberine','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5459,'mouse','Berberine','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5460,'mouse','Berberine','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5461,'mouse','Berberine','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5462,'mouse','Berberine','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5463,'mouse','Berberine','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5464,'mouse','Berberine','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5465,'mouse','Berberine','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5466,'mouse','Berberine','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5467,'mouse','Berberine','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5468,'mouse','Berberine','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5469,'mouse','Berberine','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5470,'mouse','Berberine','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5471,'mouse','Berberine','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5472,'mouse','Berberine','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5473,'mouse','Berberine','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5474,'mouse','Berberine','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5475,'mouse','Berberine','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5476,'mouse','Berberine','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5477,'mouse','Berberine','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5478,'mouse','Berberine','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5479,'mouse','Berberine','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5480,'mouse','Berberine','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5481,'mouse','Berberine','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5482,'mouse','Berberine','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5483,'mouse','Berberine','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5484,'mouse','Berberine','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5485,'mouse','Berberine','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5486,'mouse','Berberine','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5487,'mouse','Berberine','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5488,'mouse','Berberine','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5489,'mouse','Berberine','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5490,'mouse','Berberine','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5491,'mouse','Berberine','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5492,'mouse','null','In line with these clinical findings, VSL#3 and CLA significantly ameliorated gross pathology in colon, MLN and spleen, colonic histopathology such as leukocyte infiltration  and mucosal thickening , and colonic mRNA expression such as TNF-a and MCP-1 in comparison to untreated control mice with DSS colitis.',22363592),(5493,'mouse','null','In line with these clinical findings, VSL#3 and CLA significantly ameliorated gross pathology in colon, MLN and spleen, colonic histopathology such as leukocyte infiltration  and mucosal thickening , and colonic mRNA expression such as TNF-a and MCP-1 in comparison to untreated control mice with DSS colitis.',22363592),(5498,'mouse','null','Here, we tested the hypothesis that the gut microbiota could induce maturation of the adult ENS through release of 5-HT and activation of 5-HT4 receptors.',29866843),(5568,'mouse','Daidzein','We isolated, for the first time, an anaerobic gram-positive rod-shaped strain capable of producing equol from daidzein.',17046543),(5569,'mouse','Genistein','We isolated a strictly anaerobic bacterium (Mt1B8) from the mouse intestine which converted daidzein via dihydrodaidzein to equol as well as genistein via dihydrogenistein to 5-hydroxy-equol.',18539813),(5570,'mouse','Daidzein','We isolated a strictly anaerobic bacterium (Mt1B8) from the mouse intestine which converted daidzein via dihydrodaidzein to equol as well as genistein via dihydrogenistein to 5-hydroxy-equol.',18539813),(5571,'mouse','Dihydrodaidzein','Strain Mt1B8 also transformed dihydrodaidzein and dihydrogenistein to equol and 5-hydroxy-equol, respectively.',18539813),(5572,'mouse','Dihydrogenistein','Strain Mt1B8 also transformed dihydrodaidzein and dihydrogenistein to equol and 5-hydroxy-equol, respectively.',18539813),(5573,'mouse','3-Dehydroshikimate','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.',32728212),(5574,'mouse','3-Dehydroshikimate','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.',32728212),(5575,'mouse','3-Dehydroshikimate','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.',32728212),(5576,'mouse','3-Dehydroshikimate','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.',32728212),(5577,'mouse','3-Dehydroshikimate','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.',32728212),(5578,'mouse','3-Dehydroshikimate','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.',32728212),(5579,'mouse','3-Dehydroshikimate','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.',32728212),(5580,'mouse','3-Dehydroshikimate','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.',32728212),(5581,'mouse','3-Dehydroshikimate','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.',32728212),(5582,'mouse','3-Dehydroshikimate','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.',32728212),(5583,'mouse','3-Dehydroshikimate','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.',32728212),(5584,'mouse','3-Dehydroshikimate','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.',32728212),(5585,'mouse','Phytic acid','Taken together, these data demonstrate bacterial-intrinsic phytate metabolism and provoke the hypothesis that breakdown of phytate by microbiota promotes intestinal HDAC activity.',32731255),(5586,'mouse','null','Further, colonization of germ-free (GF) mice with butyrate producing bacteria,Faecalibacterium prausnitzii, inhibited IEC-intrinsic HDAC activity.',32731255),(5587,'mouse','null','CNV mice exhibited significantly higher IP3 in both the intestinal lumen (Fig. 1e) and colonic IECs (Fig. 1f), supporting that microbiota promote increased IP3. To assess the effect of IP3 on epithelial HDAC3 activity, HDAC3 was purified by immunoprecipitation from IECs and incubated with IP3. IP3 increased endogenous HDAC3 activity (Fig. 1g), supporting that IP3 directly activates HDAC3 in IECs.',32731255),(5588,'mouse','null','Although less potent than IP3, IP4 also increased HDAC activity in IECs (Extended Data Fig. 5f), suggesting that other inositol phosphates simultaneously generated by phytate metabolism may provide synergistic effects.',32731255),(5589,'mouse','Glutamic acid','Glutamate, the agonist of NMDAR and a major excitatory neurotransmitter in the mammalian central nervous system (CNS) [6], is intracellularly located and extensively distributed in the brain.',32290475),(5590,'mouse','Glutamic acid','Glutamate, the agonist of NMDAR and a major excitatory neurotransmitter in the mammalian central nervous system (CNS) [6], is intracellularly located and extensively distributed in the brain.',32290475),(5591,'mouse','Glutamic acid','Non-pathogenic bacteria with glutamate racemase including Corynebacterium glutamicum, Brevibacterium lactofermentum, Brevibacterium avium [121], Mycobacterium smegmatis[122], and Bacillus subtilis[123] convert l-glutamate to d-glutamate.',32290475),(5592,'mouse','Glutamic acid','Non-pathogenic bacteria with glutamate racemase including Corynebacterium glutamicum, Brevibacterium lactofermentum, Brevibacterium avium [121], Mycobacterium smegmatis[122], and Bacillus subtilis[124] convert l-glutamate to d-glutamate.',32290475),(5593,'mouse','null','Coryneform bacteria are mainly used to produce glutamate in industry. LAB strains such as Lactobacillus plantarum, Lactococcus lactis, and Lactobacillus paracasei can synthesize glutamate [113,114]. ',32290475),(5594,'mouse','null','Coryneform bacteria are mainly used to produce glutamate in industry. LAB strains such as Lactobacillus plantarum, Lactococcus lactis, and Lactobacillus paracasei can synthesize glutamate [113,114]. ',32290475),(5595,'mouse','null','Coryneform bacteria are mainly used to produce glutamate in industry. LAB strains such as Lactobacillus plantarum, Lactococcus lactis, and Lactobacillus paracasei can synthesize glutamate [113,114]. ',32290475),(5596,'mouse','null','Coryneform bacteria are mainly used to produce glutamate in industry. LAB strains such as Lactobacillus plantarum, Lactococcus lactis, and Lactobacillus paracasei can synthesize glutamate [113,114]. ',32290475),(5597,'mouse','null','Furthermore, benzoate, an inhibitor of d-amino acid oxidases (DAAO), can increase d-serine,thereby enhancing NMDAR activation [125,126].',32290475),(5598,'mouse','Bile acid','Reconstitution with a single bacterial species, Clostridium scindens, or its derived metabolite, the secondary bile acid deoxycholic acid, can restore pDC- and MyD88-dependent type I IFN responses to restrict systemic CHIKV infection and transmission back to vector mosquitoes. ',32668198),(5599,'mouse','Bile acid','Reconstitution with a single bacterial species, Clostridium scindens, or its derived metabolite, the secondary bile acid deoxycholic acid, can restore pDC- and MyD88-dependent type I IFN responses to restrict systemic CHIKV infection and transmission back to vector mosquitoes. ',32668198),(5600,'mouse','Tryptophan','HSD significantly reduces intestinal Lactobacillus murinus in hypertensive FVB/N mice, while L murinus can metabolize trypto phan in the diet to produce indole, thereby inhibiting T lymphocyte differentiation into Th17 cells, alleviating inflammation and lowering BP.',32078445),(5601,'mouse','null','Moreover, the culture supernatant of B fra gilis YCH46 could significantly reduce the corticoste rone level and reverse the abnormal expressions of Mr, Cyp11a1, Cyp11b1, Hsd11b1, and Hsd11b2 in high salt–fed mice colon tissues (Figure 5E through 5J). Namely, B fragilis effectively counteracted HSD effects on intesti nal corticosterone metabolism and dramatically reduced intestinal corticosterone production level.',32078445),(5602,'mouse','null','Moreover, the culture supernatant of B fra gilis YCH46 could significantly reduce the corticoste rone level and reverse the abnormal expressions of Mr, Cyp11a1, Cyp11b1, Hsd11b1, and Hsd11b2 in high salt–fed mice colon tissues (Figure 5E through 5J). Namely, B fragilis effectively counteracted HSD effects on intesti nal corticosterone metabolism and dramatically reduced intestinal corticosterone production level.',32078445),(5603,'mouse','null','Moreover, the culture supernatant of B fra gilis YCH46 could significantly reduce the corticoste rone level and reverse the abnormal expressions of Mr, Cyp11a1, Cyp11b1, Hsd11b1, and Hsd11b2 in high salt–fed mice colon tissues (Figure 5E through 5J). Namely, B fragilis effectively counteracted HSD effects on intesti nal corticosterone metabolism and dramatically reduced intestinal corticosterone production level.',32078445),(5604,'mouse','null','Moreover, the culture supernatant of B fra gilis YCH46 could significantly reduce the corticoste rone level and reverse the abnormal expressions of Mr, Cyp11a1, Cyp11b1, Hsd11b1, and Hsd11b2 in high salt–fed mice colon tissues (Figure 5E through 5J). Namely, B fragilis effectively counteracted HSD effects on intesti nal corticosterone metabolism and dramatically reduced intestinal corticosterone production level.',32078445),(5605,'mouse','null','Here, we elucidate a micro biome-dependent pathway by which SG increases CA7S production. We show that a microbial metabolite,lithocholic acid (LCA), is increased in murine portal veins post-SG and by activating the vitamin D receptor, induces hepatic mSult2A1/hSULT2A expression to drive CA7S production. An SG-induced shift in the micro biome increases gut expression of the bile acid transporters Asbt and Osta, which in turn facilitate selective transport of LCA across the gut epithelium. Cecal microbiota transplant from SG animals is sufficient to recreate the pathway in germ-free (GF) animals. \nand GLP-1 secretion, causally connecting a microbial metabolite with the improvement of diabetic phe notypes.',33434516),(5606,'mouse','null','Here, we elucidate a micro biome-dependent pathway by which SG increases CA7S production. We show that a microbial metabolite,lithocholic acid (LCA), is increased in murine portal veins post-SG and by activating the vitamin D receptor, induces hepatic mSult2A1/hSULT2A expression to drive CA7S production. An SG-induced shift in the micro biome increases gut expression of the bile acid transporters Asbt and Osta, which in turn facilitate selective transport of LCA across the gut epithelium. Cecal microbiota transplant from SG animals is sufficient to recreate the pathway in germ-free (GF) animals. \nand GLP-1 secretion, causally connecting a microbial metabolite with the improvement of diabetic phe notypes.',33434516),(5607,'mouse','null','We determined that CA7S is a potent agonist of the G-protein-coupled recep tor TGR5 (GPBAR1) and induces GLP-1 secretion. When administered to diet-induced obese (DIO) mice, CA7S in creases insulin secretion and induces systemic glucose clearance.',33434516),(5608,'mouse','null','We determined that CA7S is a potent agonist of the G-protein-coupled recep tor TGR5 (GPBAR1) and induces GLP-1 secretion. When administered to diet-induced obese (DIO) mice, CA7S in creases insulin secretion and induces systemic glucose clearance.',33434516),(5609,'mouse','null','Consistent with our in vitro results showing that LCA inhibits expression of Asbt and Osta, SG-CMT mice exhibited significantly higher levels of Asbt and Osta expression in the distal ileum compared with sham-CMT (Figure 7E).',33434516),(5610,'mouse','null','Consistent with our in vitro results showing that LCA inhibits expression of Asbt and Osta, SG-CMT mice exhibited significantly higher levels of Asbt and Osta expression in the distal ileum compared with sham-CMT (Figure 7E).',33434516),(5611,'mouse','Trimethylamine','Concentrations of pro-inflammatory cytokines com monly associated with neuroinflammation (IL1-β, IL 6, and TNF-α) and the anti-inflammatory cytokine IL-10 were measured in whole-brain lysates from young and old mice and are presented in Fig. 3a. Whole-brain concentrations of pro-inflammatory cytokines were all ~ 50�?00% higher in old vs. young mice (all p < 0.05).Old mice also tended to have (presumably compensato ry) higher brain IL-10 compared to young mice (p =0.07).',32862276),(5612,'mouse','Trimethylamine','Concentrations of pro-inflammatory cytokines com monly associated with neuroinflammation (IL1-β, IL 6, and TNF-α) and the anti-inflammatory cytokine IL-10 were measured in whole-brain lysates from young and old mice and are presented in Fig. 3a. Whole-brain concentrations of pro-inflammatory cytokines were all ~ 50�?00% higher in old vs. young mice (all p < 0.05).Old mice also tended to have (presumably compensato ry) higher brain IL-10 compared to young mice (p =0.07).',32862276),(5613,'mouse','Trimethylamine','Concentrations of pro-inflammatory cytokines com monly associated with neuroinflammation (IL1-β, IL 6, and TNF-α) and the anti-inflammatory cytokine IL-10 were measured in whole-brain lysates from young and old mice and are presented in Fig. 3a. Whole-brain concentrations of pro-inflammatory cytokines were all ~ 50�?00% higher in old vs. young mice (all p < 0.05).Old mice also tended to have (presumably compensato ry) higher brain IL-10 compared to young mice (p =0.07).',32862276),(5614,'mouse','Trimethylamine','Concentrations of pro-inflammatory cytokines com monly associated with neuroinflammation (IL1-β, IL 6, and TNF-α) and the anti-inflammatory cytokine IL-10 were measured in whole-brain lysates from young and old mice and are presented in Fig. 3a. Whole-brain concentrations of pro-inflammatory cytokines were all ~ 50�?00% higher in old vs. young mice (all p < 0.05).Old mice also tended to have (presumably compensato ry) higher brain IL-10 compared to young mice (p =0.07).',32862276),(5615,'mouse','Trimethylamine','Next, we measured the pro-inflammatory tran scription factor nuclear factor kappa B (NF-κB), which is a key regulator of pro-inflammatory cytokine signal ing. Although abundance of phosphorylated (i.e., acti vated) NF-κB was similar between brains from young and old mice (p = 0.49), old mice had ~ 50% higher abundance of total NF-κB (p < 0.001; Fig. 3b), suggest ing a greater capacity to upregulate pro-inflammatory cytokine production.',32862276),(5616,'mouse','Trimethylamine','Finally, LCN2, an established marker of reactive astrocytes involved in the innate immune response to bacteria [5, 63], was higher in brain lysates from old compared to young mice (p < 0.05;Fig. 3c).',32862276),(5617,'mouse','null','In this study, we observed that in mice exposed to HFD or without intestinal epithelial cell AhR expression, the efect of diet was more signifcant than the absence of AhR on the fecal microbiome composition. On the other hand, the absence of AhR had a more pronounced efect on the fecal metabolome than diet. Tryptophan metabolites were among those that were signifcantly altered in the metabolome, and integrated microbiome and metabolome analysis predicted three taxa—unclas sifed Clostridiales, unclassifed Desulfovibrionaceae, and Akkermansia—as key contributors to the synthesis and/or degradation of tryptophan metabolites. Our study highlights the use of multi-omic analysis to investigate the relationship between microbiome and metabolome and suggests possible taxa that can be targeted to manip ulate the microbiome for CRC prevention.',33256731),(5618,'mouse','null','In this study, we observed that in mice exposed to HFD or without intestinal epithelial cell AhR expression, the efect of diet was more signifcant than the absence of AhR on the fecal microbiome composition. On the other hand, the absence of AhR had a more pronounced efect on the fecal metabolome than diet. Tryptophan metabolites were among those that were signifcantly altered in the metabolome, and integrated microbiome and metabolome analysis predicted three taxa—unclas sifed Clostridiales, unclassifed Desulfovibrionaceae, and Akkermansia—as key contributors to the synthesis and/or degradation of tryptophan metabolites. Our study highlights the use of multi-omic analysis to investigate the relationship between microbiome and metabolome and suggests possible taxa that can be targeted to manip ulate the microbiome for CRC prevention.',33256731),(5619,'mouse','null','In this study, we observed that in mice exposed to HFD or without intestinal epithelial cell AhR expression, the efect of diet was more signifcant than the absence of AhR on the fecal microbiome composition. On the other hand, the absence of AhR had a more pronounced efect on the fecal metabolome than diet. Tryptophan metabolites were among those that were signifcantly altered in the metabolome, and integrated microbiome and metabolome analysis predicted three taxa—unclas sifed Clostridiales, unclassifed Desulfovibrionaceae, and Akkermansia—as key contributors to the synthesis and/or degradation of tryptophan metabolites. Our study highlights the use of multi-omic analysis to investigate the relationship between microbiome and metabolome and suggests possible taxa that can be targeted to manip ulate the microbiome for CRC prevention.',33256731),(5620,'mouse','null','mRNA levels of TNF-α and IL-1β in the WAT of ob/ob mice were, respectively, 17- and 4-fold higher than those of C57BL/6 J mice (Fig. 2A). Te iNOS and COX-2 in WAT were also, respectively, 5.4- and 2.2-fold higher than those of C57BL/6 J mice (Fig. 2A). Hepatic gene expression of IL-6 was signifcantly increased in ob/ob mice compared with C57BL/6 J mice although TNF-α and IL-1β were not markedly increased (Fig. 2B).',32296105),(5621,'mouse','null','mRNA levels of TNF-α and IL-1β in the WAT of ob/ob mice were, respectively, 17- and 4-fold higher than those of C57BL/6 J mice (Fig. 2A). Te iNOS and COX-2 in WAT were also, respectively, 5.4- and 2.2-fold higher than those of C57BL/6 J mice (Fig. 2A). Hepatic gene expression of IL-6 was signifcantly increased in ob/ob mice compared with C57BL/6 J mice although TNF-α and IL-1β were not markedly increased (Fig. 2B).',32296105),(5622,'mouse','null','mRNA levels of TNF-α and IL-1β in the WAT of ob/ob mice were, respectively, 17- and 4-fold higher than those of C57BL/6 J mice (Fig. 2A). Te iNOS and COX-2 in WAT were also, respectively, 5.4- and 2.2-fold higher than those of C57BL/6 J mice (Fig. 2A). Hepatic gene expression of IL-6 was signifcantly increased in ob/ob mice compared with C57BL/6 J mice although TNF-α and IL-1β were not markedly increased (Fig. 2B).',32296105),(5623,'mouse','null','mRNA levels of TNF-α and IL-1β in the WAT of ob/ob mice were, respectively, 17- and 4-fold higher than those of C57BL/6 J mice (Fig. 2A). Te iNOS and COX-2 in WAT were also, respectively, 5.4- and 2.2-fold higher than those of C57BL/6 J mice (Fig. 2A). Hepatic gene expression of IL-6 was signifcantly increased in ob/ob mice compared with C57BL/6 J mice although TNF-α and IL-1β were not markedly increased (Fig. 2B).',32296105),(5624,'mouse','null','mRNA levels of TNF-α and IL-1β in the WAT of ob/ob mice were, respectively, 17- and 4-fold higher than those of C57BL/6 J mice (Fig. 2A). Te iNOS and COX-2 in WAT were also, respectively, 5.4- and 2.2-fold higher than those of C57BL/6 J mice (Fig. 2A). Hepatic gene expression of IL-6 was signifcantly increased in ob/ob mice compared with C57BL/6 J mice although TNF-α and IL-1β were not markedly increased (Fig. 2B).',32296105),(5625,'mouse','null','As a result, stercobilin signifcantly induced NF-κB reporter gene activities in a dose-dependent manner (Fig. 6A). Additionally, ster cobilin signifcantly induced expression of several infammation-related genes, including TNF-α, IL-1β, IL-6, iNOS, and COX-2 (Fig. 6B).',32296105),(5626,'mouse','Baicalin','Compared to the wild type mice, the levels of infammatory cytokines such as IL-1β, IL-2, IL-6, IL-10, G-CSF, and GM-CSF are signifcantly increased in the intestine and colon of APCmin/+ mice in the model group. ',31650468),(5627,'mouse','Baicalin','Compared to the wild type mice, the levels of infammatory cytokines such as IL-1β, IL-2, IL-6, IL-10, G-CSF, and GM-CSF are signifcantly increased in the intestine and colon of APCmin/+ mice in the model group. ',31650468),(5628,'mouse','Baicalin','Compared to the wild type mice, the levels of infammatory cytokines such as IL-1β, IL-2, IL-6, IL-10, G-CSF, and GM-CSF are signifcantly increased in the intestine and colon of APCmin/+ mice in the model group. ',31650468),(5629,'mouse','Baicalin','Compared to the wild type mice, the levels of infammatory cytokines such as IL-1β, IL-2, IL-6, IL-10, G-CSF, and GM-CSF are signifcantly increased in the intestine and colon of APCmin/+ mice in the model group. ',31650468),(5630,'mouse','Baicalin','Compared to the wild type mice, the levels of infammatory cytokines such as IL-1β, IL-2, IL-6, IL-10, G-CSF, and GM-CSF are signifcantly increased in the intestine and colon of APCmin/+ mice in the model group. ',31650468),(5631,'mouse','Baicalin','Compared to the wild type mice, the levels of infammatory cytokines such as IL-1β, IL-2, IL-6, IL-10, G-CSF, and GM-CSF are signifcantly increased in the intestine and colon of APCmin/+ mice in the model group. ',31650468),(5632,'mouse','Tryptophan','Interestingly, in the same mice, indole induced a dramatic increase of the expression of the adrenomedullary Pnmt gene, which is involved in catecholamine biosynthesis.',32569990),(5633,'mouse','null','To illuminate the molecular mechanism underpinning the hyperlipidemia-ameliorating role of DNJ, we further examined the impact of DNJ on hepatic genes that are involved in lipid synthesis (SREBP1c, FAS, ACC, GPAT, SCD, ChREBP, SREBP2, and HMGR), transport (FATP, FABP, LXR, and CD36) and oxidation (CPT1, PPAR , AMPK, and ACO). Compared with the normal control mice, mice that received gavage of DNJ alone showed decreased transcription of lipogenic genes (Fig. 2 and Table 1), with no significant effects on genes participating in lipid transport (except CD36) and oxidation (see Fig. S2 in the supple mental material, suggesting that DNJ may prevent hyperlipidemia via inhibiting lipid biosynthesis.',33024047),(5634,'mouse','null','To illuminate the molecular mechanism underpinning the hyperlipidemia-ameliorating role of DNJ, we further examined the impact of DNJ on hepatic genes that are involved in lipid synthesis (SREBP1c, FAS, ACC, GPAT, SCD, ChREBP, SREBP2, and HMGR), transport (FATP, FABP, LXR, and CD36) and oxidation (CPT1, PPAR , AMPK, and ACO). Compared with the normal control mice, mice that received gavage of DNJ alone showed decreased transcription of lipogenic genes (Fig. 2 and Table 1), with no significant effects on genes participating in lipid transport (except CD36) and oxidation (see Fig. S2 in the supple mental material, suggesting that DNJ may prevent hyperlipidemia via inhibiting lipid biosynthesis.',33024047),(5635,'mouse','null','To illuminate the molecular mechanism underpinning the hyperlipidemia-ameliorating role of DNJ, we further examined the impact of DNJ on hepatic genes that are involved in lipid synthesis (SREBP1c, FAS, ACC, GPAT, SCD, ChREBP, SREBP2, and HMGR), transport (FATP, FABP, LXR, and CD36) and oxidation (CPT1, PPAR , AMPK, and ACO). Compared with the normal control mice, mice that received gavage of DNJ alone showed decreased transcription of lipogenic genes (Fig. 2 and Table 1), with no significant effects on genes participating in lipid transport (except CD36) and oxidation (see Fig. S2 in the supple mental material, suggesting that DNJ may prevent hyperlipidemia via inhibiting lipid biosynthesis.',33024047),(5636,'mouse','null','To illuminate the molecular mechanism underpinning the hyperlipidemia-ameliorating role of DNJ, we further examined the impact of DNJ on hepatic genes that are involved in lipid synthesis (SREBP1c, FAS, ACC, GPAT, SCD, ChREBP, SREBP2, and HMGR), transport (FATP, FABP, LXR, and CD36) and oxidation (CPT1, PPAR , AMPK, and ACO). Compared with the normal control mice, mice that received gavage of DNJ alone showed decreased transcription of lipogenic genes (Fig. 2 and Table 1), with no significant effects on genes participating in lipid transport (except CD36) and oxidation (see Fig. S2 in the supple mental material, suggesting that DNJ may prevent hyperlipidemia via inhibiting lipid biosynthesis.',33024047),(5637,'mouse','null','To illuminate the molecular mechanism underpinning the hyperlipidemia-ameliorating role of DNJ, we further examined the impact of DNJ on hepatic genes that are involved in lipid synthesis (SREBP1c, FAS, ACC, GPAT, SCD, ChREBP, SREBP2, and HMGR), transport (FATP, FABP, LXR, and CD36) and oxidation (CPT1, PPAR , AMPK, and ACO). Compared with the normal control mice, mice that received gavage of DNJ alone showed decreased transcription of lipogenic genes (Fig. 2 and Table 1), with no significant effects on genes participating in lipid transport (except CD36) and oxidation (see Fig. S2 in the supple mental material, suggesting that DNJ may prevent hyperlipidemia via inhibiting lipid biosynthesis.',33024047),(5638,'mouse','null','To illuminate the molecular mechanism underpinning the hyperlipidemia-ameliorating role of DNJ, we further examined the impact of DNJ on hepatic genes that are involved in lipid synthesis (SREBP1c, FAS, ACC, GPAT, SCD, ChREBP, SREBP2, and HMGR), transport (FATP, FABP, LXR, and CD36) and oxidation (CPT1, PPAR , AMPK, and ACO). Compared with the normal control mice, mice that received gavage of DNJ alone showed decreased transcription of lipogenic genes (Fig. 2 and Table 1), with no significant effects on genes participating in lipid transport (except CD36) and oxidation (see Fig. S2 in the supple mental material, suggesting that DNJ may prevent hyperlipidemia via inhibiting lipid biosynthesis.',33024047),(5639,'mouse','null','To illuminate the molecular mechanism underpinning the hyperlipidemia-ameliorating role of DNJ, we further examined the impact of DNJ on hepatic genes that are involved in lipid synthesis (SREBP1c, FAS, ACC, GPAT, SCD, ChREBP, SREBP2, and HMGR), transport (FATP, FABP, LXR, and CD36) and oxidation (CPT1, PPAR , AMPK, and ACO). Compared with the normal control mice, mice that received gavage of DNJ alone showed decreased transcription of lipogenic genes (Fig. 2 and Table 1), with no significant effects on genes participating in lipid transport (except CD36) and oxidation (see Fig. S2 in the supple mental material, suggesting that DNJ may prevent hyperlipidemia via inhibiting lipid biosynthesis.',33024047),(5640,'mouse','null','To illuminate the molecular mechanism underpinning the hyperlipidemia-ameliorating role of DNJ, we further examined the impact of DNJ on hepatic genes that are involved in lipid synthesis (SREBP1c, FAS, ACC, GPAT, SCD, ChREBP, SREBP2, and HMGR), transport (FATP, FABP, LXR, and CD36) and oxidation (CPT1, PPAR , AMPK, and ACO). Compared with the normal control mice, mice that received gavage of DNJ alone showed decreased transcription of lipogenic genes (Fig. 2 and Table 1), with no significant effects on genes participating in lipid transport (except CD36) and oxidation (see Fig. S2 in the supple mental material, suggesting that DNJ may prevent hyperlipidemia via inhibiting lipid biosynthesis.',33024047),(5641,'mouse','null','Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).',33024047),(5642,'mouse','null','Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).',33024047),(5643,'mouse','null','Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).',33024047),(5644,'mouse','null','Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).',33024047),(5645,'mouse','null','Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).',33024047),(5646,'mouse','null','Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).',33024047),(5647,'mouse','null','Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).',33024047),(5648,'mouse','null','Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).',33024047),(5649,'mouse','Stachyose','In the low-dose and high dose groups, elevated concentrations of acetic acid and propionic acid were produced, indicating that stachyose intake promoted the proliferation of acetic acid- and propionic acid-producing microorganisms.',33233042),(5650,'mouse','Stachyose','In the low-dose and high dose groups, elevated concentrations of acetic acid and propionic acid were produced, indicating that stachyose intake promoted the proliferation of acetic acid- and propionic acid-producing microorganisms.',33233042),(5651,'mouse','Stachyose','In the low-dose and high dose groups, elevated concentrations of acetic acid and propionic acid were produced, indicating that stachyose intake promoted the proliferation of acetic acid- and propionic acid-producing microorganisms.',33233042),(5652,'mouse','Stachyose','In the low-dose and high dose groups, elevated concentrations of acetic acid and propionic acid were produced, indicating that stachyose intake promoted the proliferation of acetic acid- and propionic acid-producing microorganisms.',33233042),(5653,'mouse','null','The expression of IL-1, IL-17 and TNF-α were higher in the intestinal tract of the germ free mice that had not received a human flora fecal transplant than in those that had received a fecal transplant, and the expression of IL-1, IL 17 and TNF-α decreased significantly after the fecal transplant (Fig. 4c, p < 0.01).',33233042),(5654,'mouse','null','The expression of IL-1, IL-17 and TNF-α were higher in the intestinal tract of the germ free mice that had not received a human flora fecal transplant than in those that had received a fecal transplant, and the expression of IL-1, IL 17 and TNF-α decreased significantly after the fecal transplant (Fig. 4c, p < 0.01).',33233042),(5655,'mouse','null','The expression of IL-1, IL-17 and TNF-α were higher in the intestinal tract of the germ free mice that had not received a human flora fecal transplant than in those that had received a fecal transplant, and the expression of IL-1, IL 17 and TNF-α decreased significantly after the fecal transplant (Fig. 4c, p < 0.01).',33233042),(5656,'mouse','null','Compared with the germ free group, the expression of NF-κB decreased significantly in the colon after transplantation of intestinal flora (p < 0.05), and the intake of stachyose downregulated the expression of NF-κB (p < 0.05).',33233042),(5657,'mouse','null','As shown in Fig. 4h and i, the expression of occludin and ZO-1 in the colon tissue of germ-free mice significantly increased after human fecal transplantation (p < 0.05).',33233042),(5658,'mouse','null','As shown in Fig. 4h and i, the expression of occludin and ZO-1 in the colon tissue of germ-free mice significantly increased after human fecal transplantation (p < 0.05).',33233042);
/*!40000 ALTER TABLE `gut_microbiota_metabolite_generation_results` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `metabolite`
--

DROP TABLE IF EXISTS `metabolite`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `metabolite` (
  `metabolite_name` longtext COLLATE utf8mb4_bin NOT NULL,
  `pubchem_id` longtext COLLATE utf8mb4_bin NOT NULL,
  `hmdb_id` longtext COLLATE utf8mb4_bin NOT NULL,
  `chebi_id` longtext COLLATE utf8mb4_bin NOT NULL
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `metabolite`
--

LOCK TABLES `metabolite` WRITE;
/*!40000 ALTER TABLE `metabolite` DISABLE KEYS */;
INSERT INTO `metabolite` VALUES ('Acetate','175','null','CHEBI:30089'),('Butyrate','104775','null','CHEBI:17968'),('Luteolin','5280445','HMDB0005800','CHEBI:15864'),('Deoxycholic acid','222528','HMDB0000626','CHEBI:28834'),('Lithocholic acid','9903','HMDB0000761','CHEBI:16325'),('(4S)-4-[(2E)-Oct-2-enoyloxy]-4-(trimethylazaniumyl)butanoate','53481667','null','CHEBI:176711'),('Bile acid','439520','null','CHEBI:22868'),('Propionate','104745','null','CHEBI:17272'),('Phenolic acid','null','null','CHEBI:166890'),('Kaempferol-3-sorphoroside','null','null','null'),('3,4-Dihydroxyphenylacetic acid','547','HMDB0001336','CHEBI:41941'),('Dihydrocaffeic acid','348154','HMDB0000423','CHEBI:48400'),('4-Hydroxyphenylacetic acid','127','HMDB0000020','CHEBI:18101'),('3-(4-Hydroxyphenyl)propionic acid','10394','HMDB0002199','CHEBI:32980'),('Phloroglucinol','359','HMDB0013675','CHEBI:16204'),('Indoxyl sulfate','10258','HMDB0000682','CHEBI:43355'),('p-Cresol sulfate','4615423','HMDB0011635','CHEBI:82914'),('Phenylacetylglutamine','92258','HMDB0006344','CHEBI:17884'),('Dihydrodaidzein','176907','HMDB0005760','CHEBI:75842'),('Equol','91469','HMDB0002209','CHEBI:34741'),('2,3-Butanediol','262','HMDB0003156','CHEBI:62064'),('2-Acetoxypropanoic acid','79041','null','null'),('Acetoin','179','HMDB0003243','CHEBI:15688'),('2,3-Butanedione','650','HMDB0003407','CHEBI:16583'),('Trimethylamine','1146','HMDB0000906','CHEBI:18139'),('3-Hydroxy-4-methoxybenzenepropanoic acid','2752054','HMDB0131138','null'),('Hesperetin dihydrochalcone','147608','HMDB0128062','null'),('Succinate','160419','null','CHEBI:30031'),('O-Desmethylangolensin','89472','HMDB0004629','CHEBI:88902'),('1-(3,4-Dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl)-2-propanol','21722175','HMDB0127722','null'),('3,4-diHPP-2-ol','null','null','null'),('3,4-diHPVL','null','null','null'),('Trimethylamine oxide','1145','HMDB0000925','CHEBI:15724'),('D-Lactic acid','61503','HMDB0001311','CHEBI:42111'),('3-Indolepropionic acid','3744','HMDB0002302','CHEBI:43580'),('Caffeic acid','689043','HMDB0001964','CHEBI:16433'),('3-(3-Hydroxyphenyl)propanoic acid','91','HMDB0000375','CHEBI:1427'),('Dehydroxychlorogenic acid','null','null','null'),('Dihydrochlorogenic acid','null','null','null'),('2,3-Dihydroxypropyl (E)-3-(3,4-dihydroxyphenyl)prop-2-enoate','5315606','null','null'),('Dihydrocaffeoyl-glycerol','null','null','null'),('3-Hydroxyphenethyl alcohol','83404','null','null'),('Phenylacetic acid','999','HMDB0000209','CHEBI:30745'),('Isoquercitrin','5280804','HMDB0037362','CHEBI:68352'),('5-OH-Equol','9795113','null','null'),('2-(4-Hydroxyphenyl)propionic acid, (2S)-','6971268','null','null'),('8-Prenylnaringenin','480764','null','CHEBI:50207'),('Alanine','5950','HMDB0062251','CHEBI:16977'),('Leucine','6106','HMDB0000687','CHEBI:15603'),('l-Isoleucine','6306','HMDB0000172','CHEBI:17191'),('Glycine','750','HMDB0000123','CHEBI:15428'),('Proline','145742','HMDB0000162','CHEBI:17203'),('Serine','5951','HMDB0000187','CHEBI:17115'),('3-Hydroxybenzoic acid','7420','HMDB0002466','CHEBI:30764'),('4-Hydroxybenzoic acid','135','HMDB0000500','CHEBI:30763'),('Sedoheptulose','5459879','null','CHEBI:16802'),('Citric acid','311','HMDB0000094','CHEBI:30769'),('Tricarballylic acid','14925','HMDB0031193','CHEBI:45969'),('Pipecolic acid','849','HMDB0000070','CHEBI:17964'),('Malic acid','525','null','CHEBI:6650'),('Tartaric acid','875','HMDB0062642','CHEBI:15674'),('Creatine','586','HMDB0000064','CHEBI:16919'),('2-Imino-1-methylimidazolidin-4-one','588','HMDB0000562','CHEBI:16737'),('Glycocholic acid','10140','HMDB0000138','CHEBI:17687'),('3-Phenylpropionic acid','107','HMDB0000764','CHEBI:28631'),('4-Pyridoxic acid','6723','HMDB0000017','CHEBI:17405'),('Enterodiol','115089','HMDB0005056','CHEBI:81556'),('Aglycone','null','null','CHEBI:166892'),('(2R)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl)-propan-2-ol','null','null','null'),('(2R)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl)propan-2-ol','null','null','null'),('Lactate','91435','null','CHEBI:24996'),('Daidzein','5281708','HMDB0003312','CHEBI:28197'),('Levodopa','6047','HMDB0000181','CHEBI:15765'),('Sulfate','1117','HMDB0001448','CHEBI:16189'),('Baohuoside I','5488822','null','CHEBI:82619'),('Icaritin','5318980','HMDB0139407','null'),('Formate','283','null','CHEBI:15740'),('L-Lactate','5460161','null','CHEBI:16651'),('Ethanol','702','HMDB0000108','CHEBI:16236'),('Platycodin D','162859','null','CHEBI:70436'),('Chrysin','5281607','HMDB0036619','CHEBI:75095'),('Apigenin','5280443','HMDB0002124','CHEBI:18388'),('3,9-Dihydroxybenzo[c]chromen-6-one','10376459','HMDB0142418','null'),('Pyruvate','107735','null','CHEBI:15361'),('Acetyl phosphate(2-)','4183249','null','CHEBI:22191'),('3-Methyloxindole','150923','null','CHEBI:17397'),('5-Hydroxyindole-3-acetic acid','1826','HMDB0000763','CHEBI:27823'),('Indole-3-acrylic acid','5375048','HMDB0000734','CHEBI:132244'),('Indole-3-carboxylic acid','69867','HMDB0003320','CHEBI:24809'),('Indole','798','HMDB0000738','CHEBI:16881'),('Oxindole','321710','HMDB0061918','CHEBI:31697'),('3-Methylindole','6736','HMDB0000466','CHEBI:9171'),('Kynurenic acid','3845','HMDB0000715','CHEBI:18344'),('Histidine','6274','HMDB0000177','CHEBI:15971'),('L-Cysteine','5862','HMDB0000574','CHEBI:17561'),('Cystine','595','null','CHEBI:17376'),('Glutathione','124886','HMDB0000125','CHEBI:16856'),('Asparagine','6267','HMDB0000168','CHEBI:17196'),('DL-Alanine','602','HMDB0062251','CHEBI:16449'),('2-Hydroxy-3-(5-hydroxy-1H-indol-3-yl)propanoic acid','192215','null','null'),('Indole-3-lactic acid','92904','null','CHEBI:24813'),('2-Hydroxy-3-(4-hydroxyphenyl)propanoic acid','9378','HMDB0000755','CHEBI:17385'),('3-(3,4-Dihydroxyphenyl)-2-hydroxypropanoic acid','439435','HMDB0003503','CHEBI:17807'),('Nicotinic acid','938','HMDB0001488','CHEBI:15940'),('(R)-3-Hydroxybutyrate','6971058','HMDB0000011','CHEBI:10983'),('Acifran','51576','null','null'),('(S)-3-Hydroxybutyric acid','94318','HMDB0000442','CHEBI:17290'),('Sodium 3-hydroxybutyrate','23676771','null','CHEBI:113373'),('Diosmetin','5281612','HMDB0029676','CHEBI:4630'),('Acacetin','5280442','null','CHEBI:15335'),('Valerate','114781','null','CHEBI:31011'),('Taurocholic acid','6675','HMDB0000036','CHEBI:28865'),('Dihydroresveratrol','185914','HMDB0240498','CHEBI:4582'),('Lunularin','181511','null','CHEBI:6568'),('3,4-Dihydroxy-trans-stilbene','10176710','null','null'),('1-(3-Chloro-2,6-dihydroxy-4-methoxyphenyl)-1-hexanone','3081033','null','null'),('Colibactin','138805674','null','CHEBI:156303'),('beta-D-Fructofuranose','439709','null','CHEBI:28645'),('Genistein','5280961','HMDB0003217','CHEBI:28088'),('Dihydrogenistein','9838356','HMDB0005897','CHEBI:34707'),('D-Mannose','18950','HMDB0000169','CHEBI:4208'),('2,2\',4,4\'-Tetrabromodiphenyl ether','95170','HMDB0037547','CHEBI:81584'),('2,2\',4,4\',5-Pentabromodiphenyl ether','36159','HMDB0037516','CHEBI:81582'),('(10-[(Acetyloxy)methyl]-9-anthryl)methyl acetate','613145','null','null'),('Isovaleric acid','10430','HMDB0000718','CHEBI:28484'),('Palmitic acid','985','HMDB0000220','CHEBI:15756'),('D-Glucuronic Acid','94715','HMDB0000127','CHEBI:42717'),('Tretinoin','444795','null','CHEBI:15367'),('10-Keto-12Z-octadecenoic acid','24970825','null','null'),('Pioglitazone','4829','HMDB0015264','CHEBI:8228'),('Hydroxyquercitrin','129887767','null','null'),('Quercetin','5280343','HMDB0005794','CHEBI:16243'),('3,4-Dihydroxybenzoic acid','72','HMDB0001856','CHEBI:36062'),('Vancomycin','14969','HMDB0014653','CHEBI:28001'),('Diosgenin','99474','null','CHEBI:4629'),('Hydroquinone','785','HMDB0002434','CHEBI:17594'),('6,7,4\'-Trihydroxyisoflavone','5284649','HMDB0031715','CHEBI:74957'),('5-(3,4-Dihydroxyphenyl)-valerolactone','45093073','null','null'),('4-Hydroxy-(3\',4\'-dihydroxyphenyl)-valeric acid','52920332','HMDB0041679','CHEBI:137478'),('Folic acid','135398658','HMDB0000121','CHEBI:27470'),('Dihydroglycitein','101101166','null','CHEBI:174736'),('Quinic acid','6508','HMDB0003072','CHEBI:17521'),('Oxalacetic acid','970','HMDB0000223','CHEBI:30744'),('Norathyriol','5281656','null','CHEBI:7622'),('Serotonin','5202','HMDB0000259','CHEBI:28790'),('Leucocianidol','71629','null','CHEBI:15758'),('Glycerol','753','HMDB0000131','CHEBI:17754'),('Genipin','442424','HMDB0035830','CHEBI:5298'),('Indole-3-carboxaldehyde','10256','HMDB0029737','CHEBI:28238'),('10-Oxo-11-octadecenoic acid','10308378','null','null'),('Ponciretin','25201019','null','null'),('Lacto-N-tetraose','440993','HMDB0006566','CHEBI:30248'),('beta-D-Gal-(1->4)-beta-D-GlcNAc-(1->3)-beta-D-Gal-(1->4)-D-Glc','9831622','HMDB0006567','CHEBI:60239'),('2,4-Dioxopentanedioic acid','444873','null','CHEBI:42056'),('6\'-Hydroxy-O-desmethylangolensin','20601635','HMDB0041695','CHEBI:88662'),('3-(2,4-Dihydroxyphenyl)propanoic acid','96384','HMDB0126386','null'),('1,3-Diphenylpropan-2-ol','138478','HMDB0135544','null'),('1-Piperazinepentanamide, N-((1S,2R)-2,3-dihydro-2-hydroxy-1H-inden-1-yl)-2-(((1,1-dimethylethyl)amino)carbonyl)-4-(furo(2,3-b)pyridin-5-ylmethyl)-gamma-hydroxy-alpha-(phenylmethyl)-, (alphaR,gammaS,2S)-','5481990','null','null'),('Dopamine','681','HMDB0000073','CHEBI:18243'),('2-Amino-1-methyl-6-phenylimidazo[4,5-b]pyridine','1530','null','CHEBI:76290'),('7-Hydroxy-5-methyl-3-phenyl-6,7,8,9-tetrahydropyrido[3\',2\':4,5]imidazo[1,2-a]pyrimidin-5-ium chloride','129849310','null','null'),('Quercitrin','5280459','HMDB0033751','CHEBI:17558'),('Urolithin A','5488186','HMDB0013695','CHEBI:168442'),('(2S)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl)propan-2-ol','null','null','null'),('Ginsenoside-Rd','24721561','HMDB0035778','CHEBI:67988'),('Loganetin','10466307','null','null'),('Arctigenin','64981','null','CHEBI:79'),('Ursodeoxycholic acid','31401','HMDB0000946','CHEBI:9907'),('Phenylalanine','6140','HMDB0000159','CHEBI:17295'),('Glutamic acid','33032','HMDB0000148','CHEBI:16015'),('L-Threonine','6288','HMDB0000167','CHEBI:16857'),('Glycitein','5317750','HMDB0005781','CHEBI:34778'),('Kaempferol','5280863','HMDB0005801','CHEBI:28499'),('5-Hydroxy-1H-imidazole-4-carboxamide','124343','null','CHEBI:88290'),('Ginsenoside Rh2','119307','null','CHEBI:77147'),('(20S)-Protopanaxadiol','11213350','null','CHEBI:75950'),('20(R)-Ginsenoside Rh2','54580480','null','CHEBI:172776'),('Protopanaxadiol','9920281','null','CHEBI:76237'),('3\'-desmethylarctigenin','null','null','null'),('Glycerophospholipids','null','null','CHEBI:37739'),('Propanol','1031','null','CHEBI:28831'),('Ethyl phenyllactate, (-)-','9877619',' HMDB0032618','null'),('(R)-3-(4-Hydroxyphenyl)lactate','9548632','null','CHEBI:10980'),('Quercimeritrin','5282160','null','CHEBI:28529'),('11-Methoxycurvularin','10381440','null','null'),('Hydrogen','783','HMDB0001362','CHEBI:18276'),('Isobutyric acid','6590','HMDB0001873','CHEBI:16135'),('(2S)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','null','null','null'),('(4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid','null','null','null'),('(4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone','null','null','null'),('(4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid','null','null','null'),('(4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone','null','null','null'),('(4R)-4-hydroxy-5-(3-hydroxyphenyl) valeric acid','null','null','null'),('(4S)-4-hydroxy-5-(3-hydroxyphenyl) valeric acid','null','null','null'),('(4R)-5-(3-hydroxyphenyl)-γ-valerolactone','null','null','null'),('(4S)-5-(3-hydroxyphenyl)-γ-valerolactone','null','null','null'),('Myricetin','5281672','HMDB0002755','CHEBI:18152'),('Quercetin','5280343','HMDB0033751','CHEBI:16243'),('type I collagen cross-linked c-telopeptide','null','null','null'),('procollagen type 1 n-terminal propeptide','null','null','null'),('p-Cresol glucuronide','154035','HMDB0011686','CHEBI:87986'),('Baicalein','5281605','null','CHEBI:2979'),('Baicalin','64982','HMDB0041832','CHEBI:2981'),('Secoisolariciresinol','65373','HMDB0013692','CHEBI:65004'),('Lariciresinol','332427','null','CHEBI:67246'),('2,3-Bis(3,4-dihydroxybenzyl)butyrolactone','500189','null','null'),('Naringenin chalcone','5280960','HMDB0029631','CHEBI:15413'),('Phloretin','4788','HMDB0003306','CHEBI:17276'),('5,7-Dihydroxy-2-(4-hydroxyphenyl)chroman-4-one','932','HMDB0128074','null'),('Cholic acid','221493','HMDB0000619','CHEBI:16359'),('Eicosapentaenoic acid','446284','HMDB0001999','CHEBI:28364'),('Chenodeoxycholic acid','10133','HMDB0000518','CHEBI:16755'),('Desoxycortone','6166','HMDB0000016','CHEBI:16973'),('7-Dehydrocholesterol','439423','HMDB0000032','CHEBI:17759'),('Retinol','445354','HMDB0000305','CHEBI:17336'),('Myristic acid','11005','HMDB0000806','CHEBI:28875'),('Benzoic acid','243','HMDB0001870','CHEBI:30746'),('Indole-3-acetic acid','802','HMDB0000197','CHEBI:16411'),('4\'-dehydroxylated (�?-epigallocatechin','null','null','null'),('1-(3,4,5-trihydroxyphenyl)-3-(2,4,6trihydroxyphenyl)propan-2-ol','null','null','null'),('1-(3,5-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl)propan-2-ol','null','null','null'),('4\'-dehydroxylated isomer (�?-gallocatechin','null','null','null'),('Phenyalanine','null','null','null'),('Indole-3-pyruvic acid','803','HMDB0060484','CHEBI:29750'),('1-Methylindole','11781','null','null'),('Phenylpyruvic acid','997','HMDB0000205','CHEBI:30851'),('Atrolactic acid','1303','HMDB0142137','CHEBI:50392'),('Tyramine','5610','HMDB0000306','CHEBI:15760'),('Valine','6287','HMDB0000883','CHEBI:16414'),('2-Hydroxy-2-methylbutanenitrile','97823','null','CHEBI:60954'),('N-Acetylputrescine','122356','HMDB0002064','CHEBI:17768'),('Betonicine','164642','HMDB0029412','CHEBI:85533'),('N-Acetyl-D-Mannosamine','439281','null','CHEBI:63153'),('2-Formaminobenzoylacetate','440807','null','CHEBI:1098'),('12-Ketolithocholic acid','3080612','HMDB0000328','CHEBI:139134'),('Hyocholate','92805','null','CHEBI:133661'),('Hypoxanthine','135398638','HMDB0000157','CHEBI:17368'),('4-Acetamidobutyric acid','18189','HMDB0003681','CHEBI:17645'),('N(6)-Methyllysine','164795','HMDB0002038','CHEBI:17604'),('N-Isovaleroylglycine','546304','HMDB0000678','CHEBI:70984'),('21H-Biline-8,12-dipropanoic acid, 3,18-diethyl-1,4,5,15,16,19,22,24-octahydro-2,7,13,17-tetramethyl-1,19-dioxo-','6433298','null','null'),('Murideoxycholate','21158657','null','CHEBI:58876'),('Pentahomomethionine','25010767','null','CHEBI:50713'),('7-Hydroxy-3-oxocholanoic acid','53477692','HMDB0000460','null'),('Nutriadeoxycholate vulpecholate','null','null','null'),('7a-hydroxy-3-oxo-5b-cholanoate','null','null','null'),('Valyl-Hydroxyproline','10242944','HMDB0029128','null'),('1-Methoxypyrene','160236','HMDB0060324','CHEBI:81610'),('Valyl-Hydroxyproline','10242944','null','null'),('Isovanillic acid 3-O-sulfate','null','null','null'),('Phenyl sulfate','18795569','null','CHEBI:85289'),('gamma-Aminobutyric acid','119','HMDB0000112','CHEBI:16865'),('Purine','1044','HMDB0001366','CHEBI:17258'),('Uric acid','1175','HMDB0000289','CHEBI:17775'),('Rheinanthrone','119396','HMDB0039449','CHEBI:174546'),('GDP-L-fucose','135412609','HMDB0001095','CHEBI:17009'),('Oxiglutatione','65359','HMDB0003337','CHEBI:17858'),('Cysteine-glutathione disulfide','3080690','HMDB0000656','null'),('Taurine','1123','HMDB0000251','CHEBI:15891'),('Histamine','774','HMDB0000870','CHEBI:18295'),('(Z)-10-Hydroxy-12-octadecenoic acid','46186046','null','null'),('Methanol, (methyl-ONN-azoxy)-','11554','null','null'),('Cyanuric acid','7956','HMDB0041861','CHEBI:38028'),('1-Phenyl-1,2-propanedione','11363','HMDB0035243','CHEBI:63552'),('2-Oxoglutaric acid','51','HMDB0000208','CHEBI:30915'),('3-(4-Hydroxy-3-methoxyphenyl)propionic acid','14340','HMDB0062121','CHEBI:86612'),('2,8-Quinolinediol','97250','HMDB0240311','CHEBI:17715'),('Ergothioneine','5351619','HMDB0003045','CHEBI:4828'),('7-Ketodeoxycholic acid','188292','HMDB0000391','CHEBI:16390'),('Isorhamnetin','5281654','HMDB0002655','CHEBI:6052'),('Lysophosphatidylethanolamine','73755142','null','null'),('4\'-Hydroxy-3\',5,6,7,8-pentamethoxyflavone','21593928','HMDB0029547','CHEBI:175929'),('4-Hydroxy-5-(3-hydroxyphenyl)valeric acid','52920334','null','CHEBI:137492'),('Epigallocatechin','72277','null','CHEBI:42255'),('Gallic acid','370','HMDB0005807','CHEBI:30778'),('Sclerotioramine','135860757','null','null'),('Sclerotiorin','6436015','null','null'),('1,4-Diaminobutane','1045','HMDB0001414','CHEBI:17148'),('Spermidine','1102','HMDB0001257','CHEBI:16610'),('Acetylcholine','187','HMDB0000895','CHEBI:15355'),('Estradiol','5757','HMDB0000151','CHEBI:16469'),('3,3-Dimethyl-1-butanol','12233','null','null'),('Lipoteichoic acid','137349712','null','null'),('Dinoprostone','5280360','null','CHEBI:15551'),('N,N,N-trimethyl-5-aminovaleric acid','null','null','null'),('3,8,9-Trihydroxy-6H-benzo[c]chromen-6-one','60198001','null','CHEBI:174255'),('5-Acetamido-2-azaniumylpentanoate','121596237','null','CHEBI:133180'),('11-oxo-mogrol','null','null','null'),('13-hydroxy-9(Z),15(Z)-octadecadienoic acid','null','null','null'),('13-oxo-9(Z),15(Z)-octadecadienoic acid','null','null','null'),('Nicotinamide','936','null','CHEBI:17154'),('Tryptamine','1150','HMDB0000303','CHEBI:16765'),('Oxyberberine','11066','null','null'),('9,11-Octadecadienoic acid','5282796','HMDB0005047','CHEBI:36025'),('1D-Myo-inositol 6-phosphate','161368','null','CHEBI:64838'),('[(1S,2R,3S,4S,5R,6S)-2,3,5-Trihydroxy-4,6-diphosphonooxycyclohexyl] dihydrogen phosphate','55310','null','null'),('1D-Myo-inositol 1,4,5,6-tetrakisphosphate(8-)','25201642','HMDB0004527','CHEBI:57627'),('D-Glutamic acid','23327','HMDB0003339','CHEBI:15966'),('Sodium benzoate','517055','HMDB0001870','CHEBI:113455'),('Arachidonic acid','444899','HMDB0001043','CHEBI:15843'),('7-Sulfocholic acid','459070','HMDB0002421','CHEBI:166724'),('Tryptophan','6305','HMDB0000929','CHEBI:16828'),('L-Urobilin','5280818','HMDB0240259','CHEBI:29023'),('1-Deoxynojirimycin','29435','HMDB0035359','CHEBI:44369'),('Stachyose','439531','HMDB0003553','CHEBI:17164');
/*!40000 ALTER TABLE `metabolite` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `metabolite_gene_change_results`
--

DROP TABLE IF EXISTS `metabolite_gene_change_results`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `metabolite_gene_change_results` (
  `index` int DEFAULT NULL,
  `host_type` text COLLATE utf8mb4_bin,
  `Alteration` text COLLATE utf8mb4_bin,
  `substrate_name` text COLLATE utf8mb4_bin,
  `throughput` text COLLATE utf8mb4_bin,
  `description` text COLLATE utf8mb4_bin,
  `Evidence` int DEFAULT NULL
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `metabolite_gene_change_results`
--

LOCK TABLES `metabolite_gene_change_results` WRITE;
/*!40000 ALTER TABLE `metabolite_gene_change_results` DISABLE KEYS */;
INSERT INTO `metabolite_gene_change_results` VALUES (3059,'human','null','D-Glucose','null','Major end products of metabolism of glucose are acetic and butyric acids.',21357455),(3060,'human','null','Salicin','null','Acid is produced from glucose, salicin, D-xylose, L-arabinose and L-rhamnose. Acid is produced weakly from D-mannose.',21357455),(3061,'human','null','D-Xylose','null','Acid is produced from glucose, salicin, D-xylose, L-arabinose and L-rhamnose. Acid is produced weakly from D-mannose.',21357455),(3062,'human','null','L-Arabinose','null','Acid is produced from glucose, salicin, D-xylose, L-arabinose and L-rhamnose. Acid is produced weakly from D-mannose.',21357455),(3063,'human','null','L-Rhamnose','null','Acid is produced from glucose, salicin, D-xylose, L-arabinose and L-rhamnose. Acid is produced weakly from D-mannose.',21357455),(3064,'human','null','D-Mannose','null','Acid is produced from glucose, salicin, D-xylose, L-arabinose and L-rhamnose. Acid is produced weakly from D-mannose.',21357455),(3065,'human','null','D-Glucose','null','Major end products of metabolism of glucose are acetic and butyric acids.',21357455),(3066,'human','null','Salicin','null','Acid is produced from glucose, salicin, D-xylose, L-arabinose and L-rhamnose. Acid is produced weakly from D-mannose.',21357455),(3067,'human','null','D-Xylose','null','Acid is produced from glucose, salicin, D-xylose, L-arabinose and L-rhamnose. Acid is produced weakly from D-mannose.',21357455),(3068,'human','null','L-Arabinose','null','Acid is produced from glucose, salicin, D-xylose, L-arabinose and L-rhamnose. Acid is produced weakly from D-mannose.',21357455),(3069,'human','null','L-Rhamnose','null','Acid is produced from glucose, salicin, D-xylose, L-arabinose and L-rhamnose. Acid is produced weakly from D-mannose.',21357455),(3070,'human','null','D-Mannose','null','Acid is produced from glucose, salicin, D-xylose, L-arabinose and L-rhamnose. Acid is produced weakly from D-mannose.',21357455),(3071,'human','null','Orientin','null','This strain, which was able to cleave the C-glycoside of orientin to produce luteolin and generate some other metabolites.',24316522),(3072,'human','null','Bile acid','null','Deoxycholic acid (DCA) and lithocholic acid (LCA) are secondary bile acids that are generated in the gut by bacterial transformation of primary bile acids, cholic acid(CA), and chenodeoxycholic acid (CDCA), respectively.The proposed pathway for this transformation by Clostridium scindens VPI 12708.',23836456),(3073,'human','null','Cholic acid','null','Deoxycholic acid (DCA) and lithocholic acid (LCA) are secondary bile acids that are generated in the gut by bacterial transformation of primary bile acids, cholic acid(CA), and chenodeoxycholic acid (CDCA), respectively.The proposed pathway for this transformation by Clostridium scindens VPI 12708.',23836456),(3074,'human','null','Chenodeoxycholic acid','null','Deoxycholic acid (DCA) and lithocholic acid (LCA) are secondary bile acids that are generated in the gut by bacterial transformation of primary bile acids, cholic acid(CA), and chenodeoxycholic acid (CDCA), respectively.The proposed pathway for this transformation by Clostridium scindens VPI 12708.',23836456),(3075,'human','null','Bile acid','null','Deoxycholic acid (DCA) and lithocholic acid (LCA) are secondary bile acids that are generated in the gut by bacterial transformation of primary bile acids, cholic acid(CA), and chenodeoxycholic acid (CDCA), respectively.The proposed pathway for this transformation by Clostridium scindens VPI 12708.',23836456),(3076,'human','null','Cholic acid','null','Deoxycholic acid (DCA) and lithocholic acid (LCA) are secondary bile acids that are generated in the gut by bacterial transformation of primary bile acids, cholic acid(CA), and chenodeoxycholic acid (CDCA), respectively.The proposed pathway for this transformation by Clostridium scindens VPI 12708.',23836456),(3077,'human','null','Chenodeoxycholic acid','null','Deoxycholic acid (DCA) and lithocholic acid (LCA) are secondary bile acids that are generated in the gut by bacterial transformation of primary bile acids, cholic acid(CA), and chenodeoxycholic acid (CDCA), respectively.The proposed pathway for this transformation by Clostridium scindens VPI 12708.',23836456),(3080,'human','increase','Cholesterol','low-throughput','Calprotectin, as the sole feature in its own measurement type, was weakly associated with a number of metabolites that were not differentially abundant in dysbiosis, as well as with the metagenomic abundance of several dysbiosis-associated ECs. Three host genes appeared in this high-significance subnetwork: ileal expression of GIP, NXPE4, and ANXA10. Expression of RNA polymerase was also a prominent node in the network, though not a hub, that was upregulated in dysbiosis. The regulation of this essential enzyme class is growth-rate-dependent, suggesting that microbial communities as a whole are more often in higher growth conditions in dysbiotic IBD.',31142855),(3081,'human','increase','Cholesterol','low-throughput','Calprotectin, as the sole feature in its own measurement type, was weakly associated with a number of metabolites that were not differentially abundant in dysbiosis, as well as with the metagenomic abundance of several dysbiosis-associated ECs. Three host genes appeared in this high-significance subnetwork: ileal expression of GIP, NXPE4, and ANXA10. Expression of RNA polymerase was also a prominent node in the network, though not a hub, that was upregulated in dysbiosis. The regulation of this essential enzyme class is growth-rate-dependent, suggesting that microbial communities as a whole are more often in higher growth conditions in dysbiotic IBD.',31142855),(3082,'human','increase','Cholesterol','low-throughput','Calprotectin, as the sole feature in its own measurement type, was weakly associated with a number of metabolites that were not differentially abundant in dysbiosis, as well as with the metagenomic abundance of several dysbiosis-associated ECs. Three host genes appeared in this high-significance subnetwork: ileal expression of GIP, NXPE4, and ANXA10. Expression of RNA polymerase was also a prominent node in the network, though not a hub, that was upregulated in dysbiosis. The regulation of this essential enzyme class is growth-rate-dependent, suggesting that microbial communities as a whole are more often in higher growth conditions in dysbiotic IBD.',31142855),(3083,'human','null','Cholesterol','null','Calprotectin, as the sole feature in its own measurement type, was weakly associated with a number of metabolites that were not differentially abundant in dysbiosis, as well as with the metagenomic abundance of several dysbiosis-associated ECs. Three host genes appeared in this high-significance subnetwork: ileal expression of GIP, NXPE4, and ANXA10. Expression of RNA polymerase was also a prominent node in the network, though not a hub, that was upregulated in dysbiosis. The regulation of this essential enzyme class is growth-rate-dependent, suggesting that microbial communities as a whole are more often in higher growth conditions in dysbiotic IBD.',31142855),(3088,'human','null','Cynaroside','null','Eubacterium ramulus, a quercetin-3-glucoside degrading anaerobic microorganism that occurs at num bers of approximately 108/g dry feces in humans, was tested for its ability to transform other flavonoids. The organism degraded luteolin-7-glucoside, rutin, quercetin, kaempferol, luteolin, eriodictyol, naringenin, taxifolin, and phloretin to phenolic acids. It hydrolyzed kaempferol-3-sorphoro side-7-glucoside to kaempferol-3-sorphoroside and trans formed 3,4-dihydroxyphenylacetic acid, a product of anaer obic quercetin degradation, very slowly to non-aromatic fermentation products.',10648107),(3089,'human','null','Taxifolin','null','Eubacterium ramulus, a quercetin-3-glucoside degrading anaerobic microorganism that occurs at num bers of approximately 108/g dry feces in humans, was tested for its ability to transform other flavonoids. The organism degraded luteolin-7-glucoside, rutin, quercetin, kaempferol, luteolin, eriodictyol, naringenin, taxifolin, and phloretin to phenolic acids. It hydrolyzed kaempferol-3-sorphoro side-7-glucoside to kaempferol-3-sorphoroside and trans formed 3,4-dihydroxyphenylacetic acid, a product of anaer obic quercetin degradation, very slowly to non-aromatic fermentation products.',10648107),(3090,'human','null','Phloretin','null','Eubacterium ramulus, a quercetin-3-glucoside degrading anaerobic microorganism that occurs at num bers of approximately 108/g dry feces in humans, was tested for its ability to transform other flavonoids. The organism degraded luteolin-7-glucoside, rutin, quercetin, kaempferol, luteolin, eriodictyol, naringenin, taxifolin, and phloretin to phenolic acids. It hydrolyzed kaempferol-3-sorphoro side-7-glucoside to kaempferol-3-sorphoroside and trans formed 3,4-dihydroxyphenylacetic acid, a product of anaer obic quercetin degradation, very slowly to non-aromatic fermentation products.',10648107),(3091,'human','null','Luteolin','null','Eubacterium ramulus, a quercetin-3-glucoside degrading anaerobic microorganism that occurs at num bers of approximately 108/g dry feces in humans, was tested for its ability to transform other flavonoids. The organism degraded luteolin-7-glucoside, rutin, quercetin, kaempferol, luteolin, eriodictyol, naringenin, taxifolin, and phloretin to phenolic acids. It hydrolyzed kaempferol-3-sorphoro side-7-glucoside to kaempferol-3-sorphoroside and trans formed 3,4-dihydroxyphenylacetic acid, a product of anaer obic quercetin degradation, very slowly to non-aromatic fermentation products.',10648107),(3092,'human','null','5,7-Dihydroxy-2-(4-hydroxyphenyl)chroman-4-one','null','Eubacterium ramulus, a quercetin-3-glucoside degrading anaerobic microorganism that occurs at num bers of approximately 108/g dry feces in humans, was tested for its ability to transform other flavonoids. The organism degraded luteolin-7-glucoside, rutin, quercetin, kaempferol, luteolin, eriodictyol, naringenin, taxifolin, and phloretin to phenolic acids. It hydrolyzed kaempferol-3-sorphoro side-7-glucoside to kaempferol-3-sorphoroside and trans formed 3,4-dihydroxyphenylacetic acid, a product of anaer obic quercetin degradation, very slowly to non-aromatic fermentation products.',10648107),(3093,'human','null','Eriodictyol','null','Eubacterium ramulus, a quercetin-3-glucoside degrading anaerobic microorganism that occurs at num bers of approximately 108/g dry feces in humans, was tested for its ability to transform other flavonoids. The organism degraded luteolin-7-glucoside, rutin, quercetin, kaempferol, luteolin, eriodictyol, naringenin, taxifolin, and phloretin to phenolic acids. It hydrolyzed kaempferol-3-sorphoro side-7-glucoside to kaempferol-3-sorphoroside and trans formed 3,4-dihydroxyphenylacetic acid, a product of anaer obic quercetin degradation, very slowly to non-aromatic fermentation products.',10648107),(3094,'human','null','Kaempferol','null','Eubacterium ramulus, a quercetin-3-glucoside degrading anaerobic microorganism that occurs at num bers of approximately 108/g dry feces in humans, was tested for its ability to transform other flavonoids. The organism degraded luteolin-7-glucoside, rutin, quercetin, kaempferol, luteolin, eriodictyol, naringenin, taxifolin, and phloretin to phenolic acids. It hydrolyzed kaempferol-3-sorphoro side-7-glucoside to kaempferol-3-sorphoroside and trans formed 3,4-dihydroxyphenylacetic acid, a product of anaer obic quercetin degradation, very slowly to non-aromatic fermentation products.',10648107),(3095,'human','null','Rutin','null','Eubacterium ramulus, a quercetin-3-glucoside degrading anaerobic microorganism that occurs at num bers of approximately 108/g dry feces in humans, was tested for its ability to transform other flavonoids. The organism degraded luteolin-7-glucoside, rutin, quercetin, kaempferol, luteolin, eriodictyol, naringenin, taxifolin, and phloretin to phenolic acids. It hydrolyzed kaempferol-3-sorphoro side-7-glucoside to kaempferol-3-sorphoroside and trans formed 3,4-dihydroxyphenylacetic acid, a product of anaer obic quercetin degradation, very slowly to non-aromatic fermentation products.',10648107),(3096,'human','null','Quercetin','null','Eubacterium ramulus, a quercetin-3-glucoside degrading anaerobic microorganism that occurs at num bers of approximately 108/g dry feces in humans, was tested for its ability to transform other flavonoids. The organism degraded luteolin-7-glucoside, rutin, quercetin, kaempferol, luteolin, eriodictyol, naringenin, taxifolin, and phloretin to phenolic acids. It hydrolyzed kaempferol-3-sorphoro side-7-glucoside to kaempferol-3-sorphoroside and trans formed 3,4-dihydroxyphenylacetic acid, a product of anaer obic quercetin degradation, very slowly to non-aromatic fermentation products.',10648107),(3097,'human','null','Isoquercitrin','null','Schneider et al. (1999) isolated strains of Eubacterium ramulus from human feces that are capable of degrading quercetin-3-glucoside to 3,4-dihydroxyphenylacetic acid (3,4-DIOHA), butyrate, and acetate.Phloroglucinol was formed as an intermediate.',10648107),(3098,'human','null','Isoquercitrin','null','Schneider et al. (1999) isolated strains of Eubacterium ramulus from human feces that are capable of degrading quercetin-3-glucoside to 3,4-dihydroxyphenylacetic acid (3,4-DIOHA), butyrate, and acetate.Phloroglucinol was formed as an intermediate.',10648107),(3099,'human','null','Isoquercitrin','null','Schneider et al. (1999) isolated strains of Eubacterium ramulus from human feces that are capable of degrading quercetin-3-glucoside to 3,4-dihydroxyphenylacetic acid (3,4-DIOHA), butyrate, and acetate.Phloroglucinol was formed as an intermediate.',10648107),(3100,'human','null','Isoquercitrin','null','In addition,the identity of fermentation products 3,4-dihydroxyphenylacetic acid, formed from quercetin-3-glucosdide, or taxifolin, 3-(3,4 dihydroxyphenyl)propionic acid, formed from luteolin or eriodictyol, and 4-hydroxyphenylacetic acid, formed from kaempferol, were confirmed by LC-MS/MS. The most intensive ion peaks in the mass spectra of these compounds are listed in Table 2.',10648107),(3101,'human','null','Taxifolin','null','In addition,the identity of fermentation products 3,4-dihydroxyphenylacetic acid, formed from quercetin-3-glucosdide, or taxifolin, 3-(3,4 dihydroxyphenyl)propionic acid, formed from luteolin or eriodictyol, and 4-hydroxyphenylacetic acid, formed from kaempferol, were confirmed by LC-MS/MS. The most intensive ion peaks in the mass spectra of these compounds are listed in Table 2.',10648107),(3102,'human','null','Luteolin','null','In addition,the identity of fermentation products 3,4-dihydroxyphenylacetic acid, formed from quercetin-3-glucosdide, or taxifolin, 3-(3,4 dihydroxyphenyl)propionic acid, formed from luteolin or eriodictyol, and 4-hydroxyphenylacetic acid, formed from kaempferol, were confirmed by LC-MS/MS. The most intensive ion peaks in the mass spectra of these compounds are listed in Table 2.',10648107),(3103,'human','null','Eriodictyol','null','In addition,the identity of fermentation products 3,4-dihydroxyphenylacetic acid, formed from quercetin-3-glucosdide, or taxifolin, 3-(3,4 dihydroxyphenyl)propionic acid, formed from luteolin or eriodictyol, and 4-hydroxyphenylacetic acid, formed from kaempferol, were confirmed by LC-MS/MS. The most intensive ion peaks in the mass spectra of these compounds are listed in Table 2.',10648107),(3104,'human','null','Kaempferol','null','In addition,the identity of fermentation products 3,4-dihydroxyphenylacetic acid, formed from quercetin-3-glucosdide, or taxifolin, 3-(3,4 dihydroxyphenyl)propionic acid, formed from luteolin or eriodictyol, and 4-hydroxyphenylacetic acid, formed from kaempferol, were confirmed by LC-MS/MS. The most intensive ion peaks in the mass spectra of these compounds are listed in Table 2.',10648107),(3105,'human','null','5,7-Dihydroxy-2-(4-hydroxyphenyl)chroman-4-one','null','This assumption is supported by the degradation of eriodictyol and naringenin (IX) to 3-(3,4-dihydroxyphenyl)propionic acid and 3-(4-hydroxyphenyl)propionic acid (XII), respectively.',10648107),(3106,'human','null','Kaempferol-3-sorphoroside-7-glucoside','null','This assumption is supported by the degradation of eriodictyol and naringenin (IX) to 3-(3,4-dihydroxyphenyl)propionic acid and 3-(4-hydroxyphenyl)propionic acid (XII), respectively.',10648107),(3107,'human','null','Quercetin','null','Luteolin-5-glucoside, diosmetin-7rutinoside, naringenin-7-neohesperidoside, (+)-catechin, and (�?-epicatechin were not degraded. Cell extracts of E. ramulus contained α- and β-D-glucosidase activities, but were devoid of α-L-rhamnosidase activity.',10648107),(3108,'human','null','null','null','Schneider et al. (1999) isolated strains of Eubacterium ramulus from human feces that are capable of degrading quercetin-3-glucoside to 3,4-dihydroxyphenylacetic acid (3,4-DIOHA), butyrate, and acetate.Phloroglucinol was formed as an intermediate.',10648107),(3109,'human','decrease','null','low-throughput','Indoxyl-sulfate (Beta(SE) = -2.74(0.24), P = 8.8x10-29),p-cresyl-sulfate (-1.99(0.24), P = 4.6x10-16), and phenylacetylglutamine (-2.73(0.25),P = 1.2x10-25) were significantly and negatively associated with eGFR after adjusting for age, sex, body mass index (BMI), metabolite batch, type 2 diabetes, family relatedness and multiple testing using Bonferroni correction (Table 2).',26241311),(3110,'human','decrease','null','low-throughput','Indoxyl-sulfate (Beta(SE) = -2.74(0.24), P = 8.8x10-29),p-cresyl-sulfate (-1.99(0.24), P = 4.6x10-16), and phenylacetylglutamine (-2.73(0.25),P = 1.2x10-25) were significantly and negatively associated with eGFR after adjusting for age, sex, body mass index (BMI), metabolite batch, type 2 diabetes, family relatedness and multiple testing using Bonferroni correction (Table 2).',26241311),(3111,'human','decrease','null','low-throughput','Indoxyl-sulfate (Beta(SE) = -2.74(0.24), P = 8.8x10-29),p-cresyl-sulfate (-1.99(0.24), P = 4.6x10-16), and phenylacetylglutamine (-2.73(0.25),P = 1.2x10-25) were significantly and negatively associated with eGFR after adjusting for age, sex, body mass index (BMI), metabolite batch, type 2 diabetes, family relatedness and multiple testing using Bonferroni correction (Table 2).',26241311),(3112,'human','increase','null','low-throughput','we demonstrated that GPR41 and GPR43 were modestly expressed in human distal renal tubules and col lecting tubules. We also found that SCFAs (acetate, propionate, and butyrate) inhibited TNF-a induced expression of a pro inflammatory factor, MCP-1, in HRCEs by reducing p38 and JNK phosphorylation via GPR41 and/or GPR43 activation involving Gi/o, particularly Gb/g. To the best of our knowledge, this is the first report to describe the anti-inflammatory effects of SCFAs through a GPR41/43/Gb/g pathway in renal parenchymal cells.',28322790),(3113,'human','increase','null','low-throughput','we demonstrated that GPR41 and GPR43 were modestly expressed in human distal renal tubules and col lecting tubules. We also found that SCFAs (acetate, propionate, and butyrate) inhibited TNF-a induced expression of a pro inflammatory factor, MCP-1, in HRCEs by reducing p38 and JNK phosphorylation via GPR41 and/or GPR43 activation involving Gi/o, particularly Gb/g. To the best of our knowledge, this is the first report to describe the anti-inflammatory effects of SCFAs through a GPR41/43/Gb/g pathway in renal parenchymal cells.',28322790),(3114,'human','decrease','null','low-throughput','we demonstrated that GPR41 and GPR43 were modestly expressed in human distal renal tubules and col lecting tubules. We also found that SCFAs (acetate, propionate, and butyrate) inhibited TNF-a induced expression of a pro inflammatory factor, MCP-1, in HRCEs by reducing p38 and JNK phosphorylation via GPR41 and/or GPR43 activation involving Gi/o, particularly Gb/g. To the best of our knowledge, this is the first report to describe the anti-inflammatory effects of SCFAs through a GPR41/43/Gb/g pathway in renal parenchymal cells.',28322790),(3115,'human','increase','null','low-throughput','we demonstrated that GPR41 and GPR43 were modestly expressed in human distal renal tubules and col lecting tubules. We also found that SCFAs (acetate, propionate, and butyrate) inhibited TNF-a induced expression of a pro inflammatory factor, MCP-1, in HRCEs by reducing p38 and JNK phosphorylation via GPR41 and/or GPR43 activation involving Gi/o, particularly Gb/g. To the best of our knowledge, this is the first report to describe the anti-inflammatory effects of SCFAs through a GPR41/43/Gb/g pathway in renal parenchymal cells.',28322790),(3116,'human','increase','null','low-throughput','we demonstrated that GPR41 and GPR43 were modestly expressed in human distal renal tubules and col lecting tubules. We also found that SCFAs (acetate, propionate, and butyrate) inhibited TNF-a induced expression of a pro inflammatory factor, MCP-1, in HRCEs by reducing p38 and JNK phosphorylation via GPR41 and/or GPR43 activation involving Gi/o, particularly Gb/g. To the best of our knowledge, this is the first report to describe the anti-inflammatory effects of SCFAs through a GPR41/43/Gb/g pathway in renal parenchymal cells.',28322790),(3117,'human','decrease','null','low-throughput','we demonstrated that GPR41 and GPR43 were modestly expressed in human distal renal tubules and col lecting tubules. We also found that SCFAs (acetate, propionate, and butyrate) inhibited TNF-a induced expression of a pro inflammatory factor, MCP-1, in HRCEs by reducing p38 and JNK phosphorylation via GPR41 and/or GPR43 activation involving Gi/o, particularly Gb/g. To the best of our knowledge, this is the first report to describe the anti-inflammatory effects of SCFAs through a GPR41/43/Gb/g pathway in renal parenchymal cells.',28322790),(3118,'human','increase','null','low-throughput','we demonstrated that GPR41 and GPR43 were modestly expressed in human distal renal tubules and col lecting tubules. We also found that SCFAs (acetate, propionate, and butyrate) inhibited TNF-a induced expression of a pro inflammatory factor, MCP-1, in HRCEs by reducing p38 and JNK phosphorylation via GPR41 and/or GPR43 activation involving Gi/o, particularly Gb/g. To the best of our knowledge, this is the first report to describe the anti-inflammatory effects of SCFAs through a GPR41/43/Gb/g pathway in renal parenchymal cells.',28322790),(3119,'human','increase','null','low-throughput','we demonstrated that GPR41 and GPR43 were modestly expressed in human distal renal tubules and col lecting tubules. We also found that SCFAs (acetate, propionate, and butyrate) inhibited TNF-a induced expression of a pro inflammatory factor, MCP-1, in HRCEs by reducing p38 and JNK phosphorylation via GPR41 and/or GPR43 activation involving Gi/o, particularly Gb/g. To the best of our knowledge, this is the first report to describe the anti-inflammatory effects of SCFAs through a GPR41/43/Gb/g pathway in renal parenchymal cells.',28322790),(3120,'human','decrease','null','low-throughput','we demonstrated that GPR41 and GPR43 were modestly expressed in human distal renal tubules and col lecting tubules. We also found that SCFAs (acetate, propionate, and butyrate) inhibited TNF-a induced expression of a pro inflammatory factor, MCP-1, in HRCEs by reducing p38 and JNK phosphorylation via GPR41 and/or GPR43 activation involving Gi/o, particularly Gb/g. To the best of our knowledge, this is the first report to describe the anti-inflammatory effects of SCFAs through a GPR41/43/Gb/g pathway in renal parenchymal cells.',28322790),(3121,'human','null','Daidzein','null','One strain (strain TM-40) produced dihydrodaidzein both from daidzein and daidzin.',17113326),(3122,'human','null','Daidzin','null','One strain (strain TM-40) produced dihydrodaidzein both from daidzein and daidzin.',17113326),(3123,'human','null','Daidzein','null','A newly identified equol-producing bacterium, Slackia sp.strain NATTS, with a high equol-producing activity was isolated from human feces in Japanese adults.',24761887),(3124,'human','null','D-Glucose','null','endogenous capability of Escherichia coli for 2,3-butanediol (2,3-BD) synthesis was explored using the anaerobic selection platform based on redox balance. To address the issue of NADH excess associated with the 2,3-BD pathway, we devised a substrate-decoupled system where a pathway intermediate is externally supplied in addition to the carbon source to decouple NADH recycling ratio from the intrinsic pathway stoichiometry.',27931827),(3125,'human','null','Pyruvate','null','endogenous capability of Escherichia coli for 2,3-butanediol (2,3-BD) synthesis was explored using the anaerobic selection platform based on redox balance. To address the issue of NADH excess associated with the 2,3-BD pathway, we devised a substrate-decoupled system where a pathway intermediate is externally supplied in addition to the carbon source to decouple NADH recycling ratio from the intrinsic pathway stoichiometry.',27931827),(3126,'human','null','D-Glucose','null','endogenous capability of Escherichia coli for 2,3-butanediol (2,3-BD) synthesis was explored using the anaerobic selection platform based on redox balance. To address the issue of NADH excess associated with the 2,3-BD pathway, we devised a substrate-decoupled system where a pathway intermediate is externally supplied in addition to the carbon source to decouple NADH recycling ratio from the intrinsic pathway stoichiometry.',27931827),(3127,'human','null','D-Glucose','null','endogenous capability of Escherichia coli for 2,3-butanediol (2,3-BD) synthesis was explored using the anaerobic selection platform based on redox balance. To address the issue of NADH excess associated with the 2,3-BD pathway, we devised a substrate-decoupled system where a pathway intermediate is externally supplied in addition to the carbon source to decouple NADH recycling ratio from the intrinsic pathway stoichiometry.',27931827),(3128,'human','null','D-Glucose','null','endogenous capability of Escherichia coli for 2,3-butanediol (2,3-BD) synthesis was explored using the anaerobic selection platform based on redox balance. To address the issue of NADH excess associated with the 2,3-BD pathway, we devised a substrate-decoupled system where a pathway intermediate is externally supplied in addition to the carbon source to decouple NADH recycling ratio from the intrinsic pathway stoichiometry.',27931827),(3129,'human','null','Pyruvate','null','endogenous capability of Escherichia coli for 2,3-butanediol (2,3-BD) synthesis was explored using the anaerobic selection platform based on redox balance. To address the issue of NADH excess associated with the 2,3-BD pathway, we devised a substrate-decoupled system where a pathway intermediate is externally supplied in addition to the carbon source to decouple NADH recycling ratio from the intrinsic pathway stoichiometry.',27931827),(3130,'human','null','D-Glucose','null','endogenous capability of Escherichia coli for 2,3-butanediol (2,3-BD) synthesis was explored using the anaerobic selection platform based on redox balance. To address the issue of NADH excess associated with the 2,3-BD pathway, we devised a substrate-decoupled system where a pathway intermediate is externally supplied in addition to the carbon source to decouple NADH recycling ratio from the intrinsic pathway stoichiometry.',27931827),(3131,'human','null','D-Glucose','null','endogenous capability of Escherichia coli for 2,3-butanediol (2,3-BD) synthesis was explored using the anaerobic selection platform based on redox balance. To address the issue of NADH excess associated with the 2,3-BD pathway, we devised a substrate-decoupled system where a pathway intermediate is externally supplied in addition to the carbon source to decouple NADH recycling ratio from the intrinsic pathway stoichiometry.',27931827),(3132,'human','null','Choline','null','Klebsiella and Escherichia harboured three of the four potential TMA-production pathways (choline, carnitine and TMAO).',28785417),(3133,'human','null','Carnitine','null','Klebsiella and Escherichia harboured three of the four potential TMA-production pathways (choline, carnitine and TMAO).',28785417),(3134,'human','null','Trimethylamine oxide','null','Klebsiella and Escherichia harboured three of the four potential TMA-production pathways (choline, carnitine and TMAO).',28785417),(3135,'human','null','Choline','null','Klebsiella and Escherichia harboured three of the four potential TMA-production pathways (choline, carnitine and TMAO).',28785417),(3136,'human','null','Carnitine','null','Klebsiella and Escherichia harboured three of the four potential TMA-production pathways (choline, carnitine and TMAO).',28785417),(3137,'human','null','Trimethylamine oxide','null','Klebsiella and Escherichia harboured three of the four potential TMA-production pathways (choline, carnitine and TMAO).',28785417),(3138,'human','null','Hesperetin dihydrochalcone','null','C.orbiscindens was not able to convert hesperetindihydrochalcone 4�?â-D-glucoside within a period of 43 h, whereas the aglycon was cleaved by this species. C. orbiscindens formed 0.18 mM 3-(3-hydroxy-4-methoxyphenyl)propionic acid from 0.26 mM hesperetin dihydrochalcone within 23 h of incubation.',15740074),(3139,'human','null','beta-D-Glucopyranoside, 3,5-dihydroxy-4-(3-(3-hydroxy-4-methoxyphenyl)-1-oxopropyl)phenyl','null','However, hesperetin dihydrochalcone 4�?â-D-glucoside was converted by E. ramulus to hesperetin dihydrochalcone and further to 3-(3-hydroxy-4-methoxyphenyl)propionic acid, but not by C. orbiscindens.',15740074),(3140,'human','null','Hesperetin dihydrochalcone','null','However, hesperetin dihydrochalcone 4�?â-D-glucoside was converted by E. ramulus to hesperetin dihydrochalcone and further to 3-(3-hydroxy-4-methoxyphenyl)propionic acid, but not by C. orbiscindens.',15740074),(3147,'human','null','Daidzein','null','Strain YY7918 converted substrates daidzein and dihydrodaidzein into S-equol.',18838805),(3148,'human','null','Dihydrodaidzein','null','Strain YY7918 converted substrates daidzein and dihydrodaidzein into S-equol.',18838805),(3156,'human','increase','null','low-throughput','As Erk is a ‘downstream�?signaling component of TLR pathways, we investigated whether succinate alone or in combination with TLR ligands was able to induce proinflammatory cytokine production by human DCs. Succinate alone did not have any great effect on a variety of cytokines and chemokines produced by activated human DCs. However, succinate augmented the expression of TNF and IL-1b by iMoDCs when used in combination with the TLR3 ligand poly(I:C) or the TLR7 agonist imiquimod.',18820681),(3157,'human','increase','null','low-throughput','As Erk is a ‘downstream�?signaling component of TLR pathways, we investigated whether succinate alone or in combination with TLR ligands was able to induce proinflammatory cytokine production by human DCs. Succinate alone did not have any great effect on a variety of cytokines and chemokines produced by activated human DCs. However, succinate augmented the expression of TNF and IL-1b by iMoDCs when used in combination with the TLR3 ligand poly(I:C) or the TLR7 agonist imiquimod.',18820681),(3158,'human','increase','null','low-throughput','Succinate alone was able to induce phosphorylation of Erk1/2 in human iMoDCs,in agreement with the reported activity of succinate on GPR91 transfectants. Polyinosinic-polycytidylic acid (poly(I:C)) alone showed a very modest activation of this signaling pathway, but when both poly(I:C) and succinate were present, Erk1/2 phosphorylation was synergistically induced.',18820681),(3159,'human','increase','null','low-throughput','Succinate alone was able to induce phosphorylation of Erk1/2 in human iMoDCs,in agreement with the reported activity of succinate on GPR91 transfectants. Polyinosinic-polycytidylic acid (poly(I:C)) alone showed a very modest activation of this signaling pathway, but when both poly(I:C) and succinate were present, Erk1/2 phosphorylation was synergistically induced.',18820681),(3160,'human','increase','null','low-throughput','In addition, activation of iMoDCs with poly(I:C) or imiquimod in the presence of succinate induced substantial enhancement of the expression of CCL2 mRNA and CXCL8 mRNA (data not shown). Therefore, we concluded that succinate potentiates the production and secretion of proinflam matory mediators by iMoDCs when the cells are stimulated through some TLRs.',18820681),(3161,'human','increase','null','low-throughput','In addition, activation of iMoDCs with poly(I:C) or imiquimod in the presence of succinate induced substantial enhancement of the expression of CCL2 mRNA and CXCL8 mRNA (data not shown). Therefore, we concluded that succinate potentiates the production and secretion of proinflam matory mediators by iMoDCs when the cells are stimulated through some TLRs.',18820681),(3162,'human','increase','null','low-throughput','butyrate enhanced the PMA-induced expression of c-fos and ERK1/2 phosphorylation, but not p38 and JNK.',23300800),(3163,'human','increase','null','low-throughput','butyrate enhanced the PMA-induced expression of c-fos and ERK1/2 phosphorylation, but not p38 and JNK.',23300800),(3164,'human','increase','null','low-throughput','butyrate enhanced the PMA-induced expression of c-fos and ERK1/2 phosphorylation, but not p38 and JNK.',23300800),(3165,'human','increase','null','low-throughput','The activation of AP-1 was strongly correlated with the presence of butyrate in the CM.',23300800),(3166,'human','increase','null','low-throughput','butyrate enhanced the PMA-induced expression of c-fos and ERK1/2 phosphorylation, but not p38 and JNK.',23300800),(3167,'human','increase','null','low-throughput','butyrate enhanced the PMA-induced expression of c-fos and ERK1/2 phosphorylation, but not p38 and JNK.',23300800),(3168,'human','increase','null','low-throughput','butyrate enhanced the PMA-induced expression of c-fos and ERK1/2 phosphorylation, but not p38 and JNK.',23300800),(3169,'human','increase','null','low-throughput','The activation of AP-1 was strongly correlated with the presence of butyrate in the CM.',23300800),(3170,'human','increase','null','low-throughput','We also measured MUC2 mRNA levels in the butyrate-treated cells, and found that 1 mM butyrate treatment for 36 h increased the cellular MUC2 mRNA levels, although the fold increase was less than that for MUC2 proteins (Fig. 3B). These results show that buty rate treatment inhibits the proliferation of LS174T cells but concomitantly increases MUC2 production per cell.',17374366),(3171,'human','decrease','null','low-throughput','Butyrate, which has inhibitory activity on many HDACs except the class III sirtuin family of HDACs, HDAC6, and HDAC10.',17374366),(3172,'human','decrease','null','low-throughput','Butyrate, which has inhibitory activity on many HDACs except the class III sirtuin family of HDACs, HDAC6, and HDAC10.',17374366),(3173,'human','decrease','null','low-throughput','Butyrate, which has inhibitory activity on many HDACs except the class III sirtuin family of HDACs, HDAC6, and HDAC10.',17374366),(3174,'human','decrease','null','low-throughput','Butyrate, which has inhibitory activity on many HDACs except the class III sirtuin family of HDACs, HDAC6, and HDAC10.',17374366),(3175,'human','decrease','null','low-throughput','Butyrate, which has inhibitory activity on many HDACs except the class III sirtuin family of HDACs, HDAC6, and HDAC10.',17374366),(3176,'human','decrease','null','low-throughput','Butyrate, which has inhibitory activity on many HDACs except the class III sirtuin family of HDACs, HDAC6, and HDAC10.',17374366),(3177,'human','decrease','null','low-throughput','Butyrate, which has inhibitory activity on many HDACs except the class III sirtuin family of HDACs, HDAC6, and HDAC10.',17374366),(3178,'human','decrease','null','low-throughput','Butyrate, which has inhibitory activity on many HDACs except the class III sirtuin family of HDACs, HDAC6, and HDAC10.',17374366),(3179,'human','decrease','null','low-throughput','Butyrate, which has inhibitory activity on many HDACs except the class III sirtuin family of HDACs, HDAC6, and HDAC10.',17374366),(3186,'human','null','Daidzein','null','One culture produced the metabolites dihydrodaidzein and O-desmethylangolensin, another produced dihydrodaidzein and equol. From the latter, a stable and transferable mixed culture transforming daidzein into equol was obtained. Molecular fingerprinting analysis (denaturing gradient gel electrophoresis) showed the presence of four bacterial species of which only the first three strains could be brought into pure culture. These strains were identified as Lactobacillus mucosae EPI2, Enterococcus faecium EPI1 and Finegoldia magna EPI3, and did not produce equol in pure culture. The fourth species was tentatively identified as Veillonella sp strain EP.',15578160),(3187,'human','null','Daidzein','null','One culture produced the metabolites dihydrodaidzein and O-desmethylangolensin, another produced dihydrodaidzein and equol. From the latter, a stable and transferable mixed culture transforming daidzein into equol was obtained. Molecular fingerprinting analysis (denaturing gradient gel electrophoresis) showed the presence of four bacterial species of which only the first three strains could be brought into pure culture. These strains were identified as Lactobacillus mucosae EPI2, Enterococcus faecium EPI1 and Finegoldia magna EPI3, and did not produce equol in pure culture. The fourth species was tentatively identified as Veillonella sp strain EP.',15578160),(3188,'human','null','Daidzein','null','One culture produced the metabolites dihydrodaidzein and O-desmethylangolensin, another produced dihydrodaidzein and equol. From the latter, a stable and transferable mixed culture transforming daidzein into equol was obtained. Molecular fingerprinting analysis (denaturing gradient gel electrophoresis) showed the presence of four bacterial species of which only the first three strains could be brought into pure culture. These strains were identified as Lactobacillus mucosae EPI2, Enterococcus faecium EPI1 and Finegoldia magna EPI3, and did not produce equol in pure culture. The fourth species was tentatively identified as Veillonella sp strain EP.',15578160),(3189,'human','null','Daidzein','null','One culture produced the metabolites dihydrodaidzein and O-desmethylangolensin, another produced dihydrodaidzein and equol. From the latter, a stable and transferable mixed culture transforming daidzein into equol was obtained. Molecular fingerprinting analysis (denaturing gradient gel electrophoresis) showed the presence of four bacterial species of which only the first three strains could be brought into pure culture. These strains were identified as Lactobacillus mucosae EPI2, Enterococcus faecium EPI1 and Finegoldia magna EPI3, and did not produce equol in pure culture. The fourth species was tentatively identified as Veillonella sp strain EP.',15578160),(3190,'human','null','Daidzein','null','One culture produced the metabolites dihydrodaidzein and O-desmethylangolensin, another produced dihydrodaidzein and equol. From the latter, a stable and transferable mixed culture transforming daidzein into equol was obtained. Molecular fingerprinting analysis (denaturing gradient gel electrophoresis) showed the presence of four bacterial species of which only the first three strains could be brought into pure culture. These strains were identified as Lactobacillus mucosae EPI2, Enterococcus faecium EPI1 and Finegoldia magna EPI3, and did not produce equol in pure culture. The fourth species was tentatively identified as Veillonella sp strain EP.',15578160),(3191,'human','null','Daidzein','null','One culture produced the metabolites dihydrodaidzein and O-desmethylangolensin, another produced dihydrodaidzein and equol. From the latter, a stable and transferable mixed culture transforming daidzein into equol was obtained. Molecular fingerprinting analysis (denaturing gradient gel electrophoresis) showed the presence of four bacterial species of which only the first three strains could be brought into pure culture. These strains were identified as Lactobacillus mucosae EPI2, Enterococcus faecium EPI1 and Finegoldia magna EPI3, and did not produce equol in pure culture. The fourth species was tentatively identified as Veillonella sp strain EP.',15578160),(3194,'human','null','null','null','Strains of Roseburia sp. and F. prausnitzii possessed butyryl coenzyme A (CoA):acetate-CoA transferase and acetate kinase activities, but butyrate kinase activity was not detectable either in growing or in stationary-phase cultures.',12324374),(3195,'human','null','null','null','Strains of Roseburia sp. and F. prausnitzii possessed butyryl coenzyme A (CoA):acetate-CoA transferase and acetate kinase activities, but butyrate kinase activity was not detectable either in growing or in stationary-phase cultures.',12324374),(3196,'human','null','null','null','Strains of Roseburia sp. and F. prausnitzii possessed butyryl coenzyme A (CoA):acetate-CoA transferase and acetate kinase activities, but butyrate kinase activity was not detectable either in growing or in stationary-phase cultures.',12324374),(3197,'human','null','null','null','On the other hand, Coprococcus sp. strain L2-50 produces acetate and possesses both butyrate kinase and butyryl-CoA transferase.',12324374),(3198,'human','null','null','null','A. muciniphila has previously been shown to produce acetate and propionate during its growth on mucin (24).',28687644),(3199,'human','null','null','null','A. muciniphila has previously been shown to produce acetate and propionate during its growth on mucin (24).',28687644),(3200,'human','null','Cianidanol','null','The conversion of both epicatechin and catechin resulted in the formation of metabolite M1 subsequently characterized as 1�?3,4‐dihydroxyphenyl)�?�?2,4,6‐trihydroxyphenyl)propan�?‐ol (3,4‐diHPP�?‐ol, structure in Fig. 3). In accordance with the initial conversion of the catechins, 3,4‐diHPP�?‐ol was formed more rapidly from catechin than from epicatechin.',21457417),(3201,'human','null','(-)-Epicatechin','null','The conversion of both epicatechin and catechin resulted in the formation of metabolite M1 subsequently characterized as 1�?3,4‐dihydroxyphenyl)�?�?2,4,6‐trihydroxyphenyl)propan�?‐ol (3,4‐diHPP�?‐ol, structure in Fig. 3). In accordance with the initial conversion of the catechins, 3,4‐diHPP�?‐ol was formed more rapidly from catechin than from epicatechin.',21457417),(3202,'human','null','1-(3,4-Dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl)-2-propanol','null','Two metabolites, M2 and M3, were first detected after 4 and 6 h, respectively, of incubation with F. plautii aK2. The concentration of M2 was maximal after 6 h and slightly decreased afterwards. The concentration of M3 increased rapidly until 8 h followed by a minor increase during further incubation. Characterization of the observed metabolites by MS analyses and taking literature data into account led to the assignment of M2 as 5�?3,4‐dihydroxyphenyl)‐γ‐valerolactone (3,4‐diHPVL) and M3 as 4‐hydroxy�?�?3,4‐dihydroxyphenyl)valeric acid (4‐H�?,4‐diHPVA) (structures in Fig. 3).',21457417),(3203,'human','null','1-(3,4-Dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl)-2-propanol','null','Two metabolites, M2 and M3, were first detected after 4 and 6 h, respectively, of incubation with F. plautii aK2. The concentration of M2 was maximal after 6 h and slightly decreased afterwards. The concentration of M3 increased rapidly until 8 h followed by a minor increase during further incubation. Characterization of the observed metabolites by MS analyses and taking literature data into account led to the assignment of M2 as 5�?3,4‐dihydroxyphenyl)‐γ‐valerolactone (3,4‐diHPVL) and M3 as 4‐hydroxy�?�?3,4‐dihydroxyphenyl)valeric acid (4‐H�?,4‐diHPVA) (structures in Fig. 3).',21457417),(3204,'human','null','null','null','Additionally,the abundances of genera Blautia (r = 0.307, p = 0.004), Bifidobacterium (r = 0.36, p = 0.001), Collinsella(r = 0.254, p = 0.018), and Lactobacillus (r = 0.428, p < 0.001) (Figure 2C) were positively correlated withthe urinary TMAO level.',30469463),(3205,'human','null','null','null','Additionally,the abundances of genera Blautia (r = 0.307, p = 0.004), Bifidobacterium (r = 0.36, p = 0.001), Collinsella(r = 0.254, p = 0.018), and Lactobacillus (r = 0.428, p < 0.001) (Figure 2C) were positively correlated withthe urinary TMAO level.',30469463),(3206,'human','null','null','null','Additionally,the abundances of genera Blautia (r = 0.307, p = 0.004), Bifidobacterium (r = 0.36, p = 0.001), Collinsella(r = 0.254, p = 0.018), and Lactobacillus (r = 0.428, p < 0.001) (Figure 2C) were positively correlated withthe urinary TMAO level.',30469463),(3207,'human','null','null','null','Additionally,the abundances of genera Blautia (r = 0.307, p = 0.004), Bifidobacterium (r = 0.36, p = 0.001), Collinsella(r = 0.254, p = 0.018), and Lactobacillus (r = 0.428, p < 0.001) (Figure 2C) were positively correlated withthe urinary TMAO level.',30469463),(3208,'human','null','null','null','The present analyses indicate that Faecalibacterium (formerly Fusobacterium) prausnitzii may make a significant contribution to  D-lactate and butyrate formation in the large intestine.',12508881),(3209,'human','null','null','null','The present analyses indicate that Faecalibacterium (formerly Fusobacterium) prausnitzii may make a significant contribution to  D-lactate and butyrate formation in the large intestine.',12508881),(3210,'human','null','null','null','The present analyses indicate that Faecalibacterium (formerly Fusobacterium) prausnitzii may make a significant contribution to  D-lactate and butyrate formation in the large intestine.',12508881),(3211,'human','null','null','null','The present analyses indicate that Faecalibacterium (formerly Fusobacterium) prausnitzii may make a significant contribution to  D-lactate and butyrate formation in the large intestine.',12508881),(3212,'human','null','null','null','The present analyses indicate that Faecalibacterium (formerly Fusobacterium) prausnitzii may make a significant contribution to  D-lactate and butyrate formation in the large intestine.',12508881),(3213,'human','null','null','null','The present analyses indicate that Faecalibacterium (formerly Fusobacterium) prausnitzii may make a significant contribution to  D-lactate and butyrate formation in the large intestine.',12508881),(3214,'human','null','null','null','The present analyses indicate that Faecalibacterium (formerly Fusobacterium) prausnitzii may make a significant contribution to  D-lactate and butyrate formation in the large intestine.',12508881),(3215,'human','null','null','null','The present analyses indicate that Faecalibacterium (formerly Fusobacterium) prausnitzii may make a significant contribution to  D-lactate and butyrate formation in the large intestine.',12508881),(3216,'human','null','null','null','A higher relative abundance of succinate-producing Prevotellaceae (P) and Veillonellaceae (V), and a lower relative abundance of succinate-consuming Odoribacteraceae (O) and Clostridaceae (C) in obese individuals, with the (P + V/O + C) ratio being a main determinant of plasma succinate.',29434314),(3217,'human','null','null','null','A higher relative abundance of succinate-producing Prevotellaceae (P) and Veillonellaceae (V), and a lower relative abundance of succinate-consuming Odoribacteraceae (O) and Clostridaceae (C) in obese individuals, with the (P + V/O + C) ratio being a main determinant of plasma succinate.',29434314),(3218,'human','null','null','null','We also found positive correlations with the butyrate producing Faecalibacterium prausnitzii, the genus Coprococcus,the class Mollicutes and several OTUs affiliated with the Ruminococcaceae as well as one from the family Chistensenellaceae.',31030641),(3219,'human','null','null','null','We also found positive correlations with the butyrate producing Faecalibacterium prausnitzii, the genus Coprococcus,the class Mollicutes and several OTUs affiliated with the Ruminococcaceae as well as one from the family Chistensenellaceae.',31030641),(3220,'human','null','null','null','We also found positive correlations with the butyrate producing Faecalibacterium prausnitzii, the genus Coprococcus,the class Mollicutes and several OTUs affiliated with the Ruminococcaceae as well as one from the family Chistensenellaceae.',31030641),(3221,'human','null','null','null','We also found positive correlations with the butyrate producing Faecalibacterium prausnitzii, the genus Coprococcus,the class Mollicutes and several OTUs affiliated with the Ruminococcaceae as well as one from the family Chistensenellaceae.',31030641),(3222,'human','null','null','null','We also found positive correlations with the butyrate producing Faecalibacterium prausnitzii, the genus Coprococcus,the class Mollicutes and several OTUs affiliated with the Ruminococcaceae as well as one from the family Chistensenellaceae.',31030641),(3223,'human','null','null','null','The strongest associations seen were positive associations with RF39 assigned to the Tenericutes phylum and negative associations with OTUs from the Lachnospiraceae family.',31030641),(3224,'human','null','null','null','The strongest associations seen were positive associations with RF39 assigned to the Tenericutes phylum and negative associations with OTUs from the Lachnospiraceae family.',31030641),(3225,'human','null','null','null','Negative correlations instead were observed with several Eubacterium species including E. dolichum that has been previously reported to be associated with increased visceral fat (Figure 1).',31030641),(3226,'human','null','null','null','Negative correlations instead were observed with several Eubacterium species including E. dolichum that has been previously reported to be associated with increased visceral fat (Figure 1).',31030641),(3227,'human','null','Chlorogenic acid','null','The substrate and nine metabolites were observed after the incubation of C-QA with the resting cells of human gut microbiota from nine different volunteers.',24550206),(3228,'human','null','Chlorogenic acid','null','The substrate and nine metabolites were observed after the incubation of C-QA with the resting cells of human gut microbiota from nine different volunteers.',24550206),(3229,'human','null','Chlorogenic acid','null','The substrate and nine metabolites were observed after the incubation of C-QA with the resting cells of human gut microbiota from nine different volunteers.',24550206),(3230,'human','null','Chlorogenic acid','null','The substrate and nine metabolites were observed after the incubation of C-QA with the resting cells of human gut microbiota from nine different volunteers.',24550206),(3231,'human','null','Chlorogenic acid','null','The substrate and nine metabolites were observed after the incubation of C-QA with the resting cells of human gut microbiota from nine different volunteers.',24550206),(3232,'human','null','Chlorogenic acid','null','The substrate and nine metabolites were observed after the incubation of C-QA with the resting cells of human gut microbiota from nine different volunteers.',24550206),(3233,'human','null','Chlorogenic acid','null','The substrate and nine metabolites were observed after the incubation of C-QA with the resting cells of human gut microbiota from nine different volunteers.',24550206),(3234,'human','null','Chlorogenic acid','null','The substrate and nine metabolites were observed after the incubation of C-QA with the resting cells of human gut microbiota from nine different volunteers.',24550206),(3235,'human','null','Chlorogenic acid','null','The substrate and nine metabolites were observed after the incubation of C-QA with the resting cells of human gut microbiota from nine different volunteers.',24550206),(3236,'human','null','Rutin','null','α-L-rhamnosidase from Bifidobacterium breve was produced by using Escherichia coli BL21 for biotransformation of rutin to isoquercitrin.',25724715),(3242,'human','null','Taxifolin','null','The isolated C. orbiscindens strain converted quercetin and taxifolin to 3,4-dihydroxyphenylacetic acid; luteolin and eriodictyol to 3-(3,4-dihydroxyphenyl)propionic acid; and apigenin, naringenin, and phloretin to 3-(4-hydroxyphenyl)propionic acid, respectively.',14532034),(3243,'human','null','Quercetin','null','The isolated C. orbiscindens strain converted quercetin and taxifolin to 3,4-dihydroxyphenylacetic acid; luteolin and eriodictyol to 3-(3,4-dihydroxyphenyl)propionic acid; and apigenin, naringenin, and phloretin to 3-(4-hydroxyphenyl)propionic acid, respectively.',14532034),(3244,'human','null','Eriodictyol','null','The isolated C. orbiscindens strain converted quercetin and taxifolin to 3,4-dihydroxyphenylacetic acid; luteolin and eriodictyol to 3-(3,4-dihydroxyphenyl)propionic acid; and apigenin, naringenin, and phloretin to 3-(4-hydroxyphenyl)propionic acid, respectively.',14532034),(3245,'human','null','Luteolin','null','The isolated C. orbiscindens strain converted quercetin and taxifolin to 3,4-dihydroxyphenylacetic acid; luteolin and eriodictyol to 3-(3,4-dihydroxyphenyl)propionic acid; and apigenin, naringenin, and phloretin to 3-(4-hydroxyphenyl)propionic acid, respectively.',14532034),(3246,'human','null','Apigenin','null','The isolated C. orbiscindens strain converted quercetin and taxifolin to 3,4-dihydroxyphenylacetic acid; luteolin and eriodictyol to 3-(3,4-dihydroxyphenyl)propionic acid; and apigenin, naringenin, and phloretin to 3-(4-hydroxyphenyl)propionic acid, respectively.',14532034),(3247,'human','null','5,7-Dihydroxy-2-(4-hydroxyphenyl)chroman-4-one','null','The isolated C. orbiscindens strain converted quercetin and taxifolin to 3,4-dihydroxyphenylacetic acid; luteolin and eriodictyol to 3-(3,4-dihydroxyphenyl)propionic acid; and apigenin, naringenin, and phloretin to 3-(4-hydroxyphenyl)propionic acid, respectively.',14532034),(3248,'human','null','Phloretin','null','The isolated C. orbiscindens strain converted quercetin and taxifolin to 3,4-dihydroxyphenylacetic acid; luteolin and eriodictyol to 3-(3,4-dihydroxyphenyl)propionic acid; and apigenin, naringenin, and phloretin to 3-(4-hydroxyphenyl)propionic acid, respectively.',14532034),(3249,'human','null','Daidzein','null','Daidzein was converted to equol by strain DZE.',18670101),(3250,'human','null','Succinate','null','PY broth tube tests indicated that strains YIT 12067T and YIT 12068 grew well only in PY medium supplemented with succinate. HPLC analysis showed that succinate was quantitatively converted to propionate. These results indicate that succinate decarboxylation stimulated the growth of strains YIT 12067T and YIT 12068.',22081579),(3251,'human','null','Succinate','null','PY broth tube tests indicated that strains YIT 12067T and YIT 12068 grew well only in PY medium supplemented with succinate. HPLC analysis showed that succinate was quantitatively converted to propionate. These results indicate that succinate decarboxylation stimulated the growth of strains YIT 12067T and YIT 12068.',22081579),(3252,'human','null','Daidzein','null','In this study, we isolated a human intestinal bacterium capable of transforming daidzein to equol and genistein to 5-hydroxy-equol.',19139227),(3253,'human','null','Genistein','null','In this study, we isolated a human intestinal bacterium capable of transforming daidzein to equol and genistein to 5-hydroxy-equol.',19139227),(3254,'human','null','Genistein 7-O-glucoside','null','Genistein-7-O-glucoside was partially transformed by way of genistein to the product 2-(4-hydroxyphenyl)-propionic acid.',11959436),(3255,'human','null','Genistein','null','Genistein was completely degraded by E. ramulus via 6\'-hydroxy-O-desmethylangolensin to 2-(4-hydroxyphenyl)-propionic acid.',11959436),(3256,'human','null','Daidzein','null','Daidzein was in part degraded to O-desmethylangolensin, the corresponding metabolite to 6\'-hydroxy-O-desmethylangolensin.',11959436),(3257,'human','null','Isoxanthohumol','null','The intestinal bacterium E. limosum was able to perform this conversion, and after strain selection, the bacterium rapidly converted all IX into 8-PN.',16076107),(3258,'human','null','null','null','In reactions using CF faecal inocula, butyrate levels were strongly correlated with the relative abundance of Clostridium sensu stricto cluster 1.',30359203),(3259,'human','null','null','null','Propionate concentrations in CF reactions were significantly correlated with the relative abundance of the Veillonella genus.',30359203),(3260,'human','null','null','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3261,'human','null','null','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3262,'human','null','null','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3263,'human','null','null','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3264,'human','null','null','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3265,'human','null','null','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3266,'human','null','null','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3267,'human','null','null','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3268,'human','null','null','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3269,'human','null','null','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3270,'human','null','null','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3271,'human','null','null','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3272,'human','null','null','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3273,'human','null','null','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3274,'human','null','null','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3275,'human','null','null','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3276,'human','null','null','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3277,'human','null','null','null','Three genera, Ruminococcus, Dorea, and Blautia which are a member of one of the most abundant families,Lachnospiraceae, are positively correlated with L-alanine, L-leucine, L-isoleucine, glycine, serine, and L-proline.',31235863),(3278,'human','null','null','null','Genus Turicibacter and species Bifdobacterium longum,Lactobacillus mucosae, Lactobacillus zeae were positively correlated with 3,4 dihydroxyhydrocinnmaic acid.',31235863),(3279,'human','null','null','null','Genus Turicibacter and species Bifdobacterium longum,Lactobacillus mucosae, Lactobacillus zeae were positively correlated with 3,4 dihydroxyhydrocinnmaic acid.',31235863),(3280,'human','null','null','null','Genus Turicibacter and species Bifdobacterium longum,Lactobacillus mucosae, Lactobacillus zeae were positively correlated with 3,4 dihydroxyhydrocinnmaic acid.',31235863),(3281,'human','null','null','null','Genus Turicibacter and species Bifdobacterium longum,Lactobacillus mucosae, Lactobacillus zeae were positively correlated with 3,4 dihydroxyhydrocinnmaic acid.',31235863),(3282,'human','null','null','null','Similarly, a positive correlation of Eubacterium with 3-hydroxybenzoic acid and 4-hydroxybenzoic acid concurs with the previous studies.',31235863),(3283,'human','null','null','null','Similarly, a positive correlation of Eubacterium with 3-hydroxybenzoic acid and 4-hydroxybenzoic acid concurs with the previous studies.',31235863),(3284,'human','null','null','null','Collinsella aerofaciens showed a positive correlation with sedoheptulose, citric acid, tricarballylic acid.',31235863),(3285,'human','null','null','null','Collinsella aerofaciens showed a positive correlation with sedoheptulose, citric acid, tricarballylic acid.',31235863),(3286,'human','null','null','null','Collinsella aerofaciens showed a positive correlation with sedoheptulose, citric acid, tricarballylic acid.',31235863),(3287,'human','null','null','null','E. coli was positively correlated with pipecolic acid.',31235863),(3288,'human','null','null','null','Genus Faecalibacterium showed a negative correlation with L-leucine and serine and species Faecalibacterium prausnitzii showed a negative correlation with serine, L-leucine, and malic acid.',31235863),(3289,'human','null','null','null','Genus Faecalibacterium showed a negative correlation with L-leucine and serine and species Faecalibacterium prausnitzii showed a negative correlation with serine, L-leucine, and malic acid.',31235863),(3290,'human','null','null','null','Genus Faecalibacterium showed a negative correlation with L-leucine and serine and species Faecalibacterium prausnitzii showed a negative correlation with serine, L-leucine, and malic acid.',31235863),(3291,'human','null','null','null','Genus Faecalibacterium showed a negative correlation with L-leucine and serine and species Faecalibacterium prausnitzii showed a negative correlation with serine, L-leucine, and malic acid.',31235863),(3292,'human','null','null','null','Genus Faecalibacterium showed a negative correlation with L-leucine and serine and species Faecalibacterium prausnitzii showed a negative correlation with serine, L-leucine, and malic acid.',31235863),(3293,'human','null','null','null','Lactobacillus ruminis showed a negative correlation with glycine.',31235863),(3294,'human','null','null','null','Genera Dialister, Catenibacterium,Turicibacter showed a positive correlation with tartaric acid and species Ruminococcus bromii, Parabacteroides distasonis, Bacteroides caccae were negatively correlated with tartaric acid.',31235863),(3295,'human','null','null','null','Genera Dialister, Catenibacterium,Turicibacter showed a positive correlation with tartaric acid and species Ruminococcus bromii, Parabacteroides distasonis, Bacteroides caccae were negatively correlated with tartaric acid.',31235863),(3296,'human','null','null','null','Genera Dialister, Catenibacterium,Turicibacter showed a positive correlation with tartaric acid and species Ruminococcus bromii, Parabacteroides distasonis, Bacteroides caccae were negatively correlated with tartaric acid.',31235863),(3297,'human','null','null','null','Genera Dialister, Catenibacterium,Turicibacter showed a positive correlation with tartaric acid and species Ruminococcus bromii, Parabacteroides distasonis, Bacteroides caccae were negatively correlated with tartaric acid.',31235863),(3298,'human','null','null','null','Genera Dialister, Catenibacterium,Turicibacter showed a positive correlation with tartaric acid and species Ruminococcus bromii, Parabacteroides distasonis, Bacteroides caccae were negatively correlated with tartaric acid.',31235863),(3299,'human','null','null','null','Genera Dialister, Catenibacterium,Turicibacter showed a positive correlation with tartaric acid and species Ruminococcus bromii, Parabacteroides distasonis, Bacteroides caccae were negatively correlated with tartaric acid.',31235863),(3300,'human','null','null','null','Genera Akkermansia, species Coprococcus catus were positively correlated with creatine and genera Mitsuokella, Weissella, Lactobacillus, species Mitsuokella multacida, Lactobacillus ruminis were negatively correlated with creatine.',31235863),(3301,'human','null','null','null','Genera Akkermansia, species Coprococcus catus were positively correlated with creatine and genera Mitsuokella, Weissella, Lactobacillus, species Mitsuokella multacida, Lactobacillus ruminis were negatively correlated with creatine.',31235863),(3302,'human','null','null','null','Genera Akkermansia, species Coprococcus catus were positively correlated with creatine and genera Mitsuokella, Weissella, Lactobacillus, species Mitsuokella multacida, Lactobacillus ruminis were negatively correlated with creatine.',31235863),(3303,'human','null','null','null','Genera Akkermansia, species Coprococcus catus were positively correlated with creatine and genera Mitsuokella, Weissella, Lactobacillus, species Mitsuokella multacida, Lactobacillus ruminis were negatively correlated with creatine.',31235863),(3304,'human','null','null','null','Genera Akkermansia, species Coprococcus catus were positively correlated with creatine and genera Mitsuokella, Weissella, Lactobacillus, species Mitsuokella multacida, Lactobacillus ruminis were negatively correlated with creatine.',31235863),(3305,'human','null','null','null','Genera Akkermansia, species Coprococcus catus were positively correlated with creatine and genera Mitsuokella, Weissella, Lactobacillus, species Mitsuokella multacida, Lactobacillus ruminis were negatively correlated with creatine.',31235863),(3306,'human','null','null','null','Genera Akkermansia, species Coprococcus catus were positively correlated with creatine and genera Mitsuokella, Weissella, Lactobacillus, species Mitsuokella multacida, Lactobacillus ruminis were negatively correlated with creatine.',31235863),(3307,'human','null','null','null','Genera Faecalibacterium, Succinivibrio, Macrococcus are negatively correlated with creatinine.',31235863),(3308,'human','null','null','null','Genera Faecalibacterium, Succinivibrio, Macrococcus are negatively correlated with creatinine.',31235863),(3309,'human','null','null','null','Genera Faecalibacterium, Succinivibrio, Macrococcus are negatively correlated with creatinine.',31235863),(3310,'human','null','null','null','Genera Bilophila, Enterococcus, Dorea,Clostridium, Phascolarctobacter and species Faecalibacterium prausnitzii, Photobacterium angustum are positively correlated with creatinine.',31235863),(3311,'human','null','null','null','Genera Bilophila, Enterococcus, Dorea,Clostridium, Phascolarctobacter and species Faecalibacterium prausnitzii, Photobacterium angustum are positively correlated with creatinine.',31235863),(3312,'human','null','null','null','Genera Bilophila, Enterococcus, Dorea,Clostridium, Phascolarctobacter and species Faecalibacterium prausnitzii, Photobacterium angustum are positively correlated with creatinine.',31235863),(3313,'human','null','null','null','Genera Bilophila, Enterococcus, Dorea,Clostridium, Phascolarctobacter and species Faecalibacterium prausnitzii, Photobacterium angustum are positively correlated with creatinine.',31235863),(3314,'human','null','null','null','Genera Bilophila, Enterococcus, Dorea,Clostridium, Phascolarctobacter and species Faecalibacterium prausnitzii, Photobacterium angustum are positively correlated with creatinine.',31235863),(3315,'human','null','null','null','Genera Bilophila, Enterococcus, Dorea,Clostridium, Phascolarctobacter and species Faecalibacterium prausnitzii, Photobacterium angustum are positively correlated with creatinine.',31235863),(3316,'human','null','null','null','Genera Escherichia, Paraprevotella, Akkermansia have a positive correlation with glycocholic acid.',31235863),(3317,'human','null','null','null','Genera Escherichia, Paraprevotella, Akkermansia have a positive correlation with glycocholic acid.',31235863),(3318,'human','null','null','null','Genera Escherichia, Paraprevotella, Akkermansia have a positive correlation with glycocholic acid.',31235863),(3319,'human','null','null','null','Genus Carnobacterium and species Ruminococcus favefaciens, Butyricicoccus pullicaecorum,Mitsuokella multacida, Bacteroides fragilis have a negative correlation with glycocholic acid.',31235863),(3320,'human','null','null','null','Genus Carnobacterium and species Ruminococcus favefaciens, Butyricicoccus pullicaecorum,Mitsuokella multacida, Bacteroides fragilis have a negative correlation with glycocholic acid.',31235863),(3321,'human','null','null','null','Genus Carnobacterium and species Ruminococcus favefaciens, Butyricicoccus pullicaecorum,Mitsuokella multacida, Bacteroides fragilis have a negative correlation with glycocholic acid.',31235863),(3322,'human','null','null','null','Genus Carnobacterium and species Ruminococcus favefaciens, Butyricicoccus pullicaecorum,Mitsuokella multacida, Bacteroides fragilis have a negative correlation with glycocholic acid.',31235863),(3323,'human','null','null','null','Genus Carnobacterium and species Ruminococcus favefaciens, Butyricicoccus pullicaecorum,Mitsuokella multacida, Bacteroides fragilis have a negative correlation with glycocholic acid.',31235863),(3324,'human','null','null','null','Genera Enterococcus,Dermacoccus, Kocuria, Roseburia and species Eggerthella lenta, Bacteroides fragilis showed a negative correlation with 3-phenylpropionic acid.',31235863),(3325,'human','null','null','null','Genera Enterococcus,Dermacoccus, Kocuria, Roseburia and species Eggerthella lenta, Bacteroides fragilis showed a negative correlation with 3-phenylpropionic acid.',31235863),(3326,'human','null','null','null','Genera Enterococcus,Dermacoccus, Kocuria, Roseburia and species Eggerthella lenta, Bacteroides fragilis showed a negative correlation with 3-phenylpropionic acid.',31235863),(3327,'human','null','null','null','Genera Enterococcus,Dermacoccus, Kocuria, Roseburia and species Eggerthella lenta, Bacteroides fragilis showed a negative correlation with 3-phenylpropionic acid.',31235863),(3328,'human','null','null','null','Genera Enterococcus,Dermacoccus, Kocuria, Roseburia and species Eggerthella lenta, Bacteroides fragilis showed a negative correlation with 3-phenylpropionic acid.',31235863),(3329,'human','null','null','null','Genera Enterococcus,Dermacoccus, Kocuria, Roseburia and species Eggerthella lenta, Bacteroides fragilis showed a negative correlation with 3-phenylpropionic acid.',31235863),(3330,'human','null','null','null','Genus Oscillospira is positively correlated with citric acid.',31235863),(3331,'human','null','null','null','Genera Acinetobacter, Microbacterium, Dysgonomonas, Bulleidia, Oscillospira and species Bacteroides caccae, Kocuria rhizophila, Pseudomonas fragi, Ruminococcus bromii were positively correlated with 4-pyridoxic acid.',31235863),(3332,'human','null','null','null','Genera Acinetobacter, Microbacterium, Dysgonomonas, Bulleidia, Oscillospira and species Bacteroides caccae, Kocuria rhizophila, Pseudomonas fragi, Ruminococcus bromii were positively correlated with 4-pyridoxic acid.',31235863),(3333,'human','null','null','null','Genera Acinetobacter, Microbacterium, Dysgonomonas, Bulleidia, Oscillospira and species Bacteroides caccae, Kocuria rhizophila, Pseudomonas fragi, Ruminococcus bromii were positively correlated with 4-pyridoxic acid.',31235863),(3334,'human','null','null','null','Genera Acinetobacter, Microbacterium, Dysgonomonas, Bulleidia, Oscillospira and species Bacteroides caccae, Kocuria rhizophila, Pseudomonas fragi, Ruminococcus bromii were positively correlated with 4-pyridoxic acid.',31235863),(3335,'human','null','null','null','Genera Acinetobacter, Microbacterium, Dysgonomonas, Bulleidia, Oscillospira and species Bacteroides caccae, Kocuria rhizophila, Pseudomonas fragi, Ruminococcus bromii were positively correlated with 4-pyridoxic acid.',31235863),(3336,'human','null','null','null','Genera Acinetobacter, Microbacterium, Dysgonomonas, Bulleidia, Oscillospira and species Bacteroides caccae, Kocuria rhizophila, Pseudomonas fragi, Ruminococcus bromii were positively correlated with 4-pyridoxic acid.',31235863),(3337,'human','null','null','null','Genera Acinetobacter, Microbacterium, Dysgonomonas, Bulleidia, Oscillospira and species Bacteroides caccae, Kocuria rhizophila, Pseudomonas fragi, Ruminococcus bromii were positively correlated with 4-pyridoxic acid.',31235863),(3338,'human','null','null','null','Genera Acinetobacter, Microbacterium, Dysgonomonas, Bulleidia, Oscillospira and species Bacteroides caccae, Kocuria rhizophila, Pseudomonas fragi, Ruminococcus bromii were positively correlated with 4-pyridoxic acid.',31235863),(3339,'human','null','null','null','Genera Acinetobacter, Microbacterium, Dysgonomonas, Bulleidia, Oscillospira and species Bacteroides caccae, Kocuria rhizophila, Pseudomonas fragi, Ruminococcus bromii were positively correlated with 4-pyridoxic acid.',31235863),(3340,'human','null','null','null','We cultured fecal specimens from healthy young adults in media containing defatted flaxseeds and detected END from the culture supernatant. Following selection through successive subcultures of the fecal microbiota with defatted flaxseeds as the only carbon source, we obtained a bacterial consortium, designated as END-49, which contained the smallest number of bacterial types still capable of metabolizing defatted flaxseeds to produce END.',20398397),(3341,'human','increase','null','low-throughput','We provide direct evidence that IS activates EGFR and downstream MAPK signaling, which stimulates AhR−ARNT nuclear translocation and results in enhanced OAT1 expression and function in the kidney.',31341083),(3342,'human','increase','null','low-throughput','We provide direct evidence that IS activates EGFR and downstream MAPK signaling, which stimulates AhR−ARNT nuclear translocation and results in enhanced OAT1 expression and function in the kidney.',31341083),(3343,'human','increase','Choline','low-throughput','Interestingly, we found that plasma TMAO was positively correlated with serum sCD14 and sCD163 levels which are biomarkers of monocyte activation and macrophage inflammation and have been reported to be associated with carotid artery atherosclerosis and CVD risk in HIV infected individuals.',29912352),(3344,'human','increase','Choline','low-throughput','Interestingly, we found that plasma TMAO was positively correlated with serum sCD14 and sCD163 levels which are biomarkers of monocyte activation and macrophage inflammation and have been reported to be associated with carotid artery atherosclerosis and CVD risk in HIV infected individuals.',29912352),(3345,'human','increase','null','low-throughput','For the PBMC assay, although all the strains tend to increase the production of IL-10 cytokine, only four strains were able to induce statistically significant increase production of this cytokine .The two most performing strains (A2-165 and 4543) belong to the phylogroup II, group B.',28713353),(3346,'human','increase','null','low-throughput','For the PBMC assay, although all the strains tend to increase the production of IL-10 cytokine, only four strains were able to induce statistically significant increase production of this cytokine .The two most performing strains (A2-165 and 4543) belong to the phylogroup II, group B.',28713353),(3347,'human','increase','null','low-throughput','For the PBMC assay, although all the strains tend to increase the production of IL-10 cytokine, only four strains were able to induce statistically significant increase production of this cytokine .The two most performing strains (A2-165 and 4543) belong to the phylogroup II, group B.',28713353),(3348,'human','increase','null','low-throughput','For the PBMC assay, although all the strains tend to increase the production of IL-10 cytokine, only four strains were able to induce statistically significant increase production of this cytokine .The two most performing strains (A2-165 and 4543) belong to the phylogroup II, group B.',28713353),(3358,'human','null','Choline','null','Further, in vitro conversion of choline to TMA was checked in presence/absence of trigonelline and in presence of trigonelline there was significant inhibition of growth of C. freundii and hence significantly reduced levels of TMA was estimated.',29980011),(3359,'human','null','Trimethylamine','null','Butin presence oftrigonelline, significantly reduced levels of TMAO was estimated which was, of course, due to flavin monooxygenase (FMO3) enzyme inhibition by trigonelline.',29980011),(3360,'human','null','Quercetin','null','The degradation of the flavonol quercetin and the flavone luteolin by Eubacterium ramulus, a strict anaerobe of the human intestinal tract, was studied. Resting cells converted these flavonoids to 3,4-dihydroxyphenylacetic acid and 3-(3,4-dihydroxyphenyl)propionic acid, respectively.',11722907),(3361,'human','null','Luteolin','null','The degradation of the flavonol quercetin and the flavone luteolin by Eubacterium ramulus, a strict anaerobe of the human intestinal tract, was studied. Resting cells converted these flavonoids to 3,4-dihydroxyphenylacetic acid and 3-(3,4-dihydroxyphenyl)propionic acid, respectively.',11722907),(3362,'human','null','Hesperidin','null','The data herein presented demonstrated that B. pseudocatenulatum can hydrolyze certain rutinose-conjugated polyphenols, such as hesperidin, thus releasing the aglycone form. Therefore, B. pseudocatenulatum may potentially contribute to the bioavailability of this class of molecules.',25875120),(3363,'human','null','Hesperidin','null','The data herein presented demonstrated that B. pseudocatenulatum can hydrolyze certain rutinose-conjugated polyphenols, such as hesperidin, thus releasing the aglycone form. Therefore, B. pseudocatenulatum may potentially contribute to the bioavailability of this class of molecules.',25875120),(3364,'human','null','Hesperidin','null','The data herein presented demonstrated that B. pseudocatenulatum can hydrolyze certain rutinose-conjugated polyphenols, such as hesperidin, thus releasing the aglycone form. Therefore, B. pseudocatenulatum may potentially contribute to the bioavailability of this class of molecules.',25875120),(3365,'human','null','Hesperidin','null','The data herein presented demonstrated that B. pseudocatenulatum can hydrolyze certain rutinose-conjugated polyphenols, such as hesperidin, thus releasing the aglycone form. Therefore, B. pseudocatenulatum may potentially contribute to the bioavailability of this class of molecules.',25875120),(3366,'human','null','Hesperidin','null','The data herein presented demonstrated that B. pseudocatenulatum can hydrolyze certain rutinose-conjugated polyphenols, such as hesperidin, thus releasing the aglycone form. Therefore, B. pseudocatenulatum may potentially contribute to the bioavailability of this class of molecules.',25875120),(3367,'human','increase','null','low-throughput','In the present study, we focused on the impact of SCFAs and the role of PPAR-γin intestinal ANGPTL4 expression. Here, we show that ANGPTL4 expression can be regulated by SCFAs both in vitro and in vivo, with butyrate being the most efficient activator.',23518684),(3368,'human','null','null','null','Faecalibacterium prausnitzii is one of the most abundant bacteria in the human gut ecosystem and it is an important supplier of butyrate to the colonic epithelium.',22357539),(3369,'human','null','Cianidanol','null','When (+)-catechin was anaerobically incubated with Eg. sp. CAT-1, and the incubation mixture was analyzed by LC/MS, two metabolites, designated CAM1 and CAM2, were detected.',23207778),(3370,'human','null','Cianidanol','null','When (+)-catechin was anaerobically incubated with Eg. sp. CAT-1, and the incubation mixture was analyzed by LC/MS, two metabolites, designated CAM1 and CAM2, were detected.',23207778),(3371,'human','decrease','null','low-throughput','On the other hand, we confirmed in both ex vivo epithelial cultures and whole mucosal samples that the inflammatory milieu, in particular the presence of TNFɑ, does impact butyrate uptake and oxidation. Indeed, TNFɑ may be responsible for the downregulation of butyrate transporters in the human intestinal epithelium observed in patient biopsies, and that may result in decreased butyrate consumption.',31211831),(3372,'human','null','Exopolysaccharides','null','In conclusion, this study revealed that relevant changes in the populations of some major anaerobic intestinal bacterial groups as well as noticeable shifts in SCFA and lactic acid production can result from the presence of bifidobacterial EPS. Microbial population dynamics correlated well and supported the SCFA and lactic acid variations detected. EPS could reach the colon by two different ways: they could be either synthesized by probiotics in food products or by intestinal bacteria in the intestinal environment, thus contributing to the modulation of the intestinal microbiota.',19735956),(3373,'human','null','null','null','These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.',18820905),(3374,'human','null','null','null','These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.',18820905),(3375,'human','null','null','null','These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.',18820905),(3376,'human','null','null','null','These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.',18820905),(3377,'human','null','null','null','These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.',18820905),(3378,'human','null','null','null','These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.',18820905),(3379,'human','null','null','null','These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.',18820905),(3380,'human','null','null','null','These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.',18820905),(3381,'human','null','null','null','These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.',18820905),(3382,'human','null','null','null','These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.',18820905),(3383,'human','null','null','null','These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.',18820905),(3384,'human','null','null','null','These results suggest that selected probiotic strains of Bifidobacterium can be used to speed up the release of daidzein, improving its bioavailability for absorption by colonic mucosa and/or biotransformation to S-equol by other intestinal microorganisms.',18820905),(3385,'human','null','null','null','We also show that E. lenta further metabolizes the dopamine produced by L-dopa decarboxylation using a distinctly microbial reaction, catechol dehydroxylation.',31196984),(3386,'human','null','null','null','Figure 3 shows the metabolites produced by pure cultures of two propionate-producing LUB strains (P-LUB; V. ratti and P.avidum), two butyrate-producing LUB strains (B-LUB; E. hallii and E. limosum); and one LUB SRB strain (D. piger) in YCFA  medium supplemented with lactose, DL-lactate, L-lactate or glucose as a sole carbon source. H2 production and  OD measurements of these pure cultures are shown in Supplementary Figs S3 and S4, respectively.',28894218),(3387,'human','null','null','null','Figure 3 shows the metabolites produced by pure cultures of two propionate-producing LUB strains (P-LUB; V. ratti and P.avidum), two butyrate-producing LUB strains (B-LUB; E. hallii and E. limosum); and one LUB SRB strain (D. piger) in YCFA  medium supplemented with lactose, DL-lactate, L-lactate or glucose as a sole carbon source. H2 production and  OD measurements of these pure cultures are shown in Supplementary Figs S3 and S4, respectively.',28894218),(3388,'human','null','null','null','Figure 3 shows the metabolites produced by pure cultures of two propionate-producing LUB strains (P-LUB; V. ratti and P.avidum), two butyrate-producing LUB strains (B-LUB; E. hallii and E. limosum); and one LUB SRB strain (D. piger) in YCFA  medium supplemented with lactose, DL-lactate, L-lactate or glucose as a sole carbon source. H2 production and  OD measurements of these pure cultures are shown in Supplementary Figs S3 and S4, respectively.',28894218),(3389,'human','null','null','null','Figure 3 shows the metabolites produced by pure cultures of two propionate-producing LUB strains (P-LUB; V. ratti and P.avidum), two butyrate-producing LUB strains (B-LUB; E. hallii and E. limosum); and one LUB SRB strain (D. piger) in YCFA  medium supplemented with lactose, DL-lactate, L-lactate or glucose as a sole carbon source. H2 production and  OD measurements of these pure cultures are shown in Supplementary Figs S3 and S4, respectively.',28894218),(3390,'human','null','null','null','Figure 3 shows the metabolites produced by pure cultures of two propionate-producing LUB strains (P-LUB; V. ratti and P.avidum), two butyrate-producing LUB strains (B-LUB; E. hallii and E. limosum); and one LUB SRB strain (D. piger) in YCFA  medium supplemented with lactose, DL-lactate, L-lactate or glucose as a sole carbon source. H2 production and  OD measurements of these pure cultures are shown in Supplementary Figs S3 and S4, respectively.',28894218),(3391,'human','null','null','null','Streptococcus sp. MRG-ICA-B isolated from human intestine were reacted with icariin under anaerobic conditions. The MRG-ICA-B and E strains hydrolyzed only the glucose moiety of icariin, and icariside II was the only metabolite.',27589718),(3392,'human','null','null','null','Enterococcus sp. MRG-ICA-E isolated from human intestine were reacted with icariin under anaerobic conditions. The MRG-ICA-B and E strains hydrolyzed only the glucose moiety of icariin, and icariside II was the only metabolite.',27589718),(3393,'human','null','null','null','Blautia sp. MRG-PMF-1 isolated from human intestine were reacted with icariin under anaerobic conditions. It not only produced icariside II  but strain metabolized icariin further to desmethylicaritin via icariside II and icaritin.',27589718),(3394,'human','null','null','null','Blautia sp. MRG-PMF-1 isolated from human intestine were reacted with icariin under anaerobic conditions. It not only produced icariside II  but strain metabolized icariin further to desmethylicaritin via icariside II and icaritin.',27589718),(3395,'human','null','Puerarin','null','Puerarin was completely converted to daidzein within 100 min by Lactococcus sp. MRG-IFC-1 and Enterococcus sp. MRG-IFC-2, which is the fastest conversion among the reported human intestinal bacteria.',25199800),(3396,'human','null','Puerarin','null','Puerarin was completely converted to daidzein within 100 min by Lactococcus sp. MRG-IFC-1 and Enterococcus sp. MRG-IFC-2, which is the fastest conversion among the reported human intestinal bacteria.',25199800),(3397,'human','null','Isoquercitrin','null','From human feces two phenotypically different types of bacteria were isolated on quercetin-3-gluco_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x0002_side as carbon and energy source. Isolates of one type were identified as strains of Enterococcus casseliflavus. The sugar moiety (4 mM) was fermented to 5.5 ± 2.1 mM formate, 2.1 ± 0.7 mM acetate,1.6 ± 0.3 mM L-lactate, and 1.3 ± 0.4 mM ethanol. The second type of isolate was identified as Eubacterium ramulus. This organism was capable of degrading the aromatic ring system. Growing cultures of Eubacterium ramulus converted 5 mM quercetin-3-glucoside to 1.7 ± 0.6 mM 3,4-dihydroxyphenylacetic acid, 7.6 ± 1.0 mM acetate, and 4.0 ± 0.4 mM butyrate.',9914304),(3398,'human','null','Isoquercitrin','null','From human feces two phenotypically different types of bacteria were isolated on quercetin-3-gluco_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x0002_side as carbon and energy source. Isolates of one type were identified as strains of Enterococcus casseliflavus. The sugar moiety (4 mM) was fermented to 5.5 ± 2.1 mM formate, 2.1 ± 0.7 mM acetate,1.6 ± 0.3 mM L-lactate, and 1.3 ± 0.4 mM ethanol. The second type of isolate was identified as Eubacterium ramulus. This organism was capable of degrading the aromatic ring system. Growing cultures of Eubacterium ramulus converted 5 mM quercetin-3-glucoside to 1.7 ± 0.6 mM 3,4-dihydroxyphenylacetic acid, 7.6 ± 1.0 mM acetate, and 4.0 ± 0.4 mM butyrate.',9914304),(3399,'human','null','Isoquercitrin','null','From human feces two phenotypically different types of bacteria were isolated on quercetin-3-gluco_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x0002_side as carbon and energy source. Isolates of one type were identified as strains of Enterococcus casseliflavus. The sugar moiety (4 mM) was fermented to 5.5 ± 2.1 mM formate, 2.1 ± 0.7 mM acetate,1.6 ± 0.3 mM L-lactate, and 1.3 ± 0.4 mM ethanol. The second type of isolate was identified as Eubacterium ramulus. This organism was capable of degrading the aromatic ring system. Growing cultures of Eubacterium ramulus converted 5 mM quercetin-3-glucoside to 1.7 ± 0.6 mM 3,4-dihydroxyphenylacetic acid, 7.6 ± 1.0 mM acetate, and 4.0 ± 0.4 mM butyrate.',9914304),(3400,'human','null','Isoquercitrin','null','From human feces two phenotypically different types of bacteria were isolated on quercetin-3-gluco_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x0002_side as carbon and energy source. Isolates of one type were identified as strains of Enterococcus casseliflavus. The sugar moiety (4 mM) was fermented to 5.5 ± 2.1 mM formate, 2.1 ± 0.7 mM acetate,1.6 ± 0.3 mM L-lactate, and 1.3 ± 0.4 mM ethanol. The second type of isolate was identified as Eubacterium ramulus. This organism was capable of degrading the aromatic ring system. Growing cultures of Eubacterium ramulus converted 5 mM quercetin-3-glucoside to 1.7 ± 0.6 mM 3,4-dihydroxyphenylacetic acid, 7.6 ± 1.0 mM acetate, and 4.0 ± 0.4 mM butyrate.',9914304),(3401,'human','null','Isoquercitrin','null','From human feces two phenotypically different types of bacteria were isolated on quercetin-3-gluco_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x0002_side as carbon and energy source. Isolates of one type were identified as strains of Enterococcus casseliflavus. The sugar moiety (4 mM) was fermented to 5.5 ± 2.1 mM formate, 2.1 ± 0.7 mM acetate,1.6 ± 0.3 mM L-lactate, and 1.3 ± 0.4 mM ethanol. The second type of isolate was identified as Eubacterium ramulus. This organism was capable of degrading the aromatic ring system. Growing cultures of Eubacterium ramulus converted 5 mM quercetin-3-glucoside to 1.7 ± 0.6 mM 3,4-dihydroxyphenylacetic acid, 7.6 ± 1.0 mM acetate, and 4.0 ± 0.4 mM butyrate.',9914304),(3402,'human','null','Isoquercitrin','null','From human feces two phenotypically different types of bacteria were isolated on quercetin-3-gluco_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x0002_side as carbon and energy source. Isolates of one type were identified as strains of Enterococcus casseliflavus. The sugar moiety (4 mM) was fermented to 5.5 ± 2.1 mM formate, 2.1 ± 0.7 mM acetate,1.6 ± 0.3 mM L-lactate, and 1.3 ± 0.4 mM ethanol. The second type of isolate was identified as Eubacterium ramulus. This organism was capable of degrading the aromatic ring system. Growing cultures of Eubacterium ramulus converted 5 mM quercetin-3-glucoside to 1.7 ± 0.6 mM 3,4-dihydroxyphenylacetic acid, 7.6 ± 1.0 mM acetate, and 4.0 ± 0.4 mM butyrate.',9914304),(3403,'human','null','Isoquercitrin','null','From human feces two phenotypically different types of bacteria were isolated on quercetin-3-gluco_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x005f_x0002_side as carbon and energy source. Isolates of one type were identified as strains of Enterococcus casseliflavus. The sugar moiety (4 mM) was fermented to 5.5 ± 2.1 mM formate, 2.1 ± 0.7 mM acetate,1.6 ± 0.3 mM L-lactate, and 1.3 ± 0.4 mM ethanol. The second type of isolate was identified as Eubacterium ramulus. This organism was capable of degrading the aromatic ring system. Growing cultures of Eubacterium ramulus converted 5 mM quercetin-3-glucoside to 1.7 ± 0.6 mM 3,4-dihydroxyphenylacetic acid, 7.6 ± 1.0 mM acetate, and 4.0 ± 0.4 mM butyrate.',9914304),(3404,'human','null','null','null','Finally, a total of seven metabolites were detected and tentatively identified, such as the demethylation metabolite (M1), deoxidation metabolites (M3, M7) and hydrolysis at the C-28 oligosaccharide metabolites (M5,M6), which were first discovered in this experiment. The results indicate that hydrolysis, demethylation, dehydroxylation, and acetylation were the major metabolic pathways of PDin vitro. Additionally, four bacterial strains from human feces including Enterococcus sp.41, Bacillus sp.46, Escherichia sp.49A and Escherichia sp.64 were detected and further identified with 16S rRNA gene sequencing due to their relatively strong metabolic capacity',27222071),(3405,'human','null','null','null','Finally, a total of seven metabolites were detected and tentatively identified, such as the demethylation metabolite (M1), deoxidation metabolites (M3, M7) and hydrolysis at the C-28 oligosaccharide metabolites (M5,M6), which were first discovered in this experiment. The results indicate that hydrolysis, demethylation, dehydroxylation, and acetylation were the major metabolic pathways of PDin vitro. Additionally, four bacterial strains from human feces including Enterococcus sp.41, Bacillus sp.46, Escherichia sp.49A and Escherichia sp.64 were detected and further identified with 16S rRNA gene sequencing due to their relatively strong metabolic capacity',27222071),(3406,'human','null','null','null','Finally, a total of seven metabolites were detected and tentatively identified, such as the demethylation metabolite (M1), deoxidation metabolites (M3, M7) and hydrolysis at the C-28 oligosaccharide metabolites (M5,M6), which were first discovered in this experiment. The results indicate that hydrolysis, demethylation, dehydroxylation, and acetylation were the major metabolic pathways of PDin vitro. Additionally, four bacterial strains from human feces including Enterococcus sp.41, Bacillus sp.46, Escherichia sp.49A and Escherichia sp.64 were detected and further identified with 16S rRNA gene sequencing due to their relatively strong metabolic capacity',27222071),(3407,'human','null','null','null','Finally, a total of seven metabolites were detected and tentatively identified, such as the demethylation metabolite (M1), deoxidation metabolites (M3, M7) and hydrolysis at the C-28 oligosaccharide metabolites (M5,M6), which were first discovered in this experiment. The results indicate that hydrolysis, demethylation, dehydroxylation, and acetylation were the major metabolic pathways of PDin vitro. Additionally, four bacterial strains from human feces including Enterococcus sp.41, Bacillus sp.46, Escherichia sp.49A and Escherichia sp.64 were detected and further identified with 16S rRNA gene sequencing due to their relatively strong metabolic capacity',27222071),(3408,'human','null','Daidzein','null','We isolated nine strains capable of metabolizing isoflavones to equol from human faeces. Seven of these strains could metabolize daidzein via dihydrodaidzein to equol, while the other two isolates could metabolize only dihydrodaidzein to equol.',18450717),(3409,'human','null','Daidzein','null','We isolated nine strains capable of metabolizing isoflavones to equol from human faeces. Seven of these strains could metabolize daidzein via dihydrodaidzein to equol, while the other two isolates could metabolize only dihydrodaidzein to equol.',18450717),(3410,'human','null','Daidzein','null','We isolated nine strains capable of metabolizing isoflavones to equol from human faeces. Seven of these strains could metabolize daidzein via dihydrodaidzein to equol, while the other two isolates could metabolize only dihydrodaidzein to equol.',18450717),(3411,'human','null','Daidzein','null','We isolated nine strains capable of metabolizing isoflavones to equol from human faeces. Seven of these strains could metabolize daidzein via dihydrodaidzein to equol, while the other two isolates could metabolize only dihydrodaidzein to equol.',18450717),(3412,'human','null','Daidzein','null','We isolated nine strains capable of metabolizing isoflavones to equol from human faeces. Seven of these strains could metabolize daidzein via dihydrodaidzein to equol, while the other two isolates could metabolize only dihydrodaidzein to equol.',18450717),(3413,'human','null','Daidzein','null','We isolated nine strains capable of metabolizing isoflavones to equol from human faeces. Seven of these strains could metabolize daidzein via dihydrodaidzein to equol, while the other two isolates could metabolize only dihydrodaidzein to equol.',18450717),(3414,'human','null','Daidzein','null','We isolated nine strains capable of metabolizing isoflavones to equol from human faeces. Seven of these strains could metabolize daidzein via dihydrodaidzein to equol, while the other two isolates could metabolize only dihydrodaidzein to equol.',18450717),(3415,'human','null','Dihydrodaidzein','null','We isolated nine strains capable of metabolizing isoflavones to equol from human faeces. Seven of these strains could metabolize daidzein via dihydrodaidzein to equol, while the other two isolates could metabolize only dihydrodaidzein to equol.',18450717),(3416,'human','null','Dihydrodaidzein','null','We isolated nine strains capable of metabolizing isoflavones to equol from human faeces. Seven of these strains could metabolize daidzein via dihydrodaidzein to equol, while the other two isolates could metabolize only dihydrodaidzein to equol.',18450717),(3425,'human','null','D-Glucose','null','Acetate is readily absorbed from the rumen and subsequently used in fatty acid metabolism.',8481001),(3426,'human','null','D-Glucose','null','Succinate and formate, on the other hand, are considered intermediates of rumen fermentation which are further metabolized by other organisms. Succinate is thought to be an important intermediate for propionate formation , with the latter being readily absorbed from the rumen for gluconeogenesis.',8481001),(3427,'human','null','D-Glucose','null','Succinate and formate, on the other hand, are considered intermediates of rumen fermentation which are further metabolized by other organisms. Formate has been shown to be metabolized to methane by rumen methanogens.',8481001),(3428,'human','null','D-Glucose','null','Lactate is also an intermediate of the overall rumen fermentation and, when present in excess, has been correlated with acidosis.',8481001),(3429,'human','null','D-Glucose','null','Lactic acid bacteria, because the bifidobacterial pathway yields 2.5 ATP molecules from 1 mol of fermented glucose, as well as 1.5 mol of acetate and 1 mol of lactate [94]. In contrast, the homofermenta tive group of lactic acid bacteria (LAB) produces 2 mol of ATP and 2 mol of lactic acid from 1 mol of glucose, whereas heterofermentative LAB produce 1 mol each of lactic acid, ethanol and ATP per 1 mol of fermented glu cose.',21484167),(3430,'human','null','D-Glucose','null','Lactic acid bacteria, because the bifidobacterial pathway yields 2.5 ATP molecules from 1 mol of fermented glucose, as well as 1.5 mol of acetate and 1 mol of lactate [94]. In contrast, the homofermenta tive group of lactic acid bacteria (LAB) produces 2 mol of ATP and 2 mol of lactic acid from 1 mol of glucose, whereas heterofermentative LAB produce 1 mol each of lactic acid, ethanol and ATP per 1 mol of fermented glu cose.',21484167),(3431,'human','null','D-Glucose','null','Lactic acid bacteria, because the bifidobacterial pathway yields 2.5 ATP molecules from 1 mol of fermented glucose, as well as 1.5 mol of acetate and 1 mol of lactate [94]. In contrast, the homofermenta tive group of lactic acid bacteria (LAB) produces 2 mol of ATP and 2 mol of lactic acid from 1 mol of glucose, whereas heterofermentative LAB produce 1 mol each of lactic acid, ethanol and ATP per 1 mol of fermented glu cose.',21484167),(3432,'human','null','null','null','Due to the fact that bifidobacteria produce lactic acid as one of their main fermentation end products, they are often included in the lactic acid bacteria (LAB).',21484167),(3433,'human','null','5,7-Dimethoxyflavone','null','Activity-guided screening for PMF-metabolizing intestinal bacteria under anaerobic conditions resulted in the isolation of a strict anaerobic bacterium, which was identified as Blautia sp. MRG-PMF1. The isolated MRGPMF1 was able to metabolize various PMFs to the corresponding demethylated flavones.The microbial conversion of bioactive 5,7-dimethoxyflavone (5,7-DMF) and 5,7,4�?trimethoxyflavone (5,7,4�?TMF) was studied in detail. 5,7-DMF and 5,7,4�?TMF were completely metabolized to 5,7-dihydroxyflavone (chrysin) and 5,7,4�?trihydroxyflavone (apigenin), respectively.',25437273),(3434,'human','null','4\',5,7-Trimethoxyflavone','null','Activity-guided screening for PMF-metabolizing intestinal bacteria under anaerobic conditions resulted in the isolation of a strict anaerobic bacterium, which was identified as Blautia sp. MRG-PMF1. The isolated MRGPMF1 was able to metabolize various PMFs to the corresponding demethylated flavones.The microbial conversion of bioactive 5,7-dimethoxyflavone (5,7-DMF) and 5,7,4�?trimethoxyflavone (5,7,4�?TMF) was studied in detail. 5,7-DMF and 5,7,4�?TMF were completely metabolized to 5,7-dihydroxyflavone (chrysin) and 5,7,4�?trihydroxyflavone (apigenin), respectively.',25437273),(3435,'human','null','Ellagic Acid','null','Four gut bacteria strains (CEBAS 4A1, 4A2, 4A3, 4A4)isolated from a healthy donor and able to produce urolithins M6, C and isourolithin A, are described in the present study.Strains were found at high concentrations (�?07 cfu g�?feces).Comparison of the 16S rRNA gene sequences of the strains showed that the four isolates belong to the same species and that they are phylogenetically members of the family Eggerthellaceae.',28824607),(3436,'human','null','Ellagic Acid','null','Four gut bacteria strains (CEBAS 4A1, 4A2, 4A3, 4A4)isolated from a healthy donor and able to produce urolithins M6, C and isourolithin A, are described in the present study.Strains were found at high concentrations (�?07 cfu g�?feces).Comparison of the 16S rRNA gene sequences of the strains showed that the four isolates belong to the same species and that they are phylogenetically members of the family Eggerthellaceae.',28824607),(3437,'human','null','Ellagic Acid','null','Four gut bacteria strains (CEBAS 4A1, 4A2, 4A3, 4A4)isolated from a healthy donor and able to produce urolithins M6, C and isourolithin A, are described in the present study.Strains were found at high concentrations (�?07 cfu g�?feces).Comparison of the 16S rRNA gene sequences of the strains showed that the four isolates belong to the same species and that they are phylogenetically members of the family Eggerthellaceae.',28824607),(3438,'human','null','Ellagic Acid','null','Four gut bacteria strains (CEBAS 4A1, 4A2, 4A3, 4A4)isolated from a healthy donor and able to produce urolithins M6, C and isourolithin A, are described in the present study.Strains were found at high concentrations (�?07 cfu g�?feces).Comparison of the 16S rRNA gene sequences of the strains showed that the four isolates belong to the same species and that they are phylogenetically members of the family Eggerthellaceae.',28824607),(3439,'human','null','null','null','Recently, we isolated a dihydrodaidzein (DHD)-producing Clostridium-like bacterium, strain TM-40, from human feces.',25045313),(3440,'human','null','DL-Alanine','null','For example, if R1 is a methyl group (-CH3), then the donor amino acid is an alanine, and the product of the oxidative deamination would be pyruvate, a central compounds of the intermediary metabolism.',22325238),(3441,'human','null','Glycine','null','For example when R2 is a hydrogen (thus the acceptor amino acid is a glycine) this product corresponds to acetyl-phosphate.',22325238),(3442,'human','increase','null','low-throughput','Having detected (ex vivo from mouse cecal contents and human stool) and identified 3-methyl-2-oxindole and 5-hydroxyindole-3-acetic acid, 3-indoleacrylic acid and indole-3 carboxylic acid as novel in vitro AHR activators.',32783770),(3443,'human','increase','null','low-throughput','Having detected (ex vivo from mouse cecal contents and human stool) and identified 3-methyl-2-oxindole and 5-hydroxyindole-3-acetic acid, 3-indoleacrylic acid and indole-3 carboxylic acid as novel in vitro AHR activators.',32783770),(3444,'human','increase','null','low-throughput','Having detected (ex vivo from mouse cecal contents and human stool) and identified 3-methyl-2-oxindole and 5-hydroxyindole-3-acetic acid, 3-indoleacrylic acid and indole-3 carboxylic acid as novel in vitro AHR activators.',32783770),(3445,'human','increase','null','low-throughput','Having detected (ex vivo from mouse cecal contents and human stool) and identified 3-methyl-2-oxindole and 5-hydroxyindole-3-acetic acid, 3-indoleacrylic acid and indole-3 carboxylic acid as novel in vitro AHR activators.',32783770),(3446,'human','increase','null','low-throughput','And yet, in Caco2 cells, indole still elicited a significant saturation of CYP1A1 expression (~15-fold, p < .001), while the remaining tryptophan metabolites elicited nonsaturable yet variable levels of CYP1A1 induction. 2-oxindole, indole-3-acetic acid, 3-methyl indole and kynurenic acid prompted significant CYP1A1 inducibility. The remaining tryptophan metabolites, however, failed to elicit significant induction.',32783770),(3447,'human','increase','null','low-throughput','And yet, in Caco2 cells, indole still elicited a significant saturation of CYP1A1 expression (~15-fold, p < .001), while the remaining tryptophan metabolites elicited nonsaturable yet variable levels of CYP1A1 induction. 2-oxindole, indole-3-acetic acid, 3-methyl indole and kynurenic acid prompted significant CYP1A1 inducibility. The remaining tryptophan metabolites, however, failed to elicit significant induction.',32783770),(3448,'human','increase','null','low-throughput','And yet, in Caco2 cells, indole still elicited a significant saturation of CYP1A1 expression (~15-fold, p < .001), while the remaining tryptophan metabolites elicited nonsaturable yet variable levels of CYP1A1 induction. 2-oxindole, indole-3-acetic acid, 3-methyl indole and kynurenic acid prompted significant CYP1A1 inducibility. The remaining tryptophan metabolites, however, failed to elicit significant induction.',32783770),(3449,'human','increase','null','low-throughput','And yet, in Caco2 cells, indole still elicited a significant saturation of CYP1A1 expression (~15-fold, p < .001), while the remaining tryptophan metabolites elicited nonsaturable yet variable levels of CYP1A1 induction. 2-oxindole, indole-3-acetic acid, 3-methyl indole and kynurenic acid prompted significant CYP1A1 inducibility. The remaining tryptophan metabolites, however, failed to elicit significant induction.',32783770),(3450,'human','increase','null','low-throughput','And yet, in Caco2 cells, indole still elicited a significant saturation of CYP1A1 expression (~15-fold, p < .001), while the remaining tryptophan metabolites elicited nonsaturable yet variable levels of CYP1A1 induction. 2-oxindole, indole-3-acetic acid, 3-methyl indole and kynurenic acid prompted significant CYP1A1 inducibility. The remaining tryptophan metabolites, however, failed to elicit significant induction.',32783770),(3451,'human','increase','null','low-throughput','These data show that B dentium–secreted products, including acetate, can modulate 5-HT production by host enterochromaffin cells.',32795610),(3452,'human','increase','null','low-throughput','Gene expression of LGR5 was increased more than 2-fold in human duodenal enteroids treated with 10 mM ACET',32240190),(3453,'human','decrease','null','low-throughput','Conversely, enteroids treated with 10 mM BUT and PROP had a greater than 10-fold decrease in LGR5 compared to controls (CON; P < 0.05; Fig 4b). Only 1mM treatment of BUT was able to significantly decrease LGR5 expression (P < 0.05; Fig 4a).',32240190),(3454,'human','decrease','null','low-throughput','Conversely, enteroids treated with 10 mM BUT and PROP had a greater than 10-fold decrease in LGR5 compared to controls (CON; P < 0.05; Fig 4b). Only 1mM treatment of BUT was able to significantly decrease LGR5 expression (P < 0.05; Fig 4a).',32240190),(3455,'human','increase','null','low-throughput','Treatment with 10 mM SCFA significantly increased ALPI expression treatments but was highest (11-fold) in enteroids treated with PROP. There was no difference in ALPI expression in enteroids treated with 1 mM SCFAs (P > 0.05).',32240190),(3456,'human','increase','null','low-throughput','Treatment with 10 mM SCFA significantly increased ALPI expression treatments but was highest (11-fold) in enteroids treated with PROP. There was no difference in ALPI expression in enteroids treated with 1 mM SCFAs (P > 0.05).',32240190),(3457,'human','increase','null','low-throughput','Treatment with 10 mM SCFA significantly increased ALPI expression treatments but was highest (11-fold) in enteroids treated with PROP. There was no difference in ALPI expression in enteroids treated with 1 mM SCFAs (P > 0.05).',32240190),(3458,'human','increase','null','low-throughput','Gene expression of MUC2 was increased 2-fold due to ACET treatment and 5-fold due to BUT treatment (P < 0.05; Fig 4b) but was not increased due to PROP treatment. There was no difference in MUC2 expression in enteroids treated with 1 mM SCFAs (P > 0.05).',32240190),(3459,'human','increase','null','low-throughput','Gene expression of MUC2 was increased 2-fold due to ACET treatment and 5-fold due to BUT treatment (P < 0.05; Fig 4b) but was not increased due to PROP treatment. There was no difference in MUC2 expression in enteroids treated with 1 mM SCFAs (P > 0.05).',32240190),(3460,'human','increase','null','low-throughput','Lysozyme (LYZ) increased due to BUT alone (P < 0.05)however there was no difference in ALPI expression in enteroids treated with 1 mM SCFAs(P > 0.05).',32240190),(3461,'human','increase','null','low-throughput','Enteroendocrine cell marker CHGA was significantly increased (170-fold) in BUT treated enteroids. There was no difference in CHGA expression in enteroids treated with 1\nmM SCFAs (P > 0.05).',32240190),(3462,'human','increase','null','low-throughput','Tuft cell marker DCLK1 was significantly elevated due to ACET(7-fold) and BUT (14-fold) treatment compared to CON (P < 0.05; Fig 4b). 1 mM ACET,BUT and PROP significantly increased DCLK1 expression (5-fold, 3.6-fold, and 5.8-fold,respectively; P < 0.05).',32240190),(3463,'human','increase','null','low-throughput','Tuft cell marker DCLK1 was significantly elevated due to ACET(7-fold) and BUT (14-fold) treatment compared to CON (P < 0.05; Fig 4b). 1 mM ACET,BUT and PROP significantly increased DCLK1 expression (5-fold, 3.6-fold, and 5.8-fold,respectively; P < 0.05).',32240190),(3464,'human','increase','null','low-throughput','Tuft cell marker DCLK1 was significantly elevated due to ACET(7-fold) and BUT (14-fold) treatment compared to CON (P < 0.05; Fig 4b). 1 mM ACET,BUT and PROP significantly increased DCLK1 expression (5-fold, 3.6-fold, and 5.8-fold,respectively; P < 0.05).',32240190),(3465,'human','increase','null','low-throughput','Anti-microbial peptide gene expression of REG3β and REG3γ (Fig 4b)were not increased due to ACET treatment, however, were significantly increased due to BUT(2.5-fold and 3-fold, respectively) compared to CON. There was no difference in REG3β and REG3γ expression in enteroids treated with 1 mM SCFAs (P > 0.05).',32240190),(3466,'human','increase','null','low-throughput','Anti-microbial peptide gene expression of REG3β and REG3γ (Fig 4b)were not increased due to ACET treatment, however, were significantly increased due to BUT(2.5-fold and 3-fold, respectively) compared to CON. There was no difference in REG3β and REG3γ expression in enteroids treated with 1 mM SCFAs (P > 0.05).',32240190),(3467,'human','increase','null','low-throughput','β-defensin 1 (DEFB1)was increased 75-fold and 66-fold respectively in BUT and PROP treated enteroids (P < 0.05; Fig4b).',32240190),(3468,'human','increase','null','low-throughput','β-defensin 1 (DEFB1)was increased 75-fold and 66-fold respectively in BUT and PROP treated enteroids (P < 0.05; Fig4b).',32240190),(3469,'human','increase','null','low-throughput','Tight junction marker expression of OCLN was only significantly elevated in ACET treated enteroids (4-fold; P < 0.05,Fig 4b).There was no difference in OCLN expression in enteroids treated with 1 mM SCFAs(P > 0.05).',32240190),(3470,'human','increase','null','low-throughput','CLDN3 and TJP1 were increased due to both ACET and BUT treatments(P < 0.05, Fig 4b). There was no difference in CLDN3 expression in enteroids treated with 1 mM SCFAs (P > 0.05), however 1 mM BUT decreased TJP1 expression more than 2-fold (P < 0.05).',32240190),(3471,'human','increase','null','low-throughput','CLDN3 and TJP1 were increased due to both ACET and BUT treatments(P < 0.05, Fig 4b). There was no difference in CLDN3 expression in enteroids treated with 1 mM SCFAs (P > 0.05), however 1 mM BUT decreased TJP1 expression more than 2-fold (P < 0.05).',32240190),(3472,'human','increase','null','low-throughput','CLDN3 and TJP1 were increased due to both ACET and BUT treatments(P < 0.05, Fig 4b). There was no difference in CLDN3 expression in enteroids treated with 1 mM SCFAs (P > 0.05), however 1 mM BUT decreased TJP1 expression more than 2-fold (P < 0.05).',32240190),(3473,'human','increase','null','low-throughput','CLDN3 and TJP1 were increased due to both ACET and BUT treatments(P < 0.05, Fig 4b). There was no difference in CLDN3 expression in enteroids treated with 1 mM SCFAs (P > 0.05), however 1 mM BUT decreased TJP1 expression more than 2-fold (P < 0.05).',32240190),(3474,'human','increase','null','low-throughput','Tight junction related marker claudin-3 (CLDN3) expression was significantly increased by 10 mM BUT and 10 mM PROP (1.5 and 1.4-fold, respectively; P < 0.05).',32240190),(3475,'human','increase','null','low-throughput','Occludin (OCLN) expression was significantly increased by 10 mM BUT treatment alone (P < 0.05).',32240190),(3476,'human','increase','null','low-throughput','Ghrelin was significantly increased by all SCFAs in cell supernatants, with BUT and PROP showing highest concentrations (P < 0.05). Ghrelin was also significantly increased by 10 mM BUT alone in basolateral media (P < 0.05; Table 1).',32240190),(3477,'human','increase','null','low-throughput','Ghrelin was significantly increased by all SCFAs in cell supernatants, with BUT and PROP showing highest concentrations (P < 0.05). Ghrelin was also significantly increased by 10 mM BUT alone in basolateral media (P < 0.05; Table 1).',32240190),(3478,'human','increase','null','low-throughput','Ghrelin was significantly increased by all SCFAs in cell supernatants, with BUT and PROP showing highest concentrations (P < 0.05). Ghrelin was also significantly increased by 10 mM BUT alone in basolateral media (P < 0.05; Table 1).',32240190),(3479,'human','increase','null','low-throughput','Interleukin 4 was significantly increased in 10 mM BUT and PROP treated 2D enteroids(P < 0.05) and was only increased significantly in basolateral media by 10 mM BUT (P < 0.05;Table 1).',32240190),(3480,'human','increase','null','low-throughput','Interleukin 4 was significantly increased in 10 mM BUT and PROP treated 2D enteroids(P < 0.05) and was only increased significantly in basolateral media by 10 mM BUT (P < 0.05;Table 1).',32240190),(3481,'human','increase','null','low-throughput','Interleukin 8 was significantly increased in cell supernatants of 10 mM BUT treated enteroids as well as basolateral media (P < 0.05). Interleukin 18 was not significantly different in cell supernatant, however, there was a tendency for 10 mM BUT to increase expression (P = 0.08;Table 1). Interleukin 18, however was increased by 10 mM BUT treatment in basolateral media(P < 0.05).',32240190),(3482,'human','null','D-Glucose','null','B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).',32873608),(3483,'human','null','D-Glucose','null','B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).',32873608),(3484,'human','null','D-Glucose','null','B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).',32873608),(3485,'human','null','D-Glucose','null','B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).',32873608),(3486,'human','null','D-Glucose','null','B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).',32873608),(3487,'human','null','D-Glucose','null','B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).',32873608),(3488,'human','null','D-Glucose','null','B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).',32873608),(3489,'human','null','D-Glucose','null','B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).',32873608),(3490,'human','null','D-Glucose','null','B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).',32873608),(3491,'human','null','D-Glucose','null','B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).',32873608),(3492,'human','null','D-Glucose','null','B. theta was grown in minimal defined medium on glucose as sole carbon and energy sources, and spent culture medium was analyzed using 1D 1H NMR to detect the secreted metabolome and to identify any new secreted metabolites (see Fig. S1 in the supplemental material). We confirmed previous observations that the major secreted metabolic products were acetate, succinate, formate, and propionate, with small amounts of lactate. In addition, we were able to detect histidine, cysteine, cystine (Cys-Cys disulfide), glutathione, asparagine, and alanine (Fig. 1).',32873608),(3493,'human','null','Levodopa','null','Here, we show that C. sporogenes deaminates levodopa, the main treatment in Parkinson’s disease, and identify the aromatic aminotransferase responsible for the initiation of the deamination pathway. The deaminate metabolite from levodopa, 3-(3,4-dihydroxyphenyl)propionic acid, elicits an inhibitory effect on ileal motility in an ex vivo model.',33076930),(3494,'human','null','5-Hydroxytryptophan','null','5-HTP was converted into two new unknown peaks (Additional File1: Fig. S2A, S2B), albeit to a much lesser extent compared to levodopa. Only the first peak could be detected and assigned by LC-MS as 5-hydroxyindole-3-lactic acid (5-HILA) by its predicted exact mass (Additional File 1:Fig. S2C).',33076930),(3495,'human','null','Tryptophan','null','Tryptophan and tyrosine were converted to their intermediates ILA (indole-3-lactic acid) and 4-HPLA (3-(4-hydroxyphenyl)lactic acid), respectively, as previously shown [11].',33076930),(3496,'human','null','Tyrosine','null','Tryptophan and tyrosine were converted to their intermediates ILA (indole-3-lactic acid) and 4-HPLA (3-(4-hydroxyphenyl)lactic acid), respectively, as previously shown [11].',33076930),(3497,'human','null','Levodopa','null','Analogous to tryptophan and tyrosine, levodopa was no longer deaminated to DHPPA but to its intermediate product 3-(3,4-dihydroxy phenyl)lactic acid (DHPLA) (Fig. 1d, e). ',33076930),(3498,'human','increase','null','low-throughput','The human ortholog HM74a is also a nicotinic acid receptor and likely has a similar role in anti-lipolysis.',15929991),(3499,'human','increase','null','low-throughput','Here we show that the fatty acid-derived ketone body (D)-OHB specifically activates PUMA-G/HM74a at concentrations observed in serum during fasting.',15929991),(3500,'human','increase','null','low-throughput','Here we show that the fatty acid-derived ketone body (D)-OHB specifically activates PUMA-G/HM74a at concentrations observed in serum during fasting.',15929991),(3501,'human','increase','null','low-throughput','Here we show that the fatty acid-derived ketone body (D)-OHB specifically activates PUMA-G/HM74a at concentrations observed in serum during fasting.',15929991),(3502,'human','increase','null','low-throughput','Both (D)- and (L)-OHB, but not acetoacetate or free acetone, elicited a Ca2 response in cells expressing HM74a but not HM74.',15929991),(3503,'human','increase','null','low-throughput','Racemic (DL)-OHB also showed some degree of activity on all receptors; however, it was more potent on HM74a and its ortholog PUMA-G  than on HM74.',15929991),(3504,'human','decrease','Diosmetin 7-O-beta-D-glucopyranoside','low-throughput','A Gram-negative anaerobic bacterium, strain 4, capable of metabolizing diosmetin-7-O-glucoside was newly isolated.Its 16S rRNA sequence displayed 99% similarity with that of Escherichia. Then strain 4 was identified as a species of thegenus Escherichia and was named Escherichia sp. 4. Additionally,  an ultraperformance liquid chromatography/quadrupole-time-of-flight mass spectrometry  technique combined with Metabolynx software method was established to screenthe metabolites of diosmetin-7-O-glucoside.',25382172),(3505,'human','null','Diosmetin 7-O-beta-D-glucopyranoside','null','A Gram-negative anaerobic bacterium, strain 4, capable of metabolizing diosmetin-7-O-glucoside was newly isolated.Its 16S rRNA sequence displayed 99% similarity with that of Escherichia. Then strain 4 was identified as a species of thegenus Escherichia and was named Escherichia sp. 4. Additionally,  an ultraperformance liquid chromatography/quadrupole-time-of-flight mass spectrometry  technique combined with Metabolynx software method was established to screenthe metabolites of diosmetin-7-O-glucoside.',25382172),(3506,'human','increase','null','low-throughput','We analyzed 50 colon adenomas from AA patients for both microsatellite instability (MSI) and the methylation status of SLC5A8 gene. This gene\'s product is involved in the transport of butyrate that has anti-proliferative properties through its effects on histone acetylation and gene expression. A proteomic analysis to check the expressed histones in adenoma and normal tissues was also performed.No association was found between SLC5A8 methylation and the MSI status. Also, there was no association between SLC5A8 methylation and the sex and age of the patients. However, there were more right sided adenomas with SLC5A8 methylation than the left sided ones. The proteomic analysis revealed distinct histone expression profiles between normal and adenoma tissues.SLC5A8 is highly methylated in AA colon adenomas which points to its potential use as a marker for early detection. The MSI rate is similar to that found in colon cancer tumors in AAs. These findings suggest that both processes stem from the same epigenetic and genetic events occurring at an early stage in colon carcinogenesis in AAs.',21687703),(3507,'human','increase','null','low-throughput','We analyzed 50 colon adenomas from AA patients for both microsatellite instability (MSI) and the methylation status of SLC5A8 gene. This gene\'s product is involved in the transport of butyrate that has anti-proliferative properties through its effects on histone acetylation and gene expression. A proteomic analysis to check the expressed histones in adenoma and normal tissues was also performed.No association was found between SLC5A8 methylation and the MSI status. Also, there was no association between SLC5A8 methylation and the sex and age of the patients. However, there were more right sided adenomas with SLC5A8 methylation than the left sided ones. The proteomic analysis revealed distinct histone expression profiles between normal and adenoma tissues.SLC5A8 is highly methylated in AA colon adenomas which points to its potential use as a marker for early detection. The MSI rate is similar to that found in colon cancer tumors in AAs. These findings suggest that both processes stem from the same epigenetic and genetic events occurring at an early stage in colon carcinogenesis in AAs.',21687703),(3508,'human','increase','null','low-throughput','We analyzed 50 colon adenomas from AA patients for both microsatellite instability (MSI) and the methylation status of SLC5A8 gene. This gene\'s product is involved in the transport of butyrate that has anti-proliferative properties through its effects on histone acetylation and gene expression. A proteomic analysis to check the expressed histones in adenoma and normal tissues was also performed.No association was found between SLC5A8 methylation and the MSI status. Also, there was no association between SLC5A8 methylation and the sex and age of the patients. However, there were more right sided adenomas with SLC5A8 methylation than the left sided ones. The proteomic analysis revealed distinct histone expression profiles between normal and adenoma tissues.SLC5A8 is highly methylated in AA colon adenomas which points to its potential use as a marker for early detection. The MSI rate is similar to that found in colon cancer tumors in AAs. These findings suggest that both processes stem from the same epigenetic and genetic events occurring at an early stage in colon carcinogenesis in AAs.',21687703),(3509,'human','null','null','null','In vitro cultivation of PD demonstrated its capacity to transform bile acids and production of succinate. Succinate supplementation in the diet decreased hyperglycemia in ob/ob mice via the activation of intestinal gluconeogenesis (IGN). Gavage with a mixture of LCA and UDCA reduced hyperlipidemia by activating the FXR pathway and repairing gut barrier integrity. Co-treatment with succinate and LCA/UDCA mirrored the benefits of LPD. The binding target of succinate was identified as fructose-1,6-bisphosphatase, the rate-limiting enzyme in IGN. The succinate and secondary bile acids produced by P. distasonis played key roles in the modulation of host metabolism.',30605678),(3510,'human','null','null','null','In vitro cultivation of PD demonstrated its capacity to transform bile acids and production of succinate. Succinate supplementation in the diet decreased hyperglycemia in ob/ob mice via the activation of intestinal gluconeogenesis (IGN). Gavage with a mixture of LCA and UDCA reduced hyperlipidemia by activating the FXR pathway and repairing gut barrier integrity. Co-treatment with succinate and LCA/UDCA mirrored the benefits of LPD. The binding target of succinate was identified as fructose-1,6-bisphosphatase, the rate-limiting enzyme in IGN. The succinate and secondary bile acids produced by P. distasonis played key roles in the modulation of host metabolism.',30605678),(3516,'human','null','Clindamycin','null','In the CDI chemostat model, clindamycin decreased valerate and deoxycholic acid concentrations and increased C difficile total viable counts (TVC) and valerate precursors, taurocholic acid, and succinate concentrations. After we stopped adding clindamycin, levels of bile acids and succinate recovered, whereas levels of valerate and valerate precursors did not. In the CDI chemostat model FMT increased valerate concentrations and decreased C difficile TVC (94% reduction), spore counts (86% reduction), and valerate precursor concentrations—concentrations of bile acids were unchanged.',30025704),(3517,'human','null','Clindamycin','null','In the CDI chemostat model, clindamycin decreased valerate and deoxycholic acid concentrations and increased C difficile total viable counts (TVC) and valerate precursors, taurocholic acid, and succinate concentrations. After we stopped adding clindamycin, levels of bile acids and succinate recovered, whereas levels of valerate and valerate precursors did not. In the CDI chemostat model FMT increased valerate concentrations and decreased C difficile TVC (94% reduction), spore counts (86% reduction), and valerate precursor concentrations—concentrations of bile acids were unchanged.',30025704),(3518,'human','null','Clindamycin','null','In the CDI chemostat model, clindamycin decreased valerate and deoxycholic acid concentrations and increased C difficile total viable counts (TVC) and valerate precursors, taurocholic acid, and succinate concentrations. After we stopped adding clindamycin, levels of bile acids and succinate recovered, whereas levels of valerate and valerate precursors did not. In the CDI chemostat model FMT increased valerate concentrations and decreased C difficile TVC (94% reduction), spore counts (86% reduction), and valerate precursor concentrations—concentrations of bile acids were unchanged.',30025704),(3519,'human','null','Clindamycin','null','In the CDI chemostat model, clindamycin decreased valerate and deoxycholic acid concentrations and increased C difficile total viable counts (TVC) and valerate precursors, taurocholic acid, and succinate concentrations. After we stopped adding clindamycin, levels of bile acids and succinate recovered, whereas levels of valerate and valerate precursors did not. In the CDI chemostat model FMT increased valerate concentrations and decreased C difficile TVC (94% reduction), spore counts (86% reduction), and valerate precursor concentrations—concentrations of bile acids were unchanged.',30025704),(3520,'human','decrease','null','low-throughput','In our study, we demonstrate that n-butyrate, a bacterial metabolite found in high abundance in the colon, regulates macrophage function through the inhibition of HDACs. n-Butyrate may behave as a microbial signal to inform the host immune system that the relative levels of n-butyrate–producing bacteria are within the desired range. In this model, HDACs act as sensors for n-butyrate, and their inhibition by high amounts of this metabolite causes intestinal macrophages to reduce production of proinflammatory mediators such as NO, IL-6, and IL-12.',24390544),(3521,'human','decrease','null','low-throughput','In our study, we demonstrate that n-butyrate, a bacterial metabolite found in high abundance in the colon, regulates macrophage function through the inhibition of HDACs. n-Butyrate may behave as a microbial signal to inform the host immune system that the relative levels of n-butyrate–producing bacteria are within the desired range. In this model, HDACs act as sensors for n-butyrate, and their inhibition by high amounts of this metabolite causes intestinal macrophages to reduce production of proinflammatory mediators such as NO, IL-6, and IL-12.',24390544),(3522,'human','decrease','null','low-throughput','In our study, we demonstrate that n-butyrate, a bacterial metabolite found in high abundance in the colon, regulates macrophage function through the inhibition of HDACs. n-Butyrate may behave as a microbial signal to inform the host immune system that the relative levels of n-butyrate–producing bacteria are within the desired range. In this model, HDACs act as sensors for n-butyrate, and their inhibition by high amounts of this metabolite causes intestinal macrophages to reduce production of proinflammatory mediators such as NO, IL-6, and IL-12.',24390544),(3523,'human','null','Resveratrol','null','In conclusion, we identified 3 microbial trans-resveratrol metabolites, 2 of which were previously unknown and we also identified 2 intestinal bacterial species that were previously not known to produce dihydroresveratrol. In addition, we showed that there can be pronounced interindividual differences in resveratrol conversion by human gut microbiota.',23283496),(3524,'human','null','Resveratrol','null','In conclusion, we identified 3 microbial trans-resveratrol metabolites, 2 of which were previously unknown and we also identified 2 intestinal bacterial species that were previously not known to produce dihydroresveratrol. In addition, we showed that there can be pronounced interindividual differences in resveratrol conversion by human gut microbiota.',23283496),(3525,'human','null','Resveratrol','null','In conclusion, we identified 3 microbial trans-resveratrol metabolites, 2 of which were previously unknown and we also identified 2 intestinal bacterial species that were previously not known to produce dihydroresveratrol. In addition, we showed that there can be pronounced interindividual differences in resveratrol conversion by human gut microbiota.',23283496),(3526,'human','null','Resveratrol','null','In conclusion, we identified 3 microbial trans-resveratrol metabolites, 2 of which were previously unknown and we also identified 2 intestinal bacterial species that were previously not known to produce dihydroresveratrol. In addition, we showed that there can be pronounced interindividual differences in resveratrol conversion by human gut microbiota.',23283496),(3527,'human','null','Resveratrol','null','In conclusion, we identified 3 microbial trans-resveratrol metabolites, 2 of which were previously unknown and we also identified 2 intestinal bacterial species that were previously not known to produce dihydroresveratrol. In addition, we showed that there can be pronounced interindividual differences in resveratrol conversion by human gut microbiota.',23283496),(3528,'human','null','Resveratrol','null','In conclusion, we identified 3 microbial trans-resveratrol metabolites, 2 of which were previously unknown and we also identified 2 intestinal bacterial species that were previously not known to produce dihydroresveratrol. In addition, we showed that there can be pronounced interindividual differences in resveratrol conversion by human gut microbiota.',23283496),(3529,'human','increase','1-((3,5-Dichloro)-2,6-dihydroxy-4-methoxyphenyl)-1-hexanone','low-throughput','DIF-3 induces proteolysis of cyclin D1 and c-Myc in HCT-116 cells via activation of GSK-3b.',25913757),(3530,'human','decrease','1-((3,5-Dichloro)-2,6-dihydroxy-4-methoxyphenyl)-1-hexanone','low-throughput','The amount of TCF7L2 increased gradually over time in control cells, but it was markedly reduced by DIF-3 in a time and dose-dependent manner.',25913757),(3531,'human','decrease','1-((3,5-Dichloro)-2,6-dihydroxy-4-methoxyphenyl)-1-hexanone','low-throughput','DIF-3 decreased the expression of cyclin D1 and cMyc to a level comparable to the reduction induced by DIF-1.',25913757),(3532,'human','decrease','1-((3,5-Dichloro)-2,6-dihydroxy-4-methoxyphenyl)-1-hexanone','low-throughput','DIF-3 decreased the expression of cyclin D1 and cMyc to a level comparable to the reduction induced by DIF-1.',25913757),(3533,'human','decrease','null','low-throughput','Butyrate-induced antimicrobial activity was associated with a shift in macrophage metabolism, a reduction in mTOR kinase activity, increased LC3-associated host defense and anti-microbial peptide production in the absence of an increased inflammatory cytokine response. Butyrate drove this monocyte to macrophage differentiation program through histone deacetylase 3 (HDAC3) inhibition. Administration of butyrate induced antimicrobial activity in intestinal macrophages in vivo and increased resistance to enteropathogens.',30683619),(3534,'human','decrease','null','low-throughput','Butyrate-induced antimicrobial activity was associated with a shift in macrophage metabolism, a reduction in mTOR kinase activity, increased LC3-associated host defense and anti-microbial peptide production in the absence of an increased inflammatory cytokine response. Butyrate drove this monocyte to macrophage differentiation program through histone deacetylase 3 (HDAC4) inhibition. Administration of butyrate induced antimicrobial activity in intestinal macrophages in vivo and increased resistance to enteropathogens.',30683619),(3535,'human','decrease','null','low-throughput','Butyrate-induced antimicrobial activity was associated with a shift in macrophage metabolism, a reduction in mTOR kinase activity, increased LC3-associated host defense and anti-microbial peptide production in the absence of an increased inflammatory cytokine response. Butyrate drove this monocyte to macrophage differentiation program through histone deacetylase 3 (HDAC5) inhibition. Administration of butyrate induced antimicrobial activity in intestinal macrophages in vivo and increased resistance to enteropathogens.',30683619),(3536,'human','null','Xyloglucan Heptasaccharide','null','Commensal bifidobacteria colonize the human gastrointestinal tract and catabolize glycans that are impervious to host digestion. Accordingly, Bifidobacterium longum typically secretes acetate and lactate as fermentative end products. This study tested the hypothesis that B. longum utilizes cranberry-derived xyloglucans in a strain-dependent manner. Interestingly, the B. longum strain that efficiently utilizes cranberry xyloglucans secretes 2.0 to 2.5 mol of acetate-lactate. The 1.5 acetate:lactate ratio theoretical yield obtained in hexose fermentations shifts during xyloglucan metabolism. Accordingly, this metabolic shift is characterized by increased acetate and formate production at the expense of lactate. α-l-Arabinofuranosidase, an arabinan endo-1,5-α-l-arabinosidase, and a β-xylosidase with a carbohydrate substrate-binding protein and carbohydrate ABC transporter membrane proteins are upregulated (>2-fold change), which suggests carbon flux through this catabolic pathway. Finally, syntrophic interactions occurred with strains that utilize carbohydrate products derived from initial degradation from heterologous bacteria.',28667113),(3537,'human','null','Xyloglucan Heptasaccharide','null','Commensal bifidobacteria colonize the human gastrointestinal tract and catabolize glycans that are impervious to host digestion. Accordingly, Bifidobacterium longum typically secretes acetate and lactate as fermentative end products. This study tested the hypothesis that B. longum utilizes cranberry-derived xyloglucans in a strain-dependent manner. Interestingly, the B. longum strain that efficiently utilizes cranberry xyloglucans secretes 2.0 to 2.5 mol of acetate-lactate. The 1.5 acetate:lactate ratio theoretical yield obtained in hexose fermentations shifts during xyloglucan metabolism. Accordingly, this metabolic shift is characterized by increased acetate and formate production at the expense of lactate. α-l-Arabinofuranosidase, an arabinan endo-1,5-α-l-arabinosidase, and a β-xylosidase with a carbohydrate substrate-binding protein and carbohydrate ABC transporter membrane proteins are upregulated (>2-fold change), which suggests carbon flux through this catabolic pathway. Finally, syntrophic interactions occurred with strains that utilize carbohydrate products derived from initial degradation from heterologous bacteria.',28667113),(3538,'human','null','null','null','We have presented direct evidence that the gut bacterial genotoxin colibactin alkylates DNA in vivo. The ability of pks+ E. coli to generate DNA adducts in mammalian cells and in mice strengthens support for the involvement of colibactin in cancer development or progression since misrepaired mono- and cross-linked adducts may generate mutations in oncogenes or tumor suppressor genes, contributing to tumorigenesis . Our findings will enable efforts to decipher the molecular details of this process. Importantly, this work has also uncovered a candidate metabolite biomarker of colibactin exposure and cancer risk. The ability to directly assess whether exposure to colibactin has occurred in animal models and human patients will help address the critical question of whether pks+ E. coli contribute to colorectal carcinogenesis in patient cohorts .',30765538),(3539,'human','null','null','null','We have presented direct evidence that the gut bacterial genotoxin colibactin alkylates DNA in vivo. The ability of pks+ E. coli to generate DNA adducts in mammalian cells and in mice strengthens support for the involvement of colibactin in cancer development or progression since misrepaired mono- and cross-linked adducts may generate mutations in oncogenes or tumor suppressor genes, contributing to tumorigenesis . Our findings will enable efforts to decipher the molecular details of this process. Importantly, this work has also uncovered a candidate metabolite biomarker of colibactin exposure and cancer risk. The ability to directly assess whether exposure to colibactin has occurred in animal models and human patients will help address the critical question of whether pks+ E. coli contribute to colorectal carcinogenesis in patient cohorts .',30765538),(3542,'human','null','Soybean Isoflavones','null','An equol-producing bacterium, strain DZET , which was isolated from human faeces, was characterized by morphological, biochemical and molecular methods. The isolate was Grampositive, obligately anaerobic, non-spore forming, asaccharolytic and rod-shaped. 16S rRNA sequence analysis showed 92.8, 91.0, 91.1 and 90.6 % similarities with Slackia faecicanis, Slackia exigua, Slackia heliotrinireducens and Slackia isoflavoniconvertens, respectively. Basedon these data, we propose a novel species of the genus Slackia, Slackia equolifaciens sp. nov.The major cellular fatty acids are C14 : 0, C18 : 1v9c and C18 : 1v9c DMA (dimethyl acetal). The DNA G+C content of the strain is 60.8 mol%. The type strain of S.equolifaciens sp. nov. is DZE.',19734283),(3543,'human','null','Daidzein','null','Equol and 5-hydroxy-equol were found in intestinal contents, feces, and urine of SIA rats but not in the corresponding samples of the control rats. 5-Hydroxy-equol was present at much lower concentrations than equol and the main metabolite produced from genistein was the intermediate dihydrogenistein. The plasma of SIA rats contained equol but no 5-hydroxy-equol. Equol formation had no effect on plasma concentrations of the insulin-like growth factor I. The concentrations of daidzein and genistein were considerably lower in all samples of the SIA rats than in those of the control rats. Male SIA rats had higher intestinal and fecal concentrations of the isoflavones and their metabolites than female SIA rats. The observed activity in the rat model indicates that S. isoflavoniconvertens is capable of contributing in vivo to the bioactivation of daidzein and genistein by formation of equol and 5-hydroxy-equol.',22113864),(3544,'human','null','Genistein','null','Equol and 5-hydroxy-equol were found in intestinal contents, feces, and urine of SIA rats but not in the corresponding samples of the control rats. 5-Hydroxy-equol was present at much lower concentrations than equol and the main metabolite produced from genistein was the intermediate dihydrogenistein. The plasma of SIA rats contained equol but no 5-hydroxy-equol. Equol formation had no effect on plasma concentrations of the insulin-like growth factor I. The concentrations of daidzein and genistein were considerably lower in all samples of the SIA rats than in those of the control rats. Male SIA rats had higher intestinal and fecal concentrations of the isoflavones and their metabolites than female SIA rats. The observed activity in the rat model indicates that S. isoflavoniconvertens is capable of contributing in vivo to the bioactivation of daidzein and genistein by formation of equol and 5-hydroxy-equol.',22113864),(3545,'human','null','null','null','The alignment results showed that homologous proteins for FOS transporters and glycosidases were distributed in 237 of the 453 strains of gut bacteria. La506 msmK encoding the ATP-binding protein and Aec45 fosGH1 encoding glycoside hydrolase were most widely distributed, in 155 and 55 strains, respectively. Seven of eight strains with both transporters and glycosidases were proven to be capable of metabolizing FOS, while five strains without either transporters or glycosidases were not. Fifteen species isolated from human feces and 11 species from the alignment search were identified to be FOS-metabolizing, of which Cronobacter sakazakii, Marvinbryantia formatexigens, Ruminococcus gnavus, and Weissella paramesenteroides are reported here for the first time.',25653214),(3546,'human','null','null','null','The alignment results showed that homologous proteins for FOS transporters and glycosidases were distributed in 237 of the 453 strains of gut bacteria. La506 msmK encoding the ATP-binding protein and Aec45 fosGH1 encoding glycoside hydrolase were most widely distributed, in 155 and 55 strains, respectively. Seven of eight strains with both transporters and glycosidases were proven to be capable of metabolizing FOS, while five strains without either transporters or glycosidases were not. Fifteen species isolated from human feces and 11 species from the alignment search were identified to be FOS-metabolizing, of which Cronobacter sakazakii, Marvinbryantia formatexigens, Ruminococcus gnavus, and Weissella paramesenteroides are reported here for the first time.',25653214),(3547,'human','null','null','null','The alignment results showed that homologous proteins for FOS transporters and glycosidases were distributed in 237 of the 453 strains of gut bacteria. La506 msmK encoding the ATP-binding protein and Aec45 fosGH1 encoding glycoside hydrolase were most widely distributed, in 155 and 55 strains, respectively. Seven of eight strains with both transporters and glycosidases were proven to be capable of metabolizing FOS, while five strains without either transporters or glycosidases were not. Fifteen species isolated from human feces and 11 species from the alignment search were identified to be FOS-metabolizing, of which Cronobacter sakazakii, Marvinbryantia formatexigens, Ruminococcus gnavus, and Weissella paramesenteroides are reported here for the first time.',25653214),(3548,'human','null','null','null','The alignment results showed that homologous proteins for FOS transporters and glycosidases were distributed in 237 of the 453 strains of gut bacteria. La506 msmK encoding the ATP-binding protein and Aec45 fosGH1 encoding glycoside hydrolase were most widely distributed, in 155 and 55 strains, respectively. Seven of eight strains with both transporters and glycosidases were proven to be capable of metabolizing FOS, while five strains without either transporters or glycosidases were not. Fifteen species isolated from human feces and 11 species from the alignment search were identified to be FOS-metabolizing, of which Cronobacter sakazakii, Marvinbryantia formatexigens, Ruminococcus gnavus, and Weissella paramesenteroides are reported here for the first time.',25653214),(3549,'human','null','Daidzin','null','Fecal bacteria from a healthy individual were screened for the specific bacteria involved in the metabolism of dietary isoflavonoids. Two strains of bacteria capable of producing primary and secondary metabolites from the natural isoflavone glycosides daidzin and genistin were detected. The metabolites were identified by comparison of their HPLC/mass, 1H NMR and UV spectra with those of standard and synthetic compounds. Both Escherichia coli HGH21 and the gram-positive strain HGH6 converted daidzin and genistin to the their respective aglycones daidzein and genistein. Under anoxic conditions, strain HGH6 further metabolized the isoflavones daidzein and genistein to dihydrodaidzein and dihydrogenistein, respectively.',11195098),(3550,'human','null','Genistin','null','Fecal bacteria from a healthy individual were screened for the specific bacteria involved in the metabolism of dietary isoflavonoids. Two strains of bacteria capable of producing primary and secondary metabolites from the natural isoflavone glycosides daidzin and genistin were detected. The metabolites were identified by comparison of their HPLC/mass, 1H NMR and UV spectra with those of standard and synthetic compounds. Both Escherichia coli HGH21 and the gram-positive strain HGH6 converted daidzin and genistin to the their respective aglycones daidzein and genistein. Under anoxic conditions, strain HGH6 further metabolized the isoflavones daidzein and genistein to dihydrodaidzein and dihydrogenistein, respectively.',11195098),(3551,'human','null','Daidzin','null','Fecal bacteria from a healthy individual were screened for the specific bacteria involved in the metabolism of dietary isoflavonoids. Two strains of bacteria capable of producing primary and secondary metabolites from the natural isoflavone glycosides daidzin and genistin were detected. The metabolites were identified by comparison of their HPLC/mass, 1H NMR and UV spectra with those of standard and synthetic compounds. Both Escherichia coli HGH21 and the gram-positive strain HGH6 converted daidzin and genistin to the their respective aglycones daidzein and genistein. Under anoxic conditions, strain HGH6 further metabolized the isoflavones daidzein and genistein to dihydrodaidzein and dihydrogenistein, respectively.',11195098),(3552,'human','null','Genistin','null','Fecal bacteria from a healthy individual were screened for the specific bacteria involved in the metabolism of dietary isoflavonoids. Two strains of bacteria capable of producing primary and secondary metabolites from the natural isoflavone glycosides daidzin and genistin were detected. The metabolites were identified by comparison of their HPLC/mass, 1H NMR and UV spectra with those of standard and synthetic compounds. Both Escherichia coli HGH21 and the gram-positive strain HGH6 converted daidzin and genistin to the their respective aglycones daidzein and genistein. Under anoxic conditions, strain HGH6 further metabolized the isoflavones daidzein and genistein to dihydrodaidzein and dihydrogenistein, respectively.',11195098),(3553,'human','null','Daidzein','null','Fecal bacteria from a healthy individual were screened for the specific bacteria involved in the metabolism of dietary isoflavonoids. Two strains of bacteria capable of producing primary and secondary metabolites from the natural isoflavone glycosides daidzin and genistin were detected. The metabolites were identified by comparison of their HPLC/mass, 1H NMR and UV spectra with those of standard and synthetic compounds. Both Escherichia coli HGH21 and the gram-positive strain HGH6 converted daidzin and genistin to the their respective aglycones daidzein and genistein. Under anoxic conditions, strain HGH6 further metabolized the isoflavones daidzein and genistein to dihydrodaidzein and dihydrogenistein, respectively.',11195098),(3554,'human','null','Genistein','null','Fecal bacteria from a healthy individual were screened for the specific bacteria involved in the metabolism of dietary isoflavonoids. Two strains of bacteria capable of producing primary and secondary metabolites from the natural isoflavone glycosides daidzin and genistin were detected. The metabolites were identified by comparison of their HPLC/mass, 1H NMR and UV spectra with those of standard and synthetic compounds. Both Escherichia coli HGH21 and the gram-positive strain HGH6 converted daidzin and genistin to the their respective aglycones daidzein and genistein. Under anoxic conditions, strain HGH6 further metabolized the isoflavones daidzein and genistein to dihydrodaidzein and dihydrogenistein, respectively.',11195098),(3555,'human','increase','null','high-throughput','Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.',31123037),(3556,'human','increase','null','high-throughput','Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.',31123037),(3557,'human','increase','null','high-throughput','Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.',31123037),(3558,'human','increase','null','high-throughput','Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.',31123037),(3559,'human','increase','null','high-throughput','Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.',31123037),(3560,'human','increase','null','high-throughput','Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.',31123037),(3561,'human','increase','null','high-throughput','Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.',31123037),(3562,'human','increase','null','high-throughput','Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.',31123037),(3563,'human','increase','null','high-throughput','Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.',31123037),(3564,'human','increase','null','high-throughput','Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.',31123037),(3565,'human','increase','null','high-throughput','Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.',31123037),(3573,'human','null','D-Glucose','null','Two anaerobic, non-spore-forming, non-motile, Gram-negative-staining bacteria, strains YIT 12060T and YIT 12061T, were isolated from human faeces. Cells of strain YIT 12060T were coccoid to rod-shaped with round ends, positive for catalase, negative for indole and oxidase production, produced succinic and acetic acids as end products of glucose metabolism in peptone/yeast extract/glucose medium and had a DNA G+C content of 55.2 mol%. Phylogenetic analysis and biochemical data supported the affiliation of strain YIT 12060T to the genus Alistipes of the family ‘Rikenellaceae�? Strain YIT 12060T therefore represents a novel species of the genus Alistipes for which the name Alistipes indistinctus sp. nov. is proposed; the type strain is YIT 12060T.',19667375),(3574,'human','null','D-Glucose','null','Two anaerobic, non-spore-forming, non-motile, Gram-negative-staining bacteria, strains YIT 12060T and YIT 12061T, were isolated from human faeces. Cells of strain YIT 12060T were coccoid to rod-shaped with round ends, positive for catalase, negative for indole and oxidase production, produced succinic and acetic acids as end products of glucose metabolism in peptone/yeast extract/glucose medium and had a DNA G+C content of 55.2 mol%. Phylogenetic analysis and biochemical data supported the affiliation of strain YIT 12060T to the genus Alistipes of the family ‘Rikenellaceae�? Strain YIT 12060T therefore represents a novel species of the genus Alistipes for which the name Alistipes indistinctus sp. nov. is proposed; the type strain is YIT 12060T.',19667375),(3575,'human','null','D-Glucose','null','Two anaerobic, non-spore-forming, non-motile, Gram-negative-staining bacteria, strains YIT 12060T and YIT 12061T, were isolated from human faeces. Cells of strain YIT 12060T were coccoid to rod-shaped with round ends, positive for catalase, negative for indole and oxidase production, produced succinic and acetic acids as end products of glucose metabolism in peptone/yeast extract/glucose medium and had a DNA G+C content of 55.2 mol%. Phylogenetic analysis and biochemical data supported the affiliation of strain YIT 12060T to the genus Alistipes of the family ‘Rikenellaceae�? Strain YIT 12060T therefore represents a novel species of the genus Alistipes for which the name Alistipes indistinctus sp. nov. is proposed; the type strain is YIT 12060T.',19667375),(3576,'human','null','D-Glucose','null','Two anaerobic, non-spore-forming, non-motile, Gram-negative-staining bacteria, strains YIT 12060T and YIT 12061T, were isolated from human faeces. Cells of strain YIT 12060T were coccoid to rod-shaped with round ends, positive for catalase, negative for indole and oxidase production, produced succinic and acetic acids as end products of glucose metabolism in peptone/yeast extract/glucose medium and had a DNA G+C content of 55.2 mol%. Phylogenetic analysis and biochemical data supported the affiliation of strain YIT 12060T to the genus Alistipes of the family ‘Rikenellaceae�? Strain YIT 12060T therefore represents a novel species of the genus Alistipes for which the name Alistipes indistinctus sp. nov. is proposed; the type strain is YIT 12060T.',19667375),(3577,'human','null','D-Glucose','null','Two anaerobic, non-spore-forming, non-motile, Gram-negative-staining bacteria, strains YIT 12060T and YIT 12061T, were isolated from human faeces. Cells of strain YIT 12060T were coccoid to rod-shaped with round ends, positive for catalase, negative for indole and oxidase production, produced succinic and acetic acids as end products of glucose metabolism in peptone/yeast extract/glucose medium and had a DNA G+C content of 55.2 mol%. Phylogenetic analysis and biochemical data supported the affiliation of strain YIT 12060T to the genus Alistipes of the family ‘Rikenellaceae�? Strain YIT 12060T therefore represents a novel species of the genus Alistipes for which the name Alistipes indistinctus sp. nov. is proposed; the type strain is YIT 12060T.',19667375),(3578,'human','null','D-Glucose','null','Two anaerobic, non-spore-forming, non-motile, Gram-negative-staining bacteria, strains YIT 12060T and YIT 12061T, were isolated from human faeces. Cells of strain YIT 12060T were coccoid to rod-shaped with round ends, positive for catalase, negative for indole and oxidase production, produced succinic and acetic acids as end products of glucose metabolism in peptone/yeast extract/glucose medium and had a DNA G+C content of 55.2 mol%. Phylogenetic analysis and biochemical data supported the affiliation of strain YIT 12060T to the genus Alistipes of the family ‘Rikenellaceae�? Strain YIT 12060T therefore represents a novel species of the genus Alistipes for which the name Alistipes indistinctus sp. nov. is proposed; the type strain is YIT 12060T.',19667375),(3579,'human','null','D-Glucose','null','Two anaerobic, non-spore-forming, pleomorphic, Gram-negative rods, designated YIT 11840T and YIT 11841T, were isolated from human faeces. The organisms were catalase-negative, produced succinic and acetic acids as end products of glucose metabolism and had DNA G+C contents of approximately 48�?9 mol%. Although the phenotypic characteristics of these two strains were very similar, analysis of their 16S rRNA sequence showed that they are only distantly related (93.8 %), indicating that they represent two different species. A comparative sequence analysis revealed that these two species are members of the family ‘Prevotellaceae�?but are phylogenetically distant (<88 % sequence similarity) from the known genera belonging to this family, including Prevotella, Hallela and Xylanibacter.',19567577),(3580,'human','null','D-Glucose','null','Two anaerobic, non-spore-forming, pleomorphic, Gram-negative rods, designated YIT 11840T and YIT 11841T, were isolated from human faeces. The organisms were catalase-negative, produced succinic and acetic acids as end products of glucose metabolism and had DNA G+C contents of approximately 48�?9 mol%. Although the phenotypic characteristics of these two strains were very similar, analysis of their 16S rRNA sequence showed that they are only distantly related (93.8 %), indicating that they represent two different species. A comparative sequence analysis revealed that these two species are members of the family ‘Prevotellaceae�?but are phylogenetically distant (<88 % sequence similarity) from the known genera belonging to this family, including Prevotella, Hallela and Xylanibacter.',19567577),(3581,'human','null','D-Glucose','null','Two anaerobic, non-spore-forming, pleomorphic, Gram-negative rods, designated YIT 11840T and YIT 11841T, were isolated from human faeces. The organisms were catalase-negative, produced succinic and acetic acids as end products of glucose metabolism and had DNA G+C contents of approximately 48�?9 mol%. Although the phenotypic characteristics of these two strains were very similar, analysis of their 16S rRNA sequence showed that they are only distantly related (93.8 %), indicating that they represent two different species. A comparative sequence analysis revealed that these two species are members of the family ‘Prevotellaceae�?but are phylogenetically distant (<88 % sequence similarity) from the known genera belonging to this family, including Prevotella, Hallela and Xylanibacter.',19567577),(3582,'human','null','D-Glucose','null','Two anaerobic, non-spore-forming, pleomorphic, Gram-negative rods, designated YIT 11840T and YIT 11841T, were isolated from human faeces. The organisms were catalase-negative, produced succinic and acetic acids as end products of glucose metabolism and had DNA G+C contents of approximately 48�?9 mol%. Although the phenotypic characteristics of these two strains were very similar, analysis of their 16S rRNA sequence showed that they are only distantly related (93.8 %), indicating that they represent two different species. A comparative sequence analysis revealed that these two species are members of the family ‘Prevotellaceae�?but are phylogenetically distant (<88 % sequence similarity) from the known genera belonging to this family, including Prevotella, Hallela and Xylanibacter.',19567577),(3583,'human','null','null','null','F. prausnitzii modulated the metabolome of the host blood by decreasing IL-1α, IL-2, and IL-5 concentrations.',25900655),(3584,'human','null','null','null','F. prausnitzii modulated the metabolome of the host blood by decreasing IL-1α, IL-2, and IL-5 concentrations.',25900655),(3585,'human','null','null','null','F. prausnitzii modulated the metabolome of the host blood by decreasing IL-1α, IL-2, and IL-5 concentrations.',25900655),(3589,'human','decrease','Retinol','low-throughput','Retinoic acid (RA), a vitamin A metabolite, regulates transcriptional programs that drive protective or pathogenic immune responses in the intestine, in a manner dependent on RA concentration. Vitamin A is obtained from diet and is metabolized by intestinal epithelial cells (IECs), which operate in intimate association with microbes and immune cells. Here we found that commensal bacteria belonging to class Clostridia modulate RA concentration in the gut by suppressing the expression of retinol dehydrogenase 7 (Rdh7) in IECs. Rdh7 expression and associated RA amounts were lower in the intestinal tissue of conventional mice, as compared to germ-free mice. Deletion of Rdh7 in IECs diminished RA signaling in immune cells, reduced the IL-22-dependent antimicrobial response, and enhanced resistance to colonization by Salmonella Typhimurium. Our findings define a regulatory circuit wherein bacterial regulation of IEC-intrinsic RA synthesis protects microbial communities in the gut from excessive immune activity, achieving a balance that prevents colonization by enteric pathogens.',30566883),(3590,'human','increase','Retinol','low-throughput','Retinoic acid (RA), a vitamin A metabolite, regulates transcriptional programs that drive protective or pathogenic immune responses in the intestine, in a manner dependent on RA concentration. Vitamin A is obtained from diet and is metabolized by intestinal epithelial cells (IECs), which operate in intimate association with microbes and immune cells. Here we found that commensal bacteria belonging to class Clostridia modulate RA concentration in the gut by suppressing the expression of retinol dehydrogenase 7 (Rdh7) in IECs. Rdh7 expression and associated RA amounts were lower in the intestinal tissue of conventional mice, as compared to germ-free mice. Deletion of Rdh7 in IECs diminished RA signaling in immune cells, reduced the IL-22-dependent antimicrobial response, and enhanced resistance to colonization by Salmonella Typhimurium. Our findings define a regulatory circuit wherein bacterial regulation of IEC-intrinsic RA synthesis protects microbial communities in the gut from excessive immune activity, achieving a balance that prevents colonization by enteric pathogens.',30566883),(3591,'human','increase','Retinol','low-throughput','Retinoic acid (RA), a vitamin A metabolite, regulates transcriptional programs that drive protective or pathogenic immune responses in the intestine, in a manner dependent on RA concentration. Vitamin A is obtained from diet and is metabolized by intestinal epithelial cells (IECs), which operate in intimate association with microbes and immune cells. Here we found that commensal bacteria belonging to class Clostridia modulate RA concentration in the gut by suppressing the expression of retinol dehydrogenase 7 (Rdh7) in IECs. Rdh7 expression and associated RA amounts were lower in the intestinal tissue of conventional mice, as compared to germ-free mice. Deletion of Rdh7 in IECs diminished RA signaling in immune cells, reduced the IL-22-dependent antimicrobial response, and enhanced resistance to colonization by Salmonella Typhimurium. Our findings define a regulatory circuit wherein bacterial regulation of IEC-intrinsic RA synthesis protects microbial communities in the gut from excessive immune activity, achieving a balance that prevents colonization by enteric pathogens.',30566883),(3596,'human','null','Trimethylamine','null','We now show gut microbes, through generation of trimethylamine N-oxide (TMAO), directly contribute to platelet hyperreactivity and enhanced thrombosis potential. Plasma TMAO levels in subjects (n > 4,000) independently predicted incident (3 years) thrombosis (heart attack, stroke) risk. Direct exposure of platelets to TMAO enhanced sub maximal stimulus-dependent platelet activation from multiple agonists through augmented Ca2+ release from intracellular stores. Animal model studies employing dietary choline or TMAO, germ-free mice, and microbial transplantation collectively confirm a role for gut microbiota and TMAO in modulating platelet hyperresponsiveness and thrombosis potential and identify microbial taxa associated with plasma TMAO and thrombosis potential.',26972052),(3597,'human','null','Trimethylamine','null','We now show gut microbes, through generation of trimethylamine N-oxide (TMAO), directly contribute to platelet hyperreactivity and enhanced thrombosis potential. Plasma TMAO levels in subjects (n > 4,000) independently predicted incident (3 years) thrombosis (heart attack, stroke) risk. Direct exposure of platelets to TMAO enhanced sub maximal stimulus-dependent platelet activation from multiple agonists through augmented Ca2+ release from intracellular stores. Animal model studies employing dietary choline or TMAO, germ-free mice, and microbial transplantation collectively confirm a role for gut microbiota and TMAO in modulating platelet hyperresponsiveness and thrombosis potential and identify microbial taxa associated with plasma TMAO and thrombosis potential.',26972052),(3609,'human','increase','Linoleic acid','low-throughput','In this study, we investigated the effects of LA and LA-derived fatty acids on the activation of peroxisome proliferator-activated receptors (PPARs) which regulate whole-body energy metabolism. None of the fatty acids activated PPARd, whereas almost all activated PPARa in luciferase assays. Two fatty acids potently activated PPARg, a master regulator of adipocyte differentiation, with 10-oxo-12(Z)-octadecenoic acid (KetoA) having the most potency. In 3T3- L1 cells, KetoA induced adipocyte differentiation via the activation of PPARg, and increased adiponectin production and insulin-stimulated glucose uptake. These findings suggest that fatty acids, including KetoA, generated in gut by lactic acid bacteria may be involved in the regulation of host energy metabolism.',25749343),(3610,'human','increase','D-Fructose','low-throughput','A high-fructose diet caused the hippocampal neuroinflammatory response, reactive gliosis, and neuronal loss in C57BL/6N mice. Depletion of the gut microbiota using broad-spectrum antibiotics suppressed the hippocampal neuroinflammatory response in fructose-fed mice, but these animals still exhibited neuronal loss. Gut microbiota compositional alteration, short-chain fatty acids (SCFAs) reduction, intestinal epithelial barrier impairment, NOD-like receptor family pyrin domain-containing 6 (NLRP6) inflammasome dysfunction, high levels of serum endotoxin, and FITC-dextran were observed in fructose-fed mice. Of note, SCFAs, as well as pioglitazone (a selective peroxisome proliferator-activated receptor gamma (PPAR-γ) agonist), shaped the gut microbiota and ameliorated intestinal epithelial barrier impairment and NLRP6 inflammasome dysfunction in fructose-fed mice. Moreover, SCFAs-mediated NLRP6 inflammasome activation was inhibited by histamine (a bacterial metabolite) in ex vivo colonic explants and suppressed in murine CT26 colon carcinoma cells transfected with NLRP6 siRNA. However, pioglitazone and GW9662 (a PPAR-γ antagonist) exerted no impact on SCFAs-mediated NLRP6 inflammasome activation in ex vivo colonic explants, suggesting that SCFAs may stimulate NLRP6 inflammasome independently of PPAR-γ activation. SCFAs and pioglitazone prevented fructose-induced hippocampal neuroinflammatory response and neuronal loss in mice. Additionally, SCFAs activated colonic NLRP6 inflammasome and increased DCX+ newborn neurons in the hippocampal DG of control mice.',31255176),(3614,'human','increase','Tryptophan','low-throughput','Indole-3-propionic acid (IPA), a tryptophan metabolite produced by gut bacteria, is a potent anti-non-alcoholic steatohepatitis (NASH) microbial metabolite. Here, we demonstrate that administration of IPA modulates the microbiota composition in the gut and inhibits microbial dysbiosis in rats fed a high-fat diet. IPA induces the expression of tight junction proteins, such as ZO-1 and Occludin, and maintains intestinal epithelium homeostasis, leading to a reduction in plasma endotoxin levels. Interestingly, IPA inhibits NF-κB signaling and reduces the levels of proinflammatory cytokines, such as TNFα, IL-1β, and IL-6, in response to endotoxin in macrophages to repress hepatic inflammation and liver injury. Moreover, IPA is sufficient to inhibit the expression of fibrogenic and collagen genes and attenuate diet-induced NASH phenotypes. The beneficial effects of IPA on the liver are likely mediated through inhibiting the production of endotoxin in the gut.',31506421),(3615,'human','decrease','Tryptophan','low-throughput','Indole-3-propionic acid (IPA), a tryptophan metabolite produced by gut bacteria, is a potent anti-non-alcoholic steatohepatitis (NASH) microbial metabolite. Here, we demonstrate that administration of IPA modulates the microbiota composition in the gut and inhibits microbial dysbiosis in rats fed a high-fat diet. IPA induces the expression of tight junction proteins, such as ZO-1 and Occludin, and maintains intestinal epithelium homeostasis, leading to a reduction in plasma endotoxin levels. Interestingly, IPA inhibits NF-κB signaling and reduces the levels of proinflammatory cytokines, such as TNFα, IL-1β, and IL-6, in response to endotoxin in macrophages to repress hepatic inflammation and liver injury. Moreover, IPA is sufficient to inhibit the expression of fibrogenic and collagen genes and attenuate diet-induced NASH phenotypes. The beneficial effects of IPA on the liver are likely mediated through inhibiting the production of endotoxin in the gut.',31506421),(3616,'human','decrease','Tryptophan','low-throughput','Indole-3-propionic acid (IPA), a tryptophan metabolite produced by gut bacteria, is a potent anti-non-alcoholic steatohepatitis (NASH) microbial metabolite. Here, we demonstrate that administration of IPA modulates the microbiota composition in the gut and inhibits microbial dysbiosis in rats fed a high-fat diet. IPA induces the expression of tight junction proteins, such as ZO-1 and Occludin, and maintains intestinal epithelium homeostasis, leading to a reduction in plasma endotoxin levels. Interestingly, IPA inhibits NF-κB signaling and reduces the levels of proinflammatory cytokines, such as TNFα, IL-1β, and IL-6, in response to endotoxin in macrophages to repress hepatic inflammation and liver injury. Moreover, IPA is sufficient to inhibit the expression of fibrogenic and collagen genes and attenuate diet-induced NASH phenotypes. The beneficial effects of IPA on the liver are likely mediated through inhibiting the production of endotoxin in the gut.',31506421),(3617,'human','decrease','Tryptophan','low-throughput','Indole-3-propionic acid (IPA), a tryptophan metabolite produced by gut bacteria, is a potent anti-non-alcoholic steatohepatitis (NASH) microbial metabolite. Here, we demonstrate that administration of IPA modulates the microbiota composition in the gut and inhibits microbial dysbiosis in rats fed a high-fat diet. IPA induces the expression of tight junction proteins, such as ZO-1 and Occludin, and maintains intestinal epithelium homeostasis, leading to a reduction in plasma endotoxin levels. Interestingly, IPA inhibits NF-κB signaling and reduces the levels of proinflammatory cytokines, such as TNFα, IL-1β, and IL-6, in response to endotoxin in macrophages to repress hepatic inflammation and liver injury. Moreover, IPA is sufficient to inhibit the expression of fibrogenic and collagen genes and attenuate diet-induced NASH phenotypes. The beneficial effects of IPA on the liver are likely mediated through inhibiting the production of endotoxin in the gut.',31506421),(3618,'human','decrease','Tryptophan','low-throughput','Indole-3-propionic acid (IPA), a tryptophan metabolite produced by gut bacteria, is a potent anti-non-alcoholic steatohepatitis (NASH) microbial metabolite. Here, we demonstrate that administration of IPA modulates the microbiota composition in the gut and inhibits microbial dysbiosis in rats fed a high-fat diet. IPA induces the expression of tight junction proteins, such as ZO-1 and Occludin, and maintains intestinal epithelium homeostasis, leading to a reduction in plasma endotoxin levels. Interestingly, IPA inhibits NF-κB signaling and reduces the levels of proinflammatory cytokines, such as TNFα, IL-1β, and IL-6, in response to endotoxin in macrophages to repress hepatic inflammation and liver injury. Moreover, IPA is sufficient to inhibit the expression of fibrogenic and collagen genes and attenuate diet-induced NASH phenotypes. The beneficial effects of IPA on the liver are likely mediated through inhibiting the production of endotoxin in the gut.',31506421),(3619,'human','null','null','null','Gut flora and plasma microbiome-related metabolites were evaluated by amplicon sequencing of the bacterial 16S ribosomal RNA gene and capillary electrophoresis time-of-flight mass spectrometry, respectively. HF patients were evaluated in both the Decomp and Comp phases during hospitalization. The phylum Actinobacteria was enriched in HF patients compared with control subjects. At the genus level, Bifiodobacterium was abundant while Megamonas was depleted in HF patients. Meanwhile, plasma concentration of trimethylamine N-oxide (TMAO), a gut microbiome-derived metabolite, was increased in HF patients. A correlation analysis revealed positive correlations between the abundance of the genus Escherichia/Shigella and levels of TMAO and indoxyl sulfate (IS, a microbe-dependent uremic toxin) in Comp HF . Escherichia/Shigella was more abundant in Decomp than in Comp HF (P=0.030).Conclusions: Our results suggest that gut microbiome composition and microbiome-related metabolites are altered in HF patient.',30487369),(3620,'human','null','null','null','Gut flora and plasma microbiome-related metabolites were evaluated by amplicon sequencing of the bacterial 16S ribosomal RNA gene and capillary electrophoresis time-of-flight mass spectrometry, respectively. HF patients were evaluated in both the Decomp and Comp phases during hospitalization. The phylum Actinobacteria was enriched in HF patients compared with control subjects. At the genus level, Bifiodobacterium was abundant while Megamonas was depleted in HF patients. Meanwhile, plasma concentration of trimethylamine N-oxide (TMAO), a gut microbiome-derived metabolite, was increased in HF patients. A correlation analysis revealed positive correlations between the abundance of the genus Escherichia/Shigella and levels of TMAO and indoxyl sulfate (IS, a microbe-dependent uremic toxin) in Comp HF . Escherichia/Shigella was more abundant in Decomp than in Comp HF (P=0.030).Conclusions: Our results suggest that gut microbiome composition and microbiome-related metabolites are altered in HF patient.',30487369),(3621,'human','null','null','null','Gut flora and plasma microbiome-related metabolites were evaluated by amplicon sequencing of the bacterial 16S ribosomal RNA gene and capillary electrophoresis time-of-flight mass spectrometry, respectively. HF patients were evaluated in both the Decomp and Comp phases during hospitalization. The phylum Actinobacteria was enriched in HF patients compared with control subjects. At the genus level, Bifiodobacterium was abundant while Megamonas was depleted in HF patients. Meanwhile, plasma concentration of trimethylamine N-oxide (TMAO), a gut microbiome-derived metabolite, was increased in HF patients. A correlation analysis revealed positive correlations between the abundance of the genus Escherichia/Shigella and levels of TMAO and indoxyl sulfate (IS, a microbe-dependent uremic toxin) in Comp HF . Escherichia/Shigella was more abundant in Decomp than in Comp HF (P=0.030).Conclusions: Our results suggest that gut microbiome composition and microbiome-related metabolites are altered in HF patient.',30487369),(3622,'human','null','null','null','Gut flora and plasma microbiome-related metabolites were evaluated by amplicon sequencing of the bacterial 16S ribosomal RNA gene and capillary electrophoresis time-of-flight mass spectrometry, respectively. HF patients were evaluated in both the Decomp and Comp phases during hospitalization. The phylum Actinobacteria was enriched in HF patients compared with control subjects. At the genus level, Bifiodobacterium was abundant while Megamonas was depleted in HF patients. Meanwhile, plasma concentration of trimethylamine N-oxide (TMAO), a gut microbiome-derived metabolite, was increased in HF patients. A correlation analysis revealed positive correlations between the abundance of the genus Escherichia/Shigella and levels of TMAO and indoxyl sulfate (IS, a microbe-dependent uremic toxin) in Comp HF . Escherichia/Shigella was more abundant in Decomp than in Comp HF (P=0.030).Conclusions: Our results suggest that gut microbiome composition and microbiome-related metabolites are altered in HF patient.',30487369),(3626,'human','null','Quercitrin','null','Ultra performance liquid chromatography/quadrupole-time-of-flight mass spectrometry (UPLC/Q-TOF MS) technique combined with MetabolynxTM software was used for analysis of the metabolites of quercitrin by the isolated human intestinal bacteria from the human feces. Four metabolites of quercitrin were detected and tentatively identified based on the characteristics of their protonated ions. The metabolites were metabolized by four main metabolic pathwaysincluding hydroxylation, demethylation, deglycosylation and ring-cleavage. Quercitrin was metabolized to the hydroxyquercitrin and desmethylquercitrin by the majority of the isolated intestinal bacteria such as Bacteroidessp. 54, and was degraded to the deglycosylated product quercetin by rhamnosidase and further ring cleavage metabolite 3,4-dihydroxybenzoic acid by the minority of the isolated bacteria such as Bacteroides sp. 45. The metabolic pathways and most of the metabolites of quercitrin were reported for the first time.',23754166),(3627,'human','null','Quercitrin','null','Ultra performance liquid chromatography/quadrupole-time-of-flight mass spectrometry (UPLC/Q-TOF MS) technique combined with MetabolynxTM software was used for analysis of the metabolites of quercitrin by the isolated human intestinal bacteria from the human feces. Four metabolites of quercitrin were detected and tentatively identified based on the characteristics of their protonated ions. The metabolites were metabolized by four main metabolic pathwaysincluding hydroxylation, demethylation, deglycosylation and ring-cleavage. Quercitrin was metabolized to the hydroxyquercitrin and desmethylquercitrin by the majority of the isolated intestinal bacteria such as Bacteroidessp. 54, and was degraded to the deglycosylated product quercetin by rhamnosidase and further ring cleavage metabolite 3,4-dihydroxybenzoic acid by the minority of the isolated bacteria such as Bacteroides sp. 45. The metabolic pathways and most of the metabolites of quercitrin were reported for the first time.',23754166),(3628,'human','null','Quercitrin','null','Ultra performance liquid chromatography/quadrupole-time-of-flight mass spectrometry (UPLC/Q-TOF MS) technique combined with MetabolynxTM software was used for analysis of the metabolites of quercitrin by the isolated human intestinal bacteria from the human feces. Four metabolites of quercitrin were detected and tentatively identified based on the characteristics of their protonated ions. The metabolites were metabolized by four main metabolic pathwaysincluding hydroxylation, demethylation, deglycosylation and ring-cleavage. Quercitrin was metabolized to the hydroxyquercitrin and desmethylquercitrin by the majority of the isolated intestinal bacteria such as Bacteroidessp. 54, and was degraded to the deglycosylated product quercetin by rhamnosidase and further ring cleavage metabolite 3,4-dihydroxybenzoic acid by the minority of the isolated bacteria such as Bacteroides sp. 45. The metabolic pathways and most of the metabolites of quercitrin were reported for the first time.',23754166),(3632,'human','decrease','null','low-throughput','Plasma from rats treated for seven days with the non-absorbed antibiotic vancomycin or a mixture of streptomycin, neomycin, polymyxin B and bacitracin was analyzed using mass spectrometry-based metabolite profiling platforms. Antibiotic-induced changes in the abundance of individual groups of intestinal microbiota dramatically altered the host\'s metabolism. Hierarchical clustering of dissimilarities separated the levels of 284 identified metabolites from treated vs. untreated rats; 193 were altered by the antibiotic treatments with a tendency towards decreased metabolite levels. Catabolism of the aromatic amino acids phenylalanine, tryptophan and tyrosine was the most affected pathway comprising 33 affected metabolites. Vancomycin-treated rats were administered amino acid metabolites prior to ischemia/reperfusion studies. Oral or intravenous pretreatment of rats with these amino acid metabolites abolished the decrease in infarct size conferred by vancomycin. Cardioprotection by vancomycin is mediated by activation of pro-survival kinases Akt, p42/44 MAPK and p39 MAPK, as well as activation of KATP channels, classical mediators and effectors of pharmacologic cardioprotection. These proteins are components of known intersecting and interdependent signaling pathways in the myocardium that determine severity of myocardial infarction.',27505423),(3633,'human','increase','null','low-throughput','Plasma from rats treated for seven days with the non-absorbed antibiotic vancomycin or a mixture of streptomycin, neomycin, polymyxin B and bacitracin was analyzed using mass spectrometry-based metabolite profiling platforms. Antibiotic-induced changes in the abundance of individual groups of intestinal microbiota dramatically altered the host\'s metabolism. Hierarchical clustering of dissimilarities separated the levels of 284 identified metabolites from treated vs. untreated rats; 193 were altered by the antibiotic treatments with a tendency towards decreased metabolite levels. Catabolism of the aromatic amino acids phenylalanine, tryptophan and tyrosine was the most affected pathway comprising 33 affected metabolites. Vancomycin-treated rats were administered amino acid metabolites prior to ischemia/reperfusion studies. Oral or intravenous pretreatment of rats with these amino acid metabolites abolished the decrease in infarct size conferred by vancomycin. Cardioprotection by vancomycin is mediated by activation of pro-survival kinases Akt, p42/44 MAPK and p40 MAPK, as well as activation of KATP channels, classical mediators and effectors of pharmacologic cardioprotection. These proteins are components of known intersecting and interdependent signaling pathways in the myocardium that determine severity of myocardial infarction.',27505423),(3634,'human','increase','null','low-throughput','Plasma from rats treated for seven days with the non-absorbed antibiotic vancomycin or a mixture of streptomycin, neomycin, polymyxin B and bacitracin was analyzed using mass spectrometry-based metabolite profiling platforms. Antibiotic-induced changes in the abundance of individual groups of intestinal microbiota dramatically altered the host\'s metabolism. Hierarchical clustering of dissimilarities separated the levels of 284 identified metabolites from treated vs. untreated rats; 193 were altered by the antibiotic treatments with a tendency towards decreased metabolite levels. Catabolism of the aromatic amino acids phenylalanine, tryptophan and tyrosine was the most affected pathway comprising 33 affected metabolites. Vancomycin-treated rats were administered amino acid metabolites prior to ischemia/reperfusion studies. Oral or intravenous pretreatment of rats with these amino acid metabolites abolished the decrease in infarct size conferred by vancomycin. Cardioprotection by vancomycin is mediated by activation of pro-survival kinases Akt, p42/44 MAPK and p41 MAPK, as well as activation of KATP channels, classical mediators and effectors of pharmacologic cardioprotection. These proteins are components of known intersecting and interdependent signaling pathways in the myocardium that determine severity of myocardial infarction.',27505423),(3635,'human','increase','null','low-throughput','Plasma from rats treated for seven days with the non-absorbed antibiotic vancomycin or a mixture of streptomycin, neomycin, polymyxin B and bacitracin was analyzed using mass spectrometry-based metabolite profiling platforms. Antibiotic-induced changes in the abundance of individual groups of intestinal microbiota dramatically altered the host\'s metabolism. Hierarchical clustering of dissimilarities separated the levels of 284 identified metabolites from treated vs. untreated rats; 193 were altered by the antibiotic treatments with a tendency towards decreased metabolite levels. Catabolism of the aromatic amino acids phenylalanine, tryptophan and tyrosine was the most affected pathway comprising 33 affected metabolites. Vancomycin-treated rats were administered amino acid metabolites prior to ischemia/reperfusion studies. Oral or intravenous pretreatment of rats with these amino acid metabolites abolished the decrease in infarct size conferred by vancomycin. Cardioprotection by vancomycin is mediated by activation of pro-survival kinases Akt, p42/44 MAPK and p42 MAPK, as well as activation of KATP channels, classical mediators and effectors of pharmacologic cardioprotection. These proteins are components of known intersecting and interdependent signaling pathways in the myocardium that determine severity of myocardial infarction.',27505423),(3636,'human','increase','null','low-throughput','Plasma from rats treated for seven days with the non-absorbed antibiotic vancomycin or a mixture of streptomycin, neomycin, polymyxin B and bacitracin was analyzed using mass spectrometry-based metabolite profiling platforms. Antibiotic-induced changes in the abundance of individual groups of intestinal microbiota dramatically altered the host\'s metabolism. Hierarchical clustering of dissimilarities separated the levels of 284 identified metabolites from treated vs. untreated rats; 193 were altered by the antibiotic treatments with a tendency towards decreased metabolite levels. Catabolism of the aromatic amino acids phenylalanine, tryptophan and tyrosine was the most affected pathway comprising 33 affected metabolites. Vancomycin-treated rats were administered amino acid metabolites prior to ischemia/reperfusion studies. Oral or intravenous pretreatment of rats with these amino acid metabolites abolished the decrease in infarct size conferred by vancomycin. Cardioprotection by vancomycin is mediated by activation of pro-survival kinases Akt, p42/44 MAPK and p43 MAPK, as well as activation of KATP channels, classical mediators and effectors of pharmacologic cardioprotection. These proteins are components of known intersecting and interdependent signaling pathways in the myocardium that determine severity of myocardial infarction.',27505423),(3637,'human','null','D-Sorbitol','null','Three sequential fermentative batches were carried out with cell recycle in four simultaneously operating bioreactors maintained at pH 6.5, 30°C, and 100 rpm. P. acidipropionici ATCC 4875 was able to produce propionic and succinic acid from sorbitol. The concentration of propionic acid decreased slightly from 39.5 ± 5.2 g L�? to 34.4 ± 1.9 g L�?,and that of succinic acid increased significantly from 6.1 ± 2.1 g L�? to 14.8 ± 0.9 g L�? through the sequential batches. In addition, a small amount of acetic acid was produced that decreased from 3.3 ± 0.4 g L�? to 2.0 ± 0.3 g L�? through the batches. The major yield for propionic acid was 0.613 g g�? in the first batch and succinic acid it was 0.212 g g�? in the third batch. The minor yield of acetic acid was 0.029 g g�? ,in the second and third batches.',25852990),(3638,'human','null','D-Sorbitol','null','Three sequential fermentative batches were carried out with cell recycle in four simultaneously operating bioreactors maintained at pH 6.5, 30°C, and 100 rpm. P. acidipropionici ATCC 4875 was able to produce propionic and succinic acid from sorbitol. The concentration of propionic acid decreased slightly from 39.5 ± 5.2 g L�? to 34.4 ± 1.9 g L�?,and that of succinic acid increased significantly from 6.1 ± 2.1 g L�? to 14.8 ± 0.9 g L�? through the sequential batches. In addition, a small amount of acetic acid was produced that decreased from 3.3 ± 0.4 g L�? to 2.0 ± 0.3 g L�? through the batches. The major yield for propionic acid was 0.613 g g�? in the first batch and succinic acid it was 0.212 g g�? in the third batch. The minor yield of acetic acid was 0.029 g g�? ,in the second and third batches.',25852990),(3639,'human','increase','null','low-throughput','In the present study, we found that TMAO significantly upregulated the expression of proinflammatory markers, such as IL-1β, IL-6, and TNF-α in TH positive neurons, as well as NMDARs, which might further result in sympathetic hyperactivation within the LSG and promote the incidence of VA.',30954457),(3640,'human','increase','null','low-throughput','In the present study, we found that TMAO significantly upregulated the expression of proinflammatory markers, such as IL-1β, IL-6, and TNF-α in TH positive neurons, as well as NMDARs, which might further result in sympathetic hyperactivation within the LSG and promote the incidence of VA.',30954457),(3641,'human','increase','null','low-throughput','In the present study, we found that TMAO significantly upregulated the expression of proinflammatory markers, such as IL-1β, IL-6, and TNF-α in TH positive neurons, as well as NMDARs, which might further result in sympathetic hyperactivation within the LSG and promote the incidence of VA.',30954457),(3642,'human','increase','4-N-(3-Chloro-4-fluorophenyl)-6-N-[1-(iodomethyl)piperidin-4-yl]pyrimido[5,4-d]pyrimidine-4,6-diamine','low-throughput','The present study identified the metabolites from DN through analysis of organ-specific biotransformation and validated the cardioprotective effects of the screened metabolites using an isoprenaline-induced myocardial ischemia rat model. The findings of the present study provide evidence that, first, diosgenin is generated from DN by intestinal microflora; second, diosgenin can protect the myocardium against ischemic insult in a dose-dependent manner, almost comparable to the effect of DN extract; and, third, DN\'s protective effect can be attributed to the increase of enzymatic and nonenzymatic antioxidants (SOD, CAT, GPx, and T-AOC) in vivo and to a decrease in lipid peroxidation. These phenomena help explain the clinical efficacy of DN as an anti-MI drug.',29085554),(3643,'human','increase','4-N-(3-Chloro-4-fluorophenyl)-6-N-[1-(iodomethyl)piperidin-4-yl]pyrimido[5,4-d]pyrimidine-4,6-diamine','low-throughput','The present study identified the metabolites from DN through analysis of organ-specific biotransformation and validated the cardioprotective effects of the screened metabolites using an isoprenaline-induced myocardial ischemia rat model. The findings of the present study provide evidence that, first, diosgenin is generated from DN by intestinal microflora; second, diosgenin can protect the myocardium against ischemic insult in a dose-dependent manner, almost comparable to the effect of DN extract; and, third, DN\'s protective effect can be attributed to the increase of enzymatic and nonenzymatic antioxidants (SOD, CAT, GPx, and T-AOC) in vivo and to a decrease in lipid peroxidation. These phenomena help explain the clinical efficacy of DN as an anti-MI drug.',29085554),(3644,'human','increase','4-N-(3-Chloro-4-fluorophenyl)-6-N-[1-(iodomethyl)piperidin-4-yl]pyrimido[5,4-d]pyrimidine-4,6-diamine','low-throughput','The present study identified the metabolites from DN through analysis of organ-specific biotransformation and validated the cardioprotective effects of the screened metabolites using an isoprenaline-induced myocardial ischemia rat model. The findings of the present study provide evidence that, first, diosgenin is generated from DN by intestinal microflora; second, diosgenin can protect the myocardium against ischemic insult in a dose-dependent manner, almost comparable to the effect of DN extract; and, third, DN\'s protective effect can be attributed to the increase of enzymatic and nonenzymatic antioxidants (SOD, CAT, GPx, and T-AOC) in vivo and to a decrease in lipid peroxidation. These phenomena help explain the clinical efficacy of DN as an anti-MI drug.',29085554),(3645,'human','decrease','Arbutin','low-throughput','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.',21889493),(3646,'human','increase','Arbutin','low-throughput','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.',21889493),(3647,'human','increase','Arbutin','low-throughput','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.',21889493),(3648,'human','decrease','Arbutin','low-throughput','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.',21889493),(3649,'human','increase','Arbutin','low-throughput','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.',21889493),(3650,'human','increase','Arbutin','low-throughput','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.',21889493),(3651,'human','decrease','Arbutin','low-throughput','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.',21889493),(3652,'human','increase','Arbutin','low-throughput','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.',21889493),(3653,'human','increase','Arbutin','low-throughput','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.',21889493),(3654,'human','decrease','Arbutin','low-throughput','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.',21889493),(3655,'human','increase','Arbutin','low-throughput','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.',21889493),(3656,'human','increase','Arbutin','low-throughput','In addition, the mixture considerably inhibited expression of Bcl-2 together with an increase in Bax expression. Likewise, activation stimulated cleavage of caspase-3 and production of reactive oxygen species in HepG2 cell cultures.',21889493),(3667,'human','null','Dihydrodaidzein','null','In this study, chemically synthesized dihydrodaidzein was used as the substrate administered to a newly isolated rodshaped, gram-negative, anaerobic bacterium from human feces which can convert dihydrodaidzein to S-equol.',29112296),(3668,'human','null','daidzein','null','Clostridium sp. strain HGH6, that is capable of reducing daidzein to dihydrodaidzein, but further metabolism to tetrahydrodaidzein or equol did not occur.',15640190),(3669,'human','decrease','null','low-throughput','Our result also demonstrated that butyrate intervention enhanced fecal SCFAs concentrations, and inhibited endotoxin levels in feces and serum. Correlation analysis indicated positive relation between endotoxin level and Desulfovibrionaceae abundance. Furthermore,  butyrate intervention inhibited expressions of IL-1β, IL-6 and MCP1/CCL2 in liver, as well as TLR4 in adipose tissue.',15640190),(3670,'human','decrease','null','low-throughput','Our result also demonstrated that butyrate intervention enhanced fecal SCFAs concentrations, and inhibited endotoxin levels in feces and serum. Correlation analysis indicated positive relation between endotoxin level and Desulfovibrionaceae abundance. Furthermore,  butyrate intervention inhibited expressions of IL-1β, IL-6 and MCP1/CCL2 in liver, as well as TLR4 in adipose tissue.',31295342),(3671,'human','decrease','null','low-throughput','Our result also demonstrated that butyrate intervention enhanced fecal SCFAs concentrations, and inhibited endotoxin levels in feces and serum. Correlation analysis indicated positive relation between endotoxin level and Desulfovibrionaceae abundance. Furthermore,  butyrate intervention inhibited expressions of IL-1β, IL-6 and MCP1/CCL2 in liver, as well as TLR4 in adipose tissue.',31295342),(3672,'human','decrease','null','low-throughput','Our result also demonstrated that butyrate intervention enhanced fecal SCFAs concentrations, and inhibited endotoxin levels in feces and serum. Correlation analysis indicated positive relation between endotoxin level and Desulfovibrionaceae abundance. Furthermore,  butyrate intervention inhibited expressions of IL-1β, IL-6 and MCP1/CCL2 in liver, as well as TLR4 in adipose tissue.',31295342),(3673,'human','null','Biochanin A','null','Eubacterium limosum (ATCC 8486), a strict anaerobe from the human intestinal tract that is capable of O-demethylation of several compounds, was tested for the ability to metabolize three methoxylated isoflavonoids, biochanin A, formononetin, and glycitein. High performance liquid chromatography elution profiles of metabolites produced from biochanin A, formononetin, and glycitein showed peaks that had identical retention times to authentic genistein, daidzein, and 6,7,4P-trihydroxyisoflavone, respectively.',31295342),(3674,'human','null','Formononetin','null','Eubacterium limosum (ATCC 8486), a strict anaerobe from the human intestinal tract that is capable of O-demethylation of several compounds, was tested for the ability to metabolize three methoxylated isoflavonoids, biochanin A, formononetin, and glycitein. High performance liquid chromatography elution profiles of metabolites produced from biochanin A, formononetin, and glycitein showed peaks that had identical retention times to authentic genistein, daidzein, and 6,7,4P-trihydroxyisoflavone, respectively.',11040423),(3675,'human','null','Glycitein','null','Eubacterium limosum (ATCC 8486), a strict anaerobe from the human intestinal tract that is capable of O-demethylation of several compounds, was tested for the ability to metabolize three methoxylated isoflavonoids, biochanin A, formononetin, and glycitein. High performance liquid chromatography elution profiles of metabolites produced from biochanin A, formononetin, and glycitein showed peaks that had identical retention times to authentic genistein, daidzein, and 6,7,4P-trihydroxyisoflavone, respectively.',11040423),(3680,'human','null','(-)-Epigallocatechin gallate','null','The results showed that L. plantarum IFPL935 exhibited higher resistance to nongalloylated monomeric flavan-3-ols, A-type dimeric procyanidins, and cranberry extract than to (�?-epicatechin-3-O-gallate and B-type dimeric procyanidins. Despite these findings,the strain was capable of rapidly degrading (�?-epicatechin-3-O-gallate, but not A- or B-type dimeric procyanidins. However, it was able to produce large changes in the phenolic profile of the cranberry extract mainly due to the catabolism of hydroxycinnamic and hydroxybenzoic acids. Of most relevance was the fact that L. plantarum IFPL935 cleaved the heterocyclic ring of monomeric flavan-3-ols, giving rise to 1-(3�?4�?dihydroxyphenyl)-3-(2�?4�?6�?trihydroxyphenyl)propan-2-ol, activity exhibited by only a few human intestinal bacteria.',22646528),(3681,'human','null','(-)-Epigallocatechin gallate','null','The results showed that L. plantarum IFPL935 exhibited higher resistance to nongalloylated monomeric flavan-3-ols, A-type dimeric procyanidins, and cranberry extract than to (�?-epicatechin-3-O-gallate and B-type dimeric procyanidins. Despite these findings,the strain was capable of rapidly degrading (�?-epicatechin-3-O-gallate, but not A- or B-type dimeric procyanidins. However, it was able to produce large changes in the phenolic profile of the cranberry extract mainly due to the catabolism of hydroxycinnamic and hydroxybenzoic acids. Of most relevance was the fact that L. plantarum IFPL935 cleaved the heterocyclic ring of monomeric flavan-3-ols, giving rise to 1-(3�?4�?dihydroxyphenyl)-3-(2�?4�?6�?trihydroxyphenyl)propan-2-ol, activity exhibited by only a few human intestinal bacteria.',22646528),(3682,'human','null','null','null','One important secondary metabolite of bifidobacteria is folate, which contributes to the one-carbon metabolism of the host and therefore enables maintenance of DNA methylation marks as explained below (41).',23594611),(3683,'human','increase','null','low-throughput','Furthermore, TMAO increased TGF-β receptor I expression, which promoted the phosphorylation of Smad2 and up-regulated the expression of α-SMA and collagen I. The ubiquitination of TGF-βRI was decreased in neonatal mouse fibroblasts after TMAO treatment. TMAO also inhibited the expression of smurf2. Inhibition of TGF-β1 receptor with the small molecule inhibitor SB431542 decreased TGF-β receptor I expression, reduced the phosphorylation of Smad2, down-regulated TMAO-induced α-SMA and collagen I expression in cardiac fibroblasts. Conclusions: Cardiac function and cardiac fibrosis were significantly exacerbated in mice fed diets supplemented with either choline or TMAO, probably through accelerating the transformation of fibroblasts into myofibroblasts, indicating activation of TGF-βRI/Smad2 pathway',31299216),(3684,'human','increase','null','low-throughput','Furthermore, TMAO increased TGF-β receptor I expression, which promoted the phosphorylation of Smad2 and up-regulated the expression of α-SMA and collagen I. The ubiquitination of TGF-βRI was decreased in neonatal mouse fibroblasts after TMAO treatment. TMAO also inhibited the expression of smurf2. Inhibition of TGF-β1 receptor with the small molecule inhibitor SB431542 decreased TGF-β receptor I expression, reduced the phosphorylation of Smad2, down-regulated TMAO-induced α-SMA and collagen I expression in cardiac fibroblasts. Conclusions: Cardiac function and cardiac fibrosis were significantly exacerbated in mice fed diets supplemented with either choline or TMAO, probably through accelerating the transformation of fibroblasts into myofibroblasts, indicating activation of TGF-βRI/Smad2 pathway',31299216),(3685,'human','decrease','null','low-throughput','Furthermore, TMAO increased TGF-β receptor I expression, which promoted the phosphorylation of Smad2 and up-regulated the expression of α-SMA and collagen I. The ubiquitination of TGF-βRI was decreased in neonatal mouse fibroblasts after TMAO treatment. TMAO also inhibited the expression of smurf2. Inhibition of TGF-β1 receptor with the small molecule inhibitor SB431542 decreased TGF-β receptor I expression, reduced the phosphorylation of Smad2, down-regulated TMAO-induced α-SMA and collagen I expression in cardiac fibroblasts. Conclusions: Cardiac function and cardiac fibrosis were significantly exacerbated in mice fed diets supplemented with either choline or TMAO, probably through accelerating the transformation of fibroblasts into myofibroblasts, indicating activation of TGF-βRI/Smad2 pathway',31299216),(3686,'human','null','Glycitein','null','The dichotomy between the biotransformation of polyphenols into their metabolites by gut microbiota and the modulation of gut microbiota composition by polyphenols contributes to positive health outcomes. Although there are many studies on the in vivo bioavailability of polyphenols, the mutual relationship between polyphenols and gut microbiota is not fully understood. This review focuses on the biotransformation of polyphenols by gut microbiota, modulation of gut microbiota by polyphenols, and the effects of these two-way mutual interactions on polyphenol bioavailability, and ultimately, human health.',26861391),(3687,'human','null','Glycitein','null','The dichotomy between the biotransformation of polyphenols into their metabolites by gut microbiota and the modulation of gut microbiota composition by polyphenols contributes to positive health outcomes. Although there are many studies on the in vivo bioavailability of polyphenols, the mutual relationship between polyphenols and gut microbiota is not fully understood. This review focuses on the biotransformation of polyphenols by gut microbiota, modulation of gut microbiota by polyphenols, and the effects of these two-way mutual interactions on polyphenol bioavailability, and ultimately, human health.',26861391),(3688,'human','null','Chlorogenic acid','null','Chlorogenic acid hydrolysing activities were essentially intracellular. These cinnamoyl esterase-producing organisms were devoid of other phenolic-degrading activities. The results show that certain gut bacteria, including some already recognized as potentially health-promoting (i.e. species belonging to the genera Bifidobacterium and Lactobacillus), are involved in the release of bioactive hydroxycinnamic acids in the human colon.',11412317),(3689,'human','null','Chlorogenic acid','null','Chlorogenic acid hydrolysing activities were essentially intracellular. These cinnamoyl esterase-producing organisms were devoid of other phenolic-degrading activities. The results show that certain gut bacteria, including some already recognized as potentially health-promoting (i.e. species belonging to the genera Bifidobacterium and Lactobacillus), are involved in the release of bioactive hydroxycinnamic acids in the human colon.',11412317),(3690,'human','null','Chlorogenic acid','null','Chlorogenic acid hydrolysing activities were essentially intracellular. These cinnamoyl esterase-producing organisms were devoid of other phenolic-degrading activities. The results show that certain gut bacteria, including some already recognized as potentially health-promoting (i.e. species belonging to the genera Bifidobacterium and Lactobacillus), are involved in the release of bioactive hydroxycinnamic acids in the human colon.',11412317),(3691,'human','null','Chlorogenic acid','null','Chlorogenic acid hydrolysing activities were essentially intracellular. These cinnamoyl esterase-producing organisms were devoid of other phenolic-degrading activities. The results show that certain gut bacteria, including some already recognized as potentially health-promoting (i.e. species belonging to the genera Bifidobacterium and Lactobacillus), are involved in the release of bioactive hydroxycinnamic acids in the human colon.',11412317),(3692,'human','null','Chlorogenic acid','null','Chlorogenic acid hydrolysing activities were essentially intracellular. These cinnamoyl esterase-producing organisms were devoid of other phenolic-degrading activities. The results show that certain gut bacteria, including some already recognized as potentially health-promoting (i.e. species belonging to the genera Bifidobacterium and Lactobacillus), are involved in the release of bioactive hydroxycinnamic acids in the human colon.',11412317),(3693,'human','null','Chlorogenic acid','null','Chlorogenic acid hydrolysing activities were essentially intracellular. These cinnamoyl esterase-producing organisms were devoid of other phenolic-degrading activities. The results show that certain gut bacteria, including some already recognized as potentially health-promoting (i.e. species belonging to the genera Bifidobacterium and Lactobacillus), are involved in the release of bioactive hydroxycinnamic acids in the human colon.',11412317),(3694,'human','null','D-Glucose','null','To gain some specific insight into the roles microorganisms might play in non-alcoholic fatty liver disease (NAFLD), some intestinal and lactic acid bacteria and one yeast  were characterized by high performance liquid chromatography for production of ethanol when grown on different carbohydrates. Highest amounts of ethanol were produced by S. cerevisiae, L. fermentum, and W. confusa on glucose and by S. cerevisiae and W. confusa on fructose.',26858714),(3695,'human','null','D-Glucose','null','To gain some specific insight into the roles microorganisms might play in non-alcoholic fatty liver disease (NAFLD), some intestinal and lactic acid bacteria and one yeast  were characterized by high performance liquid chromatography for production of ethanol when grown on different carbohydrates. Highest amounts of ethanol were produced by S. cerevisiae, L. fermentum, and W. confusa on glucose and by S. cerevisiae and W. confusa on fructose.',26858714),(3696,'human','null','D-Fructose','null','To gain some specific insight into the roles microorganisms might play in non-alcoholic fatty liver disease (NAFLD), some intestinal and lactic acid bacteria and one yeast  were characterized by high performance liquid chromatography for production of ethanol when grown on different carbohydrates. Highest amounts of ethanol were produced by S. cerevisiae, L. fermentum, and W. confusa on glucose and by S. cerevisiae and W. confusa on fructose.',26858714),(3697,'human','null','D-Glucose','null','To gain some specific insight into the roles microorganisms might play in non-alcoholic fatty liver disease (NAFLD), some intestinal and lactic acid bacteria and one yeast  were characterized by high performance liquid chromatography for production of ethanol when grown on different carbohydrates. Highest amounts of ethanol were produced by S. cerevisiae, L. fermentum, and W. confusa on glucose and by S. cerevisiae and W. confusa on fructose.',26858714),(3698,'human','null','D-Fructose','null','To gain some specific insight into the roles microorganisms might play in non-alcoholic fatty liver disease (NAFLD), some intestinal and lactic acid bacteria and one yeast  were characterized by high performance liquid chromatography for production of ethanol when grown on different carbohydrates. Highest amounts of ethanol were produced by S. cerevisiae, L. fermentum, and W. confusa on glucose and by S. cerevisiae and W. confusa on fructose.',26858714),(3699,'human','null','Phosphoenolpyruvate','null','Our attempt to revisit classic literature has revealed many gaps in our knowledge that can now be examined in the light of modern systems approaches. At the same time, descriptive sequence-based studies are serving an important complementary role in defining priorities for pursuing mechanism-level understanding of this otherwise exceedingly complex interplay between the microbial species that inhabit humans and the resulting interactions that occur between these species and the host.',22018234),(3700,'human','null','Fumarate','null','Our attempt to revisit classic literature has revealed many gaps in our knowledge that can now be examined in the light of modern systems approaches. At the same time, descriptive sequence-based studies are serving an important complementary role in defining priorities for pursuing mechanism-level understanding of this otherwise exceedingly complex interplay between the microbial species that inhabit humans and the resulting interactions that occur between these species and the host.',22018234),(3701,'human','null','Succinate','null','Our attempt to revisit classic literature has revealed many gaps in our knowledge that can now be examined in the light of modern systems approaches. At the same time, descriptive sequence-based studies are serving an important complementary role in defining priorities for pursuing mechanism-level understanding of this otherwise exceedingly complex interplay between the microbial species that inhabit humans and the resulting interactions that occur between these species and the host.',22018234),(3702,'human','null','Acetate','null','Our attempt to revisit classic literature has revealed many gaps in our knowledge that can now be examined in the light of modern systems approaches. At the same time, descriptive sequence-based studies are serving an important complementary role in defining priorities for pursuing mechanism-level understanding of this otherwise exceedingly complex interplay between the microbial species that inhabit humans and the resulting interactions that occur between these species and the host.',22018234),(3703,'human','null','D-Glucose','null','Our attempt to revisit classic literature has revealed many gaps in our knowledge that can now be examined in the light of modern systems approaches. At the same time, descriptive sequence-based studies are serving an important complementary role in defining priorities for pursuing mechanism-level understanding of this otherwise exceedingly complex interplay between the microbial species that inhabit humans and the resulting interactions that occur between these species and the host.',22018234),(3704,'human','null','D-Glucose','null','Our attempt to revisit classic literature has revealed many gaps in our knowledge that can now be examined in the light of modern systems approaches. At the same time, descriptive sequence-based studies are serving an important complementary role in defining priorities for pursuing mechanism-level understanding of this otherwise exceedingly complex interplay between the microbial species that inhabit humans and the resulting interactions that occur between these species and the host.',22018234),(3705,'human','null','Daidzein','null','Equol is a metabolite produced from daidzein by enteric microflora, and it has attracted a great deal of attention because of its protective or ameliorative ability against several sex hormone-dependent diseases (e.g., menopausal disorder and lower bone density), which is more potent than that of other isoflavonoids. We purified a novel NADP(H)-dependent daidzein reductase (L-DZNR) from Lactococcus strain 20-92 (Lactococcus 20-92; S. Uchiyama, T. Ueno, and T. Suzuki, international patent WO2005/000042) that is involved in the metabolism of soy isoflavones and equol production and converts daidzein to dihydrodaidzein.',20639368),(3706,'human','null','Daidzein','null','Equol is a metabolite produced from daidzein by enteric microflora, and it has attracted a great deal of attention because of its protective or ameliorative ability against several sex hormone-dependent diseases (e.g., menopausal disorder and lower bone density), which is more potent than that of other isoflavonoids. We purified a novel NADP(H)-dependent daidzein reductase (L-DZNR) from Lactococcus strain 20-92 (Lactococcus 20-92; S. Uchiyama, T. Ueno, and T. Suzuki, international patent WO2005/000042) that is involved in the metabolism of soy isoflavones and equol production and converts daidzein to dihydrodaidzein.',20639368),(3707,'human','null','Mangiferin','null','The C-glucosyl bond of C-glucosides generally tolerates acid and enzymatic hydrolysis. Many C-glucosides are cleaved by human intestinal bacteria. We isolated the specific bacterium involved in the metabolism of mangiferin (2-b-D-glucopyranosyl-1,3,6,7-tetrahydroxyxanthone), C-glucosyl xanthone, from a mixture of human fecal bacteria. The anaerobic Bacteroides species named MANG, transformed mangiferin to the aglycone, norathyriol, suggesting cleavage of a C-glucosyl bond.',16141538),(3708,'human','null','Mangiferin','null','The C-glucosyl bond of C-glucosides generally tolerates acid and enzymatic hydrolysis. Many C-glucosides are cleaved by human intestinal bacteria. We isolated the specific bacterium involved in the metabolism of mangiferin (2-b-D-glucopyranosyl-1,3,6,7-tetrahydroxyxanthone), C-glucosyl xanthone, from a mixture of human fecal bacteria. The anaerobic Bacteroides species named MANG, transformed mangiferin to the aglycone, norathyriol, suggesting cleavage of a C-glucosyl bond.',16141538),(3709,'human','increase','null','low-throughput','5-HT storing enterochromaffin (EC) cells are believed to respond to nutrient and gut microbial components, and 5-HT receptor-expressing afferent vagal neurons have been described to be the major sensors of nutrients in the GI-tract. However, the molecular mechanism through which EC cells sense nutrients and gut microbiota is still unclear.Nutrient metabolites apparently do not stimulate EC cells of the small intestine directly but through a paracrine mechanism involving GLP-1 secreted from neighboring enteroendocrine cells. In contrast, colonic EC cells are able to sense a multitude of different metabolites generated by the gut microbiota as well as gut hormones, including GLP-1.',29576437),(3710,'human','increase','null','low-throughput','5-HT storing enterochromaffin (EC) cells are believed to respond to nutrient and gut microbial components, and 5-HT receptor-expressing afferent vagal neurons have been described to be the major sensors of nutrients in the GI-tract. However, the molecular mechanism through which EC cells sense nutrients and gut microbiota is still unclear.Nutrient metabolites apparently do not stimulate EC cells of the small intestine directly but through a paracrine mechanism involving GLP-1 secreted from neighboring enteroendocrine cells. In contrast, colonic EC cells are able to sense a multitude of different metabolites generated by the gut microbiota as well as gut hormones, including GLP-1.',29576437),(3711,'human','increase','null','low-throughput','5-HT storing enterochromaffin (EC) cells are believed to respond to nutrient and gut microbial components, and 5-HT receptor-expressing afferent vagal neurons have been described to be the major sensors of nutrients in the GI-tract. However, the molecular mechanism through which EC cells sense nutrients and gut microbiota is still unclear.Nutrient metabolites apparently do not stimulate EC cells of the small intestine directly but through a paracrine mechanism involving GLP-1 secreted from neighboring enteroendocrine cells. In contrast, colonic EC cells are able to sense a multitude of different metabolites generated by the gut microbiota as well as gut hormones, including GLP-1.',29576437),(3712,'human','decrease','null','low-throughput','The E.limosum produced butyrate, acetate, propionate, and lactate at 0.25, 1.0, 0.025 and 0.07 mmol/L, respectively in medium. At this concentration, each acid had no growth stimulating activity on T84 cells; however, when these acids were mixed together at the above levels, it showed significantly high activity than control. Except for lactate, these acids significantly attenuated IL-6 production at just 0.1 mmol/L. In addition, under TNF-a stimulation, butyrate attenuated the production of TLR4 mRNA. The treatment with E.limosum significantly attenuated clinical and histological scores of colitis with an increase of cecal butyrate levels, compared with the control group.',16534848),(3713,'human','decrease','null','low-throughput','The E.limosum produced butyrate, acetate, propionate, and lactate at 0.25, 1.0, 0.025 and 0.07 mmol/L, respectively in medium. At this concentration, each acid had no growth stimulating activity on T84 cells; however, when these acids were mixed together at the above levels, it showed significantly high activity than control. Except for lactate, these acids significantly attenuated IL-6 production at just 0.1 mmol/L. In addition, under TNF-a stimulation, butyrate attenuated the production of TLR4 mRNA. The treatment with E.limosum significantly attenuated clinical and histological scores of colitis with an increase of cecal butyrate levels, compared with the control group.',16534848),(3714,'human','decrease','null','low-throughput','The E.limosum produced butyrate, acetate, propionate, and lactate at 0.25, 1.0, 0.025 and 0.07 mmol/L, respectively in medium. At this concentration, each acid had no growth stimulating activity on T84 cells; however, when these acids were mixed together at the above levels, it showed significantly high activity than control. Except for lactate, these acids significantly attenuated IL-6 production at just 0.1 mmol/L. In addition, under TNF-a stimulation, butyrate attenuated the production of TLR4 mRNA. The treatment with E.limosum significantly attenuated clinical and histological scores of colitis with an increase of cecal butyrate levels, compared with the control group.',16534848),(3715,'human','decrease','null','low-throughput','The E.limosum produced butyrate, acetate, propionate, and lactate at 0.25, 1.0, 0.025 and 0.07 mmol/L, respectively in medium. At this concentration, each acid had no growth stimulating activity on T84 cells; however, when these acids were mixed together at the above levels, it showed significantly high activity than control. Except for lactate, these acids significantly attenuated IL-6 production at just 0.1 mmol/L. In addition, under TNF-a stimulation, butyrate attenuated the production of TLR4 mRNA. The treatment with E.limosum significantly attenuated clinical and histological scores of colitis with an increase of cecal butyrate levels, compared with the control group.',16534848),(3716,'human','null','Rutin','null','In this paper, rutin was metabolized by human intestinal bacteria and five isolated strains including Bacillus sp. 52, Bacteroides sp. 45, 42, 22 and Veillonella sp. 32, the metabolites were identified using ultra performance liquid chromatography/quadrupole-time-of-flight mass spectrometry (UPLC–Q-TOF/MS). As a result, Bacillus sp. 52 and Bacteroides sp. 45 could metabolize rutin to quercetin 3-O-glucoside and leucocyanidin. Bacteroides sp. 42 and Veillonella sp. 32 could convert rutin to leucocyanidin. Bacteroides sp. 22 could hydrolyze rutin to quercetin-3-O-glucoside. In order to further explain the metabolism pathway of rutin,the -d-glucosidase and -l-rhamnosidase activities offive strains weredetermined.',22583754),(3717,'human','null','Rutin','null','In this paper, rutin was metabolized by human intestinal bacteria and five isolated strains including Bacillus sp. 52, Bacteroides sp. 45, 42, 22 and Veillonella sp. 32, the metabolites were identified using ultra performance liquid chromatography/quadrupole-time-of-flight mass spectrometry (UPLC–Q-TOF/MS). As a result, Bacillus sp. 52 and Bacteroides sp. 45 could metabolize rutin to quercetin 3-O-glucoside and leucocyanidin. Bacteroides sp. 42 and Veillonella sp. 32 could convert rutin to leucocyanidin. Bacteroides sp. 22 could hydrolyze rutin to quercetin-3-O-glucoside. In order to further explain the metabolism pathway of rutin,the -d-glucosidase and -l-rhamnosidase activities offive strains weredetermined.',22583754),(3718,'human','null','Rutin','null','In this paper, rutin was metabolized by human intestinal bacteria and five isolated strains including Bacillus sp. 52, Bacteroides sp. 45, 42, 22 and Veillonella sp. 32, the metabolites were identified using ultra performance liquid chromatography/quadrupole-time-of-flight mass spectrometry (UPLC–Q-TOF/MS). As a result, Bacillus sp. 52 and Bacteroides sp. 45 could metabolize rutin to quercetin 3-O-glucoside and leucocyanidin. Bacteroides sp. 42 and Veillonella sp. 32 could convert rutin to leucocyanidin. Bacteroides sp. 22 could hydrolyze rutin to quercetin-3-O-glucoside. In order to further explain the metabolism pathway of rutin,the -d-glucosidase and -l-rhamnosidase activities offive strains weredetermined.',22583754),(3719,'human','null','Rutin','null','In this paper, rutin was metabolized by human intestinal bacteria and five isolated strains including Bacillus sp. 52, Bacteroides sp. 45, 42, 22 and Veillonella sp. 32, the metabolites were identified using ultra performance liquid chromatography/quadrupole-time-of-flight mass spectrometry (UPLC–Q-TOF/MS). As a result, Bacillus sp. 52 and Bacteroides sp. 45 could metabolize rutin to quercetin 3-O-glucoside and leucocyanidin. Bacteroides sp. 42 and Veillonella sp. 32 could convert rutin to leucocyanidin. Bacteroides sp. 22 could hydrolyze rutin to quercetin-3-O-glucoside. In order to further explain the metabolism pathway of rutin,the -d-glucosidase and -l-rhamnosidase activities offive strains weredetermined.',22583754),(3720,'human','null','Rutin','null','In this paper, rutin was metabolized by human intestinal bacteria and five isolated strains including Bacillus sp. 52, Bacteroides sp. 45, 42, 22 and Veillonella sp. 32, the metabolites were identified using ultra performance liquid chromatography/quadrupole-time-of-flight mass spectrometry (UPLC–Q-TOF/MS). As a result, Bacillus sp. 52 and Bacteroides sp. 45 could metabolize rutin to quercetin 3-O-glucoside and leucocyanidin. Bacteroides sp. 42 and Veillonella sp. 32 could convert rutin to leucocyanidin. Bacteroides sp. 22 could hydrolyze rutin to quercetin-3-O-glucoside. In order to further explain the metabolism pathway of rutin,the -d-glucosidase and -l-rhamnosidase activities offive strains weredetermined.',22583754),(3721,'human','null','Rutin','null','In this paper, rutin was metabolized by human intestinal bacteria and five isolated strains including Bacillus sp. 52, Bacteroides sp. 45, 42, 22 and Veillonella sp. 32, the metabolites were identified using ultra performance liquid chromatography/quadrupole-time-of-flight mass spectrometry (UPLC–Q-TOF/MS). As a result, Bacillus sp. 52 and Bacteroides sp. 45 could metabolize rutin to quercetin 3-O-glucoside and leucocyanidin. Bacteroides sp. 42 and Veillonella sp. 32 could convert rutin to leucocyanidin. Bacteroides sp. 22 could hydrolyze rutin to quercetin-3-O-glucoside. In order to further explain the metabolism pathway of rutin,the -d-glucosidase and -l-rhamnosidase activities offive strains weredetermined.',22583754),(3722,'human','null','Rutin','null','In this paper, rutin was metabolized by human intestinal bacteria and five isolated strains including Bacillus sp. 52, Bacteroides sp. 45, 42, 22 and Veillonella sp. 32, the metabolites were identified using ultra performance liquid chromatography/quadrupole-time-of-flight mass spectrometry (UPLC–Q-TOF/MS). As a result, Bacillus sp. 52 and Bacteroides sp. 45 could metabolize rutin to quercetin 3-O-glucoside and leucocyanidin. Bacteroides sp. 42 and Veillonella sp. 32 could convert rutin to leucocyanidin. Bacteroides sp. 22 could hydrolyze rutin to quercetin-3-O-glucoside. In order to further explain the metabolism pathway of rutin,the -d-glucosidase and -l-rhamnosidase activities offive strains weredetermined.',22583754),(3723,'human','null','Phytoestrogens','null','Nine healthy male adult subjects ingested 0.3 g/kg/day flaxseeds during 1 week. Gut bacteria as well as blood and fecal metabolites were analyzed. Ingestion of flaxseeds triggered a significant increase in the blood concentration of enterolignans, accompanied by fecal excretion of propionate and glycerol. Overall diversity and composition of dominant fecal bacteria remained individual specific throughout the study.',25988339),(3724,'human','null','Phytoestrogens','null','Nine healthy male adult subjects ingested 0.3 g/kg/day flaxseeds during 2 week. Gut bacteria as well as blood and fecal metabolites were analyzed. Ingestion of flaxseeds triggered a significant increase in the blood concentration of enterolignans, accompanied by fecal excretion of propionate and glycerol. Overall diversity and composition of dominant fecal bacteria remained individual specific throughout the study.',25988339),(3725,'human','null','Geniposide','null','Genipin, a deglycosylated form of geniposide, was cytotoxic at the concentrations that geniposide was not. As metabolic activation systems for geniposide, human intestinal bacterial cultures, fecal preparation (fecalase) and intestinal microbial enzyme mix were employed in the present study. When geniposide was incubated with human intestinal bacteria, either Bifidobacterium longum HY8001 or Bacteroides fragilis, for 24 h, the cultured media caused cytotoxicity in HepG3 cells.',22553067),(3726,'human','null','Geniposide','null','Genipin, a deglycosylated form of geniposide, was cytotoxic at the concentrations that geniposide was not. As metabolic activation systems for geniposide, human intestinal bacterial cultures, fecal preparation (fecalase) and intestinal microbial enzyme mix were employed in the present study. When geniposide was incubated with human intestinal bacteria, either Bifidobacterium longum HY8001 or Bacteroides fragilis, for 24 h, the cultured media caused cytotoxicity in HepG3 cells.',22553067),(3740,'human','increase','Retinol','low-throughput','We reported here that microbiota metabolite short-chain fatty acid acetate promoted intestinal IgA responses, which was mediated by \"metabolite-sensing\" GPR43. GPR43-/- mice demonstrated lower levels of intestinal IgA and IgA+ gut bacteria compared with those in wild type (WT) mice. Feeding WT but not GPR43-/- mice acetate but not butyrate promoted intestinal IgA response independent of T cells.',27966553),(3741,'human','null','Bile acid','null','The most important bile acid transformation is 7α-dehydroxylation, producing deoxycholic acid (DCA) and lithocholic acid (LCA). The enzymatic pathway carrying out 7α-dehydroxylation is restricted to a narrow phylogenetic group of commensal bacteria, the best-characterized of which is Clostridium scindens.we used a recently established, standardized gnotobiotic mouse model that is stably associated with a simplified murine 12-species \"oligo-mouse microbiota\" (Oligo-MM12).',28066726),(3742,'human','null','Bile acid','null','The most important bile acid transformation is 7α-dehydroxylation, producing deoxycholic acid (DCA) and lithocholic acid (LCA). The enzymatic pathway carrying out 7α-dehydroxylation is restricted to a narrow phylogenetic group of commensal bacteria, the best-characterized of which is Clostridium scindens.we used a recently established, standardized gnotobiotic mouse model that is stably associated with a simplified murine 12-species \"oligo-mouse microbiota\" (Oligo-MM12).',28066726),(3743,'human','decrease','null','low-throughput','Dietary and bacterial metabolites influence immune responses. This raises the question whether the increased incidence of allergies, asthma, some autoimmune diseases, cardiovascular disease, and others might relate to intake of unhealthy foods, and the decreased intake of dietary fiber. In recent years, new knowledge on the molecular mechanisms underpinning a \'diet-gut microbiota-physiology axis\' has emerged to substantiate this idea. Fiber is fermented to short chain fatty acids (SCFAs), particularly acetate, butyrate, and propionate. These metabolites bind \'metabolite-sensing\' G-protein-coupled receptors such as GPR43, GPR41, and GPR109A. These receptors play fundamental roles in the promotion of gut homeostasis and the regulation of inflammatory responses. For instance, these receptors and their metabolites influence Treg biology, epithelial integrity, gut homeostasis, DC biology, and IgA antibody responses. The SCFAs also influence gene transcription in many cells and tissues, through their inhibition of histone deacetylase expression or function.',28658542),(3744,'human','decrease','null','low-throughput','Dietary and bacterial metabolites influence immune responses. This raises the question whether the increased incidence of allergies, asthma, some autoimmune diseases, cardiovascular disease, and others might relate to intake of unhealthy foods, and the decreased intake of dietary fiber. In recent years, new knowledge on the molecular mechanisms underpinning a \'diet-gut microbiota-physiology axis\' has emerged to substantiate this idea. Fiber is fermented to short chain fatty acids (SCFAs), particularly acetate, butyrate, and propionate. These metabolites bind \'metabolite-sensing\' G-protein-coupled receptors such as GPR43, GPR41, and GPR109A. These receptors play fundamental roles in the promotion of gut homeostasis and the regulation of inflammatory responses. For instance, these receptors and their metabolites influence Treg biology, epithelial integrity, gut homeostasis, DC biology, and IgA antibody responses. The SCFAs also influence gene transcription in many cells and tissues, through their inhibition of histone deacetylase expression or function.',28658542),(3745,'human','decrease','null','low-throughput','Dietary and bacterial metabolites influence immune responses. This raises the question whether the increased incidence of allergies, asthma, some autoimmune diseases, cardiovascular disease, and others might relate to intake of unhealthy foods, and the decreased intake of dietary fiber. In recent years, new knowledge on the molecular mechanisms underpinning a \'diet-gut microbiota-physiology axis\' has emerged to substantiate this idea. Fiber is fermented to short chain fatty acids (SCFAs), particularly acetate, butyrate, and propionate. These metabolites bind \'metabolite-sensing\' G-protein-coupled receptors such as GPR43, GPR41, and GPR109A. These receptors play fundamental roles in the promotion of gut homeostasis and the regulation of inflammatory responses. For instance, these receptors and their metabolites influence Treg biology, epithelial integrity, gut homeostasis, DC biology, and IgA antibody responses. The SCFAs also influence gene transcription in many cells and tissues, through their inhibition of histone deacetylase expression or function.',28658542),(3746,'human','decrease','null','low-throughput','Dietary and bacterial metabolites influence immune responses. This raises the question whether the increased incidence of allergies, asthma, some autoimmune diseases, cardiovascular disease, and others might relate to intake of unhealthy foods, and the decreased intake of dietary fiber. In recent years, new knowledge on the molecular mechanisms underpinning a \'diet-gut microbiota-physiology axis\' has emerged to substantiate this idea. Fiber is fermented to short chain fatty acids (SCFAs), particularly acetate, butyrate, and propionate. These metabolites bind \'metabolite-sensing\' G-protein-coupled receptors such as GPR43, GPR41, and GPR109A. These receptors play fundamental roles in the promotion of gut homeostasis and the regulation of inflammatory responses. For instance, these receptors and their metabolites influence Treg biology, epithelial integrity, gut homeostasis, DC biology, and IgA antibody responses. The SCFAs also influence gene transcription in many cells and tissues, through their inhibition of histone deacetylase expression or function.',28658542),(3747,'human','increase','null','low-throughput','Dietary and bacterial metabolites influence immune responses. This raises the question whether the increased incidence of allergies, asthma, some autoimmune diseases, cardiovascular disease, and others might relate to intake of unhealthy foods, and the decreased intake of dietary fiber. In recent years, new knowledge on the molecular mechanisms underpinning a \'diet-gut microbiota-physiology axis\' has emerged to substantiate this idea. Fiber is fermented to short chain fatty acids (SCFAs), particularly acetate, butyrate, and propionate. These metabolites bind \'metabolite-sensing\' G-protein-coupled receptors such as GPR43, GPR41, and GPR109A. These receptors play fundamental roles in the promotion of gut homeostasis and the regulation of inflammatory responses. For instance, these receptors and their metabolites influence Treg biology, epithelial integrity, gut homeostasis, DC biology, and IgA antibody responses. The SCFAs also influence gene transcription in many cells and tissues, through their inhibition of histone deacetylase expression or function.',28658542),(3748,'human','increase','Tryptophan','low-throughput','Dietary and bacterial metabolites influence immune responses. This raises the question whether the increased incidence of allergies, asthma, some autoimmune diseases, cardiovascular disease, and others might relate to intake of unhealthy foods, and the decreased intake of dietary fiber. In recent years, new knowledge on the molecular mechanisms underpinning a \'diet-gut microbiota-physiology axis\' has emerged to substantiate this idea. Fiber is fermented to short chain fatty acids (SCFAs), particularly acetate, butyrate, and propionate. These metabolites bind \'metabolite-sensing\' G-protein-coupled receptors such as GPR43, GPR41, and GPR109A. These receptors play fundamental roles in the promotion of gut homeostasis and the regulation of inflammatory responses. For instance, these receptors and their metabolites influence Treg biology, epithelial integrity, gut homeostasis, DC biology, and IgA antibody responses. The SCFAs also influence gene transcription in many cells and tissues, through their inhibition of histone deacetylase expression or function.',28658542),(3749,'human','increase','Linoleic Acid','low-throughput','In this study, we investigated the effects of novel fatty acid metabolite derivatives of linoleic acid generated by the gut lactic acid bacteria Lactobacillus plantarum on the Nrf2-ARE pathway. 10-Oxo-trans-11-octadecenoic acid (KetoC) protected HepG2 cells from cytotoxicity induced by hydrogen peroxide. KetoC also significantly increased cellular Nrf2 protein levels, ARE-dependent transcription, and the gene expression of antioxidative enzymes such as heme oxygenase-1 (HO-1), glutamate-cysteine ligase modifier subunit (GCLM), and Nad(p)h: quinone oxidoreductase 1 (NQO1) in HepG2 cells.',26879219),(3750,'human','increase','Linoleic Acid','low-throughput','In this study, we investigated the effects of novel fatty acid metabolite derivatives of linoleic acid generated by the gut lactic acid bacteria Lactobacillus plantarum on the Nrf2-ARE pathway. 10-Oxo-trans-11-octadecenoic acid (KetoC) protected HepG2 cells from cytotoxicity induced by hydrogen peroxide. KetoC also significantly increased cellular Nrf2 protein levels, ARE-dependent transcription, and the gene expression of antioxidative enzymes such as heme oxygenase-1 (HO-1), glutamate-cysteine ligase modifier subunit (GCLM), and Nad(p)h: quinone oxidoreductase 1 (NQO1) in HepG2 cells.',26879219),(3751,'human','null','Rutin','null','The average α-Lrhamnosidase activities on the p-nitrophenyl-α-L-rhamnopyranoside, rutin, and poncirin subtrates were 0.10 ± 0.07, 0.25 ± 0.08, and 0.15 ± 0.09 pmol/min/mg, respectively. To investigate the enzymatic properties, α-L-rhamnosidase-producing bacteria were isolated from the specimens, and the α-L-rhamnosidase gene was cloned from a selected organism, Bifidobacterium dentium, and expressed in E. coli. The cloned α-L-rhamnosidase gene contained a 2,673 bp sequcence encoding 890 amino acid residues. The cloned gene was expressed using the pET 26b(+) vector in E. coli BL21, and the expressed enzyme was purified using Ni(2+)-NTA and Q-HP column chromatography. The specific activity of the purified α-L-rhamnosidase was 23.3 μmol/min/mg.',25179902),(3752,'human','null','Poncirin','null','The average α-Lrhamnosidase activities on the p-nitrophenyl-α-L-rhamnopyranoside, rutin, and poncirin subtrates were 0.10 ± 0.07, 0.25 ± 0.08, and 0.15 ± 0.09 pmol/min/mg, respectively. To investigate the enzymatic properties, α-L-rhamnosidase-producing bacteria were isolated from the specimens, and the α-L-rhamnosidase gene was cloned from a selected organism, Bifidobacterium dentium, and expressed in E. coli. The cloned α-L-rhamnosidase gene contained a 2,673 bp sequcence encoding 890 amino acid residues. The cloned gene was expressed using the pET 26b(+) vector in E. coli BL21, and the expressed enzyme was purified using Ni(2+)-NTA and Q-HP column chromatography. The specific activity of the purified α-L-rhamnosidase was 23.3 μmol/min/mg.',25179902),(3753,'human','decrease','null','low-throughput','A feature of human ulcerative colitis and other colitic diseases is a change in \'healthy\' microbiota such as Bifidobacterium and Bacteriodes, and a concurrent reduction in SCFAs. Moreover, increased intake of fermentable dietary fibre, or SCFAs, seems to be clinically beneficial in the treatment of colitis. SCFAs bind the G-protein-coupled receptor 43 (GPR43, also known as FFAR2), and here we show that SCFA-GPR43 interactions profoundly affect inflammatory responses. Stimulation of GPR43 by SCFAs was necessary for the normal resolution of certain inflammatory responses, because GPR43-deficient (Gpr43(-/-)) mice showed exacerbated or unresolving inflammation in models of colitis, arthritis and asthma. This seemed to relate to increased production of inflammatory mediators by Gpr43(-/-) immune cells, and increased immune cell recruitment. Germ-free mice, which are devoid of bacteria and express little or no SCFAs, showed a similar dysregulation of certain inflammatory responses. GPR43 binding of SCFAs potentially provides a molecular link between diet, gastrointestinal bacterial metabolism, and immune and inflammatory responses.',19865172),(3754,'human','decrease','null','low-throughput','A feature of human ulcerative colitis and other colitic diseases is a change in \'healthy\' microbiota such as Bifidobacterium and Bacteriodes, and a concurrent reduction in SCFAs. Moreover, increased intake of fermentable dietary fibre, or SCFAs, seems to be clinically beneficial in the treatment of colitis. SCFAs bind the G-protein-coupled receptor 43 (GPR43, also known as FFAR2), and here we show that SCFA-GPR43 interactions profoundly affect inflammatory responses. Stimulation of GPR43 by SCFAs was necessary for the normal resolution of certain inflammatory responses, because GPR43-deficient (Gpr43(-/-)) mice showed exacerbated or unresolving inflammation in models of colitis, arthritis and asthma. This seemed to relate to increased production of inflammatory mediators by Gpr43(-/-) immune cells, and increased immune cell recruitment. Germ-free mice, which are devoid of bacteria and express little or no SCFAs, showed a similar dysregulation of certain inflammatory responses. GPR43 binding of SCFAs potentially provides a molecular link between diet, gastrointestinal bacterial metabolism, and immune and inflammatory responses.',19865172),(3755,'human','decrease','null','low-throughput','A feature of human ulcerative colitis and other colitic diseases is a change in \'healthy\' microbiota such as Bifidobacterium and Bacteriodes, and a concurrent reduction in SCFAs. Moreover, increased intake of fermentable dietary fibre, or SCFAs, seems to be clinically beneficial in the treatment of colitis. SCFAs bind the G-protein-coupled receptor 43 (GPR43, also known as FFAR2), and here we show that SCFA-GPR43 interactions profoundly affect inflammatory responses. Stimulation of GPR43 by SCFAs was necessary for the normal resolution of certain inflammatory responses, because GPR43-deficient (Gpr43(-/-)) mice showed exacerbated or unresolving inflammation in models of colitis, arthritis and asthma. This seemed to relate to increased production of inflammatory mediators by Gpr43(-/-) immune cells, and increased immune cell recruitment. Germ-free mice, which are devoid of bacteria and express little or no SCFAs, showed a similar dysregulation of certain inflammatory responses. GPR43 binding of SCFAs potentially provides a molecular link between diet, gastrointestinal bacterial metabolism, and immune and inflammatory responses.',19865172),(3756,'human','decrease','null','low-throughput','A feature of human ulcerative colitis and other colitic diseases is a change in \'healthy\' microbiota such as Bifidobacterium and Bacteriodes, and a concurrent reduction in SCFAs. Moreover, increased intake of fermentable dietary fibre, or SCFAs, seems to be clinically beneficial in the treatment of colitis. SCFAs bind the G-protein-coupled receptor 43 (GPR43, also known as FFAR2), and here we show that SCFA-GPR43 interactions profoundly affect inflammatory responses. Stimulation of GPR43 by SCFAs was necessary for the normal resolution of certain inflammatory responses, because GPR43-deficient (Gpr43(-/-)) mice showed exacerbated or unresolving inflammation in models of colitis, arthritis and asthma. This seemed to relate to increased production of inflammatory mediators by Gpr43(-/-) immune cells, and increased immune cell recruitment. Germ-free mice, which are devoid of bacteria and express little or no SCFAs, showed a similar dysregulation of certain inflammatory responses. GPR43 binding of SCFAs potentially provides a molecular link between diet, gastrointestinal bacterial metabolism, and immune and inflammatory responses.',19865172),(3757,'human','decrease','Isoflavone','low-throughput','The genome of new equol-producing bacteria, Lactobacillus paracasei JS1, was analyzed. Also, the effect of P. lobata extract fermented with L. paracasei JS1 (FPE) on the skin and intestinal immune response was examined. With gene expression analysis, it was proven that seven skin-related proteins, hyaluronan synthase-1, -2, -3, collagen, elastin, epidermal growth factor, and epidermal growth factor receptor were differentially expressed upon FPE treatment. The messenger RNA expression increased with treatment with the FPE, and a skin moisturizing effect was confirmed by a hematoxylin-eosin staining experiment. In addition, such an experiment showed that proinflammatory cytokines, tumor necrosis factor-α, cyclooxygenase-2, inducible nitric oxide synthase, interleukin-1β, -4, and -6, were reduced in large intestine when treated with FPE.',30767231),(3758,'human','decrease','Isoflavone','low-throughput','The genome of new equol-producing bacteria, Lactobacillus paracasei JS1, was analyzed. Also, the effect of P. lobata extract fermented with L. paracasei JS1 (FPE) on the skin and intestinal immune response was examined. With gene expression analysis, it was proven that seven skin-related proteins, hyaluronan synthase-1, -2, -3, collagen, elastin, epidermal growth factor, and epidermal growth factor receptor were differentially expressed upon FPE treatment. The messenger RNA expression increased with treatment with the FPE, and a skin moisturizing effect was confirmed by a hematoxylin-eosin staining experiment. In addition, such an experiment showed that proinflammatory cytokines, tumor necrosis factor-α, cyclooxygenase-2, inducible nitric oxide synthase, interleukin-1β, -4, and -6, were reduced in large intestine when treated with FPE.',30767231),(3759,'human','decrease','Isoflavone','low-throughput','The genome of new equol-producing bacteria, Lactobacillus paracasei JS1, was analyzed. Also, the effect of P. lobata extract fermented with L. paracasei JS1 (FPE) on the skin and intestinal immune response was examined. With gene expression analysis, it was proven that seven skin-related proteins, hyaluronan synthase-1, -2, -3, collagen, elastin, epidermal growth factor, and epidermal growth factor receptor were differentially expressed upon FPE treatment. The messenger RNA expression increased with treatment with the FPE, and a skin moisturizing effect was confirmed by a hematoxylin-eosin staining experiment. In addition, such an experiment showed that proinflammatory cytokines, tumor necrosis factor-α, cyclooxygenase-2, inducible nitric oxide synthase, interleukin-1β, -4, and -6, were reduced in large intestine when treated with FPE.',30767231),(3760,'human','decrease','Isoflavone','low-throughput','The genome of new equol-producing bacteria, Lactobacillus paracasei JS1, was analyzed. Also, the effect of P. lobata extract fermented with L. paracasei JS1 (FPE) on the skin and intestinal immune response was examined. With gene expression analysis, it was proven that seven skin-related proteins, hyaluronan synthase-1, -2, -3, collagen, elastin, epidermal growth factor, and epidermal growth factor receptor were differentially expressed upon FPE treatment. The messenger RNA expression increased with treatment with the FPE, and a skin moisturizing effect was confirmed by a hematoxylin-eosin staining experiment. In addition, such an experiment showed that proinflammatory cytokines, tumor necrosis factor-α, cyclooxygenase-2, inducible nitric oxide synthase, interleukin-1β, -4, and -6, were reduced in large intestine when treated with FPE.',30767231),(3761,'human','decrease','Isoflavone','low-throughput','The genome of new equol-producing bacteria, Lactobacillus paracasei JS1, was analyzed. Also, the effect of P. lobata extract fermented with L. paracasei JS1 (FPE) on the skin and intestinal immune response was examined. With gene expression analysis, it was proven that seven skin-related proteins, hyaluronan synthase-1, -2, -3, collagen, elastin, epidermal growth factor, and epidermal growth factor receptor were differentially expressed upon FPE treatment. The messenger RNA expression increased with treatment with the FPE, and a skin moisturizing effect was confirmed by a hematoxylin-eosin staining experiment. In addition, such an experiment showed that proinflammatory cytokines, tumor necrosis factor-α, cyclooxygenase-2, inducible nitric oxide synthase, interleukin-1β, -4, and -6, were reduced in large intestine when treated with FPE.',30767231),(3762,'human','decrease','Isoflavone','low-throughput','The genome of new equol-producing bacteria, Lactobacillus paracasei JS1, was analyzed. Also, the effect of P. lobata extract fermented with L. paracasei JS1 (FPE) on the skin and intestinal immune response was examined. With gene expression analysis, it was proven that seven skin-related proteins, hyaluronan synthase-1, -2, -3, collagen, elastin, epidermal growth factor, and epidermal growth factor receptor were differentially expressed upon FPE treatment. The messenger RNA expression increased with treatment with the FPE, and a skin moisturizing effect was confirmed by a hematoxylin-eosin staining experiment. In addition, such an experiment showed that proinflammatory cytokines, tumor necrosis factor-α, cyclooxygenase-2, inducible nitric oxide synthase, interleukin-1β, -4, and -6, were reduced in large intestine when treated with FPE.',30767231),(3763,'human','null','Galbeta1,3GlcNAc','null','In the current study, we isolated a novel strain of Bifidobacterium kashiwanohense, named APCKJ1, from the faeces of a four-week old breastfed infant, based on the ability of the strain to utilise the HMO component fucosyllactose. We then determined the full genome sequence of this strain, and employed the generated data to analyze fucosyllactose metabolism in B. kashiwanohense APCKJ1. Transcriptomic and growth analyses, combined with metabolite analysis, in vitro hydrolysis assays and heterologous expression, allowed us to elucidate the pathway for fucosyllactose metabolism in B. kashiwanohense APCKJ1. Homologs of the key genes for this metabolic pathway were identified in particular in infant-derived members of the Bifdobacterium genus, revealing the apparent niche-specific nature of this pathway, and allowing a broad perspective on bifidobacterial fucosyllactose and L-fucose metabolism.',31659215),(3764,'human','null','N-Acetyl-D-lactosamine','null','In the current study, we isolated a novel strain of Bifidobacterium kashiwanohense, named APCKJ1, from the faeces of a four-week old breastfed infant, based on the ability of the strain to utilise the HMO component fucosyllactose. We then determined the full genome sequence of this strain, and employed the generated data to analyze fucosyllactose metabolism in B. kashiwanohense APCKJ1. Transcriptomic and growth analyses, combined with metabolite analysis, in vitro hydrolysis assays and heterologous expression, allowed us to elucidate the pathway for fucosyllactose metabolism in B. kashiwanohense APCKJ1. Homologs of the key genes for this metabolic pathway were identified in particular in infant-derived members of the Bifdobacterium genus, revealing the apparent niche-specific nature of this pathway, and allowing a broad perspective on bifidobacterial fucosyllactose and L-fucose metabolism.',31659215),(3765,'human','increase','null','low-throughput','A Prevotella and an Atopobium strain were chosen for further study, and shown to up-regulate two PPARγ-target genes, ANGPTL4 and ADRP. The molecular mechanisms of these activations involved the phosphorylation of PPARγ through ERK1/2. The responsible metabolites were shown to be heat sensitive but markedly diverged in size, emphasizing the diversity of bioactive compounds found in the intestine. Here we describe different mechanisms by which single intestinal bacteria can directly impact their host\'s health through transcriptional regulation.',28266623),(3766,'human','increase','null','low-throughput','A Prevotella and an Atopobium strain were chosen for further study, and shown to up-regulate two PPARγ-target genes, ANGPTL4 and ADRP. The molecular mechanisms of these activations involved the phosphorylation of PPARγ through ERK1/2. The responsible metabolites were shown to be heat sensitive but markedly diverged in size, emphasizing the diversity of bioactive compounds found in the intestine. Here we describe different mechanisms by which single intestinal bacteria can directly impact their host\'s health through transcriptional regulation.',28266623),(3767,'human','increase','null','low-throughput','A Prevotella and an Atopobium strain were chosen for further study, and shown to up-regulate two PPARγ-target genes, ANGPTL4 and ADRP. The molecular mechanisms of these activations involved the phosphorylation of PPARγ through ERK1/2. The responsible metabolites were shown to be heat sensitive but markedly diverged in size, emphasizing the diversity of bioactive compounds found in the intestine. Here we describe different mechanisms by which single intestinal bacteria can directly impact their host\'s health through transcriptional regulation.',28266623),(3768,'human','increase','null','low-throughput','A Prevotella and an Atopobium strain were chosen for further study, and shown to up-regulate two PPARγ-target genes, ANGPTL4 and ADRP. The molecular mechanisms of these activations involved the phosphorylation of PPARγ through ERK1/2. The responsible metabolites were shown to be heat sensitive but markedly diverged in size, emphasizing the diversity of bioactive compounds found in the intestine. Here we describe different mechanisms by which single intestinal bacteria can directly impact their host\'s health through transcriptional regulation.',28266623),(3769,'human','null','Daidzein','null','Several kinds of carbohydrates such as sorbose, adonitol, and melezitose were found to enhance equol production from daidzein in an in vitro human fecal culture. Sorbose, one of the most effective carbohydrates, was used as a carbohydrate source for isolating the NATTS strain, which was a Gram-positive, non-spore-forming rod bacterium with high ability to convert daidzein to equol isolated from the 7th maintenance culture. The strain was found to belong to the genus Slackia family Coriobacteriaceae by 16S rRNA sequence-based analysis, and the prevalence of the Slackia sp. in Japanese adults was examined by reverse transcription-quantitative PCR (RT-qPCR), which was found to be 40% at a mean population level of 10(6) cells per gram of feces.',20237913),(3770,'human','null','Daidzin','null','Several kinds of carbohydrates such as sorbose, adonitol, and melezitose were found to enhance equol production from daidzein in an in vitro human fecal culture. Sorbose, one of the most effective carbohydrates, was used as a carbohydrate source for isolating the NATTS strain, which was a Gram-positive, non-spore-forming rod bacterium with high ability to convert daidzein to equol isolated from the 7th maintenance culture. The strain was found to belong to the genus Slackia family Coriobacteriaceae by 16S rRNA sequence-based analysis, and the prevalence of the Slackia sp. in Japanese adults was examined by reverse transcription-quantitative PCR (RT-qPCR), which was found to be 40% at a mean population level of 10(6) cells per gram of feces.',20237913),(3771,'human','null','Daidzein','null','Several kinds of carbohydrates such as sorbose, adonitol, and melezitose were found to enhance equol production from daidzein in an in vitro human fecal culture. Sorbose, one of the most effective carbohydrates, was used as a carbohydrate source for isolating the NATTS strain, which was a Gram-positive, non-spore-forming rod bacterium with high ability to convert daidzein to equol isolated from the 7th maintenance culture. The strain was found to belong to the genus Slackia family Coriobacteriaceae by 16S rRNA sequence-based analysis, and the prevalence of the Slackia sp. in Japanese adults was examined by reverse transcription-quantitative PCR (RT-qPCR), which was found to be 40% at a mean population level of 10(6) cells per gram of feces.',20237913),(3772,'human','null','Daidzein','null','Several kinds of carbohydrates such as sorbose, adonitol, and melezitose were found to enhance equol production from daidzein in an in vitro human fecal culture. Sorbose, one of the most effective carbohydrates, was used as a carbohydrate source for isolating the NATTS strain, which was a Gram-positive, non-spore-forming rod bacterium with high ability to convert daidzein to equol isolated from the 7th maintenance culture. The strain was found to belong to the genus Slackia family Coriobacteriaceae by 16S rRNA sequence-based analysis, and the prevalence of the Slackia sp. in Japanese adults was examined by reverse transcription-quantitative PCR (RT-qPCR), which was found to be 40% at a mean population level of 10(6) cells per gram of feces.',20237913),(3773,'human','null','Puerarin','null','A rod-shaped Gram-positive bacterium, strain CG19-1, capable of deglycosylating puerarin to daidzein was isolated from the corresponding suspension. However, the strictly anaerobic isolate was unable to utilize puerarin as sole carbon and energy source nor any of the tested carbohydrates. Comparative 16S rRNA sequence analysis indicated that strain CG19-1 is a new species of the Lachnospiraceae. Strain CG19-1 also converted other aromatic C-glucosides in addition to puerarin. The xanthone C-glucoside mangiferin was deglycosylated to norathyriol. The flavone C-glucosides homoorientin and vitexin were degraded to 3-(3,4-dihydroxyphenyl)propionic acid via luteolin and 3-(4-hydroxyphenyl)propionic acid respectively. In addition, strain CG19-1 converted flavonoid O-glucosides, but at rates that were lower than those of the C-glucosides tested. The isolate deglycosylated the isoflavone O-glucosides daidzin and genistin to daidzein and genistein respectively. Several O-glucosides of the flavones luteolin and apigenin undergoing deglycosylation were subsequently cleaved to 3-(3,4-dihydroxyphenyl)propionic acid and 3-(4-hydroxyphenyl)propionic acid respectively. Moreover, strain CG19-1 cleaved both O-desmethylangolensin and 6\'-hydroxy-O-desmethylangolensin to yield 2-(4-dihydroxyphenyl)propionic acid. The corresponding cleavage product, resorcinol, was only observed for O-desmethylangolensin.',20946528),(3774,'human','null','Vitexin','null','A rod-shaped Gram-positive bacterium, strain CG19-1, capable of deglycosylating puerarin to daidzein was isolated from the corresponding suspension. However, the strictly anaerobic isolate was unable to utilize puerarin as sole carbon and energy source nor any of the tested carbohydrates. Comparative 16S rRNA sequence analysis indicated that strain CG19-1 is a new species of the Lachnospiraceae. Strain CG19-1 also converted other aromatic C-glucosides in addition to puerarin. The xanthone C-glucoside mangiferin was deglycosylated to norathyriol. The flavone C-glucosides homoorientin and vitexin were degraded to 3-(3,4-dihydroxyphenyl)propionic acid via luteolin and 3-(4-hydroxyphenyl)propionic acid respectively. In addition, strain CG19-1 converted flavonoid O-glucosides, but at rates that were lower than those of the C-glucosides tested. The isolate deglycosylated the isoflavone O-glucosides daidzin and genistin to daidzein and genistein respectively. Several O-glucosides of the flavones luteolin and apigenin undergoing deglycosylation were subsequently cleaved to 3-(3,4-dihydroxyphenyl)propionic acid and 3-(4-hydroxyphenyl)propionic acid respectively. Moreover, strain CG19-1 cleaved both O-desmethylangolensin and 6\'-hydroxy-O-desmethylangolensin to yield 2-(4-dihydroxyphenyl)propionic acid. The corresponding cleavage product, resorcinol, was only observed for O-desmethylangolensin.',20946528),(3775,'human','null','O-Desmethylangolensin','null','A rod-shaped Gram-positive bacterium, strain CG19-1, capable of deglycosylating puerarin to daidzein was isolated from the corresponding suspension. However, the strictly anaerobic isolate was unable to utilize puerarin as sole carbon and energy source nor any of the tested carbohydrates. Comparative 16S rRNA sequence analysis indicated that strain CG19-1 is a new species of the Lachnospiraceae. Strain CG19-1 also converted other aromatic C-glucosides in addition to puerarin. The xanthone C-glucoside mangiferin was deglycosylated to norathyriol. The flavone C-glucosides homoorientin and vitexin were degraded to 3-(3,4-dihydroxyphenyl)propionic acid via luteolin and 3-(4-hydroxyphenyl)propionic acid respectively. In addition, strain CG19-1 converted flavonoid O-glucosides, but at rates that were lower than those of the C-glucosides tested. The isolate deglycosylated the isoflavone O-glucosides daidzin and genistin to daidzein and genistein respectively. Several O-glucosides of the flavones luteolin and apigenin undergoing deglycosylation were subsequently cleaved to 3-(3,4-dihydroxyphenyl)propionic acid and 3-(4-hydroxyphenyl)propionic acid respectively. Moreover, strain CG19-1 cleaved both O-desmethylangolensin and 6\'-hydroxy-O-desmethylangolensin to yield 2-(4-dihydroxyphenyl)propionic acid. The corresponding cleavage product, resorcinol, was only observed for O-desmethylangolensin.',20946528),(3776,'human','null','6\'-Hydroxy-O-desmethylangolensin','null','A rod-shaped Gram-positive bacterium, strain CG19-1, capable of deglycosylating puerarin to daidzein was isolated from the corresponding suspension. However, the strictly anaerobic isolate was unable to utilize puerarin as sole carbon and energy source nor any of the tested carbohydrates. Comparative 16S rRNA sequence analysis indicated that strain CG19-1 is a new species of the Lachnospiraceae. Strain CG19-1 also converted other aromatic C-glucosides in addition to puerarin. The xanthone C-glucoside mangiferin was deglycosylated to norathyriol. The flavone C-glucosides homoorientin and vitexin were degraded to 3-(3,4-dihydroxyphenyl)propionic acid via luteolin and 3-(4-hydroxyphenyl)propionic acid respectively. In addition, strain CG19-1 converted flavonoid O-glucosides, but at rates that were lower than those of the C-glucosides tested. The isolate deglycosylated the isoflavone O-glucosides daidzin and genistin to daidzein and genistein respectively. Several O-glucosides of the flavones luteolin and apigenin undergoing deglycosylation were subsequently cleaved to 3-(3,4-dihydroxyphenyl)propionic acid and 3-(4-hydroxyphenyl)propionic acid respectively. Moreover, strain CG19-1 cleaved both O-desmethylangolensin and 6\'-hydroxy-O-desmethylangolensin to yield 2-(4-dihydroxyphenyl)propionic acid. The corresponding cleavage product, resorcinol, was only observed for O-desmethylangolensin.',20946528),(3777,'human','null','(3R)-flavan-3-ols','null','A human intestinal bacterium, Eubacterium (E.) sp. strain SDG-2, was tested for its ability to metabolize various (3R)- and (3S)-flavan-3-ols and their 3-O-gallates. This bacterium cleaved the C-ring of (3R)- and (3S)-flavan-3-ols to give 1,3-diphenylpropan-2-ol derivatives, but not their 3-O-gallates. Furthermore, E. sp. strain SDG-2 had the ability of p-dehydroxylation in the B-ring of (3R)-flavan-3-ols, such as (-)-catechin, (-)-epicatechin, (-)-gallocatechin and (-)-epigallocatechin, but not of (3S)-flavan-3-ols, such as (+)-catechin and (+)-epicatechin.',11767089),(3778,'human','null','(3S)-flavan-3-ols','null','A human intestinal bacterium, Eubacterium (E.) sp. strain SDG-2, was tested for its ability to metabolize various (3R)- and (3S)-flavan-3-ols and their 3-O-gallates. This bacterium cleaved the C-ring of (3R)- and (3S)-flavan-3-ols to give 1,3-diphenylpropan-2-ol derivatives, but not their 3-O-gallates. Furthermore, E. sp. strain SDG-2 had the ability of p-dehydroxylation in the B-ring of (3R)-flavan-3-ols, such as (-)-catechin, (-)-epicatechin, (-)-gallocatechin and (-)-epigallocatechin, but not of (3S)-flavan-3-ols, such as (+)-catechin and (+)-epicatechin.',11767089),(3803,'human','increase','Ginsenoside rb1','low-throughput','Compound K significantly up‑regulated expression of p21 (CDKN1A), and downregulated the expressions of CDK6, cyclin D and cyclin E, which are G1 phase arrest regulators.',29805567),(3804,'human','decrease','Ginsenoside rb1','low-throughput','Compound K significantly up‑regulated expression of p21 (CDKN1A), and downregulated the expressions of CDK6, cyclin D and cyclin E, which are G1 phase arrest regulators.',29805567),(3805,'human','decrease','Ginsenoside rb1','low-throughput','Compound K significantly up‑regulated expression of p21 (CDKN1A), and downregulated the expressions of CDK6, cyclin D and cyclin E, which are G1 phase arrest regulators.',29805567),(3806,'human','decrease','Ginsenoside rb1','low-throughput','Compound K significantly up‑regulated expression of p21 (CDKN1A), and downregulated the expressions of CDK6, cyclin D and cyclin E, which are G1 phase arrest regulators.',29805567),(3807,'human','increase','Ginsenoside rb1','low-throughput','Compound K up-regulated the expressions of p53 and Bax,  which are pro-apoptotic genes. In contrast, compound K down-regulated the expression of Bcl2, which is an anti-apoptotic gene.',29805567),(3808,'human','increase','Ginsenoside rb1','low-throughput','Compound K up-regulated the expressions of p53 and Bax,  which are pro-apoptotic genes. In contrast, compound K down-regulated the expression of Bcl2, which is an anti-apoptotic gene.',29805567),(3809,'human','decrease','Ginsenoside rb1','low-throughput','Compound K up-regulated the expressions of p53 and Bax,  which are pro-apoptotic genes. In contrast, compound K down-regulated the expression of Bcl2, which is an anti-apoptotic gene.',29805567),(3810,'human','decrease','Ginsenoside rb1','low-throughput','Compound K significantly reduced the IL-8 production in HT-29 cells.',29805567),(3817,'human','null','Levodopa','null','Gut-associated bacteria have the abality to decarboxylate levodopa to dopamine via tyrosine decarboxylases.',30659181),(3818,'human','null','PhIP-N2-β-D-glucuronide','null','B-GUS of Faecalibacterium prausnitzii converted PhIP-G to PhIP and GDH of Flavonifractor plautii, Blautia obeum, Eubacterium hallii, and Lactobacillus reuteri converted PhIP to PhIP-M1 in the presence of glycerol. In addition, B-GUS- and GDH-positive bacteria cooperatively converted PhIP-G to PhIP-M1.',31096909),(3819,'human','null','2-Amino-1-methyl-6-phenylimidazo[4,5-b]pyridine','null','B-GUS of Faecalibacterium prausnitzii converted PhIP-G to PhIP and GDH of Flavonifractor plautii, Blautia obeum, Eubacterium hallii, and Lactobacillus reuteri converted PhIP to PhIP-M1 in the presence of glycerol. In addition, B-GUS- and GDH-positive bacteria cooperatively converted PhIP-G to PhIP-M1.',31096909),(3820,'human','null','2-Amino-1-methyl-6-phenylimidazo[4,5-b]pyridine','null','B-GUS of Faecalibacterium prausnitzii converted PhIP-G to PhIP and GDH of Flavonifractor plautii, Blautia obeum, Eubacterium hallii, and Lactobacillus reuteri converted PhIP to PhIP-M1 in the presence of glycerol. In addition, B-GUS- and GDH-positive bacteria cooperatively converted PhIP-G to PhIP-M1.',31096909),(3821,'human','null','2-Amino-1-methyl-6-phenylimidazo[4,5-b]pyridine','null','B-GUS of Faecalibacterium prausnitzii converted PhIP-G to PhIP and GDH of Flavonifractor plautii, Blautia obeum, Eubacterium hallii, and Lactobacillus reuteri converted PhIP to PhIP-M1 in the presence of glycerol. In addition, B-GUS- and GDH-positive bacteria cooperatively converted PhIP-G to PhIP-M1.',31096909),(3822,'human','null','2-Amino-1-methyl-6-phenylimidazo[4,5-b]pyridine','null','B-GUS of Faecalibacterium prausnitzii converted PhIP-G to PhIP and GDH of Flavonifractor plautii, Blautia obeum, Eubacterium hallii, and Lactobacillus reuteri converted PhIP to PhIP-M1 in the presence of glycerol. In addition, B-GUS- and GDH-positive bacteria cooperatively converted PhIP-G to PhIP-M1.',31096909),(3823,'human','null','Rutin','null','Enterococcus avium EFEL009 is capable of converting rutin to quercetin.',26505733),(3832,'human','decrease','null','low-throughput','In the coculture model, treatment with Enterococcus durans and with butyrate reduced basal as well as E coli stimulated secretion of IL-8, IL-6, and  TNF and increased secretion of IL-10.',21178864),(3833,'human','decrease','null','low-throughput','In the coculture model, treatment with Enterococcus durans and with butyrate reduced basal as well as E coli stimulated secretion of IL-8, IL-6, and  TNF and increased secretion of IL-10.',21178864),(3834,'human','decrease','null','low-throughput','In the coculture model, treatment with Enterococcus durans and with butyrate reduced basal as well as E coli stimulated secretion of IL-8, IL-6, and  TNF and increased secretion of IL-10.',21178864),(3835,'human','increase','null','low-throughput','In the coculture model, treatment with Enterococcus durans and with butyrate reduced basal as well as E coli stimulated secretion of IL-8, IL-6, and  TNF and increased secretion of IL-10.',21178864),(3836,'human','decrease','null','low-throughput','In the coculture model, treatment with Enterococcus durans and with butyrate reduced basal as well as E coli stimulated secretion of IL-8, IL-6, and  TNF and increased secretion of IL-10.',21178864),(3838,'human','null','null','null','Lactobacillus casei strain Shirota intake increased defecation frequency, normalized stool consistency, and improved constipation related symptoms in Chinese adults with constipation. Through the use of metabolomic approaches, PIPA, which is a non-volatile metabolite in the gut, was identified as a potential target mediator in LcS-mediated constipation alleviation. Furthermore, PIPA could improve constipation symptoms, as shown by increases in the small intestine propulsive rate and the number of fecal pellets in a constipation mouse model, raising its possibility in the constipation therapy. Future studies should verify the anti-constipation function of PIPA and elucidate its potential mechanisms.',30846982),(3841,'human','decrease','Geniposide','low-throughput','Western blot analysis revealed that the activated GS inhibited Bcl-2 expression with a subsequent increase in Bax expression. Likewise, GS activation by IM stimulated caspase-3 and the production of reactive oxygen species (ROS). In addition, activated GS-induced apoptosis was confirmed by apoptosis and ROS assays; N-acetyl-l-cysteine (NAC) suppressed ROS production and apoptotic cell death. Activated GS induced sustained JNK phosphorylation. Moreover, activated GS-induced cell death was reversed by SP600125. Taken together, these findings suggest that human IM is able to metabolize GS into GP, and the related biological activities induce apoptosis through ROS/JNK signaling.',22245672),(3842,'human','increase','Geniposide','low-throughput','Western blot analysis revealed that the activated GS inhibited Bcl-2 expression with a subsequent increase in Bax expression. Likewise, GS activation by IM stimulated caspase-3 and the production of reactive oxygen species (ROS). In addition, activated GS-induced apoptosis was confirmed by apoptosis and ROS assays; N-acetyl-l-cysteine (NAC) suppressed ROS production and apoptotic cell death. Activated GS induced sustained JNK phosphorylation. Moreover, activated GS-induced cell death was reversed by SP600125. Taken together, these findings suggest that human IM is able to metabolize GS into GP, and the related biological activities induce apoptosis through ROS/JNK signaling.',22245672),(3843,'human','increase','Geniposide','low-throughput','Western blot analysis revealed that the activated GS inhibited Bcl-2 expression with a subsequent increase in Bax expression. Likewise, GS activation by IM stimulated caspase-3 and the production of reactive oxygen species (ROS). In addition, activated GS-induced apoptosis was confirmed by apoptosis and ROS assays; N-acetyl-l-cysteine (NAC) suppressed ROS production and apoptotic cell death. Activated GS induced sustained JNK phosphorylation. Moreover, activated GS-induced cell death was reversed by SP600125. Taken together, these findings suggest that human IM is able to metabolize GS into GP, and the related biological activities induce apoptosis through ROS/JNK signaling.',22245672),(3844,'human','increase','Geniposide','low-throughput','Western blot analysis revealed that the activated GS inhibited Bcl-2 expression with a subsequent increase in Bax expression. Likewise, GS activation by IM stimulated caspase-3 and the production of reactive oxygen species (ROS). In addition, activated GS-induced apoptosis was confirmed by apoptosis and ROS assays; N-acetyl-l-cysteine (NAC) suppressed ROS production and apoptotic cell death. Activated GS induced sustained JNK phosphorylation. Moreover, activated GS-induced cell death was reversed by SP600125. Taken together, these findings suggest that human IM is able to metabolize GS into GP, and the related biological activities induce apoptosis through ROS/JNK signaling.',22245672),(3845,'human','decrease','null','low-throughput','We examined the effects of UV-killed F. prausnitzii, bacterial fractions,supernatants, or sterile medium on IL-8 secretion in Caco-2 cells,with or without IL-1 stimulation (SI Materials and Methods and Fig. 3A). Either UV-killed F. prausnitzii, membrane extracts, or cytoplasmic extracts, but not DNA or sterile medium, induce low levels of IL-8 secretion. UV-killed F. prausnitzii, bacterial fractions,and sterile medium did not inhibit IL-8 secretion induced by IL-1,compared with negative control. In contrast, supernatant from F.prausnitzii cultures significantly reduces IL-8 secretion induced by IL-1.',18936492),(3846,'human','decrease','null','low-throughput','We examined the effects of UV-killed F. prausnitzii, bacterial fractions,supernatants, or sterile medium on IL-8 secretion in Caco-2 cells,with or without IL-1 stimulation (SI Materials and Methods and Fig. 3A). Either UV-killed F. prausnitzii, membrane extracts, or cytoplasmic extracts, but not DNA or sterile medium, induce low levels of IL-8 secretion. UV-killed F. prausnitzii, bacterial fractions,and sterile medium did not inhibit IL-8 secretion induced by IL-1,compared with negative control. In contrast, supernatant from F.prausnitzii cultures significantly reduces IL-8 secretion induced by IL-1.',18936492),(3850,'human','increase','3,8,9-Trihydroxy-6H-benzo[c]chromen-6-one','low-throughput','The expression of p53 was significantly increased in the HCT- 116 cells (wild type p53) by Uro-A and UM-A treatments.',32179165),(3851,'human','increase','3,8,9-Trihydroxy-6H-benzo[c]chromen-6-one','low-throughput','A significant induction of the of p21Cip1/Waf1 in the cells treated with Uro-A (2.2 ± 0.2) and UM-A (2.2 ± 0.5) was observed.',32179165),(3852,'human','increase','null','low-throughput','Both gain- and loss-of-function studies employing genetic and pharmacological tools reveal PAGln mediates cellular events through G-protein coupled receptors, including α2A, α2B, and β2-adrenergic receptors.',32142679),(3853,'human','increase','null','low-throughput','Both gain- and loss-of-function studies employing genetic and pharmacological tools reveal PAGln mediates cellular events through G-protein coupled receptors, including α2A, α2B, and β2-adrenergic receptors.',32142679),(3854,'human','increase','null','low-throughput','Both gain- and loss-of-function studies employing genetic and pharmacological tools reveal PAGln mediates cellular events through G-protein coupled receptors, including α2A, α2B, and β2-adrenergic receptors.',32142679),(3855,'human','decrease','Ellagic acid','low-throughput','As a result, a 24 h treatment with UA (20 µm) significantly decreased Rac1 and PAK1 transcript levels and activity, depolymerized actin and wound healing.',31976617),(3856,'human','decrease','Ellagic acid','low-throughput','As a result, a 24 h treatment with UA (20 µm) significantly decreased Rac1 and PAK1 transcript levels and activity, depolymerized actin and wound healing.',31976617),(3857,'human','null','(-)-Epicatechin','null','All the isoflavone-metabolizing bacteria catalyzed C-ring cleavage of –EC to produce (2S)-1-(3, 4-dihydroxyphenyl)-3-(2, 4, 6-trihydroxyphenyl)propan-2-ol (1S), and the formation was accelerated in the presence of hydrogen.',26312950),(3858,'human','null','(-)-Epicatechin','null','All the isoflavone-metabolizing bacteria catalyzed C-ring cleavage of –EC to produce (2S)-1-(3, 4-dihydroxyphenyl)-3-(2, 4, 6-trihydroxyphenyl)propan-2-ol (1S), and the formation was accelerated in the presence of hydrogen.',26312950),(3859,'human','null','(-)-Epicatechin','null','All the isoflavone-metabolizing bacteria catalyzed C-ring cleavage of –EC to produce (2S)-1-(3, 4-dihydroxyphenyl)-3-(2, 4, 6-trihydroxyphenyl)propan-2-ol (1S), and the formation was accelerated in the presence of hydrogen.',26312950),(3860,'human','null','(-)-Epicatechin','null','All the isoflavone-metabolizing bacteria catalyzed C-ring cleavage of –EC to produce (2S)-1-(3, 4-dihydroxyphenyl)-3-(2, 4, 6-trihydroxyphenyl)propan-2-ol (1S), and the formation was accelerated in the presence of hydrogen.',26312950),(3861,'human','null','Cianidanol','null','The bacterial strains also showed ring-cleaving ability on +C to yield (2R)-1-(3, 4-dihydroxyphenyl)-3-(2, 4, 6-trihydroxyphenyl)propan-2-ol (1R), and this reaction was enhanced by the presence of hydrogen, with the exception of S. equolifaciens JCM 16059.',26312950),(3862,'human','null','Cianidanol','null','The bacterial strains also showed ring-cleaving ability on +C to yield (2R)-1-(3, 4-dihydroxyphenyl)-3-(2, 4, 6-trihydroxyphenyl)propan-2-ol (1R), and this reaction was enhanced by the presence of hydrogen, with the exception of S. equolifaciens JCM 16059.',26312950),(3863,'human','null','Cianidanol','null','The bacterial strains also showed ring-cleaving ability on +C to yield (2R)-1-(3, 4-dihydroxyphenyl)-3-(2, 4, 6-trihydroxyphenyl)propan-2-ol (1R), and this reaction was enhanced by the presence of hydrogen, with the exception of S. equolifaciens JCM 16059.',26312950),(3864,'human','null','Daidzein','null','Equol producing-related species Adlercreutzia equolifaciens and Bifdobacterium bifdum, showed higher relative abundance in EP than in NP.',31168326),(3865,'human','null','Daidzein','null','Equol producing-related species Adlercreutzia equolifaciens and Bifdobacterium bifdum, showed higher relative abundance in EP than in NP.',31168326),(3866,'human','null','Ginsenoside Rb1','null','The enzyme potently biotransformed ginsenoside Rb1, loganin, arctiin and arbutin to ginsenoside Rd, loganetin, arctigenin and hydroquinone, respectively, but was not active in the case of hesperidin, and kakkalide.',22466563),(3867,'human','null','Loganin','null','The enzyme potently biotransformed ginsenoside Rb1, loganin, arctiin and arbutin to ginsenoside Rd, loganetin, arctigenin and hydroquinone, respectively, but was not active in the case of hesperidin, and kakkalide.',22466563),(3868,'human','null','Arctiin','null','The enzyme potently biotransformed ginsenoside Rb1, loganin, arctiin and arbutin to ginsenoside Rd, loganetin, arctigenin and hydroquinone, respectively, but was not active in the case of hesperidin, and kakkalide.',22466563),(3869,'human','null','Arbutin','null','The enzyme potently biotransformed ginsenoside Rb1, loganin, arctiin and arbutin to ginsenoside Rd, loganetin, arctigenin and hydroquinone, respectively, but was not active in the case of hesperidin, and kakkalide.',22466563),(3870,'human','increase','null','low-throughput','Compared with the control, the stimulation of HCT116 cells with C. butyricum supernatant and sodium butyrate induced 4.5-fold and 4.2-fold upregulation of the GPR43 mRNA levels, 2.6-fold and 7.0-fold increases in the GPR109A mRNA levels.',31734354),(3871,'human','increase','null','low-throughput','Compared with the control, the stimulation of HCT116 cells with C. butyricum supernatant and sodium butyrate induced 4.5-fold and 4.2-fold upregulation of the GPR43 mRNA levels, 2.6-fold and 7.0-fold increases in the GPR109A mRNA levels.',31734354),(3872,'human','increase','null','low-throughput','The mRNA level of P21WAF1 was increased in HCT116 cells treated with C. butyricum.',31734354),(3873,'human','increase','null','low-throughput','Indole promoted mRNA expression of TJ-associated molecules, such as Cldn7, Ocln, and Tjp1 in the human adenocarcinoma cell line, Caco-2.',24278294),(3874,'human','increase','null','low-throughput','Indole promoted mRNA expression of TJ-associated molecules, such as Cldn7, Ocln, and Tjp1 in the human adenocarcinoma cell line, Caco-2.',24278294),(3875,'human','increase','null','low-throughput','Indole promoted mRNA expression of TJ-associated molecules, such as Cldn7, Ocln, and Tjp1 in the human adenocarcinoma cell line, Caco-2.',24278294),(3876,'human','increase','null','low-throughput','The function of butyrate as a GPR109A agonist was investigated using a heterologous expression system in which the coupling of pertussis toxin-sensitive Gi proteins to the receptor was monitored with the activity of inwardly-rectifying potassium (GIRK) channels as the read-out.',19276343),(3877,'human','null','null','null','The concentration of ursodesoxycholic acid in stool samples was highly positively correlated with the level of Ruminococcus.',28587349),(3878,'human','null','null','null','The concentrations of phenylalanine and glutamic acid were highly positively correlated with the levels of Phascolarctobacterium and Acidiminobacter.',28587349),(3879,'human','null','null','null','The concentrations of phenylalanine and glutamic acid were highly positively correlated with the levels of Phascolarctobacterium and Acidiminobacter.',28587349),(3880,'human','null','null','null','The concentrations of serine and threonine were moderately positively correlated with the levels of Phascolarctobacterium and Acidiminobacter.',28587349),(3881,'human','null','null','null','The concentrations of serine and threonine were moderately positively correlated with the levels of Phascolarctobacterium and Acidiminobacter.',28587349),(3882,'human','increase','Tryptophan','low-throughput','Exposure to 1 mM indole led to an increase in the expression of two Toll-like receptor (TLR) genes—TLR-3 and TLR-9.',19966295),(3883,'human','increase','Tryptophan','low-throughput','Exposure to 1 mM indole led to an increase in the expression of two Toll-like receptor (TLR) genes—TLR-3 and TLR-9.',19966295),(3884,'human','increase','Tryptophan','low-throughput','Additionally, genes involved in the modulation of TLR signaling—TOLLIP, SIGIRR, and SOCS5—were also induced by indole.',19966295),(3885,'human','increase','Tryptophan','low-throughput','Additionally, genes involved in the modulation of TLR signaling—TOLLIP, SIGIRR, and SOCS5—were also induced by indole.',19966295),(3886,'human','increase','Tryptophan','low-throughput','Additionally, genes involved in the modulation of TLR signaling—TOLLIP, SIGIRR, and SOCS5—were also induced by indole.',19966295),(3887,'human','increase','Tryptophan','low-throughput','Similarly, genes involved in the phosphoinositide-3-kinase (PI3K) pathway, which modulates TLR signaling and leads to antiinflammatory cytokine IL-10 induction, were also induced by indole.',19966295),(3888,'human','increase','Tryptophan','low-throughput','Three genes involved in PI3K signaling (PIK3CA, PIK3CB, and PIK3CD) were induced, as was the transcription factor AKT1 that is activated by signaling through the PI3K pathway.',19966295),(3889,'human','increase','Tryptophan','low-throughput','Three genes involved in PI3K signaling (PIK3CA, PIK3CB, and PIK3CD) were induced, as was the transcription factor AKT1 that is activated by signaling through the PI3K pathway.',19966295),(3890,'human','increase','Tryptophan','low-throughput','Three genes involved in PI3K signaling (PIK3CA, PIK3CB, and PIK3CD) were induced, as was the transcription factor AKT1 that is activated by signaling through the PI3K pathway.',19966295),(3891,'human','increase','Tryptophan','low-throughput','Downstream target genes of AKT1, including the cAMP response element-binding protein and IL-10, were also induced upon addition of indole.',19966295),(3892,'human','increase','Tryptophan','low-throughput','In addition, the receptor for the proinflammatory cytokine IL-2, IL-4, IL-18 binding protein, IL-24, and IL-28 were all induced, suggesting that indole exposure leads to an increase in cytokine expression and signaling in HCT-8 intestinal epithelial cells.',19966295),(3893,'human','increase','Tryptophan','low-throughput','In addition, the receptor for the proinflammatory cytokine IL-2, IL-4, IL-18 binding protein, IL-24, and IL-28 were all induced, suggesting that indole exposure leads to an increase in cytokine expression and signaling in HCT-8 intestinal epithelial cells.',19966295),(3894,'human','increase','Tryptophan','low-throughput','In addition, the receptor for the proinflammatory cytokine IL-2, IL-4, IL-18 binding protein, IL-24, and IL-28 were all induced, suggesting that indole exposure leads to an increase in cytokine expression and signaling in HCT-8 intestinal epithelial cells.',19966295),(3895,'human','increase','Tryptophan','low-throughput','In addition, the receptor for the proinflammatory cytokine IL-2, IL-4, IL-18 binding protein, IL-24, and IL-28 were all induced, suggesting that indole exposure leads to an increase in cytokine expression and signaling in HCT-8 intestinal epithelial cells.',19966295),(3896,'human','increase','Tryptophan','low-throughput','In addition, the receptor for the proinflammatory cytokine IL-2, IL-4, IL-18 binding protein, IL-24, and IL-28 were all induced, suggesting that indole exposure leads to an increase in cytokine expression and signaling in HCT-8 intestinal epithelial cells.',19966295),(3897,'human','increase','Tryptophan','low-throughput','In addition to altering the expression of cytokines, indole also induced the expression of several chemokines (CCL25, CCL4, CCL16, CXCL3, XCL1, and CX3CL1) in HCT-8 cells.',19966295),(3898,'human','increase','Tryptophan','low-throughput','In addition to altering the expression of cytokines, indole also induced the expression of several chemokines (CCL25, CCL4, CCL16, CXCL3, XCL1, and CX3CL1) in HCT-8 cells.',19966295),(3899,'human','increase','Tryptophan','low-throughput','In addition to altering the expression of cytokines, indole also induced the expression of several chemokines (CCL25, CCL4, CCL16, CXCL3, XCL1, and CX3CL1) in HCT-8 cells.',19966295),(3900,'human','increase','Tryptophan','low-throughput','In addition to altering the expression of cytokines, indole also induced the expression of several chemokines (CCL25, CCL4, CCL16, CXCL3, XCL1, and CX3CL1) in HCT-8 cells.',19966295),(3901,'human','increase','Tryptophan','low-throughput','In addition to altering the expression of cytokines, indole also induced the expression of several chemokines (CCL25, CCL4, CCL16, CXCL3, XCL1, and CX3CL1) in HCT-8 cells.',19966295),(3902,'human','increase','Tryptophan','low-throughput','In addition to altering the expression of cytokines, indole also induced the expression of several chemokines (CCL25, CCL4, CCL16, CXCL3, XCL1, and CX3CL1) in HCT-8 cells.',19966295),(3903,'human','increase','Tryptophan','low-throughput','Specifically, the expression of dual-specificity phosphatases (DUSPs)—Dusp1, Dusp3, Dusp10, Dusp16, Dusp18, and Dusp22, which function as inhibitors of MAPK signaling —was increased upon indole exposure.',19966295),(3904,'human','increase','Tryptophan','low-throughput','Specifically, the expression of dual-specificity phosphatases (DUSPs)—Dusp1, Dusp3, Dusp10, Dusp16, Dusp18, and Dusp22, which function as inhibitors of MAPK signaling —was increased upon indole exposure.',19966295),(3905,'human','increase','Tryptophan','low-throughput','Specifically, the expression of dual-specificity phosphatases (DUSPs)—Dusp1, Dusp3, Dusp10, Dusp16, Dusp18, and Dusp22, which function as inhibitors of MAPK signaling —was increased upon indole exposure.',19966295),(3906,'human','increase','Tryptophan','low-throughput','Specifically, the expression of dual-specificity phosphatases (DUSPs)—Dusp1, Dusp3, Dusp10, Dusp16, Dusp18, and Dusp22, which function as inhibitors of MAPK signaling —was increased upon indole exposure.',19966295),(3907,'human','increase','Tryptophan','low-throughput','Specifically, the expression of dual-specificity phosphatases (DUSPs)—Dusp1, Dusp3, Dusp10, Dusp16, Dusp18, and Dusp22, which function as inhibitors of MAPK signaling —was increased upon indole exposure.',19966295),(3908,'human','null','Daidzin','null','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3909,'human','null','Daidzin','null','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3910,'human','null','Daidzin','null','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3911,'human','null','Daidzin','null','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3912,'human','null','Daidzin','null','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3913,'human','null','Genistin','null','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3914,'human','null','Genistin','null','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3915,'human','null','Genistin','null','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3916,'human','null','Genistin','null','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3917,'human','null','Genistin','null','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3918,'human','null','Glycitin','null','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3919,'human','null','Glycitin','null','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3920,'human','null','Glycitin','null','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3921,'human','null','Glycitin','null','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3922,'human','null','Glycitin','null','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3923,'human','null','Astragalin','null','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3924,'human','null','Astragalin','null','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3925,'human','null','Astragalin','null','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3926,'human','null','Astragalin','null','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3927,'human','null','Astragalin','null','All selected strains were found to be active in the conversion of kaempferol 3-O-glucoside, daidzin, genistin, and glycitin into their aglyconic forms.',17439230),(3928,'human','null','gamma-Butyrobetaine hydrochloride','null','Only 1 Clostridiales bacterium, Emergencia timonensis, that converted γBB to TMA.',30530985),(3929,'human','null','Phenylalanine','null','After incubation of wild-type C. sporogenes with phenylalanine, tyrosine and tryptophan, we observed robust production of PPA, 4-OH-PPA and IPA.',29168502),(3930,'human','null','Tyrosine','null','After incubation of wild-type C. sporogenes with phenylalanine, tyrosine and tryptophan, we observed robust production of PPA, 4-OH-PPA and IPA.',29168502),(3931,'human','null','Tryptophan','null','After incubation of wild-type C. sporogenes with phenylalanine, tyrosine and tryptophan, we observed robust production of PPA, 4-OH-PPA and IPA.',29168502),(3932,'human','null','Tryptophan','null','Notably, we identified four new IPA-producing gut bacteria: Peptostreptococcus anaerobius CC14N and three strains of Clostridium cadaveris.',29168502),(3933,'human','null','Tryptophan','null','Notably, we identified four new IPA-producing gut bacteria: Peptostreptococcus anaerobius CC14N and three strains of Clostridium cadaveris.',29168502),(3934,'human','null','Tryptophan','null','Notably, we identified four new IPA-producing gut bacteria: Peptostreptococcus anaerobius CC14N and three strains of Clostridium cadaveris.',29168502),(3935,'human','null','Tryptophan','null','Notably, we identified four new IPA-producing gut bacteria: Peptostreptococcus anaerobius CC14N and three strains of Clostridium cadaveris.',29168502),(3936,'human','decrease','Urocanic acid','low-throughput','We also noted that IRS1 levels were reduced by 8 hr treatment of imidazole propionate in the human embryonic kidney cell line HEK293.',30401435),(3937,'human','increase','Urocanic acid','low-throughput','Imidazole propionate activates mTORC1.',30401435),(3938,'human','decrease','Urocanic acid','low-throughput','We also noted that IRS1 levels were reduced by 8 hr treatment of imidazole propionate in the human embryonic kidney cell line HEK293.',30401435),(3939,'human','increase','Urocanic acid','low-throughput','Imidazole propionate activates mTORC1.',30401435),(3940,'human','increase','null','low-throughput','Propionate was the most potent agonist for both GPR41 and GPR43.',12711604),(3941,'human','increase','null','low-throughput','Propionate was the most potent agonist for both GPR41 and GPR43.',12711604),(3942,'human','increase','null','low-throughput','Acetate was more selective for GPR43, whereas butyrate and isobutyrate were more active on GPR41',12711604),(3943,'human','increase','null','low-throughput','Acetate was more selective for GPR43, whereas butyrate and isobutyrate were more active on GPR41',12711604),(3949,'human','null','Ginsenoside Rg3','null','Among the bacteria isolated from human fecal microflora, Bacteroides sp., Eubacterium sp., and Bifidobacterium sp. metabolized ginsenoside Rg3 to protopanaxadiol via ginsenoside Rh2.',11824558),(3950,'human','null','Ginsenoside Rg3','null','Among the bacteria isolated from human fecal microflora, Bacteroides sp., Eubacterium sp., and Bifidobacterium sp. metabolized ginsenoside Rg3 to protopanaxadiol via ginsenoside Rh2.',11824558),(3951,'human','null','Ginsenoside Rg3','null','Among the bacteria isolated from human fecal microflora, Bacteroides sp., Eubacterium sp., and Bifidobacterium sp. metabolized ginsenoside Rg3 to protopanaxadiol via ginsenoside Rh2.',11824558),(3952,'human','null','Ginsenoside Rh2','null','Among the bacteria isolated from human fecal microflora, Bacteroides sp., Eubacterium sp., and Bifidobacterium sp. metabolized ginsenoside Rg3 to protopanaxadiol via ginsenoside Rh2.',11824558),(3953,'human','null','Ginsenoside Rh2','null','Among the bacteria isolated from human fecal microflora, Bacteroides sp., Eubacterium sp., and Bifidobacterium sp. metabolized ginsenoside Rg3 to protopanaxadiol via ginsenoside Rh2.',11824558),(3954,'human','null','(20R)-Ginsenoside Rg3','null','Among the bacteria isolated from human fecal microflora, Bacteroides sp., Eubacterium sp., and Bifidobacterium sp. metabolized ginsenoside Rg3 to protopanaxadiol via ginsenoside Rh2.',11824558),(3955,'human','null','(20R)-Ginsenoside Rg3','null','Among the bacteria isolated from human fecal microflora, Bacteroides sp., Eubacterium sp., and Bifidobacterium sp. metabolized ginsenoside Rg3 to protopanaxadiol via ginsenoside Rh2.',11824558),(3956,'human','null','20(R)-Ginsenoside Rh2','null','Among the bacteria isolated from human fecal microflora, Bacteroides sp., Eubacterium sp., and Bifidobacterium sp. metabolized ginsenoside Rg3 to protopanaxadiol via ginsenoside Rh2.',11824558),(3957,'human','null','Arctigenin','null','In the present study, we isolated one single bacterium, which we named Blautia sp. AUH-JLD56, capable of solely biotransforming arctiin or arctigenin to (-)-3\'-DMAG.',24236649),(3958,'human','null','null','null','Ruminococcus, showed positive correlations with plasma glycerophospholipids.',31748797),(3959,'human','increase','null','high-throughput','The RNA-Seq analysis showed that cytochrome P450 1A1 (Cyp1A1) is among the top 3 UroA upregulated genes.',30626868),(3960,'human','increase','null','high-throughput','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3961,'human','increase','null','high-throughput','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3962,'human','increase','null','high-throughput','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3963,'human','increase','null','high-throughput','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3964,'human','increase','null','high-throughput','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3965,'human','increase','null','high-throughput','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3966,'human','increase','null','high-throughput','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3967,'human','increase','null','high-throughput','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3968,'human','increase','null','high-throughput','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3969,'human','increase','null','high-throughput','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3970,'human','increase','null','high-throughput','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3971,'human','increase','null','high-throughput','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3972,'human','increase','null','high-throughput','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3973,'human','increase','null','high-throughput','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3974,'human','increase','null','high-throughput','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3975,'human','increase','null','high-throughput','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3976,'human','increase','null','high-throughput','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3977,'human','increase','null','high-throughput','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3978,'human','increase','null','high-throughput','The RNA-Seq analysis showed that UroA upregulated genes.',30626868),(3979,'human','decrease','null','high-throughput','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3980,'human','decrease','null','high-throughput','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3981,'human','decrease','null','high-throughput','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3982,'human','decrease','null','high-throughput','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3983,'human','decrease','null','high-throughput','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3984,'human','decrease','null','high-throughput','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3985,'human','decrease','null','high-throughput','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3986,'human','decrease','null','high-throughput','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3987,'human','decrease','null','high-throughput','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3988,'human','decrease','null','high-throughput','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3989,'human','decrease','null','high-throughput','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3990,'human','decrease','null','high-throughput','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3991,'human','decrease','null','high-throughput','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3992,'human','decrease','null','high-throughput','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3993,'human','decrease','null','high-throughput','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3994,'human','increase','null','high-throughput','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3995,'human','increase','null','high-throughput','The RNA-Seq analysis showed that UroA downregulated genes.',30626868),(3996,'human','increase','null','high-throughput','The fold changes in mRNA levels of claudin 4 (Cldn4), occludin (Ocln), and Zona occludens 1 (ZO1) in HT29 cells were determined by RT PCR method.',30626868),(3997,'human','increase','null','high-throughput','The fold changes in mRNA levels of claudin 4 (Cldn4), occludin (Ocln), and Zona occludens 1 (ZO1) in HT29 cells were determined by RT PCR method.',30626868),(3998,'human','increase','null','high-throughput','The fold changes in mRNA levels of claudin 4 (Cldn4), occludin (Ocln), and Zona occludens 1 (ZO1) in HT29 cells were determined by RT PCR method.',30626868),(3999,'human','increase','null','high-throughput','UroA/UAS03 induced NQO1 expression was also confirmed in HT29 cells (Supplementary Fig. 9d).',30626868),(4000,'human','null','L-Fucose','null','R. inulinivorans A2-194 is also able to grow on the host-derived sugar fucose, following a lag period, producing propionate and propanol as additional fermentation products.',16740940),(4001,'human','null','L-Fucose','null','R. inulinivorans A2-194 is also able to grow on the host-derived sugar fucose, following a lag period, producing propionate and propanol as additional fermentation products.',16740940),(4002,'human','null','L-Fucose','null','R. inulinivorans A2-194 is also able to grow on the host-derived sugar fucose, following a lag period, producing propionate and propanol as additional fermentation products.',16740940),(4003,'human','null','null','null','Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.',29572891),(4004,'human','null','null','null','Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.',29572891),(4005,'human','null','null','null','Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.',29572891),(4006,'human','null','null','null','Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.',29572891),(4007,'human','null','null','null','Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.',29572891),(4008,'human','null','null','null','Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.',29572891),(4009,'human','null','null','null','Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.',29572891),(4010,'human','null','null','null','Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.',29572891),(4011,'human','null','null','null','Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.',29572891),(4012,'human','null','null','null','Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.',29572891),(4013,'human','null','null','null','Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.',29572891),(4014,'human','null','null','null','Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.',29572891),(4015,'human','null','null','null','Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.',29572891),(4016,'human','null','null','null','Interestingly, among the bacterial species significantly correlated with phenyllactate and 3-(4-hydroxyphenyl)lactacte, 7 have been demonstrated to be associated with advanced fibrosis in NAFLD in a previously published study: Bacteroides caccae, Clostridium sp, Escherichia Coli, Firmicutes bacterium CAG110, Firmicutes bacterium CAG129, Firmicutes bacterium CAG176, Subdoligranulum sp.',29572891),(4023,'human','null','D-Glucose','null','The major end products of glucose metabolism are lactic, acetic and formic acids.',20023061),(4024,'human','null','D-Glucose','null','The major end products of glucose metabolism are lactic, acetic and formic acids.',20023061),(4025,'human','null','D-Glucose','null','The major end products of glucose metabolism are lactic, acetic and formic acids.',20023061),(4026,'human','null','D-Glucose','null','The major end products of glucose metabolism are lactic, acetic and formic acids.',20023061),(4027,'human','null','D-Glucose','null','The major end products of glucose metabolism were succinate and acetate.',19749029),(4028,'human','null','D-Glucose','null','The major end products of glucose metabolism were succinate and acetate.',19749029),(4029,'human','null','Avicularin','null','Avicularin (M1) was anaerobic metabolized to its aglycone quercetin (M2) and methoxylated avicularin (M3, M4), then quercetin was converted to quercetin glycosides: quercetin-3-O-rhamnoside (M5), quercetin-3-O-glucoside (M6) and quercetin-7-O-glucoside (M7) by Bacillus sp. 46.',24463398),(4030,'human','null','Quercetin','null','Avicularin (M1) was anaerobic metabolized to its aglycone quercetin (M2) and methoxylated avicularin (M3, M4), then quercetin was converted to quercetin glycosides: quercetin-3-O-rhamnoside (M5), quercetin-3-O-glucoside (M6) and quercetin-7-O-glucoside (M7) by Bacillus sp. 46.',24463398),(4031,'human','null','Quercetin','null','Avicularin (M1) was anaerobic metabolized to its aglycone quercetin (M2) and methoxylated avicularin (M3, M4), then quercetin was converted to quercetin glycosides: quercetin-3-O-rhamnoside (M5), quercetin-3-O-glucoside (M6) and quercetin-7-O-glucoside (M7) by Bacillus sp. 46.',24463398),(4032,'human','null','Quercetin','null','Avicularin (M1) was anaerobic metabolized to its aglycone quercetin (M2) and methoxylated avicularin (M3, M4), then quercetin was converted to quercetin glycosides: quercetin-3-O-rhamnoside (M5), quercetin-3-O-glucoside (M6) and quercetin-7-O-glucoside (M7) by Bacillus sp. 46.',24463398),(4033,'human','null','Avicularin','null','Avicularin (M1) was anaerobic metabolized to its aglycone quercetin (M2) and methoxylated avicularin (M3, M4), then quercetin was converted to quercetin glycosides: quercetin-3-O-rhamnoside (M5), quercetin-3-O-glucoside (M6) and quercetin-7-O-glucoside (M7) by Bacillus sp. 46.',24463398),(4034,'human','increase','null','low-throughput','Sodium acetate (10�?0 mM) significantly increased TPH1mRNA expression.',25550456),(4035,'human','null','D-Glucose','null','C. scindens fermented glucose mainly to ethanol, acetate, formate, and H2.',30737348),(4036,'human','null','D-Glucose','null','C. scindens fermented glucose mainly to ethanol, acetate, formate, and H2.',30737348),(4037,'human','null','D-Glucose','null','C. scindens fermented glucose mainly to ethanol, acetate, formate, and H2.',30737348),(4038,'human','null','D-Glucose','null','C. scindens fermented glucose mainly to ethanol, acetate, formate, and H2.',30737348),(4039,'human','null','null','null','A trace amount of propionate was detected as an end product of metabolism.',19542131),(4040,'human','null','D-Glucose','null','The two strains produced significant levels of butyric and isobutyric acids as end products from glucose.',19542124),(4041,'human','null','D-Glucose','null','The two strains produced significant levels of butyric and isobutyric acids as end products from glucose.',19542124),(4042,'human','null','D-Glucose','null','The two strains produced significant levels of butyric and isobutyric acids as end products from glucose.',19542124),(4043,'human','null','D-Glucose','null','The two strains produced significant levels of butyric and isobutyric acids as end products from glucose.',19542124),(4044,'human','increase','null','low-throughput','Butyrate stimulated a rapid twofold increase of IL- 22R1 mRNA expression that, after a total 8 h incubation period, translated to the protein level.',27801948),(4045,'human','null','(-)-Epicatechin','null','Strain MT4s-5 readily converted −EC and −C into (2S)-1-(3,4-dihydroxyphenyl)-3-(2,4,6- trihydroxyphenyl) propan-2-ol (1S) and +C into (2R)-1-(3,4- dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol (1R).',25947926),(4046,'human','null','(-)-Catechin','null','Strain MT4s-5 readily converted −EC and −C into (2S)-1-(3,4-dihydroxyphenyl)-3-(2,4,6- trihydroxyphenyl) propan-2-ol (1S) and +C into (2R)-1-(3,4- dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol (1R).',25947926),(4047,'human','null','Cianidanol','null','Strain MT4s-5 readily converted −EC and −C into (2S)-1-(3,4-dihydroxyphenyl)-3-(2,4,6- trihydroxyphenyl) propan-2-ol (1S) and +C into (2R)-1-(3,4- dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol (1R).',25947926),(4048,'human','null','(-)-Epicatechin','null','Strain JCM 9979 easily converted −EC and +C into 1S and 1R, respectively.',25947926),(4049,'human','null','Cianidanol','null','Strain JCM 9979 easily converted −EC and +C into 1S and 1R, respectively.',25947926),(4050,'human','null','(2S)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','null','F. plautii MT42 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4051,'human','null','(2S)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','null','F. plautii MT42 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4052,'human','null','(2R)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','null','F. plautii MT42 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4053,'human','null','(2R)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','null','F. plautii MT42 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4054,'human','null','(2S)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','null','Similarly, strain MT42 could convert 2S and 2R to their corresponding 4-hydroxy-5-(3-hydroxyphenyl) valeric acids (5R from 2S, 5S from 2R) and 5-(3-hydroxyphenyl)-γ-valerolactones (6R from 2S, 6S from 2R).',25947926),(4055,'human','null','(2R)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','null','Similarly, strain MT42 could convert 2S and 2R to their corresponding 4-hydroxy-5-(3-hydroxyphenyl) valeric acids (5R from 2S, 5S from 2R) and 5-(3-hydroxyphenyl)-γ-valerolactones (6R from 2S, 6S from 2R).',25947926),(4056,'human','null','(2S)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','null','Similarly, strain MT42 could convert 2S and 2R to their corresponding 4-hydroxy-5-(3-hydroxyphenyl) valeric acids (5R from 2S, 5S from 2R) and 5-(3-hydroxyphenyl)-γ-valerolactones (6R from 2S, 6S from 2R).',25947926),(4057,'human','null','(2R)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','null','Similarly, strain MT42 could convert 2S and 2R to their corresponding 4-hydroxy-5-(3-hydroxyphenyl) valeric acids (5R from 2S, 5S from 2R) and 5-(3-hydroxyphenyl)-γ-valerolactones (6R from 2S, 6S from 2R).',25947926),(4058,'human','null','(2S)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','null','F. plautii ATCC 29863 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4059,'human','null','(2S)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','null','F. plautii ATCC 29863 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4060,'human','null','(2R)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','null','F. plautii ATCC 29863 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4061,'human','null','(2R)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','null','F. plautii ATCC 29863 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4062,'human','null','(2S)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','null','F. plautii ATCC 29863 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4063,'human','null','(2R)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','null','F. plautii ATCC 29863 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4064,'human','null','(2S)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','null','F. plautii ATCC 29863 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4065,'human','null','(2R)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','null','F. plautii ATCC 29863 was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4066,'human','null','(2S)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','null','F. plautii ATCC 49531  was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4067,'human','null','(2S)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','null','F. plautii ATCC 49531  was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4068,'human','null','(2R)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','null','F. plautii ATCC 49531  was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4069,'human','null','(2R)-1-(3,4-dihydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','null','F. plautii ATCC 49531  was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4070,'human','null','(2S)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','null','F. plautii ATCC 49531  was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4071,'human','null','(2R)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','null','F. plautii ATCC 49531  was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4072,'human','null','(2S)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','null','F. plautii ATCC 49531  was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4073,'human','null','(2R)-1-(3-hydroxyphenyl)-3-(2,4,6-trihydroxyphenyl) propan-2-ol','null','F. plautii ATCC 49531  was found to produce (4R)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3R) and (4R)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4R) from 1S and (4S)-4-hydroxy-5-(3,4-dihydroxyphenyl) valeric acid (3S) and (4S)-5-(3,4-dihydroxyphenyl)-γ-valerolactone (4S) from 1R.',25947926),(4075,'human','null','Daidzein','null','The bacterium cleaved the C-ring of daidzein to produce O-demethylangolensin ( O-Dma).',12070764),(4076,'human','null','Daidzein','null','The human intestinal bacterium Slackia isoflavoniconvertens converts the main soybean isoflavones daidzein and genistein to equol and 5-hydroxy-equol, respectively.',23542626),(4077,'human','null','Genistein','null','The human intestinal bacterium Slackia isoflavoniconvertens converts the main soybean isoflavones daidzein and genistein to equol and 5-hydroxy-equol, respectively.',23542626),(4083,'human','null','D-Glucose','null','The organism produced acetate, lactate, and ethanol as the major products of glucose fermentation.',16337765),(4084,'human','null','D-Glucose','null','The organism produced acetate, lactate, and ethanol as the major products of glucose fermentation.',16337765),(4085,'human','null','D-Glucose','null','The organism produced acetate, lactate, and ethanol as the major products of glucose fermentation.',16337765),(4091,'human','null','Myricitrin','null','Specifically, the majority of the strains such as Escherichia sp. 12 could metabolize myricitrin (M1) to quercetin-3-O-rhamnoside (M2) and myricetin (M3) (Figure 1C), and the metabolites of some strains like Enterococcus sp. 45 were large amounts of myricetin (M3) and quercetin (M4) (Figure 1D), while the other strains, for example, Escherichia sp. 33, could degrade myricitrin completely to myricetin and quercetin.',24882500),(4092,'human','null','Myricitrin','null','Specifically, the majority of the strains such as Escherichia sp. 12 could metabolize myricitrin (M1) to quercetin-3-O-rhamnoside (M2) and myricetin (M3) (Figure 1C), and the metabolites of some strains like Enterococcus sp. 45 were large amounts of myricetin (M3) and quercetin (M4) (Figure 1D), while the other strains, for example, Escherichia sp. 33, could degrade myricitrin completely to myricetin and quercetin.',24882500),(4093,'human','null','Myricitrin','null','Specifically, the majority of the strains such as Escherichia sp. 12 could metabolize myricitrin (M1) to quercetin-3-O-rhamnoside (M2) and myricetin (M3) (Figure 1C), and the metabolites of some strains like Enterococcus sp. 45 were large amounts of myricetin (M3) and quercetin (M4) (Figure 1D), while the other strains, for example, Escherichia sp. 33, could degrade myricitrin completely to myricetin and quercetin.',24882500),(4094,'human','null','Myricitrin','null','Specifically, the majority of the strains such as Escherichia sp. 12 could metabolize myricitrin (M1) to quercetin-3-O-rhamnoside (M2) and myricetin (M3) (Figure 1C), and the metabolites of some strains like Enterococcus sp. 45 were large amounts of myricetin (M3) and quercetin (M4) (Figure 1D), while the other strains, for example, Escherichia sp. 33, could degrade myricitrin completely to myricetin and quercetin.',24882500),(4095,'human','null','Myricitrin','null','Specifically, the majority of the strains such as Escherichia sp. 12 could metabolize myricitrin (M1) to quercetin-3-O-rhamnoside (M2) and myricetin (M3) (Figure 1C), and the metabolites of some strains like Enterococcus sp. 45 were large amounts of myricetin (M3) and quercetin (M4) (Figure 1D), while the other strains, for example, Escherichia sp. 33, could degrade myricitrin completely to myricetin and quercetin.',24882500),(4096,'human','null','Myricitrin','null','Specifically, the majority of the strains such as Escherichia sp. 12 could metabolize myricitrin (M1) to quercetin-3-O-rhamnoside (M2) and myricetin (M3) (Figure 1C), and the metabolites of some strains like Enterococcus sp. 45 were large amounts of myricetin (M3) and quercetin (M4) (Figure 1D), while the other strains, for example, Escherichia sp. 33, could degrade myricitrin completely to myricetin and quercetin.',24882500),(4099,'human','null','D-Glucose','null','Strain M25(T) was negative for both catalase and oxidase activity, utilized carbohydrates as fermentable substrates, produced lactate and acetate as the major end products of glucose fermentation in PYG medium.',22544782),(4100,'human','null','D-Glucose','null','Strain M25(T) was negative for both catalase and oxidase activity, utilized carbohydrates as fermentable substrates, produced lactate and acetate as the major end products of glucose fermentation in PYG medium.',22544782),(4101,'human','null','Succinate','null','However, strain YIT 11850(T) was able to decarboxylate succinate to propionate.',19060046),(4102,'human','null','D-Glucose','null','Cells of the other isolate, strain YIT 11860(T), were non-motile, rod-shaped, positive for aesculin hydrolysis, negative for indole production, produced succinic and acetic acids as end products of glucose metabolism and possessed a DNA G+C content of 45.5 mol%.',19060046),(4103,'human','null','D-Glucose','null','Cells of the other isolate, strain YIT 11860(T), were non-motile, rod-shaped, positive for aesculin hydrolysis, negative for indole production, produced succinic and acetic acids as end products of glucose metabolism and possessed a DNA G+C content of 45.5 mol%.',19060046),(4104,'human','null','Cynaroside','null','Three metabolites (acetylated luteoloside, luteolin and deoxygenated luteolin) were detected in the fractions isolated from the bacterial samples.',25899973),(4106,'human','null','D-(+)-Cellobiose','null','Acetate and succinate were the major end products of cellulose and cellobiose fermentation.',21357460),(4107,'human','null','D-(+)-Cellobiose','null','Acetate and succinate were the major end products of cellulose and cellobiose fermentation.',21357460),(4112,'human','null','null','null','Acetate and butyrate were produced by strain 3BBH22T as metabolic end-products.',29745868),(4113,'human','null','null','null','Acetate and butyrate were produced by strain 3BBH22T as metabolic end-products.',29745868),(4114,'human','null','Isoquercitrin','null','Fermentation of quercetin-3-glucoside by growing cultures of Eubacterium ramulus and Enterococcus casseliflavus.',12747214),(4115,'human','null','Isoquercitrin','null','Fermentation of quercetin-3-glucoside by growing cultures of Eubacterium ramulus and Enterococcus casseliflavus.',12747214),(4116,'human','null','null','null','We also observed both P1NP and CTX-1 to be positively associated with osteopenia-enriched Allisonella, Klebsiella and Megasphaera.',32392181),(4117,'human','null','null','null','We also observed both P1NP and CTX-1 to be positively associated with osteopenia-enriched Allisonella, Klebsiella and Megasphaera.',32392181),(4118,'human','null','null','null','We also observed both P1NP and CTX-1 to be positively associated with osteopenia-enriched Allisonella, Klebsiella and Megasphaera.',32392181),(4119,'human','null','null','null','We also observed both P1NP and CTX-1 to be positively associated with osteopenia-enriched Allisonella, Klebsiella and Megasphaera.',32392181),(4120,'human','null','null','null','We also observed both P1NP and CTX-1 to be positively associated with osteopenia-enriched Allisonella, Klebsiella and Megasphaera.',32392181),(4121,'human','null','null','null','We also observed both P1NP and CTX-1 to be positively associated with osteopenia-enriched Allisonella, Klebsiella and Megasphaera.',32392181),(4122,'human','null','Daidzein','null','In conclusion, the results obtained in this study suggest that the presence of Bacteroides genus in the fecal samples of control subjects seems to be involved in the daidzein transformation to equol, an antioxidative, anti-carcinogenic molecule, which shows the most active estrogenic properties of soy isoflavones.\n',32218321),(4123,'human','null','Daidzein','null','In conclusion, the results obtained in this study suggest that the presence of Bacteroides genus in the fecal samples of control subjects seems to be involved in the daidzein transformation to equol, an antioxidative, anti-carcinogenic molecule, which shows the most active estrogenic properties of soy isoflavones.\n',32218321),(4124,'human','null','Daidzein','null','In conclusion, the results obtained in this study suggest that the presence of Bacteroides genus in the fecal samples of control subjects seems to be involved in the daidzein transformation to equol, an antioxidative, anti-carcinogenic molecule, which shows the most active estrogenic properties of soy isoflavones.\n',32218321),(4125,'human','null','Daidzein','null','In conclusion, the results obtained in this study suggest that the presence of Bacteroides genus in the fecal samples of control subjects seems to be involved in the daidzein transformation to equol, an antioxidative, anti-carcinogenic molecule, which shows the most active estrogenic properties of soy isoflavones.\n',32218321),(4126,'human','null','Daidzein','null','In conclusion, the results obtained in this study suggest that the presence of Bacteroides genus in the fecal samples of control subjects seems to be involved in the daidzein transformation to equol, an antioxidative, anti-carcinogenic molecule, which shows the most active estrogenic properties of soy isoflavones.\n',32218321),(4127,'human','null','Daidzein','null','In conclusion, the results obtained in this study suggest that the presence of Bacteroides genus in the fecal samples of control subjects seems to be involved in the daidzein transformation to equol, an antioxidative, anti-carcinogenic molecule, which shows the most active estrogenic properties of soy isoflavones.\n',32218321),(4131,'human','null','Pyruvate','null','According to the FDR-adjusted p-values (q-values) in multivariable linear regression, p-cresyl sulfate showed a significant association with six major genera, with positive correlations observed with Alistipes, Oscillibacter, and Subdoligranulum (q < 0.001, q < 0.001, and q = 0.023, respectively) and negative correlations with Lachnospira, Veillonella, and Megamonas (q = 0.039, q = 0.014, and q = 0.034, respectively). Serum p-cresyl glucuronide and indoxyl sulfate levels were positively correlated with Alistipes (q = 0.010 and q = 0.035, respectively) and Oscillibacter (q = 0.001 and q = 0.037, respectively). Serum TMAO levels were associated only with Oscillibacter (q = 0.006). Unexpectedly, we found Oscillibacter to be associated with all four measured uremic toxins, although each of these metabolites is derived from different parent compounds.',32560104),(4132,'human','null','Pyruvate','null','According to the FDR-adjusted p-values (q-values) in multivariable linear regression, p-cresyl sulfate showed a significant association with six major genera, with positive correlations observed with Alistipes, Oscillibacter, and Subdoligranulum (q < 0.001, q < 0.001, and q = 0.023, respectively) and negative correlations with Lachnospira, Veillonella, and Megamonas (q = 0.039, q = 0.014, and q = 0.034, respectively). Serum p-cresyl glucuronide and indoxyl sulfate levels were positively correlated with Alistipes (q = 0.010 and q = 0.035, respectively) and Oscillibacter (q = 0.001 and q = 0.037, respectively). Serum TMAO levels were associated only with Oscillibacter (q = 0.006). Unexpectedly, we found Oscillibacter to be associated with all four measured uremic toxins, although each of these metabolites is derived from different parent compounds.',32560104),(4133,'human','null','Pyruvate','null','According to the FDR-adjusted p-values (q-values) in multivariable linear regression, p-cresyl sulfate showed a significant association with six major genera, with positive correlations observed with Alistipes, Oscillibacter, and Subdoligranulum (q < 0.001, q < 0.001, and q = 0.023, respectively) and negative correlations with Lachnospira, Veillonella, and Megamonas (q = 0.039, q = 0.014, and q = 0.034, respectively). Serum p-cresyl glucuronide and indoxyl sulfate levels were positively correlated with Alistipes (q = 0.010 and q = 0.035, respectively) and Oscillibacter (q = 0.001 and q = 0.037, respectively). Serum TMAO levels were associated only with Oscillibacter (q = 0.006). Unexpectedly, we found Oscillibacter to be associated with all four measured uremic toxins, although each of these metabolites is derived from different parent compounds.',32560104),(4134,'human','null','Pyruvate','null','According to the FDR-adjusted p-values (q-values) in multivariable linear regression, p-cresyl sulfate showed a significant association with six major genera, with positive correlations observed with Alistipes, Oscillibacter, and Subdoligranulum (q < 0.001, q < 0.001, and q = 0.023, respectively) and negative correlations with Lachnospira, Veillonella, and Megamonas (q = 0.039, q = 0.014, and q = 0.034, respectively). Serum p-cresyl glucuronide and indoxyl sulfate levels were positively correlated with Alistipes (q = 0.010 and q = 0.035, respectively) and Oscillibacter (q = 0.001 and q = 0.037, respectively). Serum TMAO levels were associated only with Oscillibacter (q = 0.006). Unexpectedly, we found Oscillibacter to be associated with all four measured uremic toxins, although each of these metabolites is derived from different parent compounds.',32560104),(4135,'human','increase','null','low-throughput','Here, we elucidate a micro biome-dependent pathway by which SG increases CA7S production. We show that a microbial metabolite,lithocholic acid (LCA), is increased in murine portal veins post-SG and by activating the vitamin D receptor, induces hepatic mSult2A1/hSULT2A expression to drive CA7S production. An SG-induced shift in the micro biome increases gut expression of the bile acid transporters Asbt and Osta, which in turn facilitate selective transport of LCA across the gut epithelium. Cecal microbiota transplant from SG animals is sufficient to recreate the pathway in germ-free (GF) animals. \nand GLP-1 secretion, causally connecting a microbial metabolite with the improvement of diabetic phe notypes.',33434516),(4136,'human','increase','null','low-throughput','Here, we elucidate a micro biome-dependent pathway by which SG increases CA7S production. We show that a microbial metabolite,lithocholic acid (LCA), is increased in murine portal veins post-SG and by activating the vitamin D receptor, induces hepatic mSult2A1/hSULT2A expression to drive CA7S production. An SG-induced shift in the micro biome increases gut expression of the bile acid transporters Asbt and Osta, which in turn facilitate selective transport of LCA across the gut epithelium. Cecal microbiota transplant from SG animals is sufficient to recreate the pathway in germ-free (GF) animals. \nand GLP-1 secretion, causally connecting a microbial metabolite with the improvement of diabetic phe notypes.',33434516),(4137,'human','increase','Trimethylamine','low-throughput','We found that TMAO treatment activated astrocytes, as indicated by a ~ 2-fold increase in LCN2 (Fig. 6a) and a ~ 3-fold increase in CD44 (Fig. 6b), a pro-inflammatory intercellular adhesion pro tein characteristic of reactive astrocytes [10].',32862276),(4138,'human','increase','Trimethylamine','low-throughput','We found that TMAO treatment activated astrocytes, as indicated by a ~ 2-fold increase in LCN2 (Fig. 6a) and a ~ 3-fold increase in CD44 (Fig. 6b), a pro-inflammatory intercellular adhesion pro tein characteristic of reactive astrocytes [10].',32862276),(4139,'human','decrease','Baicalin','low-throughput','As shown in Fig. 5a, treatment with 10�?0 μM of BE signifcantly reduced LPS-induced IL-8 secretion. Similar inhibitory efects were observed for BI at 15�?0 μM; however, BE showed more potent efects.',31650468),(4140,'human','decrease','null','low-throughput','As shown in Fig. 5a, treatment with 10�?0 μM of BE signifcantly reduced LPS-induced IL-8 secretion. Similar inhibitory efects were observed for BI at 15�?0 μM; however, BE showed more potent efects.',31650468),(4141,'human','increase','Trimethylamine','low-throughput','Plasma TMAO was positively associated with plasma levels of TNFα and IL-12 (Figures 3A,B), but not with IL-6 (p = 0.263, rho = 0.11) or IL-8.',33042155),(4142,'human','increase','Trimethylamine','low-throughput','Plasma TMAO was positively associated with plasma levels of TNFα and IL-12 (Figures 3A,B), but not with IL-6 (p = 0.263, rho = 0.11) or IL-8.',33042155),(4143,'human','increase','Trimethylamine','low-throughput','Herein, we found that plasma levels of LPS were significantly correlated with TMAOand the TMA precursors carnitine and y-BB in CVID patients (Figures 6A–C), indicating a relation between gut leakage and TMAO pathway metabolites.',33042155),(4144,'human','null','Secoisolariciresinol diglucoside','null','Over the first 30 min at 37 1C, the amount of SDG converted to SECO by resting cells of Clostridium sp. SDG Mt85-3Db was approximately 8- and 30-fold higher than the amount converted by B. distasonis DSM 20701T and B.ovatus SDG-Mt85-3Cy, respectively.',16765860),(4145,'human','null','Secoisolariciresinol diglucoside','null','Over the first 30 min at 37 1C, the amount of SDG converted to SECO by resting cells of Clostridium sp. SDG Mt85-3Db was approximately 8- and 30-fold higher than the amount converted by B. distasonis DSM 20701T and B.ovatus SDG-Mt85-3Cy, respectively.',16765860),(4146,'human','null','Secoisolariciresinol diglucoside','null','Over the first 30 min at 37 1C, the amount of SDG converted to SECO by resting cells of Clostridium sp. SDG Mt85-3Db was approximately 8- and 30-fold higher than the amount converted by B. distasonis DSM 20701T and B.ovatus SDG-Mt85-3Cy, respectively.',16765860),(4147,'human','null','Secoisolariciresinol diglucoside','null','Over the first 30 min at 37 1C, the amount of SDG converted to SECO by resting cells of Clostridium sp. SDG Mt85-3Db was approximately 8- and 30-fold higher than the amount converted by B. distasonis DSM 20701T and B.ovatus SDG-Mt85-3Cy, respectively.',16765860),(4148,'human','null','Secoisolariciresinol diglucoside','null','Over the first 30 min at 37 1C, the amount of SDG converted to SECO by resting cells of Clostridium sp. SDG Mt85-3Db was approximately 8- and 30-fold higher than the amount converted by B. distasonis DSM 20701T and B.ovatus SDG-Mt85-3Cy, respectively.',16765860),(4149,'human','null','Secoisolariciresinol diglucoside','null','Over the first 30 min at 37 1C, the amount of SDG converted to SECO by resting cells of Clostridium sp. SDG Mt85-3Db was approximately 8- and 30-fold higher than the amount converted by B. distasonis DSM 20701T and B.ovatus SDG-Mt85-3Cy, respectively.',16765860),(4150,'human','null','Lariciresinol','null','In addition,the previously isolated strain E. lenta SECO-Mt75m3 catalysed the reduction of pinoresinol to SECO via lariciresinol.',16765860),(4151,'human','null','Pinoresinol','null','In addition,the previously isolated strain E. lenta SECO-Mt75m3 catalysed the reduction of pinoresinol to SECO via lariciresinol.',16765860),(4152,'human','null','Matairesinol','null','However, methoxy or hydroxyl groups adjacent to the targeted methoxy groups may be required by P. productus SECO-Mt75m3 to catalyse demethylation, as evident from the inability of this organism to demethylate m-anisic acid. Incubation of P.productus SECO-Mt75m3 with ferulic acid led to complete conversion of the substrate without detection of any products by HPLC, suggesting that P. productus not only catalyses the demethylation of ferulic acid but also a number of additional reactions. More interestingly, P. productus SECO-Mt75m3 catalysed as well the demethyla tion of other lignans, namely lariciresinol (Fig. 2),\nmatairesinol and pinoresinol (data not shown). In addition, the previously isolated strain E. lenta SECO-Mt75m3 catalysed the reduction of pinoresinol to SECO via lariciresinol (Fig. 3).',16765860),(4153,'human','null','Secoisolariciresinol','null','In conclusion, SDG metabolising bacteria are expected to play a major role in the activation of various dietary lignans with ensuing consequences on the bioavailability and biological activity of enterolignans. Fig. 4 summarises the new data reported above and the present knowledge about the bacterial pathways of enterolignan production.',16765860),(4154,'human','null','Secoisolariciresinol','null','In conclusion, SDG metabolising bacteria are expected to play a major role in the activation of various dietary lignans with ensuing consequences on the bioavailability and biological activity of enterolignans. Fig. 4 summarises the new data reported above and the present knowledge about the bacterial pathways of enterolignan production.',16765860),(4155,'human','null','Secoisolariciresinol','null','In conclusion, SDG metabolising bacteria are expected to play a major role in the activation of various dietary lignans with ensuing consequences on the bioavailability and biological activity of enterolignans. Fig. 4 summarises the new data reported above and the present knowledge about the bacterial pathways of enterolignan production.',16765860),(4156,'human','null','Secoisolariciresinol','null','In conclusion, SDG metabolising bacteria are expected to play a major role in the activation of various dietary lignans with ensuing consequences on the bioavailability and biological activity of enterolignans. Fig. 4 summarises the new data reported above and the present knowledge about the bacterial pathways of enterolignan production.',16765860),(4157,'human','null','Pinoresinol','null','In conclusion, SDG metabolising bacteria are expected to play a major role in the activation of various dietary lignans with ensuing consequences on the bioavailability and biological activity of enterolignans. Fig. 4 summarises the new data reported above and the present knowledge about the bacterial pathways of enterolignan production.',16765860),(4158,'human','null','5,7-Dihydroxy-2-(4-hydroxyphenyl)chroman-4-one','null','A hypothetical pathway for the degradation of narin genin by E. ramulus has been proposed by Schneider and Blaut (2000) (Fig. 1a). The first step in this pathway is the isomerization of naringenin to naringenin chalcone, which is subsequently reduced to phloretin. Phloretin is further\nhydrolyzed to phloroglucinol and 3(4-hydroxyphenyl)pro pionic acid.',15127184),(4159,'human','null','Naringenin chalcone','null','A hypothetical pathway for the degradation of narin genin by E. ramulus has been proposed by Schneider and Blaut (2000) (Fig. 1a). The first step in this pathway is the isomerization of naringenin to naringenin chalcone, which is subsequently reduced to phloretin. Phloretin is further\nhydrolyzed to phloroglucinol and 3(4-hydroxyphenyl)pro pionic acid.',15127184),(4160,'human','null','Phloretin','null','A hypothetical pathway for the degradation of narin genin by E. ramulus has been proposed by Schneider and Blaut (2000) (Fig. 1a). The first step in this pathway is the isomerization of naringenin to naringenin chalcone, which is subsequently reduced to phloretin. Phloretin is further\nhydrolyzed to phloroglucinol and 3(4-hydroxyphenyl)pro pionic acid.',15127184),(4161,'human','null','Phloretin','null','A hypothetical pathway for the degradation of narin genin by E. ramulus has been proposed by Schneider and Blaut (2000) (Fig. 1a). The first step in this pathway is the isomerization of naringenin to naringenin chalcone, which is subsequently reduced to phloretin. Phloretin is further\nhydrolyzed to phloroglucinol and 3(4-hydroxyphenyl)pro pionic acid.',15127184),(4162,'human','null','Apigenin','null','The flavone apigenin is presum ably reduced to the flavanone naringenin in a first step which is followed by an isomerization to the corresponding chalcone structure. The chalcone is reduced to the dihydrochalcone phloretin, which is subsequently hydrolyzed to phloroglucinol and 3-(4-hydroxyphenyl)propionic acid (23). Although the degradation pathway of flavones by E. ramulus has successfully been elucidated, the enzymes involved have so far escaped purification and characterization.',15466559),(4163,'human','null','5,7-Dihydroxy-2-(4-hydroxyphenyl)chroman-4-one','null','The flavone apigenin is presum ably reduced to the flavanone naringenin in a first step which\nis followed by an isomerization to the corresponding chalcone structure. The chalcone is reduced to the dihydrochalcone phloretin, which is subsequently hydrolyzed to phloroglucinol and 3-(4-hydroxyphenyl)propionic acid (23). Although the degradation pathway of flavones by E. ramulus has successfully been elucidated, the enzymes involved have so far escaped purification and characterization.',15466559),(4164,'human','null','Naringenin chalcone','null','The flavone apigenin is presum ably reduced to the flavanone naringenin in a first step which\nis followed by an isomerization to the corresponding chalcone structure. The chalcone is reduced to the dihydrochalcone phloretin, which is subsequently hydrolyzed to phloroglucinol and 3-(4-hydroxyphenyl)propionic acid (23). Although the degradation pathway of flavones by E. ramulus has successfully been elucidated, the enzymes involved have so far escaped purification and characterization.',15466559),(4165,'human','null','Phloretin','null','The flavone apigenin is presum ably reduced to the flavanone naringenin in a first step which\nis followed by an isomerization to the corresponding chalcone structure. The chalcone is reduced to the dihydrochalcone phloretin, which is subsequently hydrolyzed to phloroglucinol and 3-(4-hydroxyphenyl)propionic acid (23). Although the degradation pathway of flavones by E. ramulus has successfully been elucidated, the enzymes involved have so far escaped purification and characterization.',15466559),(4166,'human','null','Phloretin','null','The flavone apigenin is presum ably reduced to the flavanone naringenin in a first step which\nis followed by an isomerization to the corresponding chalcone structure. The chalcone is reduced to the dihydrochalcone phloretin, which is subsequently hydrolyzed to phloroglucinol and 3-(4-hydroxyphenyl)propionic acid (23). Although the degradation pathway of flavones by E. ramulus has successfully been elucidated, the enzymes involved have so far escaped purification and characterization.',15466559),(4167,'human','null','Kaempferol-3-O-gal-rham-7-O-rham','null','Bacteroides distasonis hydrolyzed robinin to kaempferol, and Bacteroides uniformis and Bacteroides ovatus converted rutin to quercetin.',19580283),(4168,'human','null','Rutin','null','Bacteroides distasonis hydrolyzed robinin to kaempferol, and Bacteroides uniformis and Bacteroides ovatus converted rutin to quercetin.',19580283),(4169,'human','null','Rutin','null','Bacteroides distasonis hydrolyzed robinin to kaempferol, and Bacteroides uniformis and Bacteroides ovatus converted rutin to quercetin.',19580283),(4170,'human','null','Glycitein','null','Glycitein is demethoxylated in vitro by Eubacterium limosum to 6,7,40-trihydroxyisoflavone .',19580283),(4171,'human','null','Daidzein','null','Strains of Bacteroide ovatus spp., Streptococcus intermedius spp., and Ru minococcus productus SNU-Julong 732 (AY310748) as well as a stable mixed culture consisting of Enterococcus faecium EPI1, Lactobacillus mucosae EPI2, Finegoldia magna EPI3, and Veillo nella sp. EP have been isolated from human feces and reported to be able to convert daidzein into equol (16, 119-122) .',19580283),(4172,'human','null','Daidzein','null','Strains of Bacteroide ovatus spp., Streptococcus intermedius spp., and Ru minococcus productus SNU-Julong 732 (AY310748) as well as a stable mixed culture consisting of Enterococcus faecium EPI1, Lactobacillus mucosae EPI2, Finegoldia magna EPI3, and Veillo nella sp. EP have been isolated from human feces and reported to be able to convert daidzein into equol (16, 119-122) .',19580283),(4173,'human','null','Daidzein','null','Strains of Bacteroide ovatus spp., Streptococcus intermedius spp., and Ru minococcus productus SNU-Julong 732 (AY310748) as well as a stable mixed culture consisting of Enterococcus faecium EPI1, Lactobacillus mucosae EPI2, Finegoldia magna EPI3, and Veillo nella sp. EP have been isolated from human feces and reported to be able to convert daidzein into equol (16, 119-122) .',19580283),(4174,'human','null','Daidzein','null','Strains of Bacteroide ovatus spp., Streptococcus intermedius spp., and Ru minococcus productus SNU-Julong 732 (AY310748) as well as a stable mixed culture consisting of Enterococcus faecium EPI1, Lactobacillus mucosae EPI2, Finegoldia magna EPI3, and Veillo nella sp. EP have been isolated from human feces and reported to be able to convert daidzein into equol (16, 119-122) .',19580283),(4175,'human','null','Daidzein','null','Strains of Bacteroide ovatus spp., Streptococcus intermedius spp., and Ru minococcus productus SNU-Julong 732 (AY310748) as well as a stable mixed culture consisting of Enterococcus faecium EPI1, Lactobacillus mucosae EPI2, Finegoldia magna EPI3, and Veillo nella sp. EP have been isolated from human feces and reported to be able to convert daidzein into equol (16, 119-122) .',19580283),(4176,'human','null','Daidzein','null','Strains of Bacteroide ovatus spp., Streptococcus intermedius spp., and Ru minococcus productus SNU-Julong 732 (AY310748) as well as a stable mixed culture consisting of Enterococcus faecium EPI1, Lactobacillus mucosae EPI2, Finegoldia magna EPI3, and Veillo nella sp. EP have been isolated from human feces and reported to be able to convert daidzein into equol (16, 119-122) .',19580283),(4177,'human','null','Daidzein','null','Strains of Bacteroide ovatus spp., Streptococcus intermedius spp., and Ru minococcus productus SNU-Julong 732 (AY310748) as well as a stable mixed culture consisting of Enterococcus faecium EPI1, Lactobacillus mucosae EPI2, Finegoldia magna EPI3, and Veillo nella sp. EP have been isolated from human feces and reported to be able to convert daidzein into equol (16, 119-122) .',19580283),(4178,'human','null','Dihydrodaidzein','null','The bacterial strain named Julong 732 is also able to metabolize dihydrodaidzein to S-equol (123). Thus, other bacteria have to be involved in the previous steps of the conversion of daidzein to equol .',19580283),(4179,'human','null','Isoxanthohumol','null','Isoxanthohumol is a prenylated flavanone from hops and beer that has interesting biological activities. It can be metabo lized in vivo by colonic microbiota to render 8-prenylnaringenin (133, 134). This metabolic O-demethylation is carried out by Eubacterium species and particularly by Eubacterium limosum.',19580283),(4180,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4181,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4182,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4183,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4184,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4185,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4186,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4187,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4188,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4189,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4190,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4191,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4192,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4193,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4194,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4195,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4196,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4197,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4198,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4199,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4200,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4201,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4202,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4203,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4204,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4205,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4206,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4207,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4208,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4209,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4210,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4211,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4212,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4213,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4214,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4215,'mouse','null','Cholesterol','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4216,'mouse','null','Cholesterol','null','Prevotella and Paraprevotella presented negative correlations with the same nine  metabolites 93-phenylpropanoic acid, cholic acid, retinol, L-glutamate, L-phenylalanine, 7-dehydrocholesterol,  chenodeoxycholic acid, (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid and 11-deoxycorticosterone) and positive  correlations with four metabolites (tetradecanoic acid, L-isoleucine, benzoic acid, and indole-3-acetic acid). Our  results also revealed clear correlations among some metabolites.',31780788),(4217,'mouse','null','Cholesterol','null','Prevotella and Paraprevotella presented negative correlations with the same nine  metabolites 93-phenylpropanoic acid, cholic acid, retinol, L-glutamate, L-phenylalanine, 7-dehydrocholesterol,  chenodeoxycholic acid, (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid and 11-deoxycorticosterone) and positive  correlations with four metabolites (tetradecanoic acid, L-isoleucine, benzoic acid, and indole-3-acetic acid). Our  results also revealed clear correlations among some metabolites.',31780788),(4218,'mouse','null','Cholesterol','null','Prevotella and Paraprevotella presented negative correlations with the same nine  metabolites 93-phenylpropanoic acid, cholic acid, retinol, L-glutamate, L-phenylalanine, 7-dehydrocholesterol,  chenodeoxycholic acid, (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid and 11-deoxycorticosterone) and positive  correlations with four metabolites (tetradecanoic acid, L-isoleucine, benzoic acid, and indole-3-acetic acid). Our  results also revealed clear correlations among some metabolites.',31780788),(4219,'mouse','null','Cholesterol','null','Prevotella and Paraprevotella presented negative correlations with the same nine  metabolites 93-phenylpropanoic acid, cholic acid, retinol, L-glutamate, L-phenylalanine, 7-dehydrocholesterol,  chenodeoxycholic acid, (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid and 11-deoxycorticosterone) and positive  correlations with four metabolites (tetradecanoic acid, L-isoleucine, benzoic acid, and indole-3-acetic acid). Our  results also revealed clear correlations among some metabolites.',31780788),(4220,'mouse','null','Cholesterol','null','Prevotella and Paraprevotella presented negative correlations with the same nine  metabolites 93-phenylpropanoic acid, cholic acid, retinol, L-glutamate, L-phenylalanine, 7-dehydrocholesterol,  chenodeoxycholic acid, (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid and 11-deoxycorticosterone) and positive  correlations with four metabolites (tetradecanoic acid, L-isoleucine, benzoic acid, and indole-3-acetic acid). Our  results also revealed clear correlations among some metabolites.',31780788),(4221,'mouse','null','Cholesterol','null','Prevotella and Paraprevotella presented negative correlations with the same nine  metabolites 93-phenylpropanoic acid, cholic acid, retinol, L-glutamate, L-phenylalanine, 7-dehydrocholesterol,  chenodeoxycholic acid, (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid and 11-deoxycorticosterone) and positive  correlations with four metabolites (tetradecanoic acid, L-isoleucine, benzoic acid, and indole-3-acetic acid). Our  results also revealed clear correlations among some metabolites.',31780788),(4222,'mouse','null','Cholesterol','null','Prevotella and Paraprevotella presented negative correlations with the same nine  metabolites 93-phenylpropanoic acid, cholic acid, retinol, L-glutamate, L-phenylalanine, 7-dehydrocholesterol,  chenodeoxycholic acid, (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid and 11-deoxycorticosterone) and positive  correlations with four metabolites (tetradecanoic acid, L-isoleucine, benzoic acid, and indole-3-acetic acid). Our  results also revealed clear correlations among some metabolites.',31780788),(4223,'mouse','null','Cholesterol','null','Prevotella and Paraprevotella presented negative correlations with the same nine  metabolites 93-phenylpropanoic acid, cholic acid, retinol, L-glutamate, L-phenylalanine, 7-dehydrocholesterol,  chenodeoxycholic acid, (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid and 11-deoxycorticosterone) and positive  correlations with four metabolites (tetradecanoic acid, L-isoleucine, benzoic acid, and indole-3-acetic acid). Our  results also revealed clear correlations among some metabolites.',31780788),(4224,'mouse','null','null','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4225,'mouse','null','null','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4226,'mouse','null','null','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4227,'mouse','null','null','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4228,'mouse','null','null','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4229,'mouse','null','null','null','unclassifed Lachnospiraceae, Oscillibacter,  Clostridium IV and Clostridium XlVa of phylum Firmicutes identifed upon RSB application showed similar  trends of correlation. Te relative abundance of the predominant RSB-enriched gut bacterial genus of unclas sifed Lachnospiraceae was positively associated with the concentrations of nine increased metabolites, includ ing cholic acid, 7-dehydrocholesterol, 11-deoxycorticosterone, retinol, chenodeoxycholic acid, L-phenylalanine,  L-glutamate, 3-phenylpropanoic acid and (5Z, 8Z, 11Z, 14Z, 17Z)-eicosapentaenoic acid, and negatively cor related with the concentrations of four metabolites showing decreasing trends, including L-isoleucine, tetra decanoic acid, indole-3-acetic acid, and benzoic acid.',31780788),(4233,'mouse','null','Epigallocatechin','null','Biotransformation of both EGC and GC was observed with Adlercreutzia equolifaciens JCM 14793, Asaccharobacter celatus JCM 14811, and Slackia equolifaciens JCM  16059, but not Slackia isoflavoniconvertens JCM 16137. With respect to the degradation of EGC, strain JCM  14793 only catalyzed 4�?dehydroxylation to produce 4�?dehydroxylated EGC (7). Strain JCM 14811 mainly  produced 7, along with a slight formation of the C ring-cleaving product 1-(3,4,5-trihydroxyphenyl)-3-(2,4,6trihydroxyphenyl)propan-2-ol (1). Strain JCM 16059 catalyzed only C ring cleavage to form 1. Interestingly,  the presence of hydrogen promoted the bioconversion of EGC by these bacteria. In addition, strain JCM  14811 revealed the ability to catalyze 4�?dehydroxylation of 1 to yield 1-(3,5-dihydroxyphenyl)-3-(2,4,6-trihy droxyphenyl)propan-2-ol (2) in the presence of hydrogen. In the case of GC, strain JCM 14793 mainly pro duced C ring-cleaving product (1) with only a very small amount of 4�?dehydroxylated GC (8), while Strain  JCM 14811 only catalyzed 4�?dehydroxylation to form 8. Strain JCM 16059 formed 1. The bioconversion of  GC by the three strains was stimulated by hydrogen. Strain JCM 14793 showed the ability to convert 1 into  2 in the presence of hydrogen as did strain JCM 14811. Furthermore, strains JCM 14793 and JCM 14811  were found to have the ability to catalyze p-dehydroxylation of the pyrogallol moiety in the EGC metabolites  4-hydroxy-5-(3,4,5-trihydroxyphenyl)valeric acid (3) and 5-(3,4,5-trihydroxyphenyl)-γ-valerolactone (4), and  this ability was enhanced by the presence of hydrogen.',25747993),(4234,'mouse','null','Epigallocatechin','null','Biotransformation of both EGC and GC was observed with  Adlercreutzia equolifaciens JCM 14793, Asaccharobacter celatus JCM 14811, and Slackia equolifaciens JCM  16059, but not Slackia isoflavoniconvertens JCM 16137. With respect to the degradation of EGC, strain JCM  14793 only catalyzed 4�?dehydroxylation to produce 4�?dehydroxylated EGC (7). Strain JCM 14811 mainly  produced 7, along with a slight formation of the C ring-cleaving product 1-(3,4,5-trihydroxyphenyl)-3-(2,4,6trihydroxyphenyl)propan-2-ol (1). Strain JCM 16059 catalyzed only C ring cleavage to form 1. Interestingly,  the presence of hydrogen promoted the bioconversion of EGC by these bacteria. In addition, strain JCM  14811 revealed the ability to catalyze 4�?dehydroxylation of 1 to yield 1-(3,5-dihydroxyphenyl)-3-(2,4,6-trihy droxyphenyl)propan-2-ol (2) in the presence of hydrogen. In the case of GC, strain JCM 14793 mainly pro duced C ring-cleaving product (1) with only a very small amount of 4�?dehydroxylated GC (8), while Strain  JCM 14811 only catalyzed 4�?dehydroxylation to form 8. Strain JCM 16059 formed 1. The bioconversion of  GC by the three strains was stimulated by hydrogen. Strain JCM 14793 showed the ability to convert 1 into  2 in the presence of hydrogen as did strain JCM 14811. Furthermore, strains JCM 14793 and JCM 14811  were found to have the ability to catalyze p-dehydroxylation of the pyrogallol moiety in the EGC metabolites  4-hydroxy-5-(3,4,5-trihydroxyphenyl)valeric acid (3) and 5-(3,4,5-trihydroxyphenyl)-γ-valerolactone (4), and  this ability was enhanced by the presence of hydrogen.',25747993),(4235,'mouse','null','Epigallocatechin','null','Biotransformation of both EGC and GC was observed with  Adlercreutzia equolifaciens JCM 14793, Asaccharobacter celatus JCM 14811, and Slackia equolifaciens JCM  16059, but not Slackia isoflavoniconvertens JCM 16137. With respect to the degradation of EGC, strain JCM  14793 only catalyzed 4�?dehydroxylation to produce 4�?dehydroxylated EGC (7). Strain JCM 14811 mainly  produced 7, along with a slight formation of the C ring-cleaving product 1-(3,4,5-trihydroxyphenyl)-3-(2,4,6trihydroxyphenyl)propan-2-ol (1). Strain JCM 16059 catalyzed only C ring cleavage to form 1. Interestingly,  the presence of hydrogen promoted the bioconversion of EGC by these bacteria. In addition, strain JCM  14811 revealed the ability to catalyze 4�?dehydroxylation of 1 to yield 1-(3,5-dihydroxyphenyl)-3-(2,4,6-trihy droxyphenyl)propan-2-ol (2) in the presence of hydrogen. In the case of GC, strain JCM 14793 mainly pro duced C ring-cleaving product (1) with only a very small amount of 4�?dehydroxylated GC (8), while Strain  JCM 14811 only catalyzed 4�?dehydroxylation to form 8. Strain JCM 16059 formed 1. The bioconversion of  GC by the three strains was stimulated by hydrogen. Strain JCM 14793 showed the ability to convert 1 into  2 in the presence of hydrogen as did strain JCM 14811. Furthermore, strains JCM 14793 and JCM 14811  were found to have the ability to catalyze p-dehydroxylation of the pyrogallol moiety in the EGC metabolites  4-hydroxy-5-(3,4,5-trihydroxyphenyl)valeric acid (3) and 5-(3,4,5-trihydroxyphenyl)-γ-valerolactone (4), and  this ability was enhanced by the presence of hydrogen.',25747993),(4236,'mouse','null','4\'-dehydroxylated (�?-epigallocatechin','null','Biotransformation of both EGC and GC was observed with  Adlercreutzia equolifaciens JCM 14793, Asaccharobacter celatus JCM 14811, and Slackia equolifaciens JCM  16059, but not Slackia isoflavoniconvertens JCM 16137. With respect to the degradation of EGC, strain JCM  14793 only catalyzed 4�?dehydroxylation to produce 4�?dehydroxylated EGC (7). Strain JCM 14811 mainly  produced 7, along with a slight formation of the C ring-cleaving product 1-(3,4,5-trihydroxyphenyl)-3-(2,4,6trihydroxyphenyl)propan-2-ol (1). Strain JCM 16059 catalyzed only C ring cleavage to form 1. Interestingly,  the presence of hydrogen promoted the bioconversion of EGC by these bacteria. In addition, strain JCM  14811 revealed the ability to catalyze 4�?dehydroxylation of 1 to yield 1-(3,5-dihydroxyphenyl)-3-(2,4,6-trihy droxyphenyl)propan-2-ol (2) in the presence of hydrogen. In the case of GC, strain JCM 14793 mainly pro duced C ring-cleaving product (1) with only a very small amount of 4�?dehydroxylated GC (8), while Strain  JCM 14811 only catalyzed 4�?dehydroxylation to form 8. Strain JCM 16059 formed 1. The bioconversion of  GC by the three strains was stimulated by hydrogen. Strain JCM 14793 showed the ability to convert 1 into  2 in the presence of hydrogen as did strain JCM 14811. Furthermore, strains JCM 14793 and JCM 14811  were found to have the ability to catalyze p-dehydroxylation of the pyrogallol moiety in the EGC metabolites  4-hydroxy-5-(3,4,5-trihydroxyphenyl)valeric acid (3) and 5-(3,4,5-trihydroxyphenyl)-γ-valerolactone (4), and  this ability was enhanced by the presence of hydrogen.',25747993),(4237,'mouse','null','null','null','Biotransformation of both EGC and GC was observed with  Adlercreutzia equolifaciens JCM 14793, Asaccharobacter celatus JCM 14811, and Slackia equolifaciens JCM  16059, but not Slackia isoflavoniconvertens JCM 16137. With respect to the degradation of EGC, strain JCM  14793 only catalyzed 4�?dehydroxylation to produce 4�?dehydroxylated EGC (7). Strain JCM 14811 mainly  produced 7, along with a slight formation of the C ring-cleaving product 1-(3,4,5-trihydroxyphenyl)-3-(2,4,6trihydroxyphenyl)propan-2-ol (1). Strain JCM 16059 catalyzed only C ring cleavage to form 1. Interestingly,  the presence of hydrogen promoted the bioconversion of EGC by these bacteria. In addition, strain JCM  14811 revealed the ability to catalyze 4�?dehydroxylation of 1 to yield 1-(3,5-dihydroxyphenyl)-3-(2,4,6-trihy droxyphenyl)propan-2-ol (2) in the presence of hydrogen. In the case of GC, strain JCM 14793 mainly pro duced C ring-cleaving product (1) with only a very small amount of 4�?dehydroxylated GC (8), while Strain  JCM 14811 only catalyzed 4�?dehydroxylation to form 8. Strain JCM 16059 formed 1. The bioconversion of  GC by the three strains was stimulated by hydrogen. Strain JCM 14793 showed the ability to convert 1 into  2 in the presence of hydrogen as did strain JCM 14811. Furthermore, strains JCM 14793 and JCM 14811  were found to have the ability to catalyze p-dehydroxylation of the pyrogallol moiety in the EGC metabolites  4-hydroxy-5-(3,4,5-trihydroxyphenyl)valeric acid (3) and 5-(3,4,5-trihydroxyphenyl)-γ-valerolactone (4), and  this ability was enhanced by the presence of hydrogen.',25747993),(4238,'mouse','null','null','null','Biotransformation of both EGC and GC was observed with  Adlercreutzia equolifaciens JCM 14793, Asaccharobacter celatus JCM 14811, and Slackia equolifaciens JCM  16059, but not Slackia isoflavoniconvertens JCM 16137. With respect to the degradation of EGC, strain JCM  14793 only catalyzed 4�?dehydroxylation to produce 4�?dehydroxylated EGC (7). Strain JCM 14811 mainly  produced 7, along with a slight formation of the C ring-cleaving product 1-(3,4,5-trihydroxyphenyl)-3-(2,4,6trihydroxyphenyl)propan-2-ol (1). Strain JCM 16059 catalyzed only C ring cleavage to form 1. Interestingly,  the presence of hydrogen promoted the bioconversion of EGC by these bacteria. In addition, strain JCM  14811 revealed the ability to catalyze 4�?dehydroxylation of 1 to yield 1-(3,5-dihydroxyphenyl)-3-(2,4,6-trihy droxyphenyl)propan-2-ol (2) in the presence of hydrogen. In the case of GC, strain JCM 14793 mainly pro duced C ring-cleaving product (1) with only a very small amount of 4�?dehydroxylated GC (8), while Strain  JCM 14811 only catalyzed 4�?dehydroxylation to form 8. Strain JCM 16059 formed 1. The bioconversion of  GC by the three strains was stimulated by hydrogen. Strain JCM 14793 showed the ability to convert 1 into  2 in the presence of hydrogen as did strain JCM 14811. Furthermore, strains JCM 14793 and JCM 14811  were found to have the ability to catalyze p-dehydroxylation of the pyrogallol moiety in the EGC metabolites  4-hydroxy-5-(3,4,5-trihydroxyphenyl)valeric acid (3) and 5-(3,4,5-trihydroxyphenyl)-γ-valerolactone (4), and  this ability was enhanced by the presence of hydrogen.',25747993),(4239,'mouse','null','1-(3,4,5-trihydroxyphenyl)-3-(2,4,6trihydroxyphenyl)propan-2-ol','null','Biotransformation of both EGC and GC was observed with  Adlercreutzia equolifaciens JCM 14793, Asaccharobacter celatus JCM 14811, and Slackia equolifaciens JCM  16059, but not Slackia isoflavoniconvertens JCM 16137. With respect to the degradation of EGC, strain JCM  14793 only catalyzed 4�?dehydroxylation to produce 4�?dehydroxylated EGC (7). Strain JCM 14811 mainly  produced 7, along with a slight formation of the C ring-cleaving product 1-(3,4,5-trihydroxyphenyl)-3-(2,4,6trihydroxyphenyl)propan-2-ol (1). Strain JCM 16059 catalyzed only C ring cleavage to form 1. Interestingly,  the presence of hydrogen promoted the bioconversion of EGC by these bacteria. In addition, strain JCM  14811 revealed the ability to catalyze 4�?dehydroxylation of 1 to yield 1-(3,5-dihydroxyphenyl)-3-(2,4,6-trihy droxyphenyl)propan-2-ol (2) in the presence of hydrogen. In the case of GC, strain JCM 14793 mainly pro duced C ring-cleaving product (1) with only a very small amount of 4�?dehydroxylated GC (8), while Strain  JCM 14811 only catalyzed 4�?dehydroxylation to form 8. Strain JCM 16059 formed 1. The bioconversion of  GC by the three strains was stimulated by hydrogen. Strain JCM 14793 showed the ability to convert 1 into  2 in the presence of hydrogen as did strain JCM 14811. Furthermore, strains JCM 14793 and JCM 14811  were found to have the ability to catalyze p-dehydroxylation of the pyrogallol moiety in the EGC metabolites  4-hydroxy-5-(3,4,5-trihydroxyphenyl)valeric acid (3) and 5-(3,4,5-trihydroxyphenyl)-γ-valerolactone (4), and  this ability was enhanced by the presence of hydrogen.',25747993),(4240,'mouse','null','(-)-Gallocatechin','null','The�?bioconversion�?of�?GC�?was�?examined�?here.�? Strain MT4s-5 was�?observed to convert GC into 1 and 8  with�?slight production of 2. In the presence of hydrogen, strain MT4s-5�?accelerated�?the�?formation�?of�?2�?(data�?not�?shown).�?The�?results�?were�?very�?similar�?to�?those�?obtained�?in�?the�?bioconversion of EGC by strain  MT4s-5.',25747993),(4241,'mouse','null','(-)-Gallocatechin','null','The�?bioconversion�?of�?GC�?was�?examined�?here.�? Strain MT4s-5 was�?observed to convert GC into 1 and 8  with�?slight production of 2. In the presence of hydrogen, strain MT4s-5�?accelerated�?the�?formation�?of�?2�?(data�?not�?shown).�?The�?results�?were�?very�?similar�?to�?those�?obtained�?in�?the�?bioconversion of EGC by strain  MT4s-5.',25747993),(4242,'mouse','null','(-)-Gallocatechin','null','The�?bioconversion�?of�?GC�?was�?examined�?here.�? Strain MT4s-5 was�?observed to convert GC into 1 and 8  with�?slight production of 2. In the presence of hydrogen, strain MT4s-5�?accelerated�?the�?formation�?of�?2�?(data�?not�?shown).�?The�?results�?were�?very�?similar�?to�?those�?obtained�?in�?the�?bioconversion of EGC by strain  MT4s-5.',25747993),(4243,'mouse','decrease','null','low-throughput','Ratelimited enzymes in the process of TG synthesis of the DGAT1/2 mRNA level in the HFD group were signifificantly increased compared with the LFD group. The DGAT2 mRNA level in the HFD-CBM group decreased in the same range as the LFD group, whereas signifificant differences of the DGAT1 mRNA level were not found between the HFD group and the HFD-CBM group.',24166662),(4249,'mouse','decrease','null','low-throughput','TMAO inhibits SIRT1 expression, induces oxidative stress, and impairs NO production both in vivo and in vitro.',29325896),(4250,'mouse','decrease','null','low-throughput','By conducting a miRNA microarray screening, we revealed that expression of miRNA-10b in macrophages can be reduced by PCA. Functional analyses demonstrated that miRNA-10b directly represses ABCA1 and ABCG1 and negatively regulates cholesterol efflux from murine- and human-derived macrophages. Further in vitro and ex vivo analyses verified that PCA accelerates macrophage cholesterol efflux, correlating with the regulation of miRNA-10b-ABCA1/ABCG1 cascade, whereas Cy-3-G consumption promoted macrophage RCT and regressed atherosclerotic lesion in a gut microbiotaendependent manner.',22821931),(4251,'mouse','increase','null','low-throughput','By conducting a miRNA microarray screening, we revealed that expression of miRNA-10b in macrophages can be reduced by PCA. Functional analyses demonstrated that miRNA-10b directly represses ABCA1 and ABCG1 and negatively regulates cholesterol efflux from murine- and human-derived macrophages. Further in vitro and ex vivo analyses verified that PCA accelerates macrophage cholesterol efflux, correlating with the regulation of miRNA-10b-ABCA1/ABCG1 cascade, whereas Cy-3-G consumption promoted macrophage RCT and regressed atherosclerotic lesion in a gut microbiotaendependent manner.',22821931),(4252,'mouse','increase','null','low-throughput','By conducting a miRNA microarray screening, we revealed that expression of miRNA-10b in macrophages can be reduced by PCA. Functional analyses demonstrated that miRNA-10b directly represses ABCA1 and ABCG1 and negatively regulates cholesterol efflux from murine- and human-derived macrophages. Further in vitro and ex vivo analyses verified that PCA accelerates macrophage cholesterol efflux, correlating with the regulation of miRNA-10b-ABCA1/ABCG1 cascade, whereas Cy-3-G consumption promoted macrophage RCT and regressed atherosclerotic lesion in a gut microbiotaendependent manner.',22821931),(4253,'mouse','decrease','null','low-throughput','We extended our study to human samples to determine the effect of bile acids on CXCL16 mRNA expression of human liver sinusoidal endothelial cells. Similar to the mouse study, primary bile acids CDCA and TCA induced CXCL16 mRNA expression in SK-HEP1 cells (Fig. 5H). Next, the correlation between bile acids and CXCL16 expression was tested in nontumor liver tissues from HCC or cholangiocarcinoma patients of the Thailand Initiative for Genomics and Expression Research in Liver Cancer (TIGER-LC) cohort (34). Primary bile acid CDCA levels correlated with CXCL16 expression (Fig. 5I), whereas inverse correlation was seen for the secondary bile acid glycolithocholate (GLCA) (fig. S6I). The primary/secondary ratio was associated with CXCL16 increase (fig. S6J), indicating that the opposing effect of bile acids on CXCL16 expression also exists in humans.',29798856),(4254,'mouse','decrease','null','low-throughput','In summary,this study showed that it is butyrate, in stead of other substances produced by F.prausnitzii,inhibiting HDAC1 to activate Foxp3 and block the IL-6/STAT3/IL-17 pathway,thus regulating Th17/Treg cells and exerting an anti-inflammatory effect in IBD animal models.',29796620),(4255,'mouse','decrease','null','low-throughput','In summary,this study showed that it is butyrate, in stead of other substances produced by F.prausnitzii,inhibiting HDAC1 to activate Foxp3 and block the IL-6/STAT3/IL-17 pathway,thus regulating Th17/Treg cells and exerting an anti-inflammatory effect in IBD animal models.',29796620),(4256,'mouse','decrease','null','low-throughput','In summary,this study showed that it is butyrate, in stead of other substances produced by F.prausnitzii,inhibiting HDAC1 to activate Foxp3 and block the IL-6/STAT3/IL-17 pathway,thus regulating Th17/Treg cells and exerting an anti-inflammatory effect in IBD animal models.',29796620),(4257,'mouse','decrease','null','low-throughput','In summary,this study showed that it is butyrate, in stead of other substances produced by F.prausnitzii,inhibiting HDAC1 to activate Foxp3 and block the IL-6/STAT3/IL-17 pathway,thus regulating Th17/Treg cells and exerting an anti-inflammatory effect in IBD animal models.',29796620),(4258,'mouse','increase','null','low-throughput','In summary,this study showed that it is butyrate, in stead of other substances produced by F.prausnitzii,inhibiting HDAC1 to activate Foxp3 and block the IL-6/STAT3/IL-17 pathway,thus regulating Th17/Treg cells and exerting an anti-inflammatory effect in IBD animal models.',29796620),(4259,'mouse','null','4-Hydroxyphenylacetic acid','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4260,'mouse','null','4-Hydroxyphenylacetic acid','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4261,'mouse','null','4-Hydroxyphenylacetic acid','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4262,'mouse','null','4-Hydroxyphenylacetic acid','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4263,'mouse','null','4-Hydroxyphenylacetic acid','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4264,'mouse','null','4-Hydroxyphenylacetic acid','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4265,'mouse','null','4-Hydroxyphenylacetic acid','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4266,'mouse','null','4-Hydroxyphenylacetic acid','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4267,'mouse','null','4-Hydroxyphenylacetic acid','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4268,'mouse','null','D-Glucose','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4269,'mouse','null','D-Glucose','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4270,'mouse','null','D-Glucose','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4271,'mouse','null','D-Glucose','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4272,'mouse','null','D-Glucose','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4273,'mouse','null','D-Glucose','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4274,'mouse','null','D-Glucose','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4275,'mouse','null','D-Glucose','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4276,'mouse','null','D-Glucose','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4277,'mouse','null','D-Glucose','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4278,'mouse','null','D-Glucose','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4279,'mouse','null','D-Glucose','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4280,'mouse','null','Indole-3-acetic acid','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4281,'mouse','null','Indole-3-acetic acid','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4282,'mouse','null','Indole-3-acetic acid','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4283,'mouse','null','Indole-3-acetic acid','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4284,'mouse','null','Indole-3-acetic acid','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4285,'mouse','null','Indole-3-acetic acid','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4286,'mouse','null','Indole-3-acetic acid','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4287,'mouse','null','Indole-3-acetic acid','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4288,'mouse','null','Indole-3-acetic acid','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4289,'mouse','null','Phenylacetic acid','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4290,'mouse','null','Phenylacetic acid','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4291,'mouse','null','Phenylacetic acid','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4292,'mouse','null','Phenylacetic acid','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4293,'mouse','null','Phenylacetic acid','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4294,'mouse','null','Phenylacetic acid','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4295,'mouse','null','Phenylacetic acid','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4296,'mouse','null','Phenylacetic acid','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4297,'mouse','null','Phenylacetic acid','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4298,'mouse','null','Phenylacetic acid','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4299,'mouse','null','Phenylacetic acid','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4300,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4301,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4302,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4303,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4304,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4305,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4306,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4307,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4308,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4309,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4310,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4311,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4312,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4313,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4314,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4315,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4316,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4317,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4318,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4319,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4320,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4321,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4322,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4323,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4324,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4325,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4326,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4327,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4328,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4329,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4330,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4331,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4332,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4333,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4334,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4335,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4336,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4337,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4338,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4339,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4340,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4341,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4342,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4343,'mouse','null','Tryptophan','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4344,'mouse','null','phenyalanine','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4345,'mouse','null','phenyalanine','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4346,'mouse','null','phenyalanine','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4347,'mouse','null','phenyalanine','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4348,'mouse','null','phenyalanine','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4349,'mouse','null','phenyalanine','null','To identify the bacteria responsible, 26 strains representing 25 of the dominant human colonic species were screened for phenyl metabolite formation. Seven strains produced significant amounts of both PAA and 4-hydroxyPAA. These included five out of seven Bacteroidetes (Bacteroides thetaiotaomicron, Bacteroides eggerthii, Bacteroides ovatus, Bac teroides fragilis, Parabacteroides distasonis), and two out of 17 Firmicutes (Eubacterium hallii and Clostridium bartlettii). These species also produced indole-3-acetic acid (IAA), the corresponding tryptophan metabolite, but C. bartlettii showed 100 times higher IAA production than the other six strains. Four strains were further tested and PAA formation was substantially increased by phenylalanine, 4-hydroxyPAA by tyrosine and IAA by tryptophan.',23349065),(4350,'mouse','increase','null','low-throughput','The main metabolites produced by B. thetaiotaomicron in the GIT were acetate (up to 6 mM) and propionate (up to 2 mM) (Figure 1D). We evaluated the effects of these bacterial metabolites on the amount of KLF4 protein produced by the mucus producing cell line HT29-MTX [31]. The amount of KLF4 protein was higher in the presence than in the absence of acetate (10 and 20 mM) (Figure 3A)whereas propionate (5 and 10 mM) and butyrate (1 and 5 mM) had no effect (Figure 3B, C).',23692866),(4351,'mouse','null','null','null','The main metabolites produced by B. thetaiotaomicron in the GIT were acetate (up to 6 mM) and propionate (up to 2 mM) (Figure 1D). We evaluated the effects of these bacterial metabolites on the amount of KLF4 protein produced by the mucus producing cell line HT29-MTX [31]. The amount of KLF4 protein was higher in the presence than in the absence of acetate (10 and 20 mM) (Figure 3A)whereas propionate (5 and 10 mM) and butyrate (1 and 5 mM) had no effect (Figure 3B, C).',23692866),(4357,'mouse','increase','Acetate','low-throughput','We first checked  in vitro  that  B. thetaiotaomicron produced acetate and propionate and that  F. prausnitzii consumed acetate and produced butyrate (Figure 4A). When both bacteria were grown together, butyrate was produced rather than acetate. The butyrate production by F. prausnitzii as higher in the presence than absence of B. thetaiotaomicron  (10.9 ± 0.3 mM versus 7.8 ± 0.5 mM) suggesting that  F. prausnitzii  synthesized butyrate from the acetate produced by  B. thetaiotaomicron .',23692866),(4361,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4362,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4363,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4364,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4365,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4366,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4367,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4368,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4369,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4370,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4371,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4372,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4373,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4374,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4375,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4376,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4377,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4378,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4379,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4380,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4381,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4382,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4383,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4384,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4385,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4386,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4387,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4388,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4389,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4390,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4391,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4392,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4393,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4394,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4395,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4396,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4397,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4398,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4399,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4400,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4401,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4402,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4403,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4404,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4405,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4406,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4407,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4408,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4409,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4410,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4411,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4412,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4413,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4414,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4415,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4416,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4417,'mouse','null','null','null','Regarding the fecal potential biomarkers, all the elevated potential biomarkers were positively correlated with g__Hungatella, and all the elevated biomarkers but for valine and 2-formaminobenzoylacetate were positively correlated with g__Parabacteroides and g__Intestinibacter. N-acetylputrescine, isoleucine, tyramine, valine, 5-hydroxyindoleacetate, 2formaminobenzoylacetate, hyocholate, and cholate were positively correlated with g__ Ruminococcus_torques_group, while isoleucine was also positively correlated with g__Blautia. N-acetyl-D-mannosamine, chenodeoxycholate, and 12-ketodeoxycholate were positively correlated to g__ Ruminococcus_1, g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae, and ursodeoxycholate and deoxycholate were positively correlated with g__Ruminococcus_1. Also, N-acetyl-D-glucosamine, nutriadeoxycholate vulpecholate, murideoxycholate, urobilin, and 7a-hydroxy-3-oxo-5b-cholanoate were positively correlated with g__norank_f_Lachnospiraceae, g__norank_f_ Ruminococcaceae, and g__unclassified_f_Ruminococcaceae.7-Hydroxy-3-oxocholanoic acid was positively correlated with g__norank_f_Lachnospiraceae, while hypoxanthine was positively correlated with g__Clostridium_sensu_stricto_1.',31214133),(4418,'mouse','increase','null','low-throughput','IVAS stimulated a dose-dependent increase in glucose transport through glucose transporter GLUT4- and PI3K-dependent mechanisms. IVAS also up regulated GLUT1, GLUT4, and PI3K p85a protein, and increased phosphorylation of Akt.',30183376),(4419,'mouse','increase','null','low-throughput','IVAS stimulated a dose-dependent increase in glucose transport through glucose transporter GLUT4- and PI3K-dependent mechanisms. IVAS also up regulated GLUT1, GLUT4, and PI3K p85a protein, and increased phosphorylation of Akt.',30183376),(4420,'mouse','increase','null','low-throughput','IVAS stimulated a dose-dependent increase in glucose transport through glucose transporter GLUT4- and PI3K-dependent mechanisms. IVAS also up regulated GLUT1, GLUT4, and PI3K p85a protein, and increased phosphorylation of Akt.',30183376),(4425,'mouse','null','D-Glucose','null','R. obeum produces acetate, R. torques produces lactate and acetate and R. gnavus produces acetate and formate on PYG medium.',24553467),(4426,'mouse','null','D-Glucose','null','R. obeum produces acetate, R. torques produces lactate and acetate and R. gnavus produces acetate and formate on PYG medium.',24553467),(4427,'mouse','null','D-Glucose','null','R. obeum produces acetate, R. torques produces lactate and acetate and R. gnavus produces acetate and formate on PYG medium.',24553467),(4428,'mouse','null','D-Glucose','null','R. obeum produces acetate, R. torques produces lactate and acetate and R. gnavus produces acetate and formate on PYG medium.',24553467),(4429,'mouse','null','D-Glucose','null','R. obeum produces acetate, R. torques produces lactate and acetate and R. gnavus produces acetate and formate on PYG medium.',24553467),(4430,'mouse','null','D-Glucose','null','R. obeum produces acetate, R. torques produces lactate and acetate and R. gnavus produces acetate and formate on PYG medium.',24553467),(4431,'mouse','null','D-Glucose','null','Propionate formation from fucose via propanediol has also been described in the human gut anaerobe Roseburia inulinivorans, which produces butyrate rather than propionate when grown on glucose.',24553467),(4432,'mouse','null','L-Fucose','null','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4433,'mouse','null','L-Fucose','null','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4434,'mouse','null','L-Fucose','null','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4435,'mouse','null','Lactic acid','null','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4436,'mouse','null','Lactic acid','null','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4437,'mouse','null','Lactic acid','null','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4438,'mouse','null','Lactic acid','null','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4439,'mouse','null','Lactic acid','null','We tested different Coprococcus strains for their ability to utilise lactate for growth. C. eutactus ATCC 27759 and C. comes ATCC 27758 could not grow on lactate, but C. catus GD/7 was able to grow with 25 mM lactate on YCFA medium containing 30 mM acetate. Propionate became the main product and substantial lactate utilisation also occurred in the presence of 10 mM fructose.',24553467),(4440,'mouse','null','Lactic acid','null','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4441,'mouse','null','Lactic acid','null','We tested different Coprococcus strains for their ability to utilise lactate for growth. C. eutactus ATCC 27759 and C. comes ATCC 27758 could not grow on lactate, but C. catus GD/7 was able to grow with 25 mM lactate on YCFA medium containing 30 mM acetate. Propionate became the main product and substantial lactate utilisation also occurred in the presence of 10 mM fructose.',24553467),(4442,'mouse','null','Lactic acid','null','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4443,'mouse','null','Lactic acid','null','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4444,'mouse','null','Lactic acid','null','We tested different Coprococcus strains for their ability to utilise lactate for growth. C. eutactus ATCC 27759 and C. comes ATCC 27758 could not grow on lactate, but C. catus GD/7 was able to grow with 25 mM lactate on YCFA medium containing 30 mM acetate. Propionate became the main product and substantial lactate utilisation also occurred in the presence of 10 mM fructose.',24553467),(4445,'mouse','null','Lactic acid','null','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4446,'mouse','null','Formate','null','We tested different Coprococcus strains for their ability to utilise lactate for growth. C. eutactus ATCC 27759 and C. comes ATCC 27758 could not grow on lactate, but C. catus GD/7 was able to grow with 25 mM lactate on YCFA medium containing 30 mM acetate. Propionate became the main product and substantial lactate utilisation also occurred in the presence of 10 mM fructose.',24553467),(4447,'mouse','null','Formate','null','We tested different Coprococcus strains for their ability to utilise lactate for growth. C. eutactus ATCC 27759 and C. comes ATCC 27758 could not grow on lactate, but C. catus GD/7 was able to grow with 25 mM lactate on YCFA medium containing 30 mM acetate. Propionate became the main product and substantial lactate utilisation also occurred in the presence of 10 mM fructose.',24553467),(4448,'mouse','null','Acrylate','null','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4449,'mouse','null','Acrylate','null','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4450,'mouse','null','Acrylate','null','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4451,'mouse','null','Butyrate','null','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4452,'mouse','null','Butyrate','null','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4453,'mouse','null','Propanediol','null','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4454,'mouse','null','Propanediol','null','The propanediol pathway has been detected in the phylogenetically distant bacteria Salmonella enterica serovar Typhimurium (Bobik et al., 1999) and R. inulinivorans (Scott et al., 2006), suggesting that it might be present in other gut bacteria.',24553467),(4455,'mouse','null','Succinate','null','The succinate pathway of propionate formation has mainly been studied in bacteria belonging to the class Negativicutes.',24553467),(4456,'mouse','null','Succinate','null','The succinate pathway of propionate formation has mainly been studied in bacteria belonging to the class Negativicutes.',24553467),(4457,'mouse','null','Succinate','null','The succinate pathway of propionate formation has mainly been studied in bacteria belonging to the class Negativicutes.',24553467),(4458,'mouse','null','Succinate','null','The succinate pathway of propionate formation has mainly been studied in bacteria belonging to the class Negativicutes.',24553467),(4459,'mouse','null','Succinate','null','The succinate pathway of propionate formation has mainly been studied in bacteria belonging to the class Negativicutes.',24553467),(4460,'mouse','null','Succinate','null','The succinate pathway of propionate formation has mainly been studied in bacteria belonging to the class Negativicutes.',24553467),(4461,'mouse','null','Succinate','null','The succinate pathway of propionate formation has mainly been studied in bacteria belonging to the class Negativicutes.',24553467),(4494,'mouse','increase','null','low-throughput','succinate augmented the expression of TNF and IL-1b by iMoDCs when used in combination with the TLR3 ligand poly(I:C) or the TLR7 agonist imiquimod.',18820681),(4495,'mouse','increase','null','low-throughput','succinate augmented the expression of TNF and IL-1b by iMoDCs when used in combination with the TLR3 ligand poly(I:C) or the TLR7 agonist imiquimod.',18820681),(4496,'mouse','increase','null','low-throughput','Our results suggest this is due to GF colonocytes adapting to their suboptimal energetic state by undergoing autophagy. Reduced ATP levels activate AMPK, which, in turn, activates p27 to prevent apoptosis and induce autophagy in these cells.',21531334),(4502,'mouse','increase','null','low-throughput','Our results suggest this is due to GF colonocytes adapting to their suboptimal energetic state by undergoing autophagy. Reduced ATP levels activate AMPK, which, in turn, activates p27 to prevent apoptosis and induce autophagy in these cells.',21531334),(4503,'mouse','increase','null','low-throughput','Our results suggest this is due to GF colonocytes adapting to their suboptimal energetic state by undergoing autophagy. Reduced ATP levels activate AMPK, which, in turn, activates p27 to prevent apoptosis and induce autophagy in these cells.',21531334),(4504,'mouse','increase','null','low-throughput','Our results suggest this is due to GF colonocytes adapting to their suboptimal energetic state by undergoing autophagy. Reduced ATP levels activate AMPK, which, in turn, activates p27 to prevent apoptosis and induce autophagy in these cells.',21531334),(4505,'mouse','increase','null','low-throughput','Our results suggest this is due to GF colonocytes adapting to their suboptimal energetic state by undergoing autophagy. Reduced ATP levels activate AMPK, which, in turn, activates p27 to prevent apoptosis and induce autophagy in these cells.',21531334),(4506,'mouse','increase','null','low-throughput','Our results suggest this is due to GF colonocytes adapting to their suboptimal energetic state by undergoing autophagy. Reduced ATP levels activate AMPK, which, in turn, activates p27 to prevent apoptosis and induce autophagy in these cells.',21531334),(4507,'mouse','decrease','null','low-throughput','We also tested whether butyrate inhibits autophagy in GF colonocytes. As expected, it decreases p27 phosphorylation and LC3-II compared to untreated GF colonocytes.',21531334),(4508,'mouse','decrease','null','low-throughput','We also tested whether butyrate inhibits autophagy in GF colonocytes. As expected, it decreases p27 phosphorylation and LC3-II compared to untreated GF colonocytes.',21531334),(4512,'mouse','null','null','null','It has been reported that B. thetaiotaomicron and B. ovatus have the ability to ferment AAA  to produce phenylacetic acid.',28628112),(4513,'mouse','null','null','null','It has been reported that B. thetaiotaomicron and B. ovatus have the ability to ferment AAA  to produce phenylacetic acid.',28628112),(4523,'mouse','increase','Tyrosine','low-throughput','Here we show using untargeted metabolomics that levels of phenyl sulfate, a gut microbiota-derived metabolite, increase with the progression of diabetes in rats overexpressing human uremic toxin transporter SLCO4C1 in the kidney, and are decreased in rats with limited proteinuria.',31015435),(4524,'mouse','increase','Tyrosine','low-throughput','although there was no difference between in the preserved tubular area and fibrotic area at the light microscopic level, the expression levels of representative inflammatory genes, TNF-α (Tnfa) and MCP-1 (Ccl2), the fibrotic gene TGF-α1 (Tgfb1), fibronectin (Fn1), and collagen I (Col1a1) were increased in PS-treated db/db kidney. It was reported that db/db mice develop renal fibrosis.',31015435),(4525,'mouse','increase','Tyrosine','low-throughput','although there was no difference between in the preserved tubular area and fibrotic area at the light microscopic level, the expression levels of representative inflammatory genes, TNF-α (Tnfa) and MCP-1 (Ccl2), the fibrotic gene TGF-α1 (Tgfb1), fibronectin (Fn1), and collagen I (Col1a1) were increased in PS-treated db/db kidney. It was reported that db/db mice develop renal fibrosis.',31015435),(4526,'mouse','increase','Tyrosine','low-throughput','although there was no difference between in the preserved tubular area and fibrotic area at the light microscopic level, the expression levels of representative inflammatory genes, TNF-α (Tnfa) and MCP-1 (Ccl2), the fibrotic gene TGF-α1 (Tgfb1), fibronectin (Fn1), and collagen I (Col1a1) were increased in PS-treated db/db kidney. It was reported that db/db mice develop renal fibrosis.',31015435),(4527,'mouse','increase','Tyrosine','low-throughput','although there was no difference between in the preserved tubular area and fibrotic area at the light microscopic level, the expression levels of representative inflammatory genes, TNF-α (Tnfa) and MCP-1 (Ccl2), the fibrotic gene TGF-α1 (Tgfb1), fibronectin (Fn1), and collagen I (Col1a1) were increased in PS-treated db/db kidney. It was reported that db/db mice develop renal fibrosis.',31015435),(4528,'mouse','increase','Tyrosine','low-throughput','although there was no difference between in the preserved tubular area and fibrotic area at the light microscopic level, the expression levels of representative inflammatory genes, TNF-α (Tnfa) and MCP-1 (Ccl2), the fibrotic gene TGF-α1 (Tgfb1), fibronectin (Fn1), and collagen I (Col1a1) were increased in PS-treated db/db kidney. It was reported that db/db mice develop renal fibrosis.',31015435),(4530,'mouse','null','Daidzein','null','Equol is a metabolite of daidzein that is produced by intestinal microbiota.The rod-shaped and Gram-positive anaerobic equol-producing intestinal bacterium Slackia TM-30 was isolated from healthy human faeces.',26455424),(4534,'mouse','increase','null','low-throughput','Live B. dentium-monoassociated mice exhibited increased colonic expression of goblet cell markers Krüppel-like factor 4 (Klf4), Trefoil factor 3 (Tff3), Relm-β, Muc2, and several glycosyltransferases compared to both heat-killed B. dentium and germfree counterparts.',31213556),(4540,'mouse','increase','null','low-throughput','We examined a number of known autophagy markers by qPCR and found that B. dentium colonization was associated with increased expression of ATG16L, ATG5, Beclin1, LAMP2, and LC3.',31213556),(4541,'mouse','increase','null','low-throughput','We examined a number of known autophagy markers by qPCR and found that B. dentium colonization was associated with increased expression of ATG16L, ATG5, Beclin1, LAMP2, and LC3.',31213556),(4542,'mouse','increase','null','low-throughput','We examined a number of known autophagy markers by qPCR and found that B. dentium colonization was associated with increased expression of ATG16L, ATG5, Beclin1, LAMP2, and LC3.',31213556),(4543,'mouse','increase','null','low-throughput','We examined a number of known autophagy markers by qPCR and found that B. dentium colonization was associated with increased expression of ATG16L, ATG5, Beclin1, LAMP2, and LC3.',31213556),(4544,'mouse','increase','null','low-throughput','We examined a number of known autophagy markers by qPCR and found that B. dentium colonization was associated with increased expression of ATG16L, ATG5, Beclin1, LAMP2, and LC3.',31213556),(4545,'mouse','increase','null','low-throughput','B. dentium can secrete products, such as acetate, that are capable of increasing MUC2 synthesis.',31213556),(4549,'mouse','null','null','null','A specific human-associated gut microbe, Clostridium orbiscindens,produced DAT and rescued antibiotic-treated influenza-infected mice.Consistent with our hypothesis that DAT protects from influenza infection by enhancing type I IFN signaling before infection.',28774928),(4550,'mouse','increase','null','low-throughput','Oxidative stress and inflammation in the kidney have been demonstrated to play a pivotal role in the initiation and progression of obesity-related renal injury [20e22]. To further determine the possible molecular mechanisms by which elevated TMAO levels induce renal interstitial fibrosis and dysfunction, we measured expression of NOX-4 (a constitutive active enzyme responsible for the production of reactive oxygen species) and proinflammatory cytokines in the kidney. As presented in Fig. 4, expression of NOX-4(Fig. 4A and B) and pro-inflammatory cytokines TNF-a (Fig. 4A and C) and IL-1 b (Fig. 4A and D) was significantly increased in mice fed a HFD, compared with mice fed a LFD, but was reduced by DMB treatment. Expression of NOX-4, TNF-a and IL-1 b in mice fed a LFD was not affected by DMB treatment.',28942145),(4551,'mouse','increase','null','low-throughput','Oxidative stress and inflammation in the kidney have been demonstrated to play a pivotal role in the initiation and progression of obesity-related renal injury [20e22]. To further determine the possible molecular mechanisms by which elevated TMAO levels induce renal interstitial fibrosis and dysfunction, we measured expression of NOX-4 (a constitutive active enzyme responsible for the production of reactive oxygen species) and proinflammatory cytokines in the kidney. As presented in Fig. 4, expression of NOX-4(Fig. 4A and B) and pro-inflammatory cytokines TNF-a (Fig. 4A and C) and IL-1 b (Fig. 4A and D) was significantly increased in mice fed a HFD, compared with mice fed a LFD, but was reduced by DMB treatment. Expression of NOX-4, TNF-a and IL-1 b in mice fed a LFD was not affected by DMB treatment.',28942145),(4552,'mouse','increase','null','low-throughput','Oxidative stress and inflammation in the kidney have been demonstrated to play a pivotal role in the initiation and progression of obesity-related renal injury [20e22]. To further determine the possible molecular mechanisms by which elevated TMAO levels induce renal interstitial fibrosis and dysfunction, we measured expression of NOX-4 (a constitutive active enzyme responsible for the production of reactive oxygen species) and proinflammatory cytokines in the kidney. As presented in Fig. 4, expression of NOX-4(Fig. 4A and B) and pro-inflammatory cytokines TNF-a (Fig. 4A and C) and IL-1 b (Fig. 4A and D) was significantly increased in mice fed a HFD, compared with mice fed a LFD, but was reduced by DMB treatment. Expression of NOX-4, TNF-a and IL-1 b in mice fed a LFD was not affected by DMB treatment.',28942145),(4553,'mouse','increase','null','low-throughput','Butyrate or niacin induces IL-18 expression in colonic epithelium in a Gpr109a-dependent manner.',24412617),(4554,'mouse','increase','null','low-throughput','Butyrate or niacin induces IL-18 expression in colonic epithelium in a Gpr109a-dependent manner.',24412617),(4555,'mouse','decrease','null','low-throughput','Our data indicate that WAT miR-181 expression is controlled by the relative abundance of microbiota-derived metabolites acting as positive and negative regulators of miR-181 expression and that  indole and indole-derived metabolites  including indoxyl sulfate  are key negative regulators of miR-181 expression in WAT.',31189717),(4556,'mouse','decrease','null','low-throughput','Our data indicate that WAT miR-181 expression is controlled by the relative abundance of microbiota-derived metabolites acting as positive and negative regulators of miR-181 expression and that  indole and indole-derived metabolites  including indoxyl sulfate  are key negative regulators of miR-181 expression in WAT.',31189717),(4560,'mouse','increase','null','low-throughput','D8 up-regulates IL-22 expression ex vivo and in vivo.',29459771),(4561,'mouse','null','null','null','Thus, exposure to S. cerevisiae in the gut led to enhanced purine metabolism and promoted increased uric acid.',28275154),(4562,'mouse','null','null','null','Thus, exposure to S. cerevisiae in the gut led to enhanced purine metabolism and promoted increased uric acid.',28275154),(4563,'mouse','increase','null','low-throughput','Butyrate promotes the increased expression of IFN-γ and granzyme B in CTLs and Tc17 cells.',30258117),(4564,'mouse','increase','null','low-throughput','Butyrate promotes the increased expression of IFN-γ and granzyme B in CTLs and Tc17 cells.',30258117),(4568,'mouse','null','Sennoside','null','We demonstrated that intestinal peristalsis was promoted by rheinanthrone produced by hydrolysis of sennoside by strain LKM512 and LKM10070.',22384059),(4569,'mouse','null','Sennoside','null','We demonstrated that intestinal peristalsis was promoted by rheinanthrone produced by hydrolysis of sennoside by strain LKM512 and LKM10070.',22384059),(4575,'mouse','null','null','null','Fecal butyrate levels significantly decreased in the DSS mice as compared to the control mice, but increased significantly in the DSS plus TN-3 mice as compared to the DSS mice.',27438072),(4576,'mouse','decrease','null','low-throughput','Indole, the most prevalent metabolite of tryptophan, is produced by a wide range of bacterial species, including those belonging to the genera Escherichia, Bacteroides, and ClostridiumProlonged exposure to indole inhibits ATP production and thus GLP-1 secretion.',25456122),(4577,'mouse','decrease','null','low-throughput','Indole, the most prevalent metabolite of tryptophan, is produced by a wide range of bacterial species, including those belonging to the genera Escherichia, Bacteroides, and ClostridiumProlonged exposure to indole inhibits ATP production and thus GLP-1 secretion.',25456122),(4578,'mouse','decrease','null','low-throughput','Indole, the most prevalent metabolite of tryptophan, is produced by a wide range of bacterial species, including those belonging to the genera Escherichia, Bacteroides, and ClostridiumProlonged exposure to indole inhibits ATP production and thus GLP-1 secretion.',25456122),(4579,'mouse','null','null','null','TL133 also increasing the caecal concentration of acetate, propionate and butyrate.',17349084),(4580,'mouse','null','null','null','TL134 also increasing the caecal concentration of acetate, propionate and butyrate.',17349084),(4581,'mouse','null','null','null','TL135 also increasing the caecal concentration of acetate, propionate and butyrate.',17349084),(4582,'mouse','null','L-Fucose','null','Accordingly, the mouse model provided L-fucose, enabling B. thetaiotaomicron to synthesize L-fucose containing polysaccharides.',23022739),(4583,'mouse','null','L-Fucose','null','Accordingly, the mouse model provided L-fucose, enabling B. thetaiotaomicron to synthesize L-fucose containing polysaccharides.',23022739),(4584,'mouse','null','L-Fucose','null','Accordingly, the mouse model provided L-fucose, enabling B. thetaiotaomicron to synthesize L-fucose containing polysaccharides.',23022739),(4585,'mouse','null','null','null','We were able to identify a few metabolites specific to the humanized Salmonella-infected group, including two metabolites in the glutathione pathway. In particular, we identified glutathione disulfide and glutathione cysteine disulfide in higher abundance in humanized Salmonella-infected mice. Salmonella infection triggers vast amounts of oxidative stress, and glutathione metabolism is important for protection against oxidative stress.',30401779),(4586,'mouse','null','null','null','We were able to identify a few metabolites specific to the humanized Salmonella-infected group, including two metabolites in the glutathione pathway. In particular, we identified glutathione disulfide and glutathione cysteine disulfide in higher abundance in humanized Salmonella-infected mice. Salmonella infection triggers vast amounts of oxidative stress, and glutathione metabolism is important for protection against oxidative stress.',30401779),(4587,'mouse','increase','null','low-throughput','SCFAs binding to GPR43 on colonic epithelial cells stimulates K þ efflux and hyperpolarization,which lead to NLRP3 inflammasome activation.',25828455),(4588,'mouse','increase','null','low-throughput','NaB corrected the HFD-induced gut microbiota imbalance in mice, while it considerably elevated the abundances of the beneficial bacteria Christensenellaceae, Blautia and Lactobacillus . These bacteria can produce butyric acid in what seems like a virtuous circle. And butyrate restored HFD induced intestinal mucosa damage, increased the expression of ZO-1 in small intestine, further decreased the levels of gut endotoxin in serum and liver compared with HF group.',28104981),(4589,'mouse','increase','null','low-throughput','NaB corrected the HFD-induced gut microbiota imbalance in mice, while it considerably elevated the abundances of the beneficial bacteria Christensenellaceae, Blautia and Lactobacillus . These bacteria can produce butyric acid in what seems like a virtuous circle. And butyrate restored HFD induced intestinal mucosa damage, increased the expression of ZO-1 in small intestine, further decreased the levels of gut endotoxin in serum and liver compared with HF group.',28104981),(4590,'mouse','increase','null','low-throughput','NaB corrected the HFD-induced gut microbiota imbalance in mice, while it considerably elevated the abundances of the beneficial bacteria Christensenellaceae, Blautia and Lactobacillus . These bacteria can produce butyric acid in what seems like a virtuous circle. And butyrate restored HFD induced intestinal mucosa damage, increased the expression of ZO-1 in small intestine, further decreased the levels of gut endotoxin in serum and liver compared with HF group.',28104981),(4591,'mouse','increase','null','low-throughput','Taken together, these experiments suggest that RYGB restores intestinal NLRP6 inflammasome expression by increasing taurine and decreasing histamine in the gut.',31602628),(4592,'mouse','decrease','null','low-throughput','Taken together, these experiments suggest that RYGB restores intestinal NLRP6 inflammasome expression by increasing taurine and decreasing histamine in the gut.',31602628),(4593,'mouse','decrease','Lipopolysaccharide','low-throughput','Equol inhibited the lipopolysaccharide (LPS)-induced TLR4 activation, MAPK activation,NF-kB-mediated transcription of inflammatory mediators, production of nitric oxide (NO), release of prostaglandin E2 (PGE-2), secretion of tumor necrosis factor-α (TNF-α) and interleukin 6 (IL-6),in Lipopolysaccharide (LPS)-activated murine microglia cells.',28264445),(4594,'mouse','decrease','Lipopolysaccharide','low-throughput','Equol inhibited the lipopolysaccharide (LPS)-induced TLR4 activation, MAPK activation,NF-kB-mediated transcription of inflammatory mediators, production of nitric oxide (NO), release of prostaglandin E2 (PGE-2), secretion of tumor necrosis factor-α (TNF-α) and interleukin 6 (IL-6),in Lipopolysaccharide (LPS)-activated murine microglia cells.',28264445),(4595,'mouse','decrease','Lipopolysaccharide','low-throughput','Equol inhibited the lipopolysaccharide (LPS)-induced TLR4 activation, MAPK activation,NF-kB-mediated transcription of inflammatory mediators, production of nitric oxide (NO), release of prostaglandin E2 (PGE-2), secretion of tumor necrosis factor-α (TNF-α) and interleukin 6 (IL-6),in Lipopolysaccharide (LPS)-activated murine microglia cells.',28264445),(4596,'mouse','decrease','Lipopolysaccharide','low-throughput','Equol inhibited the lipopolysaccharide (LPS)-induced TLR4 activation, MAPK activation,NF-kB-mediated transcription of inflammatory mediators, production of nitric oxide (NO), release of prostaglandin E2 (PGE-2), secretion of tumor necrosis factor-α (TNF-α) and interleukin 6 (IL-6),in Lipopolysaccharide (LPS)-activated murine microglia cells.',28264445),(4597,'mouse','increase','null','low-throughput','The helminth Nippostrongylus brasiliensis and a tritrichomonad protist both secreted succinate as a metabolite. In vivo sensing of the tritrichomonad required SUCNR1, whereas N. brasiliensis was SUCNR1 independent. These findings define a paradigm wherein tuft cells monitor microbial metabolites to initiate type 2 immunity and suggest the existence of other sensing pathways triggering the response to helminths.',30021144),(4598,'mouse','increase','null','low-throughput','The helminth Nippostrongylus brasiliensis and a tritrichomonad protist both secreted succinate as a metabolite. In vivo sensing of the tritrichomonad required SUCNR1, whereas N. brasiliensis was SUCNR1 independent. These findings define a paradigm wherein tuft cells monitor microbial metabolites to initiate type 2 immunity and suggest the existence of other sensing pathways triggering the response to helminths.',30021144),(4599,'mouse','increase','omega-6 Fatty Acids','low-throughput','Here, we demonstrated that gut microbiota exerted protective effects against HFD-induced obesity and improved glucose homeostasis associated with several mechanisms involving gut microbial metabolites of dietary PUFAs: (1) gut microbiota converted LA to HYA, reducing the adipose inflammation otherwise induced by excessive dietary LA via the AA cascade; (2) HYA activated GPR40 and GPR120 and promoted GLP-1 secretion; and (3) HYA suppressed lipid absorption by promoting intestinal peristalsis via EP3.',31488836),(4600,'mouse','increase','null','low-throughput','Our study shows that MAM from F. prausnitzii can restore the intestinal barrier structure and function in DM conditions via the regulation of the tight junction pathway and ZO-1 expression.Collectively, our data reveal that MAM protein upregulates ZO-1 expression in intestinal epithelial cells.',31503404),(4601,'mouse','null','Melamine','null','We further demonstrated that melamine is converted to cyanuric acid in vitro by bacteria cultured from normal rat feces;Klebsiella was subsequently identified in fecal samples by 16S ribosomal DNA sequencing. In culture, Klebsiella terrigena was shown to convert melamine to cyanuric acid directly.',23408055),(4602,'mouse','null','null','null','We demonstrate that commensal Bacteroides species limit S. Typhimurium infection in the distalgut. We find that Bacteroides spp. production of the short-chain fatty acid (SCFA) metabolic byproduct propionate directly inhibits S. Typhimurium growth in vitro by disrupting intracellular pH homeostasis and limits distal intestinal expansion and fecal shedding in vivo.',30057174),(4603,'mouse','null','null','null','We screened a collection of 79 sequenced human intestinal isolates encompassing the major phyla found in the human gut and identified nine strains capable of producing TMA from choline in vitro.',25784704),(4604,'mouse','null','null','null','We screened a collection of 79 sequenced human intestinal isolates encompassing the major phyla found in the human gut and identified nine strains capable of producing TMA from choline in vitro.',25784704),(4605,'mouse','null','null','null','We screened a collection of 79 sequenced human intestinal isolates encompassing the major phyla found in the human gut and identified nine strains capable of producing TMA from choline in vitro.',25784704),(4606,'mouse','null','null','null','We screened a collection of 79 sequenced human intestinal isolates encompassing the major phyla found in the human gut and identified nine strains capable of producing TMA from choline in vitro.',25784704),(4607,'mouse','null','null','null','We screened a collection of 79 sequenced human intestinal isolates encompassing the major phyla found in the human gut and identified nine strains capable of producing TMA from choline in vitro.',25784704),(4608,'mouse','null','null','null','We screened a collection of 79 sequenced human intestinal isolates encompassing the major phyla found in the human gut and identified nine strains capable of producing TMA from choline in vitro.',25784704),(4609,'mouse','null','null','null','We screened a collection of 79 sequenced human intestinal isolates encompassing the major phyla found in the human gut and identified nine strains capable of producing TMA from choline in vitro.',25784704),(4610,'mouse','null','null','null','We screened a collection of 79 sequenced human intestinal isolates encompassing the major phyla found in the human gut and identified nine strains capable of producing TMA from choline in vitro.',25784704),(4611,'mouse','null','null','null','We screened a collection of 79 sequenced human intestinal isolates encompassing the major phyla found in the human gut and identified nine strains capable of producing TMA from choline in vitro.',25784704),(4612,'mouse','decrease','null','low-throughput','Faecalibacterium prausnitzii produces butyrate to decrease c-Myc-related metabolism and Th17 differentiation by inhibiting histone deacetylase 3.',30809639),(4613,'mouse','null','null','null','However, PPD  synergistically enhanced APAP induced hepatic injury in vivo and in vitro. Finally, we  found Saccharomyces cerevisiae, which could reduce intestinal PPD levels, was able  to markedly alleviate APAP induced liver damage at ZT12.',29524531),(4616,'mouse','null','null','null','Accordingly, dietary succinate improved glucose and insulin tolerance in wild-type mice, but those effects were absent in mice deficient in IGN. Conventional mice colonized with the succinate producer Prevotella copri exhibited metabolic benefits, which could be related to succinate-activated IGN. Thus, microbiota-produced succinate is a previously unsuspected bacterial metabolite improving glycemic control through activation of IGN.',27411015),(4617,'mouse','increase','null','low-throughput','Butyrate can induce AMPK activation and GLUT4 expression in the adipose tissue, improving cardiovascular disease (CVD)-related metabolic disorder, resisting HFD-induced gut microbiome dysbiosis, and promoting resolvin E1 and lipoxin biosynthesis. Oral supplement of the natural fermentation product butyrate can be a potential strategy for preventing CVD.',31607907),(4618,'mouse','increase','null','low-throughput','Butyrate can induce AMPK activation and GLUT4 expression in the adipose tissue, improving cardiovascular disease (CVD)-related metabolic disorder, resisting HFD-induced gut microbiome dysbiosis, and promoting resolvin E1 and lipoxin biosynthesis. Oral supplement of the natural fermentation product butyrate can be a potential strategy for preventing CVD.',31607907),(4619,'mouse','decrease','null','low-throughput','Colonic cDNA concentrations of IL-1b and TNF-a were significantly down regulated in DSS-treated E. durans-fed mice but not in control or DSS-treated L.delbrueckii- fed mice.',17349085),(4620,'mouse','decrease','null','low-throughput','Colonic cDNA concentrations of IL-1b and TNF-a were significantly down regulated in DSS-treated E. durans-fed mice but not in control or DSS-treated L.delbrueckii- fed mice.',17349085),(4621,'mouse','increase','null','low-throughput','we demonstrate here that α-defensin 5 expression can also be regulated by microbial metabolites. Among these, lactate appeared to significantly suppress α-defensin 5 gene transcription.',27760493),(4622,'mouse','null','null','null','We report that the human gut bacterium Bacteroides thetaiotaomicron relies on the stringent response, an intracellular signaling pathway that allocates resources away from growth, to survive carbon starvation and persist in the gut. Genome-scale transcriptomics, 13C-labeling, and metabolomics analyses reveal that B. thetaiotaomicron uses the alarmone (p)ppGpp to repress multiple biosynthetic pathways and upregulate tricarboxylic acid (TCA) cycle genes in these conditions.',30008292),(4623,'mouse','increase','null','low-throughput','Lactobacilli Induce IL-22 via Iald.',23973224),(4624,'mouse','null','Ferulic acid','null','Taxonomic analysis of the cecal microbiota showed an increased abundance of Bacteroidetes, along with a reduction in the population of Firmicutes with HMPA supplementation (Figure 4B). Notably, HMPA intervention induced a drastic increase in the population of Actinobacteria, while the population of Deferribacteres in the cecal microbiota of HMPA-supplemented HFD-fed mice were decreased (Figure 4B). These results were similar to those observed with HMCA supplementation. Hierarchical clustering of individual families also confirmed the effect of HMPA intake on gut microbiota (Figure 4C). The HMPA intervention significantly increased the Bacteroidales S24-7 group and Coriobacteriaceae, which are families within the Bacteroidetes and Actinobacteria, respectively (Figure 4C). These families were also significantly increased in HMCA-supplemented HFD-fed mice (Figure 2D). Meanwhile, significant increases in Prevotellaceae and decreases in Deferribacteraceae were observed in HMPA-supplemented HFD-fed mice, but not in HMCA-supplemented HFD-fed mice (Figures 2D and 4C). These findings demonstrated that HMPA intake changed the composition of mouse gut microbiota.',31075850),(4625,'mouse','null','Ferulic acid','null','Taxonomic analysis of the cecal microbiota showed an increased abundance of Bacteroidetes, along with a reduction in the population of Firmicutes with HMPA supplementation (Figure 4B). Notably, HMPA intervention induced a drastic increase in the population of Actinobacteria, while the population of Deferribacteres in the cecal microbiota of HMPA-supplemented HFD-fed mice were decreased (Figure 4B). These results were similar to those observed with HMCA supplementation. Hierarchical clustering of individual families also confirmed the effect of HMPA intake on gut microbiota (Figure 4C). The HMPA intervention significantly increased the Bacteroidales S24-7 group and Coriobacteriaceae, which are families within the Bacteroidetes and Actinobacteria, respectively (Figure 4C). These families were also significantly increased in HMCA-supplemented HFD-fed mice (Figure 2D). Meanwhile, significant increases in Prevotellaceae and decreases in Deferribacteraceae were observed in HMPA-supplemented HFD-fed mice, but not in HMCA-supplemented HFD-fed mice (Figures 2D and 4C). These findings demonstrated that HMPA intake changed the composition of mouse gut microbiota.',31075850),(4626,'mouse','null','Ferulic acid','null','Taxonomic analysis of the cecal microbiota showed an increased abundance of Bacteroidetes, along with a reduction in the population of Firmicutes with HMPA supplementation (Figure 4B). Notably, HMPA intervention induced a drastic increase in the population of Actinobacteria, while the population of Deferribacteres in the cecal microbiota of HMPA-supplemented HFD-fed mice were decreased (Figure 4B). These results were similar to those observed with HMCA supplementation. Hierarchical clustering of individual families also confirmed the effect of HMPA intake on gut microbiota (Figure 4C). The HMPA intervention significantly increased the Bacteroidales S24-7 group and Coriobacteriaceae, which are families within the Bacteroidetes and Actinobacteria, respectively (Figure 4C). These families were also significantly increased in HMCA-supplemented HFD-fed mice (Figure 2D). Meanwhile, significant increases in Prevotellaceae and decreases in Deferribacteraceae were observed in HMPA-supplemented HFD-fed mice, but not in HMCA-supplemented HFD-fed mice (Figures 2D and 4C). These findings demonstrated that HMPA intake changed the composition of mouse gut microbiota.',31075850),(4627,'mouse','null','Ferulic acid','null','Taxonomic analysis of the cecal microbiota showed an increased abundance of Bacteroidetes, along with a reduction in the population of Firmicutes with HMPA supplementation (Figure 4B). Notably, HMPA intervention induced a drastic increase in the population of Actinobacteria, while the population of Deferribacteres in the cecal microbiota of HMPA-supplemented HFD-fed mice were decreased (Figure 4B). These results were similar to those observed with HMCA supplementation. Hierarchical clustering of individual families also confirmed the effect of HMPA intake on gut microbiota (Figure 4C). The HMPA intervention significantly increased the Bacteroidales S24-7 group and Coriobacteriaceae, which are families within the Bacteroidetes and Actinobacteria, respectively (Figure 4C). These families were also significantly increased in HMCA-supplemented HFD-fed mice (Figure 2D). Meanwhile, significant increases in Prevotellaceae and decreases in Deferribacteraceae were observed in HMPA-supplemented HFD-fed mice, but not in HMCA-supplemented HFD-fed mice (Figures 2D and 4C). These findings demonstrated that HMPA intake changed the composition of mouse gut microbiota.',31075850),(4628,'mouse','null','Ferulic acid','null','Taxonomic analysis of the cecal microbiota showed an increased abundance of Bacteroidetes, along with a reduction in the population of Firmicutes with HMPA supplementation (Figure 4B). Notably, HMPA intervention induced a drastic increase in the population of Actinobacteria, while the population of Deferribacteres in the cecal microbiota of HMPA-supplemented HFD-fed mice were decreased (Figure 4B). These results were similar to those observed with HMCA supplementation. Hierarchical clustering of individual families also confirmed the effect of HMPA intake on gut microbiota (Figure 4C). The HMPA intervention significantly increased the Bacteroidales S24-7 group and Coriobacteriaceae, which are families within the Bacteroidetes and Actinobacteria, respectively (Figure 4C). These families were also significantly increased in HMCA-supplemented HFD-fed mice (Figure 2D). Meanwhile, significant increases in Prevotellaceae and decreases in Deferribacteraceae were observed in HMPA-supplemented HFD-fed mice, but not in HMCA-supplemented HFD-fed mice (Figures 2D and 4C). These findings demonstrated that HMPA intake changed the composition of mouse gut microbiota.',31075850),(4629,'mouse','null','Ferulic acid','null','Taxonomic analysis of the cecal microbiota showed an increased abundance of Bacteroidetes, along with a reduction in the population of Firmicutes with HMPA supplementation (Figure 4B). Notably, HMPA intervention induced a drastic increase in the population of Actinobacteria, while the population of Deferribacteres in the cecal microbiota of HMPA-supplemented HFD-fed mice were decreased (Figure 4B). These results were similar to those observed with HMCA supplementation. Hierarchical clustering of individual families also confirmed the effect of HMPA intake on gut microbiota (Figure 4C). The HMPA intervention significantly increased the Bacteroidales S24-7 group and Coriobacteriaceae, which are families within the Bacteroidetes and Actinobacteria, respectively (Figure 4C). These families were also significantly increased in HMCA-supplemented HFD-fed mice (Figure 2D). Meanwhile, significant increases in Prevotellaceae and decreases in Deferribacteraceae were observed in HMPA-supplemented HFD-fed mice, but not in HMCA-supplemented HFD-fed mice (Figures 2D and 4C). These findings demonstrated that HMPA intake changed the composition of mouse gut microbiota.',31075850),(4630,'mouse','null','Ferulic acid','null','Taxonomic analysis of the cecal microbiota showed an increased abundance of Bacteroidetes, along with a reduction in the population of Firmicutes with HMPA supplementation (Figure 4B). Notably, HMPA intervention induced a drastic increase in the population of Actinobacteria, while the population of Deferribacteres in the cecal microbiota of HMPA-supplemented HFD-fed mice were decreased (Figure 4B). These results were similar to those observed with HMCA supplementation. Hierarchical clustering of individual families also confirmed the effect of HMPA intake on gut microbiota (Figure 4C). The HMPA intervention significantly increased the Bacteroidales S24-7 group and Coriobacteriaceae, which are families within the Bacteroidetes and Actinobacteria, respectively (Figure 4C). These families were also significantly increased in HMCA-supplemented HFD-fed mice (Figure 2D). Meanwhile, significant increases in Prevotellaceae and decreases in Deferribacteraceae were observed in HMPA-supplemented HFD-fed mice, but not in HMCA-supplemented HFD-fed mice (Figures 2D and 4C). These findings demonstrated that HMPA intake changed the composition of mouse gut microbiota.',31075850),(4635,'mouse','increase','null','low-throughput','To this end, DCA feeding was performed in HFD-fed mice treated with vancomycin (Figure 2(a,b)). Intriguingly, DCA feeding alone in VCM-treated HFD-fed mice was observed to induce substantial proinflammatory macrophages (iNOS+) infiltration and increase the expression of multiple proinflammatory factors (IL-6, TNF-α, IL-1β, iNOS) in the colon tissue (Figure 2(c–f)), mimicking the inflammatory intestinal damage induced by HFD.',33006494),(4636,'mouse','increase','null','low-throughput','To this end, DCA feeding was performed in HFD-fed mice treated with vancomycin (Figure 2(a,b)). Intriguingly, DCA feeding alone in VCM-treated HFD-fed mice was observed to induce substantial proinflammatory macrophages (iNOS+) infiltration and increase the expression of multiple proinflammatory factors (IL-6, TNF-α, IL-1β, iNOS) in the colon tissue (Figure 2(c–f)), mimicking the inflammatory intestinal damage induced by HFD.',33006494),(4637,'mouse','increase','null','low-throughput','To this end, DCA feeding was performed in HFD-fed mice treated with vancomycin (Figure 2(a,b)). Intriguingly, DCA feeding alone in VCM-treated HFD-fed mice was observed to induce substantial proinflammatory macrophages (iNOS+) infiltration and increase the expression of multiple proinflammatory factors (IL-6, TNF-α, IL-1β, iNOS) in the colon tissue (Figure 2(c–f)), mimicking the inflammatory intestinal damage induced by HFD.',33006494),(4638,'mouse','increase','null','low-throughput','To this end, DCA feeding was performed in HFD-fed mice treated with vancomycin (Figure 2(a,b)). Intriguingly, DCA feeding alone in VCM-treated HFD-fed mice was observed to induce substantial proinflammatory macrophages (iNOS+) infiltration and increase the expression of multiple proinflammatory factors (IL-6, TNF-α, IL-1β, iNOS) in the colon tissue (Figure 2(c–f)), mimicking the inflammatory intestinal damage induced by HFD.',33006494),(4639,'mouse','increase','null','low-throughput','Indeed, DCA stimulation could induce potent increase of mRNA expression level of M2-mAchR in macrophages, and selective inhibition of M2-mAchR significantly reduced theproduction of TNF-α, NO and IL-6 triggered by DCA (Figure 4(d,e) and Supplementary Figure 4c).',33006494),(4640,'mouse','increase','null','low-throughput','Toll-like receptor 2 (TLR2) is up-regulated upon DCA stimulation and involved in DCA-induced macrophage polarization.',33006494),(4641,'mouse','increase','null','low-throughput','Having detected (ex vivo from mouse cecal contents and human stool) and identified 3-methyl-2-oxindole and 5-hydroxyindole-3-acetic acid, 3-indoleacrylic acid and indole-3 carboxylic acid as novel in vitro AHR activators.',32783770),(4642,'mouse','increase','null','low-throughput','Having detected (ex vivo from mouse cecal contents and human stool) and identified 3-methyl-2-oxindole and 5-hydroxyindole-3-acetic acid, 3-indoleacrylic acid and indole-3 carboxylic acid as novel in vitro AHR activators.',32783770),(4643,'mouse','increase','null','low-throughput','Having detected (ex vivo from mouse cecal contents and human stool) and identified 3-methyl-2-oxindole and 5-hydroxyindole-3-acetic acid, 3-indoleacrylic acid and indole-3 carboxylic acid as novel in vitro AHR activators.',32783770),(4644,'mouse','increase','null','low-throughput','Having detected (ex vivo from mouse cecal contents and human stool) and identified 3-methyl-2-oxindole and 5-hydroxyindole-3-acetic acid, 3-indoleacrylic acid and indole-3 carboxylic acid as novel in vitro AHR activators.',32783770),(4645,'mouse','increase','null','low-throughput','At the concentrations tested, indole, 2-oxindole, kynurenic acid and indole-3-acetic acid elicited significant Cyp1a1 induction relative to vehicle (~5-fold, p < .01). 2, 8-dihydroxyquinoline induced Cyp1a1 to a lesser yet significant extent (1.3-fold, p < .01). None of the remaining tryptophan metabolites elicited significant Cyp1a1 inducibility.',32783770),(4646,'mouse','increase','null','low-throughput','At the concentrations tested, indole, 2-oxindole, kynurenic acid and indole-3-acetic acid elicited significant Cyp1a1 induction relative to vehicle (~5-fold, p < .01). 2, 8-dihydroxyquinoline induced Cyp1a1 to a lesser yet significant extent (1.3-fold, p < .01). None of the remaining tryptophan metabolites elicited significant Cyp1a1 inducibility.',32783770),(4647,'mouse','increase','null','low-throughput','At the concentrations tested, indole, 2-oxindole, kynurenic acid and indole-3-acetic acid elicited significant Cyp1a1 induction relative to vehicle (~5-fold, p < .01). 2, 8-dihydroxyquinoline induced Cyp1a1 to a lesser yet significant extent (1.3-fold, p < .01). None of the remaining tryptophan metabolites elicited significant Cyp1a1 inducibility.',32783770),(4648,'mouse','increase','null','low-throughput','At the concentrations tested, indole, 2-oxindole, kynurenic acid and indole-3-acetic acid elicited significant Cyp1a1 induction relative to vehicle (~5-fold, p < .01). 2, 8-dihydroxyquinoline induced Cyp1a1 to a lesser yet significant extent (1.3-fold, p < .01). None of the remaining tryptophan metabolites elicited significant Cyp1a1 inducibility.',32783770),(4649,'mouse','increase','null','low-throughput','At the concentrations tested, indole, 2-oxindole, kynurenic acid and indole-3-acetic acid elicited significant Cyp1a1 induction relative to vehicle (~5-fold, p < .01). 2, 8-dihydroxyquinoline induced Cyp1a1 to a lesser yet significant extent (1.3-fold, p < .01). None of the remaining tryptophan metabolites elicited significant Cyp1a1 inducibility.',32783770),(4650,'mouse','increase','null','low-throughput','Furthermore, TGP failed to blunt the increased of colonic IL-1β in colitis mice receiving ILA (Fig. 6E).',33002829),(4651,'mouse','null','Tryptophan','null','We have shown that breast milk tryptophan and Bifidobacterium infantis result in a metabolite (indole-3-lactic acid) response, which is anti-inflammatory via inhibition of the aryl hydrocarbon receptor transcription factor which stimulates an IL-8 response.',32229728),(4656,'mouse','increase','null','low-throughput','Duodenal murine enteroids were exposed to 1 and 5 mM SCFAs for 24 h. Expression of stem cell marker leucine-rich repeat-containing G-protein coupled receptor 5 (Lgr5) was increased more than 2-fold in murine enteroids treated with 5 mM ACET (P < 0.05). Conversely, enteroids treated with 5 mM BUT, or PROP had greater than 5-fold decreases in Lgr5 compared to CON (P < 0.05; Fig 2b). There was no difference in Lgr5 expression in murine enteroids treated with 1 mM SCFAs (P > 0.05; Fig 2a).',32240190),(4657,'mouse','decrease','null','low-throughput','Duodenal murine enteroids were exposed to 1 and 5 mM SCFAs for 24 h. Expression of stem cell marker leucine-rich repeat-containing G-protein coupled receptor 5 (Lgr5) was increased more than 2-fold in murine enteroids treated with 5 mM ACET (P < 0.05). Conversely, enteroids treated with 5 mM BUT, or PROP had greater than 5-fold decreases in Lgr5 compared to CON (P < 0.05; Fig 2b). There was no difference in Lgr5 expression in murine enteroids treated with 1 mM SCFAs (P > 0.05; Fig 2a).',32240190),(4658,'mouse','decrease','null','low-throughput','Duodenal murine enteroids were exposed to 1 and 5 mM SCFAs for 24 h. Expression of stem cell marker leucine-rich repeat-containing G-protein coupled receptor 5 (Lgr5) was increased more than 2-fold in murine enteroids treated with 5 mM ACET (P < 0.05). Conversely, enteroids treated with 5 mM BUT, or PROP had greater than 5-fold decreases in Lgr5 compared to CON (P < 0.05; Fig 2b). There was no difference in Lgr5 expression in murine enteroids treated with 1 mM SCFAs (P > 0.05; Fig 2a).',32240190),(4659,'mouse','increase','null','low-throughput','Enterocyte biomarker alkaline phosphatase (Alpi) was significantly increased in response to all SCFA treatments but was highest (3.5-fold;P < 0.05) in enteroids treated with PROP. There was no difference in Alpi expression in murine enteroids treated with 1 mM SCFAs (P > 0.05; Fig 2a).',32240190),(4660,'mouse','increase','null','low-throughput','Enterocyte biomarker alkaline phosphatase (Alpi) was significantly increased in response to all SCFA treatments but was highest (3.5-fold;P < 0.05) in enteroids treated with PROP. There was no difference in Alpi expression in murine enteroids treated with 1 mM SCFAs (P > 0.05; Fig 2a).',32240190),(4661,'mouse','increase','null','low-throughput','Enterocyte biomarker alkaline phosphatase (Alpi) was significantly increased in response to all SCFA treatments but was highest (3.5-fold;P < 0.05) in enteroids treated with PROP. There was no difference in Alpi expression in murine enteroids treated with 1 mM SCFAs (P > 0.05; Fig 2a).',32240190),(4662,'mouse','decrease','null','low-throughput','Gene expression of goblet cell marker mucin 2 (Muc2) was not significantly affected by ACET treatment, however, Muc2 was significantly decreased due to both BUT and PROP treatment (3-fold and 10-fold respectively;P < 0.05). Muc2 expression was not altered in murine enteroids treated with 1 mM SCFAs(P > 0.05; Fig 2a).',32240190),(4663,'mouse','decrease','null','low-throughput','Gene expression of goblet cell marker mucin 2 (Muc2) was not significantly affected by ACET treatment, however, Muc2 was significantly decreased due to both BUT and PROP treatment (3-fold and 10-fold respectively;P < 0.05). Muc2 expression was not altered in murine enteroids treated with 1 mM SCFAs(P > 0.05; Fig 2a).',32240190),(4664,'mouse','increase','null','low-throughput','Paneth cell marker lysozyme (Lyz) had a tendency (P = 0.08) to increase due to BUT treatment but there was no effect observed when treated with 1 mM SCFAs (P > 0.05;Fig 2a).',32240190),(4665,'mouse','increase','null','low-throughput','Enteroendocrine cell marker chromogranin A (Chga) was only significantly increased in BUT treated enteroids at both 1 mM and 5 mM concentrations (P < 0.05).',32240190),(4666,'mouse','increase','null','low-throughput','Tuft cell marker doublecortin-like kinase 1 (Dclk1) was significantly elevated in response to 5 mM ACET (3-fold) and PROP (2-fold) treatments compared to CON (P < 0.05; Fig 2b). There was no difference in Dclk1 expression in murine enteroids treated with 1 mM SCFAs (P > 0.05; Fig 2a).',32240190),(4667,'mouse','increase','null','low-throughput','Tuft cell marker doublecortin-like kinase 1 (Dclk1) was significantly elevated in response to 5 mM ACET (3-fold) and PROP (2-fold) treatments compared to CON (P < 0.05; Fig 2b). There was no difference in Dclk1 expression in murine enteroids treated with 1 mM SCFAs (P > 0.05; Fig 2a).',32240190),(4668,'mouse','increase','null','low-throughput','Antimicrobial peptide gene expression of c-type lectins regenerating islet-derived protein 3β (Reg3β) and 3γ (Reg3γ; Fig 2b) were increased moderately due to ACET treatment (20-fold and 40-fold), but were much more significantly increased due to BUT (150-fold and 350-fold,respectively) and PROP (250-fold, and 480-fold, respectively, P < 0.05) compared to CON.Only 1 mM BUT increased Reg3β and Reg3γ expression (80-fold and 100-fold respectively;P < 0.05).',32240190),(4669,'mouse','increase','null','low-throughput','Antimicrobial peptide gene expression of c-type lectins regenerating islet-derived protein 3β (Reg3β) and 3γ (Reg3γ; Fig 2b) were increased moderately due to ACET treatment (20-fold and 40-fold), but were much more significantly increased due to BUT (150-fold and 350-fold,respectively) and PROP (250-fold, and 480-fold, respectively, P < 0.05) compared to CON.Only 1 mM BUT increased Reg3β and Reg3γ expression (80-fold and 100-fold respectively;P < 0.05).',32240190),(4670,'mouse','increase','null','low-throughput','Antimicrobial peptide, β-defensin 1 (Defb1), was increased 3.5-fold due to ACET and BUT treatment, and 6-fold due to PROP treatment (P < 0.05); Fig 2b). 1 mM PROP was sufficient to increase Defb1 expression 2-fold compared to CON (P < 0.05).',32240190),(4671,'mouse','increase','null','low-throughput','Antimicrobial peptide, β-defensin 1 (Defb1), was increased 3.5-fold due to ACET and BUT treatment, and 6-fold due to PROP treatment (P < 0.05); Fig 2b). 1 mM PROP was sufficient to increase Defb1 expression 2-fold compared to CON (P < 0.05).',32240190),(4672,'mouse','increase','null','low-throughput','Antimicrobial peptide, β-defensin 1 (Defb1), was increased 3.5-fold due to ACET and BUT treatment, and 6-fold due to PROP treatment (P < 0.05); Fig 2b). 1 mM PROP was sufficient to increase Defb1 expression 2-fold compared to CON (P < 0.05).',32240190),(4673,'mouse','increase','null','low-throughput','Tight junction markers occludin (Ocln), and claudin-3 (Cldn3) were increased nearly 2-fold in enteroids treated with 5 mM ACET (P < 0.05).',32240190),(4674,'mouse','increase','null','low-throughput','Tight junction markers occludin (Ocln), and claudin-3 (Cldn3) were increased nearly 2-fold in enteroids treated with 5 mM ACET (P < 0.05).',32240190),(4675,'mouse','decrease','null','low-throughput','BUT decreased Cldn3 expression at 5 mM but Ocln was not altered (P > 0.05).',32240190),(4676,'mouse','decrease','null','low-throughput','5 mM PROP significantly decreased Cldn3 (P < 0.05).',32240190),(4677,'mouse','decrease','null','low-throughput','Tight junction protein 1 (Tjp1) was not altered by ACET treatment, but was decreased 1.7 and 2-fold respectively due to BUT and PROP treatment (P < 0.05; Fig 2b). There was no\ndifference in Tjp1 expression in murine enteroids treated with 1 mM SCFAs (P > 0.05; Fig 2a).',32240190),(4678,'mouse','decrease','null','low-throughput','Tight junction protein 1 (Tjp1) was not altered by ACET treatment, but was decreased 1.7 and 2-fold respectively due to BUT and PROP treatment (P < 0.05; Fig 2b). There was no\ndifference in Tjp1 expression in murine enteroids treated with 1 mM SCFAs (P > 0.05; Fig 2a).',32240190),(4679,'mouse','null','null','null','Ergothioneine, a metabolite of the gut bacterium Lactobacillus reuteri, protects against stress-induced sleep disturbances.',32467627),(4680,'mouse','decrease','null','low-throughput','Moreover, the expression of Cyp3a11 in GF mice also remained exten sively downregulated after butyrate supplementation rela tive to the corresponding conventional group (P < 0.01;P < 0.001, respectively).',32337713),(4681,'mouse','increase','null','low-throughput','In the presence of a complex microbiota, butyrate only had a significant effect on the hepatic expression of Cyp2b10 (2.85-fold higher relative to conventional mice; P < 0.05).',32337713),(4682,'mouse','decrease','null','low-throughput','Butyrate decreased the expression of Mdr1a in GF mice(P < 0.05) relative to the butyrate-treated conventional group, despite no evident changes in this isoform under GF conditions or by colonisation.',32337713),(4683,'mouse','increase','null','low-throughput','Analysis of the expression of mRNAs of three TJ proteins (ZO-1, claudin-1, and occludin) indicated greater expression in the Sham Group than in the three other groups (P < 0.05), and greater expression in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4A,B,D). In addition, expression of claudin-2, MMP9 was lower in the Sham Group than in the three other groups (P < 0.05) and lower in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4C,E).',33192557),(4684,'mouse','decrease','null','low-throughput','Analysis of the expression of mRNAs of three TJ proteins (ZO-1, claudin-1, and occludin) indicated greater expression in the Sham Group than in the three other groups (P < 0.05), and greater expression in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4A,B,D). In addition, expression of claudin-2, MMP9 was lower in the Sham Group than in the three other groups (P < 0.05) and lower in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4C,E).',33192557),(4685,'mouse','increase','null','low-throughput','Analysis of the expression of mRNAs of three TJ proteins (ZO-1, claudin-1, and occludin) indicated greater expression in the Sham Group than in the three other groups (P < 0.05), and greater expression in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4A,B,D). In addition, expression of claudin-2, MMP9 was lower in the Sham Group than in the three other groups (P < 0.05) and lower in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4C,E).',33192557),(4686,'mouse','increase','null','low-throughput','Analysis of the expression of mRNAs of three TJ proteins (ZO-1, claudin-1, and occludin) indicated greater expression in the Sham Group than in the three other groups (P < 0.05), and greater expression in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4A,B,D). In addition, expression of claudin-2, MMP9 was lower in the Sham Group than in the three other groups (P < 0.05) and lower in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4C,E).',33192557),(4687,'mouse','decrease','null','low-throughput','Analysis of the expression of mRNAs of three TJ proteins (ZO-1, claudin-1, and occludin) indicated greater expression in the Sham Group than in the three other groups (P < 0.05), and greater expression in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4A,B,D). In addition, expression of claudin-2, MMP9 was lower in the Sham Group than in the three other groups (P < 0.05) and lower in the two treatment groups than in the SAP + IAH Group (P < 0.05; Figures 4C,E).',33192557),(4688,'mouse','decrease','null','low-throughput','In addition, the DAO, LPS, FITC-dextran, and inflammatory cytokine levels of the two treatment groups were similar to each other (P > 0.05) and significantly lower than that of the SAP + IAH Group (P < 0.05; Figures 3B–E).',33192557),(4689,'mouse','decrease','null','low-throughput','In addition, the DAO, LPS, FITC-dextran, and inflammatory cytokine levels of the two treatment groups were similar to each other (P > 0.05) and significantly lower than that of the SAP + IAH Group (P < 0.05; Figures 3B–E).',33192557),(4690,'mouse','decrease','null','low-throughput','In addition, the DAO, LPS, FITC-dextran, and inflammatory cytokine levels of the two treatment groups were similar to each other (P > 0.05) and significantly lower than that of the SAP + IAH Group (P < 0.05; Figures 3B–E).',33192557),(4691,'mouse','decrease','null','low-throughput','Relative to the SAP + IAH group, the levels of claudin-2, MMP9, and TNF-α were significantly lower in the two treatment groups (P < 0.05; Figures 5A,B).',33192557),(4692,'mouse','increase','null','low-throughput','LPS significantly decreased the level of TEER (P < 0.01), while treatment with PA moderated the reduced TEER (P < 0.05). Secondly, we utilized immunoblotting to analyze the protein expression levels of ZO-1, occludin, and claudin-1. As shown in Figures 2B–E, the protein expression levels of ZO-1, occludin, and claudin-1 were lower in the LPS group than in the control group (P < 0.01). This LPS-induced alteration of tight junction protein expression was inhibited by PA (P < 0.05). These results suggest that PA can enhance and protect intestinal barrier function against LPS in IEC-6 cells.',33041816),(4693,'mouse','increase','null','low-throughput','LPS significantly decreased the level of TEER (P < 0.01), while treatment with PA moderated the reduced TEER (P < 0.05). Secondly, we utilized immunoblotting to analyze the protein expression levels of ZO-1, occludin, and claudin-1. As shown in Figures 2B–E, the protein expression levels of ZO-1, occludin, and claudin-1 were lower in the LPS group than in the control group (P < 0.01). This LPS-induced alteration of tight junction protein expression was inhibited by PA (P < 0.05). These results suggest that PA can enhance and protect intestinal barrier function against LPS in IEC-6 cells.',33041816),(4694,'mouse','increase','null','low-throughput','LPS significantly decreased the level of TEER (P < 0.01), while treatment with PA moderated the reduced TEER (P < 0.05). Secondly, we utilized immunoblotting to analyze the protein expression levels of ZO-1, occludin, and claudin-1. As shown in Figures 2B–E, the protein expression levels of ZO-1, occludin, and claudin-1 were lower in the LPS group than in the control group (P < 0.01). This LPS-induced alteration of tight junction protein expression was inhibited by PA (P < 0.05). These results suggest that PA can enhance and protect intestinal barrier function against LPS in IEC-6 cells.',33041816),(4695,'mouse','decrease','null','low-throughput','These data indicate that PA promotes epithelial cell migration via inhibition of expression of b1-integrin induced by LPS.',33041816),(4696,'mouse','decrease','null','low-throughput','Treatment with PA rescued the obvious increases in protein expression in NLRP3, caspase-1, IL-1b, and IL-18 induced by LPS.',33041816),(4697,'mouse','decrease','null','low-throughput','Treatment with PA rescued the obvious increases in protein expression in NLRP3, caspase-1, IL-1b, and IL-18 induced by LPS.',33041816),(4698,'mouse','decrease','null','low-throughput','Treatment with PA rescued the obvious increases in protein expression in NLRP3, caspase-1, IL-1b, and IL-18 induced by LPS.',33041816),(4699,'mouse','decrease','null','low-throughput','Treatment with PA rescued the obvious increases in protein expression in NLRP3, caspase-1, IL-1b, and IL-18 induced by LPS.',33041816),(4700,'mouse','decrease','null','low-throughput','The research results show that LPS strongly stimulated TLR4, the phosphorylation of IkBa and p65 in whole cell extracts and IkBa degradation in the cytoplasm. It also caused the translocation of p65 into the nucleus. However, these effects were significantly inhibited by PA (Figures 5A–G).',33041816),(4701,'mouse','increase','null','low-throughput','SLC5A8 and GPR43 expressions in butyrate‑treated CRC cells were quantified (Fig. 5B and C for SLC5A8 and GPR43, respectively). In comparison with untreated cells, the NaB‑treated SW480 cells only presented higher GPR43 (P<0.05), but the NaB‑treated SW620 cells expressed the most SLC5A8 (P<0.01) and GPR43 (P<0.05).',33101496),(4704,'mouse','increase','Cholic acid','low-throughput','Intestinal expression of farnesoid X receptor (FXR) mRNA was significantly higher, whereas hepatic expression of cholesterol-7a-hydroxylase 1 (CYP7A1) and small heterodimer partner (SHP) were significantly lower in ob/ob mice compared to those in control mice.',27111464),(4705,'mouse','increase','Cholic acid','low-throughput','FGF15 mRNA expression was also increased in ob/ob mice compared to that of the lean control mice.',27111464),(4706,'mouse','increase','Cholic acid','low-throughput','In the present study, hepatic mRNA expression of FGFR4 was significantly increased, while CYP7A1 was greatly decreased in ob/ob mice.',27111464),(4707,'mouse','decrease','Cholic acid','low-throughput','In the presentstudy, hepatic mRNA expression of FGFR4 was significantly increased, while CYP7A1 was greatly decreased in ob/ob mice.',27111464),(4708,'mouse','decrease','Cholic acid','low-throughput','Decreased intestinal taurine-conjugated BAs may also affect enterohepatic circulation of BAs and facilitate hepatic de novo lipogenesis and down-regulate fatty acid oxidation in ob/ob mice through intestinal FXR-FGF15 and hepatic FXR-SHP pathways.',27111464),(4709,'mouse','decrease','Cholic acid','low-throughput','Also, ob/ob mice showed significantly downregulated CPT-1 mRNA expression compared to their lean control mice.',27111464),(4710,'mouse','increase','Cholic acid','low-throughput','Moreover, hepatic FAS gene expression was up-regulated while both SREBP-1c and ACC were slightly increased without significant differences between two groups.',27111464),(4711,'mouse','increase','Cholic acid','low-throughput','Intestinal expression of farnesoid X receptor (FXR) mRNA was significantly higher, whereas hepatic expression of cholesterol-7a-hydroxylase 1 (CYP7A1) and small heterodimer partner (SHP) were significantly lower in ob/ob mice compared to those in control mice.',27111464),(4712,'mouse','increase','Cholic acid','low-throughput','FGF15 mRNA expression was also increased in ob/ob mice compared to that of the lean control mice.',27111464),(4713,'mouse','increase','Cholic acid','low-throughput','In the present study, hepatic mRNA expression of FGFR4 was significantly increased, while CYP7A1 was greatly decreased in ob/ob mice.',27111464),(4714,'mouse','decrease','Cholic acid','low-throughput','In the presentstudy, hepatic mRNA expression of FGFR4 was significantly increased, while CYP7A1 was greatly decreased in ob/ob mice.',27111464),(4715,'mouse','decrease','Cholic acid','low-throughput','Decreased intestinal taurine-conjugated BAs may also affect enterohepatic circulation of BAs and facilitate hepatic de novo lipogenesis and down-regulate fatty acid oxidation in ob/ob mice through intestinal FXR-FGF15 and hepatic FXR-SHP pathways.',27111464),(4716,'mouse','decrease','Cholic acid','low-throughput','Also, ob/ob mice showed significantly downregulated CPT-1 mRNA expression compared to their lean control mice.',27111464),(4717,'mouse','increase','Cholic acid','low-throughput','Moreover, hepatic FAS gene expression was up-regulated while both SREBP-1c and ACC were slightly increased without significant differences between two groups.',27111464),(4718,'mouse','increase','null','low-throughput','Intestinal expression of farnesoid X receptor (FXR) mRNA was significantly higher, whereas hepatic expression of cholesterol-7a-hydroxylase 1 (CYP7A1) and small heterodimer partner (SHP) were significantly lower in ob/ob mice compared to those in control mice.',27111464),(4719,'mouse','increase','null','low-throughput','FGF15 mRNA expression was also increased in ob/ob mice compared to that of the lean control mice.',27111464),(4720,'mouse','increase','null','low-throughput','In the present study, hepatic mRNA expression of FGFR4 was significantly increased, while CYP7A1 was greatly decreased in ob/ob mice.',27111464),(4721,'mouse','decrease','null','low-throughput','In the presentstudy, hepatic mRNA expression of FGFR4 was significantly increased, while CYP7A1 was greatly decreased in ob/ob mice.',27111464),(4722,'mouse','decrease','null','low-throughput','Decreased intestinal taurine-conjugated BAs may also affect enterohepatic circulation of BAs and facilitate hepatic de novo lipogenesis and down-regulate fatty acid oxidation in ob/ob mice through intestinal FXR-FGF15 and hepatic FXR-SHP pathways.',27111464),(4723,'mouse','decrease','null','low-throughput','Also, ob/ob mice showed significantly downregulated CPT-1 mRNA expression compared to their lean control mice.',27111464),(4724,'mouse','increase','null','low-throughput','Moreover, hepatic FAS gene expression was up-regulated while both SREBP-1c and ACC were slightly increased without significant differences between two groups.',27111464),(4725,'mouse','null','Indole','null','Production of IPA was shown to be completely dependent on the presence of gut microflora and could be established by colonization with the bacterium Clostridium sporogenes.',19234110),(4726,'mouse','null','Rutin','null','LY1 also metabolized rutin to quercetin and 3,4-DPA, although the amount of 3,4-DPA formed was minute compared to that of quercetin.',14629841),(4727,'mouse','null','Rutin','null','LY1 also metabolized rutin to quercetin and 3,4-DPA, although the amount of 3,4-DPA formed was minute compared to that of quercetin.',14629841),(4728,'mouse','null','Quercitrin','null','A significant portion of flavonoids in EGb 761 was converted to the main aglycone forms (quercetin and kaempferol) (Fig. 4B) after 36 hours.',14629841),(4729,'mouse','null','Quercitrin','null','A significant portion of flavonoids in EGb 761 was converted to the main aglycone forms (quercetin and kaempferol) (Fig. 4B) after 36 hours.',14629841),(4730,'mouse','null','Quercitrin','null','When enrichment cultures derived from either rat fecal materials was used, glycosides in EGb 761 were completely consumed but quercetin, kaempferol, and the possible isorhamnetin were only transiently detected during the growth of the bacteria in enrichment cultures.',14629841),(4731,'mouse','null','Isoquercitrin','null','A significant portion of flavonoids in EGb 761 was converted to the main aglycone forms (quercetin and kaempferol) (Fig. 4B) after 36 hours.',14629841),(4732,'mouse','null','Isoquercitrin','null','A significant portion of flavonoids in EGb 761 was converted to the main aglycone forms (quercetin and kaempferol) (Fig. 4B) after 36 hours.',14629841),(4733,'mouse','null','Isoquercitrin','null','When enrichment cultures derived from either rat fecal materials was used, glycosides in EGb 761 were completely consumed but quercetin, kaempferol, and the possible isorhamnetin were only transiently detected during the growth of the bacteria in enrichment cultures.',14629841),(4734,'mouse','null','null','null','For example, quinic acid, a cyclic polyol obtained from plant-derived products, reached its highest levels when Odoribacter splanchnicus was present but Escherichia coli was not.',24452263),(4735,'mouse','null','null','null','One such example is our observation that concentrations of a lysophosphatidylethanolamine (LysoPE15) can be increased by either B. ovatus or B. vulgatus, but reaches maximal cecal concentrations when both strains were present in a community (Fig. 3B).',24452263),(4736,'mouse','null','null','null','One such example is our observation that concentrations of a lysophosphatidylethanolamine (LysoPE15) can be increased by either B. ovatus or B. vulgatus, but reaches maximal cecal concentrations when both strains were present in a community (Fig. 3B).',24452263),(4737,'mouse','null','null','null','While another metabolite (assignment, 4�?hydroxy-3�?5,6,7,8-pentamethoxyflavone) required both B. ovatus and O.splanchnicus to consistently reach high levels.',24452263),(4738,'mouse','null','null','null','While another metabolite (assignment, 4�?hydroxy-3�?5,6,7,8-pentamethoxyflavone) required both B. ovatus and O.splanchnicus to consistently reach high levels.',24452263),(4747,'mouse','null','null','null','Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.',24452263),(4748,'mouse','null','null','null','Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.',24452263),(4749,'mouse','null','null','null','Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.',24452263),(4750,'mouse','null','null','null','Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.',24452263),(4751,'mouse','null','null','null','Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.',24452263),(4752,'mouse','null','null','null','Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.',24452263),(4753,'mouse','null','null','null','Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.',24452263),(4754,'mouse','null','null','null','Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.',24452263),(4755,'mouse','null','null','null','Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.',24452263),(4756,'mouse','null','null','null','Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.',24452263),(4757,'mouse','null','null','null','Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.',24452263),(4758,'mouse','null','null','null','Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.',24452263),(4759,'mouse','null','null','null','Colonization with B. caccae, B.thetaiotaomicron, B. massiliensis or P. distasonis led to increases in acetate, propionate and succinate relative to germ-free controls (table S12) while butyrate was only significantly elevated above concentrations observed in germ-free controls in the case of mice colonized with P. distasonis.',24452263),(4760,'mouse','increase','Lactic acid','low-throughput','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.',27448578),(4761,'mouse','increase','Lactic acid','low-throughput','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.',27448578),(4762,'mouse','increase','Lactic acid','low-throughput','SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.',27448578),(4763,'mouse','decrease','Lactic acid','low-throughput','Furthermore, DCs from mice treated with propionate, characterized by reduced expression of CD40, PD-L2 and CD86, exhibited the impaired ability to initiate Th2 effector function and further to promote Th2-mediated allergic airway inflammation.',27448578),(4764,'mouse','decrease','Lactic acid','low-throughput','Furthermore, DCs from mice treated with propionate, characterized by reduced expression of CD40, PD-L2 and CD86, exhibited the impaired ability to initiate Th2 effector function and further to promote Th2-mediated allergic airway inflammation.',27448578),(4765,'mouse','decrease','Lactic acid','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4766,'mouse','decrease','Lactic acid','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4767,'mouse','decrease','Lactic acid','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4768,'mouse','increase','Pyruvate','low-throughput','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.',27448578),(4769,'mouse','increase','Pyruvate','low-throughput','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.',27448578),(4770,'mouse','increase','Pyruvate','low-throughput','SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.',27448578),(4771,'mouse','increase','null','low-throughput','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.',27448578),(4772,'mouse','increase','null','low-throughput','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.',27448578),(4773,'mouse','increase','null','low-throughput','SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.',27448578),(4774,'mouse','increase','null','low-throughput','GPR41 and GPR43 can be efficiently activated by acetate, propionate, butyrate and other SCFAs [39], while GPR109a can be activated mainly by butyrate and niacin.',27448578),(4775,'mouse','increase','null','low-throughput','Butyrate upregulated the histone H3 acetylation of Foxp3 and promoted the differentiation of Treg, which acts as a key anti-inflammatory effector.',27448578),(4776,'mouse','decrease','null','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4777,'mouse','decrease','null','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4778,'mouse','decrease','null','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4779,'mouse','increase','null','low-throughput','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.',27448578),(4780,'mouse','increase','null','low-throughput','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.',27448578),(4781,'mouse','increase','null','low-throughput','SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.',27448578),(4782,'mouse','increase','null','low-throughput','GPR41 and GPR43 can be efficiently activated by acetate, propionate, butyrate and other SCFAs [39], while GPR109a can be activated mainly by butyrate and niacin.',27448578),(4783,'mouse','increase','null','low-throughput','Butyrate upregulated the histone H3 acetylation of Foxp3 and promoted the differentiation of Treg, which acts as a key anti-inflammatory effector.',27448578),(4784,'mouse','decrease','null','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4785,'mouse','decrease','null','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4786,'mouse','decrease','null','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4787,'mouse','increase','null','low-throughput','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.',27448578),(4788,'mouse','increase','null','low-throughput','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.',27448578),(4789,'mouse','increase','null','low-throughput','SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.',27448578),(4790,'mouse','increase','null','low-throughput','GPR41 and GPR43 can be efficiently activated by acetate, propionate, butyrate and other SCFAs [39], while GPR109a can be activated mainly by butyrate and niacin.',27448578),(4791,'mouse','increase','null','low-throughput','Butyrate upregulated the histone H3 acetylation of Foxp3 and promoted the differentiation of Treg, which acts as a key anti-inflammatory effector.',27448578),(4792,'mouse','decrease','null','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4793,'mouse','decrease','null','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4794,'mouse','decrease','null','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4795,'mouse','increase','Starch','low-throughput','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.',27448578),(4796,'mouse','increase','Starch','low-throughput','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.',27448578),(4797,'mouse','increase','Starch','low-throughput','SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.',27448578),(4798,'mouse','increase','Starch','low-throughput','GPR41 and GPR43 can be efficiently activated by acetate, propionate, butyrate and other SCFAs [39], while GPR109a can be activated mainly by butyrate and niacin.',27448578),(4799,'mouse','increase','Starch','low-throughput','Butyrate upregulated the histone H3 acetylation of Foxp3 and promoted the differentiation of Treg, which acts as a key anti-inflammatory effector.',27448578),(4800,'mouse','decrease','Starch','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4801,'mouse','decrease','Starch','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4802,'mouse','decrease','Starch','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4803,'mouse','increase','Succinate','low-throughput','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.',27448578),(4804,'mouse','increase','Succinate','low-throughput','Most notable among the SCFA targets is the mammalian G protein-coupled receptor pair of GPR41 and GPR43.',27448578),(4805,'mouse','increase','Succinate','low-throughput','SCFA affect pro-inflammatory cytokines production (e.g. IL-6, IL-8, IL-1β and TNFα) through enhancing  NF-κB activation in TLR ligand-responses in epithelial cells.',27448578),(4806,'mouse','decrease','Succinate','low-throughput','Furthermore, DCs from mice treated with propionate, characterized by reduced expression of CD40, PD-L2 and CD86, exhibited the impaired ability to initiate Th2 effector function and further to promote Th2-mediated allergic airway inflammation.',27448578),(4807,'mouse','decrease','Succinate','low-throughput','Furthermore, DCs from mice treated with propionate, characterized by reduced expression of CD40, PD-L2 and CD86, exhibited the impaired ability to initiate Th2 effector function and further to promote Th2-mediated allergic airway inflammation.',27448578),(4808,'mouse','decrease','Succinate','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4809,'mouse','decrease','Succinate','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4810,'mouse','decrease','Succinate','low-throughput','Butyrate and propionate inhibit activation of BMDC via suppressing the LPS-induced expression of costimulatory molecule CD40 and secretion of IL-6 and IL-12p40.',27448578),(4811,'mouse','increase','Tryptophan','low-throughput','Colonization of germ-free mice with wild-type Escherichia coli, but not E. coli mutants unable to generate indole, induced colonic epithelial IL-10R1.',29454749),(4812,'mouse','increase','Tryptophan','low-throughput','Colonization of germ-free mice with wild-type Escherichia coli, but not E. coli mutants unable to generate indole, induced colonic epithelial IL-10R1.',29454749),(4813,'mouse','increase','Tryptophan','low-throughput','IAld has recently been identified as an aryl hydrocarbon receptor (AHR) ligand.',29454749),(4814,'mouse','increase','Tryptophan','low-throughput','Colonization of germ-free mice with wild-type Escherichia coli, but not E. coli mutants unable to generate indole, induced colonic epithelial IL-10R1.',29454749),(4815,'mouse','increase','Tryptophan','low-throughput','Colonization of germ-free mice with wild-type Escherichia coli, but not E. coli mutants unable to generate indole, induced colonic epithelial IL-10R1.',29454749),(4816,'mouse','increase','Tryptophan','low-throughput','IAld has recently been identified as an aryl hydrocarbon receptor (AHR) ligand.',29454749),(4817,'mouse','increase','null','low-throughput','In mice, nicotinic acid inhibits lipolysis via PUMA-G, a Gi/o-coupled seven-transmembrane receptor expressed in adipocytes and activated macrophages.',15929991),(4818,'mouse','increase','null','low-throughput','Here we show that the fatty acid-derived ketone body (D)-OHB specifically activates PUMA-G/HM74a at concentrations observed in serum during fasting.',15929991),(4819,'mouse','increase','null','low-throughput','Here we show that the fatty acid-derived ketone body (D)-OHB specifically activates PUMA-G/HM74a at concentrations observed in serum during fasting.',15929991),(4820,'mouse','increase','null','low-throughput','Both (D)- and (L)-OHB, but not acetoacetate or free acetone, elicited a Ca2 response in cells expressing HM74a but not HM74.',15929991),(4821,'mouse','increase','null','low-throughput','Racemic (DL)-OHB also showed some degree of activity on all receptors; however, it was more potent on HM74a and its ortholog PUMA-G  than on HM74.',15929991),(4825,'mouse','increase','Tryptophan','high-throughput','Polybrominated diphenyl ethers (PBDEs) are persistent environmental toxicants associated with increased risk for metabolic syndrome.PBDEs altered the largest number of metabolites in the LIC; most were regulated by PBDEs in GF conditions. Importantly, intestinal microbiome was necessary for PBDE-mediated decreases in branched-chain and aromatic amino acid metabolites, including 3-indolepropionic acid, a tryptophan metabolite recently shown to be protective against inflammation and diabetes. Gene-metabolite networks revealed a positive association between the hepatic glycan synthesis gene a-1,6-mannosyltransferase (Alg12) mRNA and mannose, which are important for protein glycosylation. Glycome changes have been observed in patients with metabolic syndrome. In LIC of CV mice, 23 bacterial taxa were regulated by PBDEs. Correlations of certain taxa with distinct serum metabolites further highlight a modulatory role of the microbiome in mediating PBDE effects. In summary, PBDEs impact intermediary metabolism in an intestinal microbiome–dependent manner, suggesting that dysbiosis may contribute to PBDE-mediated toxicities that include metabolic syndrome.',31123037),(4826,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4827,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4828,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4829,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4830,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4831,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4832,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4833,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4834,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4835,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4836,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4837,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4838,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4839,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4840,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4841,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4842,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4843,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4844,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4845,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4846,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4847,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4848,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4849,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4850,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4851,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4852,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4853,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4854,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4855,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4856,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4857,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4858,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4859,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4860,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4861,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4862,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4863,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4864,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4865,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4866,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4867,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4868,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4869,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4870,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4871,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4872,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4873,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4874,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4875,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4876,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4877,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4878,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4879,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4880,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4881,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4882,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4883,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4884,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4885,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4886,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4887,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4888,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4889,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4890,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4891,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4892,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4893,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4894,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4895,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4896,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4897,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4898,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4899,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4900,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4901,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4902,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4903,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4904,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4905,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4906,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4907,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4908,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4909,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4910,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4911,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4912,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4913,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4914,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4915,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4916,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4917,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4918,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4919,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4920,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4921,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4922,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4923,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4924,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4925,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4926,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4927,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4928,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4929,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4930,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4931,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4932,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4933,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4934,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4935,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4936,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4937,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4938,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4939,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4940,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4941,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4942,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4943,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4944,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4945,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4946,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4947,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4948,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4949,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4950,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4951,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4952,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4953,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4954,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4955,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4956,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4957,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4958,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4959,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4960,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4961,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4962,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4963,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4964,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4965,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4966,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4967,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4968,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4969,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4970,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4971,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4972,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4973,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4974,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4975,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4976,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4977,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4978,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4979,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4980,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4981,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4982,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4983,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4984,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4985,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4986,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4987,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4988,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4989,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4990,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4991,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4992,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4993,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4994,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4995,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4996,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4997,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4998,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(4999,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5000,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5001,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5002,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5003,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5004,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5005,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5006,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5007,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5008,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5009,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5010,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5011,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5012,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5013,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5014,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5015,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5016,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5017,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5018,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5019,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5020,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5021,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5022,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5023,'mouse','decrease','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5024,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5025,'mouse','increase','null','high-throughput','Differential expression ratios and FDR p-value of genes involved in various intermediary metabolism pathways  in livers of PBDE-exposed CV and GF mice. Data were obtained from RNA-Seq experiments and differential expression was  determined using CuffDiff.',31123037),(5027,'mouse','increase','D-Fructose','low-throughput','A high-fructose diet caused the hippocampal neuroinflammatory response, reactive gliosis, and neuronal loss in C57BL/6N mice. Depletion of the gut microbiota using broad-spectrum antibiotics suppressed the hippocampal neuroinflammatory response in fructose-fed mice, but these animals still exhibited neuronal loss. Gut microbiota compositional alteration, short-chain fatty acids (SCFAs) reduction, intestinal epithelial barrier impairment, NOD-like receptor family pyrin domain-containing 6 (NLRP6) inflammasome dysfunction, high levels of serum endotoxin, and FITC-dextran were observed in fructose-fed mice. Of note, SCFAs, as well as pioglitazone (a selective peroxisome proliferator-activated receptor gamma (PPAR-γ) agonist), shaped the gut microbiota and ameliorated intestinal epithelial barrier impairment and NLRP6 inflammasome dysfunction in fructose-fed mice. Moreover, SCFAs-mediated NLRP6 inflammasome activation was inhibited by histamine (a bacterial metabolite) in ex vivo colonic explants and suppressed in murine CT26 colon carcinoma cells transfected with NLRP6 siRNA. However, pioglitazone and GW9662 (a PPAR-γ antagonist) exerted no impact on SCFAs-mediated NLRP6 inflammasome activation in ex vivo colonic explants, suggesting that SCFAs may stimulate NLRP6 inflammasome independently of PPAR-γ activation. SCFAs and pioglitazone prevented fructose-induced hippocampal neuroinflammatory response and neuronal loss in mice. Additionally, SCFAs activated colonic NLRP6 inflammasome and increased DCX+ newborn neurons in the hippocampal DG of control mice',31255176),(5028,'mouse','increase','D-Fructose','low-throughput','A high-fructose diet caused the hippocampal neuroinflammatory response, reactive gliosis, and neuronal loss in C57BL/6N mice. Depletion of the gut microbiota using broad-spectrum antibiotics suppressed the hippocampal neuroinflammatory response in fructose-fed mice, but these animals still exhibited neuronal loss. Gut microbiota compositional alteration, short-chain fatty acids (SCFAs) reduction, intestinal epithelial barrier impairment, NOD-like receptor family pyrin domain-containing 6 (NLRP6) inflammasome dysfunction, high levels of serum endotoxin, and FITC-dextran were observed in fructose-fed mice. Of note, SCFAs, as well as pioglitazone (a selective peroxisome proliferator-activated receptor gamma (PPAR-γ) agonist), shaped the gut microbiota and ameliorated intestinal epithelial barrier impairment and NLRP6 inflammasome dysfunction in fructose-fed mice. Moreover, SCFAs-mediated NLRP6 inflammasome activation was inhibited by histamine (a bacterial metabolite) in ex vivo colonic explants and suppressed in murine CT26 colon carcinoma cells transfected with NLRP6 siRNA. However, pioglitazone and GW9662 (a PPAR-γ antagonist) exerted no impact on SCFAs-mediated NLRP6 inflammasome activation in ex vivo colonic explants, suggesting that SCFAs may stimulate NLRP6 inflammasome independently of PPAR-γ activation. SCFAs and pioglitazone prevented fructose-induced hippocampal neuroinflammatory response and neuronal loss in mice. Additionally, SCFAs activated colonic NLRP6 inflammasome and increased DCX+ newborn neurons in the hippocampal DG of control mice',31255176),(5029,'mouse','increase','D-Fructose','low-throughput','A high-fructose diet caused the hippocampal neuroinflammatory response, reactive gliosis, and neuronal loss in C57BL/6N mice. Depletion of the gut microbiota using broad-spectrum antibiotics suppressed the hippocampal neuroinflammatory response in fructose-fed mice, but these animals still exhibited neuronal loss. Gut microbiota compositional alteration, short-chain fatty acids (SCFAs) reduction, intestinal epithelial barrier impairment, NOD-like receptor family pyrin domain-containing 6 (NLRP6) inflammasome dysfunction, high levels of serum endotoxin, and FITC-dextran were observed in fructose-fed mice. Of note, SCFAs, as well as pioglitazone (a selective peroxisome proliferator-activated receptor gamma (PPAR-γ) agonist), shaped the gut microbiota and ameliorated intestinal epithelial barrier impairment and NLRP6 inflammasome dysfunction in fructose-fed mice. Moreover, SCFAs-mediated NLRP6 inflammasome activation was inhibited by histamine (a bacterial metabolite) in ex vivo colonic explants and suppressed in murine CT26 colon carcinoma cells transfected with NLRP6 siRNA. However, pioglitazone and GW9662 (a PPAR-γ antagonist) exerted no impact on SCFAs-mediated NLRP6 inflammasome activation in ex vivo colonic explants, suggesting that SCFAs may stimulate NLRP6 inflammasome independently of PPAR-γ activation. SCFAs and pioglitazone prevented fructose-induced hippocampal neuroinflammatory response and neuronal loss in mice. Additionally, SCFAs activated colonic NLRP6 inflammasome and increased DCX+ newborn neurons in the hippocampal DG of control mice',31255176),(5030,'mouse','decrease','D-Fructose','low-throughput','A high-fructose diet caused the hippocampal neuroinflammatory response, reactive gliosis, and neuronal loss in C57BL/6N mice. Depletion of the gut microbiota using broad-spectrum antibiotics suppressed the hippocampal neuroinflammatory response in fructose-fed mice, but these animals still exhibited neuronal loss. Gut microbiota compositional alteration, short-chain fatty acids (SCFAs) reduction, intestinal epithelial barrier impairment, NOD-like receptor family pyrin domain-containing 6 (NLRP6) inflammasome dysfunction, high levels of serum endotoxin, and FITC-dextran were observed in fructose-fed mice. Of note, SCFAs, as well as pioglitazone (a selective peroxisome proliferator-activated receptor gamma (PPAR-γ) agonist), shaped the gut microbiota and ameliorated intestinal epithelial barrier impairment and NLRP6 inflammasome dysfunction in fructose-fed mice. Moreover, SCFAs-mediated NLRP6 inflammasome activation was inhibited by histamine (a bacterial metabolite) in ex vivo colonic explants and suppressed in murine CT26 colon carcinoma cells transfected with NLRP6 siRNA. However, pioglitazone and GW9662 (a PPAR-γ antagonist) exerted no impact on SCFAs-mediated NLRP6 inflammasome activation in ex vivo colonic explants, suggesting that SCFAs may stimulate NLRP6 inflammasome independently of PPAR-γ activation. SCFAs and pioglitazone prevented fructose-induced hippocampal neuroinflammatory response and neuronal loss in mice. Additionally, SCFAs activated colonic NLRP6 inflammasome and increased DCX+ newborn neurons in the hippocampal DG of control mice',31255176),(5031,'mouse','decrease','(-)-Epigallocatechin gallate','low-throughput','Inhibitory activity of angiotensin I-converting enzyme (ACE) was examined with (�?-epigallocatechin gallate (EGCG) metabolites produced by intestinal bacteria, together with tea catechins. All of the metabolites showed ACE inhibitory activities.',26323573),(5032,'mouse','decrease','(-)-Epigallocatechin gallate','low-throughput','Inhibitory activity of angiotensin I-converting enzyme (ACE) was examined with (�?-epigallocatechin gallate (EGCG) metabolites produced by intestinal bacteria, together with tea catechins. All of the metabolites showed ACE inhibitory activities.',26323573),(5033,'mouse','decrease','(-)-Epigallocatechin gallate','low-throughput','Inhibitory activity of angiotensin I-converting enzyme (ACE) was examined with (�?-epigallocatechin gallate (EGCG) metabolites produced by intestinal bacteria, together with tea catechins. All of the metabolites showed ACE inhibitory activities.',26323573),(5034,'mouse','decrease','(-)-Epigallocatechin gallate','low-throughput','Inhibitory activity of angiotensin I-converting enzyme (ACE) was examined with (�?-epigallocatechin gallate (EGCG) metabolites produced by intestinal bacteria, together with tea catechins. All of the metabolites showed ACE inhibitory activities.',26323573),(5035,'mouse','decrease','(-)-Epigallocatechin gallate','low-throughput','Inhibitory activity of angiotensin I-converting enzyme (ACE) was examined with (�?-epigallocatechin gallate (EGCG) metabolites produced by intestinal bacteria, together with tea catechins. All of the metabolites showed ACE inhibitory activities.',26323573),(5036,'mouse','decrease','(-)-Epigallocatechin gallate','low-throughput','Inhibitory activity of angiotensin I-converting enzyme (ACE) was examined with (�?-epigallocatechin gallate (EGCG) metabolites produced by intestinal bacteria, together with tea catechins. All of the metabolites showed ACE inhibitory activities.',26323573),(5037,'mouse','null','null','null','Firstly, studies have shown that both Treponema.spp and glycerophospholipids are  associated with brain development and diseases. Iron metabolism might provide  substantial contribution to the cross talk of Treponema.spp and glycerophospholipids .  Secondly, in this report, the Facklamia genus was negatively correlated with 4 saturated  and 15 unsaturated fatty acids. Among them, linoleic acid , eicosapentaenoic acid, arachidonic acid  and docosahexaenoic acid  are  well-known dietary nutrients for neural development and brain functions especially in infants and the elderly. Facklamia.spp was associated with invasive diseases although  the pathogenic potential is still unclear. Thirdly, the close correlation of Ruminococcus  gnavus (a bile salt hydrolase (BSH) bacterium) and ursodeoxycholic acid (UDCA), deoxycholic acid (DCA) and chenodeoxycholic acid (CDCA) were in line with the  consensus that BSH bacteria participant directly in the connection between brain and gut and that modulation of BSH activity may be of benefit to human health, Additionally, the links of UDCA, DCA, CDCA and Ruminococcus genuswith  typical intestinal, brain and neuropathic diseases were demonstrated separately by numerous reports which were circumstantial evidence of these association pairs. Fourthly,  oral administration of glutamic acid (the dominant neurotransmitter in brain) , GABA  and glycinewere proposed as new therapeutic strategies for  the regulation of both neurobehaviors and gastrointestinal tract health, although further  studies are needed to define their possible roles in the gut-brain axis. We here observed that these neurotransmitters were positively correlated with Haliscomenobacter.spp,  Pedobacter.spp and Dyadobacter.spp, suggesting new mechanisms underlying healthy  diet and antibiotic interventions.',29353471),(5038,'mouse','null','null','null','Firstly, studies have shown that both Treponema.spp and glycerophospholipids are  associated with brain development and diseases. Iron metabolism might provide  substantial contribution to the cross talk of Treponema.spp and glycerophospholipids .  Secondly, in this report, the Facklamia genus was negatively correlated with 4 saturated  and 15 unsaturated fatty acids. Among them, linoleic acid , eicosapentaenoic acid, arachidonic acid  and docosahexaenoic acid  are  well-known dietary nutrients for neural development and brain functions especially in infants and the elderly. Facklamia.spp was associated with invasive diseases although  the pathogenic potential is still unclear. Thirdly, the close correlation of Ruminococcus  gnavus (a bile salt hydrolase (BSH) bacterium) and ursodeoxycholic acid (UDCA), deoxycholic acid (DCA) and chenodeoxycholic acid (CDCA) were in line with the  consensus that BSH bacteria participant directly in the connection between brain and gut and that modulation of BSH activity may be of benefit to human health, Additionally, the links of UDCA, DCA, CDCA and Ruminococcus genuswith  typical intestinal, brain and neuropathic diseases were demonstrated separately by numerous reports which were circumstantial evidence of these association pairs. Fourthly,  oral administration of glutamic acid (the dominant neurotransmitter in brain) , GABA  and glycinewere proposed as new therapeutic strategies for  the regulation of both neurobehaviors and gastrointestinal tract health, although further  studies are needed to define their possible roles in the gut-brain axis. We here observed that these neurotransmitters were positively correlated with Haliscomenobacter.spp,  Pedobacter.spp and Dyadobacter.spp, suggesting new mechanisms underlying healthy  diet and antibiotic interventions.',29353471),(5039,'mouse','null','null','null','Firstly, studies have shown that both Treponema.spp and glycerophospholipids are  associated with brain development and diseases. Iron metabolism might provide  substantial contribution to the cross talk of Treponema.spp and glycerophospholipids .  Secondly, in this report, the Facklamia genus was negatively correlated with 4 saturated  and 15 unsaturated fatty acids. Among them, linoleic acid , eicosapentaenoic acid, arachidonic acid  and docosahexaenoic acid  are  well-known dietary nutrients for neural development and brain functions especially in infants and the elderly. Facklamia.spp was associated with invasive diseases although  the pathogenic potential is still unclear. Thirdly, the close correlation of Ruminococcus  gnavus (a bile salt hydrolase (BSH) bacterium) and ursodeoxycholic acid (UDCA), deoxycholic acid (DCA) and chenodeoxycholic acid (CDCA) were in line with the  consensus that BSH bacteria participant directly in the connection between brain and gut and that modulation of BSH activity may be of benefit to human health, Additionally, the links of UDCA, DCA, CDCA and Ruminococcus genuswith  typical intestinal, brain and neuropathic diseases were demonstrated separately by numerous reports which were circumstantial evidence of these association pairs. Fourthly,  oral administration of glutamic acid (the dominant neurotransmitter in brain) , GABA  and glycinewere proposed as new therapeutic strategies for  the regulation of both neurobehaviors and gastrointestinal tract health, although further  studies are needed to define their possible roles in the gut-brain axis. We here observed that these neurotransmitters were positively correlated with Haliscomenobacter.spp,  Pedobacter.spp and Dyadobacter.spp, suggesting new mechanisms underlying healthy  diet and antibiotic interventions.',29353471),(5047,'mouse','null','Vorinostat','null','Metabolite profiles of P. mallochii CCH01 were obviously changed by SAHA treatment. Four metabolites (1�?), including two new natural sclerotioramine derivatives, isochromophilone XIV (1) and isochromophilone XV (2), and two known compounds, sclerotioramine (3) and (+)- sclerotiorin (4), were isolated and purified from P. mallochii CCH01 treated with SAHA.',31632360),(5048,'mouse','null','Vorinostat','null','Metabolite profiles of P. mallochii CCH01 were obviously changed by SAHA treatment. Four metabolites (1�?), including two new natural sclerotioramine derivatives, isochromophilone XIV (1) and isochromophilone XV (2), and two known compounds, sclerotioramine (3) and (+)- sclerotiorin (4), were isolated and purified from P. mallochii CCH01 treated with SAHA.',31632360),(5053,'mouse','null','Isoxanthohumol','null','Recently, it was shown that the exposure to the potent hop phytoestrogen 8-prenylnaringenin (8-PN) depends on intestinal bacterial activation of isoxanthohumol (IX), but this occurs in only one-third of tested individuals. As the butyrate-producing Eubacterium limosum can produce 8-PN from IX, a probiotic strategy was applied to investigate whether 8-PN production could be increased in low 8-PN producers, thus balancing phytoestrogen exposure.',18567753),(5054,'mouse','decrease','P-Cresol','low-throughput','In the present study, we investigated the immunological effects of pCS derived from intestinal bacteria in mice. For this purpose, we fed mice a tyrosine-rich diet that causes the accumulation of pCS in their blood. The mice were shown to exhibit decreased Th1-driven 2, 4-dinitrofluorobenzene-induced contact hypersensitivity response. The concentration of pCS in blood was negatively correlated with the degree of the contact hypersensitivity response. In contrast, the T cell-dependent antibody response was not influenced by the accumulated pCS. We also examined the in vitro cytokine responses by T cells in the presence of pCS. The production of IFN-γ was suppressed by pCS. Further, pCS decreased the percentage of IFN-γ-producing Th1 cells. Our results suggest that intestinal bacteria-derived pCS suppressesTh1-type cellular immune responses',24161588),(5055,'mouse','increase','P-Cresol','low-throughput','In the present study, we investigated the immunological effects of pCS derived from intestinal bacteria in mice. For this purpose, we fed mice a tyrosine-rich diet that causes the accumulation of pCS in their blood. The mice were shown to exhibit decreased Th1-driven 2, 4-dinitrofluorobenzene-induced contact hypersensitivity response. The concentration of pCS in blood was negatively correlated with the degree of the contact hypersensitivity response. In contrast, the T cell-dependent antibody response was not influenced by the accumulated pCS. We also examined the in vitro cytokine responses by T cells in the presence of pCS. The production of IFN-γ was suppressed by pCS. Further, pCS decreased the percentage of IFN-γ-producing Th1 cells. Our results suggest that intestinal bacteria-derived pCS suppressesTh1-type cellular immune responses',24161588),(5056,'mouse','null','Pectin','null','The purpose of this study was to elucidate the effect of pectin on the polyamine production by defined bacterial species in vivo. Germfree male Wistar rats (n = 18) were randomly assigned to one of three treatments: (i) monoassociation with Bacteroides thetaiotaomicron + fiber-free diet; (ii) diassociation with B. thetaiotaomicron + Fusobacterium varium + fiber-free diet or (iii) diassociation with B. thetaiotaomicron + F. varium + fiber-free diet + 10% pectin. The cecal contents of monoassociated rats fed fiber-free diet contained large amounts (1.51+/-0.21 micromol/dry total cecum content) of spermidine which was the major polyamine. The cecum of diassociated rats fed the fiber-free diet contained even higher concentrations of spermidine (2.53+/-0.21 micromol/dry total cecum content) and also putrescine, which was now the dominant polyamine (putrescine 0.32+/-0.28 vs. 3.01+/-0.28 micromol/dry total cecum content; monoassociation vs. diassociation).',10801923),(5057,'mouse','null','Pectin','null','The purpose of this study was to elucidate the effect of pectin on the polyamine production by defined bacterial species in vivo. Germfree male Wistar rats (n = 18) were randomly assigned to one of three treatments: (i) monoassociation with Bacteroides thetaiotaomicron + fiber-free diet; (ii) diassociation with B. thetaiotaomicron + Fusobacterium varium + fiber-free diet or (iii) diassociation with B. thetaiotaomicron + F. varium + fiber-free diet + 10% pectin. The cecal contents of monoassociated rats fed fiber-free diet contained large amounts (1.51+/-0.21 micromol/dry total cecum content) of spermidine which was the major polyamine. The cecum of diassociated rats fed the fiber-free diet contained even higher concentrations of spermidine (2.53+/-0.21 micromol/dry total cecum content) and also putrescine, which was now the dominant polyamine (putrescine 0.32+/-0.28 vs. 3.01+/-0.28 micromol/dry total cecum content; monoassociation vs. diassociation).',10801923),(5058,'mouse','null','Pectin','null','The purpose of this study was to elucidate the effect of pectin on the polyamine production by defined bacterial species in vivo. Germfree male Wistar rats (n = 18) were randomly assigned to one of three treatments: (i) monoassociation with Bacteroides thetaiotaomicron + fiber-free diet; (ii) diassociation with B. thetaiotaomicron + Fusobacterium varium + fiber-free diet or (iii) diassociation with B. thetaiotaomicron + F. varium + fiber-free diet + 10% pectin. The cecal contents of monoassociated rats fed fiber-free diet contained large amounts (1.51+/-0.21 micromol/dry total cecum content) of spermidine which was the major polyamine. The cecum of diassociated rats fed the fiber-free diet contained even higher concentrations of spermidine (2.53+/-0.21 micromol/dry total cecum content) and also putrescine, which was now the dominant polyamine (putrescine 0.32+/-0.28 vs. 3.01+/-0.28 micromol/dry total cecum content; monoassociation vs. diassociation).',10801923),(5059,'mouse','null','Pectin','null','The purpose of this study was to elucidate the effect of pectin on the polyamine production by defined bacterial species in vivo. Germfree male Wistar rats (n = 18) were randomly assigned to one of three treatments: (i) monoassociation with Bacteroides thetaiotaomicron + fiber-free diet; (ii) diassociation with B. thetaiotaomicron + Fusobacterium varium + fiber-free diet or (iii) diassociation with B. thetaiotaomicron + F. varium + fiber-free diet + 10% pectin. The cecal contents of monoassociated rats fed fiber-free diet contained large amounts (1.51+/-0.21 micromol/dry total cecum content) of spermidine which was the major polyamine. The cecum of diassociated rats fed the fiber-free diet contained even higher concentrations of spermidine (2.53+/-0.21 micromol/dry total cecum content) and also putrescine, which was now the dominant polyamine (putrescine 0.32+/-0.28 vs. 3.01+/-0.28 micromol/dry total cecum content; monoassociation vs. diassociation).',10801923),(5088,'mouse','increase','null','low-throughput','Transverse aortic constriction (TAC) was performed to induce cardiac hypertrophy in Sprague-Dawley (SD) rats. We observed that TMAO levels were significantly elevated in SD rats after 6 weeks of TAC, suggesting the potential role of TMAO in regulating cardiac hypertrophy. In cultured cardiomyocytes, TMAO treatment stimulated cardiac hypertrophy, as indicated by increased cell area of cardiomyocytes and expression of hypertrophic markers including atrial natriuretic peptide (ANP) and beta-myosin heavy chain (β-MHC). Additionally, TMAO treatment induced cardiac hypertrophy and cardiac fibrosis in SD rats. Reducing TMAO synthesis by antibiotics (Abs) attenuated TAC-induced cardiac hypertrophy and fibrosis. Furthermore, pharmacological inhibition of Smad3 by SIS3 significantly reduced the expression of ANP and β-MHC, and cardiomyocyte cell size in TMAO-treated group. These data for the first time demonstrate that gut microbe-derived metabolite TMAO induces cardiac hypertrophy and fibrosis involving Smad3 signaling, suggesting that inhibition of gut microbes or generation of TMAO may become a potential target for the prevention and treatment of cardiac hypertrophy.',30068915),(5089,'mouse','increase','null','low-throughput','Transverse aortic constriction (TAC) was performed to induce cardiac hypertrophy in Sprague-Dawley (SD) rats. We observed that TMAO levels were significantly elevated in SD rats after 6 weeks of TAC, suggesting the potential role of TMAO in regulating cardiac hypertrophy. In cultured cardiomyocytes, TMAO treatment stimulated cardiac hypertrophy, as indicated by increased cell area of cardiomyocytes and expression of hypertrophic markers including atrial natriuretic peptide (ANP) and beta-myosin heavy chain (β-MHC). Additionally, TMAO treatment induced cardiac hypertrophy and cardiac fibrosis in SD rats. Reducing TMAO synthesis by antibiotics (Abs) attenuated TAC-induced cardiac hypertrophy and fibrosis. Furthermore, pharmacological inhibition of Smad3 by SIS3 significantly reduced the expression of ANP and β-MHC, and cardiomyocyte cell size in TMAO-treated group. These data for the first time demonstrate that gut microbe-derived metabolite TMAO induces cardiac hypertrophy and fibrosis involving Smad3 signaling, suggesting that inhibition of gut microbes or generation of TMAO may become a potential target for the prevention and treatment of cardiac hypertrophy.',30068915),(5090,'mouse','increase','Linoelaidic acid','low-throughput','Therefore, in this study, we investigated the effects of KetoA [10-oxo-12(Z)-octadecenoic acid]—a linoleic acid metabolite produced by gut lactic acid bacteria—on whole-body energy metabolism and found that dietary intake of KetoA could enhance energy expenditure in mice, thereby protecting mice from diet-induced obesity.',28754711),(5091,'mouse','increase','Linoelaidic acid','low-throughput','Ucp1 expression levels were also observed to have increased after administration of KetoA.',28754711),(5092,'mouse','increase','Linoelaidic acid','low-throughput','We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT\nmice.',28754711),(5093,'mouse','increase','Linoelaidic acid','low-throughput','We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT\nmice.',28754711),(5094,'mouse','increase','Linoelaidic acid','low-throughput','We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT\nmice.',28754711),(5095,'mouse','increase','Linoelaidic acid','low-throughput','We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT\nmice.',28754711),(5096,'mouse','increase','Linoelaidic acid','low-throughput','We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT\nmice.',28754711),(5097,'mouse','increase','(Z)-10-Hydroxy-12-octadecenoic acid','low-throughput','Therefore, in this study, we investigated the effects of KetoA [10-oxo-12(Z)-octadecenoic acid]—a linoleic acid metabolite produced by gut lactic acid bacteria—on whole-body energy metabolism and found that dietary intake of KetoA could enhance energy expenditure in mice, thereby protecting mice from diet-induced obesity.',28754711),(5098,'mouse','increase','(Z)-10-Hydroxy-12-octadecenoic acid','low-throughput','Ucp1 expression levels were also observed to have increased after administration of KetoA.',28754711),(5099,'mouse','increase','(Z)-10-Hydroxy-12-octadecenoic acid','low-throughput','We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT mice.',28754711),(5100,'mouse','increase','(Z)-10-Hydroxy-12-octadecenoic acid','low-throughput','We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT mice.',28754711),(5101,'mouse','increase','(Z)-10-Hydroxy-12-octadecenoic acid','low-throughput','We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT mice.',28754711),(5102,'mouse','increase','(Z)-10-Hydroxy-12-octadecenoic acid','low-throughput','We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT mice.',28754711),(5103,'mouse','increase','(Z)-10-Hydroxy-12-octadecenoic acid','low-throughput','We also confirmed that administration of KetoA induced not only Ucp1, but other genes that are related to BAT function, such as b3AR, Pgc1a , Cpt1b , and Tbx1  in BAT and inguinal WAT of WT mice.',28754711),(5104,'mouse','increase','null','high-throughput','It should be noted that the observed Akkermansia muciniphila-induced decrease of Fiaf is not significant. These results are partially supported by the effects of both butyrate and propionate stimulation, leading to significant increase in Fiaf expression, while the expression of Gpr43 and Pparis significantly decreased .',25118238),(5105,'mouse','decrease','null','high-throughput','Akkermansia muciniphila leads to significantly decreased expression of Gpr43 and Ppar and increased expression of Hdac3 and Hdac5.',25118238),(5106,'mouse','increase','null','high-throughput','It should be noted that the observed Akkermansia muciniphila-induced decrease of Fiaf is not significant. These results are partially supported by the effects of both butyrate and propionate stimulation, leading to significant increase in Fiaf expression, while the expression of Gpr43 and Pparis significantly decreased .',25118238),(5107,'mouse','increase','null','high-throughput','It should be noted that the observed Akkermansia muciniphila-induced decrease of Fiaf is not significant. These results are partially supported by the effects of both butyrate and propionate stimulation, leading to significant increase in Fiaf expression, while the expression of Gpr43 and Pparis significantly decreased .',25118238),(5108,'mouse','decrease','null','high-throughput','Akkermansia muciniphila leads to significantly decreased expression of Gpr43 and Ppar and increased expression of Hdac3 and Hdac5.',25118238),(5109,'mouse','increase','null','high-throughput','Acetate had no significant effect on the expression of Fiaf, Gpr43, Hdac5, or Ppar, while the expression of Hdac3 was significantly increased by acetate.',25118238),(5110,'mouse','increase','null','high-throughput','It should be noted that the observed Akkermansia muciniphila-induced decrease of Fiaf is not significant. These results are partially supported by the effects of both butyrate and propionate stimulation, leading to significant increase in Fiaf expression, while the expression of Gpr43 and Pparis significantly decreased .',25118238),(5111,'mouse','decrease','null','high-throughput','Akkermansia muciniphila leads to significantly decreased expression of Gpr43 and Ppar and increased expression of Hdac3 and Hdac5.',25118238),(5112,'mouse','increase','null','high-throughput','It should be noted that the observed Akkermansia muciniphila-induced decrease of Fiaf is not significant. These results are partially supported by the effects of both butyrate and propionate stimulation, leading to significant increase in Fiaf expression, while the expression of Gpr43 and Pparis significantly decreased .',25118238),(5113,'mouse','increase','null','high-throughput','It should be noted that the observed Akkermansia muciniphila-induced decrease of Fiaf is not significant. These results are partially supported by the effects of both butyrate and propionate stimulation, leading to significant increase in Fiaf expression, while the expression of Gpr43 and Pparis significantly decreased .',25118238),(5114,'mouse','decrease','null','high-throughput','Akkermansia muciniphila leads to significantly decreased expression of Gpr43 and Ppar and increased expression of Hdac3 and Hdac5.',25118238),(5115,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5116,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5117,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5118,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5119,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5120,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5121,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5122,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5123,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5124,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5125,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5126,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5127,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5128,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5129,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5130,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5131,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5132,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5133,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5134,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5135,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5136,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5137,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5138,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5139,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5140,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5141,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5142,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5143,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5144,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5145,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5146,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5147,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5148,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5149,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5150,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5151,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5152,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5153,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5154,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5155,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5156,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5157,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5158,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5159,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5160,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5161,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5162,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5163,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5164,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5165,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5166,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5167,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5168,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5169,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5170,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5171,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5172,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5173,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5174,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5175,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5203,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5204,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5205,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5206,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5207,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5208,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5209,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5210,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5211,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5212,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5213,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5214,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5215,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5216,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5217,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5218,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5219,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5220,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5221,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5222,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5223,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5224,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5225,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5226,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5227,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5228,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5229,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5230,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5231,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5232,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5233,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5234,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5235,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5236,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5237,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5238,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5239,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5240,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5241,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5242,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5243,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5244,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5245,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5246,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5247,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5248,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5249,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5250,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5252,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5253,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5254,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5255,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5256,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5257,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5258,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5260,'mouse','increase','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5262,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5264,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5265,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5266,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5267,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5268,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5269,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5270,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5272,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5273,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5274,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5275,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5276,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5277,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5278,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5280,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5281,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5282,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5283,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5284,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5285,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5286,'mouse','decrease','null','high-throughput','To analyze the effect of microbiota and SCFA on the main regulators of gene expression in the intestinal epithelium, we focused on the effects of these conditions on transcription factor expression (Fig. 3A). We observed that major overlap of effects on transcription factor expression was evident after treatment with Akkermansia muciniphila and propionate and butyrate. However, butyrate additionally affected expression of specific transcription factors in the intestinal epithelium of organoids not affected by other conditions (Fig. 3A). The expression of a very small numberof transcription factors was affected specifically by exposure of organoids to acetate or propionate (Fig. 3A).',25118238),(5304,'mouse','null','null','null','The results clearly showed that the high dose of PIPA could improve the intestinal movement and alleviate constipation symptoms in AD induced constipation mice.',30846982),(5305,'mouse','null','null','null','The results showed that the AD groups had lower levels of ACh and 5-HT in colonic tissue and that the H-PIPA dose induced an increase in both colonic ACh and 5-HT but without significant differences from those in the AD group.',30846982),(5306,'mouse','null','null','null','The results showed that the AD groups had lower levels of ACh and 5-HT in colonic tissue and that the H-PIPA dose induced an increase in both colonic ACh and 5-HT but without significant differences from those in the AD group.',30846982),(5307,'mouse','increase','null','low-throughput','Butyrate is a short-chain fatty acid derived from the metabolism of indigestible carbohydrates by the gut microbiota. Butyrate contributes to gut homeostasis, but it may also control inflammatory responses and host physiology in other tissues. Butyrate inhibits histone deacetylases, thereby affecting gene transcription, and also signals through the metabolite-sensing G protein receptor (GPR)109a.',31366236),(5308,'mouse','increase','null','low-throughput','The protective effect of butyrate was further confirmed in situ by enhanced expression of WT-1 (Fig. 1D),podocin, and podoplanin (Supplemental Fig. S3). In ad_x0002_dition, analysis of glomeruli ultrastructure revealed a well-preserved podocytelayer surrounding the GBM with presence of slit diaphragm and reduced FP effacement in butyrate-treated mice at early and later time points (Fig.1E). Over 60 d, all butyrate-treated mice survived ADRinduced nephropathy, compared with only about 20% of nontreated mice (Fig. 1F).',31366236),(5309,'mouse','decrease','null','low-throughput','We also assessed the gene expression levels of important fibrotic markers such as Tgf-b, Fsp1, ActaII, col4a1, and the enzymes involved in matrix remodeling, Mmp9 and Timp1 (Fig. 3C). Butyrate-treated mice showed lower levels of fibrotic markers both at d 7 and at d 14 compared with nontreated mice.',31366236),(5310,'mouse','decrease','null','low-throughput','We also assessed the gene expression levels of important fibrotic markers such as Tgf-b, Fsp1, ActaII, col4a1, and the enzymes involved in matrix remodeling, Mmp9 and Timp1 (Fig. 3C). Butyrate-treated mice showed lower levels of fibrotic markers both at d 7 and at d 14 compared with nontreated mice.',31366236),(5311,'mouse','decrease','null','low-throughput','We also assessed the gene expression levels of important fibrotic markers such as Tgf-b, Fsp1, ActaII, col4a1, and the enzymes involved in matrix remodeling, Mmp9 and Timp1 (Fig. 3C). Butyrate-treated mice showed lower levels of fibrotic markers both at d 7 and at d 14 compared with nontreated mice.',31366236),(5312,'mouse','decrease','null','low-throughput','We also assessed the gene expression levels of important fibrotic markers such as Tgf-b, Fsp1, ActaII, col4a1, and the enzymes involved in matrix remodeling, Mmp9 and Timp1 (Fig. 3C). Butyrate-treated mice showed lower levels of fibrotic markers both at d 7 and at d 14 compared with nontreated mice.',31366236),(5313,'mouse','decrease','null','low-throughput','We also assessed the gene expression levels of important fibrotic markers such as Tgf-b, Fsp1, ActaII, col4a1, and the enzymes involved in matrix remodeling, Mmp9 and Timp1 (Fig. 3C). Butyrate-treated mice showed lower levels of fibrotic markers both at d 7 and at d 14 compared with nontreated mice.',31366236),(5314,'mouse','decrease','null','low-throughput','Renal inflammation is a common finding in ADR nephropathy. Injured resident kidney cells release chemokines that promote the recruitment ofmacrophages, which play a pivotal role as disease progresses. CD68 staining in renal cortex showed, at early stages of the disease, a slightly positive staining in nontreated mice, whereby we identified several foci of infiltrating CD68-positive cells at d 14 surrounding the glomeruli (Fig. 4A). Butyrate treatment significantly reduced the infiltration of CD68-positive cells, as well as gene expression of chemokines such as ccl2 and ccl3 and important proinflammatory cytokines such as Tnf-a and Il-1b in renal tissue (Fig. 4B). Protein levels for the proinflammatory cytokines in renal tissue were also significantly reduced by butyrate treatment (Fig. 4C).',31366236),(5315,'mouse','decrease','null','low-throughput','Renal inflammation is a common finding in ADR nephropathy. Injured resident kidney cells release chemokines that promote the recruitment ofmacrophages, which play a pivotal role as disease progresses. CD68 staining in renal cortex showed, at early stages of the disease, a slightly positive staining in nontreated mice, whereby we identified several foci of infiltrating CD68-positive cells at d 14 surrounding the glomeruli (Fig. 4A). Butyrate treatment significantly reduced the infiltration of CD68-positive cells, as well as gene expression of chemokines such as ccl2 and ccl3 and important proinflammatory cytokines such as Tnf-a and Il-1b in renal tissue (Fig. 4B). Protein levels for the proinflammatory cytokines in renal tissue were also significantly reduced by butyrate treatment (Fig. 4C).',31366236),(5316,'mouse','decrease','null','low-throughput','Renal inflammation is a common finding in ADR nephropathy. Injured resident kidney cells release chemokines that promote the recruitment ofmacrophages, which play a pivotal role as disease progresses. CD68 staining in renal cortex showed, at early stages of the disease, a slightly positive staining in nontreated mice, whereby we identified several foci of infiltrating CD68-positive cells at d 14 surrounding the glomeruli (Fig. 4A). Butyrate treatment significantly reduced the infiltration of CD68-positive cells, as well as gene expression of chemokines such as ccl2 and ccl3 and important proinflammatory cytokines such as Tnf-a and Il-1b in renal tissue (Fig. 4B). Protein levels for the proinflammatory cytokines in renal tissue were also significantly reduced by butyrate treatment (Fig. 4C).',31366236),(5317,'mouse','decrease','null','low-throughput','Renal inflammation is a common finding in ADR nephropathy. Injured resident kidney cells release chemokines that promote the recruitment ofmacrophages, which play a pivotal role as disease progresses. CD68 staining in renal cortex showed, at early stages of the disease, a slightly positive staining in nontreated mice, whereby we identified several foci of infiltrating CD68-positive cells at d 14 surrounding the glomeruli (Fig. 4A). Butyrate treatment significantly reduced the infiltration of CD68-positive cells, as well as gene expression of chemokines such as ccl2 and ccl3 and important proinflammatory cytokines such as Tnf-a and Il-1b in renal tissue (Fig. 4B). Protein levels for the proinflammatory cytokines in renal tissue were also significantly reduced by butyrate treatment (Fig. 4C).',31366236),(5318,'mouse','increase','null','low-throughput','In terestingly, we identified 5 genes that had their expression switched after treatment with butyrate: Csrp2bp,Aurkc, Dot1L, Kat5, and Hdac6.',31366236),(5319,'mouse','increase','null','low-throughput','In terestingly, we identified 5 genes that had their expression switched after treatment with butyrate: Csrp2bp,Aurkc, Dot1L, Kat5, and Hdac6.',31366236),(5320,'mouse','increase','null','low-throughput','In terestingly, we identified 5 genes that had their expression switched after treatment with butyrate: Csrp2bp,Aurkc, Dot1L, Kat5, and Hdac6.',31366236),(5321,'mouse','increase','null','low-throughput','In terestingly, we identified 5 genes that had their expression switched after treatment with butyrate: Csrp2bp,Aurkc, Dot1L, Kat5, and Hdac6.',31366236),(5322,'mouse','increase','null','low-throughput','In terestingly, we identified 5 genes that had their expression switched after treatment with butyrate: Csrp2bp,Aurkc, Dot1L, Kat5, and Hdac6.',31366236),(5323,'mouse','increase','null','low-throughput','To test this hypothesis, we evaluated the histone modifications H3K27Ac and H3K4me3, which are typically regarded as markers that lead to transcriptional activation, andH3K9me3,which correlates to transcriptional repression (Fig. 5F). ADR led to significant reductions of the activation marker H3K27Ac at the promoter region of both Synpo and Nphs1 genes, which was recovered in the presence of butyrate. In parallel, the increase of the repressive marker H3K9me3 that is due to ADR at the promoter region on both Synpo and Kirrel genes was significantly reduced in the presence of butyrate. Indeed, podocytes grown in vitro showed preserved expression of synaptopodin (Fig. 5G) after treatmentwith butyrate in the presence of ADR.',31366236),(5324,'mouse','increase','null','low-throughput','To test this hypothesis, we evaluated the histone modifications H3K27Ac and H3K4me3, which are typically regarded as markers that lead to transcriptional activation, andH3K9me3,which correlates to transcriptional repression (Fig. 5F). ADR led to significant reductions of the activation marker H3K27Ac at the promoter region of both Synpo and Nphs1 genes, which was recovered in the presence of butyrate. In parallel, the increase of the repressive marker H3K9me3 that is due to ADR at the promoter region on both Synpo and Kirrel genes was significantly reduced in the presence of butyrate. Indeed, podocytes grown in vitro showed preserved expression of synaptopodin (Fig. 5G) after treatmentwith butyrate in the presence of ADR.',31366236),(5325,'mouse','increase','null','low-throughput','To test this hypothesis, we evaluated the histone modifications H3K27Ac and H3K4me3, which are typically regarded as markers that lead to transcriptional activation, andH3K9me3,which correlates to transcriptional repression (Fig. 5F). ADR led to significant reductions of the activation marker H3K27Ac at the promoter region of both Synpo and Nphs1 genes, which was recovered in the presence of butyrate. In parallel, the increase of the repressive marker H3K9me3 that is due to ADR at the promoter region on both Synpo and Kirrel genes was significantly reduced in the presence of butyrate. Indeed, podocytes grown in vitro showed preserved expression of synaptopodin (Fig. 5G) after treatmentwith butyrate in the presence of ADR.',31366236),(5326,'mouse','decrease','null','low-throughput','Podocytes grown in vitro in the presence of ADR exhibited significantly increased TRPC5 and TRPC6 gene transcriptions, which were abolished when podocytes were incubated with buty rate.',31366236),(5327,'mouse','decrease','null','low-throughput','Podocytes grown in vitro in the presence of ADR exhibited significantly increased TRPC5 and TRPC6 gene transcriptions, which were abolished when podocytes were incubated with buty rate.',31366236),(5328,'mouse','increase','null','low-throughput','Here, the presence of ADR in podocytes grown in vitro decreased the expression levels of RHOA, RAC1, and CDC42, whereas treatmentwith butyrate significantly recovered the levels of both RAC1 and CDC42 but not RHOA.',31366236),(5329,'mouse','increase','null','low-throughput','Here, the presence of ADR in podocytes grown in vitro decreased the expression levels of RHOA, RAC1, and CDC42, whereas treatmentwith butyrate significantly recovered the levels of both RAC1 and CDC42 but not RHOA.',31366236),(5330,'mouse','decrease','Ellagic acid','low-throughput','Thus, the gut bacterial metabolite UA increases miR-10a-5p levels thereby downregulating Orai1/STIM1/STIM2 expression, store operated Ca2+ entry, and proliferation of murine CD4+ T cells.',31417547),(5331,'mouse','decrease','Ellagic acid','low-throughput','Thus, the gut bacterial metabolite UA increases miR-10a-5p levels thereby downregulating Orai1/STIM1/STIM2 expression, store operated Ca2+ entry, and proliferation of murine CD4+ T cells.',31417547),(5332,'mouse','decrease','Ellagic acid','low-throughput','Thus, the gut bacterial metabolite UA increases miR-10a-5p levels thereby downregulating Orai1/STIM1/STIM2 expression, store operated Ca2+ entry, and proliferation of murine CD4+ T cells.',31417547),(5333,'mouse','increase','Ellagic acid','low-throughput','Thus, the present observations uncover a novel action of UA, i.e., the upregulation of miR-10a-5p with subsequent downregulation of store operated Ca2+ influx in CD4+ T cells.',31417547),(5337,'mouse','decrease','Cholesterol','low-throughput','Altered substrate oxidation and fecal fat excretion in LHFD-fed GF mice is linked to decreased hepatic Cyp7a1 and Nr1h4 expression.',27900259),(5338,'mouse','decrease','Cholesterol','low-throughput','Altered substrate oxidation and fecal fat excretion in LHFD-fed GF mice is linked to decreased hepatic Cyp7a1 and Nr1h4 expression.',27900259),(5339,'mouse','decrease','Linoleic acid','low-throughput','10-Hydroxy-cis-12-octadecenoic acid (HYA), a gut microbial metabolite of linoleic acid, suppressed TNF and dextran sulfate sodium-induced changes in the expression of TJ-related molecules, occludin, zonula occludens-1, and myosin light chain kinase. HYA also suppressed the expression of TNF receptor 2 (TNFR2) mRNA and protein expression in Caco-2 cells and colonic tissue. In addition, HYA suppressed the protein expression of TNFR2 in murine intestinal epithelial cells. Furthermore, HYA significantly up-regulated G protein-coupled receptor (GPR) 40 expression in Caco-2 cells.',25505251),(5340,'mouse','decrease','Linoleic acid','low-throughput','10-Hydroxy-cis-12-octadecenoic acid (HYA), a gut microbial metabolite of linoleic acid, suppressed TNF and dextran sulfate sodium-induced changes in the expression of TJ-related molecules, occludin, zonula occludens-1, and myosin light chain kinase. HYA also suppressed the expression of TNF receptor 2 (TNFR2) mRNA and protein expression in Caco-2 cells and colonic tissue. In addition, HYA suppressed the protein expression of TNFR2 in murine intestinal epithelial cells. Furthermore, HYA significantly up-regulated G protein-coupled receptor (GPR) 40 expression in Caco-2 cells.',25505251),(5341,'mouse','increase','Linoleic acid','low-throughput','10-Hydroxy-cis-12-octadecenoic acid (HYA), a gut microbial metabolite of linoleic acid, suppressed TNF and dextran sulfate sodium-induced changes in the expression of TJ-related molecules, occludin, zonula occludens-1, and myosin light chain kinase. HYA also suppressed the expression of TNF receptor 2 (TNFR2) mRNA and protein expression in Caco-2 cells and colonic tissue. In addition, HYA suppressed the protein expression of TNFR2 in murine intestinal epithelial cells. Furthermore, HYA significantly up-regulated G protein-coupled receptor (GPR) 40 expression in Caco-2 cells.',25505251),(5342,'mouse','decrease','Linoleic acid','low-throughput','IFN-TNF-induced an increase in FD-4 permeability;however, the pretreatment of HYA and linoleic acid signififcantly suppressed this increase (Fig. 2C). In contrast, the pretreatment of HYB did not alter IFN-TNF-induced changes in TER and FD-4 permeability. As shown in Fig. 2D,IFN-TNF-induced a marked increase in the secretion of IL-8, which is a major pro-inflammatory cytokine. However,the pretreatment of HYA significantly suppressed IL-8 secretion.',25505251),(5343,'mouse','increase','Linoleic acid','low-throughput','Pretreatment of HYA, but not HYB, significantly restored IFN-TNF-induced changes in the expression of occludin, ZO-1, claudin-1, and MLCK.',25505251),(5344,'mouse','increase','Linoleic acid','low-throughput','Pretreatment of HYA, but not HYB, significantly restored IFN-TNF-induced changes in the expression of occludin, ZO-1, claudin-1, and MLCK.',25505251),(5345,'mouse','increase','Linoleic acid','low-throughput','Pretreatment of HYA, but not HYB, significantly restored IFN-TNF-induced changes in the expression of occludin, ZO-1, claudin-1, and MLCK.',25505251),(5346,'mouse','decrease','Linoleic acid','low-throughput','Pretreatment of HYA, but not HYB, significantly restored IFN-TNF-induced changes in the expression of occludin, ZO-1, claudin-1, and MLCK.',25505251),(5347,'mouse','decrease','Linoleic acid','low-throughput','The pretreatment of HYA suppressed IFN-induced up-regulation of TNFR2 (but not TNFR1), and it subsequently decreased the ratio of phospho-INF-B p65 expression induced by cytokines (Fig. 3, B and C). In contrast, pretreatment of HYB did not show the same inhibitory effects as HYA.',25505251),(5348,'mouse','increase','Linoleic acid','low-throughput','It was noteworthy that the pretreatment of HYA augmented the mRNA expression of GPR40 and GPR43 but not GPR120, PPAR, and GPR41 in Caco-2 cells (Fig. 4A). However, the pretreatment of HYB did not up-regulate GPR40 and GPR43 expression (Fig. 4A).',25505251),(5349,'mouse','decrease','null','low-throughput','The expression levels of IL�? (Fig. 5A and B) and its receptors CXCR1 (Fig. 5A and C) and CXCR2 (Fig. 5A and D) were significantly increased in the non‑infarct LV of MI + VEH rats compared with the LV of SHAM + VEH rats; however, the expression of these proteins was normalized by DMB treatment.',31180562),(5350,'mouse','decrease','null','low-throughput','The expression levels of IL�? (Fig. 5A and B) and its receptors CXCR1 (Fig. 5A and C) and CXCR2 (Fig. 5A and D) were significantly increased in the non‑infarct LV of MI + VEH rats compared with the LV of SHAM + VEH rats; however, the expression of these proteins was normalized by DMB treatment.',31180562),(5351,'mouse','decrease','null','low-throughput','The expression levels of IL�? (Fig. 5A and B) and its receptors CXCR1 (Fig. 5A and C) and CXCR2 (Fig. 5A and D) were significantly increased in the non‑infarct LV of MI + VEH rats compared with the LV of SHAM + VEH rats; however, the expression of these proteins was normalized by DMB treatment.',31180562),(5352,'mouse','increase','null','low-throughput','Accordingly, in livers from TLR2-defcient mice, the expression of SASP factors, such as IL1β,  Groα, and IL6, was significantly reduced in HSCs, suggesting that TLR2-mediated signaling induced the expression of SASP factors (Fig. 2F). Notably, however, the presence of p21 and 53BP1 foci was still observed in activated HSCs from TLR2-deficient mice (Fig. 2F). The majority of senescent \nsignals are detected in the HSCs but not in HCC cells in the tumor region (Supplementary Fig. S2). Taken together, these results suggest that TLR2-mediated signaling is important for the upregulation of SASP factors in senescent HSCs but not for the induction of senescent cell-cycle arrest.',28202625),(5353,'mouse','increase','null','low-throughput','Accordingly, in livers from TLR2-defcient mice, the expression of SASP factors, such as IL1β,  Groα, and IL6, was significantly reduced in HSCs, suggesting that TLR2-mediated signaling induced the expression of SASP factors (Fig. 2F). Notably, however, the presence of p21 and 53BP1 foci was still observed in activated HSCs from TLR2-deficient mice (Fig. 2F). The majority of senescent \nsignals are detected in the HSCs but not in HCC cells in the tumor region (Supplementary Fig. S2). Taken together, these results suggest that TLR2-mediated signaling is important for the upregulation of SASP factors in senescent HSCs but not for the induction of senescent cell-cycle arrest.',28202625),(5354,'mouse','increase','null','low-throughput','Accordingly, in livers from TLR2-defcient mice, the expression of SASP factors, such as IL1β,  Groα, and IL6, was significantly reduced in HSCs, suggesting that TLR2-mediated signaling induced the expression of SASP factors (Fig. 2F). Notably, however, the presence of p21 and 53BP1 foci was still observed in activated HSCs from TLR2-deficient mice (Fig. 2F). The majority of senescent \nsignals are detected in the HSCs but not in HCC cells in the tumor region (Supplementary Fig. S2). Taken together, these results suggest that TLR2-mediated signaling is important for the upregulation of SASP factors in senescent HSCs but not for the induction of senescent cell-cycle arrest.',28202625),(5355,'mouse','increase','null','low-throughput','Consistent with our previous report, the expression of the senescence inducers p21 (encoded by Cdkn1a) and p16 (encoded by Cdkn2a) was upregulated by DCA treatment (Fig. 3B). Notably, the expression levels of TLR2 and SASP factors, such as IL1β, Groα, and IL6, were significantly enhanced by LTA in DCA-induced senescent HSCs (Fig. 3C).',28202625),(5356,'mouse','increase','null','low-throughput','Consistent with our previous report, the expression of the senescence inducers p21 (encoded by Cdkn1a) and p16 (encoded by Cdkn2a) was upregulated by DCA treatment (Fig. 3B). Notably, the expression levels of TLR2 and SASP factors, such as IL1β, Groα, and IL6, were significantly enhanced by LTA in DCA-induced senescent HSCs (Fig. 3C).',28202625),(5357,'mouse','increase','null','low-throughput','The expression of COX2, but not COX1, in DCA-treated primary HSCs was significantly induced by increasing amounts of LTA (Fig. 4C), and the COX2 expression was dependent on TLR2 (Supplementary \nFig. S3). Indeed, we confirmed that TLR2 was expressed in HSCs and this expression was enhanced by LTA alone or DCA plus LTA treatment (Supplementary Fig. S8A).',28202625),(5358,'mouse','decrease','null','low-throughput','The production of IFNγ and TNFα, which are known to activate antitumor immune responses and inhibit cancer cell growth, was significantly and specifically attenuated by PGE2.',28202625),(5359,'mouse','decrease','null','low-throughput','The production of IFNγ and TNFα, which are known to activate antitumor immune responses and inhibit cancer cell growth, was significantly and specifically attenuated by PGE2.',28202625),(5360,'mouse','increase','null','low-throughput','In contrast, the production of IL10, a potent anti-inflammatory cytokine, and IL6, a known tumor-promoting cytokine, in immune cells was slightly but significantly enhanced (Supplementary Fig. S8D).',28202625),(5361,'mouse','increase','null','low-throughput','In contrast, the production of IL10, a potent anti-inflammatory cytokine, and IL6, a known tumor-promoting cytokine, in immune cells was slightly but significantly enhanced (Supplementary Fig. S8D).',28202625),(5362,'mouse','increase','null','low-throughput','PGE2 is known to be recognized by four types of receptors: PTGER1, PTGER2, PTGER3, and PTGER4 (37, 38). In our mouse model, the expression of PTGER4, but not other PGE2 receptors, was significantly upregulated in tumor tissues (Fig. 5C),',28202625),(5363,'mouse','increase','null','low-throughput','Indole 3-propionic acid (IPA), in the context of indole, is a ligand for PXR in vivo, and IPA downregulated enterocyte TNF-a while it upregulated junctional protein-coding mRNAs. PXR_deficient mice showed a distinctly ‘‘leaky’’gut physiology coupled with upregulation of the Toll-like receptor (TLR) signaling pathway.',25065623),(5364,'mouse','decrease','null','low-throughput','Indole 3-propionic acid (IPA), in the context of indole, is a ligand for PXR in vivo, and IPA downregulated enterocyte TNF-a while it upregulated junctional protein-coding mRNAs. PXR_deficient mice showed a distinctly ‘‘leaky’’gut physiology coupled with upregulation of the Toll-like receptor (TLR) signaling pathway.',25065623),(5365,'mouse','increase','null','low-throughput','Indole 3-propionic acid (IPA), in the context of indole, is a ligand for PXR in vivo, and IPA downregulated enterocyte TNF-a while it upregulated junctional protein-coding mRNAs. PXR_deficient mice showed a distinctly ‘‘leaky’’gut physiology coupled with upregulation of the Toll-like receptor (TLR) signaling pathway.',25065623),(5366,'mouse','increase','null','low-throughput','For critical validation of the ex vivo experiments demonstrating IPA effects on junctional regulators, we coadministered C. sporogenes to germ-free mice in the presence or absence of\nL-tryptophan (Figure 1D). We verified that C. sporogenes inoculation led to production of IPA in vivo (thus, it was assumed that indoles were present) (Figure 1E). Germ-free mice exposed to\nC. sporogenes had a significant reduction in FITC-dextran recovery from the serum and this was further reduced in the presence of L-tryptophan dosing (Figure 1F). The mice intestinal mucosa\nexposed to C. sporogenes demonstrated significant induction of PXR target genes (Mdr1, Cyp3a11, Ugt1a1), which was further augmented in the presence of L-tryptophan (Figure 1G).',25065623),(5367,'mouse','increase','null','low-throughput','For critical validation of the ex vivo experiments demonstrating IPA effects on junctional regulators, we coadministered C. sporogenes to germ-free mice in the presence or absence of\nL-tryptophan (Figure 1D). We verified that C. sporogenes inoculation led to production of IPA in vivo (thus, it was assumed that indoles were present) (Figure 1E). Germ-free mice exposed to\nC. sporogenes had a significant reduction in FITC-dextran recovery from the serum and this was further reduced in the presence of L-tryptophan dosing (Figure 1F). The mice intestinal mucosa\nexposed to C. sporogenes demonstrated significant induction of PXR target genes (Mdr1, Cyp3a11, Ugt1a1), which was further augmented in the presence of L-tryptophan (Figure 1G).',25065623),(5368,'mouse','increase','null','low-throughput','For critical validation of the ex vivo experiments demonstrating IPA effects on junctional regulators, we coadministered C. sporogenes to germ-free mice in the presence or absence of\nL-tryptophan (Figure 1D). We verified that C. sporogenes inoculation led to production of IPA in vivo (thus, it was assumed that indoles were present) (Figure 1E). Germ-free mice exposed to\nC. sporogenes had a significant reduction in FITC-dextran recovery from the serum and this was further reduced in the presence of L-tryptophan dosing (Figure 1F). The mice intestinal mucosa\nexposed to C. sporogenes demonstrated significant induction of PXR target genes (Mdr1, Cyp3a11, Ugt1a1), which was further augmented in the presence of L-tryptophan (Figure 1G).',25065623),(5369,'mouse','decrease','N-Trimethyllysine','low-throughput','We found TMAVA to bind and inhibit BBOX, reducing synthesis of carnitine.We found the intestinal bacteria Enterococcus faecalis and Pseudomonas aeruginosa to metabolize trimethyllysine to TMAVA.',32105727),(5370,'mouse','decrease','N-Trimethyllysine','low-throughput','We found TMAVA to bind and inhibit BBOX, reducing synthesis of carnitine.We found the intestinal bacteria Enterococcus faecalis and Pseudomonas aeruginosa to metabolize trimethyllysine to TMAVA.',32105727),(5371,'mouse','null','null','null','In addition, Parabacteroides_sp_HGS0025 positively correlated with butyrate, IgM, IL-10, and TNF-α products in intestine and mice blood, respectively.',32014483),(5375,'mouse','increase','beta-D-Glucopyranose, 1,3,6-tris(3,4,5-trihydroxybenzoate)','low-throughput','We found that UB treatment resulted in significant p62 accumulation.',31996327),(5376,'mouse','increase','beta-D-Glucopyranose, 1,3,6-tris(3,4,5-trihydroxybenzoate)','low-throughput','The nuclear Nrf2 level was increased with UB treatment.',31996327),(5377,'mouse','increase','beta-D-Glucopyranose, 1,3,6-tris(3,4,5-trihydroxybenzoate)','low-throughput','UB treatment significantly enhanced nuclear Nrf2 level and its downstream genes, HO-1, NQO1 and GSTP1 in the rat heart subjected to IR.',31996327),(5378,'mouse','increase','beta-D-Glucopyranose, 1,3,6-tris(3,4,5-trihydroxybenzoate)','low-throughput','UB treatment significantly enhanced nuclear Nrf2 level and its downstream genes, HO-1, NQO1 and GSTP1 in the rat heart subjected to IR.',31996327),(5379,'mouse','increase','beta-D-Glucopyranose, 1,3,6-tris(3,4,5-trihydroxybenzoate)','low-throughput','UB treatment significantly enhanced nuclear Nrf2 level and its downstream genes, HO-1, NQO1 and GSTP1 in the rat heart subjected to IR.',31996327),(5380,'mouse','null','null','null','The amino acid metabolite, N-delta-acetylornithine, was notably increased by B. longum rice bran fermentation when compared to non-fermented rice bran in food, colon, and blood.',31965839),(5381,'mouse','increase','Mogroside V','low-throughput','Only 11-oxo-mogrol could increase the mRNA expression of both Camk2α and Calbindin genes',31900523),(5382,'mouse','increase','Mogroside V','low-throughput','Only 11-oxo-mogrol could increase the mRNA expression of both Camk2α and Calbindin genes',31900523),(5383,'mouse','increase','null','low-throughput','Treatment of naive T cells under the Treg-cell-polarizing conditions with butyrate enhanced histone H3 acetylation in the promoter and conserved non-coding sequence regions of the Foxp3 locus, suggesting a possible mechanism for how microbial-derived butyrate regulates the differentiation of Treg cells.',24226770),(5384,'mouse','increase','null','low-throughput','Treatment of naive T cells under the Treg-cell-polarizing conditions with butyrate enhanced histone H3 acetylation in the promoter and conserved non-coding sequence regions of the Foxp3 locus, suggesting a possible mechanism for how microbial-derived butyrate regulates the differentiation of Treg cells.',24226770),(5385,'mouse','null','Tryptophan','null','Bt and B. ovatus produced indole in both media conditions.',27916477),(5386,'mouse','null','Tryptophan','null','Bt and B. ovatus produced indole in both media conditions.',27916477),(5387,'mouse','increase','null','low-throughput','We found that butyrate enhanced phosphorylated-STAT3, mTOR, and ERK1/2 as evidenced by Western blots.',30177845),(5388,'mouse','increase','null','low-throughput','We found that butyrate enhanced phosphorylated-STAT3, mTOR, and ERK1/2 as evidenced by Western blots.',30177845),(5389,'mouse','increase','null','low-throughput','Mechanistically, SCFAs activate Th1 cell STAT3 and mTOR, and consequently upregulate transcription factor B lymphocyte-induced maturation protein 1 (Blimp-1), which mediates SCFA-induction of IL-10.',30177845),(5390,'mouse','increase','null','low-throughput','Mechanistically, SCFAs activate Th1 cell STAT3 and mTOR, and consequently upregulate transcription factor B lymphocyte-induced maturation protein 1 (Blimp-1), which mediates SCFA-induction of IL-10.',30177845),(5391,'mouse','increase','null','low-throughput','SCFA Propionate Induced Sympathetic Activation via GPR41-Gβγ in Sympathetic Neurons.',21518883),(5392,'mouse','increase','Linolenic acid','low-throughput','Treatments of BMDCs with the fatty acids significantly increased mRNA expression of Arg1 in a dose-dependent manner in comparison with that of IL-4 treatment alone.',28904023),(5393,'mouse','increase','Linolenic acid','low-throughput','Treatments of BMDCs with the fatty acids significantly increased mRNA expression of Arg1 in a dose-dependent manner in comparison with that of IL-4 treatment alone.',28904023),(5399,'mouse','increase','null','low-throughput','AM attenuates ALS in mice by increasing NAM levels',31330533),(5400,'mouse','increase','null','low-throughput','AM attenuates ALS in mice by increasing NAM levels',31330533),(5406,'mouse','decrease','null','low-throughput','Butyrate treatment significantly decreased expression of the CD11b.',31835282),(5407,'mouse','increase','Tryptophan','low-throughput','Here, we show that the biological effects of tryptamine are mediated through the 5-HT4 receptor (5-HT4R), a G-protein-coupled receptor (GPCR) uniquely expressed in the colonic epithelium.',29902441),(5408,'mouse','increase','Tryptophan','low-throughput','Here, we show that the biological effects of tryptamine are mediated through the 5-HT4 receptor (5-HT4R), a G-protein-coupled receptor (GPCR) uniquely expressed in the colonic epithelium.',29902441),(5409,'mouse','decrease','null','high-throughput','UroA or UAS03 treatment significantly reduced the LPS-induced increase in serum IL-6 and TNF-α levels (Fig. 1c). These results suggest that UAS03 is a potent structural analogue of UroA with increased anti-inflammatory activities.',30626868),(5410,'mouse','decrease','null','high-throughput','UroA or UAS03 treatment significantly reduced the LPS-induced increase in serum IL-6 and TNF-α levels (Fig. 1c). These results suggest that UAS03 is a potent structural analogue of UroA with increased anti-inflammatory activities.',30626868),(5411,'mouse','increase','null','high-throughput','UroA/UAS03 treatment attenuates TNBS-induced colitis in mice. Colitis was induced by intrarectal administration of TNBS (2.5 mg/mouse) in C57BL/6 (8 week age old, n = 5/group) mice. Mice were orally treated with vehicle or UroA (20 mg/kg) or UAS03 (20 mg/kg body weight) every 12 h post-TNBS instillation for 60 h and the experiment terminated at 72 h. Representative data from one of three independent experiments is shown. a Percent body weight loss (No TNBS- Solid black line; Veh + TNBS- Solid red line; UroA + TNBS- Solid blue line; UAS03 + TNBS- Solid purple line). b disease activity index, c intestinal permeability, d colon lengths were measured. e Gross morphological changes of colon, f ratio of colon weight/length, g colonic myeloperoxidase (MPO) levels, h serum IL-6, TNF-α, CXCL1, and IL-1β levels, i microphotographs of hematoxylin and eosin (H&E) stained sections of colons and inflammation scores are shown. Scale bar indicates 300 μm. j Cldn4 expression in the colons of these mice (n = 3) was measured by immunoblots and quantified. Statistical analysis was performed (unpaired t-test) using Graphpad Prism software. Error bars, ±SEM ***p < 0.001; **p < 0.01\n*p < 0.05. Source Data are provided as a Source Data File.',30626868),(5412,'mouse','increase','null','high-throughput','UroA/UAS03 treatment attenuates TNBS-induced colitis in mice. Colitis was induced by intrarectal administration of TNBS (2.5 mg/mouse) in C57BL/6 (8 week age old, n = 5/group) mice. Mice were orally treated with vehicle or UroA (20 mg/kg) or UAS03 (20 mg/kg body weight) every 12 h post-TNBS instillation for 60 h and the experiment terminated at 72 h. Representative data from one of three independent experiments is shown. a Percent body weight loss (No TNBS- Solid black line; Veh + TNBS- Solid red line; UroA + TNBS- Solid blue line; UAS03 + TNBS- Solid purple line). b disease activity index, c intestinal permeability, d colon lengths were measured. e Gross morphological changes of colon, f ratio of colon weight/length, g colonic myeloperoxidase (MPO) levels, h serum IL-6, TNF-α, CXCL1, and IL-1β levels, i microphotographs of hematoxylin and eosin (H&E) stained sections of colons and inflammation scores are shown. Scale bar indicates 300 μm. j Cldn4 expression in the colons of these mice (n = 3) was measured by immunoblots and quantified. Statistical analysis was performed (unpaired t-test) using Graphpad Prism software. Error bars, ±SEM ***p < 0.001; **p < 0.01\n*p < 0.05. Source Data are provided as a Source Data File.',30626868),(5416,'mouse','increase','null','low-throughput','Interestingly, metabolites produced by gut microbiota, including propionate, butyrate, and the secondary bile acid deoxycolate directly induced expression of tryptophan hydroxylase 1 (Tph1).',28961041),(5417,'mouse','increase','null','low-throughput','Interestingly, metabolites produced by gut microbiota, including propionate, butyrate, and the secondary bile acid deoxycolate directly induced expression of tryptophan hydroxylase 1 (Tph1).',28961041),(5418,'mouse','increase','null','low-throughput','Interestingly, metabolites produced by gut microbiota, including propionate, butyrate, and the secondary bile acid deoxycolate directly induced expression of tryptophan hydroxylase 1 (Tph1).',28961041),(5426,'mouse','null','null','null','CB0313.1 upregulates butyrate production.',28765642),(5432,'mouse','increase','Berberine','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5433,'mouse','increase','Berberine','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5434,'mouse','increase','Berberine','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5435,'mouse','increase','Berberine','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5436,'mouse','increase','Berberine','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5437,'mouse','decrease','Berberine','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5438,'mouse','decrease','Berberine','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5439,'mouse','decrease','Berberine','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5440,'mouse','decrease','Berberine','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5441,'mouse','decrease','Berberine','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5442,'mouse','increase','Berberine','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5443,'mouse','increase','Berberine','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5444,'mouse','increase','Berberine','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5445,'mouse','increase','Berberine','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5446,'mouse','increase','Berberine','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5447,'mouse','decrease','Berberine','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5448,'mouse','decrease','Berberine','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5449,'mouse','decrease','Berberine','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5450,'mouse','decrease','Berberine','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5451,'mouse','decrease','Berberine','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5452,'mouse','increase','Berberine','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5453,'mouse','increase','Berberine','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5454,'mouse','increase','Berberine','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5455,'mouse','increase','Berberine','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5456,'mouse','increase','Berberine','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5457,'mouse','decrease','Berberine','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5458,'mouse','decrease','Berberine','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5459,'mouse','decrease','Berberine','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5460,'mouse','decrease','Berberine','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5461,'mouse','decrease','Berberine','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5462,'mouse','increase','Berberine','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5463,'mouse','increase','Berberine','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5464,'mouse','increase','Berberine','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5465,'mouse','increase','Berberine','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5466,'mouse','increase','Berberine','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5467,'mouse','decrease','Berberine','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5468,'mouse','decrease','Berberine','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5469,'mouse','decrease','Berberine','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5470,'mouse','decrease','Berberine','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5471,'mouse','decrease','Berberine','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5472,'mouse','increase','Berberine','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5473,'mouse','increase','Berberine','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5474,'mouse','increase','Berberine','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5475,'mouse','increase','Berberine','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5476,'mouse','increase','Berberine','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5477,'mouse','decrease','Berberine','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5478,'mouse','decrease','Berberine','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5479,'mouse','decrease','Berberine','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5480,'mouse','decrease','Berberine','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5481,'mouse','decrease','Berberine','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5482,'mouse','increase','Berberine','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5483,'mouse','increase','Berberine','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5484,'mouse','increase','Berberine','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5485,'mouse','increase','Berberine','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5486,'mouse','increase','Berberine','low-throughput','By contrast, OBB and positive drugs promoted the expression of the above five TJs proteins in a dose-dependent manner.',31863867),(5487,'mouse','decrease','Berberine','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5488,'mouse','decrease','Berberine','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5489,'mouse','decrease','Berberine','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5490,'mouse','decrease','Berberine','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5491,'mouse','decrease','Berberine','low-throughput','However, these elevated pro-inflammatory cytokines were dramatically decreased by OBB.',31863867),(5492,'mouse','decrease','null','low-throughput','In line with these clinical findings, VSL#3 and CLA significantly ameliorated gross pathology in colon, MLN and spleen, colonic histopathology such as leukocyte infiltration  and mucosal thickening , and colonic mRNA expression such as TNF-a and MCP-1 in comparison to untreated control mice with DSS colitis.',22363592),(5493,'mouse','decrease','null','low-throughput','In line with these clinical findings, VSL#3 and CLA significantly ameliorated gross pathology in colon, MLN and spleen, colonic histopathology such as leukocyte infiltration  and mucosal thickening , and colonic mRNA expression such as TNF-a and MCP-1 in comparison to untreated control mice with DSS colitis.',22363592),(5498,'mouse','increase','null','low-throughput','Here, we tested the hypothesis that the gut microbiota could induce maturation of the adult ENS through release of 5-HT and activation of 5-HT4 receptors.',29866843),(5568,'mouse','null','Daidzein','null','We isolated, for the first time, an anaerobic gram-positive rod-shaped strain capable of producing equol from daidzein.',17046543),(5569,'mouse','null','Genistein','null','We isolated a strictly anaerobic bacterium (Mt1B8) from the mouse intestine which converted daidzein via dihydrodaidzein to equol as well as genistein via dihydrogenistein to 5-hydroxy-equol.',18539813),(5570,'mouse','null','Daidzein','null','We isolated a strictly anaerobic bacterium (Mt1B8) from the mouse intestine which converted daidzein via dihydrodaidzein to equol as well as genistein via dihydrogenistein to 5-hydroxy-equol.',18539813),(5571,'mouse','null','Dihydrodaidzein','null','Strain Mt1B8 also transformed dihydrodaidzein and dihydrogenistein to equol and 5-hydroxy-equol, respectively.',18539813),(5572,'mouse','null','Dihydrogenistein','null','Strain Mt1B8 also transformed dihydrodaidzein and dihydrogenistein to equol and 5-hydroxy-equol, respectively.',18539813),(5573,'mouse','increase','3-Dehydroshikimate','low-throughput','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.',32728212),(5574,'mouse','increase','3-Dehydroshikimate','low-throughput','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.',32728212),(5575,'mouse','increase','3-Dehydroshikimate','low-throughput','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.',32728212),(5576,'mouse','increase','3-Dehydroshikimate','low-throughput','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.',32728212),(5577,'mouse','increase','3-Dehydroshikimate','low-throughput','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.',32728212),(5578,'mouse','increase','3-Dehydroshikimate','low-throughput','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.',32728212),(5579,'mouse','increase','3-Dehydroshikimate','low-throughput','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.',32728212),(5580,'mouse','increase','3-Dehydroshikimate','low-throughput','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.',32728212),(5581,'mouse','increase','3-Dehydroshikimate','low-throughput','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.',32728212),(5582,'mouse','increase','3-Dehydroshikimate','low-throughput','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.',32728212),(5583,'mouse','increase','3-Dehydroshikimate','low-throughput','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.',32728212),(5584,'mouse','increase','3-Dehydroshikimate','low-throughput','Mutant p53 induces WNT suppression and differentiation of intestinal tumour organoids, an effect which is reversibly blocked by gallic acid treatment.',32728212),(5585,'mouse','increase','Phytic acid','low-throughput','Taken together, these data demonstrate bacterial-intrinsic phytate metabolism and provoke the hypothesis that breakdown of phytate by microbiota promotes intestinal HDAC activity.',32731255),(5586,'mouse','decrease','null','low-throughput','Further, colonization of germ-free (GF) mice with butyrate producing bacteria,Faecalibacterium prausnitzii, inhibited IEC-intrinsic HDAC activity.',32731255),(5587,'mouse','increase','null','low-throughput','CNV mice exhibited significantly higher IP3 in both the intestinal lumen (Fig. 1e) and colonic IECs (Fig. 1f), supporting that microbiota promote increased IP3. To assess the effect of IP3 on epithelial HDAC3 activity, HDAC3 was purified by immunoprecipitation from IECs and incubated with IP3. IP3 increased endogenous HDAC3 activity (Fig. 1g), supporting that IP3 directly activates HDAC3 in IECs.',32731255),(5588,'mouse','increase','null','low-throughput','Although less potent than IP3, IP4 also increased HDAC activity in IECs (Extended Data Fig. 5f), suggesting that other inositol phosphates simultaneously generated by phytate metabolism may provide synergistic effects.',32731255),(5589,'mouse','increase','Glutamic acid','low-throughput','Glutamate, the agonist of NMDAR and a major excitatory neurotransmitter in the mammalian central nervous system (CNS) [6], is intracellularly located and extensively distributed in the brain.',32290475),(5590,'mouse','increase','Glutamic acid','low-throughput','Glutamate, the agonist of NMDAR and a major excitatory neurotransmitter in the mammalian central nervous system (CNS) [6], is intracellularly located and extensively distributed in the brain.',32290475),(5591,'mouse','increase','Glutamic acid','low-throughput','Non-pathogenic bacteria with glutamate racemase including Corynebacterium glutamicum, Brevibacterium lactofermentum, Brevibacterium avium [121], Mycobacterium smegmatis[122], and Bacillus subtilis[123] convert l-glutamate to d-glutamate.',32290475),(5592,'mouse','increase','Glutamic acid','low-throughput','Non-pathogenic bacteria with glutamate racemase including Corynebacterium glutamicum, Brevibacterium lactofermentum, Brevibacterium avium [121], Mycobacterium smegmatis[122], and Bacillus subtilis[124] convert l-glutamate to d-glutamate.',32290475),(5593,'mouse','increase','null','low-throughput','Coryneform bacteria are mainly used to produce glutamate in industry. LAB strains such as Lactobacillus plantarum, Lactococcus lactis, and Lactobacillus paracasei can synthesize glutamate [113,114]. ',32290475),(5594,'mouse','increase','null','low-throughput','Coryneform bacteria are mainly used to produce glutamate in industry. LAB strains such as Lactobacillus plantarum, Lactococcus lactis, and Lactobacillus paracasei can synthesize glutamate [113,114]. ',32290475),(5595,'mouse','increase','null','low-throughput','Coryneform bacteria are mainly used to produce glutamate in industry. LAB strains such as Lactobacillus plantarum, Lactococcus lactis, and Lactobacillus paracasei can synthesize glutamate [113,114]. ',32290475),(5596,'mouse','increase','null','low-throughput','Coryneform bacteria are mainly used to produce glutamate in industry. LAB strains such as Lactobacillus plantarum, Lactococcus lactis, and Lactobacillus paracasei can synthesize glutamate [113,114]. ',32290475),(5597,'mouse','increase','null','low-throughput','Furthermore, benzoate, an inhibitor of d-amino acid oxidases (DAAO), can increase d-serine,thereby enhancing NMDAR activation [125,126].',32290475),(5598,'mouse','increase','Bile acid','low-throughput','Reconstitution with a single bacterial species, Clostridium scindens, or its derived metabolite, the secondary bile acid deoxycholic acid, can restore pDC- and MyD88-dependent type I IFN responses to restrict systemic CHIKV infection and transmission back to vector mosquitoes. ',32668198),(5599,'mouse','increase','Bile acid','low-throughput','Reconstitution with a single bacterial species, Clostridium scindens, or its derived metabolite, the secondary bile acid deoxycholic acid, can restore pDC- and MyD88-dependent type I IFN responses to restrict systemic CHIKV infection and transmission back to vector mosquitoes. ',32668198),(5600,'mouse','null','Tryptophan','null','HSD significantly reduces intestinal Lactobacillus murinus in hypertensive FVB/N mice, while L murinus can metabolize trypto phan in the diet to produce indole, thereby inhibiting T lymphocyte differentiation into Th17 cells, alleviating inflammation and lowering BP.',32078445),(5601,'mouse','decrease','null','low-throughput','Moreover, the culture supernatant of B fra gilis YCH46 could significantly reduce the corticoste rone level and reverse the abnormal expressions of Mr, Cyp11a1, Cyp11b1, Hsd11b1, and Hsd11b2 in high salt–fed mice colon tissues (Figure 5E through 5J). Namely, B fragilis effectively counteracted HSD effects on intesti nal corticosterone metabolism and dramatically reduced intestinal corticosterone production level.',32078445),(5602,'mouse','decrease','null','low-throughput','Moreover, the culture supernatant of B fra gilis YCH46 could significantly reduce the corticoste rone level and reverse the abnormal expressions of Mr, Cyp11a1, Cyp11b1, Hsd11b1, and Hsd11b2 in high salt–fed mice colon tissues (Figure 5E through 5J). Namely, B fragilis effectively counteracted HSD effects on intesti nal corticosterone metabolism and dramatically reduced intestinal corticosterone production level.',32078445),(5603,'mouse','decrease','null','low-throughput','Moreover, the culture supernatant of B fra gilis YCH46 could significantly reduce the corticoste rone level and reverse the abnormal expressions of Mr, Cyp11a1, Cyp11b1, Hsd11b1, and Hsd11b2 in high salt–fed mice colon tissues (Figure 5E through 5J). Namely, B fragilis effectively counteracted HSD effects on intesti nal corticosterone metabolism and dramatically reduced intestinal corticosterone production level.',32078445),(5604,'mouse','increase','null','low-throughput','Moreover, the culture supernatant of B fra gilis YCH46 could significantly reduce the corticoste rone level and reverse the abnormal expressions of Mr, Cyp11a1, Cyp11b1, Hsd11b1, and Hsd11b2 in high salt–fed mice colon tissues (Figure 5E through 5J). Namely, B fragilis effectively counteracted HSD effects on intesti nal corticosterone metabolism and dramatically reduced intestinal corticosterone production level.',32078445),(5605,'mouse','increase','null','low-throughput','Here, we elucidate a micro biome-dependent pathway by which SG increases CA7S production. We show that a microbial metabolite,lithocholic acid (LCA), is increased in murine portal veins post-SG and by activating the vitamin D receptor, induces hepatic mSult2A1/hSULT2A expression to drive CA7S production. An SG-induced shift in the micro biome increases gut expression of the bile acid transporters Asbt and Osta, which in turn facilitate selective transport of LCA across the gut epithelium. Cecal microbiota transplant from SG animals is sufficient to recreate the pathway in germ-free (GF) animals. \nand GLP-1 secretion, causally connecting a microbial metabolite with the improvement of diabetic phe notypes.',33434516),(5606,'mouse','increase','null','low-throughput','Here, we elucidate a micro biome-dependent pathway by which SG increases CA7S production. We show that a microbial metabolite,lithocholic acid (LCA), is increased in murine portal veins post-SG and by activating the vitamin D receptor, induces hepatic mSult2A1/hSULT2A expression to drive CA7S production. An SG-induced shift in the micro biome increases gut expression of the bile acid transporters Asbt and Osta, which in turn facilitate selective transport of LCA across the gut epithelium. Cecal microbiota transplant from SG animals is sufficient to recreate the pathway in germ-free (GF) animals. \nand GLP-1 secretion, causally connecting a microbial metabolite with the improvement of diabetic phe notypes.',33434516),(5607,'mouse','increase','null','low-throughput','We determined that CA7S is a potent agonist of the G-protein-coupled recep tor TGR5 (GPBAR1) and induces GLP-1 secretion. When administered to diet-induced obese (DIO) mice, CA7S in creases insulin secretion and induces systemic glucose clearance.',33434516),(5608,'mouse','increase','null','low-throughput','We determined that CA7S is a potent agonist of the G-protein-coupled recep tor TGR5 (GPBAR1) and induces GLP-1 secretion. When administered to diet-induced obese (DIO) mice, CA7S in creases insulin secretion and induces systemic glucose clearance.',33434516),(5609,'mouse','decrease','null','low-throughput','Consistent with our in vitro results showing that LCA inhibits expression of Asbt and Osta, SG-CMT mice exhibited significantly higher levels of Asbt and Osta expression in the distal ileum compared with sham-CMT (Figure 7E).',33434516),(5610,'mouse','decrease','null','low-throughput','Consistent with our in vitro results showing that LCA inhibits expression of Asbt and Osta, SG-CMT mice exhibited significantly higher levels of Asbt and Osta expression in the distal ileum compared with sham-CMT (Figure 7E).',33434516),(5611,'mouse','increase','Trimethylamine','low-throughput','Concentrations of pro-inflammatory cytokines com monly associated with neuroinflammation (IL1-β, IL 6, and TNF-α) and the anti-inflammatory cytokine IL-10 were measured in whole-brain lysates from young and old mice and are presented in Fig. 3a. Whole-brain concentrations of pro-inflammatory cytokines were all ~ 50�?00% higher in old vs. young mice (all p < 0.05).Old mice also tended to have (presumably compensato ry) higher brain IL-10 compared to young mice (p =0.07).',32862276),(5612,'mouse','increase','Trimethylamine','low-throughput','Concentrations of pro-inflammatory cytokines com monly associated with neuroinflammation (IL1-β, IL 6, and TNF-α) and the anti-inflammatory cytokine IL-10 were measured in whole-brain lysates from young and old mice and are presented in Fig. 3a. Whole-brain concentrations of pro-inflammatory cytokines were all ~ 50�?00% higher in old vs. young mice (all p < 0.05).Old mice also tended to have (presumably compensato ry) higher brain IL-10 compared to young mice (p =0.07).',32862276),(5613,'mouse','increase','Trimethylamine','low-throughput','Concentrations of pro-inflammatory cytokines com monly associated with neuroinflammation (IL1-β, IL 6, and TNF-α) and the anti-inflammatory cytokine IL-10 were measured in whole-brain lysates from young and old mice and are presented in Fig. 3a. Whole-brain concentrations of pro-inflammatory cytokines were all ~ 50�?00% higher in old vs. young mice (all p < 0.05).Old mice also tended to have (presumably compensato ry) higher brain IL-10 compared to young mice (p =0.07).',32862276),(5614,'mouse','increase','Trimethylamine','low-throughput','Concentrations of pro-inflammatory cytokines com monly associated with neuroinflammation (IL1-β, IL 6, and TNF-α) and the anti-inflammatory cytokine IL-10 were measured in whole-brain lysates from young and old mice and are presented in Fig. 3a. Whole-brain concentrations of pro-inflammatory cytokines were all ~ 50�?00% higher in old vs. young mice (all p < 0.05).Old mice also tended to have (presumably compensato ry) higher brain IL-10 compared to young mice (p =0.07).',32862276),(5615,'mouse','increase','Trimethylamine','low-throughput','Next, we measured the pro-inflammatory tran scription factor nuclear factor kappa B (NF-κB), which is a key regulator of pro-inflammatory cytokine signal ing. Although abundance of phosphorylated (i.e., acti vated) NF-κB was similar between brains from young and old mice (p = 0.49), old mice had ~ 50% higher abundance of total NF-κB (p < 0.001; Fig. 3b), suggest ing a greater capacity to upregulate pro-inflammatory cytokine production.',32862276),(5616,'mouse','increase','Trimethylamine','low-throughput','Finally, LCN2, an established marker of reactive astrocytes involved in the innate immune response to bacteria [5, 63], was higher in brain lysates from old compared to young mice (p < 0.05;Fig. 3c).',32862276),(5617,'mouse','increase','null','low-throughput','In this study, we observed that in mice exposed to HFD or without intestinal epithelial cell AhR expression, the efect of diet was more signifcant than the absence of AhR on the fecal microbiome composition. On the other hand, the absence of AhR had a more pronounced efect on the fecal metabolome than diet. Tryptophan metabolites were among those that were signifcantly altered in the metabolome, and integrated microbiome and metabolome analysis predicted three taxa—unclas sifed Clostridiales, unclassifed Desulfovibrionaceae, and Akkermansia—as key contributors to the synthesis and/or degradation of tryptophan metabolites. Our study highlights the use of multi-omic analysis to investigate the relationship between microbiome and metabolome and suggests possible taxa that can be targeted to manip ulate the microbiome for CRC prevention.',33256731),(5618,'mouse','increase','null','low-throughput','In this study, we observed that in mice exposed to HFD or without intestinal epithelial cell AhR expression, the efect of diet was more signifcant than the absence of AhR on the fecal microbiome composition. On the other hand, the absence of AhR had a more pronounced efect on the fecal metabolome than diet. Tryptophan metabolites were among those that were signifcantly altered in the metabolome, and integrated microbiome and metabolome analysis predicted three taxa—unclas sifed Clostridiales, unclassifed Desulfovibrionaceae, and Akkermansia—as key contributors to the synthesis and/or degradation of tryptophan metabolites. Our study highlights the use of multi-omic analysis to investigate the relationship between microbiome and metabolome and suggests possible taxa that can be targeted to manip ulate the microbiome for CRC prevention.',33256731),(5619,'mouse','increase','null','low-throughput','In this study, we observed that in mice exposed to HFD or without intestinal epithelial cell AhR expression, the efect of diet was more signifcant than the absence of AhR on the fecal microbiome composition. On the other hand, the absence of AhR had a more pronounced efect on the fecal metabolome than diet. Tryptophan metabolites were among those that were signifcantly altered in the metabolome, and integrated microbiome and metabolome analysis predicted three taxa—unclas sifed Clostridiales, unclassifed Desulfovibrionaceae, and Akkermansia—as key contributors to the synthesis and/or degradation of tryptophan metabolites. Our study highlights the use of multi-omic analysis to investigate the relationship between microbiome and metabolome and suggests possible taxa that can be targeted to manip ulate the microbiome for CRC prevention.',33256731),(5620,'mouse','increase','null','low-throughput','mRNA levels of TNF-α and IL-1β in the WAT of ob/ob mice were, respectively, 17- and 4-fold higher than those of C57BL/6 J mice (Fig. 2A). Te iNOS and COX-2 in WAT were also, respectively, 5.4- and 2.2-fold higher than those of C57BL/6 J mice (Fig. 2A). Hepatic gene expression of IL-6 was signifcantly increased in ob/ob mice compared with C57BL/6 J mice although TNF-α and IL-1β were not markedly increased (Fig. 2B).',32296105),(5621,'mouse','increase','null','low-throughput','mRNA levels of TNF-α and IL-1β in the WAT of ob/ob mice were, respectively, 17- and 4-fold higher than those of C57BL/6 J mice (Fig. 2A). Te iNOS and COX-2 in WAT were also, respectively, 5.4- and 2.2-fold higher than those of C57BL/6 J mice (Fig. 2A). Hepatic gene expression of IL-6 was signifcantly increased in ob/ob mice compared with C57BL/6 J mice although TNF-α and IL-1β were not markedly increased (Fig. 2B).',32296105),(5622,'mouse','increase','null','low-throughput','mRNA levels of TNF-α and IL-1β in the WAT of ob/ob mice were, respectively, 17- and 4-fold higher than those of C57BL/6 J mice (Fig. 2A). Te iNOS and COX-2 in WAT were also, respectively, 5.4- and 2.2-fold higher than those of C57BL/6 J mice (Fig. 2A). Hepatic gene expression of IL-6 was signifcantly increased in ob/ob mice compared with C57BL/6 J mice although TNF-α and IL-1β were not markedly increased (Fig. 2B).',32296105),(5623,'mouse','increase','null','low-throughput','mRNA levels of TNF-α and IL-1β in the WAT of ob/ob mice were, respectively, 17- and 4-fold higher than those of C57BL/6 J mice (Fig. 2A). Te iNOS and COX-2 in WAT were also, respectively, 5.4- and 2.2-fold higher than those of C57BL/6 J mice (Fig. 2A). Hepatic gene expression of IL-6 was signifcantly increased in ob/ob mice compared with C57BL/6 J mice although TNF-α and IL-1β were not markedly increased (Fig. 2B).',32296105),(5624,'mouse','increase','null','low-throughput','mRNA levels of TNF-α and IL-1β in the WAT of ob/ob mice were, respectively, 17- and 4-fold higher than those of C57BL/6 J mice (Fig. 2A). Te iNOS and COX-2 in WAT were also, respectively, 5.4- and 2.2-fold higher than those of C57BL/6 J mice (Fig. 2A). Hepatic gene expression of IL-6 was signifcantly increased in ob/ob mice compared with C57BL/6 J mice although TNF-α and IL-1β were not markedly increased (Fig. 2B).',32296105),(5625,'mouse','increase','null','low-throughput','As a result, stercobilin signifcantly induced NF-κB reporter gene activities in a dose-dependent manner (Fig. 6A). Additionally, ster cobilin signifcantly induced expression of several infammation-related genes, including TNF-α, IL-1β, IL-6, iNOS, and COX-2 (Fig. 6B).',32296105),(5626,'mouse','decrease','Baicalin','low-throughput','Compared to the wild type mice, the levels of infammatory cytokines such as IL-1β, IL-2, IL-6, IL-10, G-CSF, and GM-CSF are signifcantly increased in the intestine and colon of APCmin/+ mice in the model group. ',31650468),(5627,'mouse','decrease','Baicalin','low-throughput','Compared to the wild type mice, the levels of infammatory cytokines such as IL-1β, IL-2, IL-6, IL-10, G-CSF, and GM-CSF are signifcantly increased in the intestine and colon of APCmin/+ mice in the model group. ',31650468),(5628,'mouse','decrease','Baicalin','low-throughput','Compared to the wild type mice, the levels of infammatory cytokines such as IL-1β, IL-2, IL-6, IL-10, G-CSF, and GM-CSF are signifcantly increased in the intestine and colon of APCmin/+ mice in the model group. ',31650468),(5629,'mouse','decrease','Baicalin','low-throughput','Compared to the wild type mice, the levels of infammatory cytokines such as IL-1β, IL-2, IL-6, IL-10, G-CSF, and GM-CSF are signifcantly increased in the intestine and colon of APCmin/+ mice in the model group. ',31650468),(5630,'mouse','decrease','Baicalin','low-throughput','Compared to the wild type mice, the levels of infammatory cytokines such as IL-1β, IL-2, IL-6, IL-10, G-CSF, and GM-CSF are signifcantly increased in the intestine and colon of APCmin/+ mice in the model group. ',31650468),(5631,'mouse','decrease','Baicalin','low-throughput','Compared to the wild type mice, the levels of infammatory cytokines such as IL-1β, IL-2, IL-6, IL-10, G-CSF, and GM-CSF are signifcantly increased in the intestine and colon of APCmin/+ mice in the model group. ',31650468),(5632,'mouse','increase','Tryptophan','low-throughput','Interestingly, in the same mice, indole induced a dramatic increase of the expression of the adrenomedullary Pnmt gene, which is involved in catecholamine biosynthesis.',32569990),(5633,'mouse','decrease','null','low-throughput','To illuminate the molecular mechanism underpinning the hyperlipidemia-ameliorating role of DNJ, we further examined the impact of DNJ on hepatic genes that are involved in lipid synthesis (SREBP1c, FAS, ACC, GPAT, SCD, ChREBP, SREBP2, and HMGR), transport (FATP, FABP, LXR, and CD36) and oxidation (CPT1, PPAR , AMPK, and ACO). Compared with the normal control mice, mice that received gavage of DNJ alone showed decreased transcription of lipogenic genes (Fig. 2 and Table 1), with no significant effects on genes participating in lipid transport (except CD36) and oxidation (see Fig. S2 in the supple mental material, suggesting that DNJ may prevent hyperlipidemia via inhibiting lipid biosynthesis.',33024047),(5634,'mouse','decrease','null','low-throughput','To illuminate the molecular mechanism underpinning the hyperlipidemia-ameliorating role of DNJ, we further examined the impact of DNJ on hepatic genes that are involved in lipid synthesis (SREBP1c, FAS, ACC, GPAT, SCD, ChREBP, SREBP2, and HMGR), transport (FATP, FABP, LXR, and CD36) and oxidation (CPT1, PPAR , AMPK, and ACO). Compared with the normal control mice, mice that received gavage of DNJ alone showed decreased transcription of lipogenic genes (Fig. 2 and Table 1), with no significant effects on genes participating in lipid transport (except CD36) and oxidation (see Fig. S2 in the supple mental material, suggesting that DNJ may prevent hyperlipidemia via inhibiting lipid biosynthesis.',33024047),(5635,'mouse','decrease','null','low-throughput','To illuminate the molecular mechanism underpinning the hyperlipidemia-ameliorating role of DNJ, we further examined the impact of DNJ on hepatic genes that are involved in lipid synthesis (SREBP1c, FAS, ACC, GPAT, SCD, ChREBP, SREBP2, and HMGR), transport (FATP, FABP, LXR, and CD36) and oxidation (CPT1, PPAR , AMPK, and ACO). Compared with the normal control mice, mice that received gavage of DNJ alone showed decreased transcription of lipogenic genes (Fig. 2 and Table 1), with no significant effects on genes participating in lipid transport (except CD36) and oxidation (see Fig. S2 in the supple mental material, suggesting that DNJ may prevent hyperlipidemia via inhibiting lipid biosynthesis.',33024047),(5636,'mouse','decrease','null','low-throughput','To illuminate the molecular mechanism underpinning the hyperlipidemia-ameliorating role of DNJ, we further examined the impact of DNJ on hepatic genes that are involved in lipid synthesis (SREBP1c, FAS, ACC, GPAT, SCD, ChREBP, SREBP2, and HMGR), transport (FATP, FABP, LXR, and CD36) and oxidation (CPT1, PPAR , AMPK, and ACO). Compared with the normal control mice, mice that received gavage of DNJ alone showed decreased transcription of lipogenic genes (Fig. 2 and Table 1), with no significant effects on genes participating in lipid transport (except CD36) and oxidation (see Fig. S2 in the supple mental material, suggesting that DNJ may prevent hyperlipidemia via inhibiting lipid biosynthesis.',33024047),(5637,'mouse','decrease','null','low-throughput','To illuminate the molecular mechanism underpinning the hyperlipidemia-ameliorating role of DNJ, we further examined the impact of DNJ on hepatic genes that are involved in lipid synthesis (SREBP1c, FAS, ACC, GPAT, SCD, ChREBP, SREBP2, and HMGR), transport (FATP, FABP, LXR, and CD36) and oxidation (CPT1, PPAR , AMPK, and ACO). Compared with the normal control mice, mice that received gavage of DNJ alone showed decreased transcription of lipogenic genes (Fig. 2 and Table 1), with no significant effects on genes participating in lipid transport (except CD36) and oxidation (see Fig. S2 in the supple mental material, suggesting that DNJ may prevent hyperlipidemia via inhibiting lipid biosynthesis.',33024047),(5638,'mouse','decrease','null','low-throughput','To illuminate the molecular mechanism underpinning the hyperlipidemia-ameliorating role of DNJ, we further examined the impact of DNJ on hepatic genes that are involved in lipid synthesis (SREBP1c, FAS, ACC, GPAT, SCD, ChREBP, SREBP2, and HMGR), transport (FATP, FABP, LXR, and CD36) and oxidation (CPT1, PPAR , AMPK, and ACO). Compared with the normal control mice, mice that received gavage of DNJ alone showed decreased transcription of lipogenic genes (Fig. 2 and Table 1), with no significant effects on genes participating in lipid transport (except CD36) and oxidation (see Fig. S2 in the supple mental material, suggesting that DNJ may prevent hyperlipidemia via inhibiting lipid biosynthesis.',33024047),(5639,'mouse','decrease','null','low-throughput','To illuminate the molecular mechanism underpinning the hyperlipidemia-ameliorating role of DNJ, we further examined the impact of DNJ on hepatic genes that are involved in lipid synthesis (SREBP1c, FAS, ACC, GPAT, SCD, ChREBP, SREBP2, and HMGR), transport (FATP, FABP, LXR, and CD36) and oxidation (CPT1, PPAR , AMPK, and ACO). Compared with the normal control mice, mice that received gavage of DNJ alone showed decreased transcription of lipogenic genes (Fig. 2 and Table 1), with no significant effects on genes participating in lipid transport (except CD36) and oxidation (see Fig. S2 in the supple mental material, suggesting that DNJ may prevent hyperlipidemia via inhibiting lipid biosynthesis.',33024047),(5640,'mouse','decrease','null','low-throughput','To illuminate the molecular mechanism underpinning the hyperlipidemia-ameliorating role of DNJ, we further examined the impact of DNJ on hepatic genes that are involved in lipid synthesis (SREBP1c, FAS, ACC, GPAT, SCD, ChREBP, SREBP2, and HMGR), transport (FATP, FABP, LXR, and CD36) and oxidation (CPT1, PPAR , AMPK, and ACO). Compared with the normal control mice, mice that received gavage of DNJ alone showed decreased transcription of lipogenic genes (Fig. 2 and Table 1), with no significant effects on genes participating in lipid transport (except CD36) and oxidation (see Fig. S2 in the supple mental material, suggesting that DNJ may prevent hyperlipidemia via inhibiting lipid biosynthesis.',33024047),(5641,'mouse','decrease','null','low-throughput','Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).',33024047),(5642,'mouse','decrease','null','low-throughput','Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).',33024047),(5643,'mouse','decrease','null','low-throughput','Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).',33024047),(5644,'mouse','decrease','null','low-throughput','Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).',33024047),(5645,'mouse','decrease','null','low-throughput','Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).',33024047),(5646,'mouse','decrease','null','low-throughput','Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).',33024047),(5647,'mouse','decrease','null','low-throughput','Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).',33024047),(5648,'mouse','decrease','null','low-throughput','Furthermore, IPA dose dependently decreased the tran scription of the key genes involved in fatty acid (SREBP1c and FAS) and cholesterol biosynthesis (SREBP2 and HMGR) (Fig. 5D).',33024047),(5649,'mouse','null','Stachyose','null','In the low-dose and high dose groups, elevated concentrations of acetic acid and propionic acid were produced, indicating that stachyose intake promoted the proliferation of acetic acid- and propionic acid-producing microorganisms.',33233042),(5650,'mouse','null','Stachyose','null','In the low-dose and high dose groups, elevated concentrations of acetic acid and propionic acid were produced, indicating that stachyose intake promoted the proliferation of acetic acid- and propionic acid-producing microorganisms.',33233042),(5651,'mouse','null','Stachyose','null','In the low-dose and high dose groups, elevated concentrations of acetic acid and propionic acid were produced, indicating that stachyose intake promoted the proliferation of acetic acid- and propionic acid-producing microorganisms.',33233042),(5652,'mouse','null','Stachyose','null','In the low-dose and high dose groups, elevated concentrations of acetic acid and propionic acid were produced, indicating that stachyose intake promoted the proliferation of acetic acid- and propionic acid-producing microorganisms.',33233042),(5653,'mouse','decrease','null','low-throughput','The expression of IL-1, IL-17 and TNF-α were higher in the intestinal tract of the germ free mice that had not received a human flora fecal transplant than in those that had received a fecal transplant, and the expression of IL-1, IL 17 and TNF-α decreased significantly after the fecal transplant (Fig. 4c, p < 0.01).',33233042),(5654,'mouse','decrease','null','low-throughput','The expression of IL-1, IL-17 and TNF-α were higher in the intestinal tract of the germ free mice that had not received a human flora fecal transplant than in those that had received a fecal transplant, and the expression of IL-1, IL 17 and TNF-α decreased significantly after the fecal transplant (Fig. 4c, p < 0.01).',33233042),(5655,'mouse','decrease','null','low-throughput','The expression of IL-1, IL-17 and TNF-α were higher in the intestinal tract of the germ free mice that had not received a human flora fecal transplant than in those that had received a fecal transplant, and the expression of IL-1, IL 17 and TNF-α decreased significantly after the fecal transplant (Fig. 4c, p < 0.01).',33233042),(5656,'mouse','decrease','null','low-throughput','Compared with the germ free group, the expression of NF-κB decreased significantly in the colon after transplantation of intestinal flora (p < 0.05), and the intake of stachyose downregulated the expression of NF-κB (p < 0.05).',33233042),(5657,'mouse','increase','null','low-throughput','As shown in Fig. 4h and i, the expression of occludin and ZO-1 in the colon tissue of germ-free mice significantly increased after human fecal transplantation (p < 0.05).',33233042),(5658,'mouse','increase','null','low-throughput','As shown in Fig. 4h and i, the expression of occludin and ZO-1 in the colon tissue of germ-free mice significantly increased after human fecal transplantation (p < 0.05).',33233042);
/*!40000 ALTER TABLE `metabolite_gene_change_results` ENABLE KEYS */;
UNLOCK TABLES;
/*!40103 SET TIME_ZONE=@OLD_TIME_ZONE */;

/*!40101 SET SQL_MODE=@OLD_SQL_MODE */;
/*!40014 SET FOREIGN_KEY_CHECKS=@OLD_FOREIGN_KEY_CHECKS */;
/*!40014 SET UNIQUE_CHECKS=@OLD_UNIQUE_CHECKS */;
/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
/*!40111 SET SQL_NOTES=@OLD_SQL_NOTES */;

-- Dump completed on 2023-01-12 14:42:01
